{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "34930806",
   "metadata": {},
   "outputs": [
    {
     "ename": "ImportError",
     "evalue": "cannot import name 'add_enzyme_constraints' from 'mewpy' (/home/mmartins/anaconda3/envs/mcd22/lib/python3.9/site-packages/mewpy/__init__.py)",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mImportError\u001b[0m                               Traceback (most recent call last)",
      "Cell \u001b[0;32mIn[4], line 2\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[38;5;28;01mfrom\u001b[39;00m \u001b[38;5;21;01mcobra\u001b[39;00m\u001b[38;5;21;01m.\u001b[39;00m\u001b[38;5;21;01mio\u001b[39;00m \u001b[38;5;28;01mimport\u001b[39;00m read_sbml_model, write_sbml_model\n\u001b[0;32m----> 2\u001b[0m \u001b[38;5;28;01mfrom\u001b[39;00m \u001b[38;5;21;01mmewpy\u001b[39;00m \u001b[38;5;28;01mimport\u001b[39;00m add_enzyme_constraints\n\u001b[1;32m      3\u001b[0m \u001b[38;5;28;01mfrom\u001b[39;00m \u001b[38;5;21;01mmewpy\u001b[39;00m\u001b[38;5;21;01m.\u001b[39;00m\u001b[38;5;21;01msimulation\u001b[39;00m \u001b[38;5;28;01mimport\u001b[39;00m get_simulator\n\u001b[1;32m      4\u001b[0m \u001b[38;5;28;01mfrom\u001b[39;00m \u001b[38;5;21;01mmewpy\u001b[39;00m\u001b[38;5;21;01m.\u001b[39;00m\u001b[38;5;21;01msimulation\u001b[39;00m\u001b[38;5;21;01m.\u001b[39;00m\u001b[38;5;21;01menvironment\u001b[39;00m \u001b[38;5;28;01mimport\u001b[39;00m Environment \u001b[38;5;28;01mas\u001b[39;00m Environment\n",
      "\u001b[0;31mImportError\u001b[0m: cannot import name 'add_enzyme_constraints' from 'mewpy' (/home/mmartins/anaconda3/envs/mcd22/lib/python3.9/site-packages/mewpy/__init__.py)"
     ]
    }
   ],
   "source": [
    "from cobra.io import read_sbml_model, write_sbml_model\n",
    "from mewpy import add_enzyme_constraints\n",
    "from mewpy.simulation import get_simulator\n",
    "from mewpy.simulation.environment import Environment as Environment\n",
    "from mewpy.util.request import retreive_gene\n",
    "import json"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "90e537c9",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "No objective coefficients in model. Unclear what should be optimized\n"
     ]
    },
    {
     "ename": "NameError",
     "evalue": "name 'get_simulator' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "Cell \u001b[0;32mIn[2], line 2\u001b[0m\n\u001b[1;32m      1\u001b[0m model \u001b[38;5;241m=\u001b[39m read_sbml_model(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mmodels/iAF1260.xml\u001b[39m\u001b[38;5;124m\"\u001b[39m)\n\u001b[0;32m----> 2\u001b[0m sim \u001b[38;5;241m=\u001b[39m \u001b[43mget_simulator\u001b[49m(model)\n\u001b[1;32m      3\u001b[0m sim\u001b[38;5;241m.\u001b[39mset_objective(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mBIOMASS\u001b[39m\u001b[38;5;124m\"\u001b[39m)\n",
      "\u001b[0;31mNameError\u001b[0m: name 'get_simulator' is not defined"
     ]
    }
   ],
   "source": [
    "model = read_sbml_model(\"models/iAF1260.xml\")\n",
    "sim = get_simulator(model)\n",
    "sim.set_objective(\"BIOMASS\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "36147fb7",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "objective: 28.215758805752888\n",
       "Status: OPTIMAL\n",
       "Constraints: OrderedDict()\n",
       "Method:SimulationMethod.FBA"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sim.simulate()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "f24d0a33",
   "metadata": {},
   "source": [
    "### Retreive MW infos\n",
    "\n",
    "The protein IDs and MW (Da) are obtained from UniProt.\n",
    "\n",
    "Note: does not retreive Kcats from BRENDA. All catalytic rates are equal to 100000/s "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "04ee16a4",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "No comments\n",
      "No comments\n"
     ]
    }
   ],
   "source": [
    "data_all = dict()\n",
    "try:\n",
    "    with open(sim.id+\".json\", 'r') as f:\n",
    "        data = json.load(f)\n",
    "except:\n",
    "    data = dict()\n",
    "    for gene in sim.genes:\n",
    "        try:\n",
    "            info = retreive_gene(gene,\"Escherichia coli\")\n",
    "            # set the inverse kcat to a low value\n",
    "            data[gene]={'protein':info['protein'],'mw':info['mw']*0.001,'kcat':1e-5}\n",
    "            data_all[gene]=info\n",
    "        except:\n",
    "            data[gene]={'protein': gene[len(sim._g_prefix):], 'mw': 0, 'kcat': 0}\n",
    "    with open(sim.id+\".json\", \"w\") as f:\n",
    "        json.dump(data,f)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "90dcf9d9",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Genes with missing data:\n",
      "b1692\n"
     ]
    }
   ],
   "source": [
    "print(\"Genes with missing data:\")\n",
    "for k,v in data.items():\n",
    "    if v['mw']==0:\n",
    "        print(k)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "c0e69ecd",
   "metadata": {},
   "source": [
    "### Build the ec model"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "dd58c530",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Converting to irreversible: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2382/2382 [00:03<00:00, 740.61it/s]\n",
      "Splitting isozymes:   0%|                                                                                                                                                         | 0/3239 [00:00<?, ?it/s]/home/vmsapereira/anaconda3/lib/python3.9/site-packages/cobra/core/group.py:148: UserWarning: need to pass in a list\n",
      "  warn(\"need to pass in a list\")\n",
      "Splitting isozymes: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 3239/3239 [00:41<00:00, 77.28it/s]\n",
      "Adding gene species: 100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 1261/1261 [00:10<00:00, 116.79it/s]\n",
      "Adding proteins usage to reactions: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 6548/6548 [04:02<00:00, 26.96it/s]\n"
     ]
    }
   ],
   "source": [
    "ec_sim = add_enzyme_constraints(sim, data=data)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "421d4f16",
   "metadata": {},
   "source": [
    "### Protein pool usage"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "9578f5f3",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "objective: 28.215758805752927\n",
      "Status: OPTIMAL\n",
      "Constraints: OrderedDict()\n",
      "Method:SimulationMethod.FBA\n"
     ]
    }
   ],
   "source": [
    "res = ec_sim.simulate()\n",
    "print(res)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "1584e366",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Flux rate</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Reaction ID</th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>prot_pool_exchange</th>\n",
       "      <td>12.729624</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                    Flux rate\n",
       "Reaction ID                  \n",
       "prot_pool_exchange  12.729624"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "res.find('pool')"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "5d614509",
   "metadata": {},
   "source": [
    "### Save the model"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "0164c6ae",
   "metadata": {},
   "outputs": [],
   "source": [
    "write_sbml_model(ec_sim.model,\"eciAF1260.xml\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "62fef5d4",
   "metadata": {},
   "outputs": [],
   "source": [
    "env = Environment.complete(ec_sim, max_uptake=1000.0, inplace=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "8fbc4164",
   "metadata": {},
   "outputs": [],
   "source": [
    "env['prot_pool_exchange']=(0,0.8)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "74f7b5cb",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "objective: 4.4311771722358655\n",
       "Status: OPTIMAL\n",
       "Constraints: OrderedDict([('EX_g3ps_e', (-1000.0, inf)), ('EX_cynt_e', (-1000.0, inf)), ('EX_acac_e', (-1000.0, inf)), ('EX_indole_e', (-1000.0, inf)), ('EX_inost_e', (-1000.0, inf)), ('EX_3amp_e', (-1000.0, inf)), ('EX_xmp_e', (-1000.0, inf)), ('EX_lipa_cold_e', (-1000.0, inf)), ('EX_ppal_e', (-1000.0, inf)), ('EX_adocbl_e', (-1000.0, inf)), ('EX_hom__L_e', (-1000.0, inf)), ('EX_dcyt_e', (-1000.0, inf)), ('EX_kdo2lipid4_e', (-1000.0, inf)), ('EX_cys__L_e', (-1000.0, inf)), ('EX_ppt_e', (-1000.0, inf)), ('EX_gthox_e', (-1000.0, inf)), ('EX_anhgm_e', (-1000.0, inf)), ('EX_maltpt_e', (-1000.0, inf)), ('EX_pacald_e', (-1000.0, inf)), ('EX_ocdcea_e', (-1000.0, inf)), ('EX_galctn__L_e', (-1000.0, inf)), ('EX_ura_e', (-1000.0, inf)), ('EX_tcynt_e', (-1000.0, inf)), ('EX_lyx__L_e', (-1000.0, inf)), ('EX_no3_e', (-1000.0, inf)), ('EX_3hcinnm_e', (-1000.0, inf)), ('EX_skm_e', (-1000.0, inf)), ('EX_cpgn_un_e', (-1000.0, inf)), ('EX_tym_e', (-1000.0, inf)), ('EX_taur_e', (-1000.0, inf)), ('EX_gal_e', (-1000.0, inf)), ('EX_hxa_e', (-1000.0, inf)), ('EX_mso3_e', (-1000.0, inf)), ('EX_idon__L_e', (-1000.0, inf)), ('EX_nh4_e', (-1000.0, inf)), ('EX_so3_e', (-1000.0, inf)), ('EX_g6p_e', (-1000.0, inf)), ('EX_ni2_e', (-1000.0, inf)), ('EX_na1_e', (-1000.0, inf)), ('EX_hxan_e', (-1000.0, inf)), ('EX_octa_e', (-1000.0, inf)), ('EX_tungs_e', (-1000.0, inf)), ('EX_thr__L_e', (-1000.0, inf)), ('EX_tyr__L_e', (-1000.0, inf)), ('EX_csn_e', (-1000.0, inf)), ('EX_fru_e', (-1000.0, inf)), ('EX_fecrm_e', (-1000.0, inf)), ('EX_rmn_e', (-1000.0, inf)), ('EX_5dglcn_e', (-1000.0, inf)), ('EX_enter_e', (-1000.0, inf)), ('EX_thm_e', (-1000.0, inf)), ('EX_din_e', (-1000.0, inf)), ('EX_dimp_e', (-1000.0, inf)), ('EX_his__L_e', (-1000.0, inf)), ('EX_dad_2_e', (-1000.0, inf)), ('EX_dha_e', (-1000.0, inf)), ('EX_glu__L_e', (-1000.0, inf)), ('EX_glyc__R_e', (-1000.0, inf)), ('EX_metsox__S__L_e', (-1000.0, inf)), ('EX_gln__L_e', (-1000.0, inf)), ('EX_man_e', (-1000.0, inf)), ('EX_eca4colipa_e', (-1000.0, inf)), ('EX_cit_e', (-1000.0, inf)), ('EX_ocdca_e', (-1000.0, inf)), ('EX_acgal1p_e', (-1000.0, inf)), ('EX_udpgal_e', (-1000.0, inf)), ('EX_ddca_e', (-1000.0, inf)), ('EX_arab__L_e', (-1000.0, inf)), ('EX_amp_e', (-1000.0, inf)), ('EX_ppa_e', (-1000.0, inf)), ('EX_hacolipa_e', (-1000.0, inf)), ('EX_orot_e', (-1000.0, inf)), ('EX_galct__D_e', (-1000.0, inf)), ('EX_cys__D_e', (-1000.0, inf)), ('EX_h2o2_e', (-1000.0, inf)), ('EX_fe3dhbzs_e', (-1000.0, inf)), ('EX__LalaDgluMdapDala_e', (-1000.0, inf)), ('EX_g3pi_e', (-1000.0, inf)), ('EX_phe__L_e', (-1000.0, inf)), ('EX_ala__D_e', (-1000.0, inf)), ('EX_acmum_e', (-1000.0, inf)), ('EX_glcur_e', (-1000.0, inf)), ('EX_fe3dcit_e', (-1000.0, inf)), ('EX_ins_e', (-1000.0, inf)), ('EX_4hoxpacd_e', (-1000.0, inf)), ('EX_glcr_e', (-1000.0, inf)), ('EX_thrp_e', (-1000.0, inf)), ('EX_glyb_e', (-1000.0, inf)), ('EX_malt_e', (-1000.0, inf)), ('EX_asp__L_e', (-1000.0, inf)), ('EX_glyc2p_e', (-1000.0, inf)), ('EX_etoh_e', (-1000.0, inf)), ('EX_cd2_e', (-1000.0, inf)), ('EX_h2_e', (-1000.0, inf)), ('EX_crn_e', (-1000.0, inf)), ('EX_gal_bD_e', (-1000.0, inf)), ('EX_tma_e', (-1000.0, inf)), ('EX_acser_e', (-1000.0, inf)), ('EX_h2o_e', (-1000.0, inf)), ('EX_lys__L_e', (-1000.0, inf)), ('EX_cu_e', (-1000.0, inf)), ('EX_g3pe_e', (-1000.0, inf)), ('EX_no2_e', (-1000.0, inf)), ('EX_progly_e', (-1000.0, inf)), ('EX_cu2_e', (-1000.0, inf)), ('EX_gam_e', (-1000.0, inf)), ('EX_3cmp_e', (-1000.0, inf)), ('EX_xtsn_e', (-1000.0, inf)), ('EX_fum_e', (-1000.0, inf)), ('EX_gthrd_e', (-1000.0, inf)), ('EX_dump_e', (-1000.0, inf)), ('EX_galctn__D_e', (-1000.0, inf)), ('EX_cbi_e', (-1000.0, inf)), ('EX_gsn_e', (-1000.0, inf)), ('EX_uri_e', (-1000.0, inf)), ('EX_halipa_e', (-1000.0, inf)), ('EX_succ_e', (-1000.0, inf)), ('EX_arg__L_e', (-1000.0, inf)), ('EX_tmao_e', (-1000.0, inf)), ('EX_so4_e', (-1000.0, inf)), ('EX_etha_e', (-1000.0, inf)), ('EX_acolipa_e', (-1000.0, inf)), ('EX_fuc__L_e', (-1000.0, inf)), ('EX_cbl1_e', (-1000.0, inf)), ('EX_15dap_e', (-1000.0, inf)), ('EX_gtp_e', (-1000.0, inf)), ('EX_23ccmp_e', (-1000.0, inf)), ('EX_ile__L_e', (-1000.0, inf)), ('EX_adn_e', (-1000.0, inf)), ('EX_dgmp_e', (-1000.0, inf)), ('EX_glyald_e', (-1000.0, inf)), ('EX_n2o_e', (-1000.0, inf)), ('EX_udpacgal_e', (-1000.0, inf)), ('EX_psclys_e', (-1000.0, inf)), ('EX_pi_e', (-1000.0, inf)), ('EX_hdcea_e', (-1000.0, inf)), ('EX_xyl__D_e', (-1000.0, inf)), ('EX_cmp_e', (-1000.0, inf)), ('EX_co2_e', (-1000.0, inf)), ('EX_gam6p_e', (-1000.0, inf)), ('EX_12ppd__S_e', (-1000.0, inf)), ('EX_glcn_e', (-1000.0, inf)), ('EX_pser__L_e', (-1000.0, inf)), ('EX_tartr__L_e', (-1000.0, inf)), ('EX_sucr_e', (-1000.0, inf)), ('EX_ttdcea_e', (-1000.0, inf)), ('EX_gal1p_e', (-1000.0, inf)), ('EX_all__D_e', (-1000.0, inf)), ('EX_cgly_e', (-1000.0, inf)), ('EX_mn2_e', (-1000.0, inf)), ('EX_arbtn_e', (-1000.0, inf)), ('EX_f6p_e', (-1000.0, inf)), ('EX_manglyc_e', (-1000.0, inf)), ('EX_aso3_e', (-1000.0, inf)), ('EX_arbtn_fe3_e', (-1000.0, inf)), ('EX_14glucan_e', (-1000.0, inf)), ('EX_feoxam_e', (-1000.0, inf)), ('EX_mg2_e', (-1000.0, inf)), ('EX_imp_e', (-1000.0, inf)), ('EX_fald_e', (-1000.0, inf)), ('EX_nac_e', (-1000.0, inf)), ('EX_fe3hox_un_e', (-1000.0, inf)), ('EX_isetac_e', (-1000.0, inf)), ('EX_udpg_e', (-1000.0, inf)), ('EX__LalaDgluMdap_e', (-1000.0, inf)), ('EX_agm_e', (-1000.0, inf)), ('EX_cytd_e', (-1000.0, inf)), ('EX_3hpppn_e', (-1000.0, inf)), ('EX_acmana_e', (-1000.0, inf)), ('EX_acgam1p_e', (-1000.0, inf)), ('EX_mal__L_e', (-1000.0, inf)), ('EX_ag_e', (-1000.0, inf)), ('EX_dca_e', (-1000.0, inf)), ('EX_cobalt2_e', (-1000.0, inf)), ('EX_damp_e', (-1000.0, inf)), ('EX_lcts_e', (-1000.0, inf)), ('EX_ala__L_e', (-1000.0, inf)), ('EX_metsox__R__L_e', (-1000.0, inf)), ('EX_ser__L_e', (-1000.0, inf)), ('EX_ttdca_e', (-1000.0, inf)), ('EX_pro__L_e', (-1000.0, inf)), ('EX_tre_e', (-1000.0, inf)), ('EX_4abut_e', (-1000.0, inf)), ('EX_tyrp_e', (-1000.0, inf)), ('EX_met__L_e', (-1000.0, inf)), ('EX_glyc_e', (-1000.0, inf)), ('EX_gdp_e', (-1000.0, inf)), ('EX_mnl_e', (-1000.0, inf)), ('EX_frulys_e', (-1000.0, inf)), ('EX_acnam_e', (-1000.0, inf)), ('EX_acgam_e', (-1000.0, inf)), ('EX_g3pc_e', (-1000.0, inf)), ('EX_fruur_e', (-1000.0, inf)), ('EX_k_e', (-1000.0, inf)), ('EX_hdca_e', (-1000.0, inf)), ('EX_mmet_e', (-1000.0, inf)), ('EX_thym_e', (-1000.0, inf)), ('EX_lipa_e', (-1000.0, inf)), ('EX_no_e', (-1000.0, inf)), ('EX_colipa_e', (-1000.0, inf)), ('EX_akg_e', (-1000.0, inf)), ('EX_ser__D_e', (-1000.0, inf)), ('EX_val__L_e', (-1000.0, inf)), ('EX_met__D_e', (-1000.0, inf)), ('EX_23cump_e', (-1000.0, inf)), ('EX_spmd_e', (-1000.0, inf)), ('EX_alaala_e', (-1000.0, inf)), ('EX_enlipa_e', (-1000.0, inf)), ('EX_pppn_e', (-1000.0, inf)), ('EX_h2s_e', (-1000.0, inf)), ('EX_dgsn_e', (-1000.0, inf)), ('EX_o2s_e', (-1000.0, inf)), ('EX_leu__L_e', (-1000.0, inf)), ('EX_galt_e', (-1000.0, inf)), ('EX_acald_e', (-1000.0, inf)), ('EX_mobd_e', (-1000.0, inf)), ('EX_23camp_e', (-1000.0, inf)), ('EX_feoxam_un_e', (-1000.0, inf)), ('EX_g1p_e', (-1000.0, inf)), ('EX_butso3_e', (-1000.0, inf)), ('EX_r5p_e', (-1000.0, inf)), ('EX_but_e', (-1000.0, inf)), ('EX_fe2_e', (-1000.0, inf)), ('EX_ac_e', (-1000.0, inf)), ('EX_sulfac_e', (-1000.0, inf)), ('EX_glcur1p_e', (-1000.0, inf)), ('EX_urea_e', (-1000.0, inf)), ('EX_minohp_e', (-1000.0, inf)), ('EX_12ppd__R_e', (-1000.0, inf)), ('EX_malttr_e', (-1000.0, inf)), ('EX_23dappa_e', (-1000.0, inf)), ('EX_tsul_e', (-1000.0, inf)), ('EX_cpgn_e', (-1000.0, inf)), ('EX_alltn_e', (-1000.0, inf)), ('EX_g3pg_e', (-1000.0, inf)), ('EX_ptrc_e', (-1000.0, inf)), ('EX_gua_e', (-1000.0, inf)), ('EX_rib__D_e', (-1000.0, inf)), ('EX_2ddglcn_e', (-1000.0, inf)), ('EX_maltttr_e', (-1000.0, inf)), ('EX_melib_e', (-1000.0, inf)), ('EX_h_e', (-1000.0, inf)), ('EX_ethso3_e', (-1000.0, inf)), ('EX_fe3hox_e', (-1000.0, inf)), ('EX_cyan_e', (-1000.0, inf)), ('EX_23cgmp_e', (-1000.0, inf)), ('EX_thymd_e', (-1000.0, inf)), ('EX_dtmp_e', (-1000.0, inf)), ('EX_ala_B_e', (-1000.0, inf)), ('EX_malthx_e', (-1000.0, inf)), ('EX_gbbtn_e', (-1000.0, inf)), ('EX_ascb__L_e', (-1000.0, inf)), ('EX_pnto__R_e', (-1000.0, inf)), ('EX_ump_e', (-1000.0, inf)), ('EX_feenter_e', (-1000.0, inf)), ('EX_cl_e', (-1000.0, inf)), ('EX_glyclt_e', (-1000.0, inf)), ('EX_galur_e', (-1000.0, inf)), ('EX_lac__D_e', (-1000.0, inf)), ('EX_pyr_e', (-1000.0, inf)), ('EX_udpglcur_e', (-1000.0, inf)), ('EX_so2_e', (-1000.0, inf)), ('EX_xan_e', (-1000.0, inf)), ('EX_gmp_e', (-1000.0, inf)), ('EX_glyc3p_e', (-1000.0, inf)), ('EX_glc__D_e', (-1000.0, inf)), ('EX_nmn_e', (-1000.0, inf)), ('EX_peamn_e', (-1000.0, inf)), ('EX_26dap_M_e', (-1000.0, inf)), ('EX_3gmp_e', (-1000.0, inf)), ('EX_gly_e', (-1000.0, inf)), ('EX_orn_e', (-1000.0, inf)), ('EX_mal__D_e', (-1000.0, inf)), ('EX_zn2_e', (-1000.0, inf)), ('EX_3ump_e', (-1000.0, inf)), ('EX_xylu__L_e', (-1000.0, inf)), ('EX_dcmp_e', (-1000.0, inf)), ('EX_lac__L_e', (-1000.0, inf)), ('EX_ca2_e', (-1000.0, inf)), ('EX_man6p_e', (-1000.0, inf)), ('EX_acgal_e', (-1000.0, inf)), ('EX_sbt__D_e', (-1000.0, inf)), ('EX_o16a4colipa_e', (-1000.0, inf)), ('EX_dmso_e', (-1000.0, inf)), ('EX_fecrm_un_e', (-1000.0, inf)), ('EX_for_e', (-1000.0, inf)), ('EX_chol_e', (-1000.0, inf)), ('EX_asn__L_e', (-1000.0, inf)), ('EX_dopa_e', (-1000.0, inf)), ('EX_pheme_e', (-1000.0, inf)), ('EX_trp__L_e', (-1000.0, inf)), ('EX_fe3_e', (-1000.0, inf)), ('EX_34dhpac_e', (-1000.0, inf)), ('EX_o2_e', (-1000.0, inf)), ('EX_duri_e', (-1000.0, inf)), ('EX_uacgam_e', (-1000.0, inf)), ('EX_hg2_e', (-1000.0, inf)), ('EX_ade_e', (-1000.0, inf)), ('EX_dms_e', (-1000.0, inf)), ('EX_g3ps_e_REV', (-1000.0, inf)), ('EX_cynt_e_REV', (-1000.0, inf)), ('EX_acac_e_REV', (-1000.0, inf)), ('EX_indole_e_REV', (-1000.0, inf)), ('EX_inost_e_REV', (-1000.0, inf)), ('EX_3amp_e_REV', (-1000.0, inf)), ('EX_xmp_e_REV', (-1000.0, inf)), ('EX_lipa_cold_e_REV', (-1000.0, inf)), ('EX_ppal_e_REV', (-1000.0, inf)), ('EX_adocbl_e_REV', (-1000.0, inf)), ('EX_hom__L_e_REV', (-1000.0, inf)), ('EX_dcyt_e_REV', (-1000.0, inf)), ('EX_kdo2lipid4_e_REV', (-1000.0, inf)), ('EX_cys__L_e_REV', (-1000.0, inf)), ('EX_ppt_e_REV', (-1000.0, inf)), ('EX_gthox_e_REV', (-1000.0, inf)), ('EX_anhgm_e_REV', (-1000.0, inf)), ('EX_maltpt_e_REV', (-1000.0, inf)), ('EX_pacald_e_REV', (-1000.0, inf)), ('EX_ocdcea_e_REV', (-1000.0, inf)), ('EX_galctn__L_e_REV', (-1000.0, inf)), ('EX_ura_e_REV', (-1000.0, inf)), ('EX_tcynt_e_REV', (-1000.0, inf)), ('EX_lyx__L_e_REV', (-1000.0, inf)), ('EX_no3_e_REV', (-1000.0, inf)), ('EX_3hcinnm_e_REV', (-1000.0, inf)), ('EX_skm_e_REV', (-1000.0, inf)), ('EX_cpgn_un_e_REV', (-1000.0, inf)), ('EX_tym_e_REV', (-1000.0, inf)), ('EX_taur_e_REV', (-1000.0, inf)), ('EX_gal_e_REV', (-1000.0, inf)), ('EX_hxa_e_REV', (-1000.0, inf)), ('EX_mso3_e_REV', (-1000.0, inf)), ('EX_idon__L_e_REV', (-1000.0, inf)), ('EX_nh4_e_REV', (-1000.0, inf)), ('EX_so3_e_REV', (-1000.0, inf)), ('EX_g6p_e_REV', (-1000.0, inf)), ('EX_ni2_e_REV', (-1000.0, inf)), ('EX_na1_e_REV', (-1000.0, inf)), ('EX_hxan_e_REV', (-1000.0, inf)), ('EX_octa_e_REV', (-1000.0, inf)), ('EX_tungs_e_REV', (-1000.0, inf)), ('EX_thr__L_e_REV', (-1000.0, inf)), ('EX_tyr__L_e_REV', (-1000.0, inf)), ('EX_csn_e_REV', (-1000.0, inf)), ('EX_fru_e_REV', (-1000.0, inf)), ('EX_fecrm_e_REV', (-1000.0, inf)), ('EX_rmn_e_REV', (-1000.0, inf)), ('EX_5dglcn_e_REV', (-1000.0, inf)), ('EX_enter_e_REV', (-1000.0, inf)), ('EX_thm_e_REV', (-1000.0, inf)), ('EX_din_e_REV', (-1000.0, inf)), ('EX_dimp_e_REV', (-1000.0, inf)), ('EX_his__L_e_REV', (-1000.0, inf)), ('EX_dad_2_e_REV', (-1000.0, inf)), ('EX_dha_e_REV', (-1000.0, inf)), ('EX_glu__L_e_REV', (-1000.0, inf)), ('EX_glyc__R_e_REV', (-1000.0, inf)), ('EX_metsox__S__L_e_REV', (-1000.0, inf)), ('EX_gln__L_e_REV', (-1000.0, inf)), ('EX_man_e_REV', (-1000.0, inf)), ('EX_eca4colipa_e_REV', (-1000.0, inf)), ('EX_cit_e_REV', (-1000.0, inf)), ('EX_ocdca_e_REV', (-1000.0, inf)), ('EX_acgal1p_e_REV', (-1000.0, inf)), ('EX_udpgal_e_REV', (-1000.0, inf)), ('EX_ddca_e_REV', (-1000.0, inf)), ('EX_arab__L_e_REV', (-1000.0, inf)), ('EX_amp_e_REV', (-1000.0, inf)), ('EX_ppa_e_REV', (-1000.0, inf)), ('EX_hacolipa_e_REV', (-1000.0, inf)), ('EX_orot_e_REV', (-1000.0, inf)), ('EX_galct__D_e_REV', (-1000.0, inf)), ('EX_cys__D_e_REV', (-1000.0, inf)), ('EX_h2o2_e_REV', (-1000.0, inf)), ('EX_fe3dhbzs_e_REV', (-1000.0, inf)), ('EX__LalaDgluMdapDala_e_REV', (-1000.0, inf)), ('EX_g3pi_e_REV', (-1000.0, inf)), ('EX_phe__L_e_REV', (-1000.0, inf)), ('EX_ala__D_e_REV', (-1000.0, inf)), ('EX_acmum_e_REV', (-1000.0, inf)), ('EX_glcur_e_REV', (-1000.0, inf)), ('EX_fe3dcit_e_REV', (-1000.0, inf)), ('EX_ins_e_REV', (-1000.0, inf)), ('EX_4hoxpacd_e_REV', (-1000.0, inf)), ('EX_glcr_e_REV', (-1000.0, inf)), ('EX_thrp_e_REV', (-1000.0, inf)), ('EX_glyb_e_REV', (-1000.0, inf)), ('EX_malt_e_REV', (-1000.0, inf)), ('EX_asp__L_e_REV', (-1000.0, inf)), ('EX_glyc2p_e_REV', (-1000.0, inf)), ('EX_etoh_e_REV', (-1000.0, inf)), ('EX_cd2_e_REV', (-1000.0, inf)), ('EX_h2_e_REV', (-1000.0, inf)), ('EX_crn_e_REV', (-1000.0, inf)), ('EX_gal_bD_e_REV', (-1000.0, inf)), ('EX_tma_e_REV', (-1000.0, inf)), ('EX_acser_e_REV', (-1000.0, inf)), ('EX_h2o_e_REV', (-1000.0, inf)), ('EX_lys__L_e_REV', (-1000.0, inf)), ('EX_cu_e_REV', (-1000.0, inf)), ('EX_g3pe_e_REV', (-1000.0, inf)), ('EX_no2_e_REV', (-1000.0, inf)), ('EX_progly_e_REV', (-1000.0, inf)), ('EX_cu2_e_REV', (-1000.0, inf)), ('EX_gam_e_REV', (-1000.0, inf)), ('EX_3cmp_e_REV', (-1000.0, inf)), ('EX_xtsn_e_REV', (-1000.0, inf)), ('EX_fum_e_REV', (-1000.0, inf)), ('EX_gthrd_e_REV', (-1000.0, inf)), ('EX_dump_e_REV', (-1000.0, inf)), ('EX_galctn__D_e_REV', (-1000.0, inf)), ('EX_cbi_e_REV', (-1000.0, inf)), ('EX_gsn_e_REV', (-1000.0, inf)), ('EX_uri_e_REV', (-1000.0, inf)), ('EX_halipa_e_REV', (-1000.0, inf)), ('EX_succ_e_REV', (-1000.0, inf)), ('EX_arg__L_e_REV', (-1000.0, inf)), ('EX_tmao_e_REV', (-1000.0, inf)), ('EX_so4_e_REV', (-1000.0, inf)), ('EX_etha_e_REV', (-1000.0, inf)), ('EX_acolipa_e_REV', (-1000.0, inf)), ('EX_fuc__L_e_REV', (-1000.0, inf)), ('EX_cbl1_e_REV', (-1000.0, inf)), ('EX_15dap_e_REV', (-1000.0, inf)), ('EX_gtp_e_REV', (-1000.0, inf)), ('EX_23ccmp_e_REV', (-1000.0, inf)), ('EX_ile__L_e_REV', (-1000.0, inf)), ('EX_adn_e_REV', (-1000.0, inf)), ('EX_dgmp_e_REV', (-1000.0, inf)), ('EX_glyald_e_REV', (-1000.0, inf)), ('EX_n2o_e_REV', (-1000.0, inf)), ('EX_udpacgal_e_REV', (-1000.0, inf)), ('EX_psclys_e_REV', (-1000.0, inf)), ('EX_pi_e_REV', (-1000.0, inf)), ('EX_hdcea_e_REV', (-1000.0, inf)), ('EX_xyl__D_e_REV', (-1000.0, inf)), ('EX_cmp_e_REV', (-1000.0, inf)), ('EX_co2_e_REV', (-1000.0, inf)), ('EX_gam6p_e_REV', (-1000.0, inf)), ('EX_12ppd__S_e_REV', (-1000.0, inf)), ('EX_glcn_e_REV', (-1000.0, inf)), ('EX_pser__L_e_REV', (-1000.0, inf)), ('EX_tartr__L_e_REV', (-1000.0, inf)), ('EX_sucr_e_REV', (-1000.0, inf)), ('EX_ttdcea_e_REV', (-1000.0, inf)), ('EX_gal1p_e_REV', (-1000.0, inf)), ('EX_all__D_e_REV', (-1000.0, inf)), ('EX_cgly_e_REV', (-1000.0, inf)), ('EX_mn2_e_REV', (-1000.0, inf)), ('EX_arbtn_e_REV', (-1000.0, inf)), ('EX_f6p_e_REV', (-1000.0, inf)), ('EX_manglyc_e_REV', (-1000.0, inf)), ('EX_aso3_e_REV', (-1000.0, inf)), ('EX_arbtn_fe3_e_REV', (-1000.0, inf)), ('EX_14glucan_e_REV', (-1000.0, inf)), ('EX_feoxam_e_REV', (-1000.0, inf)), ('EX_mg2_e_REV', (-1000.0, inf)), ('EX_imp_e_REV', (-1000.0, inf)), ('EX_fald_e_REV', (-1000.0, inf)), ('EX_nac_e_REV', (-1000.0, inf)), ('EX_fe3hox_un_e_REV', (-1000.0, inf)), ('EX_isetac_e_REV', (-1000.0, inf)), ('EX_udpg_e_REV', (-1000.0, inf)), ('EX__LalaDgluMdap_e_REV', (-1000.0, inf)), ('EX_agm_e_REV', (-1000.0, inf)), ('EX_cytd_e_REV', (-1000.0, inf)), ('EX_3hpppn_e_REV', (-1000.0, inf)), ('EX_acmana_e_REV', (-1000.0, inf)), ('EX_acgam1p_e_REV', (-1000.0, inf)), ('EX_mal__L_e_REV', (-1000.0, inf)), ('EX_ag_e_REV', (-1000.0, inf)), ('EX_dca_e_REV', (-1000.0, inf)), ('EX_cobalt2_e_REV', (-1000.0, inf)), ('EX_damp_e_REV', (-1000.0, inf)), ('EX_lcts_e_REV', (-1000.0, inf)), ('EX_ala__L_e_REV', (-1000.0, inf)), ('EX_metsox__R__L_e_REV', (-1000.0, inf)), ('EX_ser__L_e_REV', (-1000.0, inf)), ('EX_ttdca_e_REV', (-1000.0, inf)), ('EX_pro__L_e_REV', (-1000.0, inf)), ('EX_tre_e_REV', (-1000.0, inf)), ('EX_4abut_e_REV', (-1000.0, inf)), ('EX_tyrp_e_REV', (-1000.0, inf)), ('EX_met__L_e_REV', (-1000.0, inf)), ('EX_glyc_e_REV', (-1000.0, inf)), ('EX_gdp_e_REV', (-1000.0, inf)), ('EX_mnl_e_REV', (-1000.0, inf)), ('EX_frulys_e_REV', (-1000.0, inf)), ('EX_acnam_e_REV', (-1000.0, inf)), ('EX_acgam_e_REV', (-1000.0, inf)), ('EX_g3pc_e_REV', (-1000.0, inf)), ('EX_fruur_e_REV', (-1000.0, inf)), ('EX_k_e_REV', (-1000.0, inf)), ('EX_hdca_e_REV', (-1000.0, inf)), ('EX_mmet_e_REV', (-1000.0, inf)), ('EX_thym_e_REV', (-1000.0, inf)), ('EX_lipa_e_REV', (-1000.0, inf)), ('EX_no_e_REV', (-1000.0, inf)), ('EX_colipa_e_REV', (-1000.0, inf)), ('EX_akg_e_REV', (-1000.0, inf)), ('EX_ser__D_e_REV', (-1000.0, inf)), ('EX_val__L_e_REV', (-1000.0, inf)), ('EX_met__D_e_REV', (-1000.0, inf)), ('EX_23cump_e_REV', (-1000.0, inf)), ('EX_spmd_e_REV', (-1000.0, inf)), ('EX_alaala_e_REV', (-1000.0, inf)), ('EX_enlipa_e_REV', (-1000.0, inf)), ('EX_pppn_e_REV', (-1000.0, inf)), ('EX_h2s_e_REV', (-1000.0, inf)), ('EX_dgsn_e_REV', (-1000.0, inf)), ('EX_o2s_e_REV', (-1000.0, inf)), ('EX_leu__L_e_REV', (-1000.0, inf)), ('EX_galt_e_REV', (-1000.0, inf)), ('EX_acald_e_REV', (-1000.0, inf)), ('EX_mobd_e_REV', (-1000.0, inf)), ('EX_23camp_e_REV', (-1000.0, inf)), ('EX_feoxam_un_e_REV', (-1000.0, inf)), ('EX_g1p_e_REV', (-1000.0, inf)), ('EX_butso3_e_REV', (-1000.0, inf)), ('EX_r5p_e_REV', (-1000.0, inf)), ('EX_but_e_REV', (-1000.0, inf)), ('EX_fe2_e_REV', (-1000.0, inf)), ('EX_ac_e_REV', (-1000.0, inf)), ('EX_sulfac_e_REV', (-1000.0, inf)), ('EX_glcur1p_e_REV', (-1000.0, inf)), ('EX_urea_e_REV', (-1000.0, inf)), ('EX_minohp_e_REV', (-1000.0, inf)), ('EX_12ppd__R_e_REV', (-1000.0, inf)), ('EX_malttr_e_REV', (-1000.0, inf)), ('EX_23dappa_e_REV', (-1000.0, inf)), ('EX_tsul_e_REV', (-1000.0, inf)), ('EX_cpgn_e_REV', (-1000.0, inf)), ('EX_alltn_e_REV', (-1000.0, inf)), ('EX_g3pg_e_REV', (-1000.0, inf)), ('EX_ptrc_e_REV', (-1000.0, inf)), ('EX_gua_e_REV', (-1000.0, inf)), ('EX_rib__D_e_REV', (-1000.0, inf)), ('EX_2ddglcn_e_REV', (-1000.0, inf)), ('EX_maltttr_e_REV', (-1000.0, inf)), ('EX_melib_e_REV', (-1000.0, inf)), ('EX_h_e_REV', (-1000.0, inf)), ('EX_ethso3_e_REV', (-1000.0, inf)), ('EX_fe3hox_e_REV', (-1000.0, inf)), ('EX_cyan_e_REV', (-1000.0, inf)), ('EX_23cgmp_e_REV', (-1000.0, inf)), ('EX_thymd_e_REV', (-1000.0, inf)), ('EX_dtmp_e_REV', (-1000.0, inf)), ('EX_ala_B_e_REV', (-1000.0, inf)), ('EX_malthx_e_REV', (-1000.0, inf)), ('EX_gbbtn_e_REV', (-1000.0, inf)), ('EX_ascb__L_e_REV', (-1000.0, inf)), ('EX_pnto__R_e_REV', (-1000.0, inf)), ('EX_ump_e_REV', (-1000.0, inf)), ('EX_feenter_e_REV', (-1000.0, inf)), ('EX_cl_e_REV', (-1000.0, inf)), ('EX_glyclt_e_REV', (-1000.0, inf)), ('EX_galur_e_REV', (-1000.0, inf)), ('EX_lac__D_e_REV', (-1000.0, inf)), ('EX_pyr_e_REV', (-1000.0, inf)), ('EX_udpglcur_e_REV', (-1000.0, inf)), ('EX_so2_e_REV', (-1000.0, inf)), ('EX_xan_e_REV', (-1000.0, inf)), ('EX_gmp_e_REV', (-1000.0, inf)), ('EX_glyc3p_e_REV', (-1000.0, inf)), ('EX_glc__D_e_REV', (-1000.0, inf)), ('EX_nmn_e_REV', (-1000.0, inf)), ('EX_peamn_e_REV', (-1000.0, inf)), ('EX_26dap_M_e_REV', (-1000.0, inf)), ('EX_3gmp_e_REV', (-1000.0, inf)), ('EX_gly_e_REV', (-1000.0, inf)), ('EX_orn_e_REV', (-1000.0, inf)), ('EX_mal__D_e_REV', (-1000.0, inf)), ('EX_zn2_e_REV', (-1000.0, inf)), ('EX_3ump_e_REV', (-1000.0, inf)), ('EX_xylu__L_e_REV', (-1000.0, inf)), ('EX_dcmp_e_REV', (-1000.0, inf)), ('EX_lac__L_e_REV', (-1000.0, inf)), ('EX_ca2_e_REV', (-1000.0, inf)), ('EX_man6p_e_REV', (-1000.0, inf)), ('EX_acgal_e_REV', (-1000.0, inf)), ('EX_sbt__D_e_REV', (-1000.0, inf)), ('EX_o16a4colipa_e_REV', (-1000.0, inf)), ('EX_dmso_e_REV', (-1000.0, inf)), ('EX_fecrm_un_e_REV', (-1000.0, inf)), ('EX_for_e_REV', (-1000.0, inf)), ('EX_chol_e_REV', (-1000.0, inf)), ('EX_asn__L_e_REV', (-1000.0, inf)), ('EX_dopa_e_REV', (-1000.0, inf)), ('EX_pheme_e_REV', (-1000.0, inf)), ('EX_trp__L_e_REV', (-1000.0, inf)), ('EX_fe3_e_REV', (-1000.0, inf)), ('EX_34dhpac_e_REV', (-1000.0, inf)), ('EX_o2_e_REV', (-1000.0, inf)), ('EX_duri_e_REV', (-1000.0, inf)), ('EX_uacgam_e_REV', (-1000.0, inf)), ('EX_hg2_e_REV', (-1000.0, inf)), ('EX_ade_e_REV', (-1000.0, inf)), ('EX_dms_e_REV', (-1000.0, inf)), ('prot_pool_exchange', (0, 0.8))])\n",
       "Method:SimulationMethod.FBA"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "ec_sim.simulate(constraints=env)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "e4b827b2",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'b0002': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P00561',\n",
       "  'gene': 'b0002',\n",
       "  'locus': ['b0002', 'JW0001'],\n",
       "  'synonyms': 'thrA1 thrA2',\n",
       "  'properties': {'Catalytic Activity': [('L-homoserine + NADP(+) = H(+) + L-aspartate 4-semialdehyde + NADPH',\n",
       "     '1.1.1.3'),\n",
       "    ('L-homoserine + NAD(+) = H(+) + L-aspartate 4-semialdehyde + NADH',\n",
       "     '1.1.1.3'),\n",
       "    ('ATP + L-aspartate = 4-phospho-L-aspartate + ADP', '2.7.2.4')],\n",
       "   'Regulation Activity': 'The enzyme activities are regulated allosterically by L-threonine',\n",
       "   'Pathway': 'Amino-acid biosynthesis; L-threonine biosynthesis; L-threonine from L-aspartate: step 3/5'},\n",
       "  'seq': 'MRVLKFGGTSVANAERFLRVADILESNARQGQVATVLSAPAKITNHLVAMIEKTISGQDALPNISDAERIFAELLTGLAAAQPGFPLAQLKTFVDQEFAQIKHVLHGISLLGQCPDSINAALICRGEKMSIAIMAGVLEARGHNVTVIDPVEKLLAVGHYLESTVDIAESTRRIAASRIPADHMVLMAGFTAGNEKGELVVLGRNGSDYSAAVLAACLRADCCEIWTDVDGVYTCDPRQVPDARLLKSMSYQEAMELSYFGAKVLHPRTITPIAQFQIPCLIKNTGNPQAPGTLIGASRDEDELPVKGISNLNNMAMFSVSGPGMKGMVGMAARVFAAMSRARISVVLITQSSSEYSISFCVPQSDCVRAERAMQEEFYLELKEGLLEPLAVTERLAIISVVGDGMRTLRGISAKFFAALARANINIVAIAQGSSERSISVVVNNDDATTGVRVTHQMLFNTDQVIEVFVIGVGGVGGALLEQLKRQQSWLKNKHIDLRVCGVANSKALLTNVHGLNLENWQEELAQAKEPFNLGRLIRLVKEYHLLNPVIVDCTSSQAVADQYADFLREGFHVVTPNKKANTSSMDYYHQLRYAAEKSRRKFLYDTNVGAGLPVIENLQNLLNAGDELMKFSGILSGSLSYIFGKLDEGMSFSEATTLAREMGYTEPDPRDDLSGMDVARKLLILARETGRELELADIEIEPVLPAEFNAEGDVAAFMANLSQLDDLFAARVAKARDEGKVLRYVGNIDEDGVCRVKIAEVDGNDPLFKVKNGENALAFYSHYYQPLPLVLRGYGAGNDVTAAGVFADLLRTLSWKLGV',\n",
       "  'mw': 89120.0},\n",
       " 'b0003': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P00547',\n",
       "  'gene': 'b0003',\n",
       "  'locus': ['b0003', 'JW0002'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + L-homoserine = ADP + H(+) + O-phospho-L-homoserine',\n",
       "     '2.7.1.39')],\n",
       "   'Function': 'Catalyzes the ATP-dependent phosphorylation of L-homoserine to L-homoserine phosphate. Is also able to phosphorylate the hydroxy group on gamma-carbon of L-homoserine analogs when the functional group at the alpha-position is a carboxyl, an ester, or even a hydroxymethyl group. Neither L-threonine nor L-serine are substrates of the enzyme',\n",
       "   'Pathway': 'Amino-acid biosynthesis; L-threonine biosynthesis; L-threonine from L-aspartate: step 4/5'},\n",
       "  'seq': 'MVKVYAPASSANMSVGFDVLGAAVTPVDGALLGDVVTVEAAETFSLNNLGRFADKLPSEPRENIVYQCWERFCQELGKQIPVAMTLEKNMPIGSGLGSSACSVVAALMAMNEHCGKPLNDTRLLALMGELEGRISGSIHYDNVAPCFLGGMQLMIEENDIISQQVPGFDEWLWVLAYPGIKVSTAEARAILPAQYRRQDCIAHGRHLAGFIHACYSRQPELAAKLMKDVIAEPYRERLLPGFRQARQAVAEIGAVASGISGSGPTLFALCDKPETAQRVADWLGKNYLQNQEGFVHICRLDTAGARVLEN',\n",
       "  'mw': 33624.0},\n",
       " 'b0004': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P00934',\n",
       "  'gene': 'b0004',\n",
       "  'locus': ['b0004', 'JW0003'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('H2O + O-phospho-L-homoserine = L-threonine + phosphate',\n",
       "     '4.2.3.1')],\n",
       "   'Function': 'Catalyzes the gamma-elimination of phosphate from L-phosphohomoserine and the beta-addition of water to produce L-threonine. To a lesser extent, is able to slowly catalyze the deamination of L-threonine into alpha-ketobutyrate and that of L-serine and 3-chloroalanine into pyruvate. Is also able to rapidly convert vinylglycine to threonine, which proves that the pyridoxal p-quinonoid of vinylglycine is an intermediate in the TS reaction',\n",
       "   'Regulation Activity': 'Is competitively inhibited by L-threo-3-hydroxyhomoserine phosphate',\n",
       "   'Pathway': 'Amino-acid biosynthesis; L-threonine biosynthesis; L-threonine from L-aspartate: step 5/5'},\n",
       "  'seq': 'MKLYNLKDHNEQVSFAQAVTQGLGKNQGLFFPHDLPEFSLTEIDEMLKLDFVTRSAKILSAFIGDEIPQEILEERVRAAFAFPAPVANVESDVGCLELFHGPTLAFKDFGGRFMAQMLTHIAGDKPVTILTATSGDTGAAVAHAFYGLPNVKVVILYPRGKISPLQEKLFCTLGGNIETVAIDGDFDACQALVKQAFDDEELKVALGLNSANSINISRLLAQICYYFEAVAQLPQETRNQLVVSVPSGNFGDLTAGLLAKSLGLPVKRFIAATNVNDTVPRFLHDGQWSPKATQATLSNAMDVSQPNNWPRVEELFRRKIWQLKELGYAAVDDETTQQTMRELKELGYTSEPHAAVAYRALRDQLNPGEYGLFLGTAHPAKFKESVEAILGETLDLPKELAERADLPLLSHNLPADFAALRKLMMNHQ',\n",
       "  'mw': 47114.0},\n",
       " 'b0007': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P30143',\n",
       "  'gene': 'b0007',\n",
       "  'locus': ['b0007', 'JW0006'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': []},\n",
       "  'seq': 'MPDFFSFINSVLWGSVMIYLLFGAGCWFTFRTGFVQFRYIRQFGKSLKNSIHPQPGGLTSFQSLCTSLAARVGSGNLAGVALAITAGGPGAVFWMWVAAFIGMATSFAECSLAQLYKERDVNGQFRGGPAWYMARGLGMRWMGVLFAVFLLIAYGIIFSGVQANAVARALSFSFDFPPLVTGIILAVFTLLAITRGLHGVARLMQGFVPLMAIIWVLTSLVICVMNIGQLPHVIWSIFESAFGWQEAAGGAAGYTLSQAITNGFQRSMFSNEAGMGSTPNAAAAAASWPPHPAAQGIVQMIGIFIDTLVICTASAMLILLAGNGTTYMPLEGIQLIQKAMRVLMGSWGAEFVTLVVILFAFSSIVANYIYAENNLFFLRLNNPKAIWCLRICTFATVIGGTLLSLPLMWQLADIIMACMAITNLTAILLLSPVVHTIASDYLRQRKLGVRPVFDPLRYPDIGRQLSPDAWDDVSQE',\n",
       "  'mw': 51663.0},\n",
       " 'b0008': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A870',\n",
       "  'gene': 'b0008',\n",
       "  'locus': ['b0008', 'JW0007'],\n",
       "  'synonyms': 'yaaK',\n",
       "  'properties': {'Catalytic Activity': [('D-glyceraldehyde 3-phosphate + D-sedoheptulose 7-phosphate = beta-D-fructose 6-phosphate + D-erythrose 4-phosphate',\n",
       "     '2.2.1.2')],\n",
       "   'Function': 'Transaldolase is important for the balance of metabolites in the pentose-phosphate pathway',\n",
       "   'Regulation Activity': 'Inhibited by Tris-HCl and phosphate buffer. Also competitively inhibited by sugars with L configuration at C2, such as D-arabinose-5-phosphate and L-glyceraldehyde',\n",
       "   'Pathway': 'Carbohydrate degradation; pentose phosphate pathway; D-glyceraldehyde 3-phosphate and beta-D-fructose 6-phosphate from D-ribose 5-phosphate and D-xylulose 5-phosphate (non-oxidative stage): step 2/3'},\n",
       "  'seq': 'MTDKLTSLRQYTTVVADTGDIAAMKLYQPQDATTNPSLILNAAQIPEYRKLIDDAVAWAKQQSNDRAQQIVDATDKLAVNIGLEILKLVPGRISTEVDARLSYDTEASIAKAKRLIKLYNDAGISNDRILIKLASTWQGIRAAEQLEKEGINCNLTLLFSFAQARACAEAGVFLISPFVGRILDWYKANTDKKEYAPAEDPGVVSVSEIYQYYKEHGYETVVMGASFRNIGEILELAGCDRLTIAPALLKELAESEGAIERKLSYTGEVKARPARITESEFLWQHNQDPMAVDKLAEGIRKFAIDQEKLEKMIGDLL',\n",
       "  'mw': 35219.0},\n",
       " 'b0019': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P13738',\n",
       "  'gene': 'b0019',\n",
       "  'locus': ['b0019', 'JW0018'],\n",
       "  'synonyms': 'ant',\n",
       "  'properties': {'Catalytic Activity': [('2 H(+)(out) + Na(+)(in) = 2 H(+)(in) + Na(+)(out)',\n",
       "     ''),\n",
       "    ('2 H(+)(out) + Li(+)(in) = 2 H(+)(in) + Li(+)(out)', '')],\n",
       "   'Function': 'Na(+)/H(+) antiporter that extrudes sodium in exchange for external protons (PubMed:2839489, PubMed:1645730, PubMed:8383669, PubMed:8019504, PubMed:7737413, PubMed:23836890). Plays an important role in the regulation of intracellular pH, cellular Na(+) content and cell volume (PubMed:33129932). Catalyzes the exchange of 2 H(+) per Na(+) (PubMed:8383669, PubMed:23836890). This stoichiometry applies at both neutral and alkaline pH values (PubMed:8383669). In addition, can also transport lithium and is involved in lithium detoxification (PubMed:8019504, PubMed:7737413, PubMed:22915592, PubMed:27021484). Binding of the Li(+) and H(+) ligands to NhaA is coupled and antagonistic (PubMed:27021484)',\n",
       "   'Regulation Activity': 'Activity is regulated by pH (PubMed:1645730, PubMed:23836890, PubMed:10455127, PubMed:14604993). Active at alkaline pH (PubMed:1645730, PubMed:23836890, PubMed:10455127, PubMed:14604993). Activity is strongly down-regulated below pH 6.5 and a dramatic increase in activity is observed upon increase of the pH from 6.5 to 8.5 (PubMed:1645730)'},\n",
       "  'seq': 'MKHLHRFFSSDASGGIILIIAAILAMIMANSGATSGWYHDFLETPVQLRVGSLEINKNMLLWINDALMAVFFLLVGLEVKRELMQGSLASLRQAAFPVIAAIGGMIVPALLYLAFNYADPITREGWAIPAATDIAFALGVLALLGSRVPLALKIFLMALAIIDDLGAIIIIALFYTNDLSMASLGVAAVAIAVLAVLNLCGARRTGVYILVGVVLWTAVLKSGVHATLAGVIVGFFIPLKEKHGRSPAKRLEHVLHPWVAYLILPLFAFANAGVSLQGVTLDGLTSILPLGIIAGLLIGKPLGISLFCWLALRLKLAHLPEGTTYQQIMVVGILCGIGFTMSIFIASLAFGSVDPELINWAKLGILVGSISSAVIGYSWLRVRLRPSV',\n",
       "  'mw': 41356.0},\n",
       " 'b0025': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AG40',\n",
       "  'gene': 'b0025',\n",
       "  'locus': ['b0025', 'JW0023'],\n",
       "  'synonyms': 'yaaC',\n",
       "  'properties': {'Catalytic Activity': [('ATP + riboflavin = ADP + FMN + H(+)',\n",
       "     '2.7.1.26'),\n",
       "    ('ATP + FMN + H(+) = diphosphate + FAD', '2.7.7.2')],\n",
       "   'Function': 'Catalyzes the phosphorylation of riboflavin to FMN followed by the adenylation of FMN to FAD',\n",
       "   'Pathway': 'Cofactor biosynthesis; FMN biosynthesis; FMN from riboflavin (ATP route): step 1/1'},\n",
       "  'seq': 'MKLIRGIHNLSQAPQEGCVLTIGNFDGVHRGHRALLQGLQEEGRKRNLPVMVMLFEPQPLELFATDKAPARLTRLREKLRYLAECGVDYVLCVRFDRRFAALTAQNFISDLLVKHLRVKFLAVGDDFRFGAGREGDFLLLQKAGMEYGFDITSTQTFCEGGVRISSTAVRQALADDNLALAESLLGHPFAISGRVVHGDELGRTIGFPTANVPLRRQVSPVKGVYAVEVLGLGEKPLPGVANIGTRPTVAGIRQQLEVHLLDVAMDLYGRHIQVVLRKKIRNEQRFASLDELKAQIARDELTAREFFGLTKPA',\n",
       "  'mw': 34734.0},\n",
       " 'b0026': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P00956',\n",
       "  'gene': 'b0026',\n",
       "  'locus': ['b0026', 'JW0024'],\n",
       "  'synonyms': 'ilvS',\n",
       "  'properties': {'Catalytic Activity': [('ATP + L-isoleucine + tRNA(Ile) = AMP + diphosphate + L-isoleucyl-tRNA(Ile)',\n",
       "     '6.1.1.5')],\n",
       "   'Function': \"Catalyzes the attachment of isoleucine to tRNA(Ile). As IleRS can inadvertently accommodate and process structurally similar amino acids such as valine, to avoid such errors it has two additional distinct tRNA(Ile)-dependent editing activities. One activity is designated as 'pretransfer' editing and involves the hydrolysis of activated Val-AMP. The other activity is designated 'posttransfer' editing and involves deacylation of mischarged Val-tRNA(Ile)\"},\n",
       "  'seq': 'MSDYKSTLNLPETGFPMRGDLAKREPGMLARWTDDDLYGIIRAAKKGKKTFILHDGPPYANGSIHIGHSVNKILKDIIVKSKGLSGYDSPYVPGWDCHGLPIELKVEQEYGKPGEKFTAAEFRAKCREYAATQVDGQRKDFIRLGVLGDWSHPYLTMDFKTEANIIRALGKIIGNGHLHKGAKPVHWCVDCRSALAEAEVEYYDKTSPSIDVAFQAVDQDALKAKFAVSNVNGPISLVIWTTTPWTLPANRAISIAPDFDYALVQIDGQAVILAKDLVESVMQRIGVTDYTILGTVKGAELELLRFTHPFMGFDVPAILGDHVTLDAGTGAVHTAPGHGPDDYVIGQKYGLETANPVGPDGTYLPGTYPTLDGVNVFKANDIVVALLQEKGALLHVEKMQHSYPCCWRHKTPIIFRATPQWFVSMDQKGLRAQSLKEIKGVQWIPDWGQARIESMVANRPDWCISRQRTWGVPMSLFVHKDTEELHPRTLELMEEVAKRVEVDGIQAWWDLDAKEILGDEADQYVKVPDTLDVWFDSGSTHSSVVDVRPEFAGHAADMYLEGSDQHRGWFMSSLMISTAMKGKAPYRQVLTHGFTVDGQGRKMSKSIGNTVSPQDVMNKLGADILRLWVASTDYTGEMAVSDEILKRAADSYRRIRNTARFLLANLNGFDPAKDMVKPEEMVVLDRWAVGCAKAAQEDILKAYEAYDFHEVVQRLMRFCSVEMGSFYLDIIKDRQYTAKADSVARRSCQTALYHIAEALVRWMAPILSFTADEVWGYLPGEREKYVFTGEWYEGLFGLADSEAMNDAFWDELLKVRGEVNKVIEQARADKKVGGSLEAAVTLYAEPELSAKLTALGDELRFVLLTSGATVADYNDAPADAQQSEVLKGLKVALSKAEGEKCPRCWHYTQDVGKVAEHAEICGRCVSNVAGDGEKRKFA',\n",
       "  'mw': 104297.0},\n",
       " 'b0029': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P62623',\n",
       "  'gene': 'b0029',\n",
       "  'locus': ['b0029', 'JW0027'],\n",
       "  'synonyms': 'lytB yaaE',\n",
       "  'properties': {'Catalytic Activity': [('H2O + isopentenyl diphosphate + 2 oxidized [2Fe-2S]-[ferredoxin] = (2E)-4-hydroxy-3-methylbut-2-enyl diphosphate + 2 H(+) + 2 reduced [2Fe-2S]-[ferredoxin]',\n",
       "     '1.17.7.4'),\n",
       "    ('dimethylallyl diphosphate + H2O + 2 oxidized [2Fe-2S]-[ferredoxin] = (2E)-4-hydroxy-3-methylbut-2-enyl diphosphate + 2 H(+) + 2 reduced [2Fe-2S]-[ferredoxin]',\n",
       "     '1.17.7.4')],\n",
       "   'Function': 'Catalyzes the conversion of 1-hydroxy-2-methyl-2-(E)-butenyl 4-diphosphate (HMBPP) into a mixture of isopentenyl diphosphate (IPP) and dimethylallyl diphosphate (DMAPP). Acts in the terminal step of the DOXP/MEP pathway for isoprenoid precursor biosynthesis (PubMed:11818558, PubMed:11418107, PubMed:12706830, PubMed:19569147, PubMed:22137895). In vitro, can also hydrate acetylenes to aldehydes and ketones via anti-Markovnikov/Markovnikov addition (PubMed:22948824)',\n",
       "   'Regulation Activity': 'Addition of Ca(2+), Mg(2+), Mn(2+), Ni(2+), Co(2+) or Fe(2+) decreases the catalytic activity (PubMed:15469281). Addition of Zn(2+) results in complete loss of activity (PubMed:15469281). Is potently inhibited by substrate analogs in which the hydroxy group in HMBPP is replaced by an amino or thiol group (PubMed:22012762)',\n",
       "   'Pathway': 'Isoprenoid biosynthesis; isopentenyl diphosphate biosynthesis via DXP pathway; isopentenyl diphosphate from 1-deoxy-D-xylulose 5-phosphate: step 6/6'},\n",
       "  'seq': 'MQILLANPRGFCAGVDRAISIVENALAIYGAPIYVRHEVVHNRYVVDSLRERGAIFIEQISEVPDGAILIFSAHGVSQAVRNEAKSRDLTVFDATCPLVTKVHMEVARASRRGEESILIGHAGHPEVEGTMGQYSNPEGGMYLVESPDDVWKLTVKNEEKLSFMTQTTLSVDDTSDVIDALRKRFPKIVGPRKDDICYATTNRQEAVRALAEQAEVVLVVGSKNSSNSNRLAELAQRMGKRAFLIDDAKDIQEEWVKEVKCVGVTAGASAPDILVQNVVARLQQLGGGEAIPLEGREENIVFEVPKELRVDIREVD',\n",
       "  'mw': 34775.0},\n",
       " 'b0030': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P22564',\n",
       "  'gene': 'b0030',\n",
       "  'locus': ['b0030', 'JW0028'],\n",
       "  'synonyms': 'yaaF',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Hydrolyzes both purine and pyrimidine ribonucleosides with a broad-substrate specificity with decreasing activity in the order uridine, xanthosine, inosine, adenosine, cytidine, guanosine',\n",
       "   'Regulation Activity': 'Subject to catabolite repression'},\n",
       "  'seq': 'MRLPIFLDTDPGIDDAVAIAAAIFAPELDLQLMTTVAGNVSVEKTTRNALQLLHFWNAEIPLAQGAAVPLVRAPRDAASVHGESGMAGYDFVEHNRKPLGIPAFLAIRDALMRAPEPVTLVAIGPLTNIALLLSQCPECKPYIRRLVIMGGSAGRGNCTPNAEFNIAADPEAAACVFRSGIEIVMCGLDVTNQAILTPDYLSTLPQLNRTGKMLHALFSHYRSGSMQSGLRMHDLCAIAWLVRPDLFTLKPCFVAVETQGEFTSGTTVVDIDGCLGKPANVQVALDLDVKGFQQWVAEVLALAS',\n",
       "  'mw': 32561.0},\n",
       " 'b0031': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P04036',\n",
       "  'gene': 'b0031',\n",
       "  'locus': ['b0031', 'JW0029'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(S)-2,3,4,5-tetrahydrodipicolinate + H2O + NAD(+) = (2S,4S)-4-hydroxy-2,3,4,5-tetrahydrodipicolinate + H(+) + NADH',\n",
       "     '1.17.1.8'),\n",
       "    ('(S)-2,3,4,5-tetrahydrodipicolinate + H2O + NADP(+) = (2S,4S)-4-hydroxy-2,3,4,5-tetrahydrodipicolinate + H(+) + NADPH',\n",
       "     '1.17.1.8')],\n",
       "   'Function': 'Catalyzes the conversion of 4-hydroxy-tetrahydrodipicolinate (HTPA) to tetrahydrodipicolinate. Can use both NADH and NADPH as a reductant, with NADH being twice as effective as NADPH',\n",
       "   'Regulation Activity': 'Is inhibited by 2,6-pyridine dicarboxylate (2,6-PDC or picolinate)',\n",
       "   'Pathway': 'Amino-acid biosynthesis; L-lysine biosynthesis via DAP pathway; (S)-tetrahydrodipicolinate from L-aspartate: step 4/4'},\n",
       "  'seq': 'MHDANIRVAIAGAGGRMGRQLIQAALALEGVQLGAALEREGSSLLGSDAGELAGAGKTGVTVQSSLDAVKDDFDVFIDFTRPEGTLNHLAFCRQHGKGMVIGTTGFDEAGKQAIRDAAADIAIVFAANFSVGVNVMLKLLEKAAKVMGDYTDIEIIEAHHRHKVDAPSGTALAMGEAIAHALDKDLKDCAVYSREGHTGERVPGTIGFATVRAGDIVGEHTAMFADIGERLEITHKASSRMTFANGAVRSALWLSGKESGLFDMRDVLDLNNL',\n",
       "  'mw': 28757.0},\n",
       " 'b0032': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A6F1',\n",
       "  'gene': 'b0032',\n",
       "  'locus': ['b0032', 'JW0030'],\n",
       "  'synonyms': 'pyrA',\n",
       "  'properties': {'Catalytic Activity': [('2 ATP + H2O + hydrogencarbonate + L-glutamine = 2 ADP + carbamoyl phosphate + 2 H(+) + L-glutamate + phosphate',\n",
       "     '6.3.5.5')],\n",
       "   'Pathway': 'Pyrimidine metabolism; UMP biosynthesis via de novo pathway; (S)-dihydroorotate from bicarbonate: step 1/3'},\n",
       "  'seq': 'MIKSALLVLEDGTQFHGRAIGATGSAVGEVVFNTSMTGYQEILTDPSYSRQIVTLTYPHIGNVGTNDADEESSQVHAQGLVIRDLPLIASNFRNTEDLSSYLKRHNIVAIADIDTRKLTRLLREKGAQNGCIIAGDNPDAALALEKARAFPGLNGMDLAKEVTTAEAYSWTQGSWTLTGGLPEAKKEDELPFHVVAYDFGAKRNILRMLVDRGCRLTIVPAQTSAEDVLKMNPDGIFLSNGPGDPAPCDYAITAIQKFLETDIPVFGICLGHQLLALASGAKTVKMKFGHHGGNHPVKDVEKNVVMITAQNHGFAVDEATLPANLRVTHKSLFDGTLQGIHRTDKPAFSFQGHPEASPGPHDAAPLFDHFIELIEQYRKTAK',\n",
       "  'mw': 41431.0},\n",
       " 'b0033': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P00968',\n",
       "  'gene': 'b0033',\n",
       "  'locus': ['b0033', 'JW0031'],\n",
       "  'synonyms': 'pyrA',\n",
       "  'properties': {'Catalytic Activity': [('2 ATP + H2O + hydrogencarbonate + L-glutamine = 2 ADP + carbamoyl phosphate + 2 H(+) + L-glutamate + phosphate',\n",
       "     '6.3.5.5')],\n",
       "   'Pathway': 'Pyrimidine metabolism; UMP biosynthesis via de novo pathway; (S)-dihydroorotate from bicarbonate: step 1/3'},\n",
       "  'seq': 'MPKRTDIKSILILGAGPIVIGQACEFDYSGAQACKALREEGYRVILVNSNPATIMTDPEMADATYIEPIHWEVVRKIIEKERPDAVLPTMGGQTALNCALELERQGVLEEFGVTMIGATADAIDKAEDRRRFDVAMKKIGLETARSGIAHTMEEALAVAADVGFPCIIRPSFTMGGSGGGIAYNREEFEEICARGLDLSPTKELLIDESLIGWKEYEMEVVRDKNDNCIIVCSIENFDAMGIHTGDSITVAPAQTLTDKEYQIMRNASMAVLREIGVETGGSNVQFAVNPKNGRLIVIEMNPRVSRSSALASKATGFPIAKVAAKLAVGYTLDELMNDITGGRTPASFEPSIDYVVTKIPRFNFEKFAGANDRLTTQMKSVGEVMAIGRTQQESLQKALRGLEVGATGFDPKVSLDDPEALTKIRRELKDAGADRIWYIADAFRAGLSVDGVFNLTNIDRWFLVQIEELVRLEEKVAEVGITGLNADFLRQLKRKGFADARLAKLAGVREAEIRKLRDQYDLHPVYKRVDTCAAEFATDTAYMYSTYEEECEANPSTDREKIMVLGGGPNRIGQGIEFDYCCVHASLALREDGYETIMVNCNPETVSTDYDTSDRLYFEPVTLEDVLEIVRIEKPKGVIVQYGGQTPLKLARALEAAGVPVIGTSPDAIDRAEDRERFQHAVERLKLKQPANATVTAIEMAVEKAKEIGYPLVVRPSYVLGGRAMEIVYDEADLRRYFQTAVSVSNDAPVLLDHFLDDAVEVDVDAICDGEMVLIGGIMEHIEQAGVHSGDSACSLPAYTLSQEIQDVMRQQVQKLAFELQVRGLMNVQFAVKNNEVYLIEVNPRAARTVPFVSKATGVPLAKVAARVMAGKSLAEQGVTKEVIPPYYSVKEVVLPFNKFPGVDPLLGPEMRSTGEVMGVGRTFAEAFAKAQLGSNSTMKKHGRALLSVREGDKERVVDLAAKLLKQGFELDATHGTAIVLGEAGINPRLVNKVHEGRPHIQDRIKNGEYTYIINTTSGRRAIEDSRVIRRSALQYKVHYDTTLNGGFATAMALNADATEKVISVQEMHAQIK',\n",
       "  'mw': 117842.0},\n",
       " 'b0036': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P31551',\n",
       "  'gene': 'b0036',\n",
       "  'locus': ['b0036', 'JW0035'],\n",
       "  'synonyms': 'yaaL',\n",
       "  'properties': {'Catalytic Activity': [('(R)-carnitinyl-CoA = crotonobetainyl-CoA + H2O',\n",
       "     '4.2.1.149'),\n",
       "    ('3-hydroxybutanoyl-CoA = (2E)-butenoyl-CoA + H2O', '')],\n",
       "   'Function': 'Catalyzes the reversible dehydration of L-carnitinyl-CoA to crotonobetainyl-CoA (PubMed:11551212). Can also hydrate crotonyl-CoA to hydroxybutyryl-CoA (PubMed:11551212)',\n",
       "   'Pathway': 'Amine and polyamine metabolism; carnitine metabolism'},\n",
       "  'seq': 'MSESLHLTRNGSILEITLDRPKANAIDAKTSFEMGEVFLNFRDDPQLRVAIITGAGEKFFSAGWDLKAAAEGEAPDADFGPGGFAGLTEIFNLDKPVIAAVNGYAFGGGFELALAADFIVCADNASFALPEAKLGIVPDSGGVLRLPKILPPAIVNEMVMTGRRMGAEEALRWGIVNRVVSQAELMDNARELAQQLVNSAPLAIAALKEIYRTTSEMPVEEAYRYIRSGVLKHYPSVLHSEDAIEGPLAFAEKRDPVWKGR',\n",
       "  'mw': 28190.0},\n",
       " 'b0037': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P31552',\n",
       "  'gene': 'b0037',\n",
       "  'locus': ['b0037', 'JW0036'],\n",
       "  'synonyms': 'yaaM',\n",
       "  'properties': {'Catalytic Activity': [('4-(trimethylamino)butanoate + ATP + CoA = AMP + diphosphate + gamma-butyrobetainyl-CoA',\n",
       "     '6.2.1.48'),\n",
       "    ('ATP + CoA + crotonobetaine = AMP + crotonobetainyl-CoA + diphosphate',\n",
       "     '6.2.1.48'),\n",
       "    ('(R)-carnitine + ATP + CoA = (R)-carnitinyl-CoA + AMP + diphosphate',\n",
       "     '6.2.1.48')],\n",
       "   'Function': 'Catalyzes the transfer of CoA to carnitine, generating the initial carnitinyl-CoA needed for the CaiB reaction cycle. Also has activity toward crotonobetaine and gamma-butyrobetaine',\n",
       "   'Pathway': 'Amine and polyamine metabolism; carnitine metabolism'},\n",
       "  'seq': 'MDIIGGQHLRQMWDDLADVYGHKTALICESSGGVVNRYSYLELNQEINRTANLFYTLGIRKGDKVALHLDNCPEFIFCWFGLAKIGAIMVPINARLLCEESAWILQNSQACLLVTSAQFYPMYQQIQQEDATQLRHICLTDVALPADDGVSSFTQLKNQQPATLCYAPPLSTDDTAEILFTSGTTSRPKGVVITHYNLRFAGYYSAWQCALRDDDVYLTVMPAFHIDCQCTAAMAAFSAGATFVLVEKYSARAFWGQVQKYRATVTECIPMMIRTLMVQPPSANDQQHRLREVMFYLNLSEQEKDAFCERFGVRLLTSYGMTETIVGIIGDRPGDKRRWPSIGRVGFCYEAEIRDDHNRPLPAGEIGEICIKGIPGKTIFKEYFLNPQATAKVLEADGWLHTGDTGYRDEEDFFYFVDRRCNMIKRGGENVSCVELENIIAAHPKIQDIVVVGIKDSIRDEAIKAFVVLNEGETLSEEEFFRFCEQNMAKFKVPSYLEIRKDLPRNCSGKIIRKNLK',\n",
       "  'mw': 58559.0},\n",
       " 'b0038': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P31572',\n",
       "  'gene': 'b0038',\n",
       "  'locus': ['b0038', 'JW0037'],\n",
       "  'synonyms': 'yaaN',\n",
       "  'properties': {'Catalytic Activity': [('(R)-carnitine + crotonobetainyl-CoA = (R)-carnitinyl-CoA + crotonobetaine',\n",
       "     '2.8.3.21'),\n",
       "    ('(R)-carnitine + gamma-butyrobetainyl-CoA = (R)-carnitinyl-CoA + 4-(trimethylamino)butanoate',\n",
       "     '2.8.3.21')],\n",
       "   'Function': 'Catalyzes the reversible transfer of the CoA moiety from gamma-butyrobetainyl-CoA to L-carnitine to generate L-carnitinyl-CoA and gamma-butyrobetaine. Is also able to catalyze the reversible transfer of the CoA moiety from gamma-butyrobetainyl-CoA or L-carnitinyl-CoA to crotonobetaine to generate crotonobetainyl-CoA',\n",
       "   'Pathway': 'Amine and polyamine metabolism; carnitine metabolism'},\n",
       "  'seq': 'MDHLPMPKFGPLAGLRVVFSGIEIAGPFAGQMFAEWGAEVIWIENVAWADTIRVQPNYPQLSRRNLHALSLNIFKDEGREAFLKLMETTDIFIEASKGPAFARRGITDEVLWQHNPKLVIAHLSGFGQYGTEEYTNLPAYNTIAQAFSGYLIQNGDVDQPMPAFPYTADYFSGLTATTAALAALHKVRETGKGESIDIAMYEVMLRMGQYFMMDYFNGGEMCPRMSKGKDPYYAGCGLYKCADGYIVMELVGITQIEECFKDIGLAHLLGTPEIPEGTQLIHRIECPYGPLVEEKLDAWLATHTIAEVKERFAELNIACAKVLTVPELESNPQYVARESITQWQTMDGRTCKGPNIMPKFKNNPGQIWRGMPSHGMDTAAILKNIGYSENDIQELVSKGLAKVED',\n",
       "  'mw': 45127.0},\n",
       " 'b0040': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P31553',\n",
       "  'gene': 'b0040',\n",
       "  'locus': ['b0040', 'JW0039'],\n",
       "  'synonyms': 'yaaP',\n",
       "  'properties': {'Catalytic Activity': [('(R)-carnitine(out) + 4-(trimethylamino)butanoate(in) = (R)-carnitine(in) + 4-(trimethylamino)butanoate(out)',\n",
       "     '')],\n",
       "   'Function': 'Catalyzes the exchange of L-carnitine for gamma-butyrobetaine (PubMed:12163501, PubMed:20829798, PubMed:30846799). Can also transport D-carnitine and, with lower efficiency, acetyl-L-carnitine and glycine betaine (PubMed:12163501)',\n",
       "   'Regulation Activity': 'Inhibited by the sulfhydryl reagent N-ethylmaleimide, but not by the ionophore carbonyl cyanide m-chlorophenylhydrazone (CCCP)',\n",
       "   'Pathway': 'Amine and polyamine metabolism; carnitine metabolism'},\n",
       "  'seq': 'MKNEKRKTGIEPKVFFPPLIIVGILCWLTVRDLDAANVVINAVFSYVTNVWGWAFEWYMVVMLFGWFWLVFGPYAKKRLGNEPPEFSTASWIFMMFASCTSAAVLFWGSIEIYYYISTPPFGLEPNSTGAKELGLAYSLFHWGPLPWATYSFLSVAFAYFFFVRKMEVIRPSSTLVPLVGEKHAKGLFGTIVDNFYLVALIFAMGTSLGLATPLVTECMQWLFGIPHTLQLDAIIITCWIILNAICVACGLQKGVRIASDVRSYLSFLMLGWVFIVSGASFIMNYFTDSVGMLLMYLPRMLFYTDPIAKGGFPQGWTVFYWAWWVIYAIQMSIFLARISRGRTVRELCFGMVLGLTASTWILWTVLGSNTLLLIDKNIINIPNLIEQYGVARAIIETWAALPLSTATMWGFFILCFIATVTLVNACSYTLAMSTCREVRDGEEPPLLVRIGWSILVGIIGIVLLALGGLKPIQTAIIAGGCPLFFVNIMVTLSFIKDAKQNWKD',\n",
       "  'mw': 56587.0},\n",
       " 'b0047': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P03819',\n",
       "  'gene': 'b0047',\n",
       "  'locus': ['b0047', 'JW0046'],\n",
       "  'synonyms': 'trkC',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Pore-forming subunit of a potassium efflux system that confers protection against electrophiles. Catalyzes K(+)/H(+) antiport. Can also export rubidium, lithium and sodium',\n",
       "   'Regulation Activity': 'Inhibited by glutathione. This inhibition is increased by NADH. Activated by adducts between glutathione and electrophiles'},\n",
       "  'seq': 'MDSHTLIQALIYLGSAALIVPIAVRLGLGSVLGYLIAGCIIGPWGLRLVTDAESILHFAEIGVVLMLFIIGLELDPQRLWKLRAAVFGCGALQMVICGGLLGLFCMLLGLRWQVAELIGMTLALSSTAIAMQAMNERNLMVTQMGRSAFAVLLFQDIAAIPLVAMIPLLATSSASTTMGAFALSALKVAGALVLVVLLGRYVTRPALRFVARSGLREVFSAVALFLVFGFGLLLEEVGLSMAMGAFLAGVLLASSEYRHALESDIEPFKGLLLGLFFIGVGMSIDFGTLLENPLRIVILLLGFLIIKIAMLWLIARPLQVPNKQRRWFAVLLGQGSEFAFVVFGAAQMANVLEPEWAKSLTLAVALSMAATPILLVILNRLEQSSTEEAREADEIDEEQPRVIIAGFGRFGQITGRLLLSSGVKMVVLDHDPDHIETLRKFGMKVFYGDATRMDLLESAGAAKAEVLINAIDDPQTNLQLTEMVKEHFPHLQIIARARDVDHYIRLRQAGVEKPERETFEGALKTGRLALESLGLGPYEARERADVFRRFNIQMVEEMAMVENDTKARAAVYKRTSAMLSEIITEDREHLSLIQRHGWQGTEEGKHTGNMADEPETKPSS',\n",
       "  'mw': 67796.0},\n",
       " 'b0048': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ABQ4',\n",
       "  'gene': 'b0048',\n",
       "  'locus': ['b0048', 'JW0047'],\n",
       "  'synonyms': 'tmrA',\n",
       "  'properties': {'Catalytic Activity': [('(6S)-5,6,7,8-tetrahydrofolate + NADP(+) = 7,8-dihydrofolate + H(+) + NADPH',\n",
       "     '1.5.1.3')],\n",
       "   'Function': 'Key enzyme in folate metabolism. Catalyzes an essential reaction for de novo glycine and purine synthesis, and for DNA precursor synthesis',\n",
       "   'Pathway': 'Cofactor biosynthesis; tetrahydrofolate biosynthesis; 5,6,7,8-tetrahydrofolate from 7,8-dihydrofolate: step 1/1'},\n",
       "  'seq': 'MISLIAALAVDRVIGMENAMPWNLPADLAWFKRNTLNKPVIMGRHTWESIGRPLPGRKNIILSSQPGTDDRVTWVKSVDEAIAACGDVPEIMVIGGGRVYEQFLPKAQKLYLTHIDAEVEGDTHFPDYEPDDWESVFSEFHDADAQNSHSYCFEILERR',\n",
       "  'mw': 17999.0},\n",
       " 'b0049': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P05637',\n",
       "  'gene': 'b0049',\n",
       "  'locus': ['b0049', 'JW0048'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [(\"H2O + P(1),P(4)-bis(5'-adenosyl) tetraphosphate = 2 ADP + 2 H(+)\",\n",
       "     '3.6.1.41')],\n",
       "   'Function': \"Hydrolyzes diadenosine 5',5'''-P1,P4-tetraphosphate to yield ADP\",\n",
       "   'Regulation Activity': 'Co(2+) is a strong stimulator (100-fold increase in rate of hydrolysis). Mn(2+), Cd(2+), Ni(2+), Mg(2+) and Ca(2+) are weak stimulators; the two latter act synergistically with Co(2+)'},\n",
       "  'seq': 'MATYLIGDVHGCYDELIALLHKVEFTPGKDTLWLTGDLVARGPGSLDVLRYVKSLGDSVRLVLGNHDLHLLAVFAGISRNKPKDRLTPLLEAPDADELLNWLRRQPLLQIDEEKKLVMAHAGITPQWDLQTAKECARDVEAVLSSDSYPFFLDAMYGDMPNNWSPELRGLGRLRFITNAFTRMRFCFPNGQLDMYSKESPEEAPAPLKPWFAIPGPVAEEYSIAFGHWASLEGKGTPEGIYALDTGCCWGGTLTCLRWEDKQYFVQPSNRHKDLGEAAAS',\n",
       "  'mw': 31297.0},\n",
       " 'b0052': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P19624',\n",
       "  'gene': 'b0052',\n",
       "  'locus': ['b0052', 'JW0051'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('4-(phosphooxy)-L-threonine + NAD(+) = 3-amino-2-oxopropyl phosphate + CO2 + NADH',\n",
       "     '1.1.1.262')],\n",
       "   'Function': 'Catalyzes the NAD(P)-dependent oxidation of 4-(phosphooxy)-L-threonine (HTP) into 2-amino-3-oxo-4-(phosphooxy)butyric acid which spontaneously decarboxylates to form 3-amino-2-oxopropyl phosphate (AHAP)',\n",
       "   'Pathway': \"Cofactor biosynthesis; pyridoxine 5'-phosphate biosynthesis; pyridoxine 5'-phosphate from D-erythrose 4-phosphate: step 4/5\"},\n",
       "  'seq': 'MVKTQRVVITPGEPAGIGPDLVVQLAQREWPVELVVCADATLLTNRAAMLGLPLTLRPYSPNSPAQPQTAGTLTLLPVALRAPVTAGQLAVENGHYVVETLARACDGCLNGEFAALITGPVHKGVINDAGIPFTGHTEFFEERSQAKKVVMMLATEELRVALATTHLPLRDIADAITPALLHEVIAILHHDLRTKFGIAEPRILVCGLNPHAGEGGHMGTEEIDTIIPVLNELRAQGMKLNGPLPADTLFQPKYLDNADAVLAMYHDQGLPVLKYQGFGRGVNITLGLPFIRTSVDHGTALELAGRGKADVGSFITALNLAIKMIVNTQ',\n",
       "  'mw': 35114.0},\n",
       " 'b0061': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P08203',\n",
       "  'gene': 'b0061',\n",
       "  'locus': ['b0061', 'JW0060'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('L-ribulose 5-phosphate = D-xylulose 5-phosphate',\n",
       "     '5.1.3.4')],\n",
       "   'Function': 'Involved in the degradation of L-arabinose (PubMed:13890280). Catalyzes the interconversion of L-ribulose 5-phosphate (LRu5P) and D-xylulose 5-phosphate (D-Xu5P) via a retroaldol/aldol mechanism (carbon-carbon bond cleavage analogous to a class II aldolase reaction)',\n",
       "   'Regulation Activity': 'Inhibited by glycolohydroxamate at concentration above 0.1 mM',\n",
       "   'Pathway': 'Carbohydrate degradation; L-arabinose degradation via L-ribulose; D-xylulose 5-phosphate from L-arabinose (bacterial route): step 3/3'},\n",
       "  'seq': 'MLEDLKRQVLEANLALPKHNLVTLTWGNVSAVDRERGVFVIKPSGVDYSVMTADDMVVVSIETGEVVEGTKKPSSDTPTHRLLYQAFPSIGGIVHTHSRHATIWAQAGQSIPATGTTHADYFYGTIPCTRKMTDAEINGEYEWETGNVIVETFEKQGIDAAQMPGVLVHSHGPFAWGKNAEDAVHNAIVLEEVAYMGIFCRQLAPQLPDMQQTLLDKHYLRKHGAKAYYGQ',\n",
       "  'mw': 25519.0},\n",
       " 'b0062': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P08202',\n",
       "  'gene': 'b0062',\n",
       "  'locus': ['b0062', 'JW0061'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('beta-L-arabinopyranose = L-ribulose',\n",
       "     '5.3.1.4')],\n",
       "   'Function': 'Catalyzes the conversion of L-arabinose to L-ribulose',\n",
       "   'Pathway': 'Carbohydrate degradation; L-arabinose degradation via L-ribulose; D-xylulose 5-phosphate from L-arabinose (bacterial route): step 1/3'},\n",
       "  'seq': 'MTIFDNYEVWFVIGSQHLYGPETLRQVTQHAEHVVNALNTEAKLPCKLVLKPLGTTPDEITAICRDANYDDRCAGLVVWLHTFSPAKMWINGLTMLNKPLLQFHTQFNAALPWDSIDMDFMNLNQTAHGGREFGFIGARMRQQHAVVTGHWQDKQAHERIGSWMRQAVSKQDTRHLKVCRFGDNMREVAVTDGDKVAAQIKFGFSVNTWAVGDLVQVVNSISDGDVNALVDEYESCYTMTPATQIHGKKRQNVLEAARIELGMKRFLEQGGFHAFTTTFEDLHGLKQLPGLAVQRLMQQGYGFAGEGDWKTAALLRIMKVMSTGLQGGTSFMEDYTYHFEKGNDLVLGSHMLEVCPSIAAEEKPILDVQHLGIGGKDDPARLIFNTQTGPAIVASLIDLGDRYRLLVNCIDTVKTPHSLPKLPVANALWKAQPDLPTASEAWILAGGAHHTVFSHALNLNDMRQFAEMHDIEITVIDNDTRLPAFKDALRWNEVYYGFRR',\n",
       "  'mw': 56074.0},\n",
       " 'b0063': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P08204',\n",
       "  'gene': 'b0063',\n",
       "  'locus': ['b0063', 'JW0062'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + D-ribulose = ADP + D-ribulose 5-phosphate + H(+)',\n",
       "     '2.7.1.16'),\n",
       "    ('ATP + L-ribulose = ADP + H(+) + L-ribulose 5-phosphate', '2.7.1.16')],\n",
       "   'Pathway': 'Carbohydrate degradation; L-arabinose degradation via L-ribulose; D-xylulose 5-phosphate from L-arabinose (bacterial route): step 2/3'},\n",
       "  'seq': 'MAIAIGLDFGSDSVRALAVDCATGEEIATSVEWYPRWQKGQFCDAPNNQFRHHPRDYIESMEAALKTVLAELSVEQRAAVVGIGVDSTGSTPAPIDADGNVLALRPEFAENPNAMFVLWKDHTAVEEAEEITRLCHAPGNVDYSRYIGGIYSSEWFWAKILHVTRQDSAVAQSAASWIELCDWVPALLSGTTRPQDIRRGRCSAGHKSLWHESWGGLPPASFFDELDPILNRHLPSPLFTDTWTADIPVGTLCPEWAQRLGLPESVVISGGAFDCHMGAVGAGAQPNALVKVIGTSTCDILIADKQSVGERAVKGICGQVDGSVVPGFIGLEAGQSAFGDIYAWFGRVLGWPLEQLAAQHPELKTQINASQKQLLPALTEAWAKNPSLDHLPVVLDWFNGRRTPNANQRLKGVITDLNLATDAPLLFGGLIAATAFGARAIMECFTDQGIAVNNVMALGGIARKNQVIMQACCDVLNRPLQIVASDQCCALGAAIFAAVAAKVHADIPSAQQKMASAVEKTLQPCSEQAQRFEQLYRRYQQWAMSAEQHYLPTSAPAQAAQAVATL',\n",
       "  'mw': 61089.0},\n",
       " 'b0066': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P31548',\n",
       "  'gene': 'b0066',\n",
       "  'locus': ['b0066', 'JW0065'],\n",
       "  'synonyms': 'yabJ',\n",
       "  'properties': {'Catalytic Activity': [('ATP + H2O + thiamine(out) = ADP + H(+) + phosphate + thiamine(in)',\n",
       "     '7.6.2.15')],\n",
       "   'Function': 'Part of the ABC transporter complex ThiBPQ involved in thiamine import (PubMed:12175925). Responsible for energy coupling to the transport system (Probable)'},\n",
       "  'seq': 'MLKLTDITWLYHHLPMRFSLTVERGEQVAILGPSGAGKSTLLNLIAGFLTPASGSLTIDGVDHTTMPPSRRPVSMLFQENNLFSHLTVAQNIGLGLNPGLKLNAVQQGKMHAIARQMGIDNLMARLPGELSGGQRQRVALARCLVREQPILLLDEPFSALDPALRQEMLTLVSTSCQQQKMTLLMVSHSVEDAARIATRSVVVADGRIAWQGMTNELLSGKASASALLGITG',\n",
       "  'mw': 24999.0},\n",
       " 'b0067': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P31549',\n",
       "  'gene': 'b0067',\n",
       "  'locus': ['b0067', 'JW0066'],\n",
       "  'synonyms': 'yabK',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the ABC transporter complex ThiBPQ involved in thiamine import (PubMed:12175925). Probably responsible for the translocation of the substrate across the membrane (Probable)',\n",
       "   'Regulation Activity': 'Transport is inhibited by the sulfhydryl-specific modifier N-ethylmaleimide'},\n",
       "  'seq': 'MATRRQPLIPGWLIPGVSATTLVVAVALAAFLALWWNAPQDDWVAVWQDSYLWHVVRFSFWQAFLSALLSVIPAIFLARALYRRRFPGRLALLRLCAMTLILPVLVAVFGILSVYGRQGWLATLCQSLGLEWTFSPYGLQGILLAHVFFNLPMASRLLLQALENIPGEQRQLAAQLGMRSWHFFRFVEWPWLRRQIPPVAALIFMLCFASFATVLSLGGGPQATTIELAIYQALSYDYDPARAAMLALLQMVCCLGLVLLSQRLSKAIAPGTTLLQGWRDPDDRLHSRICDTVLIVLALLLLLPPLLAVIVDGVNRQLPEVLAQPVLWQALWTSLRIALAAGVLCVVLTMMLLWSSRELRARQKMLAGQVLEMSGMLILAMPGIVLATGFFLLLNNTIGLPQSADGIVIFTNALMAIPYALKVLENPMRDITARYSMLCQSLGIEGWSRLKVVELRALKRPLAQALAFACVLSIGDFGVVALFGNDDFRTLPFYLYQQIGSYRSQDGAVTALILLLLCFLLFTVIEKLPGRNVKTD',\n",
       "  'mw': 59533.0},\n",
       " 'b0068': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P31550',\n",
       "  'gene': 'b0068',\n",
       "  'locus': ['b0068', 'JW0067'],\n",
       "  'synonyms': 'tbpA yabL',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the ABC transporter complex ThiBPQ involved in thiamine import (PubMed:12175925). Binds thiamine, thiamine phosphate and thiamine diphosphate with high affinity (PubMed:12182833, PubMed:18177053)',\n",
       "   'Regulation Activity': 'Transport is inhibited by the sulfhydryl-specific modifier N-ethylmaleimide'},\n",
       "  'seq': 'MLKKCLPLLLLCTAPVFAKPVLTVYTYDSFAADWGPGPVVKKAFEADCNCELKLVALEDGVSLLNRLRMEGKNSKADVVLGLDNNLLDAASKTGLFAKSGVAADAVNVPGGWNNDTFVPFDYGYFAFVYDKNKLKNPPQSLKELVESDQNWRVIYQDPRTSTPGLGLLLWMQKVYGDDAPQAWQKLAKKTVTVTKGWSEAYGLFLKGESDLVLSYTTSPAYHILEEKKDNYAAANFSEGHYLQVEVAARTAASKQPELAQKFLQFMVSPAFQNAIPTGNWMYPVANVTLPAGFEKLTKPATTLEFTPAEVAAQRQAWISEWQRAVSR',\n",
       "  'mw': 36163.0},\n",
       " 'b0070': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P31675',\n",
       "  'gene': 'b0070',\n",
       "  'locus': ['b0070', 'JW0069'],\n",
       "  'synonyms': 'yabM',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Involved in the efflux of sugars. The physiological role may be the detoxification of non-metabolizable sugar analogs. Can transport IPTG, lactose and glucose. Has broad substrate specificity, with preferences for glucosides or galactosides with alkyl or aryl substituents'},\n",
       "  'seq': 'MIWIMTMARRMNGVYAAFMLVAFMMGVAGALQAPTLSLFLSREVGAQPFWIGLFYTVNAIAGIGVSLWLAKRSDSQGDRRKLIIFCCLMAIGNALLFAFNRHYLTLITCGVLLASLANTAMPQLFALAREYADNSAREVVMFSSVMRAQLSLAWVIGPPLAFMLALNYGFTVMFSIAAGIFTLSLVLIAFMLPSVARVELPSENALSMQGGWQDSNVRMLFVASTLMWTCNTMYIIDMPLWISSELGLPDKLAGFLMGTAAGLEIPAMILAGYYVKRYGKRRMMVIAVAAGVLFYTGLIFFNSRMALMTLQLFNAVFIGIVAGIGMLWFQDLMPGRAGAATTLFTNSISTGVILAGVIQGAIAQSWGHFAVYWVIAVISVVALFLTAKVKDV',\n",
       "  'mw': 42713.0},\n",
       " 'b0071': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P30126',\n",
       "  'gene': 'b0071',\n",
       "  'locus': ['b0071', 'JW0070'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(2R,3S)-3-isopropylmalate = (2S)-2-isopropylmalate',\n",
       "     '4.2.1.33')],\n",
       "   'Function': 'Catalyzes the isomerization between 2-isopropylmalate and 3-isopropylmalate, via the formation of 2-isopropylmaleate',\n",
       "   'Pathway': 'Amino-acid biosynthesis; L-leucine biosynthesis; L-leucine from 3-methyl-2-oxobutanoate: step 2/4'},\n",
       "  'seq': 'MAEKFIKHTGLVVPLDAANVDTDAIIPKQFLQKVTRTGFGAHLFNDWRFLDEKGQQPNPDFVLNFPQYQGASILLARENFGCGSSREHAPWALTDYGFKVVIAPSFADIFYGNSFNNQLLPVKLSDAEVDELFALVKANPGIHFDVDLEAQEVKAGEKTYRFTIDAFRRHCMMNGLDSIGLTLQHDDAIAAYEAKQPAFMN',\n",
       "  'mw': 22487.0},\n",
       " 'b0072': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A6A6',\n",
       "  'gene': 'b0072',\n",
       "  'locus': ['b0072', 'JW0071'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(2R,3S)-3-isopropylmalate = (2S)-2-isopropylmalate',\n",
       "     '4.2.1.33')],\n",
       "   'Function': 'Catalyzes the isomerization between 2-isopropylmalate and 3-isopropylmalate, via the formation of 2-isopropylmaleate',\n",
       "   'Pathway': 'Amino-acid biosynthesis; L-leucine biosynthesis; L-leucine from 3-methyl-2-oxobutanoate: step 2/4'},\n",
       "  'seq': 'MAKTLYEKLFDAHVVYEAENETPLLYIDRHLVHEVTSPQAFDGLRAHGRPVRQPGKTFATMDHNVSTQTKDINACGEMARIQMQELIKNCKEFGVELYDLNHPYQGIVHVMGPEQGVTLPGMTIVCGDSHTATHGAFGALAFGIGTSEVEHVLATQTLKQGRAKTMKIEVQGKAAPGITAKDIVLAIIGKTGSAGGTGHVVEFCGEAIRDLSMEGRMTLCNMAIEMGAKAGLVAPDETTFNYVKGRLHAPKGKDFDDAVAYWKTLQTDEGATFDTVVTLQAEEISPQVTWGTNPGQVISVNDNIPDPASFADPVERASAEKALAYMGLKPGIPLTEVAIDKVFIGSCTNSRIEDLRAAAEIAKGRKVAPGVQALVVPGSGPVKAQAEAEGLDKIFIEAGFEWRLPGCSMCLAMNNDRLNPGERCASTSNRNFEGRQGRGGRTHLVSPAMAAAAAVTGHFADIRNIK',\n",
       "  'mw': 49882.0},\n",
       " 'b0073': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P30125',\n",
       "  'gene': 'b0073',\n",
       "  'locus': ['b0073', 'JW5807'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(2R,3S)-3-isopropylmalate + NAD(+) = 4-methyl-2-oxopentanoate + CO2 + NADH',\n",
       "     '1.1.1.85')],\n",
       "   'Function': 'Catalyzes the oxidation of 3-carboxy-2-hydroxy-4-methylpentanoate (3-isopropylmalate) to 3-carboxy-4-methyl-2-oxopentanoate. The product decarboxylates to 4-methyl-2 oxopentanoate',\n",
       "   'Regulation Activity': 'Requires K(+) ions for optimum activity',\n",
       "   'Pathway': 'Amino-acid biosynthesis; L-leucine biosynthesis; L-leucine from 3-methyl-2-oxobutanoate: step 3/4'},\n",
       "  'seq': 'MSKNYHIAVLPGDGIGPEVMTQALKVLDAVRNRFAMRITTSHYDVGGAAIDNHGQPLPPATVEGCEQADAVLFGSVGGPKWEHLPPDQQPERGALLPLRKHFKLFSNLRPAKLYQGLEAFCPLRADIAANGFDILCVRELTGGIYFGQPKGREGSGQYEKAFDTEVYHRFEIERIARIAFESARKRRHKVTSIDKANVLQSSILWREIVNEIATEYPDVELAHMYIDNATMQLIKDPSQFDVLLCSNLFGDILSDECAMITGSMGMLPSASLNEQGFGLYEPAGGSAPDIAGKNIANPIAQILSLALLLRYSLDADDAACAIERAINRALEEGIRTGDLARGAAAVSTDEMGDIIARYVAEGV',\n",
       "  'mw': 39517.0},\n",
       " 'b0074': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P09151',\n",
       "  'gene': 'b0074',\n",
       "  'locus': ['b0074', 'JW0073'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('3-methyl-2-oxobutanoate + acetyl-CoA + H2O = (2S)-2-isopropylmalate + CoA + H(+)',\n",
       "     '2.3.3.13')],\n",
       "   'Function': 'Catalyzes the condensation of the acetyl group of acetyl-CoA with 3-methyl-2-oxobutanoate (2-oxoisovalerate) to form 3-carboxy-3-hydroxy-4-methylpentanoate (2-isopropylmalate)',\n",
       "   'Pathway': 'Amino-acid biosynthesis; L-leucine biosynthesis; L-leucine from 3-methyl-2-oxobutanoate: step 1/4'},\n",
       "  'seq': 'MSQQVIIFDTTLRDGEQALQASLSVKEKLQIALALERMGVDVMEVGFPVSSPGDFESVQTIARQVKNSRVCALARCVEKDIDVAAESLKVAEAFRIHTFIATSPMHIATKLRSTLDEVIERAIYMVKRARNYTDDVEFSCEDAGRTPIADLARVVEAAINAGATTINIPDTVGYTMPFEFAGIISGLYERVPNIDKAIISVHTHDDLGLAVGNSLAAVHAGARQVEGAMNGIGERAGNCSLEEVIMAIKVRKDILNVHTAINHQEIWRTSQLVSQICNMPIPANKAIVGSGAFAHSSGIHQDGVLKNRENYEIMTPESIGLNQIQLNLTSRSGRAAVKHRMDEMGYKESEYNLDNLYDAFLKLADKKGQVFDYDLEALAFIGKQQEEPEHFRLDYFSVQSGSNDIATAAVKLACGEEVKAEAANGNGPVDAVYQAINRITEYNVELVKYSLTAKGHGKDALGQVDIVANYNGRRFHGVGLATDIVESSAKAMVHVLNNIWRAAEVEKELQRKAQHNENNKETV',\n",
       "  'mw': 57298.0},\n",
       " 'b0077': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P00893',\n",
       "  'gene': 'b0077',\n",
       "  'locus': ['b0077', 'JW0076'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('H(+) + 2 pyruvate = (2S)-2-acetolactate + CO2',\n",
       "     '2.2.1.6')],\n",
       "   'Regulation Activity': 'Sensitive to valine inhibition',\n",
       "   'Pathway': 'Amino-acid biosynthesis; L-valine biosynthesis; L-valine from pyruvate: step 1/4'},\n",
       "  'seq': 'MEMLSGAEMVVRSLIDQGVKQVFGYPGGAVLDIYDALHTVGGIDHVLVRHEQAAVHMADGLARATGEVGVVLVTSGPGATNAITGIATAYMDSIPLVVLSGQVATSLIGYDAFQECDMVGISRPVVKHSFLVKQTEDIPQVLKKAFWLAASGRPGPVVVDLPKDILNPANKLPYVWPESVSMRSYNPTTTGHKGQIKRALQTLVAAKKPVVYVGGGAITAGCHQQLKETVEALNLPVVCSLMGLGAFPATHRQALGMLGMHGTYEANMTMHNADVIFAVGVRFDDRTTNNLAKYCPNATVLHIDIDPTSISKTVTADIPIVGDARQVLEQMLELLSQESAHQPLDEIRDWWQQIEQWRARQCLKYDTHSEKIKPQAVIETLWRLTKGDAYVTSDVGQHQMFAALYYPFDKPRRWINSGGLGTMGFGLPAALGVKMALPEETVVCVTGDGSIQMNIQELSTALQYELPVLVVNLNNRYLGMVKQWQDMIYSGRHSQSYMQSLPDFVRLAEAYGHVGIQISHPHELESKLSEALEQVRNNRLVFVDVTVDGSEHVYPMQIRGGGMDEMWLSKTERT',\n",
       "  'mw': 62984.0},\n",
       " 'b0078': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P00894',\n",
       "  'gene': 'b0078',\n",
       "  'locus': ['b0078', 'JW0077'],\n",
       "  'synonyms': 'brnP',\n",
       "  'properties': {'Catalytic Activity': [('H(+) + 2 pyruvate = (2S)-2-acetolactate + CO2',\n",
       "     '2.2.1.6')],\n",
       "   'Regulation Activity': 'Sensitive to valine inhibition',\n",
       "   'Pathway': 'Amino-acid biosynthesis; L-valine biosynthesis; L-valine from pyruvate: step 1/4'},\n",
       "  'seq': 'MRRILSVLLENESGALSRVIGLFSQRGYNIESLTVAPTDDPTLSRMTIQTVGDEKVLEQIEKQLHKLVDVLRVSELGQGAHVEREIMLVKIQASGYGRDEVKRNTEIFRGQIIDVTPSLYTVQLAGTSGKLDAFLASIRDVAKIVEVARSGVVGLSRGDKIMR',\n",
       "  'mw': 17977.0},\n",
       " 'b0084': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AD68',\n",
       "  'gene': 'b0084',\n",
       "  'locus': ['b0084', 'JW0082'],\n",
       "  'synonyms': 'pbpB',\n",
       "  'properties': {'Catalytic Activity': [('Preferential cleavage: (Ac)2-L-Lys-D-Ala-|-D-Ala. Also transpeptidation of peptidyl-alanyl moieties that are N-acyl substituents of D-alanine.',\n",
       "     '3.4.16.4')],\n",
       "   'Function': 'Essential cell division protein that catalyzes cross-linking of the peptidoglycan cell wall at the division septum (PubMed:1103132, PubMed:6450748, PubMed:9614966, PubMed:3531167, PubMed:7030331). Required for localization of FtsN (PubMed:9282742)',\n",
       "   'Regulation Activity': 'Inhibited by beta-lactam antibiotics such as penicillin, moenomycin, macarbomycin, furazlocillin and piperacillin. Antibiotics inhibit the activity by binding to the catalytic serine',\n",
       "   'Pathway': 'Cell wall biogenesis; peptidoglycan biosynthesis'},\n",
       "  'seq': 'MKAAAKTQKPKRQEEHANFISWRFALLCGCILLALAFLLGRVAWLQVISPDMLVKEGDMRSLRVQQVSTSRGMITDRSGRPLAVSVPVKAIWADPKEVHDAGGISVGDRWKALANALNIPLDQLSARINANPKGRFIYLARQVNPDMADYIKKLKLPGIHLREESRRYYPSGEVTAHLIGFTNVDSQGIEGVEKSFDKWLTGQPGERIVRKDRYGRVIEDISSTDSQAAHNLALSIDERLQALVYRELNNAVAFNKAESGSAVLVDVNTGEVLAMANSPSYNPNNLSGTPKEAMRNRTITDVFEPGSTVKPMVVMTALQRGVVRENSVLNTIPYRINGHEIKDVARYSELTLTGVLQKSSNVGVSKLALAMPSSALVDTYSRFGLGKATNLGLVGERSGLYPQKQRWSDIERATFSFGYGLMVTPLQLARVYATIGSYGIYRPLSITKVDPPVPGERVFPESIVRTVVHMMESVALPGGGGVKAAIKGYRIAIKTGTAKKVGPDGRYINKYIAYTAGVAPASQPRFALVVVINDPQAGKYYGGAVSAPVFGAIMGGVLRTMNIEPDALTTGDKNEFVINQGEGTGGRS',\n",
       "  'mw': 63877.0},\n",
       " 'b0085': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P22188',\n",
       "  'gene': 'b0085',\n",
       "  'locus': ['b0085', 'JW0083'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + meso-2,6-diaminoheptanedioate + UDP-N-acetyl-alpha-D-muramoyl-L-alanyl-D-glutamate = ADP + H(+) + phosphate + UDP-N-acetyl-alpha-D-muramoyl-L-alanyl-gamma-D-glutamyl-meso-2,6-diaminoheptanedioate',\n",
       "     '6.3.2.13')],\n",
       "   'Function': 'Catalyzes the addition of meso-diaminopimelic acid to the nucleotide precursor UDP-N-acetylmuramoyl-L-alanyl-D-glutamate (UMAG) in the biosynthesis of bacterial cell-wall peptidoglycan. Is also able to use many meso-diaminopimelate analogs as substrates, although much less efficiently, but not L-lysine',\n",
       "   'Regulation Activity': 'Activated by potassium phosphate',\n",
       "   'Pathway': 'Cell wall biogenesis; peptidoglycan biosynthesis'},\n",
       "  'seq': 'MADRNLRDLLAPWVPDAPSRALREMTLDSRVAAAGDLFVAVVGHQADGRRYIPQAIAQGVAAIIAEAKDEATDGEIREMHGVPVIYLSQLNERLSALAGRFYHEPSDNLRLVGVTGTNGKTTTTQLLAQWSQLLGEISAVMGTVGNGLLGKVIPTENTTGSAVDVQHELAGLVDQGATFCAMEVSSHGLVQHRVAALKFAASVFTNLSRDHLDYHGDMEHYEAAKWLLYSEHHCGQAIINADDEVGRRWLAKLPDAVAVSMEDHINPNCHGRWLKATEVNYHDSGATIRFSSSWGDGEIESHLMGAFNVSNLLLALATLLALGYPLADLLKTAARLQPVCGRMEVFTAPGKPTVVVDYAHTPDALEKALQAARLHCAGKLWCVFGCGGDRDKGKRPLMGAIAEEFADVAVVTDDNPRTEEPRAIINDILAGMLDAGHAKVMEGRAEAVTCAVMQAKENDVVLVAGKGHEDYQIVGNQRLDYSDRVTVARLLGVIA',\n",
       "  'mw': 53344.0},\n",
       " 'b0086': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P11880',\n",
       "  'gene': 'b0086',\n",
       "  'locus': ['b0086', 'JW0084'],\n",
       "  'synonyms': 'mra',\n",
       "  'properties': {'Catalytic Activity': [('ATP + D-alanyl-D-alanine + UDP-N-acetyl-alpha-D-muramoyl-L-alanyl-gamma-D-glutamyl-meso-2,6-diaminoheptanedioate = ADP + H(+) + phosphate + UDP-N-acetyl-alpha-D-muramoyl-L-alanyl-gamma-D-glutamyl-meso-2,6-diaminopimeloyl-D-alanyl-D-alanine',\n",
       "     '6.3.2.10')],\n",
       "   'Function': 'Involved in cell wall formation. Catalyzes the final step in the synthesis of UDP-N-acetylmuramoyl-pentapeptide, the precursor of murein',\n",
       "   'Pathway': 'Cell wall biogenesis; peptidoglycan biosynthesis'},\n",
       "  'seq': 'MISVTLSQLTDILNGELQGADITLDAVTTDTRKLTPGCLFVALKGERFDAHDFADQAKAGGAGALLVSRPLDIDLPQLIVKDTRLAFGELAAWVRQQVPARVVALTGSSGKTSVKEMTAAILSQCGNTLYTAGNLNNDIGVPMTLLRLTPEYDYAVIELGANHQGEIAWTVSLTRPEAALVNNLAAAHLEGFGSLAGVAKAKGEIFSGLPENGIAIMNADNNDWLNWQSVIGSRKVWRFSPNAANSDFTATNIHVTSHGTEFTLQTPTGSVDVLLPLPGRHNIANALAAAALSMSVGATLDAIKAGLANLKAVPGRLFPIQLAENQLLLDDSYNANVGSMTAAVQVLAEMPGYRVLVVGDMAELGAESEACHVQVGEAAKAAGIDRVLSVGKQSHAISTASGVGEHFADKTALITRLKLLIAEQQVITILVKGSRSAAMEEVVRALQENGTC',\n",
       "  'mw': 47447.0},\n",
       " 'b0087': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A6W3',\n",
       "  'gene': 'b0087',\n",
       "  'locus': ['b0087', 'JW0085'],\n",
       "  'synonyms': 'murX',\n",
       "  'properties': {'Catalytic Activity': [('di-trans,octa-cis-undecaprenyl phosphate + UDP-N-acetyl-alpha-D-muramoyl-L-alanyl-gamma-D-glutamyl-meso-2,6-diaminopimeloyl-D-alanyl-D-alanine = di-trans-octa-cis-undecaprenyl diphospho-N-acetyl-alpha-D-muramoyl-L-alanyl-D-glutamyl-meso-2,6-diaminopimeloyl-D-alanyl-D-alanine + UMP',\n",
       "     '2.7.8.13')],\n",
       "   'Function': 'Catalyzes the initial step of the lipid cycle reactions in the biosynthesis of the cell wall peptidoglycan: transfers peptidoglycan precursor phospho-MurNAc-pentapeptide from UDP-MurNAc-pentapeptide onto the lipid carrier undecaprenyl phosphate, yielding undecaprenyl-pyrophosphoryl-MurNAc-pentapeptide, known as lipid I',\n",
       "   'Pathway': 'Cell wall biogenesis; peptidoglycan biosynthesis'},\n",
       "  'seq': 'MLVWLAEHLVKYYSGFNVFSYLTFRAIVSLLTALFISLWMGPRMIAHLQKLSFGQVVRNDGPESHFSKRGTPTMGGIMILTAIVISVLLWAYPSNPYVWCVLVVLVGYGVIGFVDDYRKVVRKDTKGLIARWKYFWMSVIALGVAFALYLAGKDTPATQLVVPFFKDVMPQLGLFYILLAYFVIVGTGNAVNLTDGLDGLAIMPTVFVAGGFALVAWATGNMNFASYLHIPYLRHAGELVIVCTAIVGAGLGFLWFNTYPAQVFMGDVGSLALGGALGIIAVLLRQEFLLVIMGGVFVVETLSVILQVGSFKLRGQRIFRMAPIHHHYELKGWPEPRVIVRFWIISLMLVLIGLATLKVR',\n",
       "  'mw': 39875.0},\n",
       " 'b0088': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P14900',\n",
       "  'gene': 'b0088',\n",
       "  'locus': ['b0088', 'JW0086'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + D-glutamate + UDP-N-acetyl-alpha-D-muramoyl-L-alanine = ADP + H(+) + phosphate + UDP-N-acetyl-alpha-D-muramoyl-L-alanyl-D-glutamate',\n",
       "     '6.3.2.9')],\n",
       "   'Function': 'Cell wall formation (Probable). Catalyzes the addition of glutamate to the nucleotide precursor UDP-N-acetylmuramoyl-L-alanine (UMA) (PubMed:1765076)',\n",
       "   'Pathway': 'Cell wall biogenesis; peptidoglycan biosynthesis'},\n",
       "  'seq': 'MADYQGKNVVIIGLGLTGLSCVDFFLARGVTPRVMDTRMTPPGLDKLPEAVERHTGSLNDEWLMAADLIVASPGIALAHPSLSAAADAGIEIVGDIELFCREAQAPIVAITGSNGKSTVTTLVGEMAKAAGVNVGVGGNIGLPALMLLDDECELYVLELSSFQLETTSSLQAVAATILNVTEDHMDRYPFGLQQYRAAKLRIYENAKVCVVNADDALTMPIRGADERCVSFGVNMGDYHLNHQQGETWLRVKGEKVLNVKEMKLSGQHNYTNALAALALADAAGLPRASSLKALTTFTGLPHRFEVVLEHNGVRWINDSKATNVGSTEAALNGLHVDGTLHLLLGGDGKSADFSPLARYLNGDNVRLYCFGRDGAQLAALRPEVAEQTETMEQAMRLLAPRVQPGDMVLLSPACASLDQFKNFEQRGNEFARLAKELG',\n",
       "  'mw': 46974.0},\n",
       " 'b0090': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P17443',\n",
       "  'gene': 'b0090',\n",
       "  'locus': ['b0090', 'JW0088'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('di-trans-octa-cis-undecaprenyl diphospho-N-acetyl-alpha-D-muramoyl-L-alanyl-D-glutamyl-meso-2,6-diaminopimeloyl-D-alanyl-D-alanine + UDP-N-acetyl-alpha-D-glucosamine = di-trans-octa-cis-undecaprenyl diphospho-[N-acetyl-alpha-D-glucosaminyl-(1->4)]-N-acetyl-alpha-D-muramoyl-L-alanyl-D-glutamyl-meso-2,6-diaminopimeloyl-D-alanyl-D-alanine + H(+) + UDP',\n",
       "     '2.4.1.227')],\n",
       "   'Function': 'Cell wall formation (PubMed:1649817). Catalyzes the transfer of a GlcNAc subunit on undecaprenyl-pyrophosphoryl-MurNAc-pentapeptide (lipid intermediate I) to form undecaprenyl-pyrophosphoryl-MurNAc-(pentapeptide)GlcNAc (lipid intermediate II) (PubMed:1649817, PubMed:12538870). Strongly prefers UDP to CDP, GDP, ADP and TDP (PubMed:12538870)',\n",
       "   'Pathway': 'Cell wall biogenesis; peptidoglycan biosynthesis'},\n",
       "  'seq': 'MSGQGKRLMVMAGGTGGHVFPGLAVAHHLMAQGWQVRWLGTADRMEADLVPKHGIEIDFIRISGLRGKGIKALIAAPLRIFNAWRQARAIMKAYKPDVVLGMGGYVSGPGGLAAWSLGIPVVLHEQNGIAGLTNKWLAKIATKVMQAFPGAFPNAEVVGNPVRTDVLALPLPQQRLAGREGPVRVLVVGGSQGARILNQTMPQVAAKLGDSVTIWHQSGKGSQQSVEQAYAEAGQPQHKVTEFIDDMAAAYAWADVVVCRSGALTVSEIAAAGLPALFVPFQHKDRQQYWNALPLEKAGAAKIIEQPQLSVDAVANTLAGWSRETLLTMAERARAASIPDATERVANEVSRVARA',\n",
       "  'mw': 37815.0},\n",
       " 'b0091': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P17952',\n",
       "  'gene': 'b0091',\n",
       "  'locus': ['b0091', 'JW0089'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + L-alanine + UDP-N-acetyl-alpha-D-muramate = ADP + H(+) + phosphate + UDP-N-acetyl-alpha-D-muramoyl-L-alanine',\n",
       "     '6.3.2.8')],\n",
       "   'Function': 'Cell wall formation',\n",
       "   'Pathway': 'Cell wall biogenesis; peptidoglycan biosynthesis'},\n",
       "  'seq': 'MNTQQLAKLRSIVPEMRRVRHIHFVGIGGAGMGGIAEVLANEGYQISGSDLAPNPVTQQLMNLGATIYFNHRPENVRDASVVVVSSAISADNPEIVAAHEARIPVIRRAEMLAELMRFRHGIAIAGTHGKTTTTAMVSSIYAEAGLDPTFVNGGLVKAAGVHARLGHGRYLIAEADESDASFLHLQPMVAIVTNIEADHMDTYQGDFENLKQTFINFLHNLPFYGRAVMCVDDPVIRELLPRVGRQTTTYGFSEDADVRVEDYQQIGPQGHFTLLRQDKEPMRVTLNAPGRHNALNAAAAVAVATEEGIDDEAILRALESFQGTGRRFDFLGEFPLEPVNGKSGTAMLVDDYGHHPTEVDATIKAARAGWPDKNLVMLFQPHRFTRTRDLYDDFANVLTQVDTLLMLEVYPAGEAPIPGADSRSLCRTIRGRGKIDPILVPDPARVAEMLAPVLTGNDLILVQGAGNIGKIARSLAEIKLKPQTPEEEQHD',\n",
       "  'mw': 53626.0},\n",
       " 'b0092': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P07862',\n",
       "  'gene': 'b0092',\n",
       "  'locus': ['b0092', 'JW0090'],\n",
       "  'synonyms': 'ddl',\n",
       "  'properties': {'Catalytic Activity': [('ATP + 2 D-alanine = ADP + D-alanyl-D-alanine + H(+) + phosphate',\n",
       "     '6.3.2.4')],\n",
       "   'Function': 'Cell wall formation',\n",
       "   'Pathway': 'Cell wall biogenesis; peptidoglycan biosynthesis'},\n",
       "  'seq': 'MTDKIAVLLGGTSAEREVSLNSGAAVLAGLREGGIDAYPVDPKEVDVTQLKSMGFQKVFIALHGRGGEDGTLQGMLELMGLPYTGSGVMASALSMDKLRSKLLWQGAGLPVAPWVALTRAEFEKGLSDKQLAEISALGLPVIVKPSREGSSVGMSKVVAENALQDALRLAFQHDEEVLIEKWLSGPEFTVAILGEEILPSIRIQPSGTFYDYEAKYLSDETQYFCPAGLEASQEANLQALVLKAWTTLGCKGWGRIDVMLDSDGQFYLLEANTSPGMTSHSLVPMAARQAGMSFSQLVVRILELAD',\n",
       "  'mw': 32840.0},\n",
       " 'b0096': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A725',\n",
       "  'gene': 'b0096',\n",
       "  'locus': ['b0096', 'JW0094'],\n",
       "  'synonyms': 'asmB envA',\n",
       "  'properties': {'Catalytic Activity': [('a UDP-3-O-[(3R)-3-hydroxyacyl]-N-acetyl-alpha-D-glucosamine + H2O = a UDP-3-O-[(3R)-3-hydroxyacyl]-alpha-D-glucosamine + acetate',\n",
       "     '3.5.1.108')],\n",
       "   'Function': 'Catalyzes the hydrolysis of UDP-3-O-myristoyl-N-acetylglucosamine to form UDP-3-O-myristoylglucosamine and acetate, the committed step in lipid A biosynthesis',\n",
       "   'Regulation Activity': 'Regulation occurs at the protein level, via degradation of LpxC by the FtsH protease (PubMed:10048027, PubMed:16420369, PubMed:21193611, PubMed:23417489). Degradation is growth rate-dependent. LpxC is degraded rapidly during slow growth, probably to avoid toxic overproduction of lipopolysaccharides, but is highly stable under optimal growth conditions (PubMed:23417489). Increased amounts of LpxC are made under conditions that reduce the lipid A content of cells (PubMed:8824222). Inhibited by metal chelators such as EDTA and dipicolinic acid (DPA) and by high concentrations of Zn(2+) (PubMed:10026271)',\n",
       "   'Pathway': 'Glycolipid biosynthesis; lipid IV(A) biosynthesis; lipid IV(A) from (3R)-3-hydroxytetradecanoyl-[acyl-carrier-protein] and UDP-N-acetyl-alpha-D-glucosamine: step 2/6'},\n",
       "  'seq': 'MIKQRTLKRIVQATGVGLHTGKKVTLTLRPAPANTGVIYRRTDLNPPVDFPADAKSVRDTMLCTCLVNEHDVRISTVEHLNAALAGLGIDNIVIEVNAPEIPIMDGSAAPFVYLLLDAGIDELNCAKKFVRIKETVRVEDGDKWAEFKPYNGFSLDFTIDFNHPAIDSSNQRYAMNFSADAFMRQISRARTFGFMRDIEYLQSRGLCLGGSFDCAIVVDDYRVLNEDGLRFEDEFVRHKMLDAIGDLFMCGHNIIGAFTAYKSGHALNNKLLQAVLAKQEAWEYVTFQDDAELPLAFKAPSAVLA',\n",
       "  'mw': 33956.0},\n",
       " 'b0099': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P08337',\n",
       "  'gene': 'b0099',\n",
       "  'locus': ['b0099', 'JW0097'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('8-oxo-dGTP + H2O = 8-oxo-dGMP + diphosphate + H(+)',\n",
       "     '3.6.1.55'),\n",
       "    ('8-oxo-GTP + H2O = 8-oxo-GMP + diphosphate + H(+)', ''),\n",
       "    ('8-oxo-dGDP + H2O = 8-oxo-dGMP + H(+) + phosphate', ''),\n",
       "    ('8-oxo-GDP + H2O = 8-oxo-GMP + H(+) + phosphate', '')],\n",
       "   'Function': 'Specifically hydrolyzes both 8-oxo-deoxyguanosine triphosphate (8-oxo-dGTP) and 8-oxo-guanosine triphosphate (8-oxo-GTP) to the related monophosphates, thereby cleaning up the nucleotide pools and preventing misincorporation of 8-oxoGua into DNA and RNA (PubMed:1309939, PubMed:9311918, PubMed:15850400). It prevents replicational errors by removing an oxidatively damaged form of guanine (8-oxo-dGTP) from DNA and the nucleotide pool (PubMed:1309939). 8-oxo-dGTP can be inserted opposite dA and dC residues of template DNA with almost equal efficiency thus leading to A.T to G.C transversions (PubMed:1309939). MutT may also ensure transcriptional fidelity, removing 8-oxo-GTP from the ribonucleotide triphosphate pool (PubMed:9311918). However, due to the lower efficiency of RNA polymerase 8-oxo-GTP incorporation, MutT is probably not a major contributor to transcriptional fidelity (PubMed:25294823). It also hydrolyzes 8-oxo-dGDP and 8-oxo-GDP to their monophosphate form (PubMed:15850400). In vitro, can also use dGTP, dGDP and other various nucleoside di- and triphosphates, with much lower efficiency (PubMed:1851162, PubMed:1309939, PubMed:15850400, PubMed:23481913). Works cooperatively with MutM and MutY to prevent accumulation in the DNA of oxidized guanine residues (PubMed:7739614)'},\n",
       "  'seq': 'MKKLQIAVGIIRNENNEIFITRRAADAHMANKLEFPGGKIEMGETPEQAVVRELQEEVGITPQHFSLFEKLEYEFPDRHITLWFWLVERWEGEPWGKEGQPGEWMSLVGLNADDFPPANEPVIAKLKRL',\n",
       "  'mw': 14927.0},\n",
       " 'b0103': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A6I9',\n",
       "  'gene': 'b0103',\n",
       "  'locus': ['b0103', 'JW0100'],\n",
       "  'synonyms': 'yacE',\n",
       "  'properties': {'Catalytic Activity': [(\"3'-dephospho-CoA + ATP = ADP + CoA + H(+)\",\n",
       "     '2.7.1.24')],\n",
       "   'Function': \"Catalyzes the phosphorylation of the 3'-hydroxyl group of dephosphocoenzyme A to form coenzyme A\",\n",
       "   'Pathway': 'Cofactor biosynthesis; coenzyme A biosynthesis; CoA from (R)-pantothenate: step 5/5'},\n",
       "  'seq': 'MRYIVALTGGIGSGKSTVANAFADLGINVIDADIIARQVVEPGAPALHAIADHFGANMIAADGTLQRRALRERIFANPEEKNWLNALLHPLIQQETQHQIQQATSPYVLWVVPLLVENSLYKKANRVLVVDVSPETQLKRTMQRDDVTREHVEQILAAQATREARLAVADDVIDNNGAPDAIASDVARLHAHYLQLASQFVSQEKP',\n",
       "  'mw': 22622.0},\n",
       " 'b0104': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P60560',\n",
       "  'gene': 'b0104',\n",
       "  'locus': ['b0104', 'JW0101'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('IMP + NADP(+) + NH4(+) = GMP + 2 H(+) + NADPH',\n",
       "     '1.7.1.7')],\n",
       "   'Function': 'Catalyzes the irreversible NADPH-dependent deamination of GMP to IMP. It functions in the conversion of nucleobase, nucleoside and nucleotide derivatives of G to A nucleotides, and in maintaining the intracellular balance of A and G nucleotides'},\n",
       "  'seq': 'MRIEEDLKLGFKDVLIRPKRSTLKSRSDVELERQFTFKHSGQSWSGVPIIAANMDTVGTFSMASALASFDILTAVHKHYSVEEWQAFINNSSADVLKHVMVSTGTSDADFEKTKQILDLNPALNFVCIDVANGYSEHFVQFVAKAREAWPTKTICAGNVVTGEMCEELILSGADIVKVGIGPGSVCTTRVKTGVGYPQLSAVIECADAAHGLGGMIVSDGGCTTPGDVAKAFGGGADFVMLGGMLAGHEESGGRIVEENGEKFMLFYGMSSESAMKRHVGGVAEYRAAEGKTVKLPLRGPVENTARDILGGLRSACTYVGASRLKELTKRTTFIRVQEQENRIFNNL',\n",
       "  'mw': 37384.0},\n",
       " 'b0109': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P30011',\n",
       "  'gene': 'b0109',\n",
       "  'locus': ['b0109', 'JW0105'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('CO2 + diphosphate + nicotinate beta-D-ribonucleotide = 5-phospho-alpha-D-ribose 1-diphosphate + 2 H(+) + quinolinate',\n",
       "     '2.4.2.19')],\n",
       "   'Function': 'Involved in the catabolism of quinolinic acid (QA)',\n",
       "   'Pathway': 'Cofactor biosynthesis; NAD(+) biosynthesis; nicotinate D-ribonucleotide from quinolinate: step 1/1'},\n",
       "  'seq': 'MPPRRYNPDTRRDELLERINLDIPGAVAQALREDLGGTVDANNDITAKLLPENSRSHATVITRENGVFCGKRWVEEVFIQLAGDDVTIIWHVDDGDVINANQSLFELEGPSRVLLTGERTALNFVQTLSGVASKVRHYVELLEGTNTQLLDTRKTLPGLRSALKYAVLCGGGANHRLGLSDAFLIKENHIIASGSVRQAVEKASWLHPDAPVEVEVENLEELDEALKAGADIIMLDNFETEQMREAVKRTNGKALLEVSGNVTDKTLREFAETGVDFISVGALTKHVQALDLSMRFR',\n",
       "  'mw': 32762.0},\n",
       " 'b0110': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P13016',\n",
       "  'gene': 'b0110',\n",
       "  'locus': ['b0110', 'JW0106'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('Hydrolyzes the link between N-acetylmuramoyl residues and L-amino acid residues in certain cell-wall glycopeptides.',\n",
       "     '3.5.1.28')],\n",
       "   'Function': 'Involved in cell wall peptidoglycan recycling (PubMed:19309146). Specifically cleaves the amide bond between the lactyl group of N-acetylmuramic acid and the alpha-amino group of the L-alanine in degradation products containing an anhydro N-acetylmuramyl moiety (PubMed:19309146). Is also involved in beta-lactamase induction (PubMed:2691840, PubMed:2607970)'},\n",
       "  'seq': 'MLLEQGWLVGARRVPSPHYDCRPDDETPTLLVVHNISLPPGEFGGPWIDALFTGTIDPQAHPFFAEIAHLRVSAHCLIRRDGEIVQYVPFDKRAWHAGVSQYQGRERCNDFSIGIELEGTDTLAYTDAQYQQLAAVTRALIDCYPDIAKNMTGHCDIAPDRKTDPGPAFDWARFRVLVSKETT',\n",
       "  'mw': 20536.0},\n",
       " 'b0112': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P15993',\n",
       "  'gene': 'b0112',\n",
       "  'locus': ['b0112', 'JW0108'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('H(+)(in) + L-phenylalanine(in) = H(+)(out) + L-phenylalanine(out)',\n",
       "     ''),\n",
       "    ('H(+)(in) + L-tryptophan(in) = H(+)(out) + L-tryptophan(out)', ''),\n",
       "    ('H(+)(in) + L-tyrosine(in) = H(+)(out) + L-tyrosine(out)', '')],\n",
       "   'Function': 'Permease that is involved in the active transport across the cytoplasmic membrane of all three aromatic amino acids, phenylalanine, tyrosine and tryptophan',\n",
       "   'Regulation Activity': 'Strong, mutual inhibition of uptake by tyrosine, phenylalanine, and tryptophan. Transport is also inhibited by the aromatic analogs p-fluorophenylalanine, beta-2-thienylalanine and 5-methyltryptophan'},\n",
       "  'seq': 'MMEGQQHGEQLKRGLKNRHIQLIALGGAIGTGLFLGSASVIQSAGPGIILGYAIAGFIAFLIMRQLGEMVVEEPVAGSFSHFAYKYWGSFAGFASGWNYWVLYVLVAMAELTAVGKYIQFWYPEIPTWVSAAVFFVVINAINLTNVKVFGEMEFWFAIIKVIAVVAMIIFGGWLLFSGNGGPQATVSNLWDQGGFLPHGFTGLVMMMAIIMFSFGGLELVGITAAEADNPEQSIPKATNQVIYRILIFYIGSLAVLLSLMPWTRVTADTSPFVLIFHELGDTFVANALNIVVLTAALSVYNSCVYCNSRMLFGLAQQGNAPKALASVDKRGVPVNTILVSALVTALCVLINYLAPESAFGLLMALVVSALVINWAMISLAHMKFRRAKQEQGVVTRFPALLYPLGNWICLLFMAAVLVIMLMTPGMAISVYLIPVWLIVLGIGYLFKEKTAKAVKAH',\n",
       "  'mw': 49690.0},\n",
       " 'b0114': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AFG8',\n",
       "  'gene': 'b0114',\n",
       "  'locus': ['b0114', 'JW0110'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(R)-N(6)-lipoyl-L-lysyl-[dihydrolipoyllysine-residue acetyltransferase] + H(+) + pyruvate = (R)-N(6)-(S(8)-acetyldihydrolipoyl)-L-lysyl-[dihydrolipoyllysine-residue acetyltransferase] + CO2',\n",
       "     '1.2.4.1')],\n",
       "   'Function': 'Component of the pyruvate dehydrogenase (PDH) complex, that catalyzes the overall conversion of pyruvate to acetyl-CoA and CO(2)'},\n",
       "  'seq': 'MSERFPNDVDPIETRDWLQAIESVIREEGVERAQYLIDQLLAEARKGGVNVAAGTGISNYINTIPVEEQPEYPGNLELERRIRSAIRWNAIMTVLRASKKDLELGGHMASFQSSATIYDVCFNHFFRARNEQDGGDLVYFQGHISPGVYARAFLEGRLTQEQLDNFRQEVHGNGLSSYPHPKLMPEFWQFPTVSMGLGPIGAIYQAKFLKYLEHRGLKDTSKQTVYAFLGDGEMDEPESKGAITIATREKLDNLVFVINCNLQRLDGPVTGNGKIINELEGIFEGAGWNVIKVMWGSRWDELLRKDTSGKLIQLMNETVDGDYQTFKSKDGAYVREHFFGKYPETAALVADWTDEQIWALNRGGHDPKKIYAAFKKAQETKGKATVILAHTIKGYGMGDAAEGKNIAHQVKKMNMDGVRHIRDRFNVPVSDADIEKLPYITFPEGSEEHTYLHAQRQKLHGYLPSRQPNFTEKLELPSLQDFGALLEEQSKEISTTIAFVRALNVMLKNKSIKDRLVPIIADEARTFGMEGLFRQIGIYSPNGQQYTPQDREQVAYYKEDEKGQILQEGINELGAGCSWLAAATSYSTNNLPMIPFYIYYSMFGFQRIGDLCWAAGDQQARGFLIGGTSGRTTLNGEGLQHEDGHSHIQSLTIPNCISYDPAYAYEVAVIMHDGLERMYGEKQENVYYYITTLNENYHMPAMPEGAEEGIRKGIYKLETIEGSKGKVQLLGSGSILRHVREAAEILAKDYGVGSDVYSVTSFTELARDGQDCERWNMLHPLETPRVPYIAQVMNDAPAVASTDYMKLFAEQVRTYVPADDYRVLGTDGFGRSDSRENLRHHFEVDASYVVVAALGELAKRGEIDKKVVADAIAKFNIDADKVNPRLA',\n",
       "  'mw': 99668.0},\n",
       " 'b0115': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P06959',\n",
       "  'gene': 'b0115',\n",
       "  'locus': ['b0115', 'JW0111'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(R)-N(6)-dihydrolipoyl-L-lysyl-[protein] + acetyl-CoA = (R)-N(6)-(S(8)-acetyldihydrolipoyl)-L-lysyl-[protein] + CoA',\n",
       "     '2.3.1.12')],\n",
       "   'Function': 'The pyruvate dehydrogenase complex catalyzes the overall conversion of pyruvate to acetyl-CoA and CO(2). It contains multiple copies of three enzymatic components: pyruvate dehydrogenase (E1), dihydrolipoamide acetyltransferase (E2) and lipoamide dehydrogenase (E3)'},\n",
       "  'seq': 'MAIEIKVPDIGADEVEITEILVKVGDKVEAEQSLITVEGDKASMEVPSPQAGIVKEIKVSVGDKTQTGALIMIFDSADGAADAAPAQAEEKKEAAPAAAPAAAAAKDVNVPDIGSDEVEVTEILVKVGDKVEAEQSLITVEGDKASMEVPAPFAGTVKEIKVNVGDKVSTGSLIMVFEVAGEAGAAAPAAKQEAAPAAAPAPAAGVKEVNVPDIGGDEVEVTEVMVKVGDKVAAEQSLITVEGDKASMEVPAPFAGVVKELKVNVGDKVKTGSLIMIFEVEGAAPAAAPAKQEAAAPAPAAKAEAPAAAPAAKAEGKSEFAENDAYVHATPLIRRLAREFGVNLAKVKGTGRKGRILREDVQAYVKEAIKRAEAAPAATGGGIPGMLPWPKVDFSKFGEIEEVELGRIQKISGANLSRNWVMIPHVTHFDKTDITELEAFRKQQNEEAAKRKLDVKITPVVFIMKAVAAALEQMPRFNSSLSEDGQRLTLKKYINIGVAVDTPNGLVVPVFKDVNKKGIIELSRELMTISKKARDGKLTAGEMQGGCFTISSIGGLGTTHFAPIVNAPEVAILGVSKSAMEPVWNGKEFVPRLMLPISLSFDHRVIDGADGARFITIINNTLSDIRRLVM',\n",
       "  'mw': 66096.0},\n",
       " 'b0116': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A9P0',\n",
       "  'gene': 'b0116',\n",
       "  'locus': ['b0116', 'JW0112'],\n",
       "  'synonyms': 'lpd',\n",
       "  'properties': {'Catalytic Activity': [('(R)-N(6)-dihydrolipoyl-L-lysyl-[protein] + NAD(+) = (R)-N(6)-lipoyl-L-lysyl-[protein] + H(+) + NADH',\n",
       "     '1.8.1.4')],\n",
       "   'Function': 'Lipoamide dehydrogenase is a component of the glycine cleavage system as well as of the alpha-ketoacid dehydrogenase complexes'},\n",
       "  'seq': 'MSTEIKTQVVVLGAGPAGYSAAFRCADLGLETVIVERYNTLGGVCLNVGCIPSKALLHVAKVIEEAKALAEHGIVFGEPKTDIDKIRTWKEKVINQLTGGLAGMAKGRKVKVVNGLGKFTGANTLEVEGENGKTVINFDNAIIAAGSRPIQLPFIPHEDPRIWDSTDALELKEVPERLLVMGGGIIGLEMGTVYHALGSQIDVVEMFDQVIPAADKDIVKVFTKRISKKFNLMLETKVTAVEAKEDGIYVTMEGKKAPAEPQRYDAVLVAIGRVPNGKNLDAGKAGVEVDDRGFIRVDKQLRTNVPHIFAIGDIVGQPMLAHKGVHEGHVAAEVIAGKKHYFDPKVIPSIAYTEPEVAWVGLTEKEAKEKGISYETATFPWAASGRAIASDCADGMTKLIFDKESHRVIGGAIVGTNGGELLGEIGLAIEMGCDAEDIALTIHAHPTLHESVGLAAEVFEGSITDLPNPKAKKK',\n",
       "  'mw': 50688.0},\n",
       " 'b0118': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P36683',\n",
       "  'gene': 'b0118',\n",
       "  'locus': ['b0118', 'JW0114'],\n",
       "  'synonyms': 'yacI yacJ',\n",
       "  'properties': {'Catalytic Activity': [('citrate = D-threo-isocitrate',\n",
       "     '4.2.1.3'),\n",
       "    ('(2S,3R)-3-hydroxybutane-1,2,3-tricarboxylate = 2-methyl-cis-aconitate + H2O',\n",
       "     '4.2.1.99')],\n",
       "   'Function': \"Involved in the catabolism of short chain fatty acids (SCFA) via the tricarboxylic acid (TCA)(acetyl degradation route) and the 2-methylcitrate cycle I (propionate degradation route). Catalyzes the reversible isomerization of citrate to isocitrate via cis-aconitate. Also catalyzes the hydration of 2-methyl-cis-aconitate to yield (2R,3S)-2-methylisocitrate. The apo form of AcnB functions as a RNA-binding regulatory protein. During oxidative stress inactive AcnB apo-enzyme without iron sulfur clusters binds the acnB mRNA 3' UTRs (untranslated regions), stabilizes acnB mRNA and increases AcnB synthesis, thus mediating a post-transcriptional positive autoregulatory switch. AcnB also decreases the stability of the sodA transcript\",\n",
       "   'Pathway': 'Carbohydrate metabolism; tricarboxylic acid cycle; isocitrate from oxaloacetate: step 2/2'},\n",
       "  'seq': 'MLEEYRKHVAERAAEGIAPKPLDANQMAALVELLKNPPAGEEEFLLDLLTNRVPPGVDEAAYVKAGFLAAIAKGEAKSPLLTPEKAIELLGTMQGGYNIHPLIDALDDAKLAPIAAKALSHTLLMFDNFYDVEEKAKAGNEYAKQVMQSWADAEWFLNRPALAEKLTVTVFKVTGETNTDDLSPAPDAWSRPDIPLHALAMLKNAREGIEPDQPGVVGPIKQIEALQQKGFPLAYVGDVVGTGSSRKSATNSVLWFMGDDIPHVPNKRGGGLCLGGKIAPIFFNTMEDAGALPIEVDVSNLNMGDVIDVYPYKGEVRNHETGELLATFELKTDVLIDEVRAGGRIPLIIGRGLTTKAREALGLPHSDVFRQAKDVAESDRGFSLAQKMVGRACGVKGIRPGAYCEPKMTSVGSQDTTGPMTRDELKDLACLGFSADLVMQSFCHTAAYPKPVDVNTHHTLPDFIMNRGGVSLRPGDGVIHSWLNRMLLPDTVGTGGDSHTRFPIGISFPAGSGLVAFAAATGVMPLDMPESVLVRFKGKMQPGITLRDLVHAIPLYAIKQGLLTVEKKGKKNIFSGRILEIEGLPDLKVEQAFELTDASAERSAAGCTIKLNKEPIIEYLNSNIVLLKWMIAEGYGDRRTLERRIQGMEKWLANPELLEADADAEYAAVIDIDLADIKEPILCAPNDPDDARPLSAVQGEKIDEVFIGSCMTNIGHFRAAGKLLDAHKGQLPTRLWVAPPTRMDAAQLTEEGYYSVFGKSGARIEIPGCSLCMGNQARVADGATVVSTSTRNFPNRLGTGANVFLASAELAAVAALIGKLPTPEEYQTYVAQVDKTAVDTYRYLNFNQLSQYTEKADGVIFQTAV',\n",
       "  'mw': 93498.0},\n",
       " 'b0120': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A7F6',\n",
       "  'gene': 'b0120',\n",
       "  'locus': ['b0120', 'JW0116'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('H(+) + S-adenosyl-L-methionine = CO2 + S-adenosyl 3-(methylsulfanyl)propylamine',\n",
       "     '4.1.1.50')],\n",
       "   'Function': 'Catalyzes the decarboxylation of S-adenosylmethionine to S-adenosylmethioninamine (dcAdoMet), the propylamine donor required for the synthesis of the polyamines spermine and spermidine from the diamine putrescine',\n",
       "   'Regulation Activity': 'Allosterically activated by metal cations, which are absolutely required for activity. The presumed physiological activator is Mg(2+), but can also be activated in vitro by other divalent cations such as Mn(2+), Fe(2+) and Ca(2+), by the monovalent cation Li(+), and by trivalent cations such as Eu(3+), Tb(3+) and Gd(3+). Competitively inhibited by methylglyoxal bis-guanylhydrazone. Also inhibited by Zn(2+), inhibition may be due to interaction with the active site cysteine. Inactivated by treatment with the imine reductant NaCNBH(3) only in the presence of substrate',\n",
       "   'Pathway': 'Amine and polyamine biosynthesis; S-adenosylmethioninamine biosynthesis; S-adenosylmethioninamine from S-adenosyl-L-methionine: step 1/1'},\n",
       "  'seq': 'MKKLKLHGFNNLTKSLSFCIYDICYAKTAEERDGYIAYIDELYNANRLTEILSETCSIIGANILNIARQDYEPQGASVTILVSEEPVDPKLIDKTEHPGPLPETVVAHLDKSHICVHTYPESHPEGGLCTFRADIEVSTCGVISPLKALNYLIHQLESDIVTIDYRVRGFTRDINGMKHFIDHEINSIQNFMSDDMKALYDMVDVNVYQENIFHTKMLLKEFDLKHYMFHTKPEDLTDSERQEITAALWKEMREIYYGRNMPAV',\n",
       "  'mw': 30385.0},\n",
       " 'b0121': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P09158',\n",
       "  'gene': 'b0121',\n",
       "  'locus': ['b0121', 'JW0117'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [(\"putrescine + S-adenosyl 3-(methylsulfanyl)propylamine = H(+) + S-methyl-5'-thioadenosine + spermidine\",\n",
       "     '2.5.1.16'),\n",
       "    (\"S-adenosyl 3-(methylsulfanyl)propylamine + spermidine = H(+) + S-methyl-5'-thioadenosine + spermine\",\n",
       "     '2.5.1.22'),\n",
       "    (\"cadaverine + S-adenosyl 3-(methylsulfanyl)propylamine = aminopropylcadaverine + H(+) + S-methyl-5'-thioadenosine\",\n",
       "     ''),\n",
       "    (\"S-adenosyl 3-(methylsulfanyl)propylamine + spermidine = H(+) + S-methyl-5'-thioadenosine + thermospermine\",\n",
       "     '2.5.1.79')],\n",
       "   'Function': 'Involved in the biosynthesis of polyamines which play a significant role in the structural and functional organization in the chromoid of E.coli by compacting DNA and neutralizing negative charges. Catalyzes the irreversible transfer (ping-pong mechanism) of a propylamine group from the amino donor S-adenosylmethioninamine (decarboxy-AdoMet) to putrescine (1,4-diaminobutane) to yield spermidine. Cadaverine (1,5-diaminopentane) and spermidine can also be used as the propylamine acceptor',\n",
       "   'Regulation Activity': \"The activity is thought to be regulated mainly by the availability of decarboxylated S-adenosylmethionine. Inhibited by dicyclohexylamine, S-adenosyl-1,8-diamino-3-thiooctane (AdoDATO), sulfonium-deaminated analogs of S-adenosyl(5')-3-methylthiopropylamine, p-hydroxymercuribenzoate and N-ethylmaleimide\",\n",
       "   'Pathway': 'Amine and polyamine biosynthesis; spermidine biosynthesis; spermidine from putrescine: step 1/1'},\n",
       "  'seq': 'MAEKKQWHETLHDQFGQYFAVDNVLYHEKTDHQDLIIFENAAFGRVMALDGVVQTTERDEFIYHEMMTHVPLLAHGHAKHVLIIGGGDGAMLREVTRHKNVESITMVEIDAGVVSFCRQYLPNHNAGSYDDPRFKLVIDDGVNFVNQTSQTFDVIISDCTDPIGPGESLFTSAFYEGCKRCLNPGGIFVAQNGVCFLQQEEAIDSHRKLSHYFSDVGFYQAAIPTYYGGIMTFAWATDNDALRHLSTEIIQARFLASGLKCRYYNPAIHTAAFALPQYLQDALASQPS',\n",
       "  'mw': 32321.0},\n",
       " 'b0123': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P36649',\n",
       "  'gene': 'b0123',\n",
       "  'locus': ['b0123', 'JW0119'],\n",
       "  'synonyms': 'yacK',\n",
       "  'properties': {'Catalytic Activity': [('4 Cu(+) + 4 H(+) + O2 = 4 Cu(2+) + 2 H2O',\n",
       "     '1.16.3.4')],\n",
       "   'Function': \"In vitro, in the presence of excess copper ions, exhibits ferroxidase and phenoloxidase activities (PubMed:11466290, PubMed:11527384, PubMed:11867755, PubMed:15317788, PubMed:15516598, PubMed:17804014, PubMed:25679350). Fe(2+) is an excellent substrate in the presence of excess Cu(2+), but is inactive in the absence of Cu(2+) (PubMed:15516598, PubMed:17804014). Oxidizes the phenolate iron siderophores enterobactin, 2,3-dihydroxybenzoate (2,3-DHB) and 3-hydroxyanthranilate (3-HAA) (PubMed:11466290, PubMed:15317788). Oxidation and thus inactivation of enterobactin could protect cells from the interaction of enterobactin with copper and play a central role as an interface between copper detoxification and iron homeostasis (PubMed:11466290, PubMed:15317788). Also oxidizes a variety of phenolic model substrates, including 2,2'-azinobis(3-ethylbenzthiazolinesulfonic acid) (ABTS), p-phenylenediamine (pPD), 2,6-dimethoxyphenol (2,6-DMP) and 3,4-dihydroxybenzoic acid (3,4-DHB) (PubMed:11466290, PubMed:11527384, PubMed:17804014, PubMed:25679350)\",\n",
       "   'Regulation Activity': 'Ferroxidase and phenoloxidase activities are enhanced considerably in the presence of excess copper ions (PubMed:11466290, PubMed:11527384, PubMed:11867755, PubMed:15516598, PubMed:17804014). A labile regulatory copper ion near the T1 copper site is important for the copper associated activation of enzyme activity (PubMed:12794077). Ag(+) acts as a potent inhibitor of oxidase activity by binding at Cu(+) binding sites, blocking Cu(+) substrate binding and oxidation (PubMed:21903583). pPD oxidase activity is strongly inhibited by sodium azide, an inhibitor of the electron transfer (PubMed:11527384)'},\n",
       "  'seq': 'MQRRDFLKYSVALGVASALPLWSRAVFAAERPTLPIPDLLTTDARNRIQLTIGAGQSTFGGKTATTWGYNGNLLGPAVKLQRGKAVTVDIYNQLTEETTLHWHGLEVPGEVDGGPQGIIPPGGKRSVTLNVDQPAATCWFHPHQHGKTGRQVAMGLAGLVVIEDDEILKLMLPKQWGIDDVPVIVQDKKFSADGQIDYQLDVMTAAVGWFGDTLLTNGAIYPQHAAPRGWLRLRLLNGCNARSLNFATSDNRPLYVIASDGGLLPEPVKVSELPVLMGERFEVLVEVNDNKPFDLVTLPVSQMGMAIAPFDKPHPVMRIQPIAISASGALPDTLSSLPALPSLEGLTVRKLQLSMDPMLDMMGMQMLMEKYGDQAMAGMDHSQMMGHMGHGNMNHMNHGGKFDFHHANKINGQAFDMNKPMFAAAKGQYERWVISGVGDMMLHPFHIHGTQFRILSENGKPPAAHRAGWKDTVKVEGNVSEVLVKFNHDAPKEHAYMAHCHLLEHEDTGMMLGFTV',\n",
       "  'mw': 56556.0},\n",
       " 'b0124': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P15877',\n",
       "  'gene': 'b0124',\n",
       "  'locus': ['b0124', 'JW0120'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('a ubiquinone + D-glucose = a ubiquinol + D-glucono-1,5-lactone',\n",
       "     '1.1.5.2')],\n",
       "   'Function': 'GDH is probably involved in energy conservation rather than in sugar metabolism'},\n",
       "  'seq': 'MAINNTGSRRLLVTLTALFAALCGLYLLIGGGWLVAIGGSWYYPIAGLVMLGVAWMLWRSKRAALWLYAALLLGTMIWGVWEVGFDFWALTPRSDILVFFGIWLILPFVWRRLVIPASGAVAALVVALLISGGILTWAGFNDPQEINGTLSADATPAEAISPVADQDWPAYGRNQEGQRFSPLKQINADNVHNLKEAWVFRTGDVKQPNDPGEITNEVTPIKVGDTLYLCTAHQRLFALDAASGKEKWHYDPELKTNESFQHVTCRGVSYHEAKAETASPEVMADCPRRIILPVNDGRLIAINAENGKLCETFANKGVLNLQSNMPDTKPGLYEPTSPPIITDKTIVMAGSVTDNFSTRETSGVIRGFDVNTGELLWAFDPGAKDPNAIPSDEHTFTFNSPNSWAPAAYDAKLDLVYLPMGVTTPDIWGGNRTPEQERYASSILALNATTGKLAWSYQTVHHDLWDMDLPAQPTLADITVNGQKVPVIYAPAKTGNIFVLDRRNGELVVPAPEKPVPQGAAKGDYVTPTQPFSELSFRPTKDLSGADMWGATMFDQLVCRVMFHQMRYEGIFTPPSEQGTLVFPGNLGMFEWGGISVDPNREVAIANPMALPFVSKLIPRGPGNPMEQPKDAKGTGTESGIQPQYGVPYGVTLNPFLSPFGLPCKQPAWGYISALDLKTNEVVWKKRIGTPQDSMPFPMPVPVPFNMGMPMLGGPISTAGNVLFIAATADNYLRAYNMSNGEKLWQGRLPAGGQATPMTYEVNGKQYVVISAGGHGSFGTKMGDYIVAYALPDDVK',\n",
       "  'mw': 86747.0},\n",
       " 'b0125': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A9M2',\n",
       "  'gene': 'b0125',\n",
       "  'locus': ['b0125', 'JW5009'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('diphosphate + IMP = 5-phospho-alpha-D-ribose 1-diphosphate + hypoxanthine',\n",
       "     '2.4.2.8'),\n",
       "    ('diphosphate + GMP = 5-phospho-alpha-D-ribose 1-diphosphate + guanine',\n",
       "     '2.4.2.8')],\n",
       "   'Function': \"Purine salvage pathway enzyme which catalyzes the transfer of the ribosyl-5-phosphate group from 5-phospho-alpha-D-ribose 1-diphosphate (PRPP) to the N9 position of hypoxanthine to yield IMP (inosine 5'-monophosphate). To a lesser extent, can also act on guanine leading to GMP, but shows a highly less efficient activity with xanthine\",\n",
       "   'Regulation Activity': 'Inhibited by the product IMP (PubMed:12070315). Highly inhibited by nucleoside phosphonates, which are product analogs (PubMed:23927482, Ref.6)',\n",
       "   'Pathway': 'Purine metabolism; IMP biosynthesis via salvage pathway; IMP from hypoxanthine: step 1/1'},\n",
       "  'seq': 'MKHTVEVMIPEAEIKARIAELGRQITERYKDSGSDMVLVGLLRGSFMFMADLCREVQVSHEVDFMTASSYGSGMSTTRDVKILKDLDEDIRGKDVLIVEDIIDSGNTLSKVREILSLREPKSLAICTLLDKPSRREVNVPVEFIGFSIPDEFVVGYGIDYAQRYRHLPYIGKVILLDE',\n",
       "  'mw': 20115.0},\n",
       " 'b0126': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P61517',\n",
       "  'gene': 'b0126',\n",
       "  'locus': ['b0126', 'JW0122'],\n",
       "  'synonyms': 'cynT2 yadF',\n",
       "  'properties': {'Catalytic Activity': [('H(+) + hydrogencarbonate = CO2 + H2O',\n",
       "     '4.2.1.1')]},\n",
       "  'seq': 'MKDIDTLISNNALWSKMLVEEDPGFFEKLAQAQKPRFLWIGCSDSRVPAERLTGLEPGELFVHRNVANLVIHTDLNCLSVVQYAVDVLEVEHIIICGHYGCGGVQAAVENPELGLINNWLLHIRDIWFKHSSLLGEMPQERRLDTLCELNVMEQVYNLGHSTIMQSAWKRGQKVTIHGWAYGIHDGLLRDLDVTATNRETLEQRYRHGISNLKLKHANHK',\n",
       "  'mw': 25097.0},\n",
       " 'b0131': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A790',\n",
       "  'gene': 'b0131',\n",
       "  'locus': ['b0131', 'JW0127'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('H(+) + L-aspartate = beta-alanine + CO2',\n",
       "     '4.1.1.11')],\n",
       "   'Function': 'Catalyzes the pyruvoyl-dependent decarboxylation of aspartate to produce beta-alanine',\n",
       "   'Regulation Activity': 'Inhibited by hydroxylamine, sodium borohydride, D-cycloserine, hydrazine, semicarbazine and succinic dehydrazine (PubMed:6767707). D-serine is a competitive inhibitor (PubMed:6767707). Cleavage and catalytic activity are regulated by PanZ in a coenzyme A (CoA)-dependent fashion (PubMed:23170229, PubMed:25910242)',\n",
       "   'Pathway': 'Cofactor biosynthesis; (R)-pantothenate biosynthesis; beta-alanine from L-aspartate: step 1/1'},\n",
       "  'seq': 'MIRTMLQGKLHRVKVTHADLHYEGSCAIDQDFLDAAGILENEAIDIWNVTNGKRFSTYAIAAERGSRIISVNGAAAHCASVGDIVIIASFVTMPDEEARTWRPNVAYFEGDNEMKRTAKAIPVQVA',\n",
       "  'mw': 13834.0},\n",
       " 'b0133': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P31663',\n",
       "  'gene': 'b0133',\n",
       "  'locus': ['b0133', 'JW0129'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(R)-pantoate + ATP + beta-alanine = (R)-pantothenate + AMP + diphosphate + H(+)',\n",
       "     '6.3.2.1')],\n",
       "   'Function': 'Catalyzes the condensation of pantoate with beta-alanine in an ATP-dependent reaction via a pantoyl-adenylate intermediate',\n",
       "   'Regulation Activity': 'Activation requires a combination of a divalent cation, magnesium or manganese, and a monovalent cation, potassium or ammonium. Above the optimum concentration for activation, magnesium and manganese are rather inhibitory. Also activated by 2-mercaptoethanol, dithiothreitol, cysteine and glutathione. Inhibited by divalent cations (mercury, cobalt, zinc, copper, silver), chelating agents (EDTA, EGTA and o-phenanthroline), and analogs of beta-alanine (taurine, gamma-aminobutyrate, gamma-amino-beta-hydroxybutyrate)',\n",
       "   'Pathway': 'Cofactor biosynthesis; (R)-pantothenate biosynthesis; (R)-pantothenate from (R)-pantoate and beta-alanine: step 1/1'},\n",
       "  'seq': 'MLIIETLPLLRQQIRRLRMEGKRVALVPTMGNLHDGHMKLVDEAKARADVVVVSIFVNPMQFDRPEDLARYPRTLQEDCEKLNKRKVDLVFAPSVKEIYPNGTETHTYVDVPGLSTMLEGASRPGHFRGVSTIVSKLFNLVQPDIACFGEKDFQQLALIRKMVADMGFDIEIVGVPIMRAKDGLALSSRNGYLTAEQRKIAPGLYKVLSSIADKLQAGERDLDEIITIAGQELNEKGFRADDIQIRDADTLLEVSETSKRAVILVAAWLGDARLIDNKMVELA',\n",
       "  'mw': 31598.0},\n",
       " 'b0134': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P31057',\n",
       "  'gene': 'b0134',\n",
       "  'locus': ['b0134', 'JW0130'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(6R)-5,10-methylene-5,6,7,8-tetrahydrofolate + 3-methyl-2-oxobutanoate + H2O = (6S)-5,6,7,8-tetrahydrofolate + 2-dehydropantoate',\n",
       "     '2.1.2.11')],\n",
       "   'Function': 'Catalyzes the reversible reaction in which hydroxymethyl group from 5,10-methylenetetrahydrofolate is transferred onto alpha-ketoisovalerate to form ketopantoate',\n",
       "   'Regulation Activity': 'Inhibited by isovalerate, pyruvate, 3-methyl-2-butanone, and valine. Partially inhibited by formaldehyde and tetrahydrofolate below or near the KM value. Also inhibited by later intermediates pantoate, pantothenate and coenzyme A',\n",
       "   'Pathway': 'Cofactor biosynthesis; (R)-pantothenate biosynthesis; (R)-pantoate from 3-methyl-2-oxobutanoate: step 1/2'},\n",
       "  'seq': 'MKPTTISLLQKYKQEKKRFATITAYDYSFAKLFADEGLNVMLVGDSLGMTVQGHDSTLPVTVADIAYHTAAVRRGAPNCLLLADLPFMAYATPEQAFENAATVMRAGANMVKIEGGEWLVETVQMLTERAVPVCGHLGLTPQSVNIFGGYKVQGRGDEAGDQLLSDALALEAAGAQLLVLECVPVELAKRITEALAIPVIGIGAGNVTDGQILVMHDAFGITGGHIPKFAKNFLAETGDIRAAVRQYMAEVESGVYPGEEHSFH',\n",
       "  'mw': 28237.0},\n",
       " 'b0142': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P26281',\n",
       "  'gene': 'b0142',\n",
       "  'locus': ['b0142', 'JW0138'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('6-hydroxymethyl-7,8-dihydropterin + ATP = (7,8-dihydropterin-6-yl)methyl diphosphate + AMP + H(+)',\n",
       "     '2.7.6.3')],\n",
       "   'Function': 'Catalyzes the transfer of pyrophosphate from adenosine triphosphate (ATP) to 6-hydroxymethyl-7,8-dihydropterin, an enzymatic step in folate biosynthesis pathway',\n",
       "   'Pathway': 'Cofactor biosynthesis; tetrahydrofolate biosynthesis; 2-amino-4-hydroxy-6-hydroxymethyl-7,8-dihydropteridine diphosphate from 7,8-dihydroneopterin triphosphate: step 4/4'},\n",
       "  'seq': 'MTVAYIAIGSNLASPLEQVNAALKALGDIPESHILTVSSFYRTPPLGPQDQPDYLNAAVALETSLAPEELLNHTQRIELQQGRVRKAERWGPRTLDLDIMLFGNEVINTERLTVPHYDMKNRGFMLWPLFEIAPELVFPDGEMLRQILHTRAFDKLNKW',\n",
       "  'mw': 18079.0},\n",
       " 'b0149': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P02919',\n",
       "  'gene': 'b0149',\n",
       "  'locus': ['b0149', 'JW0145'],\n",
       "  'synonyms': 'pbpF ponB',\n",
       "  'properties': {'Catalytic Activity': [('[GlcNAc-(1->4)-Mur2Ac(oyl-L-Ala-gamma-D-Glu-L-Lys-D-Ala-D-Ala)](n)-di-trans,octa-cis-undecaprenyl diphosphate + beta-D-GlcNAc-(1->4)-Mur2Ac(oyl-L-Ala-gamma-D-Glu-L-Lys-D-Ala-D-Ala)-di-trans,octa-cis-undecaprenyl diphosphate = [GlcNAc-(1->4)-Mur2Ac(oyl-L-Ala-gamma-D-Glu-L-Lys-D-Ala-D-Ala)](n+1)-di-trans-octa-cis-undecaprenyl diphosphate + di-trans,octa-cis-undecaprenyl diphosphate + H(+)',\n",
       "     '2.4.1.129'),\n",
       "    ('Preferential cleavage: (Ac)2-L-Lys-D-Ala-|-D-Ala. Also transpeptidation of peptidyl-alanyl moieties that are N-acyl substituents of D-alanine.',\n",
       "     '3.4.16.4')],\n",
       "   'Function': 'Cell wall formation. Synthesis of cross-linked peptidoglycan from the lipid intermediates. The enzyme has a penicillin-insensitive transglycosylase N-terminal domain (formation of linear glycan strands) and a penicillin-sensitive transpeptidase C-terminal domain (cross-linking of the peptide subunits)',\n",
       "   'Pathway': 'Cell wall biogenesis; peptidoglycan biosynthesis'},\n",
       "  'seq': 'MAGNDREPIGRKGKPTRPVKQKVSRRRYEDDDDYDDYDDYEDEEPMPRKGKGKGKGRKPRGKRGWLWLLLKLAIVFAVLIAIYGVYLDQKIRSRIDGKVWQLPAAVYGRMVNLEPDMTISKNEMVKLLEATQYRQVSKMTRPGEFTVQANSIEMIRRPFDFPDSKEGQVRARLTFDGDHLATIVNMENNRQFGFFRLDPRLITMISSPNGEQRLFVPRSGFPDLLVDTLLATEDRHFYEHDGISLYSIGRAVLANLTAGRTVQGASTLTQQLVKNLFLSSERSYWRKANEAYMALIMDARYSKDRILELYMNEVYLGQSGDNEIRGFPLASLYYFGRPVEELSLDQQALLVGMVKGASIYNPWRNPKLALERRNLVLRLLQQQQIIDQELYDMLSARPLGVQPRGGVISPQPAFMQLVRQELQAKLGDKVKDLSGVKIFTTFDSVAQDAAEKAAVEGIPALKKQRKLSDLETAIVVVDRFSGEVRAMVGGSEPQFAGYNRAMQARRSIGSLAKPATYLTALSQPKIYRLNTWIADAPIALRQPNGQVWSPQNDDRRYSESGRVMLVDALTRSMNVPTVNLGMALGLPAVTETWIKLGVPKDQLHPVPAMLLGALNLTPIEVAQAFQTIASGGNRAPLSALRSVIAEDGKVLYQSFPQAERAVPAQAAYLTLWTMQQVVQRGTGRQLGAKYPNLHLAGKTGTTNNNVDTWFAGIDGSTVTITWVGRDNNQPTKLYGASGAMSIYQRYLANQTPTPLNLVPPEDIADMGVDYDGNFVCSGGMRILPVWTSDPQSLCQQSEMQQQPSGNPFDQSSQPQQQPQQQPAQQEQKDSDGVAGWIKDMFGSN',\n",
       "  'mw': 94293.0},\n",
       " 'b0150': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P06971',\n",
       "  'gene': 'b0150',\n",
       "  'locus': ['b0150', 'JW0146'],\n",
       "  'synonyms': 'tonA',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Involved in the uptake of iron in complex with ferrichrome, a hydroxamate-type siderophore. Binds and transports ferrichrome-iron across the outer membrane (PubMed:1089064, PubMed:2066336). In addition to its role in ferrichrome-iron transport, transports the antibiotic albomycin, which is a structural analog of ferrichrome, and acts as a receptor for colicin M, microcin J25 and bacteriophages T1, T5, phi80 and UC-1 (PubMed:1089064, PubMed:353030, PubMed:2066336, PubMed:8617231). The energy source, which is required for all FhuA functions except infection by phage T5, is provided by the inner membrane TonB system (PubMed:353030, PubMed:9353297, PubMed:12427941)',\n",
       "   'Regulation Activity': 'Binding of ferrichrome or colicin M enhances the interaction between FhuA and TonB (PubMed:9353297). TonB activates FhuA through interaction with the beta-barrel (PubMed:12427941)'},\n",
       "  'seq': 'MARSKTAQPKHSLRKIAVVVATAVSGMSVYAQAAVEPKEDTITVTAAPAPQESAWGPAATIAARQSATGTKTDTPIQKVPQSISVVTAEEMALHQPKSVKEALSYTPGVSVGTRGASNTYDHLIIRGFAAEGQSQNNYLNGLKLQGNFYNDAVIDPYMLERAEIMRGPVSVLYGKSSPGGLLNMVSKRPTTEPLKEVQFKAGTDSLFQTGFDFSDSLDDDGVYSYRLTGLARSANAQQKGSEEQRYAIAPAFTWRPDDKTNFTFLSYFQNEPETGYYGWLPKEGTVEPLPNGKRLPTDFNEGAKNNTYSRNEKMVGYSFDHEFNDTFTVRQNLRFAENKTSQNSVYGYGVCSDPANAYSKQCAALAPADKGHYLARKYVVDDEKLQNFSVDTQLQSKFATGDIDHTLLTGVDFMRMRNDINAWFGYDDSVPLLNLYNPVNTDFDFNAKDPANSGPYRILNKQKQTGVYVQDQAQWDKVLVTLGGRYDWADQESLNRVAGTTDKRDDKQFTWRGGVNYLFDNGVTPYFSYSESFEPSSQVGKDGNIFAPSKGKQYEVGVKYVPEDRPIVVTGAVYNLTKTNNLMADPEGSFFSVEGGEIRARGVEIEAKAALSASVNVVGSYTYTDAEYTTDTTYKGNTPAQVPKHMASLWADYTFFDGPLSGLTLGTGGRYTGSSYGDPANSFKVGSYTVVDALVRYDLARVGMAGSNVALHVNNLFDREYVASCFNTYGCFWGAERQVVATATFRF',\n",
       "  'mw': 82182.0},\n",
       " 'b0151': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P07821',\n",
       "  'gene': 'b0151',\n",
       "  'locus': ['b0151', 'JW0147'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + H2O + Fe(3+)-hydroxamate complex-[hydroxamate-binding protein]Side 1 = ADP + phosphate + Fe(3+)-hydroxamate complexSide 2 + [hydroxamate-binding protein]Side 1.',\n",
       "     '7.2.2.16')],\n",
       "   'Function': 'Part of the ABC transporter complex FhuCDB involved in iron(3+)-hydroxamate import. Responsible for energy coupling to the transport system'},\n",
       "  'seq': 'MQEYTNHSDTTFALRNISFRVPGRTLLHPLSLTFPAGKVTGLIGHNGSGKSTLLKMLGRHQPPSEGEILLDAQPLESWSSKAFARKVAYLPQQLPPAEGMTVRELVAIGRYPWHGALGRFGAADREKVEEAISLVGLKPLAHRLVDSLSGGERQRAWIAMLVAQDSRCLLLDEPTSALDIAHQVDVLSLVHRLSQERGLTVIAVLHDINMAARYCDYLVALRGGEMIAQGTPAEIMRGETLEMIYGIPMGILPHPAGAAPVSFVY',\n",
       "  'mw': 28886.0},\n",
       " 'b0152': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P07822',\n",
       "  'gene': 'b0152',\n",
       "  'locus': ['b0152', 'JW0148'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the ABC transporter complex FhuCDB involved in iron(3+)-hydroxamate import. Binds the iron(3+)-hydroxamate complex and transfers it to the membrane-bound permease. Required for the transport of all iron(3+)-hydroxamate siderophores such as ferrichrome, gallichrome, desferrioxamine, coprogen, aerobactin, shizokinen, rhodotorulic acid and the antibiotic albomycin'},\n",
       "  'seq': 'MSGLPLISRRRLLTAMALSPLLWQMNTAHAAAIDPNRIVALEWLPVELLLALGIVPYGVADTINYRLWVSEPPLPDSVIDVGLRTEPNLELLTEMKPSFMVWSAGYGPSPEMLARIAPGRGFNFSDGKQPLAMARKSLTEMADLLNLQSAAETHLAQYEDFIRSMKPRFVKRGARPLLLTTLIDPRHMLVFGPNSLFQEILDEYGIPNAWQGETNFWGSTAVSIDRLAAYKDVDVLCFDHDNSKDMDALMATPLWQAMPFVRAGRFQRVPAVWFYGATLSAMHFVRVLDNAIGGKA',\n",
       "  'mw': 32998.0},\n",
       " 'b0153': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P06972',\n",
       "  'gene': 'b0153',\n",
       "  'locus': ['b0153', 'JW0149'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the ABC transporter complex FhuCDB involved in iron(3+)-hydroxamate import. Responsible for the translocation of the substrate across the membrane'},\n",
       "  'seq': 'MSKRIALFPALLLALLVIVATALTWMNFSQALPRSQWAQAAWSPDIDVIEQMIFHYSLLPRLAISLLVGAGLGLVGVLFQQVLRNPLAEPTTLGVATGAQLGITVTTLWAIPGAMASQFAAQAGACVVGLIVFGVAWGKRLSPVTLILAGLVVSLYCGAINQLLVIFHHDQLQSMFLWSTGTLTQTDWGGVERLWPQLLGGVMLTLLLLRPLTLMGLDDGVARNLGLALSLARLAALSLAIVISALLVNAVGIIGFIGLFAPLLAKMLGARRLLPRLMLASLIGALILWLSDQIILWLTRVWMEVSTGSVTALIGAPLLLWLLPRLRSISAPDMKVNDRVAAERQHVLAFALAGGVLLLMAVVVALSFGRDAHGWTWASGALLEDLMPWRWPRIMAALFAGVMLAVAGCIIQRLTGNPMASPEVLGISSGAAFGVVLMLFLVPGNAFGWLLPAGSLGAAVTLLIIMIAAGRGGFSPHRMLLAGMALSTAFTMLLMMLQASGDPRMAQVLTWISGSTYNATDAQVWRTGIVMVILLAITPLCRRWLTILPLGGDTARAVGMALTPTRIALLLLAACLTATATMTIGPLSFVGLMAPHIARMMGFRRTMPHIVISALVGGLLLVFADWCGRMVLFPFQIPAGLLSTFIGAPYFIYLLRKQSR',\n",
       "  'mw': 70423.0},\n",
       " 'b0154': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P23893',\n",
       "  'gene': 'b0154',\n",
       "  'locus': ['b0154', 'JW0150'],\n",
       "  'synonyms': 'gsa popC',\n",
       "  'properties': {'Catalytic Activity': [('(S)-4-amino-5-oxopentanoate = 5-aminolevulinate',\n",
       "     '5.4.3.8')],\n",
       "   'Function': 'Aminomutase that catalyzes the transfer of the amine group on carbon 2 of glutamate 1-semialdehyde to the adjacent carbon 1 to form 5-aminolevulinate',\n",
       "   'Pathway': 'Porphyrin-containing compound metabolism; protoporphyrin-IX biosynthesis; 5-aminolevulinate from L-glutamyl-tRNA(Glu): step 2/2'},\n",
       "  'seq': 'MSKSENLYSAARELIPGGVNSPVRAFTGVGGTPLFIEKADGAYLYDVDGKAYIDYVGSWGPMVLGHNHPAIRNAVIEAAERGLSFGAPTEMEVKMAQLVTELVPTMDMVRMVNSGTEATMSAIRLARGFTGRDKIIKFEGCYHGHADCLLVKAGSGALTLGQPNSPGVPADFAKYTLTCTYNDLASVRAAFEQYPQEIACIIVEPVAGNMNCVPPLPEFLPGLRALCDEFGALLIIDEVMTGFRVALAGAQDYYGVVPDLTCLGKIIGGGMPVGAFGGRRDVMDALAPTGPVYQAGTLSGNPIAMAAGFACLNEVAQPGVHETLDELTTRLAEGLLEAAEEAGIPLVVNHVGGMFGIFFTDAESVTCYQDVMACDVERFKRFFHMMLDEGVYLAPSAFEAGFMSVAHSMEDINNTIDAARRVFAKL',\n",
       "  'mw': 45366.0},\n",
       " 'b0155': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37019',\n",
       "  'gene': 'b0155',\n",
       "  'locus': ['b0155', 'JW5012'],\n",
       "  'synonyms': 'eriC yadQ',\n",
       "  'properties': {'Catalytic Activity': [('2 chloride(in) + H(+)(out) = 2 chloride(out) + H(+)(in)',\n",
       "     '')],\n",
       "   'Function': 'Proton-coupled chloride transporter. Functions as antiport system and exchanges two chloride ions for 1 proton. Probably acts as an electrical shunt for an outwardly-directed proton pump that is linked to amino acid decarboxylation, as part of the extreme acid resistance (XAR) response'},\n",
       "  'seq': 'MKTDTPSLETPQAARLRRRQLIRQLLERDKTPLAILFMAAVVGTLVGLAAVAFDKGVAWLQNQRMGALVHTADNYPLLLTVAFLCSAVLAMFGYFLVRKYAPEAGGSGIPEIEGALEDQRPVRWWRVLPVKFFGGLGTLGGGMVLGREGPTVQIGGNIGRMVLDIFRLKGDEARHTLLATGAAAGLAAAFNAPLAGILFIIEEMRPQFRYTLISIKAVFIGVIMSTIMYRIFNHEVALIDVGKLSDAPLNTLWLYLILGIIFGIFGPIFNKWVLGMQDLLHRVHGGNITKWVLMGGAIGGLCGLLGFVAPATSGGGFNLIPIATAGNFSMGMLVFIFVARVITTLLCFSSGAPGGIFAPMLALGTVLGTAFGMVAVELFPQYHLEAGTFAIAGMGALLAASIRAPLTGIILVLEMTDNYQLILPMIITGLGATLLAQFTGGKPLYSAILARTLAKQEAEQLARSKAASASENT',\n",
       "  'mw': 50349.0},\n",
       " 'b0158': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37028',\n",
       "  'gene': 'b0158',\n",
       "  'locus': ['b0158', 'JW0154'],\n",
       "  'synonyms': 'yadT',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the ABC transporter complex BtuCDF involved in vitamin B12 import. Binds vitamin B12 and delivers it to the periplasmic surface of BtuC'},\n",
       "  'seq': 'MAKSLFRALVALSFLAPLWLNAAPRVITLSPANTELAFAAGITPVGVSSYSDYPPQAQKIEQVSTWQGMNLERIVALKPDLVIAWRGGNAERQVDQLASLGIKVMWVDATSIEQIANALRQLAPWSPQPDKAEQAAQSLLDQYAQLKAQYADKPKKRVFLQFGINPPFTSGKESIQNQVLEVCGGENIFKDSRVPWPQVSREQVLARSPQAIVITGGPDQIPKIKQYWGEQLKIPVIPLTSDWFERASPRIILAAQQLCNALSQVD',\n",
       "  'mw': 29367.0},\n",
       " 'b0159': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AF12',\n",
       "  'gene': 'b0159',\n",
       "  'locus': ['b0159', 'JW0155'],\n",
       "  'synonyms': 'mtn pfs yadA',\n",
       "  'properties': {'Catalytic Activity': [('H2O + S-adenosyl-L-homocysteine = adenine + S-(5-deoxy-D-ribos-5-yl)-L-homocysteine',\n",
       "     '3.2.2.9'),\n",
       "    (\"H2O + S-methyl-5'-thioadenosine = 5-(methylsulfanyl)-D-ribose + adenine\",\n",
       "     '3.2.2.9'),\n",
       "    (\"5'-deoxyadenosine + H2O = 5-deoxy-D-ribose + adenine\", '3.2.2.9')],\n",
       "   'Function': \"Catalyzes the irreversible cleavage of the glycosidic bond in both 5'-methylthioadenosine (MTA) and S-adenosylhomocysteine (SAH/AdoHcy) to adenine and the corresponding thioribose, 5'-methylthioribose and S-ribosylhomocysteine, respectively (PubMed:3911944, PubMed:16101288). Also cleaves 5'-deoxyadenosine, a toxic by-product of radical S-adenosylmethionine (SAM) enzymes, into 5-deoxyribose and adenine. Thus, is required for in vivo function of the radical SAM enzymes biotin synthase and lipoic acid synthase, that are inhibited by 5'-deoxyadenosine accumulation (PubMed:15911379). Can also use 5'-isobutylthioadenosine, 5'-n-butylthioadenosine, S-adenosyl-D-homocysteine, decarboxylated adenosylhomocysteine, deaminated adenosylhomocysteine and S-2-aza-adenosylhomocysteine as substrates in vitro (PubMed:3911944)\",\n",
       "   'Regulation Activity': \"Strongly inhibited by aryl-substituted MTA analogs, alkyl-substituted MTA analogs, 5'-methylthioformycin, 5'-chloroformycin, S-formycinylhomocysteine, 5'-methylthiotubercidin, S-tubercidinylhomocysteine and S-8-aza-adenosylhomocysteine. Poorly inhibited by 5'-isobutylthioinosine, 5'-n-butylthioinosine, 5'-methylthio-3-deaza-adenosine, 5'-isobutylthio-3-deaza-adenosine, S-n-6-methyl-3-deaza-adenosylhomocysteine, S-adenosylhomocysteine sulphoxide and Sinefungin\",\n",
       "   'Pathway': \"Amino-acid biosynthesis; L-methionine biosynthesis via salvage pathway; S-methyl-5-thio-alpha-D-ribose 1-phosphate from S-methyl-5'-thioadenosine (hydrolase route): step 1/2\"},\n",
       "  'seq': 'MKIGIIGAMEEEVTLLRDKIENRQTISLGGCEIYTGQLNGTEVALLKSGIGKVAAALGATLLLEHCKPDVIINTGSAGGLAPTLKVGDIVVSDEARYHDADVTAFGYEYGQLPGCPAGFKADDKLIAAAEACIAELNLNAVRGLIVSGDAFINGSVGLAKIRHNFPQAIAVEMEATAIAHVCHNFNVPFVVVRAISDVADQQSHLSFDEFLAVAAKQSSLMVESLVQKLAHG',\n",
       "  'mw': 24354.0},\n",
       " 'b0160': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P15723',\n",
       "  'gene': 'b0160',\n",
       "  'locus': ['b0160', 'JW0156'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [(\"dGTP + H2O = 2'-deoxyguanosine + H(+) + triphosphate\",\n",
       "     '3.1.5.1')],\n",
       "   'Function': 'dGTPase preferentially hydrolyzes dGTP over the other canonical NTPs'},\n",
       "  'seq': 'MAQIDFRKKINWHRRYRSPQGVKTEHEILRIFESDRGRIINSPAIRRLQQKTQVFPLERNAAVRTRLTHSMEVQQVGRYIAKEILSRLKELKLLEAYGLDELTGPFESIVEMSCLMHDIGNPPFGHFGEAAINDWFRQRLHPEDAESQPLTDDRCSVAALRLRDGEEPLNELRRKIRQDLCHFEGNAQGIRLVHTLMRMNLTWAQVGGILKYTRPAWWRGETPETHHYLMKKPGYYLSEEAYIARLRKELNLALYSRFPLTWIMEAADDISYCVADLEDAVEKRIFTVEQLYHHLHEAWGQHEKGSLFSLVVENAWEKSRSNSLSRSTEDQFFMYLRVNTLNKLVPYAAQRFIDNLPAIFAGTFNHALLEDASECSDLLKLYKNVAVKHVFSHPDVERLELQGYRVISGLLEIYRPLLSLSLSDFTELVEKERVKRFPIESRLFHKLSTRHRLAYVEAVSKLPSDSPEFPLWEYYYRCRLLQDYISGMTDLYAWDEYRRLMAVEQ',\n",
       "  'mw': 59383.0},\n",
       " 'b0166': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A9D8',\n",
       "  'gene': 'b0166',\n",
       "  'locus': ['b0166', 'JW0161'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(S)-2,3,4,5-tetrahydrodipicolinate + H2O + succinyl-CoA = (S)-2-succinylamino-6-oxoheptanedioate + CoA',\n",
       "     '2.3.1.117')],\n",
       "   'Function': 'Catalyzes the formation of N-succinyl-2-amino-6-oxo-L-pimelate from succinyl-CoA and tetrahydrodipicolinate, a key step in lysine biosynthesis via diaminopimelate pathway',\n",
       "   'Regulation Activity': 'Inhibited by N-ethylmaleimide, p-chloromercuriphenyl sulfonate, cobalt and copper ions',\n",
       "   'Pathway': 'Amino-acid biosynthesis; L-lysine biosynthesis via DAP pathway; LL-2,6-diaminopimelate from (S)-tetrahydrodipicolinate (succinylase route): step 1/3'},\n",
       "  'seq': 'MQQLQNIIETAFERRAEITPANADTVTREAVNQVIALLDSGALRVAEKIDGQWVTHQWLKKAVLLSFRINDNQVIEGAESRYFDKVPMKFADYDEARFQKEGFRVVPPAAVRQGAFIARNTVLMPSYVNIGAYVDEGTMVDTWATVGSCAQIGKNVHLSGGVGIGGVLEPLQANPTIIEDNCFIGARSEVVEGVIVEEGSVISMGVYIGQSTRIYDRETGEIHYGRVPAGSVVVSGNLPSKDGKYSLYCAVIVKKVDAKTRGKVGINELLRTID',\n",
       "  'mw': 29892.0},\n",
       " 'b0171': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A7E9',\n",
       "  'gene': 'b0171',\n",
       "  'locus': ['b0171', 'JW0166'],\n",
       "  'synonyms': 'smbA',\n",
       "  'properties': {'Catalytic Activity': [('ATP + UMP = ADP + UDP', '2.7.4.22')],\n",
       "   'Function': 'Catalyzes the reversible phosphorylation of UMP to UDP, with ATP as the most efficient phosphate donor',\n",
       "   'Regulation Activity': 'Allosterically activated by GTP. Competitively inhibited by magnesium-free UTP. Magnesium-bound UTP is unable to inhibit enzyme activity',\n",
       "   'Pathway': 'Pyrimidine metabolism; CTP biosynthesis via de novo pathway; UDP from UMP (UMPK route): step 1/1'},\n",
       "  'seq': 'MATNAKPVYKRILLKLSGEALQGTEGFGIDASILDRMAQEIKELVELGIQVGVVIGGGNLFRGAGLAKAGMNRVVGDHMGMLATVMNGLAMRDALHRAYVNARLMSAIPLNGVCDSYSWAEAISLLRNNRVVILSAGTGNPFFTTDSAACLRGIEIEADVVLKATKVDGVFTADPAKDPTATMYEQLTYSEVLEKELKVMDLAAFTLARDHKLPIRVFNMNKPGALRRVVMGEKEGTLITE',\n",
       "  'mw': 25970.0},\n",
       " 'b0173': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P45568',\n",
       "  'gene': 'b0173',\n",
       "  'locus': ['b0173', 'JW0168'],\n",
       "  'synonyms': 'ispC yaeM',\n",
       "  'properties': {'Catalytic Activity': [('2-C-methyl-D-erythritol 4-phosphate + NADP(+) = 1-deoxy-D-xylulose 5-phosphate + H(+) + NADPH',\n",
       "     '1.1.1.267')],\n",
       "   'Function': 'Catalyzes the NADPH-dependent rearrangement and reduction of 1-deoxy-D-xylulose-5-phosphate (DXP) to 2-C-methyl-D-erythritol 4-phosphate (MEP)',\n",
       "   'Regulation Activity': 'Inhibited by fosmidomycin',\n",
       "   'Pathway': 'Isoprenoid biosynthesis; isopentenyl diphosphate biosynthesis via DXP pathway; isopentenyl diphosphate from 1-deoxy-D-xylulose 5-phosphate: step 1/6'},\n",
       "  'seq': 'MKQLTILGSTGSIGCSTLDVVRHNPEHFRVVALVAGKNVTRMVEQCLEFSPRYAVMDDEASAKLLKTMLQQQGSRTEVLSGQQAACDMAALEDVDQVMAAIVGAAGLLPTLAAIRAGKTILLANKESLVTCGRLFMDAVKQSKAQLLPVDSEHNAIFQSLPQPIQHNLGYADLEQNGVVSILLTGSGGPFRETPLRDLATMTPDQACRHPNWSMGRKISVDSATMMNKGLEYIEARWLFNASASQMEVLIHPQSVIHSMVRYQDGSVLAQLGEPDMRTPIAHTMAWPNRVNSGVKPLDFCKLSALTFAAPDYDRYPCLKLAMEAFEQGQAATTALNAANEITVAAFLAQQIRFTDIAALNLSVLEKMDMREPQCVDDVLSVDANAREVARKEVMRLAS',\n",
       "  'mw': 43388.0},\n",
       " 'b0174': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P60472',\n",
       "  'gene': 'b0174',\n",
       "  'locus': ['b0174', 'JW0169'],\n",
       "  'synonyms': 'rth uppS yaeS',\n",
       "  'properties': {'Catalytic Activity': [('(2E,6E)-farnesyl diphosphate + 8 isopentenyl diphosphate = di-trans,octa-cis-undecaprenyl diphosphate + 8 diphosphate',\n",
       "     '2.5.1.31')],\n",
       "   'Function': 'Generates ditrans,octacis-undecaprenyl pyrophosphate (UPP) from isopentenyl pyrophosphate (IPP) and farnesyl diphosphate (FPP). UPP is the precursor of glycosyl carrier lipid in the biosynthesis of bacterial cell wall polysaccharide components such as peptidoglycan and lipopolysaccharide',\n",
       "   'Regulation Activity': 'Inhibited by bisphosphonates'},\n",
       "  'seq': 'MMLSATQPLSEKLPAHGCRHVAIIMDGNGRWAKKQGKIRAFGHKAGAKSVRRAVSFAANNGIEALTLYAFSSENWNRPAQEVSALMELFVWALDSEVKSLHRHNVRLRIIGDTSRFNSRLQERIRKSEALTAGNTGLTLNIAANYGGRWDIVQGVRQLAEKVQQGNLQPDQIDEEMLNQHVCMHELAPVDLVIRTGGEHRISNFLLWQIAYAELYFTDVLWPDFDEQDFEGALNAFANRERRFGGTEPGDETA',\n",
       "  'mw': 28444.0},\n",
       " 'b0175': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ABG1',\n",
       "  'gene': 'b0175',\n",
       "  'locus': ['b0175', 'JW5810'],\n",
       "  'synonyms': 'cds',\n",
       "  'properties': {'Catalytic Activity': [('a 1,2-diacyl-sn-glycero-3-phosphate + CTP + H(+) = a CDP-1,2-diacyl-sn-glycerol + diphosphate',\n",
       "     '2.7.7.41')],\n",
       "   'Pathway': 'Phospholipid metabolism; CDP-diacylglycerol biosynthesis; CDP-diacylglycerol from sn-glycerol 3-phosphate: step 3/3'},\n",
       "  'seq': 'MLKYRLISAFVLIPVVIAALFLLPPVGFAIVTLVVCMLAAWEWGQLSGFTTRSQRVWLAVLCGLLLALMLFLLPEYHRNIHQPLVEISLWASLGWWIVALLLVLFYPGSAAIWRNSKTLRLIFGVLTIVPFFWGMLALRAWHYDENHYSGAIWLLYVMILVWGADSGAYMFGKLFGKHKLAPKVSPGKTWQGFIGGLATAAVISWGYGMWANLDVAPVTLLICSIVAALASVLGDLTESMFKREAGIKDSGHLIPGHGGILDRIDSLTAAVPVFACLLLLVFRTL',\n",
       "  'mw': 31454.0},\n",
       " 'b0179': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P21645',\n",
       "  'gene': 'b0179',\n",
       "  'locus': ['b0179', 'JW0174'],\n",
       "  'synonyms': 'firA omsA',\n",
       "  'properties': {'Catalytic Activity': [('a (3R)-hydroxyacyl-[ACP] + a UDP-3-O-[(3R)-3-hydroxyacyl]-alpha-D-glucosamine = a UDP-2-N,3-O-bis[(3R)-3-hydroxyacyl]-alpha-D-glucosamine + H(+) + holo-[ACP]',\n",
       "     '2.3.1.191'),\n",
       "    ('(3R)-hydroxytetradecanoyl-[ACP] + UDP-3-O-[(3R)-3-hydroxytetradecanoyl]-alpha-D-glucosamine = H(+) + holo-[ACP] + UDP-2-N,3-O-bis[(3R)-3-hydroxytetradecanoyl]-alpha-D-glucosamine',\n",
       "     '')],\n",
       "   'Function': \"Catalyzes the N-acylation of UDP-3-O-(hydroxytetradecanoyl)glucosamine using 3-hydroxytetradecanoyl-ACP as the acyl donor. Is involved in the biosynthesis of lipid A, a phosphorylated glycolipid that anchors the lipopolysaccharide to the outer membrane of the cell. Prefers (3R)-3-hydroxytetradecanoyl-ACP over (3R)-3-hydroxyhexadecanoyl-ACP as the acyl donor in vitro, which is consistent with the structure of E.coli lipid A that contains over 95% (R)-3-hydroxytetradecanoate at the 2 and 2' positions\",\n",
       "   'Regulation Activity': 'Inhibited by divalent cations such as Ca(2+) and Mg(2+)',\n",
       "   'Pathway': 'Glycolipid biosynthesis; lipid IV(A) biosynthesis; lipid IV(A) from (3R)-3-hydroxytetradecanoyl-[acyl-carrier-protein] and UDP-N-acetyl-alpha-D-glucosamine: step 3/6'},\n",
       "  'seq': 'MPSIRLADLAQQLDAELHGDGDIVITGVASMQSAQTGHITFMVNPKYREHLGLCQASAVVMTQDDLPFAKSAALVVKNPYLTYARMAQILDTTPQPAQNIAPSAVIDATAKLGNNVSIGANAVIESGVELGDNVIIGAGCFVGKNSKIGAGSRLWANVTIYHEIQIGQNCLIQSGTVVGADGFGYANDRGNWVKIPQIGRVIIGDRVEIGACTTIDRGALDDTIIGNGVIIDNQCQIAHNVVIGDNTAVAGGVIMAGSLKIGRYCMIGGASVINGHMEICDKVTVTGMGMVMRPITEPGVYSSGIPLQPNKVWRKTAALVMNIDDMSKRLKSLERKVNQQD',\n",
       "  'mw': 36038.0},\n",
       " 'b0180': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A6Q6',\n",
       "  'gene': 'b0180',\n",
       "  'locus': ['b0180', 'JW0175'],\n",
       "  'synonyms': 'sefA yaeA',\n",
       "  'properties': {'Catalytic Activity': [('a (3R)-hydroxyacyl-[ACP] = a (2E)-enoyl-[ACP] + H2O',\n",
       "     '4.2.1.59'),\n",
       "    ('(3R)-hydroxybutanoyl-[ACP] = (2E)-butenoyl-[ACP] + H2O', ''),\n",
       "    ('(3R)-hydroxyhexanoyl-[ACP] = (2E)-hexenoyl-[ACP] + H2O', ''),\n",
       "    ('(3R)-hydroxyoctanoyl-[ACP] = (2E)-octenoyl-[ACP] + H2O', ''),\n",
       "    ('(3R)-hydroxydecanoyl-[ACP] = (2E)-decenoyl-[ACP] + H2O', ''),\n",
       "    ('(3R)-hydroxydodecanoyl-[ACP] = (2E)-dodecenoyl-[ACP] + H2O', ''),\n",
       "    ('(3R)-hydroxytetradecanoyl-[ACP] = (2E)-tetradecenoyl-[ACP] + H2O', ''),\n",
       "    ('(3R)-hydroxyhexadecanoyl-[ACP] = (2E)-hexadecenoyl-[ACP] + H2O', ''),\n",
       "    ('(3R)-hydroxy-(9Z)-hexadecenoyl-[ACP] = (2E,9Z)-hexadecadienoyl-[ACP] + H2O',\n",
       "     ''),\n",
       "    ('(3R)-hydroxy-5-methylhexanoyl-[ACP] = (2E)-5-methylhexenoyl-[ACP] + H2O',\n",
       "     '')],\n",
       "   'Function': 'Involved in unsaturated fatty acids biosynthesis. Catalyzes the dehydration of short chain beta-hydroxyacyl-ACPs and long chain saturated and unsaturated beta-hydroxyacyl-ACPs'},\n",
       "  'seq': 'MTTNTHTLQIEEILELLPHRFPFLLVDRVLDFEEGRFLRAVKNVSVNEPFFQGHFPGKPIFPGVLILEAMAQATGILAFKSVGKLEPGELYYFAGIDEARFKRPVVPGDQMIMEVTFEKTRRGLTRFKGVALVDGKVVCEATMMCARSREA',\n",
       "  'mw': 17033.0},\n",
       " 'b0181': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A722',\n",
       "  'gene': 'b0181',\n",
       "  'locus': ['b0181', 'JW0176'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('a (3R)-hydroxyacyl-[ACP] + UDP-N-acetyl-alpha-D-glucosamine = a UDP-3-O-[(3R)-3-hydroxyacyl]-N-acetyl-alpha-D-glucosamine + holo-[ACP]',\n",
       "     '2.3.1.129')],\n",
       "   'Function': 'Involved in the biosynthesis of lipid A, a phosphorylated glycolipid that anchors the lipopolysaccharide to the outer membrane of the cell',\n",
       "   'Pathway': 'Glycolipid biosynthesis; lipid IV(A) biosynthesis; lipid IV(A) from (3R)-3-hydroxytetradecanoyl-[acyl-carrier-protein] and UDP-N-acetyl-alpha-D-glucosamine: step 1/6'},\n",
       "  'seq': 'MIDKSAFVHPTAIVEEGASIGANAHIGPFCIVGPHVEIGEGTVLKSHVVVNGHTKIGRDNEIYQFASIGEVNQDLKYAGEPTRVEIGDRNRIRESVTIHRGTVQGGGLTKVGSDNLLMINAHIAHDCTVGNRCILANNATLAGHVSVDDFAIIGGMTAVHQFCIIGAHVMVGGCSGVAQDVPPYVIAQGNHATPFGVNIEGLKRRGFSREAITAIRNAYKLIYRSGKTLDEVKPEIAELAETYPEVKAFTDFFARSTRGLIR',\n",
       "  'mw': 28080.0},\n",
       " 'b0182': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P10441',\n",
       "  'gene': 'b0182',\n",
       "  'locus': ['b0182', 'JW0177'],\n",
       "  'synonyms': 'pgsB',\n",
       "  'properties': {'Catalytic Activity': [('2-N,3-O-bis[(3R)-3-hydroxytetradecanoyl]-alpha-D-glucosaminyl 1-phosphate + UDP-2-N,3-O-bis[(3R)-3-hydroxytetradecanoyl]-alpha-D-glucosamine = H(+) + lipid A disaccharide (E. coli) + UDP',\n",
       "     ''),\n",
       "    ('a lipid X + a UDP-2-N,3-O-bis[(3R)-3-hydroxyacyl]-alpha-D-glucosamine = a lipid A disaccharide + H(+) + UDP',\n",
       "     '2.4.1.182')],\n",
       "   'Function': 'Condensation of UDP-2,3-diacylglucosamine and 2,3-diacylglucosamine-1-phosphate to form lipid A disaccharide, a precursor of lipid A, a phosphorylated glycolipid that anchors the lipopolysaccharide to the outer membrane of the cell',\n",
       "   'Pathway': 'Glycolipid biosynthesis; lipid IV(A) biosynthesis; lipid IV(A) from (3R)-3-hydroxytetradecanoyl-[acyl-carrier-protein] and UDP-N-acetyl-alpha-D-glucosamine: step 5/6'},\n",
       "  'seq': 'MTEQRPLTIALVAGETSGDILGAGLIRALKEHVPNARFVGVAGPRMQAEGCEAWYEMEELAVMGIVEVLGRLRRLLHIRADLTKRFGELKPDVFVGIDAPDFNITLEGNLKKQGIKTIHYVSPSVWAWRQKRVFKIGRATDLVLAFLPFEKAFYDKYNVPCRFIGHTMADAMPLDPDKNAARDVLGIPHDAHCLALLPGSRGAEVEMLSADFLKTAQLLRQTYPDLEIVVPLVNAKRREQFERIKAEVAPDLSVHLLDGMGREAMVASDAALLASGTAALECMLAKCPMVVGYRMKPFTFWLAKRLVKTDYVSLPNLLAGRELVKELLQEECEPQKLAAALLPLLANGKTSHAMHDTFRELHQQIRCNADEQAAQAVLELAQ',\n",
       "  'mw': 42382.0},\n",
       " 'b0185': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ABD5',\n",
       "  'gene': 'b0185',\n",
       "  'locus': ['b0185', 'JW0180'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('acetyl-CoA + N(6)-carboxybiotinyl-L-lysyl-[protein] = malonyl-CoA + N(6)-biotinyl-L-lysyl-[protein]',\n",
       "     '2.1.3.15')],\n",
       "   'Function': 'Component of the acetyl coenzyme A carboxylase (ACC) complex. First, biotin carboxylase catalyzes the carboxylation of biotin on its carrier protein (BCCP) and then the CO(2) group is transferred by the carboxyltransferase to acetyl-CoA to form malonyl-CoA',\n",
       "   'Regulation Activity': 'Competitively inhibited by pyrrolidine dione antibiotic moiramide B (CPD1)',\n",
       "   'Pathway': 'Lipid metabolism; malonyl-CoA biosynthesis; malonyl-CoA from acetyl-CoA: step 1/1'},\n",
       "  'seq': 'MSLNFLDFEQPIAELEAKIDSLTAVSRQDEKLDINIDEEVHRLREKSVELTRKIFADLGAWQIAQLARHPQRPYTLDYVRLAFDEFDELAGDRAYADDKAIVGGIARLDGRPVMIIGHQKGRETKEKIRRNFGMPAPEGYRKALRLMQMAERFKMPIITFIDTPGAYPGVGAEERGQSEAIARNLREMSRLGVPVVCTVIGEGGSGGALAIGVGDKVNMLQYSTYSVISPEGCASILWKSADKAPLAAEAMGIIAPRLKELKLIDSIIPEPLGGAHRNPEAMAASLKAQLLADLADLDVLSTEDLKNRRYQRLMSYGYA',\n",
       "  'mw': 35242.0},\n",
       " 'b0186': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P52095',\n",
       "  'gene': 'b0186',\n",
       "  'locus': ['b0186', 'JW0181'],\n",
       "  'synonyms': 'ldc ldcH',\n",
       "  'properties': {'Catalytic Activity': [('H(+) + L-lysine = cadaverine + CO2',\n",
       "     '4.1.1.18')],\n",
       "   'Function': 'Plays a role in lysine utilization by acting as a lysine decarboxylase'},\n",
       "  'seq': 'MNIIAIMGPHGVFYKDEPIKELESALVAQGFQIIWPQNSVDLLKFIEHNPRICGVIFDWDEYSLDLCSDINQLNEYLPLYAFINTHSTMDVSVQDMRMALWFFEYALGQAEDIAIRMRQYTDEYLDNITPPFTKALFTYVKERKYTFCTPGHMGGTAYQKSPVGCLFYDFFGGNTLKADVSISVTELGSLLDHTGPHLEAEEYIARTFGAEQSYIVTNGTSTSNKIVGMYAAPSGSTLLIDRNCHKSLAHLLMMNDVVPVWLKPTRNALGILGGIPRREFTRDSIEEKVAATTQAQWPVHAVITNSTYDGLLYNTDWIKQTLDVPSIHFDSAWVPYTHFHPIYQGKSGMSGERVAGKVIFETQSTHKMLAALSQASLIHIKGEYDEEAFNEAFMMHTTTSPSYPIVASVETAAAMLRGNPGKRLINRSVERALHFRKEVQRLREESDGWFFDIWQPPQVDEAECWPVAPGEQWHGFNDADADHMFLDPVKVTILTPGMDEQGNMSEEGIPAALVAKFLDERGIVVEKTGPYNLLFLFSIGIDKTKAMGLLRGLTEFKRSYDLNLRIKNMLPDLYAEDPDFYRNMRIQDLAQGIHKLIRKHDLPGLMLRAFDTLPEMIMTPHQAWQRQIKGEVETIALEQLVGRVSANMILPYPPGVPLLMPGEMLTKESRTVLDFLLMLCSVGQHYPGFETDIHGAKQDEDGVYRVRVLKMAG',\n",
       "  'mw': 80590.0},\n",
       " 'b0194': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P16659',\n",
       "  'gene': 'b0194',\n",
       "  'locus': ['b0194', 'JW0190'],\n",
       "  'synonyms': 'drpA',\n",
       "  'properties': {'Catalytic Activity': [('ATP + L-proline + tRNA(Pro) = AMP + diphosphate + L-prolyl-tRNA(Pro)',\n",
       "     '6.1.1.15')],\n",
       "   'Function': \"Catalyzes the attachment of proline to tRNA(Pro) in a two-step reaction: proline is first activated by ATP to form Pro-AMP and then transferred to the acceptor end of tRNA(Pro). As ProRS can inadvertently accommodate and process non-cognate amino acids such as alanine and cysteine, to avoid such errors it has two additional distinct editing activities against alanine. One activity is designated as 'pretransfer' editing and involves the tRNA(Pro)-independent hydrolysis of activated Ala-AMP. The other activity is designated 'posttransfer' editing and involves deacylation of mischarged Ala-tRNA(Pro). Misacylated Cys-tRNA(Pro) is not edited by ProRS, but instead may be edited in trans by YbaK\"},\n",
       "  'seq': 'MRTSQYLLSTLKETPADAEVISHQLMLRAGMIRKLASGLYTWLPTGVRVLKKVENIVREEMNNAGAIEVSMPVVQPADLWQESGRWEQYGPELLRFVDRGERPFVLGPTHEEVITDLIRNELSSYKQLPLNFYQIQTKFRDEVRPRFGVMRSREFLMKDAYSFHTSQESLQETYDAMYAAYSKIFSRMGLDFRAVQADTGSIGGSASHEFQVLAQSGEDDVVFSDTSDYAANIELAEAIAPKEPRAAATQEMTLVDTPNAKTIAELVEQFNLPIEKTVKTLLVKAVEGSSFPQVALLVRGDHELNEVKAEKLPQVASPLTFATEEEIRAVVKAGPGSLGPVNMPIPVVIDRTVAAMSDFAAGANIDGKHYFGINWDRDVATPEVADIRNVVAGDPSPDGQGRLLIKRGIEVGHIFQLGTKYSEALKASVQGEDGRNQILTMGCYGIGVTRVVAAAIEQNYDERGIVWPDAIAPFQVAILPMNMHKSFRVQELAEKLYSELRAQGIEVLLDDRKERPGVMFADMELIGIPHTIVLGDRNLDNDDIEYKYRRNGEKQLIKTGDIVEYLVKQIKG',\n",
       "  'mw': 63693.0},\n",
       " 'b0197': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P28635',\n",
       "  'gene': 'b0197',\n",
       "  'locus': ['b0197', 'JW0193'],\n",
       "  'synonyms': 'yaeC',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'This protein is a component of a D-methionine permease, a binding protein-dependent, ATP-driven transport system'},\n",
       "  'seq': 'MAFKFKTFAAVGALIGSLALVGCGQDEKDPNHIKVGVIVGAEQQVAEVAQKVAKDKYGLDVELVTFNDYVLPNEALSKGDIDANAFQHKPYLDQQLKDRGYKLVAVGNTFVYPIAGYSKKIKSLDELQDGSQVAVPNDPTNLGRSLLLLQKVGLIKLKDGVGLLPTVLDVVENPKNLKIVELEAPQLPRSLDDAQIALAVINTTYASQIGLTPAKDGIFVEDKESPYVNLIVTREDNKDAENVKKFVQAYQSDEVYEAANKVFNGGAVKGW',\n",
       "  'mw': 29432.0},\n",
       " 'b0198': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P31547',\n",
       "  'gene': 'b0198',\n",
       "  'locus': ['b0198', 'JW0194'],\n",
       "  'synonyms': 'yaeE',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': '(Microbial infection) Probably transports the toxic C-terminal region of CdiA from E.coli strain MHI813 across the inner membrane to the cytoplasm, where CdiA has a toxic effect. Toxin transport is strain-specific, mutations in this gene do not confer resistance to several other tested CdiA toxins'},\n",
       "  'seq': 'MSEPMMWLLVRGVWETLAMTFVSGFFGFVIGLPVGVLLYVTRPGQIIANAKLYRTVSAIVNIFRSIPFIILLVWMIPFTRVIVGTSIGLQAAIVPLTVGAAPFIARMVENALLEIPTGLIEASRAMGATPMQIVRKVLLPEALPGLVNAATITLITLVGYSAMGGAVGAGGLGQIGYQYGYIGYNATVMNTVLVLLVILVYLIQFAGDRIVRAVTRK',\n",
       "  'mw': 23256.0},\n",
       " 'b0199': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P30750',\n",
       "  'gene': 'b0199',\n",
       "  'locus': ['b0199', 'JW0195'],\n",
       "  'synonyms': 'abc',\n",
       "  'properties': {'Catalytic Activity': [('ATP + H2O + L-methionine(out) = ADP + H(+) + L-methionine(in) + phosphate',\n",
       "     '7.4.2.11'),\n",
       "    ('ATP + D-methionine(out) + H2O = ADP + D-methionine(in) + H(+) + phosphate',\n",
       "     '7.4.2.11')],\n",
       "   'Function': 'Part of the ABC transporter complex MetNIQ involved in methionine import (PubMed:12169620, PubMed:12218041, PubMed:12819857, PubMed:18621668, PubMed:30352853). Responsible for energy coupling to the transport system (PubMed:18621668, PubMed:25678706). It has also been shown to be involved in formyl-L-methionine transport (PubMed:12819857)',\n",
       "   'Regulation Activity': 'ATPase activity is inhibited by intracellular L-methionine (PubMed:18621668, PubMed:22095702, PubMed:25678706, PubMed:30352853). Binding of methionine to the dimerized C-terminal regulatory domain stabilizes an inward-facing, ATPase-inactive conformation of the transporter, and as a consequence, the rate of ATP hydrolysis decreases (PubMed:18621668, PubMed:22095702, PubMed:30352853). ADP is a competitive inhibitor (PubMed:25678706)'},\n",
       "  'seq': 'MIKLSNITKVFHQGTRTIQALNNVSLHVPAGQIYGVIGASGAGKSTLIRCVNLLERPTEGSVLVDGQELTTLSESELTKARRQIGMIFQHFNLLSSRTVFGNVALPLELDNTPKDEVKRRVTELLSLVGLGDKHDSYPSNLSGGQKQRVAIARALASNPKVLLCDEATSALDPATTRSILELLKDINRRLGLTILLITHEMDVVKRICDCVAVISNGELIEQDTVSEVFSHPKTPLAQKFIQSTLHLDIPEDYQERLQAEPFTDCVPMLRLEFTGQSVDAPLLSETARRFNVNNNIISAQMDYAGGVKFGIMLTEMHGTQQDTQAAIAWLQEHHVKVEVLGYV',\n",
       "  'mw': 37788.0},\n",
       " 'b0200': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P63228',\n",
       "  'gene': 'b0200',\n",
       "  'locus': ['b0200', 'JW0196'],\n",
       "  'synonyms': 'yaeD',\n",
       "  'properties': {'Catalytic Activity': [('D-glycero-beta-D-manno-heptose 1,7-bisphosphate + H2O = D-glycero-beta-D-manno-heptose 1-phosphate + phosphate',\n",
       "     '3.1.3.82')],\n",
       "   'Function': 'Converts the D-glycero-beta-D-manno-heptose 1,7-bisphosphate (beta-HBP) intermediate into D-glycero-beta-D-manno-heptose 1-phosphate by removing the phosphate group at the C-7 position',\n",
       "   'Pathway': 'Bacterial outer membrane biogenesis; LPS core biosynthesis'},\n",
       "  'seq': 'MAKSVPAIFLDRDGTINVDHGYVHEIDNFEFIDGVIDAMRELKKMGFALVVVTNQSGIARGKFTEAQFETLTEWMDWSLADRDVDLDGIYYCPHHPQGSVEEFRQVCDCRKPHPGMLLSARDYLHIDMAASYMVGDKLEDMQAAVAANVGTKVLVRTGKPITPEAENAADWVLNSLADLPQAIKKQQKPAQ',\n",
       "  'mw': 21294.0},\n",
       " 'b0207': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P30863',\n",
       "  'gene': 'b0207',\n",
       "  'locus': ['b0207', 'JW0197'],\n",
       "  'synonyms': 'yafB',\n",
       "  'properties': {'Catalytic Activity': [('2-dehydro-L-idonate + NADP(+) = 2,5-didehydro-D-gluconate + H(+) + NADPH',\n",
       "     '1.1.1.346')],\n",
       "   'Function': 'Catalyzes the reduction of 2,5-diketo-D-gluconic acid (25DKG) to 2-keto-L-gulonic acid (2KLG)'},\n",
       "  'seq': 'MAIPAFGLGTFRLKDDVVISSVITALELGYRAIDTAQIYDNEAAVGQAIAESGVPRHELYITTKIWIENLSKDKLIPSLKESLQKLRTDYVDLTLIHWPSPNDEVSVEEFMQALLEAKKQGLTREIGISNFTIPLMEKAIAAVGAENIATNQIELSPYLQNRKVVAWAKQHGIHITSYMTLAYGKALKDEVIARIAAKHNATPAQVILAWAMGEGYSVIPSSTKRKNLESNLKAQNLQLDAEDKKAIAALDCNDRLVSPEGLAPEWD',\n",
       "  'mw': 29437.0},\n",
       " 'b0212': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AC84',\n",
       "  'gene': 'b0212',\n",
       "  'locus': ['b0212', 'JW0202'],\n",
       "  'synonyms': 'yafR',\n",
       "  'properties': {'Catalytic Activity': [('an S-(2-hydroxyacyl)glutathione + H2O = a 2-hydroxy carboxylate + glutathione + H(+)',\n",
       "     '3.1.2.6'),\n",
       "    ('(R)-S-lactoylglutathione + H2O = (R)-lactate + glutathione + H(+)',\n",
       "     '3.1.2.6')],\n",
       "   'Function': 'Type II glyoxalase that catalyzes the hydrolysis of (R)-S-lactoylglutathione to (R)-lactate and glutathione (PubMed:25670698, PubMed:17196158). Is more efficient than the isozyme GloC, and plays a major contribution to methylglyoxal (MG) detoxification in E.coli (PubMed:25670698). The two isoenzymes have additive effects and ensure maximal MG degradation (PubMed:25670698)',\n",
       "   'Regulation Activity': 'Is inhibited by Cu(2+)',\n",
       "   'Pathway': 'Secondary metabolite metabolism; methylglyoxal degradation; (R)-lactate from methylglyoxal: step 2/2'},\n",
       "  'seq': 'MNLNSIPAFDDNYIWVLNDEAGRCLIVDPGDAEPVLNAIAANNWQPEAIFLTHHHHDHVGGVKELVEKFPQIVVYGPQETQDKGTTQVVKDGETAFVLGHEFSVIATPGHTLGHICYFSKPYLFCGDTLFSGGCGRLFEGTASQMYQSLKKLSALPDDTLVCCAHEYTLSNMKFALSILPHDLSINDYYRKVKELRAKNQITLPVILKNERQINVFLRTEDIDLINVINEETLLQQPEERFAWLRSKKDRF',\n",
       "  'mw': 28434.0},\n",
       " 'b0221': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'Q47146',\n",
       "  'gene': 'b0221',\n",
       "  'locus': ['b0221', 'JW5020'],\n",
       "  'synonyms': 'yafH',\n",
       "  'properties': {'Catalytic Activity': [('a medium-chain 2,3-saturated fatty acyl-CoA + H(+) + oxidized [electron-transfer flavoprotein] = a medium-chain (2E)-enoyl-CoA + reduced [electron-transfer flavoprotein]',\n",
       "     '1.3.8.7'),\n",
       "    ('a long-chain 2,3-saturated fatty acyl-CoA + H(+) + oxidized [electron-transfer flavoprotein] = a long-chain (2E)-enoyl-CoA + reduced [electron-transfer flavoprotein]',\n",
       "     '1.3.8.8')],\n",
       "   'Function': 'Catalyzes the dehydrogenation of acyl-coenzymes A (acyl-CoAs) to 2-enoyl-CoAs, the first step of the beta-oxidation cycle of fatty acid degradation. Is required for E.coli to utilize dodecanoate or oleate as the sole carbon and energy source for growth',\n",
       "   'Pathway': 'Lipid metabolism; fatty acid beta-oxidation'},\n",
       "  'seq': 'MMILSILATVVLLGALFYHRVSLFISSLILLAWTAALGVAGLWSAWVLVPLAIILVPFNFAPMRKSMISAPVFRGFRKVMPPMSRTEKEAIDAGTTWWEGDLFQGKPDWKKLHNYPQPRLTAEEQAFLDGPVEEACRMANDFQITHELADLPPELWAYLKEHRFFAMIIKKEYGGLEFSAYAQSRVLQKLSGVSGILAITVGVPNSLGPGELLQHYGTDEQKDHYLPRLARGQEIPCFALTSPEAGSDAGAIPDTGIVCMGEWQGQQVLGMRLTWNKRYITLAPIATVLGLAFKLSDPEKLLGGAEDLGITCALIPTTTPGVEIGRRHFPLNVPFQNGPTRGKDVFVPIDYIIGGPKMAGQGWRMLVECLSVGRGITLPSNSTGGVKSVALATGAYAHIRRQFKISIGKMEGIEEPLARIAGNAYVMDAAASLITYGIMLGEKPAVLSAIVKYHCTHRGQQSIIDAMDITGGKGIMLGQSNFLARAYQGAPIAITVEGANILTRSMMIFGQGAIRCHPYVLEEMEAAKNNDVNAFDKLLFKHIGHVGSNKVRSFWLGLTRGLTSSTPTGDATKRYYQHLNRLSANLALLSDVSMAVLGGSLKRRERISARLGDILSQLYLASAVLKRYDDEGRNEADLPLVHWGVQDALYQAEQAMDDLLQNFPNRVVAGLLNVVIFPTGRHYLAPSDKLDHKVAKILQVPNATRSRIGRGQYLTPSEHNPVGLLEEALVDVIAADPIHQRICKELGKNLPFTRLDELAHNALVKGLIDKDEAAILVKAEESRLRSINVDDFDPEELATKPVKLPEKVRKVEAA',\n",
       "  'mw': 89224.0},\n",
       " 'b0222': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P63224',\n",
       "  'gene': 'b0222',\n",
       "  'locus': ['b0222', 'JW0212'],\n",
       "  'synonyms': 'lpcA tfrA yafI',\n",
       "  'properties': {'Catalytic Activity': [('2 D-sedoheptulose 7-phosphate = D-glycero-alpha-D-manno-heptose 7-phosphate + D-glycero-beta-D-manno-heptose 7-phosphate',\n",
       "     '5.3.1.28')],\n",
       "   'Function': 'Catalyzes the isomerization of sedoheptulose 7-phosphate in D-glycero-D-manno-heptose 7-phosphate',\n",
       "   'Pathway': 'Bacterial outer membrane biogenesis; LPS core biosynthesis'},\n",
       "  'seq': 'MYQDLIRNELNEAAETLANFLKDDANIHAIQRAAVLLADSFKAGGKVLSCGNGGSHCDAMHFAEELTGRYRENRPGYPAIAISDVSHISCVGNDFGFNDIFSRYVEAVGREGDVLLGISTSGNSANVIKAIAAAREKGMKVITLTGKDGGKMAGTADIEIRVPHFGYADRIQEIHIKVIHILIQLIEKEMVK',\n",
       "  'mw': 20815.0},\n",
       " 'b0237': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P15288',\n",
       "  'gene': 'b0237',\n",
       "  'locus': ['b0237', 'JW0227'],\n",
       "  'synonyms': 'pepH',\n",
       "  'properties': {'Catalytic Activity': [('Hydrolysis of dipeptides, preferentially hydrophobic dipeptides including prolyl amino acids.',\n",
       "     '3.4.13.18')],\n",
       "   'Function': 'Dipeptidase with broad substrate specificity. Requires dipeptide substrates with an unblocked N-terminus and the amino group in the alpha or beta position. Non-protein amino acids and proline are not accepted in the C-terminal position, whereas some dipeptide amides and formyl amino acids are hydrolyzed. Also shows cysteinylglycinase activity, which is sufficient for E.coli to utilize cysteinylglycine as a cysteine source',\n",
       "   'Regulation Activity': 'Inhibited by metal chelators'},\n",
       "  'seq': 'MSELSQLSPQPLWDIFAKICSIPHPSYHEEQLAEYIVGWAKEKGFHVERDQVGNILIRKPATAGMENRKPVVLQAHLDMVPQKNNDTVHDFTKDPIQPYIDGEWVKARGTTLGADNGIGMASALAVLADENVVHGPLEVLLTMTEEAGMDGAFGLQGNWLQADILINTDSEEEGEIYMGCAGGIDFTSNLHLDREAVPAGFETFKLTLKGLKGGHSGGEIHVGLGNANKLLVRFLAGHAEELDLRLIDFNGGTLRNAIPREAFATIAVAADKVDVLKSLVNTYQEILKNELAEKEKNLALLLDSVANDKAALIAKSRDTFIRLLNATPNGVIRNSDVAKGVVETSLNVGVVTMTDNNVEIHCLIRSLIDSGKDYVVSMLDSLGKLAGAKTEAKGAYPGWQPDANSPVMHLVRETYQRLFNKTPNIQIIHAGLECGLFKKPYPEMDMVSIGPTITGPHSPDEQVHIESVGHYWTLLTELLKEIPAK',\n",
       "  'mw': 52915.0},\n",
       " 'b0238': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A9M5',\n",
       "  'gene': 'b0238',\n",
       "  'locus': ['b0238', 'JW0228'],\n",
       "  'synonyms': 'gpp gxu',\n",
       "  'properties': {'Catalytic Activity': [('diphosphate + GMP = 5-phospho-alpha-D-ribose 1-diphosphate + guanine',\n",
       "     ''),\n",
       "    ('diphosphate + XMP = 5-phospho-alpha-D-ribose 1-diphosphate + xanthine',\n",
       "     '2.4.2.22'),\n",
       "    ('diphosphate + IMP = 5-phospho-alpha-D-ribose 1-diphosphate + hypoxanthine',\n",
       "     '')],\n",
       "   'Function': \"Purine salvage pathway enzyme that catalyzes the transfer of the ribosyl-5-phosphate group from 5-phospho-alpha-D-ribose 1-diphosphate (PRPP) to the N9 position of the 6-oxopurines guanine and xanthine to form the corresponding ribonucleotides GMP (guanosine 5'-monophosphate) and XMP (xanthosine 5'-monophosphate), with the release of PPi. To a lesser extent, also acts on hypoxanthine (PubMed:9100006, PubMed:3886014). Does not ribophosphorylate adenine (PubMed:3886014)\",\n",
       "   'Regulation Activity': 'Inhibited by thioguanine, GMP and, to a lesser extent, by thioxanthine, azaxanthine and azaguanine (PubMed:3886014). Highly inhibited by nucleoside phosphonates, which are product analogs (PubMed:23927482)',\n",
       "   'Pathway': 'Purine metabolism; XMP biosynthesis via salvage pathway; XMP from xanthine: step 1/1'},\n",
       "  'seq': 'MSEKYIVTWDMLQIHARKLASRLMPSEQWKGIIAVSRGGLVPGALLARELGIRHVDTVCISSYDHDNQRELKVLKRAEGDGEGFIVIDDLVDTGGTAVAIREMYPKAHFVTIFAKPAGRPLVDDYVVDIPQDTWIEQPWDMGVVFVPPISGR',\n",
       "  'mw': 16971.0},\n",
       " 'b0241': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P02932',\n",
       "  'gene': 'b0241',\n",
       "  'locus': ['b0241', 'JW0231'],\n",
       "  'synonyms': 'ompE',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Uptake of inorganic phosphate, phosphorylated compounds, and some other negatively charged solutes'},\n",
       "  'seq': 'MKKSTLALVVMGIVASASVQAAEIYNKDGNKLDVYGKVKAMHYMSDNASKDGDQSYIRFGFKGETQINDQLTGYGRWEAEFAGNKAESDTAQQKTRLAFAGLKYKDLGSFDYGRNLGALYDVEAWTDMFPEFGGDSSAQTDNFMTKRASGLATYRNTDFFGVIDGLNLTLQYQGKNENRDVKKQNGDGFGTSLTYDFGGSDFAISGAYTNSDRTNEQNLQSRGTGKRAEAWATGLKYDANNIYLATFYSETRKMTPITGGFANKTQNFEAVAQYQFDFGLRPSLGYVLSKGKDIEGIGDEDLVNYIDVGATYYFNKNMSAFVDYKINQLDSDNKLNINNDDIVAVGMTYQF',\n",
       "  'mw': 38922.0},\n",
       " 'b0242': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A7B5',\n",
       "  'gene': 'b0242',\n",
       "  'locus': ['b0242', 'JW0232'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + L-glutamate = ADP + L-glutamyl 5-phosphate',\n",
       "     '2.7.2.11')],\n",
       "   'Function': 'Catalyzes the transfer of a phosphate group to glutamate to form L-glutamate 5-phosphate',\n",
       "   'Regulation Activity': 'Interaction with gamma-glutamyl phosphate reductase (proA) seems necessary for kinase activity. Requires free Mg(2+). Inhibited by proline and ADP',\n",
       "   'Pathway': 'Amino-acid biosynthesis; L-proline biosynthesis; L-glutamate 5-semialdehyde from L-glutamate: step 1/2'},\n",
       "  'seq': 'MSDSQTLVVKLGTSVLTGGSRRLNRAHIVELVRQCAQLHAAGHRIVIVTSGAIAAGREHLGYPELPATIASKQLLAAVGQSRLIQLWEQLFSIYGIHVGQMLLTRADMEDRERFLNARDTLRALLDNNIVPVINENDAVATAEIKVGDNDNLSALAAILAGADKLLLLTDQKGLYTADPRSNPQAELIKDVYGIDDALRAIAGDSVSGLGTGGMSTKLQAADVACRAGIDTIIAAGSKPGVIGDVMEGISVGTLFHAQATPLENRKRWIFGAPPAGEITVDEGATAAILERGSSLLPKGIKSVTGNFSRGEVIRICNLEGRDIAHGVSRYNSDALRRIAGHHSQEIDAILGYEYGPVAVHRDDMITR',\n",
       "  'mw': 39057.0},\n",
       " 'b0243': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P07004',\n",
       "  'gene': 'b0243',\n",
       "  'locus': ['b0243', 'JW0233'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('L-glutamate 5-semialdehyde + NADP(+) + phosphate = H(+) + L-glutamyl 5-phosphate + NADPH',\n",
       "     '1.2.1.41')],\n",
       "   'Function': 'Catalyzes the NADPH-dependent reduction of L-glutamate 5-phosphate into L-glutamate 5-semialdehyde and phosphate. The product spontaneously undergoes cyclization to form 1-pyrroline-5-carboxylate',\n",
       "   'Regulation Activity': 'Inhibited by p-chloromercuribenzoate, copper and nickel ions',\n",
       "   'Pathway': 'Amino-acid biosynthesis; L-proline biosynthesis; L-glutamate 5-semialdehyde from L-glutamate: step 2/2'},\n",
       "  'seq': 'MLEQMGIAAKQASYKLAQLSSREKNRVLEKIADELEAQSEIILNANAQDVADARANGLSEAMLDRLALTPARLKGIADDVRQVCNLADPVGQVIDGGVLDSGLRLERRRVPLGVIGVIYEARPNVTVDVASLCLKTGNAVILRGGKETCRTNAATVAVIQDALKSCGLPAGAVQAIDNPDRALVSEMLRMDKYIDMLIPRGGAGLHKLCREQSTIPVITGGIGVCHIYVDESVEIAEALKVIVNAKTQRPSTCNTVETLLVNKNIADSFLPALSKQMAESGVTLHADAAALAQLQAGPAKVVAVKAEEYDDEFLSLDLNVKIVSDLDDAIAHIREHGTQHSDAILTRDMRNAQRFVNEVDSSAVYVNASTRFTDGGQFGLGAEVAVSTQKLHARGPMGLEALTTYKWIGIGDYTIRA',\n",
       "  'mw': 44630.0},\n",
       " 'b0260': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'Q47689',\n",
       "  'gene': 'b0260',\n",
       "  'locus': ['b0260', 'JW5027'],\n",
       "  'synonyms': 'ykfD',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Transporter for the intake of S-methylmethionine'},\n",
       "  'seq': 'MQTTQQNAPLKRTMKTRHLIMLSLGGVIGTGLFFNTGYIISTTGAAGTLLAYLIGALVVWLVMQCLGELSVAMPETGAFHVYAARYLGPATGYTVAWLYWLTWTVALGSSFTAAGFCMQYWFPQVPVWVWCVVFCAIIFGLNVISTRFFAEGEFWFSLVKVVTIIAFIILGGAAIFGFIPMQDGSPAPGLSNITAEGWFPHGGLPILMTMVAVNFAFSGTELIGIAAGETENPRKVIPVAIRTTIARLIIFFIGTVFVLAALIPMQQVGVEKSPFVLVFEKVGIPYAADIFNFVILTAILSAANSGLYASGRMLWSLSNERTLPACFARVTKNGVPLTALSVSMLGGVLALFSSVVAPDTVFVALSAISGFAVVAVWLSICASHFVFRRRHLQQGKALSELHYRAPWYPLVPVLGFVLCLVACVGLAFDPAQRIALWCGLPFVALCYGAYFLTQPRNAKQEPEHVAE',\n",
       "  'mw': 50526.0},\n",
       " 'b0261': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'Q47690',\n",
       "  'gene': 'b0261',\n",
       "  'locus': ['b0261', 'JW0253'],\n",
       "  'synonyms': 'yagD',\n",
       "  'properties': {'Catalytic Activity': [('L-homocysteine + S-methyl-L-methionine = H(+) + 2 L-methionine',\n",
       "     '2.1.1.10')],\n",
       "   'Function': 'Catalyzes methyl transfer from S-methylmethionine or S-adenosylmethionine (less efficient) to homocysteine, selenohomocysteine and less efficiently selenocysteine'},\n",
       "  'seq': 'MSQNNPLRALLDKQDILLLDGAMATELEARGCNLADSLWSAKVLVENPELIREVHLDYYRAGAQCAITASYQATPAGFAARGLDEAQSKALIGKSVELARKAREAYLAENPQAGTLLVAGSVGPYGAYLADGSEYRGDYHCSVEAFQAFHRPRVEALLDAGADLLACETLPNFSEIEALAELLTAYPRARAWFSFTLRDSEHLSDGTPLRDVVALLAGYPQVVALGINCIALENTTAALQHLHGLTVLPLVVYPNSGEHYDAVSKTWHHHGEHCAQLADYLPQWQAAGARLIGGCCRTTPADIAALKARS',\n",
       "  'mw': 33423.0},\n",
       " 'b0273': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P06960',\n",
       "  'gene': 'b0273',\n",
       "  'locus': ['b0273', 'JW0266'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('carbamoyl phosphate + L-ornithine = H(+) + L-citrulline + phosphate',\n",
       "     '2.1.3.3')],\n",
       "   'Function': 'Reversibly catalyzes the transfer of the carbamoyl group from carbamoyl phosphate (CP) to the N(epsilon) atom of ornithine (ORN) to produce L-citrulline, which is a substrate for argininosuccinate synthetase, the enzyme involved in the final step in arginine biosynthesis',\n",
       "   'Pathway': 'Amino-acid biosynthesis; L-arginine biosynthesis; L-arginine from L-ornithine and carbamoyl phosphate: step 1/3'},\n",
       "  'seq': 'MSDLYKKHFLKLLDFTPAQFTSLLTLAAQLKADKKNGKEVQKLTGKNIALIFEKDSTRTRCSFEVAAFDQGARVTYLGPSGSQIGHKESIKDTARVLGRMYDGIQYRGHGQEVVETLAQYAGVPVWNGLTNEFHPTQLLADLMTMQEHLPGKAFNEMTLVYAGDARNNMGNSMLEAAALTGLDLRLLAPKACWPEESLVAECSALAEKHGGKITLTEDVAAGVKGADFIYTDVWVSMGEAKEKWAERIALLRGYQVNAQMMALTDNPNVKFLHCLPAFHDDQTTLGKQMAKEFDLHGGMEVTDEVFESAASIVFDQAENRMHTIKAVMMATLGE',\n",
       "  'mw': 36827.0},\n",
       " 'b0312': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P17445',\n",
       "  'gene': 'b0312',\n",
       "  'locus': ['b0312', 'JW0304'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('betaine aldehyde + H2O + NAD(+) = glycine betaine + 2 H(+) + NADH',\n",
       "     '1.2.1.8'),\n",
       "    ('betaine aldehyde + H2O + NADP(+) = glycine betaine + 2 H(+) + NADPH',\n",
       "     '')],\n",
       "   'Function': 'Involved in the biosynthesis of the osmoprotectant glycine betaine. Catalyzes the irreversible oxidation of betaine aldehyde to the corresponding acid. It is highly specific for betaine and has a significantly higher affinity for NAD than for NADP',\n",
       "   'Regulation Activity': 'Aldehydes with apolar groups are the most effective inhibitors. N-Methylated compounds are also inhibitory, but to a much lesser degree than the aldehydes. The alcohols are more inhibitory than the corresponding acids, and the inhibitory effect increases with the degree of methylation from ethanolamine to choline',\n",
       "   'Pathway': 'Amine and polyamine biosynthesis; betaine biosynthesis via choline pathway; betaine from betaine aldehyde: step 1/1'},\n",
       "  'seq': 'MSRMAEQQLYIHGGYTSATSGRTFETINPANGNVLATVQAAGREDVDRAVKSAQQGQKIWASMTAMERSRILRRAVDILRERNDELAKLETLDTGKAYSETSTVDIVTGADVLEYYAGLIPALEGSQIPLRETSFVYTRREPLGVVAGIGAWNYPIQIALWKSAPALAAGNAMIFKPSEVTPLTALKLAEIYSEAGLPDGVFNVLPGVGAETGQYLTEHPGIAKVSFTGGVASGKKVMANSAASSLKEVTMELGGKSPLIVFDDADLDLAADIAMMANFFSSGQVCTNGTRVFVPAKCKAAFEQKILARVERIRAGDVFDPQTNFGPLVSFPHRDNVLRYIAKGKEEGARVLCGGDVLKGDGFDNGAWVAPTVFTDCSDDMTIVREEIFGPVMSILTYESEDEVIRRANDTDYGLAAGIVTADLNRAHRVIHQLEAGICWINTWGESPAEMPVGGYKHSGIGRENGVMTLQSYTQVKSIQVEMAKFQSIF',\n",
       "  'mw': 52911.0},\n",
       " 'b0314': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ABC9',\n",
       "  'gene': 'b0314',\n",
       "  'locus': ['b0314', 'JW0306'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('choline(in) + H(+)(in) = choline(out) + H(+)(out)',\n",
       "     '')],\n",
       "   'Function': 'High-affinity uptake of choline driven by a proton-motive force',\n",
       "   'Regulation Activity': 'The choline uptake activity is completely inhibited by the protonophore carbonyl cyanide 4-(trifluoromethoxy)phenylhydrazone (FCCP)',\n",
       "   'Pathway': 'Amine and polyamine biosynthesis; betaine biosynthesis via choline pathway'},\n",
       "  'seq': 'MTDLSHSREKDKINPVVFYTSAGLILLFSLTTILFRDFSALWIGRTLDWVSKTFGWYYLLAATLYIVFVVCIACSRFGSVKLGPEQSKPEFSLLSWAAMLFAAGIGIDLMFFSVAEPVTQYMQPPEGAGQTIEAARQAMVWTLFHYGLTGWSMYALMGMALGYFSYRYNLPLTIRSALYPIFGKRINGPIGHSVDIAAVIGTIFGIATTLGIGVVQLNYGLSVLFDIPDSMAAKAALIALSVIIATISVTSGVDKGIRVLSELNVALALGLILFVLFMGDTSFLLNALVLNVGDYVNRFMGMTLNSFAFDRPVEWMNNWTLFFWAWWVAWSPFVGLFLARISRGRTIRQFVLGTLIIPFTFTLLWLSVFGNSALYEIIHGGAAFAEEAMVHPERGFYSLLAQYPAFTFSASVATITGLLFYVTSADSGALVLGNFTSQLKDINSDAPGWLRVFWSVAIGLLTLGMLMTNGISALQNTTVIMGLPFSFVIFFVMAGLYKSLKVEDYRRESANRDTAPRPLGLQDRLSWKKRLSRLMNYPGTRYTKQMMETVCYPAMEEVAQELRLRGAYVELKSLPPEEGQQLGHLDLLVHMGEEQNFVYQIWPQQYSVPGFTYRARSGKSTYYRLETFLLEGSQGNDLMDYSKEQVITDILDQYERHLNFIHLHREAPGHSVMFPDA',\n",
       "  'mw': 75842.0},\n",
       " 'b0323': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77624',\n",
       "  'gene': 'b0323',\n",
       "  'locus': ['b0323', 'JW0315'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': []},\n",
       "  'seq': 'MKELVVVAIGGNSIIKDNASQSIEHQAEAVKAVADTVLEMLASDYDIVLTHGNGPQVGLDLRRAEIAHKREGLPLTPLANCVADTQGGIGYLIQQALNNRLARHGEKKAVTVVTQVEVDKNDPGFAHPTKPIGAFFSDSQRDELQKANPDWCFVEDAGRGYRRVVASPEPKRIVEAPAIKALIQQGFVVIGAGGGGIPVVRTDAGDYQSVDAVIDKDLSTALLAREIHADILVITTGVEKVCIHFGKPQQQALDRVDIATMTRYMQEGHFPPGSMLPKIIASLTFLEQGGKEVIITTPECLPAALRGETGTHIIKT',\n",
       "  'mw': 33931.0},\n",
       " 'b0331': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77541',\n",
       "  'gene': 'b0331',\n",
       "  'locus': ['b0331', 'JW0323'],\n",
       "  'synonyms': 'yahQ',\n",
       "  'properties': {'Catalytic Activity': [('(2S,3R)-3-hydroxybutane-1,2,3-tricarboxylate = pyruvate + succinate',\n",
       "     '4.1.3.30')],\n",
       "   'Function': 'Involved in the catabolism of short chain fatty acids (SCFA) via the 2-methylcitrate cycle I (propionate degradation route). Catalyzes the thermodynamically favored C-C bond cleavage of (2R,3S)-2-methylisocitrate to yield pyruvate and succinate via an alpha-carboxy-carbanion intermediate',\n",
       "   'Pathway': 'Organic acid metabolism; propanoate degradation'},\n",
       "  'seq': 'MSLHSPGKAFRAALTKENPLQIVGTINANHALLAQRAGYQAIYLSGGGVAAGSLGLPDLGISTLDDVLTDIRRITDVCSLPLLVDADIGFGSSAFNVARTVKSMIKAGAAGLHIEDQVGAKRCGHRPNKAIVSKEEMVDRIRAAVDAKTDPDFVIMARTDALAVEGLDAAIERAQAYVEAGAEMLFPEAITELAMYRQFADAVQVPILANITEFGATPLFTTDELRSAHVAMALYPLSAFRAMNRAAEHVYNVLRQEGTQKSVIDTMQTRNELYESINYYQYEEKLDNLFARSQVK',\n",
       "  'mw': 32135.0},\n",
       " 'b0333': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P31660',\n",
       "  'gene': 'b0333',\n",
       "  'locus': ['b0333', 'JW0324'],\n",
       "  'synonyms': 'yahS yzzD',\n",
       "  'properties': {'Catalytic Activity': [('H2O + oxaloacetate + propanoyl-CoA = (2S,3S)-2-methylcitrate + CoA + H(+)',\n",
       "     '2.3.3.5'),\n",
       "    ('acetyl-CoA + H2O + oxaloacetate = citrate + CoA + H(+)', '2.3.3.16')],\n",
       "   'Function': 'Involved in the catabolism of short chain fatty acids (SCFA) via the tricarboxylic acid (TCA)(acetyl degradation route) and via the 2-methylcitrate cycle I (propionate degradation route). Catalyzes the Claisen condensation of propionyl-CoA and oxaloacetate (OAA) to yield 2-methylcitrate (2-MC) and CoA. Also catalyzes the condensation of oxaloacetate with acetyl-CoA to yield citrate but with a lower specificity',\n",
       "   'Pathway': 'Carbohydrate metabolism; tricarboxylic acid cycle; isocitrate from oxaloacetate: step 1/2'},\n",
       "  'seq': 'MSDTTILQNSTHVIKPKKSVALSGVPAGNTALCTVGKSGNDLHYRGYDILDLAKHCEFEEVAHLLIHGKLPTRDELAAYKTKLKALRGLPANVRTVLEALPAASHPMDVMRTGVSALGCTLPEKEGHTVSGARDIADKLLASLSSILLYWYHYSHNGERIQPETDDDSIGGHFLHLLHGEKPSQSWEKAMHISLVLYAEHEFNASTFTSRVIAGTGSDMYSAIIGAIGALRGPKHGGANEVSLEIQQRYETPDEAEADIRKRVENKEVVIGFGHPVYTIADPRHQVIKRVAKQLSQEGGSLKMYNIADRLETVMWESKKMFPNLDWFSAVSYNMMGVPTEMFTPLFVIARVTGWAAHIIEQRQDNKIIRPSANYVGPEDRPFVALDKRQ',\n",
       "  'mw': 43102.0},\n",
       " 'b0334': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77243',\n",
       "  'gene': 'b0334',\n",
       "  'locus': ['b0334', 'JW0325'],\n",
       "  'synonyms': 'yahT',\n",
       "  'properties': {'Catalytic Activity': [('(2S,3S)-2-methylcitrate = 2-methyl-cis-aconitate + H2O',\n",
       "     '4.2.1.79'),\n",
       "    ('citrate = D-threo-isocitrate', '4.2.1.3')],\n",
       "   'Function': 'Involved in the catabolism of short chain fatty acids (SCFA) via the tricarboxylic acid (TCA)(acetyl degradation route) and via the 2-methylcitrate cycle I (propionate degradation route). Catalyzes the dehydration of 2-methylcitrate (2-MC) to yield the cis isomer of 2-methyl-aconitate. It is also able to catalyze the dehydration of citrate and the hydration of cis-aconitate at a lower rate. Due to its broad substrate specificity, it seems to be responsible for the residual aconitase activity of the acnAB-null mutant',\n",
       "   'Pathway': 'Carbohydrate metabolism; tricarboxylic acid cycle; isocitrate from oxaloacetate: step 2/2'},\n",
       "  'seq': 'MSAQINNIRPEFDREIVDIVDYVMNYEISSKVAYDTAHYCLLDTLGCGLEALEYPACKKLLGPIVPGTVVPNGVRVPGTQFQLDPVQAAFNIGAMIRWLDFNDTWLAAEWGHPSDNLGGILATADWLSRNAVASGKAPLTMKQVLTAMIKAHEIQGCIALENSFNRVGLDHVLLVKVASTAVVAEMLGLTREEILNAVSLAWVDGQSLRTYRHAPNTGTRKSWAAGDATSRAVRLALMAKTGEMGYPSALTAPVWGFYDVSFKGESFRFQRPYGSYVMENVLFKISFPAEFHSQTAVEAAMTLYEQMQAAGKTAADIEKVTIRTHEACIRIIDKKGPLNNPADRDHCIQYMVAIPLLFGRLTAADYEDNVAQDKRIDALREKINCFEDPAFTADYHDPEKRAIANAITLEFTDGTRFEEVVVEYPIGHARRRQDGIPKLVDKFKINLARQFPTRQQQRILEVSLDRARLEQMPVNEYLDLYVI',\n",
       "  'mw': 53952.0},\n",
       " 'b0335': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77495',\n",
       "  'gene': 'b0335',\n",
       "  'locus': ['b0335', 'JW0326'],\n",
       "  'synonyms': 'yahU',\n",
       "  'properties': {'Catalytic Activity': [('ATP + CoA + propanoate = AMP + diphosphate + propanoyl-CoA',\n",
       "     '6.2.1.17')],\n",
       "   'Function': 'Catalyzes the synthesis of propionyl-CoA from propionate and CoA. Also converts acetate to acetyl-CoA but with a lower specific activity (By similarity)',\n",
       "   'Pathway': 'Organic acid metabolism; propanoate degradation'},\n",
       "  'seq': 'MSFSEFYQRSINEPEQFWAEQARRIDWQTPFTQTLDHSNPPFARWFCEGRTNLCHNAIDRWLEKQPEALALIAVSSETEEERTFTFRQLHDEVNAVASMLRSLGVQRGDRVLVYMPMIAEAHITLLACARIGAIHSVVFGGFASHSVAARIDDAKPVLIVSADAGARGGKIIPYKKLLDDAISQAQHQPRHVLLVDRGLAKMARVSGRDVDFASLRHQHIGARVPVAWLESNETSCILYTSGTTGKPKGVQRDVGGYAVALATSMDTIFGGKAGSVFFCASDIGWVVGHSYIVYAPLLAGMATIVYEGLPTWPDCGVWWTIVEKYQVSRMFSAPTAIRVLKKFPTAEIRKHDLSSLEVLYLAGEPLDEPTASWVSNTLDVPVIDNYWQTESGWPIMAIARGLDDRPTRLGSPGVPMYGYNVQLLNEVTGEPCGVNEKGMLVVEGPLPPGCIQTIWGDDGRFVKTYWSLFSRPVYATFDWGIRDADGYHFILGRTDDVINVAGHRLGTREIEESISSHPGVAEVAVVGVKDALKGQVAVAFVIPKESDSLEDRDVAHSQEKAIMALVDSQIGNFGRPAHVWFVSQLPKTRSGKMLRRTIQAICEGRDPGDLTTIDDPASLDQIRQAMEE',\n",
       "  'mw': 69351.0},\n",
       " 'b0336': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AA82',\n",
       "  'gene': 'b0336',\n",
       "  'locus': ['b0336', 'JW0327'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Required for cytosine transport into the cell'},\n",
       "  'seq': 'MSQDNNFSQGPVPQSARKGVLALTFVMLGLTFFSASMWTGGTLGTGLSYHDFFLAVLIGNLLLGIYTSFLGYIGAKTGLTTHLLARFSFGVKGSWLPSLLLGGTQVGWFGVGVAMFAIPVGKATGLDINLLIAVSGLLMTVTVFFGISALTVLSVIAVPAIACLGGYSVWLAVNGMGGLDALKAVVPAQPLDFNVALALVVGSFISAGTLTADFVRFGRNAKLAVLVAMVAFFLGNSLMFIFGAAGAAALGMADISDVMIAQGLLLPAIVVLGLNIWTTNDNALYASGLGFANITGMSSKTLSVINGIIGTVCALWLYNNFVGWLTFLSAAIPPVGGVIIADYLMNRRRYEHFATTRMMSVNWVAILAVALGIAAGHWLPGIVPVNAVLGGALSYLILNPILNRKTTAAMTHVEANSVE',\n",
       "  'mw': 43650.0},\n",
       " 'b0337': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P25524',\n",
       "  'gene': 'b0337',\n",
       "  'locus': ['b0337', 'JW0328'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('cytosine + H(+) + H2O = NH4(+) + uracil',\n",
       "     '3.5.4.1'),\n",
       "    ('H(+) + H2O + isoguanine = NH4(+) + xanthine', '')],\n",
       "   'Function': 'Catalyzes the hydrolytic deamination of cytosine to uracil. Is involved in the pyrimidine salvage pathway, which allows the cell to utilize cytosine for pyrimidine nucleotide synthesis. Is also able to catalyze deamination of isoguanine, a mutagenic oxidation product of adenine in DNA, and of isocytosine. To a lesser extent, also catalyzes the conversion of 5-fluorocytosine (5FC) to 5-fluorouracil (5FU); this activity allows the formation of a cytotoxic chemotherapeutic agent from a non-cytotoxic precursor',\n",
       "   'Regulation Activity': 'Fe(2+)-CDase is rapidly inactivated by H(2)O(2), whereas Mn(2+)-CDase, Co(2+)-CDase, and Zn(2+)-CDase are not inactivated by H(2)O(2). CDase is also inhibited by excess divalent cations (PubMed:8226944). Phosphonocytosine, a mimic of the tetrahedral reaction intermediate, inhibits the deamination of cytosine with a Ki of 52 nM (PubMed:21545144)'},\n",
       "  'seq': 'MSNNALQTIINARLPGEEGLWQIHLQDGKISAIDAQSGVMPITENSLDAEQGLVIPPFVEPHIHLDTTQTAGQPNWNQSGTLFEGIERWAERKALLTHDDVKQRAWQTLKWQIANGIQHVRTHVDVSDATLTALKAMLEVKQEVAPWIDLQIVAFPQEGILSYPNGEALLEEALRLGADVVGAIPHFEFTREYGVESLHKTFALAQKYDRLIDVHCDEIDDEQSRFVETVAALAHHEGMGARVTASHTTAMHSYNGAYTSRLFRLLKMSGINFVANPLVNIHLQGRFDTYPKRRGITRVKEMLESGINVCFGHDDVFDPWYPLGTANMLQVLHMGLHVCQLMGYGQINDGLNLITHHSARTLNLQDYGIAAGNSANLIILPAENGFDALRRQVPVRYSVRGGKVIASTQPAQTTVYLEQPEAIDYKR',\n",
       "  'mw': 47591.0},\n",
       " 'b0339': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ABE9',\n",
       "  'gene': 'b0339',\n",
       "  'locus': ['b0339', 'JW0330'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('H(+) + hydrogencarbonate = CO2 + H2O',\n",
       "     '4.2.1.1')],\n",
       "   'Function': 'Reversible hydration of carbon dioxide. Carbon dioxide formed in the bicarbonate-dependent decomposition of cyanate by cyanase (CynS) diffuses out of the cell faster than it would be hydrated to bicarbonate, so the apparent function of this enzyme is to catalyze the hydration of carbon dioxide and thus prevent depletion of cellular bicarbonate'},\n",
       "  'seq': 'MKEIIDGFLKFQREAFPKREALFKQLATQQSPRTLFISCSDSRLVPELVTQREPGDLFVIRNAGNIVPSYGPEPGGVSASVEYAVAALRVSDIVICGHSNCGAMTAIASCQCMDHMPAVSHWLRYADSARVVNEARPHSDLPSKAAAMVRENVIAQLANLQTHPSVRLALEEGRIALHGWVYDIESGSIAAFDGATRQFVPLAANPRVCAIPLRQPTAA',\n",
       "  'mw': 23764.0},\n",
       " 'b0340': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P00816',\n",
       "  'gene': 'b0340',\n",
       "  'locus': ['b0340', 'JW0331'],\n",
       "  'synonyms': 'cnt',\n",
       "  'properties': {'Catalytic Activity': [('cyanate + 3 H(+) + hydrogencarbonate = 2 CO2 + NH4(+)',\n",
       "     '4.2.1.104')],\n",
       "   'Function': 'Catalyzes the reaction of cyanate with bicarbonate to produce ammonia and carbon dioxide'},\n",
       "  'seq': 'MIQSQINRNIRLDLADAILLSKAKKDLSFAEIADGTGLAEAFVTAALLGQQALPADAARLVGAKLDLDEDSILLLQMIPLRGCIDDRIPTDPTMYRFYEMLQVYGTTLKALVHEKFGDGIISAINFKLDVKKVADPEGGERAVITLDGKYLPTKPF',\n",
       "  'mw': 17049.0},\n",
       " 'b0341': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P17583',\n",
       "  'gene': 'b0341',\n",
       "  'locus': ['b0341', 'JW0332'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'This protein is part of an active transport system that transports exogenous cyanate into E.coli cells'},\n",
       "  'seq': 'MLLVLVLIGLNMRPLLTSVGPLLPQLRQASGMSFSVAALLTALPVVTMGGLALAGSWLHQHVSERRSVAISLLLIAVGALMRELYPQSALLLSSALLGGVGIGIIQAVMPSVIKRRFQQRTPLVMGLWSAALMGGGGLGAAITPWLVQHSETWYQTLAWWALPAVVALFAWWWQSAREVASSHKTTTTPVRVVFTPRAWTLGVYFGLINGGYASLIAWLPAFYIEIGASAQYSGSLLALMTLGQAAGALLMPAMARHQDRRKLLMLALVLQLVGFCGFIWLPMQLPVLWAMVCGLGLGGAFPLCLLLALDHSVQPAIAGKLVAFMQGIGFIIAGLAPWFSGVLRSISGNYLMDWAFHALCVVGLMIITLRFAPVRFPQLWVKEA',\n",
       "  'mw': 41361.0},\n",
       " 'b0343': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P02920',\n",
       "  'gene': 'b0343',\n",
       "  'locus': ['b0343', 'JW0334'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('H(+)(in) + lactose(in) = H(+)(out) + lactose(out)',\n",
       "     ''),\n",
       "    ('H(+)(in) + melibiose(in) = H(+)(out) + melibiose(out)', '')],\n",
       "   'Function': 'Responsible for transport of beta-galactosides into the cell, with the concomitant import of a proton (symport system). Can transport lactose, melibiose, the synthetic disaccharide lactulose or the analog methyl-1-thio-beta,D-galactopyranoside (TMG), but not sucrose or fructose (PubMed:7000781, PubMed:7028742, PubMed:18177889, PubMed:1848449, PubMed:22106930). The substrate specificity is directed toward the galactopyranosyl moiety of the substrate (PubMed:22106930)',\n",
       "   'Regulation Activity': 'Inhibited by the proton ionophore carbonyl cyanide m-chlorophenylhydrazone (CCCP)'},\n",
       "  'seq': 'MYYLKNTNFWMFGLFFFFYFFIMGAYFPFFPIWLHDINHISKSDTGIIFAAISLFSLLFQPLFGLLSDKLGLRKYLLWIITGMLVMFAPFFIFIFGPLLQYNILVGSIVGGIYLGFCFNAGAPAVEAFIEKVSRRSNFEFGRARMFGCVGWALCASIVGIMFTINNQFVFWLGSGCALILAVLLFFAKTDAPSSATVANAVGANHSAFSLKLALELFRQPKLWFLSLYVIGVSCTYDVFDQQFANFFTSFFATGEQGTRVFGYVTTMGELLNASIMFFAPLIINRIGGKNALLLAGTIMSVRIIGSSFATSALEVVILKTLHMFEVPFLLVGCFKYITSQFEVRFSATIYLVCFCFFKQLAMIFMSVLAGNMYESIGFQGAYLVLGLVALGFTLISVFTLSGPGPLSLLRRQVNEVA',\n",
       "  'mw': 46503.0},\n",
       " 'b0344': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P00722',\n",
       "  'gene': 'b0344',\n",
       "  'locus': ['b0344', 'JW0335'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('Hydrolysis of terminal non-reducing beta-D-galactose residues in beta-D-galactosides.',\n",
       "     '3.2.1.23')],\n",
       "   'Regulation Activity': 'Inhibited by phenylethyl thio-beta-D-galactoside (PETG), isopropyl thio-beta-D-galactoside (IPTG), L-ribose, D-galactonolactone, lactose and 2-amino-D-galactose'},\n",
       "  'seq': 'MTMITDSLAVVLQRRDWENPGVTQLNRLAAHPPFASWRNSEEARTDRPSQQLRSLNGEWRFAWFPAPEAVPESWLECDLPEADTVVVPSNWQMHGYDAPIYTNVTYPITVNPPFVPTENPTGCYSLTFNVDESWLQEGQTRIIFDGVNSAFHLWCNGRWVGYGQDSRLPSEFDLSAFLRAGENRLAVMVLRWSDGSYLEDQDMWRMSGIFRDVSLLHKPTTQISDFHVATRFNDDFSRAVLEAEVQMCGELRDYLRVTVSLWQGETQVASGTAPFGGEIIDERGGYADRVTLRLNVENPKLWSAEIPNLYRAVVELHTADGTLIEAEACDVGFREVRIENGLLLLNGKPLLIRGVNRHEHHPLHGQVMDEQTMVQDILLMKQNNFNAVRCSHYPNHPLWYTLCDRYGLYVVDEANIETHGMVPMNRLTDDPRWLPAMSERVTRMVQRDRNHPSVIIWSLGNESGHGANHDALYRWIKSVDPSRPVQYEGGGADTTATDIICPMYARVDEDQPFPAVPKWSIKKWLSLPGETRPLILCEYAHAMGNSLGGFAKYWQAFRQYPRLQGGFVWDWVDQSLIKYDENGNPWSAYGGDFGDTPNDRQFCMNGLVFADRTPHPALTEAKHQQQFFQFRLSGQTIEVTSEYLFRHSDNELLHWMVALDGKPLASGEVPLDVAPQGKQLIELPELPQPESAGQLWLTVRVVQPNATAWSEAGHISAWQQWRLAENLSVTLPAASHAIPHLTTSEMDFCIELGNKRWQFNRQSGFLSQMWIGDKKQLLTPLRDQFTRAPLDNDIGVSEATRIDPNAWVERWKAAGHYQAEAALLQCTADTLADAVLITTAHAWQHQGKTLFISRKTYRIDGSGQMAITVDVEVASDTPHPARIGLNCQLAQVAERVNWLGLGPQENYPDRLTAACFDRWDLPLSDMYTPYVFPSENGLRCGTRELNYGPHQWRGDFQFNISRYSQQQLMETSHRHLLHAEEGTWLNIDGFHMGIGGDDSWSPSVSAEFQLSAGRYHYQLVWCQK',\n",
       "  'mw': 116483.0},\n",
       " 'b0347': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77397',\n",
       "  'gene': 'b0347',\n",
       "  'locus': ['b0347', 'JW0338'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('3-(3-hydroxyphenyl)propanoate + H(+) + NADH + O2 = 3-(2,3-dihydroxyphenyl)propanoate + H2O + NAD(+)',\n",
       "     '1.14.13.127'),\n",
       "    ('(2E)-3-(3-hydroxyphenyl)prop-2-enoate + H(+) + NADH + O2 = (2E)-3-(2,3-dihydroxyphenyl)prop-2-enoate + H2O + NAD(+)',\n",
       "     '1.14.13.127')],\n",
       "   'Function': 'Catalyzes the insertion of one atom of molecular oxygen into position 2 of the phenyl ring of 3-(3-hydroxyphenyl)propionate (3-HPP) and hydroxycinnamic acid (3HCI)',\n",
       "   'Pathway': 'Aromatic compound metabolism; 3-phenylpropanoate degradation'},\n",
       "  'seq': 'MAIQHPDIQPAVNHSVQVAIAGAGPVGLMMANYLGQMGIDVLVVEKLDKLIDYPRAIGIDDEALRTMQSVGLVDDVLPHTTPWHAMRFLTPKGRCFADIQPMTDEFGWPRRNAFIQPQVDAVMLEGVSRFPNVRCLFSRELEAFSQQDDEVTLHLKTAEGQREIVKAQWLVACDGGASFVRRTLNVPFEGKTAPNQWIVVDIANDPLSTPHIYLCCDPVRPYVSAALPHAVRRFEFMVMPGETEEQLREPQNMRKLLSKVLPNPDNVELIRQRVYTHNARLAQRFRIDRVLLAGDAAHIMPVWQGQGYNSGMRDAFNLAWKLALVIQGKARDALLDTYQQERRDHAKAMIDLSVTAGNVLAPPKRWQGTLRDGVSWLLNYLPPVKRYFLEMRFKPMPQYYGGALMREGEAKHSPVGKMFIQPKVTLENGDVTLLDNAIGANFAVIGWGCNPLWGMSDEQIQQWRALGTRFIQVVPEVQIHTAQDNHDGVLRVGDTQGRLRSWFAQHNASLVVMRPDRFVAATAIPQTLGKTLNKLASVMTLTRPDADVSVEKVA',\n",
       "  'mw': 62186.0},\n",
       " 'b0348': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ABR9',\n",
       "  'gene': 'b0348',\n",
       "  'locus': ['b0348', 'JW0339'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('3-(2,3-dihydroxyphenyl)propanoate + O2 = (2Z,4E)-2-hydroxy-6-oxonona-2,4-dienedioate + H(+)',\n",
       "     '1.13.11.16'),\n",
       "    ('(2E)-3-(2,3-dihydroxyphenyl)prop-2-enoate + O2 = (2Z,4E,7E)-2-hydroxy-6-oxonona-2,4,7-trienedioate + H(+)',\n",
       "     '1.13.11.16')],\n",
       "   'Function': 'Catalyzes the non-heme iron(II)-dependent oxidative cleavage of 2,3-dihydroxyphenylpropionic acid and 2,3-dihydroxicinnamic acid into 2-hydroxy-6-ketononadienedioate and 2-hydroxy-6-ketononatrienedioate, respectively',\n",
       "   'Pathway': 'Aromatic compound metabolism; 3-phenylpropanoate degradation'},\n",
       "  'seq': 'MHAYLHCLSHSPLVGYVDPAQEVLDEVNGVIASARERIAAFSPELVVLFAPDHYNGFFYDVMPPFCLGVGATAIGDFGSAAGELPVPVELAEACAHAVMKSGIDLAVSYCMQVDHGFAQPLEFLLGGLDKVPVLPVFINGVATPLPGFQRTRMLGEAIGRFTSTLNKRVLFLGSGGLSHQPPVPELAKADAHMRDRLLGSGKDLPASERELRQQRVISAAEKFVEDQRTLHPLNPIWDNQFMTLLEQGRIQELDAVSNEELSAIAGKSTHEIKTWVAAFAAISAFGNWRSEGRYYRPIPEWIAGFGSLSARTEN',\n",
       "  'mw': 34196.0},\n",
       " 'b0349': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77044',\n",
       "  'gene': 'b0349',\n",
       "  'locus': ['b0349', 'JW0340'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(2Z,4E)-2-hydroxy-6-oxonona-2,4-dienedioate + H2O = (2Z)-2-hydroxypenta-2,4-dienoate + H(+) + succinate',\n",
       "     '3.7.1.14'),\n",
       "    ('(2Z,4E,7E)-2-hydroxy-6-oxonona-2,4,7-trienedioate + H2O = (2Z)-2-hydroxypenta-2,4-dienoate + fumarate + H(+)',\n",
       "     '3.7.1.14')],\n",
       "   'Function': 'Catalyzes the cleavage of the C5-C6 bond of 2-hydroxy-6-oxononadienedioate and 2-hydroxy-6-oxononatrienedioate, a dienol ring fission product of the bacterial meta-cleavage pathway for degradation of phenylpropionic acid. MhpC shows some selectivity for the carboxylate of the side chain',\n",
       "   'Pathway': 'Aromatic compound metabolism; 3-phenylpropanoate degradation'},\n",
       "  'seq': 'MSYQPQTEAATSRFLNVEEAGKTLRIHFNDCGQGDETVVLLHGSGPGATGWANFSRNIDPLVEAGYRVILLDCPGWGKSDSVVNSGSRSDLNARILKSVVDQLDIAKIHLLGNSMGGHSSVAFTLKWPERVGKLVLMGGGTGGMSLFTPMPTEGIKRLNQLYRQPTIENLKLMMDIFVFDTSDLTDALFEARLNNMLSRRDHLENFVKSLEANPKQFPDFGPRLAEIKAQTLIVWGRNDRFVPMDAGLRLLSGIAGSELHIFRDCGHWAQWEHADAFNQLVLNFLARP',\n",
       "  'mw': 31937.0},\n",
       " 'b0350': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77608',\n",
       "  'gene': 'b0350',\n",
       "  'locus': ['b0350', 'JW0341'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(S)-4-hydroxy-2-oxopentanoate = (2Z)-2-hydroxypenta-2,4-dienoate + H2O',\n",
       "     '4.2.1.80')],\n",
       "   'Function': 'Catalyzes the conversion of 2-hydroxypentadienoic acid (enolic form of 2-oxopent-4-enoate) to 4-hydroxy-2-ketopentanoic acid',\n",
       "   'Regulation Activity': 'Inhibited by sodium oxalate',\n",
       "   'Pathway': 'Aromatic compound metabolism; 3-phenylpropanoate degradation'},\n",
       "  'seq': 'MTKHTLEQLAADLRRAAEQGEAIAPLRDLIGIDNAEAAYAIQHINVQHDVAQGRRVVGRKVGLTHPKVQQQLGVDQPDFGTLFADMCYGDNEIIPFSRVLQPRIEAEIALVLNRDLPATDITFDELYNAIEWVLPALEVVGSRIRDWSIQFVDTVADNASCGVYVIGGPAQRPAGLDLKNCAMKMTRNNEEVSSGRGSECLGHPLNAAVWLARKMASLGEPLRTGDIILTGALGPMVAVNAGDRFEAHIEGIGSVAATFSSAAPKGSLS',\n",
       "  'mw': 28890.0},\n",
       " 'b0351': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77580',\n",
       "  'gene': 'b0351',\n",
       "  'locus': ['b0351', 'JW0342'],\n",
       "  'synonyms': 'mhpE',\n",
       "  'properties': {'Catalytic Activity': [('acetaldehyde + CoA + NAD(+) = acetyl-CoA + H(+) + NADH',\n",
       "     '1.2.1.10')],\n",
       "   'Function': 'Catalyzes the conversion of acetaldehyde to acetyl-CoA, using NAD(+) and coenzyme A. Is the final enzyme in the meta-cleavage pathway for the degradation of 3-phenylpropanoate. Functions as a chaperone protein for folding of MhpE',\n",
       "   'Pathway': 'Aromatic compound metabolism; 3-phenylpropanoate degradation'},\n",
       "  'seq': 'MSKRKVAIIGSGNIGTDLMIKILRHGQHLEMAVMVGIDPQSDGLARARRMGVATTHEGVIGLMNMPEFADIDIVFDATSAGAHVKNDAALREAKPDIRLIDLTPAAIGPYCVPVVNLEANVDQLNVNMVTCGGQATIPMVAAVSRVARVHYAEIIASIASKSAGPGTRANIDEFTETTSRAIEVVGGAAKGKAIIVLNPAEPPLMMRDTVYVLSDEASQDDIEASINEMAEAVQAYVPGYRLKQRVQFEVIPQDKPVNLPGVGQFSGLKTAVWLEVEGAAHYLPAYAGNLDIMTSSALATAEKMAQSLARKAGEAA',\n",
       "  'mw': 33442.0},\n",
       " 'b0352': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P51020',\n",
       "  'gene': 'b0352',\n",
       "  'locus': ['b0352', 'JW0343'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(S)-4-hydroxy-2-oxopentanoate = acetaldehyde + pyruvate',\n",
       "     '4.1.3.39')],\n",
       "   'Function': 'Catalyzes the retro-aldol cleavage of 4-hydroxy-2-oxopentanoate to pyruvate and acetaldehyde. Is involved in the meta-cleavage pathway for the degradation of 3-phenylpropanoate',\n",
       "   'Pathway': 'Aromatic compound metabolism; 3-phenylpropanoate degradation'},\n",
       "  'seq': 'MNGKKLYISDVTLRDGMHAIRHQYSLENVRQIAKALDDARVDSIEVAHGDGLQGSSFNYGFGAHSDLEWIEAAADVVKHAKIATLLLPGIGTIHDLKNAWQAGARVVRVATHCTEADVSAQHIQYARELGMDTVGFLMMSHMTTPENLAKQAKLMEGYGATCIYVVDSGGAMNMSDIRDRFRALKAELKPETQTGMHAHHNLSLGVANSIAAVEEGCDRIDASLAGMGAGAGNAPLEVFIAAADKLGWQHGTDLYALMDAADDLVRPLQDRPVRVDRETLALGYAGVYSSFLRHCETAAARYGLSAVDILVELGKRRMVGGQEDMIVDVALDLRNNK',\n",
       "  'mw': 36470.0},\n",
       " 'b0353': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77589',\n",
       "  'gene': 'b0353',\n",
       "  'locus': ['b0353', 'JW5046'],\n",
       "  'synonyms': 'yaiK',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Uptake of 3-(3-hydroxyphenyl)propionate (3HPP) across the cytoplasmic membrane. Transport is driven by the proton motive force. Does not transport benzoate, 3-hydroxybenzoate or gentisate'},\n",
       "  'seq': 'MSTRTPSSSSSRLMLTIGLCFLVALMEGLDLQAAGIAAGGIAQAFALDKMQMGWIFSAGILGLLPGALVGGMLADRYGRKRILIGSVALFGLFSLATAIAWDFPSLVFARLMTGVGLGAALPNLIALTSEAAGPRFRGTAVSLMYCGVPIGAALAATLGFAGANLAWQTVFWVGGVVPLILVPLLMRWLPESAVFAGEKQSAPPLRALFAPETATATLLLWLCYFFTLLVVYMLINWLPLLLVEQGFQPSQAAGVMFALQMGAASGTLMLGALMDKLRPVTMSLLIYSGMLASLLALGTVSSFNGMLLAGFVAGLFATGGQSVLYALAPLFYSSQIRATGVGTAVAVGRLGAMSGPLLAGKMLALGTGTVGVMAASAPGILVAGLAVFILMSRRSRIQPCADA',\n",
       "  'mw': 41551.0},\n",
       " 'b0355': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P51025',\n",
       "  'gene': 'b0355',\n",
       "  'locus': ['b0355', 'JW0346'],\n",
       "  'synonyms': 'yaiM',\n",
       "  'properties': {'Catalytic Activity': [('H2O + S-formylglutathione = formate + glutathione + H(+)',\n",
       "     '3.1.2.12')],\n",
       "   'Function': 'Serine hydrolase involved in the detoxification of formaldehyde. Hydrolyzes S-formylglutathione to glutathione and formate. Shows also esterase activity against two pNP-esters (pNP-acetate and pNP-propionate), alpha-naphthyl acetate and lactoylglutathione',\n",
       "   'Regulation Activity': 'Inhibited by the sulfhydryl inhibitors (N-ethylmaleimide, iodoacetate, ZnCl(2) and CuCl(2))'},\n",
       "  'seq': 'MELIEKHVSFGGWQNMYRHYSQSLKCEMNVGVYLPPKAANEKLPVLYWLSGLTCNEQNFITKSGMQRYAAEHNIIVVAPDTSPRGSHVADADRYDLGQGAGFYLNATQAPWNEHYKMYDYIRNELPDLVMHHFPATAKKSISGHSMGGLGALVLALRNPDEYVSVSAFSPIVSPSQVPWGQQAFAAYLAENKDAWLDYDPVSLISQGQRVAEIMVDQGLSDDFYAEQLRTPNLEKICQEMNIKTLIRYHEGYDHSYYFVSSFIGEHIAYHANKLNMR',\n",
       "  'mw': 31424.0},\n",
       " 'b0356': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P25437',\n",
       "  'gene': 'b0356',\n",
       "  'locus': ['b0356', 'JW0347'],\n",
       "  'synonyms': 'adhC',\n",
       "  'properties': {'Catalytic Activity': [('NADP(+) + S-(hydroxymethyl)glutathione = H(+) + NADPH + S-formylglutathione',\n",
       "     '1.1.1.284'),\n",
       "    ('NAD(+) + S-(hydroxymethyl)glutathione = H(+) + NADH + S-formylglutathione',\n",
       "     '1.1.1.284'),\n",
       "    ('a primary alcohol + NAD(+) = an aldehyde + H(+) + NADH', '1.1.1.1'),\n",
       "    ('a secondary alcohol + NAD(+) = a ketone + H(+) + NADH', '1.1.1.1')],\n",
       "   'Function': 'Has high formaldehyde dehydrogenase activity in the presence of glutathione and catalyzes the oxidation of normal alcohols in a reaction that is not GSH-dependent. In addition, hemithiolacetals other than those formed from GSH, including omega-thiol fatty acids, also are substrates'},\n",
       "  'seq': 'MKSRAAVAFAPGKPLEIVEIDVAPPKKGEVLIKVTHTGVCHTDAFTLSGDDPEGVFPVVLGHEGAGVVVEVGEGVTSVKPGDHVIPLYTAECGECEFCRSGKTNLCVAVRETQGKGLMPDGTTRFSYNGQPLYHYMGCSTFSEYTVVAEVSLAKINPEANHEHVCLLGCGVTTGIGAVHNTAKVQPGDSVAVFGLGAIGLAVVQGARQAKAGRIIAIDTNPKKFDLARRFGATDCINPNDYDKPIKDVLLDINKWGIDHTFECIGNVNVMRAALESAHRGWGQSVIIGVAVAGQEISTRPFQLVTGRVWKGSAFGGVKGRSQLPGMVEDAMKGDIDLEPFVTHTMSLDEINDAFDLMHEGKSIRTVIRY',\n",
       "  'mw': 39359.0},\n",
       " 'b0365': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'Q47537',\n",
       "  'gene': 'b0365',\n",
       "  'locus': ['b0365', 'JW0357'],\n",
       "  'synonyms': 'ssiA yaiR',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of a binding-protein-dependent transport system for taurine'},\n",
       "  'seq': 'MAISSRNTLLAALAFIAFQAQAVNVTVAYQTSAEPAKVAQADNTFAKESGATVDWRKFDSGASIVRALASGDVQIGNLGSSPLAVAASQQVPIEVFLLASKLGNSEALVVKKTISKPEDLIGKRIAVPFISTTHYSLLAALKHWGIKPGQVEIVNLQPPAIIAAWQRGDIDGAYVWAPAVNALEKDGKVLTDSEQVGQWGAPTLDVWVVRKDFAEKHPEVVKAFAKSAIDAQQPYIANPDVWLKQPENISKLARLSGVPEGDVPGLVKGNTYLTPQQQTAELTGPVNKAIIDTAQFLKEQGKVPAVANDYSQYVTSRFVQ',\n",
       "  'mw': 34266.0},\n",
       " 'b0366': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'Q47538',\n",
       "  'gene': 'b0366',\n",
       "  'locus': ['b0366', 'JW0358'],\n",
       "  'synonyms': 'ssiB yaiQ',\n",
       "  'properties': {'Catalytic Activity': [('ATP + H2O + taurine(out) = ADP + H(+) + phosphate + taurine(in)',\n",
       "     '7.6.2.7')],\n",
       "   'Function': 'Part of the ABC transporter complex TauABC involved in taurine import. Responsible for energy coupling to the transport system'},\n",
       "  'seq': 'MLQISHLYADYGGKPALEDINLTLESGELLVVLGPSGCGKTTLLNLIAGFVPYQHGSIQLAGKRIEGPGAERGVVFQNEGLLPWRNVQDNVAFGLQLAGIEKMQRLEIAHQMLKKVGLEGAEKRYIWQLSGGQRQRVGIARALAANPQLLLLDEPFGALDAFTRDQMQTLLLKLWQETGKQVLLITHDIEEAVFMATELVLLSSGPGRVLERLPLNFARRFVAGESSRSIKSDPQFIAMREYVLSRVFEQREAFS',\n",
       "  'mw': 28297.0},\n",
       " 'b0367': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'Q47539',\n",
       "  'gene': 'b0367',\n",
       "  'locus': ['b0367', 'JW0359'],\n",
       "  'synonyms': 'ssiC yaiJ',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of a binding-protein-dependent transport system for taurine. Probably responsible for the translocation of the substrate across the membrane'},\n",
       "  'seq': 'MSVLINEKLHSRRLKWRWPLSRQVTLSIGTLAVLLTVWWTVATLQLISPLFLPPPQQVLEKLLTIAGPQGFMDATLWQHLAASLTRIMLALFAAVLFGIPVGIAMGLSPTVRGILDPIIELYRPVPPLAYLPLMVIWFGIGETSKILLIYLAIFAPVAMSALAGVKSVQQVRIRAAQSLGASRAQVLWFVILPGALPEILTGLRIGLGVGWSTLVAAELIAATRGLGFMVQSAGEFLATDVVLAGIAVIAIIAFLLELGLRALQRRLTPWHGEVQ',\n",
       "  'mw': 29812.0},\n",
       " 'b0368': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37610',\n",
       "  'gene': 'b0368',\n",
       "  'locus': ['b0368', 'JW0360'],\n",
       "  'synonyms': 'ssiD yaiG',\n",
       "  'properties': {'Catalytic Activity': [('2-oxoglutarate + O2 + taurine = aminoacetaldehyde + CO2 + H(+) + succinate + sulfite',\n",
       "     '1.14.11.17')],\n",
       "   'Function': 'Catalyzes the alpha-ketoglutarate-dependent hydroxylation of taurine yielding sulfite and aminoacetaldehyde after decomposition of an unstable intermediate (PubMed:9287300). Is required for the utilization of taurine (2-aminoethanesulfonate) as an alternative sulfur source for growth in the absence of sulfate (PubMed:8808933). To a lesser extent, pentanesulfonate, 3-(N-morpholino)propanesulfonate and 1,3-dioxo-2-isoindolineethanesulfonate are also desulfonated by this enzyme in vitro; however, desulfonation by TauD of organosulfonates other than taurine seem to be of little or no importance for sulfur metabolism in vivo (PubMed:9287300)',\n",
       "   'Regulation Activity': 'Activated by ascorbate and inhibited by divalent metal ions such as zinc, copper and cobalt',\n",
       "   'Pathway': 'Organosulfur degradation; taurine degradation via aerobic pathway; aminoacetaldehyde and sulfite from taurine: step 1/1'},\n",
       "  'seq': 'MSERLSITPLGPYIGAQISGADLTRPLSDNQFEQLYHAVLRHQVVFLRDQAITPQQQRALAQRFGELHIHPVYPHAEGVDEIIVLDTHNDNPPDNDNWHTDVTFIETPPAGAILAAKELPSTGGDTLWTSGIAAYEALSVPFRQLLSGLRAEHDFRKSFPEYKYRKTEEEHQRWREAVAKNPPLLHPVVRTHPVSGKQALFVNEGFTTRIVDVSEKESEALLSFLFAHITKPEFQVRWRWQPNDIAIWDNRVTQHYANADYLPQRRIMHRATILGDKPFYRAG',\n",
       "  'mw': 32410.0},\n",
       " 'b0369': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ACB2',\n",
       "  'gene': 'b0369',\n",
       "  'locus': ['b0369', 'JW0361'],\n",
       "  'synonyms': 'ncf',\n",
       "  'properties': {'Catalytic Activity': [('2 5-aminolevulinate = H(+) + 2 H2O + porphobilinogen',\n",
       "     '4.2.1.24')],\n",
       "   'Function': 'Catalyzes an early step in the biosynthesis of tetrapyrroles. Binds two molecules of 5-aminolevulinate per subunit, each at a distinct site, and catalyzes their condensation to form porphobilinogen',\n",
       "   'Regulation Activity': 'Allosteric enzyme. Stimulated by magnesium ions',\n",
       "   'Pathway': 'Porphyrin-containing compound metabolism; protoporphyrin-IX biosynthesis; coproporphyrinogen-III from 5-aminolevulinate: step 1/4'},\n",
       "  'seq': 'MTDLIQRPRRLRKSPALRAMFEETTLSLNDLVLPIFVEEEIDDYKAVEAMPGVMRIPEKHLAREIERIANAGIRSVMTFGISHHTDETGSDAWREDGLVARMSRICKQTVPEMIVMSDTCFCEYTSHGHCGVLCEHGVDNDATLENLGKQAVVAAAAGADFIAPSAAMDGQVQAIRQALDAAGFKDTAIMSYSTKFASSFYGPFREAAGSALKGDRKSYQMNPMNRREAIRESLLDEAQGADCLMVKPAGAYLDIVRELRERTELPIGAYQVSGEYAMIKFAALAGAIDEEKVVLESLGSIKRAGADLIFSYFALDLAEKKILR',\n",
       "  'mw': 35625.0},\n",
       " 'b0381': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A6J8',\n",
       "  'gene': 'b0381',\n",
       "  'locus': ['b0381', 'JW0372'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + 2 D-alanine = ADP + D-alanyl-D-alanine + H(+) + phosphate',\n",
       "     '6.3.2.4')],\n",
       "   'Function': 'Cell wall formation',\n",
       "   'Pathway': 'Cell wall biogenesis; peptidoglycan biosynthesis'},\n",
       "  'seq': 'MEKLRVGIVFGGKSAEHEVSLQSAKNIVDAIDKSRFDVVLLGIDKQGQWHVSDASNYLLNADDPAHIALRPSATSLAQVPGKHEHQLIDAQNGQPLPTVDVIFPIVHGTLGEDGSLQGMLRVANLPFVGSDVLASAACMDKDVTKRLLRDAGLNIAPFITLTRANRHNISFAEVESKLGLPLFVKPANQGSSVGVSKVTSEEQYAIAVDLAFEFDHKVIVEQGIKGREIECAVLGNDNPQASTCGEIVLTSDFYAYDTKYIDEDGAKVVVPAAIAPEINDKIRAIAVQAYQTLGCAGMARVDVFLTPENEVVINEINTLPGFTNISMYPKLWQASGLGYTDLITRLIELALERHAADNALKTTM',\n",
       "  'mw': 39316.0},\n",
       " 'b0383': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P00634',\n",
       "  'gene': 'b0383',\n",
       "  'locus': ['b0383', 'JW0374'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('a phosphate monoester + H2O = an alcohol + phosphate',\n",
       "     '3.1.3.1')]},\n",
       "  'seq': 'MKQSTIALALLPLLFTPVTKARTPEMPVLENRAAQGDITAPGGARRLTGDQTAALRDSLSDKPAKNIILLIGDGMGDSEITAARNYAEGAGGFFKGIDALPLTGQYTHYALNKKTGKPDYVTDSAASATAWSTGVKTYNGALGVDIHEKDHPTILEMAKAAGLATGNVSTAELQDATPAALVAHVTSRKCYGPSATSEKCPGNALEKGGKGSITEQLLNARADVTLGGGAKTFAETATAGEWQGKTLREQAQARGYQLVSDAASLNSVTEANQQKPLLGLFADGNMPVRWLGPKATYHGNIDKPAVTCTPNPQRNDSVPTLAQMTDKAIELLSKNEKGFFLQVEGASIDKQDHAANPCGQIGETVDLDEAVQRALEFAKKEGNTLVIVTADHAHASQIVAPDTKAPGLTQALNTKDGAVMVMSYGNSEEDSQEHTGSQLRIAAYGPHAANVVGLTDQTDLFYTMKAALGLK',\n",
       "  'mw': 49439.0},\n",
       " 'b0386': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A9L8',\n",
       "  'gene': 'b0386',\n",
       "  'locus': ['b0386', 'JW0377'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('L-proline + NADP(+) = 1-pyrroline-5-carboxylate + 2 H(+) + NADPH',\n",
       "     '1.5.1.2'),\n",
       "    ('L-proline + NAD(+) = 1-pyrroline-5-carboxylate + 2 H(+) + NADH',\n",
       "     '1.5.1.2')],\n",
       "   'Function': 'Catalyzes the reduction of 1-pyrroline-5-carboxylate (PCA) to L-proline. Does not catalyze the reverse reaction',\n",
       "   'Regulation Activity': 'Inhibited by p-chloromercuribenzoate',\n",
       "   'Pathway': 'Amino-acid biosynthesis; L-proline biosynthesis; L-proline from L-glutamate 5-semialdehyde: step 1/1'},\n",
       "  'seq': 'MEKKIGFIGCGNMGKAILGGLIASGQVLPGQIWVYTPSPDKVAALHDQFGINAAESAQEVAQIADIIFAAVKPGIMIKVLSEITSSLNKDSLVVSIAAGVTLDQLARALGHDRKIIRAMPNTPALVNAGMTSVTPNALVTPEDTADVLNIFRCFGEAEVIAEPMIHPVVGVSGSSPAYVFMFIEAMADAAVLGGMPRAQAYKFAAQAVMGSAKMVLETGEHPGALKDMVCSPGGTTIEAVRVLEEKGFRAAVIEAMTKCMEKSEKLSKS',\n",
       "  'mw': 28145.0},\n",
       " 'b0388': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A6E1',\n",
       "  'gene': 'b0388',\n",
       "  'locus': ['b0388', 'JW0379'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + shikimate = 3-phosphoshikimate + ADP + H(+)',\n",
       "     '2.7.1.71')],\n",
       "   'Function': 'Catalyzes the specific phosphorylation of the 3-hydroxyl group of shikimic acid using ATP as a cosubstrate',\n",
       "   'Pathway': 'Metabolic intermediate biosynthesis; chorismate biosynthesis; chorismate from D-erythrose 4-phosphate and phosphoenolpyruvate: step 5/7'},\n",
       "  'seq': 'MTQPLFLIGPRGCGKTTVGMALADSLNRRFVDTDQWLQSQLNMTVAEIVEREEWAGFRARETAALEAVTAPSTVIATGGGIILTEFNRHFMQNNGIVVYLCAPVSVLVNRLQAAPEEDLRPTLTGKPLSEEVQEVLEERDALYREVAHIIIDATNEPSQVISEIRSALAQTINC',\n",
       "  'mw': 19151.0},\n",
       " 'b0394': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P23917',\n",
       "  'gene': 'b0394',\n",
       "  'locus': ['b0394', 'JW0385'],\n",
       "  'synonyms': 'yajF',\n",
       "  'properties': {'Catalytic Activity': [('ATP + D-fructose = ADP + D-fructose 6-phosphate + H(+)',\n",
       "     '2.7.1.4')],\n",
       "   'Function': 'Catalyzes the phosphorylation of fructose to fructose-6-P. Has also low level glucokinase activity in vitro. Is not able to phosphorylate D-ribose, D-mannitol, D-sorbitol, inositol, and L-threonine'},\n",
       "  'seq': 'MRIGIDLGGTKTEVIALGDAGEQLYRHRLPTPRDDYRQTIETIATLVDMAEQATGQRGTVGMGIPGSISPYTGVVKNANSTWLNGQPFDKDLSARLQREVRLANDANCLAVSEAVDGAAAGAQTVFAVIIGTGCGAGVAFNGRAHIGGNGTAGEWGHNPLPWMDEDELRYREEVPCYCGKQGCIETFISGTGFAMDYRRLSGHALKGSEIIRLVEESDPVAELALRRYELRLAKSLAHVVNILDPDVIVLGGGMSNVDRLYQTVGQLIKQFVFGGECETPVRKAKHGDSSGVRGAAWLWPQE',\n",
       "  'mw': 32500.0},\n",
       " 'b0401': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AD99',\n",
       "  'gene': 'b0401',\n",
       "  'locus': ['b0401', 'JW0391'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Liv-II branched chain amino acid transport system, which transports leucine, valine and isoleucine'},\n",
       "  'seq': 'MTHQLRSRDIIALGFMTFALFVGAGNIIFPPMVGLQAGEHVWTAAFGFLITAVGLPVLTVVALAKVGGGVDSLSTPIGKVAGVLLATVCYLAVGPLFATPRTATVSFEVGIAPLTGDSALPLFIYSLVYFAIVILVSLYPGKLLDTVGNFLAPLKIIALVILSVAAIVWPAGSISTATEAYQNAAFSNGFVNGYLTMDTLGAMVFGIVIVNAARSRGVTEARLLTRYTVWAGLMAGVGLTLLYLALFRLGSDSASLVDQSANGAAILHAYVQHTFGGGGSFLLAALIFIACLVTAVGLTCACAEFFAQYVPLSYRTLVFILGGFSMVVSNLGLSQLIQISVPVLTAIYPPCIALVVLSFTRSWWHNSSRVIAPPMFISLLFGILDGIKASAFSDILPSWAQRLPLAEQGLAWLMPTVVMVVLAIIWDRAAGRQVTSSAH',\n",
       "  'mw': 46209.0},\n",
       " 'b0403': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P21517',\n",
       "  'gene': 'b0403',\n",
       "  'locus': ['b0403', 'JW0393'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('Hydrolysis of terminal, non-reducing (1->4)-linked alpha-D-glucose residues with release of alpha-D-glucose.',\n",
       "     '3.2.1.20')],\n",
       "   'Function': 'May play a role in regulating the intracellular level of maltotriose. Cleaves glucose from the reducing end of maltotriose and longer maltodextrins with a chain length of up to 7 glucose units'},\n",
       "  'seq': 'MLNAWHLPVPPFVKQSKDQLLITLWLTGEDPPQRIMLRTEHDNEEMSVPMHKQRSQPQPGVTAWRAAIDLSSGQPRRRYSFKLLWHDRQRWFTPQGFSRMPPARLEQFAVDVPDIGPQWAADQIFYQIFPDRFARSLPREAEQDHVYYHHAAGQEIILRDWDEPVTAQAGGSTFYGGDLDGISEKLPYLKKLGVTALYLNPVFKAPSVHKYDTEDYRHVDPQFGGDGALLRLRHNTQQLGMRLVLDGVFNHSGDSHAWFDRHNRGTGGACHNPESPWRDWYSFSDDGTALDWLGYASLPKLDYQSESLVNEIYRGEDSIVRHWLKAPWNMDGWRLDVVHMLGEAGGARNNMQHVAGITEAAKETQPEAYIVGEHFGDARQWLQADVEDAAMNYRGFTFPLWGFLANTDISYDPQQIDAQTCMAWMDNYRAGLSHQQQLRMFNQLDSHDTARFKTLLGRDIARLPLAVVWLFTWPGVPCIYYGDEVGLDGKNDPFCRKPFPWQVEKQDTALFALYQRMIALRKKSQALRHGGCQVLYAEDNVVVFVRVLNQQRVLVAINRGEACEVVLPASPFLNAVQWQCKEGHGQLTDGILALPAISATVWMN',\n",
       "  'mw': 69041.0},\n",
       " 'b0404': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P21515',\n",
       "  'gene': 'b0404',\n",
       "  'locus': ['b0404', 'JW0394'],\n",
       "  'synonyms': 'yajB',\n",
       "  'properties': {'Catalytic Activity': [(\"H2O + holo-[ACP] = (R)-4'-phosphopantetheine + apo-[ACP] + H(+)\",\n",
       "     '3.1.4.14')],\n",
       "   'Function': 'Converts holo-ACP to apo-ACP by hydrolytic cleavage of the phosphopantetheine prosthetic group from ACP'},\n",
       "  'seq': 'MNFLAHLHLAHLAESSLSGNLLADFVRGNPEESFPPDVVAGIHMHRRIDVLTDNLPEVREAREWFRSETRRVAPITLDVMWDHFLSRHWSQLSPDFPLQEFVCYAREQVMTILPDSPPRFINLNNYLWSEQWLVRYRDMDFIQNVLNGMASRRPRLDALRDSWYDLDAHYDALETRFWQFYPRMMAQASRKAL',\n",
       "  'mw': 22961.0},\n",
       " 'b0411': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A927',\n",
       "  'gene': 'b0411',\n",
       "  'locus': ['b0411', 'JW0401'],\n",
       "  'synonyms': 'nupA',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': '(Microbial infection) Serves as a receptor for CdiA-STECO31, required for adhesion between E.coli expressing CdiA-STECO31 and this strain'},\n",
       "  'seq': 'MKKTLLAAGAVLALSSSFTVNAAENDKPQYLSDWWHQSVNVVGSYHTRFGPQIRNDTYLEYEAFAKKDWFDFYGYADAPVFFGGNSDAKGIWNHGSPLFMEIEPRFSIDKLTNTDLSFGPFKEWYFANNYIYDMGRNKDGRQSTWYMGLGTDIDTGLPMSLSMNVYAKYQWQNYGAANENEWDGYRFKIKYFVPITDLWGGQLSYIGFTNFDWGSDLGDDSGNAINGIKTRTNNSIASSHILALNYDHWHYSVVARYWHDGGQWNDDAELNFGNGNFNVRSTGWGGYLVVGYNF',\n",
       "  'mw': 33589.0},\n",
       " 'b0414': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P25539',\n",
       "  'gene': 'b0414',\n",
       "  'locus': ['b0414', 'JW0404'],\n",
       "  'synonyms': 'ribG ybaE',\n",
       "  'properties': {'Catalytic Activity': [('2,5-diamino-6-hydroxy-4-(5-phosphoribosylamino)-pyrimidine + H(+) + H2O = 5-amino-6-(5-phospho-D-ribosylamino)uracil + NH4(+)',\n",
       "     '3.5.4.26'),\n",
       "    ('5-amino-6-(5-phospho-D-ribitylamino)uracil + NADP(+) = 5-amino-6-(5-phospho-D-ribosylamino)uracil + H(+) + NADPH',\n",
       "     '1.1.1.193')],\n",
       "   'Function': \"Converts 2,5-diamino-6-(ribosylamino)-4(3h)-pyrimidinone 5'-phosphate into 5-amino-6-(ribosylamino)-2,4(1h,3h)-pyrimidinedione 5'-phosphate\",\n",
       "   'Pathway': 'Cofactor biosynthesis; riboflavin biosynthesis; 5-amino-6-(D-ribitylamino)uracil from GTP: step 3/4'},\n",
       "  'seq': 'MQDEYYMARALKLAQRGRFTTHPNPNVGCVIVKDGEIVGEGYHQRAGEPHAEVHALRMAGEKAKGATAYVTLEPCSHHGRTPPCCDALIAAGVARVVASMQDPNPQVAGRGLYRLQQAGIDVSHGLMMSEAEQLNKGFLKRMRTGFPYIQLKLGASLDGRTAMASGESQWITSPQARRDVQLLRAQSHAILTSSATVLADDPALTVRWSELDEQTQALYPQQNLRQPIRIVIDSQNRVTPVHRIVQQPGETWFARTQEDSREWPETVRTLLIPEHKGHLDLVVLMMQLGKQQINSIWVEAGPTLAGALLQAGLVDELIVYIAPKLLGSDARGLCTLPGLEKLADAPQFKFKEIRHVGPDVCLHLVGA',\n",
       "  'mw': 40338.0},\n",
       " 'b0415': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P61714',\n",
       "  'gene': 'b0415',\n",
       "  'locus': ['b0415', 'JW0405'],\n",
       "  'synonyms': 'ribH ybaF',\n",
       "  'properties': {'Catalytic Activity': [('(2S)-2-hydroxy-3-oxobutyl phosphate + 5-amino-6-(D-ribitylamino)uracil = 6,7-dimethyl-8-(1-D-ribityl)lumazine + H(+) + 2 H2O + phosphate',\n",
       "     '2.5.1.78')],\n",
       "   'Function': 'Catalyzes the formation of 6,7-dimethyl-8-ribityllumazine by condensation of 5-amino-6-(D-ribitylamino)uracil with 3,4-dihydroxy-2-butanone 4-phosphate. This is the penultimate step in the biosynthesis of riboflavin',\n",
       "   'Pathway': 'Cofactor biosynthesis; riboflavin biosynthesis; riboflavin from 2-hydroxy-3-oxobutyl phosphate and 5-amino-6-(D-ribitylamino)uracil: step 1/2'},\n",
       "  'seq': 'MNIIEANVATPDARVAITIARFNNFINDSLLEGAIDALKRIGQVKDENITVVWVPGAYELPLAAGALAKTGKYDAVIALGTVIRGGTAHFEYVAGGASNGLAHVAQDSEIPVAFGVLTTESIEQAIERAGTKAGNKGAEAALTALEMINVLKAIKA',\n",
       "  'mw': 16157.0},\n",
       " 'b0417': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AGG0',\n",
       "  'gene': 'b0417',\n",
       "  'locus': ['b0417', 'JW0407'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + thiamine phosphate = ADP + thiamine diphosphate',\n",
       "     '2.7.4.16')],\n",
       "   'Function': 'Catalyzes the ATP-dependent phosphorylation of thiamine-monophosphate (TMP) to form thiamine-pyrophosphate (TPP), the active form of vitamin B1. Cannot use thiamine as substrate. Is highly specific for ATP as phosphate donor',\n",
       "   'Regulation Activity': 'Is markedly activated by the monovalent cations K(+), NH(4)(+), and Rb(+). Is significantly inhibited by ADP, AMP, p-chloromercuribenzoate, N-ethylmaleimide, pyrophosphate, and EDTA',\n",
       "   'Pathway': 'Cofactor biosynthesis; thiamine diphosphate biosynthesis; thiamine diphosphate from thiamine phosphate: step 1/1'},\n",
       "  'seq': 'MACGEFSLIARYFDRVRSSRLDVELGIGDDCALLNIPEKQTLAISTDTLVAGNHFLPDIDPADLAYKALAVNLSDLAAMGADPAWLTLALTLPDVDEAWLESFSDSLFDLLNYYDMQLIGGDTTRGPLSMTLGIHGFVPMGRALTRSGAKPGDWIYVTGTPGDSAAGLAILQNRLQVADAKDADYLIKRHLRPSPRILQGQALRDLANSAIDLSDGLISDLGHIVKASDCGARIDLALLPFSDALSRHVEPEQALRWALSGGEDYELCFTVPELNRGALDVALGHLGVPFTCIGQMTADIEGLCFIRDGEPVTLDWKGYDHFATP',\n",
       "  'mw': 35071.0},\n",
       " 'b0418': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P18200',\n",
       "  'gene': 'b0418',\n",
       "  'locus': ['b0418', 'JW0408'],\n",
       "  'synonyms': 'yajN',\n",
       "  'properties': {'Catalytic Activity': [(\"1,2-diacyl-sn-glycero-3-phospho-(1'-sn-glycero-3'-phosphate) + H2O = 1,2-diacyl-sn-glycero-3-phospho-(1'-sn-glycerol) + phosphate\",\n",
       "     '3.1.3.27')],\n",
       "   'Function': 'Lipid phosphatase which dephosphorylates phosphatidylglycerophosphate (PGP) to phosphatidylglycerol (PG)',\n",
       "   'Pathway': 'Phospholipid metabolism; phosphatidylglycerol biosynthesis; phosphatidylglycerol from CDP-diacylglycerol: step 2/2'},\n",
       "  'seq': 'MTILPRHKDVAKSRLKMSNPWHLLAVGFGSGLSPIVPGTMGSLAAIPFWYLMTFLPWQLYSLVVMLGICIGVYLCHQTAKDMGVHDHGSIVWDEFIGMWITLMALPTNDWQWVAAGFVIFRILDMWKPWPIRWFDRNVHGGMGIMIDDIVAGVISAGILYFIGHHWPLGILS',\n",
       "  'mw': 19418.0},\n",
       " 'b0420': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77488',\n",
       "  'gene': 'b0420',\n",
       "  'locus': ['b0420', 'JW0410'],\n",
       "  'synonyms': 'yajP',\n",
       "  'properties': {'Catalytic Activity': [('D-glyceraldehyde 3-phosphate + H(+) + pyruvate = 1-deoxy-D-xylulose 5-phosphate + CO2',\n",
       "     '2.2.1.7')],\n",
       "   'Function': 'Catalyzes the acyloin condensation reaction between C atoms 2 and 3 of pyruvate and glyceraldehyde 3-phosphate to yield 1-deoxy-D-xylulose-5-phosphate (DXP)',\n",
       "   'Pathway': 'Metabolic intermediate biosynthesis; 1-deoxy-D-xylulose 5-phosphate biosynthesis; 1-deoxy-D-xylulose 5-phosphate from D-glyceraldehyde 3-phosphate and pyruvate: step 1/1'},\n",
       "  'seq': 'MSFDIAKYPTLALVDSTQELRLLPKESLPKLCDELRRYLLDSVSRSSGHFASGLGTVELTVALHYVYNTPFDQLIWDVGHQAYPHKILTGRRDKIGTIRQKGGLHPFPWRGESEYDVLSVGHSSTSISAGIGIAVAAEKEGKNRRTVCVIGDGAITAGMAFEAMNHAGDIRPDMLVILNDNEMSISENVGALNNHLAQLLSGKLYSSLREGGKKVFSGVPPIKELLKRTEEHIKGMVVPGTLFEELGFNYIGPVDGHDVLGLITTLKNMRDLKGPQFLHIMTKKGRGYEPAEKDPITFHAVPKFDPSSGCLPKSSGGLPSYSKIFGDWLCETAAKDNKLMAITPAMREGSGMVEFSRKFPDRYFDVAIAEQHAVTFAAGLAIGGYKPIVAIYSTFLQRAYDQVLHDVAIQKLPVLFAIDRAGIVGADGQTHQGAFDLSYLRCIPEMVIMTPSDENECRQMLYTGYHYNDGPSAVRYPRGNAVGVELTPLEKLPIGKGIVKRRGEKLAILNFGTLMPEAAKVAESLNATLVDMRFVKPLDEALILEMAASHEALVTVEENAIMGGAGSGVNEVLMAHRKPVPVLNIGLPDFFIPQGTQEEMRAELGLDAAGMEAKIKAWLA',\n",
       "  'mw': 67617.0},\n",
       " 'b0421': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P22939',\n",
       "  'gene': 'b0421',\n",
       "  'locus': ['b0421', 'JW0411'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(2E)-geranyl diphosphate + isopentenyl diphosphate = (2E,6E)-farnesyl diphosphate + diphosphate',\n",
       "     '2.5.1.10')]},\n",
       "  'seq': 'MDFPQQLEACVKQANQALSRFIAPLPFQNTPVVETMQYGALLGGKRLRPFLVYATGHMFGVSTNTLDAPAAAVECIHAYSLIHDDLPAMDDDDLRRGLPTCHVKFGEANAILAGDALQTLAFSILSDADMPEVSDRDRISMISELASASGIAGMCGGQALDLDAEGKHVPLDALERIHRHKTGALIRAAVRLGALSAGDKGRRALPVLDKYAESIGLAFQVQDDILDVVGDTATLGKRQGADQQLGKSTYPALLGLEQARKKARDLIDDARQSLKQLAEQSLDTSALEALADYIIQRNK',\n",
       "  'mw': 32160.0},\n",
       " 'b0423': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77718',\n",
       "  'gene': 'b0423',\n",
       "  'locus': ['b0423', 'JW0413'],\n",
       "  'synonyms': 'yajK',\n",
       "  'properties': {'Catalytic Activity': [('[ThiI sulfur-carrier protein]-S-sulfanyl-L-cysteine + a uridine in tRNA + ATP + H(+) + 2 reduced [2Fe-2S]-[ferredoxin] = [ThiI sulfur-carrier protein]-L-cysteine + a 4-thiouridine in tRNA + AMP + diphosphate + 2 oxidized [2Fe-2S]-[ferredoxin]',\n",
       "     '2.8.1.4'),\n",
       "    ('[sulfur-carrier protein ThiS]-C-terminal Gly-Gly-AMP + AH2 + S-sulfanyl-L-cysteinyl-[cysteine desulfurase] = [sulfur-carrier protein ThiS]-C-terminal Gly-NH-CH2-C(O)SH + A + AMP + H(+) + L-cysteinyl-[cysteine desulfurase]',\n",
       "     '')],\n",
       "   'Function': 'Catalyzes the ATP-dependent transfer of a sulfur to tRNA to produce 4-thiouridine in position 8 of tRNAs, which functions as a near-UV photosensor. Also catalyzes the transfer of sulfur to the sulfur carrier protein ThiS, forming ThiS-thiocarboxylate. This is a step in the synthesis of thiazole, in the thiamine biosynthesis pathway. The sulfur is donated as persulfide by IscS',\n",
       "   'Pathway': 'Cofactor biosynthesis; thiamine diphosphate biosynthesis'},\n",
       "  'seq': 'MKFIIKLFPEITIKSQSVRLRFIKILTGNIRNVLKHYDETLAVVRHWDNIEVRAKDENQRLAIRDALTRIPGIHHILEVEDVPFTDMHDIFEKALVQYRDQLEGKTFCVRVKRRGKHDFSSIDVERYVGGGLNQHIESARVKLTNPDVTVHLEVEDDRLLLIKGRYEGIGGFPIGTQEDVLSLISGGFDSGVSSYMLMRRGCRVHYCFFNLGGAAHEIGVRQVAHYLWNRFGSSHRVRFVAINFEPVVGEILEKIDDGQMGVILKRMMVRAASKVAERYGVQALVTGEALGQVSSQTLTNLRLIDNVSDTLILRPLISYDKEHIINLARQIGTEDFARTMPEYCGVISKSPTVKAVKSKIEAEEEKFDFSILDKVVEEANNVDIREIAQQTEQEVVEVETVNGFGPNDVILDIRSIDEQEDKPLKVEGIDVVSLPFYKLSTKFGDLDQNKTWLLWCERGVMSRLQALYLREQGFNNVKVYRP',\n",
       "  'mw': 54973.0},\n",
       " 'b0425': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A9J4',\n",
       "  'gene': 'b0425',\n",
       "  'locus': ['b0425', 'JW0415'],\n",
       "  'synonyms': 'apbA',\n",
       "  'properties': {'Catalytic Activity': [('(R)-pantoate + NADP(+) = 2-dehydropantoate + H(+) + NADPH',\n",
       "     '1.1.1.169')],\n",
       "   'Function': 'Catalyzes the NADPH-dependent reduction of ketopantoate into pantoic acid',\n",
       "   'Pathway': 'Cofactor biosynthesis; (R)-pantothenate biosynthesis; (R)-pantoate from 3-methyl-2-oxobutanoate: step 2/2'},\n",
       "  'seq': 'MKITVLGCGALGQLWLTALCKQGHEVQGWLRVPQPYCSVNLVETDGSIFNESLTANDPDFLATSDLLLVTLKAWQVSDAVKSLASTLPVTTPILLIHNGMGTIEELQNIQQPLLMGTTTHAARRDGNVIIHVANGITHIGPARQQDGDYSYLADILQTVLPDVAWHNNIRAELWRKLAVNCVINPLTAIWNCPNGELRHHPQEIMQICEEVAAVIEREGHHTSAEDLRDYVMQVIDATAENISSMLQDIRALRHTEIDYINGFLLRRARAHGIAVPENTRLFEMVKRKESEYERIGTGLPRPW',\n",
       "  'mw': 33871.0},\n",
       " 'b0428': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AEA5',\n",
       "  'gene': 'b0428',\n",
       "  'locus': ['b0428', 'JW0418'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(2E,6E)-farnesyl diphosphate + H2O + heme b = diphosphate + Fe(II)-heme o',\n",
       "     '2.5.1.141')],\n",
       "   'Function': 'Converts heme B (protoheme IX) to heme O by substitution of the vinyl group on carbon 2 of heme B porphyrin ring with a hydroxyethyl farnesyl side group',\n",
       "   'Pathway': 'Porphyrin-containing compound metabolism; heme O biosynthesis; heme O from protoheme: step 1/1'},\n",
       "  'seq': 'MMFKQYLQVTKPGIIFGNLISVIGGFLLASKGSIDYPLFIYTLVGVSLVVASGCVFNNYIDRDIDRKMERTKNRVLVKGLISPAVSLVYATLLGIAGFMLLWFGANPLACWLGVMGFVVYVGVYSLYMKRHSVYGTLIGSLSGAAPPVIGYCAVTGEFDSGAAILLAIFSLWQMPHSYAIAIFRFKDYQAANIPVLPVVKGISVAKNHITLYIIAFAVATLMLSLGGYAGYKYLVVAAAVSVWWLGMALRGYKVADDRIWARKLFGFSIIAITALSVMMSVDFMVPDSHTLLAAVW',\n",
       "  'mw': 32248.0},\n",
       " 'b0429': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ABJ6',\n",
       "  'gene': 'b0429',\n",
       "  'locus': ['b0429', 'JW0419'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Cytochrome bo(3) ubiquinol terminal oxidase is the component of the aerobic respiratory chain of E.coli that predominates when cells are grown at high aeration. Has proton pump activity across the membrane in addition to electron transfer, pumping 2 protons/electron'},\n",
       "  'seq': 'MSHSTDHSGASHGSVKTYMTGFILSIILTVIPFWMVMTGAASPAVILGTILAMAVVQVLVHLVCFLHMNTKSDEGWNMTAFVFTVLIIAILVVGSIWIMWNLNYNMMMH',\n",
       "  'mw': 12029.0},\n",
       " 'b0430': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ABJ3',\n",
       "  'gene': 'b0430',\n",
       "  'locus': ['b0430', 'JW0420'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Cytochrome bo(3) ubiquinol terminal oxidase is the component of the aerobic respiratory chain of E.coli that predominates when cells are grown at high aeration. Has proton pump activity across the membrane in addition to electron transfer, pumping 2 protons/electron'},\n",
       "  'seq': 'MATDTLTHATAHAHEHGHHDAGGTKIFGFWIYLMSDCILFSILFATYAVLVNGTAGGPTGKDIFELPFVLVETFLLLFSSITYGMAAIAMYKNNKSQVISWLALTWLFGAGFIGMEIYEFHHLIVNGMGPDRSGFLSAFFALVGTHGLHVTSGLIWMAVLMVQIARRGLTSTNRTRIMCLSLFWHFLDVVWICVFTVVYLMGAM',\n",
       "  'mw': 22623.0},\n",
       " 'b0431': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ABI8',\n",
       "  'gene': 'b0431',\n",
       "  'locus': ['b0431', 'JW0421'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('2 a ubiquinol + n H(+)(in) + O2 = 2 a ubiquinone + n H(+)(out) + 2 H2O',\n",
       "     '7.1.1.3')],\n",
       "   'Function': 'Cytochrome bo(3) ubiquinol oxidase is the terminal enzyme in the aerobic respiratory chain of E.coli that predominates when cells are grown at high aeration. Catalyzes the four-electron reduction of O2 to water, using a ubiquinol as a membrane soluble electron donor for molecular oxygen reduction; ubiquinol-8 is the natural substrate for E.coli. Has proton pump activity across the membrane in addition to electron transfer, pumping 2 protons/electron and generating a proton motive force. All the redox centers of this enzyme complex are located within the largest subunit, subunit I. Protons are probably pumped via D- and K- channels found in this subunit (PubMed:11017202)',\n",
       "   'Regulation Activity': 'Competitively inhibited by piericidin A, non-competitively inhibited by 2-n-heptyl-4-hydroxyquinoline N-oxide, NaN(3) and KCN; 50% inhibition occurs at 2 uM, 2 uM, 15 mM and 10 uM, respectively. Inhibited by Zn(2+) and Cd(2+)'},\n",
       "  'seq': 'MFGKLSLDAVPFHEPIVMVTIAGIILGGLALVGLITYFGKWTYLWKEWLTSVDHKRLGIMYIIVAIVMLLRGFADAIMMRSQQALASAGEAGFLPPHHYDQIFTAHGVIMIFFVAMPFVIGLMNLVVPLQIGARDVAFPFLNNLSFWFTVVGVILVNVSLGVGEFAQTGWLAYPPLSGIEYSPGVGVDYWIWSLQLSGIGTTLTGINFFVTILKMRAPGMTMFKMPVFTWASLCANVLIIASFPILTVTVALLTLDRYLGTHFFTNDMGGNMMMYINLIWAWGHPEVYILILPVFGVFSEIAATFSRKRLFGYTSLVWATVCITVLSFIVWLHHFFTMGAGANVNAFFGITTMIIAIPTGVKIFNWLFTMYQGRIVFHSAMLWTIGFIVTFSVGGMTGVLLAVPGADFVLHNSLFLIAHFHNVIIGGVVFGCFAGMTYWWPKAFGFKLNETWGKRAFWFWIIGFFVAFMPLYALGFMGMTRRLSQQIDPQFHTMLMIAASGAVLIALGILCLVIQMYVSIRDRDQNRDLTGDPWGGRTLEWATSSPPPFYNFAVVPHVHERDAFWEMKEKGEAYKKPDHYEEIHMPKNSGAGIVIAAFSTIFGFAMIWHIWWLAIVGFAGMIITWIVKSFDEDVDYYVPVAEIEKLENQHFDEITKAGLKNGN',\n",
       "  'mw': 74368.0},\n",
       " 'b0432': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ABJ1',\n",
       "  'gene': 'b0432',\n",
       "  'locus': ['b0432', 'JW0422'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Cytochrome bo(3) ubiquinol terminal oxidase is the component of the aerobic respiratory chain of E.coli that predominates when cells are grown at high aeration. Has proton pump activity across the membrane in addition to electron transfer, pumping 2 protons/electron',\n",
       "   'Regulation Activity': 'Competitively inhibited by piericidin A, non-competitively inhibited by 2-n-heptyl-4-hydroxyquinoline N-oxide, NaN(3) and KCN; 50% inhibition occurs at 2 uM, 2 uM, 15 mM and 10 uM, respectively. Inhibited by Zn(2+) and Cd(2+)'},\n",
       "  'seq': 'MRLRKYNKSLGWLSLFAGTVLLSGCNSALLDPKGQIGLEQRSLILTAFGLMLIVVIPAILMAVGFAWKYRASNKDAKYSPNWSHSNKVEAVVWTVPILIIIFLAVLTWKTTHALEPSKPLAHDEKPITIEVVSMDWKWFFIYPEQGIATVNEIAFPANTPVYFKVTSNSVMNSFFIPRLGSQIYAMAGMQTRLHLIANEPGTYDGISASYSGPGFSGMKFKAIATPDRAAFDQWVAKAKQSPNTMSDMAAFEKLAAPSEYNQVEYFSNVKPDLFADVINKFMAHGKSMDMTQPEGEHSAHEGMEGMDMSHAESAH',\n",
       "  'mw': 34911.0},\n",
       " 'b0433': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AE16',\n",
       "  'gene': 'b0433',\n",
       "  'locus': ['b0433', 'JW0423'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Permease involved in cell wall peptidoglycan recycling (PubMed:8878601, PubMed:12426329). Transports, from the periplasm into the cytoplasm, the disaccharide N-acetylglucosaminyl-beta-1,4-anhydro-N-acetylmuramic acid (GlcNAc-anhMurNAc) and GlcNAc-anhMurNAc-peptides (PubMed:12426329). Transport is dependent on the proton motive force (PubMed:12426329). AmpG is also involved in beta-lactamase induction (PubMed:7773404)',\n",
       "   'Regulation Activity': 'Uptake is inhibited by carbonyl cyanide m-chlorophenylhydrazone (CCCP)'},\n",
       "  'seq': 'MSSQYLRIFQQPRSAILLILGFASGLPLALTSGTLQAWMTVENIDLKTIGFFSLVGQAYVFKFLWSPLMDRYTPPFFGRRRGWLLATQILLLVAIAAMGFLEPGTQLRWMAALAVVIAFCSASQDIVFDAWKTDVLPAEERGAGAAISVLGYRLGMLVSGGLALWLADKWLGWQGMYWLMAALLIPCIIATLLAPEPTDTIPVPKTLEQAVVAPLRDFFGRNNAWLILLLIVLYKLGDAFAMSLTTTFLIRGVGFDAGEVGVVNKTLGLLATIVGALYGGILMQRLSLFRALLIFGILQGASNAGYWLLSITDKHLYSMGAAVFFENLCGGMGTSAFVALLMTLCNKSFSATQFALLSALSAVGRVYVGPVAGWFVEAHGWSTFYLFSVAAAVPGLILLLVCRQTLEYTRVNDNFISRTAYPAGYAFAMWTLAAGVSLLAVWLLLLTMDALDLTHFSFLPALLEVGVLVALSGVVLGGLLDYLALRKTHLT',\n",
       "  'mw': 53245.0},\n",
       " 'b0446': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P46891',\n",
       "  'gene': 'b0446',\n",
       "  'locus': ['b0446', 'JW0436'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('4-amino-2-methyl-5-(diphosphooxymethyl)pyrimidine + H2O = 4-amino-2-methyl-5-(phosphooxymethyl)pyrimidine + H(+) + phosphate',\n",
       "     '')],\n",
       "   'Function': \"Catalyzes the hydrolysis of 4-amino-2-methyl-5-hydroxymethylpyrimidine pyrophosphate (HMP-PP) to 4-amino-2-methyl-5-hydroxymethylpyrimidine phosphate (HMP-P). Can also hydrolyze other substrates such as MeO-HMP-PP and 4-amino-2-trifluoromethyl 5-hydroxymethylpyrimidine pyrophosphate (CF3-HMP-PP) to give MeO-HMP-P and 4-amino-2-trifluoromethyl-5-hydroxymethylpyrimidine phosphate. This hydrolysis generates resistance to the antibiotics (bacimethrin, CF3-HMP) by reducing the formation of their toxic forms, 2'-methoxythiamin pyrophosphate (MeO-TPP) and CF3-HMP-PP. Also hydrolyzes pyridoxal-phosphate (PLP) and flavin mononucleotide (FMN), and purines (GMP and IMP) as secondary substrates\"},\n",
       "  'seq': 'MARLAAFDMDGTLLMPDHHLGEKTLSTLARLRERDITLTFATGRHALEMQHILGALSLDAYLITGNGTRVHSLEGELLHRDDLPADVAELVLYQQWDTRASMHIFNDDGWFTGKEIPALLQAFVYSGFRYQIIDVKKMPLGSVTKICFCGDHDDLTRLQIQLYEALGERAHLCFSATDCLEVLPVGCNKGAALTVLTQHLGLSLRDCMAFGDAMNDREMLVSVGSGFIMGNAMPQLRAELPHLPVIGHCRNQAVSHYLTHWLDYPHLPYSPE',\n",
       "  'mw': 30371.0},\n",
       " 'b0451': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P69681',\n",
       "  'gene': 'b0451',\n",
       "  'locus': ['b0451', 'JW0441'],\n",
       "  'synonyms': 'ybaG',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Involved in the uptake of ammonium/ammonia (NH(4)(+)/NH(3)) (PubMed:14668330, PubMed:15876187, PubMed:15563598, PubMed:15361618, PubMed:16910683, PubMed:30211659, PubMed:32662768). Transport is electrogenic (PubMed:30211659, PubMed:32662768). Following sequestration of NH(4)(+) at the periplasmic face, NH(4)(+) is deprotonated and neutral NH(3) is transported into the cytoplasm (PubMed:15876187, PubMed:16910683, PubMed:18362341, PubMed:19278252, PubMed:32662768). Neutral NH(3) and charged H(+) are carried separately across the membrane on a unique two-lane pathway, before recombining to NH(4)(+) inside the cell (PubMed:32662768). In vitro can also transport the larger analogs methylamine and methylammonium (PubMed:11847102, PubMed:14668330, PubMed:12023896, PubMed:15876187, PubMed:17998534, PubMed:18362341). Also acts as a sensor of the extracellular ammonium concentration (PubMed:14668330)',\n",
       "   'Regulation Activity': 'In the presence of high extracellular ammonium concentrations, transport activity is inhibited by interaction with the regulatory protein GlnK (PubMed:11847102, PubMed:14668330, PubMed:16864585). Ammonium transport through AmtB is required for GlnK sequestration (PubMed:14668330). Formation of the GlnK-AmtB complex is influenced by intracellular pools of the effector molecules ATP, ADP, Mg(2+) and 2-oxoglutarate (PubMed:16864585, PubMed:20639578). The rapid drop in the 2-oxoglutarate pool upon ammonium influx and a simultaneous, but transient, change in the ATP/ADP ratio promotes AmtB-GlnK complex formation (PubMed:20639578). The GlnK-AmtB interaction is also controlled by the level of intracellular glutamine and the uridylylation status of GlnK (PubMed:14668330). Inhibited by imidazole and thallium, which likely act by competitive binding to the periplasmic ammonium binding site (PubMed:18362341). Transport activity is independent of the membrane potential and of ATP hydrolysis (PubMed:15876187)'},\n",
       "  'seq': 'MKIATIKTGLASLAMLPGLVMAAPAVADKADNAFMMICTALVLFMTIPGIALFYGGLIRGKNVLSMLTQVTVTFALVCILWVVYGYSLAFGEGNNFFGNINWLMLKNIELTAVMGSIYQYIHVAFQGSFACITVGLIVGALAERIRFSAVLIFVVVWLTLSYIPIAHMVWGGGLLASHGALDFAGGTVVHINAAIAGLVGAYLIGKRVGFGKEAFKPHNLPMVFTGTAILYIGWFGFNAGSAGTANEIAALAFVNTVVATAAAILGWIFGEWALRGKPSLLGACSGAIAGLVGVTPACGYIGVGGALIIGVVAGLAGLWGVTMLKRLLRVDDPCDVFGVHGVCGIVGCIMTGIFAASSLGGVGFAEGVTMGHQLLVQLESIAITIVWSGVVAFIGYKLADLTVGLRVPEEQEREGLDVNSHGENAYNA',\n",
       "  'mw': 44515.0},\n",
       " 'b0452': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AGG2',\n",
       "  'gene': 'b0452',\n",
       "  'locus': ['b0452', 'JW0442'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('a fatty acyl-CoA + H2O = a fatty acid + CoA + H(+)',\n",
       "     '3.1.2.20'),\n",
       "    ('H2O + tetradecanoyl-CoA = CoA + H(+) + tetradecanoate', ''),\n",
       "    ('(3E,5Z)-tetradecadienoyl-CoA + H2O = (3E,5Z)-tetradecadienoate + CoA + H(+)',\n",
       "     ''),\n",
       "    ('dodecanoyl-CoA + H2O = CoA + dodecanoate + H(+)', ''),\n",
       "    ('(3Z,5E)-dodecadienoyl-CoA + H2O = (3Z,5E)-dodecadienoate + CoA + H(+)',\n",
       "     ''),\n",
       "    ('decanoyl-CoA + H2O = CoA + decanoate + H(+)', ''),\n",
       "    ('H2O + octanoyl-CoA = CoA + H(+) + octanoate', ''),\n",
       "    ('H2O + hexanoyl-CoA = CoA + H(+) + hexanoate', ''),\n",
       "    ('H2O + pentanoyl-CoA = CoA + H(+) + pentanoate', ''),\n",
       "    ('butanoyl-CoA + H2O = butanoate + CoA + H(+)', ''),\n",
       "    ('4-methylpentanoyl-CoA + H2O = 4-methylpentanoate + CoA + H(+)', ''),\n",
       "    ('(3R)-3-hydroxypentanoyl-CoA + H2O = (3R)-3-hydroxypentanoate + CoA + H(+)',\n",
       "     ''),\n",
       "    ('(3S)-3-hydroxypentanoyl-CoA + H2O = (3S)-3-hydroxypentanoate + CoA + H(+)',\n",
       "     ''),\n",
       "    ('H2O + hexadecanoyl-CoA = CoA + H(+) + hexadecanoate', ''),\n",
       "    ('H2O + octadecanoyl-CoA = CoA + H(+) + octadecanoate', ''),\n",
       "    ('(9Z)-octadecenoyl-CoA + H2O = (9Z)-octadecenoate + CoA + H(+)', ''),\n",
       "    ('(3R)-3-hydroxybutanoyl-CoA + H2O = (R)-3-hydroxybutanoate + CoA + H(+)',\n",
       "     ''),\n",
       "    ('(3S)-3-hydroxybutanoyl-CoA + H2O = (S)-3-hydroxybutanoate + CoA + H(+)',\n",
       "     ''),\n",
       "    ('(2E)-dodecenoyl-CoA + H2O = (2E)-dodecenoate + CoA + H(+)', ''),\n",
       "    ('3-oxododecanoyl-CoA + H2O = 3-oxododecanoate + CoA + H(+)', ''),\n",
       "    ('3-hydroxydodecanoyl-CoA + H2O = 3-hydroxydodecanoate + CoA + H(+)', '')],\n",
       "   'Function': 'Thioesterase that has relatively broad substrate specificity, hydrolyzing primarily medium- and long-chain acyl-CoA substrates to free fatty acids and CoA (PubMed:1645722, PubMed:24271180, PubMed:20547355). Functions in the thioesterase-dependent pathway of beta-oxidation of oleate and conjugated linoleate ((9Z,11E)-octadecadienoate or CLA), which provides all energy and carbon precursors required for the growth of E.coli. Thus, supports growth on oleate or conjugated linoleate as the sole source of carbon by hydrolyzing 3,5-tetradecadienoyl-CoA, the terminal metabolite of oleate beta-oxidation via the alternative thioesterase-dependent pathway, and 3,5-dodecadienoyl-CoA, the end product of CLA beta-oxidation, respectively (PubMed:18702504, PubMed:14707139). Seems to be involved in 3-hydroxyalkanoate production in E.coli (PubMed:20547355)',\n",
       "   'Regulation Activity': 'Inhibited by iodoacetamide and diethylpyrocarbonate in vitro, but is insensitive to iodoacetate treatment'},\n",
       "  'seq': 'MSQALKNLLTLLNLEKIEEGLFRGQSEDLGLRQVFGGQVVGQALYAAKETVPEERLVHSFHSYFLRPGDSKKPIIYDVETLRDGNSFSARRVAAIQNGKPIFYMTASFQAPEAGFEHQKTMPSAPAPDGLPSETQIAQSLAHLLPPVLKDKFICDRPLEVRPVEFHNPLKGHVAEPHRQVWIRANGSVPDDLRVHQYLLGYASDLNFLPVALQPHGIGFLEPGIQIATIDHSMWFHRPFNLNEWLLYSVESTSASSARGFVRGEFYTQDGVLVASTVQEGVMRNHN',\n",
       "  'mw': 31966.0},\n",
       " 'b0459': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77791',\n",
       "  'gene': 'b0459',\n",
       "  'locus': ['b0459', 'JW0448'],\n",
       "  'synonyms': 'ylaD',\n",
       "  'properties': {'Catalytic Activity': [('acetyl-CoA + D-maltose = 1-O-acetylmaltose + CoA',\n",
       "     '2.3.1.79')],\n",
       "   'Function': 'Catalyzes the CoA-dependent transfer of an acetyl group to maltose and other sugars (PubMed:1856235). Acetylates glucose exclusively at the C6 position and maltose at the C6 position of the non-reducing end glucosyl moiety. Is able to acetylate maltooligosaccharides (PubMed:12731863)'},\n",
       "  'seq': 'MSTEKEKMIAGELYRSADETLSRDRLRARQLIHRYNHSLAEEHTLRQQILADLFGQVTEAYIEPTFRCDYGYNIFLGNNFFANFDCVMLDVCPIRIGDNCMLAPGVHIYTATHPIDPVARNSGAELGKPVTIGNNVWIGGRAVINPGVTIGDNVVVASGAVVTKDVPDNVVVGGNPARIIKKL',\n",
       "  'mw': 20096.0},\n",
       " 'b0469': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P69503',\n",
       "  'gene': 'b0469',\n",
       "  'locus': ['b0469', 'JW0458'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('AMP + diphosphate = 5-phospho-alpha-D-ribose 1-diphosphate + adenine',\n",
       "     '2.4.2.7')],\n",
       "   'Function': 'Catalyzes a salvage reaction resulting in the formation of AMP, that is energically less costly than de novo synthesis',\n",
       "   'Pathway': 'Purine metabolism; AMP biosynthesis via salvage pathway; AMP from adenine: step 1/1'},\n",
       "  'seq': 'MTATAQQLEYLKNSIKSIQDYPKPGILFRDVTSLLEDPKAYALSIDLLVERYKNAGITKVVGTEARGFLFGAPVALGLGVGFVPVRKPGKLPRETISETYDLEYGTDQLEIHVDAIKPGDKVLVVDDLLATGGTIEATVKLIRRLGGEVADAAFIINLFDLGGEQRLEKQGITSYSLVPFPGH',\n",
       "  'mw': 19859.0},\n",
       " 'b0474': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P69441',\n",
       "  'gene': 'b0474',\n",
       "  'locus': ['b0474', 'JW0463'],\n",
       "  'synonyms': 'dnaW plsA',\n",
       "  'properties': {'Catalytic Activity': [('AMP + ATP = 2 ADP', '2.7.4.3')],\n",
       "   'Function': 'Catalyzes the reversible transfer of the terminal phosphate group between ATP and AMP. Plays an important role in cellular energy homeostasis and in adenine nucleotide metabolism',\n",
       "   'Pathway': 'Purine metabolism; AMP biosynthesis via salvage pathway; AMP from ADP: step 1/1'},\n",
       "  'seq': 'MRIILLGAPGAGKGTQAQFIMEKYGIPQISTGDMLRAAVKSGSELGKQAKDIMDAGKLVTDELVIALVKERIAQEDCRNGFLLDGFPRTIPQADAMKEAGINVDYVLEFDVPDELIVDRIVGRRVHAPSGRVYHVKFNPPKVEGKDDVTGEELTTRKDDQEETVRKRLVEYHQMTAPLIGYYSKEAEAGNTKYAKVDGTKPVAEVRADLEKILG',\n",
       "  'mw': 23586.0},\n",
       " 'b0475': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P23871',\n",
       "  'gene': 'b0475',\n",
       "  'locus': ['b0475', 'JW0464'],\n",
       "  'synonyms': 'popA visA',\n",
       "  'properties': {'Catalytic Activity': [('2 H(+) + heme b = Fe(2+) + protoporphyrin IX',\n",
       "     '4.98.1.1')],\n",
       "   'Function': 'Catalyzes the ferrous insertion into protoporphyrin IX',\n",
       "   'Pathway': 'Porphyrin-containing compound metabolism; protoheme biosynthesis; protoheme from protoporphyrin-IX: step 1/1'},\n",
       "  'seq': 'MRQTKTGILLANLGTPDAPTPEAVKRYLKQFLSDRRVVDTSRLLWWPLLRGVILPLRSPRVAKLYASVWMEGGSPLMVYSRQQQQALAQRLPEMPVALGMSYGSPSLESAVDELLAEHVDHIVVLPLYPQFSCSTVGAVWDELARILARKRSIPGISFIRDYADNHDYINALANSVRASFAKHGEPDLLLLSYHGIPQRYADEGDDYPQRCRTTTRELASALGMAPEKVMMTFQSRFGREPWLMPYTDETLKMLGEKGVGHIQVMCPGFAADCLETLEEIAEQNREVFLGAGGKKYEYIPALNATPEHIEMMANLVAAYR',\n",
       "  'mw': 35884.0},\n",
       " 'b0477': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AEW6',\n",
       "  'gene': 'b0477',\n",
       "  'locus': ['b0477', 'JW0466'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + guanosine = ADP + GMP + H(+)',\n",
       "     '2.7.1.73'),\n",
       "    ('ATP + inosine = ADP + H(+) + IMP', '2.7.1.73')],\n",
       "   'Function': 'Catalyzes the phosphorylation of guanosine and inosine to GMP and IMP, respectively (PubMed:7721718, PubMed:7665468, PubMed:10879466). Can also use deoxyguanosine and xanthosine, but not adenosine, uridine, cytidine or deoxythymidine (PubMed:7665468, PubMed:10879466). Shows a strong preference for guanosine (PubMed:7665468, PubMed:10879466). dATP can serve as a phosphate donor as well as ATP. Shows weaker activity with UTP and CTP (PubMed:7665468, PubMed:10879466)',\n",
       "   'Regulation Activity': \"Guanosine and inosine kinase activities are both inhibited by ppGpp, a nucleotide messenger universally produced in bacteria following nutrient starvation. ppGpp binds to Gsk, and inhibits its activity by blocking conformational dynamics and inducing tetramerization. Inhibiting purine nucleotide synthesis is required during starvation to maintain levels of the metabolite 5'-phosphoribosyl-1'-diphosphate (pRpp), which is required for the synthesis of histidine and tryptophan (PubMed:32857952). Activity is slightly increased in the presence of pyrimidine nucleotides, while it is markedly inhibited by GDP and GTP (PubMed:10879466). Activity is stimulated by K(+) ions (PubMed:7665468). Guanosine and inosine kinase activities are inhibited by Cu(2+) or Zn(2+) (PubMed:7665468)\",\n",
       "   'Pathway': 'Purine metabolism; GMP biosynthesis via salvage pathway'},\n",
       "  'seq': 'MKFPGKRKSKHYFPVNARDPLLQQFQPENETSAAWVVGIDQTLVDIEAKVDDEFIERYGLSAGHSLVIEDDVAEALYQELKQKNLITHQFAGGTIGNTMHNYSVLADDRSVLLGVMCSNIEIGSYAYRYLCNTSSRTDLNYLQGVDGPIGRCFTLIGESGERTFAISPGHMNQLRAESIPEDVIAGASALVLTSYLVRCKPGEPMPEATMKAIEYAKKYNVPVVLTLGTKFVIAENPQWWQQFLKDHVSILAMNEDEAEALTGESDPLLASDKALDWVDLVLCTAGPIGLYMAGFTEDEAKRKTQHPLLPGAIAEFNQYEFSRAMRHKDCQNPLRVYSHIAPYMGGPEKIMNTNGAGDGALAALLHDITANSYHRSNVPNSSKHKFTWLTYSSLAQVCKYANRVSYQVLNQHSPRLTRGLPEREDSLEESYWDR',\n",
       "  'mw': 48449.0},\n",
       " 'b0480': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P07024',\n",
       "  'gene': 'b0480',\n",
       "  'locus': ['b0480', 'JW0469'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('UDP-sugar + H2O = UMP + alpha-D-aldose 1-phosphate.',\n",
       "     '3.6.1.45'),\n",
       "    (\"a ribonucleoside 5'-phosphate + H2O = a ribonucleoside + phosphate\",\n",
       "     '3.1.3.5')],\n",
       "   'Function': 'Degradation of external UDP-glucose to uridine monophosphate and glucose-1-phosphate, which can then be used by the cell',\n",
       "   'Regulation Activity': 'The activity of this protein is inhibited by an intracellular protein inhibitor'},\n",
       "  'seq': 'MKLLQRGVALALLTTFTLASETALAYEQDKTYKITVLHTNDHHGHFWRNEYGEYGLAAQKTLVDGIRKEVAAEGGSVLLLSGGDINTGVPESDLQDAEPDFRGMNLVGYDAMAIGNHEFDNPLTVLRQQEKWAKFPLLSANIYQKSTGERLFKPWALFKRQDLKIAVIGLTTDDTAKIGNPEYFTDIEFRKPADEAKLVIQELQQTEKPDIIIAATHMGHYDNGEHGSNAPGDVEMARALPAGSLAMIVGGHSQDPVCMAAENKKQVDYVPGTPCKPDQQNGIWIVQAHEWGKYVGRADFEFRNGEMKMVNYQLIPVNLKKKVTWEDGKSERVLYTPEIAENQQMISLLSPFQNKGKAQLEVKIGETNGRLEGDRDKVRFVQTNMGRLILAAQMDRTGADFAVMSGGGIRDSIEAGDISYKNVLKVQPFGNVVVYADMTGKEVIDYLTAVAQMKPDSGAYPQFANVSFVAKDGKLNDLKIKGEPVDPAKTYRMATLNFNATGGDGYPRLDNKPGYVNTGFIDAEVLKAYIQKSSPLDVSVYEPKGEVSWQ',\n",
       "  'mw': 60824.0},\n",
       " 'b0484': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'Q59385',\n",
       "  'gene': 'b0484',\n",
       "  'locus': ['b0484', 'JW0473'],\n",
       "  'synonyms': 'atcU f834 ybaR',\n",
       "  'properties': {'Catalytic Activity': [('ATP + Cu(+)(in) + H2O = ADP + Cu(+)(out) + H(+) + phosphate',\n",
       "     '7.2.2.8')],\n",
       "   'Function': 'mRNA is subject to programmed ribosomal frameshifting which produces a cytoplasmic copper chaperone CopA(Z) that corresponds to the first HMA domain (PubMed:28107647). The soluble form is essential for cell survivial in the presence of CuSO(4); in growth competition experiments between wild-type and a version that prevents expression of CopA(Z) after 50 generations the non-CopA(Z) version is nearly extinct (PubMed:28107647). The first HMA domain (residues 1-70) can be replaced by B.subtilis Cu chaperone CopZ (PubMed:25899340)',\n",
       "   'Regulation Activity': 'Export is inhibited by vanadate (PubMed:10639134). Phosphorylation is inhibited by vanadate and sensitive to KOH and hydroxylamine; it is not inhibited by azide (PubMed:12351646). Phosphorylation is Cu(+) not Cu(2+)-dependent (PubMed:12351646). ATPase activity is inhibited by bathocuproindisulfonate (BCDS), which chelates Cu(+) but not Cu(2+), and stimulated 3-4-fold by Cu(+) (PubMed:12351646, PubMed:25899340). ATPase activity is inhibited by Cu(2+) plus DTT or Ag(+) (PubMed:12351646)'},\n",
       "  'seq': 'MSQTIDLTLDGLSCGHCVKRVKESLEQRPDVEQADVSITEAHVTGTASAEQLIETIKQAGYDASVSHPKAKPLAESSIPSEALTAVSEALPAATADDDDSQQLLLSGMSCASCVTRVQNALQSVPGVTQARVNLAERTALVMGSASPQDLVQAVEKAGYGAEAIEDDAKRRERQQETAVATMKRFRWQAIVALAVGIPVMVWGMIGDNMMVTADNRSLWLVIGLITLAVMVFAGGHFYRSAWKSLLNGAATMDTLVALGTGVAWLYSMSVNLWPQWFPMEARHLYYEASAMIIGLINLGHMLEARARQRSSKALEKLLDLTPPTARLVTDEGEKSVPLAEVQPGMLLRLTTGDRVPVDGEITQGEAWLDEAMLTGEPIPQQKGEGDSVHAGTVVQDGSVLFRASAVGSHTTLSRIIRMVRQAQSSKPEIGQLADKISAVFVPVVVVIALVSAAIWYFFGPAPQIVYTLVIATTVLIIACPCALGLATPMSIISGVGRAAEFGVLVRDADALQRASTLDTVVFDKTGTLTEGKPQVVAVKTFADVDEAQALRLAAALEQGSSHPLARAILDKAGDMQLPQVNGFRTLRGLGVSGEAEGHALLLGNQALLNEQQVGTKAIEAEITAQASQGATPVLLAVDGKAVALLAVRDPLRSDSVAALQRLHKAGYRLVMLTGDNPTTANAIAKEAGIDEVIAGVLPDGKAEAIKHLQSEGRQVAMVGDGINDAPALAQADVGIAMGGGSDVAIETAAITLMRHSLMGVADALAISRATLHNMKQNLLGAFIYNSIGIPVAAGILWPFTGTLLNPVVAGAAMALSSITVVSNANRLLRFKPKE',\n",
       "  'mw': 87873.0},\n",
       " 'b0485': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77454',\n",
       "  'gene': 'b0485',\n",
       "  'locus': ['b0485', 'JW0474'],\n",
       "  'synonyms': 'ybaS',\n",
       "  'properties': {'Catalytic Activity': [('H2O + L-glutamine = L-glutamate + NH4(+)',\n",
       "     '3.5.1.2')]},\n",
       "  'seq': 'MLDANKLQQAVDQAYTQFHSLNGGQNADYIPFLANVPGQLAAVAIVTCDGNVYSAGDSDYRFALESISKVCTLALALEDVGPQAVQDKIGADPTGLPFNSVIALELHGGKPLSPLVNAGAIATTSLINAENVEQRWQRILHIQQQLAGEQVALSDEVNQSEQTTNFHNRAIAWLLYSAGYLYCDAMEACDVYTRQCSTLLNTIELATLGATLAAGGVNPLTHKRVLQADNVPYILAEMMMEGLYGRSGDWAYRVGLPGKSGVGGGILAVVPGVMGIAAFSPPLDEDGNSVRGQKMVASVAKQLGYNVFKG',\n",
       "  'mw': 32903.0},\n",
       " 'b0494': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ADA1',\n",
       "  'gene': 'b0494',\n",
       "  'locus': ['b0494', 'JW0483'],\n",
       "  'synonyms': 'apeA pldC',\n",
       "  'properties': {'Catalytic Activity': [('a fatty acyl-CoA + H2O = a fatty acid + CoA + H(+)',\n",
       "     ''),\n",
       "    ('H2O + hexadecanoyl-CoA = CoA + H(+) + hexadecanoate', '3.1.2.2'),\n",
       "    ('(9Z)-hexadecenoyl-CoA + H2O = (9Z)-hexadecenoate + CoA + H(+)', ''),\n",
       "    ('H2O + octadecanoyl-CoA = CoA + H(+) + octadecanoate', ''),\n",
       "    ('(9Z)-octadecenoyl-CoA + H2O = (9Z)-octadecenoate + CoA + H(+)', ''),\n",
       "    ('(9Z)-octadecenoyl-[ACP] + H2O = (9Z)-octadecenoate + H(+) + holo-[ACP]',\n",
       "     '3.1.2.14'),\n",
       "    ('(11Z)-octadecenoyl-CoA + H2O = (11Z)-octadecenoate + CoA + H(+)', ''),\n",
       "    ('H2O + tetradecanoyl-CoA = CoA + H(+) + tetradecanoate', ''),\n",
       "    ('(5Z,8Z,11Z,14Z)-eicosatetraenoyl-CoA + H2O = (5Z,8Z,11Z,14Z)-eicosatetraenoate + CoA + H(+)',\n",
       "     ''),\n",
       "    ('dodecanoyl-CoA + H2O = CoA + dodecanoate + H(+)', ''),\n",
       "    ('decanoyl-CoA + H2O = CoA + decanoate + H(+)', ''),\n",
       "    ('H2O + hexanoyl-CoA = CoA + H(+) + hexanoate', ''),\n",
       "    ('a 1-acyl-sn-glycero-3-phosphocholine + H2O = a fatty acid + H(+) + sn-glycerol 3-phosphocholine',\n",
       "     '3.1.1.5'),\n",
       "    ('a phenyl acetate + H2O = a phenol + acetate + H(+)', '3.1.1.2'),\n",
       "    ('a butanoate ester + H2O = an aliphatic alcohol + butanoate + H(+)', ''),\n",
       "    ('a hexanoate ester + H2O = an aliphatic alcohol + H(+) + hexanoate', ''),\n",
       "    ('an octanoate ester + H2O = an aliphatic alcohol + H(+) + octanoate',\n",
       "     '')],\n",
       "   'Function': 'TesA is a multifunctional esterase that can act as a thioesterase, lysophospholipase and protease (PubMed:8098033, PubMed:8432696, PubMed:1864840, PubMed:4945109, PubMed:4554913, PubMed:238979, PubMed:791643, PubMed:8132479, PubMed:9070299, PubMed:10423542). TesA functions as a thioesterase specific for fatty acyl thioesters of greater than ten carbons, with highest activity on palmitoyl-CoA, cis-vaccenoyl-CoA and palmitoleoyl-CoA (PubMed:8098033, PubMed:4554913, PubMed:8132479, PubMed:9070299, PubMed:10423542). TesA also possesses an arylesterase activity towards short acyl-chain aromatic esters such as alpha-naphthyl acetate, alpha-naphthyl butyrate, benzyl acetate and phenyl acetate (PubMed:9070299). Also able to hydrolyze short acyl-chain triacylglycerols such as triacetin and tributyrin, and p-nitrophenyl esters such as p-nitrophenyl hexanoate and p-nitrophenyl butyrate (PubMed:9070299). The protease activity is mainly active on small peptides (PubMed:8432696, PubMed:9070299). TesA is also able to hydrolyze p-nitrophenyl esters of N-substituted amino acids such as N-benzyloxycarbonyl-L-Phe-p-nitrophenyl ester (Z-L-Phe-ONp) and N-benzyloxycarbonyl-L-Tyr-p-nitrophenyl ester (Z-L-Tyr-ONp), however it is unable to hydrolyze N-acetyl-L-Phe ethyl ester and its Tyr analog (PubMed:8432696, PubMed:791643, PubMed:10423542). TesA also hydrolyzes N-benzyloxycarbonyl-L-Phe beta-nitrophenyl ester (Cbz-Phe-ONap) and N-acetyl-DL-Phe-2-naphthyl ester (chymotrypsin-like specificity) (PubMed:8432696, PubMed:4945109). Shows a slow proteolytic activity against denatured casein (PubMed:4945109). The lysophospholipase activity of TesA is able to hydrolyze 1-palmitoyl-sn-glycero-3-phosphocholine, 1-acyl-sn-glycero-3-phosphoglycerol, 1- and 2-acyl-sn-glycero-3-phosphoethanolamine (PubMed:1864840, PubMed:238979, PubMed:10423542)',\n",
       "   'Regulation Activity': 'Thioesterase activity is inhibited by iodoacetamide and photoactivated methylene blue, and slowly inhibited by 2,4-dinitrofluorobenzene (PubMed:4554913). Protease and lysophospholipase activities are inhibited by diisopropylfluorophosphate (DFP) (PubMed:1864840, PubMed:4945109, PubMed:238979). Lysophospholipase activity is inhibited by Fe(2+), Fe(3+) and Al(3+) ions (PubMed:238979). Diethyl p-nitrophenyl phosphate (DENP) irreversibly inhibits both the protease and thioesterase activities (PubMed:12846577)'},\n",
       "  'seq': 'MMNFNNVFRWHLPFLFLVLLTFRAAAADTLLILGDSLSAGYRMSASAAWPALLNDKWQSKTSVVNASISGDTSQQGLARLPALLKQHQPRWVLVELGGNDGLRGFQPQQTEQTLRQILQDVKAANAEPLLMQIRLPANYGRRYNEAFSAIYPKLAKEFDVPLLPFFMEEVYLKPQWMQDDGIHPNRDAQPFIADWMAKQLQPLVNHDS',\n",
       "  'mw': 23622.0},\n",
       " 'b0505': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77731',\n",
       "  'gene': 'b0505',\n",
       "  'locus': ['b0505', 'JW0493'],\n",
       "  'synonyms': 'glxA2 ybbT',\n",
       "  'properties': {'Catalytic Activity': [('(S)-ureidoglycolate = glyoxylate + urea',\n",
       "     '4.3.2.3')],\n",
       "   'Function': 'Catalyzes the catabolism of the allantoin degradation intermediate (S)-ureidoglycolate, generating urea and glyoxylate. Involved in the anaerobic utilization of allantoin as sole nitrogen source. Reinforces the induction of genes involved in the degradation of allantoin and glyoxylate by producing glyoxylate',\n",
       "   'Pathway': 'Nitrogen metabolism; (S)-allantoin degradation'},\n",
       "  'seq': 'MKLQVLPLSQEAFSAYGDVIETQQRDFFHINNGLVERYHDLALVEILEQDCTLISINRAQPANLPLTIHELERHPLGTQAFIPMKGEVFVVVVALGDDKPDLSTLRAFITNGEQGVNYHRNVWHHPLFAWQRVTDFLTIDRGGSDNCDVESIPEQELCFA',\n",
       "  'mw': 18170.0},\n",
       " 'b0507': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AEP7',\n",
       "  'gene': 'b0507',\n",
       "  'locus': ['b0507', 'JW0495'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('2 glyoxylate + H(+) = 2-hydroxy-3-oxopropanoate + CO2',\n",
       "     '4.1.1.47')],\n",
       "   'Function': 'Catalyzes the condensation of two molecules of glyoxylate to give 2-hydroxy-3-oxopropanoate (also termed tartronate semialdehyde)',\n",
       "   'Pathway': 'Organic acid metabolism; glycolate degradation; 3-phospho-D-glycerate from glycolate: step 2/4'},\n",
       "  'seq': 'MAKMRAVDAAMYVLEKEGITTAFGVPGAAINPFYSAMRKHGGIRHILARHVEGASHMAEGYTRATAGNIGVCLGTSGPAGTDMITALYSASADSIPILCITGQAPRARLHKEDFQAVDIEAIAKPVSKMAVTVREAALVPRVLQQAFHLMRSGRPGPVLVDLPFDVQVAEIEFDPDMYEPLPVYKPAASRMQIEKAVEMLIQAERPVIVAGGGVINADAAALLQQFAELTSVPVIPTLMGWGCIPDDHELMAGMVGLQTAHRYGNATLLASDMVFGIGNRFANRHTGSVEKYTEGRKIVHIDIEPTQIGRVLCPDLGIVSDAKAALTLLVEVAQEMQKAGRLPCRKEWVADCQQRKRTLLRKTHFDNVPVKPQRVYEEMNKAFGRDVCYVTTIGLSQIAAAQMLHVFKDRHWINCGQAGPLGWTIPAALGVCAADPKRNVVAISGDFDFQFLIEELAVGAQFNIPYIHVLVNNAYLGLIRQSQRAFDMDYCVQLAFENINSSEVNGYGVDHVKVAEGLGCKAIRVFKPEDIAPAFEQAKALMAQYRVPVVVEVILERVTNISMGSELDNVMEFEDIADNAADAPTETCFMHYE',\n",
       "  'mw': 64732.0},\n",
       " 'b0508': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P30147',\n",
       "  'gene': 'b0508',\n",
       "  'locus': ['b0508', 'JW0496'],\n",
       "  'synonyms': 'gip ybbG',\n",
       "  'properties': {'Catalytic Activity': [('3-hydroxypyruvate = 2-hydroxy-3-oxopropanoate',\n",
       "     '5.3.1.22')],\n",
       "   'Function': 'Catalyzes the reversible isomerization between hydroxypyruvate and 2-hydroxy-3-oxopropanoate (also termed tartronate semialdehyde). Does not catalyze the isomerization of D-fructose to D-glucose or that of D-xylulose to D-xylose. Also does not catalyze racemization of serine, alanine, glycerate or lactate',\n",
       "   'Regulation Activity': \"Not stimulated by addition of pyridoxal 5'-phosphate (0.1 mM), FAD, NAD(+), NADP(+) or ATP (1 mM each). EDTA (10 mM) and metal ions (1 mM) such as Ca(2+), Co(2+), Mg(2+), Ni(2+), Zn(2+) do not affect the enzyme activity\"},\n",
       "  'seq': 'MLRFSANLSMLFGEYDFLARFEKAAQCGFRGVEFMFPYDYDIEELKHVLASNKLEHTLHNLPAGDWAAGERGIACIPGREEEFRDGVAAAIRYARALGNKKINCLVGKTPAGFSSEQIHATLVENLRYAANMLMKEDILLLIEPINHFDIPGFHLTGTRQALKLIDDVGCCNLKIQYDIYHMQRMEGELTNTMTQWADKIGHLQIADNPHRGEPGTGEINYDYLFKVIENSDYNGWVGCEYKPQTTTEAGLRWMDPYR',\n",
       "  'mw': 29377.0},\n",
       " 'b0509': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77161',\n",
       "  'gene': 'b0509',\n",
       "  'locus': ['b0509', 'JW0497'],\n",
       "  'synonyms': 'glxB1 ybbQ',\n",
       "  'properties': {'Catalytic Activity': [('(R)-glycerate + NADP(+) = 2-hydroxy-3-oxopropanoate + H(+) + NADPH',\n",
       "     '1.1.1.60'),\n",
       "    ('(R)-glycerate + NAD(+) = 2-hydroxy-3-oxopropanoate + H(+) + NADH',\n",
       "     '1.1.1.60')],\n",
       "   'Pathway': 'Organic acid metabolism; glycolate degradation; 3-phospho-D-glycerate from glycolate: step 3/4'},\n",
       "  'seq': 'MKLGFIGLGIMGTPMAINLARAGHQLHVTTIGPVADELLSLGAVSVETARQVTEASDIIFIMVPDTPQVEEVLFGENGCTKASLKGKTIVDMSSISPIETKRFARQVNELGGDYLDAPVSGGEIGAREGTLSIMVGGDEAVFERVKPLFELLGKNITLVGGNGDGQTCKVANQIIVALNIEAVSEALLFASKAGADPVRVRQALMGGFASSRILEVHGERMIKRTFNPGFKIALHQKDLNLALQSAKALALNLPNTATCQELFNTCAANGGSQLDHSALVQALELMANHKLA',\n",
       "  'mw': 30801.0},\n",
       " 'b0511': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P75712',\n",
       "  'gene': 'b0511',\n",
       "  'locus': ['b0511', 'JW0499'],\n",
       "  'synonyms': 'glxB2',\n",
       "  'properties': {'Catalytic Activity': [('(S)-allantoin(in) + H(+)(in) = (S)-allantoin(out) + H(+)(out)',\n",
       "     '')],\n",
       "   'Function': 'Uptake of allantoin into the cell'},\n",
       "  'seq': 'MEHQRKLFQQRGYSEDLLPKTQSQRTWKTFNYFTLWMGSVHNVPNYVMVGGFFILGLSTFSIMLAIILSAFFIAAVMVLNGAAGSKYGVPFAMILRASYGVRGALFPGLLRGGIAAIMWFGLQCYAGSLACLILIGKIWPGFLTLGGDFTLLGLSLPGLITFLIFWLVNVGIGFGGGKVLNKFTAILNPCIYIVFGGMAIWAISLVGIGPIFDYIPSGIQKAENGGFLFLVVINAVVAVWAAPAVSASDFTQNAHSFREQALGQTLGLVVAYILFAVAGVCIIAGASIHYGADTWNVLDIVQRWDSLFASFFAVLVILMTTISTNATGNIIPAGYQIAAIAPTKLTYKNGVLIASIISLLICPWKLMENQDSIYLFLDIIGGMLGPVIGVMMAHYFVVMRGQINLDELYTAPGDYKYYDNGFNLTAFSVTLVAVILSLGGKFIHFMEPLSRVSWFVGVIVAFAAYALLKKRTTAEKTGEQKTIG',\n",
       "  'mw': 52456.0},\n",
       " 'b0512': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77671',\n",
       "  'gene': 'b0512',\n",
       "  'locus': ['b0512', 'JW0500'],\n",
       "  'synonyms': 'glxB3 ybbX',\n",
       "  'properties': {'Catalytic Activity': [('(S)-allantoin + H2O = allantoate + H(+)',\n",
       "     '3.5.2.5')],\n",
       "   'Function': 'Catalyzes the conversion of allantoin (5-ureidohydantoin) to allantoic acid by hydrolytic cleavage of the five-member hydantoin ring',\n",
       "   'Regulation Activity': 'Severely inhibited by copper, and competitively inhibited by dithiothreitol (DTT). Zinc concentrations above 2.5 mM and cobalt or nickel concentrations above 1 mM are inhibitors',\n",
       "   'Pathway': 'Nitrogen metabolism; (S)-allantoin degradation; allantoate from (S)-allantoin: step 1/1'},\n",
       "  'seq': 'MSFDLIIKNGTVILENEARVVDIAVKGGKIAAIGQDLGDAKEVMDASGLVVSPGMVDAHTHISEPGRSHWEGYETGTRAAAKGGITTMIEMPLNQLPATVDRASIELKFDAAKGKLTIDAAQLGGLVSYNIDRLHELDEVGVVGFKCFVATCGDRGIDNDFRDVNDWQFFKGAQKLGELGQPVLVHCENALICDELGEEAKREGRVTAHDYVASRPVFTEVEAIRRVLYLAKVAGCRLHVCHVSSPEGVEEVTRARQEGQDVTCESCPHYFVLDTDQFEEIGTLAKCSPPIRDLENQKGMWEKLFNGEIDCLVSDHSPCPPEMKAGNIMKAWGGIAGLQSCMDVMFDEAVQKRGMSLPMFGKLMATNAADIFGLQQKGRIAPGKDADFVFIQPNSSYVLTNDDLEYRHKVSPYVGRTIGARITKTILRGDVIYDIEQGFPVAPKGQFILKHQQ',\n",
       "  'mw': 49602.0},\n",
       " 'b0514': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77364',\n",
       "  'gene': 'b0514',\n",
       "  'locus': ['b0514', 'JW0502'],\n",
       "  'synonyms': 'glxB5 ybbZ',\n",
       "  'properties': {'Catalytic Activity': [('(R)-glycerate + ATP = (2R)-3-phosphoglycerate + ADP + H(+)',\n",
       "     '2.7.1.31')],\n",
       "   'Regulation Activity': 'Inhibited by EDTA, p-hydroxy-mercuribenzoate and iodoacetate but not by NaF',\n",
       "   'Pathway': 'Organic acid metabolism; glycolate degradation; 3-phospho-D-glycerate from glycolate: step 4/4'},\n",
       "  'seq': 'MKIVIAPDSFKESLSAEKCCQAIKAGFSTLFPDANYICLPIADGGEGTVDAMVAATGGNIVTLEVCGPMGEKVNAFYGLTGDGKTAVIEMAAASGLMLVAPEKRNPLLASSFGTGELIRHALDNDIRHIILGIGGSATVDGGMGMAQALGVRFLDADGQALAANGGNLARVASIEMDECDPRLANCHIEVACDVDNPLVGARGAAAVFGPQKGATPEMVEELEQGLQNYARVLQQQTEINVCQMAGGGAAGGMGIAAAVFLNADIKPGIEIVLNAVNLAQAVQGAALVITGEGRIDSQTAGGKAPLGVASVAKQFNVPVIGIAGVLGDGVEVVHQYGIDAVFSILPRLAPLAEVLASGETNLFNSARNIACAIKIGQGIKN',\n",
       "  'mw': 38734.0},\n",
       " 'b0516': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77425',\n",
       "  'gene': 'b0516',\n",
       "  'locus': ['b0516', 'JW0504'],\n",
       "  'synonyms': 'glxB7 ylbB',\n",
       "  'properties': {'Catalytic Activity': [('allantoate + 2 H(+) + H2O = (S)-2-ureidoglycine + CO2 + NH4(+)',\n",
       "     '3.5.3.9')],\n",
       "   'Function': 'Involved in the anaerobic nitrogen utilization via the assimilation of allantoin (PubMed:10601204, PubMed:20038185). Catalyzes specifically the hydrolysis of allantoate to yield CO2, NH3 and S-ureidoglycine, which is unstable and readily undergoes a second deamination by S-ureidoglycine aminohydrolase AllE to yield S-ureidoglycolate and NH3 (PubMed:20038185). In vivo, the spontaneous release of S-ureidoglycolate and ammonia from S-ureidoglycine appears to be too slow to sustain an efficient flux of nitrogen (PubMed:20038185)',\n",
       "   'Regulation Activity': 'Sulfate could be an allosteric effector of the enzyme that is responsible for stabilizing substrate binding. In addition, this anion effector may act as a counterion during enzyme-mediated catalysis',\n",
       "   'Pathway': 'Nitrogen metabolism; (S)-allantoin degradation'},\n",
       "  'seq': 'MITHFRQAIEETLPWLSSFGADPAGGMTRLLYSPEWLETQQQFKKRMAASGLETRFDEVGNLYGRLNGTEYPQEVVLSGSHIDTVVNGGNLDGQFGALAAWLAIDWLKTQYGAPLRTVEVVAMAEEEGSRFPYVFWGSKNIFGLANPDDVRNICDAKGNSFVDAMKACGFTLPNAPLTPRQDIKAFVELHIEQGCVLESNGQSIGVVNAIVGQRRYTVTLNGESNHAGTTPMGYRRDTVYAFSRICHQSVEKAKRMGDPLVLTFGKVEPRPNTVNVVPGKTTFTIDCRHTDAAVLRDFTQQLENDMRAICDEMDIGIDIDLWMDEEPVPMNKELVATLTELCEREKLNYRVMHSGAGHDAQIFAPRVPTCMIFIPSINGISHNPAERTNITDLAEGVKTLALMLYQLAWQK',\n",
       "  'mw': 45694.0},\n",
       " 'b0517': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77555',\n",
       "  'gene': 'b0517',\n",
       "  'locus': ['b0517', 'JW0505'],\n",
       "  'synonyms': 'glxB8 ylbC',\n",
       "  'properties': {'Catalytic Activity': [('(S)-ureidoglycolate + NAD(+) = H(+) + N-carbamoyl-2-oxoglycine + NADH',\n",
       "     '1.1.1.350')],\n",
       "   'Function': 'AllD plays a pivotal role as a metabolic branch-point enzyme in nitrogen utilization via the assimilation of allantoin (PubMed:10601204). It is able to utilize allantoin as a sole source of nitrogen under anaerobic conditions (PubMed:10601204). Catalyzes the oxidation of ureidoglycolate to oxalurate (PubMed:23284870)',\n",
       "   'Pathway': 'Nitrogen metabolism; (S)-allantoin degradation; oxalurate from (S)-ureidoglycolate: step 1/1'},\n",
       "  'seq': 'MKISRETLHQLIENKLCQAGLKREHAATVAEVLVYADARGIHSHGAVRVEYYAERISKGGTNREPEFRLEETGPCSAILHADNAAGQVAAKMGMEHAIKTAQQNGVAVVGISRMGHSGAISYFVQQAARAGFIGISMCQSDPMVVPFGGAEIYYGTNPLAFAAPGEGDEILTFDMATTVQAWGKVLDARSRNMSIPDTWAVDKNGVPTTDPFAVHALLPAAGPKGYGLMMMIDVLSGVLLGLPFGRQVSSMYDDLHAGRNLGQLHIVINPNFFSSSELFRQHLSQTMRELNAITPAPGFNQVYYPGQDQDIKQRKAAVEGIEIVDDIYQYLISDALYNTSYETKNPFAQ',\n",
       "  'mw': 37967.0},\n",
       " 'b0521': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37306',\n",
       "  'gene': 'b0521',\n",
       "  'locus': ['b0521', 'JW0510'],\n",
       "  'synonyms': 'ybcF',\n",
       "  'properties': {'Catalytic Activity': [('ATP + hydrogencarbonate + NH4(+) = ADP + carbamoyl phosphate + H(+) + H2O',\n",
       "     '2.7.2.2')],\n",
       "   'Pathway': 'Metabolic intermediate metabolism; carbamoyl phosphate degradation; CO(2) and NH(3) from carbamoyl phosphate: step 1/1'},\n",
       "  'seq': 'MKTLVVALGGNALLQRGEALTAENQYRNIASAVPALARLARSYRLAIVHGNGPQVGLLALQNLAWKEVEPYPLDVLVAESQGMIGYMLAQSLSAQPQMPPVTTVLTRIEVSPDDPAFLQPEKFIGPVYQPEEQEALEAAYGWQMKRDGKYLRRVVASPQPRKILDSEAIELLLKEGHVVICSGGGGVPVTDDGAGSEAVIDKDLAAALLAEQINADGLVILTDADAVYENWGTPQQRAIRHATPDELAPFAKADGSMGPNVTAVSGYVRSRGKPAWIGALSRIEETLAGEAGTCISL',\n",
       "  'mw': 31644.0},\n",
       " 'b0522': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P09029',\n",
       "  'gene': 'b0522',\n",
       "  'locus': ['b0522', 'JW0511'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('5-amino-1-(5-phospho-beta-D-ribosyl)imidazole + ATP + hydrogencarbonate = 5-carboxyamino-1-(5-phospho-D-ribosyl)imidazole + ADP + 2 H(+) + phosphate',\n",
       "     '6.3.4.18')],\n",
       "   'Function': 'Catalyzes the ATP-dependent conversion of 5-aminoimidazole ribonucleotide (AIR) and HCO(3)(-) to N5-carboxyaminoimidazole ribonucleotide (N5-CAIR)',\n",
       "   'Pathway': 'Purine metabolism; IMP biosynthesis via de novo pathway; 5-amino-1-(5-phospho-D-ribosyl)imidazole-4-carboxylate from 5-amino-1-(5-phospho-D-ribosyl)imidazole (N5-CAIR route): step 1/2'},\n",
       "  'seq': 'MKQVCVLGNGQLGRMLRQAGEPLGIAVWPVGLDAEPAAVPFQQSVITAEIERWPETALTRELARHPAFVNRDVFPIIADRLTQKQLFDKLHLPTAPWQLLAERSEWPAVFDRLGELAIVKRRTGGYDGRGQWRLRANETEQLPAECYGECIVEQGINFSGEVSLVGARGFDGSTVFYPLTHNLHQDGILRTSVAFPQANAQQQAQAEEMLSAIMQELGYVGVMAMECFVTPQGLLINELAPRVHNSGHWTQNGASISQFELHLRAITDLPLPQPVVNNPSVMINLIGSDVNYDWLKLPLVHLHWYDKEVRPGRKVGHLNLTDSDTSRLTATLEALIPLLPPEYASGVIWAQSKFG',\n",
       "  'mw': 39461.0},\n",
       " 'b0523': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AG18',\n",
       "  'gene': 'b0523',\n",
       "  'locus': ['b0523', 'JW0512'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('5-carboxyamino-1-(5-phospho-D-ribosyl)imidazole + H(+) = 5-amino-1-(5-phospho-D-ribosyl)imidazole-4-carboxylate',\n",
       "     '5.4.99.18')],\n",
       "   'Function': 'Catalyzes the conversion of N5-carboxyaminoimidazole ribonucleotide (N5-CAIR) to 4-carboxy-5-aminoimidazole ribonucleotide (CAIR)',\n",
       "   'Pathway': 'Purine metabolism; IMP biosynthesis via de novo pathway; 5-amino-1-(5-phospho-D-ribosyl)imidazole-4-carboxylate from 5-amino-1-(5-phospho-D-ribosyl)imidazole (N5-CAIR route): step 2/2'},\n",
       "  'seq': 'MSSRNNPARVAIVMGSKSDWATMQFAAEIFEILNVPHHVEVVSAHRTPDKLFSFAESAEENGYQVIIAGAGGAAHLPGMIAAKTLVPVLGVPVQSAALSGVDSLYSIVQMPRGIPVGTLAIGKAGAANAALLAAQILATHDKELHQRLNDWRKAQTDEVLENPDPRGAA',\n",
       "  'mw': 17780.0},\n",
       " 'b0524': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P43341',\n",
       "  'gene': 'b0524',\n",
       "  'locus': ['b0524', 'JW0513'],\n",
       "  'synonyms': 'ybbF',\n",
       "  'properties': {'Catalytic Activity': [('H2O + UDP-2-N,3-O-bis[(3R)-3-hydroxytetradecanoyl]-alpha-D-glucosamine = 2-N,3-O-bis[(3R)-3-hydroxytetradecanoyl]-alpha-D-glucosaminyl 1-phosphate + 2 H(+) + UMP',\n",
       "     '3.6.1.54')],\n",
       "   'Function': 'Hydrolyzes the pyrophosphate bond of UDP-2,3-diacylglucosamine to yield 2,3-diacylglucosamine 1-phosphate (lipid X) and UMP by catalyzing the attack of water at the alpha-P atom (PubMed:12000770). Involved in the biosynthesis of lipid A, a phosphorylated glycolipid that anchors the lipopolysaccharide to the outer membrane of the cell (PubMed:12000770, PubMed:12000771). Is essential for E.coli growth (PubMed:12000771). Does not cleave the unacylated UDP-GlcNAc, the mono-acylated UDP-3-O-(R)-3-hydroxymyristoyl-GlcNAc, and CDP-diacylglycerol (PubMed:12000770)',\n",
       "   'Regulation Activity': 'Inhibited by a sulfonyl piperazine compound that shows antibacterial activity against E.coli; LpxH is the cellular target of this compound (PubMed:25733621). Inhibited by 0.01% (or more) Triton X-100 in vitro (PubMed:12000770)',\n",
       "   'Pathway': 'Glycolipid biosynthesis; lipid IV(A) biosynthesis; lipid IV(A) from (3R)-3-hydroxytetradecanoyl-[acyl-carrier-protein] and UDP-N-acetyl-alpha-D-glucosamine: step 4/6'},\n",
       "  'seq': 'MATLFIADLHLCVEEPAITAGFLRFLAGEARKADALYILGDLFEAWIGDDDPNPLHRKMAAAIKAVSDSGVPCYFIHGNRDFLLGKRFARESGMTLLPEEKVLELYGRRVLIMHGDTLCTDDAGYQAFRAKVHKPWLQTLFLALPLFVRKRIAARMRANSKEANSSKSLAIMDVNQNAVVSAMEKHQVQWLIHGHTHRPAVHELIANQQPAFRVVLGAWHTEGSMVKVTADDVELIHFPF',\n",
       "  'mw': 26894.0},\n",
       " 'b0526': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P21888',\n",
       "  'gene': 'b0526',\n",
       "  'locus': ['b0526', 'JW0515'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + L-cysteine + tRNA(Cys) = AMP + diphosphate + L-cysteinyl-tRNA(Cys)',\n",
       "     '6.1.1.16')]},\n",
       "  'seq': 'MLKIFNTLTRQKEEFKPIHAGEVGMYVCGITVYDLCHIGHGRTFVAFDVVARYLRFLGYKLKYVRNITDIDDKIIKRANENGESFVAMVDRMIAEMHKDFDALNILRPDMEPRATHHIAEIIELTEQLIAKGHAYVADNGDVMFDVPTDPTYGVLSRQDLDQLQAGARVDVVDDKRNPMDFVLWKMSKEGEPSWPSPWGAGRPGWHIECSAMNCKQLGNHFDIHGGGSDLMFPHHENEIAQSTCAHDGQYVNYWMHSGMVMVDREKMSKSLGNFFTVRDVLKYYDAETVRYFLMSGHYRSQLNYSEENLKQARAALERLYTALRGTDKTVAPAGGEAFEARFIEAMDDDFNTPEAYSVLFDMAREVNRLKAEDMAAANAMASHLRKLSAVLGLLEQEPEAFLQSGAQADDSEVAEIEALIQQRLDARKAKDWAAADAARDRLNEMGIVLEDGPQGTTWRRK',\n",
       "  'mw': 52202.0},\n",
       " 'b0529': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P24186',\n",
       "  'gene': 'b0529',\n",
       "  'locus': ['b0529', 'JW0518'],\n",
       "  'synonyms': 'ads',\n",
       "  'properties': {'Catalytic Activity': [('(6R)-5,10-methylene-5,6,7,8-tetrahydrofolate + NADP(+) = 5,10-methenyltetrahydrofolate + NADPH',\n",
       "     '1.5.1.5'),\n",
       "    ('5,10-methenyltetrahydrofolate + H2O = (6S)-10-formyltetrahydrofolate + H(+)',\n",
       "     '3.5.4.9')],\n",
       "   'Function': 'Catalyzes the oxidation of 5,10-methylenetetrahydrofolate to 5,10-methenyltetrahydrofolate and then the hydrolysis of 5,10-methenyltetrahydrofolate to 10-formyltetrahydrofolate. This enzyme is specific for NADP',\n",
       "   'Pathway': 'One-carbon metabolism; tetrahydrofolate interconversion'},\n",
       "  'seq': 'MAAKIIDGKTIAQQVRSEVAQKVQARIAAGLRAPGLAVVLVGSNPASQIYVASKRKACEEVGFVSRSYDLPETTSEAELLELIDTLNADNTIDGILVQLPLPAGIDNVKVLERIHPDKDVDGFHPYNVGRLCQRAPRLRPCTPRGIVTLLERYNIDTFGLNAVVIGASNIVGRPMSMELLLAGCTTTVTHRFTKNLRHHVENADLLIVAVGKPGFIPGDWIKEGAIVIDVGINRLENGKVVGDVVFEDAAKRASYITPVPGGVGPMTVATLIENTLQACVEYHDPQDE',\n",
       "  'mw': 31044.0},\n",
       " 'b0572': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77211',\n",
       "  'gene': 'b0572',\n",
       "  'locus': ['b0572', 'JW0561'],\n",
       "  'synonyms': 'ibeB ylcB',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Forms pores that allow passive diffusion of cations across the outer membrane. Part of a cation efflux system that mediates resistance to copper and silver. In pathogenic strains it allows the bacteria to invade brain microvascular endothelial cells (BMEC) thus allowing it to cross the blood-brain barrier and cause neonatal meningitis'},\n",
       "  'seq': 'MSPCKLLPFCVALALTGCSLAPDYQRPAMPVPQQFSLSQNGLVNAADNYQNAGWRTFFVDNQVKTLISEALVNNRDLRMATLKVQEARAQYRLTDADRYPQLNGEGSGSWSGNLKGNTATTREFSTGLNASFDLDFFGRLKNMSEAERQNYLATEEAQRAVHILLVSNVAQSYFNQQLAYAQLQIAEETLRNYQQSYAFVEKQLLTGSSNVLALEQARGVIESTRSDIAKRQGELAQANNALQLLLGSYGKLPQAQTVNSDSLQSVKLPAGLSSQILLQRPDIMEAEHALMAANANIGAARAAFFPSISLTSGISTASSDLSSLFNASSGMWNFIPKIEIPIFNAGRNQANLDIAEIRQQQSVVNYEQKIQNAFKEVADALALRQSLNDQISAQQRYLASLQITLQRARALYQHGAVSYLEVLDAERSLFATRQTLLDLNYARQVNEISLYTALGGG',\n",
       "  'mw': 50270.0},\n",
       " 'b0573': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77214',\n",
       "  'gene': 'b0573',\n",
       "  'locus': ['b0573', 'JW0562'],\n",
       "  'synonyms': 'cusX ylcC',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of a cation efflux system that mediates resistance to copper and silver. Binds one copper per polypeptide (PubMed:11399769, PubMed:24917681)'},\n",
       "  'seq': 'MKKALQVAMFSLFTVIGFNAQANEHHHETMSEAQPQVISATGVVKGIDLESKKITIHHDPIAAVNWPEMTMRFTITPQTKMSEIKTGDKVAFNFVQQGNLSLLQDIKVSQ',\n",
       "  'mw': 12251.0},\n",
       " 'b0574': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77239',\n",
       "  'gene': 'b0574',\n",
       "  'locus': ['b0574', 'JW0563'],\n",
       "  'synonyms': 'ylcD',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of a cation efflux system that mediates resistance to copper and silver'},\n",
       "  'seq': 'MKKIALIIGSMIAGGIISAAGFTWVAKAEPPAEKTSTAERKILFWYDPMYPNTRFDKPGKSPFMDMDLVPKYADEESSASGVRIDPTQTQNLGVKTATVTRGPLTFAQSFPANVSYNEYQYAIVQARAAGFIDKVYPLTVGDKVQKGTPLLDLTIPDWVEAQSEYLLLRETGGTATQTEGILERLRLAGMPEADIRRLIATQKIQTRFTLKAPIDGVITAFDLRAGMNIAKDNVVAKIQGMDPVWVTAAIPESIAWLVKDASQFTLTVPARPDKTLTIRKWTLLPGVDAATRTLQLRLEVDNADEALKPGMNAWLQLNTASEPMLLIPSQALIDTGSEQRVITVDADGRFVPKRVAVFQASQGVTALRSGLAEGEKVVSSGLFLIDSEANISGALERMRSESATHAH',\n",
       "  'mw': 44305.0},\n",
       " 'b0575': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P38054',\n",
       "  'gene': 'b0575',\n",
       "  'locus': ['b0575', 'JW0564'],\n",
       "  'synonyms': 'ybdE',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of a cation efflux system that mediates resistance to copper and silver'},\n",
       "  'seq': 'MIEWIIRRSVANRFLVLMGALFLSIWGTWTIINTPVDALPDLSDVQVIIKTSYPGQAPQIVENQVTYPLTTTMLSVPGAKTVRGFSQFGDSYVYVIFEDGTDPYWARSRVLEYLNQVQGKLPAGVSAELGPDATGVGWIYEYALVDRSGKHDLADLRSLQDWFLKYELKTIPDVAEVASVGGVVKEYQVVIDPQRLAQYGISLAEVKSALDASNQEAGGSSIELAEAEYMVRASGYLQTLDDFNHIVLKASENGVPVYLRDVAKVQIGPEMRRGIAELNGEGEVAGGVVILRSGKNAREVIAAVKDKLETLKSSLPEGVEIVTTYDRSQLIDRAIDNLSGKLLEEFIVVAVVCALFLWHVRSALVAIISLPLGLCIAFIVMHFQGLNANIMSLGGIAIAVGAMVDAAIVMIENAHKRLEEWQHQHPDATLDNKTRWQVITDASVEVGPALFISLLIITLSFIPIFTLEGQEGRLFGPLAFTKTYAMAGAALLAIVVIPILMGYWIRGKIPPESSNPLNRFLIRVYHPLLLKVLHWPKTTLLVAALSVLTVLWPLNKVGGEFLPQINEGDLLYMPSTLPGISAAEAASMLQKTDKLIMSVPEVARVFGKTGKAETATDSAPLEMVETTIQLKPQEQWRPGMTMDKIIEELDNTVRLPGLANLWVPPIRNRIDMLSTGIKSPIGIKVSGTVLADIDAMAEQIEEVARTVPGVASALAERLEGGRYINVEINREKAARYGMTVADVQLFVTSAVGGAMVGETVEGIARYPINLRYPQSWRDSPQALRQLPILTPMKQQITLADVADIKVSTGPSMLKTENARPTSWIYIDARDRDMVSVVHDLQKAIAEKVQLKPGTSVAFSGQFELLERANHKLKLMVPMTLMIIFVLLYLAFRRVGEALLIISSVPFALVGGIWLLWWMGFHLSVATGTGFIALAGVAAEFGVVMLMYLRHAIEAVPSLNNPQTFSEQKLDEALYHGAVLRVRPKAMTVAVIIAGLLPILWGTGAGSEVMSRIAAPMIGGMITAPLLSLFIIPAAYKLMWLHRHRVRK',\n",
       "  'mw': 114707.0},\n",
       " 'b0576': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P24207',\n",
       "  'gene': 'b0576',\n",
       "  'locus': ['b0576', 'JW0565'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('H(+)(in) + L-phenylalanine(in) = H(+)(out) + L-phenylalanine(out)',\n",
       "     '')],\n",
       "   'Function': 'Permease that is involved in the active transport across the cytoplasmic membrane of phenylalanine (PubMed:10735864, PubMed:1711024, PubMed:8226700, PubMed:9791098). Can also transport tyrosine, but not tryptophan (PubMed:10735864)'},\n",
       "  'seq': 'MKNASTVSEDTASNQEPTLHRGLHNRHIQLIALGGAIGTGLFLGIGPAIQMAGPAVLLGYGVAGIIAFLIMRQLGEMVVEEPVSGSFAHFAYKYWGPFAGFLSGWNYWVMFVLVGMAELTAAGIYMQYWFPDVPTWIWAAAFFIIINAVNLVNVRLYGETEFWFALIKVLAIIGMIGFGLWLLFSGHGGEKASIDNLWRYGGFFATGWNGLILSLAVIMFSFGGLELIGITAAEARDPEKSIPKAVNQVVYRILLFYIGSLVVLLALYPWVEVKSNSSPFVMIFHNLDSNVVASALNFVILVASLSVYNSGVYSNSRMLFGLSVQGNAPKFLTRVSRRGVPINSLMLSGAITSLVVLINYLLPQKAFGLLMALVVATLLLNWIMICLAHLRFRAAMRRQGRETQFKALLYPFGNYLCIAFLGMILLLMCTMDDMRLSAILLPVWIVFLFMAFKTLRRK',\n",
       "  'mw': 50677.0},\n",
       " 'b0583': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P19925',\n",
       "  'gene': 'b0583',\n",
       "  'locus': ['b0583', 'JW5085'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [(\"apo-[aryl-carrier protein] + CoA = adenosine 3',5'-bisphosphate + H(+) + holo-[aryl-carrier protein]\",\n",
       "     ''),\n",
       "    (\"apo-[peptidyl-carrier protein] + CoA = adenosine 3',5'-bisphosphate + H(+) + holo-[peptidyl-carrier protein]\",\n",
       "     '')],\n",
       "   'Function': \"Involved in the biosynthesis of the siderophore enterobactin (enterochelin), which is a macrocyclic trimeric lactone of N-(2,3-dihydroxybenzoyl)-serine. The serine trilactone serves as a scaffolding for the three catechol functionalities that provide hexadentate coordination for the tightly ligated iron(2+) atoms. Plays an essential role in the assembly of the enterobactin by catalyzing the transfer of the 4'-phosphopantetheine (Ppant) moiety from coenzyme A to the apo-domains of both EntB (ArCP domain) and EntF (PCP domain) to yield their holo-forms which make them competent for the activation of 2,3-dihydroxybenzoate (DHB) and L-serine, respectively\",\n",
       "   'Pathway': 'Siderophore biosynthesis; enterobactin biosynthesis'},\n",
       "  'seq': 'MKTTHTSLPFAGHTLHFVEFDPANFCEQDLLWLPHYAQLQHAGRKRKTEHLAGRIAAVYALREYGYKCVPAIGELRQPVWPAEVYGSISHCGTTALAVVSRQPIGIDIEEIFSVQTARELTDNIITPAEHERLADCGLAFSLALTLAFSAKESAFKASEIQTDAGFLDYQIISWNKQQVIIHRENEMFAVHWQIKEKIVITLCQHD',\n",
       "  'mw': 23259.0},\n",
       " 'b0584': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P05825',\n",
       "  'gene': 'b0584',\n",
       "  'locus': ['b0584', 'JW5086'],\n",
       "  'synonyms': 'fep feuB',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'This protein is involved in the initial step of iron uptake by binding ferrienterobactin (Fe-ENT), an iron chelatin siderophore that allows E.coli to extract iron from the environment. FepA also acts as a receptor for colicins B and D'},\n",
       "  'seq': 'MNKKIHSLALLVNLGIYGVAQAQEPTDTPVSHDDTIVVTAAEQNLQAPGVSTITADEIRKNPVARDVSKIIRTMPGVNLTGNSTSGQRGNNRQIDIRGMGPENTLILIDGKPVSSRNSVRQGWRGERDTRGDTSWVPPEMIERIEVLRGPAAARYGNGAAGGVVNIITKKGSGEWHGSWDAYFNAPEHKEEGATKRTNFSLTGPLGDEFSFRLYGNLDKTQADAWDINQGHQSARAGTYATTLPAGREGVINKDINGVVRWDFAPLQSLELEAGYSRQGNLYAGDTQNTNSDSYTRSKYGDETNRLYRQNYALTWNGGWDNGVTTSNWVQYEHTRNSRIPEGLAGGTEGKFNEKATQDFVDIDLDDVMLHSEVNLPIDFLVNQTLTLGTEWNQQRMKDLSSNTQALTGTNTGGAIDGVSTTDRSPYSKAEIFSLFAENNMELTDSTIVTPGLRFDHHSIVGNNWSPALNISQGLGDDFTLKMGIARAYKAPSLYQTNPNYILYSKGQGCYASAGGCYLQGNDDLKAETSINKEIGLEFKRDGWLAGVTWFRNDYRNKIEAGYVAVGQNAVGTDLYQWDNVPKAVVEGLEGSLNVPVSETVMWTNNITYMLKSENKTTGDRLSIIPEYTLNSTLSWQAREDLSMQTTFTWYGKQQPKKYNYKGQPAVGPETKEISPYSIVGLSATWDVTKNVSLTGGVDNLFDKRLWRAGNAQTTGDLAGANYIAGAGAYTYNEPGRTWYMSVNTHF',\n",
       "  'mw': 82107.0},\n",
       " 'b0585': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P13039',\n",
       "  'gene': 'b0585',\n",
       "  'locus': ['b0585', 'JW0576'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('Fe(III)-enterobactin + H(+) + 3 H2O = Fe(III)-[N-(2,3-dihydroxybenzoyl)-L-serine] + 2 N-(2,3-dihydroxybenzoyl)-L-serine',\n",
       "     '3.1.1.108'),\n",
       "    ('Fe(III)-enterobactin + H2O = Fe(III)-[N-(2,3-dihydroxybenzoyl)-L-serine]3 + H(+)',\n",
       "     ''),\n",
       "    ('Fe(III)-[N-(2,3-dihydroxybenzoyl)-L-serine]3 + H(+) + H2O = Fe(III)-[N-(2,3-dihydroxybenzoyl)-L-serine]2 + N-(2,3-dihydroxybenzoyl)-L-serine',\n",
       "     ''),\n",
       "    ('Fe(III)-[N-(2,3-dihydroxybenzoyl)-L-serine]2 + H(+) + H2O = Fe(III)-[N-(2,3-dihydroxybenzoyl)-L-serine] + N-(2,3-dihydroxybenzoyl)-L-serine',\n",
       "     '')],\n",
       "   'Function': 'Catalyzes the hydrolysis of ferric enterobactin (Fe-Ent) (PubMed:150859, PubMed:1534808, PubMed:8148617). Is responsible for the release of iron from ferric enterobactin (PubMed:1534808). Also catalyzes the hydrolysis of iron-free enterobactin (Ent) (PubMed:150859, PubMed:1534808, PubMed:8148617). Cleavage of ferric enterobactin results in a mixture of three hydrolysis products, 2,3-dihydroxybenzoylserine (DHBS), the linear dimer (DHBS)2 and the linear trimer (DHBS)3, while cleavage of iron-free enterobactin yields only the monomer (PubMed:8148617). Hydrolysis of ferric enterobactin is less efficient than hydrolysis of unliganded enterobactin (PubMed:150859, PubMed:1534808). It also cleaves the aluminum (III) complex at a rate similar to the ferric complex (PubMed:1534808)',\n",
       "   'Regulation Activity': 'Inhibited by N-ethylmaleimide'},\n",
       "  'seq': 'MTALKVGSESWWQSKHGPEWQRLNDEMFEVTFWWRDPQGSEEYSTIKRVWVYITGVTDHHQNSQPQSMQRIAGTNVWQWTTQLNANWRGSYCFIPTERDDIFSVPSPDRLELREGWRKLLPQAIADPLNLQSWKGGRGHAVSALEMPQAPLQPGWDCPQAPEIPAKEIIWKSERLKKSRRVWIFTTGDATAEERPLAVLLDGEFWAQSMPVWPVLTSLTHRQQLPPAVYVLIDAIDTTHRAHELPCNADFWLAVQQELLPLVKAIAPFSDRADRTVVAGQSFGGLSALYAGLHWPERFGCVLSQSGSYWWPHRGGQQEGVLLEKLKAGEVSAEGLRIVLEAGIREPMIMRANQALYAQLHPIKESIFWRQVDGGHDALCWRGGLMQGLIDLWQPLFHDRS',\n",
       "  'mw': 45652.0},\n",
       " 'b0586': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P11454',\n",
       "  'gene': 'b0586',\n",
       "  'locus': ['b0586', 'JW0578'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('3 2,3-dihydroxybenzoate + 6 ATP + 3 L-serine = 6 AMP + 6 diphosphate + enterobactin + 4 H(+)',\n",
       "     '6.3.2.14'),\n",
       "    ('ATP + holo-[peptidyl-carrier protein] + L-serine = AMP + diphosphate + L-seryl-[peptidyl-carrier protein]',\n",
       "     '6.2.1.72')],\n",
       "   'Function': 'Involved in the biosynthesis of the siderophore enterobactin (enterochelin), which is a macrocyclic trimeric lactone of N-(2,3-dihydroxybenzoyl)-serine (PubMed:216414, PubMed:1826089, PubMed:1338974, PubMed:9485415). EntF catalyzes the activation of L-serine via ATP-dependent PPi exchange reaction to form seryladenylate (PubMed:1826089, PubMed:1338974, PubMed:9485415, PubMed:10688898). Activated L-serine is loaded onto the peptidyl carrier domain via a thioester linkage to the phosphopanthetheine moiety, forming seryl-S-Ppant-EntF (PubMed:9485415). EntF acts then as the sole catalyst for the formation of the three amide and three ester linkages found in enterobactin, using seryladenylate and 2,3-dihydroxybenzoate-S-Ppant-EntB (DHB-S-Ppant-EntB) as substrates, via the formation of a DHB-Ser-S-Ppant-EntF intermediate (PubMed:9485415)',\n",
       "   'Regulation Activity': 'Adenylation activity is increased in the presence of the MbtH-like protein YbdZ',\n",
       "   'Pathway': 'Siderophore biosynthesis; enterobactin biosynthesis'},\n",
       "  'seq': 'MSQHLPLVAAQPGIWMAEKLSELPSAWSVAHYVELTGEVDSPLLARAVVAGLAQADTLRMRFTEDNGEVWQWVDDALTFELPEIIDLRTNIDPHGTAQALMQADLQQDLRVDSGKPLVFHQLIQVADNRWYWYQRYHHLLVDGFSFPAITRQIANIYCTWLRGEPTPASPFTPFADVVEEYQQYRESEAWQRDAAFWAEQRRQLPPPASLSPAPLPGRSASADILRLKLEFTDGEFRQLATQLSGVQRTDLALALAALWLGRLCNRMDYAAGFIFMRRLGSAALTATGPVLNVLPLGIHIAAQETLPELATRLAAQLKKMRRHQRYDAEQIVRDSGRAAGDEPLFGPVLNIKVFDYQLDIPDVQAQTHTLATGPVNDLELALFPDVHGDLSIEILANKQRYDEPTLIQHAERLKMLIAQFAADPALLCGDVDIMLPGEYAQLAQLNATQVEIPETTLSALVAEQAAKTPDAPALADARYLFSYREMREQVVALANLLRERGVKPGDSVAVALPRSVFLTLALHAIVEAGAAWLPLDTGYPDDRLKMMLEDARPSLLITTDDQLPRFSDVPNLTSLCYNAPLTPQGSAPLQLSQPHHTAYIIFTSGSTGRPKGVMVGQTAIVNRLLWMQNHYPLTGEDVVAQKTPCSFDVSVWEFFWPFIAGAKLVMAEPEAHRDPLAMQQFFAEYGVTTTHFVPSMLAAFVASLTPQTARQSCATLKQVFCSGEALPADLCREWQQLTGAPLHNLYGPTEAAVDVSWYPAFGEELAQVRGSSVPIGYPVWNTGLRILDAMMHPVPPGVAGDLYLTGIQLAQGYLGRPDLTASRFIADPFAPGERMYRTGDVARWLDNGAVEYLGRSDDQLKIRGQRIELGEIDRVMQALPDVEQAVTHACVINQAAATGGDARQLVGYLVSQSGLPLDTSALQAQLRETLPPHMVPVVLLQLPQLPLSANGKLDRKALPLPELKAQAPGRAPKAGSETIIAAAFSSLLGCDVQDADADFFALGGHSLLAMKLAAQLSRQVARQVTPGQVMVASTVAKLATIIDAEEDSTRRMGFETILPLREGNGPTLFCFHPASGFAWQFSVLSRYLDPQWSIIGIQSPRPNGPMQTAANLDEVCEAHLATLLEQQPHGPYYLLGYSLGGTLAQGIAARLRARGEQVAFLGLLDTWPPETQNWQEKEANGLDPEVLAEINREREAFLAAQQGSTSTELFTTIEGNYADAVRLLTTAHSVPFDGKATLFVAERTLQEGMSPERAWSPWIAELDIYRQDCAHVDIISPGTFEKIGPIIRATLNR',\n",
       "  'mw': 141991.0},\n",
       " 'b0588': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P23878',\n",
       "  'gene': 'b0588',\n",
       "  'locus': ['b0588', 'JW0580'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + Fe(III)-enterobactin(out) + H2O = ADP + Fe(III)-enterobactin(in) + H(+) + phosphate',\n",
       "     '7.2.2.17')],\n",
       "   'Function': 'Part of the ABC transporter complex FepBDGC involved in ferric enterobactin uptake (PubMed:1838574, PubMed:3011753). Responsible for energy coupling to the transport system (Probable)'},\n",
       "  'seq': 'MTESVARLRGEQLTLGYGKYTVAENLTVEIPDGHFTAIIGPNGCGKSTLLRTLSRLMTPAHGHVWLDGEHIQHYASKEVARRIGLLAQNATTPGDITVQELVARGRYPHQPLFTRWRKEDEEAVTKAMQATGITHLADQSVDTLSGGQRQRAWIAMVLAQETAIMLLDEPTTWLDISHQIDLLELLSELNREKGYTLAAVLHDLNQACRYASHLIALREGKIVAQGAPKEIVTAELIERIYGLRCMIIDDPVAGTPLVVPLGRTAPSTANS',\n",
       "  'mw': 29784.0},\n",
       " 'b0589': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P23877',\n",
       "  'gene': 'b0589',\n",
       "  'locus': ['b0589', 'JW0581'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the ABC transporter complex FepBDGC involved in ferric enterobactin uptake (PubMed:1838574, PubMed:1479347). Responsible for the translocation of the substrate across the membrane (Probable)'},\n",
       "  'seq': 'MIYVSRRLLITCLLLVSACVVAGIWGLRSGAVTLETSQVFAALMGDAPRSMTMVVTEWRLPRVLMALLIGAALGVSGAIFQSLMRNPLGSPDVMGFNTGAWSGVLVAMVLFGQDLTAIALSAMVGGIVTSLLVWLLAWRNGIDTFRLIIIGIGVRAMLVAFNTWLLLKASLETALTAGLWNAGSLNGLTWAKTSPSAPIIILMLIAAALLVRRMRLLEMGDDTACALGVSVERSRLLMMLVAVVLTAAATALAGPISFIALVAPHIARRISGTARWGLTQAALCGALLLLAADLCAQQLFMPYQLPVGVVTVSLGGIYLIVLLIQESRKK',\n",
       "  'mw': 34910.0},\n",
       " 'b0590': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P23876',\n",
       "  'gene': 'b0590',\n",
       "  'locus': ['b0590', 'JW0582'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the ABC transporter complex FepBDGC involved in ferric enterobactin uptake (PubMed:1838574, PubMed:1479347). Responsible for the translocation of the substrate across the membrane (Probable)'},\n",
       "  'seq': 'MSGSVAVTRAIAVPGLLLLLIIATALSLLIGAKSLPASVVLEAFSGTCQSADCTIVLDARLPRTLAGLLAGGALGLAGALMQTLTRNPLADPGLLGVNAGASFAIVLGAALFGYSSAQEQLAMAFAGALVASLIVAFTGSQGGGQLSPVRLTLAGVALAAVLEGLTSGIALLNPDVYDQLRFWQAGSLDIRNLHTLKVVLIPVLIAGATALLLSRALNSLSLGSDTATALGSRVARTQLIGLLAITVLCGSATAIVGPIAFIGLMMPHMARWLVGADHRWSLPVTLLATPALLLFADIIGRVIVPGELRVSVVSAFIGAPVLIFLVRRKTRGGA',\n",
       "  'mw': 33871.0},\n",
       " 'b0591': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P24077',\n",
       "  'gene': 'b0591',\n",
       "  'locus': ['b0591', 'JW0583'],\n",
       "  'synonyms': 'ybdA',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Component of an export pathway for enterobactin (PubMed:12068807). Overexpression reduces intracellular arabinose concentrations (PubMed:22952739)'},\n",
       "  'seq': 'MNKQSWLLNLSLLKTHPAFRAVFLARFISIVSLGLLGVAVPVQIQMMTHSTWQVGLSVTLTGGAMFVGLMVGGVLADRYERKKVILLARGTCGIGFIGLCLNALLPEPSLLAIYLLGLWDGFFASLGVTALLAATPALVGRENLMQAGAITMLTVRLGSVISPMIGGLLLATGGVAWNYGLAAAGTFITLLPLLSLPALPPPPQPREHPLKSLLAGFRFLLASPLVGGIALLGGLLTMASAVRVLYPALADNWQMSAAQIGFLYAAIPLGAAIGALTSGKLAHSARPGLLMLLSTLGSFLAIGLFGLMPMWILGVVCLALFGWLSAVSSLLQYTMLQTQTPEAMLGRINGLWTAQNVTGDAIGAALLGGLGAMMTPVASASASGFGLLIIGVLLLLVLVELRHFRQTPPQVTASDS',\n",
       "  'mw': 43282.0},\n",
       " 'b0592': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AEL6',\n",
       "  'gene': 'b0592',\n",
       "  'locus': ['b0592', 'JW0584'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the ABC transporter complex FepBDGC involved in ferric enterobactin uptake (PubMed:3011753). Binds ferric enterobactin (PubMed:7551033, PubMed:10986237, PubMed:25173704). Can also bind ferric enantioenterobactin, the left-handed stereoisomer of the natural E.coli siderophore, and the apo-siderophore, enterobactin, but does not bind ferric agrobactin (PubMed:10986237)'},\n",
       "  'seq': 'MRLAPLYRNALLLTGLLLSGIAAVQAADWPRQITDSRGTHTLESQPQRIVSTSVTLTGSLLAIDAPVIASGATTPNNRVADDQGFLRQWSKVAKERKLQRLYIGEPSAEAVAAQMPDLILISATGGDSALALYDQLSTIAPTLIINYDDKSWQSLLTQLGEITGHEKQAAERIAQFDKQLAAAKEQIKLPPQPVTAIVYTAAAHSANLWTPESAQGQMLEQLGFTLAKLPAGLNASQSQGKRHDIIQLGGENLAAGLNGESLFLFAGDQKDADAIYANPLLAHLPAVQNKQVYALGTETFRLDYYSAMQVLDRLKALF',\n",
       "  'mw': 34283.0},\n",
       " 'b0593': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AEJ2',\n",
       "  'gene': 'b0593',\n",
       "  'locus': ['b0593', 'JW0585'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('chorismate = isochorismate',\n",
       "     '5.4.4.2')],\n",
       "   'Function': 'Involved in the biosynthesis of the siderophore enterobactin (macrocyclic trimeric lactone of N-(2,3-dihydroxybenzoyl)-serine). Catalyzes the reversible conversion of chorismate to isochorismate',\n",
       "   'Pathway': 'Siderophore biosynthesis; enterobactin biosynthesis'},\n",
       "  'seq': 'MDTSLAEEVQQTMATLAPNRFFFMSPYRSFTTSGCFARFDEPAVNGDSPDSPFQQKLAALFADAKAQGIKNPVMVGAIPFDPRQPSSLYIPESWQSFSRQEKQASARRFTRSQSLNVVERQAIPEQTTFEQMVARAAALTATPQVDKVVLSRLIDITTDAAIDSGVLLERLIAQNPVSYNFHVPLADGGVLLGASPELLLRKDGERFSSIPLAGSARRQPDEVLDREAGNRLLASEKDRHEHELVTQAMKEVLRERSSELHVPSSPQLITTPTLWHLATPFEGKANSQENALTLACLLHPTPALSGFPHQAATQVIAELEPFDRELFGGIVGWCDSEGNGEWVVTIRCAKLRENQVRLFAGAGIVPASSPLGEWRETGVKLSTMLNVFGLH',\n",
       "  'mw': 42932.0},\n",
       " 'b0594': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P10378',\n",
       "  'gene': 'b0594',\n",
       "  'locus': ['b0594', 'JW0586'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('3 2,3-dihydroxybenzoate + 6 ATP + 3 L-serine = 6 AMP + 6 diphosphate + enterobactin + 4 H(+)',\n",
       "     '6.3.2.14'),\n",
       "    ('2,3-dihydroxybenzoate + ATP + holo-[aryl-carrier protein] = 2,3-dihydroxybenzoyl-[aryl-carrier protein] + AMP + diphosphate',\n",
       "     '6.2.1.71'),\n",
       "    (\"2,3-dihydroxybenzoyl-5'-AMP + holo-[aryl-carrier protein] = 2,3-dihydroxybenzoyl-[aryl-carrier protein] + AMP + H(+)\",\n",
       "     '')],\n",
       "   'Function': 'Involved in the biosynthesis of the siderophore enterobactin (enterochelin), which is a macrocyclic trimeric lactone of N-(2,3-dihydroxybenzoyl)-serine. The serine trilactone serves as a scaffolding for the three catechol functionalities that provide hexadentate coordination for the tightly ligated iron(2+) atoms. EntE proccesses via a two-step adenylation-ligation reaction (bi-uni-uni-bi ping-pong mechanism). First, it catalyzes the activation of the carboxylate group of 2,3-dihydroxy-benzoate (DHB), via a reversible ATP-dependent pyrophosphate exchange reactions to yield the acyladenylate intermediate 2,3-dihydroxybenzoyl-AMP. It can also transfer AMP to salicylate, 2,4-dihydroxybenzoate, gentisate and 2,3,4-trihydroxybenzoate. In the second step, DHB is transferred from 2,3-dihydroxybenzoyl-AMP onto the phosphopantetheinylated EntB (holo-EntB) to form DHB-holo-EntB. Then this product will serve in the formation of the amide bond between 2,3-dihydroxybenzoate (DHB) and L-serine. It can also transfer adenylated salicylate to holo-EntB',\n",
       "   'Regulation Activity': \"Inhibited by the adenylate analogs, 5'-O-[N-(salicyl)sulfamoyl]adenosine (Sal-AMS) and 5'-O-[N-(2,3-dihydroxybenzoyl)sulfamoyl]adenosine (DHB-AMS). Adenylation of 2,3-dihydroxybenzoate (DHB) is enhanced by a protein-protein interaction between the EntA and EntE\",\n",
       "   'Pathway': 'Siderophore biosynthesis; enterobactin biosynthesis'},\n",
       "  'seq': 'MSIPFTRWPEEFARRYREKGYWQDLPLTDILTRHAASDSIAVIDGERQLSYRELNQAADNLACSLRRQGIKPGETALVQLGNVAELYITFFALLKLGVAPVLALFSHQRSELNAYASQIEPALLIADRQHALFSGDDFLNTFVTEHSSIRVVQLLNDSGEHNLQDAINHPAEDFTATPSPADEVAYFQLSGGTTGTPKLIPRTHNDYYYSVRRSVEICQFTQQTRYLCAIPAAHNYAMSSPGSLGVFLAGGTVVLAADPSATLCFPLIEKHQVNVTALVPPAVSLWLQALIEGESRAQLASLKLLQVGGARLSATLAARIPAEIGCQLQQVFGMAEGLVNYTRLDDSAEKIIHTQGYPMCPDDEVWVADAEGNPLPQGEVGRLMTRGPYTFRGYYKSPQHNASAFDANGFYCSGDLISIDPEGYITVQGREKDQINRGGEKIAAEEIENLLLRHPAVIYAALVSMEDELMGEKSCAYLVVKEPLRAVQVRRFLREQGIAEFKLPDRVECVDSLPLTAVGKVDKKQLRQWLASRASA',\n",
       "  'mw': 59112.0},\n",
       " 'b0595': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ADI4',\n",
       "  'gene': 'b0595',\n",
       "  'locus': ['b0595', 'JW0587'],\n",
       "  'synonyms': 'entG',\n",
       "  'properties': {'Catalytic Activity': [('3 2,3-dihydroxybenzoate + 6 ATP + 3 L-serine = 6 AMP + 6 diphosphate + enterobactin + 4 H(+)',\n",
       "     '6.3.2.14'),\n",
       "    ('H2O + isochorismate = (2S,3S)-2,3-dihydroxy-2,3-dihydrobenzoate + pyruvate',\n",
       "     '3.3.2.1')],\n",
       "   'Function': 'Involved in the biosynthesis of the siderophore enterobactin (enterochelin), which is a macrocyclic trimeric lactone of N-(2,3-dihydroxybenzoyl)-serine. The serine trilactone serves as a scaffolding for the three catechol functionalities that provide hexadentate coordination for the tightly ligated iron(2+) atoms. EntB is a bifunctional protein that serves as an isochorismate lyase and an aryl carrier protein (ArCP). Catalyzes the conversion of isochorismate to 2,3-dihydro-2,3-dihydroxybenzoate (2,3-diDHB), the precursor of 2,3-dihydroxybenzoate (DHB). In the enterobactin assembly, EntB functions as an aryl carrier protein phosphopantetheinylated near the C terminus by EntD to yield holo-EntB, which is then acylated by EntE with 2,3-dihydroxybenzoyl-AMP to form DHB-holo-EntB. Then this product will serve in the formation of the amide bond between 2,3-dihydroxybenzoate (DHB) and L-serine',\n",
       "   'Regulation Activity': 'Inhibited by 3-[(carboxyethenyl)oxy]-6-hydroxy-1-benzoic acid and 3-[(carboxyethenyl)oxy]benzoic acid',\n",
       "   'Pathway': 'Siderophore biosynthesis; enterobactin biosynthesis'},\n",
       "  'seq': 'MAIPKLQAYALPESHDIPQNKVDWAFEPQRAALLIHDMQDYFVSFWGENCPMMEQVIANIAALRDYCKQHNIPVYYTAQPKEQSDEDRALLNDMWGPGLTRSPEQQKVVDRLTPDADDTVLVKWRYSAFHRSPLEQMLKESGRNQLIITGVYAHIGCMTTATDAFMRDIKPFMVADALADFSRDEHLMSLKYVAGRSGRVVMTEELLPAPIPASKAALREVILPLLDESDEPFDDDNLIDYGLDSVRMMALAARWRKVHGDIDFVMLAKNPTIDAWWKLLSREVK',\n",
       "  'mw': 32554.0},\n",
       " 'b0596': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P15047',\n",
       "  'gene': 'b0596',\n",
       "  'locus': ['b0596', 'JW0588'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(2S,3S)-2,3-dihydroxy-2,3-dihydrobenzoate + NAD(+) = 2,3-dihydroxybenzoate + H(+) + NADH',\n",
       "     '1.3.1.28')],\n",
       "   'Function': 'Involved in the biosynthesis of the siderophore enterobactin (enterochelin), which is a macrocyclic trimeric lactone of N-(2,3-dihydroxybenzoyl)-serine. Catalyzes the reversible NAD-dependent oxidation of the C3-hydroxyl group of 2,3-dihydro-2,3-dihydroxybenzoate (2,3-diDHB), producing the transient intermediate 2-hydroxy-3-oxo-4,6-cyclohexadiene-1-carboxylate, which undergoes rapid aromatization to the final product, 2,3-dihydroxybenzoate (2,3-DHB). Only the compounds with a C3-hydroxyl group such as methyl 2,3-dihydro-2,3-dihydroxybenzoate, methyl-3-hydroxy-1,4-cyclohexadiene-1-carboxylate, trans-3-hydroxy-2-cyclohexene-1-carboxylate, cis-3-hydroxy-4-cyclohexene-1-carboxylate, cis-3-hydroxycyclohexane-1-carboxylic acid are oxidized to the corresponding ketone products. The stereospecificity of the C3 allylic alcohol group oxidation is 3R in a 1R,3R dihydro substrate. It can also increase the DHB-AMP ligase activity of EntE by interaction EntE',\n",
       "   'Regulation Activity': 'Inhibited by cis-2-hydroxy-3-cyclohexen-1-carboxylate, cis-2-hydroxycyclohexane-1-carboxylate and trans-2-hydroxycyclohexane-1-carboxylate',\n",
       "   'Pathway': 'Siderophore biosynthesis; enterobactin biosynthesis'},\n",
       "  'seq': 'MDFSGKNVWVTGAGKGIGYATALAFVEAGAKVTGFDQAFTQEQYPFATEVMDVADAAQVAQVCQRLLAETERLDALVNAAGILRMGATDQLSKEDWQQTFAVNVGGAFNLFQQTMNQFRRQRGGAIVTVASDAAHTPRIGMSAYGASKAALKSLALSVGLELAGSGVRCNVVSPGSTDTDMQRTLWVSDDAEEQRIRGFGEQFKLGIPLGKIARPQEIANTILFLASDLASHITLQDIVVDGGSTLGA',\n",
       "  'mw': 26250.0},\n",
       " 'b0604': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77202',\n",
       "  'gene': 'b0604',\n",
       "  'locus': ['b0604', 'JW0597'],\n",
       "  'synonyms': 'ybdP',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Involved in disulfide bond formation. DsbG and DsbC are part of a periplasmic reducing system that controls the level of cysteine sulfenylation, and provides reducing equivalents to rescue oxidatively damaged secreted proteins such as ErfK, YbiS and YnhG. Probably also functions as a disulfide isomerase with a narrower substrate specificity than DsbC. DsbG is maintained in a reduced state by DsbD. Displays chaperone activity in both redox states in vitro'},\n",
       "  'seq': 'MLKKILLLALLPAIAFAEELPAPVKAIEKQGITIIKTFDAPGGMKGYLGKYQDMGVTIYLTPDGKHAISGYMYNEKGENLSNTLIEKEIYAPAGREMWQRMEQSHWLLDGKKDAPVIVYVFADPFCPYCKQFWQQARPWVDSGKVQLRTLLVGVIKPESPATAAAILASKDPAKTWQQYEASGGKLKLNVPANVSTEQMKVLSDNEKLMDDLGANVTPAIYYMSKENTLQQAVGLPDQKTLNIIMGNK',\n",
       "  'mw': 27495.0},\n",
       " 'b0612': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AE74',\n",
       "  'gene': 'b0612',\n",
       "  'locus': ['b0612', 'JW0604'],\n",
       "  'synonyms': 'ybdS',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Responsible for the uptake of citrate in exchange with the efflux of succinate, fumarate or tartrate. Has a relatively broad specificity for C(4)-dicarboxylates and tricarboxylates (PubMed:9696764)'},\n",
       "  'seq': 'MSLAKDNIWKLLAPLVVMGVMFLIPVPDGMPPQAWHYFAVFVAMIVGMILEPIPATAISFIAVTICVIGSNYLLFDAKELADPAFNAQKQALKWGLAGFSSTTVWLVFGAFIFALGYEVSGLGRRIALFLVKFMGKRTLTLGYAIVIIDILLAPFTPSNTARTGGTVFPVIKNLPPLFKSFPNDPSARRIGGYLMWMMVISTSLSSSMFVTGAAPNVLGLEFVSKIAGIQISWLQWFLCFLPVGVILLIIAPWLSYVLYKPEITHSEEVATWAGDELKTMGALTRREWTLIGLVLLSLGLWVFGSEVINATAVGLLAVSLMLALHVVPWKDITRYNSAWNTLVNLATLVVMANGLTRSGFIDWFAGTMSTHLEGFSPNATVIVLVLVFYFAHYLFASLSAHTATMLPVILAVGKGIPGVPMEQLCILLVLSIGIMGCLTPYATGPGVIIYGCGYVKSKDYWRLGAIFGVIYISMLLLVGWPILAMWN',\n",
       "  'mw': 53093.0},\n",
       " 'b0613': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77231',\n",
       "  'gene': 'b0613',\n",
       "  'locus': ['b0613', 'JW0605'],\n",
       "  'synonyms': 'ybdT',\n",
       "  'properties': {'Catalytic Activity': [(\"3'-dephospho-CoA + ATP = 2'-(5''-triphospho-alpha-D-ribosyl)-3'-dephospho-CoA + adenine\",\n",
       "     '2.4.2.52')],\n",
       "   'Function': \"Catalyzes the formation of 2-(5''-triphosphoribosyl)-3'-dephosphocoenzyme-A, the precursor of the prosthetic group of the holo-acyl carrier protein (gamma chain) of citrate lyase, from ATP and dephospho-CoA\"},\n",
       "  'seq': 'MSMPATSTKTTKLATSLIDEYALLGWRAMLTEVNLSPKPGLVDRINCGAHKDMALEDFHRSALAIQGWLPRFIEFGACSAEMAPEAVLHGLRPIGMACEGDMFRATAGVNTHKGSIFSLGLLCAAIGRLLQLNQPVTPTTVCSTAASFCRGLTDRELRTNNSQLTAGQRLYQQLGLTGARGEAEAGYPLVINHALPHYLTLLDQGLDPELALLDTLLLLMAINGDTNVASRGGEGGLRWLQREAQTLLQKGGIRTPADLDYLRQFDRECIERNLSPGGSADLLILTWFLAQI',\n",
       "  'mw': 31644.0},\n",
       " 'b0614': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A6G5',\n",
       "  'gene': 'b0614',\n",
       "  'locus': ['b0614', 'JW0606'],\n",
       "  'synonyms': 'ybdU',\n",
       "  'properties': {'Catalytic Activity': [(\"2'-(5''-triphospho-alpha-D-ribosyl)-3'-dephospho-CoA + apo-[citrate lyase ACP] = diphosphate + holo-[citrate lyase ACP]\",\n",
       "     '2.7.7.61')],\n",
       "   'Function': \"Transfers 2-(5''-triphosphoribosyl)-3'-dephosphocoenzyme-A on a serine residue to the apo-acyl carrier protein (gamma chain) of the citrate lyase to yield holo-acyl carrier protein\"},\n",
       "  'seq': 'MHLLPELASHHAVSIPELLVSRDERQARQHVWLKRHPVPLVSFTVVAPGPIKDSEVTRRIFNHGVTALRALAAKQGWQIQEQAALVSASGPEGMLSIAAPARDLKLATIELEHSHPLGRLWDIDVLTPEGEILSRRDYSLPPRRCLLCEQSAAVCARGKTHQLTDLLNRMEALLNDVDACNVN',\n",
       "  'mw': 20270.0},\n",
       " 'b0615': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P75726',\n",
       "  'gene': 'b0615',\n",
       "  'locus': ['b0615', 'JW5087'],\n",
       "  'synonyms': 'ybdV',\n",
       "  'properties': {'Catalytic Activity': [('citrate = acetate + oxaloacetate',\n",
       "     '4.1.3.6'),\n",
       "    ('acetyl-CoA + citrate = (3S)-citryl-CoA + acetate', '2.8.3.10')],\n",
       "   'Function': 'Represents a citrate:acetyl-ACP transferase'},\n",
       "  'seq': 'MTQKIEQSQRQERVAAWNRRAECDLAAFQNSPKQTYQAEKARDRKLCANLEEAIRRSGLQDGMTVSFHHAFRGGDLTVNMVMDVIAKMGFKNLTLASSSLSDCHAPLVEHIRQGVVTRIYTSGLRGPLAEEISRGLLAEPVQIHSHGGRVHLVQSGELNIDVAFLGVPSCDEFGNANGYTGKACCGSLGYAIVDADNAKQVVMLTEELLPYPHNPASIEQDQVDLIVKVDRVGDAAKIGAGATRMTTNPRELLIARSAADVIVNSGYFKEGFSMQTGTGGASLAVTRFLEDKMRSRDIRADFALGGITATMVDLHEKGLIRKLLDVQSFDSHAAQSLARNPNHIEISANQYANWGSKGASVDRLDVVVLSALEIDTQFNVNVLTGSDGVLRGASGGHCDTAIASALSIIVAPLVRGRIPTLVDNVLTCITPGSSVDILVTDHGIAVNPARPELAERLQEAGIKVVSIEWLRERARLLTGEPQPIEFTDRVVAVVRYRDGSVIDVVHQVKE',\n",
       "  'mw': 55173.0},\n",
       " 'b0616': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A9I1',\n",
       "  'gene': 'b0616',\n",
       "  'locus': ['b0616', 'JW0608'],\n",
       "  'synonyms': 'ybdW',\n",
       "  'properties': {'Catalytic Activity': [('citrate = acetate + oxaloacetate',\n",
       "     '4.1.3.6'),\n",
       "    ('(3S)-citryl-CoA = acetyl-CoA + oxaloacetate', '4.1.3.34')],\n",
       "   'Function': 'Represents a citryl-ACP lyase'},\n",
       "  'seq': 'MISASLQQRKTRTRRSMLFVPGANAAMVSNSFIYPADALMFDLEDSVALREKDTARRMVYHALQHPLYRDIETIVRVNALDSEWGVNDLEAVVRGGADVVRLPKTDTAQDVLDIEKEILRIEKACGREPGSTGLLAAIESPLGITRAVEIAHASERLIGIALGAEDYVRNLRTERSPEGTELLFARCSILQAARSAGIQAFDTVYSDANNEAGFLQEAAHIKQLGFDGKSLINPRQIDLLHNLYAPTQKEVDHARRVVEAAEAAAREGLGVVSLNGKMVDGPVIDRARLVLSRAELSGIREE',\n",
       "  'mw': 33110.0},\n",
       " 'b0617': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P69330',\n",
       "  'gene': 'b0617',\n",
       "  'locus': ['b0617', 'JW0609'],\n",
       "  'synonyms': 'ybdX',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Covalent carrier of the coenzyme of citrate lyase'},\n",
       "  'seq': 'MKINQPAVAGTLESGDVMIRIAPLDTQDIDLQINSSVEKQFGDAIRTTILDVLARYNVRGVQLNVDDKGALDCILRARLEALLARASGIPALPWEDCQ',\n",
       "  'mw': 10689.0},\n",
       " 'b0621': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ABP3',\n",
       "  'gene': 'b0621',\n",
       "  'locus': ['b0621', 'JW0613/JW0616'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('fumarate(in) + succinate(out) = fumarate(out) + succinate(in)',\n",
       "     ''),\n",
       "    ('fumarate(out) + 3 H(+)(out) = fumarate(in) + 3 H(+)(in)', '')],\n",
       "   'Function': 'Responsible for the transport of C4-dicarboxylates during anaerobic growth (PubMed:8955408, PubMed:10368146). Catalyzes the uptake of fumarate coupled to the export of succinate (PubMed:8955408). Can also catalyze the uptake of fumarate and the efflux of succinate, without exchange (PubMed:8955408, PubMed:10368146). Shows low rates of transport, which are sufficient for succinate export during fermentation but not for fumarate-succinate exchange in fumarate respiration, indicating that it may function in vivo as the succinate efflux carrier for glucose fermentation, even if it is also able to operate as a fumarate-succinate antiporter (PubMed:10368146)',\n",
       "   'Regulation Activity': 'Active under anaerobic conditions'},\n",
       "  'seq': 'MLTFIELLIGVVVIVGVARYIIKGYSATGVLFVGGLLLLIISAIMGHKVLPSSQASTGYSATDIVEYVKILLMSRGGDLGMMIMMLCGFAAYMTHIGANDMVVKLASKPLQYINSPYLLMIAAYFVACLMSLAVSSATGLGVLLMATLFPVMVNVGISRGAAAAICASPAAIILAPTSGDVVLAAQASEMSLIDFAFKTTLPISIAAIIGMAIAHFFWQRYLDKKEHISHEMLDVSEITTTAPAFYAILPFTPIIGVLIFDGKWGPQLHIITILVICMLIASILEFLRSFNTQKVFSGLEVAYRGMADAFANVVMLLVAAGVFAQGLSTIGFIQSLISIATSFGSASIILMLVLVILTMLAAVTTGSGNAPFYAFVEMIPKLAHSSGINPAYLTIPMLQASNLGRTLSPVSGVVVAVAGMAKISPFEVVKRTSVPVLVGLVIVIVATELMVPGTAAAVTGK',\n",
       "  'mw': 48412.0},\n",
       " 'b0622': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37001',\n",
       "  'gene': 'b0622',\n",
       "  'locus': ['b0622', 'JW0617'],\n",
       "  'synonyms': 'crcA ybeG',\n",
       "  'properties': {'Catalytic Activity': [('1-hexadecanoyl-2-acyl-sn-glycero-3-phosphocholine + lipid A (E. coli) = a 2-acyl-sn-glycero-3-phosphocholine + hepta-acyl lipid A',\n",
       "     '2.3.1.251'),\n",
       "    ('1-hexadecanoyl-2-acyl-sn-glycero-3-phosphocholine + lipid IIA = a 2-acyl-sn-glycero-3-phosphocholine + lipid IIB',\n",
       "     '2.3.1.251'),\n",
       "    ('1-hexadecanoyl-2-acyl-sn-glycero-3-phosphocholine + lipid IVA (E. coli) = a 2-acyl-sn-glycero-3-phosphocholine + lipid IVB (E. coli)',\n",
       "     '2.3.1.251')],\n",
       "   'Function': 'Transfers a palmitate residue from the sn-1 position of a phospholipid to the N-linked hydroxymyristate on the proximal unit of lipid A or its precursors. Phosphatidylglycerol (PtdGro), phosphatidylethanolamine (PtdEtn), phosphatidylserine (PtdSer) and phosphatidic acid (Ptd-OH) are all effective acyl donors',\n",
       "   'Regulation Activity': 'Inhibited by lauryldimethylamine oxide (LDAO) and dodecylphosphocholine (DPC)'},\n",
       "  'seq': 'MNVSKYVAIFSFVFIQLISVGKVFANADEWMTTFRENIAQTWQQPEHYDLYIPAITWHARFAYDKEKTDRYNERPWGGGFGLSRWDEKGNWHGLYAMAFKDSWNKWEPIAGYGWESTWRPLADENFHLGLGFTAGVTARDNWNYIPLPVLLPLASVGYGPVTFQMTYIPGTYNNGNVYFAWMRFQF',\n",
       "  'mw': 21770.0},\n",
       " 'b0632': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AEB2',\n",
       "  'gene': 'b0632',\n",
       "  'locus': ['b0632', 'JW0627'],\n",
       "  'synonyms': 'pfv',\n",
       "  'properties': {'Catalytic Activity': [('Preferential cleavage: (Ac)2-L-Lys-D-Ala-|-D-Ala. Also transpeptidation of peptidyl-alanyl moieties that are N-acyl substituents of D-alanine.',\n",
       "     '3.4.16.4'),\n",
       "    ('a beta-lactam + H2O = a substituted beta-amino acid', '3.5.2.6')],\n",
       "   'Function': 'Removes C-terminal D-alanyl residues from sugar-peptide cell wall precursors',\n",
       "   'Pathway': 'Cell wall biogenesis; peptidoglycan biosynthesis'},\n",
       "  'seq': 'MNTIFSARIMKRLALTTALCTAFISAAHADDLNIKTMIPGVPQIDAESYILIDYNSGKVLAEQNADVRRDPASLTKMMTSYVIGQAMKAGKFKETDLVTIGNDAWATGNPVFKGSSLMFLKPGMQVPVSQLIRGINLQSGNDACVAMADFAAGSQDAFVGLMNSYVNALGLKNTHFQTVHGLDADGQYSSARDMALIGQALIRDVPNEYSIYKEKEFTFNGIRQLNRNGLLWDNSLNVDGIKTGHTDKAGYNLVASATEGQMRLISAVMGGRTFKGREAESKKLLTWGFRFFETVNPLKVGKEFASEPVWFGDSDRASLGVDKDVYLTIPRGRMKDLKASYVLNSSELHAPLQKNQVVGTINFQLDGKTIEQRPLVVLQEIPEGNFFGKIIDYIKLMFHHWFG',\n",
       "  'mw': 44444.0},\n",
       " 'b0635': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AD65',\n",
       "  'gene': 'b0635',\n",
       "  'locus': ['b0635', 'JW0630'],\n",
       "  'synonyms': 'pbpA',\n",
       "  'properties': {'Catalytic Activity': [('Preferential cleavage: (Ac)2-L-Lys-D-Ala-|-D-Ala. Also transpeptidation of peptidyl-alanyl moieties that are N-acyl substituents of D-alanine.',\n",
       "     '3.4.16.4')],\n",
       "   'Function': 'Catalyzes cross-linking of the peptidoglycan cell wall (PubMed:3009484). Responsible for the determination of the rod shape of the cell (PubMed:1103132). Is probably required for lateral peptidoglycan synthesis and maintenance of the correct diameter during lateral and centripetal growth (PubMed:12519203)',\n",
       "   'Regulation Activity': 'Inhibited by mecillinam and benzylpenicillin',\n",
       "   'Pathway': 'Cell wall biogenesis; peptidoglycan biosynthesis'},\n",
       "  'seq': 'MKLQNSFRDYTAESALFVRRALVAFLGILLLTGVLIANLYNLQIVRFTDYQTRSNENRIKLVPIAPSRGIIYDRNGIPLALNRTIYQIEMMPEKVDNVQQTLDALRSVVDLTDDDIAAFRKERARSHRFTSIPVKTNLTEVQVARFAVNQYRFPGVEVKGYKRRYYPYGSALTHVIGYVSKINDKDVERLNNDGKLANYAATHDIGKLGIERYYEDVLHGQTGYEEVEVNNRGRVIRQLKEVPPQAGHDIYLTLDLKLQQYIETLLAGSRAAVVVTDPRTGGVLALVSTPSYDPNLFVDGISSKDYSALLNDPNTPLVNRATQGVYPPASTVKPYVAVSALSAGVITRNTTLFDPGWWQLPGSEKRYRDWKKWGHGRLNVTRSLEESADTFFYQVAYDMGIDRLSEWMGKFGYGHYTGIDLAEERSGNMPTREWKQKRFKKPWYQGDTIPVGIGQGYWTATPIQMSKALMILINDGIVKVPHLLMSTAEDGKQVPWVQPHEPPVGDIHSGYWELAKDGMYGVANRPNGTAHKYFASAPYKIAAKSGTAQVFGLKANETYNAHKIAERLRDHKLMTAFAPYNNPQVAVAMILENGGAGPAVGTLMRQILDHIMLGDNNTDLPAENPAVAAAEDH',\n",
       "  'mw': 70857.0},\n",
       " 'b0638': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P52086',\n",
       "  'gene': 'b0638',\n",
       "  'locus': ['b0638', 'JW0633'],\n",
       "  'synonyms': 'phpB',\n",
       "  'properties': {'Catalytic Activity': [(\"adenosylcob(III)alamin 5'-phosphate + H2O = adenosylcob(III)alamin + phosphate\",\n",
       "     '3.1.3.73'),\n",
       "    (\"alpha-ribazole 5'-phosphate + H2O = alpha-ribazole + phosphate\",\n",
       "     '3.1.3.73')],\n",
       "   'Function': \"Catalyzes the conversion of adenosylcobalamin 5'-phosphate to adenosylcobalamin (vitamin B12); involved in the assembly of the nucleotide loop of cobalamin. Also catalyzes the hydrolysis of the phospho group from alpha-ribazole 5'-phosphate to form alpha-ribazole\",\n",
       "   'Pathway': 'Nucleoside biosynthesis; alpha-ribazole biosynthesis; alpha-ribazole from 5,6-dimethylbenzimidazole: step 2/2'},\n",
       "  'seq': 'MRLWLIRHGETQANIDGLYSGHAPTPLTARGIEQAQNLHTLLHGVSFDLVLCSELERAQHTARLVLSDRQLPVQIIPELNEMFFGDWEMRHHRDLMQEDAENYSAWCNDWQHAIPTNGEGFQAFSQRVERFIARLSEFQHYQNILVVSHQGVLSLLIARLIGMPAEAMWHFRVDQGCWSAIDINQKFATLRVLNSRAIGVENA',\n",
       "  'mw': 23308.0},\n",
       " 'b0639': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A752',\n",
       "  'gene': 'b0639',\n",
       "  'locus': ['b0639', 'JW0634'],\n",
       "  'synonyms': 'ybeN',\n",
       "  'properties': {'Catalytic Activity': [('ATP + H(+) + nicotinate beta-D-ribonucleotide = deamido-NAD(+) + diphosphate',\n",
       "     '2.7.7.18')],\n",
       "   'Function': 'Catalyzes the reversible adenylation of nicotinate mononucleotide (NaMN) to nicotinic acid adenine dinucleotide (NaAD)',\n",
       "   'Regulation Activity': 'Activity is susceptible to product inhibition',\n",
       "   'Pathway': 'Cofactor biosynthesis; NAD(+) biosynthesis; deamido-NAD(+) from nicotinate D-ribonucleotide: step 1/1'},\n",
       "  'seq': 'MKSLQALFGGTFDPVHYGHLKPVETLANLIGLTRVTIIPNNVPPHRPQPEANSVQRKHMLELAIADKPLFTLDERELKRNAPSYTAQTLKEWRQEQGPDVPLAFIIGQDSLLTFPTWYEYETILDNAHLIVCRRPGYPLEMAQPQYQQWLEDHLTHNPEDLHLQPAGKIYLAETPWFNISATIIRERLQNGESCEDLLPEPVLTYINQQGLYR',\n",
       "  'mw': 24528.0},\n",
       " 'b0642': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P07813',\n",
       "  'gene': 'b0642',\n",
       "  'locus': ['b0642', 'JW0637'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + L-leucine + tRNA(Leu) = AMP + diphosphate + L-leucyl-tRNA(Leu)',\n",
       "     '6.1.1.4')]},\n",
       "  'seq': 'MQEQYRPEEIESKVQLHWDEKRTFEVTEDESKEKYYCLSMLPYPSGRLHMGHVRNYTIGDVIARYQRMLGKNVLQPIGWDAFGLPAEGAAVKNNTAPAPWTYDNIAYMKNQLKMLGFGYDWSRELATCTPEYYRWEQKFFTELYKKGLVYKKTSAVNWCPNDQTVLANEQVIDGCCWRCDTKVERKEIPQWFIKITAYADELLNDLDKLDHWPDTVKTMQRNWIGRSEGVEITFNVNDYDNTLTVYTTRPDTFMGCTYLAVAAGHPLAQKAAENNPELAAFIDECRNTKVAEAEMATMEKKGVDTGFKAVHPLTGEEIPVWAANFVLMEYGTGAVMAVPGHDQRDYEFASKYGLNIKPVILAADGSEPDLSQQALTEKGVLFNSGEFNGLDHEAAFNAIADKLTAMGVGERKVNYRLRDWGVSRQRYWGAPIPMVTLEDGTVMPTPDDQLPVILPEDVVMDGITSPIKADPEWAKTTVNGMPALRETDTFDTFMESSWYYARYTCPQYKEGMLDSEAANYWLPVDIYIGGIEHAIMHLLYFRFFHKLMRDAGMVNSDEPAKQLLCQGMVLADAFYYVGENGERNWVSPVDAIVERDEKGRIVKAKDAAGHELVYTGMSKMSKSKNNGIDPQVMVERYGADTVRLFMMFASPADMTLEWQESGVEGANRFLKRVWKLVYEHTAKGDVAALNVDALTENQKALRRDVHKTIAKVTDDIGRRQTFNTAIAAIMELMNKLAKAPTDGEQDRALMQEALLAVVRMLNPFTPHICFTLWQELKGEGDIDNAPWPVADEKAMVEDSTLVVVQVNGKVRAKITVPVDATEEQVRERAGQEHLVAKYLDGVTVRKVIYVPGKLLNLVVG',\n",
       "  'mw': 97234.0},\n",
       " 'b0650': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77319',\n",
       "  'gene': 'b0650',\n",
       "  'locus': ['b0650', 'JW0645'],\n",
       "  'synonyms': 'ybeW',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Probable chaperone. Has ATPase activity. Not stimulated by DnaJ'},\n",
       "  'seq': 'MDNAELAIGIDLGTTNSLIAVWKDGAAQLIPNKFGEYLTPSIISMDENNHILVGKPAVSRRTSHPDKTAALFKRAMGSNTNWRLGSDTFNAPELSSLVLRSLKEDAEEFLQRPIKDVVISVPAYFSDEQRKHTRLAAELAGLNAVRLINEPTAAAMAYGLHTQQNTRSLVFDLGGGTFDVTVLEYATPVIEVHASAGDNFLGGEDFTHMLVDEVLKRADVARTTLNESELAALYACVEAAKCSNQSPLHIRWQYQEETRECEFYENELEDLWLPLLNRLRVPIEQALRDARLKPSQIDSLVLVGGASQMPLVQRIAVRLFGKLPYQSYDPSTIVALGAAIQAACRLRSEDIEEVILTDICPYSLGVEVNRQGVSGIFSPIIERNTTVPVSRVETYSTMHPEQDSITVNVYQGENHKVKNNILVESFDVPLKKTGAYQSIDIRFSYDINGLLEVDVLLEDGSVKSRVINHSPVTLSAQQIEESRTRLSALKIYPRDMLINRTFKAKLEELWARALGDEREEIGRVITDFDAALQSNDMARVDEVRRRASDYLAIEIP',\n",
       "  'mw': 61986.0},\n",
       " 'b0651': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P41409',\n",
       "  'gene': 'b0651',\n",
       "  'locus': ['b0651', 'JW0646'],\n",
       "  'synonyms': 'ybeK',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Hydrolyzes with equal efficiency cytidine or uridine to ribose and cytosine or uracil, respectively',\n",
       "   'Regulation Activity': 'Subject to catabolite repression'},\n",
       "  'seq': 'MALPILLDCDPGHDDAIAIVLALASPELDVKAITSSAGNQTPEKTLRNVLRMLTLLNRTDIPVAGGAVKPLMRELIIADNVHGESGLDGPALPEPTFAPQNCTAVELMAKTLRESAEPVTIVSTGPQTNVALLLNSHPELHSKIARIVIMGGAMGLGNWTPAAEFNIYVDPEAAEIVFQSGIPVVMAGLDVTHKAQIHVEDTERFRAIGNPVSTIVAELLDFFLEYHKDEKWGFVGAPLHDPCTIAWLLKPELFTSVERWVGVETQGKYTQGMTVVDYYYLTGNKPNATVMVDVDRQGFVDLLADRLKFYA',\n",
       "  'mw': 33823.0},\n",
       " 'b0652': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AAG3',\n",
       "  'gene': 'b0652',\n",
       "  'locus': ['b0652', 'JW0647'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('a polar amino acid(out) + ATP + H2O = a polar amino acid(in) + ADP + H(+) + phosphate',\n",
       "     '7.4.2.1'),\n",
       "    ('ATP + H2O + L-glutamate(out) = ADP + H(+) + L-glutamate(in) + phosphate',\n",
       "     ''),\n",
       "    ('ATP + H2O + L-aspartate(out) = ADP + H(+) + L-aspartate(in) + phosphate',\n",
       "     '')],\n",
       "   'Function': 'Part of the ABC transporter complex GltIJKL involved in glutamate and aspartate uptake. Probably responsible for energy coupling to the transport system'},\n",
       "  'seq': 'MITLKNVSKWYGHFQVLTDCSTEVKKGEVVVVCGPSGSGKSTLIKTVNGLEPVQQGEITVDGIVVNDKKTDLAKLRSRVGMVFQHFELFPHLSIIENLTLAQVKVLKRDKAPAREKALKLLERVGLSAHANKFPAQLSGGQQQRVAIARALCMDPIAMLFDEPTSALDPEMINEVLDVMVELANEGMTMMVVTHEMGFARKVANRVIFMDEGKIVEDSPKDAFFDDPKSDRAKDFLAKILH',\n",
       "  'mw': 26661.0},\n",
       " 'b0653': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AER5',\n",
       "  'gene': 'b0653',\n",
       "  'locus': ['b0653', 'JW0648'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': '(Microbial infection) Probably transports the toxic C-terminal region of CdiA from P.luminescens strain TTO1 across the inner membrane to the cytoplasm, where CdiA has a toxic effect. Toxin transport is strain-specific, mutations in this gene do not confer resistance to several other tested CdiA toxins'},\n",
       "  'seq': 'MYEFDWSSIVPSLPYLLDGLVITLKITVTAVVIGILWGTMLAVMRLSSFAPVAWFAKAYVNVFRSIPLVMVLLWFYLIVPGFLQNVLGLSPKNDIRLISAMVAFSMFEAAYYSEIIRAGIQSISRGQSSAALALGMTHWQSMKLIILPQAFRAMVPLLLTQGIVLFQDTSLVYVLSLADFFRTASTIGERDGTQVEMILFAGFVYFVISLSASLLVSYLKRRTA',\n",
       "  'mw': 24915.0},\n",
       " 'b0654': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AER3',\n",
       "  'gene': 'b0654',\n",
       "  'locus': ['b0654', 'JW0649'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': '(Microbial infection) Probably transports the toxic C-terminal region of CdiA from P.luminescens strain TTO1 across the inner membrane to the cytoplasm, where CdiA has a toxic effect. Toxin transport is strain-specific, mutations in this gene do not confer resistance to several other tested CdiA toxins'},\n",
       "  'seq': 'MSIDWNWGIFLQQAPFGNTTYLGWIWSGFQVTIALSICAWIIAFLVGSFFGILRTVPNRFLSGLGTLYVELFRNVPLIVQFFTWYLVIPELLPEKIGMWFKAELDPNIQFFLSSMLCLGLFTAARVCEQVRAAIQSLPRGQKNAALAMGLTLPQAYRYVLLPNAYRVIVPPMTSEMMNLVKNSAIASTIGLVDMAAQAGKLLDYSAHAWESFTAITLAYVLINAFIMLVMTLVERKVRLPGNMGGK',\n",
       "  'mw': 27503.0},\n",
       " 'b0655': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37902',\n",
       "  'gene': 'b0655',\n",
       "  'locus': ['b0655', 'JW5092'],\n",
       "  'synonyms': 'ybeJ yzzK',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the ABC transporter complex GltIJKL involved in glutamate and aspartate uptake. Binds to both glutamate and aspartate'},\n",
       "  'seq': 'MQLRKPATAILALALSAGLAQADDAAPAAGSTLDKIAKNGVIVVGHRESSVPFSYYDNQQKVVGYSQDYSNAIVEAVKKKLNKPDLQVKLIPITSQNRIPLLQNGTFDFECGSTTNNVERQKQAAFSDTIFVVGTRLLTKKGGDIKDFANLKDKAVVVTSGTTSEVLLNKLNEEQKMNMRIISAKDHGDSFRTLESGRAVAFMMDDALLAGERAKAKKPDNWEIVGKPQSQEAYGCMLRKDDPQFKKLMDDTIAQVQTSGEAEKWFDKWFKNPIPPKNLNMNFELSDEMKALFKEPNDKALN',\n",
       "  'mw': 33420.0},\n",
       " 'b0657': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P23930',\n",
       "  'gene': 'b0657',\n",
       "  'locus': ['b0657', 'JW0654'],\n",
       "  'synonyms': 'cutE',\n",
       "  'properties': {'Catalytic Activity': [('a glycerophospholipid + N-terminal S-1,2-diacyl-sn-glyceryl-L-cysteinyl-[lipoprotein] = a 2-acyl-sn-glycero-3-phospholipid + H(+) + N-acyl-S-1,2-diacyl-sn-glyceryl-L-cysteinyl-[lipoprotein]',\n",
       "     '2.3.1.269')],\n",
       "   'Function': 'Catalyzes the phospholipid dependent N-acylation of the N-terminal cysteine of apolipoprotein, the last step in lipoprotein maturation (PubMed:2032623, PubMed:21676878, PubMed:28885614, PubMed:28675161). Utilizes a two-step reaction via a ping-pong mechanism (PubMed:21676878, PubMed:28675161). Lnt undergoes covalent modification in the presence of phospholipids, resulting in a thioester acyl-enzyme intermediate. It then transfers the acyl chain to the amine group of the N-terminal diacylglyceryl-modified cysteine of apolipoprotein, leading to the formation of mature triacylated lipoprotein (PubMed:21676878, PubMed:28675161). In vitro, can utilize the phospholipids phosphatidylethanolamine (PE), phosphatidylglycerol (PG), phosphatidic acid (PA) or cardiolipin (CL) (PubMed:2032623, PubMed:21676878). PE is the most efficient acyl donor (PubMed:21676878)',\n",
       "   'Pathway': 'Protein modification; lipoprotein biosynthesis (N-acyl transfer)'},\n",
       "  'seq': 'MAFASLIERQRIRLLLALLFGACGTLAFSPYDVWPAAIISLMGLQALTFNRRPLQSAAIGFCWGFGLFGSGINWVYVSIATFGGMPGPVNIFLVVLLAAYLSLYTGLFAGVLSRLWPKTTWLRVAIAAPALWQVTEFLRGWVLTGFPWLQFGYSQIDGPLKGLAPIMGVEAINFLLMMVSGLLALALVKRNWRPLVVAVVLFALPFPLRYIQWFTPQPEKTIQVSMVQGDIPQSLKWDEGQLLNTLKIYYNATAPLMGKSSLIIWPESAITDLEINQQPFLKALDGELRDKGSSLVTGIVDARLNKQNRYDTYNTIITLGKGAPYSYESADRYNKNHLVPFGEFVPLESILRPLAPFFDLPMSSFSRGPYIQPPLSANGIELTAAICYEIILGEQVRDNFRPDTDYLLTISNDAWFGKSIGPWQHFQMARMRALELARPLLRSTNNGITAVIGPQGEIQAMIPQFTREVLTTNVTPTTGLTPYARTGNWPLWVLTALFGFAAVLMSLRQRRK',\n",
       "  'mw': 57066.0},\n",
       " 'b0662': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P75728',\n",
       "  'gene': 'b0662',\n",
       "  'locus': ['b0662', 'JW0659'],\n",
       "  'synonyms': 'yleB',\n",
       "  'properties': {'Catalytic Activity': [('a 6-methoxy-3-methyl-2-all-trans-polyprenyl-1,4-benzoquinol + AH2 + O2 = A + a 3-demethylubiquinol + H2O',\n",
       "     '1.14.99.60')],\n",
       "   'Function': 'Catalyzes the hydroxylation of 2-octaprenyl-3-methyl-6-methoxy-1,4-benzoquinol during ubiquinone biosynthesis',\n",
       "   'Pathway': 'Cofactor biosynthesis; ubiquinone biosynthesis'},\n",
       "  'seq': 'MTNQPTEIAIVGGGMVGGALALGLAQHGFAVTVIEHAEPAPFVADSQPDVRISAISAASVSLLKGLGVWDAVQAMRCHPYRRLETWEWETAHVVFDAAELKLPLLGYMVENTVLQQALWQALEAHPKVTLRVPGSLIALHRHDDLQELELKGGEVIRAKLVIGADGANSQVRQMAGIGVHAWQYAQSCMLISVQCENDPGDSTWQQFTPDGPRAFLPLFDNWASLVWYDSPARIRQLQNMNMAQLQAEIAKHFPSRLGYVTPLAAGAFPLTRRHALQYVQPGLALVGDAAHTIHPLAGQGVNLGYRDVDALIDVLVNARSYGEAWASYPVLKRYQMRRMADNFIMQSGMDLFYAGFSNNLPPLRFMRNLGLMAAERAGVLKRQALKYALGL',\n",
       "  'mw': 42953.0},\n",
       " 'b0674': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P22106',\n",
       "  'gene': 'b0674',\n",
       "  'locus': ['b0674', 'JW0660'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + H2O + L-aspartate + L-glutamine = AMP + diphosphate + H(+) + L-asparagine + L-glutamate',\n",
       "     '6.3.5.4')],\n",
       "   'Function': 'Catalyzes the ATP-dependent conversion of aspartate into asparagine, using glutamine as a source of nitrogen. Can also use ammonia as the nitrogen source in vitro, albeit with lower efficiency. As nucleotide substrates, ATP and dATP are utilized at a similar rate in both the glutamine- and ammonia-dependent reactions, whereas GTP utilization is only 15% that of ATP, and CTP, UTP, ITP and XTP are very poor or not substrates. Also exhibits glutaminase activity',\n",
       "   'Regulation Activity': 'Glutamine-dependent asparagine synthesis activity can be inhibited by aspartic acid analogs (such as a sulfinate derivative and (2S,3R)-2-amino-3-methylsuccinate) in vitro; the inhibition is competitive with respect to aspartate',\n",
       "   'Pathway': 'Amino-acid biosynthesis; L-asparagine biosynthesis; L-asparagine from L-aspartate (L-Gln route): step 1/1'},\n",
       "  'seq': 'MCSIFGVFDIKTDAVELRKKALELSRLMRHRGPDWSGIYASDNAILAHERLSIVDVNAGAQPLYNQQKTHVLAVNGEIYNHQALRAEYGDRYQFQTGSDCEVILALYQEKGPEFLDDLQGMFAFALYDSEKDAYLIGRDHLGIIPLYMGYDEHGQLYVASEMKALVPVCRTIKEFPAGSYLWSQDGEIRSYYHRDWFDYDAVKDNVTDKNELRQALEDSVKSHLMSDVPYGVLLSGGLDSSIISAITKKYAARRVEDQERSEAWWPQLHSFAVGLPGSPDLKAAQEVANHLGTVHHEIHFTVQEGLDAIRDVIYHIETYDVTTIRASTPMYLMSRKIKAMGIKMVLSGEGSDEVFGGYLYFHKAPNAKELHEETVRKLLALHMYDCARANKAMSAWGVEARVPFLDKKFLDVAMRINPQDKMCGNGKMEKHILRECFEAYLPASVAWRQKEQFSDGVGYSWIDTLKEVAAQQVSDQQLETARFRFPYNTPTSKEAYLYREIFEELFPLPSAAECVPGGPSVACSSAKAIEWDEAFKKMDDPSGRAVGVHQSAYK',\n",
       "  'mw': 62659.0},\n",
       " 'b0677': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AF18',\n",
       "  'gene': 'b0677',\n",
       "  'locus': ['b0677', 'JW0663'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('H2O + N-acetyl-D-glucosamine 6-phosphate = acetate + D-glucosamine 6-phosphate',\n",
       "     '3.5.1.25')],\n",
       "   'Function': 'Involved in the first step in the biosynthesis of amino-sugar-nucleotides. Catalyzes the hydrolysis of the N-acetyl group of N-acetylglucosamine-6-phosphate (GlcNAc-6-P) to yield glucosamine 6-phosphate and acetate. In vitro, can also hydrolyze substrate analogs such as N-thioacetyl-D-glucosamine-6-phosphate, N-trifluoroacetyl-D-glucosamine-6-phosphate, N-acetyl-D-glucosamine-6-sulfate, N-acetyl-D-galactosamine-6-phosphate, and N-formyl-D-glucosamine-6-phosphate',\n",
       "   'Regulation Activity': \"Inhibited by high substrate concentration and by products glucosamine 6-phosphate and acetate. Completely inactivated by the treatment with 5,5'-dithio-bis(2-nitrobenzoate) or 2,2'-dithio-pyridine (2-DPDS). Inhibited by 1,10-phenanthroline and EDTA\",\n",
       "   'Pathway': 'Amino-sugar metabolism; N-acetylneuraminate degradation; D-fructose 6-phosphate from N-acetylneuraminate: step 4/5'},\n",
       "  'seq': 'MYALTQGRIFTGHEFLDDHAVVIADGLIKSVCPVAELPPEIEQRSLNGAILSPGFIDVQLNGCGGVQFNDTAEAVSVETLEIMQKANEKSGCTNYLPTLITTSDELMKQGVRVMREYLAKHPNQALGLHLEGPWLNLVKKGTHNPNFVRKPDAALVDFLCENADVITKVTLAPEMVPAEVISKLANAGIVVSAGHSNATLKEAKAGFRAGITFATHLYNAMPYITGREPGLAGAILDEADIYCGIIADGLHVDYANIRNAKRLKGDKLCLVTDATAPAGANIEQFIFAGKTIYYRNGLCVDENGTLSGSSLTMIEGVRNLVEHCGIALDEVLRMATLYPARAIGVEKRLGTLAAGKVANLTAFTPDFKITKTIVNGNEVVTQ',\n",
       "  'mw': 40949.0},\n",
       " 'b0678': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A759',\n",
       "  'gene': 'b0678',\n",
       "  'locus': ['b0678', 'JW0664'],\n",
       "  'synonyms': 'glmD',\n",
       "  'properties': {'Catalytic Activity': [('alpha-D-glucosamine 6-phosphate + H2O = beta-D-fructose 6-phosphate + NH4(+)',\n",
       "     '3.5.99.6')],\n",
       "   'Function': 'Catalyzes the reversible isomerization-deamination of glucosamine 6-phosphate (GlcN6P) to form fructose 6-phosphate (Fru6P) and ammonium ion',\n",
       "   'Regulation Activity': 'Allosterically activated by N-acetylglucosamine 6-phosphate (GlcNAc6P). Competitively inhibited by 2-amino-2-deoxy-D-glucitol-6-phosphate (GlcN-ol-6P)',\n",
       "   'Pathway': 'Amino-sugar metabolism; N-acetylneuraminate degradation; D-fructose 6-phosphate from N-acetylneuraminate: step 5/5'},\n",
       "  'seq': 'MRLIPLTTAEQVGKWAARHIVNRINAFKPTADRPFVLGLPTGGTPMTTYKALVEMHKAGQVSFKHVVTFNMDEYVGLPKEHPESYYSFMHRNFFDHVDIPAENINLLNGNAPDIDAECRQYEEKIRSYGKIHLFMGGVGNDGHIAFNEPASSLASRTRIKTLTHDTRVANSRFFDNDVNQVPKYALTVGVGTLLDAEEVMILVLGSQKALALQAAVEGCVNHMWTISCLQLHPKAIMVCDEPSTMELKVKTLRYFNELEAENIKGL',\n",
       "  'mw': 29774.0},\n",
       " 'b0679': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P09323',\n",
       "  'gene': 'b0679',\n",
       "  'locus': ['b0679', 'JW0665'],\n",
       "  'synonyms': 'pstN',\n",
       "  'properties': {'Catalytic Activity': [('N(pros)-phospho-L-histidyl-[protein] + N-acetyl-D-glucosamine(out) = L-histidyl-[protein] + N-acetyl-D-glucosamine 6-phosphate(in)',\n",
       "     '2.7.1.193')],\n",
       "   'Function': 'The phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS), a major carbohydrate active transport system, catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane (PubMed:4919472). This system is involved in N-acetylglucosamine transport (PubMed:4919472). It can also transport and phosphorylate the antibiotic streptozotocin (PubMed:161156). Could play a significant role in the recycling of peptidoglycan (PubMed:19617367)',\n",
       "   'Regulation Activity': 'P-chloromercuribenzoate inhibits the accumulation of both N-acetyl-D-glucosamine and antibiotic streptozotocin (2-deoxy-2-(3-methyl-3-nitrosoureido)-D-glucopyranose). N-acetyl-D-glucosamine is a competitive inhibitor for the uptake of streptozotocin'},\n",
       "  'seq': 'MNILGFFQRLGRALQLPIAVLPVAALLLRFGQPDLLNVAFIAQAGGAIFDNLALIFAIGVASSWSKDSAGAAALAGAVGYFVLTKAMVTINPEINMGVLAGIITGLVGGAAYNRWSDIKLPDFLSFFGGKRFVPIATGFFCLVLAAIFGYVWPPVQHAIHAGGEWIVSAGALGSGIFGFINRLLIPTGLHQVLNTIAWFQIGEFTNAAGTVFHGDINRFYAGDGTAGMFMSGFFPIMMFGLPGAALAMYFAAPKERRPMVGGMLLSVAVTAFLTGVTEPLEFLFMFLAPLLYLLHALLTGISLFVATLLGIHAGFSFSAGAIDYALMYNLPAASQNVWMLLVMGVIFFAIYFVVFSLVIRMFNLKTPGREDKEDEIVTEEANSNTEEGLTQLATNYIAAVGGTDNLKAIDACITRLRLTVADSARVNDTMCKRLGASGVVKLNKQTIQVIVGAKAESIGDAMKKVVARGPVAAASAEATPATAAPVAKPQAVPNAVSIAELVSPITGDVVALDQVPDEAFASKAVGDGVAVKPTDKIVVSPAAGTIVKIFNTNHAFCLETEKGAEIVVHMGIDTVALEGKGFKRLVEEGAQVSAGQPILEMDLDYLNANARSMISPVVCSNIDDFSGLIIKAQGHIVAGQTPLYEIKK',\n",
       "  'mw': 68347.0},\n",
       " 'b0680': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P00962',\n",
       "  'gene': 'b0680',\n",
       "  'locus': ['b0680', 'JW0666'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + L-glutamine + tRNA(Gln) = AMP + diphosphate + L-glutaminyl-tRNA(Gln)',\n",
       "     '6.1.1.18')]},\n",
       "  'seq': 'MSEAEARPTNFIRQIIDEDLASGKHTTVHTRFPPEPNGYLHIGHAKSICLNFGIAQDYKGQCNLRFDDTNPVKEDIEYVESIKNDVEWLGFHWSGNVRYSSDYFDQLHAYAIELINKGLAYVDELTPEQIREYRGTLTQPGKNSPYRDRSVEENLALFEKMRAGGFEEGKACLRAKIDMASPFIVMRDPVLYRIKFAEHHQTGNKWCIYPMYDFTHCISDALEGITHSLCTLEFQDNRRLYDWVLDNITIPVHPRQYEFSRLNLEYTVMSKRKLNLLVTDKHVEGWDDPRMPTISGLRRRGYTAASIREFCKRIGVTKQDNTIEMASLESCIREDLNENAPRAMAVIDPVKLVIENYQGEGEMVTMPNHPNKPEMGSRQVPFSGEIWIDRADFREEANKQYKRLVLGKEVRLRNAYVIKAERVEKDAEGNITTIFCTYDADTLSKDPADGRKVKGVIHWVSAAHALPVEIRLYDRLFSVPNPGAADDFLSVINPESLVIKQGFAEPSLKDAVAGKAFQFEREGYFCLDSRHSTAEKPVFNRTVGLRDTWAKVGE',\n",
       "  'mw': 63478.0},\n",
       " 'b0684': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P61949',\n",
       "  'gene': 'b0684',\n",
       "  'locus': ['b0684', 'JW0671'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Low-potential electron donor to a number of redox enzymes (Potential). Involved in the reactivation of inactive cob(II)alamin in methionine synthase'},\n",
       "  'seq': 'MAITGIFFGSDTGNTENIAKMIQKQLGKDVADVHDIAKSSKEDLEAYDILLLGIPTWYYGEAQCDWDDFFPTLEEIDFNGKLVALFGCGDQEDYAEYFCDALGTIRDIIEPRGATIVGHWPTAGYHFEASKGLADDDHFVGLAIDEDRQPELTAERVEKWVKQISEELHLDEILNA',\n",
       "  'mw': 19737.0},\n",
       " 'b0688': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P36938',\n",
       "  'gene': 'b0688',\n",
       "  'locus': ['b0688', 'JW0675'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('alpha-D-glucose 1-phosphate = alpha-D-glucose 6-phosphate',\n",
       "     '5.4.2.2')],\n",
       "   'Function': 'This enzyme participates in both the breakdown and synthesis of glucose'},\n",
       "  'seq': 'MAIHNRAGQPAQQSDLINVAQLTAQYYVLKPEAGNAEHAVKFGTSGHRGSAARHSFNEPHILAIAQAIAEERAKNGITGPCYVGKDTHALSEPAFISVLEVLAANGVDVIVQENNGFTPTPAVSNAILVHNKKGGPLADGIVITPSHNPPEDGGIKYNPPNGGPADTNVTKVVEDRANALLADGLKGVKRISLDEAMASGHVKEQDLVQPFVEGLADIVDMAAIQKAGLTLGVDPLGGSGIEYWKRIGEYYNLNLTIVNDQVDQTFRFMHLDKDGAIRMDCSSECAMAGLLALRDKFDLAFANDPDYDRHGIVTPAGLMNPNHYLAVAINYLFQHRPQWGKDVAVGKTLVSSAMIDRVVNDLGRKLVEVPVGFKWFVDGLFDGSFGFGGEESAGASFLRFDGTPWSTDKDGIIMCLLAAEITAVTGKNPQEHYNELAKRFGAPSYNRLQAAATSAQKAALSKLSPEMVSASTLAGDPITARLTAAPGNGASIGGLKVMTDNGWFAARPSGTEDAYKIYCESFLGEEHRKQIEKEAVEIVSEVLKNA',\n",
       "  'mw': 58361.0},\n",
       " 'b0692': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AAF1',\n",
       "  'gene': 'b0692',\n",
       "  'locus': ['b0692', 'JW0679'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('H(+)(in) + putrescine(in) = H(+)(out) + putrescine(out)',\n",
       "     ''),\n",
       "    ('L-ornithine(out) + putrescine(in) = L-ornithine(in) + putrescine(out)',\n",
       "     '')],\n",
       "   'Function': 'Catalyzes both the uptake and excretion of putrescine. The uptake of putrescine is dependent on the membrane potential and the excretion involves putrescine-ornithine antiporter activity',\n",
       "   'Regulation Activity': 'Uptake activity, but not antiporter activity, is inhibited by CCCP and N-ethylmaleimide (NEM). Uptake of putrescine is inhibited by high concentrations of ornithine'},\n",
       "  'seq': 'MSQAKSNKMGVVQLTILTMVNMMGSGIIMLPTKLAEVGTISIISWLVTAVGSMALAWAFAKCGMFSRKSGGMGGYAEYAFGKSGNFMANYTYGVSLLIANVAIAISAVGYGTELLGASLSPVQIGLATIGVLWICTVANFGGARITGQISSITVWGVIIPVVGLCIIGWFWFSPTLYVDSWNPHHAPFFSAVGSSIAMTLWAFLGLESACANTDVVENPERNVPIAVLGGTLGAAVIYIVSTNVIAGIVPNMELANSTAPFGLAFAQMFTPEVGKVIMALMVMSCCGSLLGWQFTIAQVFKSSSDEGYFPKIFSRVTKVDAPVQGMLTIVIIQSGLALMTISPSLNSQFNVLVNLAVVTNIIPYILSMAALVIIQKVANVPPSKAKVANFVAFVGAMYSFYALYSSGEEAMLYGSIVTFLGWTLYGLVSPRFELKNKHG',\n",
       "  'mw': 46495.0},\n",
       " 'b0693': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P24169',\n",
       "  'gene': 'b0693',\n",
       "  'locus': ['b0693', 'JW0680'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('H(+) + L-ornithine = CO2 + putrescine',\n",
       "     '4.1.1.17')],\n",
       "   'Function': 'The first enzyme leading to putrescine and thus polyamine synthesis',\n",
       "   'Pathway': 'Amine and polyamine biosynthesis; putrescine biosynthesis via L-ornithine pathway; putrescine from L-ornithine: step 1/1'},\n",
       "  'seq': 'MSKLKIAVSDSCPDCFTTQRECIYINESRNIDVAAIVLSLNDVTCGKLDEIDATGYGIPVFIATENQERVPAEYLPRISGVFENCESRREFYGRQLETAASHYETQLRPPFFRALVDYVNQGNSAFDCPGHQGGEFFRRHPAGNQFVEYFGEALFRADLCNADVAMGDLLIHEGAPCIAQQHAAKVFNADKTYFVLNGTSSSNKVVLNALLTPGDLVLFDRNNHKSNHHGALLQAGATPVYLETARNPYGFIGGIDAHCFEESYLRELIAEVAPQRAKEARPFRLAVIQLGTYDGTIYNARQVVDKIGHLCDYILFDSAWVGYEQFIPMMADCSPLLLDLNENDPGILVTQSVHKQQAGFSQTSQIHKKDSHIKGQQRYVPHKRMNNAFMMHASTSPFYPLFAALNINAKMHEGVSGRNMWMDCVVNGINARKLILDNCQHIRPFVPELVDGKPWQSYETAQIAVDLRFFQFVPGEHWHSFEGYAENQYFVDPCKLLLTTPGIDARNGEYEAFGVPATILANFLRENGVVPEKCDLNSILFLLTPAEDMAKLQQLVALLVRFEKLLESDAPLAEVLPSIYKQHEERYAGYTLRQLCQEMHDLYARHNVKQLQKEMFRKEHFPRVSMNPQEANYAYLRGEVELVRLPDAEGRIAAEGALPYPPGVLCVVPGEIWGGAVLRYFSALEEGINLLPGFAPELQGVYIEEHDGRKQVWCYVIKPRDAQSTLLKGEKL',\n",
       "  'mw': 82416.0},\n",
       " 'b0696': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P03961',\n",
       "  'gene': 'b0696',\n",
       "  'locus': ['b0696', 'JW0684'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the high-affinity ATP-driven potassium transport (or Kdp) system, which catalyzes the hydrolysis of ATP coupled with the electrogenic transport of potassium into the cytoplasm (PubMed:2849541, PubMed:8499455, PubMed:23930894). This subunit acts as a catalytic chaperone that increases the ATP-binding affinity of the ATP-hydrolyzing subunit KdpB by the formation of a transient KdpB/KdpC/ATP ternary complex (PubMed:21711450)'},\n",
       "  'seq': 'MSGLRPALSTFIFLLLITGGVYPLLTTVLGQWWFPWQANGSLIREGDTVRGSALIGQNFTGNGYFHGRPSATAEMPYNPQASGGSNLAVSNPELDKLIAARVAALRAANPDASASVPVELVTASASGLDNNITPQAAAWQIPRVAKARNLSVEQLTQLIAKYSQQPLVKYIGQPVVNIVELNLALDKLDE',\n",
       "  'mw': 20267.0},\n",
       " 'b0697': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P03960',\n",
       "  'gene': 'b0697',\n",
       "  'locus': ['b0697', 'JW0685'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + H2O + K(+)(out) = ADP + H(+) + K(+)(in) + phosphate',\n",
       "     '7.2.2.6')],\n",
       "   'Function': 'Part of the high-affinity ATP-driven potassium transport (or Kdp) system, which catalyzes the hydrolysis of ATP coupled with the electrogenic transport of potassium into the cytoplasm (PubMed:23930894, PubMed:2849541, PubMed:8499455). This subunit is responsible for energy coupling to the transport system and for the release of the potassium ions to the cytoplasm (PubMed:16354672, PubMed:30478378, PubMed:34272288)',\n",
       "   'Regulation Activity': \"ATPase activity is inhibited by phosphorylation at Ser-162 (PubMed:28636601, PubMed:30478378). ATPase activity is inhibited by vanadate, fluorescein isothiocyanate, N,N'-dicyclohexylcarbodiimide and N-ethylmaleimide\"},\n",
       "  'seq': 'MSRKQLALFEPTLVVQALKEAVKKLNPQAQWRNPVMFIVWIGSLLTTCISIAMASGAMPGNALFSAAISGWLWITVLFANFAEALAEGRSKAQANSLKGVKKTAFARKLREPKYGAAADKVPADQLRKGDIVLVEAGDIIPCDGEVIEGGASVDESAITGESAPVIRESGGDFASVTGGTRILSDWLVIECSVNPGETFLDRMIAMVEGAQRRKTPNEIALTILLIALTIVFLLATATLWPFSAWGGNAVSVTVLVALLVCLIPTTIGGLLSAIGVAGMSRMLGANVIATSGRAVEAAGDVDVLLLDKTGTITLGNRQASEFIPAQGVDEKTLADAAQLASLADETPEGRSIVILAKQRFNLRERDVQSLHATFVPFTAQSRMSGINIDNRMIRKGSVDAIRRHVEANGGHFPTDVDQKVDQVARQGATPLVVVEGSRVLGVIALKDIVKGGIKERFAQLRKMGIKTVMITGDNRLTAAAIAAEAGVDDFLAEATPEAKLALIRQYQAEGRLVAMTGDGTNDAPALAQADVAVAMNSGTQAAKEAGNMVDLDSNPTKLIEVVHIGKQMLMTRGSLTTFSIANDVAKYFAIIPAAFAATYPQLNALNIMCLHSPDSAILSAVIFNALIIVFLIPLALKGVSYKPLTASAMLRRNLWIYGLGGLLVPFIGIKVIDLLLTVCGLV',\n",
       "  'mw': 72199.0},\n",
       " 'b0698': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P03959',\n",
       "  'gene': 'b0698',\n",
       "  'locus': ['b0698', 'JW0686'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the high-affinity ATP-driven potassium transport (or Kdp) system, which catalyzes the hydrolysis of ATP coupled with the electrogenic transport of potassium into the cytoplasm (PubMed:23930894, PubMed:2849541, PubMed:8499455). This subunit binds the periplasmic potassium ions and delivers the ions to the membrane domain of KdpB through an intramembrane tunnel (PubMed:7896809, PubMed:30478378, PubMed:34272288)'},\n",
       "  'seq': 'MAAQGFLLIATFLLVLMVLARPLGSGLARLINDIPLPGTTGVERVLFRALGVSDREMNWKQYLCAILGLNMLGLAVLFFMLLGQHYLPLNPQQLPGLSWDLALNTAVSFVTNTNWQSYSGETTLSYFSQMAGLTVQNFLSAASGIAVIFALIRAFTRQSMSTLGNAWVDLLRITLWVLVPVALLIALFFIQQGALQNFLPYQAVNTVEGAQQLLPMGPVASQEAIKMLGTNGGGFFNANSSHPFENPTALTNFVQMLAIFLIPTALCFAFGEVMGDRRQGRMLLWAMSVIFVICVGVVMWAEVQGNPHLLALGTDSSINMEGKESRFGVLVSSLFAVVTTAASCGAVIAMHDSFTALGGMVPMWLMQIGEVVFGGVGSGLYGMMLFVLLAVFIAGLMIGRTPEYLGKKIDVREMKLTALAILVTPTLVLMGAALAMMTDAGRSAMLNPGPHGFSEVLYAVSSAANNNGSAFAGLSANSPFWNCLLAFCMFVGRFGVIIPVMAIAGSLVSKKSQAASSGTLPTHGPLFVGLLIGTVLLVGALTFIPALALGPVAEYLS',\n",
       "  'mw': 59189.0},\n",
       " 'b0720': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ABH7',\n",
       "  'gene': 'b0720',\n",
       "  'locus': ['b0720', 'JW0710'],\n",
       "  'synonyms': 'gluT icdB',\n",
       "  'properties': {'Catalytic Activity': [('acetyl-CoA + H2O + oxaloacetate = citrate + CoA + H(+)',\n",
       "     '2.3.3.16')],\n",
       "   'Regulation Activity': 'Allosterically inhibited by NADH',\n",
       "   'Pathway': 'Carbohydrate metabolism; tricarboxylic acid cycle; isocitrate from oxaloacetate: step 1/2'},\n",
       "  'seq': 'MADTKAKLTLNGDTAVELDVLKGTLGQDVIDIRTLGSKGVFTFDPGFTSTASCESKITFIDGDEGILLHRGFPIDQLATDSNYLEVCYILLNGEKPTQEQYDEFKTTVTRHTMIHEQITRLFHAFRRDSHPMAVMCGITGALAAFYHDSLDVNNPRHREIAAFRLLSKMPTMAAMCYKYSIGQPFVYPRNDLSYAGNFLNMMFSTPCEPYEVNPILERAMDRILILHADHEQNASTSTVRTAGSSGANPFACIAAGIASLWGPAHGGANEAALKMLEEISSVKHIPEFVRRAKDKNDSFRLMGFGHRVYKNYDPRATVMRETCHEVLKELGTKDDLLEVAMELENIALNDPYFIEKKLYPNVDFYSGIILKAMGIPSSMFTVIFAMARTVGWIAHWSEMHSDGMKIARPRQLYTGYEKRDFKSDIKR',\n",
       "  'mw': 48015.0},\n",
       " 'b0721': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P69054',\n",
       "  'gene': 'b0721',\n",
       "  'locus': ['b0721', 'JW0711'],\n",
       "  'synonyms': 'cybA',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Membrane-anchoring subunit of succinate dehydrogenase (SDH)',\n",
       "   'Pathway': 'Carbohydrate metabolism; tricarboxylic acid cycle'},\n",
       "  'seq': 'MIRNVKKQRPVNLDLQTIRFPITAIASILHRVSGVITFVAVGILLWLLGTSLSSPEGFEQASAIMGSFFVKFIMWGILTALAYHVVVGIRHMMMDFGYLEETFEAGKRSAKISFVITVVLSLLAGVLVW',\n",
       "  'mw': 14299.0},\n",
       " 'b0722': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AC44',\n",
       "  'gene': 'b0722',\n",
       "  'locus': ['b0722', 'JW0712'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Membrane-anchoring subunit of succinate dehydrogenase (SDH)',\n",
       "   'Pathway': 'Carbohydrate metabolism; tricarboxylic acid cycle'},\n",
       "  'seq': 'MVSNASALGRNGVHDFILVRATAIVLTLYIIYMVGFFATSGELTYEVWIGFFASAFTKVFTLLALFSILIHAWIGMWQVLTDYVKPLALRLMLQLVIVVALVVYVIYGFVVVWGV',\n",
       "  'mw': 12868.0},\n",
       " 'b0723': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AC41',\n",
       "  'gene': 'b0723',\n",
       "  'locus': ['b0723', 'JW0713'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('a quinone + succinate = a quinol + fumarate',\n",
       "     '1.3.5.1')],\n",
       "   'Function': 'Two distinct, membrane-bound, FAD-containing enzymes are responsible for the catalysis of fumarate and succinate interconversion; the fumarate reductase is used in anaerobic growth, and the succinate dehydrogenase is used in aerobic growth',\n",
       "   'Regulation Activity': 'Inhibited by oxaloacetate',\n",
       "   'Pathway': 'Carbohydrate metabolism; tricarboxylic acid cycle; fumarate from succinate (bacterial route): step 1/1'},\n",
       "  'seq': 'MKLPVREFDAVVIGAGGAGMRAALQISQSGQTCALLSKVFPTRSHTVSAQGGITVALGNTHEDNWEWHMYDTVKGSDYIGDQDAIEYMCKTGPEAILELEHMGLPFSRLDDGRIYQRPFGGQSKNFGGEQAARTAAAADRTGHALLHTLYQQNLKNHTTIFSEWYALDLVKNQDGAVVGCTALCIETGEVVYFKARATVLATGGAGRIYQSTTNAHINTGDGVGMAIRAGVPVQDMEMWQFHPTGIAGAGVLVTEGCRGEGGYLLNKHGERFMERYAPNAKDLAGRDVVARSIMIEIREGRGCDGPWGPHAKLKLDHLGKEVLESRLPGILELSRTFAHVDPVKEPIPVIPTCHYMMGGIPTKVTGQALTVNEKGEDVVVPGLFAVGEIACVSVHGANRLGGNSLLDLVVFGRAAGLHLQESIAEQGALRDASESDVEASLDRLNRWNNNRNGEDPVAIRKALQECMQHNFSVFREGDAMAKGLEQLKVIRERLKNARLDDTSSEFNTQRVECLELDNLMETAYATAVSANFRTESRGAHSRFDFPDRDDENWLCHSLYLPESESMTRRSVNMEPKLRPAFPPKIRTY',\n",
       "  'mw': 64422.0},\n",
       " 'b0724': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P07014',\n",
       "  'gene': 'b0724',\n",
       "  'locus': ['b0724', 'JW0714'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('a quinone + succinate = a quinol + fumarate',\n",
       "     '1.3.5.1')],\n",
       "   'Function': 'Two distinct, membrane-bound, FAD-containing enzymes are responsible for the catalysis of fumarate and succinate interconversion; the fumarate reductase is used in anaerobic growth, and the succinate dehydrogenase is used in aerobic growth',\n",
       "   'Pathway': 'Carbohydrate metabolism; tricarboxylic acid cycle; fumarate from succinate (bacterial route): step 1/1'},\n",
       "  'seq': 'MRLEFSIYRYNPDVDDAPRMQDYTLEADEGRDMMLLDALIQLKEKDPSLSFRRSCREGVCGSDGLNMNGKNGLACITPISALNQPGKKIVIRPLPGLPVIRDLVVDMGQFYAQYEKIKPYLLNNGQNPPAREHLQMPEQREKLDGLYECILCACCSTSCPSFWWNPDKFIGPAGLLAAYRFLIDSRDTETDSRLDGLSDAFSVFRCHSIMNCVSVCPKGLNPTRAIGHIKSMLLQRNA',\n",
       "  'mw': 26770.0},\n",
       " 'b0726': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AFG3',\n",
       "  'gene': 'b0726',\n",
       "  'locus': ['b0726', 'JW0715'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(R)-N(6)-lipoyl-L-lysyl-[dihydrolipoyllysine-residue succinyltransferase] + 2-oxoglutarate + H(+) = (R)-N(6)-(S(8)-succinyldihydrolipoyl)-L-lysyl-[dihydrolipoyllysine-residue succinyltransferase] + CO2',\n",
       "     '1.2.4.2')],\n",
       "   'Function': 'E1 component of the 2-oxoglutarate dehydrogenase (OGDH) complex which catalyzes the decarboxylation of 2-oxoglutarate, the first step in the conversion of 2-oxoglutarate to succinyl-CoA and CO(2)',\n",
       "   'Regulation Activity': 'Inhibited by oxaloacetate'},\n",
       "  'seq': 'MQNSALKAWLDSSYLSGANQSWIEQLYEDFLTDPDSVDANWRSTFQQLPGTGVKPDQFHSQTREYFRRLAKDASRYSSTISDPDTNVKQVKVLQLINAYRFRGHQHANLDPLGLWQQDKVADLDPSFHDLTEADFQETFNVGSFASGKETMKLGELLEALKQTYCGPIGAEYMHITSTEEKRWIQQRIESGRATFNSEEKKRFLSELTAAEGLERYLGAKFPGAKRFSLEGGDALIPMLKEMIRHAGNSGTREVVLGMAHRGRLNVLVNVLGKKPQDLFDEFAGKHKEHLGTGDVKYHMGFSSDFQTDGGLVHLALAFNPSHLEIVSPVVIGSVRARLDRLDEPSSNKVLPITIHGDAAVTGQGVVQETLNMSKARGYEVGGTVRIVINNQVGFTTSNPLDARSTPYCTDIGKMVQAPIFHVNADDPEAVAFVTRLALDFRNTFKRDVFIDLVCYRRHGHNEADEPSATQPLMYQKIKKHPTPRKIYADKLEQEKVATLEDATEMVNLYRDALDAGDCVVAEWRPMNMHSFTWSPYLNHEWDEEYPNKVEMKRLQELAKRISTVPEAVEMQSRVAKIYGDRQAMAAGEKLFDWGGAENLAYATLVDEGIPVRLSGEDSGRGTFFHRHAVIHNQSNGSTYTPLQHIHNGQGAFRVWDSVLSEEAVLAFEYGYATAEPRTLTIWEAQFGDFANGAQVVIDQFISSGEQKWGRMCGLVMLLPHGYEGQGPEHSSARLERYLQLCAEQNMQVCVPSTPAQVYHMLRRQALRGMRRPLVVMSPKSLLRHPLAVSSLEELANGTFLPAIGEIDELDPKGVKRVVMCSGKVYYDLLEQRRKNNQHDVAIVRIEQLYPFPHKAMQEVLQQFAHVKDFVWCQEEPLNQGAWYCSQHHFREVIPFGASLRYAGRPASASPAVGYMSVHQKQQQDLVNDALNVE',\n",
       "  'mw': 105062.0},\n",
       " 'b0727': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AFG6',\n",
       "  'gene': 'b0727',\n",
       "  'locus': ['b0727', 'JW0716'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(R)-N(6)-dihydrolipoyl-L-lysyl-[2-oxoglutarate dehydrogenase complex component E2] + succinyl-CoA = (R)-N(6)-(S(8)-succinyldihydrolipoyl)-L-lysyl-[2-oxoglutarate dehydrogenase complex component E2] + CoA',\n",
       "     '2.3.1.61')],\n",
       "   'Function': 'E2 component of the 2-oxoglutarate dehydrogenase (OGDH) complex which catalyzes the second step in the conversion of 2-oxoglutarate to succinyl-CoA and CO(2)',\n",
       "   'Pathway': 'Amino-acid degradation; L-lysine degradation via saccharopine pathway; glutaryl-CoA from L-lysine: step 6/6'},\n",
       "  'seq': 'MSSVDILVPDLPESVADATVATWHKKPGDAVVRDEVLVEIETDKVVLEVPASADGILDAVLEDEGTTVTSRQILGRLREGNSAGKETSAKSEEKASTPAQRQQASLEEQNNDALSPAIRRLLAEHNLDASAIKGTGVGGRLTREDVEKHLAKAPAKESAPAAAAPAAQPALAARSEKRVPMTRLRKRVAERLLEAKNSTAMLTTFNEVNMKPIMDLRKQYGEAFEKRHGIRLGFMSFYVKAVVEALKRYPEVNASIDGDDVVYHNYFDVSMAVSTPRGLVTPVLRDVDTLGMADIEKKIKELAVKGRDGKLTVEDLTGGNFTITNGGVFGSLMSTPIINPPQSAILGMHAIKDRPMAVNGQVEILPMMYLALSYDHRLIDGRESVGFLVTIKELLEDPTRLLLDV',\n",
       "  'mw': 44011.0},\n",
       " 'b0728': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A836',\n",
       "  'gene': 'b0728',\n",
       "  'locus': ['b0728', 'JW0717'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + CoA + succinate = ADP + phosphate + succinyl-CoA',\n",
       "     '6.2.1.5')],\n",
       "   'Function': 'Succinyl-CoA synthetase functions in the citric acid cycle (TCA), coupling the hydrolysis of succinyl-CoA to the synthesis of either ATP or GTP and thus represents the only step of substrate-level phosphorylation in the TCA. The beta subunit provides nucleotide specificity of the enzyme and binds the substrate succinate, while the binding sites for coenzyme A and phosphate are found in the alpha subunit. Can use either ATP or GTP, but prefers ATP. It can also function in the other direction for anabolic purposes, and this may be particularly important for providing succinyl-CoA during anaerobic growth when the oxidative route from 2-oxoglutarate is severely repressed',\n",
       "   'Regulation Activity': \"Exhibits two interesting properties: 'substrate synergism', in which the enzyme is most active for the catalysis of its partial reactions only when all the substrate-binding sites are occupied, and 'catalytic cooperativity' between alternating active sites in the tetramer, whereby the interaction of substrates (particularly ATP) at one site is needed to promote catalysis at the other\",\n",
       "   'Pathway': 'Carbohydrate metabolism; tricarboxylic acid cycle; succinate from succinyl-CoA (ligase route): step 1/1'},\n",
       "  'seq': 'MNLHEYQAKQLFARYGLPAPVGYACTTPREAEEAASKIGAGPWVVKCQVHAGGRGKAGGVKVVNSKEDIRAFAENWLGKRLVTYQTDANGQPVNQILVEAATDIAKELYLGAVVDRSSRRVVFMASTEGGVEIEKVAEETPHLIHKVALDPLTGPMPYQGRELAFKLGLEGKLVQQFTKIFMGLATIFLERDLALIEINPLVITKQGDLICLDGKLGADGNALFRQPDLREMRDQSQEDPREAQAAQWELNYVALDGNIGCMVNGAGLAMGTMDIVKLHGGEPANFLDVGGGATKERVTEAFKIILSDDKVKAVLVNIFGGIVRCDLIADGIIGAVAEVGVNVPVVVRLEGNNAELGAKKLADSGLNIIAAKGLTDAAQQVVAAVEGK',\n",
       "  'mw': 41393.0},\n",
       " 'b0729': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AGE9',\n",
       "  'gene': 'b0729',\n",
       "  'locus': ['b0729', 'JW0718'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + CoA + succinate = ADP + phosphate + succinyl-CoA',\n",
       "     '6.2.1.5')],\n",
       "   'Function': 'Succinyl-CoA synthetase functions in the citric acid cycle (TCA), coupling the hydrolysis of succinyl-CoA to the synthesis of either ATP or GTP and thus represents the only step of substrate-level phosphorylation in the TCA. The alpha subunit of the enzyme binds the substrates coenzyme A and phosphate, while succinate binding and nucleotide specificity is provided by the beta subunit. Can use either ATP or GTP, but prefers ATP. It can also function in the other direction for anabolic purposes, and this may be particularly important for providing succinyl-CoA during anaerobic growth when the oxidative route from 2-oxoglutarate is severely repressed',\n",
       "   'Regulation Activity': \"Exhibits two interesting properties: 'substrate synergism', in which the enzyme is most active for the catalysis of its partial reactions only when all the substrate-binding sites are occupied, and 'catalytic cooperativity' between alternating active sites in the tetramer, whereby the interaction of substrates (particularly ATP) at one site is needed to promote catalysis at the other\",\n",
       "   'Pathway': 'Carbohydrate metabolism; tricarboxylic acid cycle; succinate from succinyl-CoA (ligase route): step 1/1'},\n",
       "  'seq': 'MSILIDKNTKVICQGFTGSQGTFHSEQAIAYGTKMVGGVTPGKGGTTHLGLPVFNTVREAVAATGATASVIYVPAPFCKDSILEAIDAGIKLIITITEGIPTLDMLTVKVKLDEAGVRMIGPNCPGVITPGECKIGIQPGHIHKPGKVGIVSRSGTLTYEAVKQTTDYGFGQSTCVGIGGDPIPGSNFIDILEMFEKDPQTEAIVMIGEIGGSAEEEAAAYIKEHVTKPVVGYIAGVTAPKGKRMGHAGAIIAGGKGTADEKFAALEAAGVKTVRSLADIGEALKTVLK',\n",
       "  'mw': 29777.0},\n",
       " 'b0731': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P54745',\n",
       "  'gene': 'b0731',\n",
       "  'locus': ['b0731', 'JW0720'],\n",
       "  'synonyms': 'hrsA',\n",
       "  'properties': {'Catalytic Activity': [('(2R)-2-O-(alpha-D-mannosyl)-glycerate(out) + N(pros)-phospho-L-histidyl-[protein] = (2R)-2-O-(6-phospho-alpha-D-mannosyl)-glycerate(in) + L-histidyl-[protein]',\n",
       "     '2.7.1.195')],\n",
       "   'Function': 'The phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS), a major carbohydrate active transport system, catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane (PubMed:14645248, PubMed:9063979). This system is involved in mannosyl-D-glycerate transport (PubMed:14645248). Also involved in thermoinduction of ompC (PubMed:9063979)'},\n",
       "  'seq': 'MVLFYRAHWRDYKNDQVRIMMNLTTLTHRDALCLNARFTSREEAIHALTQRLAALGKISSTEQFLEEVYRRESLGPTALGEGLAVPHGKTAAVKEAAFAVATLSEPLQWEGVDGPEAVDLVVLLAIPPNEAGTTHMQLLTALTTRLADDEIRARIQSATTPDELLSALDDKGGTQPSASFSNAPTIVCVTACPAGIAHTYMAAEYLEKAGRKLGVNVYVEKQGANGIEGRLTADQLNSATACIFAAEVAIKESERFNGIPALSVPVAEPIRHAEALIQQALTLKRSDETRTVQQDTQPVKSVKTELKQALLSGISFAVPLIVAGGTVLAVAVLLSQIFGLQDLFNEENSWLWMYRKLGGGLLGILMVPVLAAYTAYSLADKPALAPGFAAGLAANMIGSGFLGAVVGGLIAGYLMRWVKNHLRLSSKFNGFLTFYLYPVLGTLGAGSLMLFVVGEPVAWINNSLTAWLNGLSGSNALLLGAILGFMCSFDLGGPVNKAAYAFCLGAMANGVYGPYAIFASVKMVSAFTVTASTMLAPRLFKEFEIETGKSTWLLGLAGITEGAIPMAIEDPLRVIGSFVLGSMVTGAIVGAMNIGLSTPGAGIFSLFLLHDNGAGGVMAAIGWFGAALVGAAISTAILLMWRRHAVKHGNYLTDGVMP',\n",
       "  'mw': 69668.0},\n",
       " 'b0732': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P54746',\n",
       "  'gene': 'b0732',\n",
       "  'locus': ['b0732', 'JW0721'],\n",
       "  'synonyms': 'ybgG',\n",
       "  'properties': {'Catalytic Activity': [('(2R)-2-O-(6-phospho-alpha-D-mannosyl)-glycerate + H2O = (R)-glycerate + alpha-D-mannose 6-phosphate',\n",
       "     '')],\n",
       "   'Function': 'May hydrolyze 6-phospho-mannosyl-D-glycerate to mannose-6-phosphate and glycerate'},\n",
       "  'seq': 'MKAVSRVHITPHMHWDREWYFTTEESRILLVNNMEEILCRLEQDNEYKYYVLDGQTAILEDYFAVKPENKDRVKKQVEAGKLIIGPWYTQTDTTIVSAESIVRNLMYGMRDCLAFGEPMKIGYLPDSFGMSGQLPHIYNGFGITRTMFWRGCSERHGTDKTEFLWQSSDGSEVTAQVLPLGYAIGKYLPADENGLRKRLDSYFDVLEKASVTKEILLPNGHDQMPLQQNIFEVMDKLREIYPQRKFVMSRFEEVFEKIEAQRDNLATLKGEFIDGKYMRVHRTIGSTRMDIKIAHARIENKIVNLLEPLATLAWTLGFEYHHGLLEKMWKEILKNHAHDSIGCCCSDKVHREIVARFELAEDMADNLIRFYMRKIADNMPQSDADKLVLFNLMPWPREEVINTTVRLRASQFNLRDDRGQPVPYFIRHAREIDPGLIDRQIVHYGNYDPFMEFDIQINQIVPSMGYRTLYIEANQPGNVIAAKSDAEGILENAFWQIALNEDGSLQLVDKDSGVRYDRVLQIEESSDDGDEYDYSPAKEEWVITAANAKPQCDIIHEAWQSRAVIRYDMAVPLNLSERSARQSTGRVGVVLVVTLSHNSRRIDVDINLDNQADDHRLRVLVPTPFNTDSVLADTQFGSLTRPVNDSAMNNWQQEGWKEAPVPVWNMLNYVALQEGRNGMAVFSEGLREFEVIGEEKKTFAITLLRGVGLLGKEDLLLRPGRPSGIKMPVPDSQLRGLLSCRLSLLSYTGTPTAAGVAQQARAWLTPVQCYNKIPWDVMKLNKAGFNVPESYSLLKMPPVGCLISALKKAEDRQEVILRLFNPAESATCDATVAFSREVISCSETMMDEHITTEENQGSNLSGPFLPGQSRTFSYRLA',\n",
       "  'mw': 100015.0},\n",
       " 'b0733': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ABJ9',\n",
       "  'gene': 'b0733',\n",
       "  'locus': ['b0733', 'JW0722'],\n",
       "  'synonyms': 'cyd-1',\n",
       "  'properties': {'Catalytic Activity': [('2 a ubiquinol + 4 H(+)(in) + O2(in) = 2 a ubiquinone + 4 H(+)(out) + 2 H2O(in)',\n",
       "     '7.1.1.7')],\n",
       "   'Function': 'A terminal oxidase that produces a proton motive force by the vectorial transfer of protons across the inner membrane. It is the component of the aerobic respiratory chain of E.coli that predominates when cells are grown at low aeration. Generates a proton motive force using protons and electrons from opposite sides of the membrane to generate H(2)O, transferring 1 proton/electron',\n",
       "   'Regulation Activity': '90% inhibited by cyanide and 2-heptyl-4-hydroxyquinoline N-oxide, at 1 mM and 40 uM respectively',\n",
       "   'Pathway': 'Energy metabolism; oxidative phosphorylation'},\n",
       "  'seq': 'MLDIVELSRLQFALTAMYHFLFVPLTLGMAFLLAIMETVYVLSGKQIYKDMTKFWGKLFGINFALGVATGLTMEFQFGTNWSYYSHYVGDIFGAPLAIEGLMAFFLESTFVGLFFFGWDRLGKVQHMCVTWLVALGSNLSALWILVANGWMQNPIASDFNFETMRMEMVSFSELVLNPVAQVKFVHTVASGYVTGAMFILGISAWYMLKGRDFAFAKRSFAIAASFGMAAVLSVIVLGDESGYEMGDVQKTKLAAIEAEWETQPAPAAFTLFGIPDQEEETNKFAIQIPYALGIIATRSVDTPVIGLKELMVQHEERIRNGMKAYSLLEQLRSGSTDQAVRDQFNSMKKDLGYGLLLKRYTPNVADATEAQIQQATKDSIPRVAPLYFAFRIMVACGFLLLAIIALSFWSVIRNRIGEKKWLLRAALYGIPLPWIAVEAGWFVAEYGRQPWAIGEVLPTAVANSSLTAGDLIFSMVLICGLYTLFLVAELFLMFKFARLGPSSLKTGRYHFEQSSTTTQPAR',\n",
       "  'mw': 58205.0},\n",
       " 'b0734': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ABK2',\n",
       "  'gene': 'b0734',\n",
       "  'locus': ['b0734', 'JW0723'],\n",
       "  'synonyms': 'cyd-2',\n",
       "  'properties': {'Catalytic Activity': [('2 a ubiquinol + 4 H(+)(in) + O2(in) = 2 a ubiquinone + 4 H(+)(out) + 2 H2O(in)',\n",
       "     '7.1.1.7')],\n",
       "   'Function': 'A terminal oxidase that produces a proton motive force by the vectorial transfer of protons across the inner membrane. It is the component of the aerobic respiratory chain of E.coli that predominates when cells are grown at low aeration. Generates a proton motive force using protons and electrons from opposite sides of the membrane to generate H(2)O, transferring 1 proton/electron',\n",
       "   'Regulation Activity': '90% inhibited by cyanide and 2-heptyl-4-hydroxyquinoline N-oxide, at 1 mM and 40 uM respectively',\n",
       "   'Pathway': 'Energy metabolism; oxidative phosphorylation'},\n",
       "  'seq': 'MIDYEVLRFIWWLLVGVLLIGFAVTDGFDMGVGMLTRFLGRNDTERRIMINSIAPHWDGNQVWLITAGGALFAAWPMVYAAAFSGFYVAMILVLASLFFRPVGFDYRSKIEETRWRNMWDWGIFIGSFVPPLVIGVAFGNLLQGVPFNVDEYLRLYYTGNFFQLLNPFGLLAGVVSVGMIITQGATYLQMRTVGELHLRTRATAQVAALVTLVCFALAGVWVMYGIDGYVVKSTMDHYAASNPLNKEVVREAGAWLVNFNNTPILWAIPALGVVLPLLTILTARMDKAAWAFVFSSLTLACIILTAGIAMFPFVMPSSTMMNASLTMWDATSSQLTLNVMTWVAVVLVPIILLYTAWCYWKMFGRITKEDIERNTHSLY',\n",
       "  'mw': 42453.0},\n",
       " 'b0750': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P11458',\n",
       "  'gene': 'b0750',\n",
       "  'locus': ['b0750', 'JW0733'],\n",
       "  'synonyms': 'nicA',\n",
       "  'properties': {'Catalytic Activity': [('dihydroxyacetone phosphate + iminosuccinate = H(+) + 2 H2O + phosphate + quinolinate',\n",
       "     '2.5.1.72')],\n",
       "   'Function': 'Catalyzes the condensation of iminoaspartate with dihydroxyacetone phosphate to form quinolinate',\n",
       "   'Regulation Activity': 'Activity is greater under oxic conditions and is regulated by the formation of a reversible disulfide-bond between Cys-291 and Cys-294',\n",
       "   'Pathway': 'Cofactor biosynthesis; NAD(+) biosynthesis; quinolinate from iminoaspartate: step 1/1'},\n",
       "  'seq': 'MSVMFDPDTAIYPFPPKPTPLSIDEKAYYREKIKRLLKERNAVMVAHYYTDPEIQQLAEETGGCISDSLEMARFGAKHPASTLLVAGVRFMGETAKILSPEKTILMPTLQAECSLDLGCPVEEFNAFCDAHPDRTVVVYANTSAAVKARADWVVTSSIAVELIDHLDSLGEKIIWAPDKHLGRYVQKQTGGDILCWQGACIVHDEFKTQALTRLQEEYPDAAILVHPESPQAIVDMADAVGSTSQLIAAAKTLPHQRLIVATDRGIFYKMQQAVPDKELLEAPTAGEGATCRSCAHCPWMAMNGLQAIAEALEQEGSNHEVHVDERLRERALVPLNRMLDFAATLRG',\n",
       "  'mw': 38241.0},\n",
       " 'b0751': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AFK2',\n",
       "  'gene': 'b0751',\n",
       "  'locus': ['b0751', 'JW0734'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Required for nicotinamide riboside transport across the inner membrane'},\n",
       "  'seq': 'MDFFSVQNILVHIPIGAGGYDLSWIEAVGTIAGLLCIGLASLEKISNYFFGLINVTLFGIIFFQIQLYASLLLQVFFFAANIYGWYAWSRQTSQNEAELKIRWLPLPKALSWLAVCVVSIGLMTVFINPVFAFLTRVAVMIMQALGLQVVMPELQPDAFPFWDSCMMVLSIVAMILMTRKYVENWLLWVIINVISVVIFALQGVYAMSLEYIILTFIALNGSRMWINSARERGSRALSH',\n",
       "  'mw': 26996.0},\n",
       " 'b0752': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P75757',\n",
       "  'gene': 'b0752',\n",
       "  'locus': ['b0752', 'JW0735'],\n",
       "  'synonyms': 'ybgR',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Involved in zinc efflux across the cytoplasmic membrane, thus reducing zinc accumulation in the cytoplasm and rendering bacteria more resistant to zinc. It may contribute to zinc homeostasis at low concentrations of zinc, whereas ZntA is required for growth at more toxic concentrations'},\n",
       "  'seq': 'MAHSHSHTSSHLPEDNNARRLLYAFGVTAGFMLVEVVGGFLSGSLALLADAGHMLTDTAALLFALLAVQFSRRPPTIRHTFGWLRLTTLAAFVNAIALVVITILIVWEAIERFRTPRPVEGGMMMAIAVAGLLANILSFWLLHHGSEEKNLNVRAAALHVLGDLLGSVGAIIAALIIIWTGWTPADPILSILVSLLVLRSAWRLLKDSVNELLEGAPVSLDIAELKRRMCREIPEVRNVHHVHVWMVGEKPVMTLHVQVIPPHDHDALLDQIQHYLMDHYQIEHATIQMEYQPCHGPDCHLNEGVSGHSHHHH',\n",
       "  'mw': 34678.0},\n",
       " 'b0754': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AB91',\n",
       "  'gene': 'b0754',\n",
       "  'locus': ['b0754', 'JW0737'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('D-erythrose 4-phosphate + H2O + phosphoenolpyruvate = 7-phospho-2-dehydro-3-deoxy-D-arabino-heptonate + phosphate',\n",
       "     '2.5.1.54')],\n",
       "   'Function': 'Stereospecific condensation of phosphoenolpyruvate (PEP) and D-erythrose-4-phosphate (E4P) giving rise to 3-deoxy-D-arabino-heptulosonate-7-phosphate (DAHP)',\n",
       "   'Pathway': 'Metabolic intermediate biosynthesis; chorismate biosynthesis; chorismate from D-erythrose 4-phosphate and phosphoenolpyruvate: step 1/7'},\n",
       "  'seq': 'MNYQNDDLRIKEIKELLPPVALLEKFPATENAANTVAHARKAIHKILKGNDDRLLVVIGPCSIHDPVAAKEYATRLLALREELKDELEIVMRVYFEKPRTTVGWKGLINDPHMDNSFQINDGLRIARKLLLDINDSGLPAAGEFLDMITPQYLADLMSWGAIGARTTESQVHRELASGLSCPVGFKNGTDGTIKVAIDAINAAGAPHCFLSVTKWGHSAIVNTSGNGDCHIILRGGKEPNYSAKHVAEVKEGLNKAGLPAQVMIDFSHANSSKQFKKQMDVCADVCQQIAGGEKAIIGVMVESHLVEGNQSLESGEPLAYGKSITDACIGWEDTDALLRQLANAVKARRG',\n",
       "  'mw': 38010.0},\n",
       " 'b0755': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P62707',\n",
       "  'gene': 'b0755',\n",
       "  'locus': ['b0755', 'JW0738'],\n",
       "  'synonyms': 'gpm pgm pgmA',\n",
       "  'properties': {'Catalytic Activity': [('(2R)-2-phosphoglycerate = (2R)-3-phosphoglycerate',\n",
       "     '5.4.2.11')],\n",
       "   'Function': 'Catalyzes the interconversion of 2-phosphoglycerate and 3-phosphoglycerate',\n",
       "   'Regulation Activity': 'Strongly inhibited by vanadate',\n",
       "   'Pathway': 'Carbohydrate degradation; glycolysis; pyruvate from D-glyceraldehyde 3-phosphate: step 3/5'},\n",
       "  'seq': 'MAVTKLVLVRHGESQWNKENRFTGWYDVDLSEKGVSEAKAAGKLLKEEGYSFDFAYTSVLKRAIHTLWNVLDELDQAWLPVEKSWKLNERHYGALQGLNKAETAEKYGDEQVKQWRRGFAVTPPELTKDDERYPGHDPRYAKLSEKELPLTESLALTIDRVIPYWNETILPRMKSGERVIIAAHGNSLRALVKYLDNMSEEEILELNIPTGVPLVYEFDENFKPLKRYYLGNADEIAAKAAAVANQGKAK',\n",
       "  'mw': 28556.0},\n",
       " 'b0756': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A9C3',\n",
       "  'gene': 'b0756',\n",
       "  'locus': ['b0756', 'JW0739'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('alpha-D-glucose = beta-D-glucose',\n",
       "     '5.1.3.3')],\n",
       "   'Function': 'Mutarotase converts alpha-aldose to the beta-anomer. It is active on D-glucose, L-arabinose, D-xylose, D-galactose, maltose and lactose',\n",
       "   'Pathway': 'Carbohydrate metabolism; hexose metabolism'},\n",
       "  'seq': 'MLNETPALAPDGQPYRLLTLRNNAGMVVTLMDWGATLLSARIPLSDGSVREALLGCASPECYQDQAAFLGASIGRYANRIANSRYTFDGETVTLSPSQGVNQLHGGPEGFDKRRWQIVNQNDRQVLFALSSDDGDQGFPGNLGATVQYRLTDDNRISITYRATVDKPCPVNMTNHVYFNLDGEQSDVRNHKLQILADEYLPVDEGGIPHDGLKSVAGTSFDFRSAKIIASEFLADDDQRKVKGYDHAFLLQAKGDGKKVAAHVWSADEKLQLKVYTTAPALQFYSGNFLGGTPSRGTEPYADWQGLALESEFLPDSPNHPEWPQPDCFLRPGEEYSSLTEYQFIAE',\n",
       "  'mw': 38190.0},\n",
       " 'b0757': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A6T3',\n",
       "  'gene': 'b0757',\n",
       "  'locus': ['b0757', 'JW0740'],\n",
       "  'synonyms': 'galA',\n",
       "  'properties': {'Catalytic Activity': [('alpha-D-galactose + ATP = ADP + alpha-D-galactose 1-phosphate + H(+)',\n",
       "     '2.7.1.6')],\n",
       "   'Function': 'Catalyzes the transfer of the gamma-phosphate of ATP to D-galactose to form alpha-D-galactose-1-phosphate (Gal-1-P). To a lesser extent, is also able to phosphorylate 2-deoxy-D-galactose and D-galactosamine. Is not able to use D-galacturonic acid, D-talose, L-altrose, and L-glucose as substrates',\n",
       "   'Pathway': 'Carbohydrate metabolism; galactose metabolism'},\n",
       "  'seq': 'MSLKEKTQSLFANAFGYPATHTIQAPGRVNLIGEHTDYNDGFVLPCAIDYQTVISCAPRDDRKVRVMAADYENQLDEFSLDAPIVAHENYQWANYVRGVVKHLQLRNNSFGGVDMVISGNVPQGAGLSSSASLEVAVGTVLQQLYHLPLDGAQIALNGQEAENQFVGCNCGIMDQLISALGKKDHALLIDCRSLGTKAVSMPKGVAVVIINSNFKRTLVGSEYNTRREQCETGARFFQQPALRDVTIEEFNAVAHELDPIVAKRVRHILTENARTVEAASALEQGDLKRMGELMAESHASMRDDFEITVPQIDTLVEIVKAVIGDKGGVRMTGGGFGGCIVALIPEELVPAVQQAVAEQYEAKTGIKETFYVCKPSQGAGQC',\n",
       "  'mw': 41442.0},\n",
       " 'b0758': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P09148',\n",
       "  'gene': 'b0758',\n",
       "  'locus': ['b0758', 'JW0741'],\n",
       "  'synonyms': 'galB',\n",
       "  'properties': {'Catalytic Activity': [('alpha-D-galactose 1-phosphate + UDP-alpha-D-glucose = alpha-D-glucose 1-phosphate + UDP-alpha-D-galactose',\n",
       "     '2.7.7.12')],\n",
       "   'Pathway': 'Carbohydrate metabolism; galactose metabolism'},\n",
       "  'seq': 'MTQFNPVDHPHRRYNPLTGQWILVSPHRAKRPWQGAQETPAKQVLPAHDPDCFLCAGNVRVTGDKNPDYTGTYVFTNDFAALMSDTPDAPESHDPLMRCQSARGTSRVICFSPDHSKTLPELSVAALTEIVKTWQEQTAELGKTYPWVQVFENKGAAMGCSNPHPHGQIWANSFLPNEAEREDRLQKEYFAEQKSPMLVDYVQRELADGSRTVVETEHWLAVVPYWAAWPFETLLLPKAHVLRITDLTDAQRSDLALALKKLTSRYDNLFQCSFPYSMGWHGAPFNGEENQHWQLHAHFYPPLLRSATVRKFMVGYEMLAETQRDLTAEQAAERLRAVSDIHFRESGV',\n",
       "  'mw': 39646.0},\n",
       " 'b0759': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P09147',\n",
       "  'gene': 'b0759',\n",
       "  'locus': ['b0759', 'JW0742'],\n",
       "  'synonyms': 'galD',\n",
       "  'properties': {'Catalytic Activity': [('UDP-alpha-D-glucose = UDP-alpha-D-galactose',\n",
       "     '5.1.3.2')],\n",
       "   'Function': 'Involved in the metabolism of galactose. Catalyzes the conversion of UDP-galactose (UDP-Gal) to UDP-glucose (UDP-Glc) through a mechanism involving the transient reduction of NAD. It is only active on UDP-galactose and UDP-glucose',\n",
       "   'Regulation Activity': 'Inhibited by UDP-phenol and NaBH3CN',\n",
       "   'Pathway': 'Carbohydrate metabolism; galactose metabolism'},\n",
       "  'seq': 'MRVLVTGGSGYIGSHTCVQLLQNGHDVIILDNLCNSKRSVLPVIERLGGKHPTFVEGDIRNEALMTEILHDHAIDTVIHFAGLKAVGESVQKPLEYYDNNVNGTLRLISAMRAANVKNFIFSSSATVYGDQPKIPYVESFPTGTPQSPYGKSKLMVEQILTDLQKAQPDWSIALLRYFNPVGAHPSGDMGEDPQGIPNNLMPYIAQVAVGRRDSLAIFGNDYPTEDGTGVRDYIHVMDLADGHVVAMEKLANKPGVHIYNLGAGVGNSVLDVVNAFSKACGKPVNYHFAPRREGDLPAYWADASKADRELNWRVTRTLDEMAQDTWHWQSRHPQGYPD',\n",
       "  'mw': 37265.0},\n",
       " 'b0763': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37329',\n",
       "  'gene': 'b0763',\n",
       "  'locus': ['b0763', 'JW0746'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the ABC transporter complex ModABC involved in the transport of molybdenum into the cell (PubMed:8576221). Binds molybdate with high affinity in vitro and with a similar affinity in vivo (PubMed:21861186, PubMed:28150901, PubMed:9545596, PubMed:8576221, PubMed:21784948, PubMed:8409926). Binds tungstate with high affinity in vitro (PubMed:28150901, PubMed:9545596, PubMed:8576221, PubMed:21784948). Binds unnatural anion perrhenate with high affinity in vitro (PubMed:21861186). Does not bind sulfate, phosphate, arsenate, selenate, chlorate, metavanadate, nitrate, perchlorate, permanganate or carbonate (PubMed:9545596, PubMed:28150901, PubMed:8576221)'},\n",
       "  'seq': 'MARKWLNLFAGAALSFAVAGNALADEGKITVFAAASLTNAMQDIATQFKKEKGVDVVSSFASSSTLARQIEAGAPADLFISADQKWMDYAVDKKAIDTATRQTLLGNSLVVVAPKASVQKDFTIDSKTNWTSLLNGGRLAVGDPEHVPAGIYAKEALQKLGAWDTLSPKLAPAEDVRGALALVERNEAPLGIVYGSDAVASKGVKVVATFPEDSHKKVEYPVAVVEGHNNATVKAFYDYLKGPQAAEIFKRYGFTIK',\n",
       "  'mw': 27364.0},\n",
       " 'b0764': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AF01',\n",
       "  'gene': 'b0764',\n",
       "  'locus': ['b0764', 'JW0747'],\n",
       "  'synonyms': 'chlJ',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the binding-protein-dependent transport system for molybdenum; probably responsible for the translocation of the substrate across the membrane'},\n",
       "  'seq': 'MILTDPEWQAVLLSLKVSSLAVLFSLPFGIFFAWLLVRCTFPGKALLDSVLHLPLVLPPVVVGYLLLVSMGRRGFIGERLYDWFGITFAFSWRGAVLAAAVMSFPLMVRAIRLALEGVDVKLEQAARTLGAGRWRVFFTITLPLTLPGIIVGTVLAFARSLGEFGATITFVSNIPGETRTIPSAMYTLIQTPGGESGAARLCIISIALAMISLLISEWLARISRERAGR',\n",
       "  'mw': 24939.0},\n",
       " 'b0765': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P09833',\n",
       "  'gene': 'b0765',\n",
       "  'locus': ['b0765', 'JW0748'],\n",
       "  'synonyms': 'chlD narD',\n",
       "  'properties': {'Catalytic Activity': [('ATP + H2O + molybdate(out) = ADP + H(+) + molybdate(in) + phosphate',\n",
       "     '7.3.2.5')],\n",
       "   'Function': 'Part of the ABC transporter complex ModABC involved in molybdenum import. Responsible for energy coupling to the transport system'},\n",
       "  'seq': 'MLELNFSQTLGNHCLTINETLPANGITAIFGVSGAGKTSLINAISGLTRPQKGRIVLNGRVLNDAEKGICLTPEKRRVGYVFQDARLFPHYKVRGNLRYGMSKSMVDQFDKLVALLGIEPLLDRLPGSLSGGEKQRVAIGRALLTAPELLLLDEPLASLDIPRKRELLPYLQRLTREINIPMLYVSHSLDEILHLADRVMVLENGQVKAFGALEEVWGSSVMNPWLPKEQQSSILKVTVLEHHPHYAMTALALGDQHLWVNKLDEPLQAALRIRIQASDVSLVLQPPQQTSIRNVLRAKVVNSYDDNGQVEVELEVGGKTLWARISPWARDELAIKPGLWLYAQIKSVSITA',\n",
       "  'mw': 39102.0},\n",
       " 'b0767': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P52697',\n",
       "  'gene': 'b0767',\n",
       "  'locus': ['b0767', 'JW0750'],\n",
       "  'synonyms': 'ybhE',\n",
       "  'properties': {'Catalytic Activity': [('6-phospho-D-glucono-1,5-lactone + H2O = 6-phospho-D-gluconate + H(+)',\n",
       "     '3.1.1.31')],\n",
       "   'Function': 'Catalyzes the hydrolysis of 6-phosphogluconolactone to 6-phosphogluconate',\n",
       "   'Pathway': 'Carbohydrate degradation; pentose phosphate pathway; D-ribulose 5-phosphate from D-glucose 6-phosphate (oxidative stage): step 2/3'},\n",
       "  'seq': 'MKQTVYIASPESQQIHVWNLNHEGALTLTQVVDVPGQVQPMVVSPDKRYLYVGVRPEFRVLAYRIAPDDGALTFAAESALPGSPTHISTDHQGQFVFVGSYNAGNVSVTRLEDGLPVGVVDVVEGLDGCHSANISPDNRTLWVPALKQDRICLFTVSDDGHLVAQDPAEVTTVEGAGPRHMVFHPNEQYAYCVNELNSSVDVWELKDPHGNIECVQTLDMMPENFSDTRWAADIHITPDGRHLYACDRTASLITVFSVSEDGSVLSKEGFQPTETQPRGFNVDHSGKYLIAAGQKSHHISVYEIVGEQGLLHEKGRYAVGQGPMWVVVNAH',\n",
       "  'mw': 36308.0},\n",
       " 'b0774': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P12995',\n",
       "  'gene': 'b0774',\n",
       "  'locus': ['b0774', 'JW0757'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(8S)-8-amino-7-oxononanoate + S-adenosyl-L-methionine = (7R,8S)-7,8-diammoniononanoate + S-adenosyl-4-methylsulfanyl-2-oxobutanoate',\n",
       "     '2.6.1.62')],\n",
       "   'Function': 'Catalyzes the transfer of the alpha-amino group from S-adenosyl-L-methionine (SAM) to 7-keto-8-aminopelargonic acid (KAPA) to form 7,8-diaminopelargonic acid (DAPA) (PubMed:1092681, PubMed:1092682). It is the only animotransferase known to utilize SAM as an amino donor (PubMed:1092681, PubMed:1092682). Complements a bioU deletion in Synechocystis PCC 6803 (PubMed:32042199)',\n",
       "   'Regulation Activity': 'Inhibited by amiclenomycin. S-adenosyl-L-(2-hydroxy-4-methylthio)butyric acid and adenosine are competitive inhibitors with SAM and uncompetitive inhibitors with KAPA as substrates (PubMed:1092682). S-adenosyl-L-ethionine, adenine and 8-keto-7-aminopelargonic acid are non-competitive inhibitors with both substrates (PubMed:1092682)',\n",
       "   'Pathway': 'Cofactor biosynthesis; biotin biosynthesis; 7,8-diaminononanoate from 8-amino-7-oxononanoate (SAM route): step 1/1'},\n",
       "  'seq': 'MTTDDLAFDQRHIWHPYTSMTSPLPVYPVVSAEGCELILSDGRRLVDGMSSWWAAIHGYNHPQLNAAMKSQIDAMSHVMFGGITHAPAIELCRKLVAMTPQPLECVFLADSGSVAVEVAMKMALQYWQAKGEARQRFLTFRNGYHGDTFGAMSVCDPDNSMHSLWKGYLPENLFAPAPQSRMDGEWDERDMVGFARLMAAHRHEIAAVIIEPIVQGAGGMRMYHPEWLKRIRKICDREGILLIADEIATGFGRTGKLFACEHAEIAPDILCLGKALTGGTMTLSATLTTREVAETISNGEAGCFMHGPTFMGNPLACAAANASLAILESGDWQQQVADIEVQLREQLAPARDAEMVADVRVLGAIGVVETTHPVNMAALQKFFVEQGVWIRPFGKLIYLMPPYIILPQQLQRLTAAVNRAVQDETFFCQ',\n",
       "  'mw': 47336.0},\n",
       " 'b0775': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P12996',\n",
       "  'gene': 'b0775',\n",
       "  'locus': ['b0775', 'JW0758'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [(\"(4R,5S)-dethiobiotin + [sulfur carrier]-SH + 2 reduced [2Fe-2S]-[ferredoxin] + 2 S-adenosyl-L-methionine = 2 5'-deoxyadenosine + [sulfur carrier]-H + biotin + 2 L-methionine + 2 oxidized [2Fe-2S]-[ferredoxin]\",\n",
       "     '2.8.1.6')],\n",
       "   'Function': 'Catalyzes the conversion of dethiobiotin (DTB) to biotin by the insertion of a sulfur atom into dethiobiotin via a radical-based mechanism',\n",
       "   'Regulation Activity': \"Is physiologically inhibited by accumulation of the reaction product 5'-deoxyadenosine\",\n",
       "   'Pathway': 'Cofactor biosynthesis; biotin biosynthesis; biotin from 7,8-diaminononanoate: step 2/2'},\n",
       "  'seq': 'MAHRPRWTLSQVTELFEKPLLDLLFEAQQVHRQHFDPRQVQVSTLLSIKTGACPEDCKYCPQSSRYKTGLEAERLMEVEQVLESARKAKAAGSTRFCMGAAWKNPHERDMPYLEQMVQGVKAMGLEACMTLGTLSESQAQRLANAGLDYYNHNLDTSPEFYGNIITTRTYQERLDTLEKVRDAGIKVCSGGIVGLGETVKDRAGLLLQLANLPTPPESVPINMLVKVKGTPLADNDDVDAFDFIRTIAVARIMMPTSYVRLSAGREQMNEQTQAMCFMAGANSIFYGCKLLTTPNPEEDKDLQLFRKLGLNPQQTAVLAGDNEQQQRLEQALMTPDTDEYYNAAAL',\n",
       "  'mw': 38648.0},\n",
       " 'b0776': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P12998',\n",
       "  'gene': 'b0776',\n",
       "  'locus': ['b0776', 'JW0759'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('6-carboxyhexanoyl-[ACP] + H(+) + L-alanine = (8S)-8-amino-7-oxononanoate + CO2 + holo-[ACP]',\n",
       "     '2.3.1.47')],\n",
       "   'Function': 'Catalyzes the decarboxylative condensation of pimeloyl-[acyl-carrier protein] and L-alanine to produce 8-amino-7-oxononanoate (AON), [acyl-carrier protein], and carbon dioxide. Can also use pimeloyl-CoA instead of pimeloyl-ACP as substrate, but it is believed that pimeloyl-ACP rather than pimeloyl-CoA is the physiological substrate of BioF',\n",
       "   'Regulation Activity': 'Competitively inhibited by D-alanine, 8-amino-7-hydroxy-8-phosphonononanoic acid and 2-amino-3-hydroxy-2-methylnonadioic acid',\n",
       "   'Pathway': 'Cofactor biosynthesis; biotin biosynthesis'},\n",
       "  'seq': 'MSWQEKINAALDARRAADALRRRYPVAQGAGRWLVADDRQYLNFSSNDYLGLSHHPQIIRAWQQGAEQFGIGSGGSGHVSGYSVVHQALEEELAEWLGYSRALLFISGFAANQAVIAAMMAKEDRIAADRLSHASLLEAASLSPSQLRRFAHNDVTHLARLLASPCPGQQMVVTEGVFSMDGDSAPLAEIQQVTQQHNGWLMVDDAHGTGVIGEQGRGSCWLQKVKPELLVVTFGKGFGVSGAAVLCSSTVADYLLQFARHLIYSTSMPPAQAQALRASLAVIRSDEGDARREKLAALITRFRAGVQDLPFTLADSCSAIQPLIVGDNSRALQLAEKLRQQGCWVTAIRPPTVPAGTARLRLTLTAAHEMQDIDRLLEVLHGNG',\n",
       "  'mw': 41594.0},\n",
       " 'b0778': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P13000',\n",
       "  'gene': 'b0778',\n",
       "  'locus': ['b0778', 'JW0761'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(7R,8S)-7,8-diammoniononanoate + ATP + CO2 = (4R,5S)-dethiobiotin + ADP + 3 H(+) + phosphate',\n",
       "     '6.3.3.3')],\n",
       "   'Function': 'Catalyzes a mechanistically unusual reaction, the ATP-dependent insertion of CO2 between the N7 and N8 nitrogen atoms of 7,8-diaminopelargonic acid (DAPA, also called 7,8-diammoniononanoate) to form a ureido ring. Only CTP can partially replace ATP while diaminobiotin is only 37% as effective as 7,8-diaminopelargonic acid (PubMed:4892372, PubMed:4921568). In another study both CTP and GTP (but not ITP, TTP or UTP) can partially replace ATP (PubMed:25801336)',\n",
       "   'Pathway': 'Cofactor biosynthesis; biotin biosynthesis; biotin from 7,8-diaminononanoate: step 1/2'},\n",
       "  'seq': 'MSKRYFVTGTDTEVGKTVASCALLQAAKAAGYRTAGYKPVASGSEKTPEGLRNSDALALQRNSSLQLDYATVNPYTFAEPTSPHIISAQEGRPIESLVMSAGLRALEQQADWVLVEGAGGWFTPLSDTFTFADWVTQEQLPVILVVGVKLGCINHAMLTAQVIQHAGLTLAGWVANDVTPPGKRHAEYMTTLTRMIPAPLLGEIPWLAENPENAATGKYINLALL',\n",
       "  'mw': 24140.0},\n",
       " 'b0789': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AA84',\n",
       "  'gene': 'b0789',\n",
       "  'locus': ['b0789', 'JW0772'],\n",
       "  'synonyms': 'ybhO',\n",
       "  'properties': {'Catalytic Activity': [(\"2 1,2-diacyl-sn-glycero-3-phospho-(1'-sn-glycerol) = a cardiolipin + glycerol\",\n",
       "     '')],\n",
       "   'Function': 'Catalyzes the phosphatidyl group transfer from one phosphatidylglycerol molecule to another to form cardiolipin (CL) (diphosphatidylglycerol) and glycerol. Can also catalyze phosphatidyl group transfer to water to form phosphatidate',\n",
       "   'Regulation Activity': 'Activated by phosphate. Inhibited by cardiolipin and phosphatidate'},\n",
       "  'seq': 'MKCSWREGNKIQLLENGEQYYPAVFKAIGEAQERIILETFIWFEDDVGKQLHAALLAAAQRGVKAEVLLDGYGSPDLSDEFVNELTAAGVVFRYYDPRPRLFGMRTNVFRRMHRKIVVIDARIAFIGGLNYSAEHMSSYGPEAKQDYAVRLEGPIVEDILQFELENLPGQSAARRWWRRHHKAEENRQPGEAQVLLVWRDNEEHRDDIERHYLKMLTQARREVIIANAYFFPGYRFLHALRKAARRGVRIKLIIQGEPDMPIVRVGARLLYNYLVKGGVQVFEYRRRPLHGKVALMDDHWATVGSSNLDPLSLSLNLEANVIIHDRHFNQTLRDNLNGIIAADCQQVDETMLPKRTWWNLTKSVLAFHFLRHFPALVGWLPAHTPRLAQVDPPAQPTMETQDRVETENTGVKP',\n",
       "  'mw': 47634.0},\n",
       " 'b0805': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P75780',\n",
       "  'gene': 'b0805',\n",
       "  'locus': ['b0805', 'JW0790'],\n",
       "  'synonyms': 'ybiL',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Involved in the active transport across the outer membrane of iron complexed with catecholate siderophores such as dihydroxybenzoylserine and dihydroxybenzoate. It derives its energy for transport by interacting with the trans-periplasmic membrane protein TonB. Can also transport catechol-substituted cephalosporins. Receptor for microcins M, H47 and E492'},\n",
       "  'seq': 'MENNRNFPARQFHSLTFFAGLCIGITPVAQALAAEGQTNADDTLVVEASTPSLYAPQQSADPKFSRPVADTTRTMTVISEQVIKDQGATNLTDALKNVPGVGAFFAGENGNSTTGDAIYMRGADTSNSIYIDGIRDIGSVSRDTFNTEQVEVIKGPSGTDYGRSAPTGSINMISKQPRNDSGIDASASIGSAWFRRGTLDVNQVIGDTTAVRLNVMGEKTHDAGRDKVKNERYGVAPSVAFGLGTANRLYLNYLHVTQHNTPDGGIPTIGLPGYSAPSAGTAALNHSGKVDTHNFYGTDSDYDDSTTDTATMRFEHDINDNTTIRNTTRWSRVKQDYLMTAIMGGASNITQPTSDVNSWTWSRTANTKDVSNKILTNQTNLTSTFYTGSIGHDVSTGVEFTRETQTNYGVNPVTLPAVNIYHPDSSIHPGGLTRNGANANGQTDTFAIYAFDTLQITRDFELNGGIRLDNYHTEYDSATACGGSGRGAITCPTGVAKGSPVTTVDTAKSGNLMNWKAGALYHLTENGNVYINYAVSQQPPGGNNFALAQSGSGNSANRTDFKPQKANTSEIGTKWQVLDKRLLLTAALFRTDIENEVEQNDDGTYSQYGKKRVEGYEISVAGNITPAWQVIGGYTQQKATIKNGKDVAQDGSSSLPYTPEHAFTLWSQYQATDDISVGAGARYIGSMHKGSDGAVGTPAFTEGYWVADAKLGYRVNRNLDFQLNVYNLFDTDYVASINKSGYRYHPGEPRTFLLTANMHF',\n",
       "  'mw': 81960.0},\n",
       " 'b0809': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P10346',\n",
       "  'gene': 'b0809',\n",
       "  'locus': ['b0809', 'JW0794'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the binding-protein-dependent transport system for glutamine. Probably responsible for energy coupling to the transport system'},\n",
       "  'seq': 'MIEFKNVSKHFGPTQVLHNIDLNIAQGEVVVIIGPSGSGKSTLLRCINKLEEITSGDLIVDGLKVNDPKVDERLIRQEAGMVFQQFYLFPHLTALENVMFGPLRVRGANKEEAEKLARELLAKVGLAERAHHYPSELSGGQQQRVAIARALAVKPKMMLFDEPTSALDPELRHEVLKVMQDLAEEGMTMVIVTHEIGFAEKVASRLIFIDKGRIAEDGNPQVLIKNPPSQRLQEFLQHVS',\n",
       "  'mw': 26731.0},\n",
       " 'b0810': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AEQ6',\n",
       "  'gene': 'b0810',\n",
       "  'locus': ['b0810', 'JW0795'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the binding-protein-dependent transport system for glutamine; probably responsible for the translocation of the substrate across the membrane'},\n",
       "  'seq': 'MQFDWSAIWPAIPLLIEGAKMTLWISVLGLAGGLVIGLLAGFARTFGGWIANHVALVFIEVIRGTPIVVQVMFIYFALPMAFNDLRIDPFTAAVVTIMINSGAYIAEITRGAVLSIHKGFREAGLALGLSRWETIRYVILPLALRRMLPPLGNQWIISIKDTSLFIVIGVAELTRQGQEIIAGNFRALEIWSAVAVFYLIITLVLSFILRRLERRMKIL',\n",
       "  'mw': 24364.0},\n",
       " 'b0811': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AEQ3',\n",
       "  'gene': 'b0811',\n",
       "  'locus': ['b0811', 'JW0796'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Involved in a glutamine-transport system GlnHPQ'},\n",
       "  'seq': 'MKSVLKVSLAALTLAFAVSSHAADKKLVVATDTAFVPFEFKQGDKYVGFDVDLWAAIAKELKLDYELKPMDFSGIIPALQTKNVDLALAGITITDERKKAIDFSDGYYKSGLLVMVKANNNDVKSVKDLDGKVVAVKSGTGSVDYAKANIKTKDLRQFPNIDNAYMELGTNRADAVLHDTPNILYFIKTAGNGQFKAVGDSLEAQQYGIAFPKGSDELRDKVNGALKTLRENGTYNEIYKKWFGTEPK',\n",
       "  'mw': 27190.0},\n",
       " 'b0813': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AA67',\n",
       "  'gene': 'b0813',\n",
       "  'locus': ['b0813', 'JW0798'],\n",
       "  'synonyms': 'ybiF',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Involved in the efflux of threonine and homoserine. Can also export other amino acids such as proline, serine, histidine and cysteine'},\n",
       "  'seq': 'MPGSLRKMPVWLPIVILLVAMASIQGGASLAKSLFPLVGAPGVTALRLALGTLILIAFFKPWRLRFAKEQRLPLLFYGVSLGGMNYLFYLSIQTVPLGIAVALEFTGPLAVALFSSRRPVDFVWVVLAVLGLWFLLPLGQDVSHVDLTGCALALGAGACWAIYILSGQRAGAEHGPATVAIGSLIAALIFVPIGALQAGEALWHWSVIPLGLAVAILSTALPYSLEMIALTRLPTRTFGTLMSMEPALAAVSGMIFLGETLTPIQLLALGAIIAASMGSTLTVRKESKIKELDIN',\n",
       "  'mw': 31168.0},\n",
       " 'b0822': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P75792',\n",
       "  'gene': 'b0822',\n",
       "  'locus': ['b0822', 'JW0806'],\n",
       "  'synonyms': 'supH',\n",
       "  'properties': {'Catalytic Activity': [('sugar phosphate + H2O = sugar + phosphate.',\n",
       "     '3.1.3.23')],\n",
       "   'Function': 'Catalyzes the hydrolysis of sugar phosphate to sugar and inorganic phosphate. Has a wide substrate specificity catalyzing the hydrolysis of ribose-5-phosphate, glucose-6-phosphate, fructose-1-phosphate, acetyl-phosphate, glycerol-1-phosphate, glycerol-2-phosphate, 2-deoxy-glucose-6-phosphate, mannose-6-phosphate and fructose-6-phosphate. Appears to have a low level of phosphotransferase activity using monophosphates as the phosphate donor'},\n",
       "  'seq': 'MSVKVIVTDMDGTFLNDAKTYNQPRFMAQYQELKKRGIKFVVASGNQYYQLISFFPELKDEISFVAENGALVYEHGKQLFHGELTRHESRIVIGELLKDKQLNFVACGLQSAYVSENAPEAFVALMAKHYHRLKPVKDYQEIDDVLFKFSLNLPDEQIPLVIDKLHVALDGIMKPVTSGFGFIDLIIPGLHKANGISRLLKRWDLSPQNVVAIGDSGNDAEMLKMARYSFAMGNAAENIKQIARYATDDNNHEGALNVIQAVLDNTSPFNS',\n",
       "  'mw': 30413.0},\n",
       " 'b0825': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P78055',\n",
       "  'gene': 'b0825',\n",
       "  'locus': ['b0825', 'JW5109'],\n",
       "  'synonyms': 'fsa mipB ybiZ',\n",
       "  'properties': {'Catalytic Activity': [('beta-D-fructose 6-phosphate = D-glyceraldehyde 3-phosphate + dihydroxyacetone',\n",
       "     '')],\n",
       "   'Function': 'Catalyzes the reversible formation of fructose 6-phosphate from dihydroxyacetone (DHA) and D-glyceraldehyde 3-phosphate via an aldolization reaction. Can utilize several aldehydes as acceptor compounds in vitro, and hydroxyacetone (HA) or 1-hydroxy-butan-2-one as alternative donor substrate. Is also able to catalyze the direct stereoselective self-aldol addition of glycolaldehyde to furnish D-(-)-threose, and cross-aldol reactions of glycolaldehyde to other aldehyde acceptors. Is not able to cleave fructose, fructose 1-phosphate, glucose 6-phosphate, sedoheptulose 1,7-bisphosphate, xylulose 5-phosphate, ribulose 5-phosphate, and fructose 1,6-bisphosphate; cannot use dihydroxyacetone phosphate as donor compound nor D-glyceraldehyde as acceptor. Does not display transaldolase activity',\n",
       "   'Regulation Activity': 'Inhibited by glycerol, inorganic phosphate and arabinose 5-phosphate'},\n",
       "  'seq': 'MELYLDTSDVVAVKALSRIFPLAGVTTNPSIIAAGKKPLDVVLPQLHEAMGGQGRLFAQVMATTAEGMVNDALKLRSIIADIVVKVPVTAEGLAAIKMLKAEGIPTLGTAVYGAAQGLLSALAGAEYVAPYVNRIDAQGGSGIQTVTDLHQLLKMHAPQAKVLAASFKTPRQALDCLLAGCESITLPLDVAQQMISYPAVDAAVAKFEQDWQGAFGRTSI',\n",
       "  'mw': 22997.0},\n",
       " 'b0828': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37595',\n",
       "  'gene': 'b0828',\n",
       "  'locus': ['b0828', 'JW0812'],\n",
       "  'synonyms': 'spt ybiK',\n",
       "  'properties': {'Catalytic Activity': [('Cleavage of a beta-linked Asp residue from the N-terminus of a polypeptide.',\n",
       "     '3.4.19.5')],\n",
       "   'Function': 'May be involved in glutathione, and possibly other peptide, transport, although these results could also be due to polar effects of disruption'},\n",
       "  'seq': 'MGKAVIAIHGGAGAISRAQMSLQQELRYIEALSAIVETGQKMLEAGESALDVVTEAVRLLEECPLFNAGIGAVFTRDETHELDACVMDGNTLKAGAVAGVSHLRNPVLAARLVMEQSPHVMMIGEGAENFAFARGMERVSPEIFSTSLRYEQLLAARKEGATVLDHSGAPLDEKQKMGTVGAVALDLDGNLAAATSTGGMTNKLPGRVGDSPLVGAGCYANNASVAVSCTGTGEVFIRALAAYDIAALMDYGGLSLAEACERVVMEKLPALGGSGGLIAIDHEGNVALPFNTEGMYRAWGYAGDTPTTGIYREKGDTVATQ',\n",
       "  'mw': 33394.0},\n",
       " 'b0829': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P75796',\n",
       "  'gene': 'b0829',\n",
       "  'locus': ['b0829', 'JW5897'],\n",
       "  'synonyms': 'yliA',\n",
       "  'properties': {'Catalytic Activity': [('ATP + glutathione(out) + H2O = ADP + glutathione(in) + H(+) + phosphate',\n",
       "     '7.4.2.10')],\n",
       "   'Function': 'Part of the ABC transporter complex GsiABCD involved in glutathione import. Responsible for energy coupling to the transport system',\n",
       "   'Regulation Activity': 'Inhibited by verapamil but not by carbonyl cyanide m-chlorophenylhydrazone (CCCP)'},\n",
       "  'seq': 'MPHSDELDAGNVLAVENLNIAFMQDQQKIAAVRNLSFSLQRGETLAIVGESGSGKSVTALALMRLLEQAGGLVQCDKMLLQRRSREVIELSEQNAAQMRHVRGADMAMIFQEPMTSLNPVFTVGEQIAESIRLHQNASREEAMVEAKRMLDQVRIPEAQTILSRYPHQLSGGMRQRVMIAMALSCRPAVLIADEPTTALDVTIQAQILQLIKVLQKEMSMGVIFITHDMGVVAEIADRVLVMYQGEAVETGTVEQIFHAPQHPYTRALLAAVPQLGAMKGLDYPRRFPLISLEHPAKQAPPIEQKTVVDGEPVLRVRNLVTRFPLRSGLLNRVTREVHAVEKVSFDLWPGETLSLVGESGSGKSTTGRALLRLVESQGGEIIFNGQRIDTLSPGKLQALRRDIQFIFQDPYASLDPRQTIGDSIIEPLRVHGLLPGKDAAARVAWLLERVGLLPEHAWRYPHEFSGGQRQRICIARALALNPKVIIADEAVSALDVSIRGQIINLLLDLQRDFGIAYLFISHDMAVVERISHRVAVMYLGQIVEIGPRRAVFENPQHPYTRKLLAAVPVAEPSRQRPQRVLLSDDLPSNIHLRGEEVAAVSLQCVGPGHYVAQPQSEYAFMRR',\n",
       "  'mw': 69114.0},\n",
       " 'b0830': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P75797',\n",
       "  'gene': 'b0830',\n",
       "  'locus': ['b0830', 'JW5111'],\n",
       "  'synonyms': 'yliB',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the ABC transporter complex GsiABCD involved in glutathione import (PubMed:16109926). Binds glutathione (PubMed:30515393)'},\n",
       "  'seq': 'MARAVHRSGLVALGIATALMASCAFAAKDVVVAVGSNFTTLDPYDANDTLSQAVAKSFYQGLFGLDKEMKLKNVLAESYTVSDDGITYTVKLREGIKFQDGTDFNAAAVKANLDRASDPANHLKRYNLYKNIAKTEAIDPTTVKITLKQPFSAFINILAHPATAMISPAALEKYGKEIGFYPVGTGPYELDTWNQTDFVKVKKFAGYWQPGLPKLDSITWRPVADNNTRAAMLQTGEAQFAFPIPYEQATLLEKNKNIELMASPSIMQRYISMNVTQKPFDNPKVREALNYAINRPALVKVAFAGYATPATGVVPPSIAYAQSYKPWPYDPVKARELLKEAGYPNGFSTTLWSSHNHSTAQKVLQFTQQQLAQVGIKAQVTAMDAGQRAAEVEGKGQKESGVRMFYTGWSASTGEADWALSPLFASQNWPPTLFNTAFYSNKQVDDFLAQALKTNDPAEKTRLYKAAQDIIWQESPWIPLVVEKLVSAHSKNLTGFWIMPDTGFSFEDADLQ',\n",
       "  'mw': 56470.0},\n",
       " 'b0831': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P75798',\n",
       "  'gene': 'b0831',\n",
       "  'locus': ['b0831', 'JW0815'],\n",
       "  'synonyms': 'yliC',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the ABC transporter complex GsiABCD involved in glutathione import. Probably responsible for the translocation of the substrate across the membrane'},\n",
       "  'seq': 'MLNYVIKRLLGLIPTLFIVSVLVFLFVHMLPGDPARLIAGPEADAQVIELVRQQLGLDQPLYHQFWHYISNAVQGDFGLSMVSRRPVADEIASRFMPTLWLTITSMVWAVIFGMAAGIIAAVWRNRWPDRLSMTIAVSGISFPAFALGMLLIQVFSVELGWLPTVGADSWQHYILPSLTLGAAVAAVMARFTRASFVDVLSEDYMRTARAKGVSETWVVLKHGLRNAMIPVVTMMGLQFGFLLGGSIVVEKVFNWPGLGRLLVDSVEMRDYPVIQAEILLFSLEFILINLVVDVLYAAINPAIRYK',\n",
       "  'mw': 34066.0},\n",
       " 'b0832': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P75799',\n",
       "  'gene': 'b0832',\n",
       "  'locus': ['b0832', 'JW0816'],\n",
       "  'synonyms': 'yliD',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the ABC transporter complex GsiABCD involved in glutathione import. Probably responsible for the translocation of the substrate across the membrane'},\n",
       "  'seq': 'MRLFNWRRQAVLNAMPLVKPDQVRTPWHEFWRRFRRQHMAMTAALFVILLIVVAIFARWIAPYDAENYFDYDNLNNGPSLQHWFGVDSLGRDIFSRVLVGAQISLAAGVFAVFIGAAIGTLLGLLAGYYEGWWDRLIMRICDVLFAFPGILLAIAVVAVLGSGIANVIIAVAIFSIPAFARLVRGNTLVLKQQTFIESARSIGASDMTVLLRHILPGTVSSIVVFFTMRIGTSIISAASLSFLGLGAQPPTPEWGAMLNEARADMVIAPHVAVFPALAIFLTVLAFNLLGDGLRDALDPKIKG',\n",
       "  'mw': 33238.0},\n",
       " 'b0837': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P75804',\n",
       "  'gene': 'b0837',\n",
       "  'locus': ['b0837', 'JW0821'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Aldose sugar dehydrogenase with broad substrate specificity. The physiological substrate is unknown. Can oxidize glucose to gluconolactone. Can also utilize D-arabinose, L-arabinose and 2-deoxy-glucose. Has higher activity towards oligomeric sugars, such as maltose, maltotriose or cellobiose. It may function to input sugar-derived electrons into the respiratory network'},\n",
       "  'seq': 'MHRQSFFLVPLICLSSALWAAPATVNVEVLQDKLDHPWALAFLPDNHGMLITLRGGELRHWQAGKGLSAPLSGVPDVWAHGQGGLLDVVLAPDFAQSRRIWLSYSEVGDDGKAGTAVGYGRLSDDLSKVTDFRTVFRQMPKLSTGNHFGGRLVFDGKGYLFIALGENNQRPTAQDLDKLQGKLVRLTDQGEIPDDNPFIKESGARAEIWSYGIRNPQGMAMNPWSNALWLNEHGPRGGDEINIPQKGKNYGWPLATWGINYSGFKIPEAKGEIVAGTEQPVFYWKDSPAVSGMAFYNSDKFPQWQQKLFIGALKDKDVIVMSVNGDKVTEDGRILTDRGQRIRDVRTGPDGYLYVLTDESSGELLKVSPRN',\n",
       "  'mw': 41054.0},\n",
       " 'b0839': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P08506',\n",
       "  'gene': 'b0839',\n",
       "  'locus': ['b0839', 'JW0823'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('Preferential cleavage: (Ac)2-L-Lys-D-Ala-|-D-Ala. Also transpeptidation of peptidyl-alanyl moieties that are N-acyl substituents of D-alanine.',\n",
       "     '3.4.16.4')],\n",
       "   'Function': 'Removes C-terminal D-alanyl residues from sugar-peptide cell wall precursors',\n",
       "   'Pathway': 'Cell wall biogenesis; peptidoglycan biosynthesis'},\n",
       "  'seq': 'MTQYSSLLRGLAAGSAFLFLFAPTAFAAEQTVEAPSVDARAWILMDYASGKVLAEGNADEKLDPASLTKIMTSYVVGQALKADKIKLTDMVTVGKDAWATGNPALRGSSVMFLKPGDQVSVADLNKGVIIQSGNDACIALADYVAGSQESFIGLMNGYAKKLGLTNTTFQTVHGLDAPGQFSTARDMALLGKALIHDVPEEYAIHKEKEFTFNKIRQPNRNRLLWSSNLNVDGMKTGTTAGAGYNLVASATQGDMRLISVVLGAKTDRIRFNESEKLLTWGFRFFETVTPIKPDATFVTQRVWFGDKSEVNLGAGEAGSVTIPRGQLKNLKASYTLTEPQLTAPLKKGQVVGTIDFQLNGKSIEQRPLIVMENVEEGGFFGRVWDFVMMKFHQWFGSWFS',\n",
       "  'mw': 43609.0},\n",
       " 'b0841': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P75806',\n",
       "  'gene': 'b0841',\n",
       "  'locus': ['b0841', 'JW5112'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('di-trans,octa-cis-undecaprenyl diphosphate + H2O = di-trans,octa-cis-undecaprenyl phosphate + H(+) + phosphate',\n",
       "     '3.6.1.27')],\n",
       "   'Function': 'Overexpression leads to increased undecaprenyl diphosphatase activity and to increased resistance to bacitracin. May have a preferred substrate other than undecaprenyl diphosphate in vivo'},\n",
       "  'seq': 'MLENLNLSLFSLINATPDSAPWMISLAIFIAKDLITVVPLLAVVLWLWGLTAQRQLVIKIAIALAVSLFVSWTMGHLFPHDRPFVENIGYNFLHHAADDSFPSDHGTVIFTFALAFLCWHRLWSGSLLMVLAVVIAWSRVYLGVHWPLDMLGGLLAGMIGCLSAQIIWQAMGHKLYQRLQSWYRVCFALPIRKGWVRD',\n",
       "  'mw': 22399.0},\n",
       " 'b0842': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AEY8',\n",
       "  'gene': 'b0842',\n",
       "  'locus': ['b0842', 'JW0826'],\n",
       "  'synonyms': 'cmlA cmr',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Efflux pump driven by the proton motive force. Confers resistance to a broad spectrum of chemically unrelated drugs. Confers resistance to a diverse group of cationic or zwitterionic lipophilic compounds such as ethidium bromide, tetraphenylphosphonium, rhodamine, daunomycin, benzalkonium, rifampicin, tetracycline, puromycin, and to chemically unrelated, clinically important antibiotics such as chloramphenicol, erythromycin, and certain aminoglycosides and fluoroquinolones. Overexpression results in isopropyl-beta-D-thiogalactopyranoside (IPTG) exclusion and spectinomycin sensitivity. Transport of neutral substrates is electrogenic, whereas transport of cationic substrates is electroneutral. In addition to its role in multidrug resistance, confers extreme alkaline pH resistance, allowing the growth under conditions that are close to those used normally by alkaliphiles. This activity requires Na(+) or K(+)',\n",
       "   'Regulation Activity': 'Inhibited by CCCP'},\n",
       "  'seq': 'MQNKLASGARLGRQALLFPLCLVLYEFSTYIGNDMIQPGMLAVVEQYQAGIDWVPTSMTAYLAGGMFLQWLLGPLSDRIGRRPVMLAGVVWFIVTCLAILLAQNIEQFTLLRFLQGISLCFIGAVGYAAIQESFEEAVCIKITALMANVALIAPLLGPLVGAAWIHVLPWEGMFVLFAALAAISFFGLQRAMPETATRIGEKLSLKELGRDYKLVLKNGRFVAGALALGFVSLPLLAWIAQSPIIIITGEQLSSYEYGLLQVPIFGALIAGNLLLARLTSRRTVRSLIIMGGWPIMIGLLVAAAATVISSHAYLWMTAGLSIYAFGIGLANAGLVRLTLFASDMSKGTVSAAMGMLQMLIFTVGIEISKHAWLNGGNGLFNLFNLVNGILWLSLMVIFLKDKQMGNSHEG',\n",
       "  'mw': 44321.0},\n",
       " 'b0849': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P68688',\n",
       "  'gene': 'b0849',\n",
       "  'locus': ['b0849', 'JW0833'],\n",
       "  'synonyms': 'grx',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'The disulfide bond functions as an electron carrier in the glutathione-dependent synthesis of deoxyribonucleotides by the enzyme ribonucleotide reductase. In addition, it is also involved in reducing some disulfide bonds in a coupled system with glutathione reductase'},\n",
       "  'seq': 'MQTVIFGRSGCPYCVRAKDLAEKLSNERDDFQYQYVDIRAEGITKEDLQQKAGKPVETVPQIFVDQQHIGGYTDFAAWVKENLDA',\n",
       "  'mw': 9685.0},\n",
       " 'b0854': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P31133',\n",
       "  'gene': 'b0854',\n",
       "  'locus': ['b0854', 'JW0838'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the ABC transporter complex PotFGHI involved in putrescine uptake (PubMed:8416922, PubMed:23719730). Binds putrescine (PubMed:8416922, PubMed:9651355). Imports putrescine for maintenance of the optimal concentration of polyamines necessary for cell growth in the presence of glucose (PubMed:23719730)',\n",
       "   'Regulation Activity': 'Transport is feedback inhibited by intracellular polyamines'},\n",
       "  'seq': 'MTALNKKWLSGLVAGALMAVSVGTLAAEQKTLHIYNWSDYIAPDTVANFEKETGIKVVYDVFDSNEVLEGKLMAGSTGFDLVVPSASFLERQLTAGVFQPLDKSKLPEWKNLDPELLKLVAKHDPDNKFAMPYMWATTGIGYNVDKVKAVLGENAPVDSWDLILKPENLEKLKSCGVSFLDAPEEVFATVLNYLGKDPNSTKADDYTGPATDLLLKLRPNIRYFHSSQYINDLANGDICVAIGWAGDVWQASNRAKEAKNGVNVSFSIPKEGAMAFFDVFAMPADAKNKDEAYQFLNYLLRPDVVAHISDHVFYANANKAATPLVSAEVRENPGIYPPADVRAKLFTLKVQDPKIDRVRTRAWTKVKSGK',\n",
       "  'mw': 40840.0},\n",
       " 'b0855': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P31134',\n",
       "  'gene': 'b0855',\n",
       "  'locus': ['b0855', 'JW5818'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + H2O + putrescine(out) = ADP + H(+) + phosphate + putrescine(in)',\n",
       "     '7.6.2.16')],\n",
       "   'Function': 'Part of the ABC transporter complex PotFGHI involved in putrescine uptake (PubMed:8416922, PubMed:23719730). Responsible for energy coupling to the transport system (PubMed:23719730). Imports putrescine for maintenance of the optimal concentration of polyamines necessary for cell growth in the presence of glucose (PubMed:23719730)',\n",
       "   'Regulation Activity': 'Transport is feedback inhibited by intracellular polyamines'},\n",
       "  'seq': 'MNDAIPRPQAKTRKALTPLLEIRNLTKSYDGQHAVDDVSLTIYKGEIFALLGASGCGKSTLLRMLAGFEQPSAGQIMLDGVDLSQVPPYLRPINMMFQSYALFPHMTVEQNIAFGLKQDKLPKAEIASRVNEMLGLVHMQEFAKRKPHQLSGGQRQRVALARSLAKRPKLLLLDEPMGALDKKLRDRMQLEVVDILERVGVTCVMVTHDQEEAMTMAGRIAIMNRGKFVQIGEPEEIYEHPTTRYSAEFIGSVNVFEGVLKERQEDGLVLDSPGLVHPLKVDADASVVDNVPVHVALRPEKIMLCEEPPANGCNFAVGEVIHIAYLGDLSVYHVRLKSGQMISAQLQNAHRHRKGLPTWGDEVRLCWEVDSCVVLTV',\n",
       "  'mw': 41931.0},\n",
       " 'b0856': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P31135',\n",
       "  'gene': 'b0856',\n",
       "  'locus': ['b0856', 'JW0840'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the ABC transporter complex PotFGHI involved in putrescine uptake (PubMed:8416922, PubMed:23719730). Responsible for the translocation of the substrate across the membrane (Probable). Imports putrescine for maintenance of the optimal concentration of polyamines necessary for cell growth in the presence of glucose (PubMed:23719730)',\n",
       "   'Regulation Activity': 'Transport is feedback inhibited by intracellular polyamines'},\n",
       "  'seq': 'MSTLEPAAQSKPPGGFKLWLSQLQMKHGRKLVIALPYIWLILLFLLPFLIVFKISLAEMARAIPPYTELMEWADGQLSITLNLGNFLQLTDDPLYFDAYLQSLQVAAISTFCCLLIGYPLAWAVAHSKPSTRNILLLLVILPSWTSFLIRVYAWMGILKNNGVLNNFLLWLGVIDQPLTILHTNLAVYIGIVYAYVPFMVLPIYTALIRIDYSLVEAALDLGARPLKTFFTVIVPLTKGGIIAGSMLVFIPAVGEFVIPELLGGPDSIMIGRVLWQEFFNNRDWPVASAVAIIMLLLLIVPIMWFHKHQQKSVGEHG',\n",
       "  'mw': 35490.0},\n",
       " 'b0857': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AFL1',\n",
       "  'gene': 'b0857',\n",
       "  'locus': ['b0857', 'JW0841'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the ABC transporter complex PotFGHI involved in putrescine uptake (PubMed:8416922, PubMed:23719730). Responsible for the translocation of the substrate across the membrane (Probable). Imports putrescine for maintenance of the optimal concentration of polyamines necessary for cell growth in the presence of glucose (PubMed:23719730)',\n",
       "   'Regulation Activity': 'Transport is feedback inhibited by intracellular polyamines'},\n",
       "  'seq': 'MNNLPVVRSPWRIVILLLGFTFLYAPMLMLVIYSFNSSKLVTVWAGWSTRWYGELLRDDAMMSAVGLSLTIAACAATAAAILGTIAAVVLVRFGRFRGSNGFAFMITAPLVMPDVITGLSLLLLFVALAHAIGWPADRGMLTIWLAHVTFCTAYVAVVISSRLRELDRSIEEAAMDLGATPLKVFFVITLPMIMPAIISGWLLAFTLSLDDLVIASFVSGPGATTLPMLVFSSVRMGVNPEINALATLILGAVGIVGFIAWYLMARAEKQRIRDIQRARRG',\n",
       "  'mw': 30541.0},\n",
       " 'b0860': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P30860',\n",
       "  'gene': 'b0860',\n",
       "  'locus': ['b0860', 'JW0844'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the ABC transporter complex ArtPIQMJ involved in arginine transport. Binds L-arginine with high affinity'},\n",
       "  'seq': 'MKKLVLAALLASFTFGASAAEKINFGVSATYPPFESIGANNEIVGFDIDLAKALCKQMQAECTFTNHAFDSLIPSLKFRKYDAVISGMDITPERSKQVSFTTPYYENSAVVIAKKDTYKTFADLKGKRIGMENGTTHQKYIQDQHPEVKTVSYDSYQNAFIDLKNGRIDGVFGDTAVVNEWLKTNPQLGVATEKVTDPQYFGTGLGIAVRPDNKALLEKLNNALAAIKADGTYQKISDQWFPQ',\n",
       "  'mw': 26829.0},\n",
       " 'b0861': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AE30',\n",
       "  'gene': 'b0861',\n",
       "  'locus': ['b0861', 'JW0845'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the ABC transporter complex ArtPIQMJ involved in arginine transport. Probably responsible for the translocation of the substrate across the membrane'},\n",
       "  'seq': 'MFEYLPELMKGLHTSLTLTVASLIVALILALIFTIILTLKTPVLVWLVRGYITLFTGTPLLVQIFLIYYGPGQFPTLQEYPALWHLLSEPWLCALIALSLNSAAYTTQLFYGAIRAIPEGQWQSCSALGMSKKDTLAILLPYAFKRSLSSYSNEVVLVFKSTSLAYTITLMEVMGYSQLLYGRTYDVMVFGAAGIIYLVVNGLLTLMMRLIERKALAFERRN',\n",
       "  'mw': 24914.0},\n",
       " 'b0862': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AE34',\n",
       "  'gene': 'b0862',\n",
       "  'locus': ['b0862', 'JW0846'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the ABC transporter complex ArtPIQMJ involved in arginine transport. Probably responsible for the translocation of the substrate across the membrane'},\n",
       "  'seq': 'MNEFFPLASAAGMTVGLAVCALIVGLALAMFFAVWESAKWRPVAWAGSALVTILRGLPEILVVLFIYFGSSQLLLTLSDGFTINLGFVQIPVQMDIENFDVSPFLCGVIALSLLYAAYASQTLRGALKAVPVGQWESGQALGLSKSAIFFRLVMPQMWRHALPGLGNQWLVLLKDTALVSLISVNDLMLQTKSIATRTQEPFTWYIVAAAIYLVITLLSQYILKRIDLRATRFERRPS',\n",
       "  'mw': 26217.0},\n",
       " 'b0863': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P30859',\n",
       "  'gene': 'b0863',\n",
       "  'locus': ['b0863', 'JW0847'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the ABC transporter complex ArtPIQMJ involved in arginine transport'},\n",
       "  'seq': 'MKKVLIAALIAGFSLSATAAETIRFATEASYPPFESIDANNQIVGFDVDLAQALCKEIDATCTFSNQAFDSLIPSLKFRRVEAVMAGMDITPEREKQVLFTTPYYDNSALFVGQQGKYTSVDQLKGKKVGVQNGTTHQKFIMDKHPEITTVPYDSYQNAKLDLQNGRIDGVFGDTAVVTEWLKDNPKLAAVGDKVTDKDYFGTGLGIAVRQGNTELQQKLNTALEKVKKDGTYETIYNKWFQK',\n",
       "  'mw': 26930.0},\n",
       " 'b0864': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AAF6',\n",
       "  'gene': 'b0864',\n",
       "  'locus': ['b0864', 'JW0848'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the ABC transporter complex ArtPIQMJ involved in arginine transport. Probably responsible for energy coupling to the transport system'},\n",
       "  'seq': 'MSIQLNGINCFYGAHQALFDITLDCPQGETLVLLGPSGAGKSSLLRVLNLLEMPRSGTLNIAGNHFDFTKTPSDKAIRDLRRNVGMVFQQYNLWPHLTVQQNLIEAPCRVLGLSKDQALARAEKLLERLRLKPYSDRYPLHLSGGQQQRVAIARALMMEPQVLLFDEPTAALDPEITAQIVSIIRELAETNITQVIVTHEVEVARKTASRVVYMENGHIVEQGDASCFTEPQTEAFKNYLSH',\n",
       "  'mw': 27022.0},\n",
       " 'b0870': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P75823',\n",
       "  'gene': 'b0870',\n",
       "  'locus': ['b0870', 'JW0854'],\n",
       "  'synonyms': 'ybjU',\n",
       "  'properties': {'Catalytic Activity': [('L-threonine = acetaldehyde + glycine',\n",
       "     '4.1.2.48'),\n",
       "    ('L-allo-threonine = acetaldehyde + glycine', '4.1.2.48')],\n",
       "   'Function': 'Catalyzes the cleavage of L-allo-threonine and L-threonine to glycine and acetaldehyde. L-threo-phenylserine and L-erythro-phenylserine are also good substrates'},\n",
       "  'seq': 'MIDLRSDTVTRPSRAMLEAMMAAPVGDDVYGDDPTVNALQDYAAELSGKEAAIFLPTGTQANLVALLSHCERGEEYIVGQAAHNYLFEAGGAAVLGSIQPQPIDAAADGTLPLDKVAMKIKPDDIHFARTKLLSLENTHNGKVLPREYLKEAWEFTRERNLALHVDGARIFNAVVAYGCELKEITQYCDSFTICLSKGLGTPVGSLLVGNRDYIKRAIRWRKMTGGGMRQSGILAAAGIYALKNNVARLQEDHDNAAWMAEQLREAGADVMRQDTNMLFVRVGEENAAALGEYMKARNVLINASPIVRLVTHLDVSREQLAEVAAHWRAFLAR',\n",
       "  'mw': 36495.0},\n",
       " 'b0871': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P07003',\n",
       "  'gene': 'b0871',\n",
       "  'locus': ['b0871', 'JW0855'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('a ubiquinone + H2O + pyruvate = a ubiquinol + acetate + CO2',\n",
       "     '1.2.5.1')],\n",
       "   'Function': 'A peripheral cell membrane enzyme that catalyzes the oxidative decarboxylation of pyruvate to form acetate and CO(2). It channels electrons from the cytoplasm to the respiratory chain at the cell membrane via ubiquinone (By similarity) (PubMed:18988747, PubMed:2663858, PubMed:365232). The main pathway for acetate production during stationary phase (PubMed:16080684)',\n",
       "   'Regulation Activity': 'The C-terminus inhibits activity; it has to move for the enzyme to be active (PubMed:18988747). Activated by lipid-binding, which occurs via the C-terminus; detergents also activate the enzyme (PubMed:334770, PubMed:334771, PubMed:2663858, PubMed:18988747). Specifically activated by palmitic acid; 12 (lauric) and 14-carbon (myristic) fatty acids as well as unsaturated 16 and 18-carbon fatty acids also activate the enzyme (PubMed:6385860). Lipid affinity is increased in the presence of pyruvate plus the thiamine pyrophosphate cofactor (PubMed:334770, PubMed:334771, PubMed:2663858, PubMed:18988747)'},\n",
       "  'seq': 'MKQTVAAYIAKTLESAGVKRIWGVTGDSLNGLSDSLNRMGTIEWMSTRHEEVAAFAAGAEAQLSGELAVCAGSCGPGNLHLINGLFDCHRNHVPVLAIAAHIPSSEIGSGYFQETHPQELFRECSHYCELVSSPEQIPQVLAIAMRKAVLNRGVSVVVLPGDVALKPAPEGATMHWYHAPQPVVTPEEEELRKLAQLLRYSSNIALMCGSGCAGAHKELVEFAGKIKAPIVHALRGKEHVEYDNPYDVGMTGLIGFSSGFHTMMNADTLVLLGTQFPYRAFYPTDAKIIQIDINPASIGAHSKVDMALVGDIKSTLRALLPLVEEKADRKFLDKALEDYRDARKGLDDLAKPSEKAIHPQYLAQQISHFAADDAIFTCDVGTPTVWAARYLKMNGKRRLLGSFNHGSMANAMPQALGAQATEPERQVVAMCGDGGFSMLMGDFLSVVQMKLPVKIVVFNNSVLGFVAMEMKAGGYLTDGTELHDTNFARIAEACGITGIRVEKASEVDEALQRAFSIDGPVLVDVVVAKEELAIPPQIKLEQAKGFSLYMLRAIISGRGDEVIELAKTNWLR',\n",
       "  'mw': 62011.0},\n",
       " 'b0875': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P60844',\n",
       "  'gene': 'b0875',\n",
       "  'locus': ['b0875', 'JW0859'],\n",
       "  'synonyms': 'bniP',\n",
       "  'properties': {'Catalytic Activity': [('H2O(in) = H2O(out)', '')],\n",
       "   'Function': 'Channel that permits osmotically driven movement of water in both directions. It is involved in the osmoregulation and in the maintenance of cell turgor during volume expansion in rapidly growing cells. It mediates rapid entry or exit of water in response to abrupt changes in osmolarity'},\n",
       "  'seq': 'MFRKLAAECFGTFWLVFGGCGSAVLAAGFPELGIGFAGVALAFGLTVLTMAFAVGHISGGHFNPAVTIGLWAGGRFPAKEVVGYVIAQVVGGIVAAALLYLIASGKTGFDAAASGFASNGYGEHSPGGYSMLSALVVELVLSAGFLLVIHGATDKFAPAGFAPIAIGLALTLIHLISIPVTNTSVNPARSTAVAIFQGGWALEQLWFFWVVPIVGGIIGGLIYRTLLEKRD',\n",
       "  'mw': 23703.0},\n",
       " 'b0886': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P23886',\n",
       "  'gene': 'b0886',\n",
       "  'locus': ['b0886', 'JW0869'],\n",
       "  'synonyms': 'mdrA mdrH surB',\n",
       "  'properties': {'Catalytic Activity': [('ATP + H2O + L-cysteine(in) = ADP + H(+) + L-cysteine(out) + phosphate',\n",
       "     ''),\n",
       "    ('ATP + glutathione(in) + H2O = ADP + glutathione(out) + H(+) + phosphate',\n",
       "     '')],\n",
       "   'Function': 'Conversely, a more recent study suggests an alternative function of CydDC: authors suggest that CydDC does not mediate the export of L-cysteine but rather reduces cytoplasmic L-cystine to L-cysteine (PubMed:32900959). The principle function of CydDC would be to maintain the reduced state of cytoplasmic L-cysteine, thereby providing an important connection between sulfur metabolism, oxidative stress and resistance to antibiotics (PubMed:32900959)',\n",
       "   'Regulation Activity': 'ATPase activity is stimulated by various thiol compounds (PubMed:24958725). The presence of heme leads to a further enhancement of thiol-stimulated ATPase activity, although a large excess of heme inhibits activity (PubMed:24958725). Glutathione transport is inhibited by sodium orthovanadate, an inhibitor of ABC-type transport systems, but not by the proton ionophore carbonyl cyanide m-chlorophenylhydrazone (CCCP) (PubMed:16040611)'},\n",
       "  'seq': 'MRALLPYLALYKRHKWMLSLGIVLAIVTLLASIGLLTLSGWFLSASAVAGVAGLYSFNYMLPAAGVRGAAITRTAGRYFERLVSHDATFRVLQHLRIYTFSKLLPLSPAGLARYRQGELLNRVVADVDTLDHLYLRVISPLVGAFVVIMVVTIGLSFLDFTLAFTLGGIMLLTLFLMPPLFYRAGKSTGQNLTHLRGQYRQQLTAWLQGQAELTIFGASDRYRTQLENTEIQWLEAQRRQSELTALSQAIMLLIGALAVILMLWMASGGVGGNAQPGALIALFVFCALAAFEALAPVTGAFQHLGQVIASAVRISDLTDQKPEVTFPDTQTRVADRVSLTLRDVQFTYPEQSQQALKGISLQVNAGEHIAILGRTGCGKSTLLQQLTRAWDPQQGEILLNDSPIASLNEAALRQTISVVPQRVHLFSATLRDNLLLASPGSSDEALSEILRRVGLEKLLEDAGLNSWLGEGGRQLSGGELRRLAIARALLHDAPLVLLDEPTEGLDATTESQILELLAEMMREKTVLMVTHRLRGLSRFQQIIVMDNGQIIEQGTHAELLARQGRYYQFKQGL',\n",
       "  'mw': 62920.0},\n",
       " 'b0887': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P29018',\n",
       "  'gene': 'b0887',\n",
       "  'locus': ['b0887', 'JW0870'],\n",
       "  'synonyms': 'htrD',\n",
       "  'properties': {'Catalytic Activity': [('ATP + H2O + L-cysteine(in) = ADP + H(+) + L-cysteine(out) + phosphate',\n",
       "     ''),\n",
       "    ('ATP + glutathione(in) + H2O = ADP + glutathione(out) + H(+) + phosphate',\n",
       "     '')],\n",
       "   'Function': 'Conversely, a more recent study suggests an alternative function of CydDC: authors suggest that CydDC does not mediate the export of L-cysteine but rather reduces cytoplasmic L-cystine to L-cysteine (PubMed:32900959). The principle function of CydDC would be to maintain the reduced state of cytoplasmic L-cysteine, thereby providing an important connection between sulfur metabolism, oxidative stress and resistance to antibiotics (PubMed:32900959)',\n",
       "   'Regulation Activity': 'ATPase activity is stimulated by various thiol compounds (PubMed:24958725). The presence of heme leads to a further enhancement of thiol-stimulated ATPase activity, although a large excess of heme inhibits activity (PubMed:24958725). Glutathione transport is inhibited by sodium orthovanadate, an inhibitor of ABC-type transport systems, but not by the proton ionophore carbonyl cyanide m-chlorophenylhydrazone (CCCP) (PubMed:16040611)'},\n",
       "  'seq': 'MNKSRQKELTRWLKQQSVISQRWLNISRLLGFVSGILIIAQAWFMARILQHMIMENIPREALLLPFTLLVLTFVLRAWVVWLRERVGYHAGQHIRFAIRRQVLDRLQQAGPAWIQGKPAGSWATLVLEQIDDMHDYYARYLPQMALAVSVPLLIVVAIFPSNWAAALILLGTAPLIPLFMALVGMGAADANRRNFLALARLSGHFLDRLRGMETLRIFGRGEAEIESIRSASEDFRQRTMEVLRLAFLSSGILEFFTSLSIALVAVYFGFSYLGELDFGHYDTGVTLAAGFLALILAPEFFQPLRDLGTFYHAKAQAVGAADSLKTFMETPLAHPQRGEAELASTDPVTIEAEELFITSPEGKTLAGPLNFTLPAGQRAVLVGRSGSGKSSLLNALSGFLSYQGSLRINGIELRDLSPESWRKHLSWVGQNPQLPAATLRDNVLLARPDASEQELQAALDNAWVSEFLPLLPQGVDTPVGDQAARLSVGQAQRVAVARALLNPCSLLLLDEPAASLDAHSEQRVMEALNAASLRQTTLMVTHQLEDLADWDVIWVMQDGRIIEQGRYAELSVAGGPFATLLAHRQEEI',\n",
       "  'mw': 65056.0},\n",
       " 'b0888': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A9P4',\n",
       "  'gene': 'b0888',\n",
       "  'locus': ['b0888', 'JW0871'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('[thioredoxin]-dithiol + NADP(+) = [thioredoxin]-disulfide + H(+) + NADPH',\n",
       "     '1.8.1.9')]},\n",
       "  'seq': 'MGTTKHSKLLILGSGPAGYTAAVYAARANLQPVLITGMEKGGQLTTTTEVENWPGDPNDLTGPLLMERMHEHATKFETEIIFDHINKVDLQNRPFRLNGDNGEYTCDALIIATGASARYLGLPSEEAFKGRGVSACATCDGFFYRNQKVAVIGGGNTAVEEALYLSNIASEVHLIHRRDGFRAEKILIKRLMDKVENGNIILHTNRTLEEVTGDQMGVTGVRLRDTQNSDNIESLDVAGLFVAIGHSPNTAIFEGQLELENGYIKVQSGIHGNATQTSIPGVFAAGDVMDHIYRQAITSAGTGCMAALDAERYLDGLADAK',\n",
       "  'mw': 34623.0},\n",
       " 'b0893': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A8L1',\n",
       "  'gene': 'b0893',\n",
       "  'locus': ['b0893', 'JW0876'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + L-serine + tRNA(Ser) = AMP + diphosphate + H(+) + L-seryl-tRNA(Ser)',\n",
       "     '6.1.1.11'),\n",
       "    ('ATP + L-serine + tRNA(Sec) = AMP + diphosphate + H(+) + L-seryl-tRNA(Sec)',\n",
       "     '6.1.1.11')],\n",
       "   'Function': 'Catalyzes the attachment of serine to tRNA(Ser) (PubMed:7537870). Is also able to aminoacylate tRNA(Sec) with serine, to form the misacylated tRNA L-seryl-tRNA(Sec), which will be further converted into selenocysteinyl-tRNA(Sec)',\n",
       "   'Pathway': 'Aminoacyl-tRNA biosynthesis; selenocysteinyl-tRNA(Sec) biosynthesis; L-seryl-tRNA(Sec) from L-serine and tRNA(Sec): step 1/1'},\n",
       "  'seq': 'MLDPNLLRNEPDAVAEKLARRGFKLDVDKLGALEERRKVLQVKTENLQAERNSRSKSIGQAKARGEDIEPLRLEVNKLGEELDAAKAELDALQAEIRDIALTIPNLPADEVPVGKDENDNVEVSRWGTPREFDFEVRDHVTLGEMHSGLDFAAAVKLTGSRFVVMKGQIARMHRALSQFMLDLHTEQHGYSENYVPYLVNQDTLYGTGQLPKFAGDLFHTRPLEEEADTSNYALIPTAEVPLTNLVRGEIIDEDDLPIKMTAHTPCFRSEAGSYGRDTRGLIRMHQFDKVEMVQIVRPEDSMAALEEMTGHAEKVLQLLGLPYRKIILCTGDMGFGACKTYDLEVWIPAQNTYREISSCSNVWDFQARRMQARCRSKSDKKTRLVHTLNGSGLAVGRTLVAVMENYQQADGRIEVPEVLRPYMNGLEYIG',\n",
       "  'mw': 48414.0},\n",
       " 'b0894': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P18775',\n",
       "  'gene': 'b0894',\n",
       "  'locus': ['b0894', 'JW5118'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('a menaquinone + dimethyl sulfide + H2O = a menaquinol + dimethyl sulfoxide',\n",
       "     '1.8.5.3')],\n",
       "   'Function': 'Catalyzes the reduction of dimethyl sulfoxide (DMSO) to dimethyl sulfide (DMS). DMSO reductase serves as the terminal reductase under anaerobic conditions, with DMSO being the terminal electron acceptor. Terminal reductase during anaerobic growth on various sulfoxides and N-oxide compounds. Allows E.coli to grow anaerobically on DMSO as respiratory oxidant',\n",
       "   'Regulation Activity': 'Inhibited by dithionite, sodium hydrogensulfite and tungstate'},\n",
       "  'seq': 'MKTKIPDAVLAAEVSRRGLVKTTAIGGLAMASSALTLPFSRIAHAVDSAIPTKSDEKVIWSACTVNCGSRCPLRMHVVDGEIKYVETDNTGDDNYDGLHQVRACLRGRSMRRRVYNPDRLKYPMKRVGARGEGKFERISWEEAYDIIATNMQRLIKEYGNESIYLNYGTGTLGGTMTRSWPPGNTLVARLMNCCGGYLNHYGDYSSAQIAEGLNYTYGGWADGNSPSDIENSKLVVLFGNNPGETRMSGGGVTYYLEQARQKSNARMIIIDPRYTDTGAGREDEWIPIRPGTDAALVNGLAYVMITENLVDQAFLDKYCVGYDEKTLPASAPKNGHYKAYILGEGPDGVAKTPEWASQITGVPADKIIKLAREIGSTKPAFISQGWGPQRHANGEIATRAISMLAILTGNVGINGGNSGAREGSYSLPFVRMPTLENPIQTSISMFMWTDAIERGPEMTALRDGVRGKDKLDVPIKMIWNYAGNCLINQHSEINRTHEILQDDKKCELIVVIDCHMTSSAKYADILLPDCTASEQMDFALDASCGNMSYVIFNDQVIKPRFECKTIYEMTSELAKRLGVEQQFTEGRTQEEWMRHLYAQSREAIPELPTFEEFRKQGIFKKRDPQGHHVAYKAFREDPQANPLTTPSGKIEIYSQALADIAATWELPEGDVIDPLPIYTPGFESYQDPLNKQYPLQLTGFHYKSRVHSTYGNVDVLKAACRQEMWINPLDAQKRGIHNGDKVRIFNDRGEVHIEAKVTPRMMPGVVALGEGAWYDPDAKRVDKGGCINVLTTQRPSPLAKGNPSHTNLVQVEKV',\n",
       "  'mw': 90399.0},\n",
       " 'b0895': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P18776',\n",
       "  'gene': 'b0895',\n",
       "  'locus': ['b0895', 'JW0878'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Electron transfer subunit of the terminal reductase during anaerobic growth on various sulfoxide and N-oxide compounds'},\n",
       "  'seq': 'MTTQYGFFIDSSRCTGCKTCELACKDYKDLTPEVSFRRIYEYAGGDWQEDNGVWHQNVFAYYLSISCNHCEDPACTKVCPSGAMHKREDGFVVVDEDVCIGCRYCHMACPYGAPQYNETKGHMTKCDGCYDRVAEGKKPICVESCPLRALDFGPIDELRKKHGDLAAVAPLPRAHFTKPNIVIKPNANSRPTGDTTGYLANPKEV',\n",
       "  'mw': 22869.0},\n",
       " 'b0896': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P18777',\n",
       "  'gene': 'b0896',\n",
       "  'locus': ['b0896', 'JW0879'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Terminal reductase during anaerobic growth on various sulfoxide and N-oxide compounds. DmsC anchors the DmsAB dimer to the membrane and stabilizes it'},\n",
       "  'seq': 'MGSGWHEWPLMIFTVFGQCVAGGFIVLALALLKGDLRAEAQQRVIACMFGLWVLMGIGFIASMLHLGSPMRAFNSLNRVGASALSNEIASGSIFFAVGGIGWLLAMLKKLSPALRTLWLIVTMVLGVIFVWMMVRVYNSIDTVPTWYSIWTPMGFFLTMFMGGPLLGYLLLSLAGVDGWAMRLLPAISVLALVVSGVVSVMQGAELATIHSSVQQAAALVPDYGALMSWRIVLLAVALCLWIAPQLKGYQPAVPLLSVSFILLLAGELIGRGVFYGLHMTVGMAVAS',\n",
       "  'mw': 30826.0},\n",
       " 'b0902': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A9N4',\n",
       "  'gene': 'b0902',\n",
       "  'locus': ['b0902', 'JW0885'],\n",
       "  'synonyms': 'act',\n",
       "  'properties': {'Catalytic Activity': [(\"glycyl-[formate C-acetyltransferase] + reduced [flavodoxin] + S-adenosyl-L-methionine = 5'-deoxyadenosine + glycin-2-yl radical-[formate C-acetyltransferase] + H(+) + L-methionine + semiquinone [flavodoxin]\",\n",
       "     '1.97.1.4')],\n",
       "   'Function': \"Activation of pyruvate formate-lyase 1 under anaerobic conditions by generation of an organic free radical, using S-adenosylmethionine and reduced flavodoxin as cosubstrates to produce 5'-deoxy-adenosine\"},\n",
       "  'seq': 'MSVIGRIHSFESCGTVDGPGIRFITFFQGCLMRCLYCHNRDTWDTHGGKEVTVEDLMKEVVTYRHFMNASGGGVTASGGEAILQAEFVRDWFRACKKEGIHTCLDTNGFVRRYDPVIDELLEVTDLVMLDLKQMNDEIHQNLVGVSNHRTLEFAKYLANKNVKVWIRYVVVPGWSDDDDSAHRLGEFTRDMGNVEKIELLPYHELGKHKWVAMGEEYKLDGVKPPKKETMERVKGILEQYGHKVMF',\n",
       "  'mw': 28204.0},\n",
       " 'b0903': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P09373',\n",
       "  'gene': 'b0903',\n",
       "  'locus': ['b0903', 'JW0886'],\n",
       "  'synonyms': 'pfl',\n",
       "  'properties': {'Catalytic Activity': [('acetyl-CoA + formate = CoA + pyruvate',\n",
       "     '2.3.1.54')],\n",
       "   'Function': 'Catalyzes the conversion of pyruvate to formate and acetyl-CoA (PubMed:4615902). In addition, may be involved in the control of the activity of the formate channel FocA, via direct interaction with FocA (PubMed:24887098)',\n",
       "   'Pathway': 'Fermentation; pyruvate fermentation; formate from pyruvate: step 1/1'},\n",
       "  'seq': 'MSELNEKLATAWEGFTKGDWQNEVNVRDFIQKNYTPYEGDESFLAGATEATTTLWDKVMEGVKLENRTHAPVDFDTAVASTITSHDAGYINKQLEKIVGLQTEAPLKRALIPFGGIKMIEGSCKAYNRELDPMIKKIFTEYRKTHNQGVFDVYTPDILRCRKSGVLTGLPDAYGRGRIIGDYRRVALYGIDYLMKDKLAQFTSLQADLENGVNLEQTIRLREEIAEQHRALGQMKEMAAKYGYDISGPATNAQEAIQWTYFGYLAAVKSQNGAAMSFGRTSTFLDVYIERDLKAGKITEQEAQEMVDHLVMKLRMVRFLRTPEYDELFSGDPIWATESIGGMGLDGRTLVTKNSFRFLNTLYTMGPSPEPNMTILWSEKLPLNFKKFAAKVSIDTSSLQYENDDLMRPDFNNDDYAIACCVSPMIVGKQMQFFGARANLAKTMLYAINGGVDEKLKMQVGPKSEPIKGDVLNYDEVMERMDHFMDWLAKQYITALNIIHYMHDKYSYEASLMALHDRDVIRTMACGIAGLSVAADSLSAIKYAKVKPIRDEDGLAIDFEIEGEYPQFGNNDPRVDDLAVDLVERFMKKIQKLHTYRDAIPTQSVLTITSNVVYGKKTGNTPDGRRAGAPFGPGANPMHGRDQKGAVASLTSVAKLPFAYAKDGISYTFSIVPNALGKDDEVRKTNLAGLMDGYFHHEASIEGGQHLNVNVMNREMLLDAMENPEKYPQLTIRVSGYAVRFNSLTKEQQQDVITRTFTQSM',\n",
       "  'mw': 85357.0},\n",
       " 'b0904': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AC23',\n",
       "  'gene': 'b0904',\n",
       "  'locus': ['b0904', 'JW0887'],\n",
       "  'synonyms': 'ycaE',\n",
       "  'properties': {'Catalytic Activity': [('formate(in) = formate(out)', '')],\n",
       "   'Function': 'Involved in the bidirectional transport of formate during mixed-acid fermentation (PubMed:8022272, PubMed:23335413, PubMed:24659605, PubMed:30247527, PubMed:33169422, PubMed:35084298, PubMed:35377837, PubMed:35390794). Functions to maintain relatively constant intracellular formate levels during growth, using different mechanisms for efflux and uptake of the anion (PubMed:35377837). Has a strong preference for formate as a substrate in vivo and not other acidic fermentation products (PubMed:23335413)',\n",
       "   'Regulation Activity': 'The efflux and influx of formate are mechanistically distinct processes that are controlled by the interplay between the conserved pore residues Thr-91 and His-209 (PubMed:35390794). The direction of formate translocation depends on external pH and electron donor source (PubMed:30247527). Interaction with PflB is important for optimal translocation of formate by FocA (PubMed:24887098). The N-terminal domain may also have a role in gating formate access (PubMed:33169422)'},\n",
       "  'seq': 'MKADNPFDLLLPAAMAKVAEEAGVYKATKHPLKTFYLAITAGVFISIAFVFYITATTGTGTMPFGMAKLVGGICFSLGLILCVVCGADLFTSTVLIVVAKASGRITWGQLAKNWLNVYFGNLVGALLFVLLMWLSGEYMTANGQWGLNVLQTADHKVHHTFIEAVCLGILANLMVCLAVWMSYSGRSLMDKAFIMVLPVAMFVASGFEHSIANMFMIPMGIVIRDFASPEFWTAVGSAPENFSHLTVMNFITDNLIPVTIGNIIGGGLLVGLTYWVIYLRENDHH',\n",
       "  'mw': 30991.0},\n",
       " 'b0907': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P23721',\n",
       "  'gene': 'b0907',\n",
       "  'locus': ['b0907', 'JW0890'],\n",
       "  'synonyms': 'pdxC pdxF',\n",
       "  'properties': {'Catalytic Activity': [('2-oxoglutarate + O-phospho-L-serine = 3-phosphooxypyruvate + L-glutamate',\n",
       "     '2.6.1.52'),\n",
       "    ('2-oxoglutarate + 4-(phosphooxy)-L-threonine = (R)-3-hydroxy-2-oxo-4-phosphooxybutanoate + L-glutamate',\n",
       "     '2.6.1.52')],\n",
       "   'Function': 'Catalyzes the reversible conversion of 3-phosphohydroxypyruvate to phosphoserine and of 3-hydroxy-2-oxo-4-phosphonooxybutanoate to phosphohydroxythreonine. Is involved in both pyridoxine and serine biosynthesis',\n",
       "   'Pathway': \"Cofactor biosynthesis; pyridoxine 5'-phosphate biosynthesis; pyridoxine 5'-phosphate from D-erythrose 4-phosphate: step 3/5\"},\n",
       "  'seq': 'MAQIFNFSSGPAMLPAEVLKQAQQELRDWNGLGTSVMEVSHRGKEFIQVAEEAEKDFRDLLNVPSNYKVLFCHGGGRGQFAAVPLNILGDKTTADYVDAGYWAASAIKEAKKYCTPNVFDAKVTVDGLRAVKPMREWQLSDNAAYMHYCPNETIDGIAIDETPDFGADVVVAADFSSTILSRPIDVSRYGVIYAGAQKNIGPAGLTIVIVREDLLGKANIACPSILDYSILNDNGSMFNTPPTFAWYLSGLVFKWLKANGGVAEMDKINQQKAELLYGVIDNSDFYRNDVAKANRSRMNVPFQLADSALDKLFLEESFAAGLHALKGHRVVGGMRASIYNAMPLEGVKALTDFMVEFERRHG',\n",
       "  'mw': 39783.0},\n",
       " 'b0908': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A6D3',\n",
       "  'gene': 'b0908',\n",
       "  'locus': ['b0908', 'JW0891'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('3-phosphoshikimate + phosphoenolpyruvate = 5-O-(1-carboxyvinyl)-3-phosphoshikimate + phosphate',\n",
       "     '2.5.1.19')],\n",
       "   'Function': 'Catalyzes the transfer of the enolpyruvyl moiety of phosphoenolpyruvate (PEP) to the 5-hydroxyl of shikimate-3-phosphate (S3P) to produce enolpyruvyl shikimate-3-phosphate and inorganic phosphate',\n",
       "   'Regulation Activity': 'Competitively inhibited by glyphosate (PubMed:12430021, PubMed:6229418). Inhibited by (S)- and (R)-phosphonates analogs (PubMed:15736934). Inhibited by (R)-difluoromethyl analogs of the tetrahedral reaction intermediate (PubMed:16225867). Inhibited by bromopyruvate (PubMed:1899181)',\n",
       "   'Pathway': 'Metabolic intermediate biosynthesis; chorismate biosynthesis; chorismate from D-erythrose 4-phosphate and phosphoenolpyruvate: step 6/7'},\n",
       "  'seq': 'MESLTLQPIARVDGTINLPGSKSVSNRALLLAALAHGKTVLTNLLDSDDVRHMLNALTALGVSYTLSADRTRCEIIGNGGPLHAEGALELFLGNAGTAMRPLAAALCLGSNDIVLTGEPRMKERPIGHLVDALRLGGAKITYLEQENYPPLRLQGGFTGGNVDVDGSVSSQFLTALLMTAPLAPEDTVIRIKGDLVSKPYIDITLNLMKTFGVEIENQHYQQFVVKGGQSYQSPGTYLVEGDASSASYFLAAAAIKGGTVKVTGIGRNSMQGDIRFADVLEKMGATICWGDDYISCTRGELNAIDMDMNHIPDAAMTIATAALFAKGTTTLRNIYNWRVKETDRLFAMATELRKVGAEVEEGHDYIRITPPEKLNFAEIATYNDHRMAMCFSLVALSDTPVTILDPKCTAKTFPDYFEQLARISQAA',\n",
       "  'mw': 46096.0},\n",
       " 'b0910': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A6I0',\n",
       "  'gene': 'b0910',\n",
       "  'locus': ['b0910', 'JW0893'],\n",
       "  'synonyms': 'mssA ycaF ycaG',\n",
       "  'properties': {'Catalytic Activity': [('ATP + CMP = ADP + CDP', '2.7.4.25'),\n",
       "    ('ATP + dCMP = ADP + dCDP', '2.7.4.25')],\n",
       "   'Function': 'ATP, dATP, and GTP are equally effective as phosphate donors. CMP and dCMP are the best phosphate acceptors'},\n",
       "  'seq': 'MTAIAPVITIDGPSGAGKGTLCKAMAEALQWHLLDSGAIYRVLALAALHHHVDVASEDALVPLASHLDVRFVSTNGNLEVILEGEDVSGEIRTQEVANAASQVAAFPRVREALLRRQRAFRELPGLIADGRDMGTVVFPDAPVKIFLDASSEERAHRRMLQLQEKGFSVNFERLLAEIKERDDRDRNRAVAPLVPAADALVLDSTTLSIEQVIEKALQYARQKLALA',\n",
       "  'mw': 24746.0},\n",
       " 'b0914': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P60752',\n",
       "  'gene': 'b0914',\n",
       "  'locus': ['b0914', 'JW0897'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + H2O + lipid A-core oligosaccharideSide 1 = ADP + phosphate + lipid A-core oligosaccharideSide 2.',\n",
       "     '7.5.2.6')],\n",
       "   'Function': 'Involved in lipopolysaccharide (LPS) biosynthesis. Translocates lipid A-core from the inner to the outer leaflet of the inner membrane (PubMed:8809774, PubMed:9575204, PubMed:12119303, PubMed:15304478, PubMed:28869968). Transmembrane domains (TMD) form a pore in the inner membrane and the ATP-binding domain (NBD) is responsible for energy generation (PubMed:12119303). Shows ATPase activity (PubMed:12119303, PubMed:18024585, PubMed:18344567, PubMed:19132955, PubMed:20412049, PubMed:21462989). May transport glycerophospholipids (PubMed:9575204). In proteoliposomes, mediates the ATP-dependent flipping of a variety of phospholipid and glycolipid derivatives (PubMed:20412049). May also function as a multidrug transporter (PubMed:19132955)',\n",
       "   'Regulation Activity': 'ATPase activity is stimulated by lipid A, hexaacylated lipid A or Kdo(2)-lipid A (PubMed:12119303, PubMed:18344567). Inhibited by the phosphate analog vanadate (PubMed:12119303, PubMed:18344567, PubMed:28869968). ATPase activity is also modulated by the lipid-based drugs D-20133 and D-21266, along with LY335979 and leupeptin (PubMed:18344567)'},\n",
       "  'seq': 'MHNDKDLSTWQTFRRLWPTIAPFKAGLIVAGVALILNAASDTFMLSLLKPLLDDGFGKTDRSVLVWMPLVVIGLMILRGITSYVSSYCISWVSGKVVMTMRRRLFGHMMGMPVSFFDKQSTGTLLSRITYDSEQVASSSSGALITVVREGASIIGLFIMMFYYSWQLSIILIVLAPIVSIAIRVVSKRFRNISKNMQNTMGQVTTSAEQMLKGHKEVLIFGGQEVETKRFDKVSNRMRLQGMKMVSASSISDPIIQLIASLALAFVLYAASFPSVMDSLTAGTITVVFSSMIALMRPLKSLTNVNAQFQRGMAACQTLFTILDSEQEKDEGKRVIERATGDVEFRNVTFTYPGRDVPALRNINLKIPAGKTVALVGRSGSGKSTIASLITRFYDIDEGEILMDGHDLREYTLASLRNQVALVSQNVHLFNDTVANNIAYARTEQYSREQIEEAARMAYAMDFINKMDNGLDTVIGENGVLLSGGQRQRIAIARALLRDSPILILDEATSALDTESERAIQAALDELQKNRTSLVIAHRLSTIEKADEIVVVEDGVIVERGTHNDLLEHRGVYAQLHKMQFGQ',\n",
       "  'mw': 64461.0},\n",
       " 'b0915': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P27300',\n",
       "  'gene': 'b0915',\n",
       "  'locus': ['b0915', 'JW0898'],\n",
       "  'synonyms': 'ycaH',\n",
       "  'properties': {'Catalytic Activity': [('ATP + lipid A disaccharide (E. coli) = ADP + H(+) + lipid IVA (E. coli)',\n",
       "     '2.7.1.130')],\n",
       "   'Function': \"Transfers the gamma-phosphate of ATP to the 4'-position of a tetraacyldisaccharide 1-phosphate intermediate (termed DS-1-P) to form tetraacyldisaccharide 1,4'-bis-phosphate (lipid IVA)\",\n",
       "   'Pathway': 'Glycolipid biosynthesis; lipid IV(A) biosynthesis; lipid IV(A) from (3R)-3-hydroxytetradecanoyl-[acyl-carrier-protein] and UDP-N-acetyl-alpha-D-glucosamine: step 6/6'},\n",
       "  'seq': 'MIEKIWSGESPLWRLLLPLSWLYGLVSGAIRLCYKLKLKRAWRAPVPVVVVGNLTAGGNGKTPVVVWLVEQLQQRGIRVGVVSRGYGGKAESYPLLLSADTTTAQAGDEPVLIYQRTDAPVAVSPVRSDAVKAILAQHPDVQIIVTDDGLQHYRLARDVEIVVIDGVRRFGNGWWLPAGPMRERAGRLKSVDAVIVNGGVPRSGEIPMHLLPGQAVNLRTGTRCDVAQLEHVVAMAGIGHPPRFFATLKMCGVQPEKCVPLADHQSLNHADVSALVSAGQTLVMTEKDAVKCRAFAEENWWYLPVDAQLSGDEPAKLLTQLTLLASGN',\n",
       "  'mw': 35589.0},\n",
       " 'b0918': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P04951',\n",
       "  'gene': 'b0918',\n",
       "  'locus': ['b0918', 'JW0901'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('3-deoxy-alpha-D-manno-oct-2-ulosonate + CTP = CMP-3-deoxy-beta-D-manno-octulosonate + diphosphate',\n",
       "     '2.7.7.38')],\n",
       "   'Function': 'Activates KDO (a required 8-carbon sugar) for incorporation into bacterial lipopolysaccharide in Gram-negative bacteria',\n",
       "   'Pathway': 'Bacterial outer membrane biogenesis; lipopolysaccharide biosynthesis'},\n",
       "  'seq': 'MSFVVIIPARYASTRLPGKPLVDINGKPMIVHVLERARESGAERIIVATDHEDVARAVEAAGGEVCMTRADHQSGTERLAEVVEKCAFSDDTVIVNVQGDEPMIPATIIRQVADNLAQRQVGMATLAVPIHNAEEAFNPNAVKVVLDAEGYALYFSRATIPWDRDRFAEGLETVGDNFLRHLGIYGYRAGFIRRYVNWQPSPLEHIEMLEQLRVLWYGEKIHVAVAQEVPGTGVDTPEDLERVRAEMR',\n",
       "  'mw': 27614.0},\n",
       " 'b0928': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P00509',\n",
       "  'gene': 'b0928',\n",
       "  'locus': ['b0928', 'JW0911'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('2-oxoglutarate + L-aspartate = L-glutamate + oxaloacetate',\n",
       "     '2.6.1.1')]},\n",
       "  'seq': 'MFENITAAPADPILGLADLFRADERPGKINLGIGVYKDETGKTPVLTSVKKAEQYLLENETTKNYLGIDGIPEFGRCTQELLFGKGSALINDKRARTAQTPGGTGALRVAADFLAKNTSVKRVWVSNPSWPNHKSVFNSAGLEVREYAYYDAENHTLDFDALINSLNEAQAGDVVLFHGCCHNPTGIDPTLEQWQTLAQLSVEKGWLPLFDFAYQGFARGLEEDAEGLRAFAAMHKELIVASSYSKNFGLYNERVGACTLVAADSETVDRAFSQMKAAIRANYSNPPAHGASVVATILSNDALRAIWEQELTDMRQRIQRMRQLFVNTLQEKGANRDFSFIIKQNGMFSFSGLTKEQVLRLREEFGVYAVASGRVNVAGMTPDNMAPLCEAIVAVL',\n",
       "  'mw': 43573.0},\n",
       " 'b0929': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P02931',\n",
       "  'gene': 'b0929',\n",
       "  'locus': ['b0929', 'JW0912'],\n",
       "  'synonyms': 'cmlB coa cry tolF',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': '(Microbial infection) A mixed OmpC-OmpF heterotrimer is the outer membrane receptor for toxin CdiA-EC536; polymorphisms in extracellular loops 4 and 5 of OmpC confer susceptibility to CdiA-EC536-mediated toxicity'},\n",
       "  'seq': 'MMKRNILAVIVPALLVAGTANAAEIYNKDGNKVDLYGKAVGLHYFSKGNGENSYGGNGDMTYARLGFKGETQINSDLTGYGQWEYNFQGNNSEGADAQTGNKTRLAFAGLKYADVGSFDYGRNYGVVYDALGYTDMLPEFGGDTAYSDDFFVGRVGGVATYRNSNFFGLVDGLNFAVQYLGKNERDTARRSNGDGVGGSISYEYEGFGIVGAYGAADRTNLQEAQPLGNGKKAEQWATGLKYDANNIYLAANYGETRNATPITNKFTNTSGFANKTQDVLLVAQYQFDFGLRPSIAYTKSKAKDVEGIGDVDLVNYFEVGATYYFNKNMSTYVDYIINQIDSDNKLGVGSDDTVAVGIVYQF',\n",
       "  'mw': 39333.0},\n",
       " 'b0930': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A8M0',\n",
       "  'gene': 'b0930',\n",
       "  'locus': ['b0930', 'JW0913'],\n",
       "  'synonyms': 'tss',\n",
       "  'properties': {'Catalytic Activity': [('ATP + L-asparagine + tRNA(Asn) = AMP + diphosphate + H(+) + L-asparaginyl-tRNA(Asn)',\n",
       "     '6.1.1.22')]},\n",
       "  'seq': 'MSVVPVADVLQGRVAVDSEVTVRGWVRTRRDSKAGISFLAVYDGSCFDPVQAVINNSLPNYNEDVLRLTTGCSVIVTGKVVASPGQGQQFEIQASKVEVAGWVEDPDTYPMAAKRHSIEYLREVAHLRPRTNLIGAVARVRHTLAQALHRFFNEQGFFWVSTPLITASDTEGAGEMFRVSTLDLENLPRNDQGKVDFDKDFFGKESFLTVSGQLNGETYACALSKIYTFGPTFRAENSNTSRHLAEFWMLEPEVAFANLNDIAGLAEAMLKYVFKAVLEERADDMKFFAERVDKDAVSRLERFIEADFAQVDYTDAVTILENCGRKFENPVYWGVDLSSEHERYLAEEHFKAPVVVKNYPKDIKAFYMRLNEDGKTVAAMDVLAPGIGEIIGGSQREERLDVLDERMLEMGLNKEDYWWYRDLRRYGTVPHSGFGLGFERLIAYVTGVQNVRDVIPFPRTPRNASF',\n",
       "  'mw': 52570.0},\n",
       " 'b0931': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P18133',\n",
       "  'gene': 'b0931',\n",
       "  'locus': ['b0931', 'JW0914'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('5-phospho-alpha-D-ribose 1-diphosphate + ATP + H2O + nicotinate = ADP + diphosphate + nicotinate beta-D-ribonucleotide + phosphate',\n",
       "     '6.3.4.21')],\n",
       "   'Function': 'Catalyzes the synthesis of beta-nicotinate D-ribonucleotide from nicotinate and 5-phospho-D-ribose 1-phosphate at the expense of ATP',\n",
       "   'Pathway': 'Cofactor biosynthesis; NAD(+) biosynthesis; nicotinate D-ribonucleotide from nicotinate: step 1/1'},\n",
       "  'seq': 'MTQFASPVLHSLLDTDAYKLHMQQAVFHHYYDVHVAAEFRCRGDDLLGIYADAIREQVQAMQHLRLQDDEYQWLSALPFFKADYLNWLREFRFNPEQVTVSNDNGKLDIRLSGPWREVILWEVPLLAVISEMVHRYRSPQADVAQALDTLESKLVDFSALTAGLDMSRFHLMDFGTRRRFSREVQETIVKRLQQESWFVGTSNYDLARRLSLTPMGTQAHEWFQAHQQISPDLANSQRAALAAWLEEYPDQLGIALTDCITMDAFLRDFGVEFASRYQGLRHDSGDPVEWGEKAIAHYEKLGIDPQSKTLVFSDNLDLRKAVELYRHFSSRVQLSFGIGTRLTCDIPQVKPLNIVIKLVECNGKPVAKLSDSPGKTICHDKAFVRALRKAFDLPHIKKAS',\n",
       "  'mw': 45897.0},\n",
       " 'b0932': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P04825',\n",
       "  'gene': 'b0932',\n",
       "  'locus': ['b0932', 'JW0915'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('Release of an N-terminal amino acid, Xaa-|-Yaa- from a peptide, amide or arylamide. Xaa is preferably Ala, but may be most amino acids including Pro (slow action). When a terminal hydrophobic residue is followed by a prolyl residue, the two may be released as an intact Xaa-Pro dipeptide.',\n",
       "     '3.4.11.2')],\n",
       "   'Function': 'Aminopeptidase N is involved in the degradation of intracellular peptides generated by protein breakdown during normal growth as well as in response to nutrient starvation'},\n",
       "  'seq': 'MTQQPQAKYRHDYRAPDYQITDIDLTFDLDAQKTVVTAVSQAVRHGASDAPLRLNGEDLKLVSVHINDEPWTAWKEEEGALVISNLPERFTLKIINEISPAANTALEGLYQSGDALCTQCEAEGFRHITYYLDRPDVLARFTTKIIADKIKYPFLLSNGNRVAQGELENGRHWVQWQDPFPKPCYLFALVAGDFDVLRDTFTTRSGREVALELYVDRGNLDRAPWAMTSLKNSMKWDEERFGLEYDLDIYMIVAVDFFNMGAMENKGLNIFNSKYVLARTDTATDKDYLDIERVIGHEYFHNWTGNRVTCRDWFQLSLKEGLTVFRDQEFSSDLGSRAVNRINNVRTMRGLQFAEDASPMAHPIRPDMVIEMNNFYTLTVYEKGAEVIRMIHTLLGEENFQKGMQLYFERHDGSAATCDDFVQAMEDASNVDLSHFRRWYSQSGTPIVTVKDDYNPETEQYTLTISQRTPATPDQAEKQPLHIPFAIELYDNEGKVIPLQKGGHPVNSVLNVTQAEQTFVFDNVYFQPVPALLCEFSAPVKLEYKWSDQQLTFLMRHARNDFSRWDAAQSLLATYIKLNVARHQQGQPLSLPVHVADAFRAVLLDEKIDPALAAEILTLPSVNEMAELFDIIDPIAIAEVREALTRTLATELADELLAIYNANYQSEYRVEHEDIAKRTLRNACLRFLAFGETHLADVLVSKQFHEANNMTDALAALSAAVAAQLPCRDALMQEYDDKWHQNGLVMDKWFILQATSPAANVLETVRGLLQHRSFTMSNPNRIRSLIGAFAGSNPAAFHAEDGSGYLFLVEMLTDLNSRNPQVASRLIEPLIRLKRYDAKRQEKMRAALEQLKGLENLSGDLYEKITKALA',\n",
       "  'mw': 98919.0},\n",
       " 'b0933': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AAI1',\n",
       "  'gene': 'b0933',\n",
       "  'locus': ['b0933', 'JW0916'],\n",
       "  'synonyms': 'ycbE',\n",
       "  'properties': {'Catalytic Activity': [('ATP + H2O + aliphatic sulfonate-[sulfonate-binding protein]Side 1 = ADP + phosphate + aliphatic sulfonateSide 2 + [sulfonate-binding protein]Side 1.',\n",
       "     '7.6.2.14')],\n",
       "   'Function': 'Part of the ABC transporter complex SsuABC involved in aliphatic sulfonates import. Responsible for energy coupling to the transport system (Probable)'},\n",
       "  'seq': 'MNTARLNQGTPLLLNAVSKHYAENIVLNQLDLHIPAGQFVAVVGRSGGGKSTLLRLLAGLETPTAGDVLAGTTPLAEIQEDTRMMFQDARLLPWKSVIDNVGLGLKGQWRDAARRALAAVGLENRAGEWPAALSGGQKQRVALARALIHRPGLLLLDEPLGALDALTRLEMQDLIVSLWQEHGFTVLLVTHDVSEAVAMADRVLLIEEGKIGLDLTVDIPRPRRLGSVRLAELEAEVLQRVMQRGESETRLRKQG',\n",
       "  'mw': 27738.0},\n",
       " 'b0934': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P75851',\n",
       "  'gene': 'b0934',\n",
       "  'locus': ['b0934', 'JW5121'],\n",
       "  'synonyms': 'ycbM',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of a binding-protein-dependent transport system for aliphatic sulfonates. Probably responsible for the translocation of the substrate across the membrane'},\n",
       "  'seq': 'MATPVKKWLLRVAPWFLPVGIVAVWQLASSVGWLSTRILPSPEGVVTAFWTLSASGELWQHLAISSWRALIGFSIGGSLGLILGLISGLSRWGERLLDTSIQMLRNVPHLALIPLVILWFGIDESAKIFLVALGTLFPIYINTWHGIRNIDRGLVEMARSYGLSGIPLFIHVILPGALPSIMVGVRFALGLMWLTLIVAETISANSGIGYLAMNAREFLQTDVVVVAIILYALLGKLADVSAQLLERLWLRWNPAYHLKEATV',\n",
       "  'mw': 28925.0},\n",
       " 'b0935': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P80645',\n",
       "  'gene': 'b0935',\n",
       "  'locus': ['b0935', 'JW0918'],\n",
       "  'synonyms': 'ycbN',\n",
       "  'properties': {'Catalytic Activity': [('an alkanesulfonate + FMNH2 + O2 = an aldehyde + FMN + 2 H(+) + H2O + sulfite',\n",
       "     '1.14.14.5')],\n",
       "   'Function': 'Involved in desulfonation of aliphatic sulfonates. Catalyzes the conversion of pentanesulfonate to sulfite and pentaldehyde and is able to desulfonate a wide range of sulfonated substrates including C-2 to C-10 unsubstituted linear alkanesulfonates, substituted ethanesulfonic acids and sulfonated buffers'},\n",
       "  'seq': 'MSLNMFWFLPTHGDGHYLGTEEGSRPVDHGYLQQIAQAADRLGYTGVLIPTGRSCEDAWLVAASMIPVTQRLKFLVALRPSVTSPTVAARQAATLDRLSNGRALFNLVTGSDPQELAGDGVFLDHSERYEASAEFTQVWRRLLQRETVDFNGKHIHVRGAKLLFPAIQQPYPPLYFGGSSDVAQELAAEQVDLYLTWGEPPELVKEKIEQVRAKAAAHGRKIRFGIRLHVIVRETNDEAWQAAERLISHLDDETIAKAQAAFARTDSVGQQRMAALHNGKRDNLEISPNLWAGVGLVRGGAGTALVGDGPTVAARINEYAALGIDSFVLSGYPHLEEAYRVGELLFPLLDVAIPEIPQPQPLNPQGEAVANDFIPRKVAQS',\n",
       "  'mw': 41736.0},\n",
       " 'b0936': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P75853',\n",
       "  'gene': 'b0936',\n",
       "  'locus': ['b0936', 'JW0919'],\n",
       "  'synonyms': 'ycbO',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of a binding-protein-dependent transport system for aliphatic sulfonates. Putative binding protein'},\n",
       "  'seq': 'MRNIIKLALAGLLSVSTFAVAAESSPEALRIGYQKGSIGMVLAKSHQLLEKRYPESKISWVEFPAGPQMLEALNVGSIDLGSTGDIPPIFAQAAGADLVYVGVEPPKPKAEVILVAENSPIKTVADLKGHKVAFQKGSSSHNLLLRALRQAGLKFTDIQPTYLTPADARAAFQQGNVDAWAIWDPYYSAALLQGGVRVLKDGTDLNQTGSFYLAARPYAEKNGAFIQGVLATFSEADALTRSQREQSIALLAKTMGLPAPVIASYLDHRPPTTIKPVNAEVAALQQQTADLFYENRLVPKKVDIRQRIWQPTQLEGKQL',\n",
       "  'mw': 34558.0},\n",
       " 'b0937': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P80644',\n",
       "  'gene': 'b0937',\n",
       "  'locus': ['b0937', 'JW0920'],\n",
       "  'synonyms': 'ycbP',\n",
       "  'properties': {'Catalytic Activity': [('FMNH2 + NADP(+) = FMN + 2 H(+) + NADPH',\n",
       "     '1.5.1.38')],\n",
       "   'Function': 'Catalyzes an NADPH-dependent reduction of FMN, but is also able to reduce FAD or riboflavin'},\n",
       "  'seq': 'MRVITLAGSPRFPSRSSSLLEYAREKLNGLDVEVYHWNLQNFAPEDLLYARFDSPALKTFTEQLQQADGLIVATPVYKAAYSGALKTLLDLLPERALQGKVVLPLATGGTVAHLLAVDYALKPVLSALKAQEILHGVFADDSQVIDYHHRPQFTPNLQTRLDTALETFWQALHRRDVQVPDLLSLRGNAHA',\n",
       "  'mw': 21253.0},\n",
       " 'b0945': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A7E1',\n",
       "  'gene': 'b0945',\n",
       "  'locus': ['b0945', 'JW0928'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(S)-dihydroorotate + a quinone = a quinol + orotate',\n",
       "     '1.3.5.2')],\n",
       "   'Function': 'Catalyzes the conversion of dihydroorotate to orotate with quinone as electron acceptor',\n",
       "   'Pathway': 'Pyrimidine metabolism; UMP biosynthesis via de novo pathway; orotate from (S)-dihydroorotate (quinone route): step 1/1'},\n",
       "  'seq': 'MYYPFVRKALFQLDPERAHEFTFQQLRRITGTPFEALVRQKVPAKPVNCMGLTFKNPLGLAAGLDKDGECIDALGAMGFGSIEIGTVTPRPQPGNDKPRLFRLVDAEGLINRMGFNNLGVDNLVENVKKAHYDGVLGINIGKNKDTPVEQGKDDYLICMEKIYAYAGYIAINISSPNTPGLRTLQYGEALDDLLTAIKNKQNDLQAMHHKYVPIAVKIAPDLSEEELIQVADSLVRHNIDGVIATNTTLDRSLVQGMKNCDQTGGLSGRPLQLKSTEIIRRLSLELNGRLPIIGVGGIDSVIAAREKIAAGASLVQIYSGFIFKGPPLIKEIVTHI',\n",
       "  'mw': 36775.0},\n",
       " 'b0954': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A6Q3',\n",
       "  'gene': 'b0954',\n",
       "  'locus': ['b0954', 'JW0937'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('a (3R)-hydroxyacyl-[ACP] = a (2E)-enoyl-[ACP] + H2O',\n",
       "     '4.2.1.59'),\n",
       "    ('(3R)-hydroxydecanoyl-[ACP] = (2E)-decenoyl-[ACP] + H2O', ''),\n",
       "    ('(2E)-decenoyl-[ACP] = (3Z)-decenoyl-[ACP]', '5.3.3.14'),\n",
       "    ('(3R)-hydroxybutanoyl-[ACP] = (2E)-butenoyl-[ACP] + H2O', ''),\n",
       "    ('(3R)-hydroxyhexanoyl-[ACP] = (2E)-hexenoyl-[ACP] + H2O', ''),\n",
       "    ('(3R)-hydroxyoctanoyl-[ACP] = (2E)-octenoyl-[ACP] + H2O', ''),\n",
       "    ('(3R)-hydroxydodecanoyl-[ACP] = (2E)-dodecenoyl-[ACP] + H2O', ''),\n",
       "    ('(3R)-hydroxytetradecanoyl-[ACP] = (2E)-tetradecenoyl-[ACP] + H2O', ''),\n",
       "    ('(3R)-hydroxyhexadecanoyl-[ACP] = (2E)-hexadecenoyl-[ACP] + H2O', ''),\n",
       "    ('(3R)-hydroxy-5-methylhexanoyl-[ACP] = (2E)-5-methylhexenoyl-[ACP] + H2O',\n",
       "     '')],\n",
       "   'Function': 'Necessary for the introduction of cis unsaturation into fatty acids (PubMed:8910376). Catalyzes the dehydration of (3R)-3-hydroxydecanoyl-ACP to E-(2)-decenoyl-ACP and then its isomerization to Z-(3)-decenoyl-ACP (PubMed:8910376). Can catalyze the dehydratase reaction for beta-hydroxyacyl-ACPs with saturated chain lengths up to 16:0, being most active on intermediate chain length (PubMed:8910376, PubMed:7592873, PubMed:10629181). Is inactive in the dehydration of long chain unsaturated beta-hydroxyacyl-ACP (PubMed:8910376)',\n",
       "   'Regulation Activity': 'Irreversibly inactivated by 3-decynoyl-N-acetylcysteamine (DNAC) which binds to the active site and forms an adduct (PubMed:21276098)',\n",
       "   'Pathway': 'Lipid metabolism; fatty acid biosynthesis'},\n",
       "  'seq': 'MVDKRESYTKEDLLASGRGELFGAKGPQLPAPNMLMMDRVVKMTETGGNFDKGYVEAELDINPDLWFFGCHFIGDPVMPGCLGLDAMWQLVGFYLGWLGGEGKGRALGVGEVKFTGQVLPTAKKVTYRIHFKRIVNRRLIMGLADGEVLVDGRLIYTASDLKVGLFQDTSAF',\n",
       "  'mw': 18969.0},\n",
       " 'b0957': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A910',\n",
       "  'gene': 'b0957',\n",
       "  'locus': ['b0957', 'JW0940'],\n",
       "  'synonyms': 'con tolG tut',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': '(Microbial infection) A mutation in this locus (called tolG) renders the cell tolerant to bacteriocin JF246 but does not affect its sensitivity to colicins A, C, El, E2, E3, K, Ia, or Ib (PubMed:4591955). Mutations in this protein render the bacteria partially or completely susceptible to colicin K or colicin L, for which is it probably the receptor (PubMed:6086577)'},\n",
       "  'seq': 'MKKTAIAIAVALAGFATVAQAAPKDNTWYTGAKLGWSQYHDTGFINNNGPTHENQLGAGAFGGYQVNPYVGFEMGYDWLGRMPYKGSVENGAYKAQGVQLTAKLGYPITDDLDIYTRLGGMVWRADTKSNVYGKNHDTGVSPVFAGGVEYAITPEIATRLEYQWTNNIGDAHTIGTRPDNGMLSLGVSYRFGQGEAAPVVAPAPAPAPEVQTKHFTLKSDVLFNFNKATLKPEGQAALDQLYSQLSNLDPKDGSVVVLGYTDRIGSDAYNQGLSERRAQSVVDYLISKGIPADKISARGMGESNPVTGNTCDNVKQRAALIDCLAPDRRVEIEVKGIKDVVTQPQA',\n",
       "  'mw': 37201.0},\n",
       " 'b0963': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A731',\n",
       "  'gene': 'b0963',\n",
       "  'locus': ['b0963', 'JW5129'],\n",
       "  'synonyms': 'yccG',\n",
       "  'properties': {'Catalytic Activity': [('dihydroxyacetone phosphate = methylglyoxal + phosphate',\n",
       "     '4.2.3.3')],\n",
       "   'Function': 'Catalyzes the formation of methylglyoxal from dihydroxyacetone phosphate',\n",
       "   'Regulation Activity': 'Inhibited by inorganic phosphate ions (PubMed:9665712, PubMed:15049687). Competitively inhibited by phosphoglycolate (PubMed:15049687). Competitively inhibited by phosphoglycolohydroxamate (PubMed:11389594)'},\n",
       "  'seq': 'MELTTRTLPARKHIALVAHDHCKQMLMSWVERHQPLLEQHVLYATGTTGNLISRATGMNVNAMLSGPMGGDQQVGALISEGKIDVLIFFWDPLNAVPHDPDVKALLRLATVWNIPVATNVATADFIIQSPHFNDAVDILIPDYQRYLADRLK',\n",
       "  'mw': 16919.0},\n",
       " 'b0972': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P69739',\n",
       "  'gene': 'b0972',\n",
       "  'locus': ['b0972', 'JW0954'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('A + H2 = AH2', '1.12.99.6')],\n",
       "   'Function': 'This is one of three E.coli hydrogenases synthesized in response to different physiological conditions. HYD1 is believed to have a role in hydrogen cycling during fermentative growth'},\n",
       "  'seq': 'MNNEETFYQAMRRQGVTRRSFLKYCSLAATSLGLGAGMAPKIAWALENKPRIPVVWIHGLECTCCTESFIRSAHPLAKDVILSLISLDYDDTLMAAAGTQAEEVFEDIITQYNGKYILAVEGNPPLGEQGMFCISSGRPFIEKLKRAAAGASAIIAWGTCASWGCVQAARPNPTQATPIDKVITDKPIIKVPGCPPIPDVMSAIITYMVTFDRLPDVDRMGRPLMFYGQRIHDKCYRRAHFDAGEFVQSWDDDAARKGYCLYKMGCKGPTTYNACSSTRWNDGVSFPIQSGHGCLGCAENGFWDRGSFYSRVVDIPQMGTHSTADTVGLTALGVVAAAVGVHAVASAVDQRRRHNQQPTETEHQPGNEDKQA',\n",
       "  'mw': 40681.0},\n",
       " 'b0973': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ACD8',\n",
       "  'gene': 'b0973',\n",
       "  'locus': ['b0973', 'JW0955'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('A + H2 = AH2', '1.12.99.6')],\n",
       "   'Function': 'This is one of three E.coli hydrogenases synthesized in response to different physiological conditions. HYD1 is believed to have a role in hydrogen cycling during fermentative growth'},\n",
       "  'seq': 'MSTQYETQGYTINNAGRRLVVDPITRIEGHMRCEVNINDQNVITNAVSCGTMFRGLEIILQGRDPRDAWAFVERICGVCTGVHALASVYAIEDAIGIKVPDNANIIRNIMLATLWCHDHLVHFYQLAGMDWIDVLDALKADPRKTSELAQSLSSWPKSSPGYFFDVQNRLKKFVEGGQLGIFRNGYWGHPQYKLPPEANLMGFAHYLEALDFQREIVKIHAVFGGKNPHPNWIVGGMPCAINIDESGAVGAVNMERLNLVQSIITRTADFINNVMIPDALAIGQFNKPWSEIGTGLSDKCVLSYGAFPDIANDFGEKSLLMPGGAVINGDFNNVLPVDLVDPQQVQEFVDHAWYRYPNDQVGRHPFDGITDPWYNPGDVKGSDTNIQQLNEQERYSWIKAPRWRGNAMEVGPLARTLIAYHKGDAATVESVDRMMSALNLPLSGIQSTLGRILCRAHEAQWAAGKLQYFFDKLMTNLKNGNLATASTEKWEPATWPTECRGVGFTEAPRGALGHWAAIRDGKIDLYQCVVPTTWNASPRDPKGQIGAYEAALMNTKMAIPEQPLEILRTLHSFDPCLACSTHVLGDDGSELISVQVR',\n",
       "  'mw': 66253.0},\n",
       " 'b0974': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AAM1',\n",
       "  'gene': 'b0974',\n",
       "  'locus': ['b0974', 'JW0956'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Probable b-type cytochrome'},\n",
       "  'seq': 'MQQKSDNVVSHYVFEAPVRIWHWLTVLCMAVLMVTGYFIGKPLPSVSGEATYLFYMGYIRLIHFSAGMVFTVVLLMRIYWAFVGNRYSRELFIVPVWRKSWWQGVWYEIRWYLFLAKRPSADIGHNPIAQAAMFGYFLMSVFMIITGFALYSEHSQYAIFAPFRYVVEFFYWTGGNSMDIHSWHRLGMWLIGAFVIGHVYMALREDIMSDDTVISTMVNGYRSHKFGKISNKERS',\n",
       "  'mw': 27597.0},\n",
       " 'b0978': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P26459',\n",
       "  'gene': 'b0978',\n",
       "  'locus': ['b0978', 'JW0960'],\n",
       "  'synonyms': 'cbdA cyxA',\n",
       "  'properties': {'Catalytic Activity': [('2 a ubiquinol + n H(+)(in) + O2 = 2 a ubiquinone + n H(+)(out) + 2 H2O',\n",
       "     '7.1.1.3')],\n",
       "   'Function': 'A terminal oxidase that catalyzes quinol-dependent, Na(+)-independent oxygen uptake. Prefers menadiol over other quinols although ubiquinol was not tested (PubMed:8626304). Generates a proton motive force using protons and electrons from opposite sides of the membrane to generate H(2)O, transferring 1 proton/electron',\n",
       "   'Regulation Activity': 'Inhibited by cyanide; is more sensitive to cyanide than cytochrome bd-I oxidase',\n",
       "   'Pathway': 'Energy metabolism; oxidative phosphorylation'},\n",
       "  'seq': 'MWDVIDLSRWQFALTALYHFLFVPLTLGLIFLLAIMETIYVVTGKTIYRDMTRFWGKLFGINFALGVATGLTMEFQFGTNWSFYSNYVGDIFGAPLAMEALMAFFLESTFVGLFFFGWQRLNKYQHLLVTWLVAFGSNLSALWILNANGWMQYPTGAHFDIDTLRMEMTSFSELVFNPVSQVKFVHTVMAGYVTGAMFIMAISAWYLLRGRERNVALRSFAIGSVFGTLAIIGTLQLGDSSAYEVAQVQPVKLAAMEGEWQTEPAPAPFHVVAWPEQDQERNAFALKIPALLGILATHSLDKPVPGLKNLMAETYPRLQRGRMAWLLMQEISQGNREPHVLQAFRGLEGDLGYGMLLSRYAPDMNHVTAAQYQAAMRGAIPQVAPVFWSFRIMVGCGSLLLLVMLIALVQTLRGKIDQHRWVLKMALWSLPLPWIAIEAGWFMTEFGRQPWAIQDILPTYSAHSALTTGQLAFSLIMIVGLYTLFLIAEVYLMQKYARLGPSAMQSEQPTQQQG',\n",
       "  'mw': 57920.0},\n",
       " 'b0979': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P26458',\n",
       "  'gene': 'b0979',\n",
       "  'locus': ['b0979', 'JW0961'],\n",
       "  'synonyms': 'cbdB cyxB',\n",
       "  'properties': {'Catalytic Activity': [('2 a ubiquinol + n H(+)(in) + O2 = 2 a ubiquinone + n H(+)(out) + 2 H2O',\n",
       "     '7.1.1.3')],\n",
       "   'Function': 'A terminal oxidase that catalyzes quinol-dependent, Na(+)-independent oxygen uptake. Prefers menadiol over other quinols although ubiquinol was not tested (PubMed:8626304). Generates a proton motive force using protons and electrons from opposite sides of the membrane to generate H(2)O, transferring 1 proton/electron',\n",
       "   'Regulation Activity': 'Inhibited by cyanide; is more sensitive to cyanide than cytochrome bd-I oxidase',\n",
       "   'Pathway': 'Energy metabolism; oxidative phosphorylation'},\n",
       "  'seq': 'MFDYETLRFIWWLLIGVILVVFMISDGFDMGIGCLLPLVARNDDERRIVINSVGAHWEGNQVWLILAGGALFAAWPRVYAAAFSGFYVAMILVLCSLFFRPLAFDYRGKIADARWRKMWDAGLVIGSLVPPVVFGIAFGNLLLGVPFAFTPQLRVEYLGSFWQLLTPFPLLCGLLSLGMVILQGGVWLQLKTVGVIHLRSQLATKRAALLVMLCFLLAGYWLWVGIDGFVLLAQDANGPSNPLMKLVAVLPGAWMNNFVESPVLWIFPLLGFFCPLLTVMAIYRGRPGWGFLMASLMQFGVIFTAGITLFPFVMPSSVSPISSLTLWDSTSSQLTLSIMLVIVLIFLPIVLLYTLWSYYKMWGRMTTETLRRNENELY',\n",
       "  'mw': 42424.0},\n",
       " 'b0980': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P07102',\n",
       "  'gene': 'b0980',\n",
       "  'locus': ['b0980', 'JW0963'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('1D-myo-inositol hexakisphosphate + H2O = 1D-myo-inositol 1,2,3,4,5-pentakisphosphate + phosphate',\n",
       "     ''),\n",
       "    ('1D-myo-inositol 1,2,3,4,5-pentakisphosphate + H2O = 1D-myo-inositol 2,3,4,5-tetrakisphosphate + phosphate',\n",
       "     ''),\n",
       "    ('1D-myo-inositol 2,3,4,5-tetrakisphosphate + H2O = 1D-myo-inositol 2,4,5-triphosphate + phosphate',\n",
       "     ''),\n",
       "    ('1D-myo-inositol 2,4,5-triphosphate + H2O = 1D-myo-inositol 2,5-bisphosphate + phosphate',\n",
       "     ''),\n",
       "    ('1D-myo-inositol 2,5-bisphosphate + H2O = 1D-myo-inositol 2-phosphate + phosphate',\n",
       "     ''),\n",
       "    ('GTP + H2O = GDP + H(+) + phosphate', '')],\n",
       "   'Function': \"Catalyzes the hydrolysis of phytate (or myo-inositol hexakisphosphate, an indigestible organic form of phosphorus that is found in many plant tissues) to myo-inositol and inorganic phosphate (PubMed:8387749, Ref.8, PubMed:10696472, PubMed:11035187, PubMed:30712472). Dephosphorylates phytate in a stereospecific way by sequential removal of phosphate groups to produce myo-inositol 2-monophosphate (PubMed:11035187). Also shows phosphoanhydride phosphatase activity and hydrolyzes the distal phosphoryl residues of GTP, the 5'-beta-phosphoryl residue of the regulatory nucleotide ppGpp and tripolyphosphates (PubMed:6282821, PubMed:1429631, PubMed:8407904). Does not split most phosphomonoesters with the exception of the synthetic substrate p-nitrophenyl phosphate (pNPP), 2,3-bisphosphoglycerate and fructose 1,6-bisphosphate (PubMed:6282821, Ref.8, PubMed:1429631, PubMed:8387749, PubMed:8407904, PubMed:10696472)\",\n",
       "   'Regulation Activity': 'Contains three consecutive and one non-consecutive disulfide bonds and shows a strong dependence on DsbC for its full activity (PubMed:15642731). Competitively inhibited by tartaric acid and by sodium fluorid (PubMed:6282821, PubMed:8387749)'},\n",
       "  'seq': 'MKAILIPFLSLLIPLTPQSAFAQSEPELKLESVVIVSRHGVRAPTKATQLMQDVTPDAWPTWPVKLGWLTPRGGELIAYLGHYQRQRLVADGLLAKKGCPQSGQVAIIADVDERTRKTGEAFAAGLAPDCAITVHTQADTSSPDPLFNPLKTGVCQLDNANVTDAILSRAGGSIADFTGHRQTAFRELERVLNFPQSNLCLKREKQDESCSLTQALPSELKVSADNVSLTGAVSLASMLTEIFLLQQAQGMPEPGWGRITDSHQWNTLLSLHNAQFYLLQRTPEVARSRATPLLDLIKTALTPHPPQKQAYGVTLPTSVLFIAGHDTNLANLGGALELNWTLPGQPDNTPPGGELVFERWRRLSDNSQWIQVSLVFQTLQQMRDKTPLSLNTPPGEVKLTLAGCEERNAQGMCSLAGFTQIVNEARIPACSL',\n",
       "  'mw': 47057.0},\n",
       " 'b0996': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P33226',\n",
       "  'gene': 'b0996',\n",
       "  'locus': ['b0996', 'JW0981'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the anaerobic respiratory chain of trimethylamine-N-oxide reductase TorA. Acts by transferring electrons from the membranous menaquinones to TorA. This transfer probably involves an electron transfer pathway from menaquinones to the N-terminal domain of TorC, then from the N-terminus to the C-terminus, and finally to TorA. TorC apocytochrome negatively autoregulates the torCAD operon probably by inhibiting the TorS kinase activity'},\n",
       "  'seq': 'MRKLWNALRRPSARWSVLALVAIGIVIGIALIVLPHVGIKVTSTTEFCVSCHSMQPVYEEYKQSVHFQNASGVRAECHDCHIPPDIPGMVKRKLEASNDIYQTFIAHSIDTPEKFEAKRAELAEREWARMKENNSATCRSCHNYDAMDHAKQHPEAARQMKVAAKDNQSCIDCHKGIAHQLPDMSSGFRKQFDELRASANDSGDTLYSIDIKPIYAAKGDKEASGSLLPASEVKVLKRDGDWLQIEITGWTESAGRQRVLTQFPGKRIFVASIRGDVQQQVKTLEKTTVADTNTEWSKLQATAWMKKGDMVNDIKPIWAYADSLYNGTCNQCHGAPEIAHFDANGWIGTLNGMIGFTSLDKREERTLLKYLQMNASDTAGKAHGDKKEEK',\n",
       "  'mw': 43607.0},\n",
       " 'b0997': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P33225',\n",
       "  'gene': 'b0997',\n",
       "  'locus': ['b0997', 'JW0982'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('2 Fe(III)-[cytochrome c] + H2O + trimethylamine = 2 Fe(II)-[cytochrome c] + 3 H(+) + trimethylamine N-oxide',\n",
       "     '1.7.2.3')],\n",
       "   'Function': 'Reduces trimethylamine-N-oxide (TMAO) into trimethylamine; an anaerobic reaction coupled to energy-yielding reactions'},\n",
       "  'seq': 'MNNNDLFQASRRRFLAQLGGLTVAGMLGPSLLTPRRATAAQAATDAVISKEGILTGSHWGAIRATVKDGRFVAAKPFELDKYPSKMIAGLPDHVHNAARIRYPMVRVDWLRKRHLSDTSQRGDNRFVRVSWDEALDMFYEELERVQKTHGPSALLTASGWQSTGMFHNASGMLAKAIALHGNSVGTGGDYSTGAAQVILPRVVGSMEVYEQQTSWPLVLQNSKTIVLWGSDLLKNQQANWWCPDHDVYEYYAQLKAKVAAGEIEVISIDPVVTSTHEYLGREHVKHIAVNPQTDVPLQLALAHTLYSENLYDKNFLANYCVGFEQFLPYLLGEKDGQPKDAAWAEKLTGIDAETIRGLARQMAANRTQIIAGWCVQRMQHGEQWAWMIVVLAAMLGQIGLPGGGFGFGWHYNGAGTPGRKGVILSGFSGSTSIPPVHDNSDYKGYSSTIPIARFIDAILEPGKVINWNGKSVKLPPLKMCIFAGTNPFHRHQQINRIIEGLRKLETVIAIDNQWTSTCRFADIVLPATTQFERNDLDQYGNHSNRGIIAMKQVVPPQFEARNDFDIFRELCRRFNREEAFTEGLDEMGWLKRIWQEGVQQGKGRGVHLPAFDDFWNNKEYVEFDHPQMFVRHQAFREDPDLEPLGTPSGLIEIYSKTIADMNYDDCQGHPMWFEKIERSHGGPGSQKYPLHLQSVHPDFRLHSQLCESETLRQQYTVAGKEPVFINPQDASARGIRNGDVVRVFNARGQVLAGAVVSDRYAPGVARIHEGAWYDPDKGGEPGALCKYGNPNVLTIDIGTSQLAQATSAHTTLVEIEKYNGTVEQVTAFNGPVEMVAQCEYVPASQVKS',\n",
       "  'mw': 94456.0},\n",
       " 'b1002': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P19926',\n",
       "  'gene': 'b1002',\n",
       "  'locus': ['b1002', 'JW0987'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('alpha-D-glucose 1-phosphate + H2O = D-glucose + phosphate',\n",
       "     '3.1.3.10')],\n",
       "   'Function': 'Absolutely required for the growth of E.coli in a high-phosphate medium containing G-1-P as the sole carbon source',\n",
       "   'Regulation Activity': 'Independent from inorganic phosphate availability, and apparently submitted to catabolite repression, it is positively controlled by cAMP and the cAMP receptor protein'},\n",
       "  'seq': 'MNKTLIAAAVAGIVLLASNAQAQTVPEGYQLQQVLMMSRHNLRAPLANNGSVLEQSTPNKWPEWDVPGGQLTTKGGVLEVYMGHYMREWLAEQGMVKSGECPPPYTVYAYANSLQRTVATAQFFITGAFPGCDIPVHHQEKMGTMDPTFNPVITDDSAAFSEQAVAAMEKELSKLQLTDSYQLLEKIVNYKDSPACKEKQQCSLVDGKNTFSAKYQQEPGVSGPLKVGNSLVDAFTLQYYEGFPMDQVAWGEIKSDQQWKVLSKLKNGYQDSLFTSPEVARNVAKPLVSYIDKALVTDRTSAPKITVLVGHDSNIASLLTALDFKPYQLHDQNERTPIGGKIVFQRWHDSKANRDLMKIEYVYQSAEQLRNADALTLQAPAQRVTLELSGCPIDADGFCPMDKFDSVLNEAVK',\n",
       "  'mw': 45683.0},\n",
       " 'b1006': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P75892',\n",
       "  'gene': 'b1006',\n",
       "  'locus': ['b1006', 'JW5137'],\n",
       "  'synonyms': 'ycdG',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'May function as a proton-driven pyrimidine uptake system'},\n",
       "  'seq': 'MAMFGFPHWQLKSTSTESGVVAPDERLPFAQTAVMGVQHAVAMFGATVLMPILMGLDPNLSILMSGIGTLLFFFITGGRVPSYLGSSAAFVGVVIAATGFNGQGINPNISIALGGIIACGLVYTVIGLVVMKIGTRWIERLMPPVVTGAVVMAIGLNLAPIAVKSVSASAFDSWMAVMTVLCIGLVAVFTRGMIQRLLILVGLIVACLLYGVMTNVLGLGKAVDFTLVSHAAWFGLPHFSTPAFNGQAMMLIAPVAVILVAENLGHLKAVAGMTGRNMDPYMGRAFVGDGLATMLSGSVGGSGVTTYAENIGVMAVTKVYSTLVFVAAAVIAMLLGFSPKFGALIHTIPAAVIGGASIVVFGLIAVAGARIWVQNRVDLSQNGNLIMVAVTLVLGAGDFALTLGGFTLGGIGTATFGAILLNALLSRKLVDVPPPEVVHQEP',\n",
       "  'mw': 45557.0},\n",
       " 'b1014': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P09546',\n",
       "  'gene': 'b1014',\n",
       "  'locus': ['b1014', 'JW0999'],\n",
       "  'synonyms': 'poaA',\n",
       "  'properties': {'Catalytic Activity': [('a quinone + L-proline = (S)-1-pyrroline-5-carboxylate + a quinol + H(+)',\n",
       "     '1.5.5.2'),\n",
       "    ('H2O + L-glutamate 5-semialdehyde + NAD(+) = 2 H(+) + L-glutamate + NADH',\n",
       "     '1.2.1.88')],\n",
       "   'Function': 'Oxidizes proline to glutamate for use as a carbon and nitrogen source and also function as a transcriptional repressor of the put operon',\n",
       "   'Pathway': 'Amino-acid degradation; L-proline degradation into L-glutamate; L-glutamate from L-proline: step 2/2'},\n",
       "  'seq': 'MGTTTMGVKLDDATRERIKSAATRIDRTPHWLIKQAIFSYLEQLENSDTLPELPALLSGAANESDEAPTPAEEPHQPFLDFAEQILPQSVSRAAITAAYRRPETEAVSMLLEQARLPQPVAEQAHKLAYQLADKLRNQKNASGRAGMVQGLLQEFSLSSQEGVALMCLAEALLRIPDKATRDALIRDKISNGNWQSHIGRSPSLFVNAATWGLLFTGKLVSTHNEASLSRSLNRIIGKSGEPLIRKGVDMAMRLMGEQFVTGETIAEALANARKLEEKGFRYSYDMLGEAALTAADAQAYMVSYQQAIHAIGKASNGRGIYEGPGISIKLSALHPRYSRAQYDRVMEELYPRLKSLTLLARQYDIGINIDAEESDRLEISLDLLEKLCFEPELAGWNGIGFVIQAYQKRCPLVIDYLIDLATRSRRRLMIRLVKGAYWDSEIKRAQMDGLEGYPVYTRKVYTDVSYLACAKKLLAVPNLIYPQFATHNAHTLAAIYQLAGQNYYPGQYEFQCLHGMGEPLYEQVTGKVADGKLNRPCRIYAPVGTHETLLAYLVRRLLENGANTSFVNRIADTSLPLDELVADPVTAVEKLAQQEGQTGLPHPKIPLPRDLYGHGRDNSAGLDLANEHRLASLSSALLNSALQKWQALPMLEQPVAAGEMSPVINPAEPKDIVGYVREATPREVEQALESAVNNAPIWFATPPAERAAILHRAAVLMESQMQQLIGILVREAGKTFSNAIAEVREAVDFLHYYAGQVRDDFANETHRPLGPVVCISPWNFPLAIFTGQIAAALAAGNSVLAKPAEQTPLIAAQGIAILLEAGVPPGVVQLLPGRGETVGAQLTGDDRVRGVMFTGSTEVATLLQRNIASRLDAQGRPIPLIAETGGMNAMIVDSSALTEQVVVDVLASAFDSAGQRCSALRVLCLQDEIADHTLKMLRGAMAECRMGNPGRLTTDIGPVIDSEAKANIERHIQTMRSKGRPVFQAVRENSEDAREWQSGTFVAPTLIELDDFAELQKEVFGPVLHVVRYNRNQLPELIEQINASGYGLTLGVHTRIDETIAQVTGSAHVGNLYVNRNMVGAVVGVQPFGGEGLSGTGPKAGGPLYLYRLLANRPESALAVTLARQDAKYPVDAQLKAALTQPLNALREWAANRPELQALCTQYGELAQAGTQRLLPGPTGERNTWTLLPRERVLCIADDEQDALTQLAAVLAVGSQVLWPDDALHRQLVKALPSAVSERIQLAKAENITAQPFDAVIFHGDSDQLRALCEAVAARDGTIVSVQGFARGESNILLERLYIERSLSVNTAAAGGNASLMTIG',\n",
       "  'mw': 143815.0},\n",
       " 'b1015': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P07117',\n",
       "  'gene': 'b1015',\n",
       "  'locus': ['b1015', 'JW1001'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('L-proline(in) + Na(+)(in) = L-proline(out) + Na(+)(out)',\n",
       "     '')],\n",
       "   'Function': 'Catalyzes the sodium-dependent uptake of extracellular L-proline (PubMed:3512540, PubMed:1567896, PubMed:9693004). This protein is also capable of using lithium as the transport cation (PubMed:1567896, PubMed:9693004). Also catalyzes the uptake of propionate (PubMed:17088549)',\n",
       "   'Regulation Activity': 'Activity is stimulated by phosphatidylethanolamine and phosphatidylglycerol, but not by phosphatidylcholine and cardiolipin (PubMed:3512540). Proline uptake is inhibited by the sulfhydryl reagent N-ethylmaleimide (NEM) (PubMed:3053687, PubMed:1567896). Proline, in the presence of Na(+) or Li(+), protects the carrier functions from NEM-inactivation (PubMed:1567896)'},\n",
       "  'seq': 'MAISTPMLVTFCVYIFGMILIGFIAWRSTKNFDDYILGGRSLGPFVTALSAGASDMSGWLLMGLPGAVFLSGISESWIAIGLTLGAWINWKLVAGRLRVHTEYNNNALTLPDYFTGRFEDKSRILRIISALVILLFFTIYCASGIVAGARLFESTFGMSYETALWAGAAATILYTFIGGFLAVSWTDTVQASLMIFALILTPVIVIISVGGFGDSLEVIKQKSIENVDMLKGLNFVAIISLMGWGLGYFGQPHILARFMAADSHHSIVHARRISMTWMILCLAGAVAVGFFGIAYFNDHPALAGAVNQNAERVFIELAQILFNPWIAGILLSAILAAVMSTLSCQLLVCSSAITEDLYKAFLRKHASQKELVWVGRVMVLVVALVAIALAANPENRVLGLVSYAWAGFGAAFGPVVLFSVMWSRMTRNGALAGMIIGALTVIVWKQFGWLGLYEIIPGFIFGSIGIVVFSLLGKAPSAAMQKRFAEADAHYHSAPPSRLQES',\n",
       "  'mw': 54344.0},\n",
       " 'b1033': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P75913',\n",
       "  'gene': 'b1033',\n",
       "  'locus': ['b1033', 'JW5146'],\n",
       "  'synonyms': 'ycdW',\n",
       "  'properties': {'Catalytic Activity': [('glycolate + NADP(+) = glyoxylate + H(+) + NADPH',\n",
       "     '1.1.1.79'),\n",
       "    ('(R)-glycerate + NAD(+) = 3-hydroxypyruvate + H(+) + NADH', '1.1.1.81'),\n",
       "    ('(R)-glycerate + NADP(+) = 3-hydroxypyruvate + H(+) + NADPH',\n",
       "     '1.1.1.81')],\n",
       "   'Function': 'Catalyzes the NADPH-dependent reduction of glyoxylate and hydroxypyruvate into glycolate and glycerate, respectively. Inactive towards 2-oxo-D-gluconate, 2-oxoglutarate, oxaloacetate and pyruvate. Only D- and L-glycerate are involved in the oxidative activity with NADP. Activity with NAD is very low'},\n",
       "  'seq': 'MDIIFYHPTFDTQWWIEALRKAIPQARVRAWKSGDNDSADYALVWHPPVEMLAGRDLKAVFALGAGVDSILSKLQAHPEMLNPSVPLFRLEDTGMGEQMQEYAVSQVLHWFRRFDDYRIQQNSSHWQPLPEYHREDFTIGILGAGVLGSKVAQSLQTWRFPLRCWSRTRKSWPGVQSFAGREELSAFLSQCRVLINLLPNTPETVGIINQQLLEKLPDGAYLLNLARGVHVVEDDLLAALDSGKVKGAMLDVFNREPLPPESPLWQHPRVTITPHVAAITRPAEAVEYISRTIAQLEKGERVCGQVDRARGY',\n",
       "  'mw': 35343.0},\n",
       " 'b1054': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ACV0',\n",
       "  'gene': 'b1054',\n",
       "  'locus': ['b1054', 'JW1041'],\n",
       "  'synonyms': 'htrB waaM',\n",
       "  'properties': {'Catalytic Activity': [('alpha-Kdo-(2->4)-alpha-Kdo-(2->6)-lipid IVA (E. coli) + dodecanoyl-[ACP] = alpha-Kdo-(2->4)-alpha-Kdo-(2->6)-(dodecanoyl)-lipid IVA (E. coli) + holo-[ACP]',\n",
       "     '2.3.1.241')],\n",
       "   'Function': 'Catalyzes the transfer of laurate from lauroyl-[acyl-carrier-protein] (ACP) to Kdo(2)-lipid IV(A) to form Kdo(2)-(lauroyl)-lipid IV(A). Has 10 fold selectivity for lauroyl-ACP over myristoyl-ACP. In vitro, can also catalyze a slow second acylation reaction leading to the formation of Kdo(2)-(dilauroyl)-lipid IV(A)',\n",
       "   'Pathway': 'Bacterial outer membrane biogenesis; lipopolysaccharide biosynthesis'},\n",
       "  'seq': 'MTNLPKFSTALLHPRYWLTWLGIGVLWLVVQLPYPVIYRLGCGLGKLALRFMKRRAKIVHRNLELCFPEMSEQERRKMVVKNFESVGMGLMETGMAWFWPDRRIARWTEVIGMEHIRDVQAQKRGILLVGIHFLTLELGARQFGMQEPGIGVYRPNDNPLIDWLQTWGRLRSNKSMLDRKDLKGMIKALKKGEVVWYAPDHDYGPRSSVFVPLFAVEQAATTTGTWMLARMSGACLVPFVPRRKPDGKGYQLIMLPPECSPPLDDAETTAAWMNKVVEKCIMMAPEQYMWLHRRFKTRPEGVPSRY',\n",
       "  'mw': 35407.0},\n",
       " 'b1059': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P40874',\n",
       "  'gene': 'b1059',\n",
       "  'locus': ['b1059', 'JW1046'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('H2O + N(alpha)-methyl-L-tryptophan + O2 = formaldehyde + H2O2 + L-tryptophan',\n",
       "     '')],\n",
       "   'Function': 'Catalyzes the oxidative demethylation of N-methyl-L-tryptophan. Can also use other N-methyl amino acids, including sarcosine, which, however, is a poor substrate',\n",
       "   'Regulation Activity': 'Aromatic carboxylates are competitive inhibitors'},\n",
       "  'seq': 'MKYDLIIIGSGSVGAAAGYYATRAGLNVLMTDAHMPPHQHGSHHGDTRLIRHAYGEGEKYVPLVLRAQTLWDELSRHNEEDPIFVRSGVINLGPADSTFLANVAHSAEQWQLNVEKLDAQGIMARWPEIRVPDNYIGLFETDSGFLRSELAIKTWIQLAKEAGCAQLFNCPVTAIRHDDDGVTIETADGEYQAKKAIVCAGTWVKDLLPELPVQPVRKVFAWYQADGRYSVKNKFPAFTGELPNGDQYYGFPAENDALKIGKHNGGQVIHSADERVPFAEVASDGSEAFPFLRNVLPGIGCCLYGAACTYDNSPDEDFIIDTLPGHDNTLLITGLSGHGFKFASVLGEIAADFAQDKKSDFDLTPFRLSRFQ',\n",
       "  'mw': 40902.0},\n",
       " 'b1062': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P05020',\n",
       "  'gene': 'b1062',\n",
       "  'locus': ['b1062', 'JW1049'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(S)-dihydroorotate + H2O = H(+) + N-carbamoyl-L-aspartate',\n",
       "     '3.5.2.3')],\n",
       "   'Function': 'Catalyzes the reversible cyclization of carbamoyl aspartate to dihydroorotate',\n",
       "   'Pathway': 'Pyrimidine metabolism; UMP biosynthesis via de novo pathway; (S)-dihydroorotate from bicarbonate: step 3/3'},\n",
       "  'seq': 'MTAPSQVLKIRRPDDWHLHLRDGDMLKTVVPYTSEIYGRAIVMPNLAPPVTTVEAAVAYRQRILDAVPAGHDFTPLMTCYLTDSLDPNELERGFNEGVFTAAKLYPANATTNSSHGVTSIDAIMPVLERMEKIGMPLLVHGEVTHADIDIFDREARFIESVMEPLRQRLTALKVVFEHITTKDAADYVRDGNERLAATITPQHLMFNRNHMLVGGVRPHLYCLPILKRNIHQQALRELVASGFNRVFLGTDSAPHARHRKESSCGCAGCFNAPTALGSYATVFEEMNALQHFEAFCSVNGPQFYGLPVNDTFIELVREEQQVAESIALTDDTLVPFLAGETVRWSVKQ',\n",
       "  'mw': 38827.0},\n",
       " 'b1064': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AC59',\n",
       "  'gene': 'b1064',\n",
       "  'locus': ['b1064', 'JW1051'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Involved in reducing some disulfide bonds in a coupled system with glutathione reductase. Does not act as hydrogen donor for ribonucleotide reductase'},\n",
       "  'seq': 'MKLYIYDHCPYCLKARMIFGLKNIPVELHVLLNDDAETPTRMVGQKQVPILQKDDSRYMPESMDIVHYVDKLDGKPLLTGKRSPAIEEWLRKVNGYANKLLLPRFAKSAFDEFSTPAARKYFVDKKEASAGNFADLLAHSDGLIKNISDDLRALDKLIVKPNAVNGELSEDDIQLFPLLRNLTLVAGINWPSRVADYRDNMAKQTQINLLSSMAI',\n",
       "  'mw': 24350.0},\n",
       " 'b1091': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A6R0',\n",
       "  'gene': 'b1091',\n",
       "  'locus': ['b1091', 'JW1077'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('acetyl-CoA + H(+) + malonyl-[ACP] = 3-oxobutanoyl-[ACP] + CO2 + CoA',\n",
       "     '2.3.1.180'),\n",
       "    ('H(+) + malonyl-[ACP] + propanoyl-CoA = 3-oxopentanoyl-[ACP] + CO2 + CoA',\n",
       "     '')],\n",
       "   'Function': 'Catalyzes the condensation reaction of fatty acid synthesis by the addition to an acyl acceptor of two carbons from malonyl-ACP. Catalyzes the first condensation reaction which initiates fatty acid synthesis and may therefore play a role in governing the total rate of fatty acid production. Possesses both acetoacetyl-ACP synthase and acetyl transacylase activities. Has some substrate specificity for acetyl-CoA. Its substrate specificity determines the biosynthesis of straight-chain of fatty acids instead of branched-chain (PubMed:7592873, PubMed:8631920, PubMed:8910376, PubMed:10629181). Can also use propionyl-CoA, with lower efficiency (PubMed:8631920, PubMed:10629181)',\n",
       "   'Regulation Activity': 'Negatively regulated by acyl-ACP, possibly by binding to either the free enzyme or the acyl-enzyme intermediate (PubMed:8631920). Inhibited by the SB418011 antibiotic (PubMed:11375394). Not inhibited by cerulenin, and weakly inhibited by thiolactomycin (PubMed:11375394)',\n",
       "   'Pathway': 'Lipid metabolism; fatty acid biosynthesis'},\n",
       "  'seq': 'MYTKIIGTGSYLPEQVRTNADLEKMVDTSDEWIVTRTGIRERHIAAPNETVSTMGFEAATRAIEMAGIEKDQIGLIVVATTSATHAFPSAACQIQSMLGIKGCPAFDVAAACAGFTYALSVADQYVKSGAVKYALVVGSDVLARTCDPTDRGTIIIFGDGAGAAVLAASEEPGIISTHLHADGSYGELLTLPNADRVNPENSIHLTMAGNEVFKVAVTELAHIVDETLAANNLDRSQLDWLVPHQANLRIISATAKKLGMSMDNVVVTLDRHGNTSAASVPCALDEAVRDGRIKPGQLVLLEAFGGGFTWGSALVRF',\n",
       "  'mw': 33515.0},\n",
       " 'b1092': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AAI9',\n",
       "  'gene': 'b1092',\n",
       "  'locus': ['b1092', 'JW1078'],\n",
       "  'synonyms': 'tfpA',\n",
       "  'properties': {'Catalytic Activity': [('holo-[ACP] + malonyl-CoA = CoA + malonyl-[ACP]',\n",
       "     '2.3.1.39')],\n",
       "   'Pathway': 'Lipid metabolism; fatty acid biosynthesis'},\n",
       "  'seq': 'MTQFAFVFPGQGSQTVGMLADMAASYPIVEETFAEASAALGYDLWALTQQGPAEELNKTWQTQPALLTASVALYRVWQQQGGKAPAMMAGHSLGEYSALVCAGVIDFADAVRLVEMRGKFMQEAVPEGTGAMAAIIGLDDASIAKACEEAAEGQVVSPVNFNSPGQVVIAGHKEAVERAGAACKAAGAKRALPLPVSVPSHCALMKPAADKLAVELAKITFNAPTVPVVNNVDVKCETNGDAIRDALVRQLYNPVQWTKSVEYMAAQGVEHLYEVGPGKVLTGLTKRIVDTLTASALNEPSAMAAALEL',\n",
       "  'mw': 32417.0},\n",
       " 'b1093': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AEK2',\n",
       "  'gene': 'b1093',\n",
       "  'locus': ['b1093', 'JW1079'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('a (3R)-hydroxyacyl-[ACP] + NADP(+) = a 3-oxoacyl-[ACP] + H(+) + NADPH',\n",
       "     '1.1.1.100'),\n",
       "    ('3-oxobutanoyl-[ACP] + H(+) + NADPH = (3R)-hydroxybutanoyl-[ACP] + NADP(+)',\n",
       "     ''),\n",
       "    ('3-oxopentanoyl-[ACP] + H(+) + NADPH = (3R)-hydroxypentanoyl-[ACP] + NADP(+)',\n",
       "     ''),\n",
       "    ('3-oxohexanoyl-[ACP] + H(+) + NADPH = (3R)-hydroxyhexanoyl-[ACP] + NADP(+)',\n",
       "     ''),\n",
       "    ('3-oxoheptanoyl-[ACP] + H(+) + NADPH = (3R)-hydroxyheptanoyl-[ACP] + NADP(+)',\n",
       "     ''),\n",
       "    ('3-oxooctanoyl-[ACP] + H(+) + NADPH = (3R)-hydroxyoctanoyl-[ACP] + NADP(+)',\n",
       "     ''),\n",
       "    ('3-oxononanoyl-[ACP] + H(+) + NADPH = (3R)-hydroxynonanoyl-[ACP] + NADP(+)',\n",
       "     ''),\n",
       "    ('3-oxodecanoyl-[ACP] + H(+) + NADPH = (3R)-hydroxydecanoyl-[ACP] + NADP(+)',\n",
       "     ''),\n",
       "    ('3-oxohexadecanoyl-[ACP] + H(+) + NADPH = (3R)-hydroxyhexadecanoyl-[ACP] + NADP(+)',\n",
       "     ''),\n",
       "    ('3-oxo-(9Z)-hexadecenoyl-[ACP] + H(+) + NADPH = (3R)-hydroxy-(9Z)-hexadecenoyl-[ACP] + NADP(+)',\n",
       "     ''),\n",
       "    ('4-methyl-3-oxopentanoyl-[ACP] + H(+) + NADPH = (3R)-hydroxy-4-methylpentanoyl-[ACP] + NADP(+)',\n",
       "     ''),\n",
       "    ('5-methyl-3-oxohexanoyl-[ACP] + H(+) + NADPH = (3R)-hydroxy-5-methylhexanoyl-[ACP] + NADP(+)',\n",
       "     ''),\n",
       "    ('4-methyl-3-oxohexanoyl-[ACP] + H(+) + NADPH = (3R)-hydroxy-4-methylhexanoyl-[ACP] + NADP(+)',\n",
       "     '')],\n",
       "   'Function': 'Catalyzes the NADPH-dependent reduction of beta-ketoacyl-ACP substrates to beta-hydroxyacyl-ACP products, the first reductive step in the elongation cycle of fatty acid biosynthesis',\n",
       "   'Regulation Activity': 'Inhibited by cinnamic acid derivatives',\n",
       "   'Pathway': 'Lipid metabolism; fatty acid biosynthesis'},\n",
       "  'seq': 'MNFEGKIALVTGASRGIGRAIAETLAARGAKVIGTATSENGAQAISDYLGANGKGLMLNVTDPASIESVLEKIRAEFGEVDILVNNAGITRDNLLMRMKDEEWNDIIETNLSSVFRLSKAVMRAMMKKRHGRIITIGSVVGTMGNGGQANYAAAKAGLIGFSKSLAREVASRGITVNVVAPGFIETDMTRALSDDQRAGILAQVPAGRLGGAQEIANAVAFLASDEAAYITGETLHVNGGMYMV',\n",
       "  'mw': 25560.0},\n",
       " 'b1094': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A6A8',\n",
       "  'gene': 'b1094',\n",
       "  'locus': ['b1094', 'JW1080'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Carrier of the growing fatty acid chain in fatty acid biosynthesis',\n",
       "   'Pathway': 'Lipid metabolism; fatty acid biosynthesis'},\n",
       "  'seq': 'MSTIEERVKKIIGEQLGVKQEEVTNNASFVEDLGADSLDTVELVMALEEEFDTEIPDEEAEKITTVQAAIDYINGHQA',\n",
       "  'mw': 8640.0},\n",
       " 'b1095': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AAI5',\n",
       "  'gene': 'b1095',\n",
       "  'locus': ['b1095', 'JW1081'],\n",
       "  'synonyms': 'fabJ',\n",
       "  'properties': {'Catalytic Activity': [('a fatty acyl-[ACP] + H(+) + malonyl-[ACP] = a 3-oxoacyl-[ACP] + CO2 + holo-[ACP]',\n",
       "     ''),\n",
       "    ('(9Z)-hexadecenoyl-[ACP] + H(+) + malonyl-[ACP] = 3-oxo-(11Z)-octadecenoyl-[ACP] + CO2 + holo-[ACP]',\n",
       "     '2.3.1.179'),\n",
       "    ('H(+) + hexanoyl-[ACP] + malonyl-[ACP] = 3-oxooctanoyl-[ACP] + CO2 + holo-[ACP]',\n",
       "     ''),\n",
       "    ('H(+) + malonyl-[ACP] + octanoyl-[ACP] = 3-oxodecanoyl-[ACP] + CO2 + holo-[ACP]',\n",
       "     ''),\n",
       "    ('decanoyl-[ACP] + H(+) + malonyl-[ACP] = 3-oxododecanoyl-[ACP] + CO2 + holo-[ACP]',\n",
       "     ''),\n",
       "    ('dodecanoyl-[ACP] + H(+) + malonyl-[ACP] = 3-oxotetradecanoyl-[ACP] + CO2 + holo-[ACP]',\n",
       "     ''),\n",
       "    ('H(+) + malonyl-[ACP] + tetradecanoyl-[ACP] = 3-oxohexadecanoyl-[ACP] + CO2 + holo-[ACP]',\n",
       "     '')],\n",
       "   'Function': 'Involved in the type II fatty acid elongation cycle (PubMed:9013860, PubMed:6988423). Catalyzes the elongation of a wide range of acyl-ACP by the addition of two carbons from malonyl-ACP to an acyl acceptor (PubMed:9013860, PubMed:22017312). Can efficiently catalyze the conversion of palmitoleoyl-ACP (cis-hexadec-9-enoyl-ACP) to cis-vaccenoyl-ACP (cis-octadec-11-enoyl-ACP), an essential step in the thermal regulation of fatty acid composition (PubMed:9013860, PubMed:6988423). Can use acyl chains from C-6 to C-16 (PubMed:9013860, PubMed:22017312). Is able to catalyze the condensation reaction when CoA is the carrier for both substrates (PubMed:22017312)',\n",
       "   'Pathway': 'Lipid metabolism; fatty acid biosynthesis'},\n",
       "  'seq': 'MSKRRVVVTGLGMLSPVGNTVESTWKALLAGQSGISLIDHFDTSAYATKFAGLVKDFNCEDIISRKEQRKMDAFIQYGIVAGVQAMQDSGLEITEENATRIGAAIGSGIGGLGLIEENHTSLMNGGPRKISPFFVPSTIVNMVAGHLTIMYGLRGPSISIATACTSGVHNIGHAARIIAYGDADVMVAGGAEKASTPLGVGGFGAARALSTRNDNPQAASRPWDKERDGFVLGDGAGMLVLEEYEHAKKRGAKIYAELVGFGMSSDAYHMTSPPENGAGAALAMANALRDAGIEASQIGYVNAHGTSTPAGDKAEAQAVKTIFGEAASRVLVSSTKSMTGHLLGAAGAVESIYSILALRDQAVPPTINLDNPDEGCDLDFVPHEARQVSGMEYTLCNSFGFGGTNGSLIFKKI',\n",
       "  'mw': 43046.0},\n",
       " 'b1096': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P28305',\n",
       "  'gene': 'b1096',\n",
       "  'locus': ['b1096', 'JW1082'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('4-amino-4-deoxychorismate = 4-aminobenzoate + H(+) + pyruvate',\n",
       "     '4.1.3.38')],\n",
       "   'Function': 'Involved in the biosynthesis of p-aminobenzoate (PABA), a precursor of tetrahydrofolate. Converts 4-amino-4-deoxychorismate into 4-aminobenzoate (PABA) and pyruvate',\n",
       "   'Pathway': 'Cofactor biosynthesis; tetrahydrofolate biosynthesis; 4-aminobenzoate from chorismate: step 2/2'},\n",
       "  'seq': 'MFLINGHKQESLAVSDRATQFGDGCFTTARVIDGKVSLLSAHIQRLQDACQRLMISCDFWPQLEQEMKTLAAEQQNGVLKVVISRGSGGRGYSTLNSGPATRILSVTAYPAHYDRLRNEGITLALSPVRLGRNPHLAGIKHLNRLEQVLIRSHLEQTNADEALVLDSEGWVTECCAANLFWRKGNVVYTPRLDQAGVNGIMRQFCIRLLAQSSYQLVEVQASLEESLQADEMVICNALMPVMPVCACGDVSFSSATLYEYLAPLCERPN',\n",
       "  'mw': 29715.0},\n",
       " 'b1098': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A720',\n",
       "  'gene': 'b1098',\n",
       "  'locus': ['b1098', 'JW1084'],\n",
       "  'synonyms': 'ycfG',\n",
       "  'properties': {'Catalytic Activity': [('ATP + dTMP = ADP + dTDP',\n",
       "     '2.7.4.9')],\n",
       "   'Function': 'Catalyzes the reversible phosphorylation of deoxythymidine monophosphate (dTMP) to deoxythymidine diphosphate (dTDP), using ATP as its preferred phosphoryl donor (PubMed:8631667). Situated at the junction of both de novo and salvage pathways of deoxythymidine triphosphate (dTTP) synthesis, is essential for DNA synthesis and cellular growth',\n",
       "   'Pathway': 'Pyrimidine metabolism; dTTP biosynthesis'},\n",
       "  'seq': 'MRSKYIVIEGLEGAGKTTARNVVVETLEQLGIRDMVFTREPGGTQLAEKLRSLVLDIKSVGDEVITDKAEVLMFYAARVQLVETVIKPALANGTWVIGDRHDLSTQAYQGGGRGIDQHMLATLRDAVLGDFRPDLTLYLDVTPEVGLKRARARGELDRIEQESFDFFNRTRARYLELAAQDKSIHTIDATQPLEAVMDAIRTTVTHWVKELDA',\n",
       "  'mw': 23783.0},\n",
       " 'b1101': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P69786',\n",
       "  'gene': 'b1101',\n",
       "  'locus': ['b1101', 'JW1087'],\n",
       "  'synonyms': 'glcA umg',\n",
       "  'properties': {'Catalytic Activity': [('D-glucose(out) + N(pros)-phospho-L-histidyl-[protein] = D-glucose 6-phosphate(in) + L-histidyl-[protein]',\n",
       "     '2.7.1.199')],\n",
       "   'Function': '(Microbial infection) Probably transports the toxic C-terminal region of CdiA from E.coli strains NC101 and 3006 across the inner membrane to the cytoplasm, where CdiA degrades specific tRNAs. Toxin transport is strain-specific, mutations in this gene do not confer resistance to several other tested CdiA toxins (PubMed:26305955). Probably also serves as the inner membrane receptor for CdiA-STECO31 from E.coli strain STEC_O31 (Probable)',\n",
       "   'Regulation Activity': 'Transporter activity may be inhibited by SgrT'},\n",
       "  'seq': 'MFKNAFANLQKVGKSLMLPVSVLPIAGILLGVGSANFSWLPAVVSHVMAEAGGSVFANMPLIFAIGVALGFTNNDGVSALAAVVAYGIMVKTMAVVAPLVLHLPAEEIASKHLADTGVLGGIISGAIAAYMFNRFYRIKLPEYLGFFAGKRFVPIISGLAAIFTGVVLSFIWPPIGSAIQTFSQWAAYQNPVVAFGIYGFIERCLVPFGLHHIWNVPFQMQIGEYTNAAGQVFHGDIPRYMAGDPTAGKLSGGFLFKMYGLPAAAIAIWHSAKPENRAKVGGIMISAALTSFLTGITEPIEFSFMFVAPILYIIHAILAGLAFPICILLGMRDGTSFSHGLIDFIVLSGNSSKLWLFPIVGIGYAIVYYTIFRVLIKALDLKTPGREDATEDAKATGTSEMAPALVAAFGGKENITNLDACITRLRVSVADVSKVDQAGLKKLGAAGVVVAGSGVQAIFGTKSDNLKTEMDEYIRNH',\n",
       "  'mw': 50677.0},\n",
       " 'b1102': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P16869',\n",
       "  'gene': 'b1102',\n",
       "  'locus': ['b1102', 'JW1088'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Required for the uptake of Fe(3+) via coprogen, ferrioxamine B, and rhodotorulic acid'},\n",
       "  'seq': 'MLSTQFNRDNQYQAITKPSLLAGCIALALLPSAAFAAPATEETVIVEGSATAPDDGENDYSVTSTSAGTKMQMTQRDIPQSVTIVSQQRMEDQQLQTLGEVMENTLGISKSQADSDRALYYSRGFQIDNYMVDGIPTYFESRWNLGDALSDMALFERVEVVRGATGLMTGTGNPSAAINMVRKHATSREFKGDVSAEYGSWNKERYVADLQSPLTEDGKIRARIVGGYQNNDSWLDRYNSEKTFFSGIVDADLGDLTTLSAGYEYQRIDVNSPTWGGLPRWNTDGSSNSYDRARSTAPDWAYNDKEINKVFMTLKQQFADTWQATLNATHSEVEFDSKMMYVDAYVNKADGMLVGPYSNYGPGFDYVGGTGWNSGKRKVDALDLFADGSYELFGRQHNLMFGGSYSKQNNRYFSSWANIFPDEIGSFYNFNGNFPQTDWSPQSLAQDDTTHMKSLYAATRVTLADPLHLILGARYTNWRVDTLTYSMEKNHTTPYAGLVFDINDNWSTYASYTSIFQPQNDRDSSGKYLAPITGNNYELGLKSDWMNSRLTTTLAIFRIEQDNVAQSTGTPIPGSNGETAYKAVDGTVSKGVEFELNGAITDNWQLTFGATRYIAEDNEGNAVNPNLPRTTVKMFTSYRLPVMPELTVGGGVNWQNRVYTDTVTPYGTFRAEQGSYALVDLFTRYQVTKNFSLQGNVNNLFDKTYDTNVEGSIVYGTPRNFSITGTYQF',\n",
       "  'mw': 81233.0},\n",
       " 'b1106': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P75948',\n",
       "  'gene': 'b1106',\n",
       "  'locus': ['b1106', 'JW1092'],\n",
       "  'synonyms': 'ycfN',\n",
       "  'properties': {'Catalytic Activity': [('ATP + thiamine = ADP + H(+) + thiamine phosphate',\n",
       "     '2.7.1.89')],\n",
       "   'Function': 'Catalyzes the phosphorylation of thiamine to thiamine phosphate',\n",
       "   'Pathway': 'Cofactor biosynthesis; thiamine diphosphate biosynthesis; thiamine phosphate from thiamine: step 1/1'},\n",
       "  'seq': 'MPFRSNNPITRDELLSRFFPQYHPVTTFNSGLSGGSFLIEHQGQRFVVRQPHDPDAPQSAFLRQYRALSQLPACIAPKPHLYLRDWMVVDYLPGAVKTYLPDTNELAGLLYYLHQQPRFGWRITLLPLLELYWQQSDPARRTVGWLRMLKRLRKAREPRPLRLSPLHMDVHAGNLVHSASGLKLIDWEYAGDGDIALELAAVWVENTEQHRQLVNDYATRAKIYPAQLWRQVRRWFPWLLMLKAGWFEYRWRQTGDQQFIRLADDTWRQLLIKQ',\n",
       "  'mw': 32397.0},\n",
       " 'b1107': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P75949',\n",
       "  'gene': 'b1107',\n",
       "  'locus': ['b1107', 'JW1093'],\n",
       "  'synonyms': 'ycfO',\n",
       "  'properties': {'Catalytic Activity': [('Hydrolysis of terminal non-reducing N-acetyl-D-hexosamine residues in N-acetyl-beta-D-hexosaminides.',\n",
       "     '3.2.1.52')],\n",
       "   'Function': 'Plays a role in peptidoglycan recycling by cleaving the terminal beta-1,4-linked N-acetylglucosamine (GlcNAc) from peptide-linked peptidoglycan fragments, giving rise to free GlcNAc, anhydro-N-acetylmuramic acid and anhydro-N-acetylmuramic acid-linked peptides. Cleaves GlcNAc linked beta-1,4 to MurNAc tripeptides',\n",
       "   'Regulation Activity': 'Slightly inhibited by GlcNAc and N-acetylmuramic acid and strongly inhibited by N-acetylglucosaminolactone',\n",
       "   'Pathway': 'Cell wall biogenesis; peptidoglycan recycling'},\n",
       "  'seq': 'MGPVMLDVEGYELDAEEREILAHPLVGGLILFTRNYHDPAQLRELVRQIRAASRNRLVVAVDQEGGRVQRFREGFTRLPAAQSFAALSGMEEGGKLAQEAGWLMASEMIAMDIDISFAPVLDVGHISAAIGERSYHADPQKALAIASRFIDGMHEAGMKTTGKHFPGHGAVTADSHKETPCDPRPQAEIRAKDMSVFSSLIRENKLDAIMPAHVIYSDVDPRPASGSPYWLKTVLRQELGFDGVIFSDDLSMEGAAIMGSYAERGQASLDAGCDMILVCNNRKGAVSVLDNLSPIKAERVTRLYHKGSFSRQELMDSARWKAISTRLNQLHERWQEEKAGH',\n",
       "  'mw': 37595.0},\n",
       " 'b1109': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P00393',\n",
       "  'gene': 'b1109',\n",
       "  'locus': ['b1109', 'JW1095'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('a quinone + H(+) + NADH = a quinol + NAD(+)',\n",
       "     '1.6.5.9'),\n",
       "    ('a ubiquinone + H(+) + NADH = a ubiquinol + NAD(+)', ''),\n",
       "    ('H(+) + NADH + ubiquinone-8 = NAD(+) + ubiquinol-8', ''),\n",
       "    ('2 Cu(2+) + NADH = 2 Cu(+) + H(+) + NAD(+)', '')],\n",
       "   'Function': 'It may also contribute to copper homeostasis and bacterial oxidative protection (PubMed:7487066, PubMed:10510271, PubMed:16759635, PubMed:21390523). Shows cupric reductase activity, and catalyzes the reduction of Cu(2+) to Cu(+) with NADH as electron donor (PubMed:10510271). Exhibits Cu(2+) reductase activity in the presence of either FAD or quinone, but is unable to reduce Fe(3+) (PubMed:10510271). Contains thiolate-bound copper (PubMed:12176061)',\n",
       "   'Regulation Activity': 'Requires Mg2(+) for activity in vitro (PubMed:2679883). Inhibited by 3-n-dodecylmercapto-2-hydroxy-1,4-naphthoquinone (PubMed:7009604). The quinone-dependent Cu(2+) reduction is partially inhibited by superoxide dismutase (PubMed:10510271)'},\n",
       "  'seq': 'MTTPLKKIVIVGGGAGGLEMATQLGHKLGRKKKAKITLVDRNHSHLWKPLLHEVATGSLDEGVDALSYLAHARNHGFQFQLGSVIDIDREAKTITIAELRDEKGELLVPERKIAYDTLVMALGSTSNDFNTPGVKENCIFLDNPHQARRFHQEMLNLFLKYSANLGANGKVNIAIVGGGATGVELSAELHNAVKQLHSYGYKGLTNEALNVTLVEAGERILPALPPRISAAAHNELTKLGVRVLTQTMVTSADEGGLHTKDGEYIEADLMVWAAGIKAPDFLKDIGGLETNRINQLVVEPTLQTTRDPDIYAIGDCASCPRPEGGFVPPRAQAAHQMATCAMNNILAQMNGKPLKNYQYKDHGSLVSLSNFSTVGSLMGNLTRGSMMIEGRIARFVYISLYRMHQIALHGYFKTGLMMLVGSINRVIRPRLKLH',\n",
       "  'mw': 47359.0},\n",
       " 'b1119': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P75959',\n",
       "  'gene': 'b1119',\n",
       "  'locus': ['b1119', 'JW1105'],\n",
       "  'synonyms': 'ycfX',\n",
       "  'properties': {'Catalytic Activity': [('ATP + N-acetyl-D-glucosamine = ADP + H(+) + N-acetyl-D-glucosamine 6-phosphate',\n",
       "     '2.7.1.59')],\n",
       "   'Function': 'Catalyzes the phosphorylation of N-acetyl-D-glucosamine (GlcNAc) derived from cell-wall degradation, yielding GlcNAc-6-P. Has also low level glucokinase activity in vitro',\n",
       "   'Regulation Activity': 'Strongly inhibited by ADP',\n",
       "   'Pathway': 'Cell wall biogenesis; peptidoglycan recycling'},\n",
       "  'seq': 'MYYGFDIGGTKIALGVFDSGRQLQWEKRVPTPRDSYDAFLDAVCELVAEADQRFGCKGSVGIGIPGMPETEDGTLYAANVPAASGKPLRADLSARLDRDVRLDNDANCFALSEAWDDEFTQYPLVMGLILGTGVGGGLIFNGKPITGKSYITGEFGHMRLPVDALTMMGLDFPLRRCGCGQHGCIENYLSGRGFAWLYQHYYHQPLQAPEIIALYDQGDEQARAHVERYLDLLAVCLGNILTIVDPDLVVIGGGLSNFPAITTQLADRLPRHLLPVARVPRIERARHGDAGGMRGAAFLHLTD',\n",
       "  'mw': 33043.0},\n",
       " 'b1123': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AFK9',\n",
       "  'gene': 'b1123',\n",
       "  'locus': ['b1123', 'JW1109'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Required for the activity of the bacterial periplasmic transport system of putrescine and spermidine. Polyamine binding protein'},\n",
       "  'seq': 'MKKWSRHLLAAGALALGMSAAHADDNNTLYFYNWTEYVPPGLLEQFTKETGIKVIYSTYESNETMYAKLKTYKDGAYDLVVPSTYYVDKMRKEGMIQKIDKSKLTNFSNLDPDMLNKPFDPNNDYSIPYIWGATAIGVNGDAVDPKSVTSWADLWKPEYKGSLLLTDDAREVFQMALRKLGYSGNTTDPKEIEAAYNELKKLMPNVAAFNSDNPANPYMEGEVNLGMIWNGSAFVARQAGTPIDVVWPKEGGIFWMDSLAIPANAKNKEGALKLINFLLRPDVAKQVAETIGYPTPNLAARKLLSPEVANDKTLYPDAETIKNGEWQNDVGAASSIYEEYYQKLKAGR',\n",
       "  'mw': 38867.0},\n",
       " 'b1124': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AFK6',\n",
       "  'gene': 'b1124',\n",
       "  'locus': ['b1124', 'JW1110'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Required for the activity of the bacterial periplasmic transport system of putrescine and spermidine'},\n",
       "  'seq': 'MIGRLLRGGFMTAIYAYLYIPIIILIVNSFNSSRFGINWQGFTTKWYSLLMNNDSLLQAAQHSLTMAVFSATFATLIGSLTAVALYRYRFRGKPFVSGMLFVVMMSPDIVMAISLLVLFMLLGIQLGFWSLLFSHITFCLPFVVVTVYSRLKGFDVRMLEAAKDLGASEFTILRKIILPLAMPAVAAGWVLSFTLSMDDVVVSSFVTGPSYEILPLKIYSMVKVGVSPEVNALATILLVLSLVMVIASQLIARDKTKGNTGDVK',\n",
       "  'mw': 29111.0},\n",
       " 'b1125': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AFK4',\n",
       "  'gene': 'b1125',\n",
       "  'locus': ['b1125', 'JW1111'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Required for the activity of the bacterial periplasmic transport system of putrescine and spermidine'},\n",
       "  'seq': 'MIVTIVGWLVLFVFLPNLMIIGTSFLTRDDASFVKMVFTLDNYTRLLDPLYFEVLLHSLNMALIATLACLVLGYPFAWFLAKLPHKVRPLLLFLLIVPFWTNSLIRIYGLKIFLSTKGYLNEFLLWLGVIDTPIRIMFTPSAVIIGLVYILLPFMVMPLYSSIEKLDKPLLEAARDLGASKLQTFIRIIIPLTMPGIIAGCLLVMLPAMGLFYVSDLMGGAKNLLIGNVIKVQFLNIRDWPFGAATSITLTIVMGLMLLVYWRASRLLNKKVELE',\n",
       "  'mw': 31062.0},\n",
       " 'b1126': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P69874',\n",
       "  'gene': 'b1126',\n",
       "  'locus': ['b1126', 'JW1112'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + H2O + polyamine-[polyamine-binding protein]Side 1 = ADP + phosphate + polyamineSide 2 + [polyamine-binding protein]Side 1.',\n",
       "     '7.6.2.11')],\n",
       "   'Function': 'Part of the ABC transporter complex PotABCD involved in spermidine/putrescine import. Responsible for energy coupling to the transport system',\n",
       "   'Regulation Activity': 'Strongly inhibited by carbonyl cyanide m-chlorophenylhydrazone (CCCP), and by spermidine via its interaction with the C-terminus. Also inhibited by N-ethylmaleimide and p-chloromercuribenzoic acid. Partially inhibited by KCN'},\n",
       "  'seq': 'MGQSKKLNKQPSSLSPLVQLAGIRKCFDGKEVIPQLDLTINNGEFLTLLGPSGCGKTTVLRLIAGLETVDSGRIMLDNEDITHVPAENRYVNTVFQSYALFPHMTVFENVAFGLRMQKTPAAEITPRVMEALRMVQLETFAQRKPHQLSGGQQQRVAIARAVVNKPRLLLLDESLSALDYKLRKQMQNELKALQRKLGITFVFVTHDQEEALTMSDRIVVMRDGRIEQDGTPREIYEEPKNLFVAGFIGEINMFNATVIERLDEQRVRANVEGRECNIYVNFAVEPGQKLHVLLRPEDLRVEEINDDNHAEGLIGYVRERNYKGMTLESVVELENGKMVMVSEFFNEDDPDFDHSLDQKMAINWVESWEVVLADEEHK',\n",
       "  'mw': 43028.0},\n",
       " 'b1131': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AB89',\n",
       "  'gene': 'b1131',\n",
       "  'locus': ['b1131', 'JW1117'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('N(6)-(1,2-dicarboxyethyl)-AMP = AMP + fumarate',\n",
       "     '4.3.2.2'),\n",
       "    ('(2S)-2-[5-amino-1-(5-phospho-beta-D-ribosyl)imidazole-4-carboxamido]succinate = 5-amino-1-(5-phospho-beta-D-ribosyl)imidazole-4-carboxamide + fumarate',\n",
       "     '4.3.2.2'),\n",
       "    (\"(2S)-2-amino-2'-deoxyadenylo-succinate = dZMP + fumarate\", '')],\n",
       "   'Function': \"(Microbial infection) Catalyzes the conversion of 2-amino-2'-deoxyadenylo-succinate to dZMP and fumarate, when the bacterium is infected by a phage that produces the substrate of this reaction, a step in the synthesis of dZTP (2-amino-2'-deoxyadenosine 5'-triphosphate), which is a nucleotide then used by the phage as a DNA polymerase substrate\",\n",
       "   'Pathway': 'Purine metabolism'},\n",
       "  'seq': 'MELSSLTAVSPVDGRYGDKVSALRGIFSEYGLLKFRVQVEVRWLQKLAAHAAIKEVPAFAADAIGYLDAIVASFSEEDAARIKTIERTTNHDVKAVEYFLKEKVAEIPELHAVSEFIHFACTSEDINNLSHALMLKTARDEVILPYWRQLIDGIKDLAVQYRDIPLLSRTHGQPATPSTIGKEMANVAYRMERQYRQLNQVEILGKINGAVGNYNAHIAAYPEVDWHQFSEEFVTSLGIQWNPYTTQIEPHDYIAELFDCVARFNTILIDFDRDVWGYIALNHFKQKTIAGEIGSSTMPHKVNPIDFENSEGNLGLSNAVLQHLASKLPVSRWQRDLTDSTVLRNLGVGIGYALIAYQSTLKGVSKLEVNRDHLLDELDHNWEVLAEPIQTVMRRYGIEKPYEKLKELTRGKRVDAEGMKQFIDGLALPEEEKARLKAMTPANYIGRAITMVDELK',\n",
       "  'mw': 51543.0},\n",
       " 'b1134': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AEI6',\n",
       "  'gene': 'b1134',\n",
       "  'locus': ['b1134', 'JW1120'],\n",
       "  'synonyms': 'ymfB',\n",
       "  'properties': {'Catalytic Activity': [('4-amino-2-methyl-5-(diphosphooxymethyl)pyrimidine + H2O = 4-amino-2-methyl-5-(phosphooxymethyl)pyrimidine + H(+) + phosphate',\n",
       "     ''),\n",
       "    ('H2O + thiamine diphosphate = H(+) + phosphate + thiamine phosphate',\n",
       "     '')],\n",
       "   'Function': 'Catalyzes the hydrolysis of 4-amino-2-methyl-5-hydroxymethylpyrimidine pyrophosphate (HMP-PP) to 4-amino-2-methyl-5-hydroxymethylpyrimidine phosphate (HMP-P), and hydrolysis of thiamine pyrophosphate (TPP) to thiamine monophosphate (TMP). Can hydrolyze other substrates such as MeO-HMP-PP, CF(3)-HMP-PP and MeO-TPP. Is also a non-specific nucleoside tri- and diphosphatase that releases inorganic orthophosphate'},\n",
       "  'seq': 'MFKPHVTVACVVHAEGKFLVVEETINGKALWNQPAGHLEADETLVEAAARELWEETGISAQPQHFIRMHQWIAPDKTPFLRFLFAIELEQICPTQPHDSDIDCCRWVSAEEILQASNLRSPLVAESIRCYQSGQRYPLEMIGDFNWPFTKGVI',\n",
       "  'mw': 17433.0},\n",
       " 'b1136': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P08200',\n",
       "  'gene': 'b1136',\n",
       "  'locus': ['b1136', 'JW1122'],\n",
       "  'synonyms': 'icdA icdE',\n",
       "  'properties': {'Catalytic Activity': [('D-threo-isocitrate + NADP(+) = 2-oxoglutarate + CO2 + NADPH',\n",
       "     '1.1.1.42')],\n",
       "   'Regulation Activity': 'Inhibition of this enzyme by phosphorylation regulates the branch point between the Krebs cycle and the glyoxylate bypass, which is an alternate route that accumulates carbon for biosynthesis when acetate is the sole carbon source for growth'},\n",
       "  'seq': 'MESKVVVPAQGKKITLQNGKLNVPENPIIPYIEGDGIGVDVTPAMLKVVDAAVEKAYKGERKISWMEIYTGEKSTQVYGQDVWLPAETLDLIREYRVAIKGPLTTPVGGGIRSLNVALRQELDLYICLRPVRYYQGTPSPVKHPELTDMVIFRENSEDIYAGIEWKADSADAEKVIKFLREEMGVKKIRFPEHCGIGIKPCSEEGTKRLVRAAIEYAIANDRDSVTLVHKGNIMKFTEGAFKDWGYQLAREEFGGELIDGGPWLKVKNPNTGKEIVIKDVIADAFLQQILLRPAEYDVIACMNLNGDYISDALAAQVGGIGIAPGANIGDECALFEATHGTAPKYAGQDKVNPGSIILSAEMMLRHMGWTEAADLIVKGMEGAINAKTVTYDFERLMDGAKLLKCSEFGDAIIENM',\n",
       "  'mw': 45757.0},\n",
       " 'b1185': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A6M2',\n",
       "  'gene': 'b1185',\n",
       "  'locus': ['b1185', 'JW5182'],\n",
       "  'synonyms': 'roxB ycgA',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Required for disulfide bond formation in some periplasmic proteins such as PhoA or OmpA. Acts by oxidizing the DsbA protein. PhoP-regulated transcription is redox-sensitive, being activated when the periplasm becomes more reducing (deletion of dsbA/dsbB, treatment with dithiothreitol). MgrB acts between DsbA/DsbB and PhoP/PhoQ in this pathway'},\n",
       "  'seq': 'MLRFLNQCSQGRGAWLLMAFTALALELTALWFQHVMLLKPCVLCIYERCALFGVLGAALIGAIAPKTPLRYVAMVIWLYSAFRGVQLTYEHTMLQLYPSPFATCDFMVRFPEWLPLDKWVPQVFVASGDCAERQWDFLGLEMPQWLLGIFIAYLIVAVLVVISQPFKAKKRDLFGR',\n",
       "  'mw': 20142.0},\n",
       " 'b1186': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AFA7',\n",
       "  'gene': 'b1186',\n",
       "  'locus': ['b1186', 'JW1175'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('3 H(+)(out) + 2 Na(+)(in) = 3 H(+)(in) + 2 Na(+)(out)',\n",
       "     '')],\n",
       "   'Function': 'Na(+)/H(+) antiporter that extrudes sodium in exchange for external protons (PubMed:1317851, PubMed:8093613, PubMed:8019504, PubMed:7929345). Catalyzes the exchange of 3 H(+) per 2 Na(+) (PubMed:7929345). Can also transport lithium (PubMed:8019504). Essential for regulation of intracellular pH under alkaline conditions (PubMed:7822245). Is necessary for growth on Na(+)/symport substrates such as glutamate and proline under conditions in which nhaA is not expressed, such as acidic pH and low Na(+) concentration (PubMed:8093613)',\n",
       "   'Regulation Activity': 'Activity is weakly pH-dependent'},\n",
       "  'seq': 'MEISWGRALWRNFLGQSPDWYKLALIIFLIVNPLIFLISPFVAGWLLVAEFIFTLAMALKCYPLLPGGLLAIEAVFIGMTSAEHVREEVAANLEVLLLLMFMVAGIYFMKQLLLFIFTRLLLSIRSKMLLSLSFCVAAAFLSAFLDALTVVAVVISVAVGFYGIYHRVASSRTEDTDLQDDSHIDKHYKVVLEQFRGFLRSLMMHAGVGTALGGVMTMVGEPQNLIIAKAAGWHFGDFFLRMSPVTVPVLICGLLTCLLVEKLRWFGYGETLPEKVREVLQQFDDQSRHQRTRQDKIRLIVQAIIGVWLVTALALHLAEVGLIGLSVIILATSLTGVTDEHAIGKAFTESLPFTALLTVFFSVVAVIIDQQLFSPIIQFVLQASEHAQLSLFYIFNGLLSSISDNVFVGTIYINEAKAAMESGAITLKQYELLAVAINTGTNLPSVATPNGQAAFLFLLTSALAPLIRLSYGRMVWMALPYTLVLTLVGLLCVEFTLAPVTEWFMQMGWIATL',\n",
       "  'mw': 56728.0},\n",
       " 'b1189': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A6J5',\n",
       "  'gene': 'b1189',\n",
       "  'locus': ['b1189', 'JW1178'],\n",
       "  'synonyms': 'dadR',\n",
       "  'properties': {'Catalytic Activity': [('A + a D-alpha-amino acid + H2O = a 2-oxocarboxylate + AH2 + NH4(+)',\n",
       "     '')],\n",
       "   'Function': 'Catalyzes the oxidative deamination of D-amino acids. Has broad substrate specificity; is mostly active on D-alanine, and to a lesser extent, on several other D-amino acids such as D-methionine, D-serine and D-proline, but not on L-alanine. Participates in the utilization of L-alanine and D-alanine as the sole source of carbon, nitrogen and energy for growth. Is also able to oxidize D-amino acid analogs such as 3,4-dehydro-D-proline, D-2-aminobutyrate, D-norvaline, D-norleucine, cis-4-hydroxy-D-proline, and DL-ethionine',\n",
       "   'Regulation Activity': 'Is activated by the HinT protein. Is inhibited by D-cycloserine',\n",
       "   'Pathway': 'Amino-acid degradation; D-alanine degradation; NH(3) and pyruvate from D-alanine: step 1/1'},\n",
       "  'seq': 'MRVVILGSGVVGVASAWYLNQAGHEVTVIDREPGAALETSAANAGQISPGYAAPWAAPGVPLKAIKWMFQRHAPLAVRLDGTQFQLKWMWQMLRNCDTSHYMENKGRMVRLAEYSRDCLKALRAETNIQYEGRQGGTLQLFRTEQQYENATRDIAVLEDAGVPYQLLESSRLAEVEPALAEVAHKLTGGLQLPNDETGDCQLFTQNLARMAEQAGVKFRFNTPVDQLLCDGEQIYGVKCGDEVIKADAYVMAFGSYSTAMLKGIVDIPVYPLKGYSLTIPIAQEDGAPVSTILDETYKIAITRFDNRIRVGGMAEIVGFNTELLQPRRETLEMVVRDLYPRGGHVEQATFWTGLRPMTPDGTPVVGRTRFKNLWLNTGHGTLGWTMACGSGQLLSDLLSGRTPAIPYEDLSVARYSRGFTPSRPGHLHGAHS',\n",
       "  'mw': 47607.0},\n",
       " 'b1190': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P29012',\n",
       "  'gene': 'b1190',\n",
       "  'locus': ['b1190', 'JW1179'],\n",
       "  'synonyms': 'alnB dadB',\n",
       "  'properties': {'Catalytic Activity': [('L-alanine = D-alanine', '5.1.1.1')],\n",
       "   'Function': 'Isomerizes L-alanine to D-alanine which is then oxidized to pyruvate by DadA'},\n",
       "  'seq': 'MTRPIQASLDLQALKQNLSIVRQAATHARVWSVVKANAYGHGIERIWSAIGATDGFALLNLEEAITLRERGWKGPILMLEGFFHAQDLEIYDQHRLTTCVHSNWQLKALQNARLKAPLDIYLKVNSGMNRLGFQPDRVLTVWQQLRAMANVGEMTLMSHFAEAEHPDGISGAMARIEQAAEGLECRRSLSNSAATLWHPEAHFDWVRPGIILYGASPSGQWRDIANTGLRPVMTLSSEIIGVQTLKAGERVGYGGRYTARDEQRIGIVAAGYADGYPRHAPTGTPVLVDGVRTMTVGTVSMDMLAVDLTPCPQAGIGTPVELWGKEIKIDDVAAAAGTVGYELMCALALRVPVVTV',\n",
       "  'mw': 38845.0},\n",
       " 'b1192': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P76008',\n",
       "  'gene': 'b1192',\n",
       "  'locus': ['b1192', 'JW1181'],\n",
       "  'synonyms': 'ycgQ',\n",
       "  'properties': {'Catalytic Activity': [('H2O + N-acetyl-D-glucosaminyl-N-acetylmuramoyl-L-alanyl-D-glutamyl-6-carboxy-L-lysyl-D-alanine = D-alanine + N-acetyl-D-glucosaminyl-N-acetylmuramoyl-L-alanyl-D-glutamyl-6-carboxy-L-lysyl',\n",
       "     '3.4.17.13')],\n",
       "   'Function': 'Releases the terminal D-alanine residue from the cytoplasmic tetrapeptide recycling product L-Ala-gamma-D-Glu-meso-Dap-D-Ala. To a lesser extent, can also cleave D-Ala from murein derivatives containing the tetrapeptide, i.e. MurNAc-tetrapeptide, UDP-MurNAc-tetrapeptide, GlcNAc-MurNAc-tetrapeptide, and GlcNAc-anhMurNAc-tetrapeptide. Does not act on murein sacculi or cross-linked muropeptides. The tripeptides produced by the LcdA reaction can then be reused as peptidoglycan building blocks; LcdA is thereby involved in murein recycling. Is also essential for viability during stationary phase',\n",
       "   'Regulation Activity': 'Inhibited by beta-lactams containing a D-amino acid side chain',\n",
       "   'Pathway': 'Cell wall biogenesis; peptidoglycan recycling'},\n",
       "  'seq': 'MSLFHLIAPSGYCIKQHAALRGIQRLTDAGHQVNNVEVIARRCERFAGTETERLEDLNSLARLTTPNTIVLAVRGGYGASRLLADIDWQALVARQQHDPLLICGHSDFTAIQCGLLAHGNVITFSGPMLVANFGADELNAFTEHHFWLALRNETFTIEWQGEGPTCRAEGTLWGGNLAMLISLIGTPWMPKIENGILVLEDINEHPFRVERMLLQLYHAGILPRQKAIILGSFSGSTPNDYDAGYNLESVYAFLRSRLSIPLITGLDFGHEQRTVTLPLGAHAILNNTREGTQLTISGHPVLKM',\n",
       "  'mw': 33567.0},\n",
       " 'b1193': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0C960',\n",
       "  'gene': 'b1193',\n",
       "  'locus': ['b1193', 'JW5821'],\n",
       "  'synonyms': 'mltE sltZ ycgP',\n",
       "  'properties': {'Catalytic Activity': [('Endolytic cleavage of the (1->4)-beta-glycosidic linkage between N-acetylmuramic acid (MurNAc) and N-acetylglucosamine (GlcNAc) residues in peptidoglycan with concomitant formation of a 1,6-anhydrobond in the MurNAc residue.',\n",
       "     '4.2.2.n2')],\n",
       "   'Function': 'Murein-degrading enzyme. May play a role in recycling of muropeptides during cell elongation and/or cell division. Preferentially cleaves at a distance of more than two disaccharide units from the ends of the glycan chain. Prefers cross-linked murein in vivo'},\n",
       "  'seq': 'MKLRWFAFLIVLLAGCSSKHDYTNPPWNAKVPVQRAMQWMPISQKAGAAWGVDPQLITAIIAIESGGNPNAVSKSNAIGLMQLKASTSGRDVYRRMGWSGEPTTSELKNPERNISMGAAYLNILETGPLAGIEDPKVLQYALVVSYANGAGALLRTFSSDRKKAISKINDLDADEFLEHVARNHPAPQAPRYIYKLEQALDAM',\n",
       "  'mw': 22227.0},\n",
       " 'b1197': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P13482',\n",
       "  'gene': 'b1197',\n",
       "  'locus': ['b1197', 'JW1186'],\n",
       "  'synonyms': 'osmA',\n",
       "  'properties': {'Catalytic Activity': [('alpha,alpha-trehalose + H2O = alpha-D-glucose + beta-D-glucose',\n",
       "     '3.2.1.28')],\n",
       "   'Function': 'Provides the cells with the ability to utilize trehalose at high osmolarity by splitting it into glucose molecules that can subsequently be taken up by the phosphotransferase-mediated uptake system'},\n",
       "  'seq': 'MKSPAPSRPQKMALIPACIFLCFAALSVQAEETPVTPQPPDILLGPLFNDVQNAKLFPDQKTFADAVPNSDPLMILADYRMQQNQSGFDLRHFVNVNFTLPKEGEKYVPPEGQSLREHIDGLWPVLTRSTENTEKWDSLLPLPEPYVVPGGRFREVYYWDSYFTMLGLAESGHWDKVADMVANFAHEIDTYGHIPNGNRSYYLSRSQPPFFALMVELLAQHEGDAALKQYLPQMQKEYAYWMDGVENLQAGQQEKRVVKLQDGTLLNRYWDDRDTPRPESWVEDIATAKSNPNRPATEIYRDLRSAAASGWDFSSRWMDNPQQLNTLRTTSIVPVDLNSLMFKMEKILARASKAAGDNAMANQYETLANARQKGIEKYLWNDQQGWYADYDLKSHKVRNQLTAAALFPLYVNAAAKDRANKMATATKTHLLQPGGLNTTSVKSGQQWDAPNGWAPLQWVATEGLQNYGQKEVAMDISWHFLTNVQHTYDREKKLVEKYDVSTTGTGGGGGEYPLQDGFGWTNGVTLKMLDLICPKEQPCDNVPATRPTVKSATTQPSTKEAQPTP',\n",
       "  'mw': 63637.0},\n",
       " 'b1198': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37349',\n",
       "  'gene': 'b1198',\n",
       "  'locus': ['b1198', 'JW5185'],\n",
       "  'synonyms': 'ycgC',\n",
       "  'properties': {'Catalytic Activity': [('dihydroxyacetone + phosphoenolpyruvate = dihydroxyacetone phosphate + pyruvate',\n",
       "     '2.7.1.121')],\n",
       "   'Function': 'Component of the dihydroxyacetone kinase complex, which is responsible for the phosphoenolpyruvate (PEP)-dependent phosphorylation of dihydroxyacetone. DhaM serves as the phosphoryl donor. Is phosphorylated by phosphoenolpyruvate in an EI- and HPr-dependent reaction, and a phosphorelay system on histidine residues finally leads to phosphoryl transfer to DhaL and dihydroxyacetone',\n",
       "   'Pathway': 'Polyol metabolism; glycerol degradation'},\n",
       "  'seq': 'MVNLVIVSHSSRLGEGVGELARQMLMSDSCKIAIAAGIDDPQNPIGTDAVKVMEAIESVADADHVLVMMDMGSALLSAETALELLAPEIAAKVRLCAAPLVEGTLAATVSAASGADIDKVIFDAMHALEAKREQLGLPSSDTEISDTCPAYDEEARSLAVVIKNRNGLHVRPASRLVYTLSTFNADMLLEKNGKCVTPESINQIALLQVRYNDTLRLIAKGPEAEEALIAFRQLAEDNFGETEEVAPPTLRPVPPVSGKAFYYQPVLCTVQAKSTLTVEEEQDRLRQAIDFTLLDLMTLTAKAEASGLDDIAAIFSGHHTLLDDPELLAAASELLQHEHCTAEYAWQQVLKELSQQYQQLDDEYLQARYIDVDDLLHRTLVHLTQTKEELPQFNSPTILLAENIYPSTVLQLDPAVVKGICLSAGSPVSHSALIARELGIGWICQQGEKLYAIQPEETLTLDVKTQRFNRQG',\n",
       "  'mw': 51449.0},\n",
       " 'b1199': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P76014',\n",
       "  'gene': 'b1199',\n",
       "  'locus': ['b1199', 'JW5186'],\n",
       "  'synonyms': 'ycgS',\n",
       "  'properties': {'Catalytic Activity': [('dihydroxyacetone + phosphoenolpyruvate = dihydroxyacetone phosphate + pyruvate',\n",
       "     '2.7.1.121')],\n",
       "   'Function': 'ADP-binding subunit of the dihydroxyacetone kinase, which is responsible for the phosphoenolpyruvate (PEP)-dependent phosphorylation of dihydroxyacetone (PubMed:11350937). DhaL-ADP is converted to DhaL-ATP via a phosphoryl group transfer from DhaM and transmits it to dihydroxyacetone bound to DhaK (PubMed:11350937). DhaL acts also as coactivator of the transcription activator DhaR by binding to the sensor domain of DhaR (PubMed:15616579). In the presence of dihydroxyacetone, DhaL-ADP displaces DhaK and stimulates DhaR activity (PubMed:15616579). In the absence of dihydroxyacetone, DhaL-ADP is converted by the PTS to DhaL-ATP, which does not bind to DhaR (PubMed:15616579)',\n",
       "   'Pathway': 'Polyol metabolism; glycerol degradation'},\n",
       "  'seq': 'MSLSRTQIVNWLTRCGDIFSTESEYLTGLDREIGDADHGLNMNRGFSKVVEKLPAIADKDIGFILKNTGMTLLSSVGGASGPLFGTFFIRAAQATQARQSLTLEELYQMFRDGADGVISRGKAEPGDKTMCDVWVPVVESLRQSSEQNLSVPVALEAASSIAESAAQSTITMQARKGRASYLGERSIGHQDPGATSVMFMMQMLALAAKE',\n",
       "  'mw': 22632.0},\n",
       " 'b1200': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P76015',\n",
       "  'gene': 'b1200',\n",
       "  'locus': ['b1200', 'JW5187'],\n",
       "  'synonyms': 'ycgT',\n",
       "  'properties': {'Catalytic Activity': [('dihydroxyacetone + phosphoenolpyruvate = dihydroxyacetone phosphate + pyruvate',\n",
       "     '2.7.1.121')],\n",
       "   'Function': 'Dihydroxyacetone binding subunit of the dihydroxyacetone kinase, which is responsible for the phosphoenolpyruvate (PEP)-dependent phosphorylation of dihydroxyacetone via a phosphoryl group transfer from DhaL-ATP (PubMed:15476397, PubMed:11350937). Binds covalently dihydroxyacetone in hemiaminal linkage (PubMed:15476397). DhaK acts also as corepressor of the transcription activator DhaR by binding to the sensor domain of DhaR (PubMed:24440518). In the presence of dihydroxyacetone, DhaL-ADP displaces DhaK and stimulates DhaR activity (PubMed:24440518). In the absence of dihydroxyacetone, DhaL-ADP is converted by the PTS to DhaL-ATP, which does not bind to DhaR (PubMed:24440518)',\n",
       "   'Regulation Activity': 'Inhibited by chloro-3-hydroxyacetone and D,L-glyceraldehyde',\n",
       "   'Pathway': 'Polyol metabolism; glycerol degradation'},\n",
       "  'seq': 'MKKLINDVQDVLDEQLAGLAKAHPSLTLHQDPVYVTRADAPVAGKVALLSGGGSGHEPMHCGYIGQGMLSGACPGEIFTSPTPDKIFECAMQVDGGEGVLLIIKNYTGDILNFETATELLHDSGVKVTTVVIDDDVAVKDSLYTAGRRGVANTVLIEKLVGAAAERGDSLDACAELGRKLNNQGHSIGIALGACTVPAAGKPSFTLADNEMEFGVGIHGEPGIDRRPFSSLDQTVDEMFDTLLVNGSYHRTLRFWDYQQGSWQEEQQTKQPLQSGDRVIALVNNLGATPLSELYGVYNRLTTRCQQAGLTIERNLIGAYCTSLDMTGFSITLLKVDDETLALWDAPVHTPALNWGK',\n",
       "  'mw': 38215.0},\n",
       " 'b1207': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A717',\n",
       "  'gene': 'b1207',\n",
       "  'locus': ['b1207', 'JW1198'],\n",
       "  'synonyms': 'prsA',\n",
       "  'properties': {'Catalytic Activity': [('ATP + D-ribose 5-phosphate = 5-phospho-alpha-D-ribose 1-diphosphate + AMP + H(+)',\n",
       "     '2.7.6.1')],\n",
       "   'Function': 'Involved in the biosynthesis of the central metabolite phospho-alpha-D-ribosyl-1-pyrophosphate (PRPP) via the transfer of pyrophosphoryl group from ATP to 1-hydroxyl of ribose-5-phosphate (Rib-5-P)',\n",
       "   'Regulation Activity': \"Activated by inorganic phosphate (PubMed:3009477, PubMed:10954724). In addition to form a complex with ATP, Mg(2+) also acts as a cofactor (PubMed:9125530). Strongly inhibited by ADP through competitive binding at the activation site and at a specific allosteric site (PubMed:3009477, PubMed:6290219,PubMed:9125530). Competitively inhibited by Ca(2+) and ribose 1,5-bisphosphate (Rib-1,5-P2) (PubMed:2542328, PubMed:9125530). Less strongly inhibited by AMP, alpha,beta-methylene ATP (mATP), (2',3'-dialdehyde)-ATP (oATP) and 1-alpha,2-alpha,3-alpha-trihydroxy-4-beta-cyclopentanemethanol 5-phosphate (cRib-5-P) (PubMed:2542328, PubMed:7657655, PubMed:9125530)\",\n",
       "   'Pathway': 'Metabolic intermediate biosynthesis; 5-phospho-alpha-D-ribose 1-diphosphate biosynthesis; 5-phospho-alpha-D-ribose 1-diphosphate from D-ribose 5-phosphate (route I): step 1/1'},\n",
       "  'seq': 'MPDMKLFAGNATPELAQRIANRLYTSLGDAAVGRFSDGEVSVQINENVRGGDIFIIQSTCAPTNDNLMELVVMVDALRRASAGRITAVIPYFGYARQDRRVRSARVPITAKVVADFLSSVGVDRVLTVDLHAEQIQGFFDVPVDNVFGSPILLEDMLQLNLDNPIVVSPDIGGVVRARAIAKLLNDTDMAIIDKRRPRANVSQVMHIIGDVAGRDCVLVDDMIDTGGTLCKAAEALKERGAKRVFAYATHPIFSGNAANNLRNSVIDEVVVCDTIPLSDEIKSLPNVRTLTLSGMLAEAIRRISNEESISAMFEH',\n",
       "  'mw': 34218.0},\n",
       " 'b1208': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P62615',\n",
       "  'gene': 'b1208',\n",
       "  'locus': ['b1208', 'JW1199'],\n",
       "  'synonyms': 'ipk ychB',\n",
       "  'properties': {'Catalytic Activity': [('4-CDP-2-C-methyl-D-erythritol + ATP = 4-CDP-2-C-methyl-D-erythritol 2-phosphate + ADP + H(+)',\n",
       "     '2.7.1.148')],\n",
       "   'Function': 'Catalyzes the phosphorylation of the position 2 hydroxy group of 4-diphosphocytidyl-2C-methyl-D-erythritol (PubMed:10570138, PubMed:10655484, Ref.6). Phosphorylates isopentenyl phosphate at low rates. Also acts on isopentenol, and, much less efficiently, dimethylallyl alcohol. Dimethylallyl monophosphate does not serve as a substrate (PubMed:10570138)',\n",
       "   'Pathway': 'Isoprenoid biosynthesis; isopentenyl diphosphate biosynthesis via DXP pathway; isopentenyl diphosphate from 1-deoxy-D-xylulose 5-phosphate: step 3/6'},\n",
       "  'seq': 'MRTQWPSPAKLNLFLYITGQRADGYHTLQTLFQFLDYGDTISIELRDDGDIRLLTPVEGVEHEDNLIVRAARLLMKTAADSGRLPTGSGANISIDKRLPMGGGLGGGSSNAATVLVALNHLWQCGLSMDELAEMGLTLGADVPVFVRGHAAFAEGVGEILTPVDPPEKWYLVAHPGVSIPTPVIFKDPELPRNTPKRSIETLLKCEFSNDCEVIARKRFREVDAVLSWLLEYAPSRLTGTGACVFAEFDTESEARQVLEQAPEWLNGFVAKGANLSPLHRAML',\n",
       "  'mw': 30925.0},\n",
       " 'b1210': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A6X1',\n",
       "  'gene': 'b1210',\n",
       "  'locus': ['b1210', 'JW1201'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(S)-4-amino-5-oxopentanoate + NADP(+) + tRNA(Glu) = H(+) + L-glutamyl-tRNA(Glu) + NADPH',\n",
       "     '1.2.1.70')],\n",
       "   'Function': 'Catalyzes the NADPH-dependent reduction of glutamyl-tRNA(Glu) to glutamate 1-semialdehyde (GSA). In the absence of NADPH, exhibits substrate esterase activity, leading to the release of glutamate from tRNA',\n",
       "   'Regulation Activity': \"Activated by Mg(2+) ions. Inhibited by metal-chelating agents such as EDTA, EGTA, 1,10-phenanthroline, 2,2'-dipyridyl, and by PtCl4 and KPdCl4 as well as Ni(2+) and Co(2+). Also inhibited by iodoacetamide, N-tosyl-L-phenylalanine chloromethyl ketone, and 5,5'-dithiobis(2-nitrobenzoic acid), as well as glutamycin\",\n",
       "   'Pathway': 'Porphyrin-containing compound metabolism; protoporphyrin-IX biosynthesis; 5-aminolevulinate from L-glutamyl-tRNA(Glu): step 1/2'},\n",
       "  'seq': 'MTLLALGINHKTAPVSLRERVSFSPDKLDQALDSLLAQPMVQGGVVLSTCNRTELYLSVEEQDNLQEALIRWLCDYHNLNEEDLRKSLYWHQDNDAVSHLMRVASGLDSLVLGEPQILGQVKKAFADSQKGHMKASELERMFQKSFSVAKRVRTETDIGASAVSVAFAACTLARQIFESLSTVTVLLVGAGETIELVARHLREHKVQKMIIANRTRERAQILADEVGAEVIALSDIDERLREADIIISSTASPLPIIGKGMVERALKSRRNQPMLLVDIAVPRDVEPEVGKLANAYLYSVDDLQSIISHNLAQRKAAAVEAETIVAQETSEFMAWLRAQSASETIREYRSQAEQVRDELTAKALAALEQGGDAQAIMQDLAWKLTNRLIHAPTKSLQQAARDGDNERLNILRDSLGLE',\n",
       "  'mw': 46307.0},\n",
       " 'b1215': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A715',\n",
       "  'gene': 'b1215',\n",
       "  'locus': ['b1215', 'JW1206'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('D-arabinose 5-phosphate + H2O + phosphoenolpyruvate = 3-deoxy-alpha-D-manno-2-octulosonate-8-phosphate + phosphate',\n",
       "     '2.5.1.55')],\n",
       "   'Function': 'Synthesis of KDO 8-P which is required for lipid A maturation and cellular growth',\n",
       "   'Pathway': 'Bacterial outer membrane biogenesis; lipopolysaccharide biosynthesis'},\n",
       "  'seq': 'MKQKVVSIGDINVANDLPFVLFGGMNVLESRDLAMRICEHYVTVTQKLGIPYVFKASFDKANRSSIHSYRGPGLEEGMKIFQELKQTFGVKIITDVHEPSQAQPVADVVDVIQLPAFLARQTDLVEAMAKTGAVINVKKPQFVSPGQMGNIVDKFKEGGNEKVILCDRGANFGYDNLVVDMLGFSIMKKVSGNSPVIFDVTHALQCRDPFGAASGGRRAQVAELARAGMAVGLAGLFIEAHPDPEHAKCDGPSALPLAKLEPFLKQMKAIDDLVKGFEELDTSK',\n",
       "  'mw': 30833.0},\n",
       " 'b1216': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P31801',\n",
       "  'gene': 'b1216',\n",
       "  'locus': ['b1216', 'JW1207'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('H(+)(out) + Na(+)(in) = H(+)(in) + Na(+)(out)',\n",
       "     ''),\n",
       "    ('H(+)(out) + K(+)(in) = H(+)(in) + K(+)(out)', ''),\n",
       "    ('Ca(2+)(in) + H(+)(out) = Ca(2+)(out) + H(+)(in)', '')],\n",
       "   'Function': 'Sodium exporter that functions mainly at alkaline pH (PubMed:8496184, PubMed:9518629, PubMed:8021217). Can also function as a potassium/proton and calcium/proton antiporter at alkaline pH (PubMed:8496184, PubMed:16687400, PubMed:8021217, PubMed:18342619). Does not play a major role in calcium export (PubMed:18342619). The K(+)/H(+) antiporter activity may enable E.coli to adapt to K(+) salinity stress and to maintain K(+) homeostasis (PubMed:16687400)',\n",
       "   'Regulation Activity': 'Pronounced pH dependence with sodium as substrate (PubMed:8496184). Ca(2+)/H(+) and Na(+)/H(+) antiporter activities are both inhibited by magnesium (PubMed:8496184). Ca(2+)/H(+) activity is inhibited by the proton ionophore carbonyl cyanide m-chlorophenylhydrazone (CCCP) (PubMed:8496184)'},\n",
       "  'seq': 'MSNAQEAVKTRHKETSLIFPVLALVVLFLWGSSQTLPVVIAINLLALIGILSSAFSVVRHADVLAHRLGEPYGSLILSLSVVILEVSLISALMATGDAAPTLMRDTLYSIIMIVTGGLVGFSLLLGGRKFATQYMNLFGIKQYLIALFPLAIIVLVFPMALPAANFSTGQALLVALISAAMYGVFLLIQTKTHQSLFVYEHEDDSDDDDPHHGKPSAHSSLWHAIWLIIHLIAVIAVTKMNASSLETLLDSMNAPVAFTGFLVALLILSPEGLGALKAVLNNQVQRAMNLFFGSVLATISLTVPVVTLIAFMTGNELQFALGAPEMVVMVASLVLCHISFSTGRTNVLNGAAHLALFAAYLMTIFA',\n",
       "  'mw': 39168.0},\n",
       " 'b1223': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P10903',\n",
       "  'gene': 'b1223',\n",
       "  'locus': ['b1223', 'JW1214'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('nitrate(in) + nitrite(out) = nitrate(out) + nitrite(in)',\n",
       "     '')],\n",
       "   'Function': 'Catalyzes nitrate uptake, nitrite uptake and nitrite export across the cytoplasmic membrane (PubMed:2668029, PubMed:11967075, PubMed:15667293, PubMed:16804183, PubMed:23665960, PubMed:25959928). Functions as a nitrate/nitrite exchanger, and protons are unlikely to be co-transported (PubMed:2668029, PubMed:15667293, PubMed:23665960, PubMed:25959928)'},\n",
       "  'seq': 'MSHSSAPERATGAVITDWRPEDPAFWQQRGQRIASRNLWISVPCLLLAFCVWMLFSAVAVNLPKVGFNFTTDQLFMLTALPSVSGALLRVPYSFMVPIFGGRRWTAFSTGILIIPCVWLGFAVQDTSTPYSVFIIISLLCGFAGANFASSMANISFFFPKQKQGGALGLNGGLGNMGVSVMQLVAPLVVSLSIFAVFGSQGVKQPDGTELYLANASWIWVPFLAIFTIAAWFGMNDLATSKASIKEQLPVLKRGHLWIMSLLYLATFGSFIGFSAGFAMLSKTQFPDVQILQYAFFGPFIGALARSAGGALSDRLGGTRVTLVNFILMAIFSGLLFLTLPTDGQGGSFMAFFAVFLALFLTAGLGSGSTFQMISVIFRKLTMDRVKAEGGSDERAMREAATDTAAALGFISAIGAIGGFFIPKAFGSSLALTGSPVGAMKVFLIFYIACVVITWAVYGRHSKK',\n",
       "  'mw': 49693.0},\n",
       " 'b1224': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P09152',\n",
       "  'gene': 'b1224',\n",
       "  'locus': ['b1224', 'JW1215'],\n",
       "  'synonyms': 'bisD narC',\n",
       "  'properties': {'Catalytic Activity': [('a quinol + nitrate = a quinone + H2O + nitrite',\n",
       "     '1.7.5.1')],\n",
       "   'Function': 'The nitrate reductase enzyme complex allows E.coli to use nitrate as an electron acceptor during anaerobic growth. The alpha chain is the actual site of nitrate reduction'},\n",
       "  'seq': 'MSKFLDRFRYFKQKGETFADGHGQLLNTNRDWEDGYRQRWQHDKIVRSTHGVNCTGSCSWKIYVKNGLVTWETQQTDYPRTRPDLPNHEPRGCPRGASYSWYLYSANRLKYPMMRKRLMKMWREAKALHSDPVEAWASIIEDADKAKSFKQARGRGGFVRSSWQEVNELIAASNVYTIKNYGPDRVAGFSPIPAMSMVSYASGARYLSLIGGTCLSFYDWYCDLPPASPQTWGEQTDVPESADWYNSSYIIAWGSNVPQTRTPDAHFFTEVRYKGTKTVAVTPDYAEIAKLCDLWLAPKQGTDAAMALAMGHVMLREFHLDNPSQYFTDYVRRYTDMPMLVMLEERDGYYAAGRMLRAADLVDALGQENNPEWKTVAFNTNGEMVAPNGSIGFRWGEKGKWNLEQRDGKTGEETELQLSLLGSQDEIAEVGFPYFGGDGTEHFNKVELENVLLHKLPVKRLQLADGSTALVTTVYDLTLANYGLERGLNDVNCATSYDDVKAYTPAWAEQITGVSRSQIIRIAREFADNADKTHGRSMIIVGAGLNHWYHLDMNYRGLINMLIFCGCVGQSGGGWAHYVGQEKLRPQTGWQPLAFALDWQRPARHMNSTSYFYNHSSQWRYETVTAEELLSPMADKSRYTGHLIDFNVRAERMGWLPSAPQLGTNPLTIAGEAEKAGMNPVDYTVKSLKEGSIRFAAEQPENGKNHPRNLFIWRSNLLGSSGKGHEFMLKYLLGTEHGIQGKDLGQQGGVKPEEVDWQDNGLEGKLDLVVTLDFRLSSTCLYSDIILPTATWYEKDDMNTSDMHPFIHPLSAAVDPAWEAKSDWEIYKAIAKKFSEVCVGHLGKETDIVTLPIQHDSAAELAQPLDVKDWKKGECDLIPGKTAPHIMVVERDYPATYERFTSIGPLMEKIGNGGKGIAWNTQSEMDLLRKLNYTKAEGPAKGQPMLNTAIDAAEMILTLAPETNGQVAVKAWAALSEFTGRDHTHLALNKEDEKIRFRDIQAQPRKIISSPTWSGLEDEHVSYNAGYTNVHELIPWRTLSGRQQLYQDHQWMRDFGESLLVYRPPIDTRSVKEVIGQKSNGNQEKALNFLTPHQKWGIHSTYSDNLLMLTLGRGGPVVWLSEADAKDLGIADNDWIEVFNSNGALTARAVVSQRVPAGMTMMYHAQERIVNLPGSEITQQRGGIHNSVTRITPKPTHMIGGYAHLAYGFNYYGTVGSNRDEFVVVRKMKNIDWLDGEGNDQVQESVK',\n",
       "  'mw': 140489.0},\n",
       " 'b1225': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P11349',\n",
       "  'gene': 'b1225',\n",
       "  'locus': ['b1225', 'JW1216'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('a quinol + nitrate = a quinone + H2O + nitrite',\n",
       "     '1.7.5.1')],\n",
       "   'Function': 'The nitrate reductase enzyme complex allows E.coli to use nitrate as an electron acceptor during anaerobic growth. The beta chain is an electron transfer unit containing four cysteine clusters involved in the formation of iron-sulfur centers. Electrons are transferred from the gamma chain to the molybdenum cofactor of the alpha subunit'},\n",
       "  'seq': 'MKIRSQVGMVLNLDKCIGCHTCSVTCKNVWTSREGVEYAWFNNVETKPGQGFPTDWENQEKYKGGWIRKINGKLQPRMGNRAMLLGKIFANPHLPGIDDYYEPFDFDYQNLHTAPEGSKSQPIARPRSLITGERMAKIEKGPNWEDDLGGEFDKLAKDKNFDNIQKAMYSQFENTFMMYLPRLCEHCLNPACVATCPSGAIYKREEDGIVLIDQDKCRGWRMCITGCPYKKIYFNWKSGKSEKCIFCYPRIEAGQPTVCSETCVGRIRYLGVLLYDADAIERAASTENEKDLYQRQLDVFLDPNDPKVIEQAIKDGIPLSVIEAAQQSPVYKMAMEWKLALPLHPEYRTLPMVWYVPPLSPIQSAADAGELGSNGILPDVESLRIPVQYLANLLTAGDTKPVLRALKRMLAMRHYKRAETVDGKVDTRALEEVGLTEAQAQEMYRYLAIANYEDRFVVPSSHRELAREAFPEKNGCGFTFGDGCHGSDTKFNLFNSRRIDAIDVTSKTEPHP',\n",
       "  'mw': 58066.0},\n",
       " 'b1226': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AF26',\n",
       "  'gene': 'b1226',\n",
       "  'locus': ['b1226', 'JW1217'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Chaperone required for proper molybdenum cofactor insertion and final assembly of the membrane-bound respiratory nitrate reductase 1. Required for the insertion of the molybdenum into the apo-NarG subunit, maybe by keeping NarG in an appropriate competent-open conformation for the molybdenum cofactor insertion to occur. NarJ maintains the apoNarGH complex in a soluble state. Upon insertion of the molybdenum cofactor, NarJ seems to dissociate from the activated soluble NarGH complex, before its association with the NarI subunit on the membrane'},\n",
       "  'seq': 'MIELVIVSRLLEYPDAALWQHQQEMFEAIAASKNLPKEDAHALGIFLRDLTTMDPLDAQAQYSELFDRGRATSLLLFEHVHGESRDRGQAMVDLLAQYEQHGLQLNSRELPDHLPLYLEYLAQLPQSEAVEGLKDIAPILALLSARLQQRESRYAVLFDLLLKLANTAIDSDKVAEKIADEARDDTPQALDAVWEEEQVKFFADKGCGDSAITAHQRRFAGAVAPQYLNITTGGQH',\n",
       "  'mw': 26449.0},\n",
       " 'b1227': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P11350',\n",
       "  'gene': 'b1227',\n",
       "  'locus': ['b1227', 'JW1218'],\n",
       "  'synonyms': 'chlI',\n",
       "  'properties': {'Catalytic Activity': [('a quinol + nitrate = a quinone + H2O + nitrite',\n",
       "     '1.7.5.1')],\n",
       "   'Function': 'The nitrate reductase enzyme complex allows E.coli to use nitrate as an electron acceptor during anaerobic growth. The gamma chain is a membrane-embedded heme-iron unit resembling cytochrome b, which transfers electrons from quinones to the beta subunit'},\n",
       "  'seq': 'MQFLNMFFFDIYPYIAGAVFLIGSWLRYDYGQYTWRAASSQMLDRKGMNLASNLFHIGILGIFVGHFFGMLTPHWMYEAWLPIEVKQKMAMFAGGASGVLCLIGGVLLLKRRLFSPRVRATTTGADILILSLLVIQCALGLLTIPFSAQHMDGSEMMKLVGWAQSVVTFHGGASQHLDGVAFIFRLHLVLGMTLFLLFPFSRLIHIWSVPVEYLTRKYQLVRARH',\n",
       "  'mw': 25497.0},\n",
       " 'b1232': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37051',\n",
       "  'gene': 'b1232',\n",
       "  'locus': ['b1232', 'JW1220'],\n",
       "  'synonyms': 'tgs ychI',\n",
       "  'properties': {'Catalytic Activity': [('(6S)-10-formyltetrahydrofolate + H2O = (6S)-5,6,7,8-tetrahydrofolate + formate + H(+)',\n",
       "     '3.5.1.10')],\n",
       "   'Function': \"Catalyzes the hydrolysis of 10-formyltetrahydrofolate (formyl-FH4) to formate and tetrahydrofolate (FH4). Provides the major source of formate for the PurT-dependent synthesis of 5'-phosphoribosyl-N-formylglycinamide (FGAR) during aerobic growth. Has a role in regulating the one-carbon pool\",\n",
       "   'Regulation Activity': 'Activated by methionine, inhibited by glycine',\n",
       "   'Pathway': 'Purine metabolism; IMP biosynthesis via de novo pathway; formate from 10-formyl-5,6,7,8-tetrahydrofolate: step 1/1'},\n",
       "  'seq': 'MHSLQRKVLRTICPDQKGLIARITNICYKHELNIVQNNEFVDHRTGRFFMRTELEGIFNDSTLLADLDSALPEGSVRELNPAGRRRIVILVTKEAHCLGDLLMKANYGGLDVEIAAVIGNHDTLRSLVERFDIPFELVSHEGLTRNEHDQKMADAIDAYQPDYVVLAKYMRVLTPEFVARFPNKIINIHHSFLPAFIGARPYHQAYERGVKIIGATAHYVNDNLDEGPIIMQDVIHVDHTYTAEDMMRAGRDVEKNVLSRALYKVLAQRVFVYGNRTIIL',\n",
       "  'mw': 31935.0},\n",
       " 'b1236': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AEP3',\n",
       "  'gene': 'b1236',\n",
       "  'locus': ['b1236', 'JW1224'],\n",
       "  'synonyms': 'ychD',\n",
       "  'properties': {'Catalytic Activity': [('alpha-D-glucose 1-phosphate + H(+) + UTP = diphosphate + UDP-alpha-D-glucose',\n",
       "     '2.7.7.9')],\n",
       "   'Function': 'May play a role in stationary phase survival'},\n",
       "  'seq': 'MAAINTKVKKAVIPVAGLGTRMLPATKAIPKEMLPLVDKPLIQYVVNECIAAGITEIVLVTHSSKNSIENHFDTSFELEAMLEKRVKRQLLDEVQSICPPHVTIMQVRQGLAKGLGHAVLCAHPVVGDEPVAVILPDVILDEYESDLSQDNLAEMIRRFDETGHSQIMVEPVADVTAYGVVDCKGVELAPGESVPMVGVVEKPKADVAPSNLAIVGRYVLSADIWPLLAKTPPGAGDEIQLTDAIDMLIEKETVEAYHMKGKSHDCGNKLGYMQAFVEYGIRHNTLGTEFKAWLEEEMGIKK',\n",
       "  'mw': 32942.0},\n",
       " 'b1238': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P23331',\n",
       "  'gene': 'b1238',\n",
       "  'locus': ['b1238', 'JW1226'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + thymidine = ADP + dTMP + H(+)',\n",
       "     '2.7.1.21')],\n",
       "   'Function': 'Phosphorylates both thymidine and deoxyuridine',\n",
       "   'Regulation Activity': 'Allosteric enzyme which is feedback inhibited by dTTP and activated by a number of dNDP and dNTP'},\n",
       "  'seq': 'MAQLYFYYSAMNAGKSTALLQSSYNYQERGMRTVVYTAEIDDRFGAGKVSSRIGLSSPAKLFNQNSSLFDEIRAEHEQQAIHCVLVDECQFLTRQQVYELSEVVDQLDIPVLCYGLRTDFRGELFIGSQYLLAWSDKLVELKTICFCGRKASMVLRLDQAGRPYNEGEQVVIGGNERYVSVCRKHYKEALQVDSLTAIQERHRHD',\n",
       "  'mw': 23457.0},\n",
       " 'b1241': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A9Q7',\n",
       "  'gene': 'b1241',\n",
       "  'locus': ['b1241', 'JW1228'],\n",
       "  'synonyms': 'ana',\n",
       "  'properties': {'Catalytic Activity': [('acetaldehyde + CoA + NAD(+) = acetyl-CoA + H(+) + NADH',\n",
       "     '1.2.1.10'),\n",
       "    ('ethanol + NAD(+) = acetaldehyde + H(+) + NADH', '1.1.1.1'),\n",
       "    ('a primary alcohol + NAD(+) = an aldehyde + H(+) + NADH', '1.1.1.1')],\n",
       "   'Function': 'Under fermentative conditions, catalyzes the sequential NADH-dependent reduction of acetyl-CoA to acetaldehyde and then to ethanol (PubMed:6752127, PubMed:2015910, PubMed:10612730, PubMed:10922373, PubMed:17280641). Under aerobic conditions, despite the reversibility of the two NADH-coupled reactions, wild-type E.coli cannot grow on ethanol as the sole carbon and energy source due to the down-regulation of adhE transcription and the inactivation of the enzyme by metal-catalyzed oxidation (MCO) (PubMed:10922373). Nevertheless, in the presence of oxygen, AdhE may act as a hydrogen peroxide scavenger and have a protective role against oxidative stress (PubMed:12783863)',\n",
       "   'Regulation Activity': 'Formation of filaments, called spirosomes, is critical for activity (PubMed:31586059, PubMed:32188856, PubMed:32523125). Filamentation is essential for substrate channeling between the ALDH and ADH domains and efficient coupling of the ALDH and ADH enzymatic activities, and for the regulation of enzyme activity (PubMed:32188856, PubMed:32523125). In the absence of ligands, the spirosomes are compact (PubMed:31586059, PubMed:32188856, PubMed:32523125). Binding of NAD(+) and Fe(2+) cofactors induces conformational changes and extension of the filaments, leading to the activation of the enzyme (PubMed:31586059, PubMed:32188856, PubMed:32523125). AdhE is inhibited in the presence of oxygen by metal-catalyzed oxidation (MCO) and by heat stress (PubMed:11917132, PubMed:19238259). The chaperone DnaK has a protective effect on AdhE activity under aerobic but not anaerobic conditions (PubMed:11917132). DnaK may protect AdhE and AdhE mutants against irreversible MCO damages and improve their ability to grow aerobically on ethanol (PubMed:11917132). IbpA and IbpB protect AdhE against thermal and oxidative inactivation during oxidative stress, providing that the enzyme remained soluble, and increase the efficiency of reactivation of heat-denatured AdhE by the DnaK system (PubMed:19238259). Acetaldehyde dehydrogenase activity is inhibited by guaiacol, which mimics an intermediate of ubiquinone synthesis (PubMed:10612730)'},\n",
       "  'seq': 'MAVTNVAELNALVERVKKAQREYASFTQEQVDKIFRAAALAAADARIPLAKMAVAESGMGIVEDKVIKNHFASEYIYNAYKDEKTCGVLSEDDTFGTITIAEPIGIICGIVPTTNPTSTAIFKSLISLKTRNAIIFSPHPRAKDATNKAADIVLQAAIAAGAPKDLIGWIDQPSVELSNALMHHPDINLILATGGPGMVKAAYSSGKPAIGVGAGNTPVVIDETADIKRAVASVLMSKTFDNGVICASEQSVVVVDSVYDAVRERFATHGGYLLQGKELKAVQDVILKNGALNAAIVGQPAYKIAELAGFSVPENTKILIGEVTVVDESEPFAHEKLSPTLAMYRAKDFEDAVEKAEKLVAMGGIGHTSCLYTDQDNQPARVSYFGQKMKTARILINTPASQGGIGDLYNFKLAPSLTLGCGSWGGNSISENVGPKHLINKKTVAKRAENMLWHKLPKSIYFRRGSLPIALDEVITDGHKRALIVTDRFLFNNGYADQITSVLKAAGVETEVFFEVEADPTLSIVRKGAELANSFKPDVIIALGGGSPMDAAKIMWVMYEHPETHFEELALRFMDIRKRIYKFPKMGVKAKMIAVTTTSGTGSEVTPFAVVTDDATGQKYPLADYALTPDMAIVDANLVMDMPKSLCAFGGLDAVTHAMEAYVSVLASEFSDGQALQALKLLKEYLPASYHEGSKNPVARERVHSAATIAGIAFANAFLGVCHSMAHKLGSQFHIPHGLANALLICNVIRYNANDNPTKQTAFSQYDRPQARRRYAEIADHLGLSAPGDRTAAKIEKLLAWLETLKAELGIPKSIREAGVQEADFLANVDKLSEDAFDDQCTGANPRYPLISELKQILLDTYYGRDYVEGETAAKKEAAPAKAEKKAKKSA',\n",
       "  'mw': 96127.0},\n",
       " 'b1243': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P23843',\n",
       "  'gene': 'b1243',\n",
       "  'locus': ['b1243', 'JW1235'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'This protein is a component of the oligopeptide permease, a binding protein-dependent transport system, it binds peptides up to five amino acids long with high affinity'},\n",
       "  'seq': 'MTNITKRSLVAAGVLAALMAGNVALAADVPAGVTLAEKQTLVRNNGSEVQSLDPHKIEGVPESNISRDLFEGLLVSDLDGHPAPGVAESWDNKDAKVWTFHLRKDAKWSDGTPVTAQDFVYSWQRSVDPNTASPYASYLQYGHIAGIDEILEGKKPITDLGVKAIDDHTLEVTLSEPVPYFYKLLVHPSTSPVPKAAIEKFGEKWTQPGNIVTNGAYTLKDWVVNERIVLERSPTYWNNAKTVINQVTYLPIASEVTDVNRYRSGEIDMTNNSMPIELFQKLKKEIPDEVHVDPYLCTYYYEINNQKPPFNDVRVRTALKLGMDRDIIVNKVKAQGNMPAYGYTPPYTDGAKLTQPEWFGWSQEKRNEEAKKLLAEAGYTADKPLTINLLYNTSDLHKKLAIAASSLWKKNIGVNVKLVNQEWKTFLDTRHQGTFDVARAGWCADYNEPTSFLNTMLSNSSMNTAHYKSPAFDSIMAETLKVTDEAQRTALYTKAEQQLDKDSAIVPVYYYVNARLVKPWVGGYTGKDPLDNTYTRNMYIVKH',\n",
       "  'mw': 60899.0},\n",
       " 'b1244': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AFH2',\n",
       "  'gene': 'b1244',\n",
       "  'locus': ['b1244', 'JW1236'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the binding-protein-dependent transport system for oligopeptides; probably responsible for the translocation of the substrate across the membrane'},\n",
       "  'seq': 'MLKFILRRCLEAIPTLFILITISFFMMRLAPGSPFTGERTLPPEVMANIEAKYHLNDPIMTQYFSYLKQLAHGDFGPSFKYKDYSVNDLVASSFPVSAKLGAAAFFLAVILGVSAGVIAALKQNTKWDYTVMGLAMTGVVIPSFVVAPLLVMIFAIILHWLPGGGWNGGALKFMILPMVALSLAYIASIARITRGSMIEVLHSNFIRTARAKGLPMRRIILRHALKPALLPVLSYMGPAFVGIITGSMVIETIYGLPGIGQLFVNGALNRDYSLVLSLTILVGALTILFNAIVDVLYAVIDPKIRY',\n",
       "  'mw': 33443.0},\n",
       " 'b1245': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AFH6',\n",
       "  'gene': 'b1245',\n",
       "  'locus': ['b1245', 'JW1237'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the binding-protein-dependent transport system for oligopeptides; probably responsible for the translocation of the substrate across the membrane'},\n",
       "  'seq': 'MMLSKKNSETLENFSEKLEVEGRSLWQDARRRFMHNRAAVASLIVLVLIALFVILAPMLSQFAYDDTDWAMMSSAPDMESGHYFGTDSSGRDLLVRVAIGGRISLMVGVAAALVAVVVGTLYGSLSGYLGGKVDSVMMRLLEILNSFPFMFFVILLVTFFGQNILLIFVAIGMVSWLDMARIVRGQTLSLKRKEFIEAAQVGGVSTSGIVIRHIVPNVLGVVVVYASLLVPSMILFESFLSFLGLGTQEPLSSWGALLSDGANSMEVSPWLLLFPAGFLVVTLFCFNFIGDGLRDALDPKDR',\n",
       "  'mw': 33022.0},\n",
       " 'b1246': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P76027',\n",
       "  'gene': 'b1246',\n",
       "  'locus': ['b1246', 'JW1238'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the binding-protein-dependent transport system for oligopeptides. Probably responsible for energy coupling to the transport system'},\n",
       "  'seq': 'MSVIETATVPLAQQQADALLNVKDLRVTFSTPDGDVTAVNDLNFSLRAGETLGIVGESGSGKSQTAFALMGLLAANGRIGGSATFNGREILNLPEHELNKLRAEQISMIFQDPMTSLNPYMRVGEQLMEVLMLHKNMSKAEAFEESVRMLDAVKMPEARKRMKMYPHEFSGGMRQRVMIAMALLCRPKLLIADEPTTALDVTVQAQIMTLLNELKREFNTAIIMITHDLVVVAGICDKVLVMYAGRTMEYGNARDVFYQPVHPYSIGLLNAVPRLDAEGETMLTIPGNPPNLLRLPKGCPFQPRCPHAMEICSSAPPLEEFTPGRLRACFKPVEELL',\n",
       "  'mw': 37188.0},\n",
       " 'b1247': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77737',\n",
       "  'gene': 'b1247',\n",
       "  'locus': ['b1247', 'JW1239'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the binding-protein-dependent transport system for oligopeptides. Probably responsible for energy coupling to the transport system'},\n",
       "  'seq': 'MNAVTEGRKVLLEIADLKVHFEIKDGKQWFWQPPKTLKAVDGVTLRLYEGETLGVVGESGCGKSTFARAIIGLVKATDGHVAWLGKELLGMKPDEWRAVRSDIQMIFQDPLASLNPRMTIGEIIAEPLRTYHPKMSRQEVRERVKAMMLKVGLLPNLINRYPHEFSGGQCQRIGIARALILEPKLIICDEPVSALDVSIQAQVVNLLQQLQREMGLSLIFIAHDLAVVKHISDRVLVMYLGHAVELGTYDEVYHNPLHPYTRALMSAVPIPDPDLEKNKTIQLLEGELPSPINPPSGCVFRTRCPIAGPECAKTRPVLEGSFRHSVSCLKVDPL',\n",
       "  'mw': 37198.0},\n",
       " 'b1249': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A6H8',\n",
       "  'gene': 'b1249',\n",
       "  'locus': ['b1249', 'JW1241'],\n",
       "  'synonyms': 'cls nov yciJ',\n",
       "  'properties': {'Catalytic Activity': [(\"2 1,2-diacyl-sn-glycero-3-phospho-(1'-sn-glycerol) = a cardiolipin + glycerol\",\n",
       "     '')],\n",
       "   'Function': 'Catalyzes the reversible phosphatidyl group transfer from one phosphatidylglycerol molecule to another to form cardiolipin (CL) (diphosphatidylglycerol) and glycerol'},\n",
       "  'seq': 'MTTVYTLVSWLAILGYWLLIAGVTLRILMKRRAVPSAMAWLLIIYILPLVGIIAYLAVGELHLGKRRAERARAMWPSTAKWLNDLKACKHIFAEENSSVAAPLFKLCERRQGIAGVKGNQLQLMTESDDVMQALIRDIQLARHNIEMVFYIWQPGGMADQVAESLMAAARRGIHCRLMLDSAGSVAFFRSPWPELMRNAGIEVVEALKVNLMRVFLRRMDLRQHRKMIMIDNYIAYTGSMNMVDPRYFKQDAGVGQWIDLMARMEGPIATAMGIIYSCDWEIETGKRILPPPPDVNIMPFEQASGHTIHTIASGPGFPEDLIHQALLTAAYSAREYLIMTTPYFVPSDDLLHAICTAAQRGVDVSIILPRKNDSMLVGWASRAFFTELLAAGVKIYQFEGGLLHTKSVLVDGELSLVGTVNLDMRSLWLNFEITLAIDDKGFGADLAAVQDDYISRSRLLDARLWLKRPLWQRVAERLFYFFSPLL',\n",
       "  'mw': 54822.0},\n",
       " 'b1250': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P31069',\n",
       "  'gene': 'b1250',\n",
       "  'locus': ['b1250', 'JW1242'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'K(+)-specific ion channel. May play a role in the defense against osmotic shock'},\n",
       "  'seq': 'MSHWATFKQTATNLWVTLRHDILALAVFLNGLLIFKTIYGMSVNLLDIFHIKAFSELDLSLLANAPLFMLGVFLVLNSIGLLFRAKLAWAISIILLLIALIYTLHFYPWLKFSIGFCIFTLVFLLILRKDFSHSSAAAGTIFAFISFTTLLFYSTYGALYLSEGFNPRIESLMTAFYFSIETMSTVGYGDIVPVSESARLFTISVIISGITVFATSMTSIFGPLIRGGFNKLVKGNNHTMHRKDHFIVCGHSILAINTILQLNQRGQNVTVISNLPEDDIKQLEQRLGDNADVIPGDSNDSSVLKKAGIDRCRAILALSDNDADNAFVVLSAKDMSSDVKTVLAVSDSKNLNKIKMVHPDIILSPQLFGSEILARVLNGEEINNDMLVSMLLNSGHGIFSDNDELETKADSKESAQK',\n",
       "  'mw': 46063.0},\n",
       " 'b1252': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P02929',\n",
       "  'gene': 'b1252',\n",
       "  'locus': ['b1252', 'JW5195'],\n",
       "  'synonyms': 'exbA',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Interacts with outer membrane receptor proteins that carry out high-affinity binding and energy dependent uptake into the periplasmic space of specific substrates such as cobalamin, and various iron compounds (such as iron dicitrate, enterochelin, aerobactin, etc.). In the absence of TonB these receptors bind their substrates but do not carry out active transport. TonB also interacts with some colicins and is involved in the energy-dependent, irreversible steps of bacteriophages phi 80 and T1 infection. It could act to transduce energy from the cytoplasmic membrane to specific energy-requiring processes in the outer membrane, resulting in the release into the periplasm of ligands bound by these outer membrane proteins. Implicated in hydroxy radical-mediated cell death induced by hydroxyurea treatment (PubMed:20005847)'},\n",
       "  'seq': 'MTLDLPRRFPWPTLLSVCIHGAVVAGLLYTSVHQVIELPAPAQPISVTMVTPADLEPPQAVQPPPEPVVEPEPEPEPIPEPPKEAPVVIEKPKPKPKPKPKPVKKVQEQPKRDVKPVESRPASPFENTAPARLTSSTATAATSKPVTSVASGPRALSRNQPQYPARAQALRIEGQVKVKFDVTPDGRVDNVQILSAKPANMFEREVKNAMRRWRYEPGKPGSGIVVNILFKINGTTEIQ',\n",
       "  'mw': 26094.0},\n",
       " 'b1260': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A877',\n",
       "  'gene': 'b1260',\n",
       "  'locus': ['b1260', 'JW1252'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(1S,2R)-1-C-(indol-3-yl)glycerol 3-phosphate + L-serine = D-glyceraldehyde 3-phosphate + H2O + L-tryptophan',\n",
       "     '4.2.1.20')],\n",
       "   'Function': 'The alpha subunit is responsible for the aldol cleavage of indoleglycerol phosphate to indole and glyceraldehyde 3-phosphate',\n",
       "   'Pathway': 'Amino-acid biosynthesis; L-tryptophan biosynthesis; L-tryptophan from chorismate: step 5/5'},\n",
       "  'seq': 'MERYESLFAQLKERKEGAFVPFVTLGDPGIEQSLKIIDTLIEAGADALELGIPFSDPLADGPTIQNATLRAFAAGVTPAQCFEMLALIRQKHPTIPIGLLMYANLVFNKGIDEFYAQCEKVGVDSVLVADVPVEESAPFRQAALRHNVAPIFICPPNADDDLLRQIASYGRGYTYLLSRAGVTGAENRAALPLNHLVAKLKEYNAAPPLQGFGISAPDQVKAAIDAGAAGAISGSAIVKIIEQHINEPEKMLAALKVFVQPMKAATRS',\n",
       "  'mw': 28724.0},\n",
       " 'b1261': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A879',\n",
       "  'gene': 'b1261',\n",
       "  'locus': ['b1261', 'JW1253'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(1S,2R)-1-C-(indol-3-yl)glycerol 3-phosphate + L-serine = D-glyceraldehyde 3-phosphate + H2O + L-tryptophan',\n",
       "     '4.2.1.20')],\n",
       "   'Function': 'The beta subunit is responsible for the synthesis of L-tryptophan from indole and L-serine',\n",
       "   'Pathway': 'Amino-acid biosynthesis; L-tryptophan biosynthesis; L-tryptophan from chorismate: step 5/5'},\n",
       "  'seq': 'MTTLLNPYFGEFGGMYVPQILMPALRQLEEAFVSAQKDPEFQAQFNDLLKNYAGRPTALTKCQNITAGTNTTLYLKREDLLHGGAHKTNQVLGQALLAKRMGKTEIIAETGAGQHGVASALASALLGLKCRIYMGAKDVERQSPNVFRMRLMGAEVIPVHSGSATLKDACNEALRDWSGSYETAHYMLGTAAGPHPYPTIVREFQRMIGEETKAQILEREGRLPDAVIACVGGGSNAIGMFADFINETNVGLIGVEPGGHGIETGEHGAPLKHGRVGIYFGMKAPMMQTEDGQIEESYSISAGLDFPSVGPQHAYLNSTGRADYVSITDDEALEAFKTLCLHEGIIPALESSHALAHALKMMRENPDKEQLLVVNLSGRGDKDIFTVHDILKARGEI',\n",
       "  'mw': 42983.0},\n",
       " 'b1262': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P00909',\n",
       "  'gene': 'b1262',\n",
       "  'locus': ['b1262', 'JW1254'],\n",
       "  'synonyms': 'trpF',\n",
       "  'properties': {'Catalytic Activity': [('N-(5-phospho-beta-D-ribosyl)anthranilate = 1-(2-carboxyphenylamino)-1-deoxy-D-ribulose 5-phosphate',\n",
       "     '5.3.1.24'),\n",
       "    ('1-(2-carboxyphenylamino)-1-deoxy-D-ribulose 5-phosphate + H(+) = (1S,2R)-1-C-(indol-3-yl)glycerol 3-phosphate + CO2 + H2O',\n",
       "     '4.1.1.48')],\n",
       "   'Function': 'Bifunctional enzyme that catalyzes two sequential steps of tryptophan biosynthetic pathway. The first reaction is catalyzed by the isomerase, coded by the TrpF domain; the second reaction is catalyzed by the synthase, coded by the TrpC domain',\n",
       "   'Pathway': 'Amino-acid biosynthesis; L-tryptophan biosynthesis; L-tryptophan from chorismate: step 4/5'},\n",
       "  'seq': 'MMQTVLAKIVADKAIWVEARKQQQPLASFQNEVQPSTRHFYDALQGARTAFILECKKASPSKGVIRDDFDPARIAAIYKHYASAISVLTDEKYFQGSFNFLPIVSQIAPQPILCKDFIIDPYQIYLARYYQADACLLMLSVLDDDQYRQLAAVAHSLEMGVLTEVSNEEEQERAIALGAKVVGINNRDLRDLSIDLNRTRELAPKLGHNVTVISESGINTYAQVRELSHFANGFLIGSALMAHDDLHAAVRRVLLGENKVCGLTRGQDAKAAYDAGAIYGGLIFVATSPRCVNVEQAQEVMAAAPLQYVGVFRNHDIADVVDKAKVLSLAAVQLHGNEEQLYIDTLREALPAHVAIWKALSVGETLPAREFQHVDKYVLDNGQGGSGQRFDWSLLNGQSLGNVLLAGGLGADNCVEAAQTGCAGLDFNSAVESQPGIKDARLLASVFQTLRAY',\n",
       "  'mw': 49492.0},\n",
       " 'b1263': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P00904',\n",
       "  'gene': 'b1263',\n",
       "  'locus': ['b1263', 'JW1255'],\n",
       "  'synonyms': 'trpD',\n",
       "  'properties': {'Catalytic Activity': [('chorismate + L-glutamine = anthranilate + H(+) + L-glutamate + pyruvate',\n",
       "     '4.1.3.27'),\n",
       "    ('diphosphate + N-(5-phospho-beta-D-ribosyl)anthranilate = 5-phospho-alpha-D-ribose 1-diphosphate + anthranilate',\n",
       "     '2.4.2.18')],\n",
       "   'Function': 'Part of a heterotetrameric complex that catalyzes the two-step biosynthesis of anthranilate, an intermediate in the biosynthesis of L-tryptophan. In the first step, the glutamine-binding beta subunit (TrpG) of anthranilate synthase (AS) provides the glutamine amidotransferase activity which generates ammonia as a substrate that, along with chorismate, is used in the second step, catalyzed by the large alpha subunit of AS (TrpE) to produce anthranilate. In the absence of TrpG, TrpE can synthesize anthranilate directly from chorismate and high concentrations of ammonia. In addition to synthesizing anthranilate, it also catalyzes the second step of the pathway, the transfer of the phosphoribosyl group of 5-phosphorylribose-1-pyrophosphate (PRPP) to anthranilate',\n",
       "   'Regulation Activity': 'Cooperatively feedback inhibited by tryptophan',\n",
       "   'Pathway': 'Amino-acid biosynthesis; L-tryptophan biosynthesis; L-tryptophan from chorismate: step 2/5'},\n",
       "  'seq': 'MADILLLDNIDSFTYNLADQLRSNGHNVVIYRNHIPAQTLIERLATMSNPVLMLSPGPGVPSEAGCMPELLTRLRGKLPIIGICLGHQAIVEAYGGYVGQAGEILHGKASSIEHDGQAMFAGLTNPLPVARYHSLVGSNIPAGLTINAHFNGMVMAVRHDADRVCGFQFHPESILTTQGARLLEQTLAWAQQKLEPANTLQPILEKLYQAQTLSQQESHQLFSAVVRGELKPEQLAAALVSMKIRGEHPNEIAGAATALLENAAPFPRPDYLFADIVGTGGDGSNSINISTASAFVAAACGLKVAKHGNRSVSSKSGSSDLLAAFGINLDMNADKSRQALDELGVCFLFAPKYHTGFRHAMPVRQQLKTRTLFNVLGPLINPAHPPLALIGVYSPELVLPIAETLRVLGYQRAAVVHSGGMDEVSLHAPTIVAELHDGEIKSYQLTAEDFGLTPYHQEQLAGGTPEENRDILTRLLQGKGDAAHEAAVAANVAMLMRLHGHEDLQANAQTVLEVLRSGSAYDRVTALAARG',\n",
       "  'mw': 56870.0},\n",
       " 'b1264': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P00895',\n",
       "  'gene': 'b1264',\n",
       "  'locus': ['b1264', 'JW1256'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('chorismate + L-glutamine = anthranilate + H(+) + L-glutamate + pyruvate',\n",
       "     '4.1.3.27')],\n",
       "   'Function': 'Part of a heterotetrameric complex that catalyzes the two-step biosynthesis of anthranilate, an intermediate in the biosynthesis of L-tryptophan. In the first step, the glutamine-binding beta subunit (TrpG) of anthranilate synthase (AS) provides the glutamine amidotransferase activity which generates ammonia as a substrate that, along with chorismate, is used in the second step, catalyzed by the large alpha subunit of AS (TrpE) to produce anthranilate. In the absence of TrpG, TrpE can synthesize anthranilate directly from chorismate and high concentrations of ammonia',\n",
       "   'Regulation Activity': 'Cooperatively feedback inhibited by tryptophan',\n",
       "   'Pathway': 'Amino-acid biosynthesis; L-tryptophan biosynthesis; L-tryptophan from chorismate: step 1/5'},\n",
       "  'seq': 'MQTQKPTLELLTCEGAYRDNPTALFHQLCGDRPATLLLESADIDSKDDLKSLLLVDSALRITALGDTVTIQALSGNGEALLALLDNALPAGVESEQSPNCRVLRFPPVSPLLDEDARLCSLSVFDAFRLLQNLLNVPKEEREAMFFGGLFSYDLVAGFEDLPQLSAENNCPDFCFYLAETLMVIDHQKKSTRIQASLFAPNEEEKQRLTARLNELRQQLTEAAPPLPVVSVPHMRCECNQSDEEFGGVVRLLQKAIRAGEIFQVVPSRRFSLPCPSPLAAYYVLKKSNPSPYMFFMQDNDFTLFGASPESSLKYDATSRQIEIYPIAGTRPRGRRADGSLDRDLDSRIELEMRTDHKELSEHLMLVDLARNDLARICTPGSRYVADLTKVDRYSYVMHLVSRVVGELRHDLDALHAYRACMNMGTLSGAPKVRAMQLIAEAEGRRRGSYGGAVGYFTAHGDLDTCIVIRSALVENGIATVQAGAGVVLDSVPQSEADETRNKARAVLRAIATAHHAQETF',\n",
       "  'mw': 57494.0},\n",
       " 'b1270': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A9H5',\n",
       "  'gene': 'b1270',\n",
       "  'locus': ['b1270', 'JW1262'],\n",
       "  'synonyms': 'cobA',\n",
       "  'properties': {'Catalytic Activity': [('2 ATP + 2 cob(II)yrinate a,c diamide + reduced [electron-transfer flavoprotein] = 2 adenosylcob(III)yrinate a,c-diamide + 3 H(+) + oxidized [electron-transfer flavoprotein] + 2 triphosphate',\n",
       "     '2.5.1.17'),\n",
       "    ('2 ATP + 2 cob(II)alamin + reduced [electron-transfer flavoprotein] = 2 adenosylcob(III)alamin + 3 H(+) + oxidized [electron-transfer flavoprotein] + 2 triphosphate',\n",
       "     '2.5.1.17')],\n",
       "   'Function': 'Required for both de novo synthesis of the corrin ring for the assimilation of exogenous corrinoids. Participates in the adenosylation of a variety of incomplete and complete corrinoids (By similarity)',\n",
       "   'Pathway': 'Cofactor biosynthesis; adenosylcobalamin biosynthesis; adenosylcobalamin from cob(II)yrinate a,c-diamide: step 2/7'},\n",
       "  'seq': 'MSDERYQQRQQRVKEKVDARVAQAQDERGIIIVFTGNGKGKTTAAFGTATRAVGHGKKVGVVQFIKGTWPNGERNLLEPHGVEFQVMATGFTWDTQNRESDTAACREVWQHAKRMLADSSLDMVLLDELTYMVAYDYLPLEEVVQALNERPHQQTVIITGRGCHRDILELADTVSELRPVKHAFDAGVKAQIGIDY',\n",
       "  'mw': 21999.0},\n",
       " 'b1276': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P25516',\n",
       "  'gene': 'b1276',\n",
       "  'locus': ['b1276', 'JW1268'],\n",
       "  'synonyms': 'acn',\n",
       "  'properties': {'Catalytic Activity': [('citrate = D-threo-isocitrate',\n",
       "     '4.2.1.3')],\n",
       "   'Function': \"Catalyzes the reversible isomerization of citrate to isocitrate via cis-aconitate. The apo form of AcnA functions as a RNA-binding regulatory protein which plays a role as a maintenance or survival enzyme during nutritional or oxidative stress. During oxidative stress inactive AcnA apo-enzyme without iron sulfur clusters binds the acnA mRNA 3' UTRs (untranslated regions), stabilizes acnA mRNA and increases AcnA synthesis, thus mediating a post-transcriptional positive autoregulatory switch. AcnA also enhances the stability of the sodA transcript\",\n",
       "   'Pathway': 'Carbohydrate metabolism; tricarboxylic acid cycle; isocitrate from oxaloacetate: step 2/2'},\n",
       "  'seq': 'MSSTLREASKDTLQAKDKTYHYYSLPLAAKSLGDITRLPKSLKVLLENLLRWQDGNSVTEEDIHALAGWLKNAHADREIAYRPARVLMQDFTGVPAVVDLAAMREAVKRLGGDTAKVNPLSPVDLVIDHSVTVDRFGDDEAFEENVRLEMERNHERYVFLKWGKQAFSRFSVVPPGTGICHQVNLEYLGKAVWSELQDGEWIAYPDTLVGTDSHTTMINGLGVLGWGVGGIEAEAAMLGQPVSMLIPDVVGFKLTGKLREGITATDLVLTVTQMLRKHGVVGKFVEFYGDGLDSLPLADRATIANMSPEYGATCGFFPIDAVTLDYMRLSGRSEDQVELVEKYAKAQGMWRNPGDEPIFTSTLELDMNDVEASLAGPKRPQDRVALPDVPKAFAASNELEVNATHKDRQPVDYVMNGHQYQLPDGAVVIAAITSCTNTSNPSVLMAAGLLAKKAVTLGLKRQPWVKASLAPGSKVVSDYLAKAKLTPYLDELGFNLVGYGCTTCIGNSGPLPDPIETAIKKSDLTVGAVLSGNRNFEGRIHPLVKTNWLASPPLVVAYALAGNMNINLASEPIGHDRKGDPVYLKDIWPSAQEIARAVEQVSTEMFRKEYAEVFEGTAEWKGINVTRSDTYGWQEDSTYIRLSPFFDEMQATPAPVEDIHGARILAMLGDSVTTDHISPAGSIKPDSPAGRYLQGRGVERKDFNSYGSRRGNHEVMMRGTFANIRIRNEMVPGVEGGMTRHLPDSDVVSIYDAAMRYKQEQTPLAVIAGKEYGSGSSRDWAAKGPRLLGIRVVIAESFERIHRSNLIGMGILPLEFPQGVTRKTLGLTGEEKIDIGDLQNLQPGATVPVTLTRADGSQEVVPCRCRIDTATELTYYQNDGILHYVIRNMLK',\n",
       "  'mw': 97677.0},\n",
       " 'b1277': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A7I7',\n",
       "  'gene': 'b1277',\n",
       "  'locus': ['b1277', 'JW1269'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('GTP + 4 H2O = 2,5-diamino-6-hydroxy-4-(5-phosphoribosylamino)-pyrimidine + formate + 3 H(+) + 2 phosphate',\n",
       "     '3.5.4.25')],\n",
       "   'Function': \"Catalyzes the conversion of GTP to 2,5-diamino-6-ribosylamino-4(3H)-pyrimidinone 5'-phosphate (DARP), formate and pyrophosphate\",\n",
       "   'Regulation Activity': 'Competitively inhibited by pyrophosphate',\n",
       "   'Pathway': 'Cofactor biosynthesis; riboflavin biosynthesis; 5-amino-6-(D-ribitylamino)uracil from GTP: step 1/4'},\n",
       "  'seq': 'MQLKRVAEAKLPTPWGDFLMVGFEELATGHDHVALVYGDISGHTPVLARVHSECLTGDALFSLRCDCGFQLEAALTQIAEEGRGILLYHRQEGRNIGLLNKIRAYALQDQGYDTVEANHQLGFAADERDFTLCADMFKLLGVNEVRLLTNNPKKVEILTEAGINIVERVPLIVGRNPNNEHYLDTKAEKMGHLLNK',\n",
       "  'mw': 21836.0},\n",
       " 'b1278': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A924',\n",
       "  'gene': 'b1278',\n",
       "  'locus': ['b1278', 'JW1270'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [(\"1,2-diacyl-sn-glycero-3-phospho-(1'-sn-glycero-3'-phosphate) + H2O = 1,2-diacyl-sn-glycero-3-phospho-(1'-sn-glycerol) + phosphate\",\n",
       "     '3.1.3.27'),\n",
       "    ('a 1,2-diacyl-sn-glycerol 3-diphosphate + H2O = a 1,2-diacyl-sn-glycero-3-phosphate + H(+) + phosphate',\n",
       "     '3.6.1.75'),\n",
       "    ('a 1,2-diacyl-sn-glycero-3-phosphate + H2O = a 1,2-diacyl-sn-glycerol + phosphate',\n",
       "     '3.1.3.4'),\n",
       "    ('di-trans,octa-cis-undecaprenyl diphosphate + H2O = di-trans,octa-cis-undecaprenyl phosphate + H(+) + phosphate',\n",
       "     '3.6.1.27')],\n",
       "   'Function': 'Catalyzes the dephosphorylation of diacylglycerol diphosphate (DGPP) to phosphatidate (PA) and the subsequent dephosphorylation of PA to diacylglycerol (DAG). Also has undecaprenyl pyrophosphate phosphatase activity, required for the biosynthesis of the lipid carrier undecaprenyl phosphate. Can also use lysophosphatidic acid (LPA) and phosphatidylglycerophosphate as substrates. The pattern of activities varies according to subcellular location, PGP phosphatase activity is higher in the cytoplasmic membrane, whereas PA and LPA phosphatase activities are higher in the outer membrane. Activity is independent of a divalent cation ion and insensitive to inhibition by N-ethylmaleimide',\n",
       "   'Regulation Activity': 'Inhibited by Mn(2+) ions. Inhibited by phosphatidylethanolamine (PE) (PubMed:27405756)',\n",
       "   'Pathway': 'Phospholipid metabolism; phosphatidylglycerol biosynthesis; phosphatidylglycerol from CDP-diacylglycerol: step 2/2'},\n",
       "  'seq': 'MRSIARRTAVGAALLLVMPVAVWISGWRWQPGEQSWLLKAAFWVTETVTQPWGVITHLILFGWFLWCLRFRIKAAFVLFAILAAAILVGQGVKSWIKDKVQEPRPFVIWLEKTHHIPVDEFYTLKRAERGNLVKEQLAEEKNIPQYLRSHWQKETGFAFPSGHTMFAASWALLAVGLLWPRRRTLTIAILLVWATGVMGSRLLLGMHWPRDLVVATLISWALVAVATWLAQRICGPLTPPAEENREIAQREQES',\n",
       "  'mw': 29021.0},\n",
       " 'b1281': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P08244',\n",
       "  'gene': 'b1281',\n",
       "  'locus': ['b1281', 'JW1273'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [(\"H(+) + orotidine 5'-phosphate = CO2 + UMP\",\n",
       "     '4.1.1.23')],\n",
       "   'Function': \"Catalyzes the decarboxylation of orotidine 5'-monophosphate (OMP) to uridine 5'-monophosphate (UMP)\",\n",
       "   'Pathway': 'Pyrimidine metabolism; UMP biosynthesis via de novo pathway; UMP from orotate: step 2/2'},\n",
       "  'seq': 'MTLTASSSSRAVTNSPVVVALDYHNRDDALAFVDKIDPRDCRLKVGKEMFTLFGPQFVRELQQRGFDIFLDLKFHDIPNTAAHAVAAAADLGVWMVNVHASGGARMMTAAREALVPFGKDAPLLIAVTVLTSMEASDLVDLGMTLSPADYAERLAALTQKCGLDGVVCSAQEAVRFKQVFGQEFKLVTPGIRPQGSEAGDQRRIMTPEQALSAGVDYMVIGRPVTQSVDPAQTLKAINASLQRSA',\n",
       "  'mw': 26350.0},\n",
       " 'b1288': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AEK4',\n",
       "  'gene': 'b1288',\n",
       "  'locus': ['b1288', 'JW1281'],\n",
       "  'synonyms': 'envM',\n",
       "  'properties': {'Catalytic Activity': [('a 2,3-saturated acyl-[ACP] + NAD(+) = a (2E)-enoyl-[ACP] + H(+) + NADH',\n",
       "     '1.3.1.9'),\n",
       "    ('(2E)-butenoyl-[ACP] + H(+) + NADH = butanoyl-[ACP] + NAD(+)', ''),\n",
       "    ('(2E)-decenoyl-[ACP] + H(+) + NADH = decanoyl-[ACP] + NAD(+)', ''),\n",
       "    ('(2E)-hexadecenoyl-[ACP] + H(+) + NADH = hexadecanoyl-[ACP] + NAD(+)',\n",
       "     ''),\n",
       "    ('(2E,9Z)-hexadecadienoyl-[ACP] + H(+) + NADH = (9Z)-hexadecenoyl-[ACP] + NAD(+)',\n",
       "     ''),\n",
       "    ('(2E)-5-methylhexenoyl-[ACP] + H(+) + NADH = 5-methylhexanoyl-[ACP] + NAD(+)',\n",
       "     '')],\n",
       "   'Function': 'Catalyzes the reduction of a carbon-carbon double bond in an enoyl moiety that is covalently linked to an acyl carrier protein (ACP). Involved in the elongation cycle of fatty acid which are used in the lipid metabolism and in the biotin biosynthesis',\n",
       "   'Regulation Activity': 'Inhibited by diazaborines, triclosan (5-chloro-2-2,4-dichlorophenoxyphenol), 1,4-disubstituted imidazoles, 1,4-benzodiazepine derivatives, naphthyridinone derivatives, luteolin and curcumin (PubMed:10398587, PubMed:10493822, PubMed:11514139, PubMed:12109908, PubMed:12699381, PubMed:19959361, PubMed:8119879, PubMed:8953047, PubMed:9707111). The antibiotic diazaborine interferes with the activity by binding to the protein and NAD (PubMed:8119879)',\n",
       "   'Pathway': 'Cofactor biosynthesis; biotin biosynthesis'},\n",
       "  'seq': 'MGFLSGKRILVTGVASKLSIAYGIAQAMHREGAELAFTYQNDKLKGRVEEFAAQLGSDIVLQCDVAEDASIDTMFAELGKVWPKFDGFVHSIGFAPGDQLDGDYVNAVTREGFKIAHDISSYSFVAMAKACRSMLNPGSALLTLSYLGAERAIPNYNVMGLAKASLEANVRYMANAMGPEGVRVNAISAGPIRTLAASGIKDFRKMLAHCEAVTPIRRTVTIEDVGNSAAFLCSDLSAGISGEVVHVDGGFSIAAMNELELK',\n",
       "  'mw': 27864.0},\n",
       " 'b1291': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AAH4',\n",
       "  'gene': 'b1291',\n",
       "  'locus': ['b1291', 'JW1284'],\n",
       "  'synonyms': 'trkE',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of a putrescine export transport system, does not play a role in resistance to antimicrobial peptides (PubMed:27803167). Stimulates K(+)-uptake proteins TrkG and TrkH to import K(+), may act via ATP-binding rather than ATP hydrolysis (PubMed:11700350)'},\n",
       "  'seq': 'MPLLDIRNLTIEFKTGDEWVKAVDRVSMTLTEGEIRGLVGESGSGKSLIAKAICGVNKDNWRVTADRMRFDDIDLLRLSARERRKLVGHNVSMIFQEPQSCLDPSERVGRQLMQNIPAWTYKGRWWQRFGWRKRRAIELLHRVGIKDHKDAMRSFPYELTEGECQKVMIAIALANQPRLLIADEPTNSMEPTTQAQIFRLLTRLNQNSNTTILLISHDLQMLSQWADKINVLYCGQTVETAPSKELVTMPHHPYTQALIRAIPDFGSAMPHKSRLNTLPGAIPLLEQLPIGCRLGPRCPYAQRECIVTPRLTGAKNHLYACHFPLNMEKE',\n",
       "  'mw': 37661.0},\n",
       " 'b1296': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P76037',\n",
       "  'gene': 'b1296',\n",
       "  'locus': ['b1296', 'JW1289'],\n",
       "  'synonyms': 'ycjJ',\n",
       "  'properties': {'Catalytic Activity': [('H(+)(in) + putrescine(in) = H(+)(out) + putrescine(out)',\n",
       "     '')],\n",
       "   'Function': 'Involved in the uptake of putrescine (PubMed:15590624, PubMed:19181795, PubMed:27803167, PubMed:23719730). Imports putrescine for its utilization as an energy source in the absence of glucose (PubMed:23719730)',\n",
       "   'Regulation Activity': 'Putrescine uptake is inhibited by carbonyl cyanide m-chlorophenylhydrazone (CCCP), which dissipates the proton motive force',\n",
       "   'Pathway': 'Amine and polyamine degradation; putrescine degradation'},\n",
       "  'seq': 'MAINSPLNIAAQPGKTRLRKSLKLWQVVMMGLAYLTPMTVFDTFGIVSGISDGHVPASYLLALAGVLFTAISYGKLVRQFPEAGSAYTYAQKSINPHVGFMVGWSSLLDYLFLPMINVLLAKIYLSALFPEVPPWVWVVTFVAILTAANLKSVNLVANFNTLFVLVQISIMVVFIFLVVQGLHKGEGVGTVWSLQPFISENAHLIPIITGATIVCFSFLGFDAVTTLSEETPDAARVIPKAIFLTAVYGGVIFIAASFFMQLFFPDISRFKDPDAALPEIALYVGGKLFQSIFLCTTFVNTLASGLASHASVSRLLYVMGRDNVFPERVFGYVHPKWRTPALNVIMVGIVALSALFFDLVTATALINFGALVAFTFVNLSVFNHFWRRKGMNKSWKDHFHYLLMPLVGALTVGVLWVNLESTSLTLGLVWASLGGAYLWYLIRRYRKVPLYDGDRTPVSET',\n",
       "  'mw': 50853.0},\n",
       " 'b1297': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P78061',\n",
       "  'gene': 'b1297',\n",
       "  'locus': ['b1297', 'JW5201'],\n",
       "  'synonyms': 'ycjK',\n",
       "  'properties': {'Catalytic Activity': [('ATP + L-glutamate + putrescine = ADP + gamma-L-glutamylputrescine + H(+) + phosphate',\n",
       "     '6.3.1.11')],\n",
       "   'Function': 'Involved in the breakdown of putrescine. Catalyzes the ATP-dependent gamma-glutamylation of putrescine, producing gamma-L-glutamylputrescine. Absolutely essential to utilize putrescine as both nitrogen and carbon sources and to decrease the toxicity of putrescine, which can lead to inhibition of cell growth and protein synthesis. In vitro is also able to use several diamines, and spermidine and spermine, instead of putrescine, but with a much lower activity, and cannot catalyze the gamma-glutamylation of ornithine or GABA (PubMed:18495664)',\n",
       "   'Pathway': 'Amine and polyamine degradation; putrescine degradation; 4-aminobutanoate from putrescine: step 1/4'},\n",
       "  'seq': 'METNIVEVENFVQQSEERRGSAFTQEVKRYLERYPNTQYVDVLLTDLNGCFRGKRIPVSSLKKLEKGCYFPASVFAMDILGNVVEEAGLGQEMGEPDRTCVPVLGSLTPSAADPEFIGQMLLTMVDEDGAPFDVEPRNVLNRLWQQLRQRGLFPVVAVELEFYLLDRQRDAEGYLQPPCAPGTDDRNTQSQVYSVDNLNHFADVLNDIDELAQLQLIPADGAVAEASPGQFEINLYHTDNVLEACDDALALKRLVRLMAEKHKMHATFMAKPYEEHAGSGMHIHISMQNNRGENVLSDAEGEDSPLLKKMLAGMIDLMPSSMALLAPNVNSYRRFQPGMYVPTQASWGHNNRTVALRIPCGDRHNHRVEYRVAGADANPYLVMAAIFAGILHGLDNELPLQEEVEGNGLEQEGLPFPIRQSDALGEFIENDHLRRYLGERFCHVYHACKNDELLQFERLITETEIEWMLKNA',\n",
       "  'mw': 53177.0},\n",
       " 'b1298': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P76038',\n",
       "  'gene': 'b1298',\n",
       "  'locus': ['b1298', 'JW1291'],\n",
       "  'synonyms': 'ycjL',\n",
       "  'properties': {'Catalytic Activity': [('4-(gamma-L-glutamylamino)butanoate + H2O = 4-aminobutanoate + L-glutamate',\n",
       "     '3.5.1.94')],\n",
       "   'Function': 'Involved in the breakdown of putrescine via hydrolysis of the gamma-glutamyl linkage of gamma-glutamyl-gamma-aminobutyrate',\n",
       "   'Pathway': 'Amine and polyamine degradation; putrescine degradation; 4-aminobutanoate from putrescine: step 4/4'},\n",
       "  'seq': 'MENIMNNPVIGVVMCRNRLKGHATQTLQEKYLNAIIHAGGLPIALPHALAEPSLLEQLLPKLDGIYLPGSPSNVQPHLYGENGDEPDADPGRDLLSMAIINAALERRIPIFAICRGLQELVVATGGSLHRKLCEQPELLEHREDPELPVEQQYAPSHEVQVEEGGLLSALLPECSNFWVNSLHGQGAKVVSPRLRVEARSPDGLVEAVSVINHPFALGVQWHPEWNSSEYALSRILFEGFITACQHHIAEKQRL',\n",
       "  'mw': 28013.0},\n",
       " 'b1300': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P23883',\n",
       "  'gene': 'b1300',\n",
       "  'locus': ['b1300', 'JW1293'],\n",
       "  'synonyms': 'aldH',\n",
       "  'properties': {'Catalytic Activity': [('an aldehyde + H2O + NADP(+) = a carboxylate + 2 H(+) + NADPH',\n",
       "     '1.2.1.5'),\n",
       "    ('an aldehyde + H2O + NAD(+) = a carboxylate + 2 H(+) + NADH', '1.2.1.5'),\n",
       "    ('4-(gamma-L-glutamylamino)butanal + H2O + NADP(+) = 4-(gamma-L-glutamylamino)butanoate + 2 H(+) + NADPH',\n",
       "     ''),\n",
       "    ('4-(gamma-L-glutamylamino)butanal + H2O + NAD(+) = 4-(gamma-L-glutamylamino)butanoate + 2 H(+) + NADH',\n",
       "     '')],\n",
       "   'Function': 'Catalyzes the oxidation of 3-hydroxypropionaldehyde (3-HPA) to 3-hydroxypropionic acid (3-HP) (PubMed:18668238). It acts preferentially with NAD but can also use NADP (PubMed:18668238). 3-HPA appears to be the most suitable substrate for PuuC followed by isovaleraldehyde, propionaldehyde, butyraldehyde, and valeraldehyde (PubMed:18668238). It might play a role in propionate and/or acetic acid metabolisms (PubMed:18668238). Also involved in the breakdown of putrescine through the oxidation of gamma-Glu-gamma-aminobutyraldehyde to gamma-Glu-gamma-aminobutyrate (gamma-Glu-GABA) (PubMed:15590624)',\n",
       "   'Regulation Activity': 'Lithium ions exhibits the highest inhibition (97%). To a lesser extent (5-20%), potassium, sodium, and ammonium ions also inhibit PuuC activity. Transition metals, such as copper and zinc ions inhibit PuuC activity by more than 90%. The presence of heavy metals (mercury, silver) or sodium hydrogensulfite in the reaction mixture completely inactivate PuuC; in contrast, disulfide reductants such as DTT and 2-mercaptoethanol significantly increase its activity by 75% and 27%, respectively',\n",
       "   'Pathway': 'Amine and polyamine degradation; putrescine degradation; 4-aminobutanoate from putrescine: step 3/4'},\n",
       "  'seq': 'MNFHHLAYWQDKALSLAIENRLFINGEYTAAAENETFETVDPVTQAPLAKIARGKSVDIDRAMSAARGVFERGDWSLSSPAKRKAVLNKLADLMEAHAEELALLETLDTGKPIRHSLRDDIPGAARAIRWYAEAIDKVYGEVATTSSHELAMIVREPVGVIAAIVPWNFPLLLTCWKLGPALAAGNSVILKPSEKSPLSAIRLAGLAKEAGLPDGVLNVVTGFGHEAGQALSRHNDIDAIAFTGSTRTGKQLLKDAGDSNMKRVWLEAGGKSANIVFADCPDLQQAASATAAGIFYNQGQVCIAGTRLLLEESIADEFLALLKQQAQNWQPGHPLDPATTMGTLIDCAHADSVHSFIREGESKGQLLLDGRNAGLAAAIGPTIFVDVDPNASLSREEIFGPVLVVTRFTSEEQALQLANDSQYGLGAAVWTRDLSRAHRMSRRLKAGSVFVNNYNDGDMTVPFGGYKQSGNGRDKSLHALEKFTELKTIWISLEA',\n",
       "  'mw': 53419.0},\n",
       " 'b1301': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37906',\n",
       "  'gene': 'b1301',\n",
       "  'locus': ['b1301', 'JW1294'],\n",
       "  'synonyms': 'ordL ycjA',\n",
       "  'properties': {'Catalytic Activity': [('gamma-L-glutamylputrescine + H2O + O2 = 4-(gamma-L-glutamylamino)butanal + H2O2 + NH4(+)',\n",
       "     '')],\n",
       "   'Function': 'Involved in the breakdown of putrescine via the oxidation of L-glutamylputrescine',\n",
       "   'Pathway': 'Amine and polyamine degradation; putrescine degradation; 4-aminobutanoate from putrescine: step 2/4'},\n",
       "  'seq': 'MTEHTSSYYAASANKYAPFDTLNESITCDVCVVGGGYTGLSSALHLAEAGFDVVVLEASRIGFGASGRNGGQLVNSYSRDIDVIEKSYGMDTARMLGSMMFEGGEIIRERIKRYQIDCDYRPGGLFVAMNDKQLATLEEQKENWERYGNKQLELLDANAIRREVASDRYTGALLDHSGGHIHPLNLAIGEADAIRLNGGRVYELSAVTQIQHTTPAVVRTAKGQVTAKYVIVAGNAYLGDKVEPELAKRSMPCGTQVITTERLSEDLARSLIPKNYCVEDCNYLLDYYRLTADNRLLYGGGVVYGARDPDDVERLVVPKLLKTFPQLKGVKIDYRWTGNFLLTLSRMPQFGRLDTNIYYMQGYSGHGVTCTHLAGRLIAELLRGDAERFDAFANLPHYPFPGGRTLRVPFTAMGAAYYSLRDRLGV',\n",
       "  'mw': 47170.0},\n",
       " 'b1302': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P50457',\n",
       "  'gene': 'b1302',\n",
       "  'locus': ['b1302', 'JW1295'],\n",
       "  'synonyms': 'goaG',\n",
       "  'properties': {'Catalytic Activity': [('2-oxoglutarate + 4-aminobutanoate = L-glutamate + succinate semialdehyde',\n",
       "     '2.6.1.19')],\n",
       "   'Function': 'Catalyzes the transfer of the amino group from gamma-aminobutyrate (GABA) to alpha-ketoglutarate (KG) to yield succinic semialdehyde (SSA). PuuE is important for utilization of putrescine as the sole nitrogen or carbon source',\n",
       "   'Regulation Activity': 'Completely inhibited by succinate and low-aeration conditions',\n",
       "   'Pathway': 'Amine and polyamine degradation; putrescine degradation; succinate semialdehyde from 4-aminobutanoate'},\n",
       "  'seq': 'MSNNEFHQRRLSATPRGVGVMCNFFAQSAENATLKDVEGNEYIDFAAGIAVLNTGHRHPDLVAAVEQQLQQFTHTAYQIVPYESYVTLAEKINALAPVSGQAKTAFFTTGAEAVENAVKIARAHTGRPGVIAFSGGFHGRTYMTMALTGKVAPYKIGFGPFPGSVYHVPYPSDLHGISTQDSLDAIERLFKSDIEAKQVAAIIFEPVQGEGGFNVAPKELVAAIRRLCDEHGIVMIADEVQSGFARTGKLFAMDHYADKPDLMTMAKSLAGGMPLSGVVGNANIMDAPAPGGLGGTYAGNPLAVAAAHAVLNIIDKESLCERANQLGQRLKNTLIDAKESVPAIAAVRGLGSMIAVEFNDPQTGEPSAAIAQKIQQRALAQGLLLLTCGAYGNVIRFLYPLTIPDAQFDAAMKILQDALSD',\n",
       "  'mw': 44729.0},\n",
       " 'b1308': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P23857',\n",
       "  'gene': 'b1308',\n",
       "  'locus': ['b1308', 'JW1301'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('hydrogen cyanide + thiosulfate = 2 H(+) + sulfite + thiocyanate',\n",
       "     '2.8.1.1')],\n",
       "   'Function': 'The phage shock protein (psp) operon (pspABCDE) may play a significant role in the competition for survival under nutrient- or energy-limited conditions. PspE catalyzes the sulfur-transfer reaction from thiosulfate to cyanide, to form sulfite and thiocyanate. Also able to use dithiol (dithiothreitol) as an alternate sulfur acceptor. Also possesses a very low mercaptopyruvate sulfurtransferase activity',\n",
       "   'Regulation Activity': 'Inhibited by thiosulfate above 100 mM, particularly at low cyanide concentrations (<5 mM). Inhibited by sodium sulfate or sodium chloride at 0.25 M which gives around 50% inhibition of rhodanese activity. Addition of sodium phosphate at the same concentration results in about 65% inhibition. Sulfite strongly inhibits PspE activity (1 mM sodium sulfite resulted in more than 50% inhibition of rhodanese activity)'},\n",
       "  'seq': 'MFKKGLLALALVFSLPVFAAEHWIDVRVPEQYQQEHVQGAINIPLKEVKERIATAVPDKNDTVKVYCNAGRQSGQAKEILSEMGYTHVENAGGLKDIAMPKVKG',\n",
       "  'mw': 11475.0},\n",
       " 'b1319': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P76045',\n",
       "  'gene': 'b1319',\n",
       "  'locus': ['b1319', 'JW1312'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'May act as a regulator of the RCS-phosphorelay signal transduction pathway'},\n",
       "  'seq': 'MKKLLPCTALVMCAGMACAQAEERNDWHFNIGAMYEIENVEGYGEDMDGLAEPSVYFNAANGPWRIALAYYQEGPVDYSAGKRGTWFDRPELEVHYQFLENDDFSFGLTGGFRNYGYHYVDEPGKDTANMQRWKIAPDWDVKLTDDLRFNGWLSMYKFANDLNTTGYADTRVETETGLQYTFNETVALRVNYYLERGFNMDDSRNNGEFSTQEIRAYLPLTLGNHSVTPYTRIGLDRWSNWDWQDDIEREGHDFNRVGLFYGYDFQNGLSVSLEYAFEWQDHDEGDSDKFHYAGVGVNYSF',\n",
       "  'mw': 34913.0},\n",
       " 'b1325': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P51981',\n",
       "  'gene': 'b1325',\n",
       "  'locus': ['b1325', 'JW1318'],\n",
       "  'synonyms': 'ycjH',\n",
       "  'properties': {'Catalytic Activity': [('L-alanyl-L-glutamate = L-alanyl-D-glutamate',\n",
       "     '5.1.1.20')],\n",
       "   'Function': 'Catalyzes the epimerization of L-Ala-D-Glu to L-Ala-L-Glu and has a role in the recycling of the murein peptide, of which L-Ala-D-Glu is a component. Is also able to catalyze the reverse reaction and the epimerization of all the L-Ala-X dipeptides, except L-Ala-L-Arg, L-Ala-L-Lys and L-Ala-L-Pro. Is also active with L-Gly-L-Glu, L-Phe-L-Glu, and L-Ser-L-Glu, but not with L-Glu-L-Glu, L-Lys-L-Glu, L-Pro-L-Glu, L-Lys-L-Ala, or D-Ala-D-Ala',\n",
       "   'Pathway': 'Cell wall biogenesis; peptidoglycan recycling'},\n",
       "  'seq': 'MRTVKVFEEAWPLHTPFVIARGSRSEARVVVVELEEEGIKGTGECTPYPRYGESDASVMAQIMSVVPQLEKGLTREELQKILPAGAARNALDCALWDLAARRQQQSLADLIGITLPETVITAQTVVIGTPDQMANSASTLWQAGAKLLKVKLDNHLISERMVAIRTAVPDATLIVDANESWRAEGLAARCQLLADLGVAMLEQPLPAQDDAALENFIHPLPICADESCHTRSNLKALKGRYEMVNIKLDKTGGLTEALALATEARAQGFSLMLGCMLCTSRAISAALPLVPQVSFADLDGPTWLAVDVEPALQFTTGELHL',\n",
       "  'mw': 34674.0},\n",
       " 'b1326': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ACV6',\n",
       "  'gene': 'b1326',\n",
       "  'locus': ['b1326', 'JW1319'],\n",
       "  'synonyms': 'ycjI',\n",
       "  'properties': {'Catalytic Activity': [('H2O + L-alanyl-gamma-D-glutamyl-meso-diaminoheptanedioate = L-alanyl-D-glutamate + meso-2,6-diaminoheptanedioate',\n",
       "     '')],\n",
       "   'Function': 'Involved in muropeptide degradation. Catalyzes the hydrolysis of the gamma-D-glutamyl-diaminopimelic acid (gamma-D-Glu-Dap) amide bond in the murein tripeptide L-alanyl-gamma-D-glutamyl-meso-diaminopimelic acid, leading to the formation of L-Ala-gamma-D-Glu and Dap (PubMed:12511517, PubMed:22970852). Has weak activity with L-Ala-gamma-D-Glu-L-Lys, MurNAc-tripeptide and gamma-D-Glu-meso-Dap (PubMed:22970852). Cannot hydrolyze murein tetrapeptide (PubMed:22970852)',\n",
       "   'Regulation Activity': 'Inhibited by EDTA',\n",
       "   'Pathway': 'Cell wall degradation; peptidoglycan degradation'},\n",
       "  'seq': 'MTVTRPRAERGAFPPGTEHYGRSLLGAPLIWFPAPAASRESGLILAGTHGDENSSVVTLSCALRTLTPSLRRHHVVLCVNPDGCQLGLRANANGVDLNRNFPAANWKEGETVYRWNSAAEERDVVLLTGDKPGSEPETQALCQLIHRIQPAWVVSFHDPLACIEDPRHSELGEWLAQAFELPLVTSVGYETPGSFGSWCADLNLHCITAEFPPISSDEASEKYLFAMANLLRWHPKDAIRPS',\n",
       "  'mw': 26558.0},\n",
       " 'b1329': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77348',\n",
       "  'gene': 'b1329',\n",
       "  'locus': ['b1329', 'JW1322'],\n",
       "  'synonyms': 'ynaH',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Essential for the uptake of the murein peptide L-alanyl-gamma-D-glutamyl-meso-diaminopimelate. Also transports some alpha-linked peptides such as Pro-Phe-Lys with low affinity. The transport is effected by the oligopeptide permease system'},\n",
       "  'seq': 'MKHSVSVTCCALLVSSISLSYAAEVPSGTVLAEKQELVRHIKDEPASLDPAKAVGLPEIQVIRDLFEGLVNQNEKGEIVPGVATQWKSNDNRIWTFTLRDNAKWADGTPVTAQDFVYSWQRLVDPKTLSPFAWFAALAGINNAQAIIDGKATPDQLGVTAVDAHTLKIQLDKPLPWFVNLTANFAFFPVQKANVESGKEWTKPGNLIGNGAYVLKERVVNEKLVVVPNTHYWDNAKTVLQKVTFLPINQESAATKRYLAGDIDITESFPKNMYQKLLKDIPGQVYTPPQLGTYYYAFNTQKGPTADQRVRLALSMTIDRRLMTEKVLGTGEKPAWHFTPDVTAGFTPEPSPFEQMSQEELNAQAKTLLSAAGYGPQKPLKLTLLYNTSENHQKIAIAVASMWKKNLGVDVKLQNQEWKTYIDSRNTGNFDVIRASWVGDYNEPSTFLTLLTSTHSGNISRFNNPAYDKVLAQASTENTVKARNADYNAAEKILMEQAPIAPIYQYTNGRLIKPWLKGYPINNPEDVAYSRTMYIVKH',\n",
       "  'mw': 59900.0},\n",
       " 'b1363': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P23849',\n",
       "  'gene': 'b1363',\n",
       "  'locus': ['b1363', 'JW1358'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Low-affinity potassium transport system. Interacts with Trk system potassium uptake protein TrkA. Requires TrkE (sapD) for maximal transport activity, low activity is seen in its absence; no further stimulation is seen with SapF (PubMed:11700350). Transport in the absence of SapD is dependent on a high membrane potential and a high cytoplasmic ATP concentration, suggesting this protein may be able to interact with other ATP-binding proteins (PubMed:11700350). Can transport potassium and rubidium (PubMed:7896723)'},\n",
       "  'seq': 'MNTSHVRVVTHMCGFLVWLYSLSMLPPMVVALFYKEKSLFVFFITFVIFFCIGGGAWYTTKKSGIQLRTRDGFIIIVMFWILFSVISAFPLWIDSELNLTFIDALFEGVSGITTTGATVIDDVSSLPRAYLYYRSQLNFIGGLGVIVLAVAVLPLLGIGGAKLYQSEMPGPFKDDKLTPRLADTSRTLWITYSLLGIACIVCYRLAGMPLFDAICHGISTVSLGGFSTHSESIGYFNNYLVELVAGSFSLLSAFNFTLWYIVISRKTIKPLIRDIELRFFLLIALGVIIVTSFQVWHIGMYDLHGSFIHSFFLASSMLTDNGLATQDYASWPTHTIVFLLLSSFFGGCIGSTCGGIKSLRFLILFKQSKHEINQLSHPRALLSVNVGGKIVTDRVMRSVWSFFFLYTLFTVFFILVLNGMGYDFLTSFATVAACINNMGLGFGATASSFGVLNDIAKCLMCIAMILGRLEIYPVIILFSGFFWRS',\n",
       "  'mw': 53944.0},\n",
       " 'b1377': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77747',\n",
       "  'gene': 'b1377',\n",
       "  'locus': ['b1377', 'JW1371'],\n",
       "  'synonyms': 'ynaG',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Forms pores that allow passive diffusion of small molecules across the outer membrane (By similarity). Non-specific porin'},\n",
       "  'seq': 'MKSKVLALLIPALLAAGAAHAAEVYNKDGNKLDLYGKVDGLHYFSDNSAKDGDQSYARLGFKGETQINDQLTGYGQWEYNIQANNTESSKNQSWTRLAFAGLKFADYGSFDYGRNYGVMYDIEGWTDMLPEFGGDSYTNADNFMTGRANGVATYRNTDFFGLVNGLNFAVQYQGNNEGASNGQEGTNNGRDVRHENGDGWGLSTTYDLGMGFSAGAAYTSSDRTNDQVNHTAAGGDKADAWTAGLKYDANNIYLATMYSETRNMTPFGDSDYAVANKTQNFEVTAQYQFDFGLRPAVSFLMSKGRDLHAAGGADNPAGVDDKDLVKYADIGATYYFNKNMSTYVDYKINLLDEDDSFYAANGISTDDIVALGLVYQF',\n",
       "  'mw': 41220.0},\n",
       " 'b1380': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P52643',\n",
       "  'gene': 'b1380',\n",
       "  'locus': ['b1380', 'JW1375'],\n",
       "  'synonyms': 'hslI htpH',\n",
       "  'properties': {'Catalytic Activity': [('(R)-lactate + NAD(+) = H(+) + NADH + pyruvate',\n",
       "     '1.1.1.28')],\n",
       "   'Function': 'Fermentative lactate dehydrogenase'},\n",
       "  'seq': 'MKLAVYSTKQYDKKYLQQVNESFGFELEFFDFLLTEKTAKTANGCEAVCIFVNDDGSRPVLEELKKHGVKYIALRCAGFNNVDLDAAKELGLKVVRVPAYDPEAVAEHAIGMMMTLNRRIHRAYQRTRDANFSLEGLTGFTMYGKTAGVIGTGKIGVAMLRILKGFGMRLLAFDPYPSAAALELGVEYVDLPTLFSESDVISLHCPLTPENYHLLNEAAFEQMKNGVMIVNTSRGALIDSQAAIEALKNQKIGSLGMDVYENERDLFFEDKSNDVIQDDVFRRLSACHNVLFTGHQAFLTAEALTSISQTTLQNLSNLEKGETCPNELV',\n",
       "  'mw': 36535.0},\n",
       " 'b1385': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P80668',\n",
       "  'gene': 'b1385',\n",
       "  'locus': ['b1385', 'JW1380'],\n",
       "  'synonyms': 'maoB padA ydbG',\n",
       "  'properties': {'Catalytic Activity': [('2-phenylacetaldehyde + H2O + NAD(+) = 2-phenylacetate + 2 H(+) + NADH',\n",
       "     '1.2.1.39')],\n",
       "   'Function': 'Acts almost equally well on phenylacetaldehyde, 4-hydroxyphenylacetaldehyde and 3,4-dihydroxyphenylacetaldehyde',\n",
       "   'Pathway': 'Amino-acid degradation; L-phenylalanine degradation; phenylacetate from L-phenylalanine: step 3/3'},\n",
       "  'seq': 'MTEPHVAVLSQVQQFLDRQHGLYIDGRPGPAQSEKRLAIFDPATGQEIASTADANEADVDNAVMSAWRAFVSRRWAGRLPAERERILLRFADLVEQHSEELAQLETLEQGKSIAISRAFEVGCTLNWMRYTAGLTTKIAGKTLDLSIPLPQGARYQAWTRKEPVGVVAGIVPWNFPLMIGMWKVMPALAAGCSIVIKPSETTPLTMLRVAELASEAGIPDGVFNVVTGSGAVCGAALTSHPHVAKISFTGSTATGKGIARTAADHLTRVTLELGGKNPAIVLKDADPQWVIEGLMTGSFLNQGQVCAASSRIYIEAPLFDTLVSGFEQAVKSLQVGPGMSPVAQINPLVSRAHCDKVCSFLDDAQAQQAELIRGSNGPAGEGYYVAPTLVVNPDAKLRLTREEVFGPVVNLVRVADGEEALQLANDTEYGLTASVWTQNLSQALEYSDRLQAGTVWVNSHTLIDANLPFGGMKQSGTGRDFGPDWLDGWCETKSVCVRY',\n",
       "  'mw': 53699.0},\n",
       " 'b1386': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P46883',\n",
       "  'gene': 'b1386',\n",
       "  'locus': ['b1386', 'JW1381'],\n",
       "  'synonyms': 'maoA',\n",
       "  'properties': {'Catalytic Activity': [('an aliphatic amine + H2O + O2 = an aldehyde + H2O2 + NH4(+)',\n",
       "     '1.4.3.21'),\n",
       "    ('2-phenylethylamine + H2O + O2 = 2-phenylacetaldehyde + H2O2 + NH4(+)',\n",
       "     '1.4.3.21')],\n",
       "   'Function': 'The enzyme prefers aromatic over aliphatic amines',\n",
       "   'Regulation Activity': 'Inhibited by 2-hydrazinopyridine',\n",
       "   'Pathway': 'Amino-acid degradation; L-phenylalanine degradation; phenylacetate from L-phenylalanine: step 2/3'},\n",
       "  'seq': 'MGSPSLYSARKTTLALAVALSFAWQAPVFAHGGEAHMVPMDKTLKEFGADVQWDDYAQLFTLIKDGAYVKVKPGAQTAIVNGQPLALQVPVVMKDNKAWVSDTFINDVFQSGLDQTFQVEKRPHPLNALTADEIKQAVEIVKASADFKPNTRFTEISLLPPDKEAVWAFALENKPVDQPRKADVIMLDGKHIIEAVVDLQNNKLLSWQPIKDAHGMVLLDDFASVQNIINNSEEFAAAVKKRGITDAKKVITTPLTVGYFDGKDGLKQDARLLKVISYLDVGDGNYWAHPIENLVAVVDLEQKKIVKIEEGPVVPVPMTARPFDGRDRVAPAVKPMQIIEPEGKNYTITGDMIHWRNWDFHLSMNSRVGPMISTVTYNDNGTKRKVMYEGSLGGMIVPYGDPDIGWYFKAYLDSGDYGMGTLTSPIARGKDAPSNAVLLNETIADYTGVPMEIPRAIAVFERYAGPEYKHQEMGQPNVSTERRELVVRWISTVGNYDYIFDWIFHENGTIGIDAGATGIEAVKGVKAKTMHDETAKDDTRYGTLIDHNIVGTTHQHIYNFRLDLDVDGENNSLVAMDPVVKPNTAGGPRTSTMQVNQYNIGNEQDAAQKFDPGTIRLLSNPNKENRMGNPVSYQIIPYAGGTHPVAKGAQFAPDEWIYHRLSFMDKQLWVTRYHPGERFPEGKYPNRSTHDTGLGQYSKDNESLDNTDAVVWMTTGTTHVARAEEWPIMPTEWVHTLLKPWNFFDETPTLGALKKDK',\n",
       "  'mw': 84379.0},\n",
       " 'b1398': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P76085',\n",
       "  'gene': 'b1398',\n",
       "  'locus': ['b1398', 'JW5218'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('2-phenylacetate + ATP + CoA = AMP + diphosphate + phenylacetyl-CoA',\n",
       "     '6.2.1.30')],\n",
       "   'Function': 'Catalyzes the activation of phenylacetic acid (PA) to phenylacetyl-CoA (PA-CoA)',\n",
       "   'Pathway': 'Aromatic compound metabolism; phenylacetate degradation'},\n",
       "  'seq': 'MITNTKLDPIETASVDELQALQTQRLKWTLKHAYENVPMYRRKFDAAGVHPDDFRELSDLRKFPCTTKQDLRDNYPFDTFAVPMEQVVRIHASSGTTGKPTVVGYTQNDIDNWANIVARSLRAAGGSPKDKIHVAYGYGLFTGGLGAHYGAERLGATVIPMSGGQTEKQAQLIRDFQPDMIMVTPSYCLNLIEELERQLGGDASGCSLRVGVFGAEPWTQAMRKEIERRLGITALDIYGLSEVMGPGVAMECLETTDGPTIWEDHFYPEIVNPHDGTPLADGEHGELLFTTLTKEALPVIRYRTRDLTRLLPGTARTMRRMDRISGRSDDMLIIRGVNVFPSQLEEEIVKFEHLSPHYQLEVNRRGHLDSLSVKVELKESSLTLTHEQRCQVCHQLRHRIKSMVGISTDVMIVNCGSIPRSEGKACRVFDLRNIVGA',\n",
       "  'mw': 48953.0},\n",
       " 'b1415': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P25553',\n",
       "  'gene': 'b1415',\n",
       "  'locus': ['b1415', 'JW1412'],\n",
       "  'synonyms': 'ald',\n",
       "  'properties': {'Catalytic Activity': [('(S)-lactaldehyde + H2O + NAD(+) = (S)-lactate + 2 H(+) + NADH',\n",
       "     '1.2.1.22'),\n",
       "    ('glycolaldehyde + H2O + NAD(+) = glycolate + 2 H(+) + NADH', '1.2.1.21')],\n",
       "   'Function': 'Catalyzes the irreversible oxidation of L-lactaldehyde to L-lactate (PubMed:6345530, PubMed:3298215, PubMed:3308886, PubMed:27671251). Also shows high activity with glycolaldehyde and L-glyceraldehyde (PubMed:6345530, PubMed:3308886, PubMed:3275622, PubMed:16731973, PubMed:31850327). Has weaker activity with various aldehydes such as methylglyoxal, propionaldehyde or benzaldehyde (PubMed:3308886, PubMed:16731973). Involved in the degradation of lactaldehyde produced during metabolism of L-fucose and L-rhamnose (PubMed:3298215, PubMed:3275622). It may be involved in several other metabolic pathways (PubMed:3308886)',\n",
       "   'Regulation Activity': 'Substrate inhibition is very strong with lactaldehyde, diminishing progressively with glycolaldehyde, glyceraldehyde or methylglyoxal (PubMed:3308886). Inhibited by p-hydroxy mercuribenzoate and by some cations, including Mn(2+), Ca(2+), Cu(2+) and Zn(2+) (PubMed:3308886). Inhibited by NADH (PubMed:3275622)',\n",
       "   'Pathway': 'Carbohydrate degradation; L-rhamnose degradation'},\n",
       "  'seq': 'MSVPVQHPMYIDGQFVTWRGDAWIDVVNPATEAVISRIPDGQAEDARKAIDAAERAQPEWEALPAIERASWLRKISAGIRERASEISALIVEEGGKIQQLAEVEVAFTADYIDYMAEWARRYEGEIIQSDRPGENILLFKRALGVTTGILPWNFPFFLIARKMAPALLTGNTIVIKPSEFTPNNAIAFAKIVDEIGLPRGVFNLVLGRGETVGQELAGNPKVAMVSMTGSVSAGEKIMATAAKNITKVCLELGGKAPAIVMDDADLELAVKAIVDSRVINSGQVCNCAERVYVQKGIYDQFVNRLGEAMQAVQFGNPAERNDIAMGPLINAAALERVEQKVARAVEEGARVAFGGKAVEGKGYYYPPTLLLDVRQEMSIMHEETFGPVLPVVAFDTLEDAISMANDSDYGLTSSIYTQNLNVAMKAIKGLKFGETYINRENFEAMQGFHAGWRKSGIGGADGKHGLHEYLQTQVVYLQS',\n",
       "  'mw': 52273.0},\n",
       " 'b1440': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P76108',\n",
       "  'gene': 'b1440',\n",
       "  'locus': ['b1440', 'JW1435'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(3R)-3-hydroxybutanoyl-CoA + [(3R)-hydroxybutanoate](n) = [(3R)-hydroxybutanoate](n+1) + CoA',\n",
       "     '')],\n",
       "   'Function': 'Catalyzes the formation of short polymers of R-3-hydroxybutyrate (cPHB) (PubMed:18640095). Involved in natural transformation (PubMed:26826386). Probably part of the ABC transporter complex YdcSTUV. During natural transformation, may bind dsDNA and convey it to the inner membrane channel formed by YdcV (Probable)'},\n",
       "  'seq': 'MSKTFARSSLCALSMTIMTAHAAEPPTNLDKPEGRLDIIAWPGYIERGQTDKQYDWVTQFEKETGCAVNVKTAATSDEMVSLMTKGGYDLVTASGDASLRLIMGKRVQPINTALIPNWKTLDPRVVKGDWFNVGGKVYGTPYQWGPNLLMYNTKTFPTPPDSWQVVFVEQNLPDGKSNKGRVQAYDGPIYIADAALFVKATQPQLGISDPYQLTEEQYQAVLKVLRAQHSLIHRYWHDTTVQMSDFKNEGVVASSAWPYQANALKAEGQPVATVFPKEGVTGWADTTMLHSEAKHPVCAYKWMNWSLTPKVQGDVAAWFGSLPVVPEGCKASPLLGEKGCETNGFNYFDKIAFWKTPIAEGGKFVPYSRWTQDYIAIMGGR',\n",
       "  'mw': 42295.0},\n",
       " 'b1441': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77795',\n",
       "  'gene': 'b1441',\n",
       "  'locus': ['b1441', 'JW1436'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Probably part of the ABC transporter complex YdcSTUV. Probably responsible for energy coupling to the transport system'},\n",
       "  'seq': 'MTYAVEFDNVSRLYGDVRAVDGVSIAIKDGEFFSMLGPSGSGKTTCLRLIAGFEQLSGGAISIFGKPASNLPPWERDVNTVFQDYALFPHMSILDNVAYGLMVKGVNKKQRHAMAQEALEKVALGFVHQRKPSQLSGGQRQRVAIARALVNEPRVLLLDEPLGALDLKLREQMQLELKKLQQSLGITFIFVTHDQGEALSMSDRVAVFNNGRIEQVDSPRDLYMRPRTPFVAGFVGTSNVFDGLMAEKLCGMTGSFALRPEHIRLNTPGELQANGTIQAVQYQGAATRFELKLNGGEKLLVSQANMTGEELPATLTPGQQVMVSWSRDVMVPLVEER',\n",
       "  'mw': 37041.0},\n",
       " 'b1442': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77156',\n",
       "  'gene': 'b1442',\n",
       "  'locus': ['b1442', 'JW1437'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Probably part of the ABC transporter complex YdcSTUV. Probably responsible for the translocation of the substrate across the membrane'},\n",
       "  'seq': 'MAMNVLQSPSRPGLGKVSGFFWHNPGLGLFLLLLGPLMWFGIVYFGSLLTLLWQGFYTFDDFTMSVTPELTLANIRALFNPANYDIILRTLTMAVAVTIASAILAFPMAWYMARYTSGKMKAFFYIAVMLPMWASYIVKAYAWTLLLAKDGVAQWFLQHLGLEPLLTAFLTLPAVGGNTLSTSGLGRFLVFLYIWLPFMILPVQAALERLPPSLLQASADLGARPRQTFRYVVLPLAIPGIAAGSIFTFSLTLGDFIVPQLVGPPGYFIGNMVYSQQGAIGNMPMAAAFTLVPIILIALYLAFVKRLGAFDAL',\n",
       "  'mw': 34360.0},\n",
       " 'b1443': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AFR9',\n",
       "  'gene': 'b1443',\n",
       "  'locus': ['b1443', 'JW1438'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Involved in natural transformation (PubMed:26826386). Probably part of the ABC transporter complex YdcSTUV. Probably responsible for the translocation of the substrate across the membrane. During natural transformation, may serve as the channel for dsDNA uptake (Probable)'},\n",
       "  'seq': 'MHSERAPFFLKLAAWGGVVFLHFPILIIAAYAFNTEDAAFSFPPQGLTLRWFSVAAQRSDILDAVTLSLKVAALATLIALVLGTLAAAALWRRDFFGKNAISLLLLLPIALPGIVTGLALLTAFKTINLEPGFFTIVVGHATFCVVVVFNNVIARFRRTSWSLVEASMDLGANGWQTFRYVVLPNLSSALLAGGMLAFALSFDEIIVTTFTAGHERTLPLWLLNQLGRPRDVPVTNVVALLVMLVTTLPILGAWWLTREGDNGQ',\n",
       "  'mw': 28722.0},\n",
       " 'b1444': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77674',\n",
       "  'gene': 'b1444',\n",
       "  'locus': ['b1444', 'JW1439'],\n",
       "  'synonyms': 'prr ydcW',\n",
       "  'properties': {'Catalytic Activity': [('4-aminobutanal + H2O + NAD(+) = 4-aminobutanoate + 2 H(+) + NADH',\n",
       "     '1.2.1.19'),\n",
       "    ('5-aminopentanal + H2O + NAD(+) = 5-aminopentanoate + 2 H(+) + NADH',\n",
       "     '')],\n",
       "   'Function': 'Catalyzes the oxidation 4-aminobutanal (gamma-aminobutyraldehyde) to 4-aminobutanoate (gamma-aminobutyrate or GABA) (PubMed:3510672, PubMed:16023116). This is the second step in one of two pathways for putrescine degradation, where putrescine is converted into 4-aminobutanoate via 4-aminobutanal, which allows E.coli to grow on putrescine as the sole nitrogen source (PubMed:3510672, PubMed:22636776). Also functions as a 5-aminopentanal dehydrogenase in a a L-lysine degradation pathway to succinate that proceeds via cadaverine, glutarate and L-2-hydroxyglutarate (PubMed:30498244). Can also oxidize n-alkyl medium-chain aldehydes, but with a lower catalytic efficiency (PubMed:15381418, PubMed:16023116)',\n",
       "   'Pathway': 'Amino-acid degradation'},\n",
       "  'seq': 'MQHKLLINGELVSGEGEKQPVYNPATGDVLLEIAEASAEQVDAAVRAADAAFAEWGQTTPKVRAECLLKLADVIEENGQVFAELESRNCGKPLHSAFNDEIPAIVDVFRFFAGAARCLNGLAAGEYLEGHTSMIRRDPLGVVASIAPWNYPLMMAAWKLAPALAAGNCVVLKPSEITPLTALKLAELAKDIFPAGVINILFGRGKTVGDPLTGHPKVRMVSLTGSIATGEHIISHTASSIKRTHMELGGKAPVIVFDDADIEAVVEGVRTFGYYNAGQDCTAACRIYAQKGIYDTLVEKLGAAVATLKSGAPDDESTELGPLSSLAHLERVGKAVEEAKATGHIKVITGGEKRKGNGYYYAPTLLAGALQDDAIVQKEVFGPVVSVTPFDNEEQVVNWANDSQYGLASSVWTKDVGRAHRVSARLQYGCTWVNTHFMLVSEMPHGGQKLSGYGKDMSLYGLEDYTVVRHVMVKH',\n",
       "  'mw': 50830.0},\n",
       " 'b1453': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77610',\n",
       "  'gene': 'b1453',\n",
       "  'locus': ['b1453', 'JW5234'],\n",
       "  'synonyms': 'yncF',\n",
       "  'properties': {'Catalytic Activity': []},\n",
       "  'seq': 'MSKHDTDTSDQHAAKRRWLNAHEEGYHKAMGNRQVQMIAIGGAIGTGLFLGAGARLQMAGPALALVYLICGLFSFFILRALGELVLHRPSSGSFVSYAREFLGEKAAYVAGWMYFINWAMTGIVDITAVALYMHYWGAFGGVPQWVFALAALTIVGTMNMIGVKWFAEMEFWFALIKVLAIVTFLVVGTVFLGSGQPLDGNTTGFHLITDNGGFFPHGLLPALVLIQGVVFAFASIEMVGTAAGECKDPQTMVPKAINSVIWRIGLFYVGSVVLLVMLLPWSAYQAGQSPFVTFFSKLGVPYIGSIMNIVVLTAALSSLNSGLYCTGRILRSMAMGGSAPSFMAKMSRQHVPYAGILATLVVYVVGVFLNYLVPSRVFEIVLNFASLGIIASWAFIIVCQMRLRKAIKEGKAADVSFKLPGAPFTSWLTLLFLLSVLVLMAFDYPNGTYTIAALPIIGILLVIGWFGVRKRVAEIHSTAPVVEEDEEKQEIVFKPETAS',\n",
       "  'mw': 54233.0},\n",
       " 'b1463': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77567',\n",
       "  'gene': 'b1463',\n",
       "  'locus': ['b1463', 'JW1458'],\n",
       "  'synonyms': 'yddI',\n",
       "  'properties': {'Catalytic Activity': [('acetyl-CoA + an arylamine = an N-acetylarylamine + CoA',\n",
       "     '2.3.1.5'),\n",
       "    ('acetyl-CoA + an N-hydroxyarylamine = an N-acetoxyarylamine + CoA',\n",
       "     '2.3.1.118')],\n",
       "   'Function': 'Catalyzes the acetyl-CoA-dependent N-acetylation of aromatic amines, and, probably, the O-acetylation of N-hydroxyarylamines. In vitro, catalyzes the N-acetylation of various arylamines such as aminobenzoic acid, aminophenol, aminotoluene, phenetidine, anisidine, aniline, isoniazid and 2-amino-fluorene (PubMed:10806332, PubMed:23452042). N-hydroxyarylamine O-acetyltransferase activity has not been assayed directly, however, NhoA activity is required for the mutagenicity of nitroaromatic compounds, suggesting that it also has O-acetyltransferase activity (Probable)',\n",
       "   'Regulation Activity': 'Inhibited by salicylic acid, acetylsalicylic acid, 2,6-dichrolo-4-nitrophenol, N-ethylmaleimide and iodoacetamide'},\n",
       "  'seq': 'MTPILNHYFARINWSGAAAVNIDTLRALHLKHNCTIPFENLDVLLPREIQLDNQSPEEKLVIARRGGYCFEQNGVFERVLRELGFNVRSLLGRVVLSNPPALPPRTHRLLLVELEEEKWIADVGFGGQTLTAPIRLVSDLVQTTPHGEYRLLQEGDDWVLQFNHHQHWQSMYRFDLCEQQQSDYVMGNFWSAHWPQSHFRHHLLMCRHLPDGGKLTLTNFHFTHYENGHAVEQRNLPDVASLYAVMQEQFGLGVDDAKHGFTVDELALVMAAFDTHPEAGK',\n",
       "  'mw': 32275.0},\n",
       " 'b1465': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AF32',\n",
       "  'gene': 'b1465',\n",
       "  'locus': ['b1465', 'JW1460'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('a quinol + nitrate = a quinone + H2O + nitrite',\n",
       "     '1.7.5.1')],\n",
       "   'Function': 'This is a second nitrate reductase enzyme which can substitute for the NRA enzyme and allows E.coli to use nitrate as an electron acceptor during anaerobic growth. The gamma chain is a membrane-embedded heme-iron unit resembling cytochrome b, which transfers electrons from quinones to the beta subunit'},\n",
       "  'seq': 'MIQYLNVFFYDIYPYICATVFFLGSWLRYDYGQYTWRASSSQMLDKRGMVIWSNLFHIGILGIFFGHLFGMLTPHWMYAWFLPVAAKQLMAMVLGGICGVLTLIGGAGLLWRRLTNQRVRATSTTPDIIIMSILLIQCLLGLSTIPFSAQYPDGSEMMKLVGWAQSIVTFRGGSSEMLNGVAFVFRLHLVLGMTIFLLFPFTRLVHVWSAPFEYFTRRYQIVRSRR',\n",
       "  'mw': 26018.0},\n",
       " 'b1466': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P19317',\n",
       "  'gene': 'b1466',\n",
       "  'locus': ['b1466', 'JW1461'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Chaperone required for proper molybdenum cofactor insertion and final assembly of the membrane-bound respiratory nitrate reductase 2'},\n",
       "  'seq': 'MQILKVIGLLMEYPDELLWECKEDALALIRRDAPMLTDFTHNLLNAPLLDKQAEWCEVFDRGRTTSLLLFEHVHAESRDRGQAMVDLLAEYEKVGLQLDCRELPDYLPLYLEYLSVLPDDQAKEGLLNVAPILALLGGRLKQREAPWYALFDALLQLAGSSLSSDSVTKQVNSEERDDTRQALDAVWEEEQVKFIEDNATACDSSPLNQYQRRFSQDVAPQYVDISAGGGK',\n",
       "  'mw': 26161.0},\n",
       " 'b1467': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P19318',\n",
       "  'gene': 'b1467',\n",
       "  'locus': ['b1467', 'JW1462'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('a quinol + nitrate = a quinone + H2O + nitrite',\n",
       "     '1.7.5.1')],\n",
       "   'Function': 'This is a second nitrate reductase enzyme which can substitute for the NRA enzyme and allows E.coli to use nitrate as an electron acceptor during anaerobic growth. The beta chain is an electron transfer unit containing four cysteine clusters involved in the formation of iron-sulfur centers. Electrons are transferred from the gamma chain to the molybdenum cofactor of the alpha subunit'},\n",
       "  'seq': 'MKIRSQVGMVLNLDKCIGCHTCSVTCKNVWTGREGMEYAWFNNVETKPGIGYPKNWEDQEEWQGGWVRDVNGKIRPRLGNKMGVITKIFANPVVPQIDDYYEPFTFDYEHLHSAPEGKHIPTARPRSLIDGKRMDKVIWGPNWEELLGGEFEKRARDRNFEAMQKEMYGQFENTFMMYLPRLCEHCLNPSCVATCPSGAIYKREEDGIVLIDQDKCRGWRLCISGCPYKKIYFNWKSGKSEKCIFCYPRIESGQPTVCSETCVGRIRYLGVLLYDADRIEEAASTEREVDLYERQCEVFLDPHDPSVIEEALKQGIPQNVIDAAQRSPVYKMAMDWKLALPLHPEYRTLPMVWYVPPLSPIQSYADAGGLPKSEGVLPAIESLRIPVQYLANMLSAGDTGPVLRALKRMMAMRHYMRSQTVEGVTDTRAIDEVGLSVAQVEEMYRYLAIANYEDRFVIPTSHREMAGDAFAERNGCGFTFGDGCHGSDSKFNLFNSSRIDAINITEVRDKAEGE',\n",
       "  'mw': 58558.0},\n",
       " 'b1468': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P19319',\n",
       "  'gene': 'b1468',\n",
       "  'locus': ['b1468', 'JW1463'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('a quinol + nitrate = a quinone + H2O + nitrite',\n",
       "     '1.7.5.1')],\n",
       "   'Function': 'The alpha chain is the actual site of nitrate reduction'},\n",
       "  'seq': 'MSKLLDRFRYFKQKGETFADGHGQVMHSNRDWEDSYRQRWQFDKIVRSTHGVNCTGSCSWKIYVKNGLVTWEIQQTDYPRTRPDLPNHEPRGCPRGASYSWYLYSANRLKYPLIRKRLIELWREALKQHSDPVLAWASIMNDPQKCLSYKQVRGRGGFIRSNWQELNQLIAAANVWTIKTYGPDRVAGFSPIPAMSMVSYAAGTRYLSLLGGTCLSFYDWYCDLPPASPMTWGEQTDVPESADWYNSSYIIAWGSNVPQTRTPDAHFFTEVRYKGTKTIAITPDYSEVAKLCDQWLAPKQGTDSALAMAMGHVILKEFHLDNPSDYFINYCRRYSDMPMLVMLEPRDDGSYVPGRMIRASDLVDGLGESNNPQWKTVAVNTAGELVVPNGSIGFRWGEKGKWNLESIAAGTETELSLTLLGQHDAVAGVAFPYFGGIENPHFRSVKHNPVLVRQLPVKNLTLVDGNTCPVVSVYDLVLANYGLDRGLEDENSAKDYAEIKPYTPAWGEQITGVPRQYIETIAREFADTAHKTHGRSMIILGAGVNHWYHMDMNYRGMINMLIFCGCVGQSGGGWAHYVGQEKLRPQTGWLPLAFALDWNRPPRQMNSTSFFYNHSSQWRYEKVSAQELLSPLADASKYSGHLIDFNVRAERMGWLPSAPQLGRNPLGIKAEADKAGLSPTEFTAQALKSGDLRMACEQPDSSSNHPRNLFVWRSNLLGSSGKGHEYMQKYLLGTESGIQGEELGASDGIKPEEVEWQTAAIEGKLDLLVTLDFRMSSTCLFSDIVLPTATWYEKDDMNTSDMHPFIHPLSAAVDPAWESRSDWEIYKGIAKAFSQVCVGHLGKETDVVLQPLLHDSPAELSQPCEVLDWRKGECDLIPGKTAPNIVAVERDYPATYERFTSLGPLMDKLGNGGKGISWNTQDEIDFLGKLNYTKRDGPAQGRPLIDTAIDASEVILALAPETNGHVAVKAWQALGEITGREHTHLALHKEDEKIRFRDIQAQPRKIISSPTWSGLESDHVSYNAGYTNVHELIPWRTLSGRQQLYQDHPWMRAFGESLVAYRPPIDTRSVSEMRQIPPNGFPEKALNFLTPHQKWGIHSTYSENLLMLTLSRGGPIVWISETDARELTIVDNDWVEVFNANGALTARAVVSQRVPPGMTMMYHAQERIMNIPGSEVTGMRGGIHNSVTRVCPKPTHMIGGYAQLAWGFNYYGTVGSNRDEFIMIRKMKNVNWLDDEGRDQVQEAKK',\n",
       "  'mw': 140227.0},\n",
       " 'b1469': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37758',\n",
       "  'gene': 'b1469',\n",
       "  'locus': ['b1469', 'JW1464'],\n",
       "  'synonyms': 'yddF',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Catalyzes nitrate uptake, nitrite uptake and nitrite export across the cytoplasmic membrane. May function as a nitrate/H(+) and nitrite/H(+) channel. Could confer a selective advantage during severe nutrient starvation or slow growth'},\n",
       "  'seq': 'MALQNEKNSRYLLRDWKPENPAFWENKGKHIARRNLWISVSCLLLAFCVWMLFSAVTVNLNKIGFNFTTDQLFLLTALPSVSGALLRVPYSFMVPIFGGRRWTVFSTAILIIPCVWLGIAVQNPNTPFGIFIVIALLCGFAGANFASSMGNISFFFPKAKQGSALGINGGLGNLGVSVMQLVAPLVIFVPVFAFLGVNGVPQADGSVMSLANAAWIWVPLLAIATIAAWSGMNDIASSRASIADQLPVLQRLHLWLLSLLYLATFGSFIGFSAGFAMLAKTQFPDVNILRLAFFGPFIGAIARSVGGAISDKFGGVRVTLINFIFMAIFSALLFLTLPGTGSGNFIAFYAVFMGLFLTAGLGSGSTFQMIAVIFRQITIYRVKMKGGSDEQAHKEAVTETAAALGFISAIGAVGGFFIPQAFGMSLNMTGSPVGAMKVFLIFYIVCVLLTWLVYGRRKFSQK',\n",
       "  'mw': 49890.0},\n",
       " 'b1474': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P24183',\n",
       "  'gene': 'b1474',\n",
       "  'locus': ['b1474', 'JW1470'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('a quinone + formate + H(+) = a quinol + CO2',\n",
       "     '1.17.5.3')],\n",
       "   'Function': 'Formate dehydrogenase allows E.coli to use formate as major electron donor during anaerobic respiration, when nitrate is used as electron acceptor. The alpha subunit FdnG contains the formate oxidation site. Electrons are transferred from formate to menaquinone in the gamma subunit (FdnI), through the 4Fe-4S clusters in the beta subunit (FdnH). Formate dehydrogenase-N is part of a system that generates proton motive force, together with the dissimilatory nitrate reductase (Nar)'},\n",
       "  'seq': 'MDVSRRQFFKICAGGMAGTTVAALGFAPKQALAQARNYKLLRAKEIRNTCTYCSVGCGLLMYSLGDGAKNAREAIYHIEGDPDHPVSRGALCPKGAGLLDYVNSENRLRYPEYRAPGSDKWQRISWEEAFSRIAKLMKADRDANFIEKNEQGVTVNRWLSTGMLCASGASNETGMLTQKFARSLGMLAVDNQARVUHGPTVASLAPTFGRGAMTNHWVDIKNANVVMVMGGNAAEAHPVGFRWAMEAKNNNDATLIVVDPRFTRTASVADIYAPIRSGTDITFLSGVLRYLIENNKINAEYVKHYTNASLLVRDDFAFEDGLFSGYDAEKRQYDKSSWNYQLDENGYAKRDETLTHPRCVWNLLKEHVSRYTPDVVENICGTPKADFLKVCEVLASTSAPDRTTTFLYALGWTQHTVGAQNIRTMAMIQLLLGNMGMAGGGVNALRGHSNIQGLTDLGLLSTSLPGYLTLPSEKQVDLQSYLEANTPKATLADQVNYWSNYPKFFVSLMKSFYGDAAQKENNWGYDWLPKWDQTYDVIKYFNMMDEGKVTGYFCQGFNPVASFPDKNKVVSCLSKLKYMVVIDPLVTETSTFWQNHGESNDVDPASIQTEVFRLPSTCFAEEDGSIANSGRWLQWHWKGQDAPGEARNDGEILAGIYHHLRELYQSEGGKGVEPLMKMSWNYKQPHEPQSDEVAKENNGYALEDLYDANGVLIAKKGQLLSSFAHLRDDGTTASSCWIYTGSWTEQGNQMANRDNSDPSGLGNTLGWAWAWPLNRRVLYNRASADINGKPWDPKRMLIQWNGSKWTGNDIPDFGNAAPGTPTGPFIMQPEGMGRLFAINKMAEGPFPEHYEPIETPLGTNPLHPNVVSNPVVRLYEQDALRMGKKEQFPYVGTTYRLTEHFHTWTKHALLNAIAQPEQFVEISETLAAAKGINNGDRVTVSSKRGFIRAVAVVTRRLKPLNVNGQQVETVGIPIHWGFEGVARKGYIANTLTPNVGDANSQTPEYKAFLVNIEKA',\n",
       "  'mw': 112963.0},\n",
       " 'b1475': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AAJ3',\n",
       "  'gene': 'b1475',\n",
       "  'locus': ['b1475', 'JW1471'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Formate dehydrogenase allows E.coli to use formate as major electron donor during anaerobic respiration, when nitrate is used as electron acceptor. The beta subunit FdnH is an electron transfer unit containing 4 iron-sulfur clusters; it serves as a conduit for electrons that are transferred from the formate oxidation site in the alpha subunit (FdnG) to the menaquinone associated with the gamma subunit (FdnI) of formate dehydrogenase-N. Formate dehydrogenase-N is part of a system that generates proton motive force, together with the dissimilatory nitrate reductase (Nar)'},\n",
       "  'seq': 'MAMETQDIIKRSATNSITPPSQVRDYKAEVAKLIDVSTCIGCKACQVACSEWNDIRDEVGHCVGVYDNPADLSAKSWTVMRFSETEQNGKLEWLIRKDGCMHCEDPGCLKACPSAGAIIQYANGIVDFQSENCIGCGYCIAGCPFNIPRLNKEDNRVYKCTLCVDRVSVGQEPACVKTCPTGAIHFGTKKEMLELAEQRVAKLKARGYEHAGVYNPEGVGGTHVMYVLHHADQPELYHGLPKDPKIDTSVSLWKGALKPLAAAGFIATFAGLIFHYIGIGPNKEVDDDEEDHHE',\n",
       "  'mw': 32239.0},\n",
       " 'b1476': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AEK7',\n",
       "  'gene': 'b1476',\n",
       "  'locus': ['b1476', 'JW1472'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Formate dehydrogenase allows E.coli to use formate as major electron donor during anaerobic respiration, when nitrate is used as electron acceptor. Subunit gamma is the cytochrome b556 component of the formate dehydrogenase-N, and also contains a menaquinone reduction site that receives electrons from the beta subunit (FdnH), through its hemes. Formate dehydrogenase-N is part of a system that generates proton motive force, together with the dissimilatory nitrate reductase (Nar)'},\n",
       "  'seq': 'MSKSKMIVRTKFIDRACHWTVVICFFLVALSGISFFFPTLQWLTQTFGTPQMGRILHPFFGIAIFVALMFMFVRFVHHNIPDKKDIPWLLNIVEVLKGNEHKVADVGKYNAGQKMMFWSIMSMIFVLLVTGVIIWRPYFAQYFPMQVVRYSLLIHAAAGIILIHAILIHMYMAFWVKGSIKGMIEGKVSRRWAKKHHPRWYREIEKAEAKKESEEGI',\n",
       "  'mw': 25368.0},\n",
       " 'b1478': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P39451',\n",
       "  'gene': 'b1478',\n",
       "  'locus': ['b1478', 'JW1474'],\n",
       "  'synonyms': 'yddN',\n",
       "  'properties': {'Catalytic Activity': [('a primary alcohol + NAD(+) = an aldehyde + H(+) + NADH',\n",
       "     '1.1.1.1'),\n",
       "    ('a secondary alcohol + NAD(+) = a ketone + H(+) + NADH', '1.1.1.1')],\n",
       "   'Function': 'Preferred specificity is towards 1-propanol'},\n",
       "  'seq': 'MKAAVVTKDHHVDVTYKTLRSLKHGEALLKMECCGVCHTDLHVKNGDFGDKTGVILGHEGIGVVAEVGPGVTSLKPGDRASVAWFYEGCGHCEYCNSGNETLCRSVKNAGYSVDGGMAEECIVVADYAVKVPDGLDSAAASSITCAGVTTYKAVKLSKIRPGQWIAIYGLGGLGNLALQYAKNVFNAKVIAIDVNDEQLKLATEMGADLAINSHTEDAAKIVQEKTGGAHAAVVTAVAKAAFNSAVDAVRAGGRVVAVGLPPESMSLDIPRLVLDGIEVVGSLVGTRQDLTEAFQFAAEGKVVPKVALRPLADINTIFTEMEEGKIRGRMVIDFRH',\n",
       "  'mw': 35380.0},\n",
       " 'b1479': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P26616',\n",
       "  'gene': 'b1479',\n",
       "  'locus': ['b1479', 'JW5238'],\n",
       "  'synonyms': 'sfcA',\n",
       "  'properties': {'Catalytic Activity': [('(S)-malate + NAD(+) = CO2 + NADH + pyruvate',\n",
       "     '1.1.1.38'),\n",
       "    ('H(+) + oxaloacetate = CO2 + pyruvate', '1.1.1.38')],\n",
       "   'Regulation Activity': 'Non-competitively inhibited by high concentrations of NAD(+) and L-malate. Also inhibited by CoA, acetyl-phosphate, palmitoyl-CoA, and oxaloacetate. Activated by aspartate'},\n",
       "  'seq': 'MEPKTKKQRSLYIPYAGPVLLEFPLLNKGSAFSMEERRNFNLLGLLPEVVETIEEQAERAWIQYQGFKTEIDKHIYLRNIQDTNETLFYRLVNNHLDEMMPVIYTPTVGAACERFSEIYRRSRGVFISYQNRHNMDDILQNVPNHNIKVIVVTDGERILGLGDQGIGGMGIPIGKLSLYTACGGISPAYTLPVVLDVGTNNQQLLNDPLYMGWRNPRITDDEYYEFVDEFIQAVKQRWPDVLLQFEDFAQKNAMPLLNRYRNEICSFNDDIQGTAAVTVGTLIAASRAAGGQLSEKKIVFLGAGSAGCGIAEMIISQTQREGLSEEAARQKVFMVDRFGLLTDKMPNLLPFQTKLVQKRENLSDWDTDSDVLSLLDVVRNVKPDILIGVSGQTGLFTEEIIREMHKHCPRPIVMPLSNPTSRVEATPQDIIAWTEGNALVATGSPFNPVVWKDKIYPIAQCNNAFIFPGIGLGVIASGASRITDEMLMSASETLAQYSPLVLNGEGMVLPELKDIQKVSRAIAFAVGKMAQQQGVAVKTSAEALQQAIDDNFWQAEYRDYRRTSI',\n",
       "  'mw': 63197.0},\n",
       " 'b1483': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77622',\n",
       "  'gene': 'b1483',\n",
       "  'locus': ['b1483', 'JW1478'],\n",
       "  'synonyms': 'yddO',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the ABC transporter complex DdpABCDF, which is probably involved in D,D-dipeptide transport (PubMed:9097039). Probably responsible for energy coupling to the transport system'},\n",
       "  'seq': 'MSDTLLTLRDVHINFPARKNWLGKTTEHVHAINGIDLQIRRGETLGIVGESGCGKSTLAQLLMGMLQPSHGQYIRSGSQRIMQMVFQDPLSSLNPRLPVWRIITEPLWIAKRSSEQQRRALAEELAVQVGIRPEYLDRLPHAFSGGQRQRIAIARALSSQPDVIVLDEPTSALDISVQAQILNLLVTLQENHGLTYVLISHNVSVIRHMSDRVAVMYLGQIVELGDAQQVLTAPAHPYTRLLLDSLPAIDKPLEEEWALRKTDLPGNRTLPQGCFFYERCPLATHGCEVRQSLAIREDGRELRCWRAL',\n",
       "  'mw': 34621.0},\n",
       " 'b1484': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77268',\n",
       "  'gene': 'b1484',\n",
       "  'locus': ['b1484', 'JW1479'],\n",
       "  'synonyms': 'yddP',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the ABC transporter complex DdpABCDF, which is probably involved in D,D-dipeptide transport (PubMed:9097039). Probably responsible for energy coupling to the transport system'},\n",
       "  'seq': 'MTQPVLDIQQLHLSFPGFNGDVHALNNVSLQINRGEIVGLVGESGSGKSVTAMLIMRLLPTGSYCVHRGQISLLGEDVLNAREKQLRQWRGARVAMIFQEPMTALNPTRRIGLQMMDVIRHHQPISRREARAKAIDLLEEMQIPDAVEVMSRYPFELSGGMRQRVMIALAFSCEPQLIIADEPTTALDVTVQLQVLRLLKHKARASGTAVLFISHDMAVVSQLCDSVYVMYAGSVIESGVTADVIHHPRHPYTIGLLQCAPEHGVPRQLLPAIPGTVPNLTHLPDGCAFRDRCYAAGAQCENVPALTACGDNNQRCACWYPQQEVISV',\n",
       "  'mw': 36100.0},\n",
       " 'b1485': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77463',\n",
       "  'gene': 'b1485',\n",
       "  'locus': ['b1485', 'JW1480'],\n",
       "  'synonyms': 'yddQ',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the ABC transporter complex DdpABCDF, which is probably involved in D,D-dipeptide transport. Probably responsible for the translocation of the substrate across the membrane'},\n",
       "  'seq': 'MMLSEETSAVRPQKQTRFNGAKLVWMLKGSPLTVTSAVIIVLMLLMMIFSPWLATHDPNAIDLTARLLPPSAAHWFGTDEVGRDLFSRVLVGSQQSILAGLVVVAIAGMIGSLLGCLSGVLGGRADAIIMRIMDIMLSIPSLVLTMALAAALGPSLFNAMLAIAIVRIPFYVRLARGQALVVRQYTYVQAAKTFGASRWHLINWHILRNSLPPLIVQASLDIGSAILMAATLGFIGLGAQQPSAEWGAMVANGRNYVLDQWWYCAFPGAAILLTAVGFNLFGDGIRDLLDPKAGGKQS',\n",
       "  'mw': 31971.0},\n",
       " 'b1486': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77308',\n",
       "  'gene': 'b1486',\n",
       "  'locus': ['b1486', 'JW1481'],\n",
       "  'synonyms': 'yddR',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the ABC transporter complex DdpABCDF, which is probably involved in D,D-dipeptide transport. Probably responsible for the translocation of the substrate across the membrane'},\n",
       "  'seq': 'MTFWSILRQRCWGLVLVVAGVCVITFIISHLIPGDPARLLAGDRASDAIVENIRQQLGLDQPLYVQFYRYVSDLFHGDLGTSIRTGRPVLEELRIFFPATLELAFGALLLALLIGIPLGILSAVWRNRWLDHLVRIMAITGISTPAFWLGLGVIVLFYGHLQILPGGGRLDDWLDPPTHVTGFYLLDALLEGNGEVFFNALQHLILPALTLAFVHLGIVARQIRSAMLEQLSEDYIRTARASGLPGWYIVLCYALPNALIPSITVLGLALGDLLYGAVLTETVFAWPGMGAWVVTSIQALDFPAVMGFAVVVSFAYVLVNLVVDLLYLWIDPRIGRGGGE',\n",
       "  'mw': 37345.0},\n",
       " 'b1487': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P76128',\n",
       "  'gene': 'b1487',\n",
       "  'locus': ['b1487', 'JW5240'],\n",
       "  'synonyms': 'yddS',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the ABC transporter complex DdpABCDF, which is probably involved in D,D-dipeptide transport'},\n",
       "  'seq': 'MKRSISFRPTLLALVLATNFPVAHAAVPKDMLVIGKAADPQTLDPAVTIDNNDWTVTYPSYQRLVQYKTDGDKGSTDVEGDLASSWKASDDQKEWTFTLKDNAKFADGTPVTAEAVKLSFERLLKIGQGPAEAFPKDLKIDAPDEHTVKFTLSQPFAPFLYTLANDGASIINPAVLKEHAADDARGFLAQNTAGSGPFMLKSWQKGQQLVLVPNPHYPGNKPNFKRVSVKIIGESASRRLQLSRGDIDIADALPVDQLNALKQENKVNVAEYPSLRVTYLYLNNSKAPLNQADLRRAISWSTDYQGMVNGILSGNGKQMRGPIPEGMWGYDATAMQYNHDETKAKAEWDKVTSKPTSLTFLYSDNDPNWEPIALATQSSLNKLGIIVKLEKLANATMRDRVGKGDYDIAIGNWSPDFADPYMFMNYWFESDKKGLPGNRSFYENSEVDKLLRNALATTDQTQRTRDYQQAQKIVIDDAAYVYLFQKNYQLAMNKEVKGFVFNPMLEQVFNINTMSK',\n",
       "  'mw': 57641.0},\n",
       " 'b1488': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77790',\n",
       "  'gene': 'b1488',\n",
       "  'locus': ['b1488', 'JW1483'],\n",
       "  'synonyms': 'vanX yddT',\n",
       "  'properties': {'Catalytic Activity': [('D-alanyl-D-alanine + H2O = 2 D-alanine',\n",
       "     '3.4.13.22')],\n",
       "   'Function': 'Catalyzes hydrolysis of the D-alanyl-D-alanine dipeptide. May have a role in cell-wall turnover'},\n",
       "  'seq': 'MSDTTELVDLAVIFPDLEIELKYACADNITGKAIYQQARCLLHKDAITALAKSISIAQLSGLQLVIYDAYRPQQAQAMLWQACPDPQYVVDVTVGSNHSRGTAIDLTLRDEHGNILDMGAGFDEMHERSHAYHPSVPPAAQRNRLLLNAIMTGGGFVGISSEWWHFELPQAASYPLLADQFSCFISPGTQHVS',\n",
       "  'mw': 21213.0},\n",
       " 'b1489': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P76129',\n",
       "  'gene': 'b1489',\n",
       "  'locus': ['b1489', 'JW1484'],\n",
       "  'synonyms': 'dos pdeO yddU',\n",
       "  'properties': {'Catalytic Activity': [(\"3',3'-c-di-GMP + H2O = 5'-phosphoguanylyl(3'->5')guanosine + H(+)\",\n",
       "     '3.1.4.52')],\n",
       "   'Function': 'Heme-based oxygen sensor protein displaying phosphodiesterase (PDE) activity toward c-di-GMP in response to oxygen availability. Involved in the modulation of intracellular c-di-GMP levels, in association with DosC which catalyzes the biosynthesis of c-di-GMP (diguanylate cyclase activity). Cyclic-di-GMP is a second messenger which controls cell surface-associated traits in bacteria. Has very poor PDE activity on cAMP (PubMed:15995192) but is not active with cGMP, bis(p-nitrophenyl) phosphate or p-nitrophenyl phosphate (PubMed:11970957). Via its PDE activity on c-di-GMP, DosP regulates biofilm formation through the repression of transcription of the csgBAC operon, which encodes curli structural subunits',\n",
       "   'Regulation Activity': 'Has c-di-GMP PDE activity in both Fe(2+) and Fe(3+)-bound forms; this activity is increased 6-7 fold by binding of O(2) and CO (PubMed:15995192) and NO (PubMed:17535805). Has cAMP PDE activity only when the heme is in the Fe(2+) form. cAMP PDE activity is inhibited by oxidation of the heme iron and by binding of external ligands such as CO and NO. Also strongly inhibited by etazolate hydrochloride, a selective cAMP PDE inhibitor. PDE activity is inhibited in the absence of oxygen'},\n",
       "  'seq': 'MKLTDADNAADGIFFPALEQNMMGAVLINENDEVMFFNPAAEKLWGYKREEVIGNNIDMLIPRDLRPAHPEYIRHNREGGKARVEGMSRELQLEKKDGSKIWTRFALSKVSAEGKVYYLALVRDASVEMAQKEQTRQLIIAVDHLDRPVIVLDPERHIVQCNRAFTEMFGYCISEASGMQPDTLLNIPEFPADNRIRLQQLLWKTARDQDEFLLLTRTGEKIWIKASISPVYDVLAHLQNLVMTFSDITEERQIRQLEGNILAAMCSSPPFHEMGEIICRNIESVLNESHVSLFALRNGMPIHWASSSHGAEIQNAQSWSATIRQRDGAPAGILQIKTSSGAETSAFIERVADISQHMAALALEQEKSRQHIEQLIQFDPMTGLPNRNNLHNYLDDLVDKAVSPVVYLIGVDHIQDVIDSLGYAWADQALLEVVNRFREKLKPDQYLCRIEGTQFVLVSLENDVSNITQIADELRNVVSKPIMIDDKPFPLTLSIGISYDLGKNRDYLLSTAHNAMDYIRKNGGNGWQFFSPAMNEMVKERLVLGAALKEAISNNQLKLVYQPQIFAETGELYGIEALARWHDPLHGHVPPSRFIPLAEEIGEIENIGRWVIAEACRQLAEWRSQNIHIPALSVNLSALHFRSNQLPNQVSDAMHAWGIDGHQLTVEITESMMMEHDTEIFKRIQILRDMGVGLSVDDFGTGFSGLSRLVSLPVTEIKIDKSFVDRCLTEKRILALLEAITSIGQSLNLTVVAEGVETKEQFEMLRKIHCRVIQGYFFSRPLPAEEIPGWMSSVLPLKI',\n",
       "  'mw': 90260.0},\n",
       " 'b1492': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P63235',\n",
       "  'gene': 'b1492',\n",
       "  'locus': ['b1492', 'JW1487'],\n",
       "  'synonyms': 'acsA xasA',\n",
       "  'properties': {'Catalytic Activity': [('4-aminobutanoate(in) + L-glutamate(out) = 4-aminobutanoate(out) + L-glutamate(in)',\n",
       "     '')],\n",
       "   'Function': 'Involved in glutaminase-dependent acid resistance (PubMed:8682809, PubMed:30498489). Exchanges extracellular glutamate (Glu) for intracellular gamma-aminobutyric acid (GABA) under acidic conditions (PubMed:22407317, PubMed:23589309). The protonation states of substrates are crucial for transport. Selectively transports Glu with no net charge and GABA with a positive charge (PubMed:23589309). Also efficiently transports glutamine and, to a smaller extent, methionine and leucine (PubMed:22407317). When the extracellular pH drops below 2.5, can import L-glutamine and export either glutamate or GABA (PubMed:30498489). The ability to survive the extremely acidic conditions of the stomach is essential for successful colonization of the host by commensal and pathogenic bacteria (PubMed:8682809, PubMed:30498489)',\n",
       "   'Regulation Activity': 'Shows pH-dependent activity (PubMed:22407317, PubMed:23589309). The Glu/GABA transport activity is robust at pH 4.5 and rapidly decreases with increasing pH, with no detectable activity at pH 6.5 or above (PubMed:22407317, PubMed:23589309). The Glu analog L-trans-pyrrolidine-2,4-dicarboxylic acid (L-PDC) blocks the uptake of glutamate by selective inhibition (PubMed:12855178)'},\n",
       "  'seq': 'MATSVQTGKAKQLTLLGFFAITASMVMAVYEYPTFATSGFSLVFFLLLGGILWFIPVGLCAAEMATVDGWEEGGVFAWVSNTLGPRWGFAAISFGYLQIAIGFIPMLYFVLGALSYILKWPALNEDPITKTIAALIILWALALTQFGGTKYTARIAKVGFFAGILLPAFILIALAAIYLHSGAPVAIEMDSKTFFPDFSKVGTLVVFVAFILSYMGVEASATHVNEMSNPGRDYPLAMLLLMVAAICLSSVGGLSIAMVIPGNEINLSAGVMQTFTVLMSHVAPEIEWTVRVISALLLLGVLAEIASWIVGPSRGMYVTAQKNLLPAAFAKMNKNGVPVTLVISQLVITSIALIILTNTGGGNNMSFLIALALTVVIYLCAYFMLFIGYIVLVLKHPDLKRTFNIPGGKGVKLVVAIVGLLTSIMAFIVSFLPPDNIQGDSTDMYVELLVVSFLVVLALPFILYAVHDRKGKANTGVTLEPINSQNAPKGHFFLHPRARSPHYIVMNDKKH',\n",
       "  'mw': 55077.0},\n",
       " 'b1493': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P69910',\n",
       "  'gene': 'b1493',\n",
       "  'locus': ['b1493', 'JW1488'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('H(+) + L-glutamate = 4-aminobutanoate + CO2',\n",
       "     '4.1.1.15')],\n",
       "   'Function': 'Converts glutamate to gamma-aminobutyrate (GABA), consuming one intracellular proton in the reaction. The gad system helps to maintain a near-neutral intracellular pH when cells are exposed to extremely acidic conditions. The ability to survive transit through the acidic conditions of the stomach is essential for successful colonization of the mammalian host by commensal and pathogenic bacteria'},\n",
       "  'seq': 'MDKKQVTDLRSELLDSRFGAKSISTIAESKRFPLHEMRDDVAFQIINDELYLDGNARQNLATFCQTWDDENVHKLMDLSINKNWIDKEEYPQSAAIDLRCVNMVADLWHAPAPKNGQAVGTNTIGSSEACMLGGMAMKWRWRKRMEAAGKPTDKPNLVCGPVQICWHKFARYWDVELREIPMRPGQLFMDPKRMIEACDENTIGVVPTFGVTYTGNYEFPQPLHDALDKFQADTGIDIDMHIDAASGGFLAPFVAPDIVWDFRLPRVKSISASGHKFGLAPLGCGWVIWRDEEALPQELVFNVDYLGGQIGTFAINFSRPAGQVIAQYYEFLRLGREGYTKVQNASYQVAAYLADEIAKLGPYEFICTGRPDEGIPAVCFKLKDGEDPGYTLYDLSERLRLRGWQVPAFTLGGEATDIVVMRIMCRRGFEMDFAELLLEDYKASLKYLSDHPKLQGIAQQNSFKHT',\n",
       "  'mw': 52668.0},\n",
       " 'b1519': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P76145',\n",
       "  'gene': 'b1519',\n",
       "  'locus': ['b1519', 'JW1512'],\n",
       "  'synonyms': 'yneD',\n",
       "  'properties': {'Catalytic Activity': [('S-adenosyl-L-methionine + trans-aconitate = (E)-3-(methoxycarbonyl)pent-2-enedioate + S-adenosyl-L-homocysteine',\n",
       "     '2.1.1.144')],\n",
       "   'Function': 'Catalyzes the S-adenosylmethionine monomethyl esterification of trans-aconitate at high affinity and of cis-aconitate, isocitrate, and citrate at lower velocities and affinities'},\n",
       "  'seq': 'MSDWNPSLYLHFSAERSRPAVELLARVPLENVEYVADLGCGPGNSTALLQQRWPAARITGIDSSPAMIAEARSALPDCQFVEADIRNWQPVQALDLIFANASLQWLPDHYELFPHLVSLLNPQGVLAVQMPDNWLEPTHVLMREVAWEQNYPDRGREPLAGVHAYYDILSEAGCEVDIWRTTYYHQMPSHQAIIDWVTATGLRPWLQDLTESEQQLFLKRYHQMLEEQYPLQENGQILLAFPRLFIVARRME',\n",
       "  'mw': 29006.0},\n",
       " 'b1521': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A6L7',\n",
       "  'gene': 'b1521',\n",
       "  'locus': ['b1521', 'JW1514'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('D-altronate + NAD(+) = H(+) + keto-D-tagaturonate + NADH',\n",
       "     '1.1.1.58')],\n",
       "   'Pathway': 'Carbohydrate metabolism; pentose and glucuronate interconversion'},\n",
       "  'seq': 'MKTLNRRDFPGAQYPERIIQFGEGNFLRAFVDWQIDLLNEHTDLNSGVVVVRPIETSFPPSLSTQDGLYTTIIRGLNEKGEAVSDARLIRSVNREISVYSEYDEFLKLAHNPEMRFVFSNTTEAGISYHAGDKFDDAPAVSYPAKLTRLLFERFSHFNGALDKGWIIIPCELIDYNGDALRELVLRYAQEWALPEAFIQWLDQANSFCSTLVDRIVTGYPRDEVAKLEEELGYHDGFLDTAEHFYLFVIQGPKSLATELRLDKYPLNVLIVDDIKPYKERKVAILNGAHTALVPVAFQAGLDTVGEAMNDAEICAFVEKAIYEEIIPVLDLPRDELESFASAVTGRFRNPYIKHQLLSIALNGMTKFRTRILPQLLAGQKANGTLPARLTFALAALIAFYRGERNGETYPVQDDAHWLERYQQLWSQHRDRVIGTQELVAIVLAEKDHWEQDLTQVPGLVEQVANDLDAILEKGMREAVRPLC',\n",
       "  'mw': 54808.0},\n",
       " 'b1524': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A6W0',\n",
       "  'gene': 'b1524',\n",
       "  'locus': ['b1524', 'JW1517'],\n",
       "  'synonyms': 'yneH',\n",
       "  'properties': {'Catalytic Activity': [('H2O + L-glutamine = L-glutamate + NH4(+)',\n",
       "     '3.5.1.2')]},\n",
       "  'seq': 'MAVAMDNAILENILRQVRPLIGQGKVADYIPALATVDGSRLGIAICTVDGQLFQAGDAQERFSIQSISKVLSLVVAMRHYSEEEIWQRVGKDPSGSPFNSLVQLEMEQGIPRNPFINAGALVVCDMLQGRLSAPRQRMLEVVRGLSGVSDISYDTVVARSEFEHSARNAAIAWLMKSFGNFHHDVTTVLQNYFHYCALKMSCVELARTFVFLANQGKAIHIDEPVVTPMQARQINALMATSGMYQNAGEFAWRVGLPAKSGVGGGIVAIVPHEMAIAVWSPELDDAGNSLAGIAVLEQLTKQLGRSVY',\n",
       "  'mw': 33516.0},\n",
       " 'b1528': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P31122',\n",
       "  'gene': 'b1528',\n",
       "  'locus': ['b1528', 'JW1521'],\n",
       "  'synonyms': 'ydeA',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Involved in the efflux of sugars. The physiological role may be the reduction of the intracellular concentration of toxic sugars or sugar metabolites. Transports L-arabinose and to a lesser extent IPTG. Seems to contribute to the control of the arabinose regulon'},\n",
       "  'seq': 'MTTNTVSRKVAWLRVVTLAVAAFIFNTTEFVPVGLLSDIAQSFHMQTAQVGIMLTIYAWVVALMSLPFMLMTSQVERRKLLICLFVVFIASHVLSFLSWSFTVLVISRIGVAFAHAIFWSITASLAIRMAPAGKRAQALSLIATGTALAMVLGLPLGRIVGQYFGWRMTFFAIGIGALITLLCLIKLLPLLPSEHSGSLKSLPLLFRRPALMSIYLLTVVVVTAHYTAYSYIEPFVQNIAGFSANFATALLLLLGGAGIIGSVIFGKLGNQYASALVSTAIALLLVCLALLLPAANSEIHLGVLSIFWGIAMMIIGLGMQVKVLALAPDATDVAMALFSGIFNIGIGAGALVGNQVSLHWSMSMIGYVGAVPAFAALIWSIIIFRRWPVTLEEQTQ',\n",
       "  'mw': 42538.0},\n",
       " 'b1533': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P31125',\n",
       "  'gene': 'b1533',\n",
       "  'locus': ['b1533', 'JW5250'],\n",
       "  'synonyms': 'ydeD',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'May be an export pump for cysteine and other metabolites of the cysteine pathway (such as N-acetyl-L-serine (NAS) and O-acetyl-L-serine (OAS)), and for other amino acids and their metabolites'},\n",
       "  'seq': 'MSRKDGVLALLVVVVWGLNFVVIKVGLHNMPPLMLAGLRFMLVAFPAIFFVARPKVPLNLLLGYGLTISFAQFAFLFCAINFGMPAGLASLVLQAQAFFTIMLGAFTFGERLHGKQLAGIALAIFGVLVLIEDSLNGQHVAMLGFMLTLAAAFSWACGNIFNKKIMSHSTRPAVMSLVIWSALIPIIPFFVASLILDGSATMIHSLVTIDMTTILSLMYLAFVATIVGYGIWGTLLGRYETWRVAPLSLLVPVVGLASAALLLDERLTGLQFLGAVLIMTGLYINVFGLRWRKAVKVGS',\n",
       "  'mw': 32324.0},\n",
       " 'b1539': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P39831',\n",
       "  'gene': 'b1539',\n",
       "  'locus': ['b1539', 'JW1532'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('3-hydroxypropanoate + NADP(+) = 3-oxopropanoate + H(+) + NADPH',\n",
       "     '1.1.1.298'),\n",
       "    ('L-allo-threonine + NADP(+) = aminoacetone + CO2 + NADPH', '1.1.1.381')],\n",
       "   'Function': 'NADP-dependent dehydrogenase with broad substrate specificity acting on 3-hydroxy acids. Catalyzes the NADP-dependent oxidation of L-allo-threonine to L-2-amino-3-keto-butyrate, which is spontaneously decarboxylated into aminoacetone. Also acts on D-threonine, L-serine, D-serine, D-3-hydroxyisobutyrate, L-3-hydroxyisobutyrate, D-glycerate and L-glycerate (PubMed:12535615). Able to catalyze the reduction of the malonic semialdehyde to 3-hydroxypropionic acid. YdfG is apparently supplementing RutE, the presumed malonic semialdehyde reductase involved in pyrimidine degradation since both are able to detoxify malonic semialdehyde (PubMed:20400551)'},\n",
       "  'seq': 'MIVLVTGATAGFGECITRRFIQQGHKVIATGRRQERLQELKDELGDNLYIAQLDVRNRAAIEEMLASLPAEWCNIDILVNNAGLALGMEPAHKASVEDWETMIDTNNKGLVYMTRAVLPGMVERNHGHIINIGSTAGSWPYAGGNVYGATKAFVRQFSLNLRTDLHGTAVRVTDIEPGLVGGTEFSNVRFKGDDGKAEKTYQNTVALTPEDVSEAVWWVSTLPAHVNINTLEMMPVTQSYAGLNVHRQ',\n",
       "  'mw': 27249.0},\n",
       " 'b1584': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A951',\n",
       "  'gene': 'b1584',\n",
       "  'locus': ['b1584', 'JW1576'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('acetyl-CoA + an alkane-alpha,omega-diamine = an N-acetylalkane-alpha,omega-diamine + CoA + H(+)',\n",
       "     '2.3.1.57'),\n",
       "    ('acetyl-CoA + spermidine = CoA + H(+) + N(1)-acetylspermidine',\n",
       "     '2.3.1.57'),\n",
       "    ('acetyl-CoA + spermidine = CoA + H(+) + N(8)-acetylspermidine',\n",
       "     '2.3.1.57'),\n",
       "    ('acetyl-CoA + spermine = CoA + H(+) + N(1)-acetylspermine', '2.3.1.57')],\n",
       "   'Function': 'In addition, may act as a modulator of transcription through its ability to interact with the two-component response regulator RcsB. Inhibits transcription of the small RNA regulator rprA in an acetylation-independent manner. Interaction with the DNA-binding domain of RcsB might be responsible for SpeG-dependent inhibition of RcsB-dependent rprA transcription (PubMed:30562360). SpeG does not acetylate RcsB in vitro (PubMed:30562360)',\n",
       "   'Regulation Activity': 'Heat shock, alkaline shift, ethanol treatment and poor nutrient availability produce increased concentrations of monoacetylated spermidine (PubMed:2061318). Inhibited by alkaline phosphatase, N(1),N(8)-bis(ethyl)spermidine (BESPD) and N(1),N(12)-bis(ethyl)spermidine (BESPM) (PubMed:8077207, PubMed:6297970)',\n",
       "   'Pathway': 'Amine and polyamine degradation; spermine degradation'},\n",
       "  'seq': 'MPSAHSVKLRPLEREDLRYVHQLDNNASVMRYWFEEPYEAFVELSDLYDKHIHDQSERRFVVECDGEKAGLVELVEINHVHRRAEFQIIISPEYQGKGLATRAAKLAMDYGFTVLNLYKLYLIVDKENEKAIHIYRKLGFSVEGELMHEFFINGQYRNAIRMCIFQHQYLAEHKTPGQTLLKPTAQ',\n",
       "  'mw': 21887.0},\n",
       " 'b1587': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77374',\n",
       "  'gene': 'b1587',\n",
       "  'locus': ['b1587', 'JW1579'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Terminal reductase during anaerobic growth on various sulfoxide and N-oxide compounds'},\n",
       "  'seq': 'MSKNERMVGISRRTLVKSTAIGSLALAAGGFSLPFTLRNAAAAVQQAREKVVWGACSVNCGSRCALRLHVKDNEVTWVETDNTGSDEYGNHQVRACLRGRSIRRRINHPDRLNYPMKRVGKRGEGKFERISWDEALDTIASSLKKTVEQYGNEAVYIQYSSGIVGGNMTRSSPSASAVKRLMNCYGGSLNQYGSYSTAQISCAMPYTYGSNDGNSTTDIENSKLVVMFGNNPAETRMSGGGITYLLEKAREKSNAKMIVIDPRYTDTAAGREDEWLPIRPGTDAALVAGIAWVLINENLVDQPFLDKYCVGYDEKTLPADAPKNGHYKAYILGEGDDKTAKTPQWASQITGIPEDRIIKLAREIGTAKPAYICQGWGPQRQANGELTARAIAMLPILTGNVGISGGNSGARESTYTITIERLPVLDNPVKTSISCFSWTDAIDHGPQMTAIRDGVRGKDKLDVPIKFIWNYAGNTLVNQHSDINKTHEILQDESKCEMIVVIENFMTSSAKYADILLPDLMTVEQEDIIPNDYAGNMGYLIFLQPVTSEKFERKPIYWILSEVAKRLGPDVYQKFTEGRTQEQWLQHLYAKMLAKDPALPSYDELKKMGIYKRKDPNGHFVAYKAFRDDPEANPLKTPSGKIEIYSSRLAEIARTWELEKDEVISPLPVYASTFEGWNSPERRTFPLQLFGFHYKSRTHSTYGNIDLLKAACRQEVWINPIDAQKRGIANGDMVRVFNHRGEVRLPAKVTPRILPGVSAMGQGAWHEANMSGDKIDHGGCVNTLTTLRPSPLAKGNPQHTNLVEIEKI',\n",
       "  'mw': 89780.0},\n",
       " 'b1588': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77783',\n",
       "  'gene': 'b1588',\n",
       "  'locus': ['b1588', 'JW5260'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Terminal reductase during anaerobic growth on various sulfoxide and N-oxide compounds'},\n",
       "  'seq': 'MKIHTTEALMKAEISRRSLMKTSALGSLALASSAFTLPFSQMVRAAEAPVEEKAVWSSCTVNCGSRCLLRLHVKDDTVYWVESDTTGDDVYGNHQVRACLRGRSIRRRMNHPDRLKYPMKRVGKRGEGKFERISWDEALDTISDNLRRILKDYGNEAVHVLYGTGVDGGNITNSNVPYRLMNSCGGFLSRYGSYSTAQISAAMSYMFGANDGNSPDDIANTKLVVMFGNNPAETRMSGGGVTYYVEQARERSNARMIVIDPRYNDTAAGREDEWLPIRPGTDGALACAIAWVLITENMVDQPFLDKYCVGYDEKTLPANAPRNAHYKAYILGEGPDGIAKTPEWAAKITSIPAEKIIQLAREIGSAKPAYICQGWGPQRHSNGEQTSRAIAMLSVLTGNVGINGGNSGVREGSWDLGVEWFPMLENPVKTQISVFTWTDAIDHGTEMTATRDGVRGKEKLDVPIKFLWCYASNTLINQHGDINHTHEVLQDDSKCEMIVGIDHFMTASAKYCDILLPDLMPTEQEDLISHESAGNMGYVILAQPATSAKFERKPIYWMLSEVAKRLGPDVYQTFTEGRSQHEWIKYLHAKTKERNPEMPDYEEMKTTGIFKKKCPEEHYVAFRAFREDPQANPLKTPSGKIEIYSERLAKIADTWELKKDEIIHPLPAYTPGFDGWDDPLRKTYPLQLTGFHYKARTHSSYGNIDVLQQACPQEVWINPIDAQARGIRHGDTVRVFNNNGEMLIAAKVTPRILPGVTAIGQGAWLKADMFGDRVDHGGSINILTSHRPSPLAKGNPSHSNLVQIEKV',\n",
       "  'mw': 89987.0},\n",
       " 'b1589': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AAJ1',\n",
       "  'gene': 'b1589',\n",
       "  'locus': ['b1589', 'JW1581'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Electron transfer subunit of the terminal reductase during anaerobic growth on various sulfoxide and N-oxide compounds'},\n",
       "  'seq': 'MTTQYGFFIDSSRCTGCKTCELACKDFKDLGPEVSFRRIYEYAGGDWQEDNGVWHQNVFAYYLSISCNHCDDPACTKVCPSGAMHKREDGFVVVDEDVCIGCRYCHMACPYGAPQYNAEKGHMTKCDGCYSRVAEGKQPICVESCPLRALEFGPIEELRQKHGTLAAVAPLPRAHFTKPNIVIKPNANSRPTGDTTGYLANPEEV',\n",
       "  'mw': 22752.0},\n",
       " 'b1590': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P76173',\n",
       "  'gene': 'b1590',\n",
       "  'locus': ['b1590', 'JW5261'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Terminal reductase during anaerobic growth on various sulfoxide and N-oxide compounds. The C subunit anchors the other two subunits to the membrane and stabilize the catalytic subunits (By similarity)'},\n",
       "  'seq': 'MGNGWHEWPLVIFTVLGQCVVGALIVSGIGWFAAKNDADRQRIVRGMFFLWLLMGVGFIASVMHLGSPLRAFNSLNRIGASGLSNEIAAGSIFFAVGGLWWLVAVIGKMPQALGKLWLLFSMALGVIFVWMMTCVYQIDTVPTWHNGYTTLAFFLTVLLSGPILAAAILRAARVTFNTTPFAIISVLALIACAGVIVLQGLSLASIHSSVQQASALVPDYASLQVWRVVLLCAGLGCWLCPLIRRREPHVAGLILGLILILGGEMIGRVLFYGLHMTVGMAIAG',\n",
       "  'mw': 30524.0},\n",
       " 'b1592': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P76175',\n",
       "  'gene': 'b1592',\n",
       "  'locus': ['b1592', 'JW5263'],\n",
       "  'synonyms': 'mriT ynfJ',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Probably acts as an electrical shunt for an outwardly-directed proton pump that is linked to amino acid decarboxylation, as part of the extreme acid resistance (XAR) response'},\n",
       "  'seq': 'MFHRLLIATVVGILAAFAVAGFRHAMLLLEWLFLNNDSGSLVNAATNLSPWRRLLTPALGGLAAGLLLMGWQKFTQQRPHAPTDYMEALQTDGQFDYAASLVKSLASLLVVTSGSAIGREGAMILLAALAASCFAQRFTPRQEWKLWIACGAAAGMAAAYRAPLAGSLFIAEVLFGTMMLASLGPVIISAVVALLVSNLINHSDALLYNVQLSVTVQARDYALIISTGVLAGLCGPLLLTLMNACHRGFVSLKLAPPWQLALGGLIVGLLSLFTPAVWGNGYSTVQSFLTAPPLLMIIAGIFLCKLCAVLASSGSGAPGGVFTPTLFIGLAIGMLYGRSLGLWFPDGEEITLLLGLTGMATLLAATTHAPIMSTLMICEMTGEYQLLPGLLIACVIASVISRTLHRDSIYRQHTAQHS',\n",
       "  'mw': 44153.0},\n",
       " 'b1602': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AB67',\n",
       "  'gene': 'b1602',\n",
       "  'locus': ['b1602', 'JW1594'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('H(+)(in) + NAD(+) + NADPH = H(+)(out) + NADH + NADP(+)',\n",
       "     '7.1.1.1')],\n",
       "   'Function': 'The transhydrogenation between NADH and NADP is coupled to respiration and ATP hydrolysis and functions as a proton pump across the membrane'},\n",
       "  'seq': 'MSGGLVTAAYIVAAILFIFSLAGLSKHETSRQGNNFGIAGMAIALIATIFGPDTGNVGWILLAMVIGGAIGIRLAKKVEMTEMPELVAILHSFVGLAAVLVGFNSYLHHDAGMAPILVNIHLTEVFLGIFIGAVTFTGSVVAFGKLCGKISSKPLMLPNRHKMNLAALVVSFLLLIVFVRTDSVGLQVLALLIMTAIALVFGWHLVASIGGADMPVVVSMLNSYSGWAAAAAGFMLSNDLLIVTGALVGSSGAILSYIMCKAMNRSFISVIAGGFGTDGSSTGDDQEVGEHREITAEETAELLKNSHSVIITPGYGMAVAQAQYPVAEITEKLRARGINVRFGIHPVAGRLPGHMNVLLAEAKVPYDIVLEMDEINDDFADTDTVLVIGANDTVNPAAQDDPKSPIAGMPVLEVWKAQNVIVFKRSMNTGYAGVQNPLFFKENTHMLFGDAKASVDAILKAL',\n",
       "  'mw': 48723.0},\n",
       " 'b1603': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P07001',\n",
       "  'gene': 'b1603',\n",
       "  'locus': ['b1603', 'JW1595'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('H(+)(in) + NAD(+) + NADPH = H(+)(out) + NADH + NADP(+)',\n",
       "     '7.1.1.1')],\n",
       "   'Function': 'The transhydrogenation between NADH and NADP is coupled to respiration and ATP hydrolysis and functions as a proton pump across the membrane'},\n",
       "  'seq': 'MRIGIPRERLTNETRVAATPKTVEQLLKLGFTVAVESGAGQLASFDDKAFVQAGAEIVEGNSVWQSEIILKVNAPLDDEIALLNPGTTLVSFIWPAQNPELMQKLAERNVTVMAMDSVPRISRAQSLDALSSMANIAGYRAIVEAAHEFGRFFTGQITAAGKVPPAKVMVIGAGVAGLAAIGAANSLGAIVRAFDTRPEVKEQVQSMGAEFLELDFKEEAGSGDGYAKVMSDAFIKAEMELFAAQAKEVDIIVTTALIPGKPAPKLITREMVDSMKAGSVIVDLAAQNGGNCEYTVPGEIFTTENGVKVIGYTDLPGRLPTQSSQLYGTNLVNLLKLLCKEKDGNITVDFDDVVIRGVTVIRAGEITWPAPPIQVSAQPQAAQKAAPEVKTEEKCTCSPWRKYALMALAIILFGWMASVAPKEFLGHFTVFALACVVGYYVVWNVSHALHTPLMSVTNAISGIIVVGALLQIGQGGWVSFLSFIAVLIASINIFGGFTVTQRMLKMFRKN',\n",
       "  'mw': 54623.0},\n",
       " 'b1605': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AAE5',\n",
       "  'gene': 'b1605',\n",
       "  'locus': ['b1605', 'JW1597'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('L-arginine(out) + L-ornithine(in) = L-arginine(in) + L-ornithine(out)',\n",
       "     '')],\n",
       "   'Function': 'Catalyzes electroneutral exchange between arginine and ornithine to allow high-efficiency energy conversion in the arginine deiminase pathway'},\n",
       "  'seq': 'MEKKLGLSALTALVLSSMLGAGVFSLPQNMAAVASPAALLIGWGITGAGILLLAFAMLILTRIRPELDGGIFTYAREGFGELIGFCSAWGYWLCAVIANVSYLVIVFSALSFFTDTPELRLFGDGNTWQSIVGASALLWIVHFLILRGVQTAASINLVATLAKLLPLGLFVVLAMMMFKLDTFKLDFTGLALGVPVWEQVKNTMLITLWVFIGVEGAVVVSARARNKRDVGKATLLAVLSALGVYLLVTLLSLGVVARPELAEIRNPSMAGLMVEMMGPWGEIIIAAGLIVSVCGAYLSWTIMAAEVPFLAATHKAFPRIFARQNAQAAPSASLWLTNICVQICLVLIWLTGSDYNTLLTIASEMILVPYFLVGAFLLKIATRPLHKAVGVGACIYGLWLLYASGPMHLLLSVVLYAPGLLVFLYARKTHTHDNVLNRQEMVLIGMLLIASVPATWMLVG',\n",
       "  'mw': 49501.0},\n",
       " 'b1606': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AFS3',\n",
       "  'gene': 'b1606',\n",
       "  'locus': ['b1606', 'JW1598'],\n",
       "  'synonyms': 'ydgB',\n",
       "  'properties': {'Catalytic Activity': [('(6S)-5,6,7,8-tetrahydrofolate + NADP(+) = 7,8-dihydrofolate + H(+) + NADPH',\n",
       "     '1.5.1.3'),\n",
       "    ('7,8-dihydromonapterin + H(+) + NADPH = 5,6,7,8-tetrahydromonapterin + NADP(+)',\n",
       "     '1.5.1.50')],\n",
       "   'Function': 'Catalyzes the reduction of dihydromonapterin to tetrahydromonapterin. Also has lower activity with dihydrofolate',\n",
       "   'Regulation Activity': 'Inhibited by methotrexate'},\n",
       "  'seq': 'MGKTQPLPILITGGGRRIGLALAWHFINQKQPVIVSYRTHYPAIDGLINAGAQCIQADFSTNDGVMAFADEVLKSTHGLRAILHNASAWMAEKPGAPLADVLACMMQIHVNTPYLLNHALERLLRGHGHAASDIIHFTDYVVERGSDKHIAYAASKAALDNMTRSFARKLAPEVKVNSIAPSLILFNEHDDAEYRQQALNKSLMKTAPGEKEVIDLVDYLLTSCFVTGRSFPLDGGRHLR',\n",
       "  'mw': 26348.0},\n",
       " 'b1611': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P05042',\n",
       "  'gene': 'b1611',\n",
       "  'locus': ['b1611', 'JW1603'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(S)-malate = fumarate + H2O',\n",
       "     '4.2.1.2')],\n",
       "   'Function': 'Involved in the TCA cycle. FumC seems to be a backup enzyme for FumA under conditions of iron limitation and oxidative stress (PubMed:7592392). Catalyzes the stereospecific interconversion of fumarate to L-malate (PubMed:1917897, PubMed:3282546)',\n",
       "   'Regulation Activity': 'Inhibited by ATP, citrate and S-2,3-dicarboxyaziridine',\n",
       "   'Pathway': 'Carbohydrate metabolism; tricarboxylic acid cycle; (S)-malate from fumarate: step 1/1'},\n",
       "  'seq': 'MNTVRSEKDSMGAIDVPADKLWGAQTQRSLEHFRISTEKMPTSLIHALALTKRAAAKVNEDLGLLSEEKASAIRQAADEVLAGQHDDEFPLAIWQTGSGTQSNMNMNEVLANRASELLGGVRGMERKVHPNDDVNKSQSSNDVFPTAMHVAALLALRKQLIPQLKTLTQTLNEKSRAFADIVKIGRTHLQDATPLTLGQEISGWVAMLEHNLKHIEYSLPHVAELALGGTAVGTGLNTHPEYARRVADELAVITCAPFVTAPNKFEALATCDALVQAHGALKGLAASLMKIANDVRWLASGPRCGIGEISIPENEPGSSIMPGKVNPTQCEALTMLCCQVMGNDVAINMGGASGNFELNVFRPMVIHNFLQSVRLLADGMESFNKHCAVGIEPNRERINQLLNESLMLVTALNTHIGYDKAAEIAKKAHKEGLTLKAAALALGYLSEAEFDSWVRPEQMVGSMKAGR',\n",
       "  'mw': 50489.0},\n",
       " 'b1612': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AC33',\n",
       "  'gene': 'b1612',\n",
       "  'locus': ['b1612', 'JW1604'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(S)-malate = fumarate + H2O',\n",
       "     '4.2.1.2'),\n",
       "    ('oxaloacetate = enol-oxaloacetate', '5.3.2.2')],\n",
       "   'Function': 'Catalyzes the reversible hydration of fumarate to (S)-malate. Functions as an aerobic enzyme in the direction of malate formation as part of the citric acid cycle. Accounts for about 80% of the fumarase activity when the bacteria grow aerobically. To a lesser extent, also displays D-tartrate dehydratase activity in vitro, but is not able to convert (R)-malate, L-tartrate or meso-tartrate. Can also catalyze the isomerization of enol- to keto-oxaloacetate',\n",
       "   'Regulation Activity': 'Is inactivated by oxygen, due to oxidation of the Fe-S cluster to its [3Fe-4S] form. Both the fumarase and the isomerase reactions are competitively inhibited by 3-hydroxy-2-nitropropionate. The isomerase reaction is also inhibited by fumarate and malate',\n",
       "   'Pathway': 'Carbohydrate metabolism; tricarboxylic acid cycle; (S)-malate from fumarate: step 1/1'},\n",
       "  'seq': 'MSNKPFHYQAPFPLKKDDTEYYLLTSEHVSVSEFEGQEILKVAPEALTLLARQAFHDASFMLRPAHQQQVADILRDPEASENDKYVALQFLRNSDIAAKGVLPTCQDTGTAIIVGKKGQRVWTGGGDEAALARGVYNTYIEDNLRYSQNAPLDMYKEVNTGTNLPAQIDLYAVDGDEYKFLCIAKGGGSANKTYLYQETKALLTPGKLKNYLVEKMRTLGTAACPPYHIAFVIGGTSAETNLKTVKLASAKYYDELPTEGNEHGQAFRDVELEKELLIEAQNLGLGAQFGGKYFAHDIRVIRLPRHGASCPVGMGVSCSADRNIKAKINRQGIWIEKLEHNPGKYIPEELRKAGEGEAVRVDLNRPMKEILAQLSQYPVSTRLSLNGTIIVGRDIAHAKLKERMDNGEGLPQYIKDHPIYYAGPAKTPEGYASGSLGPTTAGRMDSYVDQLQAQGGSMIMLAKGNRSQQVTDACKKHGGFYLGSIGGPAAVLAQGSIKSLECVEYPELGMEAIWKIEVEDFPAFILVDDKGNDFFQQIQLTQCTRCVK',\n",
       "  'mw': 60299.0},\n",
       " 'b1613': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P00946',\n",
       "  'gene': 'b1613',\n",
       "  'locus': ['b1613', 'JW1605'],\n",
       "  'synonyms': 'pmi',\n",
       "  'properties': {'Catalytic Activity': [('D-mannose 6-phosphate = D-fructose 6-phosphate',\n",
       "     '5.3.1.8')],\n",
       "   'Function': 'Involved in the conversion of glucose to GDP-L-fucose, which can be converted to L-fucose, a capsular polysaccharide'},\n",
       "  'seq': 'MQKLINSVQNYAWGSKTALTELYGMENPSSQPMAELWMGAHPKSSSRVQNAAGDIVSLRDVIESDKSTLLGEAVAKRFGELPFLFKVLCAAQPLSIQVHPNKHNSEIGFAKENAAGIPMDAAERNYKDPNHKPELVFALTPFLAMNAFREFSEIVSLLQPVAGAHPAIAHFLQQPDAERLSELFASLLNMQGEEKSRALAILKSALDSQQGEPWQTIRLISEFYPEDSGLFSPLLLNVVKLNPGEAMFLFAETPHAYLQGVALEVMANSDNVLRAGLTPKYIDIPELVANVKFEAKPANQLLTQPVKQGAELDFPIPVDDFAFSLHDLSDKETTISQQSAAILFCVEGDATLWKGSQQLQLKPGESAFIAANESPVTVKGHGRLARVYNKL',\n",
       "  'mw': 42850.0},\n",
       " 'b1621': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P19642',\n",
       "  'gene': 'b1621',\n",
       "  'locus': ['b1621', 'JW1613'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [(\"D-maltose(out) + N(pros)-phospho-L-histidyl-[protein] = alpha-maltose 6'-phosphate(in) + L-histidyl-[protein]\",\n",
       "     '2.7.1.208')],\n",
       "   'Function': 'The phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS), a major carbohydrate active transport system, catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane. This system is involved in maltose transport. MalX can also recognize and transport glucose even though this sugar may not represent the natural substrate of the system'},\n",
       "  'seq': 'MTAKTAPKVTLWEFFQQLGKTFMLPVALLSFCGIMLGIGSSLSSHDVITLIPVLGNPVLQAIFTWMSKIGSFAFSFLPVMFCIAIPLGLARENKGVAAFAGFIGYAVMNLAVNFWLTNKGILPTTDAAVLKANNIQSILGIQSIDTGILGAVIAGIIVWMLHERFHNIRLPDALAFFGGTRFVPIISSLVMGLVGLVIPLVWPIFAMGISGLGHMINSAGDFGPMLFGTGERLLLPFGLHHILVALIRFTDAGGTQEVCGQTVSGALTIFQAQLSCPTTHGFSESATRFLSQGKMPAFLGGLPGAALAMYHCARPENRHKIKGLLISGLIACVVGGTTEPLEFLFLFVAPVLYVIHALLTGLGFTVMSVLGVTIGNTDGNIIDFVVFGILHGLSTKWYMVPVVAAIWFVVYYVIFRFAITRFNLKTPGRDSEVASSIEKAVAGAPGKSGYNVPAILEALGGADNIVSLDNCITRLRLSVKDMSLVNVQALKDNRAIGVVQLNQHNLQVVIGPQVQSVKDEMAGLMHTVQA',\n",
       "  'mw': 56627.0},\n",
       " 'b1622': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P23256',\n",
       "  'gene': 'b1622',\n",
       "  'locus': ['b1622', 'JW1614'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('H2O + L,L-cystathionine = L-homocysteine + NH4(+) + pyruvate',\n",
       "     ''),\n",
       "    ('an S-substituted L-cysteine + H2O = a thiol + NH4(+) + pyruvate',\n",
       "     '4.4.1.13')],\n",
       "   'Function': 'Acts as a beta-cystathionase and as a repressor of the maltose regulon'},\n",
       "  'seq': 'MFDFSKVVDRHGTWCTQWDYVADRFGTADLLPFTISDMDFATAPCIIEALNQRLMHGVFGYSRWKNDEFLAAIAHWFSTQHYTAIDSQTVVYGPSVIYMVSELIRQWSETGEGVVIHTPAYDAFYKAIEGNQRTVMPVALEKQADGWFCDMGKLEAVLAKPECKIMLLCSPQNPTGKVWTCDELEIMADLCERHGVRVISDEIHMDMVWGEQPHIPWSNVARGDWALLTSGSKSFNIPALTGAYGIIENSSSRDAYLSALKGRDGLSSPSVLALTAHIAAYQQGAPWLDALRIYLKDNLTYIADKMNAAFPELNWQIPQSTYLAWLDLRPLNIDDNALQKALIEQEKVAIMPGYTYGEEGRGFVRLNAGCPRSKLEKGVAGLINAIRAVR',\n",
       "  'mw': 43642.0},\n",
       " 'b1623': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P22333',\n",
       "  'gene': 'b1623',\n",
       "  'locus': ['b1623', 'JW1615'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('adenosine + H(+) + H2O = inosine + NH4(+)',\n",
       "     '3.5.4.4'),\n",
       "    (\"2'-deoxyadenosine + H(+) + H2O = 2'-deoxyinosine + NH4(+)\", '3.5.4.4')],\n",
       "   'Function': 'Catalyzes the hydrolytic deamination of adenosine and 2-deoxyadenosine',\n",
       "   'Regulation Activity': 'Inhibited by reagents such as p-chloromercuribenzoate, and by coformycin, a structural analog of inosine'},\n",
       "  'seq': 'MIDTTLPLTDIHRHLDGNIRPQTILELGRQYNISLPAQSLETLIPHVQVIANEPDLVSFLTKLDWGVKVLASLDACRRVAFENIEDAARHGLHYVELRFSPGYMAMAHQLPVAGVVEAVIDGVREGCRTFGVQAKLIGIMSRTFGEAACQQELEAFLAHRDQITALDLAGDELGFPGSLFLSHFNRARDAGWHITVHAGEAAGPESIWQAIRELGAERIGHGVKAIEDRALMDFLAEQQIGIESCLTSNIQTSTVAELAAHPLKTFLEHGIRASINTDDPGVQGVDIIHEYTVAAPAAGLSREQIRQAQINGLEMAFLSAEEKRALREKVAAK',\n",
       "  'mw': 36397.0},\n",
       " 'b1636': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77150',\n",
       "  'gene': 'b1636',\n",
       "  'locus': ['b1636', 'JW1628'],\n",
       "  'synonyms': 'ydgS',\n",
       "  'properties': {'Catalytic Activity': [(\"ATP + pyridoxal = ADP + H(+) + pyridoxal 5'-phosphate\",\n",
       "     '2.7.1.35')],\n",
       "   'Function': \"Pyridoxal kinase involved in the salvage pathway of pyridoxal 5'-phosphate (PLP). Catalyzes the phosphorylation of pyridoxal to PLP in vivo, but shows very low activity compared to PdxK. Displays a low level of pyridoxine kinase activity when overexpressed, which is however not physiologically relevant\",\n",
       "   'Pathway': \"Cofactor metabolism; pyridoxal 5'-phosphate salvage; pyridoxal 5'-phosphate from pyridoxal: step 1/1\"},\n",
       "  'seq': 'MMKNILAIQSHVVYGHAGNSAAEFPMRRLGANVWPLNTVQFSNHTQYGKWTGCVMPPSHLTEIVQGIAAIDKLHTCDAVLSGYLGSAEQGEHILGIVRQVKAANPQAKYFCDPVMGHPEKGCIVAPGVAEFHVRHGLPASDIIAPNLVELEILCEHAVNNVEEAVLAARELIAQGPQIVLVKHLARAGYSRDRFEMLLVTADEAWHISRPLVDFGMRQPVGVGDVTSGLLLVKLLQGATLQEALEHVTAAVYEIMVTTKAMQEYELQVVAAQDRIAKPEHYFSATKL',\n",
       "  'mw': 31322.0},\n",
       " 'b1637': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AGJ9',\n",
       "  'gene': 'b1637',\n",
       "  'locus': ['b1637', 'JW1629'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + L-tyrosine + tRNA(Tyr) = AMP + diphosphate + H(+) + L-tyrosyl-tRNA(Tyr)',\n",
       "     '6.1.1.1'),\n",
       "    ('ATP + D-tyrosine + tRNA(Tyr) = AMP + D-tyrosyl-tRNA(Tyr) + diphosphate + H(+)',\n",
       "     '')],\n",
       "   'Function': 'Catalyzes the attachment of L-tyrosine to tRNA(Tyr) in a two-step reaction: tyrosine is first activated by ATP to form Tyr-AMP and then transferred to the acceptor end of tRNA(Tyr) (PubMed:4292198, PubMed:4579631). Can also mischarge tRNA(Tyr) with D-tyrosine, leading to the formation of D-tyrosyl-tRNA(Tyr), which can be hydrolyzed by the D-aminoacyl-tRNA deacylase (PubMed:4292198). In vitro, can also use the non-natural amino acid azatyrosine (PubMed:11006270)',\n",
       "   'Regulation Activity': 'Magnesium is essential for activity (PubMed:4579631). Inhibited by chloride and sulfate in the presence of 1 mM free Mg(2+). When the Mg(2+) concentration increases to 10 mM there is almost no chloride inhibition any more. Inhibited by diphosphate and AMP. Chloride strengthens the diphosphate inhibition and weakens the AMP inhibition. Chloride weakens the binding of Mg(2+) to the RNA and thereby the interaction between the enzyme and the RNA (PubMed:10572925). Acetylation at certain lysine residues could significantly impair activity (PubMed:28741290). D-tyrosine is a competitive inhibitor of L-tyrosine for the formation of tyrosyl-tRNA(Tyr) (PubMed:4292198)'},\n",
       "  'seq': 'MASSNLIKQLQERGLVAQVTDEEALAERLAQGPIALYCGFDPTADSLHLGHLVPLLCLKRFQQAGHKPVALVGGATGLIGDPSFKAAERKLNTEETVQEWVDKIRKQVAPFLDFDCGENSAIAANNYDWFGNMNVLTFLRDIGKHFSVNQMINKEAVKQRLNREDQGISFTEFSYNLLQGYDFACLNKQYGVVLQIGGSDQWGNITSGIDLTRRLHQNQVFGLTVPLITKADGTKFGKTEGGAVWLDPKKTSPYKFYQFWINTADADVYRFLKFFTFMSIEEINALEEEDKNSGKAPRAQYVLAEQVTRLVHGEEGLQAAKRITECLFSGSLSALSEADFEQLAQDGVPMVEMEKGADLMQALVDSELQPSRGQARKTIASNAITINGEKQSDPEYFFKEEDRLFGRFTLLRRGKKNYCLICWK',\n",
       "  'mw': 47527.0},\n",
       " 'b1638': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AFI7',\n",
       "  'gene': 'b1638',\n",
       "  'locus': ['b1638', 'JW1630'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [(\"H2O + O2 + pyridoxamine 5'-phosphate = H2O2 + NH4(+) + pyridoxal 5'-phosphate\",\n",
       "     '1.4.3.5'),\n",
       "    (\"O2 + pyridoxine 5'-phosphate = H2O2 + pyridoxal 5'-phosphate\",\n",
       "     '1.4.3.5')],\n",
       "   'Function': \"Catalyzes the oxidation of either pyridoxine 5'-phosphate (PNP) or pyridoxamine 5'-phosphate (PMP) into pyridoxal 5'-phosphate (PLP)\",\n",
       "   'Pathway': \"Cofactor metabolism; pyridoxal 5'-phosphate salvage; pyridoxal 5'-phosphate from pyridoxine 5'-phosphate: step 1/1\"},\n",
       "  'seq': 'MSDNDELQQIAHLRREYTKGGLRRRDLPADPLTLFERWLSQACEAKLADPTAMVVATVDEHGQPYQRIVLLKHYDEKGMVFYTNLGSRKAHQIENNPRVSLLFPWHTLERQVMVIGKAERLSTLEVMKYFHSRPRDSQIGAWVSKQSSRISARGILESKFLELKQKFQQGEVPLPSFWGGFRVSLEQIEFWQGGEHRLHDRFLYQRENDAWKIDRLAP',\n",
       "  'mw': 25545.0},\n",
       " 'b1640': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77570',\n",
       "  'gene': 'b1640',\n",
       "  'locus': ['b1640', 'JW1632'],\n",
       "  'synonyms': 'ydhH',\n",
       "  'properties': {'Catalytic Activity': [('1,6-anhydro-N-acetyl-beta-muramate + ATP + H2O = ADP + H(+) + N-acetyl-D-muramate 6-phosphate',\n",
       "     '2.7.1.170')],\n",
       "   'Function': 'Catalyzes the specific phosphorylation of 1,6-anhydro-N-acetylmuramic acid (anhMurNAc) with the simultaneous cleavage of the 1,6-anhydro ring, generating MurNAc-6-P. Is required for the utilization of anhMurNAc either imported from the medium or derived from its own cell wall murein, and thus plays a role in cell wall recycling',\n",
       "   'Regulation Activity': 'Strongly inhibited by ADP',\n",
       "   'Pathway': 'Cell wall biogenesis; peptidoglycan recycling'},\n",
       "  'seq': 'MKSGRFIGVMSGTSLDGVDVVLATIDEHRVAQLASLSWPIPVSLKQAVLDICQGQQLTLSQFGQLDTQLGQLFADAVNALLKEQNLQARDIVAIGCHGQTVWHEPTGVAPHTLQIGDNNQIVARTGITVVGDFRRRDIALGGQGAPLVPAFHHALLAHPTERRMVLNIGGIANLSLLIPGQPVGGYDTGPGNMLMDAWIWRQAGKPYDKDAEWARAGKVILPLLQNMLSDPYFSQPAPKSTGREYFNYGWLERHLRHFPGVDPRDVQATLAELTAVTISEQVLLSGGCERLMVCGGGSRNPLLMARLAALLPGTEVTTTDAVGISGDDMEALAFAWLAWRTLAGLPGNLPSVTGASQETVLGAIFPANP',\n",
       "  'mw': 39496.0},\n",
       " 'b1646': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AGD1',\n",
       "  'gene': 'b1646',\n",
       "  'locus': ['b1646', 'JW1638'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('2 H(+) + 2 superoxide = H2O2 + O2',\n",
       "     '1.15.1.1')],\n",
       "   'Function': 'Destroys radicals which are normally produced within the cells and which are toxic to biological systems',\n",
       "   'Regulation Activity': 'Inactivated by diethyldithiocarbamate (PubMed:7929223). Inhibited by cyanide (PubMed:7929223)'},\n",
       "  'seq': 'MKRFSLAILALVVATGAQAASEKVEMNLVTSQGVGQSIGSVTITETDKGLEFSPDLKALPPGEHGFHIHAKGSCQPATKDGKASAAESAGGHLDPQNTGKHEGPEGAGHLGDLPALVVNNDGKATDAVIAPRLKSLDEIKDKALMVHVGGDNMSDQPKPLGGGGERYACGVIK',\n",
       "  'mw': 17681.0},\n",
       " 'b1651': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AC81',\n",
       "  'gene': 'b1651',\n",
       "  'locus': ['b1651', 'JW1643'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(R)-S-lactoylglutathione = glutathione + methylglyoxal',\n",
       "     '4.4.1.5')],\n",
       "   'Function': 'Catalyzes the isomerization of the hemithioacetal formed spontaneously from methylglyoxal and glutathione, to S-lactoylglutathione, which is then hydrolyzed by a type II glyoxalase (GloB or GloC). Is involved in methylglyoxal (MG) detoxification (PubMed:10913283) (Probable). Involved in resistance to hypochlorous acid (HOCl), which is the active component of household bleach and a powerful antimicrobial during the innate immune response (PubMed:23536188)',\n",
       "   'Pathway': 'Secondary metabolite metabolism; methylglyoxal degradation; (R)-lactate from methylglyoxal: step 1/2'},\n",
       "  'seq': 'MRLLHTMLRVGDLQRSIDFYTKVLGMKLLRTSENPEYKYSLAFVGYGPETEEAVIELTYNWGVDKYELGTAYGHIALSVDNAAEACEKIRQNGGNVTREAGPVKGGTTVIAFVEDPDGYKIELIEEKDAGRGLGN',\n",
       "  'mw': 14920.0},\n",
       " 'b1654': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AC69',\n",
       "  'gene': 'b1654',\n",
       "  'locus': ['b1654', 'JW1646'],\n",
       "  'synonyms': 'ydhD',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Monothiol glutaredoxin involved in the biogenesis of iron-sulfur clusters'},\n",
       "  'seq': 'MSTTIEKIQRQIAENPILLYMKGSPKLPSCGFSAQAVQALAACGERFAYVDILQNPDIRAELPKYANWPTFPQLWVDGELVGGCDIVIEMYQRGELQQLIKETAAKYKSEEPDAE',\n",
       "  'mw': 12879.0},\n",
       " 'b1656': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AGD3',\n",
       "  'gene': 'b1656',\n",
       "  'locus': ['b1656', 'JW1648'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('2 H(+) + 2 superoxide = H2O2 + O2',\n",
       "     '1.15.1.1')],\n",
       "   'Function': 'Destroys superoxide anion radicals which are normally produced within the cells and which are toxic to biological systems'},\n",
       "  'seq': 'MSFELPALPYAKDALAPHISAETIEYHYGKHHQTYVTNLNNLIKGTAFEGKSLEEIIRSSEGGVFNNAAQVWNHTFYWNCLAPNAGGEPTGKVAEAIAASFGSFADFKAQFTDAAIKNFGSGWTWLVKNSDGKLAIVSTSNAGTPLTTDATPLLTVDVWEHAYYIDYRNARPGYLEHFWALVNWEFVAKNLAA',\n",
       "  'mw': 21266.0},\n",
       " 'b1661': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A9H7',\n",
       "  'gene': 'b1661',\n",
       "  'locus': ['b1661', 'JW1653'],\n",
       "  'synonyms': 'cdfA',\n",
       "  'properties': {'Catalytic Activity': [('1-acyl-2-(9Z)-enoyl-sn-glycero-3-phospholipid + S-adenosyl-L-methionine = 1-acyl-2-(9-cyclopronane)-acyl-sn-glycero-3-phospholipid + H(+) + S-adenosyl-L-homocysteine',\n",
       "     '2.1.1.79')],\n",
       "   'Function': 'Transfers a methylene group from S-adenosyl-L-methionine to the cis double bond of an unsaturated fatty acid chain resulting in the replacement of the double bond with a methylene bridge',\n",
       "   'Regulation Activity': 'Inhibited by sinefungin, A9145C and S-adenosyl-L-homocysteine',\n",
       "   'Pathway': 'Lipid metabolism; fatty acid biosynthesis'},\n",
       "  'seq': 'MSSSCIEEVSVPDDNWYRIANELLSRAGIAINGSAPADIRVKNPDFFKRVLQEGSLGLGESYMDGWWECDRLDMFFSKVLRAGLENQLPHHFKDTLRIAGARLFNLQSKKRAWIVGKEHYDLGNDLFSRMLDPFMQYSCAYWKDADNLESAQQAKLKMICEKLQLKPGMRVLDIGCGWGGLAHYMASNYDVSVVGVTISAEQQKMAQERCEGLDVTILLQDYRDLNDQFDRIVSVGMFEHVGPKNYDTYFAVVDRNLKPEGIFLLHTIGSKKTDLNVDPWINKYIFPNGCLPSVRQIAQSSEPHFVMEDWHNFGADYDTTLMAWYERFLAAWPEIADNYSERFKRMFTYYLNACAGAFRARDIQLWQVVFSRGVENGLRVAR',\n",
       "  'mw': 43909.0},\n",
       " 'b1662': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AFU8',\n",
       "  'gene': 'b1662',\n",
       "  'locus': ['b1662', 'JW1654'],\n",
       "  'synonyms': 'ribE',\n",
       "  'properties': {'Catalytic Activity': [('2 6,7-dimethyl-8-(1-D-ribityl)lumazine + H(+) = 5-amino-6-(D-ribitylamino)uracil + riboflavin',\n",
       "     '2.5.1.9')],\n",
       "   'Function': 'Catalyzes the dismutation of two molecules of 6,7-dimethyl-8-ribityllumazine, resulting in the formation of riboflavin and 5-amino-6-(D-ribitylamino)uracil',\n",
       "   'Pathway': 'Cofactor biosynthesis; riboflavin biosynthesis; riboflavin from 2-hydroxy-3-oxobutyl phosphate and 5-amino-6-(D-ribitylamino)uracil: step 2/2'},\n",
       "  'seq': 'MFTGIVQGTAKLVSIDEKPNFRTHVVELPDHMLDGLETGASVAHNGCCLTVTEINGNHVSFDLMKETLRITNLGDLKVGDWVNVERAAKFSDEIGGHLMSGHIMTTAEVAKILTSENNRQIWFKVQDSQLMKYILYKGFIGIDGISLTVGEVTPTRFCVHLIPETLERTTLGKKKLGARVNIEIDPQTQAVVDTVERVLAARENAMNQPGTEA',\n",
       "  'mw': 23445.0},\n",
       " 'b1676': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AD61',\n",
       "  'gene': 'b1676',\n",
       "  'locus': ['b1676', 'JW1666'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + pyruvate = ADP + H(+) + phosphoenolpyruvate',\n",
       "     '2.7.1.40')],\n",
       "   'Function': 'Catalyzes the formation of pyruvate in the last step of glycolysis, it is irreversible under physiological conditions. The reaction is critical for the control of metabolic flux in the second part of glycolysis',\n",
       "   'Regulation Activity': 'Belongs to type I PK; fructose 1,6-bisphosphate-activated',\n",
       "   'Pathway': 'Carbohydrate degradation; glycolysis; pyruvate from D-glyceraldehyde 3-phosphate: step 5/5'},\n",
       "  'seq': 'MKKTKIVCTIGPKTESEEMLAKMLDAGMNVMRLNFSHGDYAEHGQRIQNLRNVMSKTGKTAAILLDTKGPEIRTMKLEGGNDVSLKAGQTFTFTTDKSVIGNSEMVAVTYEGFTTDLSVGNTVLVDDGLIGMEVTAIEGNKVICKVLNNGDLGENKGVNLPGVSIALPALAEKDKQDLIFGCEQGVDFVAASFIRKRSDVIEIREHLKAHGGENIHIISKIENQEGLNNFDEILEASDGIMVARGDLGVEIPVEEVIFAQKMMIEKCIRARKVVITATQMLDSMIKNPRPTRAEAGDVANAILDGTDAVMLSGESAKGKYPLEAVSIMATICERTDRVMNSRLEFNNDNRKLRITEAVCRGAVETAEKLDAPLIVVATQGGKSARAVRKYFPDATILALTTNEKTAHQLVLSKGVVPQLVKEITSTDDFYRLGKELALQSGLAHKGDVVVMVSGALVPSGTTNTASVHVL',\n",
       "  'mw': 50729.0},\n",
       " 'b1677': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P69776',\n",
       "  'gene': 'b1677',\n",
       "  'locus': ['b1677', 'JW1667'],\n",
       "  'synonyms': 'mlpA mulI',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': \"An outer membrane lipoprotein that controls the distance between the inner and outer membranes; adding residues to Lpp increases the width of the periplasm (PubMed:29257832). The only protein known to be covalently linked to the peptidoglycan network (PGN) (PubMed:4261992, PubMed:3013869, PubMed:4245367). Also non-covalently binds the PGN (PubMed:3013869). The link between the cell outer membrane and PGN contributes to the maintenance of the structural and functional integrity of the cell envelope, and maintains the correct distance between the PGN and the outer membrane (PubMed:4261992, PubMed:3013869, PubMed:4245367, PubMed:4565677). The most abundant cellular protein in terms of copy number, there can be up to one million Lpp molecules per cell (PubMed:24766808). About one-third of Lpp is bound to the PGN (called bound or periplasmic) the rest is called free or transmembrane (PubMed:4565677). The 'free' form can be surface labeled by membrane impermeable agents and so must cross the outer membrane; it is thought that this transmembrane form is still anchored in the inner leaflet of the outer membrane (PubMed:21219470). Modeling suggests that non-covalent binding of OmpA (from the outer membrane) and TolR (from the inner membrane) to peptidoglycan maintains the position of the cell wall in the periplasm, holding it approximately equidistant from both the inner and outer membranes. Trimeric Lpp controls the width of the periplasm, adjusts its tilt angle to accommodate to the available space, and can compensate in part for an absence of OmpA (Probable). The role of the cell surface-exposed, free form (transmembrane) of Lpp is unknown (PubMed:21219470)\"},\n",
       "  'seq': 'MKATKLVLGAVILGSTLLAGCSSNAKIDQLSSDVQTLNAKVDQLSNDVNAMRSDVQAAKDDAARANQRLDNMATKYRK',\n",
       "  'mw': 8323.0},\n",
       " 'b1693': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P05194',\n",
       "  'gene': 'b1693',\n",
       "  'locus': ['b1693', 'JW1683'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('3-dehydroquinate = 3-dehydroshikimate + H2O',\n",
       "     '4.2.1.10')],\n",
       "   'Function': 'Involved in the third step of the chorismate pathway, which leads to the biosynthesis of aromatic amino acids (AroAA). Catalyzes the cis-dehydration of 3-dehydroquinate (DHQ) and introduces the first double bond of the aromatic ring to yield 3-dehydroshikimate. The reaction involves the formation of an imine intermediate between the keto group of 3-dehydroquinate and the epsilon-amino group of a Lys-170 at the active site',\n",
       "   'Regulation Activity': 'Inhibited by sodium borohydride',\n",
       "   'Pathway': 'Metabolic intermediate biosynthesis; chorismate biosynthesis; chorismate from D-erythrose 4-phosphate and phosphoenolpyruvate: step 3/7'},\n",
       "  'seq': 'MKTVTVKDLVIGTGAPKIIVSLMAKDIASVKSEALAYREADFDILEWRVDHYADLSNVESVMAAAKILRETMPEKPLLFTFRSAKEGGEQAISTEAYIALNRAAIDSGLVDMIDLELFTGDDQVKETVAYAHAHDVKVVMSNHDFHKTPEAEEIIARLRKMQSFDADIPKIALMPQSTSDVLTLLAATLEMQEQYADRPIITMSMAKTGVISRLAGEVFGSAATFGAVKKASAPGQISVNDLRTVLTILHQA',\n",
       "  'mw': 27467.0},\n",
       " 'b1701': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P38135',\n",
       "  'gene': 'b1701',\n",
       "  'locus': ['b1701', 'JW5910'],\n",
       "  'synonyms': 'ydiD',\n",
       "  'properties': {'Catalytic Activity': [('ATP + CoA + hexanoate = AMP + diphosphate + hexanoyl-CoA',\n",
       "     ''),\n",
       "    ('ATP + CoA + octanoate = AMP + diphosphate + octanoyl-CoA', ''),\n",
       "    ('ATP + CoA + dodecanoate = AMP + diphosphate + dodecanoyl-CoA', '')],\n",
       "   'Function': 'Catalyzes the esterification, concomitant with transport, of exogenous fatty acids into metabolically active CoA thioesters for subsequent degradation or incorporation into phospholipids. Is maximally active on C6:0, C8:0 and C12:0 fatty acids, while has a low activity on C14-C18 chain length fatty acids (PubMed:15213221, PubMed:19477415). Is involved in the anaerobic beta-oxidative degradation of fatty acids, which allows anaerobic growth of E.coli on fatty acids as a sole carbon and energy source in the presence of nitrate or fumarate as a terminal electron acceptor (PubMed:12535077). Can functionally replace FadD under anaerobic conditions (PubMed:12535077)',\n",
       "   'Pathway': 'Lipid metabolism; fatty acid beta-oxidation'},\n",
       "  'seq': 'MHPTGPHLGPDVLFRESNMKVTLTFNEQRRAAYRQQGLWGDASLADYWQQTARAMPDKIAVVDNHGASYTYSALDHAASCLANWMLAKGIESGDRIAFQLPGWCEFTVIYLACLKIGAVSVPLLPSWREAELVWVLNKCQAKMFFAPTLFKQTRPVDLILPLQNQLPQLQQIVGVDKLAPATSSLSLSQIIADNTSLTTAITTHGDELAAVLFTSGTEGLPKGVMLTHNNILASERAYCARLNLTWQDVFMMPAPLGHATGFLHGVTAPFLIGARSVLLDIFTPDACLALLEQQRCTCMLGATPFVYDLLNVLEKQPADLSALRFFLCGGTTIPKKVARECQQRGIKLLSVYGSTESSPHAVVNLDDPLSRFMHTDGYAAAGVEIKVVDDARKTLPPGCEGEEASRGPNVFMGYFDEPELTARALDEEGWYYSGDLCRMDEAGYIKITGRKKDIIVRGGENISSREVEDILLQHPKIHDACVVAMSDERLGERSCAYVVLKAPHHSLSLEEVVAFFSRKRVAKYKYPEHIVVIEKLPRTTSGKIQKFLLRKDIMRRLTQDVCEEIE',\n",
       "  'mw': 62759.0},\n",
       " 'b1702': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P23538',\n",
       "  'gene': 'b1702',\n",
       "  'locus': ['b1702', 'JW1692'],\n",
       "  'synonyms': 'pps',\n",
       "  'properties': {'Catalytic Activity': [('ATP + H2O + pyruvate = AMP + 2 H(+) + phosphate + phosphoenolpyruvate',\n",
       "     '2.7.9.2')],\n",
       "   'Function': 'Catalyzes the phosphorylation of pyruvate to phosphoenolpyruvate',\n",
       "   'Regulation Activity': 'Activated by a Pi-dependent pyrophosphorylation and inactivated by an ADP-dependent phosphorylation on a regulatory threonine. Both reactions are mediated by the bifunctional serine/threonine kinase and phosphorylase PpsR',\n",
       "   'Pathway': 'Carbohydrate biosynthesis; gluconeogenesis'},\n",
       "  'seq': 'MSNNGSSPLVLWYNQLGMNDVDRVGGKNASLGEMITNLSGMGVSVPNGFATTADAFNQFLDQSGVNQRIYELLDKTDIDDVTQLAKAGAQIRQWIIDTPFQPELENAIREAYAQLSADDENASFAVRSSATAEDMPDASFAGQQETFLNVQGFDAVLVAVKHVFASLFNDRAISYRVHQGYDHRGVALSAGVQRMVRSDLASSGVMFSIDTESGFDQVVFITSAWGLGEMVVQGAVNPDEFYVHKPTLAANRPAIVRRTMGSKKIRMVYAPTQEHGKQVKIEDVPQEQRDIFSLTNEEVQELAKQAVQIEKHYGRPMDIEWAKDGHTGKLFIVQARPETVRSRGQVMERYTLHSQGKIIAEGRAIGHRIGAGPVKVIHDISEMNRIEPGDVLVTDMTDPDWEPIMKKASAIVTNRGGRTCHAAIIARELGIPAVVGCGDATERMKDGENVTVSCAEGDTGYVYAELLEFSVKSSSVETMPDLPLKVMMNVGNPDRAFDFACLPNEGVGLARLEFIINRMIGVHPRALLEFDDQEPQLQNEIREMMKGFDSPREFYVGRLTEGIATLGAAFYPKRVIVRLSDFKSNEYANLVGGERYEPDEENPMLGFRGAGRYVSDSFRDCFALECEAVKRVRNDMGLTNVEIMIPFVRTVDQAKAVVEELARQGLKRGENGLKIIMMCEIPSNALLAEQFLEYFDGFSIGSNDMTQLALGLDRDSGVVSELFDERNDAVKALLSMAIRAAKKQGKYVGICGQGPSDHEDFAAWLMEEGIDSLSLNPDTVVQTWLSLAELKK',\n",
       "  'mw': 87435.0},\n",
       " 'b1704': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P00887',\n",
       "  'gene': 'b1704',\n",
       "  'locus': ['b1704', 'JW1694'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('D-erythrose 4-phosphate + H2O + phosphoenolpyruvate = 7-phospho-2-dehydro-3-deoxy-D-arabino-heptonate + phosphate',\n",
       "     '2.5.1.54')],\n",
       "   'Function': 'Stereospecific condensation of phosphoenolpyruvate (PEP) and D-erythrose-4-phosphate (E4P) giving rise to 3-deoxy-D-arabino-heptulosonate-7-phosphate (DAHP)',\n",
       "   'Pathway': 'Metabolic intermediate biosynthesis; chorismate biosynthesis; chorismate from D-erythrose 4-phosphate and phosphoenolpyruvate: step 1/7'},\n",
       "  'seq': 'MNRTDELRTARIESLVTPAELALRYPVTPGVATHVTDSRRRIEKILNGEDKRLLVIIGPCSIHDLTAAMEYATRLQSLRNQYQSRLEIVMRTYFEKPRTVVGWKGLISDPDLNGSYRVNHGLELARKLLLQVNELGVPTATEFLDMVTGQFIADLISWGAIGARTTESQIHREMASALSCPVGFKNGTDGNTRIAVDAIRAARASHMFLSPDKNGQMTIYQTSGNPYGHIIMRGGKKPNYHADDIAAACDTLHEFDLPEHLVVDFSHGNCQKQHRRQLEVCEDICQQIRNGSTAIAGIMAESFLREGTQKIVGSQPLTYGQSITDPCLGWEDTERLVEKLASAVDTRF',\n",
       "  'mw': 38735.0},\n",
       " 'b1709': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P06611',\n",
       "  'gene': 'b1709',\n",
       "  'locus': ['b1709', 'JW1699'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('an R-cob(III)alamin(out) + ATP + H2O = ADP + an R-cob(III)alamin(in) + H(+) + phosphate',\n",
       "     '7.6.2.8')],\n",
       "   'Function': 'Part of the ABC transporter complex BtuCDF involved in vitamin B12 import. Responsible for energy coupling to the transport system'},\n",
       "  'seq': 'MSIVMQLQDVAESTRLGPLSGEVRAGEILHLVGPNGAGKSTLLARMAGMTSGKGSIQFAGQPLEAWSATKLALHRAYLSQQQTPPFATPVWHYLTLHQHDKTRTELLNDVAGALALDDKLGRSTNQLSGGEWQRVRLAAVVLQITPQANPAGQLLLLDEPMNSLDVAQQSALDKILSALCQQGLAIVMSSHDLNHTLRHAHRAWLLKGGKMLASGRREEVLTPPNLAQAYGMNFRRLDIEGHRMLISTI',\n",
       "  'mw': 27081.0},\n",
       " 'b1711': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P06609',\n",
       "  'gene': 'b1711',\n",
       "  'locus': ['b1711', 'JW1701'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the ABC transporter complex BtuCDF involved in vitamin B12 import. Involved in the translocation of the substrate across the membrane'},\n",
       "  'seq': 'MLTLARQQQRQNIRWLLCLSVLMLLALLLSLCAGEQWISPGDWFTPRGELFVWQIRLPRTLAVLLVGAALAISGAVMQALFENPLAEPGLLGVSNGAGVGLIAAVLLGQGQLPNWALGLCAIAGALIITLILLRFARRHLSTSRLLLAGVALGIICSALMTWAIYFSTSVDLRQLMYWMMGGFGGVDWRQSWLMLALIPVLLWICCQSRPMNMLALGEISARQLGLPLWFWRNVLVAATGWMVGVSVALAGAIGFIGLVIPHILRLCGLTDHRVLLPGCALAGASALLLADIVARLALAAAELPIGVVTATLGAPVFIWLLLKAGR',\n",
       "  'mw': 34949.0},\n",
       " 'b1713': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P07395',\n",
       "  'gene': 'b1713',\n",
       "  'locus': ['b1713', 'JW1703'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + L-phenylalanine + tRNA(Phe) = AMP + diphosphate + H(+) + L-phenylalanyl-tRNA(Phe)',\n",
       "     '6.1.1.20')]},\n",
       "  'seq': 'MKFSELWLREWVNPAIDSDALANQITMAGLEVDGVEPVAGSFHGVVVGEVVECAQHPNADKLRVTKVNVGGDRLLDIVCGAPNCRQGLRVAVATIGAVLPGDFKIKAAKLRGEPSEGMLCSFSELGISDDHSGIIELPADAPIGTDIREYLKLDDNTIEISVTPNRADCLGIIGVARDVAVLNQLPLVQPEIVPVGATIDDTLPITVEAPEACPRYLGRVVKGINVKAPTPLWMKEKLRRCGIRSIDAVVDVTNYVLLELGQPMHAFDKDRIEGGIVVRMAKEGETLVLLDGTEAKLNADTLVIADHNKALAMGGIFGGEHSGVNDETQNVLLECAFFSPLSITGRARRHGLHTDASHRYERGVDPALQHKAMERATRLLIDICGGEAGPVIDITNEATLPKRATITLRRSKLDRLIGHHIADEQVTDILRRLGCEVTEGKDEWQAVAPSWRFDMEIEEDLVEEVARVYGYNNIPDEPVQASLIMGTHREADLSLKRVKTLLNDKGYQEVITYSFVDPKVQQMIHPGVEALLLPSPISVEMSAMRLSLWTGLLATVVYNQNRQQNRVRIFESGLRFVPDTQAPLGIRQDLMLAGVICGNRYEEHWNLAKETVDFYDLKGDLESVLDLTGKLNEVEFRAEANPALHPGQSAAIYLKGERIGFVGVVHPELERKLDLNGRTLVFELEWNKLADRVVPQAREISRFPANRRDIAVVVAENVPAADILSECKKVGVNQVVGVNLFDVYRGKGVAEGYKSLAISLILQDTSRTLEEEEIAATVAKCVEALKERFQASLRD',\n",
       "  'mw': 87378.0},\n",
       " 'b1714': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P08312',\n",
       "  'gene': 'b1714',\n",
       "  'locus': ['b1714', 'JW5277'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + L-phenylalanine + tRNA(Phe) = AMP + diphosphate + H(+) + L-phenylalanyl-tRNA(Phe)',\n",
       "     '6.1.1.20')]},\n",
       "  'seq': 'MSHLAELVASAKAAISQASDVAALDNVRVEYLGKKGHLTLQMTTLRELPPEERPAAGAVINEAKEQVQQALNARKAELESAALNARLAAETIDVSLPGRRIENGGLHPVTRTIDRIESFFGELGFTVATGPEIEDDYHNFDALNIPGHHPARADHDTFWFDTTRLLRTQTSGVQIRTMKAQQPPIRIIAPGRVYRNDYDQTHTPMFHQMEGLIVDTNISFTNLKGTLHDFLRNFFEEDLQIRFRPSYFPFTEPSAEVDVMGKNGKWLEVLGCGMVHPNVLRNVGIDPEVYSGFAFGMGMERLTMLRYGVTDLRSFFENDLRFLKQFK',\n",
       "  'mw': 36832.0},\n",
       " 'b1719': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A8M3',\n",
       "  'gene': 'b1719',\n",
       "  'locus': ['b1719', 'JW1709'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + L-threonine + tRNA(Thr) = AMP + diphosphate + H(+) + L-threonyl-tRNA(Thr)',\n",
       "     '6.1.1.3')],\n",
       "   'Function': 'ThrS is also a translational repressor protein, it controls binds its own mRNA in the operator region upstream of the start codon (PubMed:3086882). The mRNA region upstream of the start codon has a tRNA-like secondary structure; mRNA and tRNA compete for binding to ThrRS (PubMed:2254931). ThrRS represses translation by preventing the ribosome from to mRNA, and tRNA(Thr) acts as an antirepressor allowing fine level control of enzyme synthesis (PubMed:2254931). X-ray structures prove that operator mRNA and tRNA bind to overlapping sites in the protein (PubMed:10319817, PubMed:11953757)',\n",
       "   'Regulation Activity': 'Inhibited non-competitively by borrelidin (BN, KI is 3.7 nM) which binds in a 1:1 stoichiometry, inhibiting L-thr activation (PubMed:15507440). BN binds to 4 distinct subsites in the protein, preventing binding of all 3 substrates; BN also inhibits human ThrRS, and thus it is not useful as an antibiotic (PubMed:25824639)'},\n",
       "  'seq': 'MPVITLPDGSQRHYDHAVSPMDVALDIGPGLAKACIAGRVNGELVDACDLIENDAQLSIITAKDEEGLEIIRHSCAHLLGHAIKQLWPHTKMAIGPVIDNGFYYDVDLDRTLTQEDVEALEKRMHELAEKNYDVIKKKVSWHEARETFANRGESYKVSILDENIAHDDKPGLYFHEEYVDMCRGPHVPNMRFCHHFKLMKTAGAYWRGDSNNKMLQRIYGTAWADKKALNAYLQRLEEAAKRDHRKIGKQLDLYHMQEEAPGMVFWHNDGWTIFRELEVFVRSKLKEYQYQEVKGPFMMDRVLWEKTGHWDNYKDAMFTTSSENREYCIKPMNCPGHVQIFNQGLKSYRDLPLRMAEFGSCHRNEPSGSLHGLMRVRGFTQDDAHIFCTEEQIRDEVNGCIRLVYDMYSTFGFEKIVVKLSTRPEKRIGSDEMWDRAEADLAVALEENNIPFEYQLGEGAFYGPKIEFTLYDCLDRAWQCGTVQLDFSLPSRLSASYVGEDNERKVPVMIHRAILGSMERFIGILTEEFAGFFPTWLAPVQVVIMNITDSQSEYVNELTQKLSNAGIRVKADLRNEKIGFKIREHTLRRVPYMLVCGDKEVESGKVAVRTRRGKDLGSMDVNEVIEKLQQEIRSRSLKQLEE',\n",
       "  'mw': 74014.0},\n",
       " 'b1723': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P06999',\n",
       "  'gene': 'b1723',\n",
       "  'locus': ['b1723', 'JW5280'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + beta-D-fructose 6-phosphate = ADP + beta-D-fructose 1,6-bisphosphate + H(+)',\n",
       "     '2.7.1.11')],\n",
       "   'Function': 'Catalyzes the phosphorylation of D-fructose 6-phosphate to fructose 1,6-bisphosphate by ATP, the first committing step of glycolysis',\n",
       "   'Regulation Activity': 'Allosterically inhibited by ATP. Allosteric ATP-binding requires the presence of the substrate ATP. Inhibited by monovalent cations with ionic radii larger than Na(+) (e.g. K(+), Cs(+)). The monovalent cations increase the affinity of the allosteric site for ATP. PFK-2 is sensitive to inhibition by fructose 1,6-diphosphate',\n",
       "   'Pathway': 'Carbohydrate degradation; glycolysis; D-glyceraldehyde 3-phosphate and glycerone phosphate from D-glucose: step 3/4'},\n",
       "  'seq': 'MVRIYTLTLAPSLDSATITPQIYPEGKLRCTAPVFEPGGGGINVARAIAHLGGSATAIFPAGGATGEHLVSLLADENVPVATVEAKDWTRQNLHVHVEASGEQYRFVMPGAALNEDEFRQLEEQVLEIESGAILVISGSLPPGVKLEKLTQLISAAQKQGIRCIVDSSGEALSAALAIGNIELVKPNQKELSALVNRELTQPDDVRKAAQEIVNSGKAKRVVVSLGPQGALGVDSENCIQVVPPPVKSQSTVGAGDSMVGAMTLKLAENASLEEMVRFGVAAGSAATLNQGTRLCSHDDTQKIYAYLSR',\n",
       "  'mw': 32456.0},\n",
       " 'b1732': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P21179',\n",
       "  'gene': 'b1732',\n",
       "  'locus': ['b1732', 'JW1721'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('2 H2O2 = 2 H2O + O2', '1.11.1.6')],\n",
       "   'Function': 'Decomposes hydrogen peroxide into water and oxygen; serves to protect cells from the toxic effects of hydrogen peroxide'},\n",
       "  'seq': 'MSQHNEKNPHQHQSPLHDSSEAKPGMDSLAPEDGSHRPAAEPTPPGAQPTAPGSLKAPDTRNEKLNSLEDVRKGSENYALTTNQGVRIADDQNSLRAGSRGPTLLEDFILREKITHFDHERIPERIVHARGSAAHGYFQPYKSLSDITKADFLSDPNKITPVFVRFSTVQGGAGSADTVRDIRGFATKFYTEEGIFDLVGNNTPIFFIQDAHKFPDFVHAVKPEPHWAIPQGQSAHDTFWDYVSLQPETLHNVMWAMSDRGIPRSYRTMEGFGIHTFRLINAEGKATFVRFHWKPLAGKASLVWDEAQKLTGRDPDFHRRELWEAIEAGDFPEYELGFQLIPEEDEFKFDFDLLDPTKLIPEELVPVQRVGKMVLNRNPDNFFAENEQAAFHPGHIVPGLDFTNDPLLQGRLFSYTDTQISRLGGPNFHEIPINRPTCPYHNFQRDGMHRMGIDTNPANYEPNSINDNWPRETPPGPKRGGFESYQERVEGNKVRERSPSFGEYYSHPRLFWLSQTPFEQRHIVDGFSFELSKVVRPYIRERVVDQLAHIDLTLAQAVAKNLGIELTDDQLNITPPPDVNGLKKDPSLSLYAIPDGDVKGRVVAILLNDEVRSADLLAILKALKAKGVHAKLLYSRMGEVTADDGTVLPIAATFAGAPSLTVDAVIVPCGNIADIADNGDANYYLMEAYKHLKPIALAGDARKFKATIKIADQGEEGIVEADSADGSFMDELLTLMAAHRVWSRIPKIDKIPA',\n",
       "  'mw': 84163.0},\n",
       " 'b1740': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P18843',\n",
       "  'gene': 'b1740',\n",
       "  'locus': ['b1740', 'JW1729'],\n",
       "  'synonyms': 'efg ntrL',\n",
       "  'properties': {'Catalytic Activity': [('ATP + deamido-NAD(+) + NH4(+) = AMP + diphosphate + H(+) + NAD(+)',\n",
       "     '6.3.1.5')],\n",
       "   'Function': 'Catalyzes the ATP-dependent amidation of deamido-NAD to form NAD. Uses ammonia as a nitrogen source',\n",
       "   'Pathway': 'Cofactor biosynthesis; NAD(+) biosynthesis; NAD(+) from deamido-NAD(+) (ammonia route): step 1/1'},\n",
       "  'seq': 'MTLQQQIIKALGAKPQINAEEEIRRSVDFLKSYLQTYPFIKSLVLGISGGQDSTLAGKLCQMAINELRLETGNESLQFIAVRLPYGVQADEQDCQDAIAFIQPDRVLTVNIKGAVLASEQALREAGIELSDFVRGNEKARERMKAQYSIAGMTSGVVVGTDHAAEAITGFFTKYGDGGTDINPLYRLNKRQGKQLLAALACPEHLYKKAPTADLEDDRPSLPDEVALGVTYDNIDDYLEGKNVPQQVARTIENWYLKTEHKRRPPITVFDDFWKK',\n",
       "  'mw': 30637.0},\n",
       " 'b1744': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P76215',\n",
       "  'gene': 'b1744',\n",
       "  'locus': ['b1744', 'JW1733'],\n",
       "  'synonyms': 'ydjS',\n",
       "  'properties': {'Catalytic Activity': [('H2O + N-succinyl-L-glutamate = L-glutamate + succinate',\n",
       "     '3.5.1.96')],\n",
       "   'Function': 'Transforms N(2)-succinylglutamate into succinate and glutamate',\n",
       "   'Pathway': 'Amino-acid degradation; L-arginine degradation via AST pathway; L-glutamate and succinate from L-arginine: step 5/5'},\n",
       "  'seq': 'MDNFLALTLTGKKPVITEREINGVRWRWLGDGVLELTPLTPPQGALVISAGIHGNETAPVEMLDALLGAISHGEIPLRWRLLVILGNPPALKQGKRYCHSDMNRMFGGRWQLFAESGETCRARELEQCLEDFYDQGKESVRWHLDLHTAIRGSLHPQFGVLPQRDIPWDEKFLTWLGAAGLEALVFHQEPGGTFTHFSARHFGALACTLELGKALPFGQNDLRQFAVTASAIAALLSGESVGIVRTPPLRYRVVSQITRHSPSFEMHMASDTLNFMPFEKGTLLAQDGEERFTVTHDVEYVLFPNPLVALGLRAGLMLEKIS',\n",
       "  'mw': 35800.0},\n",
       " 'b1745': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P76216',\n",
       "  'gene': 'b1745',\n",
       "  'locus': ['b1745', 'JW1734'],\n",
       "  'synonyms': 'ydjT',\n",
       "  'properties': {'Catalytic Activity': [('2 H(+) + 2 H2O + N(2)-succinyl-L-arginine = CO2 + N(2)-succinyl-L-ornithine + 2 NH4(+)',\n",
       "     '3.5.3.23')],\n",
       "   'Function': 'Catalyzes the hydrolysis of N(2)-succinylarginine into N(2)-succinylornithine, ammonia and CO(2)',\n",
       "   'Pathway': 'Amino-acid degradation; L-arginine degradation via AST pathway; L-glutamate and succinate from L-arginine: step 2/5'},\n",
       "  'seq': 'MNAWEVNFDGLVGLTHHYAGLSFGNEASTRHRFQVSNPRLAAKQGLLKMKALADAGFPQAVIPPHERPFIPVLRQLGFSGSDEQVLEKVARQAPHWLSSVSSASPMWVANAATIAPSADTLDGKVHLTVANLNNKFHRSLEAPVTESLLKAIFNDEEKFSVHSALPQVALLGDEGAANHNRLGGHYGEPGMQLFVYGREEGNDTRPSRYPARQTREASEAVARLNQVNPQQVIFAQQNPDVIDQGVFHNDVIAVSNRQVLFCHQQAFARQSQLLANLRARVNGFMAIEVPATQVSVSDTVSTYLFNSQLLSRDDGSMMLVLPQECREHAGVWGYLNELLAADNPISELKVFDLRESMANGGGPACLRLRVVLTEEERRAVNPAVMMNDTLFNALNDWVDRYYRDRLTAADLADPQLLREGREALDVLSQLLNLGSVYPFQREGGGNG',\n",
       "  'mw': 49299.0},\n",
       " 'b1746': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P76217',\n",
       "  'gene': 'b1746',\n",
       "  'locus': ['b1746', 'JW5282'],\n",
       "  'synonyms': 'ydjU',\n",
       "  'properties': {'Catalytic Activity': [('H2O + N-succinyl-L-glutamate 5-semialdehyde + NAD(+) = 2 H(+) + N-succinyl-L-glutamate + NADH',\n",
       "     '1.2.1.71'),\n",
       "    ('decanal + H2O + NAD(+) = decanoate + 2 H(+) + NADH', ''),\n",
       "    ('H2O + NAD(+) + succinate semialdehyde = 2 H(+) + NADH + succinate', '')],\n",
       "   'Function': 'Catalyzes the NAD-dependent reduction of succinylglutamate semialdehyde into succinylglutamate (Probable). Also shows in vitro activity with decanal or succinic semialdehyde as the electron donor and NAD as the electron acceptor. No activity is detected with NADP as the electron acceptor. Therefore, is an aldehyde dehydrogenase with broad substrate specificity (PubMed:15808744)',\n",
       "   'Pathway': 'Amino-acid degradation; L-arginine degradation via AST pathway; L-glutamate and succinate from L-arginine: step 4/5'},\n",
       "  'seq': 'MTLWINGDWITGQGASRVKRNPVSGEVLWQGNDADAAQVEQACRAARAAFPRWARLSFAERHAVVERFAALLESNKAELTAIIARETGKPRWEAATEVTAMINKIAISIKAYHVRTGEQRSEMPDGAASLRHRPHGVLAVFGPYNFPGHLPNGHIVPALLAGNTIIFKPSELTPWSGEAVMRLWQQAGLPPGVLNLVQGGRETGQALSALEDLDGLLFTGSANTGYQLHRQLSGQPEKILALEMGGNNPLIIDEVADIDAAVHLTIQSAFVTAGQRCTCARRLLLKSGAQGDAFLARLVAVSQRLTPGNWDDEPQPFIGGLISEQAAQQVVTAWQQLEAMGGRPLLAPRLLQAGTSLLTPGIIEMTGVAGVPDEEVFGPLLRVWRYDTFDEAIRMANNTRFGLSCGLVSPEREKFDQLLLEARAGIVNWNKPLTGAASTAPFGGIGASGNHRPSAWYAADYCAWPMASLESDSLTLPATLNPGLDFSDEVVR',\n",
       "  'mw': 53026.0},\n",
       " 'b1747': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AE37',\n",
       "  'gene': 'b1747',\n",
       "  'locus': ['b1747', 'JW1736'],\n",
       "  'synonyms': 'ydjV',\n",
       "  'properties': {'Catalytic Activity': [('L-arginine + succinyl-CoA = CoA + H(+) + N(2)-succinyl-L-arginine',\n",
       "     '2.3.1.109')],\n",
       "   'Function': 'Catalyzes the transfer of succinyl-CoA to arginine to produce N(2)-succinylarginine',\n",
       "   'Pathway': 'Amino-acid degradation; L-arginine degradation via AST pathway; L-glutamate and succinate from L-arginine: step 1/5'},\n",
       "  'seq': 'MMVIRPVERSDVSALMQLASKTGGGLTSLPANEATLSARIERAIKTWQGELPKSEQGYVFVLEDSETGTVAGICAIEVAVGLNDPWYNYRVGTLVHASKELNVYNALPTLFLSNDHTGSSELCTLFLDPDWRKEGNGYLLSKSRFMFMAAFRDKFNDKVVAEMRGVIDEHGYSPFWQSLGKRFFSMDFSRADFLCGTGQKAFIAELMPKHPIYTHFLSQEAQDVIGQVHPQTAPARAVLEKEGFRYRNYIDIFDGGPTLECDIDRVRAIRKSRLVEVAEGQPAQGDFPACLVANENYHHFRVVLVRTDPATERLILTAAQLDALKCHAGDRVRLVRLCAEEKTA',\n",
       "  'mw': 38456.0},\n",
       " 'b1748': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77581',\n",
       "  'gene': 'b1748',\n",
       "  'locus': ['b1748', 'JW1737'],\n",
       "  'synonyms': 'argM cstC ydjW',\n",
       "  'properties': {'Catalytic Activity': [('2-oxoglutarate + N(2)-succinyl-L-ornithine = L-glutamate + N-succinyl-L-glutamate 5-semialdehyde',\n",
       "     '2.6.1.81')],\n",
       "   'Function': 'Catalyzes the transamination of N(2)-succinylornithine and alpha-ketoglutarate into N(2)-succinylglutamate semialdehyde and glutamate. Can also act as an acetylornithine aminotransferase. Is involved in the utilization of arginine as a nitrogen source, via the AST pathway, and seems also to play a role in ornithine catabolism (PubMed:9696779)',\n",
       "   'Pathway': 'Amino-acid degradation; L-arginine degradation via AST pathway; L-glutamate and succinate from L-arginine: step 3/5'},\n",
       "  'seq': 'MSQPITRENFDEWMIPVYAPAPFIPVRGEGSRLWDQQGKEYIDFAGGIAVNALGHAHPELREALNEQASKFWHTGNGYTNEPVLRLAKKLIDATFADRVFFCNSGAEANEAALKLARKFAHDRYGSHKSGIVAFKNAFHGRTLFTVSAGGQPAYSQDFAPLPADIRHAAYNDINSASALIDDSTCAVIVEPIQGEGGVVPASNAFLQGLRELCNRHNALLIFDEVQTGVGRTGELYAYMHYGVTPDLLTTAKALGGGFPVGALLATEECARVMTVGTHGTTYGGNPLASAVAGKVLELINTPEMLNGVKQRHDWFVERLNTINHRYGLFSEVRGLGLLIGCVLNADYAGQAKQISQEAAKAGVMVLIAGGNVVRFAPALNVSEEEVTTGLDRFAAACEHFVSRGSS',\n",
       "  'mw': 43665.0},\n",
       " 'b1759': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77788',\n",
       "  'gene': 'b1759',\n",
       "  'locus': ['b1759', 'JW1748'],\n",
       "  'synonyms': 'ynjG',\n",
       "  'properties': {'Catalytic Activity': [('CTP + H2O = CMP + diphosphate + H(+)',\n",
       "     '3.6.1.65'),\n",
       "    ('dCTP + H2O = dCMP + diphosphate + H(+)', '3.6.1.65')],\n",
       "   'Function': 'Hydrolase with a preference for pyrimidine substrates. Has high activity with 5-methyl-dCTP, and much lower activity with CTP, dCTP, 5-hydroxy-dCTP, 2-hydroxy-dATP and 8-hydroxy-dGTP'},\n",
       "  'seq': 'MKMIEVVAAIIERDGKILLAQRPAQSDQAGLWEFAGGKVEPDESQRQALVRELREELGIEATVGEYVASHQREVSGRIIHLHAWHVPDFHGTLQAHEHQALVWCSPEEALQYPLAPADIPLLEAFMALRAARPAD',\n",
       "  'mw': 15046.0},\n",
       " 'b1761': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P00370',\n",
       "  'gene': 'b1761',\n",
       "  'locus': ['b1761', 'JW1750'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('H2O + L-glutamate + NADP(+) = 2-oxoglutarate + H(+) + NADPH + NH4(+)',\n",
       "     '1.4.1.4')],\n",
       "   'Function': 'Catalyzes the reversible oxidative deamination of glutamate to alpha-ketoglutarate and ammonia',\n",
       "   'Regulation Activity': 'Competitively inhibited by homoserine and by glutamine'},\n",
       "  'seq': 'MDQTYSLESFLNHVQKRDPNQTEFAQAVREVMTTLWPFLEQNPKYRQMSLLERLVEPERVIQFRVVWVDDRNQIQVNRAWRVQFSSAIGPYKGGMRFHPSVNLSILKFLGFEQTFKNALTTLPMGGGKGGSDFDPKGKSEGEVMRFCQALMTELYRHLGADTDVPAGDIGVGGREVGFMAGMMKKLSNNTACVFTGKGLSFGGSLIRPEATGYGLVYFTEAMLKRHGMGFEGMRVSVSGSGNVAQYAIEKAMEFGARVITASDSSGTVVDESGFTKEKLARLIEIKASRDGRVADYAKEFGLVYLEGQQPWSLPVDIALPCATQNELDVDAAHQLIANGVKAVAEGANMPTTIEATELFQQAGVLFAPGKAANAGGVATSGLEMAQNAARLGWKAEKVDARLHHIMLDIHHACVEHGGEGEQTNYVQGANIAGFVKVADAMLAQGVI',\n",
       "  'mw': 48581.0},\n",
       " 'b1764': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P16456',\n",
       "  'gene': 'b1764',\n",
       "  'locus': ['b1764', 'JW1753'],\n",
       "  'synonyms': 'fdhB',\n",
       "  'properties': {'Catalytic Activity': [('ATP + H2O + hydrogenselenide = AMP + 2 H(+) + phosphate + selenophosphate',\n",
       "     '2.7.9.3')],\n",
       "   'Function': 'Synthesizes selenophosphate from selenide and ATP'},\n",
       "  'seq': 'MSENSIRLTQYSHGAGCGCKISPKVLETILHSEQAKFVDPNLLVGNETRDDAAVYDLGNGTSVISTTDFFMPIVDNPFDFGRIAATNAISDIFAMGGKPIMAIAILGWPINKLSPEIAREVTEGGRYACRQAGIALAGGHSIDAPEPIFGLAVTGIVPTERVKKNSTAQAGCKLFLTKPLGIGVLTTAEKKSLLKPEHQGLATEVMCRMNIAGASFANIEGVKAMTDVTGFGLLGHLSEMCQGAGVQARVDYEAIPKLPGVEEYIKLGAVPGGTERNFASYGHLMGEMPREVRDLLCDPQTSGGLLLAVMPEAENEVKATAAEFGIELTAIGELVPARGGRAMVEIR',\n",
       "  'mw': 36687.0},\n",
       " 'b1767': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A962',\n",
       "  'gene': 'b1767',\n",
       "  'locus': ['b1767', 'JW1756'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('H2O + L-asparagine = L-aspartate + NH4(+)',\n",
       "     '3.5.1.1')],\n",
       "   'Regulation Activity': 'Shows cooperative activation. Allosterically activated by asparagine'},\n",
       "  'seq': 'MQKKSIYVAYTGGTIGMQRSEQGYIPVSGHLQRQLALMPEFHRPEMPDFTIHEYTPLMDSSDMTPEDWQHIAEDIKAHYDDYDGFVILHGTDTMAYTASALSFMLENLGKPVIVTGSQIPLAELRSDGQINLLNALYVAANYPINEVTLFFNNRLYRGNRTTKAHADGFDAFASPNLPPLLEAGIHIRRLNTPPAPHGEGELIVHPITPQPIGVVTIYPGISADVVRNFLRQPVKALILRSYGVGNAPQNKAFLQELQEASDRGIVVVNLTQCMSGKVNMGGYATGNALAHAGVIGGADMTVEATLTKLHYLLSQELDTETIRKAMSQNLRGELTPDD',\n",
       "  'mw': 37127.0},\n",
       " 'b1768': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P21369',\n",
       "  'gene': 'b1768',\n",
       "  'locus': ['b1768', 'JW1757'],\n",
       "  'synonyms': 'nam ydjB',\n",
       "  'properties': {'Catalytic Activity': [('H2O + nicotinamide = NH4(+) + nicotinate',\n",
       "     '3.5.1.19'),\n",
       "    ('H2O + pyrazinamide = NH4(+) + pyrazine-2-carboxylate', '')],\n",
       "   'Function': 'Is also able to hydrolyze the first-line antituberculous drug pyrazinamide (PZA) into pyrazinoic acid in vitro, but this reaction is not considered to be physiologically relevant',\n",
       "   'Pathway': 'Cofactor biosynthesis; nicotinate biosynthesis; nicotinate from nicotinamide: step 1/1'},\n",
       "  'seq': 'MPPRALLLVDLQNDFCAGGALAVPEGDSTVDVANRLIDWCQSRGEAVIASQDWHPANHGSFASQHGVEPYTPGQLDGLPQTFWPDHCVQNSEGAQLHPLLHQKAIAAVFHKGENPLVDSYSAFFDNGRRQKTSLDDWLRDHEIDELIVMGLATDYCVKFTVLDALQLGYKVNVITDGCRGVNIQPQDSAHAFMEMSAAGATLYTLADWEETQG',\n",
       "  'mw': 23362.0},\n",
       " 'b1773': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77704',\n",
       "  'gene': 'b1773',\n",
       "  'locus': ['b1773', 'JW1762'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': []},\n",
       "  'seq': 'MLADIRYWENDATNKHYAIAHFNVWNAEMLMGVIDAAEEAKSPVIISFGTGFVGNTSFEDFSHMMVSMAQKATVPVITHWDHGRSMEIIHNAWTHGMNSLMRDASAFDFEENIRLTKEAVDFFHPLGIPVEAELGHVGNETVYEEALAGYHYTDPDQAAEFVERTGCDSLAVAIGNQHGVYTSEPQLNFEVVKRVRDAVSVPLVLHGASGISDADIKTAISLGIAKINIHTELCQAAMVAVKENQDQPFLHLEREVRKAVKERALEKIKLFGSDGKAE',\n",
       "  'mw': 30810.0},\n",
       " 'b1778': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A746',\n",
       "  'gene': 'b1778',\n",
       "  'locus': ['b1778', 'JW1767'],\n",
       "  'synonyms': 'yeaA',\n",
       "  'properties': {'Catalytic Activity': [('[thioredoxin]-disulfide + H2O + L-methionyl-[protein] = [thioredoxin]-dithiol + L-methionyl-(R)-S-oxide-[protein]',\n",
       "     '1.8.4.12')]},\n",
       "  'seq': 'MANKPSAEELKKNLSEMQFYVTQNHGTEPPFTGRLLHNKRDGVYHCLICDAPLFHSQTKYDSGCGWPSFYEPVSEESIRYIKDLSHGMQRIEIRCGNCDAHLGHVFPDGPQPTGERYCVNSASLRFTDGENGEEING',\n",
       "  'mw': 15451.0},\n",
       " 'b1779': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A9B2',\n",
       "  'gene': 'b1779',\n",
       "  'locus': ['b1779', 'JW1768'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('D-glyceraldehyde 3-phosphate + NAD(+) + phosphate = (2R)-3-phospho-glyceroyl phosphate + H(+) + NADH',\n",
       "     '1.2.1.12')],\n",
       "   'Function': 'Catalyzes the oxidative phosphorylation of glyceraldehyde 3-phosphate (G3P) to 1,3-bisphosphoglycerate (BPG) using the cofactor NAD. The first reaction step involves the formation of a hemiacetal intermediate between G3P and a cysteine residue, and this hemiacetal intermediate is then oxidized to a thioester, with concomitant reduction of NAD to NADH. The reduced NADH is then exchanged with the second NAD, and the thioester is attacked by a nucleophilic inorganic phosphate to produce BPG',\n",
       "   'Pathway': 'Carbohydrate degradation; glycolysis; pyruvate from D-glyceraldehyde 3-phosphate: step 1/5'},\n",
       "  'seq': 'MTIKVGINGFGRIGRIVFRAAQKRSDIEIVAINDLLDADYMAYMLKYDSTHGRFDGTVEVKDGHLIVNGKKIRVTAERDPANLKWDEVGVDVVAEATGLFLTDETARKHITAGAKKVVMTGPSKDNTPMFVKGANFDKYAGQDIVSNASCTTNCLAPLAKVINDNFGIIEGLMTTVHATTATQKTVDGPSHKDWRGGRGASQNIIPSSTGAAKAVGKVLPELNGKLTGMAFRVPTPNVSVVDLTVRLEKAATYEQIKAAVKAAAEGEMKGVLGYTEDDVVSTDFNGEVCTSVFDAKAGIALNDNFVKLVSWYDNETGYSNKVLDLIAHISK',\n",
       "  'mw': 35532.0},\n",
       " 'b1781': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P76234',\n",
       "  'gene': 'b1781',\n",
       "  'locus': ['b1781', 'JW1770'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': []},\n",
       "  'seq': 'MQQKMIQFSGDVSLPAVGQGTWYMGEDASQRKTEVAALRAGIELGLTLIDTAEMYADGGAEKVVGEALTGLREKVFLVSKVYPWNAGGQKAINACEASLRRLNTDYLDLYLLHWSGSFAFEETVAAMEKLIAQGKIRRWGVSNLDYADMQELWQLPGGNQCATNQVLYHLGSRGIEYDLLPWCQQQQMPVMAYSPLAQAGRLRNGLLKNAVVNEIAHAHNISAAQVLLAWVISHQGVMAIPKAATIAHVQQNAAVLEVELSSAELAMLDKAYPAPKGKTALDMV',\n",
       "  'mw': 30987.0},\n",
       " 'b1800': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P76251',\n",
       "  'gene': 'b1800',\n",
       "  'locus': ['b1800', 'JW1789'],\n",
       "  'synonyms': 'yeaU',\n",
       "  'properties': {'Catalytic Activity': [('(R)-malate + NAD(+) = CO2 + NADH + pyruvate',\n",
       "     '1.1.1.83')],\n",
       "   'Function': 'Catalyzes the NAD(+)-dependent oxidative decarboxylation of D-malate into pyruvate. Is essential for aerobic growth on D-malate as the sole carbon source. But is not required for anaerobic D-malate utilization, although DmlA is expressed and active in those conditions. Appears to be not able to use L-tartrate as a substrate for dehydrogenation instead of D-malate'},\n",
       "  'seq': 'MMKTMRIAAIPGDGIGKEVLPEGIRVLQAAAERWGFALSFEQMEWASCEYYSHHGKMMPDDWHEQLSRFDAIYFGAVGWPDTVPDHISLWGSLLKFRREFDQYVNLRPVRLFPGVPCPLAGKQPGDIDFYVVRENTEGEYSSLGGRVNEGTEHEVVIQESVFTRRGVDRILRYAFELAQSRPRKTLTSATKSNGLAISMPYWDERVEAMAENYPEIRWDKQHIDILCARFVMQPERFDVVVASNLFGDILSDLGPACTGTIGIAPSANLNPERTFPSLFEPVHGSAPDIYGKNIANPIATIWAGAMMLDFLGNGDERFQQAHNGILAAIEEVIAHGPKTPDMKGNATTPQVADAICKIILR',\n",
       "  'mw': 40315.0},\n",
       " 'b1801': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ABD1',\n",
       "  'gene': 'b1801',\n",
       "  'locus': ['b1801', 'JW5293'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Probable transporter whose substrate is unknown. Is not involved in aerobic D-malate transport'},\n",
       "  'seq': 'MVLLAMGLVIYLATSKYGNIRLGEGKPEYSTLSWLFMFICAGLGSSTLYWGVAEWAYYYQTPGLNIAPRSQQALEFSVPYSFFHWGISAWATYTLASLIMAYHFHVRKNKGLSLSGIIAAITGVRPQGPWGKLVDLMFLIATVGALTISLVVTAATFTRGLSALTGLPDNFTVQAFVILLSGGIFCLSSWIGINNGLQRLSKMVGWGAFLLPLLVLIVGPTEFITNSIINAIGLTTQNFLQMSLFTDPLGDGSFTRNWTVFYWLWWISYTPGVAMFVTRVSRGRKIKEVIWGLILGSTVGCWFFFGVMESYAIHQFINGVINVPQVLETLGGETAVQQVLMSLPAGKLFLAAYLGVMIIFLASHMDAVAYTMAATSTRNLQEGDDPDRGLRLFWCVVITLIPLSILFTGASLETMKTTVVLTALPFLVILLVKVGGFIRWLKQDYADIPAHQVEHYLPQTPVEALEKTPVLPAGTVFKGDN',\n",
       "  'mw': 52881.0},\n",
       " 'b1805': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P69451',\n",
       "  'gene': 'b1805',\n",
       "  'locus': ['b1805', 'JW1794'],\n",
       "  'synonyms': 'oldD',\n",
       "  'properties': {'Catalytic Activity': [('a long-chain fatty acid + ATP + CoA = a long-chain fatty acyl-CoA + AMP + diphosphate',\n",
       "     '6.2.1.3'),\n",
       "    ('ATP + CoA + hexadecanoate = AMP + diphosphate + hexadecanoyl-CoA', ''),\n",
       "    ('ATP + CoA + dodecanoate = AMP + diphosphate + dodecanoyl-CoA', ''),\n",
       "    ('(9Z)-octadecenoate + ATP + CoA = (9Z)-octadecenoyl-CoA + AMP + diphosphate',\n",
       "     ''),\n",
       "    ('ATP + CoA + tetradecanoate = AMP + diphosphate + tetradecanoyl-CoA', ''),\n",
       "    ('ATP + CoA + decanoate = AMP + decanoyl-CoA + diphosphate', ''),\n",
       "    ('ATP + CoA + octanoate = AMP + diphosphate + octanoyl-CoA', '')],\n",
       "   'Function': 'Catalyzes the esterification, concomitant with transport, of exogenous long-chain fatty acids into metabolically active CoA thioesters for subsequent degradation or incorporation into phospholipids. Activity is the highest with fatty acid substrates of > 10 carbon atoms (PubMed:15213221). Is involved in the aerobic beta-oxidative degradation of fatty acids, which allows aerobic growth of E.coli on fatty acids as a sole carbon and energy source (PubMed:12535077)',\n",
       "   'Pathway': 'Lipid metabolism; fatty acid beta-oxidation'},\n",
       "  'seq': 'MKKVWLNRYPADVPTEINPDRYQSLVDMFEQSVARYADQPAFVNMGEVMTFRKLEERSRAFAAYLQQGLGLKKGDRVALMMPNLLQYPVALFGILRAGMIVVNVNPLYTPRELEHQLNDSGASAIVIVSNFAHTLEKVVDKTAVQHVILTRMGDQLSTAKGTVVNFVVKYIKRLVPKYHLPDAISFRSALHNGYRMQYVKPELVPEDLAFLQYTGGTTGVAKGAMLTHRNMLANLEQVNATYGPLLHPGKELVVTALPLYHIFALTINCLLFIELGGQNLLITNPRDIPGLVKELAKYPFTAITGVNTLFNALLNNKEFQQLDFSSLHLSAGGGMPVQQVVAERWVKLTGQYLLEGYGLTECAPLVSVNPYDIDYHSGSIGLPVPSTEAKLVDDDDNEVPPGQPGELCVKGPQVMLGYWQRPDATDEIIKNGWLHTGDIAVMDEEGFLRIVDRKKDMILVSGFNVYPNEIEDVVMQHPGVQEVAAVGVPSGSSGEAVKIFVVKKDPSLTEESLVTFCRRQLTGYKVPKLVEFRDELPKSNVGKILRRELRDEARGKVDNKA',\n",
       "  'mw': 62332.0},\n",
       " 'b1812': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P05041',\n",
       "  'gene': 'b1812',\n",
       "  'locus': ['b1812', 'JW1801'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('chorismate + L-glutamine = 4-amino-4-deoxychorismate + L-glutamate',\n",
       "     '2.6.1.85')],\n",
       "   'Function': 'Part of a heterodimeric complex that catalyzes the two-step biosynthesis of 4-amino-4-deoxychorismate (ADC), a precursor of p-aminobenzoate (PABA) and tetrahydrofolate. In the first step, a glutamine amidotransferase (PabA) generates ammonia as a substrate that, along with chorismate, is used in the second step, catalyzed by aminodeoxychorismate synthase (PabB) to produce ADC. PabB, in the absence of PabA, can catalyze the formation of ADC in the presence of exogenous ammonia',\n",
       "   'Regulation Activity': 'Inhibited by 6-diazo-5-oxo-L-norleucine (DON). The inhibition is competitive with glutamine but uncompetitive with chorismate. Also inhibited by 2-fluorochorismate',\n",
       "   'Pathway': 'Cofactor biosynthesis; tetrahydrofolate biosynthesis; 4-aminobenzoate from chorismate: step 1/2'},\n",
       "  'seq': 'MKTLSPAVITLLWRQDAAEFYFSRLSHLPWAMLLHSGYADHPYSRFDIVVAEPICTLTTFGKETVVSESEKRTTTTDDPLQVLQQVLDRADIRPTHNEDLPFQGGALGLFGYDLGRRFESLPEIAEQDIVLPDMAVGIYDWALIVDHQRHTVSLLSHNDVNARRAWLESQQFSPQEDFTLTSDWQSNMTREQYGEKFRQVQEYLHSGDCYQVNLAQRFHATYSGDEWQAFLQLNQANRAPFSAFLRLEQGAILSLSPERFILCDNSEIQTRPIKGTLPRLPDPQEDSKQAVKLANSAKDRAENLMIVDLMRNDIGRVAVAGSVKVPELFVVEPFPAVHHLVSTITAQLPEQLHASDLLRAAFPGGSITGAPKVRAMEIIDELEPQRRNAWCGSIGYLSFCGNMDTSITIRTLTAINGQIFCSAGGGIVADSQEEAEYQETFDKVNRILKQLEK',\n",
       "  'mw': 50970.0},\n",
       " 'b1814': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P16095',\n",
       "  'gene': 'b1814',\n",
       "  'locus': ['b1814', 'JW1803'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('L-serine = NH4(+) + pyruvate',\n",
       "     '4.3.1.17')],\n",
       "   'Function': 'Deaminates also threonine, particularly when it is present in high concentration',\n",
       "   'Pathway': 'Carbohydrate biosynthesis; gluconeogenesis'},\n",
       "  'seq': 'MISLFDMFKVGIGPSSSHTVGPMKAGKQFVDDLVEKGLLDSVTRVAVDVYGSLSLTGKGHHTDIAIIMGLAGNEPATVDIDSIPGFIRDVEERERLLLAQGRHEVDFPRDNGMRFHNGNLPLHENGMQIHAYNGDEVVYSKTYYSIGGGFIVDEEHFGQDAANEVSVPYPFKSATELLAYCNETGYSLSGLAMQNELALHSKKEIDEYFAHVWQTMQACIDRGMNTEGVLPGPLRVPRRASALRRMLVSSDKLSNDPMNVIDWVNMFALAVNEENAAGGRVVTAPTNGACGIVPAVLAYYDHFIESVSPDIYTRYFMAAGAIGALYKMNASISGAEVGCQGEVGVACSMAAAGLAELLGGSPEQVCVAAEIGMEHNLGLTCDPVAGQVQVPCIERNAIASVKAINAARMALRRTSAPRVSLDKVIETMYETGKDMNAKYRETSRGGLAIKVQCD',\n",
       "  'mw': 48907.0},\n",
       " 'b1817': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P69797',\n",
       "  'gene': 'b1817',\n",
       "  'locus': ['b1817', 'JW1806'],\n",
       "  'synonyms': 'gptB ptsL',\n",
       "  'properties': {'Catalytic Activity': [('D-mannose(out) + N(pros)-phospho-L-histidyl-[protein] = D-mannose 6-phosphate(in) + L-histidyl-[protein]',\n",
       "     '2.7.1.191')],\n",
       "   'Function': 'Also functions as a receptor for bacterial chemotaxis and is required for infection of the cell by bacteriophage lambda where it most likely functions as a pore for penetration of lambda DNA'},\n",
       "  'seq': 'MTIAIVIGTHGWAAEQLLKTAEMLLGEQENVGWIDFVPGENAETLIEKYNAQLAKLDTTKGVLFLVDTWGGSPFNAASRIVVDKEHYEVIAGVNIPMLVETLMARDDDPSFDELVALAVETGREGVKALKAKPVEKAAPAPAAAAPKAAPTPAKPMGPNDYMVIGLARIDDRLIHGQVATRWTKETNVSRIIVVSDEVAADTVRKTLLTQVAPPGVTAHVVDVAKMIRVYNNPKYAGERVMLLFTNPTDVERLVEGGVKITSVNVGGMAFRQGKTQVNNAVSVDEKDIEAFKKLNARGIELEVRKVSTDPKLKMMDLISKIDK',\n",
       "  'mw': 35048.0},\n",
       " 'b1818': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P69801',\n",
       "  'gene': 'b1818',\n",
       "  'locus': ['b1818', 'JW1807'],\n",
       "  'synonyms': 'pel ptsP',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Also functions as a receptor for bacterial chemotaxis and is required for infection of the cell by bacteriophage lambda where it most likely functions as a pore for penetration of lambda DNA'},\n",
       "  'seq': 'MEITTLQIVLVFIVACIAGMGSILDEFQFHRPLIACTLVGIVLGDMKTGIIIGGTLEMIALGWMNIGAAVAPDAALASIISTILVIAGHQSIGAGIALAIPLAAAGQVLTIIVRTITVAFQHAADKAADNGNLTAISWIHVSSLFLQAMRVAIPAVIVALSVGTSEVQNMLNAIPEVVTNGLNIAGGMIVVVGYAMVINMMRAGYLMPFFYLGFVTAAFTNFNLVALGVIGTVMAVLYIQLSPKYNRVAGAPAQAAGNNDLDNELD',\n",
       "  'mw': 27636.0},\n",
       " 'b1819': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P69805',\n",
       "  'gene': 'b1819',\n",
       "  'locus': ['b1819', 'JW1808'],\n",
       "  'synonyms': 'gptB ptsM',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Also functions as a receptor for bacterial chemotaxis and is required for infection of the cell by bacteriophage lambda where it most likely functions as a pore for penetration of lambda DNA'},\n",
       "  'seq': 'MVDTTQTTTEKKLTQSDIRGVFLRSNLFQGSWNFERMQALGFCFSMVPAIRRLYPENNEARKQAIRRHLEFFNTQPFVAAPILGVTLALEEQRANGAEIDDGAINGIKVGLMGPLAGVGDPIFWGTVRPVFAALGAGIAMSGSLLGPLLFFILFNLVRLATRYYGVAYGYSKGIDIVKDMGGGFLQKLTEGASILGLFVMGALVNKWTHVNIPLVVSRITDQTGKEHVTTVQTILDQLMPGLVPLLLTFACMWLLRKKVNPLWIIVGFFVIGIAGYACGLLGL',\n",
       "  'mw': 30955.0},\n",
       " 'b1849': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P33221',\n",
       "  'gene': 'b1849',\n",
       "  'locus': ['b1849', 'JW1838'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + formate + N(1)-(5-phospho-beta-D-ribosyl)glycinamide = ADP + H(+) + N(2)-formyl-N(1)-(5-phospho-beta-D-ribosyl)glycinamide + phosphate',\n",
       "     '6.3.1.21'),\n",
       "    ('acetate + ATP = acetyl phosphate + ADP', ''),\n",
       "    ('ATP + carbamate = ADP + carbamoyl phosphate', '')],\n",
       "   'Function': 'Involved in the de novo purine biosynthesis (PubMed:8501063, PubMed:8117714, PubMed:9184151). Catalyzes the transfer of formate to 5-phospho-ribosyl-glycinamide (GAR), producing 5-phospho-ribosyl-N-formylglycinamide (FGAR) (PubMed:8501063, PubMed:8117714, PubMed:9184151). Formate is provided by PurU via hydrolysis of 10-formyl-tetrahydrofolate (PubMed:8226647). PurT is also able to cleave acetyl phosphate and carbamoyl phosphate to produce ATP with acetate and carbamate, respectively (PubMed:8117714, PubMed:9184151)',\n",
       "   'Pathway': 'Purine metabolism; IMP biosynthesis via de novo pathway; N(2)-formyl-N(1)-(5-phospho-D-ribosyl)glycinamide from N(1)-(5-phospho-D-ribosyl)glycinamide (formate route): step 1/1'},\n",
       "  'seq': 'MTLLGTALRPAATRVMLLGSGELGKEVAIECQRLGVEVIAVDRYADAPAMHVAHRSHVINMLDGDALRRVVELEKPHYIVPEIEAIATDMLIQLEEEGLNVVPCARATKLTMNREGIRRLAAEELQLPTSTYRFADSESLFREAVADIGYPCIVKPVMSSSGKGQTFIRSAEQLAQAWKYAQQGGRAGAGRVIVEGVVKFDFEITLLTVSAVDGVHFCAPVGHRQEDGDYRESWQPQQMSPLALERAQEIARKVVLALGGYGLFGVELFVCGDEVIFSEVSPRPHDTGMVTLISQDLSEFALHVRAFLGLPVGGIRQYGPAASAVILPQLTSQNVTFDNVQNAVGADLQIRLFGKPEIDGSRRLGVALATAESVVDAIERAKHAAGQVKVQG',\n",
       "  'mw': 42434.0},\n",
       " 'b1850': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A955',\n",
       "  'gene': 'b1850',\n",
       "  'locus': ['b1850', 'JW1839'],\n",
       "  'synonyms': 'hga kdgA',\n",
       "  'properties': {'Catalytic Activity': [('(4S)-4-hydroxy-2-oxoglutarate = glyoxylate + pyruvate',\n",
       "     '4.1.3.16'),\n",
       "    ('(4R)-4-hydroxy-2-oxoglutarate = glyoxylate + pyruvate', '4.1.3.16'),\n",
       "    ('2-dehydro-3-deoxy-6-phospho-D-gluconate = D-glyceraldehyde 3-phosphate + pyruvate',\n",
       "     '4.1.2.14')],\n",
       "   'Function': 'Involved in the degradation of glucose via the Entner-Doudoroff pathway. Catalyzes the reversible, stereospecific retro-aldol cleavage of 2-Keto-3-deoxy-6-phosphogluconate (KDPG) to pyruvate and D-glyceraldehyde-3-phosphate. In the synthetic direction, it catalyzes the addition of pyruvate to electrophilic aldehydes with si-facial selectivity. It accepts some nucleophiles other than pyruvate, including 2-oxobutanoate, phenylpyruvate, and fluorobutanoate. It has a preference for the S-configuration at C2 of the electrophile',\n",
       "   'Pathway': 'Carbohydrate metabolism; glyoxylate and dicarboxylate metabolism'},\n",
       "  'seq': 'MKNWKTSAESILTTGPVVPVIVVKKLEHAVPMAKALVAGGVRVLEVTLRTECAVDAIRAIAKEVPEAIVGAGTVLNPQQLAEVTEAGAQFAISPGLTEPLLKAATEGTIPLIPGISTVSELMLGMDYGLKEFKFFPAEANGGVKALQAIAGPFSQVRFCPTGGISPANYRDYLALKSVLCIGGSWLVPADALEAGDYDRITKLAREAVEGAKL',\n",
       "  'mw': 22284.0},\n",
       " 'b1851': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ADF6',\n",
       "  'gene': 'b1851',\n",
       "  'locus': ['b1851', 'JW1840'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('6-phospho-D-gluconate = 2-dehydro-3-deoxy-6-phospho-D-gluconate + H2O',\n",
       "     '4.2.1.12')],\n",
       "   'Function': 'Catalyzes the dehydration of 6-phospho-D-gluconate to 2-dehydro-3-deoxy-6-phospho-D-gluconate',\n",
       "   'Regulation Activity': 'Sensitive to oxidants such as superoxide or hydrogen peroxide',\n",
       "   'Pathway': 'Carbohydrate metabolism; Entner-Doudoroff pathway'},\n",
       "  'seq': 'MNPQLLRVTNRIIERSRETRSAYLARIEQAKTSTVHRSQLACGNLAHGFAACQPEDKASLKSMLRNNIAIITSYNDMLSAHQPYEHYPEIIRKALHEANAVGQVAGGVPAMCDGVTQGQDGMELSLLSREVIAMSAAVGLSHNMFDGALFLGVCDKIVPGLTMAALSFGHLPAVFVPSGPMASGLPNKEKVRIRQLYAEGKVDRMALLESEAASYHAPGTCTFYGTANTNQMVVEFMGMQLPGSSFVHPDSPLRDALTAAAARQVTRMTGNGNEWMPIGKMIDEKVVVNGIVALLATGGSTNHTMHLVAMARAAGIQINWDDFSDLSDVVPLMARLYPNGPADINHFQAAGGVPVLVRELLKAGLLHEDVNTVAGFGLSRYTLEPWLNNGELDWREGAEKSLDSNVIASFEQPFSHHGGTKVLSGNLGRAVMKTSAVPVENQVIEAPAVVFESQHDVMPAFEAGLLDRDCVVVVRHQGPKANGMPELHKLMPPLGVLLDRCFKIALVTDGRLSGASGKVPSAIHVTPEAYDGGLLAKVRDGDIIRVNGQTGELTLLVDEAELAAREPHIPDLSASRVGTGRELFSALREKLSGAEQGATCITF',\n",
       "  'mw': 64639.0},\n",
       " 'b1852': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AC53',\n",
       "  'gene': 'b1852',\n",
       "  'locus': ['b1852', 'JW1841'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('D-glucose 6-phosphate + NADP(+) = 6-phospho-D-glucono-1,5-lactone + H(+) + NADPH',\n",
       "     '1.1.1.49')],\n",
       "   'Function': \"Probable source of extracellular death factor (EDF, sequence Asn-Asn-Trp-Asn-Asn, NNWNN) following processing and amidation. This pentapeptide stimulates cell death mediated by MazF (PubMed:17962566). Artificial peptides with altered sequence show that NNGNN, GNWNG and NWN no longer stimulate MazF's endoribonuclease activity; other peptides (NNGN, GNWMM, NNWNG, NNNWNNN) retain MazF-stimulating activity. NNWNN, NNGN, GNWMM and NNWNG prevent cognate antitoxin MazE from inhibiting MazF; although NNNWNNN stimulates MazF it does not do so in the presence of MazE. EDF also stimulates ChpB's endoribonuclease activity in vitro; in this case NWN partially stimulates ChpB, whereas NNGNN, GNWNN, NNWNG, GNWNG and NNNWNNN do not. Only the wild-type EDF peptide prevents cognate antitoxin ChpS from inhibiting ChpB (PubMed:21419338)\",\n",
       "   'Pathway': 'Carbohydrate degradation; pentose phosphate pathway; D-ribulose 5-phosphate from D-glucose 6-phosphate (oxidative stage): step 1/3'},\n",
       "  'seq': 'MAVTQTAQACDLVIFGAKGDLARRKLLPSLYQLEKAGQLNPDTRIIGVGRADWDKAAYTKVVREALETFMKETIDEGLWDTLSARLDFCNLDVNDTAAFSRLGAMLDQKNRITINYFAMPPSTFGAICKGLGEAKLNAKPARVVMEKPLGTSLATSQEINDQVGEYFEECQVYRIDHYLGKETVLNLLALRFANSLFVNNWDNRTIDHVEITVAEEVGIEGRWGYFDKAGQMRDMIQNHLLQILCMIAMSPPSDLSADSIRDEKVKVLKSLRRIDRSNVREKTVRGQYTAGFAQGKKVPGYLEEEGANKSSNTETFVAIRVDIDNWRWAGVPFYLRTGKRLPTKCSEVVVYFKTPELNLFKESWQDLPQNKLTIRLQPDEGVDIQVLNKVPGLDHKHNLQITKLDLSYSETFNQTHLADAYERLLLETMRGIQALFVRRDEVEEAWKWVDSITEAWAMDNDAPKPYQAGTWGPVASVAMITRDGRSWNEFE',\n",
       "  'mw': 55704.0},\n",
       " 'b1854': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P21599',\n",
       "  'gene': 'b1854',\n",
       "  'locus': ['b1854', 'JW1843'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + pyruvate = ADP + H(+) + phosphoenolpyruvate',\n",
       "     '2.7.1.40')],\n",
       "   'Function': 'Catalyzes the formation of pyruvate in the last step of glycolysis, it is irreversible under physiological conditions. The reaction is critical for the control of metabolic flux in the second part of glycolysis',\n",
       "   'Regulation Activity': 'Allosterically activated by AMP and by several sugar phosphates. Belongs to type II PK',\n",
       "   'Pathway': 'Carbohydrate degradation; glycolysis; pyruvate from D-glyceraldehyde 3-phosphate: step 5/5'},\n",
       "  'seq': 'MSRRLRRTKIVTTLGPATDRDNNLEKVIAAGANVVRMNFSHGSPEDHKMRADKVREIAAKLGRHVAILGDLQGPKIRVSTFKEGKVFLNIGDKFLLDANLGKGEGDKEKVGIDYKGLPADVVPGDILLLDDGRVQLKVLEVQGMKVFTEVTVGGPLSNNKGINKLGGGLSAEALTEKDKADIKTAALIGVDYLAVSFPRCGEDLNYARRLARDAGCDAKIVAKVERAEAVCSQDAMDDIILASDVVMVARGDLGVEIGDPELVGIQKALIRRARQLNRAVITATQMMESMITNPMPTRAEVMDVANAVLDGTDAVMLSAETAAGQYPSETVAAMARVCLGAEKIPSINVSKHRLDVQFDNVEEAIAMSAMYAANHLKGVTAIITMTESGRTALMTSRISSGLPIFAMSRHERTLNLTALYRGVTPVHFDSANDGVAAASEAVNLLRDKGYLMSGDLVIVTQGDVMSTVGSTNTTRILTVE',\n",
       "  'mw': 51357.0},\n",
       " 'b1855': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P24205',\n",
       "  'gene': 'b1855',\n",
       "  'locus': ['b1855', 'JW1844'],\n",
       "  'synonyms': 'msbB',\n",
       "  'properties': {'Catalytic Activity': [('alpha-Kdo-(2->4)-alpha-Kdo-(2->6)-(dodecanoyl)-lipid IVA (E. coli) + tetradecanoyl-[ACP] = alpha-Kdo-(2->4)-alpha-Kdo-(2->6)-lipid A (E. coli) + holo-[ACP]',\n",
       "     '2.3.1.243'),\n",
       "    ('(9Z)-hexadecenoyl-(Kdo)2-lipid IVA (E. coli) + tetradecanoyl-[ACP] = ((9Z)-hexadecenoyl-tetradecanoyl)-(Kdo)2-lipid A + holo-[ACP]',\n",
       "     '')],\n",
       "   'Function': 'Catalyzes the transfer of myristate from myristoyl-[acyl-carrier-protein] (ACP) to Kdo(2)-(lauroyl)-lipid IV(A) to form Kdo(2)-lipid A. Can probably also catalyze the transfer of myristate to Kdo(2)-(palmitoleoyl)-lipid IV(A) to form the cold-adapted Kdo(2)-lipid A. In vitro, can acylate Kdo(2)-lipid IV(A), but acylation of (KDO)2-(lauroyl)-lipid IV(A) is about 100 times faster. In vitro, can use lauroyl-ACP but displays a slight kinetic preference for myristoyl-ACP',\n",
       "   'Pathway': 'Bacterial outer membrane biogenesis; lipopolysaccharide biosynthesis'},\n",
       "  'seq': 'METKKNNSEYIPEFDKSFRHPRYWGAWLGVAAMAGIALTPPKFRDPILARLGRFAGRLGKSSRRRALINLSLCFPERSEAEREAIVDEMFATAPQAMAMMAELAIRGPEKIQPRVDWQGLEIIEEMRRNNEKVIFLVPHGWAVDIPAMLMASQGQKMAAMFHNQGNPVFDYVWNTVRRRFGGRLHARNDGIKPFIQSVRQGYWGYYLPDQDHGPEHSEFVDFFATYKATLPAIGRLMKVCRARVVPLFPIYDGKTHRLTIQVRPPMDDLLEADDHTIARRMNEEVEIFVGPRPEQYTWILKLLKTRKPGEIQPYKRKDLYPIK',\n",
       "  'mw': 37410.0},\n",
       " 'b1857': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P39172',\n",
       "  'gene': 'b1857',\n",
       "  'locus': ['b1857', 'JW5831'],\n",
       "  'synonyms': 'yebL yzzP',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Involved in the high-affinity zinc uptake transport system'},\n",
       "  'seq': 'MLHKKTLLFAALSAALWGGATQAADAAVVASLKPVGFIASAIADGVTETEVLLPDGASEHDYSLRPSDVKRLQNADLVVWVGPEMEAFMQKPVSKLPGAKQVTIAQLEDVKPLLMKSIHGDDDDHDHAEKSDEDHHHGDFNMHLWLSPEIARATAVAIHGKLVELMPQSRAKLDANLKDFEAQLASTETQVGNELAPLKGKGYFVFHDAYGYFEKQFGLTPLGHFTVNPEIQPGAQRLHEIRTQLVEQKATCVFAEPQFRPAVVESVARGTSVRMGTLDPLGTNIKLGKTSYSEFLSQLANQYASCLKGD',\n",
       "  'mw': 33777.0},\n",
       " 'b1858': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A9X1',\n",
       "  'gene': 'b1858',\n",
       "  'locus': ['b1858', 'JW1847'],\n",
       "  'synonyms': 'yebM',\n",
       "  'properties': {'Catalytic Activity': [('ATP(in) + H2O(in) + Zn(2+)(out) = ADP(in) + H(+)(in) + phosphate(in) + Zn(2+)(in)',\n",
       "     '7.2.2.20')],\n",
       "   'Function': 'Part of the ABC transporter complex ZnuABC involved in zinc import. Responsible for energy coupling to the transport system',\n",
       "   'Regulation Activity': 'Inhibited by arsenate'},\n",
       "  'seq': 'MTSLVSLENVSVSFGQRRVLSDVSLELKPGKILTLLGPNGAGKSTLVRVVLGLVTPDEGVIKRNGKLRIGYVPQKLYLDTTLPLTVNRFLRLRPGTHKEDILPALKRVQAGHLINAPMQKLSGGETQRVLLARALLNRPQLLVLDEPTQGVDVNGQVALYDLIDQLRRELDCGVLMVSHDLHLVMAKTDEVLCLNHHICCSGTPEVVSLHPEFISMFGPRGAEQLGIYRHHHNHRHDLQGRIVLRRGNDRS',\n",
       "  'mw': 27867.0},\n",
       " 'b1859': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P39832',\n",
       "  'gene': 'b1859',\n",
       "  'locus': ['b1859', 'JW1848'],\n",
       "  'synonyms': 'yebI',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Involved in the high-affinity zinc uptake transport system'},\n",
       "  'seq': 'MIELLFPGWLAGIMLACAAGPLGSFVVWRRMSYFGDTLAHASLLGVAFGLLLDVNPFYAVIAVTLLLAGGLVWLEKRPQLAIDTLLGIMAHSALSLGLVVVSLMSNIRVDLMAYLFGDLLAVTPEDLISIAIGVVIVVAILFWQWRNLLSMTISPDLAFVDGVKLQRVKLLLMLVTALTIGVAMKFVGALIITSLLIIPAATARRFARTPEQMAGVAVLVGMVAVTGGLTFSAVYDTPAGPSVVLCAALLFILSMMKKQAS',\n",
       "  'mw': 27729.0},\n",
       " 'b1865': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AFC0',\n",
       "  'gene': 'b1865',\n",
       "  'locus': ['b1865', 'JW1854'],\n",
       "  'synonyms': 'ntpA',\n",
       "  'properties': {'Catalytic Activity': [(\"7,8-dihydroneopterin 3'-triphosphate + H2O = 7,8-dihydroneopterin 3'-phosphate + diphosphate + H(+)\",\n",
       "     '3.6.1.67')],\n",
       "   'Function': 'Catalyzes the hydrolysis of dihydroneopterin triphosphate to dihydroneopterin monophosphate and pyrophosphate. Required for efficient folate biosynthesis. Can also hydrolyze nucleoside triphosphates with a preference for dATP',\n",
       "   'Regulation Activity': 'Subject to product inhibition by pyrophosphate'},\n",
       "  'seq': 'MKDKVYKRPVSILVVIYAQDTKRVLMLQRRDDPDFWQSVTGSVEEGETAPQAAMREVKEEVTIDVVAEQLTLIDCQRTVEFEIFSHLRHRYAPGVTRNTESWFCLALPHERQIVFTEHLAYKWLDAPAAAALTKSWSNRQAIEQFVINAA',\n",
       "  'mw': 17306.0},\n",
       " 'b1866': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P21889',\n",
       "  'gene': 'b1866',\n",
       "  'locus': ['b1866', 'JW1855'],\n",
       "  'synonyms': 'tls',\n",
       "  'properties': {'Catalytic Activity': [('ATP + L-aspartate + tRNA(Asp) = AMP + diphosphate + L-aspartyl-tRNA(Asp)',\n",
       "     '6.1.1.12')],\n",
       "   'Function': 'Catalyzes the attachment of L-aspartate to tRNA(Asp) in a two-step reaction: L-aspartate is first activated by ATP to form Asp-AMP and then transferred to the acceptor end of tRNA(Asp). Also mischarges tRNA(Asp) with D-aspartate, although it is a poor substrate (PubMed:10918062)'},\n",
       "  'seq': 'MRTEYCGQLRLSHVGQQVTLCGWVNRRRDLGSLIFIDMRDREGIVQVFFDPDRADALKLASELRNEFCIQVTGTVRARDEKNINRDMATGEIEVLASSLTIINRADVLPLDSNHVNTEEARLKYRYLDLRRPEMAQRLKTRAKITSLVRRFMDDHGFLDIETPMLTKATPEGARDYLVPSRVHKGKFYALPQSPQLFKQLLMMSGFDRYYQIVKCFRDEDLRADRQPEFTQIDVETSFMTAPQVREVMEALVRHLWLEVKGVDLGDFPVMTFAEAERRYGSDKPDLRNPMELTDVADLLKSVEFAVFAGPANDPKGRVAALRVPGGASLTRKQIDEYGNFVKIYGAKGLAYIKVNERAKGLEGINSPVAKFLNAEIIEDILDRTAAQDGDMIFFGADNKKIVADAMGALRLKVGKDLGLTDESKWAPLWVIDFPMFEDDGEGGLTAMHHPFTSPKDMTAAELKAAPENAVANAYDMVINGYEVGGGSVRIHNGDMQQTVFGILGINEEEQREKFGFLLDALKYGTPPHAGLAFGLDRLTMLLTGTDNIRDVIAFPKTTAAACLMTEAPSFANPTALAELSIQVVKKAENN',\n",
       "  'mw': 65913.0},\n",
       " 'b1872': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P46923',\n",
       "  'gene': 'b1872',\n",
       "  'locus': ['b1872', 'JW1861'],\n",
       "  'synonyms': 'bisZ',\n",
       "  'properties': {'Catalytic Activity': [('2 Fe(III)-[cytochrome c] + H2O + trimethylamine = 2 Fe(II)-[cytochrome c] + 3 H(+) + trimethylamine N-oxide',\n",
       "     '1.7.2.3')],\n",
       "   'Function': 'Reduces trimethylamine-N-oxide (TMAO) into trimethylamine; an anaerobic reaction coupled to energy-yielding reactions. Can also reduce other N- and S-oxide compounds such as 4-methylmorpholine-N-oxide and biotin sulfoxide (BSO), but with a lower catalytic efficiency'},\n",
       "  'seq': 'MTLTRREFIKHSGIAAGALVVTSAAPLPAWAEEKGGKILTAGRWGAMNVEVKDGKIVSSTGALAKTIPNSLQSTAADQVHTTARIQHPMVRKSYLDNPLQPAKGRGEDTYVQVSWEQALKLIHEQHDRIRKANGPSAIFAGSYGWRSSGVLHKAQTLLQRYMNLAGGYSGHSGDYSTGAAQVIMPHVVGSVEVYEQQTSWPLILENSQVVVLWGMNPLNTLKIAWSSTDEQGLEYFHQLKKSGKPVIAIDPIRSETIEFFDDNATWIAPNMGTDVALMLGIAHTLMTQGKHDKVFLEKYTTGYPQFEEYLTGKSDNTPKSAVWAAEITGVPEAQIVKLAELMAANRTMLMAGWGIQRQQYGEQKHWMLVTLAAMLGQIGTPGGGFGFSYHYSNGGNPTRVGGVLPEMSAAIAGHASEAADDGGMTAIPVARIVDALENPGGKYQHNGKEQTYPNIKMIWWAGGGNFTHHQDTNRLIKAWQKPEMIVVSECYWTAAAKHADIVLPITTSFERNDLTMTGDYSNQHIVPMKQAVAPQFEARNDFDVFADLAELLKPGGKEIYTEGKDEMAWLKFFYDAAQKGARAQRVTMPMFNAFWQQNKLIEMRHSEKNEQYVRYGDFRADPVKNALGTPSGKIEIYSKTLEKFGYKDCPAHPTWLAPDEWKGTADEKQLQLLTAHPAHRLHSQLNYAELRKKYAIADREPITIHTEDAARFGIANGDLVRVWNKRGQILTGAVVTDGIKKGVVCVHEGAWPDLENGLCKNGSANVLTADIPSSQLANACAGNSALVYIEKYTGNAPKLTAFDQPAVQA',\n",
       "  'mw': 88964.0},\n",
       " 'b1873': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P52005',\n",
       "  'gene': 'b1873',\n",
       "  'locus': ['b1873', 'JW1862'],\n",
       "  'synonyms': 'yecK',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the anaerobic respiratory chain of trimethylamine-N-oxide reductase TorZ. Required for electron transfer to the TorZ terminal enzyme'},\n",
       "  'seq': 'MRGKKRIGLLFLLIAVVVGGGGLLLAQKVLHKTSDTAFCLSCHSMSKPFEEYQGTVHFSNQKGIRAECADCHIPKSGMDYLFAKLKASKDIYHEFVSGKIDSDDKFEAHRQEMAETVWKELKATDSATCRSCHSFDAMDIASQSESAQKMHNKAQKDSETCIDCHKGIAHFPPEIKMDDNAAHELESQAATSVTNGAHIYPFKTSHIGELATVNPGTDLTVVDASGKQPIVLLQGYQMQGSENTLYLAAGQRLALATLSEEGIKALTVNGEWQADEYGNQWRQASLQGALTDPALADRKPLWQYAEKLDDTYCAGCHAPIAADHYTVNAWPSIAKGMGARTSMSENELDILTRYFQYNAKDITEKQ',\n",
       "  'mw': 40286.0},\n",
       " 'b1876': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P11875',\n",
       "  'gene': 'b1876',\n",
       "  'locus': ['b1876', 'JW1865'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + L-arginine + tRNA(Arg) = AMP + diphosphate + L-arginyl-tRNA(Arg)',\n",
       "     '6.1.1.19')]},\n",
       "  'seq': 'MNIQALLSEKVRQAMIAAGAPADCEPQVRQSAKVQFGDYQANGMMAVAKKLGMAPRQLAEQVLTHLDLNGIASKVEIAGPGFINIFLDPAFLAEHVQQALASDRLGVATPEKQTIVVDYSAPNVAKEMHVGHLRSTIIGDAAVRTLEFLGHKVIRANHVGDWGTQFGMLIAWLEKQQQENAGEMELADLEGFYRDAKKHYDEDEEFAERARNYVVKLQSGDEYFREMWRKLVDITMTQNQITYDRLNVTLTRDDVMGESLYNPMLPGIVADLKAKGLAVESEGATVVFLDEFKNKEGEPMGVIIQKKDGGYLYTTTDIACAKYRYETLHADRVLYYIDSRQHQHLMQAWAIVRKAGYVPESVPLEHHMFGMMLGKDGKPFKTRAGGTVKLADLLDEALERARRLVAEKNPDMPADELEKLANAVGIGAVKYADLSKNRTTDYIFDWDNMLAFEGNTAPYMQYAYTRVLSVFRKAEIDEEQLAAAPVIIREDREAQLAARLLQFEETLTVVAREGTPHVMCAYLYDLAGLFSGFYEHCPILSAENEEVRNSRLKLAQLTAKTLKLGLDTLGIETVERM',\n",
       "  'mw': 64683.0},\n",
       " 'b1896': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P31677',\n",
       "  'gene': 'b1896',\n",
       "  'locus': ['b1896', 'JW5312'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('D-glucose 6-phosphate + UDP-alpha-D-glucose = alpha,alpha-trehalose 6-phosphate + H(+) + UDP',\n",
       "     '2.4.1.15')],\n",
       "   'Function': 'Catalyzes the transfer of glucose from UDP-alpha-D-glucose (UDP-Glc) to D-glucose 6-phosphate (Glc-6-P) to form trehalose-6-phosphate. Acts with retention of the anomeric configuration of the UDP-sugar donor. Essential for viability of the cells at low temperatures and at elevated osmotic strength',\n",
       "   'Regulation Activity': \"Activated by potassium and other monovalent cations at 0.25 M, but partially inhibited at greater concentrations (PubMed:3131312). Inhibited by validoxylamine A 6'-O-phosphate (PubMed:20077550)\",\n",
       "   'Pathway': 'Glycan biosynthesis; trehalose biosynthesis'},\n",
       "  'seq': 'MSRLVVVSNRIAPPDEHAASAGGLAVGILGALKAAGGLWFGWSGETGNEDQPLKKVKKGNITWASFNLSEQDLDEYYNQFSNAVLWPAFHYRLDLVQFQRPAWDGYLRVNALLADKLLPLLQDDDIIWIHDYHLLPFAHELRKRGVNNRIGFFLHIPFPTPEIFNALPTYDTLLEQLCDYDLLGFQTENDRLAFLDCLSNLTRVTTRSAKSHTAWGKAFRTEVYPIGIEPKEIAKQAAGPLPPKLAQLKAELKNVQNIFSVERLDYSKGLPERFLAYEALLEKYPQHHGKIRYTQIAPTSRGDVQAYQDIRHQLENEAGRINGKYGQLGWTPLYYLNQHFDRKLLMKIFRYSDVGLVTPLRDGMNLVAKEYVAAQDPANPGVLVLSQFAGAANELTSALIVNPYDRDEVAAALDRALTMSLAERISRHAEMLDVIVKNDINHWQECFISDLKQIVPRSAESQQRDKVATFPKLA',\n",
       "  'mw': 53611.0},\n",
       " 'b1897': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P31678',\n",
       "  'gene': 'b1897',\n",
       "  'locus': ['b1897', 'JW1886'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('alpha,alpha-trehalose 6-phosphate + H2O = alpha,alpha-trehalose + phosphate',\n",
       "     '3.1.3.12')],\n",
       "   'Function': 'Removes the phosphate from trehalose 6-phosphate (Tre6P) to produce free trehalose. Also catalyzes the dephosphorylation of glucose-6-phosphate (Glu6P) and 2-deoxyglucose-6-phosphate (2dGlu6P)',\n",
       "   'Pathway': 'Glycan biosynthesis; trehalose biosynthesis'},\n",
       "  'seq': 'MTEPLTETPELSAKYAWFFDLDGTLAEIKPHPDQVVVPDNILQGLQLLATASDGALALISGRSMVELDALAKPYRFPLAGVHGAERRDINGKTHIVHLPDAIARDISVQLHTVIAQYPGAELEAKGMAFALHYRQAPQHEDALMTLAQRITQIWPQMALQQGKCVVEIKPRGTSKGEAIAAFMQEAPFIGRTPVFLGDDLTDESGFAVVNRLGGMSVKIGTGATQASWRLAGVPDVWSWLEMITTALQQKRENNRSDDYESFSRSI',\n",
       "  'mw': 29175.0},\n",
       " 'b1900': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AAF3',\n",
       "  'gene': 'b1900',\n",
       "  'locus': ['b1900', 'JW1888'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + H2O + L-arabinose(out) = ADP + H(+) + L-arabinose(in) + phosphate',\n",
       "     '7.5.2.12')],\n",
       "   'Function': 'Part of the ABC transporter complex AraFGH involved in arabinose import. Responsible for energy coupling to the transport system (Probable)'},\n",
       "  'seq': 'MQQSTPYLSFRGIGKTFPGVKALTDISFDCYAGQVHALMGENGAGKSTLLKILSGNYAPTTGSVVINGQEMSFSDTTAALNAGVAIIYQELHLVPEMTVAENIYLGQLPHKGGIVNRSLLNYEAGLQLKHLGMDIDPDTPLKYLSIGQWQMVEIAKALARNAKIIAFDEPTSSLSAREIDNLFRVIRELRKEGRVILYVSHRMEEIFALSDAITVFKDGRYVKTFTDMQQVDHDALVQAMVGRDIGDIYGWQPRSYGEERLRLDAVKAPGVRTPISLAVRSGEIVGLFGLVGAGRSELMKGMFGGTQITAGQVYIDQQPIDIRKPSHAIAAGMMLCPEDRKAEGIIPVHSVRDNINISARRKHVLGGCVINNGWEENNADHHIRSLNIKTPGAEQLIMNLSGGNQQKAILGRWLSEEMKVILLDEPTRGIDVGAKHEIYNVIYALAAQGVAVLFASSDLPEVLGVADRIVVMREGEIAGELLHEQADERQALSLAMPKVSQAVA',\n",
       "  'mw': 55018.0},\n",
       " 'b1901': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P02924',\n",
       "  'gene': 'b1901',\n",
       "  'locus': ['b1901', 'JW1889'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Involved in the high-affinity L-arabinose membrane transport system. Binds with high affinity to arabinose, but can also bind D-galactose (approximately 2-fold reduction) and D-fucose (approximately 40-fold reduction)'},\n",
       "  'seq': 'MHKFTKALAAIGLAAVMSQSAMAENLKLGFLVKQPEEPWFQTEWKFADKAGKDLGFEVIKIAVPDGEKTLNAIDSLAASGAKGFVICTPDPKLGSAIVAKARGYDMKVIAVDDQFVNAKGKPMDTVPLVMMAATKIGERQGQELYKEMQKRGWDVKESAVMAITANELDTARRRTTGSMDALKAAGFPEKQIYQVPTKSNDIPGAFDAANSMLVQHPEVKHWLIVGMNDSTVLGGVRATEGQGFKAADIIGIGINGVDAVSELSKAQATGFYGSLLPSPDVHGYKSSEMLYNWVAKDVEPPKFTEVTDVVLITRDNFKEELEKKGLGGK',\n",
       "  'mw': 35541.0},\n",
       " 'b1907': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AAD4',\n",
       "  'gene': 'b1907',\n",
       "  'locus': ['b1907', 'JW1895'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('H(+)(in) + L-tyrosine(in) = H(+)(out) + L-tyrosine(out)',\n",
       "     '')],\n",
       "   'Function': 'Transports tyrosine across the cytoplasmic membrane (PubMed:6090409). The transport system is energized by the proton motive force (PubMed:6090409)',\n",
       "   'Regulation Activity': 'Inhibited by the uncoupler carbonyl cyanide m-chlorophenylhydrazone (CCCP)'},\n",
       "  'seq': 'MKNRTLGSVFIVAGTTIGAGMLAMPLAAAGVGFSVTLILLIGLWALMCYTALLLLEVYQHVPADTGLGTLAKRYLGRYGQWLTGFSMMFLMYALTAAYISGAGELLASSISDWTGISMSATAGVLLFTFVAGGVVCVGTSLVDLFNRFLFSAKIIFLVVMLVLLLPHIHKVNLLTLPLQQGLALSAIPVIFTSFGFHGSVPSIVSYMDGNIRKLRWVFIIGSAIPLVAYIFWQVATLGSIDSTTFMGLLANHAGLNGLLQALREMVASPHVELAVHLFADLALATSFLGVALGLFDYLADLFQRSNTVGGRLQTGAITFLPPLAFALFYPRGFVMALGYAGVALAVLALIIPSLLTWQSRKHNPQAGYRVKGGRPALVVVFLCGIAVIGVQFLIAAGLLPEVG',\n",
       "  'mw': 42819.0},\n",
       " 'b1912': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ABF8',\n",
       "  'gene': 'b1912',\n",
       "  'locus': ['b1912', 'JW1897'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [(\"a CDP-1,2-diacyl-sn-glycerol + sn-glycerol 3-phosphate = 1,2-diacyl-sn-glycero-3-phospho-(1'-sn-glycero-3'-phosphate) + CMP + H(+)\",\n",
       "     '2.7.8.5')],\n",
       "   'Function': 'This protein catalyzes the committed step to the synthesis of the acidic phospholipids',\n",
       "   'Pathway': 'Phospholipid metabolism; phosphatidylglycerol biosynthesis; phosphatidylglycerol from CDP-diacylglycerol: step 1/2'},\n",
       "  'seq': 'MQFNIPTLLTLFRVILIPFFVLVFYLPVTWSPFAAALIFCVAAVTDWFDGFLARRWNQSTRFGAFLDPVADKVLVAIAMVLVTEHYHSWWVTLPAATMIAREIIISALREWMAELGKRSSVAVSWIGKVKTTAQMVALAWLLWRPNIWVEYAGIALFFVAAVLTLWSMLQYLSAARADLLDQ',\n",
       "  'mw': 20701.0},\n",
       " 'b1919': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P76316',\n",
       "  'gene': 'b1919',\n",
       "  'locus': ['b1919', 'JW5313'],\n",
       "  'synonyms': 'yedO',\n",
       "  'properties': {'Catalytic Activity': [('D-cysteine + H2O = H(+) + hydrogen sulfide + NH4(+) + pyruvate',\n",
       "     '4.4.1.15')],\n",
       "   'Function': 'Catalyzes the alpha,beta-elimination reaction of D-cysteine and of several D-cysteine derivatives. It could be a defense mechanism against D-cysteine. Can also catalyze the degradation of 3-chloro-D-alanine'},\n",
       "  'seq': 'MPLHNLTRFPRLEFIGAPTPLEYLPRFSDYLGREIFIKRDDVTPMAMGGNKLRKLEFLAADALREGADTLITAGAIQSNHVRQTAAVAAKLGLHCVALLENPIGTTAENYLTNGNRLLLDLFNTQIEMCDALTDPNAQLEELATRVEAQGFRPYVIPVGGSNALGALGYVESALEIAQQCEGAVNISSVVVASGSAGTHAGLAVGLEHLMPESELIGVTVSRSVADQLPKVVNLQQAIAKELELTASAEILLWDDYFAPGYGVPNDEGMEAVKLLARLEGILLDPVYTGKAMAGLIDGISQKRFKDEGPILFIHTGGAPALFAYHPHV',\n",
       "  'mw': 35153.0},\n",
       " 'b1927': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P26612',\n",
       "  'gene': 'b1927',\n",
       "  'locus': ['b1927', 'JW1912'],\n",
       "  'synonyms': 'yedC',\n",
       "  'properties': {'Catalytic Activity': [('Endohydrolysis of (1->4)-alpha-D-glucosidic linkages in polysaccharides containing three or more (1->4)-alpha-linked D-glucose units.',\n",
       "     '3.2.1.1')]},\n",
       "  'seq': 'MRNPTLLQCFHWYYPEGGKLWPELAERADGFNDIGINMVWLPPAYKGASGGYSVGYDSYDLFDLGEFDQKGSIPTKYGDKAQLLAAIDALKRNDIAVLLDVVVNHKMGADEKEAIRVQRVNADDRTQIDEEIIECEGWTRYTFPARAGQYSQFIWDFKCFSGIDHIENPDEDGIFKIVNDYTGEGWNDQVDDELGNFDYLMGENIDFRNHAVTEEIKYWARWVMEQTQCDGFRLDAVKHIPAWFYKEWIEHVQEVAPKPLFIVAEYWSHEVDKLQTYIDQVEGKTMLFDAPLQMKFHEASRMGRDYDMTQIFTGTLVEADPFHAVTLVANHDTQPLQALEAPVEPWFKPLAYALILLRENGVPSVFYPDLYGAHYEDVGGDGQTYPIDMPIIEQLDELILARQRFAHGVQTLFFDHPNCIAFSRSGTDEFPGCVVVMSNGDDGEKTIHLGENYGNKTWRDFLGNRQERVVTDENGEATFFCNGGSVSVWVIEEVI',\n",
       "  'mw': 56639.0},\n",
       " 'b1981': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P76350',\n",
       "  'gene': 'b1981',\n",
       "  'locus': ['b1981', 'JW1962'],\n",
       "  'synonyms': 'yeeM',\n",
       "  'properties': {'Catalytic Activity': [('H(+)(in) + shikimate(in) = H(+)(out) + shikimate(out)',\n",
       "     '')],\n",
       "   'Function': 'Involved in the uptake of shikimate, an intermediate in the aromatic amino acid biosynthetic pathway'},\n",
       "  'seq': 'MDSTLISTRPDEGTLSLSRARRAALGSFAGAVVDWYDFLLYGITAALVFNREFFPQVSPAMGTLAAFATFGVGFLFRPLGGVIFGHFGDRLGRKRMLMLTVWMMGIATALIGILPSFSTIGWWAPILLVTLRAIQGFAVGGEWGGAALLSVESAPKNKKAFYSSGVQVGYGVGLLLSTGLVSLISMMTTDEQFLSWGWRIPFLFSIVLVLGALWVRNGMEESAEFEQQQHYQAAAKKRIPVIEALLRHPGAFLKIIALRLCELLTMYIVTAFALNYSTQNMGLPRELFLNIGLLVGGLSCLTIPCFAWLADRFGRRRVYITGTLIGTLSAFPFFMALEAQSIFWIVFFSIMLANIAHDMVVCVQQPMFTEMFGASYRYSGAGVGYQVASVVGGGFTPFIAAALITYFAGNWHSVAIYLLAGCLISAMTALLMKDSQRA',\n",
       "  'mw': 47817.0},\n",
       " 'b1982': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AE12',\n",
       "  'gene': 'b1982',\n",
       "  'locus': ['b1982', 'JW1963'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('AMP + H2O = adenine + D-ribose 5-phosphate',\n",
       "     '3.2.2.4')],\n",
       "   'Function': 'Catalyzes the hydrolysis of the N-glycosidic bond of AMP to form adenine and ribose 5-phosphate. Involved in regulation of AMP concentrations',\n",
       "   'Regulation Activity': 'Allosterically activated by Mg-ATP. Inhibited by inorganic phosphate and formycin monophosphate'},\n",
       "  'seq': 'MNNKGSGLTPAQALDKLDALYEQSVVALRNAIGNYITSGELPDENARKQGLFVYPSLTVTWDGSTTNPPKTRAFGRFTHAGSYTTTITRPTLFRSYLNEQLTLLYQDYGAHISVQPSQHEIPYPYVIDGSELTLDRSMSAGLTRYFPTTELAQIGDETADGIYHPTEFSPLSHFDARRVDFSLARLRHYTGTPVEHFQPFVLFTNYTRYVDEFVRWGCSQILDPDSPYIALSCAGGNWITAETEAPEEAISDLAWKKHQMPAWHLITADGQGITLVNIGVGPSNAKTICDHLAVLRPDVWLMIGHCGGLRESQAIGDYVLAHAYLRDDHVLDAVLPPDIPIPSIAEVQRALYDATKLVSGRPGEEVKQRLRTGTVVTTDDRNWELRYSASALRFNLSRAVAIDMESATIAAQGYRFRVPYGTLLCVSDKPLHGEIKLPGQANRFYEGAISEHLQIGIRAIDLLRAEGDRLHSRKLRTFNEPPFR',\n",
       "  'mw': 53995.0},\n",
       " 'b1991': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P36562',\n",
       "  'gene': 'b1991',\n",
       "  'locus': ['b1991', 'JW1969'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [(\"5,6-dimethylbenzimidazole + nicotinate beta-D-ribonucleotide = alpha-ribazole 5'-phosphate + H(+) + nicotinate\",\n",
       "     '2.4.2.21')],\n",
       "   'Function': \"Catalyzes the synthesis of alpha-ribazole-5'-phosphate from nicotinate mononucleotide (NAMN) and 5,6-dimethylbenzimidazole (DMB)\",\n",
       "   'Pathway': 'Nucleoside biosynthesis; alpha-ribazole biosynthesis; alpha-ribazole from 5,6-dimethylbenzimidazole: step 1/2'},\n",
       "  'seq': 'MQILADLLNTIPAIDSTAMSRAQRHIDGLLKPVGSLGKLEVLAIQLAGMPGLNGIPHVGKKAVLVMCADHGVWEEGVAISPKEVTAIQAENMTRGTTGVCVLAEQAGANVHVIDVGIDTAEPIPGLINMRVARGSGNIASAPAMSRRQAEKLLLDVICYTQELAKNGVTLFGVGELGMANTTPAAAIVSTITGRDPEEVVGIGANLPTDKLANKIDVVRRAITLNQPNPQDGVDVLAKVGGFDLVGIAGVMLGAASCGLPVLLDGFLSYAAALAACQMSPAIKPYLIPSHLSAEKGARIALSHLGLEPYLNMEMRLGEGSGAALAMPIIEAACAIYNNMGELAASNIVLPGNTTSDLNS',\n",
       "  'mw': 36987.0},\n",
       " 'b1992': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P36561',\n",
       "  'gene': 'b1992',\n",
       "  'locus': ['b1992', 'JW1970'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('adenosylcob(III)inamide-GDP + alpha-ribazole = adenosylcob(III)alamin + GMP + H(+)',\n",
       "     '2.7.8.26'),\n",
       "    (\"adenosylcob(III)inamide-GDP + alpha-ribazole 5'-phosphate = adenosylcob(III)alamin 5'-phosphate + GMP + H(+)\",\n",
       "     '2.7.8.26')],\n",
       "   'Function': \"Joins adenosylcobinamide-GDP and alpha-ribazole to generate adenosylcobalamin (Ado-cobalamin). Also synthesizes adenosylcobalamin 5'-phosphate from adenosylcobinamide-GDP and alpha-ribazole 5'-phosphate (By similarity)\",\n",
       "   'Pathway': 'Cofactor biosynthesis; adenosylcobalamin biosynthesis; adenosylcobalamin from cob(II)yrinate a,c-diamide: step 7/7'},\n",
       "  'seq': 'MSKLFWAMLSFITRLPVPRRWSQGLDFEHYSRGIITFPLIGLLLGAISGLVFMVLQAWCGAPLAALFSVLVLVLMTGGFHLDGLADTCDGVFSARSRDRMLEIMRDSRLGTHGGLALIFVVLAKILVLSELALRGESILASLAAACAVSRGTAALLMYRHRYAREEGLGNVFIGKIDGRQTCVTLGLAAIFAAVLLPGMHGVAAMVVTMVAIFILGQLLKRTLGGQTGDTLGAAIELGELVFLLALL',\n",
       "  'mw': 26386.0},\n",
       " 'b1993': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AE76',\n",
       "  'gene': 'b1993',\n",
       "  'locus': ['b1993', 'JW1971'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('adenosylcob(III)inamide + GTP = adenosylcob(III)inamide phosphate + GDP + H(+)',\n",
       "     '2.7.1.156'),\n",
       "    ('adenosylcob(III)inamide + ATP = adenosylcob(III)inamide phosphate + ADP + H(+)',\n",
       "     '2.7.1.156'),\n",
       "    ('adenosylcob(III)inamide phosphate + GTP + H(+) = adenosylcob(III)inamide-GDP + diphosphate',\n",
       "     '2.7.7.62')],\n",
       "   'Function': 'Catalyzes ATP-dependent phosphorylation of adenosylcobinamide and addition of GMP to adenosylcobinamide phosphate',\n",
       "   'Pathway': 'Cofactor biosynthesis; adenosylcobalamin biosynthesis; adenosylcobalamin from cob(II)yrinate a,c-diamide: step 6/7'},\n",
       "  'seq': 'MMILVTGGARSGKSRHAEALIGDSSQVLYIATSQILDDEMAARIEHHRQGRPEHWRTVERWQHLDELIHADINPNEVVLLECVTTMVTNLLFDYGGDKDPDEWDYQAMEQAINAEIQSLIAACQRCPAKVVLVTNEVGMGIVPESRLARHFRDIAGRVNQQLAAAANEVWLVVSGIGVKIK',\n",
       "  'mw': 20164.0},\n",
       " 'b2010': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P33013',\n",
       "  'gene': 'b2010',\n",
       "  'locus': ['b2010', 'JW5329'],\n",
       "  'synonyms': 'phsE yeeC',\n",
       "  'properties': {'Catalytic Activity': [('Preferential cleavage: (Ac)2-L-Lys-D-Ala-|-D-Ala. Also transpeptidation of peptidyl-alanyl moieties that are N-acyl substituents of D-alanine.',\n",
       "     '3.4.16.4')],\n",
       "   'Function': 'Removes C-terminal D-alanyl residues from sugar-peptide cell wall precursors',\n",
       "   'Pathway': 'Cell wall biogenesis; peptidoglycan biosynthesis'},\n",
       "  'seq': 'MKRRLIIAASLFVFNLSSGFAAENIPFSPQPPEIHAGSWVLMDYTTGQILTAGNEHQQRNPASLTKLMTGYVVDRAIDSHRITPDDIVTVGRDAWAKDNPVFVGSSLMFLKEGDRVSVRDLSRGLIVDSGNDACVALADYIAGGQRQFVEMMNNYAEKLHLKDTHFETVHGLDAPGQHSSAYDLAVLSRAIIHGEPEFYHMYSEKSLTWNGITQQNRNGLLWDKTMNVDGLKTGHTSGAGFNLIASAVDGQRRLIAVVMGADSAKGREEEARKLLRWGQQNFTTVQILHRGKKVGTERIWYGDKENIDLGTEQEFWMVLPKAEIPHIKAKYTLDGKELTAPISAHQRVGEIELYDRDKQVAHWPLVTLESVGEGSMFSRLSDYFHHKA',\n",
       "  'mw': 43346.0},\n",
       " 'b2019': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P60757',\n",
       "  'gene': 'b2019',\n",
       "  'locus': ['b2019', 'JW2001'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('1-(5-phospho-beta-D-ribosyl)-ATP + diphosphate = 5-phospho-alpha-D-ribose 1-diphosphate + ATP',\n",
       "     '2.4.2.17')],\n",
       "   'Function': \"Catalyzes the condensation of ATP and 5-phosphoribose 1-diphosphate to form N'-(5'-phosphoribosyl)-ATP (PR-ATP). Has a crucial role in the pathway because the rate of histidine biosynthesis seems to be controlled primarily by regulation of HisG enzymatic activity\",\n",
       "   'Regulation Activity': 'Feedback inhibited by histidine. Also inhibited by AMP and PR-ATP',\n",
       "   'Pathway': 'Amino-acid biosynthesis; L-histidine biosynthesis; L-histidine from 5-phospho-alpha-D-ribose 1-diphosphate: step 1/9'},\n",
       "  'seq': 'MTDNTRLRIAMQKSGRLSDDSRELLARCGIKINLHTQRLIAMAENMPIDILRVRDDDIPGLVMDGVVDLGIIGENVLEEELLNRRAQGEDPRYFTLRRLDFGGCRLSLATPVDEAWDGPLSLNGKRIATSYPHLLKRYLDQKGISFKSCLLNGSVEVAPRAGLADAICDLVSTGATLEANGLREVEVIYRSKACLIQRDGEMEESKQQLIDKLLTRIQGVIQARESKYIMMHAPTERLDEVIALLPGAERPTILPLAGDQQRVAMHMVSSETLFWETMEKLKALGASSILVLPIEKMME',\n",
       "  'mw': 33367.0},\n",
       " 'b2020': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P06988',\n",
       "  'gene': 'b2020',\n",
       "  'locus': ['b2020', 'JW2002'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('H2O + L-histidinol + 2 NAD(+) = 3 H(+) + L-histidine + 2 NADH',\n",
       "     '1.1.1.23')],\n",
       "   'Function': 'Catalyzes the sequential NAD-dependent oxidations of L-histidinol to L-histidinaldehyde and then to L-histidine',\n",
       "   'Pathway': 'Amino-acid biosynthesis; L-histidine biosynthesis; L-histidine from 5-phospho-alpha-D-ribose 1-diphosphate: step 9/9'},\n",
       "  'seq': 'MSFNTIIDWNSCTAEQQRQLLMRPAISASESITRTVNDILDNVKARGDEALREYSAKFDKTTVTALKVSAEEIAAASERLSDELKQAMAVAVKNIETFHTAQKLPPVDVETQPGVRCQQVTRPVASVGLYIPGGSAPLFSTVLMLATPASIAGCKKVVLCSPPPIADEILYAAQLCGVQDVFNVGGAQAIAALAFGTESVPKVDKIFGPGNAFVTEAKRQVSQRLDGAAIDMPAGPSEVLVIADSGATPDFVASDLLSQAEHGPDSQVILLTPAADMARRVAEAVERQLAELPRAETARQALNASRLIVTKDLAQCVEISNQYGPEHLIIQTRNARELVDSITSAGSVFLGDWSPESAGDYASGTNHVLPTYGYTATCSSLGLADFQKRMTVQELSKEGFSALASTIETLAAAERLTAHKNAVTLRVNALKEQA',\n",
       "  'mw': 46110.0},\n",
       " 'b2021': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P06986',\n",
       "  'gene': 'b2021',\n",
       "  'locus': ['b2021', 'JW2003'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('2-oxoglutarate + L-histidinol phosphate = 3-(imidazol-4-yl)-2-oxopropyl phosphate + L-glutamate',\n",
       "     '2.6.1.9')],\n",
       "   'Pathway': 'Amino-acid biosynthesis; L-histidine biosynthesis; L-histidine from 5-phospho-alpha-D-ribose 1-diphosphate: step 7/9'},\n",
       "  'seq': 'MSTVTITDLARENVRNLTPYQSARRLGGNGDVWLNANEYPTAVEFQLTQQTLNRYPECQPKAVIENYAQYAGVKPEQVLVSRGADEGIELLIRAFCEPGKDAILYCPPTYGMYSVSAETIGVECRTVPTLDNWQLDLQGISDKLDGVKVVYVCSPNNPTGQLINPQDFRTLLELTRGKAIVVADEAYIEFCPQASLAGWLAEYPHLAILRTLSKAFALAGLRCGFTLANEEVINLLMKVIAPYPLSTPVADIAAQALSPQGIVAMRERVAQIIAEREYLIAALKEIPCVEQVFDSETNYILARFKASSAVFKSLWDQGIILRDQNKQPSLSGCLRITVGTREESQRVIDALRAEQV',\n",
       "  'mw': 39360.0},\n",
       " 'b2022': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P06987',\n",
       "  'gene': 'b2022',\n",
       "  'locus': ['b2022', 'JW2004'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('D-erythro-1-(imidazol-4-yl)glycerol 3-phosphate = 3-(imidazol-4-yl)-2-oxopropyl phosphate + H2O',\n",
       "     '4.2.1.19'),\n",
       "    ('H2O + L-histidinol phosphate = L-histidinol + phosphate', '3.1.3.15')],\n",
       "   'Pathway': 'Amino-acid biosynthesis; L-histidine biosynthesis; L-histidine from 5-phospho-alpha-D-ribose 1-diphosphate: step 8/9'},\n",
       "  'seq': 'MSQKYLFIDRDGTLISEPPSDFQVDRFDKLAFEPGVIPELLKLQKAGYKLVMITNQDGLGTQSFPQADFDGPHNLMMQIFTSQGVQFDEVLICPHLPADECDCRKPKVKLVERYLAEQAMDRANSYVIGDRATDIQLAENMGITGLRYDRETLNWPMIGEQLTRRDRYAHVVRNTKETQIDVQVWLDREGGSKINTGVGFFDHMLDQIATHGGFRMEINVKGDLYIDDHHTVEDTGLALGEALKIALGDKRGICRFGFVLPMDECLARCALDISGRPHLEYKAEFTYQRVGDLSTEMIEHFFRSLSYTMGVTLHLKTKGKNDHHRVESLFKAFGRTLRQAIRVEGDTLPSSKGVL',\n",
       "  'mw': 40278.0},\n",
       " 'b2023': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P60595',\n",
       "  'gene': 'b2023',\n",
       "  'locus': ['b2023', 'JW2005'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('5-[(5-phospho-1-deoxy-D-ribulos-1-ylimino)methylamino]-1-(5-phospho-beta-D-ribosyl)imidazole-4-carboxamide + L-glutamine = 5-amino-1-(5-phospho-beta-D-ribosyl)imidazole-4-carboxamide + D-erythro-1-(imidazol-4-yl)glycerol 3-phosphate + H(+) + L-glutamate',\n",
       "     '4.3.2.10'),\n",
       "    ('H2O + L-glutamine = L-glutamate + NH4(+)', '3.5.1.2')],\n",
       "   'Function': 'IGPS catalyzes the conversion of PRFAR and glutamine to IGP, AICAR and glutamate. The HisH subunit catalyzes the hydrolysis of glutamine to glutamate and ammonia as part of the synthesis of IGP and AICAR. The resulting ammonia molecule is channeled to the active site of HisF',\n",
       "   'Pathway': 'Amino-acid biosynthesis; L-histidine biosynthesis; L-histidine from 5-phospho-alpha-D-ribose 1-diphosphate: step 5/9'},\n",
       "  'seq': 'MNVVILDTGCANLNSVKSAIARHGYEPKVSRDPDVVLLADKLFLPGVGTAQAAMDQVRERELFDLIKACTQPVLGICLGMQLLGRRSEESNGVDLLGIIDEDVPKMTDFGLPLPHMGWNRVYPQAGNRLFQGIEDGAYFYFVHSYAMPVNPWTIAQCNYGEPFTAAVQKDNFYGVQFHPERSGAAGAKLLKNFLEM',\n",
       "  'mw': 21653.0},\n",
       " 'b2024': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P10371',\n",
       "  'gene': 'b2024',\n",
       "  'locus': ['b2024', 'JW2006'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('1-(5-phospho-beta-D-ribosyl)-5-[(5-phospho-beta-D-ribosylamino)methylideneamino]imidazole-4-carboxamide = 5-[(5-phospho-1-deoxy-D-ribulos-1-ylimino)methylamino]-1-(5-phospho-beta-D-ribosyl)imidazole-4-carboxamide',\n",
       "     '5.3.1.16')],\n",
       "   'Pathway': 'Amino-acid biosynthesis; L-histidine biosynthesis; L-histidine from 5-phospho-alpha-D-ribose 1-diphosphate: step 4/9'},\n",
       "  'seq': 'MIIPALDLIDGTVVRLHQGDYGKQRDYGNDPLPRLQDYAAQGAEVLHLVDLTGAKDPAKRQIPLIKTLVAGVNVPVQVGGGVRSEEDVAALLEAGVARVVVGSTAVKSQDMVKGWFERFGADALVLALDVRIDEQGNKQVAVSGWQENSGVSLEQLVETYLPVGLKHVLCTDISRDGTLAGSNVSLYEEVCARYPQVAFQSSGGIGDIDDVAALRGTGVRGVIVGRALLEGKFTVKEAIACWQNA',\n",
       "  'mw': 26033.0},\n",
       " 'b2025': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P60664',\n",
       "  'gene': 'b2025',\n",
       "  'locus': ['b2025', 'JW2007'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('5-[(5-phospho-1-deoxy-D-ribulos-1-ylimino)methylamino]-1-(5-phospho-beta-D-ribosyl)imidazole-4-carboxamide + L-glutamine = 5-amino-1-(5-phospho-beta-D-ribosyl)imidazole-4-carboxamide + D-erythro-1-(imidazol-4-yl)glycerol 3-phosphate + H(+) + L-glutamate',\n",
       "     '4.3.2.10')],\n",
       "   'Function': 'IGPS catalyzes the conversion of PRFAR and glutamine to IGP, AICAR and glutamate. The HisF subunit catalyzes the cyclization activity that produces IGP and AICAR from PRFAR using the ammonia provided by the HisH subunit',\n",
       "   'Pathway': 'Amino-acid biosynthesis; L-histidine biosynthesis; L-histidine from 5-phospho-alpha-D-ribose 1-diphosphate: step 5/9'},\n",
       "  'seq': 'MLAKRIIPCLDVRDGQVVKGVQFRNHEIIGDIVPLAKRYAEEGADELVFYDITASSDGRVVDKSWVSRVAEVIDIPFCVAGGIKSLEDAAKILSFGADKISINSPALADPTLITRLADRFGVQCIVVGIDTWYDAETGKYHVNQYTGDESRTRVTQWETLDWVQEVQKRGAGEIVLNMMNQDGVRNGYDLEQLKKVREVCHVPLIASGGAGTMEHFLEAFRDADVDGALAASVFHKQIINIGELKAYLATQGVEIRIC',\n",
       "  'mw': 28454.0},\n",
       " 'b2026': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P06989',\n",
       "  'gene': 'b2026',\n",
       "  'locus': ['b2026', 'JW2008'],\n",
       "  'synonyms': 'hisIE',\n",
       "  'properties': {'Catalytic Activity': [(\"1-(5-phospho-beta-D-ribosyl)-ATP + H2O = 1-(5-phospho-beta-D-ribosyl)-5'-AMP + diphosphate + H(+)\",\n",
       "     '3.6.1.31'),\n",
       "    (\"1-(5-phospho-beta-D-ribosyl)-5'-AMP + H2O = 1-(5-phospho-beta-D-ribosyl)-5-[(5-phospho-beta-D-ribosylamino)methylideneamino]imidazole-4-carboxamide\",\n",
       "     '3.5.4.19')],\n",
       "   'Pathway': 'Amino-acid biosynthesis; L-histidine biosynthesis; L-histidine from 5-phospho-alpha-D-ribose 1-diphosphate: step 3/9'},\n",
       "  'seq': 'MLTEQQRRELDWEKTDGLMPVIVQHAVSGEVLMLGYMNPEALDKTLESGKVTFFSRTKQRLWTKGETSGNFLNVVSIAPDCDNDTLLVLANPIGPTCHKGTSSCFGDTAHQWLFLYQLEQLLAERKSADPETSYTAKLYASGTKRIAQKVGEEGVETALAATVHDRFELTNEASDLMYHLLVLLQDQGLDLTTVIENLRKRHQ',\n",
       "  'mw': 22756.0},\n",
       " 'b2027': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P76372',\n",
       "  'gene': 'b2027',\n",
       "  'locus': ['b2027', 'JW5836'],\n",
       "  'synonyms': 'cld rol wzz',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Confers a modal distribution of chain length on the O-antigen component of lipopolysaccharide (LPS). Gives rise to a reduced number of short chain molecules and increases in numbers of longer molecules',\n",
       "   'Pathway': 'Bacterial outer membrane biogenesis; lipopolysaccharide biosynthesis'},\n",
       "  'seq': 'MRVENNNVSGQNHDPEQIDLIDLLVQLWRGKMTIIISVIVAIALAIGYLAVAKEKWTSTAIITQPDVGQIAGYNNAMNVIYGQAAPKVSDLQETLIGRFSSAFSALAETLDNQEEREKLTIEPSVKNQQLPLTVSYVGQTAEGAQMKLAQYIQQVDDKVNQELEKDLKDNIALGRKNLQDSLRTQEVVAQEQKDLRIRQIQEALQYANQAQVTKPQIQQTGEDITQDTLFLLGSEALESMIKHEATRPLVFSPNYYQTRQNLLDIESLKVDDLDIHAYRYVMKPMLPIRRDSPKKAITLILAVLLGGMVGAGIVLGRNALRNYNAK',\n",
       "  'mw': 36455.0},\n",
       " 'b2028': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P76373',\n",
       "  'gene': 'b2028',\n",
       "  'locus': ['b2028', 'JW2010'],\n",
       "  'synonyms': 'pmrE udg yefA',\n",
       "  'properties': {'Catalytic Activity': [('H2O + 2 NAD(+) + UDP-alpha-D-glucose = 3 H(+) + 2 NADH + UDP-alpha-D-glucuronate',\n",
       "     '1.1.1.22')],\n",
       "   'Pathway': 'Bacterial outer membrane biogenesis; lipopolysaccharide biosynthesis'},\n",
       "  'seq': 'MKITISGTGYVGLSNGLLIAQNHEVVALDILPSRVAMLNDRISPIVDKEIQQFLQSDKIHFNATLDKNEAYRDADYVIIATPTDYDPKTNYFNTSSVESVIKDVVEINPYAVMVIKSTVPVGFTAAMHKKYRTENIIFSPEFLREGKALYDNLHPSRIVIGERSERAERFAALLQEGAIKQNIPMLFTDSTEAEAIKLFANTYLAMRVAYFNELDSYAESLGLNSRQIIEGVCLDPRIGNHYNNPSFGYGGYCLPKDTKQLLANYQSVPNNLISAIVDANRTRKDFIADAILSRKPQVVGIYRLIMKSGSDNFRASSIQGIMKRIKAKGVEVIIYEPVMKEDSFFNSRLERDLATFKQQADVIISNRMAEELKDVADKVYTRDLFGSD',\n",
       "  'mw': 43657.0},\n",
       " 'b2029': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P00350',\n",
       "  'gene': 'b2029',\n",
       "  'locus': ['b2029', 'JW2011'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('6-phospho-D-gluconate + NADP(+) = CO2 + D-ribulose 5-phosphate + NADPH',\n",
       "     '1.1.1.44')],\n",
       "   'Function': 'Catalyzes the oxidative decarboxylation of 6-phosphogluconate to ribulose 5-phosphate and CO(2), with concomitant reduction of NADP to NADPH',\n",
       "   'Pathway': 'Carbohydrate degradation; pentose phosphate pathway; D-ribulose 5-phosphate from D-glucose 6-phosphate (oxidative stage): step 3/3'},\n",
       "  'seq': 'MSKQQIGVVGMAVMGRNLALNIESRGYTVSIFNRSREKTEEVIAENPGKKLVPYYTVKEFVESLETPRRILLMVKAGAGTDAAIDSLKPYLDKGDIIIDGGNTFFQDTIRRNRELSAEGFNFIGTGVSGGEEGALKGPSIMPGGQKEAYELVAPILTKIAAVAEDGEPCVTYIGADGAGHYVKMVHNGIEYGDMQLIAEAYSLLKGGLNLTNEELAQTFTEWNNGELSSYLIDITKDIFTKKDEDGNYLVDVILDEAANKGTGKWTSQSALDLGEPLSLITESVFARYISSLKDQRVAASKVLSGPQAQPAGDKAEFIEKVRRALYLGKIVSYAQGFSQLRAASEEYNWDLNYGEIAKIFRAGCIIRAQFLQKITDAYAENPQIANLLLAPYFKQIADDYQQALRDVVAYAVQNGIPVPTFSAAVAYYDSYRAAVLPANLIQAQRDYFGAHTYKRIDKEGVFHTEWLD',\n",
       "  'mw': 51481.0},\n",
       " 'b2032': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37751',\n",
       "  'gene': 'b2032',\n",
       "  'locus': ['b2032', 'JW2017'],\n",
       "  'synonyms': 'yefI',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'May be a glycosyltransferase involved in the transfer of UDP-GalF and UDP-glucose',\n",
       "   'Pathway': 'Bacterial outer membrane biogenesis; lipopolysaccharide biosynthesis'},\n",
       "  'seq': 'MGKSIVVVSAVNFTTGGPFTILKKFLAATNNKENVSFIALVHSAKELKESYPWVKFIEFPEVKGSWLKRLHFEYVVCKKLSKELNATHWICLHDITANVVTKKRYVYCHNPAPFYKGILFREILMEPSFFLFKMLYGLIYKINIKKNTAVFVQQFWMKEKFIKKYSINNIIVSRPEIKLSDKSQLTDDDSQFKNNPSELTIFYPAVPRVFKNYELIISAARKLKEQSNIKFLLTISGTENAYAKYIISLAEGLDNVHFLGYLDKEKIDHCYNISDIVCFPSRLETWGLPLSEAKERGKWVLASDFPFTRETLGSYEKKAFFDSNNDDMLVKLIIDFKKGNLKKDISDANFIYRNENVLVGFDELVNFITEEH',\n",
       "  'mw': 43188.0},\n",
       " 'b2033': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37750',\n",
       "  'gene': 'b2033',\n",
       "  'locus': ['b2033', 'JW2018'],\n",
       "  'synonyms': 'yefH',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Putative O-acetyltransferase that transfers an O-acetyl group to the O antigen',\n",
       "   'Pathway': 'Bacterial outer membrane biogenesis; lipopolysaccharide biosynthesis'},\n",
       "  'seq': 'MILKLAKRYGLCGFIRLVRDVLLTRVFYRNCRIIRFPCYIRNDGSINFGENFTSGVGLRLDAFGRGVIFFSDNVQVNDYVHIASIESVTIGRDTLIASKVFITDHNHGSFKHSDPMSSPNIPPDMRTLESSAVVIGQRVWLGENVTVLPGTIIGNGVVVGANSVVRGSIPENTVIAGVPAKIIKKYNHETKLWEKA',\n",
       "  'mw': 21675.0},\n",
       " 'b2034': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37749',\n",
       "  'gene': 'b2034',\n",
       "  'locus': ['b2034', 'JW2019'],\n",
       "  'synonyms': 'yefG',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Involved in the transfer of galactofuranose (Galf) onto an alpha-D-gluco-configured acceptor substrate to form a beta-1,6-linkage. It uses n-octyl alpha-D-glucopyranoside as an acceptor substrate for the addition of galactofuranose from the donor substrate UDP-galactofuranose. It is not able to use beta-D-glucopyranoside isomers',\n",
       "   'Pathway': 'Bacterial outer membrane biogenesis; lipopolysaccharide biosynthesis'},\n",
       "  'seq': 'MYFLNDLNFSRRDAGFKARKDALDIASDYENISVVNIPLWGGVVQRIISSVKLSTFLCGLENKDVLIFNFPMAKPFWHILSFFHRLLKFRIVPLIHDIDELRGGGGSDSVRLATCDMVISHNPQMTKYLSKYMSQDKIKDIKIFDYLVSSDVEHRDVTDKQRGVIYAGNLSRHKCSFIYTEGCDFTLFGVNYENKDNPKYLGSFDAQSPEKINLPGMQFGLIWDGDSVETCSGAFGDYLKFNNPHKTSLYLSMELPVFIWDKAALADFIVDNRIGYAVGSIKEMQEIVDSMTIETYKQISENTKIISQKIRTGSYFRDVLEEVIDDLKTR',\n",
       "  'mw': 37757.0},\n",
       " 'b2035': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37748',\n",
       "  'gene': 'b2035',\n",
       "  'locus': ['b2035', 'JW2020'],\n",
       "  'synonyms': 'yefF',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'May link the O-antigen tetrasaccharide units into long chains, giving rise to typical smooth LPS',\n",
       "   'Pathway': 'Bacterial outer membrane biogenesis; lipopolysaccharide biosynthesis'},\n",
       "  'seq': 'MIYLVISVFLITAFICLYLKKDIFYPAVCVNIIFALVLLGYEITSDIYAFQLNDATLIFLLCNVLTFTLSCLLTESVLDLNIRKVNNAIYSIPSKKVHNVGLLVISFSMIYICMRLSNYQFGTSLLSYMNLIRDADVEDTSRNFSAYMQPIILTTFALFIWSKKFTNTKVSKTFTLLVFIVFIFAIILNTGKQIVFMVIISYAFIVGVNRVKHYVYLITAVGVLFSLYMLFLRGLPGGMAYYLSMYLVSPIIAFQEFYFQQVSNSASSHVFWFFERLMGLLTGGVSMSLHKEFVWVGLPTNVYTAFSDYVYISAELSYLMMVIHGCISGVLWRLSRNYISVKIFYSYFIYTFSFIFYHESFMTNISSWIQITLCIIVFSQFLKAQKIK',\n",
       "  'mw': 44744.0},\n",
       " 'b2036': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37747',\n",
       "  'gene': 'b2036',\n",
       "  'locus': ['b2036', 'JW2021'],\n",
       "  'synonyms': 'yefE',\n",
       "  'properties': {'Catalytic Activity': [('UDP-alpha-D-galactose = UDP-alpha-D-galactofuranose',\n",
       "     '5.4.99.9')],\n",
       "   'Function': 'Catalyzes the interconversion through a 2-keto intermediate of uridine diphosphogalactopyranose (UDP-GalP) into uridine diphosphogalactofuranose (UDP-GalF)',\n",
       "   'Pathway': 'Bacterial outer membrane biogenesis; lipopolysaccharide biosynthesis'},\n",
       "  'seq': 'MYDYIIVGSGLFGAVCANELKKLNKKVLVIEKRNHIGGNAYTEDCEGIQIHKYGAHIFHTNDKYIWDYVNDLVEFNRFTNSPLAIYKDKLFNLPFNMNTFHQMWGVKDPQEAQNIINAQKKKYGDKVPENLEEQAISLVGEDLYQALIKGYTEKQWGRSAKELPAFIIKRIPVRFTFDNNYFSDRYQGIPVGGYTKLIEKMLEGVDVKLGIDFLKDKDSLASKAHRIIYTGPIDQYFDYRFGALEYRSLKFETERHEFPNFQGNAVINFTDANVPYTRIIEHKHFDYVETKHTVVTKEYPLEWKVGDEPYYPVNDNKNMELFKKYRELASREDKVIFGGRLAEYKYYDMHQVISAALYQVKNIMSTD',\n",
       "  'mw': 42966.0},\n",
       " 'b2037': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37746',\n",
       "  'gene': 'b2037',\n",
       "  'locus': ['b2037', 'JW2022'],\n",
       "  'synonyms': 'wzx',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'May be involved in the translocation process of the nascent O-polysaccharide molecules and/or its ligation to lipid A core units',\n",
       "   'Pathway': 'Bacterial outer membrane biogenesis; LPS O-antigen biosynthesis'},\n",
       "  'seq': 'MNTNKLSLRRNVIYLAVVQGSNYLLPLLTFPYLVRTLGPENFGIFGFCQATMLYMIMFVEYGFNLTATQSIAKAADSKDKVTSIFWAVIFSKIVLIVITLIFLTSMTLLVPEYNKHAVIIWSFVPALVGNLIYPIWLFQGKEKMKWLTLSSILSRLAIIPLTFIFVNTKSDIAIAGFIQSSANLVAGIIALAIVVHEGWIGKVTLSLHNVRRSLADGFHVFISTSAISLYSTGIVIILGFISGPTSVGNFNAANTIRNALQGLLNPITQAIYPRISSTLVLNRVKGVILIKKSLTCLSLIGGAFSLILLLGASILVKISIGPGYDNAVIVLMIISPLPFLISLSNVYGIQVMLTHNYKKEFSKILIAAGLLSLLLIFPLTTLFKEIGAAITLLATECLVTSLMLMFVRNNKLLVC',\n",
       "  'mw': 45383.0},\n",
       " 'b2038': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37745',\n",
       "  'gene': 'b2038',\n",
       "  'locus': ['b2038', 'JW2023'],\n",
       "  'synonyms': 'rmlC',\n",
       "  'properties': {'Catalytic Activity': [('dTDP-4-dehydro-6-deoxy-alpha-D-glucose = dTDP-4-dehydro-beta-L-rhamnose',\n",
       "     '5.1.3.13')],\n",
       "   'Function': \"Catalyzes the epimerization of the C3' and C5'positions of dTDP-6-deoxy-D-xylo-4-hexulose, forming dTDP-6-deoxy-L-lyxo-4-hexulose\",\n",
       "   'Pathway': 'Bacterial outer membrane biogenesis; LPS O-antigen biosynthesis'},\n",
       "  'seq': 'MNVIRTEIEDVLILEPRVFGDDRGFFYESFNQSAFEHILGYPVSFVQDNHSRSSKNVLRGLHFQRGEYAQDKLVRCTHGAVFDVAVDIRPNSVSFGKWVGVLLSADNKQQLWIPKGFAHGFLVLSDIAEFQYKTTNYYHPESDCGICWNDERIAIDWPQTSGLILSPKDERLFTLDELIRLKLIA',\n",
       "  'mw': 21270.0},\n",
       " 'b2039': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37744',\n",
       "  'gene': 'b2039',\n",
       "  'locus': ['b2039', 'JW2024'],\n",
       "  'synonyms': 'rmlA rmlA1',\n",
       "  'properties': {'Catalytic Activity': [('alpha-D-glucose 1-phosphate + dTTP + H(+) = diphosphate + dTDP-alpha-D-glucose',\n",
       "     '2.7.7.24')],\n",
       "   'Function': 'Catalyzes the formation of dTDP-glucose, from dTTP and glucose 1-phosphate, as well as its pyrophosphorolysis',\n",
       "   'Pathway': 'Bacterial outer membrane biogenesis; LPS O-antigen biosynthesis'},\n",
       "  'seq': 'MKMRKGIILAGGSGTRLYPVTMAVSKQLLPIYDKPMIYYPLSTLMLAGIRDILIISTPQDTPRFQQLLGDGSQWGLNLQYKVQPSPDGLAQAFIIGEEFIGGDDCALVLGDNIFYGHDLPKLMEAAVNKESGATVFAYHVNDPERYGVVEFDKNGTAISLEEKPLEPKSNYAVTGLYFYDNDVVQMAKNLKPSARGELEITDINRIYLEQGRLSVAMMGRGYAWLDTGTHQSLIEASNFIATIEERQGLKVSCPEEIAFRKGFIDVEQVRKLAVPLIKNNYGQYLYKMTKDSN',\n",
       "  'mw': 32694.0},\n",
       " 'b2040': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37760',\n",
       "  'gene': 'b2040',\n",
       "  'locus': ['b2040', 'JW2025'],\n",
       "  'synonyms': 'rmlD',\n",
       "  'properties': {'Catalytic Activity': [('dTDP-beta-L-rhamnose + NADP(+) = dTDP-4-dehydro-beta-L-rhamnose + H(+) + NADPH',\n",
       "     '1.1.1.133')],\n",
       "   'Function': 'Involved in the biosynthesis of the dTDP-L-rhamnose which is an important component of lipopolysaccharide (LPS) (Probable). Catalyzes the reduction of dTDP-6-deoxy-L-lyxo-4-hexulose to yield dTDP-L-rhamnose (By similarity). RmlD uses NADH and NADPH nearly equally well (By similarity)',\n",
       "   'Pathway': 'Bacterial outer membrane biogenesis; LPS O-antigen biosynthesis'},\n",
       "  'seq': 'MNILLFGKTGQVGWELQRALAPLGNLIAFDVHSTDYCGDFSNPEGVAETVRSIRPDIIVNAAAHTAVDKAESEPEFAQLINATSVEAIAKAANEVGAWVIHYSTDYVFPGNGDMPWLETDATAPLNVYGETKLAGEKALQEYCAKHLIFRTSWVYAGKGNNFAKTMLRLAKEREELAVINDQFGAPTGAELLADCTAHAIRVALNKPDVAGLYHLVASGTTTWYDYAALVFEEARKAGIPLALNKLNAVPTTAYPTPARRPHNSRLNTEKFQQNFALVLPDWQVGVKRMLNELFTTTAI',\n",
       "  'mw': 32694.0},\n",
       " 'b2041': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37759',\n",
       "  'gene': 'b2041',\n",
       "  'locus': ['b2041', 'JW2026'],\n",
       "  'synonyms': 'rmlB',\n",
       "  'properties': {'Catalytic Activity': [('dTDP-alpha-D-glucose = dTDP-4-dehydro-6-deoxy-alpha-D-glucose + H2O',\n",
       "     '4.2.1.46')],\n",
       "   'Function': 'Catalyzes the dehydration of dTDP-D-glucose to form dTDP-6-deoxy-D-xylo-4-hexulose via a three-step process involving oxidation, dehydration and reduction',\n",
       "   'Pathway': 'Bacterial outer membrane biogenesis; LPS O-antigen biosynthesis'},\n",
       "  'seq': 'MKILVTGGAGFIGSAVVRHIINNTQDSVVNVDKLTYAGNRESLADVSDSERYVFEHADICDAPAMARIFAQHQPDAVMHLAAESHVDRSITGPAAFIETNIVGTYVLLEAARNYWSALDSDKKNSFRFHHISTDEVYGDLPHPDEVNNTEELPLFTETTAYAPSSPYSASKASSDHLVRAWKRTYGLPTIVTNCSNNYGPYHFPEKLIPLVILNALEGKALPIYGKGDQIRDWLYVEDHARALYTVVTEGKAGETYNIGGHNEKKNIDVVLTICDLLDEIVPKEKSYREQITYVADRPGHDRRYAIDAEKIGRALGWKPQETFESGIRKTVEWYLSNTKWVDNVKSGAYQSWIEQNYEGRQ',\n",
       "  'mw': 40558.0},\n",
       " 'b2045': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P71242',\n",
       "  'gene': 'b2045',\n",
       "  'locus': ['b2045', 'JW2030'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Pathway': 'Slime biogenesis; slime polysaccharide biosynthesis'},\n",
       "  'seq': 'MKLLILGNHTCGNRGDSAILRGLLDAINILNPHAEVDVMSRYPVSSSWLLNRPVMGDPLFLQMKQHNSAAGVVGRVKKVLRRRYQHQVLLSRVTDTGKLRNIAIAQGFTDFVRLLSGYDAIIQVGGSFFVDLYGVPQFEHALCTFMAKKPLFMIGHSVGPFQDEQFNQLANYVFGHCDALILRESVSFDLMKRSNITTAKVEHGVDTAWLVDHHTEDFTASYAVQHWLDVAAQQKTVAITLRELAPFDKRLGTTQQAYEKAFAGVVNRILDEGYQVIALSTCTGIDSYNKDDRMVALNLRQHISDPARYHVVMDELNDLEMGKILGACELTVGTRLHSAIISMNFATPAIAINYEHKSAGIMQQLGLPEMAIDIRHLLDGSLQAMVADTLGQLPALNARLSEAVSRERQTGMQMVQSVLERIGEVK',\n",
       "  'mw': 47344.0},\n",
       " 'b2048': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P24175',\n",
       "  'gene': 'b2048',\n",
       "  'locus': ['b2048', 'JW2033'],\n",
       "  'synonyms': 'cpsG rfbL',\n",
       "  'properties': {'Catalytic Activity': [('alpha-D-mannose 1-phosphate = D-mannose 6-phosphate',\n",
       "     '5.4.2.8')],\n",
       "   'Function': 'Involved in the biosynthesis of the capsular polysaccharide colanic acid',\n",
       "   'Pathway': 'Nucleotide-sugar biosynthesis; GDP-alpha-D-mannose biosynthesis; alpha-D-mannose 1-phosphate from D-fructose 6-phosphate: step 2/2'},\n",
       "  'seq': 'MKKLTCFKAYDIRGKLGEELNEDIAWRIGRAYGEFLKPKTIVLGGDVRLTSETLKLALAKGLQDAGVDVLDIGMSGTEEIYFATFHLGVDGGIEVTASHNPMDYNGMKLVREGARPISGDTGLRDVQRLAEANDFPPVDETKRGRYQQINLRDAYVDHLFGYINVKNLTPLKLVINSGNGAAGPVVDAIEARFKALGAPVELIKVHNTPDGNFPNGIPNPLLPECRDDTRNAVIKHGADMGIAFDGDFDRCFLFDEKGQFIEGYYIVGLLAEAFLEKNPGAKIIHDPRLSWNTVDVVTAAGGTPVMSKTGHAFIKERMRKEDAIYGGEMSAHHYFRDFAYCDSGMIPWLLVAELVCLKDKTLGELVRDRMAAFPASGEINSKLAQPVEAINRVEQHFSREALAVDRTDGISMTFADWRFNLRTSNTEPVVRLNVESRGDVPLMEARTRTLLTLLNE',\n",
       "  'mw': 50463.0},\n",
       " 'b2049': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P24174',\n",
       "  'gene': 'b2049',\n",
       "  'locus': ['b2049', 'JW2034'],\n",
       "  'synonyms': 'cpsB rfbM',\n",
       "  'properties': {'Catalytic Activity': [('alpha-D-mannose 1-phosphate + GTP + H(+) = diphosphate + GDP-alpha-D-mannose',\n",
       "     '2.7.7.13')],\n",
       "   'Function': 'Involved in the biosynthesis of the capsular polysaccharide colanic acid',\n",
       "   'Pathway': 'Nucleotide-sugar biosynthesis; GDP-alpha-D-mannose biosynthesis; GDP-alpha-D-mannose from alpha-D-mannose 1-phosphate (GTP route): step 1/1'},\n",
       "  'seq': 'MAQSKLYPVVMAGGSGSRLWPLSRVLYPKQFLCLKGDLTMLQTTICRLNGVECESPVVICNEQHRFIVAEQLRQLNKLTENIILEPAGRNTAPAIALAALAAKRHSPESDPLMLVLAADHVIADEDAFRAAVRNAMPYAEAGKLVTFGIVPDLPETGYGYIRRGEVSAGEQDMVAFEVAQFVEKPNLETAQAYVASGEYYWNSGMFLFRAGRYLEELKKYRPDILDACEKAMSAVDPDLNFIRVDEEAFLACPEESVDYAVMERTADAVVVPMDAGWSDVGSWSSLWEISAHTAEGNVCHGDVINHKTENSYVYAESGLVTTVGVKDLVVVQTKDAVLIADRNAVQDVKKVVEQIKADGRHEHRVHREVYRPWGKYDSIDAGDRYQVKRITVKPGEGLSVQMHHHRAEHWVVVAGTAKVTIDGDIKLLGENESIYIPLGATHCLENPGKIPLDLIEVRSGSYLEEDDVVRFADRYGRV',\n",
       "  'mw': 53016.0},\n",
       " 'b2051': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P32056',\n",
       "  'gene': 'b2051',\n",
       "  'locus': ['b2051', 'JW5335'],\n",
       "  'synonyms': 'nudD wcaH yefC',\n",
       "  'properties': {'Catalytic Activity': [('GDP-alpha-D-mannose + H2O = D-mannose + GDP + H(+)',\n",
       "     '')],\n",
       "   'Function': 'Hydrolyzes both GDP-mannose and GDP-glucose (PubMed:7592609, PubMed:10913267, PubMed:23481913). Could participate in the regulation of cell wall biosynthesis by influencing the concentration of GDP-mannose or GDP-glucose in the cell. Might also be involved in the biosynthesis of the slime polysaccharide colanic acid (PubMed:7592609, PubMed:10913267)'},\n",
       "  'seq': 'MFLRQEDFATVVRSTPLVSLDFIVENSRGEFLLGKRTNRPAQGYWFVPGGRVQKDETLEAAFERLTMAELGLRLPITAGQFYGVWQHFYDDNFSGTDFTTHYVVLGFRFRVSEEELLLPDEQHDDYRWLTSDALLASDNVHANSRAYFLAEKRTGVPGL',\n",
       "  'mw': 18273.0},\n",
       " 'b2052': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P32055',\n",
       "  'gene': 'b2052',\n",
       "  'locus': ['b2052', 'JW2037'],\n",
       "  'synonyms': 'wcaG yefB',\n",
       "  'properties': {'Catalytic Activity': [('GDP-beta-L-fucose + NADP(+) = GDP-4-dehydro-alpha-D-rhamnose + H(+) + NADPH',\n",
       "     '1.1.1.271')],\n",
       "   'Function': 'Catalyzes the two-step NADP-dependent conversion of GDP-4-dehydro-6-deoxy-D-mannose to GDP-fucose, involving an epimerase and a reductase reaction',\n",
       "   'Regulation Activity': 'Subject to product inhibition by NADP and GDP-fucose',\n",
       "   'Pathway': 'Exopolysaccharide biosynthesis; colanic acid biosynthesis'},\n",
       "  'seq': 'MSKQRVFIAGHRGMVGSAIRRQLEQRGDVELVLRTRDELNLLDSRAVHDFFASERIDQVYLAAAKVGGIVANNTYPADFIYQNMMIESNIIHAAHQNDVNKLLFLGSSCIYPKLAKQPMAESELLQGTLEPTNEPYAIAKIAGIKLCESYNRQYGRDYRSVMPTNLYGPHDNFHPSNSHVIPALLRRFHEATAQNAPDVVVWGSGTPMREFLHVDDMAAASIHVMELAHEVWLENTQPMLSHINVGTGVDCTIRELAQTIAKVVGYKGRVVFDASKPDGTPRKLLDVTRLHQLGWYHEISLEAGLASTYQWFLENQDRFRG',\n",
       "  'mw': 36141.0},\n",
       " 'b2053': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AC88',\n",
       "  'gene': 'b2053',\n",
       "  'locus': ['b2053', 'JW2038'],\n",
       "  'synonyms': 'yefA yefN',\n",
       "  'properties': {'Catalytic Activity': [('GDP-alpha-D-mannose = GDP-4-dehydro-alpha-D-rhamnose + H2O',\n",
       "     '4.2.1.47')],\n",
       "   'Function': 'Catalyzes the conversion of GDP-D-mannose to GDP-4-dehydro-6-deoxy-D-mannose',\n",
       "   'Pathway': 'Exopolysaccharide biosynthesis; colanic acid biosynthesis'},\n",
       "  'seq': 'MSKVALITGVTGQDGSYLAEFLLEKGYEVHGIKRRASSFNTERVDHIYQDPHTCNPKFHLHYGDLSDTSNLTRILREVQPDEVYNLGAMSHVAVSFESPEYTADVDAMGTLRLLEAIRFLGLEKKTRFYQASTSELYGLVQEIPQKETTPFYPRSPYAVAKLYAYWITVNYRESYGMYACNGILFNHESPRRGETFVTRKITRAIANIAQGLESCLYLGNMDSLRDWGHAKDYVKMQWMMLQQEQPEDFVIATGVQYSVRQFVEMAAAQLGIKLRFEGTGVEEKGIVVSVTGHDAPGVKPGDVIIAVDPRYFRPAEVETLLGDPTKAHEKLGWKPEITLREMVSEMVANDLEAAKKHSLLKSHGYDVAIALES',\n",
       "  'mw': 42047.0},\n",
       " 'b2065': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P28248',\n",
       "  'gene': 'b2065',\n",
       "  'locus': ['b2065', 'JW2050'],\n",
       "  'synonyms': 'dus paxA',\n",
       "  'properties': {'Catalytic Activity': [('dCTP + H(+) + H2O = dUTP + NH4(+)',\n",
       "     '3.5.4.13')],\n",
       "   'Function': 'Catalyzes the deamination of dCTP to dUTP',\n",
       "   'Regulation Activity': 'Inhibited by dTTP, which binds to the active site and stabilizes the inactive form of the enzyme (PubMed:17651436). Inhibited by inorganic phosphate (PubMed:15539408)',\n",
       "   'Pathway': 'Pyrimidine metabolism; dUMP biosynthesis; dUMP from dCTP (dUTP route): step 1/2'},\n",
       "  'seq': 'MRLCDRDIEAWLDEGRLSINPRPPVERINGATVDVRLGNKFRTFRGHTAAFIDLSGPKDEVSAALDRVMSDEIVLDEGEAFYLHPGELALAVTLESVTLPADLVGWLDGRSSLARLGLMVHVTAHRIDPGWSGCIVLEFYNSGKLPLALRPGMLIGALSFEPLSGPAVRPYNRREDAKYRNQQGAVASRIDKD',\n",
       "  'mw': 21249.0},\n",
       " 'b2066': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A8F4',\n",
       "  'gene': 'b2066',\n",
       "  'locus': ['b2066', 'JW2051'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + uridine = ADP + H(+) + UMP',\n",
       "     '2.7.1.48'),\n",
       "    ('ATP + cytidine = ADP + CMP + H(+)', '2.7.1.48')],\n",
       "   'Pathway': 'Pyrimidine metabolism; UMP biosynthesis via salvage pathway; UMP from uridine: step 1/1'},\n",
       "  'seq': 'MTDQSHQCVIIGIAGASASGKSLIASTLYRELREQVGDEHIGVIPEDCYYKDQSHLSMEERVKTNYDHPSAMDHSLLLEHLQALKRGSAIDLPVYSYVEHTRMKETVTVEPKKVIILEGILLLTDARLRDELNFSIFVDTPLDICLMRRIKRDVNERGRSMDSVMAQYQKTVRPMFLQFIEPSKQYADIIVPRGGKNRIAIDILKAKISQFFE',\n",
       "  'mw': 24353.0},\n",
       " 'b2091': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A9S3',\n",
       "  'gene': 'b2091',\n",
       "  'locus': ['b2091', 'JW2075'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('galactitol 1-phosphate + NAD(+) = H(+) + keto-D-tagatose 6-phosphate + NADH',\n",
       "     '1.1.1.251')],\n",
       "   'Function': 'Converts galactitol 1-phosphate to D-tagatose 6-phosphate'},\n",
       "  'seq': 'MKSVVNDTDGIVRVAESVIPEIKHQDEVRVKIASSGLCGSDLPRIFKNGAHYYPITLGHEFSGYIDAVGSGVDDLHPGDAVACVPLLPCFTCPECLKGFYSQCAKYDFIGSRRDGGFAEYIVVKRKNVFALPTDMPIEDGAFIEPITVGLHAFHLAQGCENKNVIIIGAGTIGLLAIQCAVALGAKSVTAIDISSEKLALAKSFGAMQTFNSSEMSAPQMQSVLRELRFNQLILETAGVPQTVELAVEIAGPHAQLALVGTLHQDLHLTSATFGKILRKELTVIGSWMNYSSPWPGQEWETASRLLTERKLSLEPLIAHRGSFESFAQAVRDIARNAMPGKVLLIP',\n",
       "  'mw': 37390.0},\n",
       " 'b2092': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P69831',\n",
       "  'gene': 'b2092',\n",
       "  'locus': ['b2092', 'JW2076'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'The phosphoenolpyruvate-dependent sugar phosphotransferase system (PTS), a major carbohydrate active transport system, catalyzes the phosphorylation of incoming sugar substrates concomitant with their translocation across the cell membrane. The enzyme II complex composed of GatA, GatB and GatC is involved in galactitol transport'},\n",
       "  'seq': 'MFSEVMRYILDLGPTVMLPIVIIIFSKILGMKAGDCFKAGLHIGIGFVGIGLVIGLMLDSIGPAAKAMAENFDLNLHVVDVGWPGSSPMTWASQIALVAIPIAILVNVAMLLTRMTRVVNVDIWNIWHMTFTGALLHLATGSWMIGMAGVVIHAAFVYKLGDWFARDTRNFFELEGIAIPHGTSAYMGPIAVLVDAIIEKIPGVNRIKFSADDIQRKFGPFGEPVTVGFVMGLIIGILAGYDVKGVLQLAVKTAAVMLLMPRVIKPIMDGLTPIAKQARSRLQAKFGGQEFLIGLDPALLLGHTAVVSASLIFIPLTILIAVCVPGNQVLPFGDLATIGFFVAMAVAVHRGNLFRTLISGVIIMSITLWIATQTIGLHTQLAANAGALKAGGMVASMDQGGSPITWLLIQVFSPQNIPGFIIIGAIYLTGIFMTWRRARGFIKQEKVVLAE',\n",
       "  'mw': 48365.0},\n",
       " 'b2093': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37188',\n",
       "  'gene': 'b2093',\n",
       "  'locus': ['b2093', 'JW2077'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('galactitol(out) + N(pros)-phospho-L-histidyl-[protein] = galactitol 1-phosphate(in) + L-histidyl-[protein]',\n",
       "     '2.7.1.200')],\n",
       "   'Function': 'The phosphoenolpyruvate-dependent sugar phosphotransferase system (PTS), a major carbohydrate active transport system, catalyzes the phosphorylation of incoming sugar substrates concomitant with their translocation across the cell membrane. The enzyme II complex composed of GatA, GatB and GatC is involved in galactitol transport. It can also use D-glucitol'},\n",
       "  'seq': 'MKRKIIVACGGAVATSTMAAEEIKELCQNHNIPVELIQCRVNEIETYMDGVHLICTTAKVDRSFGDIPLVHGMPFISGIGIEALQNKILTILQG',\n",
       "  'mw': 10222.0},\n",
       " 'b2094': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P69828',\n",
       "  'gene': 'b2094',\n",
       "  'locus': ['b2094', 'JW2078/JW2081'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'The phosphoenolpyruvate-dependent sugar phosphotransferase system (PTS), a major carbohydrate active transport system, catalyzes the phosphorylation of incoming sugar substrates concomitant with their translocation across the cell membrane. The enzyme II complex composed of GatA, GatB and GatC is involved in galactitol transport. It can also use D-glucitol'},\n",
       "  'seq': 'MTNLFVRSGISFVDRSEVLTHIGNEMLAKGVVHDTWPQALIAREAEFPTGIMLEQHAIAIPHCEAIHAKSSAIYLLRPTNKVHFQQADDDNDVAVSLVIALIVENPQQQLKLLRCLFGKLQQPDIVETLITLPETQLKEYFTKYVLDSDE',\n",
       "  'mw': 16907.0},\n",
       " 'b2095': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0C8J8',\n",
       "  'gene': 'b2095',\n",
       "  'locus': ['b2095', 'JW2082'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Component of the tagatose-1,6-bisphosphate aldolase GatYZ that is required for full activity and stability of the Y subunit. Could have a chaperone-like function for the proper and stable folding of GatY. When expressed alone, GatZ does not show any aldolase activity. Is involved in the catabolism of galactitol',\n",
       "   'Pathway': 'Carbohydrate metabolism; D-tagatose 6-phosphate degradation; D-glyceraldehyde 3-phosphate and glycerone phosphate from D-tagatose 6-phosphate: step 2/2'},\n",
       "  'seq': 'MKTLIARHKAGEHIGICSVCSAHPLVIEAALAFDRNSTRKVLIEATSNQVNQFGGYTGMTPADFREFVFTIADKVGFARERIILGGDHLGPNCWQQENADAAMEKSVELVKEYVRAGFSKIHLDASMSCAGDPIPLAPETVAERAAVLCFAAESVATDCQREQLSYVIGTEVPVPGGEASAIQSVHITHVEDAANTLRTHQKAFIARGLTEALTRVIAIVVQPGVEFDHSNIIHYQPQEAQPLAQWIENTRMVYEAHSTDYQTRTAYWELVRDHFAILKVGPALTFALREAIFALAQIEQELIAPENRSGCLAVIEEVMLDEPQYWKKYYRTGFNDSLLDIRYSLSDRIRYYWPHSRIKNSVETMMVNLEGVDIPLGMISQYLPKQFERIQSGELSAIPHQLIMDKIYDVLRAYRYGCAE',\n",
       "  'mw': 47109.0},\n",
       " 'b2096': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0C8J6',\n",
       "  'gene': 'b2096',\n",
       "  'locus': ['b2096', 'JW5343'],\n",
       "  'synonyms': 'yegF',\n",
       "  'properties': {'Catalytic Activity': [('D-tagatofuranose 1,6-bisphosphate = D-glyceraldehyde 3-phosphate + dihydroxyacetone phosphate',\n",
       "     '4.1.2.40')],\n",
       "   'Function': 'Catalytic subunit of the tagatose-1,6-bisphosphate aldolase GatYZ, which catalyzes the reversible aldol condensation of dihydroxyacetone phosphate (DHAP or glycerone-phosphate) with glyceraldehyde 3-phosphate (G3P) to produce tagatose 1,6-bisphosphate (TBP) (PubMed:8955298, PubMed:11976750). Requires GatZ subunit for full activity and stability (PubMed:11976750). Is involved in the catabolism of galactitol (PubMed:8955298, PubMed:11976750). Has a high aldolase activity for TBP and a very low one for fructose 1,6-bisphosphate (FBP) (PubMed:11976750)',\n",
       "   'Pathway': 'Carbohydrate metabolism; D-tagatose 6-phosphate degradation; D-glyceraldehyde 3-phosphate and glycerone phosphate from D-tagatose 6-phosphate: step 2/2'},\n",
       "  'seq': 'MYVVSTKQMLNNAQRGGYAVPAFNIHNLETMQVVVETAANLHAPVIIAGTPGTFTHAGTENLLALVSAMAKQYHHPLAIHLDHHTKFDDIAQKVRSGVRSVMIDASHLPFAQNISRVKEVVDFCHRFDVSVEAELGQLGGQEDDVQVNEADALYTNPAQAREFAEATGIDSLAVAIGTAHGMYASAPALDFSRLENIRQWVNLPLVLHGASGLSTKDIQQTIKLGICKINVATELKNAFSQALKNYLTEHPEATDPRDYLQSAKSAMRDVVSKVIADCGCEGRA',\n",
       "  'mw': 30812.0},\n",
       " 'b2097': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A991',\n",
       "  'gene': 'b2097',\n",
       "  'locus': ['b2097', 'JW5344'],\n",
       "  'synonyms': 'dhnA',\n",
       "  'properties': {'Catalytic Activity': [('beta-D-fructose 1,6-bisphosphate = D-glyceraldehyde 3-phosphate + dihydroxyacetone phosphate',\n",
       "     '4.1.2.13')],\n",
       "   'Function': 'Catalyzes the reversible aldol condensation/cleavage reaction between glyceraldehyde 3-phosphate and dihydroxyacetone phosphate (DHAP) to yield fructose-bisphosphate (FBP)',\n",
       "   'Regulation Activity': 'Activated by citrate'},\n",
       "  'seq': 'MTDIAQLLGKDADNLLQHRCMTIPSDQLYLPGHDYVDRVMIDNNRPPAVLRNMQTLYNTGRLAGTGYLSILPVDQGVEHSAGASFAANPLYFDPKNIVELAIEAGCNCVASTYGVLASVSRRYAHRIPFLVKLNHNETLSYPNTYDQTLYASVEQAFNMGAVAVGATIYFGSEESRRQIEEISAAFERAHELGMVTVLWAYLRNSAFKKDGVDYHVSADLTGQANHLAATIGADIVKQKMAENNGGYKAINYGYTDDRVYSKLTSENPIDLVRYQLANCYMGRAGLINSGGAAGGETDLSDAVRTAVINKRAGGMGLILGRKAFKKSMADGVKLINAVQDVYLDSKITIA',\n",
       "  'mw': 38109.0},\n",
       " 'b2103': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P76422',\n",
       "  'gene': 'b2103',\n",
       "  'locus': ['b2103', 'JW2090'],\n",
       "  'synonyms': 'thiJ',\n",
       "  'properties': {'Catalytic Activity': [('4-amino-5-hydroxymethyl-2-methylpyrimidine + ATP = 4-amino-2-methyl-5-(phosphooxymethyl)pyrimidine + ADP + H(+)',\n",
       "     '2.7.1.49'),\n",
       "    ('4-amino-2-methyl-5-(phosphooxymethyl)pyrimidine + ATP = 4-amino-2-methyl-5-(diphosphooxymethyl)pyrimidine + ADP',\n",
       "     '2.7.4.7')],\n",
       "   'Function': 'Catalyzes the phosphorylation of hydroxymethylpyrimidine phosphate (HMP-P) to HMP-PP, and of HMP to HMP-P. Shows no activity with pyridoxal, pyridoxamine or pyridoxine',\n",
       "   'Pathway': 'Cofactor biosynthesis; thiamine diphosphate biosynthesis; 4-amino-2-methyl-5-diphosphomethylpyrimidine from 5-amino-1-(5-phospho-D-ribosyl)imidazole: step 3/3'},\n",
       "  'seq': 'MKRINALTIAGTDPSGGAGIQADLKTFSALGAYGCSVITALVAQNTRGVQSVYRIEPDFVAAQLDSVFSDVRIDTTKIGMLAETDIVEAVAERLQRYQIQNVVLDTVMLAKSGDPLLSPSAVATLRSRLLPQVSLITPNLPEAAALLDAPHARTEQEMLEQGRSLLAMGCGAVLMKGGHLDDEQSPDWLFTREGEQRFTAPRIMTKNTHGTGCTLSAALAALRPRHTNWADTVQEAKSWLSSALAQADTLEVGHGIGPVHHFHAWW',\n",
       "  'mw': 28634.0},\n",
       " 'b2104': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P76423',\n",
       "  'gene': 'b2104',\n",
       "  'locus': ['b2104', 'JW2091'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('5-(2-hydroxyethyl)-4-methylthiazole + ATP = 4-methyl-5-(2-phosphooxyethyl)-thiazole + ADP + H(+)',\n",
       "     '2.7.1.50')],\n",
       "   'Function': 'Catalyzes the phosphorylation of the hydroxyl group of 4-methyl-5-beta-hydroxyethylthiazole (THZ)',\n",
       "   'Pathway': 'Cofactor biosynthesis; thiamine diphosphate biosynthesis; 4-methyl-5-(2-phosphoethyl)-thiazole from 5-(2-hydroxyethyl)-4-methylthiazole: step 1/1'},\n",
       "  'seq': 'MQVDLLGSAQSAHALHLFHQHSPLVHCMTNDVVQTFTANTLLALGASPAMVIETEEASQFAAIASALLINVGTLTQPRAQAMRAAVEQAKSSQTPWTLDPVAVGALDYRRHFCHELLSFKPAAIRGNASEIMALAGIANGGRGVDTTDAAANAIPAAQTLARETGAIVVVTGEMDYVTDGHRIIGIHGGDPLMTKVVGTGCALSAVVAACCALPGDTLENVASACHWMKQAGERAVARSEGPGSFVPHFLDALWQLTQEVQA',\n",
       "  'mw': 27339.0},\n",
       " 'b2114': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P00959',\n",
       "  'gene': 'b2114',\n",
       "  'locus': ['b2114', 'JW2101'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + L-methionine + tRNA(Met) = AMP + diphosphate + L-methionyl-tRNA(Met)',\n",
       "     '6.1.1.10')],\n",
       "   'Function': 'Is required not only for elongation of protein synthesis but also for the initiation of all mRNA translation through initiator tRNA(fMet) aminoacylation'},\n",
       "  'seq': 'MTQVAKKILVTCALPYANGSIHLGHMLEHIQADVWVRYQRMRGHEVNFICADDAHGTPIMLKAQQLGITPEQMIGEMSQEHQTDFAGFNISYDNYHSTHSEENRQLSELIYSRLKENGFIKNRTISQLYDPEKGMFLPDRFVKGTCPKCKSPDQYGDNCEVCGATYSPTELIEPKSVVSGATPVMRDSEHFFFDLPSFSEMLQAWTRSGALQEQVANKMQEWFESGLQQWDISRDAPYFGFEIPNAPGKYFYVWLDAPIGYMGSFKNLCDKRGDSVSFDEYWKKDSTAELYHFIGKDIVYFHSLFWPAMLEGSNFRKPSNLFVHGYVTVNGAKMSKSRGTFIKASTWLNHFDADSLRYYYTAKLSSRIDDIDLNLEDFVQRVNADIVNKVVNLASRNAGFINKRFDGVLASELADPQLYKTFTDAAEVIGEAWESREFGKAVREIMALADLANRYVDEQAPWVVAKQEGRDADLQAICSMGINLFRVLMTYLKPVLPKLTERAEAFLNTELTWDGIQQPLLGHKVNPFKALYNRIDMRQVEALVEASKEEVKAAAAPVTGPLADDPIQETITFDDFAKVDLRVALIENAEFVEGSDKLLRLTLDLGGEKRNVFSGIRSAYPDPQALIGRHTIMVANLAPRKMRFGISEGMVMAAGPGGKDIFLLSPDAGAKPGHQVK',\n",
       "  'mw': 76255.0},\n",
       " 'b2128': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P33359',\n",
       "  'gene': 'b2128',\n",
       "  'locus': ['b2128', 'JW2116'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of an ABC transporter complex involved in low-affinity glycine betaine uptake. Probably responsible for the translocation of the substrate across the membrane'},\n",
       "  'seq': 'MKMLRDPLFWLIALFVALIFWLPYSQPLFAALFPQLPRPVYQQESFAALALAHFWLVGISSLFAVIIGTGAGIAVTRPWGAEFRPLVETIAAVGQTFPPVAVLAIAVPVIGFGLQPAIIALILYGVLPVLQATLAGLGAIDASVTEVAKGMGMSRGQRVRKVELPLAAPVILAGVRTSVIINIGTATIASTVGASTLGTPIIIGLSGFNTAYVIQGALLVALAAIIADRLFERLVQALSQHAK',\n",
       "  'mw': 25515.0},\n",
       " 'b2129': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P33360',\n",
       "  'gene': 'b2129',\n",
       "  'locus': ['b2129', 'JW2117'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + glycine betaine(out) + H2O = ADP + glycine betaine(in) + H(+) + phosphate',\n",
       "     '')],\n",
       "   'Function': 'Part of an ABC transporter complex involved in low-affinity glycine betaine uptake. Probably responsible for energy coupling to the transport system'},\n",
       "  'seq': 'MIEFSHVSKLFGAQKAVNDLNLNFQEGSFSVLIGTSGSGKSTTLKMINRLVEHDSGEIRFAGEEIRSLPVLELRRRMGYAIQSIGLFPHWSVAQNIATVPQLQKWSRARIDDRIDELMALLGLESNLRERYPHQLSGGQQQRVGVARALAADPQVLLMDEPFGALDPVTRGALQQEMTRIHRLLGRTIVLVTHDIDEALRLAEHLVLMDHGEVVQQGNPLTMLTRPANDFVRQFFGRSELGVRLLSLRSVADYVRREERADGEALAEEMTLRDALSLFVARGCEVLPVVNMQGQPCGTLHFQDLLVEA',\n",
       "  'mw': 34425.0},\n",
       " 'b2130': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P33361',\n",
       "  'gene': 'b2130',\n",
       "  'locus': ['b2130', 'JW2118'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of an ABC transporter complex involved in low-affinity glycine betaine uptake. Probably responsible for the translocation of the substrate across the membrane'},\n",
       "  'seq': 'MTYFRINPVLALLLLLTAIAAALPFISYAPNRLVSGEGRHLWQLWPQTIWMLVGVGCAWLTACFIPGKKGSICALILAQFVFVLLVWGAGKAATQLAQNGSALARTSLGSGFWLAAALALLACSDAIRRISTHPLWRWLLHMQIAIIPLWLLYSGTLNDLSLMKEYANRQDVFDDALAQHLTLLFGAVLPALVIGVPLGIWCYFSTARQGAIFSLLNVIQTVPSVALFGLLIAPLAALVTAFPWLGTLGIAGTGMTPALIALVLYALLPLVRGVVVGLNQIPRDVLESARAMGMSGAQRFLHVQLPLALPVFLRSLRVVMVQTVGMAVIAALIGAGGFGALVFQGLLSSAIDLVLLGVIPVIVLAVLTDALFDLLIALLKVKRND',\n",
       "  'mw': 41139.0},\n",
       " 'b2131': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P33362',\n",
       "  'gene': 'b2131',\n",
       "  'locus': ['b2131', 'JW2119'],\n",
       "  'synonyms': 'osmF',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of an ABC transporter complex involved in low-affinity glycine betaine uptake. Binds glycine betaine with low affinity'},\n",
       "  'seq': 'MPLLKLWAGSLVMLAAVSLPLQAASPVKVGSKIDTEGALLGNIILQVLESHGVPTVNKVQLGTTPVVRGAITSGELDIYPEYTGNGAFFFKDENDAAWKNAQQGYEKVKKLDSEHNKLIWLTPAPANNTWTIAVRQDVAEKNKLTSLADLSRYLQEGGTFKLAASAEFIERADALPAFEKAYGFKLGQDQLLSLAGGDTAVTIKAAAQQTSGVNAAMAYGTDGPVAALGLQTLSDPQGVQPIYAPAPVVRESVLREYPQMAQWLQPVFASLDAKTLQQLNASIAVEGLDAKKVAADYLKQKGWTK',\n",
       "  'mw': 32609.0},\n",
       " 'b2132': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P33363',\n",
       "  'gene': 'b2132',\n",
       "  'locus': ['b2132', 'JW2120'],\n",
       "  'synonyms': 'yohA',\n",
       "  'properties': {'Catalytic Activity': [('Hydrolysis of terminal, non-reducing beta-D-glucosyl residues with release of beta-D-glucose.',\n",
       "     '3.2.1.21')]},\n",
       "  'seq': 'MKWLCSVGIAVSLALQPALADDLFGNHPLTPEARDAFVTELLKKMTVDEKIGQLRLISVGPDNPKEAIREMIKDGQVGAIFNTVTRQDIRAMQDQVMELSRLKIPLFFAYDVLHGQRTVFPISLGLASSFNLDAVKTVGRVSAYEAADDGLNMTWAPMVDVSRDPRWGRASEGFGEDTYLTSTMGKTMVEAMQGKSPADRYSVMTSVKHFAAYGAVEGGKEYNTVDMSPQRLFNDYMPPYKAGLDAGSGAVMVALNSLNGTPATSDSWLLKDVLRDQWGFKGITVSDHGAIKELIKHGTAADPEDAVRVALKSGINMSMSDEYYSKYLPGLIKSGKVTMAELDDAARHVLNVKYDMGLFNDPYSHLGPKESDPVDTNAESRLHRKEAREVARESLVLLKNRLETLPLKKSATIAVVGPLADSKRDVMGSWSAAGVADQSVTVLTGIKNAVGENGKVLYAKGANVTSDKGIIDFLNQYEEAVKVDPRSPQEMIDEAVQTAKQSDVVVAVVGEAQGMAHEASSRTDITIPQSQRDLIAALKATGKPLVLVLMNGRPLALVKEDQQADAILETWFAGTEGGNAIADVLFGDYNPSGKLPMSFPRSVGQIPVYYSHLNTGRPYNADKPNKYTSRYFDEANGALYPFGYGLSYTTFTVSDVKLSAPTMKRDGKVTASVQVTNTGKREGATVVQMYLQDVTASMSRPVKQLKGFEKITLKPGETQTVSFPIDIEALKFWNQQMKYDAEPGKFNVFIGTDSARVKKGEFELL',\n",
       "  'mw': 83460.0},\n",
       " 'b2133': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P06149',\n",
       "  'gene': 'b2133',\n",
       "  'locus': ['b2133', 'JW2121'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(R)-lactate + a quinone = a quinol + pyruvate',\n",
       "     '1.1.5.12')],\n",
       "   'Function': 'Catalyzes the oxidation of D-lactate to pyruvate. Electrons derived from D-lactate oxidation are transferred to the ubiquinone/cytochrome electron transfer chain, where they may be used to provide energy for the active transport of a variety of amino acids and sugars across the membrane',\n",
       "   'Regulation Activity': \"Inhibited by 2-hydroxy-3-butynoic acid, but not by p-chloromercuribenzoate, n-ethylmaleimide, or 5,5'-dithiobis(2-nitrobenzoic acid)\"},\n",
       "  'seq': 'MSSMTTTDNKAFLNELARLVGSSHLLTDPAKTARYRKGFRSGQGDALAVVFPGSLLELWRVLKACVTADKIILMQAANTGLTEGSTPNGNDYDRDVVIISTLRLDKLHVLGKGEQVLAYPGTTLYSLEKALKPLGREPHSVIGSSCIGASVIGGICNNSGGSLVQRGPAYTEMSLFARINEDGKLTLVNHLGIDLGETPEQILSKLDDDRIKDDDVRHDGRHAHDYDYVHRVRDIEADTPARYNADPDRLFESSGCAGKLAVFAVRLDTFEAEKNQQVFYIGTNQPEVLTEIRRHILANFENLPVAGEYMHRDIYDIAEKYGKDTFLMIDKLGTDKMPFFFNLKGRTDAMLEKVKFFRPHFTDRAMQKFGHLFPSHLPPRMKNWRDKYEHHLLLKMAGDGVGEAKSWLVDYFKQAEGDFFVCTPEEGSKAFLHRFAAAGAAIRYQAVHSDEVEDILALDIALRRNDTEWYEHLPPEIDSQLVHKLYYGHFMCYVFHQDYIVKKGVDVHALKEQMLELLQQRGAQYPAEHNVGHLYKAPETLQKFYRENDPTNSMNPGIGKTSKRKNWQEVE',\n",
       "  'mw': 64612.0},\n",
       " 'b2134': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AFI5',\n",
       "  'gene': 'b2134',\n",
       "  'locus': ['b2134', 'JW5355'],\n",
       "  'synonyms': 'yohB',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Cell wall formation. May play a specialized role in remodeling the cell wall. Specifically hydrolyzes the DD-diaminopimelate-alanine bonds in high-molecular-mass murein sacculi'},\n",
       "  'seq': 'MPKFRVSLFSLALMLAVPFAPQAVAKTAAATTASQPEIASGSAMIVDLNTNKVIYSNHPDLVRPIASISKLMTAMVVLDARLPLDEKLKVDISQTPEMKGVYSRVRLNSEISRKDMLLLALMSSENRAAASLAHHYPGGYKAFIKAMNAKAKSLGMNNTRFVEPTGLSVHNVSTARDLTKLLIASKQYPLIGQLSTTREDMATFSNPTYTLPFRNTNHLVYRDNWNIQLTKTGFTNAAGHCLVMRTVINNKPVALVVMDAFGKYTHFADASRLRTWIETGKVMPVPAAALSYKKQKAAQMAAAGQTAQND',\n",
       "  'mw': 33887.0},\n",
       " 'b2143': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ABF6',\n",
       "  'gene': 'b2143',\n",
       "  'locus': ['b2143', 'JW2131'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('cytidine + H(+) + H2O = NH4(+) + uridine',\n",
       "     '3.5.4.5'),\n",
       "    (\"2'-deoxycytidine + H(+) + H2O = 2'-deoxyuridine + NH4(+)\", '3.5.4.5')],\n",
       "   'Function': \"This enzyme scavenges exogenous and endogenous cytidine and 2'-deoxycytidine for UMP synthesis\"},\n",
       "  'seq': 'MHPRFQTAFAQLADNLQSALEPILADKYFPALLTGEQVSSLKSATGLDEDALAFALLPLAAACARTPLSNFNVGAIARGVSGTWYFGANMEFIGATMQQTVHAEQSAISHAWLSGEKALAAITVNYTPCGHCRQFMNELNSGLDLRIHLPGREAHALRDYLPDAFGPKDLEIKTLLMDEQDHGYALTGDALSQAAIAAANRSHMPYSKSPSGVALECKDGRIFSGSYAENAAFNPTLPPLQGALILLNLKGYDYPDIQRAVLAEKADAPLIQWDATSATLKALGCHSIDRVLLA',\n",
       "  'mw': 31540.0},\n",
       " 'b2148': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P23200',\n",
       "  'gene': 'b2148',\n",
       "  'locus': ['b2148', 'JW2135'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the ABC transporter complex MglABC involved in galactose/methyl galactoside import. Probably responsible for the translocation of the substrate across the membrane'},\n",
       "  'seq': 'MSALNKKSFLTYLKEGGIYVVLLVLLAIIIFQDPTFLSLLNLSNILTQSSVRIIIALGVAGLIVTQGTDLSAGRQVGLAAVVAATLLQSMDNANKVFPEMATMPIALVILIVCAIGAVIGLINGLIIAYLNVTPFITTLGTMIIVYGINSLYYDFVGASPISGFDSGFSTFAQGFVALGSFRLSYITFYALIAVAFVWVLWNKTRFGKNIFAIGGNPEAAKVSGVNVGLNLLMIYALSGVFYAFGGMLEAGRIGSATNNLGFMYELDAIAACVVGGVSFSGGVGTVIGVVTGVIIFTVINYGLTYIGVNPYWQYIIKGAIIIFAVALDSLKYARKK',\n",
       "  'mw': 35550.0},\n",
       " 'b2149': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AAG8',\n",
       "  'gene': 'b2149',\n",
       "  'locus': ['b2149', 'JW2136'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + D-galactose(out) + H2O = ADP + D-galactose(in) + H(+) + phosphate',\n",
       "     '7.5.2.11'),\n",
       "    ('ATP + H2O + methyl beta-D-galactoside(out) = ADP + H(+) + methyl beta-D-galactoside(in) + phosphate',\n",
       "     '')],\n",
       "   'Function': 'Part of the ABC transporter complex MglABC involved in galactose/methyl galactoside import. Responsible for energy coupling to the transport system (Probable)'},\n",
       "  'seq': 'MVSSTTPSSGEYLLEMSGINKSFPGVKALDNVNLKVRPHSIHALMGENGAGKSTLLKCLFGIYQKDSGTILFQGKEIDFHSAKEALENGISMVHQELNLVLQRSVMDNMWLGRYPTKGMFVDQDKMYRETKAIFDELDIDIDPRARVGTLSVSQMQMIEIAKAFSYNAKIVIMDEPTSSLTEKEVNHLFTIIRKLKERGCGIVYISHKMEEIFQLCDEVTVLRDGQWIATEPLAGLTMDKIIAMMVGRSLNQRFPDKENKPGEVILEVRNLTSLRQPSIRDVSFDLHKGEILGIAGLVGAKRTDIVETLFGIREKSAGTITLHGKQINNHNANEAINHGFALVTEERRSTGIYAYLDIGFNSLISNIRNYKNKVGLLDNSRMKSDTQWVIDSMRVKTPGHRTQIGSLSGGNQQKVIIGRWLLTQPEILMLDEPTRGIDVGAKFEIYQLIAELAKKGKGIIIISSEMPELLGITDRILVMSNGLVSGIVDTKTTTQNEILRLASLHL',\n",
       "  'mw': 56415.0},\n",
       " 'b2150': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AEE5',\n",
       "  'gene': 'b2150',\n",
       "  'locus': ['b2150', 'JW2137'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the ABC transporter complex MglABC involved in galactose/methyl galactoside import (Probable). In addition, binds D-galactose and D-glucose and plays a role in the chemotaxis towards these two sugars by interacting with the Trg chemoreceptor (PubMed:3057628, PubMed:4927373). Chemotaxis requires MglB, but not MglA or MglC (PubMed:6294056)'},\n",
       "  'seq': 'MNKKVLTLSAVMASMLFGAAAHAADTRIGVTIYKYDDNFMSVVRKAIEQDAKAAPDVQLLMNDSQNDQSKQNDQIDVLLAKGVKALAINLVDPAAAGTVIEKARGQNVPVVFFNKEPSRKALDSYDKAYYVGTDSKESGIIQGDLIAKHWAANQGWDLNKDGQIQFVLLKGEPGHPDAEARTTYVIKELNDKGIKTEQLQLDTAMWDTAQAKDKMDAWLSGPNANKIEVVIANNDAMAMGAVEALKAHNKSSIPVFGVDALPEALALVKSGALAGTVLNDANNQAKATFDLAKNLADGKGAADGTNWKIDNKVVRVPYVGVDKDNLAEFSKK',\n",
       "  'mw': 35713.0},\n",
       " 'b2153': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A6T5',\n",
       "  'gene': 'b2153',\n",
       "  'locus': ['b2153', 'JW2140'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [(\"GTP + H2O = 7,8-dihydroneopterin 3'-triphosphate + formate + H(+)\",\n",
       "     '3.5.4.16')],\n",
       "   'Regulation Activity': 'Allosteric enzyme. Activity is modulated by K(+), divalent cations, UTP, and tetrahydrobiopterin. Tetrahydrobiopterin is an inhibitor of this enzyme',\n",
       "   'Pathway': 'Cofactor biosynthesis; 7,8-dihydroneopterin triphosphate biosynthesis; 7,8-dihydroneopterin triphosphate from GTP: step 1/1'},\n",
       "  'seq': 'MPSLSKEAALVHEALVARGLETPLRPPVHEMDNETRKSLIAGHMTEIMQLLNLDLADDSLMETPHRIAKMYVDEIFSGLDYANFPKITLIENKMKVDEMVTVRDITLTSTCEHHFVTIDGKATVAYIPKDSVIGLSKINRIVQFFAQRPQVQERLTQQILIALQTLLGTNNVAVSIDAVHYCVKARGIRDATSATTTTSLGGLFKSSQNTRHEFLRAVRHHN',\n",
       "  'mw': 24831.0},\n",
       " 'b2154': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P33018',\n",
       "  'gene': 'b2154',\n",
       "  'locus': ['b2154', 'JW2141'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('H2O + S-formylglutathione = formate + glutathione + H(+)',\n",
       "     '3.1.2.12')],\n",
       "   'Function': 'Serine hydrolase involved in the detoxification of formaldehyde. Hydrolyzes S-formylglutathione to glutathione and formate. Shows also esterase activity against alpha-naphthyl acetate, lactoylglutathione, palmitoyl-CoA and several pNP-esters of short chain fatty acids',\n",
       "   'Regulation Activity': 'Inhibited by the sulfhydryl inhibitors (N-ethylmaleimide, iodoacetate, ZnCl(2) and CuCl(2))'},\n",
       "  'seq': 'MEMLEEHRCFEGWQQRWRHDSSTLNCPMTFSIFLPPPRDHTPPPVLYWLSGLTCNDENFTTKAGAQRVAAELGIVLVMPDTSPRGEKVANDDGYDLGQGAGFYLNATQPPWATHYRMYDYLRDELPALVQSQFNVSDRCAISGHSMGGHGALIMALKNPGKYTSVSAFAPIVNPCSVPWGIKAFSSYLGEDKNAWLEWDSCALMYASNAQDAIPTLIDQGDNDQFLADQLQPAVLAEAARQKAWPMTLRIQPGYDHSYYFIASFIEDHLRFHAQYLLK',\n",
       "  'mw': 31259.0},\n",
       " 'b2155': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P17315',\n",
       "  'gene': 'b2155',\n",
       "  'locus': ['b2155', 'JW2142'],\n",
       "  'synonyms': 'cir feuA',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Not yet known. Postulated to participate in iron transport. Outer membrane receptor for colicins IA and IB'},\n",
       "  'seq': 'MFRLNPFVRVGLCLSAISCAWPVLAVDDDGETMVVTASSVEQNLKDAPASISVITQEDLQRKPVQNLKDVLKEVPGVQLTNEGDNRKGVSIRGLDSSYTLILVDGKRVNSRNAVFRHNDFDLNWIPVDSIERIEVVRGPMSSLYGSDALGGVVNIITKKIGQKWSGTVTVDTTIQEHRDRGDTYNGQFFTSGPLIDGVLGMKAYGSLAKREKDDPQNSTTTDTGETPRIEGFSSRDGNVEFAWTPNQNHDFTAGYGFDRQDRDSDSLDKNRLERQNYSVSHNGRWDYGTSELKYYGEKVENKNPGNSSPITSESNTVDGKYTLPLTAINQFLTVGGEWRHDKLSDAVNLTGGTSSKTSASQYALFVEDEWRIFEPLALTTGVRMDDHETYGEHWSPRAYLVYNATDTVTVKGGWATAFKAPSLLQLSPDWTSNSCRGACKIVGSPDLKPETSESWELGLYYMGEEGWLEGVESSVTVFRNDVKDRISISRTSDVNAAPGYQNFVGFETGANGRRIPVFSYYNVNKARIQGVETELKIPFNDEWKLSINYTYNDGRDVSNGENKPLSDLPFHTANGTLDWKPLALEDWSFYVSGHYTGQKRADSATAKTPGGYTIWNTGAAWQVTKDVKLRAGVLNLGDKDLSRDDYSYNEDGRRYFMAVDYRF',\n",
       "  'mw': 73896.0},\n",
       " 'b2156': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P25737',\n",
       "  'gene': 'b2156',\n",
       "  'locus': ['b2156', 'JW2143'],\n",
       "  'synonyms': 'cadR',\n",
       "  'properties': {'Catalytic Activity': [('H(+)(out) + L-lysine(out) = H(+)(in) + L-lysine(in)',\n",
       "     '')],\n",
       "   'Function': 'Permease involved in lysine uptake (PubMed:1315732, PubMed:24056175). In addition, functions as a lysine sensor that mediates the lysine-dependent regulation of the transcriptional activator CadC (PubMed:18086202, PubMed:24056175). In the absence of lysine, or at low lysine concentrations, LysP inhibits CadC by an interaction with the transmembrane domain of CadC. In the presence of lysine, LysP loses its ability to interact with and inhibit CadC, and acts as a lysine permease (PubMed:18086202, PubMed:24056175)'},\n",
       "  'seq': 'MVSETKTTEAPGLRRELKARHLTMIAIGGSIGTGLFVASGATISQAGPGGALLSYMLIGLMVYFLMTSLGELAAYMPVSGSFATYGQNYVEEGFGFALGWNYWYNWAVTIAVDLVAAQLVMSWWFPDTPGWIWSALFLGVIFLLNYISVRGFGEAEYWFSLIKVTTVIVFIIVGVLMIIGIFKGAQPAGWSNWTIGEAPFAGGFAAMIGVAMIVGFSFQGTELIGIAAGESEDPAKNIPRAVRQVFWRILLFYVFAILIISLIIPYTDPSLLRNDVKDISVSPFTLVFQHAGLLSAAAVMNAVILTAVLSAGNSGMYASTRMLYTLACDGKAPRIFAKLSRGGVPRNALYATTVIAGLCFLTSMFGNQTVYLWLLNTSGMTGFIAWLGIAISHYRFRRGYVLQGHDINDLPYRSGFFPLGPIFAFILCLIITLGQNYEAFLKDTIDWGGVAATYIGIPLFLIIWFGYKLIKGTHFVRYSEMKFPQNDKK',\n",
       "  'mw': 53576.0},\n",
       " 'b2162': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P33022',\n",
       "  'gene': 'b2162',\n",
       "  'locus': ['b2162', 'JW2149'],\n",
       "  'synonyms': 'yeiK',\n",
       "  'properties': {'Catalytic Activity': [('a pyrimidine ribonucleoside + H2O = a pyrimidine nucleobase + D-ribose',\n",
       "     '3.2.2.8')],\n",
       "   'Function': 'Hydrolyzes cytidine or uridine to ribose and cytosine or uracil, respectively. Has a clear preference for cytidine over uridine. Strictly specific for ribonucleosides. Has a low but significant activity for the purine nucleoside xanthosine'},\n",
       "  'seq': 'MEKRKIILDCDPGHDDAIAIMMAAKHPAIDLLGITIVAGNQTLDKTLINGLNVCQKLEINVPVYAGMPQPIMRQQIVADNIHGETGLDGPVFEPLTRQAESTHAVKYIIDTLMASDGDITLVPVGPLSNIAVAMRMQPAILPKIREIVLMGGAYGTGNFTPSAEFNIFADPEAARVVFTSGVPLVMMGLDLTNQTVCTPDVIARMERAGGPAGELFSDIMNFTLKTQFENYGLAGGPVHDATCIGYLINPDGIKTQEMYVEVDVNSGPCYGRTVCDELGVLGKPANTKVGITIDTDWFWGLVEECVRGYIKTH',\n",
       "  'mw': 33748.0},\n",
       " 'b2167': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P20966',\n",
       "  'gene': 'b2167',\n",
       "  'locus': ['b2167', 'JW2154'],\n",
       "  'synonyms': 'ptsF',\n",
       "  'properties': {'Catalytic Activity': [('D-fructose(out) + N(pros)-phospho-L-histidyl-[protein] = D-fructose 1-phosphate(in) + L-histidyl-[protein]',\n",
       "     '2.7.1.202')],\n",
       "   'Function': 'The phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS), a major carbohydrate active transport system, catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane. The enzyme II FruAB PTS system is involved in fructose transport'},\n",
       "  'seq': 'MKTLLIIDANLGQARAYMAKTLLGAAARKAKLEIIDNPNDAEMAIVLGDSIPNDSALNGKNVWLGDISRAVAHPELFLSEAKGHAKPYTAPVAATAPVAASGPKRVVAVTACPTGVAHTFMAAEAIETEAKKRGWWVKVETRGSVGAGNAITPEEVAAADLVIVAADIEVDLAKFAGKPMYRTSTGLALKKTAQELDKAVAEATPYEPAGKAQTATTESKKESAGAYRHLLTGVSYMLPMVVAGGLCIALSFAFGIEAFKEPGTLAAALMQIGGGSAFALMVPVLAGYIAFSIADRPGLTPGLIGGMLAVSTGSGFIGGIIAGFLAGYIAKLISTQLKLPQSMEALKPILIIPLISSLVVGLAMIYLIGKPVAGILEGLTHWLQTMGTANAVLLGAILGGMMCTDMGGPVNKAAYAFGVGLLSTQTYGPMAAIMAAGMVPPLAMGLATMVARRKFDKAQQEGGKAALVLGLCFISEGAIPFAARDPMRVLPCCIVGGALTGAISMAIGAKLMAPHGGLFVLLIPGAITPVLGYLVAIIAGTLVAGLAYAFLKRPEVDAVAKAA',\n",
       "  'mw': 57519.0},\n",
       " 'b2168': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AEW9',\n",
       "  'gene': 'b2168',\n",
       "  'locus': ['b2168', 'JW2155'],\n",
       "  'synonyms': 'fpk',\n",
       "  'properties': {'Catalytic Activity': [('ATP + beta-D-fructose 1-phosphate = ADP + beta-D-fructose 1,6-bisphosphate + H(+)',\n",
       "     '2.7.1.56')],\n",
       "   'Function': 'Catalyzes the ATP-dependent phosphorylation of fructose-l-phosphate to fructose-l,6-bisphosphate (Ref.6, PubMed:10833389). Is specific for fructose-l-phosphate (Ref.6). GTP, UTP and CTP can also function as phosphoryl donors showing 60%, 20% and 10% of the activity of ATP (Ref.6)',\n",
       "   'Regulation Activity': 'Activity is markedly stimulated by KCl (PubMed:10833389). Reversibly inhibited by fructose-1,6- bisphosphate and ADP. Irreversibly inhibited by phenylmethanesulfonyl fluoride (PMSF) (Ref.6)'},\n",
       "  'seq': 'MSRRVATITLNPAYDLVGFCPEIERGEVNLVKTTGLHAAGKGINVAKVLKDLGIDVTVGGFLGKDNQDGFQQLFSELGIANRFQVVQGRTRINVKLTEKDGEVTDFNFSGFEVTPADWERFVTDSLSWLGQFDMVCVSGSLPSGVSPEAFTDWMTRLRSQCPCIIFDSSREALVAGLKAAPWLVKPNRRELEIWAGRKLPEMKDVIEAAHALREQGIAHVVISLGAEGALWVNASGEWIAKPPSVDVVSTVGAGDSMVGGLIYGLLMRESSEHTLRLATAVAALAVSQSNVGITDRPQLAAMMARVDLQPFN',\n",
       "  'mw': 33756.0},\n",
       " 'b2169': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P69811',\n",
       "  'gene': 'b2169',\n",
       "  'locus': ['b2169', 'JW2156'],\n",
       "  'synonyms': 'fpr fruF',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'The phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS), a major carbohydrate active transport system, catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane. The enzyme II FruAB PTS system is involved in fructose transport'},\n",
       "  'seq': 'MFQLSVQDIHPGEKAGDKEEAIRQVAAALVQAGNVAEGYVNGMLAREQQTSTFLGNGIAIPHGTTDTRDQVLKTGVQVFQFPEGVTWGDGQVAYVAIGIAASSDEHLGLLRQLTHVLSDDSVAEQLKSATTAEELRALLMGEKQSEQLKLDNEMLTLDIVASDLLTLQALNAARLKEAGAVDATFVTKAINEQPLNLGQGIWLSDSAEGNLRSAIAVSRAANAFDVDGETAAMLVSVAMNDDQPIAVLKRLADLLLDNKADRLLKADAATLLALLTSDDAPTDDVLSAEFVVRNEHGLHARPGTMLVNTIKQFNSDITVTNLDGTGKPANGRSLMKVVALGVKKGHRLRFTAQGADAEQALKAIGDAIAAGLGEGA',\n",
       "  'mw': 39648.0},\n",
       " 'b2170': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P33026',\n",
       "  'gene': 'b2170',\n",
       "  'locus': ['b2170', 'JW2157'],\n",
       "  'synonyms': 'yeiO',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Involved in the efflux of sugars. The physiological role may be the detoxification of non-metabolizable sugar analogs. Can transport lactose and glucose'},\n",
       "  'seq': 'MHNSPAVSSAKSFDLTSTAFLIVAFLTGIAGALQTPTLSIFLTDEVHARPAMVGFFFTGSAVIGILVSQFLAGRSDKRGDRKSLIVFCCLLGVLACTLFAWNRNYFVLLFVGVFLSSFGSTANPQMFALAREHADKTGREAVMFSSFLRAQVSLAWVIGPPLAYALAMGFSFTVMYLSAAVAFIVCGVMVWLFLPSMRKELPLATGTIEAPRRNRRDTLLLFVICTLMWGSNSLYIINMPLFIINELHLPEKLAGVMMGTAAGLEIPTMLIAGYFAKRLGKRFLMRVAAVGGVCFYAGMLMAHSPVILLGLQLLNAIFIGILGGIGMLYFQDLMPGQAGSATTLYTNTSRVGWIIAGSVAGIVAEIWNYHAVFWFAMVMIIATLFCLLRIKDV',\n",
       "  'mw': 42746.0},\n",
       " 'b2199': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ABM1',\n",
       "  'gene': 'b2199',\n",
       "  'locus': ['b2199', 'JW2187'],\n",
       "  'synonyms': 'yejT yejU',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Required for the export of heme to the periplasm for the biogenesis of c-type cytochromes'},\n",
       "  'seq': 'MWKTLHQLAIPPRLYQICGWFIPWLAIASVVVLTVGWIWGFGFAPADYQQGNSYRIIYLHVPAAIWSMGIYASMAVAAFIGLVWQMKMANLAVAAMAPIGAVFTFIALVTGSAWGKPMWGTWWVWDARLTSELVLLFLYVGVIALWHAFDDRRLAGRAAGILVLIGVVNLPIIHYSVEWWNTLHQGSTRMQQSIDPAMRSPLRWSIFGFLLLSATLTLMRMRNLILLMEKRRPWVSELILKRGRK',\n",
       "  'mw': 27885.0},\n",
       " 'b2200': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ABL8',\n",
       "  'gene': 'b2200',\n",
       "  'locus': ['b2200', 'JW2188'],\n",
       "  'synonyms': 'yejV',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Required for the export of heme to the periplasm for the biogenesis of c-type cytochromes'},\n",
       "  'seq': 'MMFWRIFRLELRVAFRHSAEIANPLWFFLIVITLFPLSIGPEPQLLARIAPGIIWVAALLSSLLALERLFRDDLQDGSLEQLMLLPLPLPAVVLAKVMAHWMVTGLPLLILSPLVAMLLGMDVYGWQVMALTLLLGTPTLGFLGAPGVALTVGLKRGGVLLSILVLPLTIPLLIFATAAMDAASMHLPVDGYLAILGALLAGTATLSPFATAAALRISIQ',\n",
       "  'mw': 23619.0},\n",
       " 'b2201': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P33931',\n",
       "  'gene': 'b2201',\n",
       "  'locus': ['b2201', 'JW5366'],\n",
       "  'synonyms': 'yejW',\n",
       "  'properties': {'Catalytic Activity': [('ATP + H2O + heme b(in) = ADP + H(+) + heme b(out) + phosphate',\n",
       "     '7.6.2.5')],\n",
       "   'Function': 'Part of the ABC transporter complex CcmAB involved in the biogenesis of c-type cytochromes; once thought to export heme, this seems not to be the case, but its exact role is uncertain. Responsible for energy coupling to the transport system'},\n",
       "  'seq': 'MGMLEARELLCERDERTLFSGLSFTLNAGEWVQITGSNGAGKTTLLRLLTGLSRPDAGEVLWQGQPLHQVRDSYHQNLLWIGHQPGIKTRLTALENLHFYHRDGDTAQCLEALAQAGLAGFEDIPVNQLSAGQQRRVALARLWLTRATLWILDEPFTAIDVNGVDRLTQRMAQHTEQGGIVILTTHQPLNVAESKIRRISLTQTRAA',\n",
       "  'mw': 23053.0},\n",
       " 'b2202': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ABL5',\n",
       "  'gene': 'b2202',\n",
       "  'locus': ['b2202', 'JW2190'],\n",
       "  'synonyms': 'yejX',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Mediates electron flow from quinones to the NapAB complex'},\n",
       "  'seq': 'MGNSDRKPGLIKRLWKWWRTPSRLALGTLLLIGFVGGIVFWGGFNTGMEKANTEEFCISCHEMRNTVYQEYMDSVHYNNRSGVRATCPDCHVPHEFVPKMIRKLKASKELYGKIFGVIDTPQKFEAHRLTMAQNEWRRMKDNNSQECRNCHNFEYMDTTAQKSVAAKMHDQAVKDGQTCIDCHKGIAHKLPDMREVEPGF',\n",
       "  'mw': 23101.0},\n",
       " 'b2203': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ABL3',\n",
       "  'gene': 'b2203',\n",
       "  'locus': ['b2203', 'JW5367'],\n",
       "  'synonyms': 'yejY',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Electron transfer subunit of the periplasmic nitrate reductase complex NapAB. Receives electrons from the membrane-anchored tetraheme c-type NapC protein and transfers these to NapA subunit, thus allowing electron flow between membrane and periplasm. Essential for periplasmic nitrate reduction with nitrate as the terminal electron acceptor'},\n",
       "  'seq': 'MKSHDLKKALCQWTAMLALVVSGAVWAANGVDFSQSPEVSGTQEGAIRMPKEQDRMPLNYVNQPPMIPHSVEGYQVTTNTNRCLQCHGVESYRTTGAPRISPTHFMDSDGKVGAEVAPRRYFCLQCHVPQADTAPIVGNTFTPSKGYGK',\n",
       "  'mw': 16297.0},\n",
       " 'b2204': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P33934',\n",
       "  'gene': 'b2204',\n",
       "  'locus': ['b2204', 'JW2192'],\n",
       "  'synonyms': 'yejZ',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Required for electron transfer from ubiquinol, via NapC, to the periplasmic nitrate reductase NapAB complex'},\n",
       "  'seq': 'MANRKRDAGREALEKKGWWRSHRWLVLRRLCQFFVLGMFLSGPWFGVWILHGNYSSSLLFDTVPLTDPLMTLQSLASGHLPATVALTGAVIITVLYALAGKRLFCSWVCPLNPITDLANWLRRRFDLNQSATIPRHIRYVLLVVILVGSALTGTLIWEWINPVSLMGRSLVMGFGSGALLILALFLFDLLVVEHGWCGHICPVGALYGVLGSKGVITVAATDRQKCNRCMDCFHVCPEPHVLRAPVLDEQSPVQVTSRDCMTCGRCVDVCSEDVFTITTRWSSGAKS',\n",
       "  'mw': 31874.0},\n",
       " 'b2205': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AAL3',\n",
       "  'gene': 'b2205',\n",
       "  'locus': ['b2205', 'JW2193'],\n",
       "  'synonyms': 'yojA yojB',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Required for electron transfer from ubiquinol, via NapC, to the periplasmic nitrate reductase NapAB complex'},\n",
       "  'seq': 'MSRSAKPQNGRRRFLRDVVRTAGGLAAVGVALGLQQQTARASGVRLRPPGAINENAFASACVRCGQCVQACPYDTLKLATLASGLSAGTPYFVARDIPCEMCEDIPCAKVCPSGALDREIESIDDARMGLAVLVDQENCLNFQGLRCDVCYRECPKIDEAITLELERNTRTGKHARFLPTVHSDACTGCGKCEKVCVLEQPAIKVLPLSLAKGELGHHYRFGWLEGNNGKS',\n",
       "  'mw': 24925.0},\n",
       " 'b2206': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P33937',\n",
       "  'gene': 'b2206',\n",
       "  'locus': ['b2206', 'JW2194'],\n",
       "  'synonyms': 'yojC yojD yojE',\n",
       "  'properties': {'Catalytic Activity': [('2 Fe(II)-[cytochrome] + 2 H(+) + nitrate = 2 Fe(III)-[cytochrome] + H2O + nitrite',\n",
       "     '1.9.6.1')],\n",
       "   'Function': 'Catalytic subunit of the periplasmic nitrate reductase complex NapAB. Receives electrons from NapB and catalyzes the reduction of nitrate to nitrite',\n",
       "   'Regulation Activity': 'Interaction in the cytoplasm with the NapD chaperone prevents premature export'},\n",
       "  'seq': 'MKLSRRSFMKANAVAAAAAAAGLSVPGVARAVVGQQEAIKWDKAPCRFCGTGCGVLVGTQQGRVVACQGDPDAPVNRGLNCIKGYFLPKIMYGKDRLTQPLLRMKNGKYDKEGEFTPITWDQAFDVMEEKFKTALKEKGPESIGMFGSGQWTIWEGYAASKLFKAGFRSNNIDPNARHCMASAVVGFMRTFGMDEPMGCYDDIEQADAFVLWGANMAEMHPILWSRITNRRLSNQNVTVAVLSTYQHRSFELADNGIIFTPQSDLVILNYIANYIIQNNAINQDFFSKHVNLRKGATDIGYGLRPTHPLEKAAKNPGSDASEPMSFEDYKAFVAEYTLEKTAEMTGVPKDQLEQLAQLYADPNKKVISYWTMGFNQHTRGVWANNLVYNLHLLTGKISQPGCGPFSLTGQPSACGTAREVGTFAHRLPADMVVTNEKHRDICEKKWNIPSGTIPAKIGLHAVAQDRALKDGKLNVYWTMCTNNMQAGPNINEERMPGWRDPRNFIIVSDPYPTVSALAADLILPTAMWVEKEGAYGNAERRTQFWRQQVQAPGEAKSDLWQLVQFSRRFKTEEVWPEDLLAKKPELRGKTLYEVLYATPEVSKFPVSELAEDQLNDESRELGFYLQKGLFEEYAWFGRGHGHDLAPFDDYHKARGLRWPVVNGKETQWRYSEGNDPYVKAGEGYKFYGKPDGKAVIFALPFEPAAEAPDEEYDLWLSTGRVLEHWHTGSMTRRVPELHRAFPEAVLFIHPLDAKARDLRRGDKVKVVSRRGEVISIVETRGRNRPPQGLVYMPFFDAAQLVNKLTLDATDPLSKETDFKKCAVKLEKV',\n",
       "  'mw': 93042.0},\n",
       " 'b2210': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P33940',\n",
       "  'gene': 'b2210',\n",
       "  'locus': ['b2210', 'JW2198'],\n",
       "  'synonyms': 'yojH',\n",
       "  'properties': {'Catalytic Activity': [('(S)-malate + a quinone = a quinol + oxaloacetate',\n",
       "     '1.1.5.4')],\n",
       "   'Pathway': 'Carbohydrate metabolism; tricarboxylic acid cycle; oxaloacetate from (S)-malate (quinone route): step 1/1'},\n",
       "  'seq': 'MKKVTAMLFSMAVGLNAVSMAAKAKASEEQETDVLLIGGGIMSATLGTYLRELEPEWSMTMVERLEGVAQESSNGWNNAGTGHSALMELNYTPQNADGSISIEKAVAINEAFQISRQFWAHQVERGVLRTPRSFINTVPHMSFVWGEDNVNFLRARYAALQQSSLFRGMRYSEDHAQIKEWAPLVMEGRDPQQKVAATRTEIGTDVNYGEITRQLIASLQKKSNFSLQLSSEVRALKRNDDNTWTVTVADLKNGTAQNIRAKFVFIGAGGAALKLLQESGIPEAKDYAGFPVGGQFLVSENPDVVNHHLAKVYGKASVGAPPMSVPHIDTRVLDGKRVVLFGPFATFSTKFLKNGSLWDLMSSTTTSNVMPMMHVGLDNFDLVKYLVSQVMLSEEDRFEALKEYYPQAKKEDWRLWQAGQRVQIIKRDAEKGGVLRLGTEVVSDQQGTIAALLGASPGASTAAPIMLNLLEKVFGDRVSSPQWQATLKAIVPSYGRKLNGDVAATERELQYTSEVLGLNYDKPQAADSTPKPQLKPQPVQKEVADIAL',\n",
       "  'mw': 60230.0},\n",
       " 'b2215': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P06996',\n",
       "  'gene': 'b2215',\n",
       "  'locus': ['b2215', 'JW2203'],\n",
       "  'synonyms': 'meoA par',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': '(Microbial infection) A mixed OmpC-OmpF heterotrimer is the outer membrane receptor for toxin CdiA-EC536; polymorphisms in extracellular loops 4 and 5 of OmpC confer susceptibility to CdiA-EC536-mediated toxicity'},\n",
       "  'seq': 'MKVKVLSLLVPALLVAGAANAAEVYNKDGNKLDLYGKVDGLHYFSDNKDVDGDQTYMRLGFKGETQVTDQLTGYGQWEYQIQGNSAENENNSWTRVAFAGLKFQDVGSFDYGRNYGVVYDVTSWTDVLPEFGGDTYGSDNFMQQRGNGFATYRNTDFFGLVDGLNFAVQYQGKNGNPSGEGFTSGVTNNGRDALRQNGDGVGGSITYDYEGFGIGGAISSSKRTDAQNTAAYIGNGDRAETYTGGLKYDANNIYLAAQYTQTYNATRVGSLGWANKAQNFEAVAQYQFDFGLRPSLAYLQSKGKNLGRGYDDEDILKYVDVGATYYFNKNMSTYVDYKINLLDDNQFTRDAGINTDNIVALGLVYQF',\n",
       "  'mw': 40368.0},\n",
       " 'b2221': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P76458',\n",
       "  'gene': 'b2221',\n",
       "  'locus': ['b2221', 'JW2215'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('acetate + an acyl-CoA = a carboxylate + acetyl-CoA',\n",
       "     '2.8.3.8'),\n",
       "    ('acetoacetate + acetyl-CoA = acetate + acetoacetyl-CoA', ''),\n",
       "    ('acetyl-CoA + butanoate = acetate + butanoyl-CoA', ''),\n",
       "    ('acetoacetate + butanoyl-CoA = acetoacetyl-CoA + butanoate', '')],\n",
       "   'Function': 'Coenzyme A transferase which is involved in short-chain fatty acid degradation and catalyzes the activation of short-chain fatty acids to their respective CoA thiolesters (PubMed:1103739, PubMed:3025185). During acetoacetate degradation, catalyzes the transfer of CoA from acetyl-CoA to acetoacetate by a mechanism involving a covalent enzyme-CoA compound as a reaction intermediate (PubMed:1103741). Utilizes a variety of short chain acyl-CoA and carboxylic acid substrates but exhibits maximal activity with normal and 3-keto substrates (PubMed:1103739)',\n",
       "   'Regulation Activity': 'Inhibited by p-chloromercuribenzoate',\n",
       "   'Pathway': 'Lipid metabolism; short-chain fatty acid metabolism'},\n",
       "  'seq': 'MKTKLMTLQDATGFFRDGMTIMVGGFMGIGTPSRLVEALLESGVRDLTLIANDTAFVDTGIGPLIVNGRVRKVIASHIGTNPETGRRMISGEMDVVLVPQGTLIEQIRCGGAGLGGFLTPTGVGTVVEEGKQTLTLDGKTWLLERPLRADLALIRAHRCDTLGNLTYQLSARNFNPLIALAADITLVEPDELVETGELQPDHIVTPGAVIDHIIVSQESK',\n",
       "  'mw': 23526.0},\n",
       " 'b2222': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P76459',\n",
       "  'gene': 'b2222',\n",
       "  'locus': ['b2222', 'JW2216'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('acetate + an acyl-CoA = a carboxylate + acetyl-CoA',\n",
       "     '2.8.3.8'),\n",
       "    ('acetoacetate + acetyl-CoA = acetate + acetoacetyl-CoA', ''),\n",
       "    ('acetyl-CoA + butanoate = acetate + butanoyl-CoA', ''),\n",
       "    ('acetoacetate + butanoyl-CoA = acetoacetyl-CoA + butanoate', '')],\n",
       "   'Function': 'Coenzyme A transferase which is involved in short-chain fatty acid degradation and catalyzes the activation of short-chain fatty acids to their respective CoA thiolesters (PubMed:1103739, PubMed:3025185). During acetoacetate degradation, catalyzes the transfer of CoA from acetyl-CoA to acetoacetate by a mechanism involving a covalent enzyme-CoA compound as a reaction intermediate (PubMed:1103741). Utilizes a variety of short chain acyl-CoA and carboxylic acid substrates but exhibits maximal activity with normal and 3-keto substrates (PubMed:1103739)',\n",
       "   'Regulation Activity': 'Inhibited by p-chloromercuribenzoate',\n",
       "   'Pathway': 'Lipid metabolism; short-chain fatty acid metabolism'},\n",
       "  'seq': 'MDAKQRIARRVAQELRDGDIVNLGIGLPTMVANYLPEGIHITLQSENGFLGLGPVTTAHPDLVNAGGQPCGVLPGAAMFDSAMSFALIRGGHIDACVLGGLQVDEEANLANWVVPGKMVPGMGGAMDLVTGSRKVIIAMEHCAKDGSAKILRRCTMPLTAQHAVHMLVTELAVFRFIDGKMWLTEIADGCDLATVRAKTEARFEVAADLNTQRGDL',\n",
       "  'mw': 22960.0},\n",
       " 'b2223': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P76460',\n",
       "  'gene': 'b2223',\n",
       "  'locus': ['b2223', 'JW2217'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'May be responsible for the uptake of short-chain fatty acids'},\n",
       "  'seq': 'MIGRISRFMTRFVSRWLPDPLIFAMLLTLLTFVIALWLTPQTPISMVKMWGDGFWNLLAFGMQMALIIVTGHALASSAPVKSLLRTAASAAKTPVQGVMLVTFFGSVACVINWGFGLVVGAMFAREVARRVPGSDYPLLIACAYIGFLTWGGGFSGSMPLLAATPGNPVEHIAGLIPVGDTLFSGFNIFITVALIVVMPFITRMMMPKPSDVVSIDPKLLMEEADFQKQLPKDAPPSERLEESRILTLIIGALGIAYLAMYFSEHGFNITINTVNLMFMIAGLLLHKTPMAYMRAISAAARSTAGILVQFPFYAGIQLMMEHSGLGGLITEFFINVANKDTFPVMTFFSSALINFAVPSGGGHWVIQGPFVIPAAQALGADLGKSVMAIAYGEQWMNMAQPFWALPALAIAGLGVRDIMGYCITALLFSGVIFVIGLTLF',\n",
       "  'mw': 47528.0},\n",
       " 'b2224': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P76461',\n",
       "  'gene': 'b2224',\n",
       "  'locus': ['b2224', 'JW2218'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('2 acetyl-CoA = acetoacetyl-CoA + CoA',\n",
       "     '2.3.1.9')],\n",
       "   'Pathway': 'Metabolic intermediate biosynthesis; (R)-mevalonate biosynthesis; (R)-mevalonate from acetyl-CoA: step 1/3'},\n",
       "  'seq': 'MKNCVIVSAVRTAIGSFNGSLASTSAIDLGATVIKAAIERAKIDSQHVDEVIMGNVLQAGLGQNPARQALLKSGLAETVCGFTVNKVCGSGLKSVALAAQAIQAGQAQSIVAGGMENMSLAPYLLDAKARSGYRLGDGQVYDVILRDGLMCATHGYHMGITAENVAKEYGITREMQDELALHSQRKAAAAIESGAFTAEIVPVNVVTRKKTFVFSQDEFPKANSTAEALGALRPAFDKAGTVTAGNASGINDGAAALVIMEESAALAAGLTPLARIKSYASGGVPPALMGMGPVPATQKALQLAGLQLADIDLIEANEAFAAQFLAVGKNLGFDSEKVNVNGGAIALGHPIGASGARILVTLLHAMQARDKTLGLATLCIGGGQGIAMVIERLN',\n",
       "  'mw': 40352.0},\n",
       " 'b2232': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P17993',\n",
       "  'gene': 'b2232',\n",
       "  'locus': ['b2232', 'JW2226'],\n",
       "  'synonyms': 'pufX yfaB',\n",
       "  'properties': {'Catalytic Activity': [('a 3-demethylubiquinol + S-adenosyl-L-methionine = a ubiquinol + H(+) + S-adenosyl-L-homocysteine',\n",
       "     '2.1.1.64'),\n",
       "    ('a 3-(all-trans-polyprenyl)benzene-1,2-diol + S-adenosyl-L-methionine = a 2-methoxy-6-(all-trans-polyprenyl)phenol + H(+) + S-adenosyl-L-homocysteine',\n",
       "     '2.1.1.222')],\n",
       "   'Function': 'O-methyltransferase that catalyzes the 2 O-methylation steps in the ubiquinone biosynthetic pathway',\n",
       "   'Pathway': 'Cofactor biosynthesis; ubiquinone biosynthesis'},\n",
       "  'seq': 'MNAEKSPVNHNVDHEEIAKFEAVASRWWDLEGEFKPLHRINPLRLGYIAERAGGLFGKKVLDVGCGGGILAESMAREGATVTGLDMGFEPLQVAKLHALESGIQVDYVQETVEEHAAKHAGQYDVVTCMEMLEHVPDPQSVVRACAQLVKPGGDVFFSTLNRNGKSWLMAVVGAEYILRMVPKGTHDVKKFIKPAELLGWVDQTSLKERHITGLHYNPITNTFKLGPGVDVNYMLHTQNK',\n",
       "  'mw': 26555.0},\n",
       " 'b2234': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P00452',\n",
       "  'gene': 'b2234',\n",
       "  'locus': ['b2234', 'JW2228'],\n",
       "  'synonyms': 'dnaF',\n",
       "  'properties': {'Catalytic Activity': [(\"[thioredoxin]-disulfide + a 2'-deoxyribonucleoside 5'-diphosphate + H2O = [thioredoxin]-dithiol + a ribonucleoside 5'-diphosphate\",\n",
       "     '1.17.4.1')],\n",
       "   'Function': 'Provides the precursors necessary for DNA synthesis. Catalyzes the biosynthesis of deoxyribonucleotides from the corresponding ribonucleotides. R1 contains the binding sites for both substrates and allosteric effectors and carries out the actual reduction of the ribonucleotide. It also provides redox-active cysteines',\n",
       "   'Regulation Activity': 'Under complex allosteric control mediated by deoxynucleoside triphosphates and ATP binding to separate specificity and activation sites on the alpha subunit. The type of nucleotide bound at the specificity site determines substrate preference. It seems probable that ATP makes the enzyme reduce CDP and UDP, dGTP favors ADP reduction and dTTP favors GDP reduction. Stimulated by ATP and inhibited by dATP binding to the activity site. In vitro, its activity is increased by dithiothreitol (DTT) or thioredoxins (non-specific). Inhibited by hydroxyurea, leads to dNTP depletion, replication fork arrest and genomic instability (PubMed:20005847)',\n",
       "   'Pathway': 'Genetic information processing; DNA replication'},\n",
       "  'seq': 'MNQNLLVTKRDGSTERINLDKIHRVLDWAAEGLHNVSISQVELRSHIQFYDGIKTSDIHETIIKAAADLISRDAPDYQYLAARLAIFHLRKKAYGQFEPPALYDHVVKMVEMGKYDNHLLEDYTEEEFKQMDTFIDHDRDMTFSYAAVKQLEGKYLVQNRVTGEIYESAQFLYILVAACLFSNYPRETRLQYVKRFYDAVSTFKISLPTPIMSGVRTPTRQFSSCVLIECGDSLDSINATSSAIVKYVSQRAGIGINAGRIRALGSPIRGGEAFHTGCIPFYKHFQTAVKSCSQGGVRGGAATLFYPMWHLEVESLLVLKNNRGVEGNRVRHMDYGVQINKLMYTRLLKGEDITLFSPSDVPGLYDAFFADQEEFERLYTKYEKDDSIRKQRVKAVELFSLMMQERASTGRIYIQNVDHCNTHSPFDPAIAPVRQSNLCLEIALPTKPLNDVNDENGEIALCTLSAFNLGAINNLDELEELAILAVRALDALLDYQDYPIPAAKRGAMGRRTLGIGVINFAYYLAKHGKRYSDGSANNLTHKTFEAIQYYLLKASNELAKEQGACPWFNETTYAKGILPIDTYKKDLDTIANEPLHYDWEALRESIKTHGLRNSTLSALMPSETSSQISNATNGIEPPRGYVSIKASKDGILRQVVPDYEHLHDAYELLWEMPGNDGYLQLVGIMQKFIDQSISANTNYDPSRFPSGKVPMQQLLKDLLTAYKFGVKTLYYQNTRDGAEDAQDDLVPSIQDDGCESGACKI',\n",
       "  'mw': 85775.0},\n",
       " 'b2235': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P69924',\n",
       "  'gene': 'b2235',\n",
       "  'locus': ['b2235', 'JW2229'],\n",
       "  'synonyms': 'ftsB',\n",
       "  'properties': {'Catalytic Activity': [(\"[thioredoxin]-disulfide + a 2'-deoxyribonucleoside 5'-diphosphate + H2O = [thioredoxin]-dithiol + a ribonucleoside 5'-diphosphate\",\n",
       "     '1.17.4.1')],\n",
       "   'Function': 'Provides the precursors necessary for DNA synthesis. Catalyzes the biosynthesis of deoxyribonucleotides from the corresponding ribonucleotides. R2 contains the tyrosyl radical required for catalysis',\n",
       "   'Regulation Activity': 'Inhibited by hydroxyurea, leads to dNTP depletion, replication fork arrest and genomic instability',\n",
       "   'Pathway': 'Genetic information processing; DNA replication'},\n",
       "  'seq': 'MAYTTFSQTKNDQLKEPMFFGQPVNVARYDQQKYDIFEKLIEKQLSFFWRPEEVDVSRDRIDYQALPEHEKHIFISNLKYQTLLDSIQGRSPNVALLPLISIPELETWVETWAFSETIHSRSYTHIIRNIVNDPSVVFDDIVTNEQIQKRAEGISSYYDELIEMTSYWHLLGEGTHTVNGKTVTVSLRELKKKLYLCLMSVNALEAIRFYVSFACSFAFAERELMEGNAKIIRLIARDEALHLTGTQHMLNLLRSGADDPEMAEIAEECKQECYDLFVQAAQQEKDWADYLFRDGSMIGLNKDILCQYVEYITNIRMQAVGLDLPFQTRSNPIPWINTWLVSDNVQVAPQEVEVSSYLVGQIDSEVDTDDLSNFQL',\n",
       "  'mw': 43517.0},\n",
       " 'b2239': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P09394',\n",
       "  'gene': 'b2239',\n",
       "  'locus': ['b2239', 'JW2233'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('a sn-glycero-3-phosphodiester + H2O = an alcohol + H(+) + sn-glycerol 3-phosphate',\n",
       "     '3.1.4.46')],\n",
       "   'Function': 'Glycerophosphoryl diester phosphodiesterase hydrolyzes deacylated phospholipids to G3P and the corresponding alcohols'},\n",
       "  'seq': 'MKLTLKNLSMAIMMSTIVMGSSAMAADSNEKIVIAHRGASGYLPEHTLPAKAMAYAQGADYLEQDLVMTKDDNLVVLHDHYLDRVTDVADRFPDRARKDGRYYAIDFTLDEIKSLKFTEGFDIENGKKVQTYPGRFPMGKSDFRVHTFEEEIEFVQGLNHSTGKNIGIYPEIKAPWFHHQEGKDIAAKTLEVLKKYGYTGKDDKVYLQCFDADELKRIKNELEPKMGMELNLVQLIAYTDWNETQQKQPDGSWVNYNYDWMFKPGAMKQVAEYADGIGPDYHMLIEETSQPGNIKLTGMVQDAQQNKLVVHPYTVRSDKLPEYTPDVNQLYDALYNKAGVNGLFTDFPDKAVKFLNKE',\n",
       "  'mw': 40843.0},\n",
       " 'b2240': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P08194',\n",
       "  'gene': 'b2240',\n",
       "  'locus': ['b2240', 'JW2234'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Responsible for glycerol-3-phosphate uptake'},\n",
       "  'seq': 'MLSIFKPAPHKARLPAAEIDPTYRRLRWQIFLGIFFGYAAYYLVRKNFALAMPYLVEQGFSRGDLGFALSGISIAYGFSKFIMGSVSDRSNPRVFLPAGLILAAAVMLFMGFVPWATSSIAVMFVLLFLCGWFQGMGWPPCGRTMVHWWSQKERGGIVSVWNCAHNVGGGIPPLLFLLGMAWFNDWHAALYMPAFCAILVALFAFAMMRDTPQSCGLPPIEEYKNDYPDDYNEKAEQELTAKQIFMQYVLPNKLLWYIAIANVFVYLLRYGILDWSPTYLKEVKHFALDKSSWAYFLYEYAGIPGTLLCGWMSDKVFRGNRGATGVFFMTLVTIATIVYWMNPAGNPTVDMICMIVIGFLIYGPVMLIGLHALELAPKKAAGTAAGFTGLFGYLGGSVAASAIVGYTVDFFGWDGGFMVMIGGSILAVILLIVVMIGEKRRHEQLLQERNGG',\n",
       "  'mw': 50310.0},\n",
       " 'b2241': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A9C0',\n",
       "  'gene': 'b2241',\n",
       "  'locus': ['b2241', 'JW2235'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('a quinone + sn-glycerol 3-phosphate = a quinol + dihydroxyacetone phosphate',\n",
       "     '1.1.5.3')],\n",
       "   'Function': 'Conversion of glycerol 3-phosphate to dihydroxyacetone. Uses fumarate or nitrate as electron acceptor',\n",
       "   'Pathway': 'Polyol metabolism; glycerol degradation via glycerol kinase pathway; glycerone phosphate from sn-glycerol 3-phosphate (anaerobic route): step 1/1'},\n",
       "  'seq': 'MKTRDSQSSDVIIIGGGATGAGIARDCALRGLRVILVERHDIATGATGRNHGLLHSGARYAVTDAESARECISENQILKRIARHCVEPTNGLFITLPEDDLSFQATFIRACEEAGISAEAIDPQQARIIEPAVNPALIGAVKVPDGTVDPFRLTAANMLDAKEHGAVILTAHEVTGLIREGATVCGVRVRNHLTGETQALHAPVVVNAAGIWGQHIAEYADLRIRMFPAKGSLLIMDHRINQHVINRCRKPSDADILVPGDTISLIGTTSLRIDYNEIDDNRVTAEEVDILLREGEKLAPVMAKTRILRAYSGVRPLVASDDDPSGRNVSRGIVLLDHAERDGLDGFITITGGKLMTYRLMAEWATDAVCRKLGNTRPCTTADLALPGSQEPAEVTLRKVISLPAPLRGSAVYRHGDRTPAWLSEGRLHRSLVCECEAVTAGEVQYAVENLNVNSLLDLRRRTRVGMGTCQGELCACRAAGLLQRFNVTTSAQSIEQLSTFLNERWKGVQPIAWGDALRESEFTRWVYQGLCGLEKEQKDAL',\n",
       "  'mw': 58958.0},\n",
       " 'b2242': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P13033',\n",
       "  'gene': 'b2242',\n",
       "  'locus': ['b2242', 'JW2236'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('a quinone + sn-glycerol 3-phosphate = a quinol + dihydroxyacetone phosphate',\n",
       "     '1.1.5.3')],\n",
       "   'Function': 'Conversion of glycerol 3-phosphate to dihydroxyacetone. Uses fumarate or nitrate as electron acceptor',\n",
       "   'Pathway': 'Polyol metabolism; glycerol degradation via glycerol kinase pathway; glycerone phosphate from sn-glycerol 3-phosphate (anaerobic route): step 1/1'},\n",
       "  'seq': 'MRFDTVIMGGGLAGLLCGLQLQKHGLRCAIVTRGQSALHFSSGSLDLLSHLPDGQPVTDIHSGLESLRQQAPAHPYSLLEPQRVLDLACQAQALIAESGAQLQGSVELAHQRVTPLGTLRSTWLSSPEVPVWPLPAKKICVVGISGLMDFQAHLAAASLRELGLAVETAEIELPELDVLRNNATEFRAVNIARFLDNEENWPLLLDALIPVANTCEMILMPACFGLADDKLWRWLNEKLPCSLMLLPTLPPSVLGIRLQNQLQRQFVRQGGVWMPGDEVKKVTCKNGVVNEIWTRNHADIPLRPRFAVLASGSFFSGGLVAERNGIREPILGLDVLQTATRGEWYKGDFFAPQPWQQFGVTTDETLRPSQAGQTIENLFAIGSVLGGFDPIAQGCGGGVCAVSALHAAQQIAQRAGGQQ',\n",
       "  'mw': 45357.0},\n",
       " 'b2243': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A996',\n",
       "  'gene': 'b2243',\n",
       "  'locus': ['b2243', 'JW2237'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Electron transfer protein; may also function as the membrane anchor for the GlpAB dimer',\n",
       "   'Pathway': 'Polyol metabolism; glycerol degradation via glycerol kinase pathway; glycerone phosphate from sn-glycerol 3-phosphate (anaerobic route): step 1/1'},\n",
       "  'seq': 'MNDTSFENCIKCTVCTTACPVSRVNPGYPGPKQAGPDGERLRLKDGALYDEALKYCINCKRCEVACPSDVKIGDIIQRARAKYDTTRPSLRNFVLSHTDLMGSVSTPFAPIVNTATSLKPVRQLLDAALKIDHRRTLPKYSFGTFRRWYRSVAAQQAQYKDQVAFFHGCFVNYNHPQLGKDLIKVLNAMGTGVQLLSKEKCCGVPLIANGFTDKARKQAITNVESIREAVGVKGIPVIATSSTCTFALRDEYPEVLNVDNKGLRDHIELATRWLWRKLDEGKTLPLKPLPLKVVYHTPCHMEKMGWTLYTLELLRNIPGLELTVLDSQCCGIAGTYGFKKENYPTSQAIGAPLFRQIEESGADLVVTDCETCKWQIEMSTSLRCEHPITLLAQALA',\n",
       "  'mw': 44108.0},\n",
       " 'b2251': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P52006',\n",
       "  'gene': 'b2251',\n",
       "  'locus': ['b2251', 'JW2245'],\n",
       "  'synonyms': 'yfaO',\n",
       "  'properties': {'Catalytic Activity': [(\"a ribonucleoside 5'-triphosphate + H2O = a ribonucleoside 5'-phosphate + diphosphate + H(+)\",\n",
       "     '3.6.1.9'),\n",
       "    (\"a 2'-deoxyribonucleoside 5'-triphosphate + H2O = a 2'-deoxyribonucleoside 5'-phosphate + diphosphate + H(+)\",\n",
       "     '3.6.1.9'),\n",
       "    ('dUTP + H2O = diphosphate + dUMP + H(+)', '3.6.1.9'),\n",
       "    ('dUTP + H2O = diphosphate + dUMP + H(+)', '3.6.1.23'),\n",
       "    ('dTTP + H2O = diphosphate + dTMP + H(+)', '3.6.1.9'),\n",
       "    ('dCTP + H2O = dCMP + diphosphate + H(+)', '3.6.1.9'),\n",
       "    ('dCTP + H2O = dCMP + diphosphate + H(+)', '3.6.1.12')],\n",
       "   'Function': 'Catalyzes the hydrolysis of nucleoside triphosphates, with a preference for pyrimidine deoxynucleoside triphosphates (dUTP, dTTP and dCTP)'},\n",
       "  'seq': 'MRQRTIVCPLIQNDGAYLLCKMADDRGVFPGQWAISGGGVEPGERIEEALRREIREELGEQLLLTEITPWTFSDDIRTKTYADGRKEEIYMIYLIFDCVSANREVKINEEFQDYAWVKPEDLVHYDLNVATRKTLRLKGLL',\n",
       "  'mw': 16371.0},\n",
       " 'b2253': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77690',\n",
       "  'gene': 'b2253',\n",
       "  'locus': ['b2253', 'JW5372'],\n",
       "  'synonyms': 'pmrH yfbE',\n",
       "  'properties': {'Catalytic Activity': [('2-oxoglutarate + UDP-4-amino-4-deoxy-beta-L-arabinose = L-glutamate + UDP-beta-L-threo-pentopyranos-4-ulose',\n",
       "     '2.6.1.87')],\n",
       "   'Function': 'Catalyzes the conversion of UDP-4-keto-arabinose (UDP-Ara4O) to UDP-4-amino-4-deoxy-L-arabinose (UDP-L-Ara4N). The modified arabinose is attached to lipid A and is required for resistance to polymyxin and cationic antimicrobial peptides',\n",
       "   'Pathway': 'Bacterial outer membrane biogenesis; lipopolysaccharide biosynthesis'},\n",
       "  'seq': 'MAEGKAMSEFLPFSRPAMGVEELAAVKEVLESGWITTGPKNQALEQAFCQLTGNQHAIAVSSATAGMHITLMALKIGKGDEVITPSLTWVSTLNMISLLGATPVMVDVDRDTLMVTPEAIESAITPRTKAIIPVHYAGAPADIDAIRAIGERYGIAVIEDAAHAVGTYYKGRHIGAKGTAIFSFHAIKNITCAEGGLIVTDNENLARQLRMLKFHGLGVDAYDRQTWGRAPQAEVLTPGYKYNLTDINAAIALTQLVKLEHLNTRRREIAQQYQQALAALPFQPLSLPAWPHVHAWHLFIIRVDEQRCGISRDALMEALKERGIGTGLHFRAAHTQKYYRERFPTLSLPNTEWNSERICSLPLFPDMTTADADHVITALQQLAGQ',\n",
       "  'mw': 42238.0},\n",
       " 'b2254': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77757',\n",
       "  'gene': 'b2254',\n",
       "  'locus': ['b2254', 'JW2248'],\n",
       "  'synonyms': 'pmrF yfbF',\n",
       "  'properties': {'Catalytic Activity': [('di-trans,octa-cis-undecaprenyl phosphate + UDP-4-deoxy-4-formamido-beta-L-arabinose = 4-deoxy-4-formamido-alpha-L-arabinopyranosyl di-trans,octa-cis-undecaprenyl phosphate + UDP',\n",
       "     '2.4.2.53')],\n",
       "   'Function': 'Catalyzes the transfer of 4-deoxy-4-formamido-L-arabinose from UDP to undecaprenyl phosphate. The modified arabinose is attached to lipid A and is required for resistance to polymyxin and cationic antimicrobial peptides',\n",
       "   'Pathway': 'Bacterial outer membrane biogenesis; lipopolysaccharide biosynthesis'},\n",
       "  'seq': 'MFEIHPVKKVSVVIPVYNEQESLPELIRRTTTACESLGKEYEILLIDDGSSDNSAHMLVEASQAENSHIVSILLNRNYGQHSAIMAGFSHVTGDLIITLDADLQNPPEEIPRLVAKADEGYDVVGTVRQNRQDSWFRKTASKMINRLIQRTTGKAMGDYGCMLRAYRRHIVDAMLHCHERSTFIPILANIFARRAIEIPVHHAEREFGESKYSFMRLINLMYDLVTCLTTTPLRMLSLLGSIIAIGGFSIAVLLVILRLTFGPQWAAEGVFMLFAVLFTFIGAQFIGMGLLGEYIGRIYTDVRARPRYFVQQVIRPSSKENE',\n",
       "  'mw': 36339.0},\n",
       " 'b2255': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77398',\n",
       "  'gene': 'b2255',\n",
       "  'locus': ['b2255', 'JW2249'],\n",
       "  'synonyms': 'pmrI yfbG',\n",
       "  'properties': {'Catalytic Activity': [('NAD(+) + UDP-alpha-D-glucuronate = CO2 + NADH + UDP-beta-L-threo-pentopyranos-4-ulose',\n",
       "     '1.1.1.305'),\n",
       "    ('(6S)-10-formyltetrahydrofolate + UDP-4-amino-4-deoxy-beta-L-arabinose = (6S)-5,6,7,8-tetrahydrofolate + H(+) + UDP-4-deoxy-4-formamido-beta-L-arabinose',\n",
       "     '2.1.2.13')],\n",
       "   'Function': 'Bifunctional enzyme that catalyzes the oxidative decarboxylation of UDP-glucuronic acid (UDP-GlcUA) to UDP-4-keto-arabinose (UDP-Ara4O) and the addition of a formyl group to UDP-4-amino-4-deoxy-L-arabinose (UDP-L-Ara4N) to form UDP-L-4-formamido-arabinose (UDP-L-Ara4FN). The modified arabinose is attached to lipid A and is required for resistance to polymyxin and cationic antimicrobial peptides',\n",
       "   'Pathway': 'Bacterial outer membrane biogenesis; lipopolysaccharide biosynthesis'},\n",
       "  'seq': 'MKTVVFAYHDMGCLGIEALLAAGYEISAIFTHTDNPGEKAFYGSVARLAAERGIPVYAPDNVNHPLWVERIAQLSPDVIFSFYYRHLIYDEILQLAPAGAFNLHGSLLPKYRGRAPLNWVLVNGETETGVTLHRMVKRADAGAIVAQLRIAIAPDDIAITLHHKLCHAARQLLEQTLPAIKHGNILEIAQRENEATCFGRRTPDDSFLEWHKPASVLHNMVRAVADPWPGAFSYVGNQKFTVWSSRVHPHASKAQPGSVISVAPLLIACGDGALEIVTGQAGDGITMQGSQLAQTLGLVQGSRLNSQPACTARRRTRVLILGVNGFIGNHLTERLLREDHYEVYGLDIGSDAISRFLNHPHFHFVEGDISIHSEWIEYHVKKCDVVLPLVAIATPIEYTRNPLRVFELDFEENLRIIRYCVKYRKRIIFPSTSEVYGMCSDKYFDEDHSNLIVGPVNKPRWIYSVSKQLLDRVIWAYGEKEGLQFTLFRPFNWMGPRLDNLNAARIGSSRAITQLILNLVEGSPIKLIDGGKQKRCFTDIRDGIEALYRIIENAGNRCDGEIINIGNPENEASIEELGEMLLASFEKHPLRHHFPPFAGFRVVESSSYYGKGYQDVEHRKPSIRNAHRCLDWEPKIDMQETIDETLDFFLRTVDLTDKPS',\n",
       "  'mw': 74289.0},\n",
       " 'b2257': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P76473',\n",
       "  'gene': 'b2257',\n",
       "  'locus': ['b2257', 'JW2251'],\n",
       "  'synonyms': 'pmrK yfbI',\n",
       "  'properties': {'Catalytic Activity': [('4-amino-4-deoxy-alpha-L-arabinopyranosyl di-trans,octa-cis-undecaprenyl phosphate + lipid IVA = lipid IIA + di-trans,octa-cis-undecaprenyl phosphate.',\n",
       "     '2.4.2.43')],\n",
       "   'Function': 'Catalyzes the transfer of the L-Ara4N moiety of the glycolipid undecaprenyl phosphate-alpha-L-Ara4N to lipid A. The modified arabinose is attached to lipid A and is required for resistance to polymyxin and cationic antimicrobial peptides',\n",
       "   'Pathway': 'Lipopolysaccharide metabolism; 4-amino-4-deoxy-beta-L-arabinose-lipid A biosynthesis'},\n",
       "  'seq': 'MKSVRYLIGLFAFIACYYLLPISTRLLWQPDETRYAEISREMLASGDWIVPHLLGLRYFEKPIAGYWINSIGQWLFGANNFGVRAGVIFATLLTAALVTWFTLRLWRDKRLALLATVIYLSLFIVYAIGTYAVLDPFIAFWLVAGMCSFWLAMQAQTWKGKSAGFLLLGITCGMGVMTKGFLALAVPVLSVLPWVATQKRWKDLFIYGWLAVISCVLTVLPWGLAIAQREPNFWHYFFWVEHIQRFALDDAQHRAPFWYYVPVIIAGSLPWLGLLPGALYTGWKNRKHSATVYLLSWTIMPLLFFSVAKGKLPTYILSCFASLAMLMAHYALLAAKNNPLALRINGWINIAFGVTGIIATFVVSPWGPMNTPVWQTFESYKVFCAWSIFSLWAFFGWYTLTNVEKTWPFAALCPLGLALLVGFSIPDRVMEGKHPQFFVEMTQESLQPSRYILTDSVGVAAGLAWSLQRDDIIMYRQTGELKYGLNYPDAKGRFVSGDEFANWLNQHRQEGIITLVLSVDRDEDINSLAIPPADAIDRQERLVLIQYRPK',\n",
       "  'mw': 62543.0},\n",
       " 'b2260': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37353',\n",
       "  'gene': 'b2260',\n",
       "  'locus': ['b2260', 'JW2255'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('2-succinylbenzoate + ATP + CoA = 2-succinylbenzoyl-CoA + AMP + diphosphate',\n",
       "     '6.2.1.26')],\n",
       "   'Function': 'Converts 2-succinylbenzoate (OSB) to 2-succinylbenzoyl-CoA (OSB-CoA)',\n",
       "   'Pathway': 'Quinol/quinone metabolism; menaquinone biosynthesis'},\n",
       "  'seq': 'MIFSDWPWRHWRQVRGETIALRLNDEQLNWRELCARVDELASGFAVQGVVEGSGVMLRAWNTPQTLLAWLALLQCGARVLPVNPQLPQPLLEELLPNLTLQFALVPDGENTFPALTSLHIQLVEGAHAATWQPTRLCSMTLTSGSTGLPKAAVHTYQAHLASAQGVLSLIPFGDHDDWLLSLPLFHVSGQGIMWRWLYAGARMTVRDKQPLEQMLAGCTHASLVPTQLWRLLVNRSSVSLKAVLLGGAAIPVELTEQAREQGIRCFCGYGLTEFASTVCAKEADGLADVGSPLPGREVKIVNNEVWLRAASMAEGYWRNGQLVSLVNDEGWYATRDRGEMHNGKLTIVGRLDNLFFSGGEGIQPEEVERVIAAHPAVLQVFIVPVADKEFGHRPVAVMEYDHESVDLSEWVKDKLARFQQPVRWLTLPPELKNGGIKISRQALKEWVQRQQ',\n",
       "  'mw': 50185.0},\n",
       " 'b2261': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P29208',\n",
       "  'gene': 'b2261',\n",
       "  'locus': ['b2261', 'JW2256'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(1R,6R)-6-hydroxy-2-succinyl-cyclohexa-2,4-diene-1-carboxylate = 2-succinylbenzoate + H2O',\n",
       "     '4.2.1.113')],\n",
       "   'Function': 'Converts 2-succinyl-6-hydroxy-2,4-cyclohexadiene-1-carboxylate (SHCHC) to 2-succinylbenzoate (OSB) (PubMed:10194342, PubMed:8335646). Does not show detectable N-acylamino acid racemase (NAAAR) activity with N-acetyl-S-methionine as substrate (PubMed:10194342)',\n",
       "   'Pathway': 'Quinol/quinone metabolism; menaquinone biosynthesis'},\n",
       "  'seq': 'MRSAQVYRWQIPMDAGVVLRDRRLKTRDGLYVCLREGEREGWGEISPLPGFSQETWEEAQSVLLAWVNNWLAGDCELPQMPSVAFGVSCALAELTDTLPQAANYRAAPLCNGDPDDLILKLADMPGEKVAKVKVGLYEAVRDGMVVNLLLEAIPDLHLRLDANRAWTPLKGQQFAKYVNPDYRDRIAFLEEPCKTRDDSRAFARETGIAIAWDESLREPDFAFVAEEGVRAVVIKPTLTGSLEKVREQVQAAHALGLTAVISSSIESSLGLTQLARIAAWLTPDTIPGLDTLDLMQAQQVRRWPGSTLPVVEVDALERLL',\n",
       "  'mw': 35477.0},\n",
       " 'b2262': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ABU0',\n",
       "  'gene': 'b2262',\n",
       "  'locus': ['b2262', 'JW2257'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('2-succinylbenzoyl-CoA + H(+) = 1,4-dihydroxy-2-naphthoyl-CoA + H2O',\n",
       "     '4.1.3.36')],\n",
       "   'Function': 'Converts o-succinylbenzoyl-CoA (OSB-CoA) to 1,4-dihydroxy-2-naphthoyl-CoA (DHNA-CoA)',\n",
       "   'Regulation Activity': 'Inhibited by sulfite and nitrate',\n",
       "   'Pathway': 'Quinol/quinone metabolism; menaquinone biosynthesis'},\n",
       "  'seq': 'MIYPDEAMLYAPVEWHDCSEGFEDIRYEKSTDGIAKITINRPQVRNAFRPLTVKEMIQALADARYDDNIGVIILTGAGDKAFCSGGDQKVRGDYGGYKDDSGVHHLNVLDFQRQIRTCPKPVVAMVAGYSIGGGHVLHMMCDLTIAADNAIFGQTGPKVGSFDGGWGASYMARIVGQKKAREIWFLCRQYDAKQALDMGLVNTVVPLADLEKETVRWCREMLQNSPMALRCLKAALNADCDGQAGLQELAGNATMLFYMTEEGQEGRNAFNQKRQPDFSKFKRNP',\n",
       "  'mw': 31633.0},\n",
       " 'b2264': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P17109',\n",
       "  'gene': 'b2264',\n",
       "  'locus': ['b2264', 'JW5374'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('2-oxoglutarate + H(+) + isochorismate = 5-enolpyruvoyl-6-hydroxy-2-succinyl-cyclohex-3-ene-1-carboxylate + CO2',\n",
       "     '2.2.1.9')],\n",
       "   'Function': 'Catalyzes the thiamine diphosphate-dependent decarboxylation of 2-oxoglutarate and the subsequent addition of the resulting succinic semialdehyde-thiamine pyrophosphate anion to isochorismate to yield 2-succinyl-5-enolpyruvyl-6-hydroxy-3-cyclohexene-1-carboxylate (SEPHCHC)',\n",
       "   'Pathway': 'Quinol/quinone metabolism; menaquinone biosynthesis'},\n",
       "  'seq': 'MSVSAFNRRWAAVILEALTRHGVRHICIAPGSRSTPLTLAAAENSAFIHHTHFDERGLGHLALGLAKVSKQPVAVIVTSGTAVANLYPALIEAGLTGEKLILLTADRPPELIDCGANQAIRQPGMFASHPTHSISLPRPTQDIPARWLVSTIDHALGTLHAGGVHINCPFAEPLYGEMDDTGLSWQQRLGDWWQDDKPWLREAPRLESEKQRDWFFWRQKRGVVVAGRMSAEEGKKVALWAQTLGWPLIGDVLSQTGQPLPCADLWLGNAKATSELQQAQIVVQLGSSLTGKRLLQWQASCEPEEYWIVDDIEGRLDPAHHRGRRLIANIADWLELHPAEKRQPWCVEIPRLAEQAMQAVIARRDAFGEAQLAHRICDYLPEQGQLFVGNSLVVRLIDALSQLPAGYPVYSNRGASGIDGLLSTAAGVQRASGKPTLAIVGDLSALYDLNALALLRQVSAPLVLIVVNNNGGQIFSLLPTPQSERERFYLMPQNVHFEHAAAMFELKYHRPQNWQELETAFADAWRTPTTTVIEMVVNDTDGAQTLQQLLAQVSHL',\n",
       "  'mw': 61367.0},\n",
       " 'b2265': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P38051',\n",
       "  'gene': 'b2265',\n",
       "  'locus': ['b2265', 'JW2260'],\n",
       "  'synonyms': 'yfbA',\n",
       "  'properties': {'Catalytic Activity': [('chorismate = isochorismate',\n",
       "     '5.4.4.2')],\n",
       "   'Function': 'Catalyzes the conversion of chorismate to isochorismate. Can also catalyze the reverse reaction, but with a lower efficiency',\n",
       "   'Pathway': 'Quinol/quinone metabolism; menaquinone biosynthesis'},\n",
       "  'seq': 'MQSLTTALENLLRHLSQEIPATPGIRVIDIPFPLKDAFDALSWLASQQTYPQFYWQQRNGDEEAVVLGAITRFTSLDQAQRFLRQHPEHADLRIWGLNAFDPSQGNLLLPRLEWRRCGGKATLRLTLFSESSLQHDAIQAKEFIATLVSIKPLPGLHLTTTREQHWPDKTGWTQLIELATKTIAEGELDKVVLARATDLHFASPVNAAAMMAASRRLNLNCYHFYMAFDGENAFLGSSPERLWRRRDKALRTEALAGTVANNPDDKQAQQLGEWLMADDKNQRENMLVVEDICQRLQADTQTLDVLPPQVLRLRKVQHLRRCIWTSLNKADDVICLHQLQPTAAVAGLPRDLARQFIARHEPFTREWYAGSAGYLSLQQSEFCVSLRSAKISGNVVRLYAGAGIVRGSDPEQEWQEIDNKAAGLRTLLQME',\n",
       "  'mw': 48765.0},\n",
       " 'b2276': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AFF0',\n",
       "  'gene': 'b2276',\n",
       "  'locus': ['b2276', 'JW2271'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('a quinone + 5 H(+)(in) + NADH = a quinol + 4 H(+)(out) + NAD(+)',\n",
       "     '')],\n",
       "   'Function': 'NDH-1 shuttles electrons from NADH, via FMN and iron-sulfur (Fe-S) centers, to quinones in the respiratory chain. The immediate electron acceptor for the enzyme in this species is believed to be ubiquinone. Couples the redox reaction to proton translocation (for every two electrons transferred, four hydrogen ions are translocated across the cytoplasmic membrane), and thus conserves the redox energy in a proton gradient'},\n",
       "  'seq': 'MTITPQNLIALLPLLIVGLTVVVVMLSIAWRRNHFLNATLSVIGLNAALVSLWFVGQAGAMDVTPLMRVDGFAMLYTGLVLLASLATCTFAYPWLEGYNDNKDEFYLLVLIAALGGILLANANHLASLFLGIELISLPLFGLVGYAFRQKRSLEASIKYTILSAAASSFLLFGMALVYAQSGDLSFVALGKNLGDGMLNEPLLLAGFGLMIVGLGFKLSLVPFHLWTPDVYQGAPAPVSTFLATASKIAIFGVVMRLFLYAPVGDSEAIRVVLAIIAFASIIFGNLMALSQTNIKRLLGYSSISHLGYLLVALIALQTGEMSMEAVGVYLAGYLFSSLGAFGVVSLMSSPYRGPDADSLFSYRGLFWHRPILAAVMTVMMLSLAGIPMTLGFIGKFYVLAVGVQAHLWWLVGAVVVGSAIGLYYYLRVAVSLYLHAPEQPGRDAPSNWQYSAGGIVVLISALLVLVLGVWPQPLISIVRLAMPLM',\n",
       "  'mw': 52044.0},\n",
       " 'b2277': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AFE8',\n",
       "  'gene': 'b2277',\n",
       "  'locus': ['b2277', 'JW2272'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('a quinone + 5 H(+)(in) + NADH = a quinol + 4 H(+)(out) + NAD(+)',\n",
       "     '')],\n",
       "   'Function': 'NDH-1 shuttles electrons from NADH, via FMN and iron-sulfur (Fe-S) centers, to quinones in the respiratory chain. The immediate electron acceptor for the enzyme in this species is believed to be ubiquinone. Couples the redox reaction to proton translocation (for every two electrons transferred, four hydrogen ions are translocated across the cytoplasmic membrane), and thus conserves the redox energy in a proton gradient'},\n",
       "  'seq': 'MLLPWLILIPFIGGFLCWQTERFGVKVPRWIALITMGLTLALSLQLWLQGGYSLTQSAGIPQWQSEFDMPWIPRFGISIHLAIDGLSLLMVVLTGLLGVLAVLCSWKEIEKYQGFFHLNLMWILGGVIGVFLAIDMFLFFFFWEMMLVPMYFLIALWGHKASDGKTRITAATKFFIYTQASGLVMLIAILALVFVHYNATGVWTFNYEELLNTPMSSGVEYLLMLGFFIAFAVKMPVVPLHGWLPDAHSQAPTAGSVDLAGILLKTAAYGLLRFSLPLFPNASAEFAPIAMWLGVIGIFYGAWMAFAQTDIKRLIAYTSVSHMGFVLIAIYTGSQLAYQGAVIQMIAHGLSAAGLFILCGQLYERIHTRDMRMMGGLWSKMKWLPALSLFFAVATLGMPGTGNFVGEFMILFGSFQVVPVITVISTFGLVFASVYSLAMLHRAYFGKAKSQIASQELPGMSLRELFMILLLVVLLVLLGFYPQPILDTSHSAIGNIQQWFVNSVTTTRP',\n",
       "  'mw': 56525.0},\n",
       " 'b2278': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P33607',\n",
       "  'gene': 'b2278',\n",
       "  'locus': ['b2278', 'JW2273'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('a quinone + 5 H(+)(in) + NADH = a quinol + 4 H(+)(out) + NAD(+)',\n",
       "     '')],\n",
       "   'Function': 'NDH-1 shuttles electrons from NADH, via FMN and iron-sulfur (Fe-S) centers, to quinones in the respiratory chain. The immediate electron acceptor for the enzyme in this species is believed to be ubiquinone. Couples the redox reaction to proton translocation (for every two electrons transferred, four hydrogen ions are translocated across the cytoplasmic membrane), and thus conserves the redox energy in a proton gradient'},\n",
       "  'seq': 'MNMLALTIILPLIGFVLLAFSRGRWSENVSAIVGVGSVGLAALVTAFIGVDFFANGEQTYSQPLWTWMSVGDFNIGFNLVLDGLSLTMLSVVTGVGFLIHMYASWYMRGEEGYSRFFAYTNLFIASMVVLVLADNLLLMYLGWEGVGLCSYLLIGFYYTDPKNGAAAMKAFVVTRVGDVFLAFALFILYNELGTLNFREMVELAPAHFADGNNMLMWATLMLLGGAVGKSAQLPLQTWLADAMAGPTPVSALIHAATMVTAGVYLIARTHGLFLMTPEVLHLVGIVGAVTLLLAGFAALVQTDIKRVLAYSTMSQIGYMFLALGVQAWDAAIFHLMTHAFFKALLFLASGSVILACHHEQNIFKMGGLRKSIPLVYLCFLVGGAALSALPLVTAGFFSKDEILAGAMANGHINLMVAGLVGAFMTSLYTFRMIFIVFHGKEQIHAHAVKGVTHSLPLIVLLILSTFVGALIVPPLQGVLPQTTELAHGSMLTLEITSGVVAVVGILLAAWLWLGKRTLVTSIANSAPGRLLGTWWYNAWGFDWLYDKVFVKPFLGIAWLLKRDPLNSMMNIPAVLSRFAGKGLLLSENGYLRWYVASMSIGAVVVLALLMVLR',\n",
       "  'mw': 66438.0},\n",
       " 'b2279': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AFE4',\n",
       "  'gene': 'b2279',\n",
       "  'locus': ['b2279', 'JW2274'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('a quinone + 5 H(+)(in) + NADH = a quinol + 4 H(+)(out) + NAD(+)',\n",
       "     '')],\n",
       "   'Function': 'There are 2 NADH dehydrogenases in E.coli, however only this complex is able to use dNADH (reduced nicotinamide hypoxanthine dinucleotide, deamino-NADH) and dNADH-DB (dimethoxy-5-methyl-6-decyl-1,4-benzoquinone) as substrates'},\n",
       "  'seq': 'MIPLQHGLILAAILFVLGLTGLVIRRNLLFMLIGLEIMINASALAFVVAGSYWGQTDGQVMYILAISLAAAEASIGLALLLQLHRRRQNLNIDSVSEMRG',\n",
       "  'mw': 10845.0},\n",
       " 'b2280': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AFE0',\n",
       "  'gene': 'b2280',\n",
       "  'locus': ['b2280', 'JW2275'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('a quinone + 5 H(+)(in) + NADH = a quinol + 4 H(+)(out) + NAD(+)',\n",
       "     '')],\n",
       "   'Function': 'NDH-1 shuttles electrons from NADH, via FMN and iron-sulfur (Fe-S) centers, to quinones in the respiratory chain. The immediate electron acceptor for the enzyme in this species is believed to be ubiquinone. Couples the redox reaction to proton translocation (for every two electrons transferred, four hydrogen ions are translocated across the cytoplasmic membrane), and thus conserves the redox energy in a proton gradient'},\n",
       "  'seq': 'MEFAFYICGLIAILATLRVITHTNPVHALLYLIISLLAISGVFFSLGAYFAGALEIIVYAGAIMVLFVFVVMMLNLGGSEIEQERQWLKPQVWIGPAILSAIMLVVIVYAILGVNDQGIDGTPISAKAVGITLFGPYVLAVELASMLLLAGLVVAFHVGREERAGEVLSNRKDDSAKRKTEEHA',\n",
       "  'mw': 19875.0},\n",
       " 'b2281': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AFD6',\n",
       "  'gene': 'b2281',\n",
       "  'locus': ['b2281', 'JW2276'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('a quinone + 5 H(+)(in) + NADH = a quinol + 4 H(+)(out) + NAD(+)',\n",
       "     '')],\n",
       "   'Function': 'NDH-1 shuttles electrons from NADH, via FMN and iron-sulfur (Fe-S) centers, to quinones in the respiratory chain. The immediate electron acceptor for the enzyme in this species is believed to be ubiquinone. Couples the redox reaction to proton translocation (for every two electrons transferred, four hydrogen ions are translocated across the cytoplasmic membrane), and thus conserves the redox energy in a proton gradient'},\n",
       "  'seq': 'MTLKELLVGFGTQVRSIWMIGLHAFAKRETRMYPEEPVYLPPRYRGRIVLTRDPDGEERCVACNLCAVACPVGCISLQKAETKDGRWYPEFFRINFSRCIFCGLCEEACPTTAIQLTPDFEMGEYKRQDLVYEKEDLLISGPGKYPEYNFYRMAGMAIDGKDKGEAENEAKPIDVKSLLP',\n",
       "  'mw': 20538.0},\n",
       " 'b2282': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AFD4',\n",
       "  'gene': 'b2282',\n",
       "  'locus': ['b2282', 'JW2277'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('a quinone + 5 H(+)(in) + NADH = a quinol + 4 H(+)(out) + NAD(+)',\n",
       "     '')],\n",
       "   'Function': 'NDH-1 shuttles electrons from NADH, via FMN and iron-sulfur (Fe-S) centers, to quinones in the respiratory chain. The immediate electron acceptor for the enzyme in this species is believed to be ubiquinone. Couples the redox reaction to proton translocation (for every two electrons transferred, four hydrogen ions are translocated across the cytoplasmic membrane), and thus conserves the redox energy in a proton gradient. This subunit may bind ubiquinone'},\n",
       "  'seq': 'MSWISPELIEILLTILKAVVILLVVVTCGAFMSFGERRLLGLFQNRYGPNRVGWGGSLQLVADMIKMFFKEDWIPKFSDRVIFTLAPMIAFTSLLLAFAIVPVSPGWVVADLNIGILFFLMMAGLAVYAVLFAGWSSNNKYSLLGAMRASAQTLSYEVFLGLSLMGVVAQAGSFNMTDIVNSQAHVWNVIPQFFGFITFAIAGVAVCHRHPFDQPEAEQELADGYHIEYSGMKFGLFFVGEYIGIVTISALMVTLFFGGWQGPLLPPFIWFALKTAFFMMMFILIRASLPRPRYDQVMSFGWKICLPLTLINLLVTAAVILWQAQ',\n",
       "  'mw': 36219.0},\n",
       " 'b2283': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P33602',\n",
       "  'gene': 'b2283',\n",
       "  'locus': ['b2283', 'JW2278'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('a quinone + 5 H(+)(in) + NADH = a quinol + 4 H(+)(out) + NAD(+)',\n",
       "     '')],\n",
       "   'Function': 'NDH-1 shuttles electrons from NADH, via FMN and iron-sulfur (Fe-S) centers, to quinones in the respiratory chain. The immediate electron acceptor for the enzyme in this species is believed to be ubiquinone. Couples the redox reaction to proton translocation (for every two electrons transferred, four hydrogen ions are translocated across the cytoplasmic membrane), and thus conserves the redox energy in a proton gradient'},\n",
       "  'seq': 'MATIHVDGKEYEVNGADNLLEACLSLGLDIPYFCWHPALGSVGACRQCAVKQYQNAEDTRGRLVMSCMTPASDGTFISIDDEEAKQFRESVVEWLMTNHPHDCPVCEEGGNCHLQDMTVMTGHSFRRYRFTKRTHRNQDLGPFISHEMNRCIACYRCVRYYKDYADGTDLGVYGAHDNVYFGRPEDGTLESEFSGNLVEICPTGVFTDKTHSERYNRKWDMQFAPSICQQCSIGCNISPGERYGELRRIENRYNGTVNHYFLCDRGRFGYGYVNLKDRPRQPVQRRGDDFITLNAEQAMQGAADILRQSKKVIGIGSPRASVESNFALRELVGEENFYTGIAHGEQERLQLALKVLREGGIYTPALREIESYDAVLVLGEDVTQTGARVALAVRQAVKGKAREMAAAQKVADWQIAAILNIGQRAKHPLFVTNVDDTRLDDIAAWTYRAPVEDQARLGFAIAHALDNSAPAVDGIEPELQSKIDVIVQALAGAKKPLIISGTNAGSLEVIQAAANVAKALKGRGADVGITMIARSVNSMGLGIMGGGSLEEALTELETGRADAVVVLENDLHRHASAIRVNAALAKAPLVMVVDHQRTAIMENAHLVLSAASFAESDGTVINNEGRAQRFFQVYDPAYYDSKTVMLESWRWLHSLHSTLLSREVDWTQLDHVIDAVVAKIPELAGIKDAAPDATFRIRGQKLAREPHRYSGRTAMRANISVHEPRQPQDIDTMFTFSMEGNNQPTAHRSQVPFAWAPGWNSPQAWNKFQDEVGGKLRFGDPGVRLFETSENGLDYFTSVPARFQPQDGKWRIAPYYHLFGSDELSQRAPVFQSRMPQPYIKLNPADAAKLGVNAGTRVSFSYDGNTVTLPVEIAEGLTAGQVGLPMGMSGIAPVLAGAHLEDLKEAQQ',\n",
       "  'mw': 100299.0},\n",
       " 'b2284': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P31979',\n",
       "  'gene': 'b2284',\n",
       "  'locus': ['b2284', 'JW2279'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('a quinone + 5 H(+)(in) + NADH = a quinol + 4 H(+)(out) + NAD(+)',\n",
       "     '')],\n",
       "   'Function': 'NDH-1 shuttles electrons from NADH, via FMN and iron-sulfur (Fe-S) centers, to quinones in the respiratory chain. The immediate electron acceptor for the enzyme in this species is believed to be ubiquinone. Couples the redox reaction to proton translocation (for every two electrons transferred, four hydrogen ions are translocated across the cytoplasmic membrane), and thus conserves the redox energy in a proton gradient'},\n",
       "  'seq': 'MKNIIRTPETHPLTWRLRDDKQPVWLDEYRSKNGYEGARKALTGLSPDEIVNQVKDAGLKGRGGAGFSTGLKWSLMPKDESMNIRYLLCNADEMEPGTYKDRLLMEQLPHLLVEGMLISAFALKAYRGYIFLRGEYIEAAVNLRRAIAEATEAGLLGKNIMGTGFDFELFVHTGAGRYICGEETALINSLEGRRANPRSKPPFPATSGAWGKPTCVNNVETLCNVPAILANGVEWYQNISKSKDAGTKLMGFSGRVKNPGLWELPFGTTAREILEDYAGGMRDGLKFKAWQPGGAGTDFLTEAHLDLPMEFESIGKAGSRLGTALAMAVDHEINMVSLVRNLEEFFARESCGWCTPCRDGLPWSVKILRALERGEGQPGDIETLEQLCRFLGPGKTFCAHAPGAVEPLQSAIKYFREEFEAGIKQPFSNTHLINGIQPNLLKERW',\n",
       "  'mw': 49292.0},\n",
       " 'b2285': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AFD1',\n",
       "  'gene': 'b2285',\n",
       "  'locus': ['b2285', 'JW2280'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('a quinone + 5 H(+)(in) + NADH = a quinol + 4 H(+)(out) + NAD(+)',\n",
       "     '')],\n",
       "   'Function': 'NDH-1 shuttles electrons from NADH, via FMN and iron-sulfur (Fe-S) centers, to quinones in the respiratory chain. The immediate electron acceptor for the enzyme in this species is believed to be ubiquinone. Couples the redox reaction to proton translocation (for every two electrons transferred, four hydrogen ions are translocated across the cytoplasmic membrane), and thus conserves the redox energy in a proton gradient'},\n",
       "  'seq': 'MHENQQPQTEAFELSAAEREAIEHEMHHYEDPRAASIEALKIVQKQRGWVPDGAIHAIADVLGIPASDVEGVATFYSQIFRQPVGRHVIRYCDSVVCHINGYQGIQAALEKKLNIKPGQTTFDGRFTLLPTCCLGNCDKGPNMMIDEDTHAHLTPEAIPELLERYK',\n",
       "  'mw': 18590.0},\n",
       " 'b2286': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P33599',\n",
       "  'gene': 'b2286',\n",
       "  'locus': ['b2286', 'JW5375'],\n",
       "  'synonyms': 'nuoCD nuoD',\n",
       "  'properties': {'Catalytic Activity': [('a quinone + 5 H(+)(in) + NADH = a quinol + 4 H(+)(out) + NAD(+)',\n",
       "     '')],\n",
       "   'Function': 'NDH-1 shuttles electrons from NADH, via FMN and iron-sulfur (Fe-S) centers, to quinones in the respiratory chain. The immediate electron acceptor for the enzyme in this species is believed to be ubiquinone. Couples the redox reaction to proton translocation (for every two electrons transferred, four hydrogen ions are translocated across the cytoplasmic membrane), and thus conserves the redox energy in a proton gradient'},\n",
       "  'seq': 'MTDLTAQEPAWQTRDHLDDPVIGELRNRFGPDAFTVQATRTGVPVVWIKREQLLEVGDFLKKLPKPYVMLFDLHGMDERLRTHREGLPAADFSVFYHLISIDRNRDIMLKVALAENDLHVPTFTKLFPNANWYERETWDLFGITFDGHPNLRRIMMPQTWKGHPLRKDYPARATEFSPFELTKAKQDLEMEALTFKPEEWGMKRGTENEDFMFLNLGPNHPSAHGAFRIVLQLDGEEIVDCVPDIGYHHRGAEKMGERQSWHSYIPYTDRIEYLGGCVNEMPYVLAVEKLAGITVPDRVNVIRVMLSELFRINSHLLYISTFIQDVGAMTPVFFAFTDRQKIYDLVEAITGFRMHPAWFRIGGVAHDLPRGWDRLLREFLDWMPKRLASYEKAALQNTILKGRSQGVAAYGAKEALEWGTTGAGLRATGIDFDVRKARPYSGYENFDFEIPVGGGVSDCYTRVMLKVEELRQSLRILEQCLNNMPEGPFKADHPLTTPPPKERTLQHIETLITHFLQVSWGPVMPANESFQMIEATKGINSYYLTSDGSTMSYRTRVRTPSFAHLQQIPAAIRGSLVSDLIVYLGSIDFVMSDVDR',\n",
       "  'mw': 68236.0},\n",
       " 'b2287': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AFC7',\n",
       "  'gene': 'b2287',\n",
       "  'locus': ['b2287', 'JW5875'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('a quinone + 5 H(+)(in) + NADH = a quinol + 4 H(+)(out) + NAD(+)',\n",
       "     '')],\n",
       "   'Function': 'NDH-1 shuttles electrons from NADH, via FMN and iron-sulfur (Fe-S) centers, to quinones in the respiratory chain. The immediate electron acceptor for the enzyme in this species is ubiquinone. Couples the redox reaction to proton translocation (for every two electrons transferred, four hydrogen ions are translocated across the cytoplasmic membrane), and thus conserves the redox energy in a proton gradient'},\n",
       "  'seq': 'MDYTLTRIDPNGENDRYPLQKQEIVTDPLEQEVNKNVFMGKLNDMVNWGRKNSIWPYNFGLSCCYVEMVTSFTAVHDVARFGAEVLRASPRQADLMVVAGTCFTKMAPVIQRLYDQMLEPKWVISMGACANSGGMYDIYSVVQGVDKFIPVDVYIPGCPPRPEAYMQALMLLQESIGKERRPLSWVVGDQGVYRANMQSERERKRGERIAVTNLRTPDEI',\n",
       "  'mw': 25056.0},\n",
       " 'b2288': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AFC3',\n",
       "  'gene': 'b2288',\n",
       "  'locus': ['b2288', 'JW2283'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('a quinone + 5 H(+)(in) + NADH = a quinol + 4 H(+)(out) + NAD(+)',\n",
       "     '')],\n",
       "   'Function': 'NDH-1 shuttles electrons from NADH, via FMN and iron-sulfur (Fe-S) centers, to quinones in the respiratory chain. The immediate electron acceptor for the enzyme in this species is believed to be ubiquinone. Couples the redox reaction to proton translocation (for every two electrons transferred, four hydrogen ions are translocated across the cytoplasmic membrane), and thus conserves the redox energy in a proton gradient'},\n",
       "  'seq': 'MSMSTSTEVIAHHWAFAIFLIVAIGLCCLMLVGGWFLGGRARARSKNVPFESGIDSVGSARLRLSAKFYLVAMFFVIFDVEALYLFAWSTSIRESGWVGFVEAAIFIFVLLAGLVYLVRIGALDWTPARSRRERMNPETNSIANRQR',\n",
       "  'mw': 16457.0},\n",
       " 'b2291': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P76491',\n",
       "  'gene': 'b2291',\n",
       "  'locus': ['b2291', 'JW2288'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [(\"a 2'-deoxyribonucleoside 5'-phosphate + H2O = a 2'-deoxyribonucleoside + phosphate\",\n",
       "     '3.1.3.89'),\n",
       "    (\"dAMP + H2O = 2'-deoxyadenosine + phosphate\", ''),\n",
       "    (\"dGMP + H2O = 2'-deoxyguanosine + phosphate\", ''),\n",
       "    ('dTMP + H2O = phosphate + thymidine', ''),\n",
       "    (\"dUMP + H2O = 2'-deoxyuridine + phosphate\", ''),\n",
       "    (\"dIMP + H2O = 2'-deoxyinosine + phosphate\", ''),\n",
       "    (\"dCMP + H2O = 2'-deoxycytidine + phosphate\", '')],\n",
       "   'Function': \"Essential component of the deoxycytidine triphosphate (dCTP) pathway for de novo synthesis of thymidylate. Catalyzes the strictly specific dephosphorylation of 2'-deoxyribonucleoside 5'-monophosphates (dAMP, dGMP, dTMP, dUMP, dIMP and dCMP) and does not dephosphorylate 5'-ribonucleotides or ribonucleoside 3'-monophosphates\",\n",
       "   'Regulation Activity': 'Inhibited by both ribo- and deoxyribonucleoside di- and triphosphates'},\n",
       "  'seq': 'MKQSHFFAHLSRLKLINRWPLMRNVRTENVSEHSLQVAMVAHALAAIKNRKFGGNVNAERIALLAMYHDASEVLTGDLPTPVKYFNSQIAQEYKAIEKIAQQKLVDMVPEELRDIFAPLIDEHAYSDEEKSLVKQADALCAYLKCLEELAAGNNEFLLAKTRLEATLEARRSQEMDYFMEIFVPSFHLSLDEISQDSPL',\n",
       "  'mw': 22708.0},\n",
       " 'b2296': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A6A3',\n",
       "  'gene': 'b2296',\n",
       "  'locus': ['b2296', 'JW2293'],\n",
       "  'synonyms': 'ack',\n",
       "  'properties': {'Catalytic Activity': [('acetate + ATP = acetyl phosphate + ADP',\n",
       "     '2.7.2.1')],\n",
       "   'Function': 'Catalyzes the formation of acetyl phosphate from acetate and ATP. Can also catalyze the reverse reaction. During anaerobic growth of the organism, this enzyme is also involved in the synthesis of most of the ATP formed catabolically. The main pathway for acetate production during exponential phase (PubMed:16080684)',\n",
       "   'Pathway': 'Metabolic intermediate biosynthesis; acetyl-CoA biosynthesis; acetyl-CoA from acetate: step 1/2'},\n",
       "  'seq': 'MSSKLVLVLNCGSSSLKFAIIDAVNGEEYLSGLAECFHLPEARIKWKMDGNKQEAALGAGAAHSEALNFIVNTILAQKPELSAQLTAIGHRIVHGGEKYTSSVVIDESVIQGIKDAASFAPLHNPAHLIGIEEALKSFPQLKDKNVAVFDTAFHQTMPEESYLYALPYNLYKEHGIRRYGAHGTSHFYVTQEAAKMLNKPVEELNIITCHLGNGGSVSAIRNGKCVDTSMGLTPLEGLVMGTRSGDIDPAIIFHLHDTLGMSVDAINKLLTKESGLLGLTEVTSDCRYVEDNYATKEDAKRAMDVYCHRLAKYIGAYTALMDGRLDAVVFTGGIGENAAMVRELSLGKLGVLGFEVDHERNLAARFGKSGFINKEGTRPAVVIPTNEELVIAQDASRLTA',\n",
       "  'mw': 43290.0},\n",
       " 'b2297': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A9M8',\n",
       "  'gene': 'b2297',\n",
       "  'locus': ['b2297', 'JW2294'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('acetyl-CoA + phosphate = acetyl phosphate + CoA',\n",
       "     '2.3.1.8')],\n",
       "   'Function': 'Involved in acetate metabolism (PubMed:16080684). Catalyzes the reversible interconversion of acetyl-CoA and acetyl phosphate (PubMed:16080684). The direction of the overall reaction changes depending on growth conditions (PubMed:16080684). On minimal medium acetyl-CoA is generated. In rich medium acetyl-CoA is converted to acetate and allowing the cell to dump the excess of acetylation potential in exchange for energy in the form of ATP (PubMed:15687190). The main pathway for acetate production during exponential phase (PubMed:16080684)',\n",
       "   'Regulation Activity': 'Inhibited by NADH and ATP. Pyruvate and PEP act as activators of the acetyl phosphate forming reaction while inhibiting the formation of acetyl-CoA',\n",
       "   'Pathway': 'Metabolic intermediate biosynthesis; acetyl-CoA biosynthesis; acetyl-CoA from acetate: step 2/2'},\n",
       "  'seq': 'MSRIIMLIPTGTSVGLTSVSLGVIRAMERKGVRLSVFKPIAQPRTGGDAPDQTTTIVRANSSTTTAAEPLKMSYVEGLLSSNQKDVLMEEIVANYHANTKDAEVVLVEGLVPTRKHQFAQSLNYEIAKTLNAEIVFVMSQGTDTPEQLKERIELTRNSFGGAKNTNITGVIVNKLNAPVDEQGRTRPDLSEIFDDSSKAKVNNVDPAKLQESSPLPVLGAVPWSFDLIATRAIDMARHLNATIINEGDINTRRVKSVTFCARSIPHMLEHFRAGSLLVTSADRPDVLVAACLAAMNGVEIGALLLTGGYEMDARISKLCERAFATGLPVFMVNTNTWQTSLSLQSFNLEVPVDDHERIEKVQEYVANYINADWIESLTATSERSRRLSPPAFRYQLTELARKAGKRIVLPEGDEPRTVKAAAICAERGIATCVLLGNPAEINRVAASQGVELGAGIEIVDPEVVRESYVGRLVELRKNKGMTETVAREQLEDNVVLGTLMLEQDEVDGLVSGAVHTTANTIRPPLQLIKTAPGSSLVSSVFFMLLPEQVYVYGDCAINPDPTAEQLAEIAIQSADSAAAFGIEPRVAMLSYSTGTSGAGSDVEKVREATRLAQEKRPDLMIDGPLQYDAAVMADVAKSKAPNSPVAGRATVFIFPDLNTGNTTYKAVQRSADLISIGPMLQGMRKPVNDLSRGALVDDIVYTIALTAIQSAQQQ',\n",
       "  'mw': 77172.0},\n",
       " 'b2303': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AC19',\n",
       "  'gene': 'b2303',\n",
       "  'locus': ['b2303', 'JW2300'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [(\"7,8-dihydroneopterin 3'-triphosphate = 7,8-dihydromonapterin 3'-triphosphate\",\n",
       "     '5.1.99.7')],\n",
       "   'Function': \"Catalyzes the epimerization of carbon 2' of the side chain of 7,8-dihydroneopterin triphosphate (H2NTP) to form 7,8-dihydromonapterin triphosphate (H2MTP) (PubMed:9182560) (PubMed:9651328). Is required for tetrahydromonapterin biosynthesis, a major pterin in E.coli (PubMed:19897652)\"},\n",
       "  'seq': 'MAQPAAIIRIKNLRLRTFIGIKEEEINNRQDIVINVTIHYPADKARTSEDINDALNYRTVTKNIIQHVENNRFSLLEKLTQDVLDIAREHHWVTYAEVEIDKLHALRYADSVSMTLSWQR',\n",
       "  'mw': 14082.0},\n",
       " 'b2306': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P07109',\n",
       "  'gene': 'b2306',\n",
       "  'locus': ['b2306', 'JW2303'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the histidine permease ABC transporter. Also part of a lysine/arginine/ornithine transporter. Responsible for energy coupling to the transport system (By similarity)'},\n",
       "  'seq': 'MSENKLNVIDLHKRYGEHEVLKGVSLQANAGDVISIIGSSGSGKSTFLRCINFLEKPSEGSIVVNGQTINLVRDKDGQLKVADKNQLRLLRTRLTMVFQHFNLWSHMTVLENVMEAPIQVLGLSKQEARERAVKYLAKVGIDERAQGKYPVHLSGGQQQRVSIARALAMEPEVLLFDEPTSALDPELVGEVLRIMQQLAEEGKTMVVVTHEMGFARHVSTHVIFLHQGKIEEEGAPEQLFGNPQSPRLQRFLKGSLK',\n",
       "  'mw': 28653.0},\n",
       " 'b2307': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AEU3',\n",
       "  'gene': 'b2307',\n",
       "  'locus': ['b2307', 'JW2304'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the histidine permease ABC transporter. Also part of a lysine/arginine/ornithine transporter. Probably responsible for the translocation of the substrate across the membrane. Required to relay the ATPase-inducing signal from the solute-binding protein to HisP (By similarity)'},\n",
       "  'seq': 'MIEILHEYWKPLLWTDGYRFTGVAITLWLLILSVVIGGVLALFLAIGRVSSNKYIQFPIWLFTYIFRGTPLYVQLLVFYSGMYTLEIVKGTEFLNAFFRSGLNCTVLALTLNTCAYTTEIFAGAIRSVPHGEIEAARAYGFSTFKMYRCIILPSALRIALPAYSNEVILMLHSTALAFTATVPDLLKIARDINAATYQPFTAFGIAAVLYLIISYVLISLFRRAEKRWLQHVKPSSTH',\n",
       "  'mw': 26870.0},\n",
       " 'b2308': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P52094',\n",
       "  'gene': 'b2308',\n",
       "  'locus': ['b2308', 'JW2305'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the histidine permease ABC transporter. Also part of a lysine/arginine/ornithine transporter. Probably responsible for the translocation of the substrate across the membrane. Required to relay the ATPase-inducing signal from the solute-binding protein to HisP (By similarity)'},\n",
       "  'seq': 'MLYGFSGVILQGALVTLELAISSVVLAVIIGLIGAGGKLSQNRLSGLIFEGYTTLIRGVPDLVLMLLIFYGLQIALNTVTEAMGVGQIDIDPMVAGIITLGFIYGAYFTETFRGAFMAVPKGHIEAATAFGFTRGQVFRRIMFPSMMRYALPGIGNNWQVILKSTALVSLLGLEDVVKATQLAGKSTWEPFYFAIVCGVIYLVFTTVSNGVLLFLERRYSVGVKRADL',\n",
       "  'mw': 24649.0},\n",
       " 'b2309': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AEU0',\n",
       "  'gene': 'b2309',\n",
       "  'locus': ['b2309', 'JW2306'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the histidine permease ABC transporter. Binds histidine. Interacts with HisQMP and stimulates ATPase activity of HisP, which results in histidine translocation (By similarity)'},\n",
       "  'seq': 'MKKLVLSLSLVLAFSSATAAFAAIPQNIRIGTDPTYAPFESKNSQGELVGFDIDLAKELCKRINTQCTFVENPLDALIPSLKAKKIDAIMSSLSITEKRQQEIAFTDKLYAADSRLVVAKNSDIQPTVESLKGKRVGVLQGTTQETFGNEHWAPKGIEIVSYQGQDNIYSDLTAGRIDAAFQDEVAASEGFLKQPVGKDYKFGGPSVKDEKLFGVGTGMGLRKEDNELREALNKAFAEMRADGTYEKLAKKYFDFDVYGG',\n",
       "  'mw': 28483.0},\n",
       " 'b2310': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P09551',\n",
       "  'gene': 'b2310',\n",
       "  'locus': ['b2310', 'JW2307'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of an ABC transporter involved in lysine, arginine and ornithine transport. Stimulates ATPase activity of HisP (By similarity)'},\n",
       "  'seq': 'MKKSILALSLLVGLSTAASSYAALPETVRIGTDTTYAPFSSKDAKGDFVGFDIDLGNEMCKRMQVKCTWVASDFDALIPSLKAKKIDAIISSLSITDKRQQEIAFSDKLYAADSRLIAAKGSPIQPTLDSLKGKHVGVLQGSTQEAYANETWRSKGVDVVAYANQDLVYSDLAAGRLDAALQDEVAASEGFLKQPAGKDFAFAGSSVKDKKYFGDGTGVGLRKDDAELTAAFNKALGELRQDGTYDKMAKKYFDFNVYGD',\n",
       "  'mw': 27992.0},\n",
       " 'b2311': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AG03',\n",
       "  'gene': 'b2311',\n",
       "  'locus': ['b2311', 'JW2308'],\n",
       "  'synonyms': 'dedF',\n",
       "  'properties': {'Catalytic Activity': [('dimethylallyl phosphate + FMNH2 = phosphate + prenyl-FMNH2',\n",
       "     '2.5.1.129')],\n",
       "   'Function': 'Flavin prenyltransferase that catalyzes the synthesis of the prenylated FMN cofactor (prenyl-FMN) for 4-hydroxy-3-polyprenylbenzoic acid decarboxylase UbiD. The prenyltransferase is metal-independent and links a dimethylallyl moiety from dimethylallyl monophosphate (DMAP) to the flavin N5 and C6 atoms of FMN (By similarity). Acts in concert with UbiD to perform the decarboxylation of 4-hydroxy-3-octaprenyl-benzoate, a step in the biosynthesis of coenzyme Q (PubMed:16923914, PubMed:17889824)'},\n",
       "  'seq': 'MKRLIVGISGASGAIYGVRLLQVLRDVTDIETHLVMSQAARQTLSLETDFSLREVQALADVTHDARDIAASISSGSFQTLGMVILPCSIKTLSGIVHSYTDGLLTRAADVVLKERRPLVLCVRETPLHLGHLRLMTQAAEIGAVIMPPVPAFYHRPQSLDDVINQTVNRVLDQFAITLPEDLFARWQGA',\n",
       "  'mw': 20695.0},\n",
       " 'b2312': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AG16',\n",
       "  'gene': 'b2312',\n",
       "  'locus': ['b2312', 'JW2309'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('5-phospho-beta-D-ribosylamine + diphosphate + L-glutamate = 5-phospho-alpha-D-ribose 1-diphosphate + H2O + L-glutamine',\n",
       "     '2.4.2.14')],\n",
       "   'Function': 'Catalyzes the formation of phosphoribosylamine from phosphoribosylpyrophosphate (PRPP) and glutamine. Can also use NH(3) in place of glutamine',\n",
       "   'Regulation Activity': 'Inhibited by iodoacetamide and by the glutamine analogs chloroketone and DON',\n",
       "   'Pathway': 'Purine metabolism; IMP biosynthesis via de novo pathway; N(1)-(5-phospho-D-ribosyl)glycinamide from 5-phospho-alpha-D-ribose 1-diphosphate: step 1/2'},\n",
       "  'seq': 'MCGIVGIAGVMPVNQSIYDALTVLQHRGQDAAGIITIDANNCFRLRKANGLVSDVFEARHMQRLQGNMGIGHVRYPTAGSSSASEAQPFYVNSPYGITLAHNGNLTNAHELRKKLFEEKRRHINTTSDSEILLNIFASELDNFRHYPLEADNIFAAIAATNRLIRGAYACVAMIIGHGMVAFRDPNGIRPLVLGKRDIDENRTEYMVASESVALDTLGFDFLRDVAPGEAIYITEEGQLFTRQCADNPVSNPCLFEYVYFARPDSFIDKISVYSARVNMGTKLGEKIAREWEDLDIDVVIPIPETSCDIALEIARILGKPYRQGFVKNRYVGRTFIMPGQQLRRKSVRRKLNANRAEFRDKNVLLVDDSIVRGTTSEQIIEMAREAGAKKVYLASAAPEIRFPNVYGIDMPSATELIAHGREVDEIRQIIGADGLIFQDLNDLIDAVRAENPDIQQFECSVFNGVYVTKDVDQGYLDFLDTLRNDDAKAVQRQNEVENLEMHNEG',\n",
       "  'mw': 56488.0},\n",
       " 'b2315': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P08192',\n",
       "  'gene': 'b2315',\n",
       "  'locus': ['b2315', 'JW2312'],\n",
       "  'synonyms': 'dedC',\n",
       "  'properties': {'Catalytic Activity': [('7,8-dihydropteroate + ATP + L-glutamate = 7,8-dihydrofolate + ADP + H(+) + phosphate',\n",
       "     '6.3.2.12'),\n",
       "    ('(6S)-5,6,7,8-tetrahydrofolyl-(gamma-L-Glu)(n) + ATP + L-glutamate = (6S)-5,6,7,8-tetrahydrofolyl-(gamma-L-Glu)(n+1) + ADP + H(+) + phosphate',\n",
       "     '6.3.2.17'),\n",
       "    ('10-formyltetrahydrofolyl-(gamma-L-Glu)(n) + ATP + L-glutamate = 10-formyltetrahydrofolyl-(gamma-L-Glu)(n+1) + ADP + H(+) + phosphate',\n",
       "     '6.3.2.17'),\n",
       "    ('(6R)-5,10-methylenetetrahydrofolyl-(gamma-L-Glu)(n) + ATP + L-glutamate = (6R)-5,10-methylenetetrahydrofolyl-(gamma-L-Glu)(n+1) + ADP + H(+) + phosphate',\n",
       "     '6.3.2.17')],\n",
       "   'Function': 'Functions in two distinct reactions of the de novo folate biosynthetic pathway. Catalyzes the addition of a glutamate residue to dihydropteroate (7,8-dihydropteroate or H2Pte) to form dihydrofolate (7,8-dihydrofolate monoglutamate or H2Pte-Glu). Also catalyzes successive additions of L-glutamate to tetrahydrofolate or 10-formyltetrahydrofolate or 5,10-methylenetetrahydrofolate, leading to folylpolyglutamate derivatives',\n",
       "   'Pathway': 'Cofactor biosynthesis; tetrahydrofolylpolyglutamate biosynthesis'},\n",
       "  'seq': 'MIIKRTPQAASPLASWLSYLENLHSKTIDLGLERVSLVAARLGVLKPAPFVFTVAGTNGKGTTCRTLESILMAAGYKVGVYSSPHLVRYTERVRVQGQELPESAHTASFAEIESARGDISLTYFEYGTLSALWLFKQAQLDVVILEVGLGGRLDATNIVDADVAVVTSIALDHTDWLGPDRESIGREKAGIFRSEKPAIVGEPEMPSTIADVAQEKGALLQRRGVEWNYSVTDHDWAFSDAHGTLENLPLPLVPQPNAATALAALRASGLEVSENAIRDGIASAILPGRFQIVSESPRVIFDVAHNPHAAEYLTGRMKALPKNGRVLAVIGMLHDKDIAGTLAWLKSVVDDWYCAPLEGPRGATAEQLLEHLGNGKSFDSVAQAWDAAMADAKAEDTVLVCGSFHTVAHVMEVIDARRSGGK',\n",
       "  'mw': 45406.0},\n",
       " 'b2316': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A9Q5',\n",
       "  'gene': 'b2316',\n",
       "  'locus': ['b2316', 'JW2313'],\n",
       "  'synonyms': 'dedB usg',\n",
       "  'properties': {'Catalytic Activity': [('acetyl-CoA + N(6)-carboxybiotinyl-L-lysyl-[protein] = malonyl-CoA + N(6)-biotinyl-L-lysyl-[protein]',\n",
       "     '2.1.3.15')],\n",
       "   'Function': \"Controls translation of mRNA for both itself and the alpha-subunit (accA) by binding to a probable hairpin in the 5' of the mRNA. Binding to mRNA inhibits translation; this is partially relieved by acetyl-CoA. Increasing amounts of mRNA also inhibit enzyme activity\",\n",
       "   'Regulation Activity': 'Competitively inhibited by pyrrolidine dione antibiotic moiramide B (CPD1)',\n",
       "   'Pathway': 'Lipid metabolism; malonyl-CoA biosynthesis; malonyl-CoA from acetyl-CoA: step 1/1'},\n",
       "  'seq': 'MSWIERIKSNITPTRKASIPEGVWTKCDSCGQVLYRAELERNLEVCPKCDHHMRMTARNRLHSLLDEGSLVELGSELEPKDVLKFRDSKKYKDRLASAQKETGEKDALVVMKGTLYGMPVVAAAFEFAFMGGSMGSVVGARFVRAVEQALEDNCPLICFSASGGARMQEALMSLMQMAKTSAALAKMQERGLPYISVLTDPTMGGVSASFAMLGDLNIAEPKALIGFAGPRVIEQTVREKLPPGFQRSEFLIEKGAIDMIVRRPEMRLKLASILAKLMNLPAPNPEAPREGVVVPPVPDQEPEA',\n",
       "  'mw': 33322.0},\n",
       " 'b2320': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P05459',\n",
       "  'gene': 'b2320',\n",
       "  'locus': ['b2320', 'JW2317'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('4-phospho-D-erythronate + NAD(+) = (R)-3-hydroxy-2-oxo-4-phosphooxybutanoate + H(+) + NADH',\n",
       "     '1.1.1.290')],\n",
       "   'Function': 'Catalyzes the oxidation of erythronate-4-phosphate to 3-hydroxy-2-oxo-4-phosphonooxybutanoate',\n",
       "   'Pathway': \"Cofactor biosynthesis; pyridoxine 5'-phosphate biosynthesis; pyridoxine 5'-phosphate from D-erythrose 4-phosphate: step 2/5\"},\n",
       "  'seq': 'MKILVDENMPYARDLFSRLGEVTAVPGRPIPVAQLADADALMVRSVTKVNESLLAGKPIKFVGTATAGTDHVDEAWLKQAGIGFSAAPGCNAIAVVEYVFSSLLMLAERDGFSLYDRTVGIVGVGNVGRRLQARLEALGIKTLLCDPPRADRGDEGDFRSLDELVQRADILTFHTPLFKDGPYKTLHLADEKLIRSLKPGAILINACRGAVVDNTALLTCLNEGQKLSVVLDVWEGEPELNVELLKKVDIGTSHIAGYTLEGKARGTTQVFEAYSKFIGHEQHVALDTLLPAPEFGRITLHGPLDQPTLKRLVHLVYDVRRDDAPLRKVAGIPGEFDKLRKNYLERREWSSLYVICDDASAASLLCKLGFNAVHHPAR',\n",
       "  'mw': 41368.0},\n",
       " 'b2323': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A953',\n",
       "  'gene': 'b2323',\n",
       "  'locus': ['b2323', 'JW2320'],\n",
       "  'synonyms': 'fabC',\n",
       "  'properties': {'Catalytic Activity': [('a fatty acyl-[ACP] + H(+) + malonyl-[ACP] = a 3-oxoacyl-[ACP] + CO2 + holo-[ACP]',\n",
       "     '2.3.1.41'),\n",
       "    ('(3Z)-decenoyl-[ACP] + H(+) + malonyl-[ACP] = 3-oxo-(5Z)-dodecenoyl-[ACP] + CO2 + holo-[ACP]',\n",
       "     ''),\n",
       "    ('H(+) + hexanoyl-[ACP] + malonyl-[ACP] = 3-oxooctanoyl-[ACP] + CO2 + holo-[ACP]',\n",
       "     ''),\n",
       "    ('H(+) + malonyl-[ACP] + octanoyl-[ACP] = 3-oxodecanoyl-[ACP] + CO2 + holo-[ACP]',\n",
       "     ''),\n",
       "    ('decanoyl-[ACP] + H(+) + malonyl-[ACP] = 3-oxododecanoyl-[ACP] + CO2 + holo-[ACP]',\n",
       "     ''),\n",
       "    ('dodecanoyl-[ACP] + H(+) + malonyl-[ACP] = 3-oxotetradecanoyl-[ACP] + CO2 + holo-[ACP]',\n",
       "     ''),\n",
       "    ('H(+) + malonyl-[ACP] + tetradecanoyl-[ACP] = 3-oxohexadecanoyl-[ACP] + CO2 + holo-[ACP]',\n",
       "     ''),\n",
       "    ('(7Z)-tetradecenoyl-[ACP] + H(+) + malonyl-[ACP] = 3-oxo-(9Z)-hexadecenoyl-[ACP] + CO2 + holo-[ACP]',\n",
       "     '')],\n",
       "   'Function': 'Involved in the type II fatty acid elongation cycle. Catalyzes the elongation of a wide range of acyl-ACP by the addition of two carbons from malonyl-ACP to an acyl acceptor (PubMed:9013860, PubMed:8910376, PubMed:19679654, PubMed:22017312). Can also use unsaturated fatty acids (PubMed:3076377, PubMed:8910376, PubMed:19679654). Catalyzes a key reaction in unsaturated fatty acid (UFA) synthesis, the elongation of the cis-3-decenoyl-ACP produced by FabA (PubMed:19679654). Can use acyl chains from C-6 to C-14 (PubMed:9013860, PubMed:8910376, PubMed:19679654, PubMed:22017312). Has an absolute requirement for an ACP substrate as the acyl donor, and no activity is detected when both substrates are based on CoA (PubMed:22017312)',\n",
       "   'Pathway': 'Lipid metabolism; fatty acid biosynthesis'},\n",
       "  'seq': 'MKRAVITGLGIVSSIGNNQQEVLASLREGRSGITFSQELKDSGMRSHVWGNVKLDTTGLIDRKVVRFMSDASIYAFLSMEQAIADAGLSPEAYQNNPRVGLIAGSGGGSPRFQVFGADAMRGPRGLKAVGPYVVTKAMASGVSACLATPFKIHGVNYSISSACATSAHCIGNAVEQIQLGKQDIVFAGGGEELCWEMACEFDAMGALSTKYNDTPEKASRTYDAHRDGFVIAGGGGMVVVEELEHALARGAHIYAEIVGYGATSDGADMVAPSGEGAVRCMKMAMHGVDTPIDYLNSHGTSTPVGDVKELAAIREVFGDKSPAISATKAMTGHSLGAAGVQEAIYSLLMLEHGFIAPSINIEELDEQAAGLNIVTETTDRELTTVMSNSFGFGGTNATLVMRKLKD',\n",
       "  'mw': 42613.0},\n",
       " 'b2328': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0C0T5',\n",
       "  'gene': 'b2328',\n",
       "  'locus': ['b2328', 'JW2325'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Murein endopeptidase that cleaves the D-alanyl-meso-2,6-diamino-pimelyl amide bond that connects peptidoglycan strands. Likely plays a role in the removal of murein from the sacculus and could also play a role in the integration of nascent murein strands into the sacculus',\n",
       "   'Regulation Activity': 'Inhibited by Zn(2+) at 10 mM and by metal chelating agents EDTA and 1,10-phenanthroline'},\n",
       "  'seq': 'MNKTAIALLALLASSASLAATPWQKITQPVPGSAQSIGSFSNGCIVGADTLPIQSEHYQVMRTDQRRYFGHPDLVMFIQRLSSQVSNLGMGTVLIGDMGMPAGGRFNGGHASHQTGLDVDIFLQLPKTRWTSAQLLRPQALDLVSRDGKHVVSTLWKPEIFSLIKLAAQDKDVTRIFVNPAIKQQLCLDAGTDRDWLRKVRPWFQHRAHMHVRLRCPADSLECEDQPLPPSGDGCGAELQSWFEPPKPGTTKPEKKTPPPLPPSCQALLDEHVI',\n",
       "  'mw': 30137.0},\n",
       " 'b2329': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P12008',\n",
       "  'gene': 'b2329',\n",
       "  'locus': ['b2329', 'JW2326'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('5-O-(1-carboxyvinyl)-3-phosphoshikimate = chorismate + phosphate',\n",
       "     '4.2.3.5')],\n",
       "   'Function': 'Catalyzes the anti-1,4-elimination of the C-3 phosphate and the C-6 proR hydrogen from 5-enolpyruvylshikimate-3-phosphate (EPSP) to yield chorismate, which is the branch point compound that serves as the starting substrate for the three terminal pathways of aromatic amino acid biosynthesis. This reaction introduces a second double bond into the aromatic ring system. It uses NADPH to reduce FMN',\n",
       "   'Regulation Activity': 'Competitively inhibited by 6R-6-fluoro-EPSP',\n",
       "   'Pathway': 'Metabolic intermediate biosynthesis; chorismate biosynthesis; chorismate from D-erythrose 4-phosphate and phosphoenolpyruvate: step 7/7'},\n",
       "  'seq': 'MAGNTIGQLFRVTTFGESHGLALGCIVDGVPPGIPLTEADLQHDLDRRRPGTSRYTTQRREPDQVKILSGVFEGVTTGTSIGLLIENTDQRSQDYSAIKDVFRPGHADYTYEQKYGLRDYRGGGRSSARETAMRVAAGAIAKKYLAEKFGIEIRGCLTQMGDIPLDIKDWSQVEQNPFFCPDPDKIDALDELMRALKKEGDSIGAKVTVVASGVPAGLGEPVFDRLDADIAHALMSINAVKGVEIGDGFDVVALRGSQNRDEITKDGFQSNHAGGILGGISSGQQIIAHMALKPTSSITVPGRTINRFGEEVEMITKGRHDPCVGIRAVPIAEAMLAIVLMDHLLRQRAQNADVKTDIPRW',\n",
       "  'mw': 39137.0},\n",
       " 'b2341': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77399',\n",
       "  'gene': 'b2341',\n",
       "  'locus': ['b2341', 'JW2338'],\n",
       "  'synonyms': 'yfcX',\n",
       "  'properties': {'Catalytic Activity': [('a (3S)-3-hydroxyacyl-CoA = a (2E)-enoyl-CoA + H2O',\n",
       "     '4.2.1.17'),\n",
       "    ('a 4-saturated-(3S)-3-hydroxyacyl-CoA = a (3E)-enoyl-CoA + H2O',\n",
       "     '4.2.1.17'),\n",
       "    ('a (3S)-3-hydroxyacyl-CoA + NAD(+) = a 3-oxoacyl-CoA + H(+) + NADH',\n",
       "     '1.1.1.35'),\n",
       "    ('(3S)-3-hydroxybutanoyl-CoA = (3R)-3-hydroxybutanoyl-CoA', '5.1.2.3')],\n",
       "   'Function': 'Catalyzes the formation of a hydroxyacyl-CoA by addition of water on enoyl-CoA. Also exhibits 3-hydroxyacyl-CoA epimerase and 3-hydroxyacyl-CoA dehydrogenase activities. Strongly involved in the anaerobic degradation of long and medium-chain fatty acids in the presence of nitrate and weakly involved in the aerobic degradation of long-chain fatty acids',\n",
       "   'Pathway': 'Lipid metabolism; fatty acid beta-oxidation'},\n",
       "  'seq': 'MEMTSAFTLNVRLDNIAVITIDVPGEKMNTLKAEFASQVRAIIKQLRENKELRGVVFVSAKPDNFIAGADINMIGNCKTAQEAEALARQGQQLMAEIHALPIQVIAAIHGACLGGGLELALACHGRVCTDDPKTVLGLPEVQLGLLPGSGGTQRLPRLIGVSTALEMILTGKQLRAKQALKLGLVDDVVPHSILLEAAVELAKKERPSSRPLPVRERILAGPLGRALLFKMVGKKTEHKTQGNYPATERILEVVETGLAQGTSSGYDAEARAFGELAMTPQSQALRSIFFASTDVKKDPGSDAPPAPLNSVGILGGGLMGGGIAYVTACKAGIPVRIKDINPQGINHALKYSWDQLEGKVRRRHLKASERDKQLALISGTTDYRGFAHRDLIIEAVFENLELKQQMVAEVEQNCAAHTIFASNTSSLPIGDIAAHATRPEQVIGLHFFSPVEKMPLVEIIPHAGTSAQTIATTVKLAKKQGKTPIVVRDKAGFYVNRILAPYINEAIRMLTQGERVEHIDAALVKFGFPVGPIQLLDEVGIDTGTKIIPVLEAAYGERFSAPANVVSSILNDDRKGRKNGRGFYLYGQKGRKSKKQVDPAIYPLIGTQGQGRISAPQVAERCVMLMLNEAVRCVDEQVIRSVRDGDIGAVFGIGFPPFLGGPFRYIDSLGAGEVVAIMQRLATQYGSRFTPCERLVEMGARGESFWKTTATDLQ',\n",
       "  'mw': 77072.0},\n",
       " 'b2342': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P76503',\n",
       "  'gene': 'b2342',\n",
       "  'locus': ['b2342', 'JW2339'],\n",
       "  'synonyms': 'yfcY',\n",
       "  'properties': {'Catalytic Activity': [('acetyl-CoA + an acyl-CoA = a 3-oxoacyl-CoA + CoA',\n",
       "     '2.3.1.16')],\n",
       "   'Function': 'Catalyzes the final step of fatty acid oxidation in which acetyl-CoA is released and the CoA ester of a fatty acid two carbons shorter is formed. Strongly involved in the anaerobic degradation of long and medium-chain fatty acids in the presence of nitrate and weakly involved in the aerobic degradation of long-chain fatty acids',\n",
       "   'Pathway': 'Lipid metabolism; fatty acid beta-oxidation'},\n",
       "  'seq': 'MGQVLPLVTRQGDRIAIVSGLRTPFARQATAFHGIPAVDLGKMVVGELLARSEIPAEVIEQLVFGQVVQMPEAPNIAREIVLGTGMNVHTDAYSVSRACATSFQAVANVAESLMAGTIRAGIAGGADSSSVLPIGVSKKLARVLVDVNKARTMSQRLKLFSRLRLRDLMPVPPAVAEYSTGLRMGDTAEQMAKTYGITREQQDALAHRSHQRAAQAWSDGKLKEEVMTAFIPPYKQPLVEDNNIRGNSSLADYAKLRPAFDRKHGTVTAANSTPLTDGAAAVILMTESRAKELGLVPLGYLRSYAFTAIDVWQDMLLGPAWSTPLALERAGLTMSDLTLIDMHEAFAAQTLANIQLLGSERFAREALGRAHATGEVDDSKFNVLGGSIAYGHPFAATGARMITQTLHELRRRGGGFGLVTACAAGGLGAAMVLEAE',\n",
       "  'mw': 46531.0},\n",
       " 'b2344': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P10384',\n",
       "  'gene': 'b2344',\n",
       "  'locus': ['b2344', 'JW2341'],\n",
       "  'synonyms': 'ttr',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Involved in translocation of long-chain fatty acids across the outer membrane. It is a receptor for the bacteriophage T2. FadL may form a specific channel'},\n",
       "  'seq': 'MSQKTLFTKSALAVAVALISTQAWSAGFQLNEFSSSGLGRAYSGEGAIADDAGNVSRNPALITMFDRPTFSAGAVYIDPDVNISGTSPSGRSLKADNIAPTAWVPNMHFVAPINDQFGWGASITSNYGLATEFNDTYAGGSVGGTTDLETMNLNLSGAYRLNNAWSFGLGFNAVYARAKIERFAGDLGQLVAGQIMQSPAGQTQQGQALAATANGIDSNTKIAHLNGNQWGFGWNAGILYELDKNNRYALTYRSEVKIDFKGNYSSDLNRAFNNYGLPIPTATGGATQSGYLTLNLPEMWEVSGYNRVDPQWAIHYSLAYTSWSQFQQLKATSTSGDTLFQKHEGFKDAYRIALGTTYYYDDNWTFRTGIAFDDSPVPAQNRSISIPDQDRFWLSAGTTYAFNKDASVDVGVSYMHGQSVKINEGPYQFESEGKAWLFGTNFNYAF',\n",
       "  'mw': 48542.0},\n",
       " 'b2366': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P00926',\n",
       "  'gene': 'b2366',\n",
       "  'locus': ['b2366', 'JW2363'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('D-serine = NH4(+) + pyruvate',\n",
       "     '4.3.1.18')]},\n",
       "  'seq': 'MENAKMNSLIAQYPLVKDLVALKETTWFNPGTTSLAEGLPYVGLTEQDVQDAHARLSRFAPYLAKAFPETAATGGIIESELVAIPAMQKRLEKEYQQPISGQLLLKKDSHLPISGSIKARGGIYEVLAHAEKLALEAGLLTLDDDYSKLLSPEFKQFFSQYSIAVGSTGNLGLSIGIMSARIGFKVTVHMSADARAWKKAKLRSHGVTVVEYEQDYGVAVEEGRKAAQSDPNCFFIDDENSRTLFLGYSVAGQRLKAQFAQQGRIVDADNPLFVYLPCGVGGGPGGVAFGLKLAFGDHVHCFFAEPTHSPCMLLGVHTGLHDQISVQDIGIDNLTAADGLAVGRASGFVGRAMERLLDGFYTLSDQTMYDMLGWLAQEEGIRLEPSALAGMAGPQRVCASVSYQQMHGFSAEQLRNTTHLVWATGGGMVPEEEMNQYLAKGR',\n",
       "  'mw': 47901.0},\n",
       " 'b2378': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ACV2',\n",
       "  'gene': 'b2378',\n",
       "  'locus': ['b2378', 'JW2375'],\n",
       "  'synonyms': 'ddg',\n",
       "  'properties': {'Catalytic Activity': [('(9Z)-hexadecenoyl-[ACP] + alpha-Kdo-(2->4)-alpha-Kdo-(2->6)-lipid IVA (E. coli) = (9Z)-hexadecenoyl-(Kdo)2-lipid IVA (E. coli) + holo-[ACP]',\n",
       "     '2.3.1.242')],\n",
       "   'Function': 'Catalyzes the transfer of palmitoleate from palmitoleoyl-[acyl-carrier-protein] (ACP) to Kdo(2)-lipid IV(A) to form Kdo(2)-(palmitoleoyl)-lipid IV(A). Required for the biosynthesis of a distinct molecular species of lipid A, which is present only in cells grown at low temperatures. It may confer a selective advantage to cells growing at lower temperatures by making the outer membrane a more effective barrier to harmful chemicals',\n",
       "   'Pathway': 'Bacterial outer membrane biogenesis; lipopolysaccharide biosynthesis'},\n",
       "  'seq': 'MFPQCKFSREFLHPRYWLTWFGLGVLWLWVQLPYPVLCFLGTRIGAMARPFLKRRESIARKNLELCFPQHSAEEREKMIAENFRSLGMALVETGMAWFWPDSRVRKWFDVEGLDNLKRAQMQNRGVMVVGVHFMSLELGGRVMGLCQPMMATYRPHNNQLMEWVQTRGRMRSNKAMIGRNNLRGIVGALKKGEAVWFAPDQDYGRKGSSFAPFFAVENVATTNGTYVLSRLSGAAMLTVTMVRKADYSGYRLFITPEMEGYPTDENQAAAYMNKIIEKEIMRAPEQYLWIHRRFKTRPVGESSLYI',\n",
       "  'mw': 35493.0},\n",
       " 'b2388': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A6V8',\n",
       "  'gene': 'b2388',\n",
       "  'locus': ['b2388', 'JW2385'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + D-glucose = ADP + D-glucose 6-phosphate + H(+)',\n",
       "     '2.7.1.2')],\n",
       "   'Function': 'An ATP-dependent kinase that phosphorylates glucose to make glucose-6-phosphate, not active on fructose, galactose or mannose. Not highly important in wild-type E.coli as glucose is transported into the cell by the PTS system as glucose 6-phosphate'},\n",
       "  'seq': 'MTKYALVGDVGGTNARLALCDIASGEISQAKTYSGLDYPSLEAVIRVYLEEHKVEVKDGCIAIACPITGDWVAMTNHTWAFSIAEMKKNLGFSHLEIINDFTAVSMAIPMLKKEHLIQFGGAEPVEGKPIAVYGAGTGLGVAHLVHVDKRWVSLPGEGGHVDFAPNSEEEAIILEILRAEIGHVSAERVLSGPGLVNLYRAIVKADNRLPENLKPKDITERALADSCTDCRRALSLFCVIMGRFGGNLALNLGTFGGVFIAGGIVPRFLEFFKASGFRAAFEDKGRFKEYVHDIPVYLIVHDNPGLLGSGAHLRQTLGHIL',\n",
       "  'mw': 34723.0},\n",
       " 'b2392': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A769',\n",
       "  'gene': 'b2392',\n",
       "  'locus': ['b2392', 'JW2388'],\n",
       "  'synonyms': 'yfeP',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'H(+)-stimulated, divalent metal cation uptake system. Involved in manganese and iron uptake. Can also transport cadmium, cobalt, zinc and to a lesser extent nickel and copper. Involved in response to reactive oxygen'},\n",
       "  'seq': 'MTNYRVESSSGRAARKMRLALMGPAFIAAIGYIDPGNFATNIQAGASFGYQLLWVVVWANLMAMLIQILSAKLGIATGKNLAEQIRDHYPRPVVWFYWVQAEIIAMATDLAEFIGAAIGFKLILGVSLLQGAVLTGIATFLILMLQRRGQKPLEKVIGGLLLFVAAAYIVELIFSQPNLAQLGKGMVIPSLPTSEAVFLAAGVLGATIMPHVIYLHSSLTQHLHGGSRQQRYSATKWDVAIAMTIAGFVNLAMMATAAAAFHFSGHTGVADLDEAYLTLQPLLSHAAATVFGLSLVAAGLSSTVVGTLAGQVVMQGFIRFHIPLWVRRTVTMLPSFIVILMGLDPTRILVMSQVLLSFGIALALVPLLIFTSDSKLMGDLVNSKRVKQTGWVIVVLVVALNIWLLVGTALGL',\n",
       "  'mw': 44194.0},\n",
       " 'b2393': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AFF2',\n",
       "  'gene': 'b2393',\n",
       "  'locus': ['b2393', 'JW2389'],\n",
       "  'synonyms': 'cru',\n",
       "  'properties': {'Catalytic Activity': [('adenosine(in) + H(+)(in) = adenosine(out) + H(+)(out)',\n",
       "     ''),\n",
       "    ('H(+)(in) + uridine(in) = H(+)(out) + uridine(out)', ''),\n",
       "    ('H(+)(in) + thymidine(in) = H(+)(out) + thymidine(out)', ''),\n",
       "    ('cytidine(in) + H(+)(in) = cytidine(out) + H(+)(out)', ''),\n",
       "    (\"2'-deoxycytidine(in) + H(+)(in) = 2'-deoxycytidine(out) + H(+)(out)\",\n",
       "     '')],\n",
       "   'Function': \"Nucleoside transport protein that can transport adenosine, uridine, thymidine, cytidine and deoxycytidine (PubMed:374403, PubMed:11466294, PubMed:14668133, PubMed:15678184). Shows weak activity with inosine and xanthosine (PubMed:11466294, PubMed:14668133). Transport is driven by a proton motive force (PubMed:374403, PubMed:14668133). Does not transport guanosine, deoxyguanosine, hypoxanthine or uracil (PubMed:374403, PubMed:14668133). Also shows activity with the chemotherapeutic drugs 3'-azido-3'-deoxythymidine (AZT), 2',3'- dideoxycytidine (ddC) and 2'-deoxy-2',2'-difluorocytidine (gemcitabine) (PubMed:14668133)\",\n",
       "   'Regulation Activity': 'Transport is inhibited by the proton uncoupler dinitrophenol (PubMed:374403). Inhibited by the nucleoside antibiotic showdomycin (PubMed:374403)'},\n",
       "  'seq': 'MDRVLHFVLALAVVAILALLVSSDRKKIRIRYVIQLLVIEVLLAWFFLNSDVGLGFVKGFSEMFEKLLGFANEGTNFVFGSMNDQGLAFFFLKVLCPIVFISALIGILQHIRVLPVIIRAIGFLLSKVNGMGKLESFNAVSSLILGQSENFIAYKDILGKISRNRMYTMAATAMSTVSMSIVGAYMTMLEPKYVVAALVLNMFSTFIVLSLINPYRVDASEENIQMSNLHEGQSFFEMLGEYILAGFKVAIIVAAMLIGFIALIAALNALFATVTGWFGYSISFQGILGYIFYPIAWVMGVPSSEALQVGSIMATKLVSNEFVAMMDLQKIASTLSPRAEGIISVFLVSFANFSSIGIIAGAVKGLNEEQGNVVSRFGLKLVYGSTLVSVLSASIAALVL',\n",
       "  'mw': 43476.0},\n",
       " 'b2400': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P04805',\n",
       "  'gene': 'b2400',\n",
       "  'locus': ['b2400', 'JW2395'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + L-glutamate + tRNA(Glu) = AMP + diphosphate + L-glutamyl-tRNA(Glu)',\n",
       "     '6.1.1.17')],\n",
       "   'Function': 'Phosphorylation of GltX by HipA prevents it from being charged, leading to an increase in uncharged tRNA(Glu). This induces amino acid starvation and the stringent response via RelA/SpoT and increased (p)ppGpp levels, which inhibits replication, transcription, translation and cell wall synthesis, reducing growth and leading to multidrug resistance and persistence (PubMed:24095282, PubMed:24343429). Overexpression of GltX prevents HipA-induced growth arrest, persister formation and increases in (p)ppGpp levels (PubMed:24343429, PubMed:28430938)',\n",
       "   'Regulation Activity': 'Inhibited by 1,10-phenanthroline (PubMed:8218204). Phosphorylation of Ser-239 by HipA inhibits aminoacylation of tRNA(Glu) (PubMed:24095282, PubMed:24343429)'},\n",
       "  'seq': 'MKIKTRFAPSPTGYLHVGGARTALYSWLFARNHGGEFVLRIEDTDLERSTPEAIEAIMDGMNWLSLEWDEGPYYQTKRFDRYNAVIDQMLEEGTAYKCYCSKERLEALREEQMAKGEKPRYDGRCRHSHEHHADDEPCVVRFANPQEGSVVFDDQIRGPIEFSNQELDDLIIRRTDGSPTYNFCVVVDDWDMEITHVIRGEDHINNTPRQINILKALKAPVPVYAHVSMINGDDGKKLSKRHGAVSVMQYRDDGYLPEALLNYLVRLGWSHGDQEIFTREEMIKYFTLNAVSKSASAFNTDKLLWLNHHYINALPPEYVATHLQWHIEQENIDTRNGPQLADLVKLLGERCKTLKEMAQSCRYFYEDFAEFDADAAKKHLRPVARQPLEVVRDKLAAITDWTAENVHHAIQATADELEVGMGKVGMPLRVAVTGAGQSPALDVTVHAIGKTRSIERINKALDFIAERENQQ',\n",
       "  'mw': 53816.0},\n",
       " 'b2406': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P45562',\n",
       "  'gene': 'b2406',\n",
       "  'locus': ['b2406', 'JW2397'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('H(+)(in) + xanthosine(in) = H(+)(out) + xanthosine(out)',\n",
       "     '')],\n",
       "   'Function': 'Uptake of xanthosine (PubMed:11466294). Can also transport other nucleosides such as inosine, adenosine, cytidine, uridine and thymidine (PubMed:11466294). Transport is driven by a proton motive force (PubMed:11466294)',\n",
       "   'Regulation Activity': 'Transport is abolished by the proton uncoupler 2,4-dinitrophenol'},\n",
       "  'seq': 'MSIAMRLKVMSFLQYFIWGSWLVTLGSYMINTLHFTGANVGMVYSSKGIAAIIMPGIMGIIADKWLRAERAYMLCHLVCAGVLFYAASVTDPDMMFWVMLVNAMAFMPTIALSNSVSYSCLAQAGLDPVTAFPPIRVFGTVGFIVAMWAVSLLHLELSSLQLYIASGASLLLSAYALTLPKIPVAEKKATTSLASKLGLDAFVLFKNPRMAIFFLFAMMLGAVLQITNVFGNPFLHDFARNPEFADSFVVKYPSILLSVSQMAEVGFILTIPFFLKRFGIKTVMLMSMVAWTLRFGFFAYGDPSTTGFILLLLSMIVYGCAFDFFNISGSVFVEQEVDSSIRASAQGLFMTMVNGVGAWVGSILSGMAVDYFSVDGVKDWQTIWLVFAGYALFLAVIFFFGFKYNHDPEKIKHRAVTH',\n",
       "  'mw': 46140.0},\n",
       " 'b2407': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P45563',\n",
       "  'gene': 'b2407',\n",
       "  'locus': ['b2407', 'JW2398'],\n",
       "  'synonyms': 'pndA',\n",
       "  'properties': {'Catalytic Activity': [('a purine D-ribonucleoside + phosphate = a purine nucleobase + alpha-D-ribose 1-phosphate',\n",
       "     '2.4.2.1')],\n",
       "   'Function': 'The purine nucleoside phosphorylases catalyze the phosphorolytic breakdown of the N-glycosidic bond in the beta-(deoxy)ribonucleoside molecules, with the formation of the corresponding free purine bases and pentose-1-phosphate. This protein can degrade all purine nucleosides including xanthosine, inosine and guanosine, but cannot cleave adenosine, deoxyadenosine or hypoxanthine arabinoside. Has a preference for the neutral over the monoanionic form of xanthosine',\n",
       "   'Regulation Activity': 'Rapidly inactivated by p-chloromercuriphenylsulfonic acid (p-CMB). Dithiothreitol incubation restores the activity',\n",
       "   'Pathway': 'Purine metabolism; purine nucleoside salvage'},\n",
       "  'seq': 'MSQVQFSHNPLFCIDIIKTYKPDFTPRVAFILGSGLGALADQIENAVAISYEKLPGFPVSTVHGHAGELVLGHLQGVPVVCMKGRGHFYEGRGMTIMTDAIRTFKLLGCELLFCTNAAGSLRPEVGAGSLVALKDHINTMPGTPMVGLNDDRFGERFFSLANAYDAEYRALLQKVAKEEGFPLTEGVFVSYPGPNFETAAEIRMMQIIGGDVVGMSVVPEVISARHCDLKVVAVSAITNMAEGLSDVKLSHAQTLAAAELSKQNFINLICGFLRKIA',\n",
       "  'mw': 29835.0},\n",
       " 'b2411': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P15042',\n",
       "  'gene': 'b2411',\n",
       "  'locus': ['b2411', 'JW2403'],\n",
       "  'synonyms': 'dnaL lig lop pdeC',\n",
       "  'properties': {'Catalytic Activity': [(\"NAD(+) + (deoxyribonucleotide)n-3'-hydroxyl + 5'-phospho-(deoxyribonucleotide)m = (deoxyribonucleotide)n+m + AMP + beta-nicotinamide D-nucleotide.\",\n",
       "     '6.5.1.2')],\n",
       "   'Function': \"DNA ligase that catalyzes the formation of phosphodiester linkages between 5'-phosphoryl and 3'-hydroxyl groups in double-stranded DNA using NAD as a coenzyme and as the energy source for the reaction. It is essential for DNA replication and repair of damaged DNA\"},\n",
       "  'seq': 'MESIEQQLTELRTTLRHHEYLYHVMDAPEIPDAEYDRLMRELRELETKHPELITPDSPTQRVGAAPLAAFSQIRHEVPMLSLDNVFDEESFLAFNKRVQDRLKNNEKVTWCCELKLDGLAVSILYENGVLVSAATRGDGTTGEDITSNVRTIRAIPLKLHGENIPARLEVRGEVFLPQAGFEKINEDARRTGGKVFANPRNAAAGSLRQLDPRITAKRPLTFFCYGVGVLEGGELPDTHLGRLLQFKKWGLPVSDRVTLCESAEEVLAFYHKVEEDRPTLGFDIDGVVIKVNSLAQQEQLGFVARAPRWAVAFKFPAQEQMTFVRDVEFQVGRTGAITPVARLEPVHVAGVLVSNATLHNADEIERLGLRIGDKVVIRRAGDVIPQVVNVVLSERPEDTREVVFPTHCPVCGSDVERVEGEAVARCTGGLICGAQRKESLKHFVSRRAMDVDGMGDKIIDQLVEKEYVHTPADLFKLTAGKLTGLERMGPKSAQNVVNALEKAKETTFARFLYALGIREVGEATAAGLAAYFGTLEALEAASIEELQKVPDVGIVVASHVHNFFAEESNRNVISELLAEGVHWPAPIVINAEEIDSPFAGKTVVLTGSLSQMSRDDAKARLVELGAKVAGSVSKKTDLVIAGEAAGSKLAKAQELGIEVIDEAEMLRLLGS',\n",
       "  'mw': 73606.0},\n",
       " 'b2414': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ABK5',\n",
       "  'gene': 'b2414',\n",
       "  'locus': ['b2414', 'JW2407'],\n",
       "  'synonyms': 'cysZ',\n",
       "  'properties': {'Catalytic Activity': [('hydrogen sulfide + O-acetyl-L-serine = acetate + L-cysteine',\n",
       "     '2.5.1.47'),\n",
       "    ('3-chloro-L-alanine + thioglycolate = chloride + H(+) + S-carboxymethyl-L-cysteine',\n",
       "     '4.5.1.5')],\n",
       "   'Function': '(Microbial infection) In addition to its role in cysteine synthesis, stimulates the tRNase activity of CdiA-CT from E.coli strain 536 / UPEC; stimulation does not require O-acetylserine sulfhydrylase activity. CdiA is the toxic component of a toxin-immunity protein module, which functions as a cellular contact-dependent growth inhibition (CDI) system. CDI modules allow bacteria to communicate with and inhibit the growth of closely related neighboring bacteria in a contact-dependent fashion (experiments done in strains BW25113 and X90, both K12 derivatives). This protein is not required for CDI of strain EC93, whose toxin may function by forming inner cell membrane pores (PubMed:22333533). CysK stabilizes CdiA-CT, allowing it to bind tRNA substrate; neither CdiA-CT nor CysK bind tRNA alone in vitro (PubMed:27531961)',\n",
       "   'Pathway': 'Amino-acid biosynthesis; L-cysteine biosynthesis; L-cysteine from L-serine: step 2/2'},\n",
       "  'seq': 'MSKIFEDNSLTIGHTPLVRLNRIGNGRILAKVESRNPSFSVKCRIGANMIWDAEKRGVLKPGVELVEPTSGNTGIALAYVAAARGYKLTLTMPETMSIERRKLLKALGANLVLTEGAKGMKGAIQKAEEIVASNPEKYLLLQQFSNPANPEIHEKTTGPEIWEDTDGQVDVFIAGVGTGGTLTGVSRYIKGTKGKTDLISVAVEPTDSPVIAQALAGEEIKPGPHKIQGIGAGFIPANLDLKLVDKVIGITNEEAISTARRLMEEEGILAGISSGAAVAAALKLQEDESFTNKNIVVILPSSGERYLSTALFADLFTEKELQQ',\n",
       "  'mw': 34490.0},\n",
       " 'b2415': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AA04',\n",
       "  'gene': 'b2415',\n",
       "  'locus': ['b2415', 'JW2408'],\n",
       "  'synonyms': 'hpr',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'General (non sugar-specific) component of the phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS). This major carbohydrate active-transport system catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane. The phosphoryl group from phosphoenolpyruvate (PEP) is transferred to the phosphoryl carrier protein HPr by enzyme I. Phospho-HPr then transfers it to the PTS EIIA domain'},\n",
       "  'seq': 'MFQQEVTITAPNGLHTRPAAQFVKEAKGFTSEITVTSNGKSASAKSLFKLQTLGLTQGTVVTISAEGEDEQKAVEHLVKLMAELE',\n",
       "  'mw': 9119.0},\n",
       " 'b2416': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P08839',\n",
       "  'gene': 'b2416',\n",
       "  'locus': ['b2416', 'JW2409'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('L-histidyl-[protein] + phosphoenolpyruvate = N(pros)-phospho-L-histidyl-[protein] + pyruvate',\n",
       "     '2.7.3.9')],\n",
       "   'Function': 'General (non sugar-specific) component of the phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS). This major carbohydrate active-transport system catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane. Enzyme I transfers the phosphoryl group from phosphoenolpyruvate (PEP) to the phosphoryl carrier protein (HPr) (PubMed:7876255, PubMed:12705838, PubMed:17053069). Can also use (Z)-3-fluoro-PEP (ZFPEP), (Z)-3-methyl-PEP (ZMePEP), (Z)-3-chloro-PEP (ZClPEP) and (E)-3-chloro-PEP (EClPEP) as alternative phosphoryl donors (PubMed:12705838)',\n",
       "   'Regulation Activity': 'Inhibited by oxalate'},\n",
       "  'seq': 'MISGILASPGIAFGKALLLKEDEIVIDRKKISADQVDQEVERFLSGRAKASAQLETIKTKAGETFGEEKEAIFEGHIMLLEDEELEQEIIALIKDKHMTADAAAHEVIEGQASALEELDDEYLKERAADVRDIGKRLLRNILGLKIIDLSAIQDEVILVAADLTPSETAQLNLKKVLGFITDAGGRTSHTSIMARSLELPAIVGTGSVTSQVKNDDYLILDAVNNQVYVNPTNEVIDKMRAVQEQVASEKAELAKLKDLPAITLDGHQVEVCANIGTVRDVEGAERNGAEGVGLYRTEFLFMDRDALPTEEEQFAAYKAVAEACGSQAVIVRTMDIGGDKELPYMNFPKEENPFLGWRAIRIAMDRREILRDQLRAILRASAFGKLRIMFPMIISVEEVRALRKEIEIYKQELRDEGKAFDESIEIGVMVETPAAATIARHLAKEVDFFSIGTNDLTQYTLAVDRGNDMISHLYQPMSPSVLNLIKQVIDASHAEGKWTGMCGELAGDERATLLLLGMGLDEFSMSAISIPRIKKIIRNTNFEDAKVLAEQALAQPTTDELMTLVNKFIEEKTIC',\n",
       "  'mw': 63562.0},\n",
       " 'b2417': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P69783',\n",
       "  'gene': 'b2417',\n",
       "  'locus': ['b2417', 'JW2410'],\n",
       "  'synonyms': 'gsr iex tgs treD',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'The phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS), a major carbohydrate active transport system, catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane (PubMed:3129430, PubMed:17158705). The enzyme II complex composed of PtsG and Crr is involved in glucose transport (PubMed:2657735). The non-phosphorylated EIII-Glc is an inhibitor for uptake of certain sugars such as maltose, melibiose, lactose, and glycerol. Phosphorylated EIII-Glc, however, may be an activator for adenylate cyclase. It is an important regulatory protein for cell metabolism (PubMed:789369)'},\n",
       "  'seq': 'MGLFDKLKSLVSDDKKDTGTIEIIAPLSGEIVNIEDVPDVVFAEKIVGDGIAIKPTGNKMVAPVDGTIGKIFETNHAFSIESDSGVELFVHFGIDTVELKGEGFKRIAEEGQRVKVGDTVIEFDLPLLEEKAKSTLTPVVISNMDEIKELIKLSGSVTVGETPVIRIKK',\n",
       "  'mw': 18251.0},\n",
       " 'b2418': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P40191',\n",
       "  'gene': 'b2418',\n",
       "  'locus': ['b2418', 'JW2411'],\n",
       "  'synonyms': 'yfeI',\n",
       "  'properties': {'Catalytic Activity': [(\"ATP + pyridoxal = ADP + H(+) + pyridoxal 5'-phosphate\",\n",
       "     '2.7.1.35'),\n",
       "    (\"ATP + pyridoxine = ADP + H(+) + pyridoxine 5'-phosphate\", '2.7.1.35'),\n",
       "    (\"ATP + pyridoxamine = ADP + H(+) + pyridoxamine 5'-phosphate\",\n",
       "     '2.7.1.35')],\n",
       "   'Function': \"B6-vitamer kinase involved in the salvage pathway of pyridoxal 5'-phosphate (PLP). Catalyzes the phosphorylation of pyridoxine (PN), pyridoxal (PL), and pyridoxamine (PM), forming their respective 5'-phosphorylated esters, i.e. PNP, PLP and PMP\",\n",
       "   'Regulation Activity': 'Is activated by the monovalent cation potassium',\n",
       "   'Pathway': \"Cofactor metabolism; pyridoxal 5'-phosphate salvage; pyridoxamine 5'-phosphate from pyridoxamine: step 1/1\"},\n",
       "  'seq': 'MSSLLLFNDKSRALQADIVAVQSQVVYGSVGNSIAVPAIKQNGLNVFAVPTVLLSNTPHYDTFYGGAIPDEWFSGYLRALQERDALRQLRAVTTGYMGTASQIKILAEWLTALRKDHPDLLIMVDPVIGDIDSGIYVKPDLPEAYRQYLLPLAQGITPNIFELEILTGKNCRDLDSAIAAAKSLLSDTLKWVVVTSASGNEENQEMQVVVVTADSVNVISHSRVKTDLKGTGDLFCAQLISGLLKGKALTDAVHRAGLRVLEVMRYTQQHESDELILPPLAEA',\n",
       "  'mw': 30847.0},\n",
       " 'b2421': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P16703',\n",
       "  'gene': 'b2421',\n",
       "  'locus': ['b2421', 'JW2414'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('hydrogen sulfide + O-acetyl-L-serine = acetate + L-cysteine',\n",
       "     '2.5.1.47')],\n",
       "   'Function': 'Two cysteine synthase enzymes are found. Both catalyze the same reaction. Cysteine synthase B can also use thiosulfate in place of sulfide to give cysteine thiosulfonate as a product',\n",
       "   'Pathway': 'Amino-acid biosynthesis; L-cysteine biosynthesis; L-cysteine from L-serine: step 2/2'},\n",
       "  'seq': 'MSTLEQTIGNTPLVKLQRMGPDNGSEVWLKLEGNNPAGSVKDRAALSMIVEAEKRGEIKPGDVLIEATSGNTGIALAMIAALKGYRMKLLMPDNMSQERRAAMRAYGAELILVTKEQGMEGARDLALEMANRGEGKLLDQFNNPDNPYAHYTTTGPEIWQQTGGRITHFVSSMGTTGTITGVSRFMREQSKPVTIVGLQPEEGSSIPGIRRWPTEYLPGIFNASLVDEVLDIHQRDAENTMRELAVREGIFCGVSSGGAVAGALRVAKANPDAVVVAIICDRGDRYLSTGVFGEEHFSQGAGI',\n",
       "  'mw': 32664.0},\n",
       " 'b2422': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P16676',\n",
       "  'gene': 'b2422',\n",
       "  'locus': ['b2422', 'JW2415'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + H2O + sulfate(out) = ADP + H(+) + phosphate + sulfate(in)',\n",
       "     '7.3.2.3'),\n",
       "    ('ATP + H2O + thiosulfate(out) = ADP + H(+) + phosphate + thiosulfate(in)',\n",
       "     '7.3.2.3')],\n",
       "   'Function': 'Part of the ABC transporter complex CysAWTP involved in sulfate/thiosulfate import. Responsible for energy coupling to the transport system'},\n",
       "  'seq': 'MSIEIANIKKSFGRTQVLNDISLDIPSGQMVALLGPSGSGKTTLLRIIAGLEHQTSGHIRFHGTDVSRLHARDRKVGFVFQHYALFRHMTVFDNIAFGLTVLPRRERPNAAAIKAKVTKLLEMVQLAHLADRYPAQLSGGQKQRVALARALAVEPQILLLDEPFGALDAQVRKELRRWLRQLHEELKFTSVFVTHDQEEATEVADRVVVMSQGNIEQADAPDQVWREPATRFVLEFMGEVNRLQGTIRGGQFHVGAHRWPLGYTPAYQGPVDLFLRPWEVDISRRTSLDSPLPVQVLEASPKGHYTQLVVQPLGWYNEPLTVVMHGDDAPQRGERLFVGLQHARLYNGDERIETRDEELALAQSA',\n",
       "  'mw': 41059.0},\n",
       " 'b2423': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AEB0',\n",
       "  'gene': 'b2423',\n",
       "  'locus': ['b2423', 'JW2416'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the ABC transporter complex CysAWTP (TC 3.A.1.6.1) involved in sulfate/thiosulfate import. Probably responsible for the translocation of the substrate across the membrane'},\n",
       "  'seq': 'MAEVTQLKRYDARPINWGKWFLIGIGMLVSAFILLVPMIYIFVQAFSKGLMPVLQNLADPDMLHAIWLTVMIALIAVPVNLVFGILLAWLVTRFNFPGRQLLLTLLDIPFAVSPVVAGLVYLLFYGSNGPLGGWLDEHNLQIMFSWPGMVLVTIFVTCPFVVRELVPVMLSQGSQEDEAAILLGASGWQMFRRVTLPNIRWALLYGVVLTNARAIGEFGAVSVVSGSIRGETLSLPLQIELLEQDYNTVGSFTAAALLTLMAIITLFLKSMLQWRLENQEKRAQQEEHHEH',\n",
       "  'mw': 32538.0},\n",
       " 'b2424': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P16701',\n",
       "  'gene': 'b2424',\n",
       "  'locus': ['b2424', 'JW2417'],\n",
       "  'synonyms': 'cysT',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the ABC transporter complex CysAWTP (TC 3.A.1.6.1) involved in sulfate/thiosulfate import. Probably responsible for the translocation of the substrate across the membrane'},\n",
       "  'seq': 'MFAVSSRRVLPGFTLSLGTSLLFVCLILLLPLSALVMQLAQMSWAQYWEVITNPQVVAAYKVTLLSAFVASIFNGVFGLLMAWILTRYRFPGRTLLDALMDLPFALPTAVAGLTLASLFSVNGFYGEWLAKFDIKVTYTWLGIAVAMAFTSIPFVVRTVQPVLEELGPEYEEAAETLGATRWQSFCKVVLPELSPALVAGVALSFTRSLGEFGAVIFIAGNIAWKTEVTSLMIFVRLQEFDYPAASAIASVILAASLLLLFSINTLQSRFGRRVVGH',\n",
       "  'mw': 30292.0},\n",
       " 'b2425': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P16700',\n",
       "  'gene': 'b2425',\n",
       "  'locus': ['b2425', 'JW2418'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the ABC transporter complex CysAWTP (TC 3.A.1.6.1) involved in sulfate/thiosulfate import. This protein specifically binds thiosulfate and is involved in its transmembrane transport'},\n",
       "  'seq': 'MAVNLLKKNSLALVASLLLAGHVQATELLNSSYDVSRELFAALNPPFEQQWAKDNGGDKLTIKQSHAGSSKQALAILQGLKADVVTYNQVTDVQILHDKGKLIPADWQSRLPNNSSPFYSTMGFLVRKGNPKNIHDWNDLVRSDVKLIFPNPKTSGNARYTYLAAWGAADKADGGDKGKTEQFMTQFLKNVEVFDTGGRGATTTFAERGLGDVLISFESEVNNIRKQYEAQGFEVVIPKTNILAEFPVAWVDKNVQANGTEKAAKAYLNWLYSPQAQTIITDYYYRVNNPEVMDKLKDKFPQTELFRVEDKFGSWPEVMKTHFTSGGELDKLLAAGRN',\n",
       "  'mw': 37615.0},\n",
       " 'b2429': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77272',\n",
       "  'gene': 'b2429',\n",
       "  'locus': ['b2429', 'JW2422'],\n",
       "  'synonyms': 'yfeV',\n",
       "  'properties': {'Catalytic Activity': [('N(pros)-phospho-L-histidyl-[protein] + N-acetyl-beta-D-muramate(out) = L-histidyl-[protein] + N-acetyl-beta-D-muramate 6-phosphate(in)',\n",
       "     '2.7.1.192')],\n",
       "   'Function': 'The phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS), a major carbohydrate active transport system, catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane. This system is involved in N-acetylmuramic acid (MurNAc) transport, yielding cytoplasmic MurNAc-6-P. Is responsible for growth on MurNAc as the sole source of carbon and energy. Is also able to take up anhydro-N-acetylmuramic acid (anhMurNAc), but cannot phosphorylate the carbon 6, probably because of the 1,6-anhydro ring'},\n",
       "  'seq': 'MAKEISSELLNTILTRVGGPGNIASCGNCMTRLRLGVHDSSLVDPNIKTLEGVKGVILTSDQVQVVFGPGKAHRAAKAMSELLGEAPVQDAAEIAAQNKRQLKAKQTSGVQQFLAKFATIFTPLIPGFIAAGLLLGIATLIATVMHVPADAQGTLPDALNFMKVFSKGLFTFLVILVGYNAAQAFGGTGVNGAIIAALFLLGYNPAATTGYYAGFHDFFGLPIDPRGNIIGVLIAAWACARIEGMVRRFMPDDLDMLLTSLITLLITATLAYLIIMPLGGWLFEGMSWLFMHLNSNPFGCAVLAGLFLIAVVFGVHQGFIPVYLALMDSQGFNSLFPILSMAGAGQVGAALALYWRAQPHSALRSQVRGAIIPGLLGVGEPLIYGVTLPRMKPFVTACLGGAAGGLFIGLIAWWGLPMGLNSAFGPSGLVALPLMTSAQGILPAMAVYAGGILVAWVCGFIFTTLFGCRNVNLD',\n",
       "  'mw': 49802.0},\n",
       " 'b2435': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P36548',\n",
       "  'gene': 'b2435',\n",
       "  'locus': ['b2435', 'JW2428'],\n",
       "  'synonyms': 'yfeE',\n",
       "  'properties': {'Catalytic Activity': [('Hydrolyzes the link between N-acetylmuramoyl residues and L-amino acid residues in certain cell-wall glycopeptides.',\n",
       "     '3.5.1.28')],\n",
       "   'Function': 'Cell-wall hydrolase involved in septum cleavage during cell division. Can also act as powerful autolysin in the presence of murein synthesis inhibitors'},\n",
       "  'seq': 'MSTFKPLKTLTSRRQVLKAGLAALTLSGMSQAIAKDELLKTSNGHSKPKAKKSGGKRVVVLDPGHGGIDTGAIGRNGSKEKHVVLAIAKNVRSILRNHGIDARLTRSGDTFIPLYDRVEIAHKHGADLFMSIHADGFTNPKAAGASVFALSNRGASSAMAKYLSERENRADEVAGKKATDKDHLLQQVLFDLVQTDTIKNSLTLGSHILKKIKPVHKLHSRNTEQAAFVVLKSPSVPSVLVETSFITNPEEERLLGTAAFRQKIATAIAEGVISYFHWFDNQKAHSKKR',\n",
       "  'mw': 31412.0},\n",
       " 'b2436': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P36553',\n",
       "  'gene': 'b2436',\n",
       "  'locus': ['b2436', 'JW2429'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('coproporphyrinogen III + 2 H(+) + O2 = 2 CO2 + 2 H2O + protoporphyrinogen IX',\n",
       "     '1.3.3.3')],\n",
       "   'Function': 'Involved in the heme biosynthesis. Catalyzes the aerobic oxidative decarboxylation of propionate groups of rings A and B of coproporphyrinogen-III to yield the vinyl groups in protoporphyrinogen-IX',\n",
       "   'Regulation Activity': 'Inhibited by protoporphyrinogen-IX, by metal chelator (EGTA) and by low concentrations of heavy metal ions',\n",
       "   'Pathway': 'Porphyrin-containing compound metabolism; protoporphyrin-IX biosynthesis; protoporphyrinogen-IX from coproporphyrinogen-III (O2 route): step 1/1'},\n",
       "  'seq': 'MKPDAHQVKQFLLNLQDTICQQLTAVDGAEFVEDSWQREAGGGGRSRVLRNGGVFEQAGVNFSHVHGEAMPASATAHRPELAGRSFEAMGVSLVVHPHNPYVPTSHANVRFFIAEKPGADPVWWFGGGFDLTPFYGFEEDAIHWHRTARDLCLPFGEDVYPRYKKWCDEYFYLKHRNEQRGIGGLFFDDLNTPDFDRCFAFMQAVGKGYTDAYLPIVERRKAMAYGERERNFQLYRRGRYVEFNLVWDRGTLFGLQTGGRTESILMSMPPLVRWEYDYQPKDGSPEAALSEFIKVRDWV',\n",
       "  'mw': 34323.0},\n",
       " 'b2440': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P19636',\n",
       "  'gene': 'b2440',\n",
       "  'locus': ['b2440', 'JW2433'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ethanolamine = acetaldehyde + NH4(+)',\n",
       "     '4.3.1.7')],\n",
       "   'Function': 'Catalyzes the deamination of various vicinal amino-alcohols to oxo compounds (PubMed:19762342). Ethanolamine ammonia-lyase (EAL) allows bacteria to utilize ethanolamine as the sole source of nitrogen and carbon in the presence of external vitamin B12. It is spontaneously inactivated by its substrate and reactivated by EutA (PubMed:15466038). Directly targeted to the BMC. May play a role in bacterial microcompartment (BMC) assembly or maintenance (By similarity)',\n",
       "   'Regulation Activity': 'Inhibited by 5-adeninylpentylcobalamin (AdePeCbl), a cofactor analog (PubMed:19762342). Irreversibly inhibited during catalysis by cleavage of the Co-C bond of the cobalamin coenzyme (Probable) (PubMed:19762342). Reactivated by EutA, which probably involves an ATP-dependent cobalamin exchange (PubMed:15466038)',\n",
       "   'Pathway': 'Amine and polyamine degradation; ethanolamine degradation'},\n",
       "  'seq': 'MDQKQIEEIVRSVMASMGQAAPAPSEAKCATTNCAAPVTSESCALDLGSAEAKAWIGVENPHRADVLTELRRSTVARVCTGRAGPRPRTQALLRFLADHSRSKDTVLKEVPEEWVKAQGLLEVRSEISDKNLYLTRPDMGRRLCAEAVEALKAQCVANPDVQVVISDGLSTDAITVNYEEILPPLMAGLKQAGLKVGTPFFVRYGRVKIEDQIGEILGAKVVILLVGERPGLGQSESLSCYAVYSPRMATTVEADRTCISNIHQGGTPPVEAAAVIVDLAKRMLEQKASGINMTR',\n",
       "  'mw': 31782.0},\n",
       " 'b2441': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AEJ6',\n",
       "  'gene': 'b2441',\n",
       "  'locus': ['b2441', 'JW2434'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ethanolamine = acetaldehyde + NH4(+)',\n",
       "     '4.3.1.7')],\n",
       "   'Function': 'Catalyzes the deamination of various vicinal amino-alcohols to oxo compounds (PubMed:19762342). Allows this organism to utilize ethanolamine as the sole source of nitrogen and carbon in the presence of external vitamin B12. It is spontaneously inactivated by its substrate and reactivated by EutA (PubMed:15466038)',\n",
       "   'Regulation Activity': 'Inhibited by 5-adeninylpentylcobalamin (AdePeCbl), a cofactor analog (PubMed:19762342). Irreversibly inhibited during catalysis by cleavage of the Co-C bond of the cobalamin coenzyme (Probable) (PubMed:19762342). Reactivated by EutA, which probably involves an ATP-dependent cobalamin exchange (PubMed:15466038)',\n",
       "   'Pathway': 'Amine and polyamine degradation; ethanolamine degradation'},\n",
       "  'seq': 'MKLKTTLFGNVYQFKDVKEVLAKANELRSGDVLAGVAAASSQERVAAKQVLSEMTVADIRNNPVIAYEDDCVTRLIQDDVNETAYNQIKNWSISELREYVLSDETSVDDIAFTRKGLTSEVVAAVAKICSNADLIYGAKKMPVIKKANTTIGIPGTFSARLQPNDTRDDVQSIAAQIYEGLSFGVGDAVIGVNPVTDDVENLSRVLDTIYGVIDKFNIPTQGCVLAHVTTQIEAIRRGAPGGLIFQSICGSEKGLKEFGVELAMLDEARAVGAEFNRIAGENCLYFETGQGSALSAGANFGADQVTMEARNYGLARHYDPFIVNTVVGFIGPEYLYNDRQIIRAGLEDHFMGKLSGISMGCDCCYTNHADADQNLNENLMILLATAGCNYIMGMPLGDDIMLNYQTTAFHDTATVRQLLNLRPSPEFERWLESMGIMANGRLTKRAGDPSLFF',\n",
       "  'mw': 49403.0},\n",
       " 'b2458': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77218',\n",
       "  'gene': 'b2458',\n",
       "  'locus': ['b2458', 'JW2442'],\n",
       "  'synonyms': 'eutI ypfA',\n",
       "  'properties': {'Catalytic Activity': [('acetyl-CoA + phosphate = acetyl phosphate + CoA',\n",
       "     '2.3.1.8')],\n",
       "   'Function': 'When expressed independently of the eut operon it can restore growth to a double pta-acs deletion mutant',\n",
       "   'Regulation Activity': 'Activated by pyruvate, unaffected by NADH, ATP and PEP, unlike Pta',\n",
       "   'Pathway': 'Amine and polyamine degradation; ethanolamine degradation'},\n",
       "  'seq': 'MIIERCRELALRAPARVVFPDALDQRVLKAAQYLHQQGLATPILVANPFELRQFALSHGVAMDGLQVIDPHGNLAMREEFAHRWLARAGEKTPPDALEKLTDPLMFAAAMVSAGKADVCIAGNLSSTANVLRAGLRIIGLQPGCKTLSSIFLMLPQYSGPALGFADCSVVPQPTAAQLADIALASAETWRAITGEEPRVAMLSFSSNGSARHPCVANVQQATEIVRERAPKLVVDGELQFDAAFVPEVAAQKAPASPLQGKANVMVFPSLEAGNIGYKIAQRLGGYRAVGPLIQGLAAPMHDLSRGCSVQEIIELALVAAVPRQTEVNRESSLQTLVE',\n",
       "  'mw': 36067.0},\n",
       " 'b2463': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P76558',\n",
       "  'gene': 'b2463',\n",
       "  'locus': ['b2463', 'JW2447'],\n",
       "  'synonyms': 'ypfF',\n",
       "  'properties': {'Catalytic Activity': [('(S)-malate + NADP(+) = CO2 + NADPH + pyruvate',\n",
       "     '1.1.1.40'),\n",
       "    ('H(+) + oxaloacetate = CO2 + pyruvate', '1.1.1.40')],\n",
       "   'Function': 'Catalyzes the decarboxylation of malate to pyruvate (PubMed:17557829). In vitro, shows malolactic enzyme activity in the presence of NADPH. However, it is unlikely that this activity is of relevance in E.coli, which produces little NADPH (PubMed:33824210)',\n",
       "   'Regulation Activity': 'Inhibited by 4 mM Mg(2+) and acetyl-CoA, competitively inhibited by fumarate and oxaloacetate. Activated by glutamate and aspartate, glucose-6-phosphate, acetyl-phosphate and 2 mM KCl'},\n",
       "  'seq': 'MDDQLKQSALDFHEFPVPGKIQVSPTKPLATQRDLALAYSPGVAAPCLEIEKDPLKAYKYTARGNLVAVISNGTAVLGLGNIGALAGKPVMEGKGVLFKKFAGIDVFDIEVDELDPDKFIEVVAALEPTFGGINLEDIKAPECFYIEQKLRERMNIPVFHDDQHGTAIISTAAILNGLRVVEKNISDVRMVVSGAGAAAIACMNLLVALGLQKHNIVVCDSKGVIYQGREPNMAETKAAYAVVDDGKRTLDDVIEGADIFLGCSGPKVLTQEMVKKMARAPMILALANPEPEILPPLAKEVRPDAIICTGRSDYPNQVNNVLCFPFIFRGALDVGATAINEEMKLAAVRAIAELAHAEQSEVVASAYGDQDLSFGPEYIIPKPFDPRLIVKIAPAVAKAAMESGVATRPIADFDVYIDKLTEFVYKTNLFMKPIFSQARKAPKRVVLPEGEEARVLHATQELVTLGLAKPILIGRPNVIEMRIQKLGLQIKAGVDFEIVNNESDPRFKEYWTEYFQIMKRRGVTQEQAQRALISNPTVIGAIMVQRGEADAMICGTVGDYHEHFSVVKNVFGYRDGVHTAGAMNALLLPSGNTFIADTYVNDEPDAEELAEITLMAAETVRRFGIEPRVALLSHSNFGSSDCPSSSKMRQALELVRERAPELMIDGEMHGDAALVEAIRNDRMPDSSLKGSANILVMPNMEAARISYNLLRVSSSEGVTVGPVLMGVAKPVHVLTPIASVRRIVNMVALAVVEAQTQPL',\n",
       "  'mw': 82417.0},\n",
       " 'b2464': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A867',\n",
       "  'gene': 'b2464',\n",
       "  'locus': ['b2464', 'JW2448'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('D-glyceraldehyde 3-phosphate + D-sedoheptulose 7-phosphate = beta-D-fructose 6-phosphate + D-erythrose 4-phosphate',\n",
       "     '2.2.1.2')],\n",
       "   'Function': 'Transaldolase is important for the balance of metabolites in the pentose-phosphate pathway',\n",
       "   'Pathway': 'Carbohydrate degradation; pentose phosphate pathway; D-glyceraldehyde 3-phosphate and beta-D-fructose 6-phosphate from D-ribose 5-phosphate and D-xylulose 5-phosphate (non-oxidative stage): step 2/3'},\n",
       "  'seq': 'MNELDGIKQFTTVVADSGDIESIRHYHPQDATTNPSLLLKAAGLSQYEHLIDDAIAWGKKNGKTQEQQVVAACDKLAVNFGAEILKIVPGRVSTEVDARLSFDKEKSIEKARHLVDLYQQQGVEKSRILIKLASTWEGIRAAEELEKEGINCNLTLLFSFAQARACAEAGVFLISPFVGRIYDWYQARKPMDPYVVEEDPGVKSVRNIYDYYKQHHYETIVMGASFRRTEQILALTGCDRLTIAPNLLKELQEKVSPVVRKLIPPSQTFPRPAPMSEAEFRWEHNQDAMAVEKLSEGIRLFAVDQRKLEDLLAAKL',\n",
       "  'mw': 35659.0},\n",
       " 'b2465': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P33570',\n",
       "  'gene': 'b2465',\n",
       "  'locus': ['b2465', 'JW2449'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('D-glyceraldehyde 3-phosphate + D-sedoheptulose 7-phosphate = aldehydo-D-ribose 5-phosphate + D-xylulose 5-phosphate',\n",
       "     '2.2.1.1')],\n",
       "   'Function': 'Catalyzes the reversible transfer of a two-carbon ketol group from sedoheptulose-7-phosphate to glyceraldehyde-3-phosphate, producing xylulose-5-phosphate and ribose-5-phosphate. Catalyzes the transfer of a two-carbon ketol group from a ketose donor to an aldose acceptor, via a covalent intermediate with the cofactor thiamine pyrophosphate'},\n",
       "  'seq': 'MSRKDLANAIRALSMDAVQKANSGHPGAPMGMADIAEVLWNDFLKHNPTDPTWYDRDRFILSNGHASMLLYSLLHLTGYDLPLEELKNFRQLHSKTPGHPEIGYTPGVETTTGPLGQGLANAVGLAIAERTLAAQFNQPDHEIVDHFTYVFMGDGCLMEGISHEVCSLAGTLGLGKLIGFYDHNGISIDGETEGWFTDDTAKRFEAYHWHVIHEIDGHDPQAVKEAILEAQSVKDKPSLIICRTVIGFGSPNKAGKEEAHGAPLGEEEVALARQKLGWHHPPFEIPKEIYHAWDAREKGEKAQQSWNEKFAAYKKAHPQLAEEFTRRMSGGLPKDWEKTTQKYINELQANPAKIATRKASQNTLNAYGPMLPELLGGSADLAPSNLTIWKGSVSLKEDPAGNYIHYGVREFGMTAIANGIAHHGGFVPYTATFLMFVEYARNAARMAALMKARQIMVYTHDSIGLGEDGPTHQAVEQLASLRLTPNFSTWRPCDQVEAAVGWKLAVERHNGPTALILSRQNLAQVERTPDQVKEIARGGYVLKDSGGKPDIILIATGSEMEITLQAAEKLAGEGRNVRVVSLPSTDIFDAQDEEYRESVLPSNVAARVAVEAGIADYWYKYVGLKGAIVGMTGYGESAPADKLFPFFGFTAENIVAKAHKVLGVKGA',\n",
       "  'mw': 73043.0},\n",
       " 'b2467': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37128',\n",
       "  'gene': 'b2467',\n",
       "  'locus': ['b2467', 'JW2451'],\n",
       "  'synonyms': 'yffH',\n",
       "  'properties': {'Catalytic Activity': [('GDP-alpha-D-mannose + H2O = alpha-D-mannose 1-phosphate + GMP + 2 H(+)',\n",
       "     ''),\n",
       "    ('GDP-alpha-D-glucose + H2O = alpha-D-glucose 1-phosphate + GMP + 2 H(+)',\n",
       "     '')],\n",
       "   'Function': 'Nucleoside diphosphate sugar hydrolase that hydrolyzes GDP-mannose as its preferred substrate, yielding GMP and mannose-1-phosphate. Can also hydrolyze the pyrophosphate bond of other sugar nucleotides such as IDP-ribose, GDP-glucose, and to a lesser extent, ADP-ribose, ADP-glucose and UDP-glucose. Shows no activity toward Nudix substrates FAD, CDP-ethanolamine, CDP-choline, NAD(+), diadenosine pentaphosphate, GTP, UTP, ATP, or CTP'},\n",
       "  'seq': 'MTQQITLIKDKILSDNYFTLHNITYDLTRKDGEVIRHKREVYDRGNGATILLYNTKKKTVVLIRQFRVATWVNGNESGQLIESCAGLLDNDEPEVCIRKEAIEETGYEVGEVRKLFELYMSPGGVTELIHFFIAEYSDNQRANAGGGVEDEDIEVLELPFSQALEMIKTGEIRDGKTVLLLNYLQTSHLMD',\n",
       "  'mw': 21749.0},\n",
       " 'b2472': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AED7',\n",
       "  'gene': 'b2472',\n",
       "  'locus': ['b2472', 'JW2456'],\n",
       "  'synonyms': 'msgB',\n",
       "  'properties': {'Catalytic Activity': [('H2O + N-succinyl-(2S,6S)-2,6-diaminoheptanedioate = (2S,6S)-2,6-diaminoheptanedioate + succinate',\n",
       "     '3.5.1.18')],\n",
       "   'Function': 'Catalyzes the hydrolysis of N-succinyl-L,L-diaminopimelic acid (SDAP), forming succinate and LL-2,6-diaminoheptanedioate (DAP), an intermediate involved in the bacterial biosynthesis of lysine and meso-diaminopimelic acid, an essential component of bacterial cell walls',\n",
       "   'Pathway': 'Amino-acid biosynthesis; L-lysine biosynthesis via DAP pathway; LL-2,6-diaminopimelate from (S)-tetrahydrodipicolinate (succinylase route): step 3/3'},\n",
       "  'seq': 'MSCPVIELTQQLIRRPSLSPDDAGCQALLIERLQAIGFTVERMDFADTQNFWAWRGQGETLAFAGHTDVVPPGDADRWINPPFEPTIRDGMLFGRGAADMKGSLAAMVVAAERFVAQHPNHTGRLAFLITSDEEASAHNGTVKVVEALMARNERLDYCLVGEPSSIEVVGDVVKNGRRGSLTCNLTIHGVQGHVAYPHLADNPVHRAAPFLNELVAIEWDQGNEFFPATSMQIANIQAGTGSNNVIPGELFVQFNFRFSTELTDEMIKAQVLALLEKHQLRYTVDWWLSGQPFLTARGKLVDAVVNAVEHYNEIKPQLLTTGGTSDGRFIARMGAQVVELGPVNATIHKINECVNAADLQLLARMYQRIMEQLVA',\n",
       "  'mw': 41269.0},\n",
       " 'b2476': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A7D7',\n",
       "  'gene': 'b2476',\n",
       "  'locus': ['b2476', 'JW2461'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('5-amino-1-(5-phospho-D-ribosyl)imidazole-4-carboxylate + ATP + L-aspartate = (2S)-2-[5-amino-1-(5-phospho-beta-D-ribosyl)imidazole-4-carboxamido]succinate + ADP + 2 H(+) + phosphate',\n",
       "     '6.3.2.6')],\n",
       "   'Function': 'Converts 4-carboxy-5-aminoimidazole ribonucleotide (CAIR) to 4-(N-succinylcarboxamide)-5-aminoimidazole ribonucleotide (SAICAR)',\n",
       "   'Pathway': 'Purine metabolism; IMP biosynthesis via de novo pathway; 5-amino-1-(5-phospho-D-ribosyl)imidazole-4-carboxamide from 5-amino-1-(5-phospho-D-ribosyl)imidazole-4-carboxylate: step 1/2'},\n",
       "  'seq': 'MQKQAELYRGKAKTVYSTENPDLLVLEFRNDTSAGDGARIEQFDRKGMVNNKFNYFIMSKLAEAGIPTQMERLLSDTECLVKKLDMVPVECVVRNRAAGSLVKRLGIEEGIELNPPLFDLFLKNDAMHDPMVNESYCETFGWVSKENLARMKELTYKANDVLKKLFDDAGLILVDFKLEFGLYKGEVVLGDEFSPDGSRLWDKETLEKMDKDRFRQSLGGLIEAYEAVARRLGVQLD',\n",
       "  'mw': 26995.0},\n",
       " 'b2478': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A6L2',\n",
       "  'gene': 'b2478',\n",
       "  'locus': ['b2478', 'JW2463'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('L-aspartate 4-semialdehyde + pyruvate = (2S,4S)-4-hydroxy-2,3,4,5-tetrahydrodipicolinate + H(+) + H2O',\n",
       "     '4.3.3.7')],\n",
       "   'Function': 'Catalyzes the condensation of (S)-aspartate-beta-semialdehyde [(S)-ASA] and pyruvate to 4-hydroxy-tetrahydrodipicolinate (HTPA)',\n",
       "   'Regulation Activity': 'Is allosterically regulated by the feedback inhibitor (S)-lysine. Is inhibited by pyruvate analogs such as 3-fluoropyruvate, 2-ketobutyrate, glyoxylate, and beta-hydroxypyruvate. Is not inhibited by its substrate, (S)-ASA',\n",
       "   'Pathway': 'Amino-acid biosynthesis; L-lysine biosynthesis via DAP pathway; (S)-tetrahydrodipicolinate from L-aspartate: step 3/4'},\n",
       "  'seq': 'MFTGSIVAIVTPMDEKGNVCRASLKKLIDYHVASGTSAIVSVGTTGESATLNHDEHADVVMMTLDLADGRIPVIAGTGANATAEAISLTQRFNDSGIVGCLTVTPYYNRPSQEGLYQHFKAIAEHTDLPQILYNVPSRTGCDLLPETVGRLAKVKNIIGIKEATGNLTRVNQIKELVSDDFVLLSGDDASALDFMQLGGHGVISVTANVAARDMAQMCKLAAEGHFAEARVINQRLMPLHNKLFVEPNPIPVKWACKELGLVATDTLRLPMTPITDSGRETVRAALKHAGLL',\n",
       "  'mw': 31270.0},\n",
       " 'b2480': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AE52',\n",
       "  'gene': 'b2480',\n",
       "  'locus': ['b2480', 'JW2465'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('[thioredoxin]-dithiol + a hydroperoxide = [thioredoxin]-disulfide + an alcohol + H2O',\n",
       "     '1.11.1.24')],\n",
       "   'Function': 'Thiol-specific peroxidase that catalyzes the reduction of hydrogen peroxide and organic hydroperoxides to water and alcohols, respectively. Plays a role in cell protection against oxidative stress by detoxifying peroxides and as sensor of hydrogen peroxide-mediated signaling events'},\n",
       "  'seq': 'MNPLKAGDIAPKFSLPDQDGEQVNLTDFQGQRVLVYFYPKAMTPGCTVQACGLRDNMDELKKAGVDVLGISTDKPEKLSRFAEKELLNFTLLSDEDHQVCEQFGVWGEKSFMGKTYDGIHRISFLIDADGKIEHVFDDFKTSNHHDVVLNWLKEHA',\n",
       "  'mw': 17634.0},\n",
       " 'b2481': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P23481',\n",
       "  'gene': 'b2481',\n",
       "  'locus': ['b2481', 'JW2466'],\n",
       "  'synonyms': 'yffE',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Probable electron transfer protein for hydrogenase 4'},\n",
       "  'seq': 'MNRFVVAEPLWCTGCNTCLAACSDVHKTQGLQQHPRLALAKTSTITAPVVCHHCEEAPCLQVCPVNAISQRDDAIQLNESLCIGCKLCAVVCPFGAISASGSRPVNAHAQYVFQAEGSLKDGEENAPTQHALLRWEPGVQTVAVKCDLCDFLPEGPACVRACPNQALRLITGDSLQRQMKEKQRLAASWFANGGEDPLSLTQEQR',\n",
       "  'mw': 22154.0},\n",
       " 'b2482': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P23482',\n",
       "  'gene': 'b2482',\n",
       "  'locus': ['b2482', 'JW2467'],\n",
       "  'synonyms': 'yffF',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Possible component of hydrogenase 4'},\n",
       "  'seq': 'MDALQLLTWSLILYLFASLASLFLLGLDRLAIKLSGITSLVGGVIGIISGITQLHAGVTLVARFAPPFEFADLTLRMDSLSAFMVLVISLLVVVCSLYSLTYMREYEGKGAAAMGFFMNIFIASMVALLVMDNAFWFIVLFEMMSLSSWFLVIARQDKTSINAGMLYFFIAHAGSVLIMIAFLLMGRESGSLDFASFRTLSLSPGLASAVFLLAFFGFGAKAGMMPLHSWLPRAHPAAPSHASALMSGVMVKIGIFGILKVAMDLLAQTGLPLWWGILVMAIGAISALLGVLYALAEQDIKRLLAWSTVENVGIILLAVGVAMVGLSLHDPLLTVVGLLGALFHLLNHALFKGLLFLGAGAIISRLHTHDMEKMGALAKRMPWTAAACLIGCLAISAIPPLNGFISEWYTWQSLFSLSRVEAVALQLAGPIAMVMLAVTGGLAVMCFVKMYGITFCGAPRSTHAEEAQEVPNTMIVAMLLLAALCVLIALSASWLAPKIMHIAHAFTNTPPATVASGIALVPGTFHTQVTPSLLLLLLLAMPLLPGLYWLWCRSRRAAFRRTGDAWACGYGWENAMAPSGNGVMQPLRVVFSALFRLRQQLDPTLRLNKGLAHVTARAQSTEPFWDERVIRPIVSATQRLAKEIQHLQSGDFRLYCLYVVAALVVLLIAIAV',\n",
       "  'mw': 72583.0},\n",
       " 'b2483': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77858',\n",
       "  'gene': 'b2483',\n",
       "  'locus': ['b2483', 'JW2468'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Possible component of hydrogenase 4'},\n",
       "  'seq': 'MRQTLCDGYLVIFALAQAVILLMLTPLFTGISRQIRARMHSRRGPGIWQDYRDIHKLFKRQEVAPTSSGLMFRLMPWVLISSMLVLAMALPLFITVSPFAGGGDLITLIYLLALFRFFFALSGLDTGSPFAGVGASRELTLGILVEPMLILSLLVLALIAGSTHIEMISNTLAMGWNSPLTTVLALLACGFACFIEMGKIPFDVAEAEQELQEGPLTEYSGAGLALAKWGLGLKQVVMASLFVALFLPFGRAQELSLACLLTSLVVTLLKVLLIFVLASIAENTLARGRFLLIHHVTWLGFSLAALAWVFWLTGL',\n",
       "  'mw': 34359.0},\n",
       " 'b2484': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77416',\n",
       "  'gene': 'b2484',\n",
       "  'locus': ['b2484', 'JW2469'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Possible component of hydrogenase 4'},\n",
       "  'seq': 'MENLALTTLLLPFIGALVVSFSPQRRAAEWGVLFAALTTLCMLSLISAFYQADKVAVTLTLVNVGDVALFGLVIDRVSTLILFVVVFLGLLVTIYSTGYLTDKNREHPHNGTNRYYAFLLVFIGAMAGLVLSSTLLGQLLFFEITGGCSWALISYYQSDKAQRSALKALLITHIGSLGLYLAAATLFLQTGTFALSAMSELHGDARYLVYGGILFAAWGKSAQLPMQAWLPDAMEAPTPISAYLHAASMVKVGVYIFARAIIDGGNIPHVIGGVGMVMALVTILYGFLMYLPQQDMKRLLAWSTITQLGWMFFGLSLSIFGSRLALEGSIAYIVNHAFAKSLFFLVAGALSYSCGTRLLPRLRGVLHTLPLPGVGFCVAALAITGVPPFNGFFSKFPLFAAGFALSVEYWILLPAMILLMIESVASFAWFIRWFGRVVPGKPSEAVADAAPLPGSMRLVLIVLIVMSLISSVIAATWLQ',\n",
       "  'mw': 51755.0},\n",
       " 'b2485': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AEW1',\n",
       "  'gene': 'b2485',\n",
       "  'locus': ['b2485', 'JW2470'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Possible component of hydrogenase 4'},\n",
       "  'seq': 'MTGSMIVNNLAGLMMLTSLFVISVKSYRLSCGFYACQSLVLVSIFATLSCLFAAEQLLIWSASAFITKVLLVPLIMTYAARNIPQNIPEKALFGPAMMALLAALIVLLCAFVVQPVKLPMATGLKPALAVALGHFLLGLLCIVSQRNILRQIFGYCLMENGSHLVLALLAWRAPELVEIGIATDAIFAVIVMVLLARKIWRTHGTLDVNNLTALKG',\n",
       "  'mw': 23361.0},\n",
       " 'b2486': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77437',\n",
       "  'gene': 'b2486',\n",
       "  'locus': ['b2486', 'JW2471'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Possible component of hydrogenase 4'},\n",
       "  'seq': 'MSYSVMFALLLLTPLLFSLLCFACRKRRLSATRTVTVLHSLGITLLLILALWVVQTAADAGEIFAAGLWLHIDGLGGLFLAILGVIGFLTGIYSIGYMRHEVAHGELSPVTLCDYYGFFHLFLFTMLLVVTSNNLIVMWAAIEATTLSSAFLVGIYGQRSSLEAAWKYIIICTVGVAFGLFGTVLVYANAASVMPQAEMAIFWSEVLKQSSLLDPTLMLLAFVFLLIGFGTKTGLFPMHAWLPDAHSEAPSPVSALLSAVLLNCALLVLIRYYIIICQAIGSDFPNRLLLIFGMLSVAVAAFFILVQRDIKRLLAYSSVENMGLVAVELGIGGPLGIFAALLHILNHSLAKTLLFCGSGNVLLKYGTRDLNVVCGMLKIMPFTAVLFGGGALALAGMPPFNIFLSEFMTITAGLARNHLLIIVLLLLLLTLVLAGLVRMAARVLMAKPPQAVNRGDLGWLTTSPMVILLVMMLAMGTHIPQPVIRILAGASTIVLSGTHDLPAQRSTWHDFLPSGTASVSEKHSER',\n",
       "  'mw': 56767.0},\n",
       " 'b2487': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77329',\n",
       "  'gene': 'b2487',\n",
       "  'locus': ['b2487', 'JW2472'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Possible component of hydrogenase 4'},\n",
       "  'seq': 'MNVNSSSNRGEAILAALKTQFPGAVLDEERQTPEQVTITVKINLLPDVVQYLYYQHDGWLPVLFGNDERTLNGHYAVYYALSMEGAEKCWIVVKALVDADSREFPSVTPRVPAAVWGEREIRDMYGLIPVGLPDQRRLVLPDDWPEDMHPLRKDAMDYRLRPEPTTDSETYPFINEGNSDARVIPVGPLHITSDEPGHFRLFVDGEQIVDADYRLFYVHRGMEKLAETRMGYNEVTFLSDRVCGICGFAHSVAYTNSVENALGIEVPQRAHTIRSILLEVERLHSHLLNLGLSCHFVGFDTGFMQFFRVREKSMTMAELLIGSRKTYGLNLIGGVRRDILKEQRLQTLKLVREMRADVSELVEMLLATPNMEQRTQGIGILDRQIARDLRFDHPYADYGNIPKTLFTFTGGDVFSRVMVRVKETFDSLAMLEFALDNMPDTPLLTEGFSYKPHAFALGFVEAPRGEDVHWSMLGDNQKLFRWRCRAATYANWPVLRYMLRGNTVSDAPLIIGSLDPCYSCTDRVTLVDVRKRQSKTVPYKEIERYGIDRNRSPLK',\n",
       "  'mw': 63383.0},\n",
       " 'b2488': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77423',\n",
       "  'gene': 'b2488',\n",
       "  'locus': ['b2488', 'JW2473'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Probable electron transfer protein for hydrogenase 4'},\n",
       "  'seq': 'MLKLLKTIMRAGTATVKYPFAPLEVSPGFRGKPDLMPSQCIACGACACACPANALTIQTDDQQNSRTWQLYLGRCIYCGRCEEVCPTRAIQLTNNFELTVTNKADLYTRATFHLQRCSRCERPFAPQKTIALAAELLAQQQNAPQNREMLWAQASVCPECKQRATLINDDTDVLLVAKEQL',\n",
       "  'mw': 20159.0},\n",
       " 'b2489': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77668',\n",
       "  'gene': 'b2489',\n",
       "  'locus': ['b2489', 'JW5805'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Possible component of hydrogenase 4'},\n",
       "  'seq': 'MSPVLTQHVSQPITLDEQTQKMKRHLLQDIRRSAYVYRVDCGGCNACEIEIFAAITPVFDAERFGIKVVSSPRHADILLFTGAVTRAMRMPALRAYESAPDHKICVSYGACGVGGGIFHDLYSVWGGSDTIVPIDVWIPGCPPTPAATIHGFAVALGLLQQKIHAVDYRDPTGVTMQPLWPQIPPSQRIAIEREARRLAGYRQGREICDRLLRHLSDDPTGNRVNTWLRDADDPRLNSIVQQLFRVLRGLHD',\n",
       "  'mw': 28101.0},\n",
       " 'b2490': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77453',\n",
       "  'gene': 'b2490',\n",
       "  'locus': ['b2490', 'JW2475'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Possible component of hydrogenase 4'},\n",
       "  'seq': 'MTEECGEIVFWTLRKKFVASSDEMPEHSSQVMYYSLAIGHHVGVIDCLNVAFRCPLTEYEDWLALVEEEQARRKMLGVMTFGEIVIDASHTALLTRAFAPLADDATSVWQARSIQFIHLLDEIVQEPAIYLMARKIA',\n",
       "  'mw': 15577.0},\n",
       " 'b2492': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77733',\n",
       "  'gene': 'b2492',\n",
       "  'locus': ['b2492', 'JW2477'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('formate(in) = formate(out)', '')],\n",
       "   'Function': 'Involved in the bidirectional transport of formate during mixed-acid fermentation',\n",
       "   'Regulation Activity': 'The direction of formate translocation depends on external pH and electron donor source'},\n",
       "  'seq': 'MRNKLSFDLQLSARKAAIAERIAAHKIARSKVSVFLMAMSAGVFMAIGFTFYLSVIADAPSSQALTHLVGGLCFTLGFILLAVCGTSLFTSSVMTVMAKSRGVISWRTWLINALLVACGNLAGIACFSLLIWFSGLVMSENAMWGVAVLHCAEGKMHHTFTESVSLGIMCNLMVCLALWMSYCGRSLCDKIVAMILPITLFVASGFEHCIANLFVIPFAIAIRHFAPPPFWQLAHSSADNFPALTVSHFITANLLPVMLGNIIGGAVLVSMCYRAIYLRQEP',\n",
       "  'mw': 30565.0},\n",
       " 'b2497': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AGM7',\n",
       "  'gene': 'b2497',\n",
       "  'locus': ['b2497', 'JW2482'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('H(+)(in) + uracil(in) = H(+)(out) + uracil(out)',\n",
       "     '')],\n",
       "   'Function': 'Transport of uracil in the cell'},\n",
       "  'seq': 'MTRRAIGVSERPPLLQTIPLSLQHLFAMFGATVLVPVLFHINPATVLLFNGIGTLLYLFICKGKIPAYLGSSFAFISPVLLLLPLGYEVALGGFIMCGVLFCLVSFIVKKAGTGWLDVLFPPAAMGAIVAVIGLELAGVAAGMAGLLPAEGQTPDSKTIIISITTLAVTVLGSVLFRGFLAIIPILIGVLVGYALSFAMGIVDTTPIINAHWFALPTLYTPRFEWFAILTILPAALVVIAEHVGHLVVTANIVKKDLLRDPGLHRSMFANGLSTVISGFFGSTPNTTYGENIGVMAITRVYSTWVIGGAAIFAILLSCVGKLAAAIQMIPLPVMGGVSLLLYGVIGASGIRVLIESKVDYNKAQNLILTSVILIIGVSGAKVNIGAAELKGMALATIVGIGLSLIFKLISVLRPEEVVLDAEDADITDK',\n",
       "  'mw': 45060.0},\n",
       " 'b2498': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A8F0',\n",
       "  'gene': 'b2498',\n",
       "  'locus': ['b2498', 'JW2483'],\n",
       "  'synonyms': 'uraP',\n",
       "  'properties': {'Catalytic Activity': [('diphosphate + UMP = 5-phospho-alpha-D-ribose 1-diphosphate + uracil',\n",
       "     '2.4.2.9')],\n",
       "   'Function': 'Catalyzes the conversion of uracil and 5-phospho-alpha-D-ribose 1-diphosphate (PRPP) to UMP and diphosphate',\n",
       "   'Regulation Activity': 'Allosterically activated by GTP',\n",
       "   'Pathway': 'Pyrimidine metabolism; UMP biosynthesis via salvage pathway; UMP from uracil: step 1/1'},\n",
       "  'seq': 'MKIVEVKHPLVKHKLGLMREQDISTKRFRELASEVGSLLTYEATADLETEKVTIEGWNGPVEIDQIKGKKITVVPILRAGLGMMDGVLENVPSARISVVGMYRNEETLEPVPYFQKLVSNIDERMALIVDPMLATGGSVIATIDLLKKAGCSSIKVLVLVAAPEGIAALEKAHPDVELYTASIDQGLNEHGYIIPGLGDAGDKIFGTK',\n",
       "  'mw': 22533.0},\n",
       " 'b2499': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P08178',\n",
       "  'gene': 'b2499',\n",
       "  'locus': ['b2499', 'JW2484'],\n",
       "  'synonyms': 'purG',\n",
       "  'properties': {'Catalytic Activity': [('2-formamido-N(1)-(5-O-phospho-beta-D-ribosyl)acetamidine + ATP = 5-amino-1-(5-phospho-beta-D-ribosyl)imidazole + ADP + H(+) + phosphate',\n",
       "     '6.3.3.1')],\n",
       "   'Pathway': 'Purine metabolism; IMP biosynthesis via de novo pathway; 5-amino-1-(5-phospho-D-ribosyl)imidazole from N(2)-formyl-N(1)-(5-phospho-D-ribosyl)glycinamide: step 2/2'},\n",
       "  'seq': 'MTDKTSLSYKDAGVDIDAGNALVGRIKGVVKKTRRPEVMGGLGGFGALCALPQKYREPVLVSGTDGVGTKLRLAMDLKRHDTIGIDLVAMCVNDLVVQGAEPLFFLDYYATGKLDVDTASAVISGIAEGCLQSGCSLVGGETAEMPGMYHGEDYDVAGFCVGVVEKSEIIDGSKVSDGDVLIALGSSGPHSNGYSLVRKILEVSGCDPQTTELDGKPLADHLLAPTRIYVKSVLELIEKVDVHAIAHLTGGGFWENIPRVLPDNTQAVIDESSWQWPEVFNWLQTAGNVEHHEMYRTFNCGVGMIIALPAPEVDKALALLNANGENAWKIGIIKASDSEQRVVIE',\n",
       "  'mw': 36854.0},\n",
       " 'b2500': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P08179',\n",
       "  'gene': 'b2500',\n",
       "  'locus': ['b2500', 'JW2485'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(6S)-10-formyltetrahydrofolate + N(1)-(5-phospho-beta-D-ribosyl)glycinamide = (6S)-5,6,7,8-tetrahydrofolate + H(+) + N(2)-formyl-N(1)-(5-phospho-beta-D-ribosyl)glycinamide',\n",
       "     '2.1.2.2')],\n",
       "   'Function': 'Catalyzes the transfer of a formyl group from 10-formyltetrahydrofolate to 5-phospho-ribosyl-glycinamide (GAR), producing 5-phospho-ribosyl-N-formylglycinamide (FGAR) and tetrahydrofolate',\n",
       "   'Regulation Activity': 'Inhibited by N10-(bromoacetyl)-5,8-dideazafolate',\n",
       "   'Pathway': 'Purine metabolism; IMP biosynthesis via de novo pathway; N(2)-formyl-N(1)-(5-phospho-D-ribosyl)glycinamide from N(1)-(5-phospho-D-ribosyl)glycinamide (10-formyl THF route): step 1/1'},\n",
       "  'seq': 'MNIVVLISGNGSNLQAIIDACKTNKIKGTVRAVFSNKADAFGLERARQAGIATHTLIASAFDSREAYDRELIHEIDMYAPDVVVLAGFMRILSPAFVSHYAGRLLNIHPSLLPKYPGLHTHRQALENGDEEHGTSVHFVTDELDGGPVILQAKVPVFAGDSEDDITARVQTQEHAIYPLVISWFADGRLKMHENAAWLDGQRLPPQGYAADE',\n",
       "  'mw': 23238.0},\n",
       " 'b2501': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A7B1',\n",
       "  'gene': 'b2501',\n",
       "  'locus': ['b2501', 'JW2486'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('[phosphate](n) + ATP = [phosphate](n+1) + ADP',\n",
       "     '2.7.4.1')],\n",
       "   'Function': \"Catalyzes the reversible transfer of the terminal phosphate of ATP to form a long-chain polyphosphate (polyP). Can form linear polymers of orthophosphate with chain lengths up to 1000 or more. Can use GTP instead of ATP, but the efficiency of GTP is 5% that of ATP. Also exhibits several other enzymatic activities, which include: ATP synthesis from polyP in the presence of excess ADP, general nucleoside-diphosphate kinase activity, linear guanosine 5'-tetraphosphate (ppppG) synthesis and autophosphorylation\"},\n",
       "  'seq': 'MGQEKLYIEKELSWLSFNERVLQEAADKSNPLIERMRFLGIYSNNLDEFYKVRFAELKRRIIISEEQGSNSHSRHLLGKIQSRVLKADQEFDGLYNELLLEMARNQIFLINERQLSVNQQNWLRHYFKQYLRQHITPILINPDTDLVQFLKDDYTYLAVEIIRGDTIRYALLEIPSDKVPRFVNLPPEAPRRRKPMILLDNILRYCLDDIFKGFFDYDALNAYSMKMTRDAEYDLVHEMEASLMELMSSSLKQRLTAEPVRFVYQRDMPNALVEVLREKLTISRYDSIVPGGRYHNFKDFINFPNVGKANLVNKPLPRLRHIWFDKAQFRNGFDAIRERDVLLYYPYHTFEHVLELLRQASFDPSVLAIKINIYRVAKDSRIIDSMIHAAHNGKKVTVVVELQARFDEEANIHWAKRLTEAGVHVIFSAPGLKIHAKLFLISRKENGEVVRYAHIGTGNFNEKTARLYTDYSLLTADARITNEVRRVFNFIENPYRPVTFDYLMVSPQNSRRLLYEMVDREIANAQQGLPSGITLKLNNLVDKGLVDRLYAASSSGVPVNLLVRGMCSLIPNLEGISDNIRAISIVDRYLEHDRVYIFENGGDKKVYLSSADWMTRNIDYRIEVATPLLDPRLKQRVLDIIDILFSDTVKARYIDKELSNRYVPRGNRRKVRAQLAIYDYIKSLEQPE',\n",
       "  'mw': 80432.0},\n",
       " 'b2502': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AFL6',\n",
       "  'gene': 'b2502',\n",
       "  'locus': ['b2502', 'JW2487'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('[phosphate](n) + H2O = [phosphate](n-1) + H(+) + phosphate',\n",
       "     '3.6.1.11')],\n",
       "   'Function': 'Degradation of inorganic polyphosphates (polyP). Releases orthophosphate processively from the ends of the polyP chain. Has a strong preference for long-chain polyphosphates and has only weak affinity for smaller size polyP of about 15 residues',\n",
       "   'Regulation Activity': 'Activity is redox-regulated. Highly sensitive to inactivation by oxidation (PubMed:24560923). Strongly stimulated by potassium salts (PubMed:8380170)'},\n",
       "  'seq': 'MPIHDKSPRPQEFAAVDLGSNSFHMVIARVVDGAMQIIGRLKQRVHLADGLGPDNMLSEEAMTRGLNCLSLFAERLQGFSPASVCIVGTHTLRQALNATDFLKRAEKVIPYPIEIISGNEEARLIFMGVEHTQPEKGRKLVIDIGGGSTELVIGENFEPILVESRRMGCVSFAQLYFPGGVINKENFQRARMAAAQKLETLTWQFRIQGWNVAMGASGTIKAAHEVLMEMGEKDGIITPERLEKLVKEVLRHRNFASLSLPGLSEERKTVFVPGLAILCGVFDALAIRELRLSDGALREGVLYEMEGRFRHQDVRSRTASSLANQYHIDSEQARRVLDTTMQMYEQWREQQPKLAHPQLEALLRWAAMLHEVGLNINHSGLHRHSAYILQNSDLPGFNQEQQLMMATLVRYHRKAIKLDDLPRFTLFKKKQFLPLIQLLRLGVLLNNQRQATTTPPTLTLITDDSHWTLRFPHDWFSQNALVLLDLEKEQEYWEGVAGWRLKIEEESTPEIAA',\n",
       "  'mw': 58136.0},\n",
       " 'b2507': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P04079',\n",
       "  'gene': 'b2507',\n",
       "  'locus': ['b2507', 'JW2491'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + H2O + L-glutamine + XMP = AMP + diphosphate + GMP + 2 H(+) + L-glutamate',\n",
       "     '6.3.5.2')],\n",
       "   'Function': 'Catalyzes the synthesis of GMP from XMP',\n",
       "   'Regulation Activity': \"Inhibited by 2-fluoroinosine 5'-monophosphate (F-IMP) and by N(2)-hydroxyguanosine 5'-monophosphate (N(2)-OH-GMP)\",\n",
       "   'Pathway': 'Purine metabolism; GMP biosynthesis; GMP from XMP (L-Gln route): step 1/1'},\n",
       "  'seq': 'MTENIHKHRILILDFGSQYTQLVARRVRELGVYCELWAWDVTEAQIRDFNPSGIILSGGPESTTEENSPRAPQYVFEAGVPVFGVCYGMQTMAMQLGGHVEASNEREFGYAQVEVVNDSALVRGIEDALTADGKPLLDVWMSHGDKVTAIPSDFITVASTESCPFAIMANEEKRFYGVQFHPEVTHTRQGMRMLERFVRDICQCEALWTPAKIIDDAVARIREQVGDDKVILGLSGGVDSSVTAMLLHRAIGKNLTCVFVDNGLLRLNEAEQVLDMFGDHFGLNIVHVPAEDRFLSALAGENDPEAKRKIIGRVFVEVFDEEALKLEDVKWLAQGTIYPDVIESAASATGKAHVIKSHHNVGGLPKEMKMGLVEPLKELFKDEVRKIGLELGLPYDMLYRHPFPGPGLGVRVLGEVKKEYCDLLRRADAIFIEELRKADLYDKVSQAFTVFLPVRSVGVMGDGRKYDWVVSLRAVETIDFMTAHWAHLPYDFLGRVSNRIINEVNGISRVVYDISGKPPATIEWE',\n",
       "  'mw': 58679.0},\n",
       " 'b2508': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ADG7',\n",
       "  'gene': 'b2508',\n",
       "  'locus': ['b2508', 'JW5401'],\n",
       "  'synonyms': 'guaR',\n",
       "  'properties': {'Catalytic Activity': [('H2O + IMP + NAD(+) = H(+) + NADH + XMP',\n",
       "     '1.1.1.205')],\n",
       "   'Function': \"Catalyzes the conversion of inosine 5'-phosphate (IMP) to xanthosine 5'-phosphate (XMP), the first committed and rate-limiting step in the de novo synthesis of guanine nucleotides, and therefore plays an important role in the regulation of cell growth\",\n",
       "   'Regulation Activity': 'Mycophenolic acid (MPA) is a non-competitive inhibitor that prevents formation of the closed enzyme conformation by binding to the same site as the amobile flap. In contrast, mizoribine monophosphate (MZP) is a competitive inhibitor that induces the closed conformation. MPA is a potent inhibitor of mammalian IMPDHs but a poor inhibitor of the bacterial enzymes. MZP is a more potent inhibitor of bacterial IMPDH. IMP dehydrogenase subunit of E.coli contains a cysteine at the IMP binding site and is inhibited in a simple competitive manner by GMP. It does not exhibit allosteric properties as does IMP dehydrogenase from B.subtilis or S.typhimurium',\n",
       "   'Pathway': 'Purine metabolism; XMP biosynthesis via de novo pathway; XMP from IMP: step 1/1'},\n",
       "  'seq': 'MLRIAKEALTFDDVLLVPAHSTVLPNTADLSTQLTKTIRLNIPMLSAAMDTVTEARLAIALAQEGGIGFIHKNMSIERQAEEVRRVKKHESGVVTDPQTVLPTTTLREVKELTERNGFAGYPVVTEENELVGIITGRDVRFVTDLNQPVSVYMTPKERLVTVREGEAREVVLAKMHEKRVEKALVVDDEFHLIGMITVKDFQKAERKPNACKDEQGRLRVGAAVGAGAGNEERVDALVAAGVDVLLIDSSHGHSEGVLQRIRETRAKYPDLQIIGGNVATAAGARALAEAGCSAVKVGIGPGSICTTRIVTGVGVPQITAVADAVEALEGTGIPVIADGGIRFSGDIAKAIAAGASAVMVGSMLAGTEESPGEIELYQGRSYKSYRGMGSLGAMSKGSSDRYFQSDNAADKLVPEGIEGRVAYKGRLKEIIHQQMGGLRSCMGLTGCGTIDELRTKAEFVRISGAGIQESHVHDVTITKESPNYRLGS',\n",
       "  'mw': 52022.0},\n",
       " 'b2514': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P60906',\n",
       "  'gene': 'b2514',\n",
       "  'locus': ['b2514', 'JW2498'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + L-histidine + tRNA(His) = AMP + diphosphate + H(+) + L-histidyl-tRNA(His)',\n",
       "     '6.1.1.21')]},\n",
       "  'seq': 'MAKNIQAIRGMNDYLPGETAIWQRIEGTLKNVLGSYGYSEIRLPIVEQTPLFKRAIGEVTDVVEKEMYTFEDRNGDSLTLRPEGTAGCVRAGIEHGLLYNQEQRLWYIGPMFRHERPQKGRYRQFHQLGCEVFGLQGPDIDAELIMLTARWWRALGISEHVTLELNSIGSLEARANYRDALVAFLEQHKEKLDEDCKRRMYTNPLRVLDSKNPEVQALLNDAPALGDYLDEESREHFAGLCKLLESAGIAYTVNQRLVRGLDYYNRTVFEWVTNSLGSQGTVCAGGRYDGLVEQLGGRATPAVGFAMGLERLVLLVQAVNPEFKADPVVDIYLVASGADTQSAAMALAERLRDELPGVKLMTNHGGGNFKKQFARADKWGARVAVVLGESEVANGTAVVKDLRSGEQTAVAQDSVAAHLRTLLG',\n",
       "  'mw': 47029.0},\n",
       " 'b2515': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P62620',\n",
       "  'gene': 'b2515',\n",
       "  'locus': ['b2515', 'JW2499'],\n",
       "  'synonyms': 'gcpE',\n",
       "  'properties': {'Catalytic Activity': [('(2E)-4-hydroxy-3-methylbut-2-enyl diphosphate + 2 H(+) + H2O + oxidized [flavodoxin] = 2-C-methyl-D-erythritol 2,4-cyclic diphosphate + reduced [flavodoxin]',\n",
       "     '1.17.7.3')],\n",
       "   'Function': 'Converts 2C-methyl-D-erythritol 2,4-cyclodiphosphate (ME-2,4cPP) into 1-hydroxy-2-methyl-2-(E)-butenyl 4-diphosphate, using flavodoxin as the reducing agent',\n",
       "   'Pathway': 'Isoprenoid biosynthesis; isopentenyl diphosphate biosynthesis via DXP pathway; isopentenyl diphosphate from 1-deoxy-D-xylulose 5-phosphate: step 5/6'},\n",
       "  'seq': 'MHNQAPIQRRKSTRIYVGNVPIGDGAPIAVQSMTNTRTTDVEATVNQIKALERVGADIVRVSVPTMDAAEAFKLIKQQVNVPLVADIHFDYRIALKVAEYGVDCLRINPGNIGNEERIRMVVDCARDKNIPIRIGVNAGSLEKDLQEKYGEPTPQALLESAMRHVDHLDRLNFDQFKVSVKASDVFLAVESYRLLAKQIDQPLHLGITEAGGARSGAVKSAIGLGLLLSEGIGDTLRVSLAADPVEEIKVGFDILKSLRIRSRGINFIACPTCSRQEFDVIGTVNALEQRLEDIITPMDVSIIGCVVNGPGEALVSTLGVTGGNKKSGLYEDGVRKDRLDNNDMIDQLEARIRAKASQLDEARRIDVQQVEK',\n",
       "  'mw': 40684.0},\n",
       " 'b2518': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A763',\n",
       "  'gene': 'b2518',\n",
       "  'locus': ['b2518', 'JW2502'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [(\"a 2'-deoxyribonucleoside 5'-diphosphate + ATP = a 2'-deoxyribonucleoside 5'-triphosphate + ADP\",\n",
       "     '2.7.4.6'),\n",
       "    (\"a ribonucleoside 5'-diphosphate + ATP = a ribonucleoside 5'-triphosphate + ADP\",\n",
       "     '2.7.4.6')],\n",
       "   'Function': 'Major role in the synthesis of nucleoside triphosphates other than ATP. The ATP gamma phosphate is transferred to the NDP beta phosphate via a ping-pong mechanism, using a phosphorylated active-site intermediate'},\n",
       "  'seq': 'MAIERTFSIIKPNAVAKNVIGNIFARFEAAGFKIVGTKMLHLTVEQARGFYAEHDGKPFFDGLVEFMTSGPIVVSVLEGENAVQRHRDLLGATNPANALAGTLRADYADSLTENGTHGSDSVESAAREIAYFFGEGEVCPRTR',\n",
       "  'mw': 15463.0},\n",
       " 'b2519': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P76577',\n",
       "  'gene': 'b2519',\n",
       "  'locus': ['b2519', 'JW2503'],\n",
       "  'synonyms': 'yfgN',\n",
       "  'properties': {'Catalytic Activity': [('[GlcNAc-(1->4)-Mur2Ac(oyl-L-Ala-gamma-D-Glu-L-Lys-D-Ala-D-Ala)](n)-di-trans,octa-cis-undecaprenyl diphosphate + beta-D-GlcNAc-(1->4)-Mur2Ac(oyl-L-Ala-gamma-D-Glu-L-Lys-D-Ala-D-Ala)-di-trans,octa-cis-undecaprenyl diphosphate = [GlcNAc-(1->4)-Mur2Ac(oyl-L-Ala-gamma-D-Glu-L-Lys-D-Ala-D-Ala)](n+1)-di-trans-octa-cis-undecaprenyl diphosphate + di-trans,octa-cis-undecaprenyl diphosphate + H(+)',\n",
       "     '2.4.1.129')],\n",
       "   'Function': 'Cell wall formation. The enzyme has a penicillin-insensitive transglycosylase N-terminal domain (formation of linear glycan strands) and a transpeptidase C-terminal domain which may not be functional',\n",
       "   'Regulation Activity': 'Transglycosylase activity can be inhibited by moenomycin',\n",
       "   'Pathway': 'Cell wall biogenesis; peptidoglycan biosynthesis'},\n",
       "  'seq': 'MPRLLTKRGCWITLAAAPFLLFLAAWGADKLWPLPLHEVNPARVVVAQDGTPLWRFADADGIWRYPVTIEDVSPRYLEALINYEDRWFWKHPGVNPFSVARAAWQDLTSGRVISGGSTLTMQVARLLDPHPKTFGGKIRQLWRALQLEWHLSKREILTLYLNRAPFGGTLQGIGAASWAYLGKSPANLSYSEAAMLAVLPQAPSRLRPDRWPERAEAARNKVLERMAVQGVWSREQVKESREEPIWLAPRQMPQLAPLFSRMMLGKSKSDKITTTLDAGLQRRLEELAQNWKGRLPPRSSLAMIVVDHTDMRVRGWVGSVDLNDDSRFGHVDMVNSIRSPGSVLKPFVYGLALDEGLIHPASLLQDVPRRTGDYRPGNFDSGFHGPISMSEALVRSLNLPAVQVLEAYGPKRFAAKLRNVGLPLYLPNGAAPNLSLILGGAGAKLEDMAAAYTAFARHGKAGKLRLQPDDPLLERPLMSSGAAWIIRRIMADEAQPLPDSALPRVAPLAWKTGTSYGYRDAWAIGVNARYVIGIWTGRPDGTPVVGQFGFASAVPLLNQVNNILLSRSANLPEDPRPNSVTRGVICWPGGQSLPEGDGNCRRRLATWLLDGSQPPTLLLPEQEGINGIRFPIWLDENGKRVAADCPQARQEMINVWPLPLEPWLPASERRAVRLPPASTSCPPYGHDAQLPLQLTGVRDGAIIKRLPGAAEATLPLQSSGGAGERWWFLNGEPLTERGRNVTLHLTDKGDYQLLVMDDVGQIATVKFVMQ',\n",
       "  'mw': 85067.0},\n",
       " 'b2521': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P31142',\n",
       "  'gene': 'b2521',\n",
       "  'locus': ['b2521', 'JW2505'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('2-oxo-3-sulfanylpropanoate + [thioredoxin]-dithiol = [thioredoxin]-disulfide + H(+) + hydrogen sulfide + pyruvate',\n",
       "     '2.8.1.2')],\n",
       "   'Function': 'Catalyzes the transfer of sulfur from 3-mercaptopyruvate to a thiol-containing acceptor to form an intramolecular disulfide releasing hydrogen sulfide and pyruvate (Probable). May be involved in the enhancement of bacterial growth inhibition by serine (PubMed:7982894)'},\n",
       "  'seq': 'MSTTWFVGADWLAEHIDDPEIQIIDARMASPGQEDRNVAQEYLNGHIPGAVFFDIEALSDHTSPLPHMLPRPETFAVAMRELGVNQDKHLIVYDEGNLFSAPRAWWMLRTFGVEKVSILGGGLAGWQRDDLLLEEGAVELPEGEFNAAFNPEAVVKVTDVLLASHENTAQIIDARPAARFNAEVDEPRPGLRRGHIPGALNVPWTELVREGELKTTDELDAIFFGRGVSYDKPIIVSCGSGVTAAVVLLALATLDVPNVKLYDGAWSEWGARADLPVEPVK',\n",
       "  'mw': 30812.0},\n",
       " 'b2523': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37095',\n",
       "  'gene': 'b2523',\n",
       "  'locus': ['b2523', 'JW2507'],\n",
       "  'synonyms': 'yfhI',\n",
       "  'properties': {'Catalytic Activity': [('Release of an N-terminal amino acid, Xaa, from a peptide or arylamide. Xaa is preferably Glu or Asp but may be other amino acids, including Leu, Met, His, Cys and Gln.',\n",
       "     '3.4.11.23')],\n",
       "   'Function': 'Probably plays an important role in intracellular peptide degradation (PubMed:20067529)'},\n",
       "  'seq': 'MTEAMKITLSTQPADARWGEKATYSINNDGITLHLNGADDLGLIQRAARKIDGLGIKHVQLSGEGWDADRCWAFWQGYKAPKGTRKVVWPDLDDAQRQELDNRLMIIDWVRDTINAPAEELGPSQLAQRAVDLISNVAGDRVTYRITKGEDLREQGYMGLHTVGRGSERSPVLLALDYNPTGDKEAPVYACLVGKGITFDSGGYSIKQTAFMDSMKSDMGGAATVTGALAFAITRGLNKRVKLFLCCADNLISGNAFKLGDIITYRNGKKVEVMNTDAEGRLVLADGLIDASAQKPEMIIDAATLTGAAKTALGNDYHALFSFDDALAGRLLASAAQENEPFWRLPLAEFHRSQLPSNFAELNNTGSAAYPAGASTAAGFLSHFVENYQQGWLHIDCSATYRKAPVEQWSAGATGLGVRTIANLLTA',\n",
       "  'mw': 46180.0},\n",
       " 'b2530': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A6B7',\n",
       "  'gene': 'b2530',\n",
       "  'locus': ['b2530', 'JW2514'],\n",
       "  'synonyms': 'nuvC yfhO yzzO',\n",
       "  'properties': {'Catalytic Activity': [('[sulfur carrier]-H + L-cysteine = [sulfur carrier]-SH + L-alanine',\n",
       "     '2.8.1.7')],\n",
       "   'Function': \"Master enzyme that delivers sulfur to a number of partners involved in Fe-S cluster assembly, tRNA modification or cofactor biosynthesis. Catalyzes the removal of elemental sulfur from cysteine to produce alanine. Functions as a sulfur delivery protein for Fe-S cluster synthesis onto IscU, an Fe-S scaffold assembly protein, as well as other S acceptor proteins. Preferentially binds to disordered IscU on which the Fe-S is assembled, IscU converts to the structured state and then dissociates from IscS to transfer the Fe-S to an acceptor protein. Also functions as a selenium delivery protein in the pathway for the biosynthesis of selenophosphate. Transfers sulfur onto 'Cys-456' of ThiI and onto 'Cys-19' of TusA in transpersulfidation reactions\",\n",
       "   'Regulation Activity': 'Treatment with N-ethylmaleimide inhibits sulfur transfer. Activated by ThiI and TusA',\n",
       "   'Pathway': 'Cofactor biosynthesis; iron-sulfur cluster biosynthesis'},\n",
       "  'seq': 'MKLPIYLDYSATTPVDPRVAEKMMQFMTMDGTFGNPASRSHRFGWQAEEAVDIARNQIADLVGADPREIVFTSGATESDNLAIKGAANFYQKKGKHIITSKTEHKAVLDTCRQLEREGFEVTYLAPQRNGIIDLKELEAAMRDDTILVSIMHVNNEIGVVQDIAAIGEMCRARGIIYHVDATQSVGKLPIDLSQLKVDLMSFSGHKIYGPKGIGALYVRRKPRVRIEAQMHGGGHERGMRSGTLPVHQIVGMGEAYRIAKEEMATEMERLRGLRNRLWNGIKDIEEVYLNGDLEHGAPNILNVSFNYVEGESLIMALKDLAVSSGSACTSASLEPSYVLRALGLNDELAHSSIRFSLGRFTTEEEIDYTIELVRKSIGRLRDLSPLWEMYKQGVDLNSIEWAHH',\n",
       "  'mw': 45090.0},\n",
       " 'b2533': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ADG4',\n",
       "  'gene': 'b2533',\n",
       "  'locus': ['b2533', 'JW2517'],\n",
       "  'synonyms': 'ssyA',\n",
       "  'properties': {'Catalytic Activity': [('a myo-inositol phosphate + H2O = myo-inositol + phosphate',\n",
       "     '3.1.3.25'),\n",
       "    ('1D-myo-inositol 1-phosphate + H2O = myo-inositol + phosphate',\n",
       "     '3.1.3.25'),\n",
       "    ('1D-myo-inositol 3-phosphate + H2O = myo-inositol + phosphate',\n",
       "     '3.1.3.25')],\n",
       "   'Function': \"Required for growth at low temperatures (PubMed:2138605, PubMed:6389495, PubMed:1847383, PubMed:8002619, PubMed:8589060, PubMed:17652087, PubMed:26980831). Identified as a suppressor (ssyA3) of a temperature-sensitive, protein export missense mutation of secY (secY24), allows growth at 42 but not 30 degrees Celsius (PubMed:6389495). Identified as a suppressor (suhB2) of an rpoH missense mutation (rpoH15), allowing growth at 37 and 40 but not 25, 30 or 34 degrees Celsius, increases expression of RpoH (PubMed:2138605). Identified as a suppressor of a dnaB helicase missense mutation (dnaB121), restores growth at 42 but not 30 degrees Celsius (PubMed:1847383). In both suhB2 and ssyA3 there is an insertion in the 5' region of the gene which prevents SuhB protein expression (PubMed:8002619, PubMed:8589060). Missense mutant suhB10 is suppressed by mutations in RNase III (rnc), showing genetic interaction between them (PubMed:8589060). Deletion of suhB is suppressed by mutations in RNase III, by a mutation in nusA or deletion of nusB, indicating that in the absence of SuhB the Nus complex inhibits growth (PubMed:26980831)\",\n",
       "   'Regulation Activity': 'Inhibited by Li(+) (PubMed:8002619, PubMed:10747806) (Probable). Li(+) binds to Asp-84, Asp-87 and Asp-212 (PubMed:22384802)'},\n",
       "  'seq': 'MHPMLNIAVRAARKAGNLIAKNYETPDAVEASQKGSNDFVTNVDKAAEAVIIDTIRKSYPQHTIITEESGELEGTDQDVQWVIDPLDGTTNFIKRLPHFAVSIAVRIKGRTEVAVVYDPMRNELFTATRGQGAQLNGYRLRGSTARDLDGTILATGFPFKAKQYATTYINIVGKLFNECADFRRTGSAALDLAYVAAGRVDGFFEIGLRPWDFAAGELLVREAGGIVSDFTGGHNYMLTGNIVAGNPRVVKAMLANMRDELSDALKR',\n",
       "  'mw': 29172.0},\n",
       " 'b2536': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'Q47142',\n",
       "  'gene': 'b2536',\n",
       "  'locus': ['b2536', 'JW2520'],\n",
       "  'synonyms': 'yfhS',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Probable permease involved in the uptake of 3-phenylpropionic acid'},\n",
       "  'seq': 'MVLQSTRWLALGYFTYFFSYGIFLPFWSVWLKGIGLTPETIGLLLGAGLVARFLGSLLIAPRVSDPSRLISALRVLALLTLLFAVAFWAGAHVAWLMLVMIGFNLFFSPLVPLTDALANTWQKQFPLDYGKVRLWGSVAFVIGSALTGKLVTMFDYRVILALLTLGVASMLLGFLIRPTIQPQGASRQQESTGWSAWLALVRQNWRFLACVCLLQGAHAAYYGFSAIYWQAAGYSASAVGYLWSLGVVAEVIIFALSNKLFRRCSARDMLLISAICGVVRWGIMGATTALPWLIVVQILHCGTFTVCHLAAMRYIAARQGSEVIRLQAVYSAVAMGGSIAIMTVFAGFLYQYLGHGVFWVMALVALPAMFLRPKVVPSC',\n",
       "  'mw': 41593.0},\n",
       " 'b2538': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ABR5',\n",
       "  'gene': 'b2538',\n",
       "  'locus': ['b2538', 'JW2522'],\n",
       "  'synonyms': 'digA hcaA hcaA1 phdC1 yfhU',\n",
       "  'properties': {'Catalytic Activity': [('3-phenylpropanoate + H(+) + NADH + O2 = 3-(cis-5,6-dihydroxycyclohexa-1,3-dien-1-yl)propanoate + NAD(+)',\n",
       "     '1.14.12.19'),\n",
       "    ('(E)-cinnamate + H(+) + NADH + O2 = (2E)-3-(cis-5,6-dihydroxycyclohexa-1,3-dien-1-yl)prop-2-enoate + NAD(+)',\n",
       "     '1.14.12.19')],\n",
       "   'Function': 'Part of the multicomponent 3-phenylpropionate dioxygenase. Converts 3-phenylpropionic acid (PP) and cinnamic acid (CI) into 3-phenylpropionate-dihydrodiol (PP-dihydrodiol) and cinnamic acid-dihydrodiol (CI-dihydrodiol), respectively',\n",
       "   'Pathway': 'Aromatic compound metabolism; 3-phenylpropanoate degradation'},\n",
       "  'seq': 'MTTPSDLNIYQLIDTQNGRVTPRIYTDPDIYQLELERIFGRCWLFLAHESQIPKPGDFFNTYMGEDAVVVVRQKDGSIKAFLNQCRHRAMRVSYADCGNTRAFTCPYHGWSYGINGELIDVPLEPRAYPQGLCKSHWGLNEVPCVESYKGLIFGNWDTSAPGLRDYLGDIAWYLDGMLDRREGGTEIVGGVQKWVINCNWKFPAEQFASDQYHALFSHASAVQVLGAKDDGSDKRLGDGQTARPVWETAKDALQFGQDGHGSGFFFTEKPDANVWVDGAVSSYYRETYAEAEQRLGEVRALRLAGHNNIFPTLSWLNGTATLRVWHPRGPDQVEVWAFCITDKAASDEVKAAFENSATRAFGPAGFLEQDDSENWCEIQKLLKGHRARNSKLCLEMGLGQEKRRDDGIPGITNYIFSETAARGMYQRWADLLSSESWQEVLDKTAAYQQEVMK',\n",
       "  'mw': 51109.0},\n",
       " 'b2539': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'Q47140',\n",
       "  'gene': 'b2539',\n",
       "  'locus': ['b2539', 'JW2523'],\n",
       "  'synonyms': 'digB hcaA2 hcaB phdC2 yfhV',\n",
       "  'properties': {'Catalytic Activity': [('3-phenylpropanoate + H(+) + NADH + O2 = 3-(cis-5,6-dihydroxycyclohexa-1,3-dien-1-yl)propanoate + NAD(+)',\n",
       "     '1.14.12.19'),\n",
       "    ('(E)-cinnamate + H(+) + NADH + O2 = (2E)-3-(cis-5,6-dihydroxycyclohexa-1,3-dien-1-yl)prop-2-enoate + NAD(+)',\n",
       "     '1.14.12.19')],\n",
       "   'Function': 'Part of the multicomponent 3-phenylpropionate dioxygenase. Converts 3-phenylpropionic acid (PP) and cinnamic acid (CI) into 3-phenylpropionate-dihydrodiol (PP-dihydrodiol) and cinnamic acid-dihydrodiol (CI-dihydrodiol), respectively',\n",
       "   'Pathway': 'Aromatic compound metabolism; 3-phenylpropanoate degradation'},\n",
       "  'seq': 'MSAQVSLELHHRISQFLFHEASLLDDWKFRDWLAQLDEEIRYTMRTTVNAQTRDRRKGVQPPTTWIFNDTKDQLERRIARLETGMAWAEEPPSRTRHLISNCQISETDIPNVFAVRVNYLLYRAQKERDETFYVGTRFDKVRRLEDDNWRLLERDIVLDQAVITSHNLSVLF',\n",
       "  'mw': 20579.0},\n",
       " 'b2540': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ABW0',\n",
       "  'gene': 'b2540',\n",
       "  'locus': ['b2540', 'JW2524'],\n",
       "  'synonyms': 'hcaA3 phdB yfhW',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the multicomponent 3-phenylpropionate dioxygenase, that converts 3-phenylpropionic acid (PP) and cinnamic acid (CI) into 3-phenylpropionate-dihydrodiol (PP-dihydrodiol) and cinnamic acid-dihydrodiol (CI-dihydrodiol), respectively. This protein seems to be a 2Fe-2S ferredoxin',\n",
       "   'Pathway': 'Aromatic compound metabolism; 3-phenylpropanoate degradation'},\n",
       "  'seq': 'MNRIYACPVADVPEGEALRIDTSPVIALFNVGGEFYAINDRCSHGNASMSEGYLEDDATVECPLHAASFCLKTGKALCLPATDPLTTYPVHVEGGDIFIDLPEAQP',\n",
       "  'mw': 11329.0},\n",
       " 'b2541': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0CI31',\n",
       "  'gene': 'b2541',\n",
       "  'locus': ['b2541', 'JW2525'],\n",
       "  'synonyms': 'phdD yfhX',\n",
       "  'properties': {'Catalytic Activity': [('3-(cis-5,6-dihydroxycyclohexa-1,3-dien-1-yl)propanoate + NAD(+) = 3-(2,3-dihydroxyphenyl)propanoate + H(+) + NADH',\n",
       "     '1.3.1.87'),\n",
       "    ('(2E)-3-(cis-5,6-dihydroxycyclohexa-1,3-dien-1-yl)prop-2-enoate + NAD(+) = (2E)-3-(2,3-dihydroxyphenyl)prop-2-enoate + H(+) + NADH',\n",
       "     '1.3.1.87')],\n",
       "   'Function': 'Converts 3-phenylpropionate-dihydrodiol (PP-dihydrodiol) and cinnamic acid-dihydrodiol (CI-dihydrodiol) into 3-(2,3-dihydroxylphenyl)propanoic acid (DHPP) and 2,3-dihydroxicinnamic acid (DHCI), respectively',\n",
       "   'Pathway': 'Aromatic compound metabolism; 3-phenylpropanoate degradation'},\n",
       "  'seq': 'MSDLHNESIFITGGGSGLGLALVERFIEEGAQVATLELSAAKVASLRQRFGEHILAVEGNVTCYADYQRAVDQILTRSGKLDCFIGNAGIWDHNASLVNTPAETLETGFHELFNVNVLGYLLGAKACAPALIASEGSMIFTLSNAAWYPGGGGPLYTASKHAATGLIRQLAYELAPKVRVNGVGPCGMASDLRGPQALGQSETSIMQSLTPEKIAAILPLQFFPQPADFTGPYVMLTSRRNNRALSGVMINADAGLAIRGIRHVAAGLDL',\n",
       "  'mw': 28500.0},\n",
       " 'b2542': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77650',\n",
       "  'gene': 'b2542',\n",
       "  'locus': ['b2542', 'JW2526'],\n",
       "  'synonyms': 'hcaA4 phdA yfhY',\n",
       "  'properties': {'Catalytic Activity': [('H(+) + NAD(+) + 2 reduced [2Fe-2S]-[ferredoxin] = NADH + 2 oxidized [2Fe-2S]-[ferredoxin]',\n",
       "     '1.18.1.3')],\n",
       "   'Function': 'Part of the multicomponent 3-phenylpropionate dioxygenase, that converts 3-phenylpropionic acid (PP) and cinnamic acid (CI) into 3-phenylpropionate-dihydrodiol (PP-dihydrodiol) and cinnamic acid-dihydrodiol (CI-dihydrodiol), respectively',\n",
       "   'Pathway': 'Aromatic compound metabolism; 3-phenylpropanoate degradation'},\n",
       "  'seq': 'MKEKTIIIVGGGQAAAMAAASLRQQGFTGELHLFSDERHLPYERPPLSKSMLLEDSPQLQQVLPANWWQENNVHLHSGVTIKTLGRDTRELVLTNGESWHWDQLFIATGAAARPLPLLDALGERCFTLRHAGDAARLREVLQPERSVVIIGAGTIGLELAASATQRRCKVTVIELAATVMGRNAPPPVQRYLLQRHQQAGVRILLNNAIEHVVDGEKVELTLQSGETLQADVVIYGIGISANEQLAREANLDTANGIVIDEACRTCDPAIFAGGDVAITRLDNGALHRCESWENANNQAQIAAAAMLGLPLPLLPPPWFWSDQYSDNLQFIGDMRGDDWLCRGNPETQKAIWFNLQNGVLIGAVTLNQGREIRPIRKWIQSGKTFDAKLLIDENIALKSL',\n",
       "  'mw': 43978.0},\n",
       " 'b2551': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A825',\n",
       "  'gene': 'b2551',\n",
       "  'locus': ['b2551', 'JW2535'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(6R)-5,10-methylene-5,6,7,8-tetrahydrofolate + glycine + H2O = (6S)-5,6,7,8-tetrahydrofolate + L-serine',\n",
       "     '2.1.2.1'),\n",
       "    ('L-allo-threonine = acetaldehyde + glycine', ''),\n",
       "    ('5,10-methenyltetrahydrofolate + H2O = (6S)-5-formyl-5,6,7,8-tetrahydrofolate + H(+)',\n",
       "     '')],\n",
       "   'Function': 'Catalyzes the reversible interconversion of serine and glycine with tetrahydrofolate (THF) serving as the one-carbon carrier. This reaction serves as the major source of one-carbon groups required for the biosynthesis of purines, thymidylate, methionine, and other important biomolecules (PubMed:3891721, PubMed:1517215, PubMed:7925461, PubMed:10858298, PubMed:19883126). Also exhibits THF-independent aldolase activity toward beta-hydroxyamino acids, producing glycine and aldehydes, via a retro-aldol mechanism. Thus, is able to catalyze the cleavage of allothreonine and 3-phenylserine (PubMed:3891721, PubMed:1517215, PubMed:10858298, PubMed:19883126). Also catalyzes the irreversible conversion of 5,10-methenyltetrahydrofolate to 5-formyltetrahydrofolate (PubMed:2201683, PubMed:10858298)',\n",
       "   'Pathway': 'Amino-acid biosynthesis; glycine biosynthesis; glycine from L-serine: step 1/1'},\n",
       "  'seq': 'MLKREMNIADYDAELWQAMEQEKVRQEEHIELIASENYTSPRVMQAQGSQLTNKYAEGYPGKRYYGGCEYVDIVEQLAIDRAKELFGADYANVQPHSGSQANFAVYTALLEPGDTVLGMNLAHGGHLTHGSPVNFSGKLYNIVPYGIDATGHIDYADLEKQAKEHKPKMIIGGFSAYSGVVDWAKMREIADSIGAYLFVDMAHVAGLVAAGVYPNPVPHAHVVTTTTHKTLAGPRGGLILAKGGSEELYKKLNSAVFPGGQGGPLMHVIAGKAVALKEAMEPEFKTYQQQVAKNAKAMVEVFLERGYKVVSGGTDNHLFLVDLVDKNLTGKEADAALGRANITVNKNSVPNDPKSPFVTSGIRVGTPAITRRGFKEAEAKELAGWMCDVLDSINDEAVIERIKGKVLDICARYPVYA',\n",
       "  'mw': 45317.0},\n",
       " 'b2552': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P24232',\n",
       "  'gene': 'b2552',\n",
       "  'locus': ['b2552', 'JW2536'],\n",
       "  'synonyms': 'fsrB hmpA',\n",
       "  'properties': {'Catalytic Activity': [('NADPH + 2 nitric oxide + 2 O2 = H(+) + NADP(+) + 2 nitrate',\n",
       "     '1.14.12.17'),\n",
       "    ('NADH + 2 nitric oxide + 2 O2 = H(+) + NAD(+) + 2 nitrate',\n",
       "     '1.14.12.17')],\n",
       "   'Function': 'Various electron acceptors are also reduced by HMP in vitro, including dihydropterine, ferrisiderophores, ferric citrate, cytochrome c, nitrite, S-nitrosoglutathione, and alkylhydroperoxides. However, it is unknown if these reactions are of any biological significance in vivo'},\n",
       "  'seq': 'MLDAQTIATVKATIPLLVETGPKLTAHFYDRMFTHNPELKEIFNMSNQRNGDQREALFNAIAAYASNIENLPALLPAVEKIAQKHTSFQIKPEQYNIVGEHLLATLDEMFSPGQEVLDAWGKAYGVLANVFINREAEIYNENASKAGGWEGTRDFRIVAKTPRSALITSFELEPVDGGAVAEYRPGQYLGVWLKPEGFPHQEIRQYSLTRKPDGKGYRIAVKREEGGQVSNWLHNHANVGDVVKLVAPAGDFFMAVADDTPVTLISAGVGQTPMLAMLDTLAKAGHTAQVNWFHAAENGDVHAFADEVKELGQSLPRFTAHTWYRQPSEADRAKGQFDSEGLMDLSKLEGAFSDPTMQFYLCGPVGFMQFTAKQLVDLGVKQENIHYECFGPHKVL',\n",
       "  'mw': 43868.0},\n",
       " 'b2557': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P15254',\n",
       "  'gene': 'b2557',\n",
       "  'locus': ['b2557', 'JW2541'],\n",
       "  'synonyms': 'purI',\n",
       "  'properties': {'Catalytic Activity': [('ATP + H2O + L-glutamine + N(2)-formyl-N(1)-(5-phospho-beta-D-ribosyl)glycinamide = 2-formamido-N(1)-(5-O-phospho-beta-D-ribosyl)acetamidine + ADP + H(+) + L-glutamate + phosphate',\n",
       "     '6.3.5.3')],\n",
       "   'Function': 'Phosphoribosylformylglycinamidine synthase involved in the purines biosynthetic pathway. Catalyzes the ATP-dependent conversion of formylglycinamide ribonucleotide (FGAR) and glutamine to yield formylglycinamidine ribonucleotide (FGAM) and glutamate',\n",
       "   'Pathway': 'Purine metabolism; IMP biosynthesis via de novo pathway; 5-amino-1-(5-phospho-D-ribosyl)imidazole from N(2)-formyl-N(1)-(5-phospho-D-ribosyl)glycinamide: step 1/2'},\n",
       "  'seq': 'MMEILRGSPALSAFRINKLLARFQAARLPVHNIYAEYVHFADLNAPLNDDEHAQLERLLKYGPALASHAPQGKLLLVTPRPGTISPWSSKATDIAHNCGLQQVNRLERGVAYYIEAGTLTNEQWQQVTAELHDRMMETVFFALDDAEQLFAHHQPTPVTSVDLLGQGRQALIDANLRLGLALAEDEIDYLQDAFTKLGRNPNDIELYMFAQANSEHCRHKIFNADWVIDGEQQPKSLFKMIKNTFETTPDHVLSAYKDNAAVMEGSEVGRYFADHETGRYDFHQEPAHILMKVETHNHPTAISPWPGAATGSGGEIRDEGATGRGAKPKAGLVGFSVSNLRIPGFEQPWEEDFGKPERIVTALDIMTEGPLGGAAFNNEFGRPALNGYFRTYEEKVNSHNGEELRGYHKPIMLAGGIGNIRADHVQKGEINVGAKLVVLGGPAMNIGLGGGAASSMASGQSDADLDFASVQRDNPEMERRCQEVIDRCWQLGDANPILFIHDVGAGGLSNAMPELVSDGGRGGKFELREILSDEPGMSPLEIWCNESQERYVLAVAADQLPLFDELCKRERAPYAVIGEATEELHLSLHDRHFDNQPIDLPLDVLLGKTPKMTRDVQTLKAKGDALAREGITIADAVKRVLHLPTVAEKTFLVTIGDRSVTGMVARDQMVGPWQVPVANCAVTTASLDSYYGEAMAIGERAPVALLDFAASARLAVGEALTNIAATQIGDIKRIKLSANWMAAAGHPGEDAGLYEAVKAVGEELCPALGLTIPVGKDSMSMKTRWQEGNEEREMTSPLSLVISAFARVEDVRHTITPQLSTEDNALLLIDLGKGNNALGATALAQVYRQLGDKPADVRDVAQLKGFYDAIQALVAQRKLLAYHDRSDGGLLVTLAEMAFAGHCGIDADIATLGDDRLAALFNEELGAVIQVRAADREAVESVLAQHGLADCVHYVGQAVSGDRFVITANGQTVFSESRTTLRVWWAETTWQMQRLRDNPECADQEHQAKSNDADPGLNVKLSFDINEDVAAPYIATGARPKVAVLREQGVNSHVEMAAAFHRAGFDAIDVHMSDLLTGRTGLEDFHALVACGGFSYGDVLGAGEGWAKSILFNDRVRDEFATFFHRPQTLALGVCNGCQMMSNLRELIPGSELWPRFVRNTSDRFEARFSLVEVTQSPSLLLQGMVGSQMPIAVSHGEGRVEVRDAAHLAALESKGLVALRYVDNFGKVTETYPANPNGSPNGITAVTTESGRVTIMMPHPERVFRTVSNSWHPENWGEDGPWMRIFRNARKQLG',\n",
       "  'mw': 141403.0},\n",
       " 'b2563': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P24224',\n",
       "  'gene': 'b2563',\n",
       "  'locus': ['b2563', 'JW2547'],\n",
       "  'synonyms': 'dpj',\n",
       "  'properties': {'Catalytic Activity': [(\"apo-[ACP] + CoA = adenosine 3',5'-bisphosphate + H(+) + holo-[ACP]\",\n",
       "     '2.7.8.7')],\n",
       "   'Function': \"Transfers the 4'-phosphopantetheine moiety from coenzyme A to the 'Ser-36' of acyl-carrier-protein\"},\n",
       "  'seq': 'MAILGLGTDIVEIARIEAVIARSGDRLARRVLSDNEWAIWKTHHQPVRFLAKRFAVKEAAAKAFGTGIRNGLAFNQFEVFNDELGKPRLRLWGEALKLAEKLGVANMHVTLADERHYACATVIIES',\n",
       "  'mw': 14052.0},\n",
       " 'b2564': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A794',\n",
       "  'gene': 'b2564',\n",
       "  'locus': ['b2564', 'JW2548'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [(\"1-deoxy-D-xylulose 5-phosphate + 3-amino-2-oxopropyl phosphate = H(+) + 2 H2O + phosphate + pyridoxine 5'-phosphate\",\n",
       "     '2.6.99.2')],\n",
       "   'Function': \"Catalyzes the complicated ring closure reaction between the two acyclic compounds 1-deoxy-D-xylulose-5-phosphate (DXP) and 3-amino-2-oxopropyl phosphate (1-amino-acetone-3-phosphate or AAP) to form pyridoxine 5'-phosphate (PNP) and inorganic phosphate\",\n",
       "   'Pathway': \"Cofactor biosynthesis; pyridoxine 5'-phosphate biosynthesis; pyridoxine 5'-phosphate from D-erythrose 4-phosphate: step 5/5\"},\n",
       "  'seq': 'MAELLLGVNIDHIATLRNARGTAYPDPVQAAFIAEQAGADGITVHLREDRRHITDRDVRILRQTLDTRMNLEMAVTEEMLAIAVETKPHFCCLVPEKRQEVTTEGGLDVAGQRDKMRDACKRLADAGIQVSLFIDADEEQIKAAAEVGAPFIEIHTGCYADAKTDAEQAQELARIAKAATFAASLGLKVNAGHGLTYHNVKAIAAIPEMHELNIGHAIIGRAVMTGLKDAVAEMKRLMLEARG',\n",
       "  'mw': 26384.0},\n",
       " 'b2574': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P10902',\n",
       "  'gene': 'b2574',\n",
       "  'locus': ['b2574', 'JW2558'],\n",
       "  'synonyms': 'nicB',\n",
       "  'properties': {'Catalytic Activity': [('L-aspartate + O2 = H2O2 + iminosuccinate',\n",
       "     '1.4.3.16'),\n",
       "    ('fumarate + L-aspartate = iminosuccinate + succinate', '')],\n",
       "   'Function': 'Catalyzes the oxidation of L-aspartate to iminoaspartate, the first step in the de novo biosynthesis of NAD(+) (PubMed:2841129, PubMed:7033218, PubMed:2187483, PubMed:8706749, PubMed:8706750, PubMed:11294641). Can use either oxygen or fumarate as electron acceptors, which allows the enzyme to be functional under aerobic and anaerobic conditions (PubMed:8706750, PubMed:11294641, PubMed:12200425). In vivo, fumarate is used under anaerobic conditions, and oxygen is the predominant electron acceptor under aerobic conditions due to the lower fumarate levels (PubMed:20149100). In vitro, fumarate is a more efficient electron acceptor and is kinetically superior to oxygen (PubMed:12200425, PubMed:20149100)',\n",
       "   'Regulation Activity': 'Inhibited by the product iminoaspartate (PubMed:8706749). Competitively inhibited by mesotartrate (PubMed:2187483). NAD acts as a competitive inhibitor to FAD (PubMed:7033218, PubMed:2187483). Inhibited by iodoacetic acid, diethylpyrocarbonate and tetranitromethane (PubMed:2187483)',\n",
       "   'Pathway': 'Cofactor biosynthesis; NAD(+) biosynthesis; iminoaspartate from L-aspartate (oxidase route): step 1/1'},\n",
       "  'seq': 'MNTLPEHSCDVLIIGSGAAGLSLALRLADQHQVIVLSKGPVTEGSTFYAQGGIAAVFDETDSIDSHVEDTLIAGAGICDRHAVEFVASNARSCVQWLIDQGVLFDTHIQPNGEESYHLTREGGHSHRRILHAADATGREVETTLVSKALNHPNIRVLERSNAVDLIVSDKIGLPGTRRVVGAWVWNRNKETVETCHAKAVVLATGGASKVYQYTTNPDISSGDGIAMAWRAGCRVANLEFNQFHPTALYHPQARNFLLTEALRGEGAYLKRPDGTRFMPDFDERGELAPRDIVARAIDHEMKRLGADCMFLDISHKPADFIRQHFPMIYEKLLGLGIDLTQEPVPIVPAAHYTCGGVMVDDHGRTDVEGLYAIGEVSYTGLHGANRMASNSLLECLVYGWSAAEDITRRMPYAHDISTLPPWDESRVENPDERVVIQHNWHELRLFMWDYVGIVRTTKRLERALRRITMLQQEIDEYYAHFRVSNNLLELRNLVQVAELIVRCAMMRKESRGLHFTLDYPELLTHSGPSILSPGNHYINR',\n",
       "  'mw': 60337.0},\n",
       " 'b2578': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P38101',\n",
       "  'gene': 'b2578',\n",
       "  'locus': ['b2578', 'JW2562'],\n",
       "  'synonyms': 'yfiK',\n",
       "  'properties': {'Catalytic Activity': [('O-acetyl-L-serine(in) = O-acetyl-L-serine(out)',\n",
       "     ''),\n",
       "    ('L-cysteine(in) = L-cysteine(out)', '')],\n",
       "   'Function': 'Exporter of O-acetylserine (OAS) and cysteine'},\n",
       "  'seq': 'MTPTLLSAFWTYTLITAMTPGPNNILALSSATSHGFRQSTRVLAGMSLGFLIVMLLCAGISFSLAVIDPAAVHLLSWAGAAYIVWLAWKIATSPTKEDGLQAKPISFWASFALQFVNVKIILYGVTALSTFVLPQTQALSWVVGVSVLLAMIGTFGNVCWALAGHLFQRLFRQYGRQLNIVLALLLVYCAVRIFY',\n",
       "  'mw': 21248.0},\n",
       " 'b2579': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P68066',\n",
       "  'gene': 'b2579',\n",
       "  'locus': ['b2579', 'JW2563'],\n",
       "  'synonyms': 'yfiD',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Acts as a radical domain for damaged PFL and possibly other radical proteins'},\n",
       "  'seq': 'MITGIQITKAANDDLLNSFWLLDSEKGEARCIVAKAGYAEDEVVAVSKLGDIEYREVPVEVKPEVRVEGGQHLNVNVLRRETLEDAVKHPEKYPQLTIRVSGYAVRFNSLTPEQQRDVIARTFTESL',\n",
       "  'mw': 14284.0},\n",
       " 'b2582': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AGG4',\n",
       "  'gene': 'b2582',\n",
       "  'locus': ['b2582', 'JW2566'],\n",
       "  'synonyms': 'yfiG',\n",
       "  'properties': {'Catalytic Activity': [('[protein]-dithiol + NAD(+) = [protein]-disulfide + H(+) + NADH',\n",
       "     '1.8.1.8'),\n",
       "    ('[protein]-dithiol + NADP(+) = [protein]-disulfide + H(+) + NADPH',\n",
       "     '1.8.1.8')],\n",
       "   'Function': 'Efficient electron donor for the essential enzyme ribonucleotide reductase. Is also able to reduce the interchain disulfide bridges of insulin'},\n",
       "  'seq': 'MNTVCTHCQAINRIPDDRIEDAAKCGRCGHDLFDGEVINATGETLDKLLKDDLPVVIDFWAPWCGPCRNFAPIFEDVAQERSGKVRFVKVNTEAERELSSRFGIRSIPTIMIFKNGQVVDMLNGAVPKAPFDSWLNESL',\n",
       "  'mw': 15555.0},\n",
       " 'b2585': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P23830',\n",
       "  'gene': 'b2585',\n",
       "  'locus': ['b2585', 'JW2569'],\n",
       "  'synonyms': 'pss',\n",
       "  'properties': {'Catalytic Activity': [('a CDP-1,2-diacyl-sn-glycerol + L-serine = a 1,2-diacyl-sn-glycero-3-phospho-L-serine + CMP + H(+)',\n",
       "     '2.7.8.8')]},\n",
       "  'seq': 'MLSKFKRNKHQQHLAQLPKISQSVDDVDFFYAPADFRETLLEKIASAKQRICIVALYLEQDDGGKGILNALYEAKRQRPELDVRVLVDWHRAQRGRIGAAASNTNADWYCRMAQENPGVDVPVYGVPINTREALGVLHFKGFIIDDSVLYSGASLNDVYLHQHDKYRYDRYHLIRNRKMSDIMFEWVTQNIMNGRGVNRLDDVNRPKSPEIKNDIRLFRQELRDAAYHFQGDADNDQLSVTPLVGLGKSSLLNKTIFHLMPCAEQKLTICTPYFNLPAILVRNIIQLLREGKKVEIIVGDKTANDFYIPEDEPFKIIGALPYLYEINLRRFLSRLQYYVNTDQLVVRLWKDDDNTYHLKGMWVDDKWMLITGNNLNPRAWRLDLENAILIHDPQLELAPQREKELELIREHTTIVKHYRDLQSIADYPVKVRKLIRRLRRIRIDRLISRIL',\n",
       "  'mw': 52802.0},\n",
       " 'b2587': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AEX3',\n",
       "  'gene': 'b2587',\n",
       "  'locus': ['b2587', 'JW2571'],\n",
       "  'synonyms': 'witA',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Uptake of alpha-ketoglutarate across the boundary membrane with the concomitant import of a cation (symport system)'},\n",
       "  'seq': 'MAESTVTADSKLTSSDTRRRIWAIVGASSGNLVEWFDFYVYSFCSLYFAHIFFPSGNTTTQLLQTAGVFAAGFLMRPIGGWLFGRIADKHGRKKSMLLSVCMMCFGSLVIACLPGYETIGTWAPALLLLARLFQGLSVGGEYGTSATYMSEVAVEGRKGFYASFQYVTLIGGQLLALLVVVVLQHTMEDAALREWGWRIPFALGAVLAVVALWLRRQLDETSQQETRALKEAGSLKGLWRNRRAFIMVLGFTAAGSLCFYTFTTYMQKYLVNTAGMHANVASGIMTAALFVFMLIQPLIGALSDKIGRRTSMLCFGSLAAIFTVPILSALQNVSSPYAAFGLVMCALLIVSFYTSISGILKAEMFPAQVRALGVGLSYAVANAIFGGSAEYVALSLKSIGMETAFFWYVTLMAVVAFLVSLMLHRKGKGMRL',\n",
       "  'mw': 47052.0},\n",
       " 'b2599': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A9J8',\n",
       "  'gene': 'b2599',\n",
       "  'locus': ['b2599', 'JW2580'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('chorismate = prephenate',\n",
       "     '5.4.99.5'),\n",
       "    ('H(+) + prephenate = 3-phenylpyruvate + CO2 + H2O', '4.2.1.51')],\n",
       "   'Function': 'Catalyzes the Claisen rearrangement of chorismate to prephenate and the decarboxylation/dehydration of prephenate to phenylpyruvate',\n",
       "   'Regulation Activity': 'Both activities are inhibited by L-phenylalanine',\n",
       "   'Pathway': 'Metabolic intermediate biosynthesis; prephenate biosynthesis; prephenate from chorismate: step 1/1'},\n",
       "  'seq': 'MTSENPLLALREKISALDEKLLALLAERRELAVEVGKAKLLSHRPVRDIDRERDLLERLITLGKAHHLDAHYITRLFQLIIEDSVLTQQALLQQHLNKINPHSARIAFLGPKGSYSHLAARQYAARHFEQFIESGCAKFADIFNQVETGQADYAVVPIENTSSGAINDVYDLLQHTSLSIVGEMTLTIDHCLLVSGTTDLSTINTVYSHPQPFQQCSKFLNRYPHWKIEYTESTSAAMEKVAQAKSPHVAALGSEAGGTLYGLQVLERIEANQRQNFTRFVVLARKAINVSDQVPAKTTLLMATGQQAGALVEALLVLRNHNLIMTRLESRPIHGNPWEEMFYLDIQANLESAEMQKALKELGEITRSMKVLGCYPSENVVPVDPT',\n",
       "  'mw': 43111.0},\n",
       " 'b2600': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P07023',\n",
       "  'gene': 'b2600',\n",
       "  'locus': ['b2600', 'JW2581'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('chorismate = prephenate',\n",
       "     '5.4.99.5'),\n",
       "    ('NAD(+) + prephenate = 3-(4-hydroxyphenyl)pyruvate + CO2 + NADH',\n",
       "     '1.3.1.12')],\n",
       "   'Pathway': 'Metabolic intermediate biosynthesis; prephenate biosynthesis; prephenate from chorismate: step 1/1'},\n",
       "  'seq': 'MVAELTALRDQIDEVDKALLNLLAKRLELVAEVGEVKSRFGLPIYVPEREASMLASRRAEAEALGVPPDLIEDVLRRVMRESYSSENDKGFKTLCPSLRPVVIVGGGGQMGRLFEKMLTLSGYQVRILEQHDWDRAADIVADAGMVIVSVPIHVTEQVIGKLPPLPKDCILVDLASVKNGPLQAMLVAHDGPVLGLHPMFGPDSGSLAKQVVVWCDGRKPEAYQWFLEQIQVWGARLHRISAVEHDQNMAFIQALRHFATFAYGLHLAEENVQLEQLLALSSPIYRLELAMVGRLFAQDPQLYADIIMSSERNLALIKRYYKRFGEAIELLEQGDKQAFIDSFRKVEHWFGDYAQRFQSESRVLLRQANDNRQ',\n",
       "  'mw': 42043.0},\n",
       " 'b2601': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P00888',\n",
       "  'gene': 'b2601',\n",
       "  'locus': ['b2601', 'JW2582'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('D-erythrose 4-phosphate + H2O + phosphoenolpyruvate = 7-phospho-2-dehydro-3-deoxy-D-arabino-heptonate + phosphate',\n",
       "     '2.5.1.54')],\n",
       "   'Function': 'Stereospecific condensation of phosphoenolpyruvate (PEP) and D-erythrose-4-phosphate (E4P) giving rise to 3-deoxy-D-arabino-heptulosonate-7-phosphate (DAHP)',\n",
       "   'Regulation Activity': 'Specifically feedback inhibited by tyrosine with 50% inhibition observed at 9 microM tyrosine, pH 7.0',\n",
       "   'Pathway': 'Metabolic intermediate biosynthesis; chorismate biosynthesis; chorismate from D-erythrose 4-phosphate and phosphoenolpyruvate: step 1/7'},\n",
       "  'seq': 'MQKDALNNVHITDEQVLMTPEQLKAAFPLSLQQEAQIADSRKSISDIIAGRDPRLLVVCGPCSIHDPETALEYARRFKALAAEVSDSLYLVMRVYFEKPRTTVGWKGLINDPHMDGSFDVEAGLQIARKLLLELVNMGLPLATEALDPNSPQYLGDLFSWSAIGARTTESQTHREMASGLSMPVGFKNGTDGSLATAINAMRAAAQPHRFVGINQAGQVALLQTQGNPDGHVILRGGKAPNYSPADVAQCEKEMEQAGLRPSLMVDCSHGNSNKDYRRQPAVAESVVAQIKDGNRSIIGLMIESNIHEGNQSSEQPRSEMKYGVSVTDACISWEMTDALLREIHQDLNGQLTARVA',\n",
       "  'mw': 38804.0},\n",
       " 'b2615': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A7B3',\n",
       "  'gene': 'b2615',\n",
       "  'locus': ['b2615', 'JW2596'],\n",
       "  'synonyms': 'yfjB yfjE',\n",
       "  'properties': {'Catalytic Activity': [('ATP + NAD(+) = ADP + H(+) + NADP(+)',\n",
       "     '2.7.1.23')],\n",
       "   'Function': \"Involved in the regulation of the intracellular balance of NAD and NADP, and is a key enzyme in the biosynthesis of NADP. Catalyzes specifically the phosphorylation on 2'-hydroxyl of the adenosine moiety of NAD to yield NADP. It can use ATP and other nucleoside triphosphates (UTP, CTP, GTP, dATP, TTP) as phosphoryl donors, while nucleoside mono- or diphosphates and poly(P) cannot\",\n",
       "   'Regulation Activity': 'Competitively and allosterically inhibited by NADH and NADPH at physiological concentrations, whereas inhibition by NADP is only slight. Inhibited by p-chloromercuribenzoate and HgCl(2)'},\n",
       "  'seq': 'MNNHFKCIGIVGHPRHPTALTTHEMLYRWLCTKGYEVIVEQQIAHELQLKNVKTGTLAEIGQLADLAVVVGGDGNMLGAARTLARYDIKVIGINRGNLGFLTDLDPDNAQQQLADVLEGHYISEKRFLLEAQVCQQDCQKRISTAINEVVLHPGKVAHMIEFEVYIDEIFAFSQRSDGLIISTPTGSTAYSLSAGGPILTPSLDAITLVPMFPHTLSARPLVINSSSTIRLRFSHRRNDLEISCDSQIALPIQEGEDVLIRRCDYHLNLIHPKDYSYFNTLSTKLGWSKKLF',\n",
       "  'mw': 32566.0},\n",
       " 'b2661': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P25526',\n",
       "  'gene': 'b2661',\n",
       "  'locus': ['b2661', 'JW2636'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('H2O + NADP(+) + succinate semialdehyde = 2 H(+) + NADPH + succinate',\n",
       "     '1.2.1.79'),\n",
       "    ('5-oxopentanoate + H2O + NADP(+) = glutarate + 2 H(+) + NADPH', '')],\n",
       "   'Function': 'Catalyzes the NADP(+)-dependent oxidation of succinate semialdehyde to succinate (PubMed:20174634). Thereby functions in a GABA degradation pathway that allows some E.coli strains to utilize GABA as a nitrogen source for growth (PubMed:7011797). Also catalyzes the conversion of glutarate semialdehyde to glutarate, as part of a L-lysine degradation pathway that proceeds via cadaverine, glutarate and L-2-hydroxyglutarate (PubMed:30498244)',\n",
       "   'Pathway': 'Amino-acid degradation'},\n",
       "  'seq': 'MKLNDSNLFRQQALINGEWLDANNGEAIDVTNPANGDKLGSVPKMGADETRAAIDAANRALPAWRALTAKERATILRNWFNLMMEHQDDLARLMTLEQGKPLAEAKGEISYAASFIEWFAEEGKRIYGDTIPGHQADKRLIVIKQPIGVTAAITPWNFPAAMITRKAGPALAAGCTMVLKPASQTPFSALALAELAIRAGVPAGVFNVVTGSAGAVGNELTSNPLVRKLSFTGSTEIGRQLMEQCAKDIKKVSLELGGNAPFIVFDDADLDKAVEGALASKFRNAGQTCVCANRLYVQDGVYDRFAEKLQQAVSKLHIGDGLDNGVTIGPLIDEKAVAKVEEHIADALEKGARVVCGGKAHERGGNFFQPTILVDVPANAKVSKEETFGPLAPLFRFKDEADVIAQANDTEFGLAAYFYARDLSRVFRVGEALEYGIVGINTGIISNEVAPFGGIKASGLGREGSKYGIEDYLEIKYMCIGL',\n",
       "  'mw': 51720.0},\n",
       " 'b2662': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P22256',\n",
       "  'gene': 'b2662',\n",
       "  'locus': ['b2662', 'JW2637'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('2-oxoglutarate + 4-aminobutanoate = L-glutamate + succinate semialdehyde',\n",
       "     '2.6.1.19'),\n",
       "    ('2-oxoglutarate + 5-aminopentanoate = 5-oxopentanoate + L-glutamate',\n",
       "     '2.6.1.48')],\n",
       "   'Function': 'Pyridoxal phosphate-dependent enzyme that catalyzes transamination between primary amines and alpha-keto acids. Catalyzes the transfer of the amino group from gamma-aminobutyrate (GABA) to alpha-ketoglutarate (KG) to yield succinic semialdehyde (SSA) and glutamate (PubMed:15723541, PubMed:30498244). Thereby functions in a GABA degradation pathway that allows some E.coli strains to utilize GABA as a nitrogen source for growth (PubMed:12446648). Also catalyzes the conversion of 5-aminovalerate to glutarate semialdehyde, as part of a L-lysine degradation pathway that proceeds via cadaverine, glutarate and L-2-hydroxyglutarate (PubMed:30498244)',\n",
       "   'Pathway': 'Amino-acid degradation'},\n",
       "  'seq': 'MNSNKELMQRRSQAIPRGVGQIHPIFADRAENCRVWDVEGREYLDFAGGIAVLNTGHLHPKVVAAVEAQLKKLSHTCFQVLAYEPYLELCEIMNQKVPGDFAKKTLLVTTGSEAVENAVKIARAATKRSGTIAFSGAYHGRTHYTLALTGKVNPYSAGMGLMPGHVYRALYPCPLHGISEDDAIASIHRIFKNDAAPEDIAAIVIEPVQGEGGFYASSPAFMQRLRALCDEHGIMLIADEVQSGAGRTGTLFAMEQMGVAPDLTTFAKSIAGGFPLAGVTGRAEVMDAVAPGGLGGTYAGNPIACVAALEVLKVFEQENLLQKANDLGQKLKDGLLAIAEKHPEIGDVRGLGAMIAIELFEDGDHNKPDAKLTAEIVARARDKGLILLSCGPYYNVLRILVPLTIEDAQIRQGLEIISQCFDEAKQ',\n",
       "  'mw': 45775.0},\n",
       " 'b2663': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P25527',\n",
       "  'gene': 'b2663',\n",
       "  'locus': ['b2663', 'JW2638'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('4-aminobutanoate(in) + H(+)(in) = 4-aminobutanoate(out) + H(+)(out)',\n",
       "     '')],\n",
       "   'Function': 'Transporter for gamma-aminobutyrate (GABA) (PubMed:8297211, PubMed:8557687). Transport is driven by the membrane potential (PubMed:8297211). Can also transport a number of GABA analogs such as nipecotic acid or muscimol (PubMed:8557687)',\n",
       "   'Regulation Activity': 'Uptake is stimulated by ammonium sulfate and abolished by 2,4-dinitrophenol (PubMed:8297211). Is affected both topologically and kinetically by phospholipid composition of the membrane (PubMed:15890647). In cells lacking phosphatidylethanolamine (PE), the N-terminal hairpin is inverted relative to the membrane and the rate of GABA transport is reduced by more than 99% (PubMed:15890647)',\n",
       "   'Pathway': 'Amino-acid degradation; 4-aminobutanoate degradation'},\n",
       "  'seq': 'MGQSSQPHELGGGLKSRHVTMLSIAGVIGASLFVGSSVAIAEAGPAVLLAYLFAGLLVVMIMRMLAEMAVATPDTGSFSTYADKAIGRWAGYTIGWLYWWFWVLVIPLEANIAAMILHSWVPGIPIWLFSLVITLALTGSNLLSVKNYGEFEFWLALCKVIAILAFIFLGAVAISGFYPYAEVSGISRLWDSGGFMPNGFGAVLSAMLITMFSFMGAEIVTIAAAESDTPEKHIVRATNSVIWRISIFYLCSIFVVVALIPWNMPGLKAVGSYRSVLELLNIPHAKLIMDCVILLSVTSCLNSALYTASRMLYSLSRRGDAPAVMGKINRSKTPYVAVLLSTGAAFLTVVVNYYAPAKVFKFLIDSSGAIALLVYLVIAVSQLRMRKILRAEGSEIRLRMWLYPWLTWLVIGFITFVLVVMLFRPAQQLEVISTGLLAIGIICTVPIMARWKKLVLWQKTPVHNTR',\n",
       "  'mw': 51080.0},\n",
       " 'b2675': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P39452',\n",
       "  'gene': 'b2675',\n",
       "  'locus': ['b2675', 'JW2650'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [(\"[thioredoxin]-disulfide + a 2'-deoxyribonucleoside 5'-diphosphate + H2O = [thioredoxin]-dithiol + a ribonucleoside 5'-diphosphate\",\n",
       "     '1.17.4.1')],\n",
       "   'Function': 'Provides the precursors necessary for DNA synthesis. Catalyzes the biosynthesis of deoxyribonucleotides from the corresponding ribonucleotides. R1E contains the binding sites for both substrates and allosteric effectors and carries out the actual reduction of the ribonucleotide',\n",
       "   'Regulation Activity': 'Under complex allosteric control mediated by deoxynucleoside triphosphates and ATP binding. The type of nucleotide bound at the specificity site determines substrate preference. It seems probable that ATP makes the enzyme reduce CDP and UDP, dGTP favors ADP reduction and dTTP favors GDP reduction (By similarity)',\n",
       "   'Pathway': 'Genetic information processing; DNA replication'},\n",
       "  'seq': 'MATTTAECLTQETMDYHALNAMLNLYDSAGRIQFDKDRQAVDAFIATHVRPNSVTFSSQQQRLNWLVNEGYYDESVLNRYSRDFVITLFTHAHTSGFRFQTFLGAWKFYTSYTLKTFDGKRYLEDFADRVTMVALTLAQGDETLALQLTDEMLSGRFQPATPTFLNCGKQQRGELVSCFLLRIEDNMESIGRAVNSALQLSKRGGGVAFLLSNLREAGAPIKRIENQSSGVIPVMKMLEDAFSYANQLGARQGAGAVYLHAHHPDILRFLDTKRENADEKIRIKTLSLGVVIPDITFHLAKENAQMALFSPYDVERVYGKPFADVAISQHYDELVADERIRKKYLNARDFFQRLAEIQFESGYPYIMYEDTVNRANPIAGRINMSNLCSEILQVNSASEYDENLDYTRTGHDISCNLGSLNIAHTMDSPDFARTVETAVRGLTAVSDMSHIRSVPSIEAGNAASHAIGLGQMNLHGYLAREGIAYGSPEALDFTNLYFYAITWHALRTSMLLARERGETFAGFKQSRYASGEYFSQYLQGNWQPKTAKVGELFTRSGITLPTREMWAQLRDDVMRYGIYNQNLQAVPPTGSISYINHATSSIHPIVAKVEIRKEGKTGRVYYPAPFMTNENLALYQDAYEIGAEKIIDTYAEATRHVDQGLSLTLFFPDTATTRDINKAQIYAWRKGIKTLYYIRLRQMALEGTEIEGCVSCAL',\n",
       "  'mw': 80479.0},\n",
       " 'b2676': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37146',\n",
       "  'gene': 'b2676',\n",
       "  'locus': ['b2676', 'JW2651'],\n",
       "  'synonyms': 'ygaD',\n",
       "  'properties': {'Catalytic Activity': [(\"[thioredoxin]-disulfide + a 2'-deoxyribonucleoside 5'-diphosphate + H2O = [thioredoxin]-dithiol + a ribonucleoside 5'-diphosphate\",\n",
       "     '1.17.4.1')],\n",
       "   'Function': 'Provides the precursors necessary for DNA synthesis. Catalyzes the biosynthesis of deoxyribonucleotides from the corresponding ribonucleotides. R2F contains the tyrosyl radical required for catalysis',\n",
       "   'Pathway': 'Genetic information processing; DNA replication'},\n",
       "  'seq': 'MKLSRISAINWNKISDDKDLEVWNRLTSNFWLPEKVPLSNDIPAWQTLTVVEQQLTMRVFTGLTLLDTLQNVIGAPSLMPDALTPHEEAVLSNISFMEAVHARSYSSIFSTLCQTKDVDAAYAWSEENAPLQRKAQIIQQHYRGDDPLKKKIASVFLESFLFYSGFWLPMYFSSRGKLTNTADLIRLIIRDEAVHGYYIGYKYQKNMEKISLGQREELKSFAFDLLLELYDNELQYTDELYAETPWADDVKAFLCYNANKALMNLGYEPLFPAEMAEVNPAILAALSPNADENHDFFSGSGSSYVMGKAVETEDEDWNF',\n",
       "  'mw': 36443.0},\n",
       " 'b2677': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P14175',\n",
       "  'gene': 'b2677',\n",
       "  'locus': ['b2677', 'JW2652'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the ProU ABC transporter complex involved in glycine betaine and proline betaine uptake (PubMed:3305496, PubMed:7898450, PubMed:23249124). Probably responsible for energy coupling to the transport system (Probable)'},\n",
       "  'seq': 'MAIKLEIKNLYKIFGEHPQRAFKYIEQGLSKEQILEKTGLSLGVKDASLAIEEGEIFVIMGLSGSGKSTMVRLLNRLIEPTRGQVLIDGVDIAKISDAELREVRRKKIAMVFQSFALMPHMTVLDNTAFGMELAGINAEERREKALDALRQVGLENYAHSYPDELSGGMRQRVGLARALAINPDILLMDEAFSALDPLIRTEMQDELVKLQAKHQRTIVFISHDLDEAMRIGDRIAIMQNGEVVQVGTPDEILNNPANDYVRTFFRGVDISQVFSAKDIARRTPNGLIRKTPGFGPRSALKLLQDEDREYGYVIERGNKFVGAVSIDSLKTALTQQQGLDAALIDAPLAVDAQTPLSELLSHVGQAPCAVPVVDEDQQYVGIISKGMLLRALDREGVNNG',\n",
       "  'mw': 44163.0},\n",
       " 'b2678': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P14176',\n",
       "  'gene': 'b2678',\n",
       "  'locus': ['b2678', 'JW2653'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the ProU ABC transporter complex involved in glycine betaine and proline betaine uptake (PubMed:3305496, PubMed:7898450, PubMed:23249124). Probably responsible for the translocation of the substrate across the membrane (Probable)'},\n",
       "  'seq': 'MADQNNPWDTTPAADSAAQSADAWGTPTTAPTDGGGADWLTSTPAPNVEHFNILDPFHKTLIPLDSWVTEGIDWVVTHFRPVFQGVRVPVDYILNGFQQLLLGMPAPVAIIVFALIAWQISGVGMGVATLVSLIAIGAIGAWSQAMVTLALVLTALLFCIVIGLPLGIWLARSPRAAKIIRPLLDAMQTTPAFVYLVPIVMLFGIGNVPGVVVTIIFALPPIIRLTILGINQVPADLIEASRSFGASPRQMLFKVQLPLAMPTIMAGVNQTLMLALSMVVIASMIAVGGLGQMVLRGIGRLDMGLATVGGVGIVILAIILDRLTQAVGRDSRSRGNRRWYTTGPVGLLTRPFIK',\n",
       "  'mw': 37620.0},\n",
       " 'b2679': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AFM2',\n",
       "  'gene': 'b2679',\n",
       "  'locus': ['b2679', 'JW2654'],\n",
       "  'synonyms': 'proU',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the ProU ABC transporter complex involved in glycine betaine and proline betaine uptake. Binds glycine betaine and proline betaine with high affinity'},\n",
       "  'seq': 'MRHSVLFATAFATLISTQTFAADLPGKGITVNPVQSTITEETFQTLLVSRALEKLGYTVNKPSEVDYNVGYTSLASGDATFTAVNWTPLHDNMYEAAGGDKKFYREGVFVNGAAQGYLIDKKTADQYKITNIAQLKDPKIAKLFDTNGDGKADLTGCNPGWGCEGAINHQLAAYELTNTVTHNQGNYAAMMADTISRYKEGKPVFYYTWTPYWVSNELKPGKDVVWLQVPFSALPGDKNADTKLPNGANYGFPVSTMHIVANKAWAEKNPAAAKLFAIMQLPVADINAQNAIMHDGKASEGDIQGHVDGWIKAHQQQFDGWVNEALAAQK',\n",
       "  'mw': 36023.0},\n",
       " 'b2687': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P45578',\n",
       "  'gene': 'b2687',\n",
       "  'locus': ['b2687', 'JW2662'],\n",
       "  'synonyms': 'ygaG',\n",
       "  'properties': {'Catalytic Activity': [('S-(5-deoxy-D-ribos-5-yl)-L-homocysteine = (S)-4,5-dihydroxypentane-2,3-dione + L-homocysteine',\n",
       "     '4.4.1.21')],\n",
       "   'Function': 'Involved in the synthesis of autoinducer 2 (AI-2) which is secreted by bacteria and is used to communicate both the cell density and the metabolic potential of the environment. The regulation of gene expression in response to changes in cell density is called quorum sensing. Catalyzes the transformation of S-ribosylhomocysteine (RHC) to homocysteine (HC) and 4,5-dihydroxy-2,3-pentadione (DPD)'},\n",
       "  'seq': 'MPLLDSFTVDHTRMEAPAVRVAKTMNTPHGDAITVFDLRFCVPNKEVMPERGIHTLEHLFAGFMRNHLNGNGVEIIDISPMGCRTGFYMSLIGTPDEQRVADAWKAAMEDVLKVQDQNQIPELNVYQCGTYQMHSLQEAQDIARSILERDVRINSNEELALPKEKLQELHI',\n",
       "  'mw': 19416.0},\n",
       " 'b2688': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A6W9',\n",
       "  'gene': 'b2688',\n",
       "  'locus': ['b2688', 'JW2663'],\n",
       "  'synonyms': 'gsh-I',\n",
       "  'properties': {'Catalytic Activity': [('ATP + L-cysteine + L-glutamate = ADP + gamma-L-glutamyl-L-cysteine + H(+) + phosphate',\n",
       "     '6.3.2.2')],\n",
       "   'Pathway': 'Sulfur metabolism; glutathione biosynthesis; glutathione from L-cysteine and L-glutamate: step 1/2'},\n",
       "  'seq': 'MIPDVSQALAWLEKHPQALKGIQRGLERETLRVNADGTLATTGHPEALGSALTHKWITTDFAEALLEFITPVDGDIEHMLTFMRDLHRYTARNMGDERMWPLSMPCYIAEGQDIELAQYGTSNTGRFKTLYREGLKNRYGALMQTISGVHYNFSLPMAFWQAKCGDISGADAKEKISAGYFRVIRNYYRFGWVIPYLFGASPAICSSFLQGKPTSLPFEKTECGMYYLPYATSLRLSDLGYTNKSQSNLGITFNDLYEYVAGLKQAIKTPSEEYAKIGIEKDGKRLQINSNVLQIENELYAPIRPKRVTRSGESPSDALLRGGIEYIEVRSLDINPFSPIGVDEQQVRFLDLFMVWCALADAPEMSSSELACTRVNWNRVILEGRKPGLTLGIGCETAQFPLPQVGKDLFRDLKRVAQTLDSINGGEAYQKVCDELVACFDNPDLTFSARILRSMIDTGIGGTGKAFAEAYRNLLREEPLEILREEDFVAEREASERRQQEMEAADTEPFAVWLEKHA',\n",
       "  'mw': 58269.0},\n",
       " 'b2690': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P77475',\n",
       "  'gene': 'b2690',\n",
       "  'locus': ['b2690', 'JW2665'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Catalyzes strongly the dephosphorylation of fructose-1-phosphate (Fru1P) and slightly the dephosphorylation of 6-phosphogluconate (6P-Glu). It has low beta-phosphoglucomutase activity'},\n",
       "  'seq': 'MYERYAGLIFDMDGTILDTEPTHRKAWREVLGHYGLQYDIQAMIALNGSPTWRIAQAIIELNQADLDPHALAREKTEAVRSMLLDSVEPLPLVDVVKSWHGRRPMAVGTGSESAIAEALLAHLGLRHYFDAVVAADHVKHHKPAPDTFLLCAQRMGVQPTQCVVFEDADFGIQAARAAGMDAVDVRLL',\n",
       "  'mw': 20780.0},\n",
       " 'b2697': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P00957',\n",
       "  'gene': 'b2697',\n",
       "  'locus': ['b2697', 'JW2667'],\n",
       "  'synonyms': 'lovB',\n",
       "  'properties': {'Catalytic Activity': [('ATP + L-alanine + tRNA(Ala) = AMP + diphosphate + L-alanyl-tRNA(Ala)',\n",
       "     '6.1.1.7')],\n",
       "   'Function': 'Attaches Ala to transfer-messenger RNA (tmRNA, also known as 10Sa RNA, the product of the ssrA gene). tmRNA plays a major role in rescue of stalled ribosomes via trans-translation'},\n",
       "  'seq': 'MSKSTAEIRQAFLDFFHSKGHQVVASSSLVPHNDPTLLFTNAGMNQFKDVFLGLDKRNYSRATTSQRCVRAGGKHNDLENVGYTARHHTFFEMLGNFSFGDYFKHDAIQFAWELLTSEKWFALPKERLWVTVYESDDEAYEIWEKEVGIPRERIIRIGDNKGAPYASDNFWQMGDTGPCGPCTEIFYDHGDHIWGGPPGSPEEDGDRYIEIWNIVFMQFNRQADGTMEPLPKPSVDTGMGLERIAAVLQHVNSNYDIDLFRTLIQAVAKVTGATDLSNKSLRVIADHIRSCAFLIADGVMPSNENRGYVLRRIIRRAVRHGNMLGAKETFFYKLVGPLIDVMGSAGEDLKRQQAQVEQVLKTEEEQFARTLERGLALLDEELAKLSGDTLDGETAFRLYDTYGFPVDLTADVCRERNIKVDEAGFEAAMEEQRRRAREASGFGADYNAMIRVDSASEFKGYDHLELNGKVTALFVDGKAVDAINAGQEAVVVLDQTPFYAESGGQVGDKGELKGANFSFAVEDTQKYGQAIGHIGKLAAGSLKVGDAVQADVDEARRARIRLNHSATHLMHAALRQVLGTHVSQKGSLVNDKVLRFDFSHNEAMKPEEIRAVEDLVNTQIRRNLPIETNIMDLEAAKAKGAMALFGEKYDERVRVLSMGDFSTELCGGTHASRTGDIGLFRIISESGTAAGVRRIEAVTGEGAIATVHADSDRLSEVAHLLKGDSNNLADKVRSVLERTRQLEKELQQLKEQAAAQESANLSSKAIDVNGVKLLVSELSGVEPKMLRTMVDDLKNQLGSTIIVLATVVEGKVSLIAGVSKDVTDRVKAGELIGMVAQQVGGKGGGRPDMAQAGGTDAAALPAALASVKGWVSAKLQ',\n",
       "  'mw': 96032.0},\n",
       " 'b2701': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P41052',\n",
       "  'gene': 'b2701',\n",
       "  'locus': ['b2701', 'JW2671'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('Exolytic cleavage of the (1->4)-beta-glycosidic linkage between N-acetylmuramic acid (MurNAc) and N-acetylglucosamine (GlcNAc) residues in peptidoglycan, from either the reducing or the non-reducing ends of the peptidoglycan chains, with concomitant formation of a 1,6-anhydrobond in the MurNAc residue.',\n",
       "     '4.2.2.n1')],\n",
       "   'Function': 'Murein-degrading enzyme. Catalyzes the cleavage of the glycosidic bonds between N-acetylmuramic acid and N-acetylglucosamine residues in peptidoglycan. May play a role in recycling of muropeptides during cell elongation and/or cell division'},\n",
       "  'seq': 'MFKRRYVTLLPLFVLLAACSSKPKPTETDTTTGTPSGGFLLEPQHNVMQMGGDFANNPNAQQFIDKMVNKHGFDRQQLQEILSQAKRLDSVLRLMDNQAPTTSVKPPSGPNGAWLRYRKKFITPDNVQNGVVFWNQYEDALNRAWQVYGVPPEIIVGIIGVETRWGRVMGKTRILDALATLSFNYPRRAEYFSGELETFLLMARDEQDDPLNLKGSFAGAMGYGQFMPSSYKQYAVDFSGDGHINLWDPVDAIGSVANYFKAHGWVKGDQVAVMANGQAPGLPNGFKTKYSISQLAAAGLTPQQPLGNHQQASLLRLDVGTGYQYWYGLPNFYTITRYNHSTHYAMAVWQLGQAVALARVQ',\n",
       "  'mw': 40256.0},\n",
       " 'b2702': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P56579',\n",
       "  'gene': 'b2702',\n",
       "  'locus': ['b2702', 'JW5429'],\n",
       "  'synonyms': 'gutA sbl',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'The phosphoenolpyruvate-dependent sugar phosphotransferase system (PTS), a major carbohydrate active transport system, catalyzes the phosphorylation of incoming sugar substrates concomitant with their translocation across the cell membrane. The enzyme II complex composed of SrlA, SrlB and SrlE is involved in glucitol/sorbitol transport. It can also use D-mannitol'},\n",
       "  'seq': 'MIETITHGAEWFIGLFQKGGEVFTGMVTGILPLLISLLVIMNALINFIGQHRIERFAQRCAGNPVSRYLLLPCIGTFVFCNPMTLSLGRFMPEKYKPSYYAAASYSCHSMNGLFPHINPGELFVYLGIASGLTTLNLPLGPLAVSYLLVGLVTNFFRGWVTDLTTAIFEKKMGIQLEQKVHLAGATS',\n",
       "  'mw': 20580.0},\n",
       " 'b2703': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P56580',\n",
       "  'gene': 'b2703',\n",
       "  'locus': ['b2703', 'JW5430'],\n",
       "  'synonyms': 'gutA gutE',\n",
       "  'properties': {'Catalytic Activity': [('D-sorbitol(out) + N(pros)-phospho-L-histidyl-[protein] = D-sorbitol 6-phosphate(in) + L-histidyl-[protein]',\n",
       "     '2.7.1.198')],\n",
       "   'Function': 'The phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS), a major carbohydrate active transport system, catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane. The enzyme II complex composed of SrlA, SrlB and SrlE is involved in glucitol/sorbitol transport. It can also use D-mannitol'},\n",
       "  'seq': 'MTHIRIEKGTGGWGGPLELKATPGKKIVYITAGTRPAIVDKLAQLTGWQAIDGFKEGEPAEAEIGVAVIDCGGTLRCGIYPKRRIPTINIHSTGKSGPLAQYIVEDIYVSGVKEENITVVGDATPQPSSVGRDYDTSKKITEQSDGLLAKVGMGMGSTVAVLFQSGRDTIDTVLKTILPFMAFVSALIGIIMASGLGDWIAHGLAPLASHPLGLVMLALICSFPLLSPFLGPGAVIAQVIGVLIGVQIGLGNIPPHLALPALFAINAQAACDFIPVGLSLAEARQDTVRVGVPSVLVSRFLTGAPTVLIAWFVSGFIYQ',\n",
       "  'mw': 33332.0},\n",
       " 'b2704': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P05706',\n",
       "  'gene': 'b2704',\n",
       "  'locus': ['b2704', 'JW2673'],\n",
       "  'synonyms': 'gutB',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'The phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS), a major carbohydrate active transport system, catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane. The enzyme II complex composed of SrlA, SrlB and SrlE is involved in glucitol/sorbitol transport. It can also use D-mannitol'},\n",
       "  'seq': 'MTVIYQTTITRIGASAIDALSDQMLITFREGAPADLEEYCFIHCHGELKGALHPGLQFSLGQHRYPVTAVGSVAEDNLRELGHVTLRFDGLNEAEFPGTVHVAGPVPDDIAPGSVLKFESVKE',\n",
       "  'mw': 13304.0},\n",
       " 'b2705': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P05707',\n",
       "  'gene': 'b2705',\n",
       "  'locus': ['b2705', 'JW2674'],\n",
       "  'synonyms': 'gutD',\n",
       "  'properties': {'Catalytic Activity': [('D-sorbitol 6-phosphate + NAD(+) = beta-D-fructose 6-phosphate + H(+) + NADH',\n",
       "     '1.1.1.140')],\n",
       "   'Pathway': 'Carbohydrate metabolism; D-sorbitol degradation; D-fructose 6-phosphate from D-sorbitol 6-phosphate: step 1/1'},\n",
       "  'seq': 'MNQVAVVIGGGQTLGAFLCHGLAAEGYRVAVVDIQSDKAANVAQEINAEYGESMAYGFGADATSEQSVLALSRGVDEIFGRVDLLVYSAGIAKAAFISDFQLGDFDRSLQVNLVGYFLCAREFSRLMIRDGIQGRIIQINSKSGKVGSKHNSGYSAAKFGGVGLTQSLALDLAEYGITVHSLMLGNLLKSPMFQSLLPQYATKLGIKPDQVEQYYIDKVPLKRGCDYQDVLNMLLFYASPKASYCTGQSINVTGGQVMF',\n",
       "  'mw': 27858.0},\n",
       " 'b2708': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P17115',\n",
       "  'gene': 'b2708',\n",
       "  'locus': ['b2708', 'JW5431'],\n",
       "  'synonyms': 'srlQ',\n",
       "  'properties': {'Catalytic Activity': [('D-arabinose 5-phosphate = D-ribulose 5-phosphate',\n",
       "     '5.3.1.13')],\n",
       "   'Function': 'Catalyzes the reversible aldol-ketol isomerization between D-ribulose 5-phosphate (Ru5P) and D-arabinose 5-phosphate (A5P). It appears that the physiological function of G-API may be to synthesize the regulatory molecule A5P, which in turn participates in the induction of the gut operon through an unknown mechanism. It is also able of sustaining the biosynthetic pathway of 3-deoxy-D-manno-octulosonate (KDO), a unique 8-carbon sugar component of lipopolysaccharides (LPSs)'},\n",
       "  'seq': 'MSEALLNAGRQTLMLELQEASRLPERLGDDFVRAANIILHCEGKVVVSGIGKSGHIGKKIAATLASTGTPAFFVHPAEALHGDLGMIESRDVMLFISYSGGAKELDLIIPRLEDKSIALLAMTGKPTSPLGLAAKAVLDISVEREACPMHLAPTSSTVNTLMMGDALAMAVMQARGFNEEDFARSHPAGALGARLLNKVHHLMRRDDAIPQVALTASVMDAMLELSRTGLGLVAVCDAQQQVQGVFTDGDLRRWLVGGGALTTPVNEAMTVGGTTLQSQSRAIDAKEILMKRKITAAPVVDENGKLTGAINLQDFYQAGII',\n",
       "  'mw': 34031.0},\n",
       " 'b2710': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'Q46877',\n",
       "  'gene': 'b2710',\n",
       "  'locus': ['b2710', 'JW2680'],\n",
       "  'synonyms': 'flrD ygaI ygaJ ygaK',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Anaerobic nitric oxide reductase; uses NADH to detoxify nitric oxide (NO), protecting several 4Fe-4S NO-sensitive enzymes. Has at least 2 reductase partners, only one of which (NorW, flavorubredoxin reductase) has been identified. NO probably binds to the di-iron center; electrons enter from the reductase at rubredoxin and are transferred sequentially to the FMN center and the di-iron center. Also able to function as an aerobic oxygen reductase',\n",
       "   'Pathway': 'Nitrogen metabolism; nitric oxide reduction'},\n",
       "  'seq': 'MSIVVKNNIHWVGQRDWEVRDFHGTEYKTLRGSSYNSYLIREEKNVLIDTVDHKFSREFVQNLRNEIDLADIDYIVINHAEEDHAGALTELMAQIPDTPIYCTANAIDSINGHHHHPEWNFNVVKTGDTLDIGNGKQLIFVETPMLHWPDSMMTYLTGDAVLFSNDAFGQHYCDEHLFNDEVDQTELFEQCQRYYANILTPFSRLVTPKITEILGFNLPVDMIATSHGVVWRDNPTQIVELYLKWAADYQEDRITIFYDTMSNNTRMMADAIAQGIAETDPRVAVKIFNVARSDKNEILTNVFRSKGVLVGTSTMNNVMMPKIAGLVEEMTGLRFRNKRASAFGSHGWSGGAVDRLSTRLQDAGFEMSLSLKAKWRPDQDALKLCREHGREIARQWALAPLPQSTVNTVVKEETSATTTADLGPRMQCSVCQWIYDPAKGEPMQDVAPGTPWSEVPDNFLCPECSLGKDVFEELASEAK',\n",
       "  'mw': 54234.0},\n",
       " 'b2711': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37596',\n",
       "  'gene': 'b2711',\n",
       "  'locus': ['b2711', 'JW2681'],\n",
       "  'synonyms': 'flrR ygbD',\n",
       "  'properties': {'Catalytic Activity': [('H(+) + NAD(+) + 2 reduced [nitric oxide reductase rubredoxin domain] = NADH + 2 oxidized [nitric oxide reductase rubredoxin domain]',\n",
       "     '')],\n",
       "   'Function': 'One of at least two accessory proteins for anaerobic nitric oxide (NO) reductase. Reduces the rubredoxin moiety of NO reductase',\n",
       "   'Pathway': 'Nitrogen metabolism; nitric oxide reduction'},\n",
       "  'seq': 'MSNGIVIIGSGFAARQLVKNIRKQDATIPLTLIAADSMDEYNKPDLSHVISQGQRADDLTRQTAGEFAEQFNLHLFPQTWVTDIDAEARVVKSQNNQWQYDKLVLATGASAFVPPVPGRELMLTLNSQQEYRACETQLRDARRVLIVGGGLIGSELAMDFCRAGKAVTLIDNAASILASLMPPEVSSRLQHRLTEMGVHLLLKSQLQGLEKTDSGIQATLDRQRNIEVDAVIAATGLRPETALARRAGLTINRGVCVDSYLQTSNTDIYALGDCAEINGQVLPFLQPIQLSAMVLAKNLLGNNTPLKLPAMLVKIKTPELPLHLAGETQRQDLRWQINTERQGMVARGVDDADQLRAFVVSEDRMKEAFGLLKTLPM',\n",
       "  'mw': 41404.0},\n",
       " 'b2719': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P16433',\n",
       "  'gene': 'b2719',\n",
       "  'locus': ['b2719', 'JW2689'],\n",
       "  'synonyms': 'hevG',\n",
       "  'properties': {'Catalytic Activity': []},\n",
       "  'seq': 'MSNLLGPRDANGIPVPMTVDESIASMKASLLKKIKRSAYVYRVDCGGCNGCEIEIFATLSPLFDAERFGIKVVPSPRHADILLFTGAVTRAMRSPALRAWQSAPDPKICISYGACGNSGGIFHDLYCVWGGTDKIVPVDVYIPGCPPTPAATLYGFAMALGLLEQKIHARGPGELDEQPAEILHGDMVQPLRVKVDREARRLAGYRYGRQIADDYLTQLGQGEEQVARWLEAENDPRLNEIVSHLNHVVEEARIR',\n",
       "  'mw': 27999.0},\n",
       " 'b2720': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P16432',\n",
       "  'gene': 'b2720',\n",
       "  'locus': ['b2720', 'JW2690'],\n",
       "  'synonyms': 'hevF',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Probable electron transfer protein for hydrogenase 3'},\n",
       "  'seq': 'MFTFIKKVIKTGTATSSYPLEPIAVDKNFRGKPEQNPQQCIGCAACVNACPSNALTVETDLATGELAWEFNLGHCIFCGRCEEVCPTAAIKLSQEYELAVWKKEDFLQQSRFALCNCRVCNRPFAVQKEIDYAIALLKHNGDSRAENHRESFETCPECKRQKCLVPSDRIELTRHMKEAI',\n",
       "  'mw': 20309.0},\n",
       " 'b2721': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P16431',\n",
       "  'gene': 'b2721',\n",
       "  'locus': ['b2721', 'JW2691'],\n",
       "  'synonyms': 'hevE',\n",
       "  'properties': {'Catalytic Activity': []},\n",
       "  'seq': 'MSEEKLGQHYLAALNEAFPGVVLDHAWQTKDQLTVTVKVNYLPEVVEFLYYKQGGWLSVLFGNDERKLNGHYAVYYVLSMEKGTKCWITVRVEVDANKPEYPSVTPRVPAAVWGEREVRDMYGLIPVGLPDERRLVLPDDWPDELYPLRKDSMDYRQRPAPTTDAETYEFINELGDKKNNVVPIGPLHVTSDEPGHFRLFVDGENIIDADYRLFYVHRGMEKLAETRMGYNEVTFLSDRVCGICGFAHSTAYTTSVENAMGIQVPERAQMIRAILLEVERLHSHLLNLGLACHFTGFDSGFMQFFRVRETSMKMAEILTGARKTYGLNLIGGIRRDLLKDDMIQTRQLAQQMRREVQELVDVLLSTPNMEQRTVGIGRLDPEIARDFSNVGPMVRASGHARDTRADHPFVGYGLLPMEVHSEQGCDVISRLKVRINEVYTALNMIDYGLDNLPGGPLMVEGFTYIPHRFALGFAEAPRGDDIHWSMTGDNQKLYRWRCRAATYANWPTLRYMLRGNTVSDAPLIIGSLDPCYSCTDRMTVVDVRKKKSKVVPYKELERYSIERKNSPLK',\n",
       "  'mw': 64980.0},\n",
       " 'b2722': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P16430',\n",
       "  'gene': 'b2722',\n",
       "  'locus': ['b2722', 'JW2692'],\n",
       "  'synonyms': 'hevD',\n",
       "  'properties': {'Catalytic Activity': []},\n",
       "  'seq': 'MSVLYPLIQALVLFAVAPLLSGITRVARARLHNRRGPGVLQEYRDIIKLLGRQSVGPDASGWVFRLTPYVMVGVMLTIATALPVVTVGSPLPQLGDLITLLYLFAIARFFFAISGLDTGSPFTAIGASREAMLGVLVEPMLLLGLWVAAQVAGSTNISNITDTVYHWPLSQSIPLVLALCACAFATFIEMGKLPFDLAEAEQELQEGPLSEYSGSGFGVMKWGISLKQLVVLQMFVGVFIPWGQMETFTAGGLLLALVIAIVKLVVGVLVIALFENSMARLRLDITPRITWAGFGFAFLAFVSLLAA',\n",
       "  'mw': 33029.0},\n",
       " 'b2723': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P16429',\n",
       "  'gene': 'b2723',\n",
       "  'locus': ['b2723', 'JW2693'],\n",
       "  'synonyms': 'hevC',\n",
       "  'properties': {'Catalytic Activity': []},\n",
       "  'seq': 'MSAISLINSGVAWFVAAAVLAFLFSFQKALSGWIAGIGGAVGSLYTAAAGFTVLTGAVGVSGALSLVSYDVQISPLNAIWLITLGLCGLFVSLYNIDWHRHAQVKCNGLQINMLMAAAVCAVIASNLGMFVVMAEIMALCAVFLTSNSKEGKLWFALGRLGTLLLAIACWLLWQRYGTLDLRLLDMRMQQLPLGSDIWLLGVIGFGLLAGIIPLHGWVPQAHANASAPAAALFSTVVMKIGLLGILTLSLLGGNAPLWWGIALLVLGMITAFVGGLYALVEHNIQRLLAYHTLENIGIILLGLGAGVTGIALEQPALIALGLVGGLYHLLNHSLFKSVLFLGAGSVWFRTGHRDIEKLGGIGKKMPVISIAMLVGLMAMAALPPLNGFAGEWVIYQSFFKLSNSGAFVARLLGPLLAVGLAITGALAVMCMAKVYGVTFLGAPRTKEAENATCAPLLMSVSVVALAICCVIGGVAAPWLLPMLSAAVPLPLEPANTTVSQPMITLLLIACPLLPFIIMAICKGDRLPSRSRGAAWVCGYDHEKSMVITAHGFAMPVKQAFAPVLKLRKWLNPVSLVPGWQCEGSALLFRRMALVELAVLVVIIVSRGA',\n",
       "  'mw': 64077.0},\n",
       " 'b2724': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AAK1',\n",
       "  'gene': 'b2724',\n",
       "  'locus': ['b2724', 'JW2694'],\n",
       "  'synonyms': 'hevB',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Probable electron transfer protein for hydrogenase 3'},\n",
       "  'seq': 'MNRFVIADSTLCIGCHTCEAACSETHRQHGLQSMPRLRVMLNEKESAPQLCHHCEDAPCAVVCPVNAITRVDGAVQLNESLCVSCKLCGIACPFGAIEFSGSRPLDIPANANTPKAPPAPPAPARVSTLLDWVPGIRAIAVKCDLCSFDEQGPACVRMCPTKALHLVDNTDIARVSKRKRELTFNTDFGDLTLFQQAQSGEAK',\n",
       "  'mw': 21873.0},\n",
       " 'b2738': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'Q46890',\n",
       "  'gene': 'b2738',\n",
       "  'locus': ['b2738', 'JW2708'],\n",
       "  'synonyms': 'ygbL',\n",
       "  'properties': {'Catalytic Activity': [('3-dehydro-4-O-phospho-D-erythronate + H(+) = CO2 + dihydroxyacetone phosphate',\n",
       "     '4.1.1.104'),\n",
       "    ('3-dehydro-4-O-phospho-L-erythronate + H(+) = CO2 + dihydroxyacetone phosphate',\n",
       "     '4.1.1.104')],\n",
       "   'Function': 'Catalyzes the decarboxylation of 3-oxo-tetronate 4-phosphate to dihydroxyacetone phosphate (DHAP) and CO(2)'},\n",
       "  'seq': 'MSDFAKVEQSLREEMTRIASSFFQRGYATGSAGNLSLLLPDGNLLATPTGSCLGNLDPQRLSKVAADGEWLSGDKPSKEVLFHLALYRNNPRCKAVVHLHSTWSTALSCLQGLDSSNVIRPFTPYVVMRMGNVPLVPYYRPGDKRIAQDLAELAADNQAFLLANHGPVVCGESLQEAANNMEELEETAKLIFILGDRPIRYLTAGEIAELRS',\n",
       "  'mw': 23222.0},\n",
       " 'b2744': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A840',\n",
       "  'gene': 'b2744',\n",
       "  'locus': ['b2744', 'JW2714'],\n",
       "  'synonyms': 'ygbC',\n",
       "  'properties': {'Catalytic Activity': [(\"a ribonucleoside 5'-phosphate + H2O = a ribonucleoside + phosphate\",\n",
       "     '3.1.3.5'),\n",
       "    (\"a ribonucleoside 3'-phosphate + H2O = a ribonucleoside + phosphate\",\n",
       "     '3.1.3.6'),\n",
       "    ('[phosphate](n) + H2O = [phosphate](n-1) + H(+) + phosphate', '3.6.1.11'),\n",
       "    ('AMP + H2O = adenosine + phosphate', ''),\n",
       "    ('GMP + H2O = guanosine + phosphate', ''),\n",
       "    (\"dGMP + H2O = 2'-deoxyguanosine + phosphate\", ''),\n",
       "    (\"3'-AMP + H2O = adenosine + phosphate\", ''),\n",
       "    (\"3'-CMP + H2O = cytidine + phosphate\", '')],\n",
       "   'Function': \"Nucleotidase with a broad substrate specificity as it can dephosphorylate various ribo- and deoxyribonucleoside 5'-monophosphates and ribonucleoside 3'-monophosphates with highest affinity to 3'-AMP (PubMed:15489502). Also hydrolyzes polyphosphate (exopolyphosphatase activity) with the preference for short-chain-length substrates (P20-25) (PubMed:15489502). Might be involved in the regulation of dNTP and NTP pools, and in the turnover of 3'-mononucleotides produced by numerous intracellular RNases (T1, T2, and F) during the degradation of various RNAs (PubMed:15489502). Also plays a significant physiological role in stress-response and is required for the survival of E.coli in stationary growth phase (PubMed:15489502)\",\n",
       "   'Regulation Activity': \"Inhibited by various ribo- or deoxyribonucleoside 5'-triphosphates but is insensitive to nucleoside diphosphates\"},\n",
       "  'seq': 'MRILLSNDDGVHAPGIQTLAKALREFADVQVVAPDRNRSGASNSLTLESSLRTFTFENGDIAVQMGTPTDCVYLGVNALMRPRPDIVVSGINAGPNLGDDVIYSGTVAAAMEGRHLGFPALAVSLDGHKHYDTAAAVTCSILRALCKEPLRTGRILNINVPDLPLDQIKGIRVTRCGTRHPADQVIPQQDPRGNTLYWIGPPGGKCDAGPGTDFAAVDEGYVSITPLHVDLTAHSAQDVVSDWLNSVGVGTQW',\n",
       "  'mw': 26900.0},\n",
       " 'b2746': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P62617',\n",
       "  'gene': 'b2746',\n",
       "  'locus': ['b2746', 'JW2716'],\n",
       "  'synonyms': 'mecS ygbB',\n",
       "  'properties': {'Catalytic Activity': [('4-CDP-2-C-methyl-D-erythritol 2-phosphate = 2-C-methyl-D-erythritol 2,4-cyclic diphosphate + CMP',\n",
       "     '4.6.1.12'),\n",
       "    ('4-CDP-2-C-methyl-D-erythritol = 2-C-methyl-D-erythritol 3,4-cyclophosphate + CMP + H(+)',\n",
       "     '')],\n",
       "   'Function': 'Involved in the biosynthesis of isopentenyl diphosphate (IPP) and dimethylallyl diphosphate (DMAPP), two major building blocks of isoprenoid compounds. Catalyzes the conversion of 4-diphosphocytidyl-2-C-methyl-D-erythritol 2-phosphate (CDP-ME2P) to 2-C-methyl-D-erythritol 2,4-cyclodiphosphate (ME-CPP) with a corresponding release of cytidine 5-monophosphate (CMP). Also converts 4-diphosphocytidyl-2-C-methyl-D-erythritol into 2-C-methyl-D-erythritol 3,4-cyclophosphate and CMP',\n",
       "   'Regulation Activity': 'Activated by 2C-methyl-D-erythritol 4-phosphate (MEP)',\n",
       "   'Pathway': 'Isoprenoid biosynthesis; isopentenyl diphosphate biosynthesis via DXP pathway; isopentenyl diphosphate from 1-deoxy-D-xylulose 5-phosphate: step 4/6'},\n",
       "  'seq': 'MRIGHGFDVHAFGGEGPIIIGGVRIPYEKGLLAHSDGDVALHALTDALLGAAALGDIGKLFPDTDPAFKGADSRELLREAWRRIQAKGYTLGNVDVTIIAQAPKMLPHIPQMRVFIAEDLGCHMDDVNVKATTTEKLGFTGRGEGIACEAVALLIKATK',\n",
       "  'mw': 16898.0},\n",
       " 'b2747': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'Q46893',\n",
       "  'gene': 'b2747',\n",
       "  'locus': ['b2747', 'JW2717'],\n",
       "  'synonyms': 'ygbP',\n",
       "  'properties': {'Catalytic Activity': [('2-C-methyl-D-erythritol 4-phosphate + CTP + H(+) = 4-CDP-2-C-methyl-D-erythritol + diphosphate',\n",
       "     '2.7.7.60')],\n",
       "   'Function': 'Catalyzes the formation of 4-diphosphocytidyl-2-C-methyl-D-erythritol from CTP and 2-C-methyl-D-erythritol 4-phosphate (MEP)',\n",
       "   'Pathway': 'Isoprenoid biosynthesis; isopentenyl diphosphate biosynthesis via DXP pathway; isopentenyl diphosphate from 1-deoxy-D-xylulose 5-phosphate: step 2/6'},\n",
       "  'seq': 'MATTHLDVCAVVPAAGFGRRMQTECPKQYLSIGNQTILEHSVHALLAHPRVKRVVIAISPGDSRFAQLPLANHPQITVVDGGDERADSVLAGLKAAGDAQWVLVHDAARPCLHQDDLARLLALSETSRTGGILAAPVRDTMKRAEPGKNAIAHTVDRNGLWHALTPQFFPRELLHDCLTRALNEGATITDEASALEYCGFHPQLVEGRADNIKVTRPEDLALAEFYLTRTIHQENT',\n",
       "  'mw': 25737.0},\n",
       " 'b2750': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A6J1',\n",
       "  'gene': 'b2750',\n",
       "  'locus': ['b2750', 'JW2720'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [(\"adenosine 5'-phosphosulfate + ATP = 3'-phosphoadenylyl sulfate + ADP + H(+)\",\n",
       "     '2.7.1.25')],\n",
       "   'Function': 'Catalyzes the synthesis of activated sulfate',\n",
       "   'Pathway': 'Sulfur metabolism; hydrogen sulfide biosynthesis; sulfite from sulfate: step 2/3'},\n",
       "  'seq': 'MALHDENVVWHSHPVTVQQRELHHGHRGVVLWFTGLSGSGKSTVAGALEEALHKLGVSTYLLDGDNVRHGLCSDLGFSDADRKENIRRVGEVANLMVEAGLVVLTAFISPHRAERQMVRERVGEGRFIEVFVDTPLAICEARDPKGLYKKARAGELRNFTGIDSVYEAPESAEIHLNGEQLVTNLVQQLLDLLRQNDIIRS',\n",
       "  'mw': 22321.0},\n",
       " 'b2751': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P23845',\n",
       "  'gene': 'b2751',\n",
       "  'locus': ['b2751', 'JW2721'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [(\"ATP + H(+) + sulfate = adenosine 5'-phosphosulfate + diphosphate\",\n",
       "     '2.7.7.4')],\n",
       "   'Function': \"With CysD forms the ATP sulfurylase (ATPS) that catalyzes the adenylation of sulfate producing adenosine 5'-phosphosulfate (APS) and diphosphate, the first enzymatic step in sulfur assimilation pathway. APS synthesis involves the formation of a high-energy phosphoric-sulfuric acid anhydride bond driven by GTP hydrolysis by CysN coupled to ATP hydrolysis by CysD\",\n",
       "   'Regulation Activity': 'GTPase activity is coupled to stimulation of the rate of APS formation',\n",
       "   'Pathway': 'Sulfur metabolism; hydrogen sulfide biosynthesis; sulfite from sulfate: step 1/3'},\n",
       "  'seq': 'MNTALAQQIANEGGVEAWMIAQQHKSLLRFLTCGSVDDGKSTLIGRLLHDTRQIYEDQLSSLHNDSKRHGTQGEKLDLALLVDGLQAEREQGITIDVAYRYFSTEKRKFIIADTPGHEQYTRNMATGASTCELAILLIDARKGVLDQTRRHSFISTLLGIKHLVVAINKMDLVDYSEETFTRIREDYLTFAGQLPGNLDIRFVPLSALEGDNVASQSESMPWYSGPTLLEVLETVEIQRVVDAQPMRFPVQYVNRPNLDFRGYAGTLASGRVEVGQRVKVLPSGVESNVARIVTFDGDREEAFAGEAITLVLTDEIDISRGDLLLAADEALPAVQSASVDVVWMAEQPLSPGQSYDIKIAGKKTRARVDGIRYQVDINNLTQREVENLPLNGIGLVDLTFDEPLVLDRYQQNPVTGGLIFIDRLSNVTVGAGMVHEPVSQATAAPSEFSAFELELNALVRRHFPHWGARDLLGDK',\n",
       "  'mw': 52558.0},\n",
       " 'b2752': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P21156',\n",
       "  'gene': 'b2752',\n",
       "  'locus': ['b2752', 'JW2722'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [(\"ATP + H(+) + sulfate = adenosine 5'-phosphosulfate + diphosphate\",\n",
       "     '2.7.7.4')],\n",
       "   'Function': \"With CysN forms the ATP sulfurylase (ATPS) that catalyzes the adenylation of sulfate producing adenosine 5'-phosphosulfate (APS) and diphosphate, the first enzymatic step in sulfur assimilation pathway. APS synthesis involves the formation of a high-energy phosphoric-sulfuric acid anhydride bond driven by GTP hydrolysis by CysN coupled to ATP hydrolysis by CysD\",\n",
       "   'Regulation Activity': 'Stimulated by an intrinsic GTPase (Probably CysN)',\n",
       "   'Pathway': 'Sulfur metabolism; hydrogen sulfide biosynthesis; sulfite from sulfate: step 1/3'},\n",
       "  'seq': 'MDQIRLTHLRQLEAESIHIIREVAAEFSNPVMLYSIGKDSSVMLHLARKAFYPGTLPFPLLHVDTGWKFREMYEFRDRTAKAYGCELLVHKNPEGVAMGINPFVHGSAKHTDIMKTEGLKQALNKYGFDAAFGGARRDEEKSRAKERIYSFRDRFHRWDPKNQRPELWHNYNGQINKGESIRVFPLSNWTEQDIWQYIWLENIDIVPLYLAAERPVLERDGMLMMIDDNRIDLQPGEVIKKRMVRFRTLGCWPLTGAVESNAQTLPEIIEEMLVSTTSERQGRVIDRDQAGSMELKKRQGYF',\n",
       "  'mw': 35188.0},\n",
       " 'b2762': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P17854',\n",
       "  'gene': 'b2762',\n",
       "  'locus': ['b2762', 'JW2732'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [(\"[thioredoxin]-disulfide + adenosine 3',5'-bisphosphate + 2 H(+) + sulfite = 3'-phosphoadenylyl sulfate + [thioredoxin]-dithiol\",\n",
       "     '1.8.4.8')],\n",
       "   'Function': \"Catalyzes the formation of sulfite from phosphoadenosine 5'-phosphosulfate (PAPS) using thioredoxin as an electron donor\",\n",
       "   'Regulation Activity': 'Severely inhibited by reagents which covalently modify Cys groups in proteins',\n",
       "   'Pathway': 'Sulfur metabolism; hydrogen sulfide biosynthesis; sulfite from sulfate: step 3/3'},\n",
       "  'seq': 'MSKLDLNALNELPKVDRILALAETNAELEKLDAEGRVAWALDNLPGEYVLSSSFGIQAAVSLHLVNQIRPDIPVILTDTGYLFPETYRFIDELTDKLKLNLKVYRATESAAWQEARYGKLWEQGVEGIEKYNDINKVEPMNRALKELNAQTWFAGLRREQSGSRANLPVLAIQRGVFKVLPIIDWDNRTIYQYLQKHGLKYHPLWDEGYLSVGDTHTTRKWEPGMAEEETRFFGLKRECGLHEG',\n",
       "  'mw': 27976.0},\n",
       " 'b2763': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P17846',\n",
       "  'gene': 'b2763',\n",
       "  'locus': ['b2763', 'JW2733'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('3 H2O + hydrogen sulfide + 3 NADP(+) = 4 H(+) + 3 NADPH + sulfite',\n",
       "     '1.8.1.2')],\n",
       "   'Function': 'Component of the sulfite reductase complex that catalyzes the 6-electron reduction of sulfite to sulfide. This is one of several activities required for the biosynthesis of L-cysteine from sulfate',\n",
       "   'Pathway': 'Sulfur metabolism; hydrogen sulfide biosynthesis; hydrogen sulfide from sulfite (NADPH route): step 1/1'},\n",
       "  'seq': 'MSEKHPGPLVVEGKLTDAERMKHESNYLRGTIAEDLNDGLTGGFKGDNFLLIRFHGMYQQDDRDIRAERAEQKLEPRHAMLLRCRLPGGVITTKQWQAIDKFAGENTIYGSIRLTNRQTFQFHGILKKNVKPVHQMLHSVGLDALATANDMNRNVLCTSNPYESQLHAEAYEWAKKISEHLLPRTRAYAEIWLDQEKVATTDEEPILGQTYLPRKFKTTVVIPPQNDIDLHANDMNFVAIAENGKLVGFNLLVGGGLSIEHGNKKTYARTASEFGYLPLEHTLAVAEAVVTTQRDWGNRTDRKNAKTKYTLERVGVETFKAEVERRAGIKFEPIRPYEFTGRGDRIGWVKGIDDNWHLTLFIENGRILDYPARPLKTGLLEIAKIHKGDFRITANQNLIIAGVPESEKAKIEKIAKESGLMNAVTPQRENSMACVSFPTCPLAMAEAERFLPSFIDNIDNLMAKHGVSDEHIVMRVTGCPNGCGRAMLAEVGLVGKAPGRYNLHLGGNRIGTRIPRMYKENITEPEILASLDELIGRWAKEREAGEGFGDFTVRAGIIRPVLDPARDLWD',\n",
       "  'mw': 63998.0},\n",
       " 'b2764': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P38038',\n",
       "  'gene': 'b2764',\n",
       "  'locus': ['b2764', 'JW2734'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('3 H2O + hydrogen sulfide + 3 NADP(+) = 4 H(+) + 3 NADPH + sulfite',\n",
       "     '1.8.1.2')],\n",
       "   'Function': 'Component of the sulfite reductase complex that catalyzes the 6-electron reduction of sulfite to sulfide. This is one of several activities required for the biosynthesis of L-cysteine from sulfate. The flavoprotein component catalyzes the electron flow from NADPH -> FAD -> FMN to the hemoprotein component',\n",
       "   'Pathway': 'Sulfur metabolism; hydrogen sulfide biosynthesis; hydrogen sulfide from sulfite (NADPH route): step 1/1'},\n",
       "  'seq': 'MTTQVPPSALLPLNPEQLARLQAATTDLTPTQLAWVSGYFWGVLNQQPAALAATPAPAAEMPGITIISASQTGNARRVAEALRDDLLAAKLNVKLVNAGDYKFKQIASEKLLIVVTSTQGEGEPPEEAVALHKFLFSKKAPKLENTAFAVFSLGDSSYEFFCQSGKDFDSKLAELGGERLLDRVDADVEYQAAASEWRARVVDALKSRAPVAAPSQSVATGAVNEIHTSPYSKDAPLVASLSVNQKITGRNSEKDVRHIEIDLGDSGMRYQPGDALGVWYQNDPALVKELVELLWLKGDEPVTVEGKTLPLNEALQWHFELTVNTANIVENYATLTRSETLLPLVGDKAKLQHYAATTPIVDMVRFSPAQLDAEALINLLRPLTPRLYSIASSQAEVENEVHVTVGVVRYDVEGRARAGGASSFLADRVEEEGEVRVFIEHNDNFRLPANPETPVIMIGPGTGIAPFRAFMQQRAADEAPGKNWLFFGNPHFTEDFLYQVEWQRYVKDGVLTRIDLAWSRDQKEKVYVQDKLREQGAELWRWINDGAHIYVCGDANRMAKDVEQALLEVIAEFGGMDTEAADEFLSELRVERRYQRDVY',\n",
       "  'mw': 66270.0},\n",
       " 'b2779': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A6P9',\n",
       "  'gene': 'b2779',\n",
       "  'locus': ['b2779', 'JW2750'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(2R)-2-phosphoglycerate = H2O + phosphoenolpyruvate',\n",
       "     '4.2.1.11')],\n",
       "   'Function': 'Catalyzes the reversible conversion of 2-phosphoglycerate into phosphoenolpyruvate. It is essential for the degradation of carbohydrates via glycolysis. It is also a component of the RNA degradosome, a multi-enzyme complex involved in RNA processing and messenger RNA degradation. Its interaction with RNase E is important for the turnover of mRNA, in particular on transcripts encoding enzymes of energy-generating metabolic routes. Its presence in the degradosome is required for the response to excess phosphosugar. May play a regulatory role in the degradation of specific RNAs, such as ptsG mRNA, therefore linking cellular metabolic status with post-translational gene regulation',\n",
       "   'Regulation Activity': 'The covalent binding to the substrate at Lys-342 causes inactivation of the enzyme, and possibly serves as a signal for the export of the protein. Inhibited by fluoride ion',\n",
       "   'Pathway': 'Carbohydrate degradation; glycolysis; pyruvate from D-glyceraldehyde 3-phosphate: step 4/5'},\n",
       "  'seq': 'MSKIVKIIGREIIDSRGNPTVEAEVHLEGGFVGMAAAPSGASTGSREALELRDGDKSRFLGKGVTKAVAAVNGPIAQALIGKDAKDQAGIDKIMIDLDGTENKSKFGANAILAVSLANAKAAAAAKGMPLYEHIAELNGTPGKYSMPVPMMNIINGGEHADNNVDIQEFMIQPVGAKTVKEAIRMGSEVFHHLAKVLKAKGMNTAVGDEGGYAPNLGSNAEALAVIAEAVKAAGYELGKDITLAMDCAASEFYKDGKYVLAGEGNKAFTSEEFTHFLEELTKQYPIVSIEDGLDESDWDGFAYQTKVLGDKIQLVGDDLFVTNTKILKEGIEKGIANSILIKFNQIGSLTETLAAIKMAKDAGYTAVISHRSGETEDATIADLAVGTAAGQIKTGSMSRSDRVAKYNQLIRIEEALGEKAPYNGRKEIKGQA',\n",
       "  'mw': 45655.0},\n",
       " 'b2780': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A7E5',\n",
       "  'gene': 'b2780',\n",
       "  'locus': ['b2780', 'JW2751'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + H2O + L-glutamine + UTP = ADP + CTP + 2 H(+) + L-glutamate + phosphate',\n",
       "     '6.3.4.2'),\n",
       "    ('H2O + L-glutamine = L-glutamate + NH4(+)', ''),\n",
       "    ('ATP + NH4(+) + UTP = ADP + CTP + 2 H(+) + phosphate', '')],\n",
       "   'Function': 'Catalyzes the ATP-dependent amination of UTP to CTP with either L-glutamine or ammonia as the source of nitrogen. Regulates intracellular CTP levels through interactions with the four ribonucleotide triphosphates',\n",
       "   'Regulation Activity': 'Allosterically activated by GTP, when glutamine is the substrate; GTP has no effect on the reaction when ammonia is the substrate (PubMed:4550559). The allosteric effector GTP functions by stabilizing the protein conformation that binds the tetrahedral intermediate(s) formed during glutamine hydrolysis (PubMed:11336655). Also activated by magnesium; the enzyme requires more Mg(2+) for full catalytic activity than required simply to complex the nucleotide substrates (PubMed:8385490). Inhibited by the product CTP, via allosteric rather than competitive inhibition (PubMed:8385490, PubMed:16216072). Also inhibited by divalent metal ions such as copper and zinc (PubMed:8385490). Is potently inhibited by the intermediate analog inhibitor glutamate gamma-semialdehyde (PubMed:11336655)',\n",
       "   'Pathway': 'Pyrimidine metabolism; CTP biosynthesis via de novo pathway; CTP from UDP: step 2/2'},\n",
       "  'seq': 'MTTNYIFVTGGVVSSLGKGIAAASLAAILEARGLNVTIMKLDPYINVDPGTMSPIQHGEVFVTEDGAETDLDLGHYERFIRTKMSRRNNFTTGRIYSDVLRKERRGDYLGATVQVIPHITNAIKERVLEGGEGHDVVLVEIGGTVGDIESLPFLEAIRQMAVEIGREHTLFMHLTLVPYMAASGEVKTKPTQHSVKELLSIGIQPDILICRSDRAVPANERAKIALFCNVPEKAVISLKDVDSIYKIPGLLKSQGLDDYICKRFSLNCPEANLSEWEQVIFEEANPVSEVTIGMVGKYIELPDAYKSVIEALKHGGLKNRVSVNIKLIDSQDVETRGVEILKGLDAILVPGGFGYRGVEGMITTARFARENNIPYLGICLGMQVALIDYARHVANMENANSTEFVPDCKYPVVALITEWRDENGNVEVRSEKSDLGGTMRLGAQQCQLVDDSLVRQLYNAPTIVERHRHRYEVNNMLLKQIEDAGLRVAGRSGDDQLVEIIEVPNHPWFVACQFHPEFTSTPRDGHPLFAGFVKAASEFQKRQAK',\n",
       "  'mw': 60374.0},\n",
       " 'b2781': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AEY3',\n",
       "  'gene': 'b2781',\n",
       "  'locus': ['b2781', 'JW2752'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + H2O = AMP + diphosphate + H(+)',\n",
       "     '3.6.1.8')],\n",
       "   'Function': 'Involved in the regulation of bacterial cell survival under conditions of nutritional stress. Regulates the type II MazE-MazF toxin-antitoxin (TA) system which mediates programmed cell death (PCD). This is achieved by lowering the cellular concentration of (p)ppGpp produced by RelA under amino acid starvation, thus protecting the cell from the toxicity of MazF. Reduction of (p)ppGpp can be achieved by direct degradation of (p)ppGpp or by degradation of NTPs, which are substrates for (p)ppGpp synthesis by RelA',\n",
       "   'Regulation Activity': 'Inhibited by the MazE-MazF complex'},\n",
       "  'seq': 'MNQIDRLLTIMQRLRDPENGCPWDKEQTFATIAPYTLEETYEVLDAIAREDFDDLRGELGDLLFQVVFYAQMAQEEGRFDFNDICAAISDKLERRHPHVFADSSAENSSEVLARWEQIKTEERAQKAQHSALDDIPRSLPALMRAQKIQKRCANVGFDWTTLGPVVDKVYEEIDEVMYEARQAVVDQAKLEEEMGDLLFATVNLARHLGTKAEIALQKANEKFERRFREVERIVAARGLEMTGVDLETMEEVWQQVKRQEIDL',\n",
       "  'mw': 30412.0},\n",
       " 'b2784': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AG20',\n",
       "  'gene': 'b2784',\n",
       "  'locus': ['b2784', 'JW2755'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [(\"ATP + GTP = AMP + guanosine 3'-diphosphate 5'-triphosphate\",\n",
       "     '2.7.6.5')],\n",
       "   'Function': \"In eubacteria ppGpp (guanosine 3'-diphosphate 5'-diphosphate) is a mediator of the stringent response which coordinates a variety of cellular activities in response to changes in nutritional abundance. This enzyme catalyzes the formation of pppGpp which is then hydrolyzed to form ppGpp. The second messengers ppGpp and c-di-GMP together control biofilm formation in response to translational stress; ppGpp represses biofilm formation while c-di-GMP induces it. ppGpp activates transcription of CsrA-antagonistic small RNAs CsrB and CsrC, which down-regulate CsrA's action on translation during the stringent response (PubMed:21488981)\",\n",
       "   'Pathway': 'Purine metabolism; ppGpp biosynthesis; ppGpp from GTP: step 1/2'},\n",
       "  'seq': 'MVAVRSAHINKAGEFDPEKWIASLGITSQKSCECLAETWAYCLQQTQGHPDASLLLWRGVEMVEILSTLSMDIDTLRAALLFPLADANVVSEDVLRESVGKSVVNLIHGVRDMAAIRQLKATHTDSVSSEQVDNVRRMLLAMVDDFRCVVIKLAERIAHLREVKDAPEDERVLAAKECTNIYAPLANRLGIGQLKWELEDYCFRYLHPTEYKRIAKLLHERRLDREHYIEEFVGHLRAEMKAEGVKAEVYGRPKHIYSIWRKMQKKNLAFDELFDVRAVRIVAERLQDCYAALGIVHTHYRHLPDEFDDYVANPKPNGYQSIHTVVLGPGGKTVEIQIRTKQMHEDAELGVAAHWKYKEGAAAGGARSGHEDRIAWLRKLIAWQEEMADSGEMLDEVRSQVFDDRVYVFTPKGDVVDLPAGSTPLDFAYHIHSDVGHRCIGAKIGGRIVPFTYQLQMGDQIEIITQKQPNPSRDWLNPNLGYVTTSRGRSKIHAWFRKQDRDKNILAGRQILDDELEHLGISLKEAEKHLLPRYNFNDVDELLAAIGGGDIRLNQMVNFLQSQFNKPSAEEQDAAALKQLQQKSYTPQNRSKDNGRVVVEGVGNLMHHIARCCQPIPGDEIVGFITQGRGISVHRADCEQLAELRSHAPERIVDAVWGESYSAGYSLVVRVVANDRSGLLRDITTILANEKVNVLGVASRSDTKQQLATIDMTIEIYNLQVLGRVLGKLNQVPDVIDARRLHGS',\n",
       "  'mw': 83876.0},\n",
       " 'b2787': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AES2',\n",
       "  'gene': 'b2787',\n",
       "  'locus': ['b2787', 'JW2758'],\n",
       "  'synonyms': 'ygcX',\n",
       "  'properties': {'Catalytic Activity': [('D-glucarate = 5-dehydro-4-deoxy-D-glucarate + H2O',\n",
       "     '4.2.1.40')],\n",
       "   'Function': 'Catalyzes the dehydration of glucarate or L-idarate to 5-keto-4-deoxy-D-glucarate (5-kdGluc) (PubMed:9772162). Also catalyzes the epimerization of D-glucarate and L-idarate (PubMed:11513584)',\n",
       "   'Pathway': 'Carbohydrate acid metabolism; D-glucarate degradation; 2,5-dioxopentanoate from D-glucarate: step 1/2'},\n",
       "  'seq': 'MSSQFTTPVVTEMQVIPVAGHDSMLMNLSGAHAPFFTRNIVIIKDNSGHTGVGEIPGGEKIRKTLEDAIPLVVGKTLGEYKNVLTLVRNTFADRDAGGRGLQTFDLRTTIHVVTGIEAAMLDLLGQHLGVNVASLLGDGQQRSEVEMLGYLFFVGNRKATPLPYQSQPDDSCDWYRLRHEEAMTPDAVVRLAEAAYEKYGFNDFKLKGGVLAGEEEAESIVALAQRFPQARITLDPNGAWSLNEAIKIGKYLKGSLAYAEDPCGAEQGFSGREVMAEFRRATGLPTATNMIATDWRQMGHTLSLQSVDIPLADPHFWTMQGSVRVAQMCHEFGLTWGSHSNNHFDISLAMFTHVAAAAPGKITAIDTHWIWQEGNQRLTKEPFEIKGGLVQVPEKPGLGVEIDMDQVMKAHELYQKHGLGARDDAMGMQYLIPGWTFDNKRPCMVR',\n",
       "  'mw': 49141.0},\n",
       " 'b2788': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'Q46915',\n",
       "  'gene': 'b2788',\n",
       "  'locus': ['b2788', 'JW2759'],\n",
       "  'synonyms': 'ygcY',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Does not seem to have an in-vivo activity on glucarate or idarate. Its real substrate is unknown'},\n",
       "  'seq': 'MATQSSPVITDMKVIPVAGHDSMLLNIGGAHNAYFTRNIVVLTDNAGHTGIGEAPGGDVIYQTLVDAIPMVLGQEVARLNKVVQQVHKGNQAADFDTFGKGAWTFELRVNAVAALEAALLDLLGKALNVPVCELLGPGKQREAITVLGYLFYIGDRTKTDLPYVENTPGNHEWYQLRHQKAMNSEAVVRLAEASQDRYGFKDFKLKGGVLPGEQEIDTVRALKKRFPDARITVDPNGAWLLDEAISLCKGLNDVLTYAEDPCGAEQGFSGREVMAEFRRATGLPVATNMIATNWREMGHAVMLNAVDIPLADPHFWTLSGAVRVAQLCDDWGLTWGCHSNNHFDISLAMFTHVGAAAPGNPTAIDTHWIWQEGDCRLTQNPLEIKNGKIAVPDAPGLGVELDWEQVQKAHEAYKRLPGGARNDAGPMQYLIPGWTFDRKRPVFGRH',\n",
       "  'mw': 48850.0},\n",
       " 'b2789': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'Q46916',\n",
       "  'gene': 'b2789',\n",
       "  'locus': ['b2789', 'JW2760'],\n",
       "  'synonyms': 'ygcZ',\n",
       "  'properties': {'Catalytic Activity': [('galactarate(in) + H(+)(in) = galactarate(out) + H(+)(out)',\n",
       "     ''),\n",
       "    ('D-glucarate(in) + H(+)(in) = D-glucarate(out) + H(+)(out)', ''),\n",
       "    ('(R)-glycerate(in) + H(+)(in) = (R)-glycerate(out) + H(+)(out)', '')],\n",
       "   'Function': 'Probably involved in the uptake of galactarate and/or D-glucarate (Probable). May also transport D-glycerate (Probable)'},\n",
       "  'seq': 'MSSLSQAASSVEKRTNARYWIVVMLFIVTSFNYGDRATLSIAGSEMAKDIGLDPVGMGYVFSAFSWAYVIGQIPGGWLLDRFGSKRVYFWSIFIWSMFTLLQGFVDIFSGFGIIVALFTLRFLVGLAEAPSFPGNSRIVAAWFPAQERGTAVSIFNSAQYFATVIFAPIMGWLTHEVGWSHVFFFMGGLGIVISFIWLKVIHEPNQHPGVNKKELEYIAAGGALINMDQQNTKVKVPFSVKWGQIKQLLGSRMMIGVYIGQYCINALTYFFITWFPVYLVQARGMSILKAGFVASVPAVCGFIGGVLGGIISDWLMRRTGSLNIARKTPIVMGMLLSMVMVFCNYVNVEWMIIGFMALAFFGKGIGALGWAVMADTAPKEISGLSGGLFNMFGNISGIVTPIAIGYIVGTTGSFNGALIYVGVHALIAVLSYLVLVGDIKRIELKPVAGQ',\n",
       "  'mw': 49142.0},\n",
       " 'b2796': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AAD6',\n",
       "  'gene': 'b2796',\n",
       "  'locus': ['b2796', 'JW2767'],\n",
       "  'synonyms': 'dcrA',\n",
       "  'properties': {'Catalytic Activity': [('H(+)(in) + L-serine(in) = H(+)(out) + L-serine(out)',\n",
       "     '')],\n",
       "   'Function': '(Microbial infection) May function as an inner membrane receptor of colicin V (ColV), a peptide antibiotic secreted by some members of the Enterobacteriaceae to kill closely related bacterial cells',\n",
       "   'Regulation Activity': 'Serine transport is strongly inhibited by KCN, an inhibitor of the respiratory chain, or by CCCP'},\n",
       "  'seq': 'METTQTSTIASKDSRSAWRKTDTMWMLGLYGTAIGAGVLFLPINAGVGGMIPLIIMAILAFPMTFFAHRGLTRFVLSGKNPGEDITEVVEEHFGIGAGKLITLLYFFAIYPILLVYSVAITNTVESFMSHQLGMTPPPRAILSLILIVGMMTIVRFGEQMIVKAMSILVFPFVGVLMLLALYLIPQWNGAALETLSLDTASATGNGLWMTLWLAIPVMVFSFNHSPIISSFAVAKREEYGDMAEQKCSKILAFAHIMMVLTVMFFVFSCVLSLTPADLAAAKEQNISILSYLANHFNAPVIAWMAPIIAIIAITKSFLGHYLGAREGFNGMVIKSLRGKGKSIEINKLNRITALFMLVTTWIVATLNPSILGMIETLGGPIIAMILFLMPMYAIQKVPAMRKYSGHISNVFVVVMGLIAISAIFYSLFS',\n",
       "  'mw': 46906.0},\n",
       " 'b2797': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P30744',\n",
       "  'gene': 'b2797',\n",
       "  'locus': ['b2797', 'JW2768'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('L-serine = NH4(+) + pyruvate',\n",
       "     '4.3.1.17')],\n",
       "   'Function': 'Deaminates also threonine, particularly when it is present in high concentration',\n",
       "   'Pathway': 'Carbohydrate biosynthesis; gluconeogenesis'},\n",
       "  'seq': 'MISVFDIFKIGIGPSSSHTVGPMKAGKQFTDDLIARNLLKDVTRVVVDVYGSLSLTGKGHHTDIAIIMGLAGNLPDTVDIDSIPSFIQDVNTHGRLMLANGQHEVEFPVDQCMNFHADNLSLHENGMRITALAGDKVVYSQTYYSIGGGFIVDEEHFGQQDSAPVEVPYPYSSAADLQKHCQETGLSLSGLMMKNELALHSKEELEQHLANVWEVMRGGIERGISTEGVLPGKLRVPRRAAALRRMLVSQDKTTTDPMAVVDWINMFALAVNEENAAGGRVVTAPTNGACGIIPAVLAYYDKFIREVNANSLARYLLVASAIGSLYKMNASISGAEVGCQGEVGVACSMAAAGLAELLGASPAQVCIAAEIAMEHNLGLTCDPVAGQVQVPCIERNAIAAVKAVNAARMALRRTSEPRVCLDKVIETMYETGKDMNAKYRETSRGGLAMKIVACD',\n",
       "  'mw': 48753.0},\n",
       " 'b2799': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A9S1',\n",
       "  'gene': 'b2799',\n",
       "  'locus': ['b2799', 'JW2770'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(R)-propane-1,2-diol + NAD(+) = (R)-lactaldehyde + H(+) + NADH',\n",
       "     '1.1.1.77'),\n",
       "    ('(S)-propane-1,2-diol + NAD(+) = (S)-lactaldehyde + H(+) + NADH',\n",
       "     '1.1.1.77')],\n",
       "   'Regulation Activity': 'Inhibited by Zn(2+)',\n",
       "   'Pathway': 'Carbohydrate degradation; L-fucose degradation'},\n",
       "  'seq': 'MANRMILNETAWFGRGAVGALTDEVKRRGYQKALIVTDKTLVQCGVVAKVTDKMDAAGLAWAIYDGVVPNPTITVVKEGLGVFQNSGADYLIAIGGGSPQDTCKAIGIISNNPEFADVRSLEGLSPTNKPSVPILAIPTTAGTAAEVTINYVITDEEKRRKFVCVDPHDIPQVAFIDADMMDGMPPALKAATGVDALTHAIEGYITRGAWALTDALHIKAIEIIAGALRGSVAGDKDAGEEMALGQYVAGMGFSNVGLGLVHGMAHPLGAFYNTPHGVANAILLPHVMRYNADFTGEKYRDIARVMGVKVEGMSLEEARNAAVEAVFALNRDVGIPPHLRDVGVRKEDIPALAQAALDDVCTGGNPREATLEDIVELYHTAW',\n",
       "  'mw': 40513.0},\n",
       " 'b2800': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AB87',\n",
       "  'gene': 'b2800',\n",
       "  'locus': ['b2800', 'JW2771'],\n",
       "  'synonyms': 'fucC prd',\n",
       "  'properties': {'Catalytic Activity': [('L-fuculose 1-phosphate = (S)-lactaldehyde + dihydroxyacetone phosphate',\n",
       "     '4.1.2.17')],\n",
       "   'Function': 'Involved in the degradation of L-fucose and D-arabinose (PubMed:13898172). Catalyzes the reversible cleavage of L-fuculose 1-phosphate (Fuc1P) to yield dihydroxyacetone phosphate (DHAP) and L-lactaldehyde (PubMed:13898172, Ref.8, Ref.9, PubMed:10821675, PubMed:11054289). Also able to catalyze the reversible cleavage of D-ribulose 1-phosphate, but FucA has a higher affinity for L-fuculose 1-phosphate and L-lactaldehyde than for D-ribulose 1-phosphate and glycolaldehyde, respectively (PubMed:4928018). FucA possesses a high specificity for the dihydroxyacetone phosphate (DHAP), but accepts a great variety of different aldehydes and has a strong preference for L-configurated alpha-hydroxy aldehydes (PubMed:13898172, Ref.8, PubMed:10821675). FucA generates a vicinal diol unit having the absolute (3R,4R)-cis configuration (D-erythro) (Ref.8, PubMed:10821675)',\n",
       "   'Regulation Activity': 'Inhibited by phosphoglycolohydroxamate (PGH)',\n",
       "   'Pathway': 'Carbohydrate degradation; L-fucose degradation; L-lactaldehyde and glycerone phosphate from L-fucose: step 3/3'},\n",
       "  'seq': 'MERNKLARQIIDTCLEMTRLGLNQGTAGNVSVRYQDGMLITPTGIPYEKLTESHIVFIDGNGKHEEGKLPSSEWRFHMAAYQSRPDANAVVHNHAVHCTAVSILNRSIPAIHYMIAAAGGNSIPCAPYATFGTRELSEHVALALKNRKATLLQHHGLIACEVNLEKALWLAHEVEVLAQLYLTTLAITDPVPVLSDEEIAVVLEKFKTYGLRIEE',\n",
       "  'mw': 23775.0},\n",
       " 'b2801': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P11551',\n",
       "  'gene': 'b2801',\n",
       "  'locus': ['b2801', 'JW2772'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('H(+)(in) + L-fucose(in) = H(+)(out) + L-fucose(out)',\n",
       "     ''),\n",
       "    ('D-arabinose(out) + H(+)(out) = D-arabinose(in) + H(+)(in)', ''),\n",
       "    ('H(+)(out) + L-galactose(out) = H(+)(in) + L-galactose(in)', '')],\n",
       "   'Function': 'Mediates the uptake of L-fucose across the boundary membrane with the concomitant transport of protons into the cell (symport system) (PubMed:2829831, PubMed:8052131). Can also transport L-galactose and D-arabinose, but at reduced rates compared with L-fucose. Is not able to transport L-rhamnose and L-arabinose (PubMed:2829831). Binds D-arabinose with the highest affinity, followed by L-fucose, and then by L-galactose (PubMed:9826601)'},\n",
       "  'seq': 'MGNTSIQTQSYRAVDKDAGQSRSYIIPFALLCSLFFLWAVANNLNDILLPQFQQAFTLTNFQAGLIQSAFYFGYFIIPIPAGILMKKLSYKAGIITGLFLYALGAALFWPAAEIMNYTLFLVGLFIIAAGLGCLETAANPFVTVLGPESSGHFRLNLAQTFNSFGAIIAVVFGQSLILSNVPHQSQDVLDKMSPEQLSAYKHSLVLSVQTPYMIIVAIVLLVALLIMLTKFPALQSDNHSDAKQGSFSASLSRLARIRHWRWAVLAQFCYVGAQTACWSYLIRYAVEEIPGMTAGFAANYLTGTMVCFFIGRFTGTWLISRFAPHKVLAAYALIAMALCLISAFAGGHVGLIALTLCSAFMSIQYPTIFSLGIKNLGQDTKYGSSFIVMTIIGGGIVTPVMGFVSDAAGNIPTAELIPALCFAVIFIFARFRSQTATN',\n",
       "  'mw': 47545.0},\n",
       " 'b2802': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P69922',\n",
       "  'gene': 'b2802',\n",
       "  'locus': ['b2802', 'JW2773'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('L-fucose = L-fuculose', '5.3.1.25'),\n",
       "    ('D-arabinose = D-ribulose', '5.3.1.3'),\n",
       "    ('L-xylopyranose = L-xylulose', '')],\n",
       "   'Function': 'Converts the aldose L-fucose into the corresponding ketose L-fuculose (PubMed:13319278, PubMed:4928018, PubMed:4632320, PubMed:8564401, PubMed:9367760). Also converts D-arabinose into D-ribulose (PubMed:13319278, PubMed:4928018, PubMed:4632320). In addition, catalyzes the isomerization of L-xylulose to L-xylose (PubMed:22133443)',\n",
       "   'Regulation Activity': 'Inhibited by ribitol, L-arabitol and dulcitol (PubMed:4632320). Isomerization of L-xylulose to L-xylose is inhibited by xylitol (PubMed:22133443)',\n",
       "   'Pathway': 'Carbohydrate degradation; L-fucose degradation; L-lactaldehyde and glycerone phosphate from L-fucose: step 1/3'},\n",
       "  'seq': 'MKKISLPKIGIRPVIDGRRMGVRESLEEQTMNMAKATAALLTEKLRHACGAAVECVISDTCIAGMAEAAACEEKFSSQNVGLTITVTPCWCYGSETIDMDPTRPKAIWGFNGTERPGAVYLAAALAAHSQKGIPAFSIYGHDVQDADDTSIPADVEEKLLRFARAGLAVASMKGKSYLSLGGVSMGIAGSIVDHNFFESWLGMKVQAVDMTELRRRIDQKIYDEAELEMALAWADKNFRYGEDENNKQYQRNAEQSRAVLRESLLMAMCIRDMMQGNSKLADIGRVEESLGYNAIAAGFQGQRHWTDQYPNGDTAEAILNSSFDWNGVREPFVVATENDSLNGVAMLMGHQLTGTAQVFADVRTYWSPEAIERVTGHKLDGLAEHGIIHLINSGSAALDGSCKQRDSEGNPTMKPHWEISQQEADACLAATEWCPAIHEYFRGGGYSSRFLTEGGVPFTMTRVNIIKGLGPVLQIAEGWSVELPKDVHDILNKRTNSTWPTTWFAPRLTGKGPFTDVYSVMANWGANHGVLTIGHVGADFITLASMLRIPVCMHNVEETKVYRPSAWAAHGMDIEGQDYRACQNYGPLYKR',\n",
       "  'mw': 64977.0},\n",
       " 'b2803': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P11553',\n",
       "  'gene': 'b2803',\n",
       "  'locus': ['b2803', 'JW2774'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + L-fuculose = ADP + H(+) + L-fuculose 1-phosphate',\n",
       "     '2.7.1.51')],\n",
       "   'Function': 'Catalyzes the phosphorylation of L-fuculose. Can also phosphorylate, with lower efficiency, D-ribulose, D-xylulose and D-fructose',\n",
       "   'Pathway': 'Carbohydrate degradation; L-fucose degradation; L-lactaldehyde and glycerone phosphate from L-fucose: step 2/3'},\n",
       "  'seq': 'MKQEVILVLDCGATNVRAIAVNRQGKIVARASTPNASDIAMENNTWHQWSLDAILQRFADCCRQINSELTECHIRGIAVTTFGVDGALVDKQGNLLYPIISWKCPRTAAVMDNIERLISAQRLQAISGVGAFSFNTLYKLVWLKENHPQLLERAHAWLFISSLINHRLTGEFTTDITMAGTSQMLDIQQRDFSPQILQATGIPRRLFPRLVEAGEQIGTLQNSAAAMLGLPVGIPVISAGHDTQFALFGAGAEQNEPVLSSGTWEILMVRSAQVDTSLLSQYAGSTCELDSQAGLYNPGMQWLASGVLEWVRKLFWTAETPWQMLIEEARLIAPGADGVKMQCDLLSCQNAGWQGVTLNTTRGHFYRAALEGLTAQLQRNLQMLEKIGHFKASELLLVGGGSRNTLWNQIKANMLDIPVKVLDDAETTVAGAALFGWYGVGEFNSPEEARAQIHYQYRYFYPQTEPEFIEEV',\n",
       "  'mw': 52259.0},\n",
       " 'b2810': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'Q46925',\n",
       "  'gene': 'b2810',\n",
       "  'locus': ['b2810', 'JW2781'],\n",
       "  'synonyms': 'ygdJ',\n",
       "  'properties': {'Catalytic Activity': [('[sulfur carrier]-H + L-cysteine = [sulfur carrier]-SH + L-alanine',\n",
       "     '2.8.1.7'),\n",
       "    ('AH2 + L-selenocysteine = A + H(+) + hydrogenselenide + L-alanine',\n",
       "     '4.4.1.16'),\n",
       "    ('3-sulfino-L-alanine + H2O = H(+) + L-alanine + sulfite', '')],\n",
       "   'Function': 'Catalyzes the removal of elemental sulfur and selenium atoms from L-cysteine, L-cystine, L-selenocysteine, and L-selenocystine to produce L-alanine, and transiently retains the released sulfur atom on a cysteine residue, in the form of a persulfide. Can also desulfinate L-cysteine sulfinate (3-sulfino-L-alanine), which is the best substrate of the enzyme. Functions as a selenium delivery protein in the pathway for the biosynthesis of selenophosphate. Seems to participate in Fe/S biogenesis by recruiting the SufBCD-SufE proteins. Transfers sulfur to CsdE that increases the cysteine desulfurase activity of CsdA. Can also transfer sulfur directly to TcdA/CsdL in vitro. Appears to support the function of TcdA in the generation of cyclic threonylcarbamoyladenosine at position 37 (ct(6)A37) in tRNAs that read codons beginning with adenine',\n",
       "   'Regulation Activity': 'Cysteine desulfurase activity is increased 2-fold in the presence of CsdE'},\n",
       "  'seq': 'MNVFNPAQFRAQFPALQDAGVYLDSAATALKPEAVVEATQQFYSLSAGNVHRSQFAEAQRLTARYEAAREKVAQLLNAPDDKTIVWTRGTTESINMVAQCYARPRLQPGDEIIVSVAEHHANLVPWLMVAQQTGAKVVKLPLNAQRLPDVDLLPELITPRSRILALGQMSNVTGGCPDLARAITFAHSAGMVVMVDGAQGAVHFPADVQQLDIDFYAFSGHKLYGPTGIGVLYGKSELLEAMSPWLGGGKMVHEVSFDGFTTQSAPWKLEAGTPNVAGVIGLSAALEWLADYDINQAESWSRSLATLAEDALAKRPGFRSFRCQDSSLLAFDFAGVHHSDMVTLLAEYGIALRAGQHCAQPLLAELGVTGTLRASFAPYNTKSDVDALVNAVDRALELLVD',\n",
       "  'mw': 43234.0},\n",
       " 'b2813': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A935',\n",
       "  'gene': 'b2813',\n",
       "  'locus': ['b2813', 'JW2784'],\n",
       "  'synonyms': 'mlt ygdM',\n",
       "  'properties': {'Catalytic Activity': [('Exolytic cleavage of the (1->4)-beta-glycosidic linkage between N-acetylmuramic acid (MurNAc) and N-acetylglucosamine (GlcNAc) residues in peptidoglycan, from either the reducing or the non-reducing ends of the peptidoglycan chains, with concomitant formation of a 1,6-anhydrobond in the MurNAc residue.',\n",
       "     '4.2.2.n1')],\n",
       "   'Function': 'Murein-degrading enzyme. May play a role in recycling of muropeptides during cell elongation and/or cell division. Degrades murein glycan strands and insoluble, high-molecular weight murein sacculi'},\n",
       "  'seq': 'MKGRWVKYLLMGTVVAMLAACSSKPTDRGQQYKDGKFTQPFSLVNQPDAVGAPINAGDFAEQINHIRNSSPRLYGNQSNVYNAVQEWLRAGGDTRNMRQFGIDAWQMEGADNYGNVQFTGYYTPVIQARHTRQGEFQYPIYRMPPKRGRLPSRAEIYAGALSDKYILAYSNSLMDNFIMDVQGSGYIDFGDGSPLNFFSYAGKNGHAYRSIGKVLIDRGEVKKEDMSMQAIRHWGETHSEAEVRELLEQNPSFVFFKPQSFAPVKGASAVPLVGRASVASDRSIIPPGTTLLAEVPLLDNNGKFNGQYELRLMVALDVGGAIKGQHFDIYQGIGPEAGHRAGWYNHYGRVWVLKTAPGAGNVFSG',\n",
       "  'mw': 40411.0},\n",
       " 'b2817': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P63883',\n",
       "  'gene': 'b2817',\n",
       "  'locus': ['b2817', 'JW5449'],\n",
       "  'synonyms': 'ygdN',\n",
       "  'properties': {'Catalytic Activity': [('Hydrolyzes the link between N-acetylmuramoyl residues and L-amino acid residues in certain cell-wall glycopeptides.',\n",
       "     '3.5.1.28')],\n",
       "   'Function': 'Cell-wall hydrolase involved in septum cleavage during cell division. Can also act as powerful autolysin in the presence of murein synthesis inhibitors'},\n",
       "  'seq': 'MSGSNTAISRRRLLQGAGAMWLLSVSQVSLAAVSQVVAVRVWPASSYTRVTVESNRQLKYKQFALSNPERVVVDIEDVNLNSVLKGMAAQIRADDPFIKSARVGQFDPQTVRMVFELKQNVKPQLFALAPVAGFKERLVMDLYPANAQDMQDPLLALLEDYNKGDLEKQVPPAQSGPQPGKAGRDRPIVIMLDPGHGGEDSGAVGKYKTREKDVVLQIARRLRSLIEKEGNMKVYMTRNEDIFIPLQVRVAKAQKQRADLFVSIHADAFTSRQPSGSSVFALSTKGATSTAAKYLAQTQNASDLIGGVSKSGDRYVDHTMFDMVQSLTIADSLKFGKAVLNKLGKINKLHKNQVEQAGFAVLKAPDIPSILVETAFISNVEEERKLKTATFQQEVAESILAGIKAYFADGATLARRG',\n",
       "  'mw': 45634.0},\n",
       " 'b2818': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A6C5',\n",
       "  'gene': 'b2818',\n",
       "  'locus': ['b2818', 'JW2786'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('acetyl-CoA + L-glutamate = CoA + H(+) + N-acetyl-L-glutamate',\n",
       "     '2.3.1.1')],\n",
       "   'Regulation Activity': 'Feedback inhibition by L-arginine',\n",
       "   'Pathway': 'Amino-acid biosynthesis; L-arginine biosynthesis; N(2)-acetyl-L-ornithine from L-glutamate: step 1/4'},\n",
       "  'seq': 'MVKERKTELVEGFRHSVPYINTHRGKTFVIMLGGEAIEHENFSSIVNDIGLLHSLGIRLVVVYGARPQIDANLAAHHHEPLYHKNIRVTDAKTLELVKQAAGTLQLDITARLSMSLNNTPLQGAHINVVSGNFIIAQPLGVDDGVDYCHSGRIRRIDEDAIHRQLDSGAIVLMGPVAVSVTGESFNLTSEEIATQLAIKLKAEKMIGFCSSQGVTNDDGDIVSELFPNEAQARVEAQEEKGDYNSGTVRFLRGAVKACRSGVRRCHLISYQEDGALLQELFSRDGIGTQIVMESAEQIRRATINDIGGILELIRPLEQQGILVRRSREQLEMEIDKFTIIQRDNTTIACAALYPFPEEKIGEMACVAVHPDYRSSSRGEVLLERIAAQAKQSGLSKLFVLTTRSIHWFQERGFTPVDIDLLPESKKQLYNYQRKSKVLMADLG',\n",
       "  'mw': 49195.0},\n",
       " 'b2827': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A884',\n",
       "  'gene': 'b2827',\n",
       "  'locus': ['b2827', 'JW2795'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(6R)-5,10-methylene-5,6,7,8-tetrahydrofolate + dUMP = 7,8-dihydrofolate + dTMP',\n",
       "     '2.1.1.45')],\n",
       "   'Function': \"Catalyzes the reductive methylation of 2'-deoxyuridine-5'-monophosphate (dUMP) to 2'-deoxythymidine-5'-monophosphate (dTMP) while utilizing 5,10-methylenetetrahydrofolate (mTHF) as the methyl donor and reductant in the reaction, yielding dihydrofolate (DHF) as a by-product (PubMed:3286637, PubMed:2223754, PubMed:9826509). This enzymatic reaction provides an intracellular de novo source of dTMP, an essential precursor for DNA biosynthesis. This protein also binds to its mRNA thus repressing its own translation (PubMed:7708505)\",\n",
       "   'Pathway': 'Pyrimidine metabolism; dTTP biosynthesis'},\n",
       "  'seq': 'MKQYLELMQKVLDEGTQKNDRTGTGTLSIFGHQMRFNLQDGFPLVTTKRCHLRSIIHELLWFLQGDTNIAYLHENNVTIWDEWADENGDLGPVYGKQWRAWPTPDGRHIDQITTVLNQLKNDPDSRRIIVSAWNVGELDKMALAPCHAFFQFYVADGKLSCQLYQRSCDVFLGLPFNIASYALLVHMMAQQCDLEVGDFVWTGGDTHLYSNHMDQTHLQLSREPRPLPKLIIKRKPESIFDYRFEDFEIEGYDPHPGIKAPVAI',\n",
       "  'mw': 30480.0},\n",
       " 'b2835': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P39196',\n",
       "  'gene': 'b2835',\n",
       "  'locus': ['b2835', 'JW2803'],\n",
       "  'synonyms': 'ygeD',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Catalyzes the facilitated diffusion of 2-acyl-glycero-3-phosphoethanolamine (2-acyl-GPE) into the cell'},\n",
       "  'seq': 'MSESVHTNTSLWSKGMKAVIVAQFLSAFGDNALLFATLALLKAQFYPEWSQPILQMVFVGAYILFAPFVGQVADSFAKGRVMMFANGLKLLGAASICFGINPFLGYTLVGVGAAAYSPAKYGILGELTTGSKLVKANGLMEASTIAAILLGSVAGGVLADWHVLVALAACALAYGGAVVANIYIPKLAAARPGQSWNLINMTRSFLNACTSLWRNGETRFSLVGTSLFWGAGVTLRFLLVLWVPVALGITDNATPTYLNAMVAIGIVVGAGAAAKLVTLETVSRCMPAGILIGVVVLIFSLQHELLPAYALLMLIGVMGGFFVVPLNALLQERGKKSVGAGNAIAVQNLGENSAMLLMLGIYSLAVMIGIPVVPIGIGFGALFALAITALWIWQRRH',\n",
       "  'mw': 41656.0},\n",
       " 'b2836': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P31119',\n",
       "  'gene': 'b2836',\n",
       "  'locus': ['b2836', 'JW2804'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('a 2-acyl-sn-glycero-3-phosphoethanolamine + a fatty acyl-[ACP] = a 1,2-diacyl-sn-glycero-3-phosphoethanolamine + holo-[ACP]',\n",
       "     '2.3.1.40'),\n",
       "    ('a long-chain fatty acid + ATP + holo-[ACP] = a long-chain fatty acyl-[ACP] + AMP + diphosphate',\n",
       "     '6.2.1.20')],\n",
       "   'Function': 'Plays a role in lysophospholipid acylation. Transfers fatty acids to the 1-position via an enzyme-bound acyl-ACP intermediate in the presence of ATP and magnesium. Its physiological function is to regenerate phosphatidylethanolamine from 2-acyl-glycero-3-phosphoethanolamine (2-acyl-GPE) formed by transacylation reactions or degradation by phospholipase A1'},\n",
       "  'seq': 'MLFSFFRNLCRVLYRVRVTGDTQALKGERVLITPNHVSFIDGILLGLFLPVRPVFAVYTSISQQWYMRWLKSFIDFVPLDPTQPMAIKHLVRLVEQGRPVVIFPEGRITTTGSLMKIYDGAGFVAAKSGATVIPVRIEGAELTHFSRLKGLVKRRLFPQITLHILPPTQVAMPDAPRARDRRKIAGEMLHQIMMEARMAVRPRETLYESLLSAMYRFGAGKKCVEDVNFTPDSYRKLLTKTLFVGRILEKYSVEGERIGLMLPNAGISAAVIFGAIARRRMPAMMNYTAGVKGLTSAITAAEIKTIFTSRQFLDKGKLWHLPEQLTQVRWVYLEDLKADVTTADKVWIFAHLLMPRLAQVKQQPEEEALILFTSGSEGHPKGVVHSHKSILANVEQIKTIADFTTNDRFMSALPLFHSFGLTVGLFTPLLTGAEVFLYPSPLHYRIVPELVYDRSCTVLFGTSTFLGHYARFANPYDFYRLRYVVAGAEKLQESTKQLWQDKFGLRILEGYGVTECAPVVSINVPMAAKPGTVGRILPGMDARLLSVPGIEEGGRLQLKGPNIMNGYLRVEKPGVLEVPTAENVRGEMERGWYDTGDIVRFDEQGFVQIQGRAKRFAKIAGEMVSLEMVEQLALGVSPDKVHATAIKSDASKGEALVLFTTDNELTRDKLQQYAREHGVPELAVPRDIRYLKQMPLLGSGKPDFVTLKSWVDEAEQHDE',\n",
       "  'mw': 80700.0},\n",
       " 'b2838': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P00861',\n",
       "  'gene': 'b2838',\n",
       "  'locus': ['b2838', 'JW2806'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('H(+) + meso-2,6-diaminoheptanedioate = CO2 + L-lysine',\n",
       "     '4.1.1.20')],\n",
       "   'Function': 'Specifically catalyzes the decarboxylation of meso-diaminopimelate (meso-DAP) to L-lysine. Is not active against the DD- or LL-isomers of diaminopimelate',\n",
       "   'Regulation Activity': 'Is activated by 2,3-dimercaptopropan-1-ol',\n",
       "   'Pathway': 'Amino-acid biosynthesis; L-lysine biosynthesis via DAP pathway; L-lysine from DL-2,6-diaminopimelate: step 1/1'},\n",
       "  'seq': 'MPHSLFSTDTDLTAENLLRLPAEFGCPVWVYDAQIIRRQIAALKQFDVVRFAQKACSNIHILRLMREQGVKVDSVSLGEIERALAAGYNPQTHPDDIVFTADVIDQATLERVSELQIPVNAGSVDMLDQLGQVSPGHRVWLRVNPGFGHGHSQKTNTGGENSKHGIWYTDLPAALDVIQRHHLQLVGIHMHIGSGVDYAHLEQVCGAMVRQVIEFGQDLQAISAGGGLSVPYQQGEEAVDTEHYYGLWNAAREQIARHLGHPVKLEIEPGRFLVAQSGVLITQVRSVKQMGSRHFVLVDAGFNDLMRPAMYGSYHHISALAADGRSLEHAPTVETVVAGPLCESGDVFTQQEGGNVETRALPEVKAGDYLVLHDTGAYGASMSSNYNSRPLLPEVLFDNGQARLIRRRQTIEELLALELL',\n",
       "  'mw': 46177.0},\n",
       " 'b2841': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AE24',\n",
       "  'gene': 'b2841',\n",
       "  'locus': ['b2841', 'JW2809'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('H(+)(in) + L-arabinose(in) = H(+)(out) + L-arabinose(out)',\n",
       "     ''),\n",
       "    ('D-fucose(in) + H(+)(in) = D-fucose(out) + H(+)(out)', '')],\n",
       "   'Function': 'Uptake of L-arabinose across the cytoplasmic membrane with the concomitant transport of protons into the cell (symport system) (PubMed:7030324, PubMed:6282256, PubMed:2836407). D-fucose, a nonmetabolizable analog of L-arabinose, is also a good substrate (PubMed:6282256)',\n",
       "   'Regulation Activity': 'Symport activity is inhibited in the presence of the uncoupling agents carbonyl cyanide m-chlorophenylhydrazone (CCCP) and tetrachlorosalicylanilide and 2,4-dinitrophenol (PubMed:6282256). Activity is partially inhibited by N-ethylmaleimide (PubMed:7030324)'},\n",
       "  'seq': 'MVTINTESALTPRSLRDTRRMNMFVSVAAAVAGLLFGLDIGVIAGALPFITDHFVLTSRLQEWVVSSMMLGAAIGALFNGWLSFRLGRKYSLMAGAILFVLGSIGSAFATSVEMLIAARVVLGIAVGIASYTAPLYLSEMASENVRGKMISMYQLMVTLGIVLAFLSDTAFSYSGNWRAMLGVLALPAVLLIILVVFLPNSPRWLAEKGRHIEAEEVLRMLRDTSEKAREELNEIRESLKLKQGGWALFKINRNVRRAVFLGMLLQAMQQFTGMNIIMYYAPRIFKMAGFTTTEQQMIATLVVGLTFMFATFIAVFTVDKAGRKPALKIGFSVMALGTLVLGYCLMQFDNGTASSGLSWLSVGMTMMCIAGYAMSAAPVVWILCSEIQPLKCRDFGITCSTTTNWVSNMIIGATFLTLLDSIGAAGTFWLYTALNIAFVGITFWLIPETKNVTLEHIERKLMAGEKLRNIGV',\n",
       "  'mw': 51684.0},\n",
       " 'b2866': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'Q46799',\n",
       "  'gene': 'b2866',\n",
       "  'locus': ['b2866', 'JW5462'],\n",
       "  'synonyms': 'ygeS',\n",
       "  'properties': {'Catalytic Activity': [('H2O + NAD(+) + xanthine = H(+) + NADH + urate',\n",
       "     '1.17.1.4'),\n",
       "    ('H2O + hypoxanthine + NAD(+) = H(+) + NADH + xanthine', '1.17.1.4')],\n",
       "   'Function': 'Presumed to be a dehydrogenase, but possibly an oxidase. Participates in limited purine salvage (requires aspartate) but does not support aerobic growth on purines as the sole carbon source (purine catabolism). Deletion results in increased adenine sensitivity, suggesting that this protein contributes to the conversion of adenine to guanine nucleotides during purine salvage',\n",
       "   'Pathway': 'Purine metabolism; hypoxanthine degradation; urate from hypoxanthine: step 2/2'},\n",
       "  'seq': 'MRVDAIAKVTGRARYTDDYVMAGMCYAKYVRSPIAHGYAVSINDEQARSLPGVLAIFTWEDVPDIPFATAGHAWTLDENKRDTADRALLTRHVRHHGDAVAIVVARDELTAEKAAQLVSIEWQELPVITTPEAALAEDAAPIHNGGNLLKQSTMSTGNVQQTIDAADYQVQGHYQTPVIQHCHMESVTSLAWMEDDSRITIVSSTQIPHIVRRVVGQALDIPWSCVRVIKPFVGGGFGNKQDVLEEPMAAFLTSKLGGIPVKVSLSREECFLATRTRHAFTIDGQMGVNRDGTLKGYSLDVLSNTGAYASHGHSIASAGGNKVAYLYPRCAYAYSSKTCYTNLPSAGAMRGYGAPQVVFAVESMLDDAATALGIDPVEIRLRNAAREGDANPLTGKRIYSAGLPECLEKGRKIFEWEKRRAECQNQQGNLRRGVGVACFSYTSNTWPVGVEIAGARLLMNQDGTINVQSGATEIGQGADTVFSQMVAETVGVPVSDVRVISTQDTDVTPFDPGAFASRQSYVAAPALRSAALLLKEKIIAHAAVMLHQSAMNLTLIKGHIVLVERPEEPLMSLKDLAMDAFYHPERGGQLSAESSIKTTTNPPAFGCTFVDLTVDIALCKVTINRILNVHDSGHILNPLLAEGQVHGGMGMGIGWALFEEMIIDAKSGVVRNPNLLDYKMPTMPDLPQLESAFVEINEPQSAYGHKSLGEPPIIPVAAAIRNAVKMATGVAINTLPLTPKRLYEEFHLAGLI',\n",
       "  'mw': 81321.0},\n",
       " 'b2867': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'Q46800',\n",
       "  'gene': 'b2867',\n",
       "  'locus': ['b2867', 'JW2835'],\n",
       "  'synonyms': 'ygeT',\n",
       "  'properties': {'Catalytic Activity': [('H2O + NAD(+) + xanthine = H(+) + NADH + urate',\n",
       "     '1.17.1.4'),\n",
       "    ('H2O + hypoxanthine + NAD(+) = H(+) + NADH + xanthine', '1.17.1.4')],\n",
       "   'Function': 'Presumed to be a dehydrogenase, but possibly an oxidase. Participates in limited purine salvage (requires aspartate) but does not support aerobic growth on purines as the sole carbon source (purine catabolism)',\n",
       "   'Pathway': 'Purine metabolism; hypoxanthine degradation; urate from hypoxanthine: step 2/2'},\n",
       "  'seq': 'MFDFASYHRAATLADAINLLADNPQAKLLAGGTDVLIQLHHHNDRYRHIVDIHNLAELRGITLAEDGSLRIGSATTFTQLIEDPITQRHLPALCAAATSIAGPQIRNVATYGGNICNGATSADSATPTLIYDAKLEIHSPRGVRFVPINGFHTGPGKVSLEHDEILVAFHFPPQPKEHAGSAHFKYAMRDAMDISTIGCAAHCRLDNGNFSELRLAFGVAAPTPIRCQHAEQTAQNAPLNLQTLEAISESVLQDVAPRSSWRASKEFRLHLIQTMTKKVISEAVAAAGGKLQ',\n",
       "  'mw': 31557.0},\n",
       " 'b2868': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'Q46801',\n",
       "  'gene': 'b2868',\n",
       "  'locus': ['b2868', 'JW2836'],\n",
       "  'synonyms': 'ygeU',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Iron-sulfur subunit of the xanthine dehydrogenase complex',\n",
       "   'Pathway': 'Purine metabolism; hypoxanthine degradation; urate from hypoxanthine: step 1/2'},\n",
       "  'seq': 'MNHSETITIECTINGMPFQLHAAPGTPLSELLREQGLLSVKQGCCVGECGACTVLVDGTAIDSCLYLAAWAEGKEIRTLEGEAKGGKLSHVQQAYAKSGAVQCGFCTPGLIMATTAMLAKPREKPLTITEIRRGLAGNLCRCTGYQMIVNTVLDCEKTK',\n",
       "  'mw': 16922.0},\n",
       " 'b2871': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P66899',\n",
       "  'gene': 'b2871',\n",
       "  'locus': ['b2871', 'JW2839'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(S)-2,3-diaminopropanoate + H(+) + H2O = 2 NH4(+) + pyruvate',\n",
       "     '4.3.1.15'),\n",
       "    ('(R)-2,3-diaminopropanoate + H(+) + H2O = 2 NH4(+) + pyruvate',\n",
       "     '4.3.1.15')],\n",
       "   'Function': 'Catalyzes the alpha,beta-elimination reaction of both L- and D-alpha,beta-diaminopropionate (DAP) to form pyruvate and ammonia. In vitro the D-isomer of serine is degraded to pyruvate, though very poorly; other amino acids (L-serine, D- and L-threonine, D- and L-beta-Cl-alanine) are not substrates. In vivo allows poor growth on L-DAP or a DL-DAP mixture but not on D-DAP alone, this may be due to a poor promoter. DL-DAP is toxic in the absence of this enzyme, it may inhibit enzymes involved in the synthesis of pyruvate and aspartate, as well as amino acids derived from them'},\n",
       "  'seq': 'MSVFSLKIDIADNKFFNGETSPLFSQSQAKLARQFHQKIAGYRPTPLCALDDLANLFGVKKILVKDESKRFGLNAFKMLGGAYAIAQLLCEKYHLDIETLSFEHLKNAIGEKMTFATTTDGNHGRGVAWAAQQLGQNAVIYMPKGSAQERVDAILNLGAECIVTDMNYDDTVRLTMQHAQQHGWEVVQDTAWEGYTKIPTWIMQGYATLADEAVEQMREMGVTPTHVLLQAGVGAMAGGVLGYLVDVYSPQNLHSIIVEPDKADCIYRSGVKGDIVNVGGDMATIMAGLACGEPNPLGWEILRNCATQFISCQDSVAALGMRVLGNPYGNDPRIISGESGAVGLGVLAAVHYHPQRQSLMEKLALNKDAVVLVISTEGDTDVKHYREVVWEGKHAVAP',\n",
       "  'mw': 43328.0},\n",
       " 'b2874': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'Q46807',\n",
       "  'gene': 'b2874',\n",
       "  'locus': ['b2874', 'JW2842'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': []},\n",
       "  'seq': 'MSKKIVLALGGNALGDDLAGQMKAVKITSQAIVDLIAQGHEVIVTHGNGPQVGMINQAFEAAAKTEAHSPMLPMSVCVALSQGYIGYDLQNALREELLSRGINKPVATLVTQVEVDANDPAFLNPTKPIGSFFTEQEAEQLTKQGYTLKEDAGRGYRRVVASPKPVDIIEKETVKALVDAGQVVITVGGGGIPVIREGNHLRGASAVIDKDWASARLAEMIDADMLIILTAVEKVAINFGKENEQWLDRLSLSDAERFIEEGHFAKGSMLPKVEAAASFARSRAGREALITVLSKAKEGIEGKTGTVICQ',\n",
       "  'mw': 33071.0},\n",
       " 'b2883': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P76641',\n",
       "  'gene': 'b2883',\n",
       "  'locus': ['b2883', 'JW5466'],\n",
       "  'synonyms': 'ygfP',\n",
       "  'properties': {'Catalytic Activity': [('guanine + H(+) + H2O = NH4(+) + xanthine',\n",
       "     '3.5.4.3')],\n",
       "   'Function': 'Catalyzes the hydrolytic deamination of guanine, producing xanthine and ammonia',\n",
       "   'Pathway': 'Purine metabolism; guanine degradation; xanthine from guanine: step 1/1'},\n",
       "  'seq': 'MMSGEHTLKAVRGSFIDVTRTIDNPEEIASALRFIEDGLLLIKQGKVEWFGEWENGKHQIPDTIRVRDYRGKLIVPGFVDTHIHYPQSEMVGAYGEQLLEWLNKHTFPTERRYEDLEYAREMSAFFIKQLLRNGTTTALVFGTVHPQSVDALFEAASHINMRMIAGKVMMDRNAPDYLLDTAESSYHQSKELIERWHKNGRLLYAITPRFAPTSSPEQMAMAQRLKEEYPDTWVHTHLCENKDEIAWVKSLYPDHDGYLDVYHQYGLTGKNCVFAHCVHLEEKEWDRLSETKSSIAFCPTSNLYLGSGLFNLKKAWQKKVKVGMGTDIGAGTTFNMLQTLNEAYKVLQLQGYRLSAYEAFYLATLGGAKSLGLDDLIGNFLPGKEADFVVMEPTATPLQQLRYDNSVSLVDKLFVMMTLGDDRSIYRTYVDGRLVYERN',\n",
       "  'mw': 50244.0},\n",
       " 'b2889': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'Q46822',\n",
       "  'gene': 'b2889',\n",
       "  'locus': ['b2889', 'JW2857'],\n",
       "  'synonyms': 'ygfV',\n",
       "  'properties': {'Catalytic Activity': [('isopentenyl diphosphate = dimethylallyl diphosphate',\n",
       "     '5.3.3.2')],\n",
       "   'Function': 'Catalyzes the 1,3-allylic rearrangement of the homoallylic substrate isopentenyl (IPP) to its highly electrophilic allylic isomer, dimethylallyl diphosphate (DMAPP)',\n",
       "   'Pathway': 'Isoprenoid biosynthesis; dimethylallyl diphosphate biosynthesis; dimethylallyl diphosphate from isopentenyl diphosphate: step 1/1'},\n",
       "  'seq': 'MQTEHVILLNAQGVPTGTLEKYAAHTADTRLHLAFSSWLFNAKGQLLVTRRALSKKAWPGVWTNSVCGHPQLGESNEDAVIRRCRYELGVEITPPESIYPDFRYRATDPSGIVENEVCPVFAARTTSALQINDDEVMDYQWCDLADVLHGIDATPWAFSPWMVMQATNREARKRLSAFTQLK',\n",
       "  'mw': 20508.0},\n",
       " 'b2890': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A8N3',\n",
       "  'gene': 'b2890',\n",
       "  'locus': ['b2890', 'JW2858'],\n",
       "  'synonyms': 'asuD herC',\n",
       "  'properties': {'Catalytic Activity': [('ATP + L-lysine + tRNA(Lys) = AMP + diphosphate + L-lysyl-tRNA(Lys)',\n",
       "     '6.1.1.6')]},\n",
       "  'seq': 'MSEQHAQGADAVVDLNNELKTRREKLANLREQGIAFPNDFRRDHTSDQLHAEFDGKENEELEALNIEVAVAGRMMTRRIMGKASFVTLQDVGGRIQLYVARDDLPEGVYNEQFKKWDLGDILGAKGKLFKTKTGELSIHCTELRLLTKALRPLPDKFHGLQDQEARYRQRYLDLISNDESRNTFKVRSQILSGIRQFMVNRGFMEVETPMMQVIPGGAAARPFITHHNALDLDMYLRIAPELYLKRLVVGGFERVFEINRNFRNEGISVRHNPEFTMMELYMAYADYKDLIELTESLFRTLAQDILGKTEVTYGDVTLDFGKPFEKLTMREAIKKYRPETDMADLDNFDSAKAIAESIGIHVEKSWGLGRIVTEIFEEVAEAHLIQPTFITEYPAEVSPLARRNDVNPEITDRFEFFIGGREIGNGFSELNDAEDQAQRFLDQVAAKDAGDDEAMFYDEDYVTALEHGLPPTAGLGIGIDRMVMLFTNSHTIRDVILFPAMRPVK',\n",
       "  'mw': 57603.0},\n",
       " 'b2893': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AEG6',\n",
       "  'gene': 'b2893',\n",
       "  'locus': ['b2893', 'JW2861'],\n",
       "  'synonyms': 'xprA',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Acts as a disulfide isomerase, interacting with incorrectly folded proteins to correct non-native disulfide bonds. DsbG and DsbC are part of a periplasmic reducing system that controls the level of cysteine sulfenylation, and provides reducing equivalents to rescue oxidatively damaged secreted proteins. Acts by transferring its disulfide bond to other proteins and is reduced in the process. DsbC is reoxidized by DsbD'},\n",
       "  'seq': 'MKKGFMLFTLLAAFSGFAQADDAAIQQTLAKMGIKSSDIQPAPVAGMKTVLTNSGVLYITDDGKHIIQGPMYDVSGTAPVNVTNKMLLKQLNALEKEMIVYKAPQEKHVITVFTDITCGYCHKLHEQMADYNALGITVRYLAFPRQGLDSDAEKEMKAIWCAKDKNKAFDDVMAGKSVAPASCDVDIADHYALGVQLGVSGTPAVVLSNGTLVPGYQPPKEMKEFLDEHQKMTSGK',\n",
       "  'mw': 25622.0},\n",
       " 'b2895': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ABY4',\n",
       "  'gene': 'b2895',\n",
       "  'locus': ['b2895', 'JW2863'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Low-potential electron donor to a number of redox enzymes'},\n",
       "  'seq': 'MNMGLFYGSSTCYTEMAAEKIRDIIGPELVTLHNLKDDSPKLMEQYDVLILGIPTWDFGEIQEDWEAVWDQLDDLNLEGKIVALYGLGDQLGYGEWFLDALGMLHDKLSTKGVKFVGYWPTEGYEFTSPKPVIADGQLFVGLALDETNQYDLSDERIQSWCEQILNEMAEHYA',\n",
       "  'mw': 19700.0},\n",
       " 'b2901': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'Q46829',\n",
       "  'gene': 'b2901',\n",
       "  'locus': ['b2901', 'JW2869'],\n",
       "  'synonyms': 'bglD yqfC',\n",
       "  'properties': {'Catalytic Activity': [('6-phospho-beta-D-glucosyl-(1->4)-D-glucose + H2O = D-glucose + D-glucose 6-phosphate',\n",
       "     '3.2.1.86')],\n",
       "   'Function': 'Catalyzes the hydrolysis of phosphorylated beta-glucosides into glucose-6-phosphate (G-6-P) and aglycone. It has a high affinity for phosphorylated aromatic beta-glucosides (p-nitrophenyl-beta-glucoside, phenyl beta-glucoside, arbutin), with the exception of phosphorylated salicin, and a low affinity for phosphorylated beta-methyl-glucoside. Apparently, it has only a very limited role in the utilization of external beta-glucosides'},\n",
       "  'seq': 'MIVKKLTLPKDFLWGGAVAAHQVEGGWNKGGKGPSICDVLTGGAHGVPREITKEVLPGKYYPNHEAVDFYGHYKEDIKLFAEMGFKCFRTSIAWTRIFPKGDEAQPNEEGLKFYDDMFDELLKYNIEPVITLSHFEMPLHLVQQYGSWTNRKVVDFFVRFAEVVFERYKHKVKYWMTFNEINNQRNWRAPLFGYCCSGVVYTEHENPEETMYQVLHHQFVASALAVKAARRINPEMKVGCMLAMVPLYPYSCNPDDVMFAQESMRERYVFTDVQLRGYYPSYVLNEWERRGFNIKMEDGDLDVLREGTCDYLGFSYYMTNAVKAEGGTGDAISGFEGSVPNPYVKASDWGWQIDPVGLRYALCELYERYQRPLFIVENGFGAYDKVEEDGSINDDYRIDYLRAHIEEMKKAVTYDGVDLMGYTPWGCIDCVSFTTGQYSKRYGFIYVNKHDDGTGDMSRSRKKSFNWYKEVIASNGEKL',\n",
       "  'mw': 55361.0},\n",
       " 'b2903': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P33195',\n",
       "  'gene': 'b2903',\n",
       "  'locus': ['b2903', 'JW2871'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(R)-N(6)-lipoyl-L-lysyl-[glycine-cleavage complex H protein] + glycine + H(+) = (R)-N(6)-(S(8)-aminomethyldihydrolipoyl)-L-lysyl-[glycine-cleavage complex H protein] + CO2',\n",
       "     '1.4.4.2')],\n",
       "   'Function': 'The glycine cleavage system catalyzes the degradation of glycine. The P protein binds the alpha-amino group of glycine through its pyridoxal phosphate cofactor; CO(2) is released and the remaining methylamine moiety is then transferred to the lipoamide cofactor of the H protein'},\n",
       "  'seq': 'MTQTLSQLENSGAFIERHIGPDAAQQQEMLNAVGAQSLNALTGQIVPKDIQLATPPQVGAPATEYAALAELKAIASRNKRFTSYIGMGYTAVQLPPVILRNMLENPGWYTAYTPYQPEVSQGRLEALLNFQQVTLDLTGLDMASASLLDEATAAAEAMAMAKRVSKLKNANRFFVASDVHPQTLDVVRTRAETFGFEVIVDDAQKVLDHQDVFGVLLQQVGTTGEIHDYTALISELKSRKIVVSVAADIMALVLLTAPGKQGADIVFGSAQRFGVPMGYGGPHAAFFAAKDEYKRSMPGRIIGVSKDAAGNTALRMAMQTREQHIRREKANSNICTSQVLLANIASLYAVYHGPVGLKRIANRIHRLTDILAAGLQQKGLKLRHAHYFDTLCVEVADKAGVLTRAEAAEINLRSDILNAVGITLDETTTRENVMQLFNVLLGDNHGLDIDTLDKDVAHDSRSIQPAMLRDDEILTHPVFNRYHSETEMMRYMHSLERKDLALNQAMIPLGSCTMKLNAAAEMIPITWPEFAELHPFCPPEQAEGYQQMIAQLADWLVKLTGYDAVCMQPNSGAQGEYAGLLAIRHYHESRNEGHRDICLIPASAHGTNPASAHMAGMQVVVVACDKNGNIDLTDLRAKAEQAGDNLSCIMVTYPSTHGVYEETIREVCEVVHQFGGQVYLDGANMNAQVGITSPGFIGADVSHLNLHKTFCIPHGGGGPGMGPIGVKAHLAPFVPGHSVVQIEGMLTRQGAVSAAPFGSASILPISWMYIRMMGAEGLKKASQVAILNANYIASRLQDAFPVLYTGRDGRVAHECILDIRPLKEETGISELDIAKRLIDYGFHAPTMSFPVAGTLMVEPTESESKVELDRFIDAMLAIRAEIDQVKAGVWPLEDNPLVNAPHIQSELVAEWAHPYSREVAVFPAGVADKYWPTVKRLDDVYGDRNLFCSCVPISEYQ',\n",
       "  'mw': 104376.0},\n",
       " 'b2904': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A6T9',\n",
       "  'gene': 'b2904',\n",
       "  'locus': ['b2904', 'JW2872'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'The glycine cleavage system catalyzes the degradation of glycine. The H protein shuttles the methylamine group of glycine from the P protein to the T protein'},\n",
       "  'seq': 'MSNVPAELKYSKEHEWLRKEADGTYTVGITEHAQELLGDMVFVDLPEVGATVSAGDDCAVAESVKAASDIYAPVSGEIVAVNDALSDSPELVNSEPYAGGWIFKIKASDESELESLLDATAYEALLEDE',\n",
       "  'mw': 13811.0},\n",
       " 'b2905': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P27248',\n",
       "  'gene': 'b2905',\n",
       "  'locus': ['b2905', 'JW2873'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(6S)-5,6,7,8-tetrahydrofolate + (R)-N(6)-(S(8)-aminomethyldihydrolipoyl)-L-lysyl-[protein] = (6R)-5,10-methylene-5,6,7,8-tetrahydrofolate + (R)-N(6)-dihydrolipoyl-L-lysyl-[protein] + NH4(+)',\n",
       "     '2.1.2.10')],\n",
       "   'Function': 'The glycine cleavage system catalyzes the degradation of glycine'},\n",
       "  'seq': 'MAQQTPLYEQHTLCGARMVDFHGWMMPLHYGSQIDEHHAVRTDAGMFDVSHMTIVDLRGSRTREFLRYLLANDVAKLTKSGKALYSGMLNASGGVIDDLIVYYFTEDFFRLVVNSATREKDLSWITQHAEPFGIEITVRDDLSMIAVQGPNAQAKAATLFNDAQRQAVEGMKPFFGVQAGDLFIATTGYTGEAGYEIALPNEKAADFWRALVEAGVKPCGLGARDTLRLEAGMNLYGQEMDETISPLAANMGWTIAWEPADRDFIGREALEVQREHGTEKLVGLVMTEKGVLRNELPVRFTDAQGNQHEGIITSGTFSPTLGYSIALARVPEGIGETAIVQIRNREMPVKVTKPVFVRNGKAVA',\n",
       "  'mw': 40147.0},\n",
       " 'b2907': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P25534',\n",
       "  'gene': 'b2907',\n",
       "  'locus': ['b2907', 'JW2875'],\n",
       "  'synonyms': 'visB',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Is likely an oxygenase that introduces the hydroxyl group at carbon four of 2-octaprenyl-6-methoxyphenol resulting in the formation of 2-octaprenyl-6-methoxy-1,4-benzoquinol',\n",
       "   'Pathway': 'Cofactor biosynthesis; ubiquinone biosynthesis'},\n",
       "  'seq': 'MSVIIVGGGMAGATLALAISRLSHGALPVHLIEATAPESHAHPGFDGRAIALAAGTCQQLARIGVWQSLADCATAITTVHVSDRGHAGFVTLAAEDYQLAALGQVVELHNVGQRLFALLRKAPGVTLHCPDRVANVARTQSHVEVTLESGETLTGRVLVAADGTHSALATACGVDWQQEPYEQLAVIANVATSVAHEGRAFERFTQHGPLAMLPMSDGRCSLVWCHPLERREEVLSWSDEKFCRELQSAFGWRLGKITHAGKRSAYPLALTHAARSITHRTVLVGNAAQTLHPIAGQGFNLGMRDVMSLAETLTQAQERGEDMGDYGVLCRYQQRRQSDREATIGVTDSLVHLFANRWAPLVVGRNIGLMTMELFTPARDVLAQRTLGWVAR',\n",
       "  'mw': 42288.0},\n",
       " 'b2913': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A9T0',\n",
       "  'gene': 'b2913',\n",
       "  'locus': ['b2913', 'JW2880'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(2R)-3-phosphoglycerate + NAD(+) = 3-phosphooxypyruvate + H(+) + NADH',\n",
       "     '1.1.1.95'),\n",
       "    ('(R)-2-hydroxyglutarate + NAD(+) = 2-oxoglutarate + H(+) + NADH',\n",
       "     '1.1.1.399')],\n",
       "   'Function': 'Catalyzes the reversible oxidation of 3-phospho-D-glycerate to 3-phosphonooxypyruvate, the first step of the phosphorylated L-serine biosynthesis pathway. Also catalyzes the reversible oxidation of 2-hydroxyglutarate to 2-oxoglutarate',\n",
       "   'Regulation Activity': 'Displays feedback inhibition by L-serine. Inhibited by glycine',\n",
       "   'Pathway': 'Amino-acid biosynthesis; L-serine biosynthesis; L-serine from 3-phospho-D-glycerate: step 1/3'},\n",
       "  'seq': 'MAKVSLEKDKIKFLLVEGVHQKALESLRAAGYTNIEFHKGALDDEQLKESIRDAHFIGLRSRTHLTEDVINAAEKLVAIGCFCIGTNQVDLDAAAKRGIPVFNAPFSNTRSVAELVIGELLLLLRGVPEANAKAHRGVWNKLAAGSFEARGKKLGIIGYGHIGTQLGILAESLGMYVYFYDIENKLPLGNATQVQHLSDLLNMSDVVSLHVPENPSTKNMMGAKEISLMKPGSLLINASRGTVVDIPALCDALASKHLAGAAIDVFPTEPATNSDPFTSPLCEFDNVLLTPHIGGSTQEAQENIGLEVAGKLIKYSDNGSTLSAVNFPEVSLPLHGGRRLMHIHENRPGVLTALNKIFAEQGVNIAAQYLQTSAQMGYVVIDIEADEDVAEKALQAMKAIPGTIRARLLY',\n",
       "  'mw': 44176.0},\n",
       " 'b2914': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A7Z0',\n",
       "  'gene': 'b2914',\n",
       "  'locus': ['b2914', 'JW5475'],\n",
       "  'synonyms': 'ygfC',\n",
       "  'properties': {'Catalytic Activity': [('aldehydo-D-ribose 5-phosphate = D-ribulose 5-phosphate',\n",
       "     '5.3.1.6')],\n",
       "   'Function': 'Involved in the first step of the non-oxidative branch of the pentose phosphate pathway. It catalyzes the reversible conversion of ribose-5-phosphate to ribulose 5-phosphate. Can also act on D-ribose-5-diphosphate and D-ribose-5-triphosphate as substrate',\n",
       "   'Regulation Activity': 'Inhibited by arabinose-5-phosphate, D-erythrose-4-phosphate and D-erythronic acid',\n",
       "   'Pathway': 'Carbohydrate degradation; pentose phosphate pathway; D-ribose 5-phosphate from D-ribulose 5-phosphate (non-oxidative stage): step 1/1'},\n",
       "  'seq': 'MTQDELKKAVGWAALQYVQPGTIVGVGTGSTAAHFIDALGTMKGQIEGAVSSSDASTEKLKSLGIHVFDLNEVDSLGIYVDGADEINGHMQMIKGGGAALTREKIIASVAEKFICIADASKQVDILGKFPLPVEVIPMARSAVARQLVKLGGRPEYRQGVVTDNGNVILDVHGMEILDPIAMENAINAIPGVVTVGLFANRGADVALIGTPDGVKTIVK',\n",
       "  'mw': 22860.0},\n",
       " 'b2917': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P27253',\n",
       "  'gene': 'b2917',\n",
       "  'locus': ['b2917', 'JW2884'],\n",
       "  'synonyms': 'sbm yliK',\n",
       "  'properties': {'Catalytic Activity': [('(R)-methylmalonyl-CoA = succinyl-CoA',\n",
       "     '5.4.99.2')],\n",
       "   'Function': 'Catalyzes the interconversion of succinyl-CoA and methylmalonyl-CoA. Could be part of a pathway that converts succinate to propionate'},\n",
       "  'seq': 'MSNVQEWQQLANKELSRREKTVDSLVHQTAEGIAIKPLYTEADLDNLEVTGTLPGLPPYVRGPRATMYTAQPWTIRQYAGFSTAKESNAFYRRNLAAGQKGLSVAFDLATHRGYDSDNPRVAGDVGKAGVAIDTVEDMKVLFDQIPLDKMSVSMTMNGAVLPVLAFYIVAAEEQGVTPDKLTGTIQNDILKEYLCRNTYIYPPKPSMRIIADIIAWCSGNMPRFNTISISGYHMGEAGANCVQQVAFTLADGIEYIKAAISAGLKIDDFAPRLSFFFGIGMDLFMNVAMLRAARYLWSEAVSGFGAQDPKSLALRTHCQTSGWSLTEQDPYNNVIRTTIEALAATLGGTQSLHTNAFDEALGLPTDFSARIARNTQIIIQEESELCRTVDPLAGSYYIESLTDQIVKQARAIIQQIDEAGGMAKAIEAGLPKRMIEEASAREQSLIDQGKRVIVGVNKYKLDHEDETDVLEIDNVMVRNEQIASLERIRATRDDAAVTAALNALTHAAQHNENLLAAAVNAARVRATLGEISDALEVAFDRYLVPSQCVTGVIAQSYHQSEKSASEFDAIVAQTEQFLADNGRRPRILIAKMGQDGHDRGAKVIASAYSDLGFDVDLSPMFSTPEEIARLAVENDVHVVGASSLAAGHKTLIPELVEALKKWGREDICVVAGGVIPPQDYAFLQERGVAAIYGPGTPMLDSVRDVLNLISQHHD',\n",
       "  'mw': 77871.0},\n",
       " 'b2919': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P52045',\n",
       "  'gene': 'b2919',\n",
       "  'locus': ['b2919', 'JW2886'],\n",
       "  'synonyms': 'mmcD ygfG',\n",
       "  'properties': {'Catalytic Activity': [('(R)-methylmalonyl-CoA + H(+) = CO2 + propanoyl-CoA',\n",
       "     '')],\n",
       "   'Function': 'Catalyzes the decarboxylation of (R)-methylmalonyl-CoA to propionyl-CoA. Could be part of a pathway that converts succinate to propanoate'},\n",
       "  'seq': 'MSYQYVNVVTINKVAVIEFNYGRKLNALSKVFIDDLMQALSDLNRPEIRCIILRAPSGSKVFSAGHDIHELPSGGRDPLSYDDPLRQITRMIQKFPKPIISMVEGSVWGGAFEMIMSSDLIIAASTSTFSMTPVNLGVPYNLVGIHNLTRDAGFHIVKELIFTASPITAQRALAVGILNHVVEVEELEDFTLQMAHHISEKAPLAIAVIKEELRVLGEAHTMNSDEFERIQGMRRAVYDSEDYQEGMNAFLEKRKPNFVGH',\n",
       "  'mw': 29173.0},\n",
       " 'b2920': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P52043',\n",
       "  'gene': 'b2920',\n",
       "  'locus': ['b2920', 'JW2887'],\n",
       "  'synonyms': 'ygfH',\n",
       "  'properties': {'Catalytic Activity': [('propanoyl-CoA + succinate = propanoate + succinyl-CoA',\n",
       "     '')],\n",
       "   'Function': 'Catalyzes the transfer of coenzyme A from propionyl-CoA to succinate. Could be part of a pathway that converts succinate to propionate'},\n",
       "  'seq': 'METQWTRMTANEAAEIIQHNDMVAFSGFTPAGSPKALPTAIARRANEQHEAKKPYQIRLLTGASISAAADDVLSDADAVSWRAPYQTSSGLRKKINQGAVSFVDLHLSEVAQMVNYGFFGDIDVAVIEASALAPDGRVWLTSGIGNAPTWLLRAKKVIIELNHYHDPRVAELADIVIPGAPPRRNSVSIFHAMDRVGTRYVQIDPKKIVAVVETNLPDAGNMLDKQNPMCQQIADNVVTFLLQEMAHGRIPPEFLPLQSGVGNINNAVMARLGENPVIPPFMMYSEVLQESVVHLLETGKISGASASSLTISADSLRKIYDNMDYFASRIVLRPQEISNNPEIIRRLGVIALNVGLEFDIYGHANSTHVAGVDLMNGIGGSGDFERNAYLSIFMAPSIAKEGKISTVVPMCSHVDHSEHSVKVIITEQGIADLRGLSPLQRARTIIDNCAHPMYRDYLHRYLENAPGGHIHHDLSHVFDLHRNLIATGSMLG',\n",
       "  'mw': 53824.0},\n",
       " 'b2923': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P11667',\n",
       "  'gene': 'b2923',\n",
       "  'locus': ['b2923', 'JW2890'],\n",
       "  'synonyms': 'yggA',\n",
       "  'properties': {'Catalytic Activity': [('L-arginine(in) = L-arginine(out)',\n",
       "     '')],\n",
       "   'Function': 'Involved in the export of arginine. Important to control the intracellular level of arginine and the correct balance between arginine and lysine. May also be involved in the export of canavanine (a plant-derived antimetabolite)'},\n",
       "  'seq': 'MFSYYFQGLALGAAMILPLGPQNAFVMNQGIRRQYHIMIALLCAISDLVLICAGIFGGSALLMQSPWLLALVTWGGVAFLLWYGFGAFKTAMSSNIELASAEVMKQGRWKIIATMLAVTWLNPHVYLDTFVVLGSLGGQLDVEPKRWFALGTISASFLWFFGLALLAAWLAPRLRTAKAQRIINLVVGCVMWFIALQLARDGIAHAQALFS',\n",
       "  'mw': 23176.0},\n",
       " 'b2925': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AB71',\n",
       "  'gene': 'b2925',\n",
       "  'locus': ['b2925', 'JW2892'],\n",
       "  'synonyms': 'fba fda',\n",
       "  'properties': {'Catalytic Activity': [('beta-D-fructose 1,6-bisphosphate = D-glyceraldehyde 3-phosphate + dihydroxyacetone phosphate',\n",
       "     '4.1.2.13'),\n",
       "    ('D-erythrose 4-phosphate + dihydroxyacetone phosphate = D-sedoheptulose 1,7-bisphosphate',\n",
       "     '')],\n",
       "   'Function': 'Catalyzes the aldol condensation of dihydroxyacetone phosphate (DHAP or glycerone-phosphate) with glyceraldehyde 3-phosphate (G3P) to form fructose 1,6-bisphosphate (FBP) in gluconeogenesis and the reverse reaction in glycolysis (PubMed:10712619). In addition, is involved in the utilization of D-sedoheptulose 7-phosphate, an intermediate of the pentose phosphate pathway, via the sedoheptulose bisphospate bypass pathway (PubMed:19756045). Catalyzes the cleavage of D-sedoheptulose 1,7-bisphosphate to glyceraldehyde 3-phosphate and erythrose 4-phosphate (PubMed:19756045)',\n",
       "   'Pathway': 'Carbohydrate degradation; glycolysis; D-glyceraldehyde 3-phosphate and glycerone phosphate from D-glucose: step 4/4'},\n",
       "  'seq': 'MSKIFDFVKPGVITGDDVQKVFQVAKENNFALPAVNCVGTDSINAVLETAAKVKAPVIVQFSNGGASFIAGKGVKSDVPQGAAILGAISGAHHVHQMAEHYGVPVILHTDHCAKKLLPWIDGLLDAGEKHFAATGKPLFSSHMIDLSEESLQENIEICSKYLERMSKIGMTLEIELGCTGGEEDGVDNSHMDASALYTQPEDVDYAYTELSKISPRFTIAASFGNVHGVYKPGNVVLTPTILRDSQEYVSKKHNLPHNSLNFVFHGGSGSTAQEIKDSVSYGVVKMNIDTDTQWATWEGVLNYYKANEAYLQGQLGNPKGEDQPNKKYYDPRVWLRAGQTSMIARLEKAFQELNAIDVL',\n",
       "  'mw': 39147.0},\n",
       " 'b2926': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A799',\n",
       "  'gene': 'b2926',\n",
       "  'locus': ['b2926', 'JW2893'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(2R)-3-phosphoglycerate + ATP = (2R)-3-phospho-glyceroyl phosphate + ADP',\n",
       "     '2.7.2.3')],\n",
       "   'Pathway': 'Carbohydrate degradation; glycolysis; pyruvate from D-glyceraldehyde 3-phosphate: step 2/5'},\n",
       "  'seq': 'MSVIKMTDLDLAGKRVFIRADLNVPVKDGKVTSDARIRASLPTIELALKQGAKVMVTSHLGRPTEGEYNEEFSLLPVVNYLKDKLSNPVRLVKDYLDGVDVAEGELVVLENVRFNKGEKKDDETLSKKYAALCDVFVMDAFGTAHRAQASTHGIGKFADVACAGPLLAAELDALGKALKEPARPMVAIVGGSKVSTKLTVLDSLSKIADQLIVGGGIANTFIAAQGHDVGKSLYEADLVDEAKRLLTTCNIPVPSDVRVATEFSETAPATLKSVNDVKADEQILDIGDASAQELAEILKNAKTILWNGPVGVFEFPNFRKGTEIVANAIADSEAFSIAGGGDTLAAIDLFGIADKISYISTGGGAFLEFVEGKVLPAVAMLEERAKK',\n",
       "  'mw': 41118.0},\n",
       " 'b2927': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A9B6',\n",
       "  'gene': 'b2927',\n",
       "  'locus': ['b2927', 'JW2894'],\n",
       "  'synonyms': 'gapB',\n",
       "  'properties': {'Catalytic Activity': [('D-erythrose 4-phosphate + H2O + NAD(+) = 4-phospho-D-erythronate + 2 H(+) + NADH',\n",
       "     '1.2.1.72')],\n",
       "   'Function': 'Catalyzes the NAD-dependent conversion of D-erythrose 4-phosphate to 4-phosphoerythronate',\n",
       "   'Pathway': \"Cofactor biosynthesis; pyridoxine 5'-phosphate biosynthesis; pyridoxine 5'-phosphate from D-erythrose 4-phosphate: step 1/5\"},\n",
       "  'seq': 'MTVRVAINGFGRIGRNVVRALYESGRRAEITVVAINELADAAGMAHLLKYDTSHGRFAWEVRQERDQLFVGDDAIRVLHERSLQSLPWRELGVDVVLDCTGVYGSREHGEAHIAAGAKKVLFSHPGSNDLDATVVYGVNQDQLRAEHRIVSNASCTTNCIIPVIKLLDDAYGIESGTVTTIHSAMHDQQVIDAYHPDLRRTRAASQSIIPVDTKLAAGITRFFPQFNDRFEAIAVRVPTINVTAIDLSVTVKKPVKANEVNLLLQKAAQGAFHGIVDYTELPLVSVDFNHDPHSAIVDGTQTRVSGAHLIKTLVWCDNEWGFANRMLDTTLAMATVAFR',\n",
       "  'mw': 37299.0},\n",
       " 'b2935': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P27302',\n",
       "  'gene': 'b2935',\n",
       "  'locus': ['b2935', 'JW5478'],\n",
       "  'synonyms': 'tkt',\n",
       "  'properties': {'Catalytic Activity': [('D-glyceraldehyde 3-phosphate + D-sedoheptulose 7-phosphate = aldehydo-D-ribose 5-phosphate + D-xylulose 5-phosphate',\n",
       "     '2.2.1.1')],\n",
       "   'Function': 'Catalyzes the transfer of a two-carbon ketol group from a ketose donor to an aldose acceptor, via a covalent intermediate with the cofactor thiamine pyrophosphate. Thus, catalyzes the reversible transfer of a two-carbon ketol group from sedoheptulose-7-phosphate to glyceraldehyde-3-phosphate, producing xylulose-5-phosphate and ribose-5-phosphate'},\n",
       "  'seq': 'MSSRKELANAIRALSMDAVQKAKSGHPGAPMGMADIAEVLWRDFLKHNPQNPSWADRDRFVLSNGHGSMLIYSLLHLTGYDLPMEELKNFRQLHSKTPGHPEVGYTAGVETTTGPLGQGIANAVGMAIAEKTLAAQFNRPGHDIVDHYTYAFMGDGCMMEGISHEVCSLAGTLKLGKLIAFYDDNGISIDGHVEGWFTDDTAMRFEAYGWHVIRDIDGHDAASIKRAVEEARAVTDKPSLLMCKTIIGFGSPNKAGTHDSHGAPLGDAEIALTREQLGWKYAPFEIPSEIYAQWDAKEAGQAKESAWNEKFAAYAKAYPQEAAEFTRRMKGEMPSDFDAKAKEFIAKLQANPAKIASRKASQNAIEAFGPLLPEFLGGSADLAPSNLTLWSGSKAINEDAAGNYIHYGVREFGMTAIANGISLHGGFLPYTSTFLMFVEYARNAVRMAALMKQRQVMVYTHDSIGLGEDGPTHQPVEQVASLRVTPNMSTWRPCDQVESAVAWKYGVERQDGPTALILSRQNLAQQERTEEQLANIARGGYVLKDCAGQPELIFIATGSEVELAVAAYEKLTAEGVKARVVSMPSTDAFDKQDAAYRESVLPKAVTARVAVEAGIADYWYKYVGLNGAIVGMTTFGESAPAELLFEEFGFTVDNVVAKAKELL',\n",
       "  'mw': 72212.0},\n",
       " 'b2937': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P60651',\n",
       "  'gene': 'b2937',\n",
       "  'locus': ['b2937', 'JW2904'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('agmatine + H2O = putrescine + urea',\n",
       "     '3.5.3.11')],\n",
       "   'Function': 'Catalyzes the formation of putrescine from agmatine',\n",
       "   'Regulation Activity': 'The expression of auh activity is antagonistically regulated by cyclic AMP and agmatine. In the presence of the cAMP receptor protein, cAMP represses the expression of AUH, while agmatine induces it',\n",
       "   'Pathway': 'Amine and polyamine biosynthesis; putrescine biosynthesis via agmatine pathway; putrescine from agmatine: step 1/1'},\n",
       "  'seq': 'MSTLGHQYDNSLVSNAFGFLRLPMNFQPYDSDADWVITGVPFDMATSGRAGGRHGPAAIRQVSTNLAWEHNRFPWNFDMRERLNVVDCGDLVYAFGDAREMSEKLQAHAEKLLAAGKRMLSFGGDHFVTLPLLRAHAKHFGKMALVHFDAHTDTYANGCEFDHGTMFYTAPKEGLIDPNHSVQIGIRTEFDKDNGFTVLDACQVNDRSVDDVIAQVKQIVGDMPVYLTFDIDCLDPAFAPGTGTPVIGGLTSDRAIKLVRGLKDLNIVGMDVVEVAPAYDQSEITALAAATLALEMLYIQAAKKGE',\n",
       "  'mw': 33557.0},\n",
       " 'b2938': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P21170',\n",
       "  'gene': 'b2938',\n",
       "  'locus': ['b2938', 'JW2905'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('H(+) + L-arginine = agmatine + CO2',\n",
       "     '4.1.1.19')],\n",
       "   'Function': 'Catalyzes the biosynthesis of agmatine from arginine',\n",
       "   'Regulation Activity': 'Down-regulated by polyamine end products putrescine and spermidine',\n",
       "   'Pathway': 'Amine and polyamine biosynthesis; agmatine biosynthesis; agmatine from L-arginine: step 1/1'},\n",
       "  'seq': 'MSDDMSMGLPSSAGEHGVLRSMQEVAMSSQEASKMLRTYNIAWWGNNYYDVNELGHISVCPDPDVPEARVDLAQLVKTREAQGQRLPALFCFPQILQHRLRSINAAFKRARESYGYNGDYFLVYPIKVNQHRRVIESLIHSGEPLGLEAGSKAELMAVLAHAGMTRSVIVCNGYKDREYIRLALIGEKMGHKVYLVIEKMSEIAIVLDEAERLNVVPRLGVRARLASQGSGKWQSSGGEKSKFGLAATQVLQLVETLREAGRLDSLQLLHFHLGSQMANIRDIATGVRESARFYVELHKLGVNIQCFDVGGGLGVDYEGTRSQSDCSVNYGLNEYANNIIWAIGDACEENGLPHPTVITESGRAVTAHHTVLVSNIIGVERNEYTVPTAPAEDAPRALQSMWETWQEMHEPGTRRSLREWLHDSQMDLHDIHIGYSSGIFSLQERAWAEQLYLSMCHEVQKQLDPQNRAHRPIIDELQERMADKMYVNFSLFQSMPDAWGIDQLFPVLPLEGLDQVPERRAVLLDITCDSDGAIDHYIDGDGIATTMPMPEYDPENPPMLGFFMVGAYQEILGNMHNLFGDTEAVDVFVFPDGSVEVELSDEGDTVADMLQYVQLDPKTLLTQFRDQVKKTDLDAELQQQFLEEFEAGLYGYTYLEDE',\n",
       "  'mw': 73898.0},\n",
       " 'b2942': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A817',\n",
       "  'gene': 'b2942',\n",
       "  'locus': ['b2942', 'JW2909'],\n",
       "  'synonyms': 'metX',\n",
       "  'properties': {'Catalytic Activity': [('ATP + H2O + L-methionine = diphosphate + phosphate + S-adenosyl-L-methionine',\n",
       "     '2.5.1.6')],\n",
       "   'Function': 'Catalyzes the formation of S-adenosylmethionine (AdoMet) from methionine and ATP. The overall synthetic reaction is composed of two sequential steps, AdoMet formation and the subsequent tripolyphosphate hydrolysis which occurs prior to release of AdoMet from the enzyme (PubMed:6251075, PubMed:7629147, PubMed:7629176, PubMed:9753435, PubMed:10551856, PubMed:10660564). Is essential for growth (PubMed:11952912)',\n",
       "   'Pathway': 'Amino-acid biosynthesis; S-adenosyl-L-methionine biosynthesis; S-adenosyl-L-methionine from L-methionine: step 1/1'},\n",
       "  'seq': 'MAKHLFTSESVSEGHPDKIADQISDAVLDAILEQDPKARVACETYVKTGMVLVGGEITTSAWVDIEEITRNTVREIGYVHSDMGFDANSCAVLSAIGKQSPDINQGVDRADPLEQGAGDQGLMFGYATNETDVLMPAPITYAHRLVQRQAEVRKNGTLPWLRPDAKSQVTFQYDDGKIVGIDAVVLSTQHSEEIDQKSLQEAVMEEIIKPILPAEWLTSATKFFINPTGRFVIGGPMGDCGLTGRKIIVDTYGGMARHGGGAFSGKDPSKVDRSAAYAARYVAKNIVAAGLADRCEIQVSYAIGVAEPTSIMVETFGTEKVPSEQLTLLVREFFDLRPYGLIQMLDLLHPIYKETAAYGHFGREHFPWEKTDKAQLLRDAAGLK',\n",
       "  'mw': 41952.0},\n",
       " 'b2943': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AEP1',\n",
       "  'gene': 'b2943',\n",
       "  'locus': ['b2943', 'JW2910'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Uptake of galactose across the boundary membrane with the concomitant transport of protons into the cell (symport system)'},\n",
       "  'seq': 'MPDAKKQGRSNKAMTFFVCFLAALAGLLFGLDIGVIAGALPFIADEFQITSHTQEWVVSSMMFGAAVGAVGSGWLSFKLGRKKSLMIGAILFVAGSLFSAAAPNVEVLILSRVLLGLAVGVASYTAPLYLSEIAPEKIRGSMISMYQLMITIGILGAYLSDTAFSYTGAWRWMLGVIIIPAILLLIGVFFLPDSPRWFAAKRRFVDAERVLLRLRDTSAEAKRELDEIRESLQVKQSGWALFKENSNFRRAVFLGVLLQVMQQFTGMNVIMYYAPKIFELAGYTNTTEQMWGTVIVGLTNVLATFIAIGLVDRWGRKPTLTLGFLVMAAGMGVLGTMMHIGIHSPSAQYFAIAMLLMFIVGFAMSAGPLIWVLCSEIQPLKGRDFGITCSTATNWIANMIVGATFLTMLNTLGNANTFWVYAALNVLFILLTLWLVPETKHVSLEHIERNLMKGRKLREIGAHD',\n",
       "  'mw': 50983.0},\n",
       " 'b2947': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P04425',\n",
       "  'gene': 'b2947',\n",
       "  'locus': ['b2947', 'JW2914'],\n",
       "  'synonyms': 'gsh-II',\n",
       "  'properties': {'Catalytic Activity': [('ATP + gamma-L-glutamyl-L-cysteine + glycine = ADP + glutathione + H(+) + phosphate',\n",
       "     '6.3.2.3')],\n",
       "   'Regulation Activity': 'Inhibited by 7,8-dihydrofolate, methotrexate and trimethoprim',\n",
       "   'Pathway': 'Sulfur metabolism; glutathione biosynthesis; glutathione from L-cysteine and L-glutamate: step 2/2'},\n",
       "  'seq': 'MIKLGIVMDPIANINIKKDSSFAMLLEAQRRGYELHYMEMGDLYLINGEARAHTRTLNVKQNYEEWFSFVGEQDLPLADLDVILMRKDPPFDTEFIYATYILERAEEKGTLIVNKPQSLRDCNEKLFTAWFSDLTPETLVTRNKAQLKAFWEKHSDIILKPLDGMGGASIFRVKEGDPNLGVIAETLTEHGTRYCMAQNYLPAIKDGDKRVLVVDGEPVPYCLARIPQGGETRGNLAAGGRGEPRPLTESDWKIARQIGPTLKEKGLIFVGLDIIGDRLTEINVTSPTCIREIEAEFPVSITGMLMDAIEARLQQQ',\n",
       "  'mw': 35561.0},\n",
       " 'b2954': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P52061',\n",
       "  'gene': 'b2954',\n",
       "  'locus': ['b2954', 'JW2921'],\n",
       "  'synonyms': 'yggV',\n",
       "  'properties': {'Catalytic Activity': [('H2O + XTP = diphosphate + H(+) + XMP',\n",
       "     '3.6.1.66'),\n",
       "    ('dITP + H2O = dIMP + diphosphate + H(+)', '3.6.1.66'),\n",
       "    ('H2O + ITP = diphosphate + H(+) + IMP', '3.6.1.66')],\n",
       "   'Function': \"Pyrophosphatase that catalyzes the hydrolysis of nucleoside triphosphates to their monophosphate derivatives, with a high preference for the non-canonical purine nucleotides XTP (xanthosine triphosphate), dITP (deoxyinosine triphosphate) and ITP (PubMed:12297000, PubMed:17976651). Can also efficiently hydrolyze 2'-deoxy-N-6-hydroxylaminopurine triphosphate (dHAPTP) (PubMed:17090528). Seems to function as a house-cleaning enzyme that removes non-canonical purine nucleotides from the nucleotide pool, thus preventing their incorporation into DNA/RNA and avoiding chromosomal lesions (PubMed:12297000, PubMed:12730170, PubMed:17090528). To a much lesser extent, is also able to hydrolyze GTP, dGTP and dUTP, but shows very low activity toward the canonical nucleotides dATP, dCTP and dTTP and toward 8-oxo-dGTP, purine deoxyribose triphosphate, 2-aminopurine deoxyribose triphosphate and 2,6-diaminopurine deoxyribose triphosphate (PubMed:12297000, PubMed:17090528)\"},\n",
       "  'seq': 'MQKVVLATGNVGKVRELASLLSDFGLDIVAQTDLGVDSAEETGLTFIENAILKARHAAKVTALPAIADDSGLAVDVLGGAPGIYSARYSGEDATDQKNLQKLLETMKDVPDDQRQARFHCVLVYLRHAEDPTPLVCHGSWPGVITREPAGTGGFGYDPIFFVPSEGKTAAELTREEKSAISHRGQALKLLLDALRNG',\n",
       "  'mw': 21039.0},\n",
       " 'b2957': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P00805',\n",
       "  'gene': 'b2957',\n",
       "  'locus': ['b2957', 'JW2924'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('H2O + L-asparagine = L-aspartate + NH4(+)',\n",
       "     '3.5.1.1')]},\n",
       "  'seq': 'MEFFKKTALAALVMGFSGAALALPNITILATGGTIAGGGDSATKSNYTVGKVGVENLVNAVPQLKDIANVKGEQVVNIGSQDMNDNVWLTLAKKINTDCDKTDGFVITHGTDTMEETAYFLDLTVKCDKPVVMVGAMRPSTSMSADGPFNLYNAVVTAADKASANRGVLVVMNDTVLDGRDVTKTNTTDVATFKSVNYGPLGYIHNGKIDYQRTPARKHTSDTPFDVSKLNELPKVGIVYNYANASDLPAKALVDAGYDGIVSAGVGNGNLYKSVFDTLATAAKTGTAVVRSSRVPTGATTQDAEVDDAKYGFVASGTLNPQKARVLLQLALTQTKDPQQIQQIFNQY',\n",
       "  'mw': 36851.0},\n",
       " 'b2963': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0C066',\n",
       "  'gene': 'b2963',\n",
       "  'locus': ['b2963', 'JW5481'],\n",
       "  'synonyms': 'yggZ',\n",
       "  'properties': {'Catalytic Activity': [('Exolytic cleavage of the (1->4)-beta-glycosidic linkage between N-acetylmuramic acid (MurNAc) and N-acetylglucosamine (GlcNAc) residues in peptidoglycan, from either the reducing or the non-reducing ends of the peptidoglycan chains, with concomitant formation of a 1,6-anhydrobond in the MurNAc residue.',\n",
       "     '4.2.2.n1')],\n",
       "   'Function': 'Murein-degrading enzyme. May play a role in recycling of muropeptides during cell elongation and/or cell division'},\n",
       "  'seq': 'MKKYLALALIAPLLISCSTTKKGDTYNEAWVKDTNGFDILMGQFAHNIENIWGFKEVVIAGPKDYVKYTDQYQTRSHINFDDGTITIETIAGTEPAAHLRRAIIKTLLMGDDPSSVDLYSDVDDITISKEPFLYGQVVDNTGQPIRWEGRASNFADYLLKNRLKSRSNGLRIIYSVTINMVPNHLDKRAHKYLGMVRQASRKYGVDESLILAIMQTESSFNPYAVSRSDALGLMQVVQHTAGKDVFRSQGKSGTPSRSFLFDPASNIDTGTAYLAMLNNVYLGGIDNPTSRRYAVITAYNGGAGSVLRVFSNDKIQAANIINTMTPGDVYQTLTTRHPSAESRRYLYKVNTAQKSYRRR',\n",
       "  'mw': 40113.0},\n",
       " 'b2964': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AFF4',\n",
       "  'gene': 'b2964',\n",
       "  'locus': ['b2964', 'JW2932'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('adenosine(in) + H(+)(in) = adenosine(out) + H(+)(out)',\n",
       "     ''),\n",
       "    ('H(+)(in) + uridine(in) = H(+)(out) + uridine(out)', ''),\n",
       "    ('H(+)(in) + thymidine(in) = H(+)(out) + thymidine(out)', ''),\n",
       "    ('cytidine(in) + H(+)(in) = cytidine(out) + H(+)(out)', ''),\n",
       "    (\"2'-deoxycytidine(in) + H(+)(in) = 2'-deoxycytidine(out) + H(+)(out)\",\n",
       "     ''),\n",
       "    ('guanosine(in) + H(+)(in) = guanosine(out) + H(+)(out)', ''),\n",
       "    ('H(+)(in) + inosine(in) = H(+)(out) + inosine(out)', '')],\n",
       "   'Function': 'Broad-specificity transporter of purine and pyrimidine nucleosides (PubMed:374403, PubMed:15678184, PubMed:3311747, PubMed:11466294, PubMed:15513740). Can transport adenosine, uridine, thymidine, cytidine, deoxycytidine, guanosine and inosine (PubMed:374403, PubMed:15678184, PubMed:3311747, PubMed:11466294, PubMed:15513740). Can also transport xanthosine, but with a very low affinity (PubMed:11466294). Transport is driven by a proton motive force (PubMed:374403, PubMed:15513740)',\n",
       "   'Regulation Activity': 'Inhibited by the protonophore uncouplers 2,4-dinitrophenol and carbonyl cyanide m-chlorophenylhydrazone (CCCP), and by valinomycin (PubMed:15513740, PubMed:374403). Inhibited by the nucleoside antibiotic showdomycin (PubMed:374403)'},\n",
       "  'seq': 'MNLKLQLKILSFLQFCLWGSWLTTLGSYMFVTLKFDGASIGAVYSSLGIAAVFMPALLGIVADKWLSAKWVYAICHTIGAITLFMAAQVTTPEAMFLVILINSFAYMPTLGLINTISYYRLQNAGMDIVTDFPPIRIWGTIGFIMAMWVVSLSGFELSHMQLYIGAALSAILVLFTLTLPHIPVAKQQANQSWTTLLGLDAFALFKNKRMAIFFIFSMLLGAELQITNMFGNTFLHSFDKDPMFASSFIVQHASIIMSISQISETLFILTIPFFLSRYGIKNVMMISIVAWILRFALFAYGDPTPFGTVLLVLSMIVYGCAFDFFNISGSVFVEKEVSPAIRASAQGMFLMMTNGFGCILGGIVSGKVVEMYTQNGITDWQTVWLIFAGYSVVLAFAFMAMFKYKHVRVPTGTQTVSH',\n",
       "  'mw': 46389.0},\n",
       " 'b2965': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P21169',\n",
       "  'gene': 'b2965',\n",
       "  'locus': ['b2965', 'JW5482'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('H(+) + L-ornithine = CO2 + putrescine',\n",
       "     '4.1.1.17')],\n",
       "   'Pathway': 'Amine and polyamine biosynthesis; putrescine biosynthesis via L-ornithine pathway; putrescine from L-ornithine: step 1/1'},\n",
       "  'seq': 'MKSMNIAASSELVSRLSSHRRVVALGDTDFTDVAAVVITAADSRSGILALLKRTGFHLPVFLYSEHAVELPAGVTAVINGNEQQWLELESAACQYEENLLPPFYDTLTQYVEMGNSTFACPGHQHGAFFKKHPAGRHFYDFFGENVFRADMCNADVKLGDLLIHEGSAKDAQKFAAKVFHADKTYFVLNGTSAANKVVTNALLTRGDLVLFDRNNHKSNHHGALIQAGATPVYLEASRNPFGFIGGIDAHCFNEEYLRQQIRDVAPEKADLPRPYRLAIIQLGTYDGTVYNARQVIDTVGHLCDYILFDSAWVGYEQFIPMMADSSPLLLELNENDPGIFVTQSVHKQQAGFSQTSQIHKKDNHIRGQARFCPHKRLNNAFMLHASTSPFYPLFAALDVNAKIHEGESGRRLWAECVEIGIEARKAILARCKLFRPFIPPVVDGKLWQDYPTSVLASDRRFFSFEPGAKWHGFEGYAADQYFVDPCKLLLTTPGIDAETGEYSDFGVPATILAHYLRENGIVPEKCDLNSILFLLTPAESHEKLAQLVAMLAQFEQHIEDDSPLVEVLPSVYNKYPVRYRDYTLRQLCQEMHDLYVSFDVKDLQKAMFRQQSFPSVVMNPQDAHSAYIRGDVELVRIRDAEGRIAAEGALPYPPGVLCVVPGEVWGGAVQRYFLALEEGVNLLPGFSPELQGVYSETDADGVKRLYGYVLK',\n",
       "  'mw': 79416.0},\n",
       " 'b2975': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'Q46839',\n",
       "  'gene': 'b2975',\n",
       "  'locus': ['b2975', 'JW2942'],\n",
       "  'synonyms': 'yghK',\n",
       "  'properties': {'Catalytic Activity': [('glycolate(in) + H(+)(in) = glycolate(out) + H(+)(out)',\n",
       "     ''),\n",
       "    ('(S)-lactate(in) + H(+)(in) = (S)-lactate(out) + H(+)(out)', ''),\n",
       "    ('(R)-lactate(in) + H(+)(in) = (R)-lactate(out) + H(+)(out)', '')],\n",
       "   'Function': 'Uptake of glycolate across the membrane (PubMed:11283302, PubMed:11785976). Can also transport L-lactate and D-lactate (PubMed:11283302, PubMed:11785976). Seems to be driven by a proton motive force (PubMed:11785976)',\n",
       "   'Regulation Activity': 'Inhibited by the proton ionophore carbonyl cyanide m-chlorophenylhydrazone (CCCP)'},\n",
       "  'seq': 'MVTWTQMYMPMGGLGLSALVALIPIIFFFVALAVLRLKGHVAGAITLILSILIAIFAFKMPIDMAFAAAGYGFIYGLWPIAWIIVAAVFLYKLTVASGQFDIIRSSVISITDDQRLQVLLIGFSFGALLEGAAGFGAPVAITGALLVGLGFKPLYAAGLCLIANTAPVAFGALGVPILVAGQVTGIDPFHIGAMAGRQLPFLSVLVPFWLVAMMDGWKGVKETWPAALVAGGSFAVTQFFTSNYIGPELPDITSALVSIVSLALFLKVWRPKNTETAISMGQSAGAMVVNKPSSGGPVPSEYSLGQIIRAWSPFLILTVLVTIWTMKPFKALFAPGGAFYSLVINFQIPHLHQQVLKAAPIVAQPTPMDAVFKFDPLSAGGTAIFIAAIISIFILGVGIKKGIGVFAETLISLKWPILSIGMVLAFAFVTNYSGMSTTLALVLAGTGVMFPFFSPFLGWLGVFLTGSDTSSNALFGSLQSTTAQQINVSDTLLVAANTSGGVTGKMISPQSIAVACAATGMVGRESELFRYTVKHSLIFASVIGIITLLQAYVFTGMLVS',\n",
       "  'mw': 58920.0},\n",
       " 'b2976': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37330',\n",
       "  'gene': 'b2976',\n",
       "  'locus': ['b2976', 'JW2943'],\n",
       "  'synonyms': 'glc',\n",
       "  'properties': {'Catalytic Activity': [('acetyl-CoA + glyoxylate + H2O = (S)-malate + CoA + H(+)',\n",
       "     '2.3.3.9')],\n",
       "   'Function': 'Involved in the glycolate utilization. Catalyzes the condensation and subsequent hydrolysis of acetyl-coenzyme A (acetyl-CoA) and glyoxylate to form malate and CoA',\n",
       "   'Pathway': 'Carbohydrate metabolism; glyoxylate cycle; (S)-malate from isocitrate: step 2/2'},\n",
       "  'seq': 'MSQTITQSRLRIDANFKRFVDEEVLPGTGLDAAAFWRNFDEIVHDLAPENRQLLAERDRIQAALDEWHRSNPGPVKDKAAYKSFLRELGYLVPQPERVTVETTGIDSEITSQAGPQLVVPAMNARYALNAANARWGSLYDALYGSDIIPQEGAMVSGYDPQRGEQVIAWVRRFLDESLPLENGSYQDVVAFKVVDKQLRIQLKNGKETTLRTPAQFVGYRGDAAAPTCILLKNNGLHIELQIDANGRIGKDDPAHINDVIVEAAISTILDCEDSVAAVDAEDKILLYRNLLGLMQGTLQEKMEKNGRQIVRKLNDDRHYTAADGSEISLHGRSLLFIRNVGHLMTIPVIWDSEGNEIPEGILDGVMTGAIALYDLKVQKNSRTGSVYIVKPKMHGPQEVAFANKLFTRIETMLGMAPNTLKMGIMDEERRTSLNLRSCIAQARNRVAFINTGFLDRTGDEMHSVMEAGPMLRKNQMKSTPWIKAYERNNVLSGLFCGLRGKAQIGKGMWAMPDLMADMYSQKGDQLRAGANTAWVPSPTAATLHALHYHQTNVQSVQANIAQTEFNAEFEPLLDDLLTIPVAENANWSAQEIQQELDNNVQGILGYVVRWVEQGIGCSKVPDIHNVALMEDRATLRISSQHIANWLRHGILTKEQVQASLENMAKVVDQQNAGDPAYRPMAGNFANSCAFKAASDLIFLGVKQPNGYTEPLLHAWRLREKESH',\n",
       "  'mw': 80489.0},\n",
       " 'b2979': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AEP9',\n",
       "  'gene': 'b2979',\n",
       "  'locus': ['b2979', 'JW2946'],\n",
       "  'synonyms': 'gox yghM',\n",
       "  'properties': {'Catalytic Activity': [('A + glycolate = AH2 + glyoxylate',\n",
       "     '1.1.99.14'),\n",
       "    ('(R)-lactate + A = AH2 + pyruvate', '')],\n",
       "   'Function': 'Component of a complex that catalyzes the oxidation of glycolate to glyoxylate (PubMed:4557653, PubMed:8606183). Is required for E.coli to grow on glycolate as a sole source of carbon (PubMed:8606183). Is also able to oxidize D-lactate ((R)-lactate) with a similar rate (PubMed:4557653). Does not link directly to O(2), and 2,6-dichloroindophenol (DCIP) and phenazine methosulfate (PMS) can act as artificial electron acceptors in vitro, but the physiological molecule that functions as primary electron acceptor during glycolate oxidation is unknown (PubMed:4557653)',\n",
       "   'Regulation Activity': 'In vitro the glycolate oxidase activity is inhibited by the sulfhydryl inhibitors CuSO4 and PCMB, by KCN, but not by the metal complexing agent EDTA'},\n",
       "  'seq': 'MSILYEERLDGALPDVDRTSVLMALREHVPGLEILHTDEEIIPYECDGLSAYRTRPLLVVLPKQMEQVTAILAVCHRLRVPVVTRGAGTGLSGGALPLEKGVLLVMARFKEILDINPVGRRARVQPGVRNLAISQAVAPHNLYYAPDPSSQIACSIGGNVAENAGGVHCLKYGLTVHNLLKIEVQTLDGEALTLGSDALDSPGFDLLALFTGSEGMLGVTTEVTVKLLPKPPVARVLLASFDSVEKAGLAVGDIIANGIIPGGLEMMDNLSIRAAEDFIHAGYPVDAEAILLCELDGVESDVQEDCERVNDILLKAGATDVRLAQDEAERVRFWAGRKNAFPAVGRISPDYYCMDGTIPRRALPGVLEGIARLSQQYDLRVANVFHAGDGNMHPLILFDANEPGEFARAEELGGKILELCVEVGGSISGEHGIGREKINQMCAQFNSDEITTFHAVKAAFDPDGLLNPGKNIPTLHRCAEFGAMHVHHGHLPFPELERF',\n",
       "  'mw': 53812.0},\n",
       " 'b2987': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P43676',\n",
       "  'gene': 'b2987',\n",
       "  'locus': ['b2987', 'JW2955'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('H(+)(in) + phosphate(in) = H(+)(out) + phosphate(out)',\n",
       "     '')],\n",
       "   'Function': 'Low-affinity inorganic phosphate transporter'},\n",
       "  'seq': 'MLNLFVGLDIYTGLLLLLALAFVLFYEAINGFHDTANAVAAVIYTRAMQPQLAVVMAAFFNFFGVLLGGLSVAYAIVHMLPTDLLLNMGSTHGLAMVFSMLLAAIIWNLGTWFFGLPASSSHTLIGAIIGIGLTNALLTGSSVMDALNLREVTKIFSSLIVSPIVGLVIAGGLIFLLRRYWSGTKKRDRIHRIPEDRKKKKGKRKPPFWTRIALIVSAAGVAFSHGANDGQKGIGLVMLVLVGIAPAGFVVNMNASGYEITRTRDAVTNFEHYLQQHPELPQKLIAMEPPLPAASTDGTQVTEFHCHPANTFDAIARVKTMLPGNMESYEPLSVSQRSQLRRIMLCISDTSAKLAKLPGVSKEDQNLLKKLRSDMLSTIEYAPVWIIMAVALALGIGTMIGWRRVAMTIGEKIGKRGMTYAQGMAAQMTAAVSIGLASYIGMPVSTTHVLSSAVAGTMVVDGGGLQRKTVTSILMAWVFTLPAAIFLSGGLYWIALQLI',\n",
       "  'mw': 53809.0},\n",
       " 'b2988': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AES0',\n",
       "  'gene': 'b2988',\n",
       "  'locus': ['b2988', 'JW2956'],\n",
       "  'synonyms': 'gsp',\n",
       "  'properties': {'Catalytic Activity': [('ATP + glutathione + spermidine = ADP + glutathionylspermidine + H(+) + phosphate',\n",
       "     '6.3.1.8'),\n",
       "    ('glutathionylspermidine + H2O = glutathione + spermidine', '3.5.1.78')],\n",
       "   'Function': 'Catalyzes the formation of an amide bond between glutathione (GSH) and spermidine coupled with hydrolysis of ATP; also catalyzes the opposing reaction, i.e. the hydrolysis of glutathionylspermidine (Gsp) back to glutathione and spermidine. The amidase active site can also hydrolyze Gsp-disulfide (Gsp-S-S-Gsp) to Gsp-SG and Gsp S-thiolated proteins (GspSSPs) to GSH S-thiolated protein (GSSPs). Likely acts synergistically with glutaredoxin to regulate the redox environment of E.coli and defend against oxidative damage. In vitro, the amidase active site also catalyzes hydrolysis of amide and ester derivatives of glutathione (e.g. glutathione ethyl ester and glutathione amide) but lacks activity toward acetylspermidine (N1 and N8) and acetylspermine (N1)',\n",
       "   'Regulation Activity': 'When exposed to oxidative stress, Gsp amidase activity is transiently inhibited in vivo by oxidation of the catalytic Cys-59 thiol to sulfenic acid; this modification does not affect Gsp synthetase activity. Gsp amidase activity is negatively autoregulated by the Gsp synthetase domain, and is activated by the Gsp synthetase substrates, GSH and ATP-Mg(2+); the occupancy of the synthetase active site may initiate communication through the protein as manifest by the release of inhibition of the amidase activity. A tetrahedral phosphonate analog of glutathionylspermidine, designed as a mimic of the proposed tetrahedral intermediate for either reaction, inhibits the synthetase activity (Ki of 10 uM) but does not inhibit the amidase activity. Amidase activity is inhibited by iodoacetamide in vitro',\n",
       "   'Pathway': 'Amine and polyamine metabolism; spermidine metabolism'},\n",
       "  'seq': 'MSKGTTSQDAPFGTLLGYAPGGVAIYSSDYSSLDPQEYEDDAVFRSYIDDEYMGHKWQCVEFARRFLFLNYGVVFTDVGMAWEIFSLRFLREVVNDNILPLQAFPNGSPRAPVAGALLIWDKGGEFKDTGHVAIITQLHGNKVRIAEQNVIHSPLPQGQQWTRELEMVVENGCYTLKDTFDDTTILGWMIQTEDTEYSLPQPEIAGELLKISGARLENKGQFDGKWLDEKDPLQNAYVQANGQVINQDPYHYYTITESAEQELIKATNELHLMYLHATDKVLKDDNLLALFDIPKILWPRLRLSWQRRRHHMITGRMDFCMDERGLKVYEYNADSASCHTEAGLILERWAEQGYKGNGFNPAEGLINELAGAWKHSRARPFVHIMQDKDIEENYHAQFMEQALHQAGFETRILRGLDELGWDAAGQLIDGEGRLVNCVWKTWAWETAFDQIREVSDREFAAVPIRTGHPQNEVRLIDVLLRPEVLVFEPLWTVIPGNKAILPILWSLFPHHRYLLDTDFTVNDELVKTGYAVKPIAGRCGSNIDLVSHHEEVLDKTSGKFAEQKNIYQQLWCLPKVDGKYIQVCTFTVGGNYGGTCLRGDESLVIKKESDIEPLIVVKK',\n",
       "  'mw': 70532.0},\n",
       " 'b2994': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ACE0',\n",
       "  'gene': 'b2994',\n",
       "  'locus': ['b2994', 'JW2962'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('A + H2 = AH2', '1.12.99.6')],\n",
       "   'Function': 'This is one of three E.coli hydrogenases synthesized in response to different physiological conditions. HYD2 is involved in hydrogen uptake'},\n",
       "  'seq': 'MSQRITIDPVTRIEGHLRIDCEIENGVVSKAWASGTMWRGMEEIVKNRDPRDAWMIVQRICGVCTTTHALSSVRAAESALNIDVPVNAQYIRNIILAAHTTHDHIVHFYQLSALDWVDITSALQADPTKASEMLKGVSTWHLNSPEEFTKVQNKIKDLVASGQLGIFANGYWGHPAMKLPPEVNLIAVAHYLQALECQRDANRVVALLGGKTPHIQNLAVGGVANPINLDGLGVLNLERLMYIKSFIDKLSDFVEQVYKVDTAVIAAFYPEWLTRGKGAVNYLSVPEFPTDSKNGSFLFPGGYIENADLSSYRPITSHSDEYLIKGIQESAKHSWYKDEAPQAPWEGTTIPAYDGWSDDGKYSWVKSPTFYGKTVEVGPLANMLVKLAAGRESTQNKLNEIVAIYQKLTGNTLEVAQLHSTLGRIIGRTVHCCELQDILQNQYSALITNIGKGDHTTFVKPNIPATGEFKGVGFLEAPRGMLSHWMVIKDGIISNYQAVVPSTWNSGPRNFNDDVGPYEQSLVGTPVADPNKPLEVVRTIHSFDPCMACAVHVVDADGNEVVSVKVL',\n",
       "  'mw': 62491.0},\n",
       " 'b2997': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P69741',\n",
       "  'gene': 'b2997',\n",
       "  'locus': ['b2997', 'JW2965'],\n",
       "  'synonyms': 'yghV',\n",
       "  'properties': {'Catalytic Activity': [('A + H2 = AH2', '1.12.99.6')],\n",
       "   'Function': 'This is one of three E.coli hydrogenases synthesized in response to different physiological conditions. HYD2 is involved in hydrogen uptake'},\n",
       "  'seq': 'MTGDNTLIHSHGINRRDFMKLCAALAATMGLSSKAAAEMAESVTNPQRPPVIWIGAQECTGCTESLLRATHPTVENLVLETISLEYHEVLSAAFGHQVEENKHNALEKYKGQYVLVVDGSIPLKDNGIYCMVAGEPIVDHIRKAAEGAAAIIAIGSCSAWGGVAAAGVNPTGAVSLQEVLPGKTVINIPGCPPNPHNFLATVAHIITYGKPPKLDDKNRPTFAYGRLIHEHCERRPHFDAGRFAKEFGDEGHREGWCLYHLGCKGPETYGNCSTLQFCDVGGVWPVAIGHPCYGCNEEGIGFHKGIHQLANVENQTPRSQKPDVNAKEGGNVSAGAIGLLGGVVGLVAGVSVMAVRELGRQQKKDNADSRGE',\n",
       "  'mw': 39652.0},\n",
       " 'b3001': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'Q46851',\n",
       "  'gene': 'b3001',\n",
       "  'locus': ['b3001', 'JW2970'],\n",
       "  'synonyms': 'mgrA yghZ',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Catalyzes the stereospecific, NADPH-dependent reduction of L-glyceraldehyde 3-phosphate (L-GAP). The physiological role of gpr is the detoxification of L-GAP, which may be formed by non-enzymatic racemization of GAP. Also involved in the stress response as a methylglyoxal reductase which converts the toxic metabolite methylglyoxal to acetol in vitro and in vivo'},\n",
       "  'seq': 'MVWLANPERYGQMQYRYCGKSGLRLPALSLGLWHNFGHVNALESQRAILRKAFDLGITHFDLANNYGPPPGSAEENFGRLLREDFAAYRDELIISTKAGYDMWPGPYGSGGSRKYLLASLDQSLKRMGLEYVDIFYSHRVDENTPMEETASALAHAVQSGKALYVGISSYSPERTQKMVELLREWKIPLLIHQPSYNLLNRWVDKSGLLDTLQNNGVGCIAFTPLAQGLLTGKYLNGIPQDSRMHREGNKVRGLTPKMLTEANLNSLRLLNEMAQQRGQSMAQMALSWLLKDDRVTSVLIGASRAEQLEENVQALNNLTFSTKELAQIDQHIADGELNLWQASSDK',\n",
       "  'mw': 38832.0},\n",
       " 'b3005': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ABV2',\n",
       "  'gene': 'b3005',\n",
       "  'locus': ['b3005', 'JW2973'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Involved in the TonB-dependent energy-dependent transport of various receptor-bound substrates'},\n",
       "  'seq': 'MAMHLNENLDDNGEMHDINVTPFIDVMLVLLIIFMVAAPLATVDVKVNLPASTSTPQPRPEKPVYLSVKADNSMFIGNDPVTDETMITALNALTEGKKDTTIFFRADKTVDYETLMKVMDTLHQAGYLKIGLVGEETAKAK',\n",
       "  'mw': 15527.0},\n",
       " 'b3006': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ABU7',\n",
       "  'gene': 'b3006',\n",
       "  'locus': ['b3006', 'JW2974'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Involved in the TonB-dependent energy-dependent transport of various receptor-bound substrates. Protects ExbD from proteolytic degradation and functionally stabilizes TonB'},\n",
       "  'seq': 'MGNNLMQTDLSVWGMYQHADIVVKCVMIGLILASVVTWAIFFSKSVEFFNQKRRLKREQQLLAEARSLNQANDIAADFGSKSLSLHLLNEAQNELELSEGSDDNEGIKERTSFRLERRVAAVGRQMGRGNGYLATIGAISPFVGLFGTVWGIMNSFIGIAQTQTTNLAVVAPGIAEALLATAIGLVAAIPAVVIYNVFARQIGGFKAMLGDVAAQVLLLQSRDLDLEASAAAHPVRVAQKLRAG',\n",
       "  'mw': 26287.0},\n",
       " 'b3008': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P06721',\n",
       "  'gene': 'b3008',\n",
       "  'locus': ['b3008', 'JW2975'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('H2O + L,L-cystathionine = L-homocysteine + NH4(+) + pyruvate',\n",
       "     ''),\n",
       "    ('H2O + L-cysteine = H(+) + hydrogen sulfide + NH4(+) + pyruvate',\n",
       "     '4.4.1.28'),\n",
       "    ('an S-substituted L-cysteine + H2O = a thiol + NH4(+) + pyruvate',\n",
       "     '4.4.1.13')],\n",
       "   'Function': 'Primarily catalyzes the cleavage of cystathionine to homocysteine, pyruvate and ammonia during methionine biosynthesis (PubMed:7049234). Also exhibits cysteine desulfhydrase activity, producing sulfide from cysteine (PubMed:12883870). In addition, under certain growth conditions, exhibits significant alanine racemase coactivity (PubMed:21193606)',\n",
       "   'Regulation Activity': 'L-cysteine inhibits cystathionine beta-lyase activity competitively (PubMed:7049234). Inhibited by aminoethoxyvinylglycine (AVG) (PubMed:9376370)',\n",
       "   'Pathway': 'Amino-acid biosynthesis; L-methionine biosynthesis via de novo pathway; L-homocysteine from L-cystathionine: step 1/1'},\n",
       "  'seq': 'MADKKLDTQLVNAGRSKKYTLGAVNSVIQRASSLVFDSVEAKKHATRNRANGELFYGRRGTLTHFSLQQAMCELEGGAGCVLFPCGAAAVANSILAFIEQGDHVLMTNTAYEPSQDFCSKILSKLGVTTSWFDPLIGADIVKHLQPNTKIVFLESPGSITMEVHDVPAIVAAVRSVVPDAIIMIDNTWAAGVLFKALDFGIDVSIQAATKYLVGHSDAMIGTAVCNARCWEQLRENAYLMGQMVDADTAYITSRGLRTLGVRLRQHHESSLKVAEWLAEHPQVARVNHPALPGSKGHEFWKRDFTGSSGLFSFVLKKKLNNEELANYLDNFSLFSMAYSWGGYESLILANQPEHIAAIRPQGEIDFSGTLIRLHIGLEDVDDLIADLDAGFARIV',\n",
       "  'mw': 43212.0},\n",
       " 'b3012': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'Q46857',\n",
       "  'gene': 'b3012',\n",
       "  'locus': ['b3012', 'JW5499'],\n",
       "  'synonyms': 'yqhE',\n",
       "  'properties': {'Catalytic Activity': [('2-dehydro-D-gluconate + NADP(+) = 2,5-didehydro-D-gluconate + H(+) + NADPH',\n",
       "     '1.1.1.274')],\n",
       "   'Function': 'Catalyzes the reduction of 2,5-diketo-D-gluconic acid (25DKG) to 2-keto-L-gulonic acid (2KLG). It is also capable of stereoselective -keto ester reductions on ethyl acetoacetate and other 2-substituted derivatives'},\n",
       "  'seq': 'MANPTVIKLQDGNVMPQLGLGVWQASNEEVITAIQKALEVGYRSIDTAAAYKNEEGVGKALKNASVNREELFITTKLWNDDHKRPREALLDSLKKLQLDYIDLYLMHWPVPAIDHYVEAWKGMIELQKEGLIKSIGVCNFQIHHLQRLIDETGVTPVINQIELHPLMQQRQLHAWNATHKIQTESWSPLAQGGKGVFDQKVIRDLADKYGKTPAQIVIRWHLDSGLVVIPKSVTPSRIAENFDVWDFRLDKDELGEIAKLDQGKRLGPDPDQFGG',\n",
       "  'mw': 31110.0},\n",
       " 'b3018': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P26647',\n",
       "  'gene': 'b3018',\n",
       "  'locus': ['b3018', 'JW2986'],\n",
       "  'synonyms': 'parF',\n",
       "  'properties': {'Catalytic Activity': [('a 1-acyl-sn-glycero-3-phosphate + an acyl-CoA = a 1,2-diacyl-sn-glycero-3-phosphate + CoA',\n",
       "     '2.3.1.51'),\n",
       "    ('a 1-acyl-sn-glycero-3-phosphate + a fatty acyl-[ACP] = a 1,2-diacyl-sn-glycero-3-phosphate + holo-[ACP]',\n",
       "     '2.3.1.n4')],\n",
       "   'Function': 'Converts lysophosphatidic acid (LPA) into phosphatidic acid by incorporating an acyl moiety at the 2 position. This enzyme can utilize either acyl-CoA or acyl-ACP as the fatty acyl donor',\n",
       "   'Pathway': 'Phospholipid metabolism; CDP-diacylglycerol biosynthesis; CDP-diacylglycerol from sn-glycerol 3-phosphate: step 2/3'},\n",
       "  'seq': 'MLYIFRLIITVIYSILVCVFGSIYCLFSPRNPKHVATFGHMFGRLAPLFGLKVECRKPTDAESYGNAIYIANHQNNYDMVTASNIVQPPTVTVGKKSLLWIPFFGQLYWLTGNLLIDRNNRTKAHGTIAEVVNHFKKRRISIWMFPEGTRSRGRGLLPFKTGAFHAAIAAGVPIIPVCVSTTSNKINLNRLHNGLVIVEMLPPIDVSQYGKDQVRELAAHCRSIMEQKIAELDKEVAEREAAGKV',\n",
       "  'mw': 27453.0},\n",
       " 'b3028': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AEY5',\n",
       "  'gene': 'b3028',\n",
       "  'locus': ['b3028', 'JW2996'],\n",
       "  'synonyms': 'mda66',\n",
       "  'properties': {'Catalytic Activity': [('a quinone + H(+) + NADPH = a quinol + NADP(+)',\n",
       "     '1.6.5.10'),\n",
       "    ('H(+) + menadione + NADPH = menadiol + NADP(+)', '')],\n",
       "   'Function': 'NADPH-specific quinone reductase (PubMed:8611590). Is most active with quinone derivatives and ferricyanide as electron acceptors (PubMed:8611590). Can use menadione, 1,4-naphthoquinone and 1,4-benzoquinone (PubMed:8611590). May work in concert with YgiN to form a quinone redox cycle (PubMed:15613473)'},\n",
       "  'seq': 'MSNILIINGAKKFAHSNGQLNDTLTEVADGTLRDLGHDVRIVRADSDYDVKAEVQNFLWADVVIWQMPGWWMGAPWTVKKYIDDVFTEGHGTLYASDGRTRKDPSKKYGSGGLVQGKKYMLSLTWNAPMEAFTEKDQFFHGVGVDGVYLPFHKANQFLGMEPLPTFIANDVIKMPDVPRYTEEYRKHLVEIFG',\n",
       "  'mw': 21891.0},\n",
       " 'b3029': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ADU2',\n",
       "  'gene': 'b3029',\n",
       "  'locus': ['b3029', 'JW2997'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('menadiol + 2 O2 = 2 H(+) + menadione + 2 superoxide',\n",
       "     '')],\n",
       "   'Function': 'Can oxidize menadiol to menadione (PubMed:15613473). May work in concert with MdaB to form a quinone redox cycle (PubMed:15613473)',\n",
       "   'Regulation Activity': 'Does not appear to require metal ions or other cofactors for catalysis (PubMed:15613473). The reaction is insensitive to heptyl-4-hydroxyquinoline-N-oxide (HOQNO), a typical inhibitor of single electron reduction reactions (PubMed:15613473)'},\n",
       "  'seq': 'MLTVIAEIRTRPGQHHRQAVLDQFAKIVPTVLKEEGCHGYAPMVDCAAGVSFQSMAPDSIVMIEQWESIAHLEAHLQTPHMKAYSEAVKGDVLEMNIRILQPGI',\n",
       "  'mw': 11532.0},\n",
       " 'b3040': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A8H3',\n",
       "  'gene': 'b3040',\n",
       "  'locus': ['b3040', 'JW3008'],\n",
       "  'synonyms': 'ygiE',\n",
       "  'properties': {'Catalytic Activity': [('Zn(2+)(in) = Zn(2+)(out)', ''),\n",
       "    ('Cd(2+)(in) = Cd(2+)(out)', ''),\n",
       "    ('Co(2+)(in) = Co(2+)(out)', ''),\n",
       "    ('Fe(2+)(in) = Fe(2+)(out)', ''),\n",
       "    ('Mn(2+)(in) = Mn(2+)(out)', ''),\n",
       "    ('Cu(2+)(in) = Cu(2+)(out)', '')],\n",
       "   'Function': 'Mediates zinc uptake (PubMed:11790762, PubMed:15716430, PubMed:20225068, PubMed:34793140). Has broad metal specificity and can also mediate the uptake of cadmium, cobalt, iron, manganese and possibly copper (PubMed:11790762, PubMed:15716430, PubMed:20225068, PubMed:34793140). Has a slight preference for zinc (PubMed:20225068)',\n",
       "   'Regulation Activity': 'Uptake of metal ions is decreased in the presence of the ionophores FCCP or CCCP, but no clear contribution of the proton-motive force could be demonstrated (PubMed:20225068). Bicarbonate or other ions do not influence metal-uptake (PubMed:20225068)'},\n",
       "  'seq': 'MSVPLILTILAGAATFIGAFLGVLGQKPSNRLLAFSLGFAAGIMLLISLMEMLPAALAAEGMSPVLGYGMFIFGLLGYFGLDRMLPHAHPQDLMQKSVQPLPKSIKRTAILLTLGISLHNFPEGIATFVTASSNLELGFGIALAVALHNIPEGLAVAGPVYAATGSKRTAILWAGISGLAEILGGVLAWLILGSMISPVVMAAIMAAVAGIMVALSVDELMPLAKEIDPNNNPSYGVLCGMSVMGFSLVLLQTAGIG',\n",
       "  'mw': 26485.0},\n",
       " 'b3041': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A7J0',\n",
       "  'gene': 'b3041',\n",
       "  'locus': ['b3041', 'JW3009'],\n",
       "  'synonyms': 'htrP',\n",
       "  'properties': {'Catalytic Activity': [('D-ribulose 5-phosphate = (2S)-2-hydroxy-3-oxobutyl phosphate + formate + H(+)',\n",
       "     '4.1.99.12')],\n",
       "   'Function': 'Catalyzes the conversion of D-ribulose 5-phosphate to formate and 3,4-dihydroxy-2-butanone 4-phosphate',\n",
       "   'Pathway': 'Cofactor biosynthesis; riboflavin biosynthesis; 2-hydroxy-3-oxobutyl phosphate from D-ribulose 5-phosphate: step 1/1'},\n",
       "  'seq': 'MNQTLLSSFGTPFERVENALAALREGRGVMVLDDEDRENEGDMIFPAETMTVEQMALTIRHGSGIVCLCITEDRRKQLDLPMMVENNTSAYGTGFTVTIEAAEGVTTGVSAADRITTVRAAIADGAKPSDLNRPGHVFPLRAQAGGVLTRGGHTEATIDLMTLAGFKPAGVLCELTNDDGTMARAPECIEFANKHNMALVTIEDLVAYRQAHERKAS',\n",
       "  'mw': 23353.0},\n",
       " 'b3052': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P76658',\n",
       "  'gene': 'b3052',\n",
       "  'locus': ['b3052', 'JW3024'],\n",
       "  'synonyms': 'rfaE waaE yqiF',\n",
       "  'properties': {'Catalytic Activity': [('ATP + D-glycero-beta-D-manno-heptose 7-phosphate = ADP + D-glycero-beta-D-manno-heptose 1,7-bisphosphate + H(+)',\n",
       "     '2.7.1.167'),\n",
       "    ('ATP + D-glycero-beta-D-manno-heptose 1-phosphate + H(+) = ADP-D-glycero-beta-D-manno-heptose + diphosphate',\n",
       "     '2.7.7.70')],\n",
       "   'Function': 'Catalyzes the ADP transfer from ATP to D-glycero-beta-D-manno-heptose 1-phosphate, yielding ADP-D-glycero-beta-D-manno-heptose',\n",
       "   'Pathway': 'Bacterial outer membrane biogenesis; LPS core biosynthesis'},\n",
       "  'seq': 'MKVTLPEFERAGVMVVGDVMLDRYWYGPTSRISPEAPVPVVKVNTIEERPGGAANVAMNIASLGANARLVGLTGIDDAARALSKSLADVNVKCDFVSVPTHPTITKLRVLSRNQQLIRLDFEEGFEGVDPQPLHERINQALSSIGALVLSDYAKGALASVQQMIQLARKAGVPVLIDPKGTDFERYRGATLLTPNLSEFEAVVGKCKTEEEIVERGMKLIADYELSALLVTRSEQGMSLLQPGKAPLHMPTQAQEVYDVTGAGDTVIGVLAATLAAGNSLEEACFFANAAAGVVVGKLGTSTVSPIELENAVRGRADTGFGVMTEEELKLAVAAARKRGEKVVMTNGVFDILHAGHVSYLANARKLGDRLIVAVNSDASTKRLKGDSRPVNPLEQRMIVLGALEAVDWVVSFEEDTPQRLIAGILPDLLVKGGDYKPEEIAGSKEVWANGGEVLVLNFEDGCSTTNIIKKIQQDKKG',\n",
       "  'mw': 51051.0},\n",
       " 'b3057': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P60932',\n",
       "  'gene': 'b3057',\n",
       "  'locus': ['b3057', 'JW3029'],\n",
       "  'synonyms': 'bacA upk',\n",
       "  'properties': {'Catalytic Activity': [('di-trans,octa-cis-undecaprenyl diphosphate + H2O = di-trans,octa-cis-undecaprenyl phosphate + H(+) + phosphate',\n",
       "     '3.6.1.27')],\n",
       "   'Function': 'Catalyzes the dephosphorylation of undecaprenyl diphosphate (UPP). Confers resistance to bacitracin'},\n",
       "  'seq': 'MSDMHSLLIAAILGVVEGLTEFLPVSSTGHMIIVGHLLGFEGDTAKTFEVVIQLGSILAVVVMFWRRLFGLIGIHFGRPLQHEGESKGRLTLIHILLGMIPAVVLGLLFHDTIKSLFNPINVMYALVVGGLLLIAAECLKPKEPRAPGLDDMTYRQAFMIGCFQCLALWPGFSRSGATISGGMLMGVSRYAASEFSFLLAVPMMMGATALDLYKSWGFLTSGDIPMFAVGFITAFVVALIAIKTFLQLIKRISFIPFAIYRFIVAAAVYVVFF',\n",
       "  'mw': 29759.0},\n",
       " 'b3058': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AC16',\n",
       "  'gene': 'b3058',\n",
       "  'locus': ['b3058', 'JW3030'],\n",
       "  'synonyms': 'ygiG',\n",
       "  'properties': {'Catalytic Activity': [('7,8-dihydroneopterin = 6-hydroxymethyl-7,8-dihydropterin + glycolaldehyde',\n",
       "     '4.1.2.25'),\n",
       "    ('7,8-dihydroneopterin = 7,8-dihydromonapterin', '5.1.99.8')],\n",
       "   'Function': \"Catalyzes the conversion of 7,8-dihydroneopterin to 6-hydroxymethyl-7,8-dihydropterin. Can use L-threo-dihydroneopterin and D-erythro-dihydroneopterin as substrates for the formation of 6-hydroxymethyldihydropterin, but it can also catalyze the epimerization of carbon 2' of dihydroneopterin to dihydromonapterin at appreciable velocity\",\n",
       "   'Pathway': 'Cofactor biosynthesis; tetrahydrofolate biosynthesis; 2-amino-4-hydroxy-6-hydroxymethyl-7,8-dihydropteridine diphosphate from 7,8-dihydroneopterin triphosphate: step 3/4'},\n",
       "  'seq': 'MDIVFIEQLSVITTIGVYDWEQTIEQKLVFDIEMAWDNRKAAKSDDVADCLSYADIAETVVSHVEGARFALVERVAEEVAELLLARFNSPWVRIKLSKPGAVARAANVGVIIERGNNLKENN',\n",
       "  'mw': 13620.0},\n",
       " 'b3061': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P05847',\n",
       "  'gene': 'b3061',\n",
       "  'locus': ['b3061', 'JW3033'],\n",
       "  'synonyms': 'ygjA',\n",
       "  'properties': {'Catalytic Activity': [('(2R,3R)-tartrate = H2O + oxaloacetate',\n",
       "     '4.2.1.32')]},\n",
       "  'seq': 'MMSESNKQQAVNKLTEIVANFTAMISTRMPDDVVDKLKQLKDAETSSMGKIIYHTMFDNMQKAIDLNRPACQDTGEIMFFVKVGSRFPLLGELQSILKQAVEEATVKAPLRHNAVEIFDEVNTGKNTGSGVPWVTWDIIPDNDDAEIEVYMAGGGCTLPGRSKVLMPSEGYEGVVKFVFENISTLAVNACPPVLVGVGIATSVETAAVLSRKAILRPIGSRHPNPKAAELELRLEEGLNRLGIGPQGLTGNSSVMGVHIESAARHPSTIGVAVSTGCWAHRRGTLLVHADLTFENLSHTRSAL',\n",
       "  'mw': 32734.0},\n",
       " 'b3062': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AC35',\n",
       "  'gene': 'b3062',\n",
       "  'locus': ['b3062', 'JW3034'],\n",
       "  'synonyms': 'ygjB',\n",
       "  'properties': {'Catalytic Activity': [('(2R,3R)-tartrate = H2O + oxaloacetate',\n",
       "     '4.2.1.32')]},\n",
       "  'seq': 'MKKILTTPIKAEDLQDIRVGDVIYLTGTLVTCRDVCHRRLIELKRPIPYDLNGKAIFHAGPIVRKNGDKWEMVSVGPTTSMRMESFEREFIEQTGVKLVVGKGGMGPLTEEGCQKFKALHVIFPAGCAVLAATQVEEIEEVHWTELGMPESLWVCRVKEFGPLIVSIDTHGNNLIAENKKLFAERRDPIVEEICEHVHYIK',\n",
       "  'mw': 22679.0},\n",
       " 'b3063': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P39414',\n",
       "  'gene': 'b3063',\n",
       "  'locus': ['b3063', 'JW3035'],\n",
       "  'synonyms': 'ygjC ygjE',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Catalyzes the uptake of tartrate in exchange for intracellular succinate. Essential for anaerobic L-tartrate fermentation'},\n",
       "  'seq': 'MKPSTEWWRYLAPLAVIAIIALLPVPAGLENHTWLYFAVFTGVIVGLILEPVPGAVVAMVGISIIAILSPWLLFSPEQLAQPGFKFTAKSLSWAVSGFSNSVIWLIFAAFMFGTGYEKTGLGRRIALILVKKMGHRTLFLGYAVMFSELILAPVTPSNSARGAGIIYPIIRNLPPLYQSQPNDSSSRSIGSYIMWMGIVADCVTSAIFLTAMAPNLLLIGLMKSASHATLSWGDWFLGMLPLSILLVLLVPWLAYVLYPPVLKSGDQVPRWAETELQAMGPLCSREKRMLGLMVGALVLWIFGGDYIDAAMVGYSVVALMLLLRIISWDDIVSNKAAWNVFFWLASLITLATGLNNTGFISWFGKLLAGSLSGYSPTMVMVALIVVFYLLRYFFASATAYTSALAPMMIAAALAMPEIPLPVFCLMVGAAIGLGSILTPYATGPSPIYYGSGYLPTADYWRLGAIFGLIFLVLLVITGLLWMPVVLL',\n",
       "  'mw': 52906.0},\n",
       " 'b3073': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P42588',\n",
       "  'gene': 'b3073',\n",
       "  'locus': ['b3073', 'JW5510'],\n",
       "  'synonyms': 'ygjG',\n",
       "  'properties': {'Catalytic Activity': [('2-oxoglutarate + an alkane-alpha,omega-diamine = an omega-aminoaldehyde + L-glutamate',\n",
       "     '2.6.1.29'),\n",
       "    ('2-oxoglutarate + putrescine = 1-pyrroline + H2O + L-glutamate',\n",
       "     '2.6.1.82'),\n",
       "    ('2-oxoglutarate + cadaverine = 5-aminopentanal + L-glutamate', '')],\n",
       "   'Function': 'Catalyzes the aminotransferase reaction from putrescine to 2-oxoglutarate, leading to glutamate and 4-aminobutanal, which spontaneously cyclizes to form 1-pyrroline (PubMed:12617754, PubMed:3510672). This is the first step in one of two pathways for putrescine degradation, where putrescine is converted into 4-aminobutanoate (gamma-aminobutyrate or GABA) via 4-aminobutanal, which allows E.coli to grow on putrescine as the sole nitrogen source (PubMed:3510672, PubMed:22636776). Also functions as a cadaverine transaminase in a a L-lysine degradation pathway to succinate that proceeds via cadaverine, glutarate and L-2-hydroxyglutarate (PubMed:12617754, PubMed:30498244). Is also able to transaminate spermidine, in lower extent, but not ornithine. Alpha-ketobutyrate and pyruvate can also act as amino acceptors, although much less efficiently (PubMed:12617754)',\n",
       "   'Pathway': 'Amino-acid degradation'},\n",
       "  'seq': 'MNRLPSSASALACSAHALNLIEKRTLDHEEMKALNREVIEYFKEHVNPGFLEYRKSVTAGGDYGAVEWQAGSLNTLVDTQGQEFIDCLGGFGIFNVGHRNPVVVSAVQNQLAKQPLHSQELLDPLRAMLAKTLAALTPGKLKYSFFCNSGTESVEAALKLAKAYQSPRGKFTFIATSGAFHGKSLGALSATAKSTFRKPFMPLLPGFRHVPFGNIEAMRTALNECKKTGDDVAAVILEPIQGEGGVILPPPGYLTAVRKLCDEFGALMILDEVQTGMGRTGKMFACEHENVQPDILCLAKALGGGVMPIGATIATEEVFSVLFDNPFLHTTTFGGNPLACAAALATINVLLEQNLPAQAEQKGDMLLDGFRQLAREYPDLVQEARGKGMLMAIEFVDNEIGYNFASEMFRQRVLVAGTLNNAKTIRIEPPLTLTIEQCELVIKAARKALAAMRVSVEEA',\n",
       "  'mw': 49661.0},\n",
       " 'b3089': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AGE4',\n",
       "  'gene': 'b3089',\n",
       "  'locus': ['b3089', 'JW3060'],\n",
       "  'synonyms': 'ygjU',\n",
       "  'properties': {'Catalytic Activity': [('L-serine(in) + Na(+)(in) = L-serine(out) + Na(+)(out)',\n",
       "     ''),\n",
       "    ('L-threonine(in) + Na(+)(in) = L-threonine(out) + Na(+)(out)', '')],\n",
       "   'Function': 'Involved in the import of serine and threonine into the cell, with the concomitant import of sodium (symport system)',\n",
       "   'Regulation Activity': 'Activated by valinomycin and inhibited by monensin'},\n",
       "  'seq': 'MTTQRSPGLFRRLAHGSLVKQILVGLVLGILLAWISKPAAEAVGLLGTLFVGALKAVAPILVLMLVMASIANHQHGQKTNIRPILFLYLLGTFSAALAAVVFSFAFPSTLHLSSSAGDISPPSGIVEVMRGLVMSMVSNPIDALLKGNYIGILVWAIGLGFALRHGNETTKNLVNDMSNAVTFMVKLVIRFAPIGIFGLVSSTLATTGFSTLWGYAQLLVVLVGCMLLVALVVNPLLVWWKIRRNPFPLVLLCLRESGVYAFFTRSSAANIPVNMALCEKLNLDRDTYSVSIPLGATINMAGAAITITVLTLAAVNTLGIPVDLPTALLLSVVASLCACGASGVAGGSLLLIPLACNMFGISNDIAMQVVAVGFIIGVLQDSCETALNSSTDVLFTAAACQAEDDRLANSALRN',\n",
       "  'mw': 43478.0},\n",
       " 'b3091': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P42604',\n",
       "  'gene': 'b3091',\n",
       "  'locus': ['b3091', 'JW3062'],\n",
       "  'synonyms': 'ygjW',\n",
       "  'properties': {'Catalytic Activity': [('D-altronate = 2-dehydro-3-deoxy-D-gluconate + H2O',\n",
       "     '4.2.1.7')],\n",
       "   'Function': 'Catalyzes the dehydration of D-altronate',\n",
       "   'Regulation Activity': 'Is inhibited by high concentrations of Fe(2+) (> 2 mM), and by EDTA or other iron chelators in vitro',\n",
       "   'Pathway': 'Carbohydrate metabolism; pentose and glucuronate interconversion'},\n",
       "  'seq': 'MQYIKIHALDNVAVALADLAEGTEVSVDNQTVTLRQDVARGHKFALTDIAKGANVIKYGLPIGYALADIAAGVHVHAHNTRTNLSDLDQYRYQPDFQDLPAQAADREVQIYRRANGDVGVRNELWILPTVGCVNGIARQIQNRFLKETNNAEGTDGVFLFSHTYGCSQLGDDHINTRTMLQNMVRHPNAGAVLVIGLGCENNQVAAFRETLGDIDPERVHFMICQQQDDEIEAGIEHLHQLYNVMRNDKREPGKLSELKFGLECGGSDGLSGITANPMLGRFSDYVIANGGTTVLTEVPEMFGAEQLLMDHCRDEATFEKLVTMVNDFKQYFIAHDQPIYENPSPGNKAGGITTLEDKSLGCTQKAGSSVVVDVLRYGERLKTPGLNLLSAPGNDAVATSALAGAGCHMVLFSTGRGTPYGGFVPTVKIATNSELAAKKKHWIDFDAGQLIHGKAMPQLLEEFIDTIVEFANGKQTCNERNDFRELAIFKSGVTL',\n",
       "  'mw': 54093.0},\n",
       " 'b3092': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A8G3',\n",
       "  'gene': 'b3092',\n",
       "  'locus': ['b3092', 'JW3063'],\n",
       "  'synonyms': 'ygjX ygjY ygjZ',\n",
       "  'properties': {'Catalytic Activity': [('D-glucuronate = D-fructuronate',\n",
       "     '5.3.1.12'),\n",
       "    ('aldehydo-D-galacturonate = keto-D-tagaturonate', '5.3.1.12')],\n",
       "   'Pathway': 'Carbohydrate metabolism; pentose and glucuronate interconversion'},\n",
       "  'seq': 'MTPFMTEDFLLDTEFARRLYHDYAKDQPIFDYHCHLPPQQIAEDYRFKNLYDIWLKGDHYKWRAMRTNGVAERLCTGDASDREKFDAWAATVPHTIGNPLYHWTHLELRRPFGITGKLLSPSTADEIWNECNELLAQDNFSARGIMQQMNVKMVGTTDDPIDSLEHHAEIAKDGSFTIKVLPSWRPDKAFNIEQATFNDYMAKLGEVSDTDIRRFADLQTALTKRLDHFAAHGCKVSDHALDVVMFAEANEAELDSILARRLAGETLSEHEVAQFKTAVLVFLGAEYARRGWVQQYHIGALRNNNLRQFKLLGPDVGFDSINDRPMAEELSKLLSKQNEENLLPKTILYCLNPRDNEVLGTMIGNFQGEGMPGKMQFGSGWWFNDQKDGMERQMTQLAQLGLLSRFVGMLTDSRSFLSYTRHEYFRRILCQMIGRWVEAGEAPADINLLGEMVKNICFNNARDYFAIELN',\n",
       "  'mw': 53987.0},\n",
       " 'b3093': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AA78',\n",
       "  'gene': 'b3093',\n",
       "  'locus': ['b3093', 'JW3064'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('aldehydo-D-glucuronate(in) + H(+)(in) = aldehydo-D-glucuronate(out) + H(+)(out)',\n",
       "     ''),\n",
       "    ('aldehydo-D-galacturonate(out) + H(+)(out) = aldehydo-D-galacturonate(in) + H(+)(in)',\n",
       "     ''),\n",
       "    ('D-glucose(out) + H(+)(out) = D-glucose(in) + H(+)(in)', '')],\n",
       "   'Function': 'Transport of aldohexuronates such as D-glucuronate and D-galacturonate (PubMed:783117, PubMed:6343797). Under anaerobic conditions, can play a critical role as a D-glucose transporter in the absence of sugar-PTS system (PubMed:32038601)'},\n",
       "  'seq': 'MRKIKGLRWYMIALVTLGTVLGYLTRNTVAAAAPTLMEELNISTQQYSYIIAAYSAAYTVMQPVAGYVLDVLGTKIGYAMFAVLWAVFCGATALAGSWGGLAVARGAVGAAEAAMIPAGLKASSEWFPAKERSIAVGYFNVGSSIGAMIAPPLVVWAIVMHSWQMAFIISGALSFIWAMAWLIFYKHPRDQKHLTDEERDYIINGQEAQHQVSTAKKMSVGQILRNRQFWGIALPRFLAEPAWGTFNAWIPLFMFKVYGFNLKEIAMFAWMPMLFADLGCILGGYLPPLFQRWFGVNLIVSRKMVVTLGAVLMIGPGMIGLFTNPYVAIMLLCIGGFAHQALSGALITLSSDVFGRNEVATANGLTGMSAWLASTLFALVVGALADTIGFSPLFAVLAVFDLLGALVIWTVLQNKPAIEVAQETHNDPAPQH',\n",
       "  'mw': 46892.0},\n",
       " 'b3114': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P42632',\n",
       "  'gene': 'b3114',\n",
       "  'locus': ['b3114', 'JW5522'],\n",
       "  'synonyms': 'yhaS',\n",
       "  'properties': {'Catalytic Activity': [('2-oxobutanoate + CoA = formate + propanoyl-CoA',\n",
       "     ''),\n",
       "    ('acetyl-CoA + formate = CoA + pyruvate', '2.3.1.54')],\n",
       "   'Function': 'Catalyzes the cleavage of 2-ketobutyrate to propionyl-CoA and formate. It can also use pyruvate as substrate',\n",
       "   'Regulation Activity': 'Dependent on PFL-activase',\n",
       "   'Pathway': 'Amino-acid degradation; L-threonine degradation via propanoate pathway; propanoate from L-threonine: step 2/4'},\n",
       "  'seq': 'MKVDIDTSDKLYADAWLGFKGTDWKNEINVRDFIQHNYTPYEGDESFLAEATPATTELWEKVMEGIRIENATHAPVDFDTNIATTITAHDAGYINQPLEKIVGLQTDAPLKRALHPFGGINMIKSSFHAYGREMDSEFEYLFTDLRKTHNQGVFDVYSPDMLRCRKSGVLTGLPDGYGRGRIIGDYRRVALYGISYLVRERELQFADLQSRLEKGEDLEATIRLREELAEHRHALLQIQEMAAKYGFDISRPAQNAQEAVQWLYFAYLAAVKSQNGGAMSLGRTASFLDIYIERDFKAGVLNEQQAQELIDHFIMKIRMVRFLRTPEFDSLFSGDPIWATEVIGGMGLDGRTLVTKNSFRYLHTLHTMGPAPEPNLTILWSEELPIAFKKYAAQVSIVTSSLQYENDDLMRTDFNSDDYAIACCVSPMVIGKQMQFFGARANLAKTLLYAINGGVDEKLKIQVGPKTAPLMDDVLDYDKVMDSLDHFMDWLAVQYISALNIIHYMHDKYSYEASLMALHDRDVYRTMACGIAGLSVATDSLSAIKYARVKPIRDENGLAVDFEIDGEYPQYGNNDERVDSIACDLVERFMKKIKALPTYRNAVPTQSILTITSNVVYGQKTGNTPDGRRAGTPFAPGANPMHGRDRKGAVASLTSVAKLPFTYAKDGISYTFSIVPAALGKEDPVRKTNLVGLLDGYFHHEADVEGGQHLNVNVMNREMLLDAIEHPEKYPNLTIRVSGYAVRFNALTREQQQDVISRTFTQAL',\n",
       "  'mw': 85936.0},\n",
       " 'b3115': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P11868',\n",
       "  'gene': 'b3115',\n",
       "  'locus': ['b3115', 'JW5806'],\n",
       "  'synonyms': 'yhaA',\n",
       "  'properties': {'Catalytic Activity': [('ATP + propanoate = ADP + propanoyl phosphate',\n",
       "     '2.7.2.15')],\n",
       "   'Function': 'Catalyzes the conversion of propionyl phosphate and ADP to propionate and ATP. It can also use acetyl phosphate as phosphate group acceptor',\n",
       "   'Pathway': 'Amino-acid degradation; L-threonine degradation via propanoate pathway; propanoate from L-threonine: step 4/4'},\n",
       "  'seq': 'MNEFPVVLVINCGSSSIKFSVLDASDCEVLMSGIADGINSENAFLSVNGGEPAPLAHHSYEGALKAIAFELEKRNLNDSVALIGHRIAHGGSIFTESAIITDEVIDNIRRVSPLAPLHNYANLSGIESAQQLFPGVTQVAVFDTSFHQTMAPEAYLYGLPWKYYEELGVRRYGFHGTSHRYVSQRAHSLLNLAEDDSGLVVAHLGNGASICAVRNGQSVDTSMGMTPLEGLMMGTRSGDVDFGAMSWVASQTNQSLGDLERVVNKESGLLGISGLSSDLRVLEKAWHEGHERAQLAIKTFVHRIARHIAGHAASLRRLDGIIFTGGIGENSSLIRRLVMEHLAVLGLEIDTEMNNRSNSCGERIVSSENARVICAVIPTNEEKMIALDAIHLGKVNAPAEFA',\n",
       "  'mw': 43384.0},\n",
       " 'b3116': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AAD8',\n",
       "  'gene': 'b3116',\n",
       "  'locus': ['b3116', 'JW3087'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('H(+)(in) + L-threonine(in) = H(+)(out) + L-threonine(out)',\n",
       "     ''),\n",
       "    ('H(+)(in) + L-serine(in) = H(+)(out) + L-serine(out)', '')],\n",
       "   'Function': 'Involved in the import of threonine and serine into the cell, with the concomitant import of a proton (symport system)',\n",
       "   'Regulation Activity': 'Inhibited by the proton ionophore carbonyl cyanide m-chlorophenylhydrazone (CCCP) (PubMed:2115866, PubMed:9498571). Partially inhibited by the respiratory chain inhibitor KCN (PubMed:2115866). Activity is Na(+) independent (PubMed:2115866, PubMed:9498571)'},\n",
       "  'seq': 'MSTSDSIVSSQTKQSSWRKSDTTWTLGLFGTAIGAGVLFFPIRAGFGGLIPILLMLVLAYPIAFYCHRALARLCLSGSNPSGNITETVEEHFGKTGGVVITFLYFFAICPLLWIYGVTITNTFMTFWENQLGFAPLNRGFVALFLLLLMAFVIWFGKDLMVKVMSYLVWPFIASLVLISLSLIPYWNSAVIDQVDLGSLSLTGHDGILITVWLGISIMVFSFNFSPIVSSFVVSKREEYEKDFGRDFTERKCSQIISRASMLMVAVVMFFAFSCLFTLSPANMAEAKAQNIPVLSYLANHFASMTGTKTTFAITLEYAASIIALVAIFKSFFGHYLGTLEGLNGLVLKFGYKGDKTKVSLGKLNTISMIFIMGSTWVVAYANPNILDLIEAMGAPIIASLLCLLPMYAIRKAPSLAKYRGRLDNVFVTVIGLLTILNIVYKLF',\n",
       "  'mw': 48879.0},\n",
       " 'b3117': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AGF6',\n",
       "  'gene': 'b3117',\n",
       "  'locus': ['b3117', 'JW3088'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('L-threonine = 2-oxobutanoate + NH4(+)',\n",
       "     '4.3.1.19'),\n",
       "    ('L-serine = NH4(+) + pyruvate', '4.3.1.17')],\n",
       "   'Function': 'Catalyzes the anaerobic formation of alpha-ketobutyrate and ammonia from threonine in a two-step reaction. The first step involved a dehydration of threonine and a production of enamine intermediates (aminocrotonate), which tautomerizes to its imine form (iminobutyrate). Both intermediates are unstable and short-lived. The second step is the nonenzymatic hydrolysis of the enamine/imine intermediates to form 2-ketobutyrate and free ammonia. In the low water environment of the cell, the second step is accelerated by RidA. TdcB also dehydrates serine to yield pyruvate via analogous enamine/imine intermediates',\n",
       "   'Regulation Activity': 'Each protein molecule can bind up to four molecules of AMP, which act as an allosteric activator to the enzyme. The enzyme is also inhibited by alpha-keto acids and other catabolites',\n",
       "   'Pathway': 'Amino-acid degradation; L-threonine degradation via propanoate pathway; propanoate from L-threonine: step 1/4'},\n",
       "  'seq': 'MHITYDLPVAIDDIIEAKQRLAGRIYKTGMPRSNYFSERCKGEIFLKFENMQRTGSFKIRGAFNKLSSLTDAEKRKGVVACSAGNHAQGVSLSCAMLGIDGKVVMPKGAPKSKVAATCDYSAEVVLHGDNFNDTIAKVSEIVEMEGRIFIPPYDDPKVIAGQGTIGLEIMEDLYDVDNVIVPIGGGGLIAGIAVAIKSINPTIRVIGVQSENVHGMAASFHSGEITTHRTTGTLADGCDVSRPGNLTYEIVRELVDDIVLVSEDEIRNSMIALIQRNKVVTEGAGALACAALLSGKLDQYIQNRKTVSIISGGNIDLSRVSQITGFVDA',\n",
       "  'mw': 35232.0},\n",
       " 'b3124': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P23524',\n",
       "  'gene': 'b3124',\n",
       "  'locus': ['b3124', 'JW3093'],\n",
       "  'synonyms': 'yhaD',\n",
       "  'properties': {'Catalytic Activity': [('(R)-glycerate + ATP = (2R)-2-phosphoglycerate + ADP + H(+)',\n",
       "     '2.7.1.165')],\n",
       "   'Function': 'Catalyzes the transfer of the phosphate group from adenosine triphosphate (ATP) to (R)-glycerate to form (2R)-2-phosphoglycerate, an enzymatic step in (L)-glucarate/galactarate catabolic pathway'},\n",
       "  'seq': 'MKIVIAPDSYKESLSASEVAQAIEKGFREIFPDAQYVSVPVADGGEGTVEAMIAATQGAERHAWVTGPLGEKVNASWGISGDGKTAFIEMAAASGLELVPAEKRDPLVTTSRGTGELILQALESGATNIIIGIGGSATNDGGAGMVQALGAKLCDANGNEIGFGGGSLNTLNDIDISGLDPRLKDCVIRVACDVTNPLVGDNGASRIFGPQKGASEAMIVELDNNLSHYAEVIKKALHVDVKDVPGAGAAGGMGAALMAFLGAELKSGIEIVTTALNLEEHIHDCTLVITGEGRIDSQSIHGKVPIGVANVAKKYHKPVIGIAGSLTDDVGVVHQHGIDAVFSVLTSIGTLDEAFRGAYDNICRASRNIAATLAIGMRNAG',\n",
       "  'mw': 39104.0},\n",
       " 'b3125': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ABQ2',\n",
       "  'gene': 'b3125',\n",
       "  'locus': ['b3125', 'JW5526'],\n",
       "  'synonyms': 'yhaE',\n",
       "  'properties': {'Catalytic Activity': [('(R)-glycerate + NADP(+) = 2-hydroxy-3-oxopropanoate + H(+) + NADPH',\n",
       "     '1.1.1.60'),\n",
       "    ('(R)-glycerate + NAD(+) = 2-hydroxy-3-oxopropanoate + H(+) + NADH',\n",
       "     '1.1.1.60')],\n",
       "   'Function': 'Catalyzes the reduction of tatronate semialdehyde to D-glycerate',\n",
       "   'Pathway': 'Carbohydrate acid metabolism; galactarate degradation; D-glycerate from galactarate: step 3/3'},\n",
       "  'seq': 'MKVGFIGLGIMGKPMSKNLLKAGYSLVVADRNPEAIADVIAAGAETASTAKAIAEQCDVIITMLPNSPHVKEVALGENGIIEGAKPGTVLIDMSSIAPLASREISEALKAKGIDMLDAPVSGGEPKAIDGTLSVMVGGDKAIFDKYYDLMKAMAGSVVHTGEIGAGNVTKLANQVIVALNIAAMSEALTLATKAGVNPDLVYQAIRGGLAGSTVLDAKAPMVMDRNFKPGFRIDLHIKDLANALDTSHGVGAQLPLTAAVMEMMQALRADGLGTADHSALACYYEKLAKVEVTR',\n",
       "  'mw': 30427.0},\n",
       " 'b3126': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P23522',\n",
       "  'gene': 'b3126',\n",
       "  'locus': ['b3126', 'JW3095'],\n",
       "  'synonyms': 'yhaF',\n",
       "  'properties': {'Catalytic Activity': [('5-dehydro-4-deoxy-D-glucarate = 2-hydroxy-3-oxopropanoate + pyruvate',\n",
       "     ''),\n",
       "    ('2-dehydro-3-deoxy-D-glucarate = 2-hydroxy-3-oxopropanoate + pyruvate',\n",
       "     '4.1.2.20')],\n",
       "   'Function': 'Catalyzes the reversible retro-aldol cleavage of both 5-keto-4-deoxy-D-glucarate and 2-keto-3-deoxy-D-glucarate to pyruvate and tartronic semialdehyde',\n",
       "   'Regulation Activity': 'Inhibited by acetate, chloride and bromide ions, nitrate, fluoride, cyanide, EDTA and pyrophosphate',\n",
       "   'Pathway': 'Carbohydrate acid metabolism; galactarate degradation; D-glycerate from galactarate: step 2/3'},\n",
       "  'seq': 'MNNDVFPNKFKAALAAKQVQIGCWSALSNPISTEVLGLAGFDWLVLDGEHAPNDISTFIPQLMALKGSASAPVVRVPTNEPVIIKRLLDIGFYNFLIPFVETKEEAELAVASTRYPPEGIRGVSVSHRANMFGTVADYFAQSNKNITILVQIESQQGVDNVDAIAATEGVDGIFVGPSDLAAALGHLGNASHPDVQKAIQHIFNRASAHGKPSGILAPVEADARRYLEWGATFVAVGSDLGVFRSATQKLADTFKK',\n",
       "  'mw': 27384.0},\n",
       " 'b3127': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AA80',\n",
       "  'gene': 'b3127',\n",
       "  'locus': ['b3127', 'JW3096'],\n",
       "  'synonyms': 'yhaU',\n",
       "  'properties': {'Catalytic Activity': [('galactarate(in) + H(+)(in) = galactarate(out) + H(+)(out)',\n",
       "     ''),\n",
       "    ('D-glucarate(in) + H(+)(in) = D-glucarate(out) + H(+)(out)', ''),\n",
       "    ('(R)-glycerate(in) + H(+)(in) = (R)-glycerate(out) + H(+)(out)', '')],\n",
       "   'Function': 'Probably involved in the uptake of galactarate and/or D-glucarate (Probable). May also transport D-glycerate (Probable)'},\n",
       "  'seq': 'MILDTVDEKKKGVHTRYLILLIIFIVTAVNYADRATLSIAGTEVAKELQLSAVSMGYIFSAFGWAYLLMQIPGGWLLDKFGSKKVYTYSLFFWSLFTFLQGFVDMFPLAWAGISMFFMRFMLGFSEAPSFPANARIVAAWFPTKERGTASAIFNSAQYFSLALFSPLLGWLTFAWGWEHVFTVMGVIGFVLTALWIKLIHNPTDHPRMSAEELKFISENGAVVDMDHKKPGSAAASGPKLHYIKQLLSNRMMLGVFFGQYFINTITWFFLTWFPIYLVQEKGMSILKVGLVASIPALCGFAGGVLGGVFSDYLIKRGLSLTLARKLPIVLGMLLASTIILCNYTNNTTLVVMLMALAFFGKGFGALGWPVISDTAPKEIVGLCGGVFNVFGNVASIVTPLVIGYLVSELHSFNAALVFVGCSALMAMVCYLFVVGDIKRMELQK',\n",
       "  'mw': 49009.0},\n",
       " 'b3128': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P39829',\n",
       "  'gene': 'b3128',\n",
       "  'locus': ['b3128', 'JW3097'],\n",
       "  'synonyms': 'yhaG',\n",
       "  'properties': {'Catalytic Activity': [('galactarate = 5-dehydro-4-deoxy-D-glucarate + H2O',\n",
       "     '4.2.1.42')],\n",
       "   'Function': 'Catalyzes the dehydration of galactarate to form 5-dehydro-4-deoxy-D-glucarate (5-KDG)',\n",
       "   'Pathway': 'Carbohydrate acid metabolism; galactarate degradation; D-glycerate from galactarate: step 1/3'},\n",
       "  'seq': 'MANIEIRQETPTAFYIKVHDTDNVAIIVNDNGLKAGTRFPDGLELIEHIPQGHKVALLDIPANGEIIRYGEVIGYAVRAIPRGSWIDESMVVLPEAPPLHTLPLATKVPEPLPPLEGYTFEGYRNADGSVGTKNLLGITTSVHCVAGVVDYVVKIIERDLLPKYPNVDGVVGLNHLYGCGVAINAPAAVVPIRTIHNISLNPNFGGEVMVIGLGCEKLQPERLLTGTDDVQAIPVESASIVSLQDEKHVGFQSMVEDILQIAERHLQKLNQRQRETCPASELVVGMQCGGSDAFSGVTANPAVGYASDLLVRCGATVMFSEVTEVRDAIHLLTPRAVNEEVGKRLLEEMEWYDNYLNMGKTDRSANPSPGNKKGGLANVVEKALGSIAKSGKSAIVEVLSPGQRPTKRGLIYAATPASDFVCGTQQVASGITVQVFTTGRGTPYGLMAVPVIKMATRTELANRWFDLMDINAGTIATGEETIEEVGWKLFHFILDVASGKKKTFSDQWGLHNQLAVFNPAPVT',\n",
       "  'mw': 56402.0},\n",
       " 'b3132': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0C8K0',\n",
       "  'gene': 'b3132',\n",
       "  'locus': ['b3132', 'JW3101'],\n",
       "  'synonyms': 'agaZ yhaX',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Component of the tagatose-1,6-bisphosphate aldolase KbaYZ that is required for full activity and stability of the Y subunit. Could have a chaperone-like function for the proper and stable folding of KbaY. When expressed alone, KbaZ does not show any aldolase activity',\n",
       "   'Pathway': 'Carbohydrate metabolism; D-tagatose 6-phosphate degradation; D-glyceraldehyde 3-phosphate and glycerone phosphate from D-tagatose 6-phosphate: step 2/2'},\n",
       "  'seq': 'MKHLTEMVRQHKAGKTNGIYAVCSAHPLVLEAAIRYASANQTPLLIEATSNQVDQFGGYTGMTPADFRGFVCQLADSLNFPQDALILGGDHLGPNRWQNLPAAQAMANADDLIKSYVAAGFKKIHLDCSMSCQDDPIPLTDDIVAERAARLAKVAEETCLEHFGEADLEYVIGTEVPVPGGAHETLSELAVTTPDAARATLEAHRHAFEKQGLNAIWPRIIALVVQPGVEFDHTNVIDYQPAKASALSQMVENYETLIFEAHSTDYQTPQSLRQLVIDHFAILKVGPALTFALREALFSLAAIEEELVPAKACSGLRQVLEDVMLDRPEYWQSHYHGDGNARRLARGYSYSDRVRYYWPDSQIDDAFAHLVRNLADSPIPLPLISQYLPLQYVKVRSGELQPTPRELIINHIQDILAQYHTACEGQ',\n",
       "  'mw': 47192.0},\n",
       " 'b3137': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AB74',\n",
       "  'gene': 'b3137',\n",
       "  'locus': ['b3137', 'JW3106'],\n",
       "  'synonyms': 'agaY kba yraC',\n",
       "  'properties': {'Catalytic Activity': [('D-tagatofuranose 1,6-bisphosphate = D-glyceraldehyde 3-phosphate + dihydroxyacetone phosphate',\n",
       "     '4.1.2.40')],\n",
       "   'Function': 'Catalytic subunit of the tagatose-1,6-bisphosphate aldolase KbaYZ, which catalyzes the reversible aldol condensation of dihydroxyacetone phosphate (DHAP or glycerone-phosphate) with glyceraldehyde 3-phosphate (G3P) to produce tagatose 1,6-bisphosphate (TBP). Requires KbaZ subunit for full activity and stability',\n",
       "   'Pathway': 'Carbohydrate metabolism; D-tagatose 6-phosphate degradation; D-glyceraldehyde 3-phosphate and glycerone phosphate from D-tagatose 6-phosphate: step 2/2'},\n",
       "  'seq': 'MSIISTKYLLQDAQANGYAVPAFNIHNAETIQAILEVCSEMRSPVILAGTPGTFKHIALEEIYALCSAYSTTYNMPLALHLDHHESLDDIRRKVHAGVRSAMIDGSHFPFAENVKLVKSVVDFCHSQDCSVEAELGRLGGVEDDMSVDAESAFLTDPQEAKRFVELTGVDSLAVAIGTAHGLYSKTPKIDFQRLAEIREVVDVPLVLHGASDVPDEFVRRTIELGVTKVNVATELKIAFAGAVKAWFAENPQGNDPRYYMRVGMDAMKEVVRNKINVCGSANRISA',\n",
       "  'mw': 31294.0},\n",
       " 'b3161': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AAD2',\n",
       "  'gene': 'b3161',\n",
       "  'locus': ['b3161', 'JW3130'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('H(+)(in) + L-tryptophan(in) = H(+)(out) + L-tryptophan(out)',\n",
       "     '')],\n",
       "   'Function': 'Involved in the transport of tryptophan into the cell'},\n",
       "  'seq': 'MATLTTTQTSPSLLGGVVIIGGTIIGAGMFSLPVVMSGAWFFWSMAALIFTWFCMLHSGLMILEANLNYRIGSSFDTITKDLLGKGWNVVNGISIAFVLYILTYAYISASGSILHHTFAEMSLNVPARAAGFGFALLVAFVVWLSTKAVSRMTAIVLGAKVITFFLTFGSLLGHVQPATLFNVAESNASYAPYLLMTLPFCLASFGYHGNVPSLMKYYGKDPKTIVKCLVYGTLMALALYTIWLLATMGNIPRPEFIGIAEKGGNIDVLVQALSGVLNSRSLDLLLVVFSNFAVASSFLGVTLGLFDYLADLFGFDDSAVGRLKTALLTFAPPVVGGLLFPNGFLYAIGYAGLAATIWAAIVPALLARASRKRFGSPKFRVWGGKPMIALILVFGVGNALVHILSSFNLLPVYQ',\n",
       "  'mw': 44333.0},\n",
       " 'b3172': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A6E4',\n",
       "  'gene': 'b3172',\n",
       "  'locus': ['b3172', 'JW3140'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + L-aspartate + L-citrulline = 2-(N(omega)-L-arginino)succinate + AMP + diphosphate + H(+)',\n",
       "     '6.3.4.5')],\n",
       "   'Pathway': 'Amino-acid biosynthesis; L-arginine biosynthesis; L-arginine from L-ornithine and carbamoyl phosphate: step 2/3'},\n",
       "  'seq': 'MTTILKHLPVGQRIGIAFSGGLDTSAALLWMRQKGAVPYAYTANLGQPDEEDYDAIPRRAMEYGAENARLIDCRKQLVAEGIAAIQCGAFHNTTGGLTYFNTTPLGRAVTGTMLVAAMKEDGVNIWGDGSTYKGNDIERFYRYGLLTNAELQIYKPWLDTDFIDELGGRHEMSEFMIACGFDYKMSVEKAYSTDSNMLGATHEAKDLEYLNSSVKIVNPIMGVKFWDESVKIPAEEVTVRFEQGHPVALNGKTFSDDVEMMLEANRIGGRHGLGMSDQIENRIIEAKSRGIYEAPGMALLHIAYERLLTGIHNEDTIEQYHAHGRQLGRLLYQGRWFDSQALMLRDSLQRWVASQITGEVTLELRRGNDYSILNTVSENLTYKPERLTMEKGDSVFSPDDRIGQLTMRNLDITDTREKLFGYAKTGLLSSSAASGVPQVENLENKGQ',\n",
       "  'mw': 49898.0},\n",
       " 'b3176': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P31120',\n",
       "  'gene': 'b3176',\n",
       "  'locus': ['b3176', 'JW3143'],\n",
       "  'synonyms': 'mrsA yhbF',\n",
       "  'properties': {'Catalytic Activity': [('alpha-D-glucosamine 1-phosphate = D-glucosamine 6-phosphate',\n",
       "     '5.4.2.10')],\n",
       "   'Function': 'Catalyzes the conversion of glucosamine-6-phosphate to glucosamine-1-phosphate. Can also catalyze the formation of glucose-6-P from glucose-1-P, although at a 1400-fold lower rate',\n",
       "   'Regulation Activity': 'Autophosphorylation is inhibited by inorganic phosphate or EDTA'},\n",
       "  'seq': 'MSNRKYFGTDGIRGRVGDAPITPDFVLKLGWAAGKVLARHGSRKIIIGKDTRISGYMLESALEAGLAAAGLSALFTGPMPTPAVAYLTRTFRAEAGIVISASHNPFYDNGIKFFSIDGTKLPDAVEEAIEAEMEKEISCVDSAELGKASRIVDAAGRYIEFCKATFPNELSLSELKIVVDCANGATYHIAPNVLRELGANVIAIGCEPNGVNINAEVGATDVRALQARVLAEKADLGIAFDGDGDRVIMVDHEGNKVDGDQIMYIIAREGLRQGQLRGGAVGTLMSNMGLELALKQLGIPFARAKVGDRYVLEKMQEKGWRIGAENSGHVILLDKTTTGDGIVAGLQVLAAMARNHMSLHDLCSGMKMFPQILVNVRYTAGSGDPLEHESVKAVTAEVEAALGNRGRVLLRKSGTEPLIRVMVEGEDEAQVTEFAHRIADAVKAV',\n",
       "  'mw': 47544.0},\n",
       " 'b3177': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AC13',\n",
       "  'gene': 'b3177',\n",
       "  'locus': ['b3177', 'JW3144'],\n",
       "  'synonyms': 'dhpS',\n",
       "  'properties': {'Catalytic Activity': [('(7,8-dihydropterin-6-yl)methyl diphosphate + 4-aminobenzoate = 7,8-dihydropteroate + diphosphate',\n",
       "     '2.5.1.15')],\n",
       "   'Function': 'Catalyzes the condensation of para-aminobenzoate (pABA) with 6-hydroxymethyl-7,8-dihydropterin diphosphate (DHPt-PP) to form 7,8-dihydropteroate (H2Pte), the immediate precursor of folate derivatives',\n",
       "   'Pathway': 'Cofactor biosynthesis; tetrahydrofolate biosynthesis; 7,8-dihydrofolate from 2-amino-4-hydroxy-6-hydroxymethyl-7,8-dihydropteridine diphosphate and 4-aminobenzoate: step 1/2'},\n",
       "  'seq': 'MKLFAQGTSLDLSHPHVMGILNVTPDSFSDGGTHNSLIDAVKHANLMINAGATIIDVGGESTRPGAAEVSVEEELQRVIPVVEAIAQRFEVWISVDTSKPEVIRESAKVGAHIINDIRSLSEPGALEAAAETGLPVCLMHMQGNPKTMQEAPKYDDVFAEVNRYFIEQIARCEQAGIAKEKLLLDPGFGFGKNLSHNYSLLARLAEFHHFNLPLLVGMSRKSMIGQLLNVGPSERLSGSLACAVIAAMQGAHIIRVHDVKETVEAMRVVEATLSAKENKRYE',\n",
       "  'mw': 30615.0},\n",
       " 'b3182': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P24228',\n",
       "  'gene': 'b3182',\n",
       "  'locus': ['b3182', 'JW3149'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('Preferential cleavage: (Ac)2-L-Lys-D-Ala-|-D-Ala. Also transpeptidation of peptidyl-alanyl moieties that are N-acyl substituents of D-alanine.',\n",
       "     '3.4.16.4')],\n",
       "   'Function': 'Not involved in transpeptidation but exclusively catalyzes a DD-carboxypeptidase and DD-endopeptidase reaction',\n",
       "   'Pathway': 'Cell wall biogenesis; peptidoglycan biosynthesis'},\n",
       "  'seq': 'MRFSRFIIGLTSCIAFSVQAANVDEYITQLPAGANLALMVQKVGASAPAIDYHSQQMALPASTQKVITALAALIQLGPDFRFTTTLETKGNVENGVLKGDLVARFGADPTLKRQDIRNMVATLKKSGVNQIDGNVLIDTSIFASHDKAPGWPWNDMTQCFSAPPAAAIVDRNCFSVSLYSAPKPGDMAFIRVASYYPVTMFSQVRTLPRGSAEAQYCELDVVPGDLNRFTLTGCLPQRSEPLPLAFAVQDGASYAGAILKDELKQAGITWSGTLLRQTQVNEPGTVVASKQSAPLHDLLKIMLKKSDNMIADTVFRMIGHARFNVPGTWRAGSDAVRQILRQQAGVDIGNTIIADGSGLSRHNLIAPATMMQVLQYIAQHDNELNFISMLPLAGYDGSLQYRAGLHQAGVDGKVSAKTGSLQGVYNLAGFITTASGQRMAFVQYLSGYAVEPADQRNRRIPLVRFESRLYKDIYQNN',\n",
       "  'mw': 51798.0},\n",
       " 'b3187': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AD57',\n",
       "  'gene': 'b3187',\n",
       "  'locus': ['b3187', 'JW3154'],\n",
       "  'synonyms': 'cel yhbD',\n",
       "  'properties': {'Catalytic Activity': [('(2E,6E)-farnesyl diphosphate + 5 isopentenyl diphosphate = all-trans-octaprenyl diphosphate + 5 diphosphate',\n",
       "     '2.5.1.90')],\n",
       "   'Function': 'Supplies octaprenyl diphosphate, the precursor for the side chain of the isoprenoid quinones ubiquinone and menaquinone'},\n",
       "  'seq': 'MNLEKINELTAQDMAGVNAAILEQLNSDVQLINQLGYYIVSGGGKRIRPMIAVLAARAVGYEGNAHVTIAALIEFIHTATLLHDDVVDESDMRRGKATANAAFGNAASVLVGDFIYTRAFQMMTSLGSLKVLEVMSEAVNVIAEGEVLQLMNVNDPDITEENYMRVIYSKTARLFEAAAQCSGILAGCTPEEEKGLQDYGRYLGTAFQLIDDLLDYNADGEQLGKNVGDDLNEGKPTLPLLHAMHHGTPEQAQMIRTAIEQGNGRHLLEPVLEAMNACGSLEWTRQRAEEEADKAIAALQVLPDTPWREALIGLAHIAVQRDR',\n",
       "  'mw': 35217.0},\n",
       " 'b3189': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A749',\n",
       "  'gene': 'b3189',\n",
       "  'locus': ['b3189', 'JW3156'],\n",
       "  'synonyms': 'murZ',\n",
       "  'properties': {'Catalytic Activity': [('phosphoenolpyruvate + UDP-N-acetyl-alpha-D-glucosamine = phosphate + UDP-N-acetyl-3-O-(1-carboxyvinyl)-alpha-D-glucosamine',\n",
       "     '2.5.1.7')],\n",
       "   'Function': 'Cell wall formation (PubMed:1512209). Adds enolpyruvyl to UDP-N-acetylglucosamine (PubMed:1512209, PubMed:20392080). Target for the antibiotic fosfomycin',\n",
       "   'Regulation Activity': \"Competitively inhibited by UDP-N-acetylglucosamine 3'-phosphate, with a Ki of 7 uM (PubMed:21445328). In vitro inhibited by covalent binding of fosfomycin and the fungal product terreic acid in the presence of substrate UDP-N-acetylglucosamine, with an inactivation rate constant of 102 M(-1)sec(-1) for terreic acid (PubMed:20392080)\",\n",
       "   'Pathway': 'Cell wall biogenesis; peptidoglycan biosynthesis'},\n",
       "  'seq': 'MDKFRVQGPTKLQGEVTISGAKNAALPILFAALLAEEPVEIQNVPKLKDVDTSMKLLSQLGAKVERNGSVHIDARDVNVFCAPYDLVKTMRASIWALGPLVARFGQGQVSLPGGCTIGARPVDLHISGLEQLGATIKLEEGYVKASVDGRLKGAHIVMDKVSVGATVTIMCAATLAEGTTIIENAAREPEIVDTANFLITLGAKISGQGTDRIVIEGVERLGGGVYRVLPDRIETGTFLVAAAISRGKIICRNAQPDTLDAVLAKLRDAGADIEVGEDWISLDMHGKRPKAVNVRTAPHPAFPTDMQAQFTLLNLVAEGTGFITETVFENRFMHVPELSRMGAHAEIESNTVICHGVEKLSGAQVMATDLRASASLVLAGCIAEGTTVVDRIYHIDRGYERIEDKLRALGANIERVKGE',\n",
       "  'mw': 44818.0},\n",
       " 'b3196': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P45394',\n",
       "  'gene': 'b3196',\n",
       "  'locus': ['b3196', 'JW3163'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': []},\n",
       "  'seq': 'MLLATALLIVGLLLVVYSADRLVFAASILCRTFGIPPLIIGMTVVSIGTSLPEVIVSLAASLHEQRDLAVGTALGSNIINILLILGLAALVRPFTVHSDVLRRELPLMLLVSVVAGSVLYDGQLSRSDGIFLLFLAVLWLLFIVKLARQAERQGTDSLTREQLAELPRDGGLPVAFLWLGIALIIMPVATRMVVDNATVLANYFAISELTMGLTAIAIGTSLPELATAIAGVRKGENDIAVGNIIGANIFNIVIVLGLPALITPGEIDPLAYSRDYSVMLLVSIIFALLCWRRSPQPGRGVGVLLTGGFIVWLAMLYWLSPILVE',\n",
       "  'mw': 34741.0},\n",
       " 'b3197': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P45395',\n",
       "  'gene': 'b3197',\n",
       "  'locus': ['b3197', 'JW3164'],\n",
       "  'synonyms': 'yrbH',\n",
       "  'properties': {'Catalytic Activity': [('D-arabinose 5-phosphate = D-ribulose 5-phosphate',\n",
       "     '5.3.1.13')],\n",
       "   'Function': 'Involved in the biosynthesis of 3-deoxy-D-manno-octulosonate (KDO), a unique 8-carbon sugar component of lipopolysaccharides (LPSs). KdsD is not essential in the KDO biosynthesis and can be substituted by GutQ. Catalyzes the reversible aldol-ketol isomerization between D-ribulose 5-phosphate (Ru5P) and D-arabinose 5-phosphate (A5P)',\n",
       "   'Regulation Activity': 'Completely inhibited by 10 uM of nickel, copper, cadmium and mercury ions. Inhibited by zinc with an IC(50) of 1-3 uM. Metal ion inhibition may be a mechanism to control activity in vivo',\n",
       "   'Pathway': 'Bacterial outer membrane biogenesis; lipopolysaccharide biosynthesis'},\n",
       "  'seq': 'MSHVELQPGFDFQQAGKEVLAIERECLAELDQYINQNFTLACEKMFWCKGKVVVMGMGKSGHIGRKMAATFASTGTPSFFVHPGEAAHGDLGMVTPQDVVIAISNSGESSEITALIPVLKRLHVPLICITGRPESSMARAADVHLCVKVAKEACPLGLAPTSSTTATLVMGDALAVALLKARGFTAEDFALSHPGGALGRKLLLRVNDIMHTGDEIPHVKKTASLRDALLEVTRKNLGMTVICDDNMMIEGIFTDGDLRRVFDMGVDVRQLSIADVMTPGGIRVRPGILAVEALNLMQSRHITSVMVADGDHLLGVLHMHDLLRAGVV',\n",
       "  'mw': 35196.0},\n",
       " 'b3198': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ABZ4',\n",
       "  'gene': 'b3198',\n",
       "  'locus': ['b3198', 'JW3165'],\n",
       "  'synonyms': 'yrbI yrbJ',\n",
       "  'properties': {'Catalytic Activity': [('3-deoxy-alpha-D-manno-2-octulosonate-8-phosphate + H2O = 3-deoxy-alpha-D-manno-oct-2-ulosonate + phosphate',\n",
       "     '3.1.3.45')],\n",
       "   'Function': 'Catalyzes the hydrolysis of 3-deoxy-D-manno-octulosonate 8-phosphate (KDO 8-P) to 3-deoxy-D-manno-octulosonate (KDO) and inorganic phosphate',\n",
       "   'Pathway': 'Bacterial outer membrane biogenesis; lipopolysaccharide biosynthesis'},\n",
       "  'seq': 'MSKAGASLATCYGPVSADVIAKAENIRLLILDVDGVLSDGLIYMGNNGEELKAFNVRDGYGIRCALTSDIEVAIITGRKAKLVEDRCATLGITHLYQGQSNKLIAFSDLLEKLAIAPENVAYVGDDLIDWPVMEKVGLSVAVADAHPLLIPRADYVTRIAGGRGAVREVCDLLLLAQGKLDEAKGQSI',\n",
       "  'mw': 19997.0},\n",
       " 'b3212': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P09831',\n",
       "  'gene': 'b3212',\n",
       "  'locus': ['b3212', 'JW3179'],\n",
       "  'synonyms': 'aspB',\n",
       "  'properties': {'Catalytic Activity': [('2 L-glutamate + NADP(+) = 2-oxoglutarate + H(+) + L-glutamine + NADPH',\n",
       "     '1.4.1.13')],\n",
       "   'Function': 'Catalyzes the conversion of L-glutamine and 2-oxoglutarate into two molecules of L-glutamate',\n",
       "   'Pathway': 'Energy metabolism; nitrogen metabolism'},\n",
       "  'seq': 'MLYDKSLERDNCGFGLIAHIEGEPSHKVVRTAIHALARMQHRGAILADGKTGDGCGLLLQKPDRFFRIVAQERGWRLAKNYAVGMLFLNKDPELAAAARRIVEEELQRETLSIVGWRDVPTNEGVLGEIALSSLPRIEQIFVNAPAGWRPRDMERRLFIARRRIEKRLEADKDFYVCSLSNLVNIYKGLCMPTDLPRFYLDLADLRLESAICLFHQRFSTNTVPRWPLAQPFRYLAHNGEINTITGNRQWARARTYKFQTPLIPDLHDAAPFVNETGSDSSSMDNMLELLLAGGMDIIRAMRLLVPPAWQNNPDMDPELRAFFDFNSMHMEPWDGPAGIVMSDGRFAACNLDRNGLRPARYVITKDKLITCASEVGIWDYQPDEVVEKGRVGPGELMVIDTRSGRILHSAETDDDLKSRHPYKEWMEKNVRRLVPFEDLPDEEVGSRELDDDTLASYQKQFNYSAEELDSVIRVLGENGQEAVGSMGDDTPFAVLSSQPRIIYDYFRQQFAQVTNPPIDPLREAHVMSLATSIGREMNVFCEAEGQAHRLSFKSPILLYSDFKQLTTMKEEHYRADTLDITFDVTKTTLEATVKELCDKAEKMVRSGTVLLVLSDRNIAKDRLPVPAPMAVGAIQTRLVDQSLRCDANIIVETASARDPHHFAVLLGFGATAIYPYLAYETLGRLVDTHAIAKDYRTVMLNYRNGINKGLYKIMSKMGISTIASYRCSKLFEAVGLHDDVVGLCFQGAVSRIGGASFEDFQQDLLNLSKRAWLARKPISQGGLLKYVHGGEYHAYNPDVVRTLQQAVQSGEYSDYQEYAKLVNERPATTLRDLLAITPGENAVNIADVEPASELFKRFDTAAMSIGALSPEAHEALAEAMNSIGGNSNSGEGGEDPARYGTNKVSRIKQVASGRFGVTPAYLVNADVIQIKVAQGAKPGEGGQLPGDKVTPYIAKLRYSVPGVTLISPPPHHDIYSIEDLAQLIFDLKQVNPKAMISVKLVSEPGVGTIATGVAKAYADLITIAGYDGGTGASPLSSVKYAGCPWELGLVETQQALVANGLRHKIRLQVDGGLKTGVDIIKAAILGAESFGFGTGPMVALGCKYLRICHLNNCATGVATQDDKLRKNHYHGLPFKVTNYFEFIARETRELMAQLGVTRLVDLIGRTDLLKELDGFTAKQQKLALSKLLETAEPHPGKALYCTENNPPFDNGLLNAQLLQQAKPFVDERQSKTFWFDIRNTDRSVGASLSGYIAQTHGDQGLAADPIKAYFNGTAGQSFGVWNAGGVELYLTGDANDYVGKGMAGGLIAIRPPVGSAFRSHEASIIGNTCLYGATGGRLYAAGRAGERFGVRNSGAITVVEGIGDNGCEYMTGGIVCILGKTGVNFGAGMTGGFAYVLDESGDFRKRVNPELVEVLSVDALAIHEEHLRGLITEHVQHTGSQRGEEILANWSTFATKFALVKPKSSDVKALLGHRSRSAAELRVQAQ',\n",
       "  'mw': 163297.0},\n",
       " 'b3213': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P09832',\n",
       "  'gene': 'b3213',\n",
       "  'locus': ['b3213', 'JW3180'],\n",
       "  'synonyms': 'aspB',\n",
       "  'properties': {'Catalytic Activity': [('2 L-glutamate + NADP(+) = 2-oxoglutarate + H(+) + L-glutamine + NADPH',\n",
       "     '1.4.1.13')],\n",
       "   'Function': 'Catalyzes the conversion of L-glutamine and 2-oxoglutarate into two molecules of L-glutamate',\n",
       "   'Pathway': 'Energy metabolism; nitrogen metabolism'},\n",
       "  'seq': 'MSQNVYQFIDLQRVDPPKKPLKIRKIEFVEIYEPFSEGQAKAQADRCLSCGNPYCEWKCPVHNYIPNWLKLANEGRIFEAAELSHQTNTLPEVCGRVCPQDRLCEGSCTLNDEFGAVTIGNIERYINDKAFEMGWRPDMSGVKQTGKKVAIIGAGPAGLACADVLTRNGVKAVVFDRHPEIGGLLTFGIPAFKLEKEVMTRRREIFTGMGIEFKLNTEVGRDVQLDDLLSDYDAVFLGVGTYQSMRGGLENEDADGVYAALPFLIANTKQLMGFGETRDEPFVSMEGKRVVVLGGGDTAMDCVRTSVRQGAKHVTCAYRRDEENMPGSRREVKNAREEGVEFKFNVQPLGIEVNGNGKVSGVKMVRTEMGEPDAKGRRRAEIVAGSEHIVPADAVIMAFGFRPHNMEWLAKHSVELDSQGRIIAPEGSDNAFQTSNPKIFAGGDIVRGSDLVVTAIAEGRKAADGIMNWLEV',\n",
       "  'mw': 52015.0},\n",
       " 'b3222': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P45425',\n",
       "  'gene': 'b3222',\n",
       "  'locus': ['b3222', 'JW5538'],\n",
       "  'synonyms': 'yhcI',\n",
       "  'properties': {'Catalytic Activity': [('ATP + N-acetyl-D-mannosamine = ADP + H(+) + N-acetyl-D-mannosamine 6-phosphate',\n",
       "     '2.7.1.60')],\n",
       "   'Function': 'Catalyzes the phosphorylation of N-acetylmannosamine (ManNAc) to ManNAc-6-P. Has also low level glucokinase activity in vitro',\n",
       "   'Pathway': 'Amino-sugar metabolism; N-acetylneuraminate degradation; D-fructose 6-phosphate from N-acetylneuraminate: step 2/5'},\n",
       "  'seq': 'MTTLAIDIGGTKLAAALIGADGQIRDRRELPTPASQTPEALRDALSALVSPLQAHAQRVAIASTGIIRDGSLLALNPHNLGGLLHFPLVKTLEQLTNLPTIAINDAQAAAWAEFQALDGDITDMVFITVSTGVGGGVVSGCKLLTGPGGLAGHIGHTLADPHGPVCGCGRTGCVEAIASGRGIAAAAQGELAGADAKTIFTRAGQGDEQAQQLIHRSARTLARLIADIKATTDCQCVVVGGSVGLAEGYLALVETYLAQEPAAFHVDLLAAHYRHDAGLLGAALLAQGEKL',\n",
       "  'mw': 29644.0},\n",
       " 'b3223': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A761',\n",
       "  'gene': 'b3223',\n",
       "  'locus': ['b3223', 'JW3192'],\n",
       "  'synonyms': 'yhcJ',\n",
       "  'properties': {'Catalytic Activity': [('an N-acyl-D-glucosamine 6-phosphate = an N-acyl-D-mannosamine 6-phosphate',\n",
       "     '5.1.3.9')],\n",
       "   'Function': 'Converts N-acetylmannosamine-6-phosphate (ManNAc-6-P) to N-acetylglucosamine-6-phosphate (GlcNAc-6-P)',\n",
       "   'Pathway': 'Amino-sugar metabolism; N-acetylneuraminate degradation; D-fructose 6-phosphate from N-acetylneuraminate: step 3/5'},\n",
       "  'seq': 'MSLLAQLDQKIAANGGLIVSCQPVPDSPLDKPEIVAAMALAAEQAGAVAIRIEGVANLQATRAVVSVPIIGIVKRDLEDSPVRITAYIEDVDALAQAGADIIAIDGTDRPRPVPVETLLARIHHHGLLAMTDCSTPEDGLACQKLGAEIIGTTLSGYTTPETPEEPDLALVKTLSDAGCRVIAEGRYNTPAQAADAMRHGAWAVTVGSAITRLEHICQWYNTAMKKAVL',\n",
       "  'mw': 24074.0},\n",
       " 'b3224': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P41036',\n",
       "  'gene': 'b3224',\n",
       "  'locus': ['b3224', 'JW3193'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('H(+)(in) + N-acetylneuraminate(in) = H(+)(out) + N-acetylneuraminate(out)',\n",
       "     '')],\n",
       "   'Function': 'Catalyzes the proton-dependent transport of sialic acid (PubMed:22167185, PubMed:23848303). Can transport the common sialic acid N-acetylneuraminic acid (Neu5Ac) and the related sialic acids N-glycolylneuraminic acid (Neu5Gc) and 3-keto-3-deoxy-D-glycero-D-galactonononic acid (KDN) (PubMed:23848303). Functions as a bidirectional transporter in vitro (PubMed:22167185)'},\n",
       "  'seq': 'MSTTTQNIPWYRHLNRAQWRAFSAAWLGYLLDGFDFVLIALVLTEVQGEFGLTTVQAASLISAAFISRWFGGLMLGAMGDRYGRRLAMVTSIVLFSAGTLACGFAPGYITMFIARLVIGMGMAGEYGSSATYVIESWPKHLRNKASGFLISGFSVGAVVAAQVYSLVVPVWGWRALFFIGILPIIFALWLRKNIPEAEDWKEKHAGKAPVRTMVDILYRGEHRIANIVMTLAAATALWFCFAGNLQNAAIVAVLGLLCAAIFISFMVQSAGKRWPTGVMLMVVVLFAFLYSWPIQALLPTYLKTDLAYNPHTVANVLFFSGFGAAVGCCVGGFLGDWLGTRKAYVCSLLASQLLIIPVFAIGGANVWVLGLLLFFQQMLGQGIAGILPKLIGGYFDTDQRAAGLGFTYNVGALGGALAPIIGALIAQRLDLGTALASLSFSLTFVVILLIGLDMPSRVQRWLRPEALRTHDAIDGKPFSGAVPFGSAKNDLVKTKS',\n",
       "  'mw': 53551.0},\n",
       " 'b3225': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A6L4',\n",
       "  'gene': 'b3225',\n",
       "  'locus': ['b3225', 'JW3194'],\n",
       "  'synonyms': 'npl',\n",
       "  'properties': {'Catalytic Activity': [('aceneuramate = N-acetyl-D-mannosamine + pyruvate',\n",
       "     '4.1.3.3')],\n",
       "   'Function': 'Catalyzes the reversible aldol cleavage of N-acetylneuraminic acid (sialic acid; Neu5Ac) to form pyruvate and N-acetylmannosamine (ManNAc) via a Schiff base intermediate (PubMed:1646603). Experiments show the true substrate is aceneuramate (linearized Neu5Ac), which forms spontaneously at alkaline pH. Linear aceneuramate can be provided by NanQ (PubMed:33895133)',\n",
       "   'Regulation Activity': 'Inhibited by reduction with NaBH(4), and by Cu(2+) ions, p-chloromercuribenzoate and N-bromosuccinimide (PubMed:1646603). Co(2+), Mn(2+) and Ni(2+) stimulate activity, perhaps by enhancing the opening of the substrate Neu5Ac (PubMed:33895133)',\n",
       "   'Pathway': 'Amino-sugar metabolism; N-acetylneuraminate degradation; D-fructose 6-phosphate from N-acetylneuraminate: step 1/5'},\n",
       "  'seq': 'MATNLRGVMAALLTPFDQQQALDKASLRRLVQFNIQQGIDGLYVGGSTGEAFVQSLSEREQVLEIVAEEAKGKIKLIAHVGCVSTAESQQLAASAKRYGFDAVSAVTPFYYPFSFEEHCDHYRAIIDSADGLPMVVYNIPALSGVKLTLDQINTLVTLPGVGALKQTSGDLYQMEQIRREHPDLVLYNGYDEIFASGLLAGADGGIGSTYNIMGWRYQGIVKALKEGDIQTAQKLQTECNKVIDLLIKTGVFRGLKTVLHYMDVVSVPLCRKPFGPVDEKYLPELKALAQQLMQERG',\n",
       "  'mw': 32593.0},\n",
       " 'b3236': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P61889',\n",
       "  'gene': 'b3236',\n",
       "  'locus': ['b3236', 'JW3205'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(S)-malate + NAD(+) = H(+) + NADH + oxaloacetate',\n",
       "     '1.1.1.37')],\n",
       "   'Function': 'Catalyzes the reversible oxidation of malate to oxaloacetate'},\n",
       "  'seq': 'MKVAVLGAAGGIGQALALLLKTQLPSGSELSLYDIAPVTPGVAVDLSHIPTAVKIKGFSGEDATPALEGADVVLISAGVARKPGMDRSDLFNVNAGIVKNLVQQVAKTCPKACIGIITNPVNTTVAIAAEVLKKAGVYDKNKLFGVTTLDIIRSNTFVAELKGKQPGEVEVPVIGGHSGVTILPLLSQVPGVSFTEQEVADLTKRIQNAGTEVVEAKAGGGSATLSMGQAAARFGLSLVRALQGEQGVVECAYVEGDGQYARFFSQPLLLGKNGVEERKSIGTLSAFEQNALEGMLDTLKKDIALGEEFVNK',\n",
       "  'mw': 32337.0},\n",
       " 'b3255': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0ABD8',\n",
       "  'gene': 'b3255',\n",
       "  'locus': ['b3255', 'JW3223'],\n",
       "  'synonyms': 'fabE',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'This protein is a component of the acetyl coenzyme A carboxylase complex; first, biotin carboxylase catalyzes the carboxylation of the carrier protein and then the transcarboxylase transfers the carboxyl group to form malonyl-CoA',\n",
       "   'Pathway': 'Lipid metabolism; fatty acid biosynthesis'},\n",
       "  'seq': 'MDIRKIKKLIELVEESGISELEISEGEESVRISRAAPAASFPVMQQAYAAPMMQQPAQSNAAAPATVPSMEAPAAAEISGHIVRSPMVGTFYRTPSPDAKAFIEVGQKVNVGDTLCIVEAMKMMNQIEADKSGTVKAILVESGQPVEFDEPLVVIE',\n",
       "  'mw': 16687.0},\n",
       " 'b3256': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P24182',\n",
       "  'gene': 'b3256',\n",
       "  'locus': ['b3256', 'JW3224'],\n",
       "  'synonyms': 'fabG',\n",
       "  'properties': {'Catalytic Activity': [('ATP + hydrogencarbonate + N(6)-biotinyl-L-lysyl-[protein] = ADP + H(+) + N(6)-carboxybiotinyl-L-lysyl-[protein] + phosphate',\n",
       "     '6.3.4.14')],\n",
       "   'Function': 'This protein is a component of the acetyl coenzyme A carboxylase complex; first, biotin carboxylase catalyzes the carboxylation of the carrier protein and then the transcarboxylase transfers the carboxyl group to form malonyl-CoA',\n",
       "   'Pathway': 'Lipid metabolism; malonyl-CoA biosynthesis; malonyl-CoA from acetyl-CoA: step 1/1'},\n",
       "  'seq': 'MLDKIVIANRGEIALRILRACKELGIKTVAVHSSADRDLKHVLLADETVCIGPAPSVKSYLNIPAIISAAEITGAVAIHPGYGFLSENANFAEQVERSGFIFIGPKAETIRLMGDKVSAIAAMKKAGVPCVPGSDGPLGDDMDKNRAIAKRIGYPVIIKASGGGGGRGMRVVRGDAELAQSISMTRAEAKAAFSNDMVYMEKYLENPRHVEIQVLADGQGNAIYLAERDCSMQRRHQKVVEEAPAPGITPELRRYIGERCAKACVDIGYRGAGTFEFLFENGEFYFIEMNTRIQVEHPVTEMITGVDLIKEQLRIAAGQPLSIKQEEVHVRGHAVECRINAEDPNTFLPSPGKITRFHAPGGFGVRWESHIYAGYTVPPYYDSMIGKLICYGENRDVAIARMKNALQELIIDGIKTNVDLQIRIMNDENFQHGGTNIHYLEKKLGLQEK',\n",
       "  'mw': 49321.0},\n",
       " 'b3258': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P16256',\n",
       "  'gene': 'b3258',\n",
       "  'locus': ['b3258', 'JW3226'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('(R)-pantothenate(in) + Na(+)(in) = (R)-pantothenate(out) + Na(+)(out)',\n",
       "     '')],\n",
       "   'Function': 'Catalyzes the sodium-dependent uptake of extracellular pantothenate',\n",
       "   'Regulation Activity': 'Pantothenate uptake is not reduced in osmotically shocked cells or by ATP depletion with arsenate, but is reduced greater than 90% by the dissipation of the membrane electrochemical gradient with 2,4-dinitrophenol'},\n",
       "  'seq': 'MQLEVILPLVAYLVVVFGISVYAMRKRSTGTFLNEYFLGSRSMGGIVLAMTLTATYISASSFIGGPGAAYKYGLGWVLLAMIQLPAVWLSLGILGKKFAILARRYNAVTLNDMLFARYQSRLLVWLASLSLLVAFVGAMTVQFIGGARLLETAAGIPYETGLLIFGISIALYTAFGGFRASVLNDTMQGLVMLIGTVVLLIGVVHAAGGLSNAVQTLQTIDPQLVTPQGADDILSPAFMTSFWVLVCFGVIGLPHTAVRCISYKDSKAVHRGIIIGTIVVAILMFGMHLAGALGRAVIPDLTVPDLVIPTLMVKVLPPFAAGIFLAAPMAAIMSTINAQLLQSSATIIKDLYLNIRPDQMQNETRLKRMSAVITLVLGALLLLAAWKPPEMIIWLNLLAFGGLEAVFLWPLVLGLYWERANAKGALSAMIVGGVLYAVLATLNIQYLGFHPIVPSLLLSLLAFLVGNRFGTSVPQATVLTTDK',\n",
       "  'mw': 51717.0},\n",
       " 'b3265': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P24180',\n",
       "  'gene': 'b3265',\n",
       "  'locus': ['b3265', 'JW3233'],\n",
       "  'synonyms': 'envC',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the tripartite efflux system AcrEF-TolC. Involved in the efflux of indole and organic solvents'},\n",
       "  'seq': 'MTKHARFFLLPSFILISAALIAGCNDKGEEKAHVGEPQVTVHIVKTAPLEVKTELPGRTNAYRIAEVRPQVSGIVLNRNFTEGSDVQAGQSLYQIDPATYQANYDSAKGELAKSEAAAAIAHLTVKRYVPLVGTKYISQQEYDQAIADARQADAAVIAAKATVESARINLAYTKVTAPISGRIGKSTVTEGALVTNGQTTELATVQQLDPIYVDVTQSSNDFMRLKQSVEQGNLHKENATSNVELVMENGQTYPLKGTLQFSDVTVDESTGSITLRAVFPNPQHTLLPGMFVRARIDEGVQPDAILIPQQGVSRTPRGDATVLIVNDKSQVEARPVVASQAIGDKWLISEGLKSGDQVIVSGLQKARPGEQVKATTDTPADTASK',\n",
       "  'mw': 41318.0},\n",
       " 'b3266': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P24181',\n",
       "  'gene': 'b3266',\n",
       "  'locus': ['b3266', 'JW3234'],\n",
       "  'synonyms': 'envD',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the tripartite efflux system AcrEF-TolC. Involved in the efflux of indole and organic solvents'},\n",
       "  'seq': 'MANFFIRRPIFAWVLAIILMMAGALAILQLPVAQYPTIAPPAVSVSANYPGADAQTVQDTVTQVIEQNMNGIDNLMYMSSTSDSAGSVTITLTFQSGTDPDIAQVQVQNKLQLATPLLPQEVQQQGISVEKSSSSYLMVAGFVSDNPGTTQDDISDYVASNVKDTLSRLNGVGDVQLFGAQYAMRIWLDADLLNKYKLTPVDVINQLKVQNDQIAAGQLGGTPALPGQQLNASIIAQTRFKNPEEFGKVTLRVNSDGSVVRLKDVARVELGGENYNVIARINGKPAAGLGIKLATGANALDTAKAIKAKLAELQPFFPQGMKVLYPYDTTPFVQLSIHEVVKTLFEAIMLVFLVMYLFLQNMRATLIPTIAVPVVLLGTFAILAAFGYSINTLTMFGMVLAIGLLVDDAIVVVENVERVMMEDKLPPKEATEKSMSQIQGALVGIAMVLSAVFIPMAFFGGSTGAIYRQFSITIVSAMALSVLVALILTPALCATLLKPVSAEHHENKGGFFGWFNTTFDHSVNHYTNSVGKILGSTGRYLLIYALIVAGMVVLFLRLPSSFLPEEDQGVFLTMIQLPAGATQERTQKVLDQVTDYYLKNEKANVESVFTVNGFSFSGQAQNAGMAFVSLKPWEERNGDENSAEAVIHRAKMELGKIRDGFVIPFNMPAIVELGTATGFDFELIDQAGLGHDALTQARNQLLGMAAQHPASLVSVRPNGLEDTAQFKLEVDQEKAQALGVSLSDINQTISTALGGTYVNDFIDRGRVKKLYVQADAKFRMLPEDVDKLYVRSANGEMVPFSAFTTSHWVYGSPRLERYNGLPSMEIQGEAAPGTSSGDAMALMENLASKLPAGIGYDWTGMSYQERLSGNQAPALVAISFVVVFLCLAALYESWSIPVSVMLVVPLGIVGVLLAATLFNQKNDVYFMVGLLTTIGLSAKNAILIVEFAKDLMEKEGKGVVEATLMAVRMRLRPILMTSLAFILGVLPLAISNGAGSGAQNAVGIGVMGGMVSATLLAIFFVPVFFVVIRRCFKG',\n",
       "  'mw': 111454.0},\n",
       " 'b3281': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P15770',\n",
       "  'gene': 'b3281',\n",
       "  'locus': ['b3281', 'JW3242'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('NADP(+) + shikimate = 3-dehydroshikimate + H(+) + NADPH',\n",
       "     '1.1.1.25')],\n",
       "   'Function': 'Involved in the biosynthesis of the chorismate, which leads to the biosynthesis of aromatic amino acids. Catalyzes the reversible NADPH linked reduction of 3-dehydroshikimate (DHSA) to yield shikimate (SA). It displays no activity in the presence of NAD',\n",
       "   'Pathway': 'Metabolic intermediate biosynthesis; chorismate biosynthesis; chorismate from D-erythrose 4-phosphate and phosphoenolpyruvate: step 4/7'},\n",
       "  'seq': 'METYAVFGNPIAHSKSPFIHQQFAQQLNIEHPYGRVLAPINDFINTLNAFFSAGGKGANVTVPFKEEAFARADELTERAALAGAVNTLMRLEDGRLLGDNTDGVGLLSDLERLSFIRPGLRILLIGAGGASRGVLLPLLSLDCAVTITNRTVSRAEELAKLFAHTGSIQALSMDELEGHEFDLIINATSSGISGDIPAIPSSLIHPGIYCYDMFYQKGKTPFLAWCEQRGSKRNADGLGMLVAQAAHAFLLWHGVLPDVEPVIKQLQEELSA',\n",
       "  'mw': 29414.0},\n",
       " 'b3288': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P23882',\n",
       "  'gene': 'b3288',\n",
       "  'locus': ['b3288', 'JW3249'],\n",
       "  'synonyms': 'yhdD',\n",
       "  'properties': {'Catalytic Activity': [('(6S)-10-formyltetrahydrofolate + L-methionyl-tRNA(fMet) = (6S)-5,6,7,8-tetrahydrofolate + H(+) + N-formyl-L-methionyl-tRNA(fMet)',\n",
       "     '2.1.2.9')],\n",
       "   'Function': 'Attaches a formyl group to the free amino group of methionyl-tRNA(fMet). The formyl group appears to play a dual role in the initiator identity of N-formylmethionyl-tRNA by promoting its recognition by IF2 and preventing the misappropriation of this tRNA by the elongation apparatus',\n",
       "   'Regulation Activity': 'Activity is optimum in the presence of Mg(2+) and K(+)'},\n",
       "  'seq': 'MSESLRIIFAGTPDFAARHLDALLSSGHNVVGVFTQPDRPAGRGKKLMPSPVKVLAEEKGLPVFQPVSLRPQENQQLVAELQADVMVVVAYGLILPKAVLEMPRLGCINVHGSLLPRWRGAAPIQRSLWAGDAETGVTIMQMDVGLDTGDMLYKLSCPITAEDTSGTLYDKLAELGPQGLITTLKQLADGTAKPEVQDETLVTYAEKLSKEEARIDWSLSAAQLERCIRAFNPWPMSWLEIEGQPVKVWKASVIDTATNAAPGTILEANKQGIQVATGDGILNLLSLQPAGKKAMSAQDLLNSRREWFVPGNRLV',\n",
       "  'mw': 34168.0},\n",
       " 'b3290': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AGI8',\n",
       "  'gene': 'b3290',\n",
       "  'locus': ['b3290', 'JW3251'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the constitutive potassium transport systems TrkG and TrkH. May regulate the transport activity of TrkG and TrkH systems. Binds to NAD(+) and NADH'},\n",
       "  'seq': 'MKIIILGAGQVGGTLAENLVGENNDITVVDTNGERLRTLQDKFDLRVVQGHGSHPRVLREAGADDADMLVAVTSSDETNMVACQVAYSLFNTPNRIARIRSPDYVRDADKLFHSDAVPIDHLIAPEQLVIDNIYRLIEYPGALQVVNFAEGKVSLAVVKAYYGGPLIGNALSTMREHMPHIDTRVAAIFRHDRPIRPQGSTIVEAGDEVFFIAASQHIRAVMSELQRLEKPYKRIMLVGGGNIGAGLARRLEKDYSVKLIERNQQRAAELAEKLQNTIVFFGDASDQELLAEEHIDQVDLFIAVTNDDEANIMSAMLAKRMGAKKVMVLIQRRAYVDLVQGSVIDIAISPQQATISALLSHVRKADIVGVSSLRRGVAEAIEAVAHGDESTSRVVGRVIDEIKLPPGTIIGAVVRGNDVMIANDNLRIEQGDHVIMFLTDKKFITDVERLFQPSPFFL',\n",
       "  'mw': 50368.0},\n",
       " 'b3350': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P45522',\n",
       "  'gene': 'b3350',\n",
       "  'locus': ['b3350', 'JW3313'],\n",
       "  'synonyms': 'trkB',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Pore-forming subunit of a potassium efflux system that confers protection against electrophiles. Catalyzes K(+)/H(+) antiport',\n",
       "   'Regulation Activity': 'Activated by adducts between glutathione and electrophiles'},\n",
       "  'seq': 'MEGSDFLLAGVLFLFAAVAAVPLASRLGIGAVLGYLLAGIAIGPWGLGFISDVDEILHFSELGVVFLMFIIGLELNPSKLWQLRRSIFGVGAAQVLLSAALLAGLLMLTDFAWQAAVVGGIGLAMSSTAMALQLMREKGMNRSESGQLGFSVLLFQDLAVIPALALVPLLAGSADEHFDWMKVGMKVLAFVGMLIGGRYLLRPVFRFIAASGVREVFTAATLLLVLGSALFMDALGLSMALGTFIAGVLLAESEYRHELETAIDPFKGLLLGLFFISVGMSLNLGVLYTHLLWVVISVVVLVAVKILVLYLLARLYGVRSSERMQFAGVLSQGGEFAFVLFSTASSQRLFQGDQMALLLVTVTLSMMTTPLLMKLVDKWLSRQFNGPEEEDEKPWVNDDKPQVIVVGFGRFGQVIGRLLMANKMRITVLERDISAVNLMRKYGYKVYYGDATQVDLLRSAGAEAAESIVITCNEPEDTMKLVEICQQHFPHLHILARARGRVEAHELLQAGVTQFSRETFSSALELGRKTLVTLGMHPHQAQRAQLHFRRLDMRMLRELIPMHADTVQISRAREARRELEEIFQREMQQERRQLDGWDEFE',\n",
       "  'mw': 66411.0},\n",
       " 'b3359': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P18335',\n",
       "  'gene': 'b3359',\n",
       "  'locus': ['b3359', 'JW3322'],\n",
       "  'synonyms': 'dapC dtu',\n",
       "  'properties': {'Catalytic Activity': [('2-oxoglutarate + N(2)-acetyl-L-ornithine = L-glutamate + N-acetyl-L-glutamate 5-semialdehyde',\n",
       "     '2.6.1.11'),\n",
       "    ('2-oxoglutarate + N-succinyl-(2S,6S)-2,6-diaminoheptanedioate = (S)-2-succinylamino-6-oxoheptanedioate + L-glutamate',\n",
       "     '2.6.1.17')],\n",
       "   'Function': 'Involved in both the arginine and lysine biosynthetic pathways',\n",
       "   'Pathway': 'Amino-acid biosynthesis; L-lysine biosynthesis via DAP pathway; LL-2,6-diaminopimelate from (S)-tetrahydrodipicolinate (succinylase route): step 2/3'},\n",
       "  'seq': 'MAIEQTAITRATFDEVILPIYAPAEFIPVKGQGSRIWDQQGKEYVDFAGGIAVTALGHCHPALVNALKTQGETLWHISNVFTNEPALRLGRKLIEATFAERVVFMNSGTEANETAFKLARHYACVRHSPFKTKIIAFHNAFHGRSLFTVSVGGQPKYSDGFGPKPADIIHVPFNDLHAVKAVMDDHTCAVVVEPIQGEGGVTAATPEFLQGLRELCDQHQALLVFDEVQCGMGRTGDLFAYMHYGVTPDILTSAKALGGGFPISAMLTTAEIASAFHPGSHGSTYGGNPLACAVAGAAFDIINTPEVLEGIQAKRQRFVDHLQKIDQQYDVFSDIRGMGLLIGAELKPQYKGRARDFLYAGAEAGVMVLNAGPDVMRFAPSLVVEDADIDEGMQRFAHAVAKVVGA',\n",
       "  'mw': 43767.0},\n",
       " 'b3360': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P00903',\n",
       "  'gene': 'b3360',\n",
       "  'locus': ['b3360', 'JW3323'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('chorismate + L-glutamine = 4-amino-4-deoxychorismate + L-glutamate',\n",
       "     '2.6.1.85')],\n",
       "   'Function': 'Part of a heterodimeric complex that catalyzes the two-step biosynthesis of 4-amino-4-deoxychorismate (ADC), a precursor of p-aminobenzoate (PABA) and tetrahydrofolate. In the first step, a glutamine amidotransferase (PabA) generates ammonia as a substrate that, along with chorismate, is used in the second step, catalyzed by aminodeoxychorismate synthase (PabB) to produce ADC. PabA converts glutamine into glutamate only in the presence of stoichiometric amounts of PabB',\n",
       "   'Regulation Activity': 'Inhibited by 6-diazo-5-oxo-L-norleucine (DON). The inhibition is competitive with glutamine, but uncompetitive with chorismate',\n",
       "   'Pathway': 'Cofactor biosynthesis; tetrahydrofolate biosynthesis; 4-aminobenzoate from chorismate: step 1/2'},\n",
       "  'seq': 'MILLIDNYDSFTWNLYQYFCELGADVLVKRNDALTLADIDALKPQKIVISPGPCTPDEAGISLDVIRHYAGRLPILGVCLGHQAMAQAFGGKVVRAAKVMHGKTSPITHNGEGVFRGLANPLTVTRYHSLVVEPDSLPACFDVTAWSETREIMGIRHRQWDLEGVQFHPESILSEQGHQLLANFLHR',\n",
       "  'mw': 20772.0},\n",
       " 'b3365': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P08201',\n",
       "  'gene': 'b3365',\n",
       "  'locus': ['b3365', 'JW3328'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('2 H2O + 3 NAD(+) + NH4(+) = 5 H(+) + 3 NADH + nitrite',\n",
       "     '1.7.1.15')],\n",
       "   'Pathway': 'Nitrogen metabolism; nitrate reduction (assimilation)'},\n",
       "  'seq': 'MSKVRLAIIGNGMVGHRFIEDLLDKSDAANFDITVFCEEPRIAYDRVHLSSYFSHHTAEELSLVREGFYEKHGIKVLVGERAITINRQEKVIHSSAGRTVFYDKLIMATGSYPWIPPIKGSDTQDCFVYRTIEDLNAIESCARRSKRGAVVGGGLLGLEAAGALKNLGIETHVIEFAPMLMAEQLDQMGGEQLRRKIESMGVRVHTSKNTLEIVQEGVEARKTMRFADGSELEVDFIVFSTGIRPRDKLATQCGLDVAPRGGIVINDSCQTSDPDIYAIGECASWNNRVFGLVAPGYKMAQVAVDHILGSENAFEGADLSAKLKLLGVDVGGIGDAHGRTPGARSYVYLDESKEIYKRLIVSEDNKTLLGAVLVGDTSDYGNLLQLVLNAIELPENPDSLILPAHSGSGKPSIGVDKLPDSAQICSCFDVTKGDLIAAINKGCHTVAALKAETKAGTGCGGCIPLVTQVLNAELAKQGIEVNNNLCEHFAYSRQELFHLIRVEGIKTFEELLAKHGKGYGCEVCKPTVGSLLASCWNEYILKPEHTPLQDSNDNFLANIQKDGTYSVIPRSPGGEITPEGLMAVGRIAREFNLYTKITGSQRLAMFGAQKDDLPEIWRQLIEAGFETGHAYAKALRMAKTCVGSTWCRYGVGDSVGLGVELENRYKGIRTPHKMKFGVSGCTRECSEAQGKDVGIIATEKGWNLYVCGNGGMKPRHADLLAADIDRETLIKYLDRFMMFYIRTADKLTRTAPWLENLEGGIDYLKAVIIDDKLGLNAHLEEEMARLREAVLCEWTETVNTPSAQTRFKHFINSDKRDPNVQMVPEREQHRPATPYERIPVTLVEDNA',\n",
       "  'mw': 93121.0},\n",
       " 'b3366': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A9I8',\n",
       "  'gene': 'b3366',\n",
       "  'locus': ['b3366', 'JW3329'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('2 H2O + 3 NAD(+) + NH4(+) = 5 H(+) + 3 NADH + nitrite',\n",
       "     '1.7.1.15')],\n",
       "   'Function': 'Required for activity of the reductase'},\n",
       "  'seq': 'MSQWKDICKIDDILPETGVCALLGDEQVAIFRPYHSDQVFAISNIDPFFESSVLSRGLIAEHQGELWVASPLKKQRFRLSDGLCMEDEQFSVKHYEARVKDGVVQLRG',\n",
       "  'mw': 12284.0},\n",
       " 'b3367': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AC26',\n",
       "  'gene': 'b3367',\n",
       "  'locus': ['b3367', 'JW3330'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Catalyzes nitrite uptake and nitrite export across the cytoplasmic membrane. Is up to 10-fold more active than NarK or NarU in nitrite uptake for subsequent reduction in the cytoplasm by the NirB/NirD nitrite reductase'},\n",
       "  'seq': 'MFTDTINKCAANAARIARLSANNPLGFWVSSAMAGAYVGLGIILIFTLGNLLDPSVRPLVMGATFGIALTLVIIAGSELFTGHTMFLTFGVKAGSISHGQMWAILPQTWLGNLVGSVFVAMLYSWGGGSLLPVDTSIVHSVALAKTTAPAMVLFFKGALCNWLVCLAIWMALRTEGAAKFIAIWWCLLAFIASGYEHSIANMTLFALSWFGNHSEAYTLAGIGHNLLWVTLGNTLSGAVFMGLGYWYATPKANRPVADKFNQTETAAG',\n",
       "  'mw': 28563.0},\n",
       " 'b3368': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AEA8',\n",
       "  'gene': 'b3368',\n",
       "  'locus': ['b3368', 'JW3331'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('2 S-adenosyl-L-methionine + uroporphyrinogen III = H(+) + precorrin-2 + 2 S-adenosyl-L-homocysteine',\n",
       "     '2.1.1.107'),\n",
       "    ('NAD(+) + precorrin-2 = 2 H(+) + NADH + sirohydrochlorin', '1.3.1.76'),\n",
       "    ('2 H(+) + siroheme = Fe(2+) + sirohydrochlorin', '4.99.1.4')],\n",
       "   'Function': 'Multifunctional enzyme that catalyzes the SAM-dependent methylations of uroporphyrinogen III at position C-2 and C-7 to form precorrin-2 via precorrin-1. Then it catalyzes the NAD-dependent ring dehydrogenation of precorrin-2 to yield sirohydrochlorin. Finally, it catalyzes the ferrochelation of sirohydrochlorin to yield siroheme',\n",
       "   'Pathway': 'Porphyrin-containing compound metabolism; siroheme biosynthesis; sirohydrochlorin from precorrin-2: step 1/1'},\n",
       "  'seq': 'MDHLPIFCQLRDRDCLIVGGGDVAERKARLLLDAGARLTVNALAFIPQFTAWADAGMLTLVEGPFDESLLDTCWLAIAATDDDALNQRVSEAAEARRIFCNVVDAPKAASFIMPSIIDRSPLMVAVSSGGTSPVLARLLREKLESLLPLHLGQVAKYAGQLRGRVKQQFATMGERRRFWEKLFVNDRLAQSLANNDQKAITETTEQLINEPLDHRGEVVLVGAGPGDAGLLTLKGLQQIQQADVVVYDRLVSDDIMNLVRRDADRVFVGKRAGYHCVPQEEINQILLREAQKGKRVVRLKGGDPFIFGRGGEELETLCNAGIPFSVVPGITAASGCSAYSGIPLTHRDYAQSVRLITGHLKTGGELDWENLAAEKQTLVFYMGLNQAATIQQKLIEHGMPGEMPVAIVENGTAVTQRVIDGTLTQLGELAQQMNSPSLIIIGRVVGLRDKLNWFSNH',\n",
       "  'mw': 49951.0},\n",
       " 'b3370': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P45539',\n",
       "  'gene': 'b3370',\n",
       "  'locus': ['b3370', 'JW3333'],\n",
       "  'synonyms': 'yhfM',\n",
       "  'properties': {'Catalytic Activity': [('N(6)-(D-fructosyl)-L-lysine(in) = N(6)-(D-fructosyl)-L-lysine(out)',\n",
       "     ''),\n",
       "    ('N(6)-(D-psicosyl)-L-lysine(in) = N(6)-(D-psicosyl)-L-lysine(out)', '')],\n",
       "   'Function': 'Is likely involved in the transport of fructoselysine and psicoselysine to the cytoplasm, where they are degraded',\n",
       "   'Pathway': 'Carbohydrate metabolism; fructoselysine degradation'},\n",
       "  'seq': 'MGSQELQRKLGFWAVLAIAVGTTVGSGIFVSVGEVAKAAGTPWLTVLAFVIGGLIVIPQMCVYAELSTAYPENGADYVYLKNAGSRPLAFLSGWASFWANDAPSLSIMALAIVSNLGFLTPIDPLLGKFIAAGLIIAFMLLHLRSVEGGAAFQTLITIAKIIPFTIVIGLGIFWFKAENFAAPTTTAIGATGSFMALLAGISATSWSYTGMASICYMTGEIKNPGKTMPRALIGSCLLVLVLYTLLALVISGLMPFDKLANSETPISDALTWIPALGSTAGIFVAITAMIVILGSLSSCVMYQPRLEYAMAKDNLFFKCFGHVHPKYNTPDVSIILQGALGIFFIFVSDLTSLLGYFTLVMCFKNTLTFGSIIWCRKRDDYKPLWRTPAFGLMTTLAIASSLILVASTFVWAPIPGLICAVIVIATGLPAYAFWAKRSRQLNALS',\n",
       "  'mw': 47577.0},\n",
       " 'b3371': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AC00',\n",
       "  'gene': 'b3371',\n",
       "  'locus': ['b3371', 'JW5700'],\n",
       "  'synonyms': 'yhfN',\n",
       "  'properties': {'Catalytic Activity': [('H2O + N(6)-(6-phospho-D-fructosyl)-L-lysine = D-glucose 6-phosphate + L-lysine',\n",
       "     '')],\n",
       "   'Function': 'Catalyzes the reversible conversion of fructoselysine 6-phosphate to glucose 6-phosphate and lysine (PubMed:12147680). Functions in a fructoselysine degradation pathway that allows E.coli to grow on fructoselysine or psicoselysine (PubMed:14641112)',\n",
       "   'Regulation Activity': 'Strongly inhibited by ZnCl(2)',\n",
       "   'Pathway': 'Carbohydrate metabolism; fructoselysine degradation; D-glucose 6-phosphate and lysine from fructoselysine: step 2/2'},\n",
       "  'seq': 'MLDIDKSTVDFLVTENMVQEVEKVLSHDVPLVHAIVEEMVKRDIDRIYFVACGSPLNAAQTAKHLADRFSDLQVYAISGWEFCDNTPYRLDDRCAVIGVSDYGKTEEVIKALELGRACGALTAAFTKRADSPITSAAEFSIDYQADCIWEIHLLLCYSVVLEMITRLAPNAEIGKIKNDLKQLPNALGHLVRTWEEKGRQLGELASQWPMIYTVAAGPLRPLGYKEGIVTLMEFTWTHGCVIESGEFRHGPLEIVEPGVPFLFLLGNDESRHTTERAINFVKQRTDNVIVIDYAEISQGLHPWLAPFLMFVPMEWLCYYLSIYKDHNPDERRYYGGLVEY',\n",
       "  'mw': 38569.0},\n",
       " 'b3374': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P45543',\n",
       "  'gene': 'b3374',\n",
       "  'locus': ['b3374', 'JW3337'],\n",
       "  'synonyms': 'yhfQ',\n",
       "  'properties': {'Catalytic Activity': [('ATP + N(6)-(D-fructosyl)-L-lysine = ADP + H(+) + N(6)-(6-phospho-D-fructosyl)-L-lysine',\n",
       "     '2.7.1.218')],\n",
       "   'Function': 'Catalyzes the ATP-dependent phosphorylation of fructoselysine to fructoselysine 6-phosphate (PubMed:12147680). Functions in a fructoselysine degradation pathway that allows E.coli to grow on fructoselysine or psicoselysine (PubMed:12147680, PubMed:14641112). To a much lesser extenst, is also able to phosphorylate psicoselysine (PubMed:14641112)',\n",
       "   'Pathway': 'Carbohydrate metabolism; fructoselysine degradation; D-glucose 6-phosphate and lysine from fructoselysine: step 1/2'},\n",
       "  'seq': 'MKTLATIGDNCVDIYPQLNKAFSGGNAVNVAVYCTRYGIQPGCITWVGDDDYGTKLKQDLARMGVDISHVHTKHGVTAQTQVELHDNDRVFGDYTEGVMADFALSEEDYAWLAQYDIVHAAIWGHAEDAFPQLHAAGKLTAFDFSDKWDSPLWQTLVPHLDFAFASAPQEDETLRLKMKAIVARGAGTVIVTLGENGSIAWDGAQFWRQAPEPVTVIDTMGAGDSFIAGFLCGWSAGMTLPQAIAQGTACAAKTIQYHGAW',\n",
       "  'mw': 28332.0},\n",
       " 'b3380': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P45549',\n",
       "  'gene': 'b3380',\n",
       "  'locus': ['b3380', 'JW3343'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': []},\n",
       "  'seq': 'MARFVVLVIDSFGVGAMKDVTLVRPQDAGANTCGHILSQLPHLQLPTLETLGLINALGYAPGDMQPSDSATWGVAELQHEGGDTFMGHQEILGTRPLPPLRMPFRDVIDRVEQALVSAGWQVERRGDDLQFLWVNQAVAIGDNLEADLGQVYNITANLSVISFDDAIKIGRIVREQVQVGRVITFGGLLTDSQRILDAAESKEGRFIGINAPRSGAYDNGFQVVHMGYGVDEKVQVPQKLYEAGVPTVLVGKVADIVNNPYGVSWQNLVDSQRIMDITLNEFNTHPTAFICTNIQETDLAGHAEDVARYAERLQVVDRNLARLVEAMQPDDCLVVMADHGNDPTIGHSHHTREVVPVLVYQQGMIATQLGVRTTLSDVGATVCEFFRAPPPQNGRSFLSSLRFAGDTL',\n",
       "  'mw': 44560.0},\n",
       " 'b3384': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P00954',\n",
       "  'gene': 'b3384',\n",
       "  'locus': ['b3384', 'JW3347'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + L-tryptophan + tRNA(Trp) = AMP + diphosphate + H(+) + L-tryptophanyl-tRNA(Trp)',\n",
       "     '6.1.1.2')],\n",
       "   'Function': 'Catalyzes the attachment of tryptophan to tRNA(Trp). Amino acylates tRNA(Trp) with both L- and D-tryptophan, although D-tryptophan is a poor substrate (PubMed:10918062)'},\n",
       "  'seq': 'MTKPIVFSGAQPSGELTIGNYMGALRQWVNMQDDYHCIYCIVDQHAITVRQDAQKLRKATLDTLALYLACGIDPEKSTIFVQSHVPEHAQLGWALNCYTYFGELSRMTQFKDKSARYAENINAGLFDYPVLMAADILLYQTNLVPVGEDQKQHLELSRDIAQRFNALYGEIFKVPEPFIPKSGARVMSLLEPTKKMSKSDDNRNNVIGLLEDPKSVVKKIKRAVTDSDEPPVVRYDVQNKAGVSNLLDILSAVTGQSIPELEKQFEGKMYGHLKGEVADAVSGMLTELQERYHRFRNDEAFLQQVMKDGAEKASAHASRTLKAVYEAIGFVAKP',\n",
       "  'mw': 37438.0},\n",
       " 'b3385': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P32662',\n",
       "  'gene': 'b3385',\n",
       "  'locus': ['b3385', 'JW3348'],\n",
       "  'synonyms': 'yhfE',\n",
       "  'properties': {'Catalytic Activity': [('2-phosphoglycolate + H2O = glycolate + phosphate',\n",
       "     '3.1.3.18')],\n",
       "   'Function': \"Specifically catalyzes the dephosphorylation of 2-phosphoglycolate (2P-Gly). Is involved in the dissimilation of the intracellular 2-phosphoglycolate formed during the DNA repair of 3'-phosphoglycolate ends, a major class of DNA lesions induced by oxidative stress\",\n",
       "   'Pathway': 'Organic acid metabolism; glycolate biosynthesis; glycolate from 2-phosphoglycolate: step 1/1'},\n",
       "  'seq': 'MNKFEDIRGVAFDLDGTLVDSAPGLAAAVDMALYALELPVAGEERVITWIGNGADVLMERALTWARQERATQRKTMGKPPVDDDIPAEEQVRILRKLFDRYYGEVAEEGTFLFPHVADTLGALQAKGLPLGLVTNKPTPFVAPLLEALDIAKYFSVVIGGDDVQNKKPHPDPLLLVAERMGIAPQQMLFVGDSRNDIQAAKAAGCPSVGLTYGYNYGEAIDLSQPDVIYQSINDLLPALGLPHSENQESKND',\n",
       "  'mw': 27389.0},\n",
       " 'b3386': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AG07',\n",
       "  'gene': 'b3386',\n",
       "  'locus': ['b3386', 'JW3349'],\n",
       "  'synonyms': 'dod yhfD',\n",
       "  'properties': {'Catalytic Activity': [('D-ribulose 5-phosphate = D-xylulose 5-phosphate',\n",
       "     '5.1.3.1')],\n",
       "   'Function': 'Catalyzes the reversible epimerization of D-ribulose 5-phosphate to D-xylulose 5-phosphate',\n",
       "   'Pathway': 'Carbohydrate degradation'},\n",
       "  'seq': 'MKQYLIAPSILSADFARLGEDTAKALAAGADVVHFDVMDNHYVPNLTIGPMVLKSLRNYGITAPIDVHLMVKPVDRIVPDFAAAGASIITFHPEASEHVDRTLQLIKENGCKAGLVFNPATPLSYLDYVMDKLDVILLMSVNPGFGGQSFIPQTLDKLREVRRRIDESGFDIRLEVDGGVKVNNIGEIAAAGADMFVAGSAIFDQPDYKKVIDEMRSELAKVSHE',\n",
       "  'mw': 24554.0},\n",
       " 'b3389': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P07639',\n",
       "  'gene': 'b3389',\n",
       "  'locus': ['b3389', 'JW3352'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('7-phospho-2-dehydro-3-deoxy-D-arabino-heptonate = 3-dehydroquinate + phosphate',\n",
       "     '4.2.3.4')],\n",
       "   'Function': 'Catalyzes the conversion of 3-deoxy-D-arabino-heptulosonate 7-phosphate (DAHP) to dehydroquinate (DHQ)',\n",
       "   'Pathway': 'Metabolic intermediate biosynthesis; chorismate biosynthesis; chorismate from D-erythrose 4-phosphate and phosphoenolpyruvate: step 2/7'},\n",
       "  'seq': 'MERIVVTLGERSYPITIASGLFNEPASFLPLKSGEQVMLVTNETLAPLYLDKVRGVLEQAGVNVDSVILPDGEQYKSLAVLDTVFTALLQKPHGRDTTLVALGGGVVGDLTGFAAASYQRGVRFIQVPTTLLSQVDSSVGGKTAVNHPLGKNMIGAFYQPASVVVDLDCLKTLPPRELASGLAEVIKYGIILDGAFFNWLEENLDALLRLDGPAMAYCIRRCCELKAEVVAADERETGLRALLNLGHTFGHAIEAEMGYGNWLHGEAVAAGMVMAARTSERLGQFSSAETQRIITLLKRAGLPVNGPREMSAQAYLPHMLRDKKVLAGEMRLILPLAIGKSEVRSGVSHELVLNAIADCQSA',\n",
       "  'mw': 38881.0},\n",
       " 'b3390': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A6D7',\n",
       "  'gene': 'b3390',\n",
       "  'locus': ['b3390', 'JW5947'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + shikimate = 3-phosphoshikimate + ADP + H(+)',\n",
       "     '2.7.1.71')],\n",
       "   'Function': 'Catalyzes the specific phosphorylation of the 3-hydroxyl group of shikimic acid using ATP as a cosubstrate',\n",
       "   'Pathway': 'Metabolic intermediate biosynthesis; chorismate biosynthesis; chorismate from D-erythrose 4-phosphate and phosphoenolpyruvate: step 5/7'},\n",
       "  'seq': 'MAEKRNIFLVGPMGAGKSTIGRQLAQQLNMEFYDSDQEIEKRTGADVGWVFDLEGEEGFRDREEKVINELTEKQGIVLATGGGSVKSRETRNRLSARGVVVYLETTIEKQLARTQRDKKRPLLHVETPPREVLEALANERNPLYEEIADVTIRTDDQSAKVVANQIIHMLESN',\n",
       "  'mw': 19538.0},\n",
       " 'b3396': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P02918',\n",
       "  'gene': 'b3396',\n",
       "  'locus': ['b3396', 'JW3359'],\n",
       "  'synonyms': 'ponA',\n",
       "  'properties': {'Catalytic Activity': [('[GlcNAc-(1->4)-Mur2Ac(oyl-L-Ala-gamma-D-Glu-L-Lys-D-Ala-D-Ala)](n)-di-trans,octa-cis-undecaprenyl diphosphate + beta-D-GlcNAc-(1->4)-Mur2Ac(oyl-L-Ala-gamma-D-Glu-L-Lys-D-Ala-D-Ala)-di-trans,octa-cis-undecaprenyl diphosphate = [GlcNAc-(1->4)-Mur2Ac(oyl-L-Ala-gamma-D-Glu-L-Lys-D-Ala-D-Ala)](n+1)-di-trans-octa-cis-undecaprenyl diphosphate + di-trans,octa-cis-undecaprenyl diphosphate + H(+)',\n",
       "     '2.4.1.129'),\n",
       "    ('Preferential cleavage: (Ac)2-L-Lys-D-Ala-|-D-Ala. Also transpeptidation of peptidyl-alanyl moieties that are N-acyl substituents of D-alanine.',\n",
       "     '3.4.16.4')],\n",
       "   'Function': 'Cell wall formation. Synthesis of cross-linked peptidoglycan from the lipid intermediates. The enzyme has a penicillin-insensitive transglycosylase N-terminal domain (formation of linear glycan strands) and a penicillin-sensitive transpeptidase C-terminal domain (cross-linking of the peptide subunits)',\n",
       "   'Pathway': 'Cell wall biogenesis; peptidoglycan biosynthesis'},\n",
       "  'seq': 'MKFVKYFLILAVCCILLGAGSIYGLYRYIEPQLPDVATLKDVRLQIPMQIYSADGELIAQYGEKRRIPVTLDQIPPEMVKAFIATEDSRFYEHHGVDPVGIFRAASVALFSGHASQGASTITQQLARNFFLSPERTLMRKIKEVFLAIRIEQLLTKDEILELYLNKIYLGYRAYGVGAAAQVYFGKTVDQLTLNEMAVIAGLPKAPSTFNPLYSMDRAVARRNVVLSRMLDEGYITQQQFDQTRTEAINANYHAPEIAFSAPYLSEMVRQEMYNRYGESAYEDGYRIYTTITRKVQQAAQQAVRNNVLDYDMRHGYRGPANVLWKVGESAWDNNKITDTLKALPTYGPLLPAAVTSANPQQATAMLADGSTVALSMEGVRWARPYRSDTQQGPTPRKVTDVLQTGQQIWVRQVGDAWWLAQVPEVNSALVSINPQNGAVMALVGGFDFNQSKFNRATQALRQVGSNIKPFLYTAAMDKGLTLASMLNDVPISRWDASAGSDWQPKNSPPQYAGPIRLRQGLGQSKNVVMVRAMRAMGVDYAAEYLQRFGFPAQNIVHTESLALGSASFTPMQVARGYAVMANGGFLVDPWFISKIENDQGGVIFEAKPKVACPECDIPVIYGDTQKSNVLENNDVEDVAISREQQNVSVPMPQLEQANQALVAKTGAQEYAPHVINTPLAFLIKSALNTNIFGEPGWQGTGWRAGRDLQRRDIGGKTGTTNSSKDAWFSGYGPGVVTSVWIGFDDHRRNLGHTTASGAIKDQISGYEGGAKSAQPAWDAYMKAVLEGVPEQPLTPPPGIVTVNIDRSTGQLANGGNSREEYFIEGTQPTQQAVHEVGTTIIDNGEAQELF',\n",
       "  'mw': 93636.0},\n",
       " 'b3397': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P45799',\n",
       "  'gene': 'b3397',\n",
       "  'locus': ['b3397', 'JW3360'],\n",
       "  'synonyms': 'yrfE',\n",
       "  'properties': {'Catalytic Activity': [('ADP-D-ribose + H2O = AMP + D-ribose 5-phosphate + 2 H(+)',\n",
       "     '')],\n",
       "   'Function': \"Active on adenosine(5')triphospho(5')adenosine (Ap3A), ADP-ribose, NADH, adenosine(5')diphospho(5')adenosine (Ap2A)\"},\n",
       "  'seq': 'MSKSLQKPTILNVETVARSRLFTVESVDLEFSNGVRRVYERMRPTNREAVMIVPIVDDHLILIREYAVGTESYELGFSKGLIDPGESVYEAANRELKEEVGFGANDLTFLKKLSMAPSYFSSKMNIVVAQDLYPESLEGDEPEPLPQVRWPLAHMMDLLEDPDFNEARNVSALFLVREWLKGQGRV',\n",
       "  'mw': 21153.0},\n",
       " 'b3403': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P22259',\n",
       "  'gene': 'b3403',\n",
       "  'locus': ['b3403', 'JW3366'],\n",
       "  'synonyms': 'pck',\n",
       "  'properties': {'Catalytic Activity': [('ATP + oxaloacetate = ADP + CO2 + phosphoenolpyruvate',\n",
       "     '4.1.1.49')],\n",
       "   'Function': 'Involved in the gluconeogenesis. Catalyzes the conversion of oxaloacetate (OAA) to phosphoenolpyruvate (PEP) through direct phosphoryl transfer between the nucleoside triphosphate and OAA',\n",
       "   'Regulation Activity': 'Allosterically activated by calcium. It may represent the only case of a monomeric, allosteric enzyme',\n",
       "   'Pathway': 'Carbohydrate biosynthesis; gluconeogenesis'},\n",
       "  'seq': 'MRVNNGLTPQELEAYGISDVHDIVYNPSYDLLYQEELDPSLTGYERGVLTNLGAVAVDTGIFTGRSPKDKYIVRDDTTRDTFWWADKGKGKNDNKPLSPETWQHLKGLVTRQLSGKRLFVVDAFCGANPDTRLSVRFITEVAWQAHFVKNMFIRPSDEELAGFKPDFIVMNGAKCTNPQWKEQGLNSENFVAFNLTERMQLIGGTWYGGEMKKGMFSMMNYLLPLKGIASMHCSANVGEKGDVAVFFGLSGTGKTTLSTDPKRRLIGDDEHGWDDDGVFNFEGGCYAKTIKLSKEAEPEIYNAIRRDALLENVTVREDGTIDFDDGSKTENTRVSYPIYHIDNIVKPVSKAGHATKVIFLTADAFGVLPPVSRLTADQTQYHFLSGFTAKLAGTERGITEPTPTFSACFGAAFLSLHPTQYAEVLVKRMQAAGAQAYLVNTGWNGTGKRISIKDTRAIIDAILNGSLDNAETFTLPMFNLAIPTELPGVDTKILDPRNTYASPEQWQEKAETLAKLFIDNFDKYTDTPAGAALVAAGPKL',\n",
       "  'mw': 59643.0},\n",
       " 'b3409': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P33650',\n",
       "  'gene': 'b3409',\n",
       "  'locus': ['b3409', 'JW3372'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Transporter of a GTP-driven Fe(2+) uptake system, probably couples GTP-binding to channel opening and Fe(2+) uptake (PubMed:12446835, PubMed:19629046). A guanine nucleotide-binding protein (G proteins) in which the guanine nucleotide binding site alternates between an active, GTP-bound state and an inactive, GDP-bound state. This protein has fast intrinsic GDP release, mediated by the G5 loop (about residues 149-158). Presumably GTP hydrolysis leads to conformational changes and channel closing (PubMed:19629046). A GDP release mechanism involving a conformational change of the G5 loop (and thus the removal of the nucleotide-binding and stabilizing interactions) would significantly reduce the affinity for GDP, and conceivably be sufficient for catalysing nucleotide release (PubMed:25374115)'},\n",
       "  'seq': 'MKKLTIGLIGNPNSGKTTLFNQLTGSRQRVGNWAGVTVERKEGQFSTTDHQVTLVDLPGTYSLTTISSQTSLDEQIACHYILSGDADLLINVVDASNLERNLYLTLQLLELGIPCIVALNMLDIAEKQNIRIEIDALSARLGCPVIPLVSTRGRGIEALKLAIDRYKANENVELVHYAQPLLNEADSLAKVMPSDIPLKQRRWLGLQMLEGDIYSRAYAGEASQHLDAALARLRNEMDDPALHIADARYQCIAAICDVVSNTLTAEPSRFTTAVDKIVLNRFLGLPIFLFVMYLMFLLAINIGGALQPLFDVGSVALFVHGIQWIGYTLHFPDWLTIFLAQGLGGGINTVLPLVPQIGMMYLFLSFLEDSGYMARAAFVMDRLMQALGLPGKSFVPLIVGFGCNVPSVMGARTLDAPRERLMTIMMAPFMSCGARLAIFAVFAAAFFGQNGALAVFSLYMLGIVMAVLTGLMLKYTIMRGEATPFVMELPVYHVPHVKSLIIQTWQRLKGFVLRAGKVIIIVSIFLSAFNSFSLSGKIVDNINDSALASVSRVITPVFKPIGVHEDNWQATVGLFTGAMAKEVVVGTLNTLYTAENIQDEEFNPAEFNLGEELFSAIDETWQSLKDTFSLSVLMNPIEASKGDGEMGTGAMGVMDQKFGSAAAAYSYLIFVLLYVPCISVMGAIARESSRGWMGFSILWGLNIAYSLATLFYQVASYSQHPTYSLVCILAVILFNIVVIGLLRRARSRVDIELLATRKSVSSCCAASTTGDCH',\n",
       "  'mw': 84474.0},\n",
       " 'b3415': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P39835',\n",
       "  'gene': 'b3415',\n",
       "  'locus': ['b3415', 'JW5690'],\n",
       "  'synonyms': 'gntM usgA yhgC',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the gluconate utilization system Gnt-I; high-affinity intake of gluconate',\n",
       "   'Pathway': 'Carbohydrate acid metabolism; D-gluconate degradation'},\n",
       "  'seq': 'MPLVIVAIGVILLLLLMIRFKMNGFIALVLVALAVGLMQGMPLDKVIGSIKAGVGGTLGSLALIMGFGAMLGKMLADCGGAQRIATTLIAKFGKKHIQWAVVLTGFTVGFALFYEVGFVLMLPLVFTIAASANIPLLYVGVPMAAALSVTHGFLPPHPGPTAIATIFNADMGKTLLYGTILAIPTVILAGPVYARVLKGIDKPIPEGLYSAKTFSEEEMPSFGVSVWTSLVPVVLMAMRAIAEMILPKGHAFLPVAEFLGDPVMATLIAVLIAMFTFGLNRGRSMDQINDTLVSSIKIIAMMLLIIGGGGAFKQVLVDSGVDKYIASMMHETNISPLLMAWSIAAVLRIALGSATVAAITAGGIAAPLIATTGVSPELMVIAVGSGSVIFSHVNDPGFWLFKEYFNLTIGETIKSWSMLETIISVCGLVGCLLLNMVI',\n",
       "  'mw': 45967.0},\n",
       " 'b3416': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P15977',\n",
       "  'gene': 'b3416',\n",
       "  'locus': ['b3416', 'JW3379'],\n",
       "  'synonyms': 'malA',\n",
       "  'properties': {'Catalytic Activity': [('Transfers a segment of a (1->4)-alpha-D-glucan to a new position in an acceptor, which may be glucose or a (1->4)-alpha-D-glucan.',\n",
       "     '2.4.1.25')]},\n",
       "  'seq': 'MESKRLDNAALAAGISPNYINAHGKPQSISAETKRRLLDAMHQRTATKVAVTPVPNVMVYTSGKKMPMVVEGSGEYSWLLTTEEGTQYKGHVTGGKAFNLPTKLPEGYHTLTLTQDDQRAHCRVIVAPKRCYEPQALLNKQKLWGACVQLYTLRSEKNWGIGDFGDLKAMLVDVAKRGGSFIGLNPIHALYPANPESASPYSPSSRRWLNVIYIDVNAVEDFHLSEEAQAWWQLPTTQQTLQQARDADWVDYSTVTALKMTALRMAWKGFAQRDDEQMAAFRQFVAEQGDSLFWQAAFDALHAQQVKEDEMRWGWPAWPEMYQNVDSPEVRQFCEEHRDDVDFYLWLQWLAYSQFAACWEISQGYEMPIGLYRDLAVGVAEGGAETWCDRELYCLKASVGAPPDILGPLGQNWGLPPMDPHIITARAYEPFIELLRANMQNCGALRIDHVMSMLRLWWIPYGETADQGAYVHYPVDDLLSILALESKRHRCMVIGEDLGTVPVEIVGKLRSSGVYSYKVLYFENDHEKTFRAPKAYPEQSMAVAATHDLPTLRGYWECGDLTLGKTLGLYPDEVVLRGLYQDRELAKQGLLDALHKYGCLPKRAGHKASLMSMTPTLNRGLQRYIADSNSALLGLQPEDWLDMAEPVNIPGTSYQYKNWRRKLSATLESMFADDGVNKLLKDLDRRRRAAAKKK',\n",
       "  'mw': 78503.0},\n",
       " 'b3417': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P00490',\n",
       "  'gene': 'b3417',\n",
       "  'locus': ['b3417', 'JW5689'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('[(1->4)-alpha-D-glucosyl](n) + phosphate = [(1->4)-alpha-D-glucosyl](n-1) + alpha-D-glucose 1-phosphate',\n",
       "     '2.4.1.1')],\n",
       "   'Function': 'Phosphorylase is an important allosteric enzyme in carbohydrate metabolism. Enzymes from different sources differ in their regulatory mechanisms and in their natural substrates. However, all known phosphorylases share catalytic and structural properties'},\n",
       "  'seq': 'MSQPIFNDKQFQEALSRQWQRYGLNSAAEMTPRQWWLAVSEALAEMLRAQPFAKPVANQRHVNYISMEFLIGRLTGNNLLNLGWYQDVQDSLKAYDINLTDLLEEEIDPALGNGGLGRLAACFLDSMATVGQSATGYGLNYQYGLFRQSFVDGKQVEAPDDWHRSNYPWFRHNEALDVQVGIGGKVTKDGRWEPEFTITGQAWDLPVVGYRNGVAQPLRLWQATHAHPFDLTKFNDGDFLRAEQQGINAEKLTKVLYPNDNHTAGKKLRLMQQYFQCACSVADILRRHHLAGRKLHELADYEVIQLNDTHPTIAIPELLRVLIDEHQMSWDDAWAITSKTFAYTNHTLMPEALERWDVKLVKGLLPRHMQIINEINTRFKTLVEKTWPGDEKVWAKLAVVHDKQVHMANLCVVGGFAVNGVAALHSDLVVKDLFPEYHQLWPNKFHNVTNGITPRRWIKQCNPALAALLDKSLQKEWANDLDQLINLEKFADDAKFRQQYREIKQANKVRLAEFVKVRTGIEINPQAIFDIQIKRLHEYKRQHLNLLHILALYKEIRENPQADRVPRVFLFGAKAAPGYYLAKNIIFAINKVADVINNDPLVGDKLKVVFLPDYCVSAAEKLIPAADISEQISTAGKEASGTGNMKLALNGALTVGTLDGANVEIAEKVGEENIFIFGHTVEQVKAILAKGYDPVKWRKKDKVLDAVLKELESGKYSDGDKHAFDQMLHSIGKQGGDPYLVMADFAAYVEAQKQVDVLYRDQEAWTRAAILNTARCGMFSSDRSIRDYQARIWQAKR',\n",
       "  'mw': 90522.0},\n",
       " 'b3425': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A6V5',\n",
       "  'gene': 'b3425',\n",
       "  'locus': ['b3425', 'JW3388'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('hydrogen cyanide + thiosulfate = 2 H(+) + sulfite + thiocyanate',\n",
       "     '2.8.1.1')],\n",
       "   'Function': 'Catalyzes, although with low efficiency, the sulfur transfer reaction from thiosulfate to cyanide. The relatively low affinity of GlpE for both thiosulfate and cyanide suggests that these compounds are not the physiological substrates. Thioredoxin 1 or related dithiol proteins could instead be the physiological sulfur-acceptor substrate. Possible association with the metabolism of glycerol-phosphate remains to be elucidated'},\n",
       "  'seq': 'MDQFECINVADAHQKLQEKEAVLVDIRDPQSFAMGHAVQAFHLTNDTLGAFMRDNDFDTPVMVMCYHGNSSKGAAQYLLQQGYDVVYSIDGGFEAWQRQFPAEVAYGA',\n",
       "  'mw': 12082.0},\n",
       " 'b3426': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P13035',\n",
       "  'gene': 'b3426',\n",
       "  'locus': ['b3426', 'JW3389'],\n",
       "  'synonyms': 'glyD',\n",
       "  'properties': {'Catalytic Activity': [('a quinone + sn-glycerol 3-phosphate = a quinol + dihydroxyacetone phosphate',\n",
       "     '1.1.5.3')],\n",
       "   'Function': 'Conversion of glycerol 3-phosphate to dihydroxyacetone. Uses molecular oxygen or nitrate as electron acceptor',\n",
       "   'Pathway': 'Polyol metabolism; glycerol degradation via glycerol kinase pathway; glycerone phosphate from sn-glycerol 3-phosphate (aerobic route): step 1/1'},\n",
       "  'seq': 'METKDLIVIGGGINGAGIAADAAGRGLSVLMLEAQDLACATSSASSKLIHGGLRYLEHYEFRLVSEALAEREVLLKMAPHIAFPMRFRLPHRPHLRPAWMIRIGLFMYDHLGKRTSLPGSTGLRFGANSVLKPEIKRGFEYSDCWVDDARLVLANAQMVVRKGGEVLTRTRATSARRENGLWIVEAEDIDTGKKYSWQARGLVNATGPWVKQFFDDGMHLPSPYGIRLIKGSHIVVPRVHTQKQAYILQNEDKRIVFVIPWMDEFSIIGTTDVEYKGDPKAVKIEESEINYLLNVYNTHFKKQLSRDDIVWTYSGVRPLCDDESDSPQAITRDYTLDIHDENGKAPLLSVFGGKLTTYRKLAEHALEKLTPYYQGIGPAWTKESVLPGGAIEGDRDDYAARLRRRYPFLTESLARHYARTYGSNSELLLGNAGTVSDLGEDFGHEFYEAELKYLVDHEWVRRADDALWRRTKQGMWLNADQQSRVSQWLVEYTQQRLSLAS',\n",
       "  'mw': 56751.0},\n",
       " 'b3428': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AC86',\n",
       "  'gene': 'b3428',\n",
       "  'locus': ['b3428', 'JW3391'],\n",
       "  'synonyms': 'glgY',\n",
       "  'properties': {'Catalytic Activity': [('[(1->4)-alpha-D-glucosyl](n) + phosphate = [(1->4)-alpha-D-glucosyl](n-1) + alpha-D-glucose 1-phosphate',\n",
       "     '2.4.1.1')],\n",
       "   'Function': 'Phosphorylase is an important allosteric enzyme in carbohydrate metabolism. Enzymes from different sources differ in their regulatory mechanisms and in their natural substrates. However, all known phosphorylases share catalytic and structural properties'},\n",
       "  'seq': 'MNAPFTYSSPTLSVEALKHSIAYKLMFTIGKDPVVANKHEWLNATLFAVRDRLVERWLRSNRAQLSQETRQVYYLSMEFLIGRTLSNAMLSLGIYEDVQGALEAMGLNLEELIDEENDPGLGNGGLGRLAACFLDSLATLGLPGRGYGIRYDYGMFKQNIVNGSQKESPDYWLEYGNPWEFKRHNTRYKVRFGGRIQQEGKKTRWIETEEILGVAYDQIIPGYDTDATNTLRLWSAQASSEINLGKFNQGDYFAAVEDKNHSENVSRVLYPDDSTYSGRELRLRQEYFLVSSTIQDILSRHYQLHKTYDNLADKIAIHLNDTHPVLSIPEMMRLLIDEHQFSWDDAFEVCCQVFSYTNHTLMSEALETWPVDMLGKILPRHLQIIFEINDYFLKTLQEQYPNDTDLLGRASIIDESNGRRVRMAWLAVVVSHKVNGVSELHSNLMVQSLFADFAKIFPGRFTNVTNGVTPRRWLAVANPSLSAVLDEHLGRNWRTDLSLLNELQQHCDFPMVNHAVHQAKLENKKRLAEYIAQQLNVVVNPKALFDVQIKRIHEYKRQLMNVLHVITRYNRIKADPDAKWVPRVNIFGGKAASAYYMAKHIIHLINDVAKVINNDPQIGDKLKVVFIPNYSVSLAQLIIPAADLSEQISLAGTEASGTSNMKFALNGALTIGTLDGANVEMLDHVGADNIFIFGNTAEEVEELRRQGYKPREYYEKDEELHQVLTQIGSGVFSPEDPGRYRDLVDSLINFGDHYQVLADYRSYVDCQDKVDELYELQEEWTAKAMLNIANMGYFSSDRTIKEYADHIWHIDPVRL',\n",
       "  'mw': 93173.0},\n",
       " 'b3429': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A6U8',\n",
       "  'gene': 'b3429',\n",
       "  'locus': ['b3429', 'JW3392'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('[(1->4)-alpha-D-glucosyl](n) + ADP-alpha-D-glucose = [(1->4)-alpha-D-glucosyl](n+1) + ADP + H(+)',\n",
       "     '2.4.1.21')],\n",
       "   'Function': 'Synthesizes alpha-1,4-glucan chains using ADP-glucose',\n",
       "   'Pathway': 'Glycan biosynthesis; glycogen biosynthesis'},\n",
       "  'seq': 'MQVLHVCSEMFPLLKTGGLADVIGALPAAQIADGVDARVLLPAFPDIRRGVTDAQVVSRRDTFAGHITLLFGHYNGVGIYLIDAPHLYDRPGSPYHDTNLFAYTDNVLRFALLGWVGAEMASGLDPFWRPDVVHAHDWHAGLAPAYLAARGRPAKSVFTVHNLAYQGMFYAHHMNDIQLPWSFFNIHGLEFNGQISFLKAGLYYADHITAVSPTYAREITEPQFAYGMEGLLQQRHREGRLSGVLNGVDEKIWSPETDLLLASRYTRDTLEDKAENKRQLQIAMGLKVDDKVPLFAVVSRLTSQKGLDLVLEALPGLLEQGGQLALLGAGDPVLQEGFLAAAAEYPGQVGVQIGYHEAFSHRIMGGADVILVPSRFEPCGLTQLYGLKYGTLPLVRRTGGLADTVSDCSLENLADGVASGFVFEDSNAWSLLRAIRRAFVLWSRPSLWRFVQRQAMAMDFSWQVAAKSYRELYYRLK',\n",
       "  'mw': 52822.0},\n",
       " 'b3430': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A6V1',\n",
       "  'gene': 'b3430',\n",
       "  'locus': ['b3430', 'JW3393'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('alpha-D-glucose 1-phosphate + ATP + H(+) = ADP-alpha-D-glucose + diphosphate',\n",
       "     '2.7.7.27')],\n",
       "   'Function': 'Involved in the biosynthesis of ADP-glucose, a building block required for the elongation reactions to produce glycogen. Catalyzes the reaction between ATP and alpha-D-glucose 1-phosphate (G1P) to produce pyrophosphate and ADP-Glc',\n",
       "   'Regulation Activity': 'Allosterically activated by fructose-1,6-bisphosphate (F16BP), hexanediol 1,6-bisphosphate, NADPH and pyridoxal phosphate. Inhibited by AMP and by high concentrations of MgCl(2)',\n",
       "   'Pathway': 'Glycan biosynthesis; glycogen biosynthesis'},\n",
       "  'seq': 'MVSLEKNDHLMLARQLPLKSVALILAGGRGTRLKDLTNKRAKPAVHFGGKFRIIDFALSNCINSGIRRMGVITQYQSHTLVQHIQRGWSFFNEEMNEFVDLLPAQQRMKGENWYRGTADAVTQNLDIIRRYKAEYVVILAGDHIYKQDYSRMLIDHVEKGARCTVACMPVPIEEASAFGVMAVDENDKIIEFVEKPANPPSMPNDPSKSLASMGIYVFDADYLYELLEEDDRDENSSHDFGKDLIPKITEAGLAYAHPFPLSCVQSDPDAEPYWRDVGTLEAYWKANLDLASVVPELDMYDRNWPIRTYNESLPPAKFVQDRSGSHGMTLNSLVSGGCVISGSVVVQSVLFSRVRVNSFCNIDSAVLLPEVWVGRSCRLRRCVIDRACVIPEGMVIGENAEEDARRFYRSEEGIVLVTREMLRKLGHKQER',\n",
       "  'mw': 48698.0},\n",
       " 'b3431': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P15067',\n",
       "  'gene': 'b3431',\n",
       "  'locus': ['b3431', 'JW3394'],\n",
       "  'synonyms': 'glyX',\n",
       "  'properties': {'Catalytic Activity': [('Hydrolysis of (1->6)-alpha-D-glucosidic linkages to branches with degrees of polymerization of three or four glucose residues in limit dextrin.',\n",
       "     '3.2.1.196')],\n",
       "   'Function': 'Removes maltotriose and maltotetraose chains that are attached by 1,6-alpha-linkage to the limit dextrin main chain, generating a debranched limit dextrin. Shows only very little activity with native glycogen',\n",
       "   'Regulation Activity': 'Inhibited by iodoacetate, p-chloromercuribenzoate, HgCl(2), cupric sulfate and ammonium sulfate',\n",
       "   'Pathway': 'Glycan degradation; glycogen degradation'},\n",
       "  'seq': 'MTQLAIGKPAPLGAHYDGQGVNFTLFSAHAERVELCVFDANGQEHRYDLPGHSGDIWHGYLPDARPGLRYGYRVHGPWQPAEGHRFNPAKLLIDPCARQIDGEFKDNPLLHAGHNEPDYRDNAAIAPKCVVVVDHYDWEDDAPPRTPWGSTIIYEAHVKGLTYLHPEIPVEIRGTYKALGHPVMINYLKQLGITALELLPVAQFASEPRLQRMGLSNYWGYNPVAMFALHPAYACSPETALDEFRDAIKALHKAGIEVILDIVLNHSAELDLDGPLFSLRGIDNRSYYWIREDGDYHNWTGCGNTLNLSHPAVVDYASACLRYWVETCHVDGFRFDLAAVMGRTPEFRQDAPLFTAIQNCPVLSQVKLIAEPWDIAPGGYQVGNFPPLFAEWNDHFRDAARRFWLHYDLPLGAFAGRFAASSDVFKRNGRLPSAAINLVTAHDGFTLRDCVCFNHKHNEANGEENRDGTNNNYSNNHGKEGLGGSLDLVERRRDSIHALLTTLLLSQGTPMLLAGDEHGHSQHGNNNAYCQDNQLTWLDWSQASSGLTAFTAALIHLRKRIPALVENRWWEEGDGNVRWLNRYAQPLSTDEWQNGPKQLQILLSDRFLIAINATLEVTEIVLPAGEWHAIPPFAGEDNPVITAVWQGPAHGLCVFQR',\n",
       "  'mw': 73577.0},\n",
       " 'b3432': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P07762',\n",
       "  'gene': 'b3432',\n",
       "  'locus': ['b3432', 'JW3395'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('Transfers a segment of a (1->4)-alpha-D-glucan chain to a primary hydroxy group in a similar glucan chain.',\n",
       "     '2.4.1.18')],\n",
       "   'Function': 'Catalyzes the formation of the alpha-1,6-glucosidic linkages in glycogen by scission of a 1,4-alpha-linked oligosaccharide from growing alpha-1,4-glucan chains and the subsequent attachment of the oligosaccharide to the alpha-1,6 position',\n",
       "   'Pathway': 'Glycan biosynthesis; glycogen biosynthesis'},\n",
       "  'seq': 'MSDRIDRDVINALIAGHFADPFSVLGMHKTTAGLEVRALLPDATDVWVIEPKTGRKLAKLECLDSRGFFSGVIPRRKNFFRYQLAVVWHGQQNLIDDPYRFGPLIQEMDAWLLSEGTHLRPYETLGAHADTMDGVTGTRFSVWAPNARRVSVVGQFNYWDGRRHPMRLRKESGIWELFIPGAHNGQLYKYEMIDANGNLRLKSDPYAFEAQMRPETASLICGLPEKVVQTEERKKANQFDAPISIYEVHLGSWRRHTDNNFWLSYRELADQLVPYAKWMGFTHLELLPINEHPFDGSWGYQPTGLYAPTRRFGTRDDFRYFIDAAHAAGLNVILDWVPGHFPTDDFALAEFDGTNLYEHSDPREGYHQDWNTLIYNYGRREVSNFLVGNALYWIERFGIDALRVDAVASMIYRDYSRKEGEWIPNEFGGRENLEAIEFLRNTNRILGEQVSGAVTMAEESTDFPGVSRPQDMGGLGFWYKWNLGWMHDTLDYMKLDPVYRQYHHDKLTFGILYNYTENFVLPLSHDEVVHGKKSILDRMPGDAWQKFANLRAYYGWMWAFPGKKLLFMGNEFAQGREWNHDASLDWHLLEGGDNWHHGVQRLVRDLNLTYRHHKAMHELDFDPYGFEWLVVDDKERSVLIFVRRDKEGNEIIVASNFTPVPRHDYRFGINQPGKWREILNTDSMHYHGSNAGNGGTVHSDEIASHGRQHSLSLTLPPLATIWLVREAE',\n",
       "  'mw': 84337.0},\n",
       " 'b3433': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A9Q9',\n",
       "  'gene': 'b3433',\n",
       "  'locus': ['b3433', 'JW3396'],\n",
       "  'synonyms': 'hom',\n",
       "  'properties': {'Catalytic Activity': [('L-aspartate 4-semialdehyde + NADP(+) + phosphate = 4-phospho-L-aspartate + H(+) + NADPH',\n",
       "     '1.2.1.11')],\n",
       "   'Function': 'Catalyzes the NADPH-dependent formation of L-aspartate-semialdehyde (L-ASA) by the reductive dephosphorylation of L-aspartyl-4-phosphate',\n",
       "   'Regulation Activity': 'Is inhibited by L- and D-cystine, and by other cystine derivatives, via the formation of a covalently bound cysteine at the active site Cys-135',\n",
       "   'Pathway': 'Amino-acid biosynthesis; L-threonine biosynthesis; L-threonine from L-aspartate: step 2/5'},\n",
       "  'seq': 'MKNVGFIGWRGMVGSVLMQRMVEERDFDAIRPVFFSTSQLGQAAPSFGGTTGTLQDAFDLEALKALDIIVTCQGGDYTNEIYPKLRESGWQGYWIDAASSLRMKDDAIIILDPVNQDVITDGLNNGIRTFVGGNCTVSLMLMSLGGLFANDLVDWVSVATYQAASGGGARHMRELLTQMGHLYGHVADELATPSSAILDIERKVTTLTRSGELPVDNFGVPLAGSLIPWIDKQLDNGQSREEWKGQAETNKILNTSSVIPVDGLCVRVGALRCHSQAFTIKLKKDVSIPTVEELLAAHNPWAKVVPNDREITMRELTPAAVTGTLTTPVGRLRKLNMGPEFLSAFTVGDQLLWGAAEPLRRMLRQLA',\n",
       "  'mw': 40018.0},\n",
       " 'b3437': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P46859',\n",
       "  'gene': 'b3437',\n",
       "  'locus': ['b3437', 'JW3400'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + D-gluconate = 6-phospho-D-gluconate + ADP + H(+)',\n",
       "     '2.7.1.12')],\n",
       "   'Pathway': 'Carbohydrate acid metabolism; D-gluconate degradation'},\n",
       "  'seq': 'MSTTNHDHHIYVLMGVSGSGKSAVASEVAHQLHAAFLDGDFLHPRRNIEKMASGEPLNDDDRKPWLQALNDAAFAMQRTNKVSLIVCSALKKHYRDLLREGNPNLSFIYLKGDFDVIESRLKARKGHFFKTQMLVTQFETLQEPGADETDVLVVDIDQPLEGVVASTIEVIKKGK',\n",
       "  'mw': 19543.0},\n",
       " 'b3447': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P18956',\n",
       "  'gene': 'b3447',\n",
       "  'locus': ['b3447', 'JW3412'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('an alpha-amino acid + an N-terminal (5-L-glutamyl)-[peptide] = 5-L-glutamyl amino acid + N-terminal L-alpha-aminoacyl-[peptide]',\n",
       "     '2.3.2.2'),\n",
       "    ('glutathione + H2O = L-cysteinylglycine + L-glutamate', '3.4.19.13'),\n",
       "    ('an S-substituted glutathione + H2O = an S-substituted L-cysteinylglycine + L-glutamate',\n",
       "     '3.4.19.13')],\n",
       "   'Function': 'Cleaves the gamma-glutamyl bond of periplasmic glutathione (gamma-Glu-Cys-Gly), glutathione conjugates, and other gamma-glutamyl compounds. The metabolism of glutathione releases free glutamate and the dipeptide cysteinyl-glycine, which is hydrolyzed to cysteine and glycine by dipeptidases; it may function in amino acid uptake/salvage, or possibly in peptidoglycan linkage. Catalyzes the hydrolysis and transpeptidation of many gamma-glutamyl compounds (including some D-gamma-glutamyl substrates), with a preference for basic and aromatic amino acids as acceptors (PubMed:2877974). The KM values for gamma-glutamyl acceptors are so high that it has been proposed transpeptidation is not the physiological role in E.coli (PubMed:2877974, PubMed:8104180)',\n",
       "   'Regulation Activity': 'Transferase and hydrolase activities are inhibited by L-Ala and L-Gln, and also by GGT affinity labeling reagents such as azaserine and 6-diazo-5-oxo-nor-leucine',\n",
       "   'Pathway': 'Sulfur metabolism; glutathione metabolism'},\n",
       "  'seq': 'MIKPTFLRRVAIAALLSGSCFSAAAAPPAPPVSYGVEEDVFHPVRAKQGMVASVDATATQVGVDILKEGGNAVDAAVAVGYALAVTHPQAGNLGGGGFMLIRSKNGNTTAIDFREMAPAKATRDMFLDDQGNPDSKKSLTSHLASGTPGTVAGFSLALDKYGTMPLNKVVQPAFKLARDGFIVNDALADDLKTYGSEVLPNHENSKAIFWKEGEPLKKGDTLVQANLAKSLEMIAENGPDEFYKGTIAEQIAQEMQKNGGLITKEDLAAYKAVERTPISGDYRGYQVYSMPPPSSGGIHIVQILNILENFDMKKYGFGSADAMQIMAEAEKYAYADRSEYLGDPDFVKVPWQALTNKAYAKSIADQIDINKAKPSSEIRPGKLAPYESNQTTHYSVVDKDGNAVAVTYTLNTTFGTGIVAGESGILLNNQMDDFSAKPGVPNVYGLVGGDANAVGPNKRPLSSMSPTIVVKDGKTWLVTGSPGGSRIITTVLQMVVNSIDYGLNVAEATNAPRFHHQWLPDELRVEKGFSPDTLKLLEAKGQKVALKEAMGSTQSIMVGPDGELYGASDPRSVDDLTAGY',\n",
       "  'mw': 61768.0},\n",
       " 'b3449': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P10908',\n",
       "  'gene': 'b3449',\n",
       "  'locus': ['b3449', 'JW3414'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('a sn-glycero-3-phosphodiester + H2O = an alcohol + H(+) + sn-glycerol 3-phosphate',\n",
       "     '3.1.4.46')],\n",
       "   'Function': 'Glycerophosphoryl diester phosphodiesterase hydrolyzes deacylated phospholipids to G3P and the corresponding alcohols'},\n",
       "  'seq': 'MSNWPYPRIVAHRGGGKLAPENTLASIDVGAKYGHKMIEFDAKLSKDGEIFLLHDDNLERTSNGWGVAGELNWQDLLRVDAGSWYSKMFKGEPLPLLSQVAERCREHGMMANIEIKPTTGTGPLTGKMVALAARELWAGMTPPLLSSFEIDALEAAQQAAPELPRGLLLDEWRDDWRELTARLGCVSIHLNHKLLNKARVMQLKDAGLRILVYTVNKPQRAAELLRWGVDCICTDAIDVIGPNFTAQ',\n",
       "  'mw': 27410.0},\n",
       " 'b3450': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P10907',\n",
       "  'gene': 'b3450',\n",
       "  'locus': ['b3450', 'JW3415'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + H2O + sn-glycerol 3-phosphate(out) = ADP + H(+) + phosphate + sn-glycerol 3-phosphate(in)',\n",
       "     '7.6.2.10'),\n",
       "    ('ATP + glycerol 2-phosphate(out) + H2O = ADP + glycerol 2-phosphate(in) + H(+) + phosphate',\n",
       "     '')],\n",
       "   'Function': 'Part of the ABC transporter complex UgpBAEC involved in sn-glycerol-3-phosphate (G3P) import (PubMed:363686, PubMed:7042685, PubMed:2842304, PubMed:8282692, PubMed:23013274). Responsible for energy coupling to the transport system (Probable). Can also transport glycerophosphoryl diesters, which are hydrolyzed to G3P and alcohol during transport (PubMed:2842304). The G3P moiety can be detected in the cytoplasm whereas the corresponding alcohol is usually found in the culture medium (PubMed:2842304). It was proposed by Yang et al that the complex could also transport glycerol-2-phosphate (G2P) in vivo, but it was shown later by Wuttge et al that UgpB does not bind G2P, questioning this transport activity (PubMed:19429609, PubMed:23013274). G2P might be converted in the periplasm to G3P before its transport (PubMed:23013274)',\n",
       "   'Regulation Activity': 'ATPase activity is stimulated when UgpB is bound to G3P (PubMed:23013274). Transport is inhibited in vivo by increasing levels of internal phosphate (PubMed:8282692, PubMed:7836304). However, ATPase activity in proteoliposomes is neither inhibited by phosphate nor by the signal transducing protein PhoU or the phosphodiesterase UgpQ (PubMed:23013274). Activated by gluconate and inhibited by fumarate (PubMed:363686)'},\n",
       "  'seq': 'MAGLKLQAVTKSWDGKTQVIKPLTLDVADGEFIVMVGPSGCGKSTLLRMVAGLERVTEGDIWINDQRVTEMEPKDRGIAMVFQNYALYPHMSVEENMAWGLKIRGMGKQQIAERVKEAARILELDGLLKRRPRELSGGQRQRVAMGRAIVRDPAVFLFDEPLSNLDAKLRVQMRLELQQLHRRLKTTSLYVTHDQVEAMTLAQRVMVMNGGVAEQIGTPVEVYEKPASLFVASFIGSPAMNLLTGRVNNEGTHFELDGGIELPLNGGYRQYAGRKMTLGIRPEHIALSSQAEGGVPMVMDTLEILGADNLAHGRWGEQKLVVRLAHQERPTAGSTLWLHLAENQLHLFDGETGQRV',\n",
       "  'mw': 39524.0},\n",
       " 'b3451': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P10906',\n",
       "  'gene': 'b3451',\n",
       "  'locus': ['b3451', 'JW3416'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the ABC transporter complex UgpBAEC involved in sn-glycerol-3-phosphate (G3P) import (PubMed:2842304, PubMed:23013274). Probably responsible for the translocation of the substrate across the membrane (Probable). Can also transport glycerophosphoryl diesters, which are hydrolyzed to G3P and alcohol during transport (PubMed:2842304). The G3P moiety can be detected in the cytoplasm whereas the corresponding alcohol is usually found in the culture medium (PubMed:2842304). It was proposed by Yang et al that the complex could also transport glycerol-2-phosphate (G2P) in vivo, but it was shown later by Wuttge et al that UgpB does not bind G2P, questioning this transport activity (PubMed:19429609, PubMed:23013274). G2P might be converted in the periplasm to G3P before its transport (PubMed:23013274)'},\n",
       "  'seq': 'MIENRPWLTIFSHTMLILGIAVILFPLYVAFVAATLDKQAVYAAPMTLIPGTHLLENIHNIWVNGVGTNSAPFWRMLLNSFVMAFSITLGKITVSMLSAFAIVWFRFPLRNLFFWMIFITLMLPVEVRIFPTVEVIANLQMLDSYAGLTLPLMASATATFLFRQFFMTLPDELVEAARIDGASPMRFFCDIVFPLSKTNLAALFVITFIYGWNQYLWPLLIITDVDLGTTVAGIKGMIATGEGTTEWNSVMVAMLLTLIPPVVIVLVMQRAFVRGLVDSEK',\n",
       "  'mw': 31500.0},\n",
       " 'b3452': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P10905',\n",
       "  'gene': 'b3452',\n",
       "  'locus': ['b3452', 'JW3417'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the ABC transporter complex UgpBAEC involved in sn-glycerol-3-phosphate (G3P) import (PubMed:2842304, PubMed:23013274). Probably responsible for the translocation of the substrate across the membrane (Probable). Can also transport glycerophosphoryl diesters, which are hydrolyzed to G3P and alcohol during transport (PubMed:2842304). The G3P moiety can be detected in the cytoplasm whereas the corresponding alcohol is usually found in the culture medium (PubMed:2842304). It was proposed by Yang et al that the complex could also transport glycerol-2-phosphate (G2P) in vivo, but it was shown later by Wuttge et al that UgpB does not bind G2P, questioning this transport activity (PubMed:19429609, PubMed:23013274). G2P might be converted in the periplasm to G3P before its transport (PubMed:23013274)'},\n",
       "  'seq': 'MSSSRPVFRSRWLPYLLVAPQLIITVIFFIWPAGEALWYSLQSVDPFGFSSQFVGLDNFVTLFHDSYYLDSFWTTIKFSTFVTVSGLLVSLFFAALVEYIVRGSRFYQTLMLLPYAVAPAVAAVLWIFLFNPGRGLITHFLAEFGYDWNHAQNSGQAMFLVVFASVWKQISYNFLFFYAALQSIPRSLIEAAAIDGAGPIRRFFKIALPLIAPVSFFLLVVNLVYAFFDTFPVIDAATSGGPVQATTTLIYKIYREGFTGLDLASSAAQSVVLMFLVIVLTVVQFRYVESKVRYQ',\n",
       "  'mw': 33264.0},\n",
       " 'b3453': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AG80',\n",
       "  'gene': 'b3453',\n",
       "  'locus': ['b3453', 'JW3418'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': \"In the absence of G3P, UgpB can serve as a potent molecular chaperone that exhibits anti-aggregation activity against bile salt-induced protein aggregation (PubMed:32882062). The substrate G3P, which is known to accumulate in the later compartments of the digestive system, triggers a functional switch between UgpB's activity as a molecular chaperone and its activity as a G3P transporter (PubMed:32882062). Chaperone and G3P binding activities are likely mutually exclusive activities (PubMed:32882062)\",\n",
       "   'Regulation Activity': 'The chaperone activity is activated by the transition from pH 2 to 7 and inhibited by the addition of G3P'},\n",
       "  'seq': 'MKPLHYTASALALGLALMGNAQAVTTIPFWHSMEGELGKEVDSLAQRFNAENPDYKIVPTYKGNYEQNLSAGIAAFRTGNAPAILQVYEVGTATMMASKAIKPVYDVFKEAGIQFDESQFVPTVSGYYSDSKTGHLLSQPFNSSTPVLYYNKDAFKKAGLDPEQPPKTWQDLADYAAKLKASGMKCGYASGWQGWIQLENFSAWNGLPFASKNNGFDGTDAVLEFNKPEQVKHIAMLEEMNKKGDFSYVGRKDESTEKFYNGDCAMTTASSGSLANIREYAKFNYGVGMMPYDADAKDAPQNAIIGGASLWVMQGKDKETYTGVAKFLDFLAKPENAAEWHQKTGYLPITKAAYDLTREQGFYEKNPGADTATRQMLNKPPLPFTKGLRLGNMPQIRVIVDEELESVWTGKKTPQQALDTAVERGNQLLRRFEKSTKS',\n",
       "  'mw': 48449.0},\n",
       " 'b3454': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P22731',\n",
       "  'gene': 'b3454',\n",
       "  'locus': ['b3454', 'JW3419'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Component of the leucine-specific transport system'},\n",
       "  'seq': 'MEKVMLSFDKVSAHYGKIQALHEVSLHINQGEIVTLIGANGAGKTTLLGTLCGDPRATSGRIVFDDKDITDWQTAKIMREAVAIVPEGRRVFSRMTVEENLAMGGFFAERDQFQERIKWVYELFPRLHERRIQRAGTMSGGEQQMLAIGRALMSNPRLLLLDEPSLGLAPIIIQQIFDTIEQLREQGMTIFLVEQNANQALKLADRGYVLENGHVVLSDTGDALLANEAVRSAYLGG',\n",
       "  'mw': 26310.0},\n",
       " 'b3455': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A9S7',\n",
       "  'gene': 'b3455',\n",
       "  'locus': ['b3455', 'JW3420'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Component of the leucine-specific transport system'},\n",
       "  'seq': 'MSQPLLSVNGLMMRFGGLLAVNNVNLELYPQEIVSLIGPNGAGKTTVFNCLTGFYKPTGGTILLRDQHLEGLPGQQIARMGVVRTFQHVRLFREMTVIENLLVAQHQQLKTGLFSGLLKTPSFRRAQSEALDRAATWLERIGLLEHANRQASNLAYGDQRRLEIARCMVTQPEILMLDEPAAGLNPKETKELDELIAELRNHHNTTILLIEHDMKLVMGISDRIYVVNQGTPLANGTPEQIRNNPDVIRAYLGEA',\n",
       "  'mw': 28427.0},\n",
       " 'b3456': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P22729',\n",
       "  'gene': 'b3456',\n",
       "  'locus': ['b3456', 'JW3421'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the binding-protein-dependent transport system for branched-chain amino acids. Probably responsible for the translocation of the substrates across the membrane'},\n",
       "  'seq': 'MKPMHIAMALLSAAMFFVLAGVFMGVQLELDGTKLVVDTASDVRWQWVFIGTAVVFFFQLLRPAFQKGLKSVSGPKFILPAIDGSTVKQKLFLVALLVLAVAWPFMVSRGTVDIATLTMIYIILGLGLNVVVGLSGLLVLGYGGFYAIGAYTFALLNHYYGLGFWTCLPIAGLMAAAAGFLLGFPVLRLRGDYLAIVTLGFGEIVRILLLNNTEITGGPNGISQIPKPTLFGLEFSRTAREGGWDTFSNFFGLKYDPSDRVIFLYLVALLLVVLSLFVINRLLRMPLGRAWEALREDEIACRSLGLSPRRIKLTAFTISAAFAGFAGTLFAARQGFVSPESFTFAESAFVLAIVVLGGMGSQFAVILAAILLVVSRELMRDFNEYSMLMLGGLMVLMMIWRPQGLLPMTRPQLKLKNGAAKGEQA',\n",
       "  'mw': 46269.0},\n",
       " 'b3457': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AEX7',\n",
       "  'gene': 'b3457',\n",
       "  'locus': ['b3457', 'JW3422'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the binding-protein-dependent transport system for branched-chain amino acids. Probably responsible for the translocation of the substrates across the membrane'},\n",
       "  'seq': 'MSEQFLYFLQQMFNGVTLGSTYALIAIGYTMVYGIIGMINFAHGEVYMIGSYVSFMIIAALMMMGIDTGWLLVAAGFVGAIVIASAYGWSIERVAYRPVRNSKRLIALISAIGMSIFLQNYVSLTEGSRDVALPSLFNGQWVVGHSENFSASITTMQAVIWIVTFLAMLALTIFIRYSRMGRACRACAEDLKMASLLGINTDRVIALTFVIGAAMAAVAGVLLGQFYGVINPYIGFMAGMKAFTAAVLGGIGSIPGAMIGGLILGIAEALSSAYLSTEYKDVVSFALLILVLLVMPTGILGRPEVEKV',\n",
       "  'mw': 32982.0},\n",
       " 'b3458': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P04816',\n",
       "  'gene': 'b3458',\n",
       "  'locus': ['b3458', 'JW3423'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'This protein is a component of the leucine-specific transport system, which is one of the two periplasmic binding protein-dependent transport systems of the high-affinity transport of the branched-chain amino acids in E.coli'},\n",
       "  'seq': 'MKRNAKTIIAGMIALAISHTAMADDIKVAVVGAMSGPIAQWGDMEFNGARQAIKDINAKGGIKGDKLVGVEYDDACDPKQAVAVANKIVNDGIKYVIGHLCSSSTQPASDIYEDEGILMISPGATNPELTQRGYQHIMRTAGLDSSQGPTAAKYILETVKPQRIAIIHDKQQYGEGLARSVQDGLKAANANVVFFDGITAGEKDFSALIARLKKENIDFVYYGGYYPEMGQMLRQARSVGLKTQFMGPEGVGNASLSNIAGDAAEGMLVTMPKRYDQDPANQGIVDALKADKKDPSGPYVWITYAAVQSLATALERTGSDEPLALVKDLKANGANTVIGPLNWDEKGDLKGFDFGVFQWHADGSSTAAK',\n",
       "  'mw': 39379.0},\n",
       " 'b3460': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AD96',\n",
       "  'gene': 'b3460',\n",
       "  'locus': ['b3460', 'JW3425'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'This protein is a component of the leucine, isoleucine, valine, (threonine) transport system, which is one of the two periplasmic binding protein-dependent transport systems of the high-affinity transport of the branched-chain amino acids'},\n",
       "  'seq': 'MNIKGKALLAGCIALAFSNMALAEDIKVAVVGAMSGPVAQYGDQEFTGAEQAVADINAKGGIKGNKLQIVKYDDACDPKQAVAVANKVVNDGIKYVIGHLCSSSTQPASDIYEDEGILMITPAATAPELTARGYQLILRTTGLDSDQGPTAAKYILEKVKPQRIAIVHDKQQYGEGLARAVQDGLKKGNANVVFFDGITAGEKDFSTLVARLKKENIDFVYYGGYHPEMGQILRQARAAGLKTQFMGPEGVANVSLSNIAGESAEGLLVTKPKNYDQVPANKPIVDAIKAKKQDPSGAFVWTTYAALQSLQAGLNQSDDPAEIAKYLKANSVDTVMGPLTWDEKGDLKGFEFGVFDWHANGTATDAK',\n",
       "  'mw': 39076.0},\n",
       " 'b3469': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37617',\n",
       "  'gene': 'b3469',\n",
       "  'locus': ['b3469', 'JW3434'],\n",
       "  'synonyms': 'yhhO',\n",
       "  'properties': {'Catalytic Activity': [('ATP + H2O + Pb(2+)(in) = ADP + H(+) + Pb(2+)(out) + phosphate',\n",
       "     ''),\n",
       "    ('ATP + H2O + Zn(2+)(in) = ADP + H(+) + phosphate + Zn(2+)(out)',\n",
       "     '7.2.2.12'),\n",
       "    ('ATP + Cd(2+)(in) + H2O = ADP + Cd(2+)(out) + H(+) + phosphate',\n",
       "     '7.2.2.21')],\n",
       "   'Function': 'Confers resistance to zinc, cadmium and lead (PubMed:9405611, PubMed:9364914, PubMed:9830000, PubMed:10660539, PubMed:17326661). Couples the hydrolysis of ATP with the export of zinc, cadmium or lead, with highest activity when the metals are present as metal-thiolate complexes (PubMed:9405611, PubMed:10660539, PubMed:17326661). Can also bind nickel, copper, cobalt and mercury (PubMed:10660539, PubMed:17326661)',\n",
       "   'Regulation Activity': 'Inhibited by orthovanadate'},\n",
       "  'seq': 'MSTPDNHGKKAPQFAAFKPLTTVQNANDCCCDGACSSTPTLSENVSGTRYSWKVSGMDCAACARKVENAVRQLAGVNQVQVLFATEKLVVDADNDIRAQVESALQKAGYSLRDEQAAEEPQASRLKENLPLITLIVMMAISWGLEQFNHPFGQLAFIATTLVGLYPIARQALRLIKSGSYFAIETLMSVAAIGALFIGATAEAAMVLLLFLIGERLEGWAASRARQGVSALMALKPETATRLRKGEREEVAINSLRPGDVIEVAAGGRLPADGKLLSPFASFDESALTGESIPVERATGDKVPAGATSVDRLVTLEVLSEPGASAIDRILKLIEEAEERRAPIERFIDRFSRIYTPAIMAVALLVTLVPPLLFAASWQEWIYKGLTLLLIGCPCALVISTPAAITSGLAAAARRGALIKGGAALEQLGRVTQVAFDKTGTLTVGKPRVTAIHPATGISESELLTLAAAVEQGATHPLAQAIVREAQVAELAIPTAESQRALVGSGIEAQVNGERVLICAAGKHPADAFTGLINELESAGQTVVLVVRNDDVLGVIALQDTLRADAATAISELNALGVKGVILTGDNPRAAAAIAGELGLEFKAGLLPEDKVKAVTELNQHAPLAMVGDGINDAPAMKAAAIGIAMGSGTDVALETADAALTHNHLRGLVQMIELARATHANIRQNITIALGLKGIFLVTTLLGMTGLWLAVLADTGATVLVTANALRLLRRR',\n",
       "  'mw': 76840.0},\n",
       " 'b3475': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37623',\n",
       "  'gene': 'b3475',\n",
       "  'locus': ['b3475', 'JW3440'],\n",
       "  'synonyms': 'yhhU',\n",
       "  'properties': {'Catalytic Activity': [(\"apo-[ACP] + CoA = adenosine 3',5'-bisphosphate + H(+) + holo-[ACP]\",\n",
       "     '2.7.8.7')],\n",
       "   'Function': 'May be involved in an alternative pathway for phosphopantetheinyl transfer and holo-ACP synthesis in E.coli. The native apo-protein substrate is unknown. Is able to functionally replace AcpS in vivo but only when expressed at high levels'},\n",
       "  'seq': 'MYRIVLGKVSTLSAAPLPPGLREQAPQGPRRERWLAGRALLSHTLSPLPEIIYGEQGKPAFAPEMPLWFNLSHSGDDIALLLSDEGEVGCDIEVIRPRANWRWLANAVFSLGEHAEMDAVHPDQQLEMFWRIWTRKEAIVKQRGGSAWQIVSVDSTYHSSLSVSHCQLENLSLAICTPTPFTLTADSVQWIDSVN',\n",
       "  'mw': 21768.0},\n",
       " 'b3476': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P33590',\n",
       "  'gene': 'b3476',\n",
       "  'locus': ['b3476', 'JW3441'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Involved in a nickel transport system, probably represents the nickel binder'},\n",
       "  'seq': 'MLSTLRRTLFALLACASFIVHAAAPDEITTAWPVNVGPLNPHLYTPNQMFAQSMVYEPLVKYQADGSVIPWLAKSWTHSEDGKTWTFTLRDDVKFSNGEPFDAEAAAENFRAVLDNRQRHAWLELANQIVDVKALSKTELQITLKSAYYPFLQELALPRPFRFIAPSQFKNHETMNGIKAPIGTGPWILQESKLNQYDVFVRNENYWGEKPAIKKITFNVIPDPTTRAVAFETGDIDLLYGNEGLLPLDTFARFSQNPAYHTQLSQPIETVMLALNTAKAPTNELAVREALNYAVNKKSLIDNALYGTQQVADTLFAPSVPYANLGLKPSQYDPQKAKALLEKAGWTLPAGKDIREKNGQPLRIELSFIGTDALSKSMAEIIQADMRQIGADVSLIGEEESSIYARQRDGRFGMIFHRTWGAPYDPHAFLSSMRVPSHADFQAQQGLADKPLIDKEIGEVLATHDETQRQALYRDILTRLHDEAVYLPISYISMMVVSKPELGNIPYAPIATEIPFEQIKPVKP',\n",
       "  'mw': 58719.0},\n",
       " 'b3477': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P33591',\n",
       "  'gene': 'b3477',\n",
       "  'locus': ['b3477', 'JW3442'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Involved in a nickel transport system, probably translocates nickel through the bacterial inner membrane'},\n",
       "  'seq': 'MLRYVLRRFLLLIPMVLAASVIIFLMLRLGTGDPALDYLRLSNLPPTPEMLASTRTMLGLDQPLYVQYGTWLWKALHLDFGISFASQRPVLDDMLNFLPATLELAGAALVLILLTSVPLGIWAARHRDRLPDFAVRFIAFLGVSMPNFWLAFLLVMAFSVYLQWLPAMGYGGWQHIILPAVSIAFMSLAINARLLRASMLDVAGQRHVTWARLRGLNDKQTERRHILRNASLPMITAVGMHIGELIGGTMIIENIFAWPGVGRYAVSAIFNRDYPVIQCFTLMMVVVFVVCNLIVDLLNAALDPRIRRHEGAHA',\n",
       "  'mw': 35248.0},\n",
       " 'b3478': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AFA9',\n",
       "  'gene': 'b3478',\n",
       "  'locus': ['b3478', 'JW3443'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Involved in a nickel transport system, probably translocates nickel through the bacterial inner membrane'},\n",
       "  'seq': 'MNFFLSSRWSVRLALIIIALLALIALTSQWWLPYDPQAIDLPSRLLSPDAQHWLGTDHLGRDIFSRLMAATRVSLGSVMACLLLVLTLGLVIGGSAGLIGGRVDQATMRVADMFMTFPTSILSFFMVGVLGTGLTNVIIAIALSHWAWYARMVRSLVISLRQREFVLASRLSGAGHVRVFVDHLAGAVIPSLLVLATLDIGHMMLHVAGMSFLGLGVTAPTAEWGVMINDARQYIWTQPLQMFWPGLALFISVMAFNLVGDALRDHLDPHLVTEHAH',\n",
       "  'mw': 30362.0},\n",
       " 'b3479': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P33593',\n",
       "  'gene': 'b3479',\n",
       "  'locus': ['b3479', 'JW3444'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + H2O + Ni(2+)(out) = ADP + H(+) + Ni(2+)(in) + phosphate',\n",
       "     '7.2.2.11')],\n",
       "   'Function': 'Part of the ABC transporter complex NikABCDE involved in nickel import. Responsible for energy coupling to the transport system'},\n",
       "  'seq': 'MPQQIELRNIALQAAQPLVHGVSLTLQRGRVLALVGGSGSGKSLTCAATLGILPAGVRQTAGEILADGKPVSPCALRGIKIATIMQNPRSAFNPLHTMHTHARETCLALGKPADDATLTAAIEAVGLENAARVLKLYPFEMSGGMLQRMMIAMAVLCESPFIIADEPTTDLDVVAQARILDLLESIMQKQAPGMLLVTHDMGVVARLADDVAVMSDGKIVEQGDVETLFNAPKHTVTRSLVSAHLALYGMELAS',\n",
       "  'mw': 26820.0},\n",
       " 'b3480': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P33594',\n",
       "  'gene': 'b3480',\n",
       "  'locus': ['b3480', 'JW3445'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + H2O + Ni(2+)(out) = ADP + H(+) + Ni(2+)(in) + phosphate',\n",
       "     '7.2.2.11')],\n",
       "   'Function': 'Part of the ABC transporter complex NikABCDE involved in nickel import. Responsible for energy coupling to the transport system'},\n",
       "  'seq': 'MTLLNISGLSHHYAHGGFNGKHQHQAVLNNVSLTLKSGETVALLGRSGCGKSTLARLLVGLESPAQGNISWRGEPLAKLNRAQRKAFRRDIQMVFQDSISAVNPRKTVREILREPMRHLLSLKKSEQLARASEMLKAVDLDDSVLDKRPPQLSGGQLQRVCLARALAVEPKLLILDEAVSNLDLVLQAGVIRLLKKLQQQFGTACLFITHDLRLVERFCQRVMVMDNGQIVETQVVGEKLTFSSDAGRVLQNAVLPAFPVRRRTTEKV',\n",
       "  'mw': 29722.0},\n",
       " 'b3493': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AFJ7',\n",
       "  'gene': 'b3493',\n",
       "  'locus': ['b3493', 'JW3460'],\n",
       "  'synonyms': 'pit',\n",
       "  'properties': {'Catalytic Activity': [('H(+)(in) + phosphate(in) = H(+)(out) + phosphate(out)',\n",
       "     ''),\n",
       "    ('arsenate(in) + H(+)(in) = arsenate(out) + H(+)(out)', '')],\n",
       "   'Function': 'Low-affinity inorganic phosphate transporter (PubMed:328484, PubMed:6998957, PubMed:8110778, PubMed:11489853). Mediates proton-driven uptake of soluble neutral metal phosphate (MeHP04) complexes (PubMed:8110778). It can use Mg(2+), Ca(2+), Co(2+) and Mn(2+) (PubMed:8110778). Activity impacts bacterial growth in low Mg(2+) conditions (PubMed:30276893). Is also involved in Zn(2+) uptake, probably via formation of a ZnHPO4 complex (PubMed:10713426). Can also transport arsenate (PubMed:328484). Involved in the uptake of tellurite (PubMed:23189244)',\n",
       "   'Regulation Activity': 'Phosphate uptake is dependent on the presence of divalent cations, which form soluble metal phosphate (MeHP04) complexes (PubMed:8110778, PubMed:10713426). In Mg(2+)-limiting conditions, the PhoQP-regulated small protein MgtS and sRNA MgrR both regulate the PitA transporter, leading to increased intracellular Mg(2+) (PubMed:30276893). Metal phosphate uptake is inhibited at low internal pH (PubMed:8110778). Inhibited by the uncoupler carbonyl cyanide m-chlorophenylhydrazone (CCCP) (PubMed:328484). Is also inhibited by the energy uncoupler 2,4-dinitrophenol and the sulfhydryl reagent N-ethylmaleimide (PubMed:6998957)'},\n",
       "  'seq': 'MLHLFAGLDLHTGLLLLLALAFVLFYEAINGFHDTANAVATVIYTRAMRSQLAVVMAAVFNFLGVLLGGLSVAYAIVHMLPTDLLLNMGSSHGLAMVFSMLLAAIIWNLGTWYFGLPASSSHTLIGAIIGIGLTNALMTGTSVVDALNIPKVLSIFGSLIVSPIVGLVFAGGLIFLLRRYWSGTKKRARIHLTPAEREKKDGKKKPPFWTRIALILSAIGVAFSHGANDGQKGIGLVMLVLIGVAPAGFVVNMNATGYEITRTRDAINNVEAYFEQHPALLKQATGADQLVPAPEAGATQPAEFHCHPSNTINALNRLKGMLTTDVESYDKLSLDQRSQMRRIMLCVSDTIDKVVKMPGVSADDQRLLKKLKSDMLSTIEYAPVWIIMAVALALGIGTMIGWRRVATTIGEKIGKKGMTYAQGMSAQMTAAVSIGLASYTGMPVSTTHVLSSSVAGTMVVDGGGLQRKTVTSILMAWVFTLPAAVLLSGGLYWLSLQFL',\n",
       "  'mw': 53389.0},\n",
       " 'b3500': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P06715',\n",
       "  'gene': 'b3500',\n",
       "  'locus': ['b3500', 'JW3467'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('2 glutathione + NADP(+) = glutathione disulfide + H(+) + NADPH',\n",
       "     '1.8.1.7')],\n",
       "   'Function': 'Maintains high levels of reduced glutathione in the cytosol'},\n",
       "  'seq': 'MTKHYDYIAIGGGSGGIASINRAAMYGQKCALIEAKELGGTCVNVGCVPKKVMWHAAQIREAIHMYGPDYGFDTTINKFNWETLIASRTAYIDRIHTSYENVLGKNNVDVIKGFARFVDAKTLEVNGETITADHILIATGGRPSHPDIPGVEYGIDSDGFFALPALPERVAVVGAGYIAVELAGVINGLGAKTHLFVRKHAPLRSFDPMISETLVEVMNAEGPQLHTNAIPKAVVKNTDGSLTLELEDGRSETVDCLIWAIGREPANDNINLEAAGVKTNEKGYIVVDKYQNTNIEGIYAVGDNTGAVELTPVAVAAGRRLSERLFNNKPDEHLDYSNIPTVVFSHPPIGTVGLTEPQAREQYGDDQVKVYKSSFTAMYTAVTTHRQPCRMKLVCVGSEEKIVGIHGIGFGMDEMLQGFAVALKMGATKKDFDNTVAIHPTAAEEFVTMR',\n",
       "  'mw': 48773.0},\n",
       " 'b3502': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AB93',\n",
       "  'gene': 'b3502',\n",
       "  'locus': ['b3502', 'JW3469'],\n",
       "  'synonyms': 'arsF',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Involved in arsenical resistance. Thought to form the channel of an arsenite pump (By similarity)'},\n",
       "  'seq': 'MLLAGAIFVLTIVLVIWQPKGLGIGWSATLGAVLALVTGVVHPGDIPVVWNIVWNATAAFIAVIIISLLLDESGFFEWAALHVSRWGNGRGRLLFTWIVLLGAAVAALFANDGAALILTPIVIAMLLALGFSKGTTLAFVMAAGFIADTASLPLIVSNLVNIVSADFFGLGFREYASVMVPVDIAAIVATLVMLHLYFRKDIPQNYDMALLKSPAEAIKDPATFKTGWVVLLLLLVGFFVLEPLGIPVSAIAAVGALILFVVAKRGHAINTGKVLRGAPWQIVIFSLGMYLVVYGLRNAGLTEYLSGVLNVLADNGLWAATLGTGFLTAFLSSIMNNMPTVLVGALSIDGSTASGVIKEAMVYANVIGCDLGPKITPIGSLATLLWLHVLSQKNMTISWGYYFRTGIIMTLPVLFVTLAALALRLSFTL',\n",
       "  'mw': 45497.0},\n",
       " 'b3503': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AB96',\n",
       "  'gene': 'b3503',\n",
       "  'locus': ['b3503', 'JW3470'],\n",
       "  'synonyms': 'arsG',\n",
       "  'properties': {'Catalytic Activity': [('[glutaredoxin]-dithiol + arsenate + glutathione + H(+) = arsenite + glutathionyl-S-S-[glutaredoxin] + H2O',\n",
       "     '1.20.4.1')],\n",
       "   'Function': 'Involved in resistance to arsenate. Catalyzes the reduction of arsenate [As(V)] to arsenite [As(III)]'},\n",
       "  'seq': 'MSNITIYHNPACGTSRNTLEMIRNSGTEPTIIHYLETPPTRDELVKLIADMGISVRALLRKNVEPYEELGLAEDKFTDDRLIDFMLQHPILINRPIVVTPLGTRLCRPSEVVLEILPDAQKGAFSKEDGEKVVDEAGKRLK',\n",
       "  'mw': 15853.0},\n",
       " 'b3517': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P69908',\n",
       "  'gene': 'b3517',\n",
       "  'locus': ['b3517', 'JW3485'],\n",
       "  'synonyms': 'gadS',\n",
       "  'properties': {'Catalytic Activity': [('H(+) + L-glutamate = 4-aminobutanoate + CO2',\n",
       "     '4.1.1.15')],\n",
       "   'Function': 'Converts glutamate to gamma-aminobutyrate (GABA), consuming one intracellular proton in the reaction. The gad system helps to maintain a near-neutral intracellular pH when cells are exposed to extremely acidic conditions. The ability to survive transit through the acidic conditions of the stomach is essential for successful colonization of the mammalian host by commensal and pathogenic bacteria'},\n",
       "  'seq': 'MDQKLLTDFRSELLDSRFGAKAISTIAESKRFPLHEMRDDVAFQIINDELYLDGNARQNLATFCQTWDDENVHKLMDLSINKNWIDKEEYPQSAAIDLRCVNMVADLWHAPAPKNGQAVGTNTIGSSEACMLGGMAMKWRWRKRMEAAGKPTDKPNLVCGPVQICWHKFARYWDVELREIPMRPGQLFMDPKRMIEACDENTIGVVPTFGVTYTGNYEFPQPLHDALDKFQADTGIDIDMHIDAASGGFLAPFVAPDIVWDFRLPRVKSISASGHKFGLAPLGCGWVIWRDEEALPQELVFNVDYLGGQIGTFAINFSRPAGQVIAQYYEFLRLGREGYTKVQNASYQVAAYLADEIAKLGPYEFICTGRPDEGIPAVCFKLKDGEDPGYTLYDLSERLRLRGWQVPAFTLGGEATDIVVMRIMCRRGFEMDFAELLLEDYKASLKYLSDHPKLQGIAQQNSFKHT',\n",
       "  'mw': 52685.0},\n",
       " 'b3519': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P62601',\n",
       "  'gene': 'b3519',\n",
       "  'locus': ['b3519', 'JW3487'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('alpha,alpha-trehalose + H2O = alpha-D-glucose + beta-D-glucose',\n",
       "     '3.2.1.28')],\n",
       "   'Function': 'Hydrolyzes trehalose to glucose. Could be involved, in cells returning to low osmolarity conditions, in the utilization of the accumulated cytoplasmic trehalose, which was synthesized in response to high osmolarity',\n",
       "   'Regulation Activity': 'Activity decreases with increasing salt concentrations',\n",
       "   'Pathway': 'Glycan degradation; trehalose degradation; D-glucose from alpha,alpha-trehalose: step 1/1'},\n",
       "  'seq': 'MLNQKIQNPNPDELMIEVDLCYELDPYELKLDEMIEAEPEPEMIEGLPASDALTPADRYLELFEHVQSAKIFPDSKTFPDCAPKMDPLDILIRYRKVRRHRDFDLRKFVENHFWLPEVYSSEYVSDPQNSLKEHIDQLWPVLTREPQDHIPWSSLLALPQSYIVPGGRFSETYYWDSYFTMLGLAESGREDLLKCMADNFAWMIENYGHIPNGNRTYYLSRSQPPVFALMVELFEEDGVRGARRYLDHLKMEYAFWMDGAESLIPNQAYRHVVRMPDGSLLNRYWDDRDTPRDESWLEDVETAKHSGRPPNEVYRDLRAGAASGWDYSSRWLRDTGRLASIRTTQFIPIDLNAFLFKLESAIANISALKGEKETEALFRQKASARRDAVNRYLWDDENGIYRDYDWRREQLALFSAAAIVPLYVGMANHEQADRLANAVRSRLLTPGGILASEYETGEQWDKPNGWAPLQWMAIQGFKMYGDDLLGDEIARSWLKTVNQFYLEQHKLIEKYHIADGVPREGGGGEYPLQDGFGWTNGVVRRLIGLYGEP',\n",
       "  'mw': 63697.0},\n",
       " 'b3526': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37647',\n",
       "  'gene': 'b3526',\n",
       "  'locus': ['b3526', 'JW5668'],\n",
       "  'synonyms': 'yhjI',\n",
       "  'properties': {'Catalytic Activity': [('2-dehydro-3-deoxy-D-gluconate + ATP = 2-dehydro-3-deoxy-6-phospho-D-gluconate + ADP + H(+)',\n",
       "     '2.7.1.45')],\n",
       "   'Function': 'Catalyzes the phosphorylation of 2-keto-3-deoxygluconate (KDG) to produce 2-keto-3-deoxy-6-phosphogluconate (KDPG)',\n",
       "   'Pathway': 'Carbohydrate acid metabolism; 2-dehydro-3-deoxy-D-gluconate degradation; D-glyceraldehyde 3-phosphate and pyruvate from 2-dehydro-3-deoxy-D-gluconate: step 1/2'},\n",
       "  'seq': 'MSKKIAVIGECMIELSEKGADVKRGFGGDTLNTSVYIARQVDPAALTVHYVTALGTDSFSQQMLDAWHGENVDTSLTQRMENRLPGLYYIETDSTGERTFYYWRNEAAAKFWLESEQSAAICEELANFDYLYLSGISLAILSPTSREKLLSLLRECRANGGKVIFDNNYRPRLWASKEETQQVYQQMLECTDIAFLTLDDEDALWGQQPVEDVIARTHNAGVKEVVVKRGADSCLVSIAGEGLVDVPAVKLPKEKVIDTTAAGDSFSAGYLAVRLTGGSAEDAAKRGHLTASTVIQYRGAIIPREAMPA',\n",
       "  'mw': 33962.0},\n",
       " 'b3528': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A830',\n",
       "  'gene': 'b3528',\n",
       "  'locus': ['b3528', 'JW3496'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Responsible for the aerobic transport of the dicarboxylates fumarate, L- and D-malate and to a lesser extent succinate, from the periplasm across the inner membrane'},\n",
       "  'seq': 'MKTSLFKSLYFQVLTAIAIGILLGHFYPEIGEQMKPLGDGFVKLIKMIIAPVIFCTVVTGIAGMESMKAVGRTGAVALLYFEIVSTIALIIGLIIVNVVQPGAGMNVDPATLDAKAVAVYADQAKDQGIVAFIMDVIPASVIGAFASGNILQVLLFAVLFGFALHRLGSKGQLIFNVIESFSQVIFGIINMIMRLAPIGAFGAMAFTIGKYGVGTLVQLGQLIICFYITCILFVVLVLGSIAKATGFSIFKFIRYIREELLIVLGTSSSESALPRMLDKMEKLGCRKSVVGLVIPTGYSFNLDGTSIYLTMAAVFIAQATNSQMDIVHQITLLIVLLLSSKGAAGVTGSGFIVLAATLSAVGHLPVAGLALILGIDRFMSEARALTNLVGNGVATIVVAKWVKELDHKKLDDVLNNRAPDGKTHELSS',\n",
       "  'mw': 45436.0},\n",
       " 'b3540': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37313',\n",
       "  'gene': 'b3540',\n",
       "  'locus': ['b3540', 'JW3509'],\n",
       "  'synonyms': 'dppE',\n",
       "  'properties': {'Catalytic Activity': [('a dipeptide(out) + ATP + H2O = a dipeptide(in) + ADP + H(+) + phosphate',\n",
       "     '7.4.2.9')],\n",
       "   'Function': 'When a foreign outer membrane heme receptor is expressed in E.coli, DppABCDF can also transport heme and its precursor, 5-aminolevulinic acid (ALA), from the periplasm into the cytoplasm'},\n",
       "  'seq': 'MSTQEATLQQPLLQAIDLKKHYPVKKGMFAPERLVKALDGVSFNLERGKTLAVVGESGCGKSTLGRLLTMIEMPTGGELYYQGQDLLKHDPQAQKLRRQKIQIVFQNPYGSLNPRKKVGQILEEPLLINTSLSKEQRREKALSMMAKVGLKTEHYDRYPHMFSGGQRQRIAIARGLMLDPDVVIADEPVSALDVSVRAQVLNLMMDLQQELGLSYVFISHDLSVVEHIADEVMVMYLGRCVEKGTKDQIFNNPRHPYTQALLSATPRLNPDDRRERIKLSGELPSPLNPPPGCAFNARCRRRFGPCTQLQPQLKDYGGQLVACFAVDQDENPQR',\n",
       "  'mw': 37560.0},\n",
       " 'b3541': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AAG0',\n",
       "  'gene': 'b3541',\n",
       "  'locus': ['b3541', 'JW3510'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('a dipeptide(out) + ATP + H2O = a dipeptide(in) + ADP + H(+) + phosphate',\n",
       "     '7.4.2.9')],\n",
       "   'Function': 'When a foreign outer membrane heme receptor is expressed in E.coli, DppABCDF can also transport heme and its precursor, 5-aminolevulinic acid (ALA), from the periplasm into the cytoplasm'},\n",
       "  'seq': 'MALLNVDKLSVHFGDESAPFRAVDRISYSVKQGEVVGIVGESGSGKSVSSLAIMGLIDYPGRVMAEKLEFNGQDLQRISEKERRNLVGAEVAMIFQDPMTSLNPCYTVGFQIMEAIKVHQGGNKSTRRQRAIDLLNQVGIPDPASRLDVYPHQLSGGMSQRVMIAMAIACRPKLLIADEPTTALDVTIQAQIIELLLELQQKENMALVLITHDLALVAEAAHKIIVMYAGQVVETGDAHAIFHAPRHPYTQALLRALPEFAQDKERLASLPGVVPGKYDRPNGCLLNPRCPYATDRCRAEEPALNMLADGRQSKCHYPLDDAGRPTL',\n",
       "  'mw': 35844.0},\n",
       " 'b3542': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AEG1',\n",
       "  'gene': 'b3542',\n",
       "  'locus': ['b3542', 'JW3511'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'When a foreign outer membrane heme receptor is expressed in E.coli, DppABCDF can also transport heme and its precursor, 5-aminolevulinic acid (ALA), from the periplasm into the cytoplasm'},\n",
       "  'seq': 'MSQVTENKVISAPVPMTPLQEFWHYFKRNKGAVVGLVYVVIVLFIAIFANWIAPYNPAEQFRDALLAPPAWQEGGSMAHLLGTDDVGRDVLSRLMYGARLSLLVGCLVVVLSLIMGVILGLIAGYFGGLVDNIIMRVVDIMLALPSLLLALVLVAIFGPSIGNAALALTFVALPHYVRLTRAAVLVEVNRDYVTASRVAGAGAMRQMFINIFPNCLAPLIVQASLGFSNAILDMAALGFLGMGAQPPTPEWGTMLSDVLQFAQSAWWVVTFPGLAILLTVLAFNLMGDGLRDALDPKLKQ',\n",
       "  'mw': 32308.0},\n",
       " 'b3543': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AEF8',\n",
       "  'gene': 'b3543',\n",
       "  'locus': ['b3543', 'JW3512'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'When a foreign outer membrane heme receptor is expressed in E.coli, DppABCDF can also transport heme and its precursor, 5-aminolevulinic acid (ALA), from the periplasm into the cytoplasm'},\n",
       "  'seq': 'MLQFILRRLGLVIPTFIGITLLTFAFVHMIPGDPVMIMAGERGISPERHAQLLAELGLDKPMWQQYLHYIWGVMHGDLGISMKSRIPVWEEFVPRFQATLELGVCAMIFATAVGIPVGVLAAVKRGSIFDHTAVGLALTGYSMPIFWWGMMLIMLVSVHWNLTPVSGRVSDMVFLDDSNPLTGFMLIDTAIWGEDGNFIDAVAHMILPAIVLGTIPLAVIVRMTRSSMLEVLGEDYIRTARAKGLTRMRVIIVHALRNAMLPVVTVIGLQVGTLLAGAILTETIFSWPGLGRWLIDALQRRDYPVVQGGVLLVATMIILVNLLVDLLYGVVNPRIRHKK',\n",
       "  'mw': 37497.0},\n",
       " 'b3544': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P23847',\n",
       "  'gene': 'b3544',\n",
       "  'locus': ['b3544', 'JW3513'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Binds heme. When a foreign outer membrane heme receptor is expressed in E.coli, DppABCDF can also transport heme and its precursor, 5-aminolevulinic acid (ALA), from the periplasm into the cytoplasm',\n",
       "   'Regulation Activity': 'Heme binding is inhibited by dipeptide'},\n",
       "  'seq': 'MRISLKKSGMLKLGLSLVAMTVAASVQAKTLVYCSEGSPEGFNPQLFTSGTTYDASSVPLYNRLVEFKIGTTEVIPGLAEKWEVSEDGKTYTFHLRKGVKWHDNKEFKPTRELNADDVVFSFDRQKNAQNPYHKVSGGSYEYFEGMGLPELISEVKKVDDNTVQFVLTRPEAPFLADLAMDFASILSKEYADAMMKAGTPEKLDLNPIGTGPFQLQQYQKDSRIRYKAFDGYWGTKPQIDTLVFSITPDASVRYAKLQKNECQVMPYPNPADIARMKQDKSINLMEMPGLNVGYLSYNVQKKPLDDVKVRQALTYAVNKDAIIKAVYQGAGVSAKNLIPPTMWGYNDDVQDYTYDPEKAKALLKEAGLEKGFSIDLWAMPVQRPYNPNARRMAEMIQADWAKVGVQAKIVTYEWGEYLKRAKDGEHQTVMMGWTGDNGDPDNFFATLFSCAASEQGSNYSKWCYKPFEDLIQPARATDDHNKRVELYKQAQVVMHDQAPALIIAHSTVFEPVRKEVKGYVVDPLGKHHFENVSIE',\n",
       "  'mw': 60294.0},\n",
       " 'b3546': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37661',\n",
       "  'gene': 'b3546',\n",
       "  'locus': ['b3546', 'JW5660'],\n",
       "  'synonyms': 'yhjW',\n",
       "  'properties': {'Catalytic Activity': [('a 1,2-diacyl-sn-glycero-3-phosphoethanolamine + alpha-Kdo-(2->4)-alpha-Kdo-(2->6)-lipid A (E. coli) = 7-O-[2-aminoethoxy(hydroxy)phosphoryl]-alpha-Kdo-(2->4)-alpha-Kdo-(2->6)-lipid A + a 1,2-diacyl-sn-glycerol',\n",
       "     '2.7.8.42'),\n",
       "    ('a 1,2-diacyl-sn-glycero-3-phosphoethanolamine + alpha-Kdo-(2->4)-alpha-Kdo-(2->6)-lipid IVA (E. coli) = 7-O-[2-aminoethoxy(hydroxy)phosphoryl]-alpha-Kdo-(2->4)-alpha-Kdo-(2->6)-lipid IVA (E. coli) + a 1,2-diacyl-sn-glycerol',\n",
       "     '2.7.8.42')],\n",
       "   'Function': 'Catalyzes the addition of a phosphoethanolamine (pEtN) moiety to the outer 3-deoxy-D-manno-octulosonic acid (Kdo) residue of a Kdo(2)-lipid A. Phosphatidylethanolamines with one unsaturated acyl group functions as pEtN donors and the reaction releases diacylglycerol',\n",
       "   'Regulation Activity': 'Inhibited by calcium concentrations higher than 1 mM'},\n",
       "  'seq': 'MRYIKSITQQKLSFLLAIYIGLFMNGAVFYRRFGSYAHDFTVWKGISAVVELAATVLVTFFLLRLLSLFGRRSWRILASLVVLFSAGASYYMTFLNVVIGYGIIASVMTTDIDLSKEVVGLNFILWLIAVSALPLILIWNNRCRYTLLRQLRTPGQRIRSLAVVVLAGIMVWAPIRLLDIQQKKVERATGVDLPSYGGVVANSYLPSNWLSALGLYAWARVDESSDNNSLLNPAKKFTYQAPQNVDDTYVVFIIGETTRWDHMGIFGYERNTTPKLAQEKNLAAFRGYSCDTATKLSLRCMFVRQGGAEDNPQRTLKEQNIFAVLKQLGFSSDLYAMQSEMWFYSNTMADNIAYREQIGAEPRNRGKPVDDMLLVDEMQQSLGRNPDGKHLIILHTKGSHFNYTQRYPRSFAQWKPECIGVDSGCTKAQMINSYDNSVTYVDHFISSVIDQVRDKKAIVFYAADHGESINEREHLHGTPRELAPPEQFRVPMMVWMSDKYLENPANAQAFAQLKKEADMKVPRRHVELYDTIMGCLGYTSPDGGINENNNWCHIPQAKEAAAN',\n",
       "  'mw': 63804.0},\n",
       " 'b3551': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P20099',\n",
       "  'gene': 'b3551',\n",
       "  'locus': ['b3551', 'JW5940'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('[thioredoxin]-disulfide + H2O + L-methionine = [thioredoxin]-dithiol + L-methionine (S)-S-oxide',\n",
       "     '1.8.4.13')],\n",
       "   'Function': 'This enzyme may serve as a scavenger, allowing the cell to utilize biotin sulfoxide as a biotin source. It reduces a spontaneous oxidation product of biotin, D-biotin D-sulfoxide (BSO or BDS), back to biotin. Also exhibits methionine-(S)-sulfoxide (Met-S-SO) reductase activity, acting specifically on the (S) enantiomer in the free, but not the protein-bound form. It thus plays a role in assimilation of oxidized methionines'},\n",
       "  'seq': 'MANSSSRYSVLTAAHWGPMLVETDGETVFSSRGALATGMENSLQSAVRDQVHSNTRVRFPMVRKGFLASPENPQGIRGQDEFVRVSWDEALDLIHQQHKRIREAYGPASIFAGSYGWRSNGVLHKASTLLQRYMALAGGYTGHLGDYSTGAAQAIMPYVVGGSEVYQQQTSWPLVLEHSDVVVLWSANPLNTLKIAWNASDEQGLSYFSALRDSGKKLICIDPMRSETVDFFGDKMEWVAPHMGTDVALMLGIAHTLVENGWHDEAFLARCTTGYAVFASYLLGESDGIAKTAEWAAEICGVGAAKIRELAAIFHQNTTMLMAGWGMQRQQFGEQKHWMIVTLAAMLGQIGTPGGGFGLSYHFANGGNPTRRSAVLSSMQGSLPGGCDAVDKIPVARIVEALENPGGAYQHNGMNRHFPDIRFIWWAGGANFTHHQDTNRLIRAWQKPELVVISECFWTAAAKHADIVLPATTSFERNDLTMTGDYSNQHLVPMKQVVPPRYEARNDFDVFAELSERWEKGGYARFTEGKSELQWLETFYNVARQRGASQQVELPPFAEFWQANQLIEMPENPDSERFIRFADFCRDPLAHPLKTASGKIEIFSQRIADYGYPDCPGHPMWLEPDEWQGNAEPEQLQVLSAHPAHRLHSQLNYSSLRELYAVANREPVTIHPDDAQERGIQDGDTVRLWNARGQILAGAVISEGIKPGVICIHEGAWPDLDLTADGICKNGAVNVLTKDLPSSRLGNGCAGNTALAWLEKYNGPELTLTAFEPPASS',\n",
       "  'mw': 85851.0},\n",
       " 'b3553': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37666',\n",
       "  'gene': 'b3553',\n",
       "  'locus': ['b3553', 'JW5656'],\n",
       "  'synonyms': 'tkrA yiaE',\n",
       "  'properties': {'Catalytic Activity': [('glycolate + NADP(+) = glyoxylate + H(+) + NADPH',\n",
       "     '1.1.1.79'),\n",
       "    ('(R)-glycerate + NAD(+) = 3-hydroxypyruvate + H(+) + NADH', '1.1.1.81'),\n",
       "    ('(R)-glycerate + NADP(+) = 3-hydroxypyruvate + H(+) + NADPH', '1.1.1.81'),\n",
       "    ('D-gluconate + NADP(+) = 2-dehydro-D-gluconate + H(+) + NADPH',\n",
       "     '1.1.1.215')],\n",
       "   'Function': 'Catalyzes the NADPH-dependent reduction of glyoxylate and hydroxypyruvate into glycolate and glycerate, respectively. Can also reduce 2,5-diketo-D-gluconate (25DKG) to 5-keto-D-gluconate (5KDG), 2-keto-D-gluconate (2KDG) to D-gluconate, and 2-keto-L-gulonate (2KLG) to L-idonate (IA), but it is not its physiological function. Inactive towards 2-oxoglutarate, oxaloacetate, pyruvate, 5-keto-D-gluconate, D-fructose and L-sorbose. Activity with NAD is very low'},\n",
       "  'seq': 'MKPSVILYKALPDDLLQRLQEHFTVHQVANLSPQTVEQNAAIFAEAEGLLGSNENVNAALLEKMPKLRATSTISVGYDNFDVDALTARKILLMHTPTVLTETVADTLMALVLSTARRVVEVAERVKAGEWTASIGPDWYGTDVHHKTLGIVGMGRIGMALAQRAHFGFNMPILYNARRHHKEAEERFNARYCDLDTLLQESDFVCLILPLTDETHHLFGAEQFAKMKSSAIFINAGRGPVVDENALIAALQKGEIHAAGLDVFEQEPLSVDSPLLSMANVVAVPHIGSATHETRYGMAACAVDNLIDALQGKVEKNCVNPHVAD',\n",
       "  'mw': 35396.0},\n",
       " 'b3559': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P00961',\n",
       "  'gene': 'b3559',\n",
       "  'locus': ['b3559', 'JW3530'],\n",
       "  'synonyms': 'glyS(B)',\n",
       "  'properties': {'Catalytic Activity': [('ATP + glycine + tRNA(Gly) = AMP + diphosphate + glycyl-tRNA(Gly)',\n",
       "     '6.1.1.14')]},\n",
       "  'seq': 'MSEKTFLVEIGTEELPPKALRSLAESFAANFTAELDNAGLAHGTVQWFAAPRRLALKVANLAEAQPDREIEKRGPAIAQAFDAEGKPSKAAEGWARGCGITVDQAERLTTDKGEWLLYRAHVKGESTEALLPNMVATSLAKLPIPKLMRWGASDVHFVRPVHTVTLLLGDKVIPATILGIQSDRVIRGHRFMGEPEFTIDNADQYPEILRERGKVIADYEERKAKIKADAEEAARKIGGNADLSESLLEEVASLVEWPVVLTAKFEEKFLAVPAEALVYTMKGDQKYFPVYANDGKLLPNFIFVANIESKDPQQIISGNEKVVRPRLADAEFFFNTDRKKRLEDNLPRLQTVLFQQQLGTLRDKTDRIQALAGWIAEQIGADVNHATRAGLLSKCDLMTNMVFEFTDTQGVMGMHYARHDGEAEDVAVALNEQYQPRFAGDDLPSNPVACALAIADKMDTLAGIFGIGQHPKGDKDPFALRRAALGVLRIIVEKNLNLDLQTLTEEAVRLYGDKLTNANVVDDVIDFMLGRFRAWYQDEGYTVDTIQAVLARRPTRPADFDARMKAVSHFRTLDAAAALAAANKRVSNILAKSDEVLSDRVNASTLKEPEEIKLAMQVVVLRDKLEPYFTEGRYQDALVELAELREPVDAFFDKVMVMVDDKELRINRLTMLEKLRELFLRVADISLLQ',\n",
       "  'mw': 76813.0},\n",
       " 'b3560': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P00960',\n",
       "  'gene': 'b3560',\n",
       "  'locus': ['b3560', 'JW3531'],\n",
       "  'synonyms': 'glyS(A)',\n",
       "  'properties': {'Catalytic Activity': [('ATP + glycine + tRNA(Gly) = AMP + diphosphate + glycyl-tRNA(Gly)',\n",
       "     '6.1.1.14')]},\n",
       "  'seq': 'MQKFDTRTFQGLILTLQDYWARQGCTIVQPLDMEVGAGTSHPMTCLRELGPEPMAAAYVQPSRRPTDGRYGENPNRLQHYYQFQVVIKPSPDNIQELYLGSLKELGMDPTIHDIRFVEDNWENPTLGAWGLGWEVWLNGMEVTQFTYFQQVGGLECKPVTGEITYGLERLAMYIQGVDSVYDLVWSDGPLGKTTYGDVFHQNEVEQSTYNFEYADVDFLFTCFEQYEKEAQQLLALENPLPLPAYERILKAAHSFNLLDARKAISVTERQRYILRIRTLTKAVAEAYYASREALGFPMCNKDK',\n",
       "  'mw': 34774.0},\n",
       " 'b3564': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P09099',\n",
       "  'gene': 'b3564',\n",
       "  'locus': ['b3564', 'JW3536'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + D-xylulose = ADP + D-xylulose 5-phosphate + H(+)',\n",
       "     '2.7.1.17'),\n",
       "    ('1-deoxy-D-xylulose + ATP = 1-deoxy-D-xylulose 5-phosphate + ADP + H(+)',\n",
       "     '')],\n",
       "   'Function': 'Catalyzes the phosphorylation of D-xylulose to D-xylulose 5-phosphate (PubMed:17123542). Also catalyzes the phosphorylation of 1-deoxy-D-xylulose to 1-deoxy-D-xylulose 5-phosphate, with lower efficiency (PubMed:11168365). Can also use D-ribulose, xylitol and D-arabitol, but D-xylulose is preferred over the other substrates (PubMed:17123542). Has a weak substrate-independent Mg-ATP-hydrolyzing activity (PubMed:17123542)',\n",
       "   'Regulation Activity': 'Sugar binding is accompanied by a dramatic hinge-bending movement that enhances interactions with Mg-ATP'},\n",
       "  'seq': 'MYIGIDLGTSGVKVILLNEQGEVVAAQTEKLTVSRPHPLWSEQDPEQWWQATDRAMKALGDQHSLQDVKALGIAGQMHGATLLDAQQRVLRPAILWNDGRCAQECTLLEARVPQSRVITGNLMMPGFTAPKLLWVQRHEPEIFRQIDKVLLPKDYLRLRMTGEFASDMSDAAGTMWLDVAKRDWSDVMLQACDLSRDQMPALYEGSEITGALLPEVAKAWGMATVPVVAGGGDNAAGAVGVGMVDANQAMLSLGTSGVYFAVSEGFLSKPESAVHSFCHALPQRWHLMSVMLSAASCLDWAAKLTGLSNVPALIAAAQQADESAEPVWFLPYLSGERTPHNNPQAKGVFFGLTHQHGPNELARAVLEGVGYALADGMDVVHACGIKPQSVTLIGGGARSEYWRQMLADISGQQLDYRTGGDVGPALGAARLAQIAANPEKSLIELLPQLPLEQSHLPDAQRYAAYQPRRETFRRLYQQLLPLMA',\n",
       "  'mw': 52618.0},\n",
       " 'b3565': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P00944',\n",
       "  'gene': 'b3565',\n",
       "  'locus': ['b3565', 'JW3537'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('alpha-D-xylose = alpha-D-xylulofuranose',\n",
       "     '5.3.1.5')]},\n",
       "  'seq': 'MQAYFDQLDRVRYEGSKSSNPLAFRHYNPDELVLGKRMEEHLRFAACYWHTFCWNGADMFGVGAFNRPWQQPGEALALAKRKADVAFEFFHKLHVPFYCFHDVDVSPEGASLKEYINNFAQMVDVLAGKQEESGVKLLWGTANCFTNPRYGAGAATNPDPEVFSWAATQVVTAMEATHKLGGENYVLWGGREGYETLLNTDLRQEREQLGRFMQMVVEHKHKIGFQGTLLIEPKPQEPTKHQYDYDAATVYGFLKQFGLEKEIKLNIEANHATLAGHSFHHEIATAIALGLFGSVDANRGDAQLGWDTDQFPNSVEENALVMYEILKAGGFTTGGLNFDAKVRRQSTDKYDLFYGHIGAMDTMALALKIAARMIEDGELDKRIAQRYSGWNSELGQQILKGQMSLADLAKYAQEHHLSPVHQSGRQEQLENLVNHYLFDK',\n",
       "  'mw': 49742.0},\n",
       " 'b3566': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37387',\n",
       "  'gene': 'b3566',\n",
       "  'locus': ['b3566', 'JW3538'],\n",
       "  'synonyms': 'xylT',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Involved in the high-affinity D-xylose membrane transport system. Binds with high affinity to xylose'},\n",
       "  'seq': 'MKIKNILLTLCTSLLLTNVAAHAKEVKIGMAIDDLRLERWQKDRDIFVKKAESLGAKVFVQSANGNEETQMSQIENMINRGVDVLVIIPYNGQVLSNVVKEAKQEGIKVLAYDRMINDADIDFYISFDNEKVGELQAKALVDIVPQGNYFLMGGSPVDNNAKLFRAGQMKVLKPYVDSGKIKVVGDQWVDGWLPENALKIMENALTANNNKIDAVVASNDATAGGAIQALSAQGLSGKVAISGQDADLAGIKRIAAGTQTMTVYKPITLLANTAAEIAVELGNGQEPKADTTLNNGLKDVPSRLLTPIDVNKNNIKDTVIKDGFHKESEL',\n",
       "  'mw': 35734.0},\n",
       " 'b3567': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37388',\n",
       "  'gene': 'b3567',\n",
       "  'locus': ['b3567', 'JW3539'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('ATP + D-xylose(out) + H2O = ADP + D-xylose(in) + H(+) + phosphate',\n",
       "     '7.5.2.10')],\n",
       "   'Function': 'Part of the ABC transporter complex XylFGH involved in xylose import. Responsible for energy coupling to the transport system (Probable). The XylFGH system can also transport ribose in absence of xylose'},\n",
       "  'seq': 'MPYLLEMKNITKTFGSVKAIDNVCLRLNAGEIVSLCGENGSGKSTLMKVLCGIYPHGSYEGEIIFAGEEIQASHIRDTERKGIAIIHQELALVKELTVLENIFLGNEITHNGIMDYDLMTLRCQKLLAQVSLSISPDTRVGDLGLGQQQLVEIAKALNKQVRLLILDEPTASLTEQETSILLDIIRDLQQHGIACIYISHKLNEVKAISDTICVIRDGQHIGTRDAAGMSEDDIITMMVGRELTALYPNEPHTTGDEILRIEHLTAWHPVNRHIKRVNDVSFSLKRGEILGIAGLVGAGRTETIQCLFGVWPGQWEGKIYIDGKQVDIRNCQQAIAQGIAMVPEDRKRDGIVPVMAVGKNITLAALNKFTGGISQLDDAAEQKCILESIQQLKVKTSSPDLAIGRLSGGNQQKAILARCLLLNPRILILDEPTRGIDIGAKYEIYKLINQLVQQGIAVIVISSELPEVLGLSDRVLVMHEGKLKANLINHNLTQEQVMEAALRSEHHVEKQSV',\n",
       "  'mw': 56470.0},\n",
       " 'b3568': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AGI4',\n",
       "  'gene': 'b3568',\n",
       "  'locus': ['b3568', 'JW3540'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the binding-protein-dependent transport system for D-xylose. Probably responsible for the translocation of the substrate across the membrane'},\n",
       "  'seq': 'MSKSNPSEVKLAVPTSGGFSGLKSLNLQVFVMIAAIIAIMLFFTWTTDGAYLSARNVSNLLRQTAITGILAVGMVFVIISAEIDLSVGSMMGLLGGVAAICDVWLGWPLPLTIIVTLVLGLLLGAWNGWWVAYRKVPSFIVTLAGMLAFRGILIGITNGTTVSPTSAAMSQIGQSYLPASTGFIIGALGLMAFVGWQWRGRMRRQALGLQSPASTAVVGRQALTAIIVLGAIWLLNDYRGVPTPVLLLTLLLLGGMFMATRTAFGRRIYAIGGNLEAARLSGINVERTKLAVFAINGLMVAIAGLILSSRLGAGSPSAGNIAELDAIAACVIGGTSLAGGVGSVAGAVMGAFIMASLDNGMSMMDVPTFWQYIVKGAILLLAVWMDSATKRRS',\n",
       "  'mw': 41031.0},\n",
       " 'b3571': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P25718',\n",
       "  'gene': 'b3571',\n",
       "  'locus': ['b3571', 'JW3543'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('Endohydrolysis of (1->4)-alpha-D-glucosidic linkages in polysaccharides containing three or more (1->4)-alpha-linked D-glucose units.',\n",
       "     '3.2.1.1')],\n",
       "   'Function': 'Since only maltooligosaccharides up to a chain length of 6 glucose units are actively transported through the cytoplasmic membrane via the membrane-bound complex of three proteins, MalF, MalG, and MalK, longer maltooligosaccharides must first be degraded by the periplasmic alpha-amylase, the MalS protein'},\n",
       "  'seq': 'MKLAACFLTLLPGFAVAASWTSPGFPAFSEQGTGTFVSHAQLPKGTRPLTLNFDQQCWQPADAIKLNQMLSLQPCSNTPPQWRLFRDGEYTLQIDTRSGTPTLMISIQNAAEPVASLVRECPKWDGLPLTVDVSATFPEGAAVRDYYSQQIAIVKNGQIMLQPAATSNGLLLLERAETDTSAPFDWHNATVYFVLTDRFENGDPSNDQSYGRHKDGMAEIGTFHGGDLRGLTNKLDYLQQLGVNALWISAPFEQIHGWVGGGTKGDFPHYAYHGYYTQDWTNLDANMGNEADLRTLVDSAHQRGIRILFDVVMNHTGYATLADMQEYQFGALYLSGDEVKKSLGERWSDWKPAAGQTWHSFNDYINFSDKTGWDKWWGKNWIRTDIGDYDNPGFDDLTMSLAFLPDIKTESTTASGLPVFYKNKMDTHAKAIDGYTPRDYLTHWLSQWVRDYGIDGFRVDTAKHVELPAWQQLKTEASAALREWKKANPDKALDDKPFWMTGEAWGHGVMQSDYYRHGFDAMINFDYQEQAAKAVDCLAQMDTTWQQMAEKLQGFNVLSYLSSHDTRLFREGGDKAAELLLLAPGAVQIFYGDESSRPFGPTGSDPLQGTRSDMNWQDVSGKSAASVAHWQKISQFRARHPAIGAGKQTTLLLKQGYGFVREHGDDKVLVVWAGQQ',\n",
       "  'mw': 75713.0},\n",
       " 'b3572': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P09053',\n",
       "  'gene': 'b3572',\n",
       "  'locus': ['b3572', 'JW5652'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('L-valine + pyruvate = 3-methyl-2-oxobutanoate + L-alanine',\n",
       "     '2.6.1.66')],\n",
       "   'Function': 'Involved in the biosynthesis of alanine'},\n",
       "  'seq': 'MTFSLFGDKFTRHSGITLLMEDLNDGLRTPGAIMLGGGNPAQIPEMQDYFQTLLTDMLESGKATDALCNYDGPQGKTELLTLLAGMLREKLGWDIEPQNIALTNGSQSAFFYLFNLFAGRRADGRVKKVLFPLAPEYIGYADAGLEEDLFVSARPNIELLPEGQFKYHVDFEHLHIGEETGMICVSRPTNPTGNVITDEELLKLDALANQHGIPLVIDNAYGVPFPGIIFSEARPLWNPNIVLCMSLSKLGLPGSRCGIIIANEKIITAITNMNGIISLAPGGIGPAMMCEMIKRNDLLRLSETVIKPFYYQRVQETIAIIRRYLPENRCLIHKPEGAIFLWLWFKDLPITTKQLYQRLKARGVLMVPGHNFFPGLDKPWPHTHQCMRMNYVPEPEKIEAGVKILAEEIERAWAESH',\n",
       "  'mw': 46711.0},\n",
       " 'b3575': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37672',\n",
       "  'gene': 'b3575',\n",
       "  'locus': ['b3575', 'JW3547'],\n",
       "  'synonyms': 'yiaK',\n",
       "  'properties': {'Catalytic Activity': [('3-dehydro-L-gulonate + NAD(+) = 2,3-dioxo-L-gulonate + H(+) + NADH',\n",
       "     '1.1.1.130'),\n",
       "    ('3-dehydro-L-gulonate + NADP(+) = 2,3-dioxo-L-gulonate + H(+) + NADPH',\n",
       "     '1.1.1.130')],\n",
       "   'Function': 'Catalyzes the reduction of 2,3-diketo-L-gulonate in the presence of NADH, to form 3-keto-L-gulonate',\n",
       "   'Regulation Activity': 'The enzyme is inhibited by tartrate and D-malate'},\n",
       "  'seq': 'MKVTFEQLKAAFNRVLISRGVDSETADACAEMFARTTESGVYSHGVNRFPRFIQQLENGDIIPDAQPKRITSLGAIEQWDAQRSIGNLTAKKMMDRAIELAADHGIGLVALRNANHWMRGGSYGWQAAEKGYIGICWTNSIAVMPPWGAKECRIGTNPLIVAIPSTPITMVDMSMSMFSYGMLEVNRLAGRQLPVDGGFDDEGNLTKEPGVIEKNRRILPMGYWKGSGMSIVLDMIATLLSDGASVAEVTQDNSDEYGISQIFIAIEVDKLIDGPTRDAKLQRIMDYVTSAERADENQAIRLPGHEFTTLLAENRRNGITVDDSVWAKIQAL',\n",
       "  'mw': 36573.0},\n",
       " 'b3577': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37674',\n",
       "  'gene': 'b3577',\n",
       "  'locus': ['b3577', 'JW3549'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the tripartite ATP-independent periplasmic (TRAP) transport system YiaMNO involved in the uptake of 2,3-diketo-L-gulonate'},\n",
       "  'seq': 'MKKILEAILAINLAVLSCIVFINIILRYGFQTSILSVDELSRYLFVWLTFIGAIVAFMDNAHVQVTFLVEKLSPAWQRRVALVTHSLILFICGALAWGATLKTIQDWSDYSPILGLPIGLMYAACLPTSLVIAFFELRHLYQLITRSNSLTSPPQGA',\n",
       "  'mw': 17516.0},\n",
       " 'b3578': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37675',\n",
       "  'gene': 'b3578',\n",
       "  'locus': ['b3578', 'JW5651'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the tripartite ATP-independent periplasmic (TRAP) transport system YiaMNO involved in the uptake of 2,3-diketo-L-gulonate'},\n",
       "  'seq': 'MAVLIFLGCLLGGIAIGLPIAWALLLCGAALMFWLDMFDVQIMAQTLVNGADSFSLLAIPFFVLAGEIMNAGGLSKRIVDLPMKLVGHKPGGLGYVGVLAAMIMASLSGSAVADTAAVAALLVPMMRSANYPVNRAAGLIASGGIIAPIIPPSIPFIIFGVSSGLSISKLFMAGIAPGMMMGATLMLTWWWQASRLNLPRQQKATMQEIWHSFVSGIWALFLPVIIIGGFRSGLFTPTEAGAVAAFYALFVATVIYREMTFATLWHVLIGAAKTTSVVMFLVASAQVSAWLITIAELPMMVSDLLQPLVDSPRLLFIVIMVAILIVGMVMDLTPTVLILTPVLMPLVKEAGIDPIYFGVMFIINCSIGLITPPIGNVLNVISGVAKLKFDDAVRGVFPYVLVLYSLLVVFVFIPDLIILPLKWIN',\n",
       "  'mw': 45368.0},\n",
       " 'b3579': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37676',\n",
       "  'gene': 'b3579',\n",
       "  'locus': ['b3579', 'JW3551'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'Part of the tripartite ATP-independent periplasmic (TRAP) transport system YiaMNO involved in the uptake of 2,3-diketo-L-gulonate. This protein specifically binds 2,3-diketo-L-gulonate. Is not able to bind either L-ascorbate or dehydroascorbate'},\n",
       "  'seq': 'MKLRSVTYALFIAGLAAFSTSSLAAQSLRFGYETSQTDSQHIAAKKFNDLLQERTKGELKLKLFPDSTLGNAQAMISGVRGGTIDMEMSGSNNFAGLSPVMNLLDVPFLFRDTAHAHKTLDGKVGDDLKASLEGKGLKVLAYWENGWRDVTNSRAPVKTPADLKGLKIRTNNSPMNIAAFKVFGANPIPMPFAEVYTGLETRTIDAQEHPINVVWSAKFFEVQKFLSLTHHAYSPLLVVINKAKFDGLSPEFQQALVSSAQEAGNYQRKLVAEDQQKIIDGMKEAGVEVITDLDRKAFSDALGNQVRDMFVKDVPQGADLLKAVDEVQ',\n",
       "  'mw': 35970.0},\n",
       " 'b3580': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37677',\n",
       "  'gene': 'b3580',\n",
       "  'locus': ['b3580', 'JW3552'],\n",
       "  'synonyms': 'lyxK sgbK xylK yiaP',\n",
       "  'properties': {'Catalytic Activity': [('ATP + L-xylulose = ADP + H(+) + L-xylulose 5-phosphate',\n",
       "     '2.7.1.53'),\n",
       "    ('3-dehydro-L-gulonate + ATP = 3-dehydro-L-gulonate 6-phosphate + ADP + H(+)',\n",
       "     '')],\n",
       "   'Function': 'Catalyzes the phosphorylation of L-xylulose and 3-keto-L-gulonate. Is involved in L-lyxose utilization via xylulose, and may also be involved in the utilization of 2,3-diketo-L-gulonate'},\n",
       "  'seq': 'MTQYWLGLDCGGSWLKAGLYDREGREAGVQRLPLCALSPQPGWAERDMAELWQCCMAVIRALLTHSGVSGEQIVGIGISAQGKGLFLLDKNDKPLGNAILSSDRRAMEIVRRWQEDGIPEKLYPLTRQTLWTGHPVSLLRWLKEHEPERYAQIGCVMMTHDYLRWCLTGVKGCEESNISESNLYNMSLGEYDPCLTDWLGIAEINHALPPVVGSAEICGEITAQTAALTGLKAGTPVVGGLFDVVSTALCAGIEDEFTLNAVMGTWAVTSGITRGLRDGEAHPYVYGRYVNDGEFIVHEASPTSSGNLEWFTAQWGEISFDEINQAVASLPKAGGDLFFLPFLYGSNAGLEMTSGFYGMQAIHTRAHLLQAIYEGVVFSHMTHLNRMRERFTDVHTLRVTGGPAHSDVWMQMLADVSGLRIELPQVEETGCFGAALAARVGTGVYHNFSEAQRDLRHPVRTLLPDMTAHQLYQKKYQRYQHLIAALQGFHARIKEHTL',\n",
       "  'mw': 55155.0},\n",
       " 'b3581': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37678',\n",
       "  'gene': 'b3581',\n",
       "  'locus': ['b3581', 'JW3553'],\n",
       "  'synonyms': 'yiaQ',\n",
       "  'properties': {'Catalytic Activity': [('3-dehydro-L-gulonate 6-phosphate + H(+) = CO2 + L-xylulose 5-phosphate',\n",
       "     '4.1.1.85')],\n",
       "   'Function': 'Catalyzes the decarboxylation of 3-keto-L-gulonate-6-P into L-xylulose-5-P. May be involved in the utilization of 2,3-diketo-L-gulonate'},\n",
       "  'seq': 'MSRPLLQLALDHSSLEAAQRDVTLLKDSVDIVEAGTILCLNEGLGAVKALREQCPDKIIVADWKVADAGETLAQQAFGAGANWMTIICAAPLATVEKGHAMAQRCGGEIQIELFGNWTLDDARDWHRIGVRQAIYHRGRDAQASGQQWGEADLARMKALSDIGLELSITGGITPADLPLFKDIRVKAFIAGRALAGAANPAQVAGDFHAQIDAIWGGARA',\n",
       "  'mw': 23445.0},\n",
       " 'b3583': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37680',\n",
       "  'gene': 'b3583',\n",
       "  'locus': ['b3583', 'JW3555'],\n",
       "  'synonyms': 'yiaS',\n",
       "  'properties': {'Catalytic Activity': [('L-ribulose 5-phosphate = D-xylulose 5-phosphate',\n",
       "     '5.1.3.4')],\n",
       "   'Function': 'Catalyzes the interconversion of L-ribulose 5-phosphate (LRu5P) and D-xylulose 5-phosphate (D-Xu5P) via a retroaldol/aldol mechanism (carbon-carbon bond cleavage analogous to a class II aldolase reaction). May be involved in the utilization of 2,3-diketo-L-gulonate'},\n",
       "  'seq': 'MLEQLKADVLAANLALPAHHLVTFTWGNVSAVDETRQWMVIKPSGVEYDVMTADDMVVVEIASGKVVEGSKKPSSDTPTHLALYRRYAEIGGIVHTHSRHATIWSQAGLDLPAWGTTHADYFYGAIPCTRQMTAEEINGEYEYQTGEVIIETFEERGRSPAQIPAVLVHSHGPFAWGKNAADAVHNAVVLEECAYMGLFSRQLAPQLPAMQNELLDKHYLRKHGANAYYGQ',\n",
       "  'mw': 25561.0},\n",
       " 'b3588': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37685',\n",
       "  'gene': 'b3588',\n",
       "  'locus': ['b3588', 'JW3561'],\n",
       "  'synonyms': 'yiaX',\n",
       "  'properties': {'Catalytic Activity': [('an aldehyde + H2O + NADP(+) = a carboxylate + 2 H(+) + NADPH',\n",
       "     '1.2.1.4'),\n",
       "    ('acetaldehyde + H2O + NADP(+) = acetate + 2 H(+) + NADPH', '1.2.1.4'),\n",
       "    ('chloroacetaldehyde + H2O + NADP(+) = chloroacetate + 2 H(+) + NADPH',\n",
       "     ''),\n",
       "    ('H2O + NADP(+) + propanal = 2 H(+) + NADPH + propanoate', '')],\n",
       "   'Function': 'Catalyzes the NADP(+)-dependent oxidation of diverse aldehydes to their corresponding carboxylic acids, with a preference for acetaldehyde and chloroacetaldehyde (PubMed:15659684). May play a role in detoxifying aldehydes present during stationary phase (Probable). Cannot use NAD(+) instead of NADP(+) as the electron acceptor. To a lesser extent is also able to oxidize propionaldehyde (propanal), benzaldehyde, mafosfamide, and 4-hydroperoxycyclophosphamide. Does not use either glyceraldehyde or glycolaldehyde as substrates (PubMed:15659684)',\n",
       "   'Regulation Activity': 'Magnesium increases enzyme activity with various substrates'},\n",
       "  'seq': 'MTNNPPSAQIKPGEYGFPLKLKARYDNFIGGEWVAPADGEYYQNLTPVTGQLLCEVASSGKRDIDLALDAAHKVKDKWAHTSVQDRAAILFKIADRMEQNLELLATAETWDNGKPIRETSAADVPLAIDHFRYFASCIRAQEGGISEVDSETVAYHFHEPLGVVGQIIPWNFPLLMASWKMAPALAAGNCVVLKPARLTPLSVLLLMEIVGDLLPPGVVNVVNGAGGVIGEYLATSKRIAKVAFTGSTEVGQQIMQYATQNIIPVTLELGGKSPNIFFADVMDEEDAFFDKALEGFALFAFNQGEVCTCPSRALVQESIYERFMERAIRRVESIRSGNPLDSVTQMGAQVSHGQLETILNYIDIGKKEGADVLTGGRRKLLEGELKDGYYLEPTILFGQNNMRVFQEEIFGPVLAVTTFKTMEEALELANDTQYGLGAGVWSRNGNLAYKMGRGIQAGRVWTNCYHAYPAHAAFGGYKQSGIGRETHKMMLEHYQQTKCLLVSYSDKPLGLF',\n",
       "  'mw': 56306.0},\n",
       " 'b3591': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A821',\n",
       "  'gene': 'b3591',\n",
       "  'locus': ['b3591', 'JW3564'],\n",
       "  'synonyms': 'fdhA',\n",
       "  'properties': {'Catalytic Activity': [('H(+) + L-seryl-tRNA(Sec) + selenophosphate = L-selenocysteinyl-tRNA(Sec) + phosphate',\n",
       "     '2.9.1.1')],\n",
       "   'Function': 'Converts seryl-tRNA(Sec) to selenocysteinyl-tRNA(Sec) required for selenoprotein biosynthesis. Requires selenophosphate as the selenium-donor molecule',\n",
       "   'Pathway': 'Aminoacyl-tRNA biosynthesis; selenocysteinyl-tRNA(Sec) biosynthesis; selenocysteinyl-tRNA(Sec) from L-seryl-tRNA(Sec) (bacterial route): step 1/1'},\n",
       "  'seq': 'MTTETRSLYSQLPAIDRLLRDSSFLSLRDTYGHTRVVELLRQMLDEAREVIRGSQTLPAWCENWAQEVDARLTKEAQSALRPVINLTGTVLHTNLGRALQAEAAVEAVAQAMRSPVTLEYDLDDAGRGHRDRALAQLLCRITGAEDACIVNNNAAAVLLMLAATASGKEVVVSRGELVEIGGAFRIPDVMRQAGCTLHEVGTTNRTHANDYRQAVNENTALLMKVHTSNYSIQGFTKAIDEAELVALGKELDVPVVTDLGSGSLVDLSQYGLPKEPMPQELIAAGVSLVSFSGDKLLGGPQAGIIVGKKEMIARLQSHPLKRALRADKMTLAALEATLRLYLHPEALSEKLPTLRLLTRSAEVIQIQAQRLQAPLAAHYGAEFAVQVMPCLSQIGSGSLPVDRLPSAALTFTPHDGRGSHLESLAARWRELPVPVIGRIYDGRLWLDLRCLEDEQRFLEMLLK',\n",
       "  'mw': 50607.0},\n",
       " 'b3599': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P00550',\n",
       "  'gene': 'b3599',\n",
       "  'locus': ['b3599', 'JW3573'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('D-mannitol(out) + N(pros)-phospho-L-histidyl-[protein] = D-mannitol 1-phosphate(in) + L-histidyl-[protein]',\n",
       "     '2.7.1.197')],\n",
       "   'Function': 'The phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS), a major carbohydrate active transport system, catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane. This system is involved in D-mannitol transport (PubMed:368051, PubMed:6427236, PubMed:2123863). Also able to use D-mannonic acid (PubMed:6427236)'},\n",
       "  'seq': 'MSSDIKIKVQSFGRFLSNMVMPNIGAFIAWGIITALFIPTGWLPNETLAKLVGPMITYLLPLLIGYTGGKLVGGERGGVVGAITTMGVIVGADMPMFLGSMIAGPLGGWCIKHFDRWVDGKIKSGFEMLVNNFSAGIIGMILAILAFLGIGPIVEALSKMLAAGVNFMVVHDMLPLASIFVEPAKILFLNNAINHGIFSPLGIQQSHELGKSIFFLIEANPGPGMGVLLAYMFFGRGSAKQSAGGAAIIHFLGGIHEIYFPYVLMNPRLILAVILGGMTGVFTLTILGGGLVSPASPGSILAVLAMTPKGAYFANIAGVCAAMAVSFVVSAILLKTSKVKEEDDIEAATRRMQDMKAESKGASPLSAGDVTNDLSHVRKIIVACDAGMGSSAMGAGVLRKKIQDAGLSQISVTNSAINNLPPDVDLVITHRDLTERAMRQVPQAQHISLTNFLDSGLYTSLTERLVAAQRHTANEEKVKDSLKDSFDDSSANLFKLGAENIFLGRKAATKEEAIRFAGEQLVKGGYVEPEYVQAMLDREKLTPTYLGESIAVPHGTVEAKDRVLKTGVVFCQYPEGVRFGEEEDDIARLVIGIAARNNEHIQVITSLTNALDDESVIERLAHTTSVDEVLELLAGRK',\n",
       "  'mw': 67972.0},\n",
       " 'b3600': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P09424',\n",
       "  'gene': 'b3600',\n",
       "  'locus': ['b3600', 'JW3574'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('D-mannitol 1-phosphate + NAD(+) = beta-D-fructose 6-phosphate + H(+) + NADH',\n",
       "     '1.1.1.17')]},\n",
       "  'seq': 'MKALHFGAGNIGRGFIGKLLADAGIQLTFADVNQVVLDALNARHSYQVHVVGETEQVDTVSGVNAVSSIGDDVVDLIAQVDLVTTAVGPVVLERIAPAIAKGQVKRKEQGNESPLNIIACENMVRGTTQLKGHVMNALPEDAKAWVEEHVGFVDSAVDRIVPPSASATNDPLEVTVETFSEWIVDKTQFKGALPNIPGMELTDNLMAFVERKLFTLNTGHAITAYLGKLAGHQTIRDAILDEKIRAVVKGAMEESGAVLIKRYGFDADKHAAYIQKILGRFENPYLKDDVERVGRQPLRKLSAGDRLIKPLLGTLEYGLPHKNLIEGIAAAMHFRSEDDPQAQELAALIADKGPQAALAQISGLDANSEVVSEAVTAYKAMQ',\n",
       "  'mw': 41139.0},\n",
       " 'b3603': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P33231',\n",
       "  'gene': 'b3603',\n",
       "  'locus': ['b3603', 'JW3578'],\n",
       "  'synonyms': 'lctP',\n",
       "  'properties': {'Catalytic Activity': [('(S)-lactate(in) + H(+)(in) = (S)-lactate(out) + H(+)(out)',\n",
       "     ''),\n",
       "    ('(R)-lactate(in) + H(+)(in) = (R)-lactate(out) + H(+)(out)', ''),\n",
       "    ('glycolate(in) + H(+)(in) = glycolate(out) + H(+)(out)', '')],\n",
       "   'Function': 'Uptake of L-lactate across the membrane (PubMed:11283302, PubMed:11785976). Can also transport D-lactate and glycolate (PubMed:11283302, PubMed:11785976). Seems to be driven by a proton motive force (PubMed:11785976)',\n",
       "   'Regulation Activity': 'Inhibited by the proton ionophore carbonyl cyanide m-chlorophenylhydrazone (CCCP)'},\n",
       "  'seq': 'MNLWQQNYDPAGNIWLSSLIASLPILFFFFALIKLKLKGYVAASWTVAIALAVALLFYKMPVANALASVVYGFFYGLWPIAWIIIAAVFVYKISVKTGQFDIIRSSILSITPDQRLQMLIVGFCFGAFLEGAAGFGAPVAITAALLVGLGFKPLYAAGLCLIVNTAPVAFGAMGIPILVAGQVTGIDSFEIGQMVGRQLPFMTIIVLFWIMAIMDGWRGIKETWPAVVVAGGSFAIAQYLSSNFIGPELPDIISSLVSLLCLTLFLKRWQPVRVFRFGDLGASQVDMTLAHTGYTAGQVLRAWTPFLFLTATVTLWSIPPFKALFASGGALYEWVINIPVPYLDKLVARMPPVVSEATAYAAVFKFDWFSATGTAILFAALLSIVWLKMKPSDAISTFGSTLKELALPIYSIGMVLAFAFISNYSGLSSTLALALAHTGHAFTFFSPFLGWLGVFLTGSDTSSNALFAALQATAAQQIGVSDLLLVAANTTGGVTGKMISPQSIAIACAAVGLVGKESDLFRFTVKHSLIFTCIVGVITTLQAYVLTWMIP',\n",
       "  'mw': 59168.0},\n",
       " 'b3605': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P33232',\n",
       "  'gene': 'b3605',\n",
       "  'locus': ['b3605', 'JW3580'],\n",
       "  'synonyms': 'lctD',\n",
       "  'properties': {'Catalytic Activity': [('(S)-lactate + A = AH2 + pyruvate',\n",
       "     '')],\n",
       "   'Function': 'Catalyzes the conversion of L-lactate to pyruvate. Seems to be a primary dehydrogenase in the respiratory chain. To a lesser extent, can also oxidize DL-alpha-hydroxybutyrate, but not D-lactate',\n",
       "   'Regulation Activity': 'Is activated by 2-fold in the presence of high concentrations of sodium and potassium ions. Is almost completely inhibited by a high concentration of pyruvate (10 mM)'},\n",
       "  'seq': 'MIISAASDYRAAAQRILPPFLFHYMDGGAYSEYTLRRNVEDLSEVALRQRILKNMSDLSLETTLFNEKLSMPVALAPVGLCGMYARRGEVQAAKAADAHGIPFTLSTVSVCPIEEVAPAIKRPMWFQLYVLRDRGFMRNALERAKAAGCSTLVFTVDMPTPGARYRDAHSGMSGPNAAMRRYLQAVTHPQWAWDVGLNGRPHDLGNISAYLGKPTGLEDYIGWLGNNFDPSISWKDLEWIRDFWDGPMVIKGILDPEDARDAVRFGADGIVVSNHGGRQLDGVLSSARALPAIADAVKGDIAILADSGIRNGLDVVRMIALGADTVLLGRAFLYALATAGQAGVANLLNLIEKEMKVAMTLTGAKSISEITQDSLVQGLGKELPAALAPMAKGNAA',\n",
       "  'mw': 42728.0},\n",
       " 'b3607': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A9D4',\n",
       "  'gene': 'b3607',\n",
       "  'locus': ['b3607', 'JW3582'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('acetyl-CoA + L-serine = CoA + O-acetyl-L-serine',\n",
       "     '2.3.1.30')],\n",
       "   'Regulation Activity': 'Sensitive to feedback inhibition by L-cysteine',\n",
       "   'Pathway': 'Amino-acid biosynthesis; L-cysteine biosynthesis; L-cysteine from L-serine: step 1/2'},\n",
       "  'seq': 'MSCEELEIVWNNIKAEARTLADCEPMLASFYHATLLKHENLGSALSYMLANKLSSPIMPAIAIREVVEEAYAADPEMIASAACDIQAVRTRDPAVDKYSTPLLYLKGFHALQAYRIGHWLWNQGRRALAIFLQNQVSVTFQVDIHPAAKIGRGIMLDHATGIVVGETAVIENDVSILQSVTLGGTGKSGGDRHPKIREGVMIGAGAKILGNIEVGRGAKIGAGSVVLQPVPPHTTAAGVPARIVGKPDSDKPSMDMDQHFNGINHTFEYGDGI',\n",
       "  'mw': 29317.0},\n",
       " 'b3608': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0A6S7',\n",
       "  'gene': 'b3608',\n",
       "  'locus': ['b3608', 'JW3583'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('NAD(+) + sn-glycerol 3-phosphate = dihydroxyacetone phosphate + H(+) + NADH',\n",
       "     '1.1.1.94'),\n",
       "    ('NADP(+) + sn-glycerol 3-phosphate = dihydroxyacetone phosphate + H(+) + NADPH',\n",
       "     '1.1.1.94')],\n",
       "   'Pathway': 'Membrane lipid metabolism; glycerophospholipid metabolism'},\n",
       "  'seq': 'MNQRNASMTVIGAGSYGTALAITLARNGHEVVLWGHDPEHIATLERDRCNAAFLPDVPFPDTLHLESDLATALAASRNILVVVPSHVFGEVLRQIKPLMRPDARLVWATKGLEAETGRLLQDVAREALGDQIPLAVISGPTFAKELAAGLPTAISLASTDQTFADDLQQLLHCGKSFRVYSNPDFIGVQLGGAVKNVIAIGAGMSDGIGFGANARTALITRGLAEMSRLGAALGADPATFMGMAGLGDLVLTCTDNQSRNRRFGMMLGQGMDVQSAQEKIGQVVEGYRNTKEVRELAHRFGVEMPITEEIYQVLYCGKNAREAALTLLGRARKDERSSH',\n",
       "  'mw': 36362.0},\n",
       " 'b3610': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AC62',\n",
       "  'gene': 'b3610',\n",
       "  'locus': ['b3610', 'JW3585'],\n",
       "  'synonyms': 'yibM',\n",
       "  'properties': {'Catalytic Activity': [],\n",
       "   'Function': 'The disulfide bond functions as an electron carrier in the glutathione-dependent synthesis of deoxyribonucleotides by the enzyme ribonucleotide reductase. In addition, it is also involved in reducing some disulfide bonds in a coupled system with glutathione reductase'},\n",
       "  'seq': 'MANVEIYTKETCPYCHRAKALLSSKGVSFQELPIDGNAAKREEMIKRSGRTTVPQIFIDAQHIGGCDDLYALDARGGLDPLLK',\n",
       "  'mw': 9137.0},\n",
       " 'b3612': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P37689',\n",
       "  'gene': 'b3612',\n",
       "  'locus': ['b3612', 'JW3587'],\n",
       "  'synonyms': 'pgmI yibO',\n",
       "  'properties': {'Catalytic Activity': [('(2R)-2-phosphoglycerate = (2R)-3-phosphoglycerate',\n",
       "     '5.4.2.12')],\n",
       "   'Function': 'Catalyzes the interconversion of 2-phosphoglycerate (2-PGA) and 3-phosphoglycerate (3-PGA)',\n",
       "   'Regulation Activity': 'Insensitive to vanadate',\n",
       "   'Pathway': 'Carbohydrate degradation; glycolysis; pyruvate from D-glyceraldehyde 3-phosphate: step 3/5'},\n",
       "  'seq': 'MLVSKKPMVLVILDGYGYREEQQDNAIFSAKTPVMDALWANRPHTLIDASGLEVGLPDRQMGNSEVGHVNLGAGRIVYQDLTRLDVEIKDRAFFANPVLTGAVDKAKNAGKAVHIMGLLSAGGVHSHEDHIMAMVELAAERGAEKIYLHAFLDGRDTPPRSAESSLKKFEEKFAALGKGRVASIIGRYYAMDRDNRWDRVEKAYDLLTLAQGEFQADTAVAGLQAAYARDENDEFVKATVIRAEGQPDAAMEDGDALIFMNFRADRAREITRAFVNADFDGFARKKVVNVDFVMLTEYAADIKTAVAYPPASLVNTFGEWMAKNDKTQLRISETEKYAHVTFFFNGGVEESFKGEDRILINSPKVATYDLQPEMSSAELTEKLVAAIKSGKYDTIICNYPNGDMVGHTGVMEAAVKAVEALDHCVEEVAKAVESVGGQLLITADHGNAEQMRDPATGQAHTAHTNLPVPLIYVGDKNVKAVEGGKLSDIAPTMLSLMGMEIPQEMTGKPLFIVE',\n",
       "  'mw': 56194.0},\n",
       " 'b3616': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P07913',\n",
       "  'gene': 'b3616',\n",
       "  'locus': ['b3616', 'JW3591'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('L-threonine + NAD(+) = (2S)-2-amino-3-oxobutanoate + H(+) + NADH',\n",
       "     '1.1.1.103')],\n",
       "   'Function': 'Catalyzes the NAD(+)-dependent oxidation of L-threonine to 2-amino-3-ketobutyrate. To a lesser extent, also catalyzes the oxidation of D-allo-threonine and L-threonine amide, but not that of D-threonine and L-allothreonine. Cannot utilize NADP(+) instead of NAD(+)',\n",
       "   'Regulation Activity': 'Is totally inhibited by EDTA in vitro',\n",
       "   'Pathway': 'Amino-acid degradation; L-threonine degradation via oxydo-reductase pathway; glycine from L-threonine: step 1/2'},\n",
       "  'seq': 'MKALSKLKAEEGIWMTDVPVPELGHNDLLIKIRKTAICGTDVHIYNWDEWSQKTIPVPMVVGHEYVGEVVGIGQEVKGFKIGDRVSGEGHITCGHCRNCRGGRTHLCRNTIGVGVNRPGCFAEYLVIPAFNAFKIPDNISDDLAAIFDPFGNAVHTALSFDLVGEDVLVSGAGPIGIMAAAVAKHVGARNVVITDVNEYRLELARKMGITRAVNVAKENLNDVMAELGMTEGFDVGLEMSGAPPAFRTMLDTMNHGGRIAMLGIPPSDMSIDWTKVIFKGLFIKGIYGREMFETWYKMAALIQSGLDLSPIITHRFSIDDFQKGFDAMRSGQSGKVILSWD',\n",
       "  'mw': 37239.0},\n",
       " 'b3617': {'organism': 'Escherichia coli (strain K12)',\n",
       "  'protein': 'P0AB77',\n",
       "  'gene': 'b3617',\n",
       "  'locus': ['b3617', 'JW3592'],\n",
       "  'synonyms': '',\n",
       "  'properties': {'Catalytic Activity': [('acetyl-CoA + glycine = (2S)-2-amino-3-oxobutanoate + CoA',\n",
       "     '2.3.1.29')],\n",
       "   'Function': 'Catalyzes the cleavage of 2-amino-3-ketobutyrate to glycine and acetyl-CoA',\n",
       "   'Pathway': 'Amino-acid degradation; L-threonine degradation via oxydo-reductase pathway; glycine from L-threonine: step 2/2'},\n",
       "  'seq': 'MRGEFYQQLTNDLETARAEGLFKEERIITSAQQADITVADGSHVINFCANNYLGLANHPDLIAAAKAGMDSHGFGMASVRFICGTQDSHKELEQKLAAFLGMEDAILYSSCFDANGGLFETLLGAEDAIISDALNHASIIDGVRLCKAKRYRYANNDMQELEARLKEAREAGARHVLIATDGVFSMDGVIANLKGVCDLADKYDALVMVDDSHAVGFVGENGRGSHEYCDVMGRVDIITGTLGKALGGASGGYTAARKEVVEWLRQRSRPYLFSNSLAPAIVAASIKVLEMVEAGSELRDRLWANARQFREQMSAAGFTLAGADHAIIPVMLGDAVVAQKFARELQKEGIYVTGFFYPVVPKGQARIRTQMSAAHTPEQITRAVEAFTRIGKQLGVIA',\n",
       "  'mw': 43117.0},\n",
       " ...}"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data_all"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "4210d055",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
